
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            NAT10-mediated ac4C modification of Lipin1 mRNA contributes to the pathogenesis of PWMI - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="FF2082FE89B753831A82FE0024112021.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="jciinsight">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333955/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="JCI Insight">
<meta name="citation_title" content="NAT10-mediated ac4C modification of Lipin1 mRNA contributes to the pathogenesis of PWMI">
<meta name="citation_author" content="Xinyu Li">
<meta name="citation_author_institution" content="Department of Cell Biology and Neurobiology, Xuzhou Key Laboratory of Neurobiology, Xuzhou Medical University, Xuzhou, Jiangsu Province, China.">
<meta name="citation_author_institution" content="Department of Neurology, and">
<meta name="citation_author" content="Meng Zhang">
<meta name="citation_author_institution" content="Department of Urology, Affiliated Hospital of Xuzhou Medical University, Xuzhou, Jiangsu Province, China.">
<meta name="citation_author" content="Yanan Liu">
<meta name="citation_author_institution" content="Department of Cell Biology and Neurobiology, Xuzhou Key Laboratory of Neurobiology, Xuzhou Medical University, Xuzhou, Jiangsu Province, China.">
<meta name="citation_author" content="Chunjie Guo">
<meta name="citation_author_institution" content="Department of Cell Biology and Neurobiology, Xuzhou Key Laboratory of Neurobiology, Xuzhou Medical University, Xuzhou, Jiangsu Province, China.">
<meta name="citation_author" content="Yiwei Liu">
<meta name="citation_author_institution" content="Department of Cell Biology and Neurobiology, Xuzhou Key Laboratory of Neurobiology, Xuzhou Medical University, Xuzhou, Jiangsu Province, China.">
<meta name="citation_author" content="Lei Han">
<meta name="citation_author_institution" content="Department of Cell Biology and Neurobiology, Xuzhou Key Laboratory of Neurobiology, Xuzhou Medical University, Xuzhou, Jiangsu Province, China.">
<meta name="citation_author" content="Zhaowei Feng">
<meta name="citation_author_institution" content="Department of Neurology, Second Affiliated Hospital of Xuzhou Medical University, Xuzhou, Jiangsu Province, China.">
<meta name="citation_author" content="Xiue Wei">
<meta name="citation_author_institution" content="Department of Neurology, Second Affiliated Hospital of Xuzhou Medical University, Xuzhou, Jiangsu Province, China.">
<meta name="citation_author" content="Ruiqin Yao">
<meta name="citation_author_institution" content="Department of Cell Biology and Neurobiology, Xuzhou Key Laboratory of Neurobiology, Xuzhou Medical University, Xuzhou, Jiangsu Province, China.">
<meta name="citation_publication_date" content="2025 Aug 8">
<meta name="citation_volume" content="10">
<meta name="citation_issue" content="15">
<meta name="citation_firstpage" content="e193712">
<meta name="citation_doi" content="10.1172/jci.insight.193712">
<meta name="citation_pmid" content="40779453">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333955/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333955/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333955/pdf/jciinsight-10-193712.pdf">
<meta name="description" content="Preterm white matter injury (PWMI) is a leading cause of cerebral palsy and chronic neurological disabilities in premature infants. It is characterized by defects in oligodendrocyte precursor cell (OPC) differentiation and dysmyelination. Currently, ...">
<meta name="og:title" content="NAT10-mediated ac4C modification of Lipin1 mRNA contributes to the pathogenesis of PWMI">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Preterm white matter injury (PWMI) is a leading cause of cerebral palsy and chronic neurological disabilities in premature infants. It is characterized by defects in oligodendrocyte precursor cell (OPC) differentiation and dysmyelination. Currently, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333955/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12333955">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1172/jci.insight.193712"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/jciinsight-10-193712.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12333955%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12333955/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12333955/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333955/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-jciinsight.gif" alt="JCI Insight logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to JCI Insight" title="Link to JCI Insight" shape="default" href="https://insight.jci.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">JCI Insight</button></div>. 2025 Aug 8;10(15):e193712. doi: <a href="https://doi.org/10.1172/jci.insight.193712" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1172/jci.insight.193712</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22JCI%20Insight%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22JCI%20Insight%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22JCI%20Insight%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22JCI%20Insight%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>NAT10-mediated ac4C modification of Lipin1 mRNA contributes to the pathogenesis of PWMI</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Xinyu Li</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xinyu Li</span></h3>
<div class="p">
<sup>1</sup>Department of Cell Biology and Neurobiology, Xuzhou Key Laboratory of Neurobiology, Xuzhou Medical University, Xuzhou, Jiangsu Province, China.</div>
<div class="p">
<sup>2</sup>Department of Neurology, and</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xinyu Li</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Meng Zhang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Meng Zhang</span></h3>
<div class="p">
<sup>3</sup>Department of Urology, Affiliated Hospital of Xuzhou Medical University, Xuzhou, Jiangsu Province, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Meng Zhang</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yanan Liu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yanan Liu</span></h3>
<div class="p">
<sup>1</sup>Department of Cell Biology and Neurobiology, Xuzhou Key Laboratory of Neurobiology, Xuzhou Medical University, Xuzhou, Jiangsu Province, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yanan Liu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Chunjie Guo</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Chunjie Guo</span></h3>
<div class="p">
<sup>1</sup>Department of Cell Biology and Neurobiology, Xuzhou Key Laboratory of Neurobiology, Xuzhou Medical University, Xuzhou, Jiangsu Province, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chunjie Guo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yiwei Liu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yiwei Liu</span></h3>
<div class="p">
<sup>1</sup>Department of Cell Biology and Neurobiology, Xuzhou Key Laboratory of Neurobiology, Xuzhou Medical University, Xuzhou, Jiangsu Province, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yiwei Liu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Lei Han</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Lei Han</span></h3>
<div class="p">
<sup>1</sup>Department of Cell Biology and Neurobiology, Xuzhou Key Laboratory of Neurobiology, Xuzhou Medical University, Xuzhou, Jiangsu Province, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lei Han</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feng%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Zhaowei Feng</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Zhaowei Feng</span></h3>
<div class="p">
<sup>4</sup>Department of Neurology, Second Affiliated Hospital of Xuzhou Medical University, Xuzhou, Jiangsu Province, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Feng%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhaowei Feng</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Xiue Wei</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Xiue Wei</span></h3>
<div class="p">
<sup>4</sup>Department of Neurology, Second Affiliated Hospital of Xuzhou Medical University, Xuzhou, Jiangsu Province, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiue Wei</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yao%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Ruiqin Yao</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Ruiqin Yao</span></h3>
<div class="p">
<sup>1</sup>Department of Cell Biology and Neurobiology, Xuzhou Key Laboratory of Neurobiology, Xuzhou Medical University, Xuzhou, Jiangsu Province, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yao%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ruiqin Yao</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="A1">
<sup>1</sup>Department of Cell Biology and Neurobiology, Xuzhou Key Laboratory of Neurobiology, Xuzhou Medical University, Xuzhou, Jiangsu Province, China.</div>
<div id="A2">
<sup>2</sup>Department of Neurology, and</div>
<div id="A3">
<sup>3</sup>Department of Urology, Affiliated Hospital of Xuzhou Medical University, Xuzhou, Jiangsu Province, China.</div>
<div id="A4">
<sup>4</sup>Department of Neurology, Second Affiliated Hospital of Xuzhou Medical University, Xuzhou, Jiangsu Province, China.</div>
<div class="author-notes p">
<div class="fn" id="corresp1">
<sup>✉</sup><p class="display-inline">Address correspondence to: Ruiqin Yao, Department of Cell Biology and Neurobiology, Xuzhou Key Laboratory of Neurobiology, Xuzhou Medical University, Xuzhou, Jiangsu Province, 221004, China. Phone: 15862141412; Email: <span>wenxi_yao@163.com</span>. Or to: Xiue Wei, Department of Neurology, Second Affiliated Hospital of Xuzhou Medical University, Xuzhou, Jiangsu Province, 221006, China. Phone: 15005206672; Email: <span>wxeqq@163.com</span>.</p>
</div>
<div class="fn" id="fn1">
<sup>#</sup><p class="display-inline"><strong>Authorship note:</strong> XL, MZ, and Yanan Liu contributed equally to this work.</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 21; Accepted 2025 Jun 11; Collection date 2025 Aug 8.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Li et al.</div>
<p>This work is licensed under the Creative Commons Attribution 4.0 International License. To view a copy of this license, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12333955  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40779453/" class="usa-link">40779453</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Preterm white matter injury (PWMI) is a leading cause of cerebral palsy and chronic neurological disabilities in premature infants. It is characterized by defects in oligodendrocyte precursor cell (OPC) differentiation and dysmyelination. Currently, there are no effective therapeutic strategies available in clinical practice. Lipid homeostasis plays a crucial role in myelin development, yet the function of Lipin1 — a key phosphatidic acid phosphatase involved in phospholipid synthesis — remains unclear. In this study, we identified a significant downregulation of Lipin1 in OPCs from PWMI mice, which impaired OPC differentiation and myelin formation. Conversely, Lipin1 overexpression in these mice promoted OPC maturation and enhanced myelin development. We found evidence that <em>N</em>-acetyltransferase 10 (NAT10) acts as a regulator of Lipin1 expression through RNA pull-down and mass spectrometry. NAT10-mediated <em>N</em><sup>4</sup>-acetylcytidine (ac4C) modification enhanced Lipin1 mRNA stability and translation, and NAT10 knockdown in OPCs impaired myelination, highlighting its crucial role in Lipin1-mediated myelination. Our study revealed that the downregulation of Lipin1 impaired OPC differentiation and myelination in PWMI, with NAT10-mediated ac4C modification playing a critical role in regulating Lipin1 expression. These findings highlight Lipin1 and NAT10 as promising therapeutic targets for treating myelination defects in PWMI, warranting further investigation into their potential in preterm birth–related neurological disorders.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> Cell biology, Neuroscience</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Keywords:</strong> Behavior, Epigenetics, Neurodevelopment</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333955_jciinsight-10-193712-g155.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7374/12333955/c779f55ceff3/jciinsight-10-193712-g155.jpg" loading="lazy" height="638" width="747" alt="graphic file with name jciinsight-10-193712-g155.jpg"></a></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p>NAT10-mediated ac4C modification regulates Lipin1 expression, contributing to the pathogenesis of PWMI and offering potential therapeutic targets for preterm birth-related neurological disorders.</p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Preterm white matter injury (PWMI) is a leading cause of lifelong neurological impairments in preterm infants, including motor disabilities and cognitive deficits, with severe cases resulting in cerebral palsy (CP) (<a href="#B1" class="usa-link" aria-describedby="B1">1</a>–<a href="#B3" class="usa-link" aria-describedby="B3">3</a>). PWMI is primarily associated with perinatal hypoxia-ischemia (HI) and its severity increases with decreasing gestational age, which causes damage to the developing brain (<a href="#B4" class="usa-link" aria-describedby="B4">4</a>, <a href="#B5" class="usa-link" aria-describedby="B5">5</a>). During HI conditions, oligodendrocyte progenitor cells (OPCs), which are essential for myelination, are particularly vulnerable, with limited antioxidant defenses and high mitochondrial oxygen consumption being the main factors contributing to their susceptibility (<a href="#B6" class="usa-link" aria-describedby="B6">6</a>–<a href="#B8" class="usa-link" aria-describedby="B8">8</a>). Although OPCs may undergo compensatory proliferation, their failure to differentiate leads to impaired axonal myelination, particularly in the corpus callosum (<a href="#B9" class="usa-link" aria-describedby="B9">9</a>–<a href="#B11" class="usa-link" aria-describedby="B11">11</a>). Despite advances in neonatal care, effective treatments for PWMI remain limited, making it crucial to investigate the key factors hindering OPC differentiation in PWMI for the prevention and treatment of this condition.</p>
<p>Emerging evidence suggests that lipid metabolism plays a vital role not only in the structural assembly of myelin membranes but also in regulating the lineage progression of OPCs (<a href="#B12" class="usa-link" aria-describedby="B12">12</a>–<a href="#B14" class="usa-link" aria-describedby="B14">14</a>). Among lipid metabolic enzymes, Lipin1, a phosphatidic acid phosphatase (PAP), catalyzes the conversion of phosphatidic acid (PA) to diacylglycerol (DG), an essential precursor for lipid biosynthesis and a second messenger in signaling cascades (<a href="#B15" class="usa-link" aria-describedby="B15">15</a>, <a href="#B16" class="usa-link" aria-describedby="B16">16</a>). DG activates protein kinase C (PKC), which promotes OPC maturation and myelin production (<a href="#B17" class="usa-link" aria-describedby="B17">17</a>). In peripheral nervous system models, Lipin1 deficiency has been associated with demyelination due to PA accumulation (<a href="#B18" class="usa-link" aria-describedby="B18">18</a>). However, the role of Lipin1 in central nervous system myelination, particularly in the context of PWMI, remains poorly understood.</p>
<p>To address this gap, we investigated whether Lipin1 downregulation occurs in the brains of PWMI mice and assessed its functional consequences. We observed that Lipin1 expression is significantly downregulated following HI injury. Gain- and loss-of-function studies revealed that Lipin1 is a crucial driver of OPC differentiation and myelination under both physiological and HI conditions. To further explore the upstream regulatory mechanism of Lipin1 expression, we performed RNA pull-down combined with mass spectrometry analysis, leading to the identification of <em>N</em>-acetyltransferase 10 (NAT10) as a potential regulator. NAT10 is the only known enzyme mediating the <em>N</em><sup>4</sup>-acetylcytidine (ac4C) modification, a rare RNA acetylation mark that enhances mRNA stability and translation in eukaryotic cells (<a href="#B19" class="usa-link" aria-describedby="B19">19</a>). Recent studies have shown that ac4C plays a crucial role in regulating gene expression during development and stress responses (<a href="#B20" class="usa-link" aria-describedby="B20">20</a>, <a href="#B21" class="usa-link" aria-describedby="B21">21</a>). In our study, we demonstrated that NAT10-mediated ac4C modification enhances Lipin1 mRNA stability and translation. NAT10 knockdown significantly impaired OPC differentiation and myelination, highlighting what we believe to be a previously unrecognized epitranscriptomic mechanism in the pathogenesis of PWMI. Taken together, our study revealed a regulatory axis involving NAT10, ac4C, and Lipin1 that is critical for OPC maturation and myelin formation. These findings not only expand our understanding of the molecular basis of PWMI but also point to potential therapeutic targets for the prevention and treatment of PWMI.</p></section><section id="sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="sec3"><h3 class="pmc_sec_title">Myelination deficits and behavioral abnormalities in PWMI mice.</h3>
<p>To investigate myelination after HI injury, we assessed myelin development at multiple time points using immunofluorescence (IF), Western blotting (WB), and transmission electron microscopy (TEM) (<a href="#F1" class="usa-link">Figure 1A</a>). IF analysis showed a significant decrease in myelin basic protein (MBP) fluorescence intensity in the ischemic corpus callosum [HI (R)] of PWMI mice compared with the contralateral side [HI (L)] and the Sham group (<a href="#F1" class="usa-link">Figure 1, B and C</a>). Consistently, WB confirmed decreased MBP levels in HI (R) from 14 days after HI was induced (P3+14) to P3+28 (<a href="#F1" class="usa-link">Figure 1, D and E</a>). TEM analysis revealed thinner myelin sheaths, an increased G-ratio, and a reduced density of myelinated axons in PWMI mice compared with Sham mice (<a href="#F1" class="usa-link">Figure 1, F–I</a>). These results demonstrate that HI injury leads to significant myelination deficits in the corpus callosum. Platelet-derived growth factor receptor α (PDGFR-α) was used as a marker to monitor OPC differentiation. In the Sham group, PDGFR-α levels decreased over time as OPCs differentiated into mature oligodendrocytes. In contrast, PWMI mice exhibited a distinct pattern; PDGFR-α was significantly reduced at P3+7 but remained abnormally elevated at P3+14, P3+21, and P3+28 in HI (R) compared with both Sham and HI (L) sides (<a href="#F1" class="usa-link">Figure 1, J and K</a>). Moreover, WB analysis showed that PDGFR-α expression decreased during the acute phase (P3+1 to P3+7) but paradoxically increased during the subacute phase (P3+7 to P3+21) (<a href="#F1" class="usa-link">Figure 1, L and M</a>). These findings suggest that HI injury not only delays OPC differentiation but also leads to dysregulation of the OPC pool over time.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Figure 1. Myelination deficits in PWMI model mice.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333955_jciinsight-10-193712-g144.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7374/12333955/b0124a587d61/jciinsight-10-193712-g144.jpg" loading="lazy" height="1078" width="765" alt="Figure 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Experimental diagram. (<strong>B</strong> and <strong>C</strong>) The MBP levels in corpus callosum were detected by immunofluorescent staining at 14, 21, and 28 days after modeling and statistical analysis. (<strong>D</strong> and <strong>E</strong>) The changes in MBP content in the corpus callosum region of mice at 3, 7, 14, 21, and 28 days after modeling were detected by Western blot analysis and statistical analysis. Representative of 3 independent experiments. (<strong>F</strong>) TEM images of the myelin sheath of the corpus callosum (cc) of 2 groups of mice. The high-magnification images are zoomed-in views of the red boxes in the low-magnification images. Scale bars: 1 μm (left) and 200 nm (right). (<strong>G</strong> and <strong>H</strong>) G-ratio distribution chart and statistical chart. (<strong>I</strong>) Percentage diagram of the number of myelinated axons. The number of evaluated myelinated axons was <em>n</em> = 50/mouse. (<strong>J</strong> and <strong>K</strong>) The level of PDGFR-α and the number of PDGFR-α<sup>+</sup> cells in the corpus callosum of PWMI mice at different time points were detected by immunofluorescent staining. (<strong>L</strong> and <strong>M</strong>) Western blot analysis of PDGFR-α levels in corpus callosum of PWMI mice at different time points and statistical analysis. Representative of 3 independent experiments. Statistics: 1-way ANOVA with LSD post hoc analysis (<strong>C</strong>, <strong>E</strong>, and <strong>K</strong>), 2-tailed Student’s <em>t</em> test (<strong>H</strong> and <strong>I</strong>), and 1-way ANOVA with Dunnett’s test (<strong>M</strong>). HI (R), ischemic side; HI (L), ischemic side. Scale bar: 100 μm. <em>n</em> = 3. The results are expressed as mean ± SEM.</p></figcaption></figure><p>To evaluate the functional consequences of myelination deficits, behavioral tests were conducted, including the Morris water maze (MWM), Y-maze, open field, rotarod, and gait analysis. PWMI mice exhibited reduced body weight (<a href="#sd" class="usa-link">Supplemental Figure 1A</a>; supplemental material available online with this article; <a href="https://doi.org/10.1172/jci.insight.193712DS1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1172/jci.insight.193712DS1</a>). In the MWM, PWMI mice had a longer latency to find the platform (<a href="#sd" class="usa-link">Supplemental Figure 1</a>, B and C), crossed the original platform location fewer times, and spent less time in the target quadrant after platform removal compared with Sham mice (<a href="#sd" class="usa-link">Supplemental Figure 1</a>, D and E). In the Y-maze, they entered the new arm fewer times and spent less time there (<a href="#sd" class="usa-link">Supplemental Figure 1</a>, F and G). The open field test showed reduced total distance and average speed in PWMI mice (<a href="#sd" class="usa-link">Supplemental Figure 1</a>, H–J). In the rotarod, PWMI mice ran for less time compared with Sham (<a href="#sd" class="usa-link">Supplemental Figure 1K</a>). Gait analysis revealed longer stance, swing, and propulsion durations for both forepaws and hindpaws in PWMI mice (<a href="#sd" class="usa-link">Supplemental Figure 1</a>, L–N). These findings indicate deficits in spatial memory and motor function after early postnatal HI.</p></section><section id="sec4"><h3 class="pmc_sec_title">Lipin1 expression is reduced in OPCs of PWMI mice.</h3>
<p>Lipin1, a PAP, is critical for maintaining lipid homeostasis, an essential process for myelination (<a href="#B22" class="usa-link" aria-describedby="B22">22</a>). WB analysis showed that in Sham mice, Lipin1 levels in the corpus callosum decreased from early postnatal stages, stabilizing by P3+14 (<a href="#F2" class="usa-link">Figure 2, A–C</a>). In contrast, PWMI mice exhibited significantly reduced Lipin1 expression at P3+3, P3+14, and P3+21 (<a href="#F2" class="usa-link">Figure 2, B and C</a>). As an internal control for antibody specificity, a negative control (secondary antibody only, no primary antibody) was included in the WB assay, which showed no visible bands, confirming the specificity of Lipin1 detection (<a href="#F2" class="usa-link">Figure 2B</a>). Additionally, PA levels were elevated and DG levels were decreased in PWMI mouse brains (<a href="#F2" class="usa-link">Figure 2D</a>), suggesting disrupted lipid metabolism. IF staining revealed a marked reduction in Lipin1<sup>+</sup>/Olig2<sup>+</sup> double-positive cells in the corpus callosum of PWMI mice (<a href="#F2" class="usa-link">Figure 2, E and F</a>). Lipin1 expression at P3+3 and P3+14 was notably lower in PWMI mice (<a href="#F2" class="usa-link">Figure 2, G and H</a>). Consistently, in primary OPCs subjected to oxygen-glucose deprivation (OGD), Lipin1 levels were significantly reduced at multiple time points after reoxygenation (<a href="#F2" class="usa-link">Figure 2, I and J</a>). These results highlight that HI injury leads to a significant decrease in Lipin1 expression in the corpus callosum and OPCs, implicating its role in impaired myelination.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Figure 2. Reduced expression of Lipin1 in the OPCs of PWMI mice.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333955_jciinsight-10-193712-g145.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7374/12333955/212410fed857/jciinsight-10-193712-g145.jpg" loading="lazy" height="1056" width="748" alt="Figure 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Experimental diagram. (<strong>B</strong> and <strong>C</strong>) Western blot analysis was conducted to detect changes in Lipin1 levels in the corpus callosum of mice from both groups at different time points. NC, negative control (sample incubated with secondary antibody only, no primary antibody), confirming the specificity of the Lipin1 signal. Statistical analysis was performed (<em>n</em> = 3). Representative of 3 independent experiments. (<strong>D</strong>) PA and DG levels in the brain tissue of mice from 2 groups (<em>n</em> = 6). (<strong>E</strong> and <strong>F</strong>) Lipin1 levels (green) in oligodendrocytes (Olig2, red) at different time points were detected by immunofluorescent staining, followed by statistical analysis (<em>n</em> = 3). Scale bars: 100 μm (low magnification) and 25 μm (high magnification). (<strong>G</strong> and <strong>H</strong>) mRNA levels of Lipin1 in the P3+3 and P3+14 corpus callosum of 2 groups of mice (<em>n</em> = 5). (<strong>I</strong> and <strong>J</strong>) Western blot analysis of Lipin1 levels in primary OPCs from control and OGD groups of mice, followed by statistical analysis (<em>n</em> = 3). Representative of 3 independent experiments. Statistics: 2-tailed Student’s <em>t</em> test (<strong>C</strong>, <strong>D</strong>, and <strong>F</strong>–<strong>H</strong>) and 1-way ANOVA with LSD post hoc analysis (<strong>J</strong>). The results are expressed as mean ± SEM. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001.</p></figcaption></figure></section><section id="sec5"><h3 class="pmc_sec_title">Lipin1 regulates OPC differentiation, myelination, and behavioral changes in mice.</h3>
<p>To investigate the effect of Lipin1 on myelination, Lipin1-knockdown lentivirus was injected into the corpus callosum (<a href="#F3" class="usa-link">Figure 3A</a>). IF imaging confirmed successful transfection (<a href="#F3" class="usa-link">Figure 3B</a>), and WB analysis validated effective Lipin1 knockdown (<a href="#F3" class="usa-link">Figure 3, C and D</a>). Knockdown of Lipin1 increased PA levels and decreased DG levels (<a href="#F3" class="usa-link">Figure 3E</a>). Differentiation of OPCs was impaired, as evidenced by significantly reduced MBP and CC1 (known as adenomatous polyposis coli protein) levels and increased PDGFR-α levels in the shLipin1 group (<a href="#F3" class="usa-link">Figure 3, C and D</a>). IF analysis further confirmed reduced MBP levels in the corpus callosum of the shLipin1 group (<a href="#F3" class="usa-link">Figure 3, F–H</a>) and a lower CC1<sup>+</sup>/PDGFR-α<sup>+</sup> cell ratio (<a href="#F3" class="usa-link">Figure 3, I and J</a>). TEM analysis showed thinner myelin (<a href="#F3" class="usa-link">Figure 3K</a>) and a higher G-ratio in the shLipin1 group (<a href="#F3" class="usa-link">Figure 3, L and M</a>), along with fewer myelinated axons (<a href="#F3" class="usa-link">Figure 3N</a>). These results indicate that downregulation of Lipin1 in the corpus callosum impairs OPC differentiation and myelination.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Figure 3. Reduced Lipin1 expression in the corpus callosum impairs OPC differentiation and myelination.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333955_jciinsight-10-193712-g146.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7374/12333955/40b2a6fc8e96/jciinsight-10-193712-g146.jpg" loading="lazy" height="1034" width="747" alt="Figure 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Experimental diagram. (<strong>B</strong>) Stereotaxic injection coordinates and infection area map of corpus callosum. Scale bar: 100 μm. (<strong>C</strong> and <strong>D</strong>) Western blot and statistical analysis of Lipin1, MBP, CC1, and PDGFR-α levels in the corpus callosum of C57BL/6J mice 7 days after stereotaxic injection of lentivirus (<em>n</em> = 3). Representative of 3 independent experiments. (<strong>E</strong>) PA and DG levels in the brain tissue of mice from both groups (<em>n</em> = 6). (<strong>F</strong>–<strong>H</strong>) MBP levels in the corpus callosum (cc) and striatum of C57BL/6J mice 7 days after stereotactic lentivirus injection were detected by immunofluorescent staining and statistically analyzed. Scale bar: 100 μm. (<strong>I</strong> and <strong>J</strong>) Statistical analysis of the CC1<sup>+</sup>/PDGFR-α<sup>+</sup> (red/green) cell ratio and positive cells in the corpus callosum of C57BL/6J mice 7 days after stereotactic lentivirus injection was performed using immunofluorescent staining. Scale bars: 100 μm (low magnification) and 25 μm (high magnification). <em>n</em> = 3. (<strong>K</strong>) Two sets of TEM images of the myelin sheath in the corpus callosum. The high-magnification images are zoomed-in views of the red boxes in the low-magnification images. Scale bars: 1 μm (low magnification) and 200 nm (high magnification). (<strong>L</strong> and <strong>M</strong>) G-ratio distribution chart and statistical chart. (<strong>N</strong>) Percentage plot showing the number of myelinated axons. The number of myelinated axons evaluated was <em>n</em> = 50 per mouse, with 3 mice per group. Statistics: 2-tailed Student’s <em>t</em> test (<strong>D</strong>, <strong>E</strong>, <strong>G</strong>, <strong>H</strong>, <strong>J</strong>, <strong>M</strong>, and <strong>N</strong>). The results are expressed as mean ± SEM. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01.</p></figcaption></figure><p>Behavioral tests demonstrated that Lipin1 downregulation caused impaired motor and cognitive functions. The shLipin1 group showed reduced activity and speed in the open field test (<a href="#sd" class="usa-link">Supplemental Figure 2</a>, A–C), longer stance and swing durations in gait analysis (<a href="#sd" class="usa-link">Supplemental Figure 2</a>, D–F), and shorter running times on the rotarod (<a href="#sd" class="usa-link">Supplemental Figure 2G</a>). Spatial memory was also compromised, with longer latency and fewer platform crossings in the MWM (<a href="#sd" class="usa-link">Supplemental Figure 2</a>, H–K), and fewer entries and shorter stay durations in the novel arm of the Y-maze (<a href="#sd" class="usa-link">Supplemental Figure 2</a>, L and M). Previous studies have shown that structural damage to the corpus callosum leads to deficits in locomotion, spatial learning, and memory (<a href="#B23" class="usa-link" aria-describedby="B23">23</a>), supporting the functional relevance of our findings.</p>
<p>To confirm the functional role of Lipin1, we overexpressed Lipin1 in the corpus callosum of PWMI mice (<a href="#F4" class="usa-link">Figure 4A</a>). WB and IF analyses showed restoration of CC1 and MBP levels and normalization of PDGFR-α levels (<a href="#F4" class="usa-link">Figure 4, B–F and H–J</a>). Additionally, chemical fluorescence analysis revealed correction of PA/DG metabolic imbalance (<a href="#F4" class="usa-link">Figure 4G</a>). TEM confirmed thicker myelin and a lower G-ratio in Lipin1-overexpressing PWMI mice (<a href="#sd" class="usa-link">Supplemental Figure 3</a>, A and B). Behaviorally, Lipin1 overexpression improved motor performance (open field, gait analysis, rotarod) and enhanced spatial memory (MWM, Y-maze) (<a href="#sd" class="usa-link">Supplemental Figure 3</a>, C–O). These results indicate that restoring Lipin1 expression alleviates both myelination deficits and behavioral impairments in PWMI mice.</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Figure 4. Overexpression of Lipin1 in the corpus callosum promotes OPC differentiation and myelination in PWMI mice.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333955_jciinsight-10-193712-g147.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7374/12333955/a9f642163ec5/jciinsight-10-193712-g147.jpg" loading="lazy" height="1040" width="728" alt="Figure 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Experimental diagram. (<strong>B</strong> and <strong>C</strong>) Western blot analysis of Lipin1, CC1, and PDGFR-α levels in the corpus callosum of mice 7 days after stereotaxic injection of lentivirus, with statistical analysis (<em>n</em> = 3). Representative of 3 independent experiments. (<strong>D</strong>) PA and DG levels in the brain tissue of mice from both groups (<em>n</em> = 6). (<strong>E</strong>–<strong>G</strong>) Immunofluorescent staining showing MBP levels in the corpus callosum and striatum of the 3 groups of mice, with statistical analysis (<em>n</em> = 3). Scale bar: 100 μm. (<strong>H</strong>–<strong>J</strong>) Immunofluorescence analysis of the number of CC1<sup>+</sup> (red) and PDGFR-α<sup>+</sup> (green) cells in the corpus callosum 7 days after lentiviral injection, and the ratio of positive cells, with statistical analysis. The high-magnification images are zoomed-in views of the white dashed boxes in the low-magnification images. Scale bars: 100 μm (low magnification) and 25 μm (high magnification). <em>n</em> = 3. Statistics: 1-way ANOVA with LSD post hoc analysis (<strong>C</strong>, <strong>E</strong>–<strong>G</strong>, <strong>I</strong>, and <strong>J</strong>). The results are expressed as mean ± SEM. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001. cc, corpus callosum.</p></figcaption></figure></section><section id="sec6"><h3 class="pmc_sec_title">Reduced NAT10 expression in OPCs of PWMI mice.</h3>
<p>To investigate the mechanisms regulating Lipin1 expression in PWMI mice, we performed RNA pull-down assays in primary cultured OPCs to identify proteins interacting with Lipin1 mRNA (<a href="#F5" class="usa-link">Figure 5A</a>). Gene Ontology (GO) enrichment analysis revealed that NAT10, an acetyltransferase involved in mRNA ac4C modification, was enriched in the “post-transcriptional regulation of gene expression” pathway (<a href="#F5" class="usa-link">Figure 5B</a> and <a href="#sd" class="usa-link">Supplemental Table 1</a>). Mass spectrometry analysis further confirmed the presence of NAT10 in the pull-down complexes (<a href="#F5" class="usa-link">Figure 5C</a>). We further predicted potential acetylation sites on Lipin1 mRNA using the PACES tool (<a href="http://rnanut.net/paces/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://rnanut.net/paces/</a>), which identified a highly conserved acetylation site (<a href="#F5" class="usa-link">Figure 5D</a>).</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Figure 5. Reduced NAT10 expression in OPCs of PWMI mice.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333955_jciinsight-10-193712-g148.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7374/12333955/b9d2ea2f32c9/jciinsight-10-193712-g148.jpg" loading="lazy" height="884" width="743" alt="Figure 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Mass spectrometry results analysis and screening of Lipin1 posttranslational regulatory proteins. (<strong>B</strong>) GO enrichment analysis of Lipin1 mRNA binding protein pool. (<strong>C</strong>) Secondary mass spectrum of NAT10. (<strong>D</strong>) Prediction of potential ac4C sites on Lipin1 mRNA by PACES. (<strong>E</strong> and <strong>F</strong>) Immunofluorescent staining to detect and analyze the number of PDGFR-α<sup>+</sup>/NAT10<sup>+</sup> (green/red) double-positive cells in the corpus callosum (cc) of mice. Scale bar: 50 μm. (<strong>G</strong> and <strong>H</strong>) Western blot analysis of NAT10 levels in the corpus callosum of mice and statistical analysis. NC, negative control (sample incubated with secondary antibody only, no primary antibody), confirming the specificity of the NAT10 signal. Representative of 3 independent experiments. (<strong>I</strong> and <strong>J</strong>) Dot blot analysis of RNA ac4C modification levels in the corpus callosum region and statistical analysis. Representative of 3 independent experiments. (<strong>K</strong> and <strong>L</strong>) Western blot analysis of NAT10 levels in primary cultured mouse OPCs after OGD and statistical analysis. <em>n</em> = 3. Statistics: 1-way ANOVA with LSD post hoc analysis (<strong>F</strong>, <strong>H</strong>, and <strong>J</strong>) and 2-tailed Student’s <em>t</em> test (<strong>L</strong>). The results are expressed as mean ± SEM. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01.</p></figcaption></figure><p>To assess the levels of NAT10 protein and RNA ac4C modification in the corpus callosum of PWMI mice, tissues were collected 14 days after modeling for WB, IF, and dot blot experiments. IF results showed a significant reduction in the number of PDGFR-α<sup>+</sup>/NAT10<sup>+</sup> double-positive cells in the corpus callosum of the HI (R) side compared with the Sham group and the HI (L) side (<a href="#F5" class="usa-link">Figure 5, E and F</a>). Consistently, WB analysis demonstrated markedly decreased NAT10 protein levels in the HI (R) corpus callosum (<a href="#F5" class="usa-link">Figure 5, G and H</a>). To confirm antibody specificity, a negative control in WB (secondary antibody only, no primary antibody) was performed, which showed no visible bands, indicating the specificity of the NAT10 signal (<a href="#F5" class="usa-link">Figure 5G</a>). Dot blot analysis revealed a significant reduction in RNA ac4C modification levels in the corpus callosum of the HI (R) side compared with the Sham group and HI (L) side (<a href="#F5" class="usa-link">Figure 5, I and J</a>). Similarly, primary OPC cultures demonstrated decreased NAT10 levels after 1 hour of OGD and 6 hours of reoxygenation (<a href="#F5" class="usa-link">Figure 5, K and L</a>). These findings indicate that NAT10 and RNA ac4C modification are significantly reduced in PWMI, suggesting their involvement in the pathology.</p></section><section id="sec7"><h3 class="pmc_sec_title">NAT10 regulates ac4C acetylation of Lipin1 mRNA in OPCs.</h3>
<p>Given the acetyltransferase activity of NAT10, we next examined whether NAT10 regulates Lipin1 expression via mRNA acetylation. Based on PACES predictions, we identified a conserved acetylation site on Lipin1 mRNA and constructed a mutated Lipin1 plasmid (Mut) by replacing TCCCT with AGGGA. RNA pull-down assays showed that NAT10 directly binds to this acetylation site but the interaction was abolished upon mutation (<a href="#F6" class="usa-link">Figure 6A</a>), suggesting that NAT10 specifically targets this site. RNA immunoprecipitation (RIP) followed by reverse transcription quantitative real-time PCR (RT-qPCR) confirmed a significant enrichment of Lipin1 mRNA in NAT10-immunoprecipitated complexes compared with IgG controls (<a href="#F6" class="usa-link">Figure 6, B and C</a>). To further validate the functional role of NAT10, we knocked down NAT10 in OPCs using siRNA (siNAT10). Both WB and RT-qPCR results showed a marked reduction in Lipin1 expression (<a href="#F6" class="usa-link">Figure 6, D–G</a>), highlighting NAT10’s positive regulation of Lipin1. Furthermore, ac4C-RIP experiments in NAT10-knockdown OPCs showed a decrease in ac4C-modified Lipin1 mRNA levels (<a href="#F6" class="usa-link">Figure 6H</a>). Moreover, actinomycin D chase experiments demonstrated that Lipin1 mRNA stability was significantly impaired upon NAT10 knockdown (<a href="#F6" class="usa-link">Figure 6I</a>), and sucrose density gradient centrifugation revealed fewer polysome complexes and decreased polysomal Lipin1 mRNA (<a href="#F6" class="usa-link">Figure 6, J and K</a>). Together, these results suggest that NAT10 may regulate Lipin1 expression by promoting ac4C modification, thereby enhancing mRNA stability and translation efficiency.</p>
<figure class="fig xbox font-sm" id="F6"><h4 class="obj_head">Figure 6. NAT10 regulated the ac4C acetylation of Lipin1 mRNA in OPCs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333955_jciinsight-10-193712-g149.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7374/12333955/59d4ccc2fee2/jciinsight-10-193712-g149.jpg" loading="lazy" height="800" width="799" alt="Figure 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) RNA pull-down assay to verify the interaction between Lipin1 acetylation sites and NAT10 after constructing ac4C-site mutation plasmids. (<strong>B</strong> and <strong>C</strong>) In RIP experiments, Western blotting confirmed the expression of NAT10, and RT-qPCR measured the relative mRNA abundance of Lipin1. (<strong>D</strong>–<strong>F</strong>) Western blot analysis of NAT10 and Lipin1 protein levels in primary cultured OPCs and statistical analysis. Representative of 3 independent experiments. (<strong>G</strong>) RT-qPCR analysis of Lipin1 mRNA levels in primary cultured OPCs. (<strong>H</strong>) ac4C-RIP analysis of Lipin1 mRNA levels modified by ac4C in OPCs after NAT10 knockdown. (<strong>I</strong>) Cells treated with actinomycin D for 0, 2, and 4 hours to inhibit total RNA transcription, and RT-qPCR analysis of relative Lipin1 mRNA expression in mice. *siCtrl vs. siNAT10-1; <sup>#</sup>siCtrl vs. siNAT10-2. (<strong>J</strong> and <strong>K</strong>) Sucrose density gradient centrifugation assay to validate the regulation of Lipin1 translation efficiency by NAT10. The position of mRNA in the sucrose gradient reflects its translation status: mRNA coprecipitating with ribonucleoproteins (RNPs) or ribosomal subunits indicates lack of translation, while coprecipitation with polysomes suggests active translation. Statistics: 2-tailed Student’s <em>t</em> test (<strong>C</strong>), 1-way ANOVA with LSD post hoc analysis (<strong>E</strong>–<strong>H</strong> and <strong>K</strong>), and 2-way ANOVA with Tukey’s multiple-comparison test (<strong>I</strong>). *siCtrl vs. siNAT10-1; <sup>#</sup>siCtrl vs. siNAT10-2. <em>n</em> = 3. Results are presented as mean ± SEM. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001; <sup>##</sup><em>P</em> &lt; 0.01, <sup>###</sup><em>P</em> &lt; 0.001.</p></figcaption></figure></section><section id="sec8"><h3 class="pmc_sec_title">NAT10 downregulation inhibits OPC differentiation by suppressing Lipin1 expression in vitro.</h3>
<p>To determine the biological impact of NAT10 on OPC function, we evaluated OPC differentiation following NAT10 knockdown. IF results revealed a significant decrease in MBP<sup>+</sup> cells and an increase in PDGFR-α<sup>+</sup> cells in the siNAT10 group compared with the siCtrl group (<a href="#F7" class="usa-link">Figure 7, A–D</a>). WB analysis confirmed reduced MBP levels and elevated PDGFR-α expression following NAT10 knockdown (<a href="#F7" class="usa-link">Figure 7, E and F</a>). To assess whether Lipin1 mediates this effect, we performed a rescue experiment by overexpressing Lipin1 in NAT10-deficient OPCs. Overexpression of Lipin1 significantly reversed the changes in PDGFR-α and MBP levels induced by NAT10 knockdown (<a href="#F7" class="usa-link">Figure 7, G–K</a>). These findings suggest that NAT10 promotes OPC differentiation at least in part through the regulation of Lipin1 expression.</p>
<figure class="fig xbox font-sm" id="F7"><h4 class="obj_head">Figure 7. Downregulation of NAT10 inhibits OPC differentiation by suppressing Lipin1 expression in vitro.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333955_jciinsight-10-193712-g150.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7374/12333955/c4b035476220/jciinsight-10-193712-g150.jpg" loading="lazy" height="753" width="729" alt="Figure 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/F7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Immunofluorescent staining to detect the number of MBP<sup>+</sup> (green) and PDGFR-α<sup>+</sup> (red) cells during differentiation in primary mouse OPCs with NAT10 knockdown. Scale bar: 100 μm. (<strong>B</strong>–<strong>D</strong>) Statistical analysis of the number of MBP<sup>+</sup> cells and PDGFR-α<sup>+</sup> cells, and the MBP<sup>+</sup>/PDGFR-α<sup>+</sup> cell ratio in primary mouse OPCs. (<strong>E</strong> and <strong>F</strong>) Western blot analysis of MBP and PDGFR-α protein levels in primary cultured OPCs and statistical analysis. Representative of 3 independent experiments. (<strong>G</strong>–<strong>K</strong>) Western blot analysis of NAT10, Lipin1, PDGFR-α, and MBP protein levels in OPCs under different treatments and statistical analysis. Representative of 3 independent experiments. Statistics: 1-way ANOVA with LSD post hoc analysis (<strong>B</strong>–<strong>D</strong> and <strong>H</strong>–<strong>K</strong>) and 1-way ANOVA with Dunnett’s test (<strong>F</strong>). <em>n</em> = 3. Results are presented as mean ± SEM. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001.</p></figcaption></figure></section></section><section id="sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p>Recent studies have emphasized the crucial role of lipid metabolic balance in OPC differentiation and myelination development. Specifically, the accumulation of 8,9-unsaturated sterol intermediates in the cholesterol synthesis pathway has been identified as a central mechanism that promotes OPC differentiation and maturation (<a href="#B24" class="usa-link" aria-describedby="B24">24</a>). During development, OPCs progressively reorganize their lipid metabolism and alter membrane lipid composition, significantly increasing the biosynthesis of cholesterol and galactolipids while reducing the relative amounts of phospholipids and proteins (<a href="#B12" class="usa-link" aria-describedby="B12">12</a>). Furthermore, targeted metabolomics studies of postmortem brain tissue from patients with CP have revealed significant dysregulation in glycerophospholipid metabolism (<a href="#B25" class="usa-link" aria-describedby="B25">25</a>). Glycerophospholipids are mainly synthesized within the brain and not only serve as structural components of cell membranes but also act as second messengers and precursors, participating in membrane fusion, apoptosis, and the regulation of membrane-bound enzymes and ion channels (<a href="#B26" class="usa-link" aria-describedby="B26">26</a>–<a href="#B28" class="usa-link" aria-describedby="B28">28</a>). Lipin1 protein plays a central role in the penultimate step of the glycerol phosphate pathway and catalyzes the conversion of PA to DG (<a href="#B29" class="usa-link" aria-describedby="B29">29</a>). The molecular and cellular roles of Lipin1 in OL development and regeneration in vivo have yet to be investigated.</p>
<p>Lipin1-deficient mice exhibit peripheral demyelination, hyperlipidemia, and lipodystrophy, highlighting its role in lipid metabolism and myelin formation (<a href="#B30" class="usa-link" aria-describedby="B30">30</a>, <a href="#B31" class="usa-link" aria-describedby="B31">31</a>). Our study found that Lipin1 expression in OPCs is significantly downregulated after HI injury, accompanied by elevated PA and reduced DG in the brain. Lipin1 expression in the corpus callosum decreases 14 days after HI but increases later, potentially regulated by lipid metabolites activating peroxisome proliferator-activated receptor α (PPARα) (<a href="#B32" class="usa-link" aria-describedby="B32">32</a>). We showed that Lipin1 downregulation impairs OPC differentiation and myelination, while its overexpression improves these processes and motor function in PWMI mice. The corpus callosum is the largest white matter structure in the brain and plays a critical role in interhemispheric information transfer. Its proper myelination is essential for coordinated motor function and cognitive processes (<a href="#B33" class="usa-link" aria-describedby="B33">33</a>). Clinical and preclinical studies have shown that damage to the corpus callosum is associated with deficits in locomotion, learning, and memory (<a href="#B34" class="usa-link" aria-describedby="B34">34</a>). The observed behavioral abnormalities in our study — including impaired motor coordination and spatial memory — are likely attributable to hypomyelination of the corpus callosum. This highlights the functional importance of Lipin1 in maintaining the structural and physiological integrity of this key brain region.</p>
<p>Our study provides what we believe to be the first evidence implicating Lipin1 in OPC differentiation. As a PAP, Lipin1 regulates glycerophospholipid metabolism and triglyceride (TG) synthesis by converting PA to DG. In the glycerol phosphate pathway, the final and only committed step is to form a TG by covalently joining a fatty acyl-CoA and a DG molecule (<a href="#B35" class="usa-link" aria-describedby="B35">35</a>). Lipin1 deficiency has been shown to promote axonal regeneration after injury by modulating TG hydrolysis and phospholipid synthesis (<a href="#B36" class="usa-link" aria-describedby="B36">36</a>). Additionally, reduced Lipin1 levels in hyperglycemia disrupt hippocampal mitochondrial dynamics and impair cognitive function in diabetic encephalopathy (<a href="#B37" class="usa-link" aria-describedby="B37">37</a>). These findings emphasize Lipin1’s critical role in disease pathology. Moreover, DG, via the Kennedy pathway, produces phosphatidylethanolamine and phosphatidylcholine, which are essential for myelin integrity (<a href="#B15" class="usa-link" aria-describedby="B15">15</a>, <a href="#B22" class="usa-link" aria-describedby="B22">22</a>), and phosphatidylethanolamine deficiency is linked to myelin dysfunction (<a href="#B38" class="usa-link" aria-describedby="B38">38</a>). PA and DG also act as second messengers, activating protein kinase C (PKC) and mechanistic target of rapamycin (mTOR), which regulate myelination (<a href="#B39" class="usa-link" aria-describedby="B39">39</a>–<a href="#B41" class="usa-link" aria-describedby="B41">41</a>). Besides its enzymatic activity, Lipin1 also regulates gene expression by translocating to the nucleus and interacting with transcription factors (<a href="#B16" class="usa-link" aria-describedby="B16">16</a>), highlighting its role in cellular homeostasis and lipid metabolism.</p>
<p>To investigate the mechanisms behind Lipin1 downregulation in PWMI, we used RNA pull-down to isolate proteins binding to Lipin1 mRNA from primary OPCs, followed by mass spectrometry analysis. We focused on NAT10 as a key modulator of these processes because it is the only acetyltransferase known to catalyze the addition of ac4C to RNA substrates, with acetyl-CoA serving as the acetyl donor (<a href="#B20" class="usa-link" aria-describedby="B20">20</a>, <a href="#B42" class="usa-link" aria-describedby="B42">42</a>). ac4C modifications are common in human mRNA, ensuring accurate codon reading during translation (<a href="#B43" class="usa-link" aria-describedby="B43">43</a>). Using the PACES database, we predicted acetylation sites on Lipin1 mRNA, and RIP and RNA pull-down experiments in OPCs confirmed that NAT10 mediates its ac4C modification by directly binding to the mRNA. Arango et al. utilized transcriptome-wide techniques to examine the distribution and functional impact of ac4C in mRNA, revealing that NAT10-mediated ac4C modification enhances the stability and translation efficiency of target mRNAs (<a href="#B19" class="usa-link" aria-describedby="B19">19</a>). In our study, we found that HI injury decreases NAT10 levels in OPCs, reducing Lipin1 mRNA stability and translation, thereby inhibiting Lipin1 expression. The mechanisms underlying NAT10 downregulation after HI injury remain to be fully elucidated. However, we speculate that the HI environment may suppress NAT10 expression through several pathways. First, HI-induced oxidative stress and inflammatory signaling (e.g., via TNF-α or IL-1β) may inhibit NAT10 transcription or promote its degradation, as has been shown for other RNA-modifying enzymes under stress conditions (<a href="#B44" class="usa-link" aria-describedby="B44">44</a>). Second, metabolic stress such as reduced acetyl-CoA availability — a key substrate for NAT10’s enzymatic activity — could impair both its function and expression (<a href="#B45" class="usa-link" aria-describedby="B45">45</a>). Third, epigenetic modifications such as promoter methylation or histone deacetylation may also suppress NAT10 expression in response to HI-induced cellular reprogramming (<a href="#B46" class="usa-link" aria-describedby="B46">46</a>). Further studies are needed to explore these mechanisms.</p>
<p>While most studies on NAT10 have focused on its role in cancer, where it regulates gene expression to promote tumor progression and resistance to therapy (<a href="#B47" class="usa-link" aria-describedby="B47">47</a>–<a href="#B49" class="usa-link" aria-describedby="B49">49</a>), lipid metabolism dysregulation has emerged as one of the most prominent metabolic changes associated with cancer (<a href="#B50" class="usa-link" aria-describedby="B50">50</a>). NAT10-mediated ac4C modification stabilizes lipid metabolism–related genes, including ELOVL6, ACSLs and ACAT1 (<a href="#B51" class="usa-link" aria-describedby="B51">51</a>). Our findings suggest that NAT10, through its regulation of Lipin1, contributes to myelination disruption in PWMI, implicating it in the disorder’s pathogenesis. NAT10-mediated ac4C modification has been linked to several human diseases, including depression (<a href="#B21" class="usa-link" aria-describedby="B21">21</a>), myocardial infarction (<a href="#B52" class="usa-link" aria-describedby="B52">52</a>), interstitial cystitis (<a href="#B53" class="usa-link" aria-describedby="B53">53</a>), and acquired immune deficiency syndrome (AIDS) (<a href="#B54" class="usa-link" aria-describedby="B54">54</a>). Inhibition of NAT10 can improve nuclear shape, repair DNA damage, and reverse cell proliferation defects, offering potential treatment for Hutchinson-Gilford progeria syndrome (<a href="#B55" class="usa-link" aria-describedby="B55">55</a>). Yang et al. found that overexpressing NAT10 in ovariectomized mice reversed bone loss (<a href="#B56" class="usa-link" aria-describedby="B56">56</a>), further supporting NAT10 as a promising therapeutic target for various diseases. Our study connects NAT10-mediated ac4C modification to myelination defects in PWMI, suggesting a potential therapeutic strategy for addressing this disorder.</p>
<p>Our study has limitations, including the need for further exploration of lipid metabolic pathways and OPC differentiation. Detailed lipidomic profiling of OPCs and myelin may reveal additional contributors to myelination defects. Moreover, using a single PWMI animal model limits clinical relevance. Future research with preterm birth models or iPSC-derived OPCs is needed for validation. In conclusion, our study provides further mechanistic insights into the molecular mechanisms underlying myelination defects in PWMI, not only broadening our understanding of lipid metabolism in OPC biology but also suggesting that targeting Lipin1 and NAT10 could offer potential therapeutic avenues for treating myelination defects in PWMI.</p></section><section id="sec10"><h2 class="pmc_sec_title">Methods</h2>
<section id="sec11"><h3 class="pmc_sec_title">Sex as a biological variable</h3>
<p>Both male and female neonatal mice were used without sex-based distinction. No sex-related differences were observed, and the results are considered relevant to both sexes.</p></section><section id="sec12"><h3 class="pmc_sec_title">Animals</h3>
<p>Specific pathogen–free–grade wild-type C57BL/6J mice were purchased from the Animal Center of Xuzhou Medical University. After 1 week of adaptation to the new environment, the mice were mated at the onset of puberty (males at 8 weeks of age, females at 6 weeks of age), with 1 male and 2 females per cage. Female mice were checked daily for the presence of a vaginal plug to determine the start of pregnancy, after which males were promptly separated. The day of birth was considered day 0, and subsequent days were numbered accordingly. All animals were housed in individually ventilated cage mouse housing systems with free access to food and water. The lighting cycle followed a 12-hour light/dark schedule, with room temperature maintained at 20°C–22°C and relative humidity at 50%–60%. The mouse feed was obtained from the Animal Experiment Center of Xuzhou Medical University, and drinking water was replaced every 3 days.</p></section><section id="sec13"><h3 class="pmc_sec_title">Development of the PWMI mouse model</h3>
<p>Unsexed 3-day-old C57BL/6J pups were anesthetized using isoflurane (3% for induction and 1.5% for maintenance). Upon achieving anesthesia, the pups were placed in a supine position on a thermostatically controlled surgical table maintained at 37°C, and their heads and limbs were gently but securely immobilized. Under a stereomicroscope (Olympus SZ61), the right common carotid artery of each pup was exposed, carefully isolated from surrounding nerves and veins, and ligated with 6-0 surgical silk (Ethicon). The fat and fascia layers were then closed sequentially, and the incision was sutured. The total duration of the surgical procedure was limited to 5 minutes. Following the surgery, the pups were placed in an anoxic chamber for 1.5 hours, which contained a gas mixture of 8% O<sub>2</sub> and 92% N<sub>2</sub>. Oxygen and nitrogen concentrations were monitored in real time using a gas analyzer (BioSpherix ProOx 110). After the anoxia period, the pups were returned to their mother’s cage. Sham-operated animals underwent the same procedure, except the carotid artery was not ligated. All surgical instruments were sterilized before use, and all procedures were performed in accordance with institutional guidelines for animal care and use.</p></section><section id="sec14"><h3 class="pmc_sec_title">IF staining</h3>
<p>The brain sections (thickness: 20 μm, prepared by freezing microtome) were incubated in 5% bovine serum albumin (BSA) (VICMED, VIC018) and 0.3% Triton X-100 (VICMED, VIC398) in 0.01 mol/L PBS for 30 minutes at 37°C to block nonspecific reactions. For cell samples, 200 μL of 4% paraformaldehyde (Solarbio, P1110) was added to each well of a 24-well plate and incubated at room temperature for 20 minutes. Then, the samples were washed with PBS for 5 minutes, repeating the wash 3 times. Membrane permeabilization and blocking were performed simultaneously with 0.3% Triton X-100 and 5% BSA in PBS for 30 minutes at room temperature. Sections or cells were incubated overnight at 4°C with primary antibodies anti-MBP (Santa Cruz Biotechnology, sc-376995; 1:200), anti–PDGFR-α (Abcam, ab203491; 1:300), anti-Olig2 (Millipore, MABN50; 1:500), anti-Lipin1 (Abcam, ab181389; 1:200), anti-CC1 (Thermo Fisher Scientific, PA5-30580; 1:200), and anti-NAT10 (Proteintech, 13365-1-AP; 1:500). After primary antibody incubation, sections and cells were washed 3 times with PBS (5 minutes each), and then incubated with fluorescently labeled secondary antibodies (goat anti–mouse IgG Alexa Fluor 488, goat anti–rabbit IgG Alexa Fluor 594; both from Abcam; 1:500) for 1 hour at room temperature in the dark. After secondary antibody incubation, sections and cells were washed with PBS 3 times. Nuclei were counterstained with DAPI (Solarbio, C0060) for 5 minutes at room temperature, followed by final washes. Imaging was performed with a fluorescence microscope (Olympus BX53, equipped with DP80 camera and CellSens Dimension software). All images were acquired with consistent exposure times and gain settings. For each staining set, at least 3 independent biological replicates were performed.</p></section><section id="sec15"><h3 class="pmc_sec_title">WB analysis</h3>
<p>The brain tissue and cells samples were rapidly isolated to perform WB detection. WB analysis was performed as previously described (<a href="#B57" class="usa-link" aria-describedby="B57">57</a>). Total proteins were extracted by ultrasonically homogenizing brain tissues in ice-cold RIPA lysis buffer (Beyotime, P0013B) supplemented with 1% PMSF (Beyotime, ST506). For cytoplasmic protein isolation, a cytoplasmic protein extraction kit (Beyotime, P0027) was utilized according to the manufacturer’s guidelines. Protein concentrations were quantified using the BCA Protein Assay Kit (Beyotime, P0012). An equivalent amount of protein (20–30 μg) from each sample was mixed with 5× SDS loading buffer, denatured at 95°C for 5 minutes, and separated in 10% SDS-polyacrylamide gels. Subsequently, proteins were transferred onto 0.22 μm PVDF membranes (Millipore, IPVH00010) by wet electrotransfer. Following transfer, membranes were blocked in PBST containing 5% skim milk for 1 hour at room temperature and incubated overnight at 4°C with the following primary antibodies: anti-MBP (Santa Cruz Biotechnology, sc-376995; 1:1000), anti–PDGFR-α (Abcam, ab203491; 1:1000), anti-Lipin1 (Abcam, ab181389; 1:1000), anti-CC1 (Thermo Fisher Scientific, PA5-30580; 1:1000), anti-NAT10 (Proteintech, 13365-1-AP; 1:1000), and anti–β-actin (Proteintech, 66009-1-Ig; 1:5000). As a negative control, membranes were also incubated with secondary antibodies only, omitting the primary antibodies, to assess nonspecific binding. After washing, membranes were incubated for 2 hours at room temperature with either IRDye 680RD Goat anti-Rabbit IgG (LI-COR, 926–68071; 1:10,000) or IRDye 800CW Goat anti-Mouse IgG (LI-COR, 926–32210; 1:20,000) secondary antibodies. Protein bands were visualized and analyzed using an Odyssey infrared imaging system (LI-COR Biosciences). Band intensities were quantified using ImageJ software version 1.8.0 (NIH). All experiments were independently repeated at least 3 times.</p></section><section id="sec16"><h3 class="pmc_sec_title">TEM</h3>
<p>For TEM analysis, mouse brains were perfused with an electron microscope fixative (containing 2% glutaraldehyde and 4% paraformaldehyde), and the corpus callosum was isolated on ice. A tissue block (~1 mm³) was cut from the corpus callosum, maintaining consistent sampling across groups. The tissue was then fixed overnight in 2.5% glutaraldehyde at 4°C, followed by PBS washes (10 minutes, repeated 3 times). Afterward, the samples were fixed in 1% osmium tetroxide in phosphate buffer at room temperature for 2 hours, followed by washing in ddH<sub>2</sub>O (10 minutes, repeated 3 times). The samples were dehydrated through a graded ethanol series (30%, 50%, 70%, 80%, 90%, 95%, 100%) and then transitioned into 100% acetone. Dehydration was followed by infiltration with a 1:1 mixture of acetone and 618 epoxy resin for 2–4 hours, then with a 2:1 acetone/resin mixture overnight. The samples were subsequently embedded in pure resin for 5–8 hours (3 exchanges), placed into embedding molds, and polymerized overnight at 37°C. After polymerization, the samples were sectioned into 50-nm-thick slices, stained with lead citrate for 15 minutes, and washed with ddH<sub>2</sub>O. Sections were dried at room temperature and observed under TEM. High-magnification images (×6000 magnification) were captured, and the G-ratio was calculated as the ratio of the inner diameter to the total outer diameter of the myelinated axon. Three samples per group were analyzed.</p></section><section id="sec17"><h3 class="pmc_sec_title">Behavioral experiment methodology</h3>
<section id="sec18"><h4 class="pmc_sec_title">MWM assessment.</h4>
<p>The MWM was used to evaluate spatial location learning and memory as described in previous studies (<a href="#B57" class="usa-link" aria-describedby="B57">57</a>). The MWM test was performed using a circular pool (diameter 1.2 m, height 40 cm) filled with water maintained at 25°C, equipped with a movable platform, temperature control, and a video tracking system (ANY-maze, Stoelting Co.). Each group included 8 mice, acclimated to the testing room 2–3 hours before testing, which was conducted between 9:00 am and 12:00 pm. The platform (8 cm diameter) was submerged 0.5–1 cm below the water surface. To enhance tracking, powdered milk was added, and visual cues were placed around the pool, which was divided into quadrants in the ANY-maze software. During training, mice were placed on the platform for 30 seconds, then released from random quadrants facing the wall to search for the platform. Each mouse underwent 2 sessions daily (2 trials per session) for 4 consecutive days. If a mouse failed to find the platform within 60 seconds, it was guided to the platform and allowed to remain for 30 seconds. On the fifth day, a probe trial was conducted by removing the platform; mice were released from the opposite quadrant, and swimming behavior was recorded for 60 seconds. The number of platform crossings, time spent, and entries into each quadrant were analyzed automatically by ANY-maze. All data were recorded using a computerized video system (Ethovision 3.1, Noldus Instruments).</p></section><section id="sec19"><h4 class="pmc_sec_title">Open field test.</h4>
<p>The open field test was conducted using a 50 cm × 50 cm × 30 cm arena and the ANY-maze automated tracking system. Mice were acclimated for 2 hours before testing. Each mouse was placed in the center of the arena for 5 minutes of free exploration while the system recorded total distance traveled, average speed, and time spent in the central and peripheral zones. After each trial, the arena was cleaned with 75% ethanol and paper towels.</p></section><section id="sec20"><h4 class="pmc_sec_title">Y-maze.</h4>
<p>In the Y-maze test, animals were first placed in the maze with 1 arm closed for 3 minutes of exploration in the remaining 2 arms. After 4 hours, a recall phase was conducted where all arms were opened, allowing the animal to explore freely for 3 minutes. The number of entries, exploration time, and path length in each arm were recorded to assess memory and spontaneous alternation behavior.</p></section><section id="sec21"><h4 class="pmc_sec_title">Rotarod test.</h4>
<p>In the rotarod test, animals underwent a training phase where they start at 4 rpm for 1 minute, then progress to 10 rpm for 2 minutes, and finally 30 rpm for 3 minutes, with 15-minute rest periods between each stage. After a 24-hour rest, the formal test began by placing the mouse on a rotating rod (accelerated to 20 rpm) and recording the latency to fall. If a mouse fell, it was promptly removed and returned to its cage. After 5 minutes, any remaining mice were moved to their cage. The maximum score is 300 seconds, with 3 trials conducted per mouse, and the average latency was used for statistical analysis.</p></section><section id="sec22"><h4 class="pmc_sec_title">Gait analysis.</h4>
<p>In the gait analysis, mice were acclimated to the testing environment 2 days prior using the VisuGait system (Zhenghua Biological Instrument and Equipment Co., Ltd.). During the training phase, mice were placed in the gait tunnel with a cage mate for 5–10 minutes to familiarize themselves with the environment, without testing. Afterward, they were individually placed at one end of the tunnel, with the other end blocked, and allowed to run twice daily for 1 week. In the formal testing phase, the tunnel exit was blocked, and the mice were placed at the entrance. Data were recorded as the mouse passed through the camera zone, with each mouse tested 5 times. The gait data, including stance duration, swing duration, and propulsion time, were averaged and analyzed for statistical significance.</p></section></section><section id="sec23"><h3 class="pmc_sec_title">Measurement of PA and DG</h3>
<p>PA was measured by using the PicoProbe Phosphatidic Acid Assay Kit (Abcam, ab273335) according to the manufacturer’s instructions. Briefly, brain tissue was homogenized in the provided PA assay buffer, and lipid extraction was performed according to the protocol. The lipids were solubilized in a 5% Triton X-100 solution. Parallel preparations of “Sample” and “Sample background control” were made. A standard curve was created using the PA standard solution. The converter mix was added exclusively to the sample and standard wells. After incubation at 45°C for 1 hour, reaction mix was introduced into all wells and incubated at 37°C for 30 minutes. Fluorescence was measured with an excitation/emission of 535/587 nm. The PA concentration was normalized based on the recorded values.</p>
<p>DG was measured by using the Diacylglycerol Assay Kit (Abcam, ab242293) according to the manufacturer’s instructions. Brain tissue was homogenized and mixed with methanol, NaCl, and chloroform, followed by phase separation through centrifugation. The organic phase was washed twice with a pre-equilibrated upper phase solution, then dried under nitrogen flow and resuspended in assay buffer. For the assay, DG standards, samples, and blanks were prepared in a 96-well plate. A kinase mixture was added to half of the sample wells, and buffer to the other half, followed by incubation at 37°C for 2 hours. After transferring the mixture to a suitable plate, lipase solution was added, and incubation continued for 30 minutes. The detection enzyme mixture was then added, and fluorescence was measured at Ex/Em = 530–560 nm/585–595 nm after 10 minutes of incubation at room temperature. Results were calculated by subtracting background values and generating a standard curve for further analysis.</p></section><section id="sec24"><h3 class="pmc_sec_title">RT-qPCR</h3>
<p>Total RNA was extracted using TRIzol reagent (Invitrogen, 15596018) following the manufacturer’s protocol. RNA concentration and purity were assessed using a NanoDrop 2000 spectrophotometer (Thermo Fisher Scientific). Reverse transcription was carried out with 1 μg of total RNA using the HiScript III 1st Strand cDNA Synthesis Kit (Vazyme, R312-01) according to the manufacturer’s instructions. qPCR was performed using ChamQ Universal SYBR qPCR Master Mix (Vazyme, Q711-02) on a LightCycler R480 PCR System (Roche) with the following cycling conditions: 95°C for 30 seconds, followed by 40 cycles of 95°C for 10 seconds and 60°C for 30 seconds (annealing temperature). Melting curve analysis was conducted to verify the specificity of amplification. Primer sequences, expected amplicon sizes (in bp), and GenBank accession numbers are provided in <a href="#sd" class="usa-link">Supplemental Table 2</a>. Relative gene expression was calculated using the 2<sup>–ΔΔCt</sup> method, and Gapdh was used as the internal reference gene (housekeeping gene). All primer pairs were validated using NCBI Primer-BLAST (<a href="https://www.ncbi.nlm.nih.gov/tools/primer-blast/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/tools/primer-blast/</a>) to ensure specificity. No off-target amplification was observed, and representative screenshots of BLAST results are presented in <a href="#sd" class="usa-link">Supplemental Figure 4</a>.</p></section><section id="sec25"><h3 class="pmc_sec_title">Virus packaging</h3>
<p>Lipin1 overexpression and RNA interference lentiviruses and their respective blank vectors were obtained from Shanghai GeneChem Corporation. The titer of the lentiviruses was 2 × 10<sup>9</sup> TU/mL. The sequences for Lipin1 shRNA and its blank vector are provided in <a href="#sd" class="usa-link">Supplemental Table 3</a>. These lentiviruses were administered in vivo via stereotaxic injection into the corpus callosum to modulate the expression of Lipin1.</p></section><section id="sec26"><h3 class="pmc_sec_title">Stereotaxic injection</h3>
<p>The stereotaxic injection procedure began by anesthetizing mice with an intraperitoneal injection of sodium pentobarbital (50 μg/g) and securing them on a stereotaxic frame, ensuring that the Bregma and Lambda points were level. After disinfecting the skin with povidone-iodine, an incision was made, and the periosteum was cleaned with 3% hydrogen peroxide to expose the Bregma point. A microsyringe was positioned so that the needle touched the Bregma, and the coordinates were recorded as the origin. The final injection coordinates were determined based on the Mouse Brain Atlas (AP: –1.89 mm, ML: 1.68 mm, DV: –1.5 mm). The virus was injected at 0.2 μL/min for 10 minutes, and the needle remained in place for another 10 minutes before being slowly withdrawn. The incision was sutured, and the mice were returned to their cages for recovery.</p></section><section id="sec27"><h3 class="pmc_sec_title">Culture of primary OPCs</h3>
<section id="sec28"><h4 class="pmc_sec_title">Isolation of mixed glial cells.</h4>
<p>Newborn mice (0–2 days) were anesthetized on crushed ice, and after disinfection with 75% ethanol, brain tissues were quickly extracted and placed in prechilled DMEM/F-12 medium. The cortical tissues were isolated under a stereomicroscope, and the meninges and non-cortical regions were removed. The cortices were minced and transferred to a 15 mL centrifuge tube containing prechilled DMEM/F-12 medium. The tissue was gently triturated 10–15 times using a sterile Pasteur pipette, allowed to settle for 2 minutes, and the supernatant was transferred to a new sterile tube. This process was repeated until the brain tissues were dissociated into single-cell suspensions. The suspensions were centrifuged at 145<em>g</em> for 5 minutes, and the supernatant was discarded. The cell pellet was resuspended in a small volume of complete medium and filtered through a 40-μm nylon mesh. The filtered cells were seeded into poly-D-lysine–coated T25 flasks at a density of 1.0 × 10<sup>6</sup> to 2.0 × 10<sup>6</sup> cells/flask and cultured in DMEM/F-12 medium containing 10% FBS in a 5% CO<sub>2</sub> incubator. Media were changed every 2 days, and after about 1 week, OPCs were isolated by shaking.</p></section><section id="sec29"><h4 class="pmc_sec_title">Purification and culture of OPCs.</h4>
<p>For OPC purification, cells were detached using a non-trypsin cell dissociation solution and centrifuged. OPCs were plated at a density of 2 × 10<sup>4</sup> cells/mL.</p></section><section id="sec30"><h4 class="pmc_sec_title">Induced differentiation of OPCs.</h4>
<p>For passaging and differentiation, bFGF (10 ng/mL) was added to the medium to promote OPC proliferation. When cells reached confluence, they were passaged at a 1:2 ratio. To induce OPC differentiation, 15 nM triiodothyronine was added to the medium, and the supply of PDGF and bFGF was discontinued. The medium was replaced every 3 days, and cells were cultured for 5–10 days to induce maturation.</p></section></section><section id="sec31"><h3 class="pmc_sec_title">siRNA transfection</h3>
<p>The medium was replaced with 1.5 mL of serum-free medium in the culture dish. To prepare the transfection complex, 3 μL of siRNA was added to 250 μL of serum-free medium, mixed, and incubated for 5 minutes. In a separate tube, 6 μL of SilentFect Lipid Reagent (Bio-Rad) was mixed with 250 μL serum-free medium and incubated for 5 minutes. Afterward, the SilentFect mixture was combined with the siRNA solution and incubated for 20 minutes. The transfection mixture was added to the culture dish, ensuring even distribution, and incubated at 37°C in a 5% CO<sub>2</sub> incubator. The medium was replaced after 6–8 hours, and protein or RNA was harvested 24–96 hours later for further analysis.</p></section><section id="sec32"><h3 class="pmc_sec_title">RNA pull-down</h3>
<p>Lipin1 probes with 3′ biotin labels were designed and synthesized by Guangzhou RiboBio Co., Ltd. Both Lipin1 sense and antisense probes were dissolved in nuclease-free water to a concentration of 100 μM. Prior to experiments, protease inhibitor cocktail (Beyotime, P1005) and RNase inhibitor (Vazyme, R411-01) were thawed at room temperature. Lysis buffer was freshly prepared by mixing WB/IP lysis buffer (Beyotime, P0013) with protease inhibitor cocktail at a 1:99 v/v ratio, and RNase inhibitor was added to a final concentration of 1 U/μL to prevent RNA degradation. A minimum of 20–30 million cells were digested with trypsin, collected, and lysed in 1 mL of lysis buffer on ice. After centrifuging at 17,000<em>g</em> for 15 minutes, the supernatant was collected, and 1 μL of either Lipin1 or antisense probe was added. The mixture was incubated on a rotator at 4°C for 6 hours or overnight, followed by the addition of 50 μL streptavidin magnetic beads (Thermo Fisher Scientific, 88817) for 3 hours. After magnetic separation, the beads were washed 3 times with lysis buffer and resuspended in 60 μL 2× SDS-PAGE buffer, and then heated at 100°C for 10 minutes for further analysis or storage.</p></section><section id="sec33"><h3 class="pmc_sec_title">Mass spectrometry analysis</h3>
<p>Mass spectrometry analysis was performed by Shanghai Zhongke Xin Life Biotechnology Co., Ltd. Protein samples were first reduced with 10 mM dithiothreitol (DTT) at 56°C for 1 hour and subsequently alkylated with 20 mM iodoacetamide at room temperature in the dark for 45 minutes. After reduction and alkylation, proteins were enzymatically digested with sequencing-grade modified trypsin at an enzyme-to-substrate ratio of 1:50 (w/w) at 37°C for 20 hours. After desalting and lyophilization, the digest was redissolved in 0.1% formic acid solution and stored at –20°C. For analysis, a mobile phase of 0.1% formic acid in water (A) and 0.1% formic acid in acetonitrile (B) was used. The peptides were loaded onto a C18 reversed-phase analytical column (Thermo Fisher Scientific, 75 μm × 250 mm, 2 μm particle size) and separated using a linear gradient of 5%–35% B over 90 minutes at a flow rate of 300 nL/min. Mass spectrometry was conducted in positive ion mode with a scan range of 300–1800 <em>m</em>/<em>z</em>, a resolution of 70,000 at 200 <em>m</em>/<em>z</em>, and an AGC target of 1 × 10<sup>6</sup>. After each full scan, 20 MS2 scans were acquired using HCD activation, with a collision energy of 27 eV. Data quality control was performed by monitoring the reproducibility of identified peptides across biological replicates and extracting specific marker peaks. The data were analyzed using Proteome Discoverer 2.5 (Thermo Fisher Scientific), and protein identification was achieved by searching the appropriate database.</p></section><section id="sec34"><h3 class="pmc_sec_title">Dot blot assay</h3>
<p>Total RNA was extracted from tissues using TRIzol reagent according to the manufacturer’s instructions. The RNA concentration was measured using a NanoDrop 2000 spectrophotometer and RNA integrity was confirmed by agarose gel electrophoresis. RNA was extracted from tissues and denatured by heating at 95°C for 3 minutes, then placed on ice. Two microliters of the denatured RNA was applied to a Hybond-N+ membrane (Sigma-Aldrich), crosslinked with UV light for 1 minute, and repeated once. The membrane was washed in washing buffer, blocked for 1 hour, and incubated overnight at 4°C with anti-ac4C antibody (Abcam, ab252215; 1:1000). After 3 washes with TBST (5 minutes each), the membrane was incubated with HRP-conjugated goat anti-rabbit IgG secondary antibody for 1 hour at room temperature. Finally, the membrane was washed and exposed to ECL Western Blotting Reagents (Thermo Fisher Scientific, 32106) for 5 minutes before imaging.</p></section><section id="sec35"><h3 class="pmc_sec_title">RIP</h3>
<p>Proteinase inhibitor cocktail (Sigma-Aldrich, P8340) and RNase inhibitor (Thermo Fisher Scientific, EN0531) were thawed at room temperature. WB and IP lysis buffers were prepared by mixing 1× PBS with 0.5% NP-40, 1 mM DTT, 1 mM EDTA, 1 mM PMSF, and 1× proteinase inhibitor cocktail. The RNase inhibitor was added to achieve a final concentration of 1 U/μL, ensuring the RNA was protected from degradation during the process. The WB and IP lysis buffers were mixed with the proteinase inhibitor cocktail in a 99:1 ratio, and RNase inhibitor was added to achieve a final concentration of 1 U/μL, serving as the lysis buffer. Cells were collected, centrifuged, and resuspended in 1 mL of the prepared lysis buffer. The lysates were subjected to repeated freeze-thaw cycles in liquid nitrogen to disrupt the cell membranes. Afterward, the lysates were centrifuged at 17,000<em>g</em> for 15 minutes at 4°C to remove cellular debris, and the supernatant was carefully collected. To capture RNA-protein complexes, the supernatant was incubated with 2 μg of specific antibody (anti-FLAG, Abcam, ab1253; or anti-Myc, Cell Signaling Technology, 2276) overnight at 4°C with gentle rotation. Protein A/G magnetic beads (Thermo Fisher Scientific, 88802) were then added to the lysate-antibody mixture and incubated for an additional 3 hours at 4°C. Afterward, the mixture was washed 3 times with lysis buffer, and RNA was extracted using 1 mL of TRIzol reagent for subsequent analysis. The RNA yield and quality were assessed by NanoDrop spectrophotometer, and RT-qPCR was performed to verify the RNA-protein interaction.</p></section><section id="sec36"><h3 class="pmc_sec_title">Sucrose density gradient centrifugation</h3>
<p>Cells were treated with 100 μg/mL cycloheximide (Sigma-Aldrich, C7698), washed with PBS, and collected by centrifugation at 101<em>g</em> for 5 minutes at 4°C. The cell pellet was resuspended in 1 mL of cold extraction buffer containing 20 mM Tris-HCl (pH 7.5), 150 mM NaCl, 1% NP-40, 1 mM DTT, 1 mM EDTA, protease inhibitors (Sigma-Aldrich, P8340), and RNase inhibitors (Thermo Fisher Scientific, EN0531), followed by incubation on ice for 30 minutes with gentle agitation to lyse the cells. The lysate was centrifuged, and the supernatant was used for gradient preparation. A sucrose gradient consisting of 10%, 20%, 30%, 40%, and 50% concentrations was prepared by layering the solutions in ultracentrifuge tubes and incubating at 4°C overnight. The lysate was carefully layered onto the gradient, and centrifugation was performed at 190,000<em>g</em> for 90 minutes at 4°C. After centrifugation, the gradient was fractionated into 18 parts, and RNA was extracted from each fraction using TRIzol reagent according to the manufacturer’s instructions. The RNA was then analyzed by RT-qPCR to quantify specific RNA levels in each fraction. Ct values were calculated, and the distribution of RNA across the gradient was plotted to generate a curve representing the relative abundance of RNA in each fraction.</p></section><section id="sec37"><h3 class="pmc_sec_title">Actinomycin D experiment</h3>
<p>Cells were seeded in 6-well plates for 24 hours and treated with 5 μg/mL actinomycin D solution (Sigma-Aldrich, A1410). Samples were collected at 0, 2, and 4 hours after treatment to assess RNA stability. For RNA extraction, the culture medium was discarded, and cells were washed with PBS to remove excess serum. One milliliter of TRIzol was added to each well, and the cells were homogenized with RNase-free tips. The mixture was allowed to stand at room temperature for 5 minutes, and the liquid was transferred to RNase-free 1.5 mL EP tubes. The remaining RNA extraction steps followed the RNA extraction procedure described earlier in this section as part of the RT-qPCR analysis. Target RNA levels were assessed by RT-qPCR.</p></section><section id="sec38"><h3 class="pmc_sec_title">Statistics</h3>
<p>All histograms and line charts in this study were created using GraphPad Prism 9.0, while data analysis was performed with SPSS software version 24.0. The Shapiro-Wilk test was employed to assess normality, and Levene’s test was used to check for homogeneity of variance. For comparisons between 2 unpaired groups, parametric data were analyzed using a 2-tailed Student’s <em>t</em> test, and nonparametric data were assessed using the Mann-Whitney test. For comparisons among 3 or 4 groups, 1-way ANOVA was applied, followed by LSD post hoc analysis when <em>F</em> ratios were significant (<em>P</em> &gt; 0.05) or Dunnett’s test when <em>F</em> ratios were not significant (<em>P</em> &lt; 0.05). Two-way ANOVA with Tukey’s multiple-comparison test was used to determine significant differences in behavioral parameters. No outlier tests were performed, and no data points were excluded. A <em>P</em> value of less than 0.05 was considered statistically significant.</p></section><section id="sec39"><h3 class="pmc_sec_title">Study approval</h3>
<p>All animal care and experimental procedures were performed according to the NIH <em>Guide for the Care and Use of Laboratory Animals</em> (National Academies Press, 2011) and approved by the Ethics Committee of Experimental Animal Center of Xuzhou Medical University (approval no. SYXK2005-0018).</p></section><section id="sec40"><h3 class="pmc_sec_title">Data availability</h3>
<p>All data supporting the findings of this study are available within the article and its supplemental materials. In addition, all underlying data values for graphs and reported means in both main and <a href="#sd" class="usa-link">Supplemental figures</a> are provided in the <a href="#sd" class="usa-link">Supporting Data Values</a> file, with separate tabs corresponding to each figure panel.</p></section></section><section id="sec41"><h2 class="pmc_sec_title">Author contributions</h2>
<p>RY, XL, and MZ designed the research studies. MZ, YL, CG, YL, and LH conducted experiments. MZ and ZF acquired data. XL and XW analyzed data. XL and RY wrote the manuscript. All authors approved the final version of the manuscript.</p></section><section id="sd"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="SD1"><div class="caption p"><span>Supplemental data</span></div>
<div class="media p" id="d67e837"><div class="caption">
<a href="/articles/instance/12333955/bin/jciinsight-10-193712-s151.pdf" data-ga-action="click_feat_suppl" class="usa-link">jciinsight-10-193712-s151.pdf</a><sup> (1.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD2"><div class="caption p"><span>Unedited blot and gel images</span></div>
<div class="media p" id="d67e841"><div class="caption">
<a href="/articles/instance/12333955/bin/jciinsight-10-193712-s152.pdf" data-ga-action="click_feat_suppl" class="usa-link">jciinsight-10-193712-s152.pdf</a><sup> (510.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD3"><div class="caption p"><span>Supporting data values</span></div>
<div class="media p" id="d67e845"><div class="caption">
<a href="/articles/instance/12333955/bin/jciinsight-10-193712-s153.xlsx" data-ga-action="click_feat_suppl" class="usa-link">jciinsight-10-193712-s153.xlsx</a><sup> (104.6KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>This study was supported by the National Natural Science Foundation of China (grants 82471754, 82201545, and 81771337).</p></section><section id="notes1"><section id="sec43"><h3 class="pmc_sec_title">Version 1. 08/08/2025</h3>
<p>Electronic publication</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn2"><p><strong>Conflict of interest:</strong> The authors have declared that no conflict of interest exists.</p></div>
<div class="fn p" id="fn3"><p><strong>Copyright:</strong> © 2025, Li et al. This is an open access article published under the terms of the Creative Commons Attribution 4.0 International License.</p></div>
<div class="fn p" id="fn4"><p><strong>Reference information:</strong><em> JCI Insight</em>. 2025;10(15):e193712.https://doi.org/10.1172/jci.insight.193712.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Xinyu Li, Email: 15996963677@163.com.</p>
<p>Meng Zhang, Email: 13407542587@163.com.</p>
<p>Yanan Liu, Email: anatomy105@163.com.</p>
<p>Chunjie Guo, Email: 13937092159@163.com.</p>
<p>Yiwei Liu, Email: wei_LLLLL000@163.com.</p>
<p>Lei Han, Email: 18056720707@163.com.</p>
<p>Zhaowei Feng, Email: zhwfeng141592@163.com.</p>
<p>Xiue Wei, Email: wxeqq@163.com.</p>
<p>Ruiqin Yao, Email: wenxi_yao@163.com.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>Amankwah N, et al.  Cerebral palsy in Canada, 2011-2031: results of a microsimulation modelling study of epidemiological and cost impacts. Health Promot Chronic Dis Prev Can. 2020;40(2):25–37. doi: 10.24095/hpcdp.40.2.01.</cite> [<a href="https://doi.org/10.24095/hpcdp.40.2.01" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7053851/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32049464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Health%20Promot%20Chronic%20Dis%20Prev%20Can&amp;title=Cerebral%20palsy%20in%20Canada,%202011-2031:%20results%20of%20a%20microsimulation%20modelling%20study%20of%20epidemiological%20and%20cost%20impacts&amp;author=N%20Amankwah&amp;volume=40&amp;issue=2&amp;publication_year=2020&amp;pages=25-37&amp;pmid=32049464&amp;doi=10.24095/hpcdp.40.2.01&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>Jarjour IT. Neurodevelopmental outcome after extreme prematurity: a review of the literature. Pediatr Neurol. 2015;52(2):143–152. doi: 10.1016/j.pediatrneurol.2014.10.027.</cite> [<a href="https://doi.org/10.1016/j.pediatrneurol.2014.10.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25497122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr%20Neurol&amp;title=Neurodevelopmental%20outcome%20after%20extreme%20prematurity:%20a%20review%20of%20the%20literature&amp;author=IT%20Jarjour&amp;volume=52&amp;issue=2&amp;publication_year=2015&amp;pages=143-152&amp;pmid=25497122&amp;doi=10.1016/j.pediatrneurol.2014.10.027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>Lin CY, et al.  Neurodevelopmental outcomes at 2 and 5 years of age in very-low-birth-weight preterm infants born between 2002 and 2009: a prospective cohort study in Taiwan. Pediatr Neonatol. 2020;61(1):36–44. doi: 10.1016/j.pedneo.2019.05.006.</cite> [<a href="https://doi.org/10.1016/j.pedneo.2019.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31278044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr%20Neonatol&amp;title=Neurodevelopmental%20outcomes%20at%202%20and%205%20years%20of%20age%20in%20very-low-birth-weight%20preterm%20infants%20born%20between%202002%20and%202009:%20a%20prospective%20cohort%20study%20in%20Taiwan&amp;author=CY%20Lin&amp;volume=61&amp;issue=1&amp;publication_year=2020&amp;pages=36-44&amp;pmid=31278044&amp;doi=10.1016/j.pedneo.2019.05.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>Wang J, et al.  Perinatal risk factors influencing neonatal hypoxic ischemic encephalopathy in southern China: a case-control study. Am J Perinatol. 2021;38(s 01):e182–e186. doi: 10.1055/s-0040-1708884.</cite> [<a href="https://doi.org/10.1055/s-0040-1708884" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32219797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am%20J%20Perinatol&amp;title=Perinatal%20risk%20factors%20influencing%20neonatal%20hypoxic%20ischemic%20encephalopathy%20in%20southern%20China:%20a%20case-control%20study&amp;author=J%20Wang&amp;volume=38&amp;issue=s%2001&amp;publication_year=2021&amp;pages=e182-e186&amp;pmid=32219797&amp;doi=10.1055/s-0040-1708884&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>Peebles PJ, et al.  Antenatal and intrapartum risk factors for neonatal hypoxic ischemic encephalopathy. J Perinatol. 2020;40(1):63–69. doi: 10.1038/s41372-019-0531-6.</cite> [<a href="https://doi.org/10.1038/s41372-019-0531-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31611618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Perinatol&amp;title=Antenatal%20and%20intrapartum%20risk%20factors%20for%20neonatal%20hypoxic%20ischemic%20encephalopathy&amp;author=PJ%20Peebles&amp;volume=40&amp;issue=1&amp;publication_year=2020&amp;pages=63-69&amp;pmid=31611618&amp;doi=10.1038/s41372-019-0531-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>Back SA. White matter injury in the preterm infant: pathology and mechanisms. Acta Neuropathol. 2017;134(3):331–349. doi: 10.1007/s00401-017-1718-6.</cite> [<a href="https://doi.org/10.1007/s00401-017-1718-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5973818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28534077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Neuropathol&amp;title=White%20matter%20injury%20in%20the%20preterm%20infant:%20pathology%20and%20mechanisms&amp;author=SA%20Back&amp;volume=134&amp;issue=3&amp;publication_year=2017&amp;pages=331-349&amp;pmid=28534077&amp;doi=10.1007/s00401-017-1718-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>Back SA, et al.  Maturation-dependent vulnerability of oligodendrocytes to oxidative stress-induced death caused by glutathione depletion. J Neurosci. 1998;18(16):6241–6253. doi: 10.1523/JNEUROSCI.18-16-06241.1998.</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.18-16-06241.1998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6793198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9698317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&amp;title=Maturation-dependent%20vulnerability%20of%20oligodendrocytes%20to%20oxidative%20stress-induced%20death%20caused%20by%20glutathione%20depletion&amp;author=SA%20Back&amp;volume=18&amp;issue=16&amp;publication_year=1998&amp;pages=6241-6253&amp;pmid=9698317&amp;doi=10.1523/JNEUROSCI.18-16-06241.1998&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>Yates N, et al.  Preventing brain injury in the preterm infant-current controversies and potential therapies. Int J Mol Sci. 2021;22(4):1671.  doi: 10.3390/ijms22041671.</cite> [<a href="https://doi.org/10.3390/ijms22041671" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7915709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33562339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Preventing%20brain%20injury%20in%20the%20preterm%20infant-current%20controversies%20and%20potential%20therapies&amp;author=N%20Yates&amp;volume=22&amp;issue=4&amp;publication_year=2021&amp;pages=1671&amp;pmid=33562339&amp;doi=10.3390/ijms22041671&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>Segovia KN, et al.  Arrested oligodendrocyte lineage maturation in chronic perinatal white matter injury. Ann Neurol. 2008;63(4):520–530. doi: 10.1002/ana.21359.</cite> [<a href="https://doi.org/10.1002/ana.21359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3140464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18393269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann%20Neurol&amp;title=Arrested%20oligodendrocyte%20lineage%20maturation%20in%20chronic%20perinatal%20white%20matter%20injury&amp;author=KN%20Segovia&amp;volume=63&amp;issue=4&amp;publication_year=2008&amp;pages=520-530&amp;pmid=18393269&amp;doi=10.1002/ana.21359&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>Motavaf M, Piao X. Oligodendrocyte development and implication in perinatal white matter injury. Front Cell Neurosci. 2021;15:764486.  doi: 10.3389/fncel.2021.764486.</cite> [<a href="https://doi.org/10.3389/fncel.2021.764486" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8599582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34803612/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Neurosci&amp;title=Oligodendrocyte%20development%20and%20implication%20in%20perinatal%20white%20matter%20injury&amp;author=M%20Motavaf&amp;author=X%20Piao&amp;volume=15&amp;publication_year=2021&amp;pages=764486&amp;pmid=34803612&amp;doi=10.3389/fncel.2021.764486&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>Kolczynska K, et al.  Diacylglycerol-evoked activation of PKC and PKD isoforms in regulation of glucose and lipid metabolism: a review. Lipids Health Dis. 2020;19(1):113.  doi: 10.1186/s12944-020-01286-8.</cite> [<a href="https://doi.org/10.1186/s12944-020-01286-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7257441/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32466765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lipids%20Health%20Dis&amp;title=Diacylglycerol-evoked%20activation%20of%20PKC%20and%20PKD%20isoforms%20in%20regulation%20of%20glucose%20and%20lipid%20metabolism:%20a%20review&amp;author=K%20Kolczynska&amp;volume=19&amp;issue=1&amp;publication_year=2020&amp;pages=113&amp;pmid=32466765&amp;doi=10.1186/s12944-020-01286-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>Marangon D, et al.  Regulation of oligodendrocyte functions: targeting lipid metabolism and extracellular matrix for myelin repair. J Clin Med. 2020;9(2):470.  doi: 10.3390/jcm9020470.</cite> [<a href="https://doi.org/10.3390/jcm9020470" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7073561/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32046349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Med&amp;title=Regulation%20of%20oligodendrocyte%20functions:%20targeting%20lipid%20metabolism%20and%20extracellular%20matrix%20for%20myelin%20repair&amp;author=D%20Marangon&amp;volume=9&amp;issue=2&amp;publication_year=2020&amp;pages=470&amp;pmid=32046349&amp;doi=10.3390/jcm9020470&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>Chrast R, et al.  Lipid metabolism in myelinating glial cells: lessons from human inherited disorders and mouse models. J Lipid Res. 2011;52(3):419–434. doi: 10.1194/jlr.R009761.</cite> [<a href="https://doi.org/10.1194/jlr.R009761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3035679/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21062955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Lipid%20Res&amp;title=Lipid%20metabolism%20in%20myelinating%20glial%20cells:%20lessons%20from%20human%20inherited%20disorders%20and%20mouse%20models&amp;author=R%20Chrast&amp;volume=52&amp;issue=3&amp;publication_year=2011&amp;pages=419-434&amp;pmid=21062955&amp;doi=10.1194/jlr.R009761&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B14">
<span class="label">14.</span><cite>Mallard C, et al.  Astrocytes and microglia in acute cerebral injury underlying cerebral palsy associated with preterm birth. Pediatr Res. 2014;75(1–2):234–240. doi: 10.1038/pr.2013.188.</cite> [<a href="https://doi.org/10.1038/pr.2013.188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11908707/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24336433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr%20Res&amp;title=Astrocytes%20and%20microglia%20in%20acute%20cerebral%20injury%20underlying%20cerebral%20palsy%20associated%20with%20preterm%20birth&amp;author=C%20Mallard&amp;volume=75&amp;issue=1%E2%80%932&amp;publication_year=2014&amp;pages=234-240&amp;pmid=24336433&amp;doi=10.1038/pr.2013.188&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B15">
<span class="label">15.</span><cite>Péterfy M, et al.  Insulin-stimulated interaction with 14-3-3 promotes cytoplasmic localization of lipin-1 in adipocytes. J Biol Chem. 2010;285(6):3857–3864. doi: 10.1074/jbc.M109.072488.</cite> [<a href="https://doi.org/10.1074/jbc.M109.072488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2823528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19955570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Insulin-stimulated%20interaction%20with%2014-3-3%20promotes%20cytoplasmic%20localization%20of%20lipin-1%20in%20adipocytes&amp;author=M%20P%C3%A9terfy&amp;volume=285&amp;issue=6&amp;publication_year=2010&amp;pages=3857-3864&amp;pmid=19955570&amp;doi=10.1074/jbc.M109.072488&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B16">
<span class="label">16.</span><cite>Finck BN, et al.  Lipin 1 is an inducible amplifier of the hepatic PGC-1alpha/PPARalpha regulatory pathway. Cell Metab. 2006;4(3):199–210. doi: 10.1016/j.cmet.2006.08.005.</cite> [<a href="https://doi.org/10.1016/j.cmet.2006.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16950137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab&amp;title=Lipin%201%20is%20an%20inducible%20amplifier%20of%20the%20hepatic%20PGC-1alpha/PPARalpha%20regulatory%20pathway&amp;author=BN%20Finck&amp;volume=4&amp;issue=3&amp;publication_year=2006&amp;pages=199-210&amp;pmid=16950137&amp;doi=10.1016/j.cmet.2006.08.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B17">
<span class="label">17.</span><cite>Damato M, et al.  Protein kinase C activation drives a differentiation program in an oligodendroglial precursor model through the modulation of specific biological networks. Int J Mol Sci. 2021;22(10):5245.  doi: 10.3390/ijms22105245.</cite> [<a href="https://doi.org/10.3390/ijms22105245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8156399/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34063504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Protein%20kinase%20C%20activation%20drives%20a%20differentiation%20program%20in%20an%20oligodendroglial%20precursor%20model%20through%20the%20modulation%20of%20specific%20biological%20networks&amp;author=M%20Damato&amp;volume=22&amp;issue=10&amp;publication_year=2021&amp;pages=5245&amp;pmid=34063504&amp;doi=10.3390/ijms22105245&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B18">
<span class="label">18.</span><cite>Nadra K, et al.  Phosphatidic acid mediates demyelination in Lpin1 mutant mice. Genes Dev. 2008;22(12):1647–1661. doi: 10.1101/gad.1638008.</cite> [<a href="https://doi.org/10.1101/gad.1638008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2428062/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18559480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev&amp;title=Phosphatidic%20acid%20mediates%20demyelination%20in%20Lpin1%20mutant%20mice&amp;author=K%20Nadra&amp;volume=22&amp;issue=12&amp;publication_year=2008&amp;pages=1647-1661&amp;pmid=18559480&amp;doi=10.1101/gad.1638008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B19">
<span class="label">19.</span><cite>Arango D, et al.  Acetylation of cytidine in mRNA promotes translation efficiency. Cell. 2018;175(7):1872–1886. doi: 10.1016/j.cell.2018.10.030.</cite> [<a href="https://doi.org/10.1016/j.cell.2018.10.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6295233/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30449621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Acetylation%20of%20cytidine%20in%20mRNA%20promotes%20translation%20efficiency&amp;author=D%20Arango&amp;volume=175&amp;issue=7&amp;publication_year=2018&amp;pages=1872-1886&amp;pmid=30449621&amp;doi=10.1016/j.cell.2018.10.030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B20">
<span class="label">20.</span><cite>Ito S, et al.  Human NAT10 is an ATP-dependent RNA acetyltransferase responsible for N4-acetylcytidine formation in 18 S ribosomal RNA (rRNA) J Biol Chem. 2014;289(52):35724–35730. doi: 10.1074/jbc.C114.602698.</cite> [<a href="https://doi.org/10.1074/jbc.C114.602698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4276842/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25411247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Human%20NAT10%20is%20an%20ATP-dependent%20RNA%20acetyltransferase%20responsible%20for%20N4-acetylcytidine%20formation%20in%2018%20S%20ribosomal%20RNA%20(rRNA)&amp;author=S%20Ito&amp;volume=289&amp;issue=52&amp;publication_year=2014&amp;pages=35724-35730&amp;pmid=25411247&amp;doi=10.1074/jbc.C114.602698&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B21">
<span class="label">21.</span><cite>Guo XF, et al.  Elevation of N-acetyltransferase 10 in hippocampal neurons mediates depression- and anxiety-like behaviors. Brain Res Bull. 2022;185:91–98. doi: 10.1016/j.brainresbull.2022.05.004.</cite> [<a href="https://doi.org/10.1016/j.brainresbull.2022.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35550155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain%20Res%20Bull&amp;title=Elevation%20of%20N-acetyltransferase%2010%20in%20hippocampal%20neurons%20mediates%20depression-%20and%20anxiety-like%20behaviors&amp;author=XF%20Guo&amp;volume=185&amp;publication_year=2022&amp;pages=91-98&amp;pmid=35550155&amp;doi=10.1016/j.brainresbull.2022.05.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B22">
<span class="label">22.</span><cite>Tang SQ, et al.  [Research and development of Lipin family.] Yi Chuan. 2010;32(10):981–993. doi: 10.3724/SP.J.1005.2010.00981.</cite> [<a href="https://doi.org/10.3724/SP.J.1005.2010.00981" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20943485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Yi%20Chuan&amp;title=%5BResearch%20and%20development%20of%20Lipin%20family.%5D&amp;author=SQ%20Tang&amp;volume=32&amp;issue=10&amp;publication_year=2010&amp;pages=981-993&amp;pmid=20943485&amp;doi=10.3724/SP.J.1005.2010.00981&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B23">
<span class="label">23.</span><cite>Paul LK. Developmental malformation of the corpus callosum: a review of typical callosal development and examples of developmental disorders with callosal involvement. J Neurodev Disord. 2011;3(1):3–27. doi: 10.1007/s11689-010-9059-y.</cite> [<a href="https://doi.org/10.1007/s11689-010-9059-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3163989/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21484594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neurodev%20Disord&amp;title=Developmental%20malformation%20of%20the%20corpus%20callosum:%20a%20review%20of%20typical%20callosal%20development%20and%20examples%20of%20developmental%20disorders%20with%20callosal%20involvement&amp;author=LK%20Paul&amp;volume=3&amp;issue=1&amp;publication_year=2011&amp;pages=3-27&amp;pmid=21484594&amp;doi=10.1007/s11689-010-9059-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B24">
<span class="label">24.</span><cite>Pleshinger MJ, et al.  Inhibition of SC4MOL and HSD17B7 shifts cellular sterol composition and promotes oligodendrocyte formation. RSC Chem Biol. 2022;3(1):56–68. doi: 10.1039/D1CB00145K.</cite> [<a href="https://doi.org/10.1039/D1CB00145K" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8729178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35128409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Chem%20Biol&amp;title=Inhibition%20of%20SC4MOL%20and%20HSD17B7%20shifts%20cellular%20sterol%20composition%20and%20promotes%20oligodendrocyte%20formation&amp;author=MJ%20Pleshinger&amp;volume=3&amp;issue=1&amp;publication_year=2022&amp;pages=56-68&amp;pmid=35128409&amp;doi=10.1039/D1CB00145K&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B25">
<span class="label">25.</span><cite>Alpay Savasan Z, et al.  Metabolomic profiling of cerebral palsy brain tissue reveals novel central biomarkers and biochemical pathways associated with the disease: a pilot Study. Metabolites. 2019;9(2):27.  doi: 10.3390/metabo9020027.</cite> [<a href="https://doi.org/10.3390/metabo9020027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6409919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30717353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Metabolites&amp;title=Metabolomic%20profiling%20of%20cerebral%20palsy%20brain%20tissue%20reveals%20novel%20central%20biomarkers%20and%20biochemical%20pathways%20associated%20with%20the%20disease:%20a%20pilot%20Study&amp;author=Z%20Alpay%20Savasan&amp;volume=9&amp;issue=2&amp;publication_year=2019&amp;pages=27&amp;pmid=30717353&amp;doi=10.3390/metabo9020027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B26">
<span class="label">26.</span><cite>Farooqui AA, et al.  Glycerophospholipids in brain: their metabolism, incorporation into membranes, functions, and involvement in neurological disorders. Chem Phys Lipids. 2000;106(1):1–29. doi: 10.1016/S0009-3084(00)00128-6.</cite> [<a href="https://doi.org/10.1016/S0009-3084(00)00128-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10878232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem%20Phys%20Lipids&amp;title=Glycerophospholipids%20in%20brain:%20their%20metabolism,%20incorporation%20into%20membranes,%20functions,%20and%20involvement%20in%20neurological%20disorders&amp;author=AA%20Farooqui&amp;volume=106&amp;issue=1&amp;publication_year=2000&amp;pages=1-29&amp;pmid=10878232&amp;doi=10.1016/S0009-3084(00)00128-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B27">
<span class="label">27.</span><cite>Hermansson M, et al.  Mechanisms of glycerophospholipid homeostasis in mammalian cells. Prog Lipid Res. 2011;50(3):240–257. doi: 10.1016/j.plipres.2011.02.004.</cite> [<a href="https://doi.org/10.1016/j.plipres.2011.02.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21382416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Prog%20Lipid%20Res&amp;title=Mechanisms%20of%20glycerophospholipid%20homeostasis%20in%20mammalian%20cells&amp;author=M%20Hermansson&amp;volume=50&amp;issue=3&amp;publication_year=2011&amp;pages=240-257&amp;pmid=21382416&amp;doi=10.1016/j.plipres.2011.02.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B28">
<span class="label">28.</span><cite>Yang Y, et al.  Phospholipid subcellular localization and dynamics. J Biol Chem. 2018;293(17):6230–6240. doi: 10.1074/jbc.R117.000582.</cite> [<a href="https://doi.org/10.1074/jbc.R117.000582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5925819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29588369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Phospholipid%20subcellular%20localization%20and%20dynamics&amp;author=Y%20Yang&amp;volume=293&amp;issue=17&amp;publication_year=2018&amp;pages=6230-6240&amp;pmid=29588369&amp;doi=10.1074/jbc.R117.000582&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B29">
<span class="label">29.</span><cite>Han GS, et al.  The Saccharomyces cerevisiae Lipin homolog is a Mg2+-dependent phosphatidate phosphatase enzyme. J Biol Chem. 2006;281(14):9210–9218. doi: 10.1074/jbc.M600425200.</cite> [<a href="https://doi.org/10.1074/jbc.M600425200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1424669/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16467296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=The%20Saccharomyces%20cerevisiae%20Lipin%20homolog%20is%20a%20Mg2+-dependent%20phosphatidate%20phosphatase%20enzyme&amp;author=GS%20Han&amp;volume=281&amp;issue=14&amp;publication_year=2006&amp;pages=9210-9218&amp;pmid=16467296&amp;doi=10.1074/jbc.M600425200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B30">
<span class="label">30.</span><cite>Langner CA, et al.  The fatty liver dystrophy (fld) mutation. A new mutant mouse with a developmental abnormality in triglyceride metabolism and associated tissue-specific defects in lipoprotein lipase and hepatic lipase activities. J Biol Chem. 1989;264(14):7994–8003. doi: 10.1016/S0021-9258(18)83141-6.</cite> [<a href="https://doi.org/10.1016/S0021-9258(18)83141-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2722772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=The%20fatty%20liver%20dystrophy%20(fld)%20mutation.%20A%20new%20mutant%20mouse%20with%20a%20developmental%20abnormality%20in%20triglyceride%20metabolism%20and%20associated%20tissue-specific%20defects%20in%20lipoprotein%20lipase%20and%20hepatic%20lipase%20activities&amp;author=CA%20Langner&amp;volume=264&amp;issue=14&amp;publication_year=1989&amp;pages=7994-8003&amp;pmid=2722772&amp;doi=10.1016/S0021-9258(18)83141-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B31">
<span class="label">31.</span><cite>Reue K, et al.  Adipose tissue deficiency, glucose intolerance, and increased atherosclerosis result from mutation in the mouse fatty liver dystrophy (fld) gene. J Lipid Res. 2000;41(7):1067–1076. doi: 10.1016/S0022-2275(20)32011-3.</cite> [<a href="https://doi.org/10.1016/S0022-2275(20)32011-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10884287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Lipid%20Res&amp;title=Adipose%20tissue%20deficiency,%20glucose%20intolerance,%20and%20increased%20atherosclerosis%20result%20from%20mutation%20in%20the%20mouse%20fatty%20liver%20dystrophy%20(fld)%20gene&amp;author=K%20Reue&amp;volume=41&amp;issue=7&amp;publication_year=2000&amp;pages=1067-1076&amp;pmid=10884287&amp;doi=10.1016/S0022-2275(20)32011-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B32">
<span class="label">32.</span><cite>Zhang P, Reue K. Lipin proteins and glycerolipid metabolism: roles at the ER membrane and beyond. Biochim Biophys Acta Biomembr. 2017;1859(9 pt b):1583–1595. doi: 10.1016/j.bbamem.2017.04.007.</cite> [<a href="https://doi.org/10.1016/j.bbamem.2017.04.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5688847/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28411173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta%20Biomembr&amp;title=Lipin%20proteins%20and%20glycerolipid%20metabolism:%20roles%20at%20the%20ER%20membrane%20and%20beyond&amp;author=P%20Zhang&amp;author=K%20Reue&amp;volume=1859&amp;issue=9%20pt%20b&amp;publication_year=2017&amp;pages=1583-1595&amp;pmid=28411173&amp;doi=10.1016/j.bbamem.2017.04.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B33">
<span class="label">33.</span><cite>Paul LK, et al.  Agenesis of the corpus callosum: genetic, developmental and functional aspects of connectivity. Nat Rev Neurosci. 2007;8(4):287–299. doi: 10.1038/nrn2107.</cite> [<a href="https://doi.org/10.1038/nrn2107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17375041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Neurosci&amp;title=Agenesis%20of%20the%20corpus%20callosum:%20genetic,%20developmental%20and%20functional%20aspects%20of%20connectivity&amp;author=LK%20Paul&amp;volume=8&amp;issue=4&amp;publication_year=2007&amp;pages=287-299&amp;pmid=17375041&amp;doi=10.1038/nrn2107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B34">
<span class="label">34.</span><cite>Siffredi V, et al.  Corpus callosum structural characteristics in very preterm children and adolescents: developmental trajectory and relationship to cognitive functioning. Dev Cogn Neurosci. 2023;60:101211.  doi: 10.1016/j.dcn.2023.101211.</cite> [<a href="https://doi.org/10.1016/j.dcn.2023.101211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9925611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36780739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev%20Cogn%20Neurosci&amp;title=Corpus%20callosum%20structural%20characteristics%20in%20very%20preterm%20children%20and%20adolescents:%20developmental%20trajectory%20and%20relationship%20to%20cognitive%20functioning&amp;author=V%20Siffredi&amp;volume=60&amp;publication_year=2023&amp;pages=101211&amp;pmid=36780739&amp;doi=10.1016/j.dcn.2023.101211&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B35">
<span class="label">35.</span><cite>Han GS, et al.  The cellular functions of the yeast lipin homolog PAH1p are dependent on its phosphatidate phosphatase activity. J Biol Chem. 2007;282(51):37026–37035. doi: 10.1074/jbc.M705777200.</cite> [<a href="https://doi.org/10.1074/jbc.M705777200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17971454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=The%20cellular%20functions%20of%20the%20yeast%20lipin%20homolog%20PAH1p%20are%20dependent%20on%20its%20phosphatidate%20phosphatase%20activity&amp;author=GS%20Han&amp;volume=282&amp;issue=51&amp;publication_year=2007&amp;pages=37026-37035&amp;pmid=17971454&amp;doi=10.1074/jbc.M705777200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B36">
<span class="label">36.</span><cite>Yang C, et al.  Rewiring neuronal glycerolipid metabolism determines the extent of axon regeneration. Neuron. 2020;105(2):276–292. doi: 10.1016/j.neuron.2019.10.009.</cite> [<a href="https://doi.org/10.1016/j.neuron.2019.10.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6975164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31786011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Rewiring%20neuronal%20glycerolipid%20metabolism%20determines%20the%20extent%20of%20axon%20regeneration&amp;author=C%20Yang&amp;volume=105&amp;issue=2&amp;publication_year=2020&amp;pages=276-292&amp;pmid=31786011&amp;doi=10.1016/j.neuron.2019.10.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B37">
<span class="label">37.</span><cite>Han X, et al.  Phosphatidate phosphatase Lipin1 involves in diabetic encephalopathy pathogenesis via regulating synaptic mitochondrial dynamics. Redox Biol. 2024;69:102996.  doi: 10.1016/j.redox.2023.102996.</cite> [<a href="https://doi.org/10.1016/j.redox.2023.102996" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10770635/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38103341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Redox%20Biol&amp;title=Phosphatidate%20phosphatase%20Lipin1%20involves%20in%20diabetic%20encephalopathy%20pathogenesis%20via%20regulating%20synaptic%20mitochondrial%20dynamics&amp;author=X%20Han&amp;volume=69&amp;publication_year=2024&amp;pages=102996&amp;pmid=38103341&amp;doi=10.1016/j.redox.2023.102996&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B38">
<span class="label">38.</span><cite>Fitzner D, et al.  Cell-type- and brain-region-resolved mouse brain lipidome. Cell Rep. 2020;32(11):108132.  doi: 10.1016/j.celrep.2020.108132.</cite> [<a href="https://doi.org/10.1016/j.celrep.2020.108132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32937123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Cell-type-%20and%20brain-region-resolved%20mouse%20brain%20lipidome&amp;author=D%20Fitzner&amp;volume=32&amp;issue=11&amp;publication_year=2020&amp;pages=108132&amp;pmid=32937123&amp;doi=10.1016/j.celrep.2020.108132&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B39">
<span class="label">39.</span><cite>Shirai Y, Saito N. Diacylglycerol kinase as a possible therapeutic target for neuronal diseases. J Biomed Sci. 2014;21(1):28.  doi: 10.1186/1423-0127-21-28.</cite> [<a href="https://doi.org/10.1186/1423-0127-21-28" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4005014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24708409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biomed%20Sci&amp;title=Diacylglycerol%20kinase%20as%20a%20possible%20therapeutic%20target%20for%20neuronal%20diseases&amp;author=Y%20Shirai&amp;author=N%20Saito&amp;volume=21&amp;issue=1&amp;publication_year=2014&amp;pages=28&amp;pmid=24708409&amp;doi=10.1186/1423-0127-21-28&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B40">
<span class="label">40.</span><cite>Beirowski B, et al.  Sir-two-homolog 2 (Sirt2) modulates peripheral myelination through polarity protein Par-3/atypical protein kinase C (aPKC) signaling. Proc Natl Acad Sci U S A. 2011;108(43):E952–E961. doi: 10.1073/pnas.1104969108.</cite> [<a href="https://doi.org/10.1073/pnas.1104969108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3203793/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21949390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Sir-two-homolog%202%20(Sirt2)%20modulates%20peripheral%20myelination%20through%20polarity%20protein%20Par-3/atypical%20protein%20kinase%20C%20(aPKC)%20signaling&amp;author=B%20Beirowski&amp;volume=108&amp;issue=43&amp;publication_year=2011&amp;pages=E952-E961&amp;pmid=21949390&amp;doi=10.1073/pnas.1104969108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B41">
<span class="label">41.</span><cite>Musah AS, et al.  Mechanistic target of rapamycin regulates the oligodendrocyte cytoskeleton during myelination. J Neurosci. 2020;40(15):2993–3007. doi: 10.1523/JNEUROSCI.1434-18.2020.</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.1434-18.2020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7141876/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32139584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&amp;title=Mechanistic%20target%20of%20rapamycin%20regulates%20the%20oligodendrocyte%20cytoskeleton%20during%20myelination&amp;author=AS%20Musah&amp;volume=40&amp;issue=15&amp;publication_year=2020&amp;pages=2993-3007&amp;pmid=32139584&amp;doi=10.1523/JNEUROSCI.1434-18.2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B42">
<span class="label">42.</span><cite>Karthiya R, et al.  Emerging role of N4-acetylcytidine modification of RNA in gene regulation and cellular functions. Mol Biol Rep. 2020;47(11):9189–9199. doi: 10.1007/s11033-020-05963-w.</cite> [<a href="https://doi.org/10.1007/s11033-020-05963-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33174082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Rep&amp;title=Emerging%20role%20of%20N4-acetylcytidine%20modification%20of%20RNA%20in%20gene%20regulation%20and%20cellular%20functions&amp;author=R%20Karthiya&amp;volume=47&amp;issue=11&amp;publication_year=2020&amp;pages=9189-9199&amp;pmid=33174082&amp;doi=10.1007/s11033-020-05963-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B43">
<span class="label">43.</span><cite>Sharma S, et al.  Yeast Kre33 and human NAT10 are conserved 18S rRNA cytosine acetyltransferases that modify tRNAs assisted by the adaptor Tan1/THUMPD1. Nucleic Acids Res. 2015;43(4):2242–2258. doi: 10.1093/nar/gkv075.</cite> [<a href="https://doi.org/10.1093/nar/gkv075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4344512/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25653167/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Yeast%20Kre33%20and%20human%20NAT10%20are%20conserved%2018S%20rRNA%20cytosine%20acetyltransferases%20that%20modify%20tRNAs%20assisted%20by%20the%20adaptor%20Tan1/THUMPD1&amp;author=S%20Sharma&amp;volume=43&amp;issue=4&amp;publication_year=2015&amp;pages=2242-2258&amp;pmid=25653167&amp;doi=10.1093/nar/gkv075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B44">
<span class="label">44.</span><cite>Liao H, et al.  Ropinirole suppresses LPS-induced periodontal inflammation by inhibiting the NAT10 in an ac4C-dependent manner. BMC Oral Health. 2024;24(1):510.  doi: 10.1186/s12903-024-04250-5.</cite> [<a href="https://doi.org/10.1186/s12903-024-04250-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11059654/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38689229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Oral%20Health&amp;title=Ropinirole%20suppresses%20LPS-induced%20periodontal%20inflammation%20by%20inhibiting%20the%20NAT10%20in%20an%20ac4C-dependent%20manner&amp;author=H%20Liao&amp;volume=24&amp;issue=1&amp;publication_year=2024&amp;pages=510&amp;pmid=38689229&amp;doi=10.1186/s12903-024-04250-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B45">
<span class="label">45.</span><cite>Pietrocola F, et al.  Acetyl coenzyme A: a central metabolite and second messenger. Cell Metab. 2015;21(6):805–821. doi: 10.1016/j.cmet.2015.05.014.</cite> [<a href="https://doi.org/10.1016/j.cmet.2015.05.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26039447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab&amp;title=Acetyl%20coenzyme%20A:%20a%20central%20metabolite%20and%20second%20messenger&amp;author=F%20Pietrocola&amp;volume=21&amp;issue=6&amp;publication_year=2015&amp;pages=805-821&amp;pmid=26039447&amp;doi=10.1016/j.cmet.2015.05.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B46">
<span class="label">46.</span><cite>Liu X, et al.  Deacetylation of NAT10 by Sirt1 promotes the transition from rRNA biogenesis to autophagy upon energy stress. Nucleic Acids Res. 2018;46(18):9601–9616. doi: 10.1093/nar/gky777.</cite> [<a href="https://doi.org/10.1093/nar/gky777" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6182161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30165671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Deacetylation%20of%20NAT10%20by%20Sirt1%20promotes%20the%20transition%20from%20rRNA%20biogenesis%20to%20autophagy%20upon%20energy%20stress&amp;author=X%20Liu&amp;volume=46&amp;issue=18&amp;publication_year=2018&amp;pages=9601-9616&amp;pmid=30165671&amp;doi=10.1093/nar/gky777&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B47">
<span class="label">47.</span><cite>Wei R, et al.  NAT10 promotes cell proliferation by acetylating CEP170 mRNA to enhance translation efficiency in multiple myeloma. Acta Pharm Sin B. 2022;12(8):3313–3325. doi: 10.1016/j.apsb.2022.01.015.</cite> [<a href="https://doi.org/10.1016/j.apsb.2022.01.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9366180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35967285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Pharm%20Sin%20B&amp;title=NAT10%20promotes%20cell%20proliferation%20by%20acetylating%20CEP170%20mRNA%20to%20enhance%20translation%20efficiency%20in%20multiple%20myeloma&amp;author=R%20Wei&amp;volume=12&amp;issue=8&amp;publication_year=2022&amp;pages=3313-3325&amp;pmid=35967285&amp;doi=10.1016/j.apsb.2022.01.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B48">
<span class="label">48.</span><cite>Zhang Y, et al.  NAT10 promotes gastric cancer metastasis via N4-acetylated COL5A1. Signal Transduct Target Ther. 2021;6(1):173.  doi: 10.1038/s41392-021-00489-4.</cite> [<a href="https://doi.org/10.1038/s41392-021-00489-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8093205/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33941767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduct%20Target%20Ther&amp;title=NAT10%20promotes%20gastric%20cancer%20metastasis%20via%20N4-acetylated%20COL5A1&amp;author=Y%20Zhang&amp;volume=6&amp;issue=1&amp;publication_year=2021&amp;pages=173&amp;pmid=33941767&amp;doi=10.1038/s41392-021-00489-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B49">
<span class="label">49.</span><cite>Wang G, et al.  NAT10-mediated mRNA N4-acetylcytidine modification promotes bladder cancer progression. Clin Transl Med. 2022;12(5):e738.  doi: 10.1002/ctm2.738.</cite> [<a href="https://doi.org/10.1002/ctm2.738" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9076013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35522942/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Med&amp;title=NAT10-mediated%20mRNA%20N4-acetylcytidine%20modification%20promotes%20bladder%20cancer%20progression&amp;author=G%20Wang&amp;volume=12&amp;issue=5&amp;publication_year=2022&amp;pages=e738&amp;pmid=35522942&amp;doi=10.1002/ctm2.738&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B50">
<span class="label">50.</span><cite>Bian X, et al.  Lipid metabolism and cancer. J Exp Med. 2021;218(1):e20201606.  doi: 10.1084/jem.20201606.</cite> [<a href="https://doi.org/10.1084/jem.20201606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7754673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33601415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&amp;title=Lipid%20metabolism%20and%20cancer&amp;author=X%20Bian&amp;volume=218&amp;issue=1&amp;publication_year=2021&amp;pages=e20201606&amp;pmid=33601415&amp;doi=10.1084/jem.20201606&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B51">
<span class="label">51.</span><cite>Dalhat MH, et al.  NAT10: an RNA cytidine transferase regulates fatty acid metabolism in cancer cells. Clin Transl Med. 2022;12(9):e1045.  doi: 10.1002/ctm2.1045.</cite> [<a href="https://doi.org/10.1002/ctm2.1045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9505754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36149760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Med&amp;title=NAT10:%20an%20RNA%20cytidine%20transferase%20regulates%20fatty%20acid%20metabolism%20in%20cancer%20cells&amp;author=MH%20Dalhat&amp;volume=12&amp;issue=9&amp;publication_year=2022&amp;pages=e1045&amp;pmid=36149760&amp;doi=10.1002/ctm2.1045&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B52">
<span class="label">52.</span><cite>Wang K, et al.  PIWI-interacting RNA HAAPIR regulates cardiomyocyte death after myocardial infarction by promoting NAT10-Mediated ac4 C acetylation of Tfec mRNA. Adv Sci (Weinh) 2022;9(8):e2106058.  doi: 10.1002/advs.202106058.</cite> [<a href="https://doi.org/10.1002/advs.202106058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8922123/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35138696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv%20Sci%20(Weinh)&amp;title=PIWI-interacting%20RNA%20HAAPIR%20regulates%20cardiomyocyte%20death%20after%20myocardial%20infarction%20by%20promoting%20NAT10-Mediated%20ac4%20C%20acetylation%20of%20Tfec%20mRNA&amp;author=K%20Wang&amp;volume=9&amp;issue=8&amp;publication_year=2022&amp;pages=e2106058&amp;pmid=35138696&amp;doi=10.1002/advs.202106058&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B53">
<span class="label">53.</span><cite>Parsons CL, et al.  Role of urinary cations in the aetiology of bladder symptoms and interstitial cystitis. BJU Int. 2014;114(2):286–293. doi: 10.1111/bju.12603.</cite> [<a href="https://doi.org/10.1111/bju.12603" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24325253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BJU%20Int&amp;title=Role%20of%20urinary%20cations%20in%20the%20aetiology%20of%20bladder%20symptoms%20and%20interstitial%20cystitis&amp;author=CL%20Parsons&amp;volume=114&amp;issue=2&amp;publication_year=2014&amp;pages=286-293&amp;pmid=24325253&amp;doi=10.1111/bju.12603&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B54">
<span class="label">54.</span><cite>Tsai K, et al.  Acetylation of cytidine residues boosts HIV-1 gene expression by increasing viral RNA stability. Cell Host Microbe. 2020;28(2):306–312. doi: 10.1016/j.chom.2020.05.011.</cite> [<a href="https://doi.org/10.1016/j.chom.2020.05.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7429276/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32533923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20Microbe&amp;title=Acetylation%20of%20cytidine%20residues%20boosts%20HIV-1%20gene%20expression%20by%20increasing%20viral%20RNA%20stability&amp;author=K%20Tsai&amp;volume=28&amp;issue=2&amp;publication_year=2020&amp;pages=306-312&amp;pmid=32533923&amp;doi=10.1016/j.chom.2020.05.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B55">
<span class="label">55.</span><cite>Lo Cicero A, Nissan X. Pluripotent stem cells to model Hutchinson-Gilford progeria syndrome (HGPS): current trends and future perspectives for drug discovery. Ageing Res Rev. 2015;24(pt b):343–348. doi: 10.1016/j.arr.2015.10.002.</cite> [<a href="https://doi.org/10.1016/j.arr.2015.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26474742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ageing%20Res%20Rev&amp;title=Pluripotent%20stem%20cells%20to%20model%20Hutchinson-Gilford%20progeria%20syndrome%20(HGPS):%20current%20trends%20and%20future%20perspectives%20for%20drug%20discovery&amp;author=A%20Lo%20Cicero&amp;author=X%20Nissan&amp;volume=24&amp;issue=pt%20b&amp;publication_year=2015&amp;pages=343-348&amp;pmid=26474742&amp;doi=10.1016/j.arr.2015.10.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B56">
<span class="label">56.</span><cite>Yang W, et al.  ac4C acetylation of RUNX2 catalyzed by NAT10 spurs osteogenesis of BMSCs and prevents ovariectomy-induced bone loss. Mol Ther Nucleic Acids. 2021;26:135–147. doi: 10.1016/j.omtn.2021.06.022.</cite> [<a href="https://doi.org/10.1016/j.omtn.2021.06.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8413676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34513300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Ther%20Nucleic%20Acids&amp;title=ac4C%20acetylation%20of%20RUNX2%20catalyzed%20by%20NAT10%20spurs%20osteogenesis%20of%20BMSCs%20and%20prevents%20ovariectomy-induced%20bone%20loss&amp;author=W%20Yang&amp;volume=26&amp;publication_year=2021&amp;pages=135-147&amp;pmid=34513300&amp;doi=10.1016/j.omtn.2021.06.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B57">
<span class="label">57.</span><cite>Li X, et al.  Gut microbiota modification by diosgenin mediates antiepileptic effects in a mouse model of epilepsy. J Neurochem. 2024;168(12):3982–4000. doi: 10.1111/jnc.16033.</cite> [<a href="https://doi.org/10.1111/jnc.16033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38115597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neurochem&amp;title=Gut%20microbiota%20modification%20by%20diosgenin%20mediates%20antiepileptic%20effects%20in%20a%20mouse%20model%20of%20epilepsy&amp;author=X%20Li&amp;volume=168&amp;issue=12&amp;publication_year=2024&amp;pages=3982-4000&amp;pmid=38115597&amp;doi=10.1111/jnc.16033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Supplemental data</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333955/bin/jciinsight-10-193712-s151.pdf" data-ga-action="click_feat_suppl" class="usa-link">jciinsight-10-193712-s151.pdf</a><sup> (1.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><span>Unedited blot and gel images</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333955/bin/jciinsight-10-193712-s152.pdf" data-ga-action="click_feat_suppl" class="usa-link">jciinsight-10-193712-s152.pdf</a><sup> (510.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><span>Supporting data values</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333955/bin/jciinsight-10-193712-s153.xlsx" data-ga-action="click_feat_suppl" class="usa-link">jciinsight-10-193712-s153.xlsx</a><sup> (104.6KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data supporting the findings of this study are available within the article and its supplemental materials. In addition, all underlying data values for graphs and reported means in both main and <a href="#sd" class="usa-link">Supplemental figures</a> are provided in the <a href="#sd" class="usa-link">Supporting Data Values</a> file, with separate tabs corresponding to each figure panel.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from JCI Insight are provided here courtesy of <strong>American Society for Clinical Investigation</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1172/jci.insight.193712"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/jciinsight-10-193712.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12333955/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12333955/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12333955%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333955/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12333955/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12333955/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40779453/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12333955/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40779453/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12333955/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12333955/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="AI4GHHWBrqw4RROL6Dcz9imd4xpIVnoREIasGxaQpSob74DWl0VqGPA3cCVPjqjz">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
