
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Leveraging unmanned aerial vehicle derived multispectral data for improved genomic prediction in potato (Solanum tuberosum) - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="4DDE470489B70DD31B4704001683578D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12290324/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Leveraging unmanned aerial vehicle derived multispectral data for improved genomic prediction in potato (Solanum tuberosum)">
<meta name="citation_author" content="Muyideen Yusuf">
<meta name="citation_author_institution" content="Department of Horticultural Science, University of Minnesota, Saint Paul, Minnesota, USA">
<meta name="citation_author" content="Xiaoxi Meng">
<meta name="citation_author_institution" content="Department of Horticultural Science, University of Minnesota, Saint Paul, Minnesota, USA">
<meta name="citation_author" content="Thomas R Stefaniak">
<meta name="citation_author_institution" content="Department of Horticultural Science, University of Minnesota, Saint Paul, Minnesota, USA">
<meta name="citation_author" content="Osval A Montesinos‐López">
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, Colima, México">
<meta name="citation_author" content="Laura M Shannon">
<meta name="citation_author_institution" content="Department of Horticultural Science, University of Minnesota, Saint Paul, Minnesota, USA">
<meta name="citation_publication_date" content="2025 Jul 24">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70082">
<meta name="citation_doi" content="10.1002/tpg2.70082">
<meta name="citation_pmid" content="40708258">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12290324/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12290324/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12290324/pdf/TPG2-18-e70082.pdf">
<meta name="description" content="Multispectral leaf canopy reflectance as measured by unmanned aerial vehicles is the result of genetic and environmental interactions driving plant physiochemical processes. These measures can then be used to construct relationship matrices for ...">
<meta name="og:title" content="Leveraging unmanned aerial vehicle derived multispectral data for improved genomic prediction in potato (Solanum tuberosum)">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Multispectral leaf canopy reflectance as measured by unmanned aerial vehicles is the result of genetic and environmental interactions driving plant physiochemical processes. These measures can then be used to construct relationship matrices for ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12290324/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12290324">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70082"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70082.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12290324%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12290324/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12290324/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12290324/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70082" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Jul 24;18(3):e70082. doi: <a href="https://doi.org/10.1002/tpg2.70082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70082</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Leveraging unmanned aerial vehicle derived multispectral data for improved genomic prediction in potato (<em>Solanum tuberosum</em>)</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yusuf%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Muyideen Yusuf</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Muyideen Yusuf</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Horticultural Science, University of Minnesota, Saint Paul, Minnesota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yusuf%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Muyideen Yusuf</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meng%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Xiaoxi Meng</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Xiaoxi Meng</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Horticultural Science, University of Minnesota, Saint Paul, Minnesota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meng%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaoxi Meng</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stefaniak%20TR%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Thomas R Stefaniak</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Thomas R Stefaniak</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Horticultural Science, University of Minnesota, Saint Paul, Minnesota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stefaniak%20TR%22%5BAuthor%5D" class="usa-link"><span class="name western">Thomas R Stefaniak</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Montesinos%E2%80%90L%C3%B3pez%20OA%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Osval A Montesinos‐López</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Osval A Montesinos‐López</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Facultad de Telemática, Universidad de Colima, Colima, México</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Montesinos%E2%80%90L%C3%B3pez%20OA%22%5BAuthor%5D" class="usa-link"><span class="name western">Osval A Montesinos‐López</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shannon%20LM%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Laura M Shannon</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Laura M Shannon</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Horticultural Science, University of Minnesota, Saint Paul, Minnesota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shannon%20LM%22%5BAuthor%5D" class="usa-link"><span class="name western">Laura M Shannon</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270082-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Horticultural Science, University of Minnesota, Saint Paul, Minnesota, USA</div>
<div id="tpg270082-aff-0002">
<sup>
<sup>2</sup>
</sup>Facultad de Telemática, Universidad de Colima, Colima, México</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Laura M. Shannon, Department of Horticultural Science, University of Minnesota, Saint Paul, MN 55108, USA. Email: <span>lmshannon@umn.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 2; Received 2024 Oct 29; Accepted 2025 Jun 9; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12290324  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40708258/" class="usa-link">40708258</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Multispectral leaf canopy reflectance as measured by unmanned aerial vehicles is the result of genetic and environmental interactions driving plant physiochemical processes. These measures can then be used to construct relationship matrices for modeling genetic main effects. This type of phenotypic prediction is particularly relevant for trials with many entries, such as those used in early generation potato (<em>Solanum tuberosum</em>) breeding. We compared three methods for making predictions in our potato breeding program: first, using multispectral‐derived relationship matrices; second, using the traditional approach based on genomic derived relationships; and third, using a combination of both. Multispectral bands were collected at five different time points for two market classes of potato: chipping and fresh market. We modeled genetic main effects for yield and quality traits at each time point and all stages combined. Models with multispectral relationship matrices exhibited better prediction accuracy for yield and roundness than genomic only models and models featuring spectra plus genomic kernels outperformed both single‐kernel predictions in terms of accuracy for most traits. Time points were variably informative depending on the trait measured, however, for all traits combining across time points performed as well or better than single time point models. Similarly, using feature selection to limit our models to important variables did not improve prediction accuracy significantly. This work highlights two potential uses for spectral data in genomic prediction: first, as an alternative to genetic data and second, in combination with genetic data to increase precision of selection.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270082-list-10001" class="list" style="list-style-type:disc">
<li><p>Using multispectral drone imagery generates more accurate potato yield predictions for genomic selection.</p></li>
<li><p>Prediction of yield, shape, skin color, and specific gravity is improved by combining phenomic and genomic data.</p></li>
<li><p>Combining spectral data across time points generates the best predictions for all traits.</p></li>
<li><p>Limiting the models to key variables does not improve prediction as compared to models including all the data.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270082-dl-0001">
<dt>ANOVA</dt>
<dd><p>analysis of variance</p></dd>
<dt>BLUEs</dt>
<dd><p>best linear unbiased estimates</p></dd>
<dt>BLUPs</dt>
<dd><p>best linear unbiased predictions</p></dd>
<dt>GLMNET</dt>
<dd><p>elastic‐net regularized general linear models</p></dd>
<dt>GS</dt>
<dd><p>genomic selection</p></dd>
<dt>HTP</dt>
<dd><p>high‐throughput phenotyping</p></dd>
<dt>l/w</dt>
<dd><p>length–width ratio</p></dd>
<dt>LSD</dt>
<dd><p>least significant difference</p></dd>
<dt>MAS</dt>
<dd><p>marker‐assisted selection</p></dd>
<dt>NIR</dt>
<dd><p>near‐infrared</p></dd>
<dt>SA</dt>
<dd><p>simulation annealing</p></dd>
<dt>SF</dt>
<dd><p>selection by filtering</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
<dt>UAV</dt>
<dd><p>unmanned aerial vehicle</p></dd>
<dt>UMN</dt>
<dd><p>University of Minnesota</p></dd>
</dl>
<section id="tpg270082-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Cultivated potato (<em>Solanum tuberosum</em> L.) is the most consumed nongrain crop in the world (FAO, <a href="#tpg270082-bib-0016" class="usa-link" aria-describedby="tpg270082-bib-0016"><sup>2024</sup></a>). However, breeding progress, particularly for yield, has been slow in comparison to other staple crops (Douches et al., <a href="#tpg270082-bib-0012" class="usa-link" aria-describedby="tpg270082-bib-0012">1996</a>; Jansky, <a href="#tpg270082-bib-0028" class="usa-link" aria-describedby="tpg270082-bib-0028"><sup>2009</sup></a>). This is in part due to US cultivated potato's clonal nature and its highly heterozygous (Hardigan et al., <a href="#tpg270082-bib-0024" class="usa-link" aria-describedby="tpg270082-bib-0024">2017</a>; Hoopes et al., <a href="#tpg270082-bib-0027" class="usa-link" aria-describedby="tpg270082-bib-0027"><sup>2022</sup></a>; Tuttle et al., <a href="#tpg270082-bib-0065" class="usa-link" aria-describedby="tpg270082-bib-0065"><sup>2024</sup></a>) autotetraploid genome. Historically, potatoes have been bred using F1 crosses followed by approximately a decade of winnowing through selection to identify potential varieties (Bradshaw, <a href="#tpg270082-bib-0006" class="usa-link" aria-describedby="tpg270082-bib-0006">2017</a>; Jansky, <a href="#tpg270082-bib-0028" class="usa-link" aria-describedby="tpg270082-bib-0028"><sup>2009</sup></a>). In many programs, F1 seed is generated in greenhouses and then ramped up in small seed increase plots for 2 years before there is sufficient seed for a preliminary yield trial. When breeders rely on traditional phenotypic selection, this results in 5–7 year breeding cycles.</p>
<p>Marker‐assisted selection (MAS) can speed the breeding process (Ortiz, <a href="#tpg270082-bib-0048" class="usa-link" aria-describedby="tpg270082-bib-0048">2020</a>; Slater et al., <a href="#tpg270082-bib-0059" class="usa-link" aria-describedby="tpg270082-bib-0059"><sup>2016</sup></a>). Genetic markers applied within the first or second field year offer greater efficiency over traditional visual screening methods (Bradshaw, <a href="#tpg270082-bib-0006" class="usa-link" aria-describedby="tpg270082-bib-0006">2017</a>). While MAS can be highly efficient, its effectiveness is generally limited to traits with simple genetic architecture and few large‐effect genes, such as self‐compatibility (Clot et al., <a href="#tpg270082-bib-0008" class="usa-link" aria-describedby="tpg270082-bib-0008">2020</a>; Kaiser et al., <a href="#tpg270082-bib-0034" class="usa-link" aria-describedby="tpg270082-bib-0034"><sup>2021</sup></a>), maturity (Gutaker et al., <a href="#tpg270082-bib-0022" class="usa-link" aria-describedby="tpg270082-bib-0022">2019</a>; Kloosterman et al., <a href="#tpg270082-bib-0035" class="usa-link" aria-describedby="tpg270082-bib-0035"><sup>2013</sup></a>), oblong versus circular shape (Endelman et al., <a href="#tpg270082-bib-0015" class="usa-link" aria-describedby="tpg270082-bib-0015">2024</a>; van Eck et al., <a href="#tpg270082-bib-0066" class="usa-link" aria-describedby="tpg270082-bib-0066"><sup>2022</sup></a>), and disease resistance (Asano &amp; Endelman, <a href="#tpg270082-bib-0005" class="usa-link" aria-describedby="tpg270082-bib-0005">2023</a>; Herrera et al., <a href="#tpg270082-bib-0025" class="usa-link" aria-describedby="tpg270082-bib-0025"><sup>2018</sup></a>; Meade et al., <a href="#tpg270082-bib-0040" class="usa-link" aria-describedby="tpg270082-bib-0040"><sup>2020</sup></a>; Nie et al., <a href="#tpg270082-bib-0047" class="usa-link" aria-describedby="tpg270082-bib-0047"><sup>2016</sup></a>; Prodhomme et al., <a href="#tpg270082-bib-0051" class="usa-link" aria-describedby="tpg270082-bib-0051"><sup>2019</sup></a>).</p>
<p>For complex traits genomic selection (GS) can be effective at increasing genetic gain and shortening breeding cycles (Endelman et al., <a href="#tpg270082-bib-0014" class="usa-link" aria-describedby="tpg270082-bib-0014">2018</a>; Gemenet et al., <a href="#tpg270082-bib-0021" class="usa-link" aria-describedby="tpg270082-bib-0021"><sup>2020</sup></a>; Habyarimana et al., <a href="#tpg270082-bib-0023" class="usa-link" aria-describedby="tpg270082-bib-0023"><sup>2017</sup></a>; Martins et al., <a href="#tpg270082-bib-0039" class="usa-link" aria-describedby="tpg270082-bib-0039"><sup>2023</sup></a>; Slater et al., <a href="#tpg270082-bib-0059" class="usa-link" aria-describedby="tpg270082-bib-0059"><sup>2016</sup></a>; Sood et al., <a href="#tpg270082-bib-0061" class="usa-link" aria-describedby="tpg270082-bib-0061"><sup>2020</sup></a>; Stich &amp; Van Inghelandt, <a href="#tpg270082-bib-0063" class="usa-link" aria-describedby="tpg270082-bib-0063"><sup>2018</sup></a>; Yusuf et al., <a href="#tpg270082-bib-0075" class="usa-link" aria-describedby="tpg270082-bib-0075"><sup>2024</sup></a>). GS estimates genome‐wide marker effects for any given trait using individuals that have been both genotyped and phenotyped within a training population (Meuwissen et al., <a href="#tpg270082-bib-0041" class="usa-link" aria-describedby="tpg270082-bib-0041">2001</a>). However, even with decreasing sequencing costs, genotyping the tens of thousands of unique genetic individuals in early generations of potato breeding programs is expensive. Moreover, the successful implementation of GS depends on precise and accurate quantitative phenotypes (Yusuf et al., <a href="#tpg270082-bib-0075" class="usa-link" aria-describedby="tpg270082-bib-0075">2024</a>), which remains a bottleneck, especially at the early testing stages of the breeding pipeline.</p>
<p>Advances in high‐throughput phenotyping (HTP) are expected to alleviate some of these challenges, enhancing the efficiency and accuracy of GS (Hickey et al., <a href="#tpg270082-bib-0026" class="usa-link" aria-describedby="tpg270082-bib-0026">2019</a>; Slater et al., <a href="#tpg270082-bib-0059" class="usa-link" aria-describedby="tpg270082-bib-0059"><sup>2016</sup></a>). HTP can enhance morphological trait scoring in plant breeding, accelerate the phenotyping processes, and increase precision (Hickey et al., <a href="#tpg270082-bib-0026" class="usa-link" aria-describedby="tpg270082-bib-0026">2019</a>; Shakoor et al., <a href="#tpg270082-bib-0056" class="usa-link" aria-describedby="tpg270082-bib-0056"><sup>2017</sup></a>). Image‐based phenotyping of tubers has been implemented to develop quantitative measures for quality traits, such as shape and color, in potatoes (Caraza‐Harter &amp; Endelman, <a href="#tpg270082-bib-0007" class="usa-link" aria-describedby="tpg270082-bib-0007">2020</a>; Feldman et al., <a href="#tpg270082-bib-0018" class="usa-link" aria-describedby="tpg270082-bib-0018"><sup>2024</sup></a>; Miller et al., <a href="#tpg270082-bib-0042" class="usa-link" aria-describedby="tpg270082-bib-0042"><sup>2022</sup></a>; Neilson et al., <a href="#tpg270082-bib-0046" class="usa-link" aria-describedby="tpg270082-bib-0046"><sup>2021</sup></a>). The phenotypic data derived from these images allows for genomic prediction of quality traits (Yusuf et al., <a href="#tpg270082-bib-0075" class="usa-link" aria-describedby="tpg270082-bib-0075">2024</a>).</p>
<p>Advances in sensor technology, machine learning, and data analytics are further enhancing the efficiency and accuracy of HTP in plant breeding (Hickey et al., <a href="#tpg270082-bib-0026" class="usa-link" aria-describedby="tpg270082-bib-0026">2019</a>). Both ground‐based and aerial HTP platforms have been deployed for phenotyping plants, either by directly capturing specific traits or indirectly examining phenotypic differences among plants using spectral reflectance data (Song et al., <a href="#tpg270082-bib-0060" class="usa-link" aria-describedby="tpg270082-bib-0060">2021</a>). Aerial phenotyping offers the significant advantage of covering a wider expanse of field trials within a short timeframe. One such aerial platform is the unmanned aerial vehicle (UAV), which is equipped with multi‐sensor cameras capable of capturing wavelengths near and beyond the visible spectrum. These wavelengths reflect different physiochemical properties related to plant structure, health, and nutrient status based on canopy leaf absorption and reflectance (Alkhaled et al., <a href="#tpg270082-bib-0004" class="usa-link" aria-describedby="tpg270082-bib-0004">2023</a>). Studies in several crops have reported significant improvements in prediction accuracy using HTP temporal data, both for direct predictions and within GS models (Abdelhakim et al., <a href="#tpg270082-bib-0001" class="usa-link" aria-describedby="tpg270082-bib-0001">2024</a>; Montesinos‐López et al., <a href="#tpg270082-bib-0043" class="usa-link" aria-describedby="tpg270082-bib-0043"><sup>2023</sup></a>; Silva et al., <a href="#tpg270082-bib-0057" class="usa-link" aria-describedby="tpg270082-bib-0057"><sup>2022</sup></a>; Xu et al., <a href="#tpg270082-bib-0071" class="usa-link" aria-describedby="tpg270082-bib-0071"><sup>2023</sup></a>).</p>
<p>Despite its benefits, the use of temporal HTP data presents challenges due to its large volume and high dimensionality, which can complicate interpretation and increase computational demands when used directly for prediction. An efficient method for GS involves using the genomic best linear unbiased prediction model, where relationship matrices constructed from marker data among individuals are used as covariance in a GS model to estimate genomic estimated breeding values (VanRaden, <a href="#tpg270082-bib-0067" class="usa-link" aria-describedby="tpg270082-bib-0067">2008</a>). This approach can be extended to incorporate multispectral data in a similar manner. Implementing a phenomic relationship matrix, either as a predictor or in combination with GS, improves predictions in multiple crops (Krause et al., <a href="#tpg270082-bib-0036" class="usa-link" aria-describedby="tpg270082-bib-0036">2019</a>; Maggiorelli et al., <a href="#tpg270082-bib-0038" class="usa-link" aria-describedby="tpg270082-bib-0038"><sup>2024</sup></a>; Mróz et al., <a href="#tpg270082-bib-0045" class="usa-link" aria-describedby="tpg270082-bib-0045"><sup>2024</sup></a>).</p>
<p>We aimed to apply this combined phenomic and GS method to our breeding program and extend it to predict quality traits as well as yield. Specifically, we address the following questions: (1) Can UAV‐derived multispectral data predict yield, specific gravity, and shape in chipping potatoes and yield, shape, and skin color in fresh market potatoes? (2) How do those predictions compare with genomic‐based predictions? (3) Can multispectral and genomic data be used in combination for prediction? (4) What time point is most appropriate for collecting multispectral data for phenomic prediction? (4) Which combination of spectral variables is most predictive?</p></section><section id="tpg270082-sec-0020"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270082-sec-0030"><h3 class="pmc_sec_title">2.1. Plant materials</h3>
<p>We evaluated two sets of advanced potato breeding clones in 2022 and 2023 at the Sand Plains Research Farm in Becker, MN. The first set included 226 unique chipping clones, while the second contained 87 unique fresh market yellow‐ and red‐skinned clones. In addition to clones from the University of Minnesota (UMN) breeding program, the chips set included 10 breeding clones from the University of Wisconsin Potato Breeding Program and five commonly grown varieties, namely, Atlantic (Webb et al., <a href="#tpg270082-bib-0069" class="usa-link" aria-describedby="tpg270082-bib-0069">1978</a>), Superior (Rieman, <a href="#tpg270082-bib-0053" class="usa-link" aria-describedby="tpg270082-bib-0053">1962</a>), Lamoka (De Jong et al., <a href="#tpg270082-bib-0010" class="usa-link" aria-describedby="tpg270082-bib-0010">2017</a>), Snowden, and Cascade. The fresh market clones include the UMN breeding program material and four commonly grown varieties, namely, Red Norland (Johansen et al., <a href="#tpg270082-bib-0031" class="usa-link" aria-describedby="tpg270082-bib-0031">1959</a>), Modoc, Chieftain (Weigle et al., <a href="#tpg270082-bib-0070" class="usa-link" aria-describedby="tpg270082-bib-0070">1968</a>), and Yukon Gold (Johnston &amp; Rowberry, <a href="#tpg270082-bib-0032" class="usa-link" aria-describedby="tpg270082-bib-0032">1981</a>). Clones were evaluated in an augmented block design with row/column information and repeated checks randomly distributed across blocks for each market class and years. Checks were replicated at least twice within each block, while clones were replicated once or twice for both chipping and fresh market clones in each year. All plots contained 15 seed pieces with 30 cm in‐row spacing and 91 cm between rows.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270082-sec-0040"><h4 class="pmc_sec_title">Core Ideas</h4>
<ul id="tpg270082-list-0001" class="list" style="list-style-type:disc">
<li><p>Using multispectral drone imagery generates more accurate potato yield predictions for genomic selection.</p></li>
<li><p>Prediction of yield, shape, skin color, and specific gravity is improved by combining phenomic and genomic data.</p></li>
<li><p>Combining spectral data across time points generates the best predictions for all traits.</p></li>
<li><p>Limiting the models to key variables does not improve prediction as compared to models including all the data.</p></li>
</ul></section></section></section><section id="tpg270082-sec-0050"><h3 class="pmc_sec_title">2.2. Phenotyping</h3>
<p>We collected 2 years of phenotype data (Table <a href="#tpg270082-tbl-0001" class="usa-link">1</a>). Yield, specific gravity, and roundness were collected for the chips, while yield, redness, lightness, and length–width ratio (l/w) were collected for the fresh market clones. Yield is reported as the total plot weight for all harvested tubers in Mg ha<sup>−1</sup>. The yield was measured on a tuber basis and summed over plots using an AccuVision Dual View LED Grader System from Exeter. Specific gravity was determined by weighing a 10‐tuber sample from each plot in the air and then weighing it again when submerged in water. The specific gravity was then calculated by dividing the weight in air by the difference between the weight in air and the weight in water.</p>
<section class="tw xbox font-sm" id="tpg270082-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Timing of drone flights and number of clones evaluated each year.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Market class</th>
<th align="left" rowspan="1" colspan="1">Evaluation year</th>
<th align="left" rowspan="1" colspan="1">Total unique entries</th>
<th align="left" rowspan="1" colspan="1">Common clones across years</th>
<th align="left" rowspan="1" colspan="1">Estimated average growth stages/drone flight dates</th>
<th align="left" rowspan="1" colspan="1">Days after planting</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Chipping</td>
<td align="left" rowspan="1" colspan="1">2022</td>
<td align="left" rowspan="1" colspan="1">42</td>
<td rowspan="2" align="left" colspan="1">31</td>
<td align="left" rowspan="1" colspan="1">
<p>Canopy cover 50%: June 28</p>
<p>Canopy cover flower: July 5</p>
<p>Peak flowering: July 12</p>
<p>Vine biomass: July 19</p>
<p>Senescence: August 5</p>
</td>
<td align="left" rowspan="1" colspan="1">
<p>36</p>
<p>45</p>
<p>52</p>
<p>59</p>
<p>76</p>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">2023</td>
<td align="left" rowspan="1" colspan="1">215</td>
<td align="left" rowspan="1" colspan="1">
<p>Canopy cover 50%: June 20</p>
<p>Canopy cover flower: July 5</p>
<p>Peak flowering: July 13</p>
<p>Vine biomass: July 18</p>
<p>Senescence: August 8</p>
</td>
<td align="left" rowspan="1" colspan="1">
<p>30</p>
<p>45</p>
<p>53</p>
<p>58</p>
<p>79</p>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Fresh market</td>
<td align="left" rowspan="1" colspan="1">2022</td>
<td align="left" rowspan="1" colspan="1">25</td>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">
<p>Canopy cover 50%: June 28</p>
<p>Canopy cover flower: July 5</p>
<p>Peak flowering: July 12</p>
<p>Vine biomass: July 19</p>
<p>Senescence: August 5</p>
</td>
<td align="left" rowspan="1" colspan="1">
<p>36</p>
<p>45</p>
<p>52</p>
<p>59</p>
<p>76</p>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">2023</td>
<td align="left" rowspan="1" colspan="1">77</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">
<p>Canopy cover 50%: June 20</p>
<p>Canopy cover flower: July 5</p>
<p>Peak flowering: July 13</p>
<p>Vine biomass: July 18</p>
<p>Senescence: August 8</p>
</td>
<td align="left" rowspan="1" colspan="1">
<p>30</p>
<p>45</p>
<p>53</p>
<p>58</p>
<p>79</p>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270082-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>All other traits were measured using image analysis. Images of samples of 10 harvested and washed tubers from each plot were taken using an Ortery Photosimilie 200 Lightbox with a Canon Rebel T6i camera following the methods from Caraza‐Harter and Endelman (<a href="#tpg270082-bib-0007" class="usa-link" aria-describedby="tpg270082-bib-0007">2020</a>). Trait measurement was performed using the potato image analysis R package, TubAR (Miller et al., <a href="#tpg270082-bib-0042" class="usa-link" aria-describedby="tpg270082-bib-0042">2022</a>). Note that l/w and roundness were calculated for each tuber, and the median value was reported. Tuber skin color was quantified in terms of redness and lightness. The median tuber value for each image is used for analysis.</p></section><section id="tpg270082-sec-0060"><h3 class="pmc_sec_title">2.3. Multispectral data</h3>
<p>Multispectral reflectance data were captured in the summers of 2022 and 2023 for five different time points using a UAV with a Sentera 6x multispectral camera and 6x thermal sensors (Sentera). The camera is equipped with high radiometric calibration tools for correction of lighting variation and to provide a consistent reflectance surface spanning both visible and near‐infrared spectral bands. Raw multispectral reflectance mean value was obtained per plot by Sentera from image preprocessing and radiometric calibration (<a href="https://support.sentera.com/portal/en/kb/articles/capturing-your-refelectance-panel" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://support.sentera.com/portal/en/kb/articles/capturing‐your‐refelectance‐panel</a>) for five separate color bands (red = 650 nm, green = 545 nm, blue = 445 nm, near‐infrared [NIR] = 840 nm, and RedEdge = 720 nm). Using the raw spectral bands, eight different vegetation indices commonly used in potatoes (Alkhaled et al., <a href="#tpg270082-bib-0004" class="usa-link" aria-describedby="tpg270082-bib-0004">2023</a>) were estimated. Normalized difference vegetation index (Rouse et al., <a href="#tpg270082-bib-0078" class="usa-link" aria-describedby="tpg270082-bib-0078">1974</a>), normalized difference red edge (Gitelson &amp; Merzlyak, <a href="#tpg270082-bib-0079" class="usa-link" aria-describedby="tpg270082-bib-0079"><sup>1994a</sup></a>), green leaf index, chlorophyll index green, green normalized difference vegetation index (Gitelson &amp; Merzlyak, <a href="#tpg270082-bib-0080" class="usa-link" aria-describedby="tpg270082-bib-0080"><sup>1994b</sup></a>), normalized difference water index (McFeeters, <a href="#tpg270082-bib-0081" class="usa-link" aria-describedby="tpg270082-bib-0081">1994</a>), chlorophyll index red edge (Gitelson et al., <a href="#tpg270082-bib-0082" class="usa-link" aria-describedby="tpg270082-bib-0082">2012</a>), and transformed chlorophyll absorption in reflectance index/optimized soil‐adjusted vegetation index (Haboudane et al., <a href="#tpg270082-bib-0083" class="usa-link" aria-describedby="tpg270082-bib-0083">2010</a>).</p>
<p>Five flight dates were used each summer based on estimated average phenological growth stages translating to canopy cover at 50%, canopy cover closure, peak flowering, peak flowering/biomass, and senescence (Table <a href="#tpg270082-tbl-0001" class="usa-link">1</a>). The timing of phenological stage varies by genotype and market class, so an estimate was made for the whole field.</p></section><section id="tpg270082-sec-0070"><h3 class="pmc_sec_title">2.4. Genotyping data</h3>
<p>Clones were genotyped with a combination of platforms as described in Yusuf et al. (<a href="#tpg270082-bib-0075" class="usa-link" aria-describedby="tpg270082-bib-0075">2024</a>). Specifically, both the SolCAP potato single nucleotide polymorphism (SNP) array (Felcher et al., <a href="#tpg270082-bib-0017" class="usa-link" aria-describedby="tpg270082-bib-0017">2012</a>; Vos et al., <a href="#tpg270082-bib-0068" class="usa-link" aria-describedby="tpg270082-bib-0068"><sup>2015</sup></a>) and FlexSeq (targeted genotyping by sequencing) genotyping from LGC Rapid Genomics. The FlexSeq was developed using 8K markers from the SolCAP array to maintain backward compatibility with existing markers as described for the DArTag genotyping platform in Endelman et al. (<a href="#tpg270082-bib-0015" class="usa-link" aria-describedby="tpg270082-bib-0015">2024</a>). A total of 181 chipping clones and 57 fresh market clones were genotyped using FlexSeq, while 45 chippers and 30 fresh market clones were genotyped with the SNP array panels. The two genotypic datasets were merged, and the FlexSeq markers were imputed to high density (21k) using the <em>L2H</em> function of the R package polyBreedR (Endelman et al., <a href="#tpg270082-bib-0015" class="usa-link" aria-describedby="tpg270082-bib-0015">2024</a>). The merging and imputation of all the datasets resulted in a total of 21,531 markers for the fresh market and 21,210 for chips, which we used to carry out further analysis.</p></section><section id="tpg270082-sec-0080"><h3 class="pmc_sec_title">2.5. Statistical analysis and prediction models</h3>
<section id="tpg270082-sec-0090"><h4 class="pmc_sec_title">2.5.1. Phenotypic data analysis</h4>
<p>Within each year and market class, we accounted for spatial variation and computed best linear unbiased estimates (BLUEs) for all traits and for all multispectral data using a spatial 2D spline model, implemented with SpATS (Rodríguez‐Álvarez et al., <a href="#tpg270082-bib-0054" class="usa-link" aria-describedby="tpg270082-bib-0054">2018</a>), within the StageWise package (Endelman, <a href="#tpg270082-bib-0013" class="usa-link" aria-describedby="tpg270082-bib-0013">2023</a>) in R (R Core Team, <a href="#tpg270082-bib-0052" class="usa-link" aria-describedby="tpg270082-bib-0052"><sup>2022</sup></a>) based on row‐column information. Normality was assessed visually based on <em>Q–Q</em>‐plots produced by StageWise. Splines and residuals estimated from the checks (replicated clones) were used to adjust the variance‐covariance of the unreplicated clones to obtain adjusted means (BLUEs) computed based on the following model:
</p>
<table class="disp-formula p" id="tpg270082-disp-0001"><tr>
<td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi>u</mi><mo linebreak="goodbreak">+</mo><msub><mi>g</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>r</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>c</mi><mi>k</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi></mrow></msub><mo>,</mo></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where <em>y<sub>ijk</sub>
</em> is the response variable for the <em>i</em>th genotype in <em>j</em>th row and <em>k</em>th column, <em>μ</em> is the overall mean, <em>g<sub>i</sub>
</em> is the fixed effect for the genotypic response, <em>r<sub>j</sub>
</em> is the random row effect where <em>r<sub>j</sub>
</em> ∼ <em>N</em> (0, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>r</mi><mn>2</mn></msubsup></mrow></math></span>), <em>c<sub>k</sub>
</em> is the random column effect where <em>c<sub>j</sub>
</em> ∼ <em>N</em> (0, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>c</mi><mn>2</mn></msubsup></mrow></math></span>), and <em>e<sub>ijk</sub>
</em> is the residual effect where <em>e<sub>ijk</sub>
</em> ∼ <em>N</em> (0, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow></math></span>).</p>
<p>Broad‐sense heritability (<em>H</em>
<sup>2</sup>) was computed on a plot basis within each year and market class for all traits and for all multispectral data at each time point with genotype as random from Equation (<a href="#tpg270082-disp-0001" class="usa-link">1</a>). This was computed based on genetic <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-5" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup></mrow></math></span> and residual <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup><mspace width="0.28em"></mspace></mrow></mrow></math></span> variance component as:
</p>
<table class="disp-formula p" id="tpg270082-disp-0002"><tr>
<td class="formula"><math id="jats-math-7" display="block"><mrow><mrow><msubsup><mi>H</mi><mi>g</mi><mn>2</mn></msubsup><mo linebreak="badbreak">=</mo><mfrac><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup><mo>+</mo><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow></mfrac><mo>.</mo></mrow></mrow></math></td>
<td class="label">(2)</td>
</tr></table></section><section id="tpg270082-sec-0100"><h4 class="pmc_sec_title">2.5.2. Relationship matrices</h4>
<p>To explore the effect of different data types and data collection dates, we generated a series of relationship matrices. Genomic relationship matrices (<strong>G</strong>) were first constructed as implemented in the StageWise package (Endelman, <a href="#tpg270082-bib-0013" class="usa-link" aria-describedby="tpg270082-bib-0013">2023</a>). A separate matrix was generated for each market class. Multispectral‐derived relationship matrices (<strong>W</strong>) were also computed in base R, across clone‐year using the BLUEs for each spectra band and a combination of both spectra bands and indices for each of the time points individually and all time points combined. Once again, each market class was treated separately. The matrices were computed as follows:
</p>
<table class="disp-formula p" id="tpg270082-disp-0003"><tr>
<td class="formula"><math id="jats-math-8" display="block"><mrow><mrow><mi>W</mi><mo linebreak="badbreak">=</mo><mfrac><mrow><mi>Z</mi><msup><mi>Z</mi><mi>T</mi></msup></mrow><mi>n</mi></mfrac></mrow></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><mi>n</mi></mrow></math></span> is the number of bands or bands plus indices included in the model <strong>
<em>Z</em>
</strong> is the scaled matrix that corresponds to the BLUEs of the multispectral band or indices and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><mrow><mi>ZT</mi><mspace width="0.28em"></mspace></mrow></mrow></math></span> is the transpose of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline"><mrow><mi>Z</mi></mrow></math></span>.</p></section><section id="tpg270082-sec-0110"><h4 class="pmc_sec_title">2.5.3. Prediction models</h4>
<p>The BLUEs for each trait, within each market class, were used as the response to fit both a genomic linear unbiased predictor model and multispectral linear unbiased predictor model using the BGLR (P. Pérez &amp; de los Campos, <a href="#tpg270082-bib-0050" class="usa-link" aria-describedby="tpg270082-bib-0050"><sup>2014</sup></a>) package in R (R Core Team, <a href="#tpg270082-bib-0022" class="usa-link" aria-describedby="tpg270082-bib-0022"><sup>2023</sup></a>). The genomic prediction model was used as a base model for comparison with the multispectral prediction models.</p>
<p>The genomic prediction model was fit with <strong>G</strong> as covariance:
</p>
<table class="disp-formula p" id="tpg270082-disp-0004"><tr>
<td class="formula"><math id="jats-math-12" display="block"><mrow><mrow><mi>BLUE</mi><mfenced separators="" open="[" close="]"><msub><mi>g</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mfenced><mo linebreak="badbreak">=</mo><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">=</mo><msub><mi>E</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>g</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>s</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>,</mo></mrow></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<p>where <em>g<sub>ij</sub>
</em> is the genotypic value for individual <em>i</em> in year <em>j</em>, <em>E<sub>j</sub>
</em> is the fixed effect for year <em>j</em>, <em>g<sub>i</sub>
</em> is the random effect for genotype <em>i</em>, and <em>s<sub>ij</sub>
</em> is the residual effect with normal distribution with mean 0 and variance <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline"><mrow><msup><mi mathvariant="normal">σ</mi><mn>2</mn></msup></mrow></math></span>. The genetic effect <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-14" display="inline"><mrow><mrow><mi>g</mi><mo>=</mo><msup><mrow><mo stretchy="false">(</mo><mrow><msub><mi>g</mi><mn>1</mn></msub><mo>,</mo><mtext>…</mtext><mo>,</mo><msub><mi>g</mi><mi>I</mi></msub></mrow><mo stretchy="false">)</mo></mrow><mi>T</mi></msup></mrow></mrow></math></span> follows a normal distribution with <strong>g</strong> ∼ <em>N</em> (0, <strong>G</strong>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-15" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup></mrow></math></span>), where <strong>G</strong> is the genomic relationship matrix and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-16" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup></mrow></math></span> is the genomic variance. This model trained with Equation (<a href="#tpg270082-disp-0004" class="usa-link">4</a>) was called model <strong>G</strong>. The base model was further modified by incorporating the effect of genotype by environment (G <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-17" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><mo>×</mo><mspace width="0.28em"></mspace></mrow></mrow></math></span> E), and Equation (<a href="#tpg270082-disp-0004" class="usa-link">4</a>) was modified as:
</p>
<table class="disp-formula p" id="tpg270082-disp-0005"><tr>
<td class="formula"><math id="jats-math-18" display="block"><mrow><mrow><mi>BLUE</mi><mfenced separators="" open="[" close="]"><msub><mi>g</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mfenced><mo linebreak="badbreak">=</mo><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">=</mo><msub><mi>E</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>g</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><mi>g</mi><msub><mi>E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>s</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>,</mo></mrow></mrow></math></td>
<td class="label">(5)</td>
</tr></table>
<p>where <em>E<sub>j</sub>
</em>, <em>g<sub>i</sub>
</em>, and <em>s<sub>ij</sub>
</em> are defined in Equation (<a href="#tpg270082-disp-0004" class="usa-link">4</a>). <em>gE<sub>ij</sub>
</em> denotes the G <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-19" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><mo>×</mo><mspace width="0.28em"></mspace></mrow></mrow></math></span> E interaction term with a multivariate normal distribution <strong>gE</strong> = <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-20" display="inline"><mrow><msup><mrow><mo stretchy="false">(</mo><mi>g</mi><msub><mi>E</mi><mn>11</mn></msub><mo>,</mo><mtext>…</mtext><mo>,</mo><mi>g</mi><msub><mi>E</mi><mrow><mn>1</mn><mi>I</mi></mrow></msub><mo>,</mo><mtext>…</mtext><mo>,</mo><mi>g</mi><msub><mi>E</mi><mrow><mi>I</mi><mi>J</mi></mrow></msub><mo stretchy="false">)</mo></mrow><mi>T</mi></msup></mrow></math></span> ∼<em>N</em> [0, (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-21" display="inline"><mrow><mrow><mi>g</mi><mi>T</mi><mi mathvariant="bold">Z</mi><mi>g</mi></mrow></mrow></math></span>
<strong>G</strong>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-22" display="inline"><mrow><mrow><mi mathvariant="bold">Z</mi><mi>g</mi><mi>T</mi></mrow></mrow></math></span>)<img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4103/12290324/755fc0422bbd/TPG2-18-e70082-e024.jpg" loading="lazy" alt="Inline graphic">(<strong>Z</strong>
<sub>E</sub>
<strong>Z</strong>
<sub>E</sub>
<sup>T</sup>) <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-24" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>g</mi><mi>E</mi></mrow><mn>2</mn></msubsup></mrow></math></span>], where <strong>G</strong> is the genomic relationship matrix, <strong>Z</strong>
<sub>g</sub> and <strong>Z</strong>
<sub>E</sub> are the incidence matrix for genomic and year effects, <img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4103/12290324/755fc0422bbd/TPG2-18-e70082-e007.jpg" loading="lazy" alt="Inline graphic"> denotes the Hadamard product, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-26" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>g</mi><mi>E</mi></mrow><mn>2</mn></msubsup></mrow></math></span> is the variance component of the genotype by environment interaction (Jarquín et al., <a href="#tpg270082-bib-0029" class="usa-link" aria-describedby="tpg270082-bib-0029">2014</a>). The model trained with Equation (<a href="#tpg270082-disp-0005" class="usa-link">5</a>), was denoted as model <strong>G</strong> + <strong>GE</strong>.</p>
<p>The model incorporating multispectral data was fit with <strong>W</strong> as covariance in a similar approach with genomic prediction:</p>
<table class="disp-formula p" id="tpg270082-disp-0006"><tr>
<td class="formula"><math id="jats-math-27" display="block"><mrow><mrow><mi>BLUE</mi><mfenced separators="" open="[" close="]"><msub><mi>g</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mfenced><mo linebreak="badbreak">=</mo><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">=</mo><msub><mi>E</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><mi>w</mi><msub><mi>E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>s</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>,</mo></mrow></mrow></math></td>
<td class="label">(6)</td>
</tr></table>
<p>where <em>E<sub>j</sub>
</em> and <em>s<sub>ij</sub>
</em> are defined in Equation (<a href="#tpg270082-disp-0004" class="usa-link">4</a>). <em>wE<sub>ij</sub>
</em> is the random effect of the multispectral band and or indices of genotype <em>i</em> in unique year <em>j</em>, which follows a joint normal distribution with <strong>wE</strong> ∼ <em>N</em> (0, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-28" display="inline"><mrow><mrow><mi mathvariant="bold">W</mi><msubsup><mi mathvariant="normal">σ</mi><mi>Ew</mi><mn>2</mn></msubsup></mrow></mrow></math></span>), where <strong>W</strong> is the multispectral relationship matrix for clone‐year combination and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-29" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>E</mi><mi>w</mi></mrow><mn>2</mn></msubsup></mrow></math></span> is the multispectral variance component for the clone‐year combination. The model in Equation (<a href="#tpg270082-disp-0006" class="usa-link">6</a>) was denoted as model <strong>W</strong>.</p>
<p>The model given in Equation (<a href="#tpg270082-disp-0006" class="usa-link">6</a>) was further improved by taking into account the effect of genotypes (<em>g<sub>i</sub>
</em>) as proposed by Lopez‐Cruz et al. (<a href="#tpg270082-bib-0037" class="usa-link" aria-describedby="tpg270082-bib-0037">2015</a>) and Krause et al. (<a href="#tpg270082-bib-0036" class="usa-link" aria-describedby="tpg270082-bib-0036">2019</a>):
</p>
<table class="disp-formula p" id="tpg270082-disp-0007"><tr>
<td class="formula"><math id="jats-math-30" display="block"><mrow><mrow><mi>BLUE</mi><mfenced separators="" open="[" close="]"><msub><mi>g</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mfenced><mo linebreak="badbreak">=</mo><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">=</mo><msub><mi>E</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>g</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><mi>w</mi><msub><mi>E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>s</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>,</mo></mrow></mrow></math></td>
<td class="label">(7)</td>
</tr></table>
<p>where all the components of Equation (<a href="#tpg270082-disp-0007" class="usa-link">7</a>) were described in the previous models, and this model was denoted as model <strong>G</strong> + <strong>WE</strong>.</p></section></section><section id="tpg270082-sec-0120"><h3 class="pmc_sec_title">2.6. Assessment of prediction models</h3>
<p>We evaluated the prediction performance under two types of cross‐validation of interest to potato breeding programs. First, we evaluated the prediction accuracy of tested lines in tested environments. Second, we evaluated the prediction performance of tested lines in untested environments (Alemu et al., <a href="#tpg270082-bib-0003" class="usa-link" aria-describedby="tpg270082-bib-0003">2024</a>). For the first evaluation, we assessed model performance using prediction ability across years randomly based on a 10‐fold cross‐validation scheme repeated 10 times by splitting data into random train‐test partitions. Each fold was partitioned with 20% of data for testing, while the remaining 80% was for training. The prediction ability was evaluated as the Pearson's correlation between the predicted value and the true value for each phenotypic trait. For the second type of cross‐validation, we partitioned clones into years and used the year 2023 as training set and year 2022 as testing set. Once again, the prediction ability was measured as the Pearson's correlation between the predicted values and the true values for each phenotypic trait.</p></section><section id="tpg270082-sec-0130"><h3 class="pmc_sec_title">2.7. Timepoint significance for prediction</h3>
<p>To identify the time point most significant for prediction and to recommend specific time points for prediction, we compared the prediction models based on multispectral matrices (<strong>W</strong> or <strong>G + WE</strong>) developed from the different time points and identified significant time points for prediction using the analysis of variance (ANOVA). Mean separation was carried out using least significant difference (LSD). The model was fitted as follows:
</p>
<table class="disp-formula p" id="tpg270082-disp-0008"><tr>
<td class="formula"><math id="jats-math-31" display="block"><mrow><mrow><msub><mi>Y</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi mathvariant="normal">α</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi mathvariant="normal">β</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">α</mi><msub><mi mathvariant="normal">β</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi></mrow></msub><mo>,</mo></mrow></mrow></math></td>
<td class="label">(8)</td>
</tr></table>
<p>where <em>Y<sub>ijk</sub>
</em> is the response variable corresponding to the <em>k</em>th mean prediction ability of the <em>i</em>th multispectral model and <em>j</em>th time point. <em>μ</em> is the overall mean, <em>α<sub>i</sub>
</em> is the effect for the multispectral model, <em>β<sub>j</sub>
</em> is the effect for the time point, <em>αβ<sub>ij</sub>
</em> is the interaction effect between the models and the time points, and <em>ϵ<sub>ijk</sub>
</em> is the residual that is assumed to be normally distributed with mean 0 and variance <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-32" display="inline"><mrow><msup><mi mathvariant="normal">σ</mi><mn>2</mn></msup></mrow></math></span>.</p></section><section id="tpg270082-sec-0140"><h3 class="pmc_sec_title">2.8. Features selection and significance for prediction</h3>
<p>Intercorrelated variables can lead to multicollinearity in regression equations, resulting in underperformance of prediction models (Yu et al., <a href="#tpg270082-bib-0074" class="usa-link" aria-describedby="tpg270082-bib-0074">2015</a>). Such scenarios are common for multispectral data, especially when multiple bands and indices are combined across timepoints (Galán et al., <a href="#tpg270082-bib-0020" class="usa-link" aria-describedby="tpg270082-bib-0020">2020</a>). We therefore carried out supervised feature selection with the combined stage multispectral data for yield of chipping potatoes using the Lasso and elastic‐net regularized generalized linear models (GLMNET) machine learning algorithm to identify the most important variables contributing to prediction (Friedman et al., <a href="#tpg270082-bib-0019" class="usa-link" aria-describedby="tpg270082-bib-0019">2010</a>).</p>
<p>In addition to the internal feature selection in the GLMNET algorithm, we used two other known feature selection wrappers within the caret R package, selection by filtering (SF) and simulation annealing (SA). Each of the features selected by the algorithm and wrappers was used to fit a new model. The set of selected features that produced the best prediction accuracies was used to fit a GS model. These new models were compared to models created based on the complete set of variables without feature selection in terms of prediction ability.</p></section></section><section id="tpg270082-sec-0150"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270082-sec-0160"><h3 class="pmc_sec_title">3.1. Phenotypic data analysis</h3>
<p>The traits measured for the chips included yield, specific gravity, and roundness, while for fresh market clones, we measured yield, redness, lightness, roundness, and l/w. BLUEs for each trait in both market classes were approximately normally distributed based on visual assessment of <em>Q</em>–<em>Q</em> plots (Figure <a href="#tpg270082-fig-0001" class="usa-link">1</a>). The BLUEs for the traits and the multispectral data were utilized for further analysis, with the trait BLUEs assigned as response variables. For the chips, broad‐sense heritability for yield was between 51% and 53%, specific gravity ranged between 64% and 66%, and roundness ranged between 27% and 77% (Figure <a href="#tpg270082-fig-0002" class="usa-link">2</a>). Within the fresh market class clones, broad‐sense heritability for yield ranged from 49% to 80%, redness ranged between 58% and 78%, lightness between 79% and 84%, roundness between 69% and 81%, and l/w between 48% and 65%.</p>
<figure class="fig xbox font-sm" id="tpg270082-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4103/12290324/4263a99d26bc/TPG2-18-e70082-g005.jpg" loading="lazy" id="jats-graphic-1" height="969" width="640" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270082-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Trait best linear unbiased estimates (BLUEs) distribution for (A) chipping clones and (B) fresh market clones.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270082-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4103/12290324/b8afd96cb3f6/TPG2-18-e70082-g006.jpg" loading="lazy" id="jats-graphic-3" height="272" width="546" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270082-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Broad‐sense heritability for both chipping clones and fresh market clones.</p></figcaption></figure><p>Multispectral data showed low to moderate heritability across market clones and years. Heritability for bands and indices across time points within the chips clones ranged from 8% to 72%, while the fresh market clones ranged between 0% and 84% (Figure <a href="#tpg270082-supinfo-0001" class="usa-link">S1</a>). Heritabilities were generally lower for bands and indices from the fresh market clones in 2022 (Figure <a href="#tpg270082-supinfo-0001" class="usa-link">S2</a>), with blue and red bands in the earlier time points (stage 1 and stage 2) exhibiting the lowest (&lt;1%) heritability. Blue and red bands also exhibited low heritability in later time points (stage 4 and stage 5) in 2023.</p></section><section id="tpg270082-sec-0170"><h3 class="pmc_sec_title">3.2. Prediction models</h3>
<section id="tpg270082-sec-0180"><h4 class="pmc_sec_title">3.2.1. Multi‐year genomic and multispectral prediction model</h4>
<p>We compared all the models described above for the chips. Models featuring only multispectral bands (Figure <a href="#tpg270082-fig-0003" class="usa-link">3</a>) performed indistinguishably from those based on both bands and indices (Figure <a href="#tpg270082-supinfo-0001" class="usa-link">S3</a>). The model combining genomic and multispectral data, <strong>G</strong> + <strong>WE</strong>, consistently outperformed the other models for all traits when data from all stages were combined. Both the models containing multispectral data, <strong>W</strong> and <strong>G</strong> + <strong>WE</strong>, outperformed the genomic only models, <strong>G</strong> and <strong>G</strong> + <strong>GE</strong>, for yield prediction at all time points and were highest at the combined stages. The mean prediction abilities at the combined stages for <strong>G</strong> and <strong>G</strong> + <strong>GE</strong> were both 0.43, while <strong>W</strong> and <strong>G</strong> + <strong>WE</strong> models were 0.72 and 0.74, respectively. In early time points, the models with genetic components, <strong>G</strong>, <strong>G</strong> + <strong>GE</strong>, and <strong>G</strong> + <strong>WE</strong>, performed better than the <strong>W</strong> only model for specific gravity. However, at stage 5 (senescence) and for the combined stage, models <strong>G</strong>, <strong>W</strong>, and <strong>G</strong> + <strong>GE</strong> performed similarly, although all were inferior to <strong>G</strong> + <strong>WE</strong> with a mean prediction ability of 0.36 at stage 5 and 0.39 for the combined stages. Prediction ability was generally lower for roundness. Similar to specific gravity, genetic models for roundness prediction performed better earlier in the growing season, while multispectral prediction improved as the season went on, but the combination of the two was the most successful with a mean prediction ability of 0.22 at stage 5 and 0.21 for the combined stages.</p>
<figure class="fig xbox font-sm" id="tpg270082-fig-0003"><h5 class="obj_head">FIGURE 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4103/12290324/14d89b96d3cd/TPG2-18-e70082-g007.jpg" loading="lazy" id="jats-graphic-5" height="310" width="546" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270082-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of models including genomic (<strong>G</strong>), genome by environment (<strong>GE</strong>), and/or multispectral (<strong>W</strong>) relationship matrices for chip traits across time points. Comparison is based on prediction ability in terms of Pearson's correlation of model‐predicted values to phenotypic observations across both years.</p></figcaption></figure><p>We repeated the above analysis for the fresh market clones to investigate the efficacy of models featuring multispectral data for prediction of quality traits. As with the chips, we found that including only bands (Figure <a href="#tpg270082-fig-0004" class="usa-link">4</a>) as opposed to combining bands and indices (Figure <a href="#tpg270082-supinfo-0001" class="usa-link">S4</a>) did not significantly affect prediction accuracy. For yield and color traits, the <strong>G</strong> + <strong>WE</strong> model performed the best. As in the chips, yield was best predicted by the two models including multispectral data. For the combined stages, the <strong>G</strong> + <strong>WE</strong> model had a prediction ability of 0.63, while that of the <strong>W</strong> model was 0.61. For both, redness and lightness models, including the genomic relationship matrix, performed best, although the difference was more pronounced for lightness across all stages. For both traits, the combined model (<strong>G</strong> + <strong>WE</strong>) with combined stages performed best, with a prediction ability of 0.43 for redness and 0.32 for lightness. For roundness, <strong>W</strong> was the strongest model with other models performing similarly, while for l/w, all models including the genomic relationship matrix, performed similarly, while <strong>W</strong> was the worst. Overall, prediction abilities were lowest for l/w.</p>
<figure class="fig xbox font-sm" id="tpg270082-fig-0004"><h5 class="obj_head">FIGURE 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4103/12290324/0c5b9953bf1d/TPG2-18-e70082-g003.jpg" loading="lazy" id="jats-graphic-7" height="310" width="546" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270082-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of models including genomic (<strong>G</strong>), genome by environment (<strong>GE</strong>), and/or multispectral (<strong>W</strong>) relationship matrices for traits measured in fresh market clones across time points. Comparison is based on prediction ability in terms of Pearson's correlation of model‐predicted values to phenotypic observations across both years.</p></figcaption></figure></section><section id="tpg270082-sec-0190"><h4 class="pmc_sec_title">3.2.2. Year‐specific genomic and multispectral prediction model</h4>
<p>In order to evaluate our ability to predict phenotypes in untested environments, we used years as a proxy for environment. We divided our data based on the year in which the data were gathered and assigned our largest dataset (2023) as the training set and our smallest (2022) as the test set. This analysis was only carried out in the chipping potatoes due to population size. Prediction ability was significant (<em>p</em> &lt; 0.05) for yield and specific gravity but not roundness (Figure <a href="#tpg270082-fig-0005" class="usa-link">5</a>; Figure <a href="#tpg270082-supinfo-0001" class="usa-link">S5</a>). These results were consistent across time points and for all time points combined.</p>
<figure class="fig xbox font-sm" id="tpg270082-fig-0005"><h5 class="obj_head">FIGURE 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4103/12290324/45b17ffb6fa9/TPG2-18-e70082-g004.jpg" loading="lazy" id="jats-graphic-9" height="228" width="546" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270082-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of year‐specific prediction ability for models including genome (<strong>G</strong>), genome by environment (<strong>GE</strong>), and/or multispectral (<strong>W</strong>) data combined across time points for chipping clones. Prediction ability is the Pearson's correlation of the model prediction to the observed phenotypic value.</p></figcaption></figure></section></section><section id="tpg270082-sec-0200"><h3 class="pmc_sec_title">3.3. Significance of time points for prediction</h3>
<p>Drone images were collected at five time points, determined by estimated average plant growth stage. We calculated relationship matrices based on each time point and all time points combined. We compared the prediction models based on multispectral matrices (<strong>W</strong> or <strong>G + WE</strong>) from the different stages and identified significant time points for prediction of yield and quality traits at harvest using ANOVA and mean separation using LSD. Time point was significant for all traits in both market classes. The time points that produced the best predictions varied by trait; however, the combined stage model was among the best performers for all traits in both market classes (Figure <a href="#tpg270082-fig-0006" class="usa-link">6</a>). Within the chips, combined analysis of all time points was the best predictor of yield. However, the earliest stage predicted yield equally well for the fresh market clones. Combined analysis was equivalent to analysis using either of the last two time points for specific gravity. Time points 4 and 5 were equally effective to combined analysis for predicting roundness in chipping clones, while stage 3 (peak flowering) was the best for roundness in fresh market clones. The last time point was similarly effective to combined analysis for lightness, and both the first and last time points were equivalent to combined analysis for predicting redness.</p>
<figure class="fig xbox font-sm" id="tpg270082-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4103/12290324/a81aae24731c/TPG2-18-e70082-g001.jpg" loading="lazy" id="jats-graphic-11" height="551" width="546" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270082-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The influence of time point of spectral data collection on prediction ability for various traits in (A) chipping potatoes and (B) fresh market clones.</p></figcaption></figure></section><section id="tpg270082-sec-0210"><h3 class="pmc_sec_title">3.4. Variable importance and significance for prediction</h3>
<p>The models using combined multispectral data across time points showed improved performance for most traits evaluated. Considering the risk of multicollinearity in multistage models (Yu et al., <a href="#tpg270082-bib-0074" class="usa-link" aria-describedby="tpg270082-bib-0074">2015</a>), we evaluated the importance and contribution of each variable to our yield prediction models for the chipping germplasm using a machine learning variable importance selection algorithm and wrappers. Specifically, we used the GLMNET algorithm, which provides a robust approach to coefficient shrinkage and regularization of important variables (Friedman et al., <a href="#tpg270082-bib-0019" class="usa-link" aria-describedby="tpg270082-bib-0019">2010</a>).</p>
<p>The GLMNET internal features selection algorithm and wrappers (Table <a href="#tpg270082-tbl-0002" class="usa-link">2</a>) selected different numbers of important multispectral variables contributing to prediction models. Performance for prediction based on the selected variables shows the wrappers performed better than the internal GLMNET features selection (Table <a href="#tpg270082-tbl-0002" class="usa-link">2</a>). The variables selected from the 65 total variables were 47 for GLMNET, 26 for SA, and 46 with SF. Most of the selected multispectral variables overlapped across the two wrappers (Table <a href="#tpg270082-supinfo-0001" class="usa-link">S1</a>).</p>
<section class="tw xbox font-sm" id="tpg270082-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>GLMNET Prediction accuracy in terms of Pearson's correlation (<em>R</em>) using selected multispectral variables.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Population set</th>
<th align="left" rowspan="1" colspan="1">GLMNET (<em>R</em>)</th>
<th align="left" rowspan="1" colspan="1">GLMNET_SA (<em>R</em>)</th>
<th align="left" rowspan="1" colspan="1">GLMNET_SF (<em>R</em>)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Training</td>
<td align="left" rowspan="1" colspan="1">0.70</td>
<td align="left" rowspan="1" colspan="1">0.75</td>
<td align="left" rowspan="1" colspan="1">0.76</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Testing</td>
<td align="left" rowspan="1" colspan="1">0.71</td>
<td align="left" rowspan="1" colspan="1">0.72</td>
<td align="left" rowspan="1" colspan="1">0.74</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270082-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270082-tbl2-note-0001"><p>Abbreviations: GLMNET, elastic‐net regularized general linear models; SA, simulation annealing; SF, selection by filtering.</p></div></div></section><p>Selected variables included multiple bands and indices from different time points. We compared a model including only the selected multispectral variables (<strong>W</strong>) and one combining genomic and selected multispectral variables (<strong>G</strong> + <strong>WE</strong>) to the genomic models (<strong>G</strong> and <strong>G</strong> + <strong>GE</strong>). The mean prediction ability from selected variables was 0.73 with SA wrapper and 0.75 with SF wrapper (Figure <a href="#tpg270082-fig-0007" class="usa-link">7</a>). These results showed no significant improvement compared with the use of the total unselected variables with 0.74 (Figure <a href="#tpg270082-fig-0007" class="usa-link">7</a>).</p>
<figure class="fig xbox font-sm" id="tpg270082-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4103/12290324/6d38ed475981/TPG2-18-e70082-g002.jpg" loading="lazy" id="jats-graphic-13" height="310" width="546" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270082-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Prediction ability for chipping clone traits across time points using subsets of the multispectral variables. SA, simulation annealing; SF, selection by filtering.</p></figcaption></figure></section></section><section id="tpg270082-sec-0220"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<section id="tpg270082-sec-0230"><h3 class="pmc_sec_title">4.1. Heritability of traits and spectra data</h3>
<p>Yield and quality traits exhibited moderate heritabilities consistent with previous estimates (Miller et al., <a href="#tpg270082-bib-0042" class="usa-link" aria-describedby="tpg270082-bib-0042">2022</a>; Yusuf et al., <a href="#tpg270082-bib-0075" class="usa-link" aria-describedby="tpg270082-bib-0075"><sup>2024</sup></a>), indicating a strong genetic basis for improvement. We observed variability between years for roundness in chips and yield, l/w, and redness in fresh market clones. Shape is known to have a strong environmental component (Jones et al., <a href="#tpg270082-bib-0033" class="usa-link" aria-describedby="tpg270082-bib-0033">2021</a>; Stefaniak et al., <a href="#tpg270082-bib-0062" class="usa-link" aria-describedby="tpg270082-bib-0062"><sup>2021</sup></a>; Yusuf et al., <a href="#tpg270082-bib-0075" class="usa-link" aria-describedby="tpg270082-bib-0075"><sup>2024</sup></a>) as does yield (Agha et al., <a href="#tpg270082-bib-0002" class="usa-link" aria-describedby="tpg270082-bib-0002">2024</a>; Schmitz Carley et al., <a href="#tpg270082-bib-0055" class="usa-link" aria-describedby="tpg270082-bib-0055"><sup>2019</sup></a>; Yusuf et al., <a href="#tpg270082-bib-0075" class="usa-link" aria-describedby="tpg270082-bib-0075"><sup>2024</sup></a>), which could explain the year‐to‐year variation. Heritability of the multispectral bands and indices was more variable, ranging from low to moderate across different market classes and years. One reason for this may be that cultivated potato exhibits variation for flowering time and maturity, and so single flight time points did not catch all individuals at the same growth stage. Low heritability of multispectral variables has also been observed in wheat (Krause et al., <a href="#tpg270082-bib-0036" class="usa-link" aria-describedby="tpg270082-bib-0036">2019</a>; Mróz et al., <a href="#tpg270082-bib-0045" class="usa-link" aria-describedby="tpg270082-bib-0045"><sup>2024</sup></a>).</p></section><section id="tpg270082-sec-0240"><h3 class="pmc_sec_title">4.2. How does phenomic prediction compare to genomic prediction?</h3>
<p>The most dramatic benefit of implementing phenomic prediction was an increase in prediction ability for yield in both market classes. This is consistent across time points and with previous studies in potato (Maggiorelli et al., <a href="#tpg270082-bib-0038" class="usa-link" aria-describedby="tpg270082-bib-0038">2024</a>), wheat (Krause et al., <a href="#tpg270082-bib-0036" class="usa-link" aria-describedby="tpg270082-bib-0036">2019</a>; Montesinos‐López et al., <a href="#tpg270082-bib-0044" class="usa-link" aria-describedby="tpg270082-bib-0044"><sup>2017</sup></a>; Mróz et al., <a href="#tpg270082-bib-0045" class="usa-link" aria-describedby="tpg270082-bib-0045"><sup>2024</sup></a>), rye (Galán et al., <a href="#tpg270082-bib-0020" class="usa-link" aria-describedby="tpg270082-bib-0020">2020</a>), corn (DeSalvio et al., <a href="#tpg270082-bib-0011" class="usa-link" aria-describedby="tpg270082-bib-0011">2024</a>), and soybean (Zhu et al., <a href="#tpg270082-bib-0077" class="usa-link" aria-describedby="tpg270082-bib-0077">2021</a>).</p>
<p>We further assessed the predictive ability of the multispectral model on traits other than yield. Phenomic models also outperformed genomic models for predicting roundness in later time points but were less successful in predicting skin color, l/w, or specific gravity. It is possible that better targeting of relevant spectral bands and indices would improve our ability to use phenomic prediction for quality traits. Physiochemical traits of canopy leaves, such as starch content, are known to be associated with specific wavelengths of leaf absorption (Alkhaled et al., <a href="#tpg270082-bib-0004" class="usa-link" aria-describedby="tpg270082-bib-0004">2023</a>; Yang et al., <a href="#tpg270082-bib-0072" class="usa-link" aria-describedby="tpg270082-bib-0072"><sup>2022</sup></a>; Zhou et al., <a href="#tpg270082-bib-0076" class="usa-link" aria-describedby="tpg270082-bib-0076"><sup>2018</sup></a>). If leaf and tuber starch content are correlated, this suggests a potential strategy for the prediction of specific gravity. In a variety of crops, imaging spectroscopy during the growing season, particularly from the NIR spectrum, accurately predicts harvest traits (Dallinger et al., <a href="#tpg270082-bib-0009" class="usa-link" aria-describedby="tpg270082-bib-0009">2023</a>; Su &amp; Xue, <a href="#tpg270082-bib-0064" class="usa-link" aria-describedby="tpg270082-bib-0064"><sup>2021</sup></a>).</p>
<p>Trait prediction from spectral imagery could be a particularly powerful tool in potato breeding. In general, potato breeding starts with a set of F1 crosses which are evaluated visually in the first field year from single plants. Breeding programs plant tens of thousands of individuals and select approximately 1%. Genotyping at this time point is prohibitively expensive, but phenotyping with drones would not be. For most programs, sufficient seed for yield trials is not generated until field year 3. If yield could be predicted from drone imagery in year 1, it could dramatically improve early generation selection accuracy and shorten breeding cycles. However, this depends on the accuracy of year‐to‐year predictions from single plants in new year‐environments, and phenomic prediction across environments is generally less successful than it is within an environment (Montesinos‐López et al., <a href="#tpg270082-bib-0044" class="usa-link" aria-describedby="tpg270082-bib-0044">2017</a>).</p></section><section id="tpg270082-sec-0250"><h3 class="pmc_sec_title">4.3. Is combining genomic and phenomic prediction effective?</h3>
<p>Genomic prediction often relies on incorporating more explanatory inputs to optimize and enhance prediction accuracy. Several studies have explored the integration of additional inputs into GS, such as environmental covariates (Jarquín et al., <a href="#tpg270082-bib-0030" class="usa-link" aria-describedby="tpg270082-bib-0030">2017</a>; Lopez‐Cruz et al., <a href="#tpg270082-bib-0037" class="usa-link" aria-describedby="tpg270082-bib-0037"><sup>2015</sup></a>), omics data (B. C. Perez et al., <a href="#tpg270082-bib-0049" class="usa-link" aria-describedby="tpg270082-bib-0049"><sup>2022</sup></a>; Ye et al., <a href="#tpg270082-bib-0073" class="usa-link" aria-describedby="tpg270082-bib-0073"><sup>2020</sup></a>), and phenomics (Galán et al., <a href="#tpg270082-bib-0020" class="usa-link" aria-describedby="tpg270082-bib-0020">2020</a>; Krause et al., <a href="#tpg270082-bib-0036" class="usa-link" aria-describedby="tpg270082-bib-0036"><sup>2019</sup></a>; Montesinos‐López et al., <a href="#tpg270082-bib-0044" class="usa-link" aria-describedby="tpg270082-bib-0044"><sup>2017</sup></a>; Mróz et al., <a href="#tpg270082-bib-0045" class="usa-link" aria-describedby="tpg270082-bib-0045"><sup>2024</sup></a>; Zhu et al., <a href="#tpg270082-bib-0077" class="usa-link" aria-describedby="tpg270082-bib-0077"><sup>2021</sup></a>), including spectral information, to achieve better optimization and prediction accuracy.</p>
<p>In this study, the integration of multispectral information through additional modeling of genotype by environment interactions (G <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-33" display="inline"><mrow><mo>×</mo></mrow></math></span> E) using spectral reflectance significantly improved genomic prediction for most of the traits considered, except for the l/w, which experienced a reduction of 7%. The inclusion of spectral data enhanced prediction accuracy for yield by over 30%, while improvements for other traits ranged from 3% to 11% across different market classes and time points. These findings suggest that multispectral information can capture additional effects not accounted for by marker effects alone, which is particularly important in clonal crops like potatoes. Previous studies in potato found that multispectral data were also highly predictive of maturity, foliage development, and emergence, and the only traits for which integrating multispectral with genetic data did not improve predictions were discoloration after cooking and polyphenol oxidase activity (Maggiorelli et al., <a href="#tpg270082-bib-0038" class="usa-link" aria-describedby="tpg270082-bib-0038">2024</a>).</p>
<p>Our findings are in strong agreement with previous studies that have utilized spectral information to improve prediction accuracy in GS, particularly for yield, using the combined relationship matrices (Galán et al., <a href="#tpg270082-bib-0020" class="usa-link" aria-describedby="tpg270082-bib-0020">2020</a>; Krause et al., <a href="#tpg270082-bib-0036" class="usa-link" aria-describedby="tpg270082-bib-0036"><sup>2019</sup></a>; Maggiorelli et al., <a href="#tpg270082-bib-0038" class="usa-link" aria-describedby="tpg270082-bib-0038"><sup>2024</sup></a>; Mróz et al., <a href="#tpg270082-bib-0045" class="usa-link" aria-describedby="tpg270082-bib-0045"><sup>2024</sup></a>). Notably, our approach differs slightly from that previously used in potato (Maggiorelli et al., <a href="#tpg270082-bib-0038" class="usa-link" aria-describedby="tpg270082-bib-0038">2024</a>) as we modeled the phenomic data as a G <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-34" display="inline"><mrow><mo>×</mo></mrow></math></span> E effect instead of using weighted matrices to avoid the confounding effects arising from multiple BLUPs (Krause et al., <a href="#tpg270082-bib-0036" class="usa-link" aria-describedby="tpg270082-bib-0036">2019</a>; Lopez‐Cruz et al., <a href="#tpg270082-bib-0037" class="usa-link" aria-describedby="tpg270082-bib-0037"><sup>2015</sup></a>). Our results also align with studies employing other linear and nonlinear modeling approaches, such as partial least squares, ordinary least squares, and functional regression, to predict yield using spectral information (Montesinos‐López et al., <a href="#tpg270082-bib-0043" class="usa-link" aria-describedby="tpg270082-bib-0043">2023</a>, <a href="#tpg270082-bib-0044" class="usa-link" aria-describedby="tpg270082-bib-0044">2017</a>).</p></section><section id="tpg270082-sec-0260"><h3 class="pmc_sec_title">4.4. What drone flight time points are most informative?</h3>
<p>We collected data at five time points based on the estimated growth stage of the plants. In order to test if fewer drone flights would be sufficient, we evaluated the performance of phenomic prediction models at each stage. The results from the ANOVA highlighted the significance and effectiveness of certain time points for prediction. However, for all traits the combined stage performed as well or better than individual time point analysis. The time points that were most predictive varied by trait and market class. A previous study in potato using a similar approach also found that combining their three time points lead to the highest prediction accuracies (Maggiorelli et al., <a href="#tpg270082-bib-0038" class="usa-link" aria-describedby="tpg270082-bib-0038">2024</a>). The only trait measured in both studies is yield, and our prediction accuracies for yield in both market classes (0.74 and 0.63) were slightly higher than the 0.61 reported by Maggiorelli et al. (<a href="#tpg270082-bib-0038" class="usa-link" aria-describedby="tpg270082-bib-0038">2024</a>). This is evidence that adding time points, especially between 30 and 61 days, the first two time points in Maggiorelli et al. (<a href="#tpg270082-bib-0038" class="usa-link" aria-describedby="tpg270082-bib-0038">2024</a>), improves prediction. We conclude that continuing to measure at all five time points is important to trait prediction, and it is possible adding additional time points would improve our models.</p>
<p>It is important to note that maturity is both highly variable and highly heritable in cultivated potato (Gutaker et al., <a href="#tpg270082-bib-0022" class="usa-link" aria-describedby="tpg270082-bib-0022">2019</a>; Kloosterman et al., <a href="#tpg270082-bib-0035" class="usa-link" aria-describedby="tpg270082-bib-0035"><sup>2013</sup></a>). At any particular flight time, different genotypes were likely at slightly different growth stages. Therefore, even if aboveground reflectance at a particular growth stage was the strongest predictor for a harvest phenotype, it might not be captured in this analysis, further supporting the need to measure multiple time points for accurate prediction.</p></section><section id="tpg270082-sec-0270"><h3 class="pmc_sec_title">4.5. Can we improve prediction accuracy by selecting key variables?</h3>
<p>Including too many non‐predictive variables in a prediction model can reduce power and result in multicollinearity, leading to underperformance of the model (Yu et al., <a href="#tpg270082-bib-0074" class="usa-link" aria-describedby="tpg270082-bib-0074">2015</a>). This is a known problem in multispectral data especially data combined across time points (Galán et al., <a href="#tpg270082-bib-0020" class="usa-link" aria-describedby="tpg270082-bib-0020">2020</a>), as well as in GS where including only trait‐associated markers can quadruple prediction ability (Singh et al., <a href="#tpg270082-bib-0058" class="usa-link" aria-describedby="tpg270082-bib-0058">2023</a>). Combining genomic and phenomic data across all time points exhibited the best predictive performance for yield; we tested if that performance could be improved by identifying the most crucial spectral variables and including only those variables in the model. Although we used multiple machine learning algorithms to define sets of predictive spectra, all subsets performed equally well to the total set. We have yet to reach the point of saturating the model and could potentially improve it with more spectral data.</p></section></section><section id="tpg270082-sec-0280"><h2 class="pmc_sec_title">5. CONCLUSION</h2>
<p>In this study, we have established the value of using multispectral data for both phenomic and GS in potatoes. By utilizing a multispectral relationship matrix, we can achieve prediction accuracy for agronomic and quality traits comparable to genomic prediction, offering a viable alternative when genetic markers are unavailable. Additionally, we have shown that integrating multispectral data into GS can further enhance prediction accuracy. We found no evidence that we were overfitting the model with too many non‐predictive variables, as our most accurate predictions resulted from combining data across time points, and feature selection made no difference.</p>
<p>This work serves as a proof of concept for using multispectral data derived from UAVs. Our approach was implemented with advanced breeding clones, where full‐sib families are limited, and there is minimal Mendelian sampling within the population. It is also important to note that we only included 2 years of data from one trial site. While more environments would give us a better sense of patterns of variation, this is a representative experiment for a potato breeding program. Potato breeding tends to be seed limited, with each plant only producing ∼10 seed pieces for the next generation, resulting in multi‐year periods before multi‐location trials are possible (Agha et al., <a href="#tpg270082-bib-0002" class="usa-link" aria-describedby="tpg270082-bib-0002">2024</a>). This long lag time can result in long slow breeding cycles and minimal genetic gain, especially for yield (Douches et al., <a href="#tpg270082-bib-0012" class="usa-link" aria-describedby="tpg270082-bib-0012">1996</a>). Genomic and phenomic prediction approaches aim to speed this process, allowing us to make breeding decisions based on fewer years of data (Slater et al., <a href="#tpg270082-bib-0059" class="usa-link" aria-describedby="tpg270082-bib-0059">2016</a>). While our model will be improved as we add data from future years, we were able to generate high prediction accuracies, which will be used for crossing decisions based on the 2 years of data available. Moving forward, we will focus on applying this concept to the screening of early‐generation clonal selections, where large number of entries require evaluation.</p></section><section id="tpg270082-sec-0290"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Muyideen Yusuf</strong>: Conceptualization; data curation; formal analysis; investigation; writing—original draft; writing—review and editing. <strong>Xiaoxi Meng</strong>: Conceptualization; funding acquisition; investigation; writing—review and editing. <strong>Thomas R. Stefaniak</strong>: Investigation; project administration; writing—review and editing. <strong>Osval A. Montesinos‐López</strong>: Formal analysis; writing—review and editing. <strong>Laura M. Shannon</strong>: Conceptualization; funding acquisition; investigation; project administration; supervision; writing—original draft; writing—review and editing.</p></section><section id="tpg270082-sec-0310"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>Laura M. Shannon is a member of the editorial board of The Plant Genome.</p></section><section id="sec31"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270082-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270082-supitem-0001"><div class="caption p">
<p>
<strong>Table S1</strong>. Variable features based on combined stages versus selected.</p>
<p>
<strong>Figure S1</strong>. Broad‐sense heritability of multispectral bands and indices within the chips market.</p>
<p>
<strong>Figure S2</strong>. Broad‐sense heritability of multispectral bands and indices within the fresh market.</p>
<p>
<strong>Figure S3</strong>. Prediction ability for traits within the chips market clones across growth stages using multispectral bands and Indices.</p>
<p>
<strong>Figure S4</strong>. Prediction ability for traits within the fresh market clones across growth stages using multispectral bands and Indices.</p>
<p>
<strong>Figure S5</strong>. Prediction ability for 2022 using 2023 within the chips market clones across growth stages using multispectral bands and Indices.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12290324/bin/TPG2-18-e70082-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70082-s001.pdf</a><sup> (4MB, pdf) </sup>
</div></div></section></section><section id="tpg270082-sec-0300" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>Breeding is a team effort. Husain Agha, Sydney Berry, Johnathan Boecker, Sapphire Coronejo, Linnea Johnston, Miranda Gregory, Megan Harder, Jessi Huege, John Larsen, Julia Lau, Adalyn Pederson, Heather Tuttle, Logan Rodewald, Lauren Sexton, Adynn Stedille,Timileyin Sunmonu, Ena Vigil, and Dat Vo assisted with planting, harvest, plant care, phenotyping, and tissue collection for genotyping. Mark Peterson and Ron Faber run the research site at the SPRF and managed plant care throughout each summer. Sentera flew the drones and returned spectral band information, flight dates were determined in partnership with Carl Rosen's lab. We are grateful to Jeffrey Endelman for allowing us to include clones from the University of Wisconsin potato breeding program in the study. Early drafts of this manuscript benefited from the comments of Dr. Rex Bernardo and Dr. Zhanyou Xu. Drone flights and analysis were funded by the Minnesota Department of Agriculture SCBG: 21SCBPMN1102. Genotyping was funded by the USDA NIFA through the polyploid tools SCRI‐SREP: 2020‐51181‐32156. Plant care and phenotyping were funded by the Minnesota Department of Agriculture through AGREETT and the USDA NIFA awards 2021‐34141‐35447, and 2023‐34141‐41020, and Potatoes USA.</p></section><section id="notes1"><p>


Yusuf, M.
, 
Meng, X.
, 
Stefaniak, T. R.
, 
Montesinos‐López, O. A.
, &amp; 
Shannon, L. M.
 (2025). Leveraging unmanned aerial vehicle derived multispectral data for improved genomic prediction in potato (<em>Solanum tuberosum</em>). The Plant Genome, 18, e70082. 10.1002/tpg2.70082

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="tpg270082-note-0001"><p>Assigned to Associate Editor Diego Jarquin.</p></div>
<div class="fn p" id="tpg270082-note-0002">
<p>
<strong>Present address</strong>
</p>
<p>Xiaoxi Meng, Department of Epidemiology and Cancer Control, St. Jude's Children's Research Hospital, Memphis, TN 38105, USA.</p>
</div>
</div></section></section><section id="tpg270082-sec-0330"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All data and scripts used for this work are available at: <a href="https://github.com/shannonlabumn/GS_multispectra_analysis.git" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/shannonlabumn/GS_multispectra_analysis.git</a>.</p></section><section id="tpg270082-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270082-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270082-bib-0001">
<cite>

Abdelhakim, L. O. A.
, 
Pleskačová, B.
, 
Rodriguez‐Granados, N. Y.
, 
Sasidharan, R.
, 
Perez‐Borroto, L. S.
, 
Sonnewald, S.
, 
Gruden, K.
, 
Vothknecht, U. C.
, 
Teige, M.
, &amp; 
Panzarová, K.
 (2024). High throughput image‐based phenotyping for determining morphological and physiological responses to single and combined stresses in potato. Journal of Visualized Experiments, 208, e66255. 10.3791/66255
</cite> [<a href="https://doi.org/10.3791/66255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38912820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Visualized%20Experiments&amp;title=High%20throughput%20image%E2%80%90based%20phenotyping%20for%20determining%20morphological%20and%20physiological%20responses%20to%20single%20and%20combined%20stresses%20in%20potato&amp;author=L.%20O.%20A.%20Abdelhakim&amp;author=B.%20Pleska%C4%8Dov%C3%A1&amp;author=N.%20Y.%20Rodriguez%E2%80%90Granados&amp;author=R.%20Sasidharan&amp;author=L.%20S.%20Perez%E2%80%90Borroto&amp;volume=208&amp;publication_year=2024&amp;pages=e66255&amp;pmid=38912820&amp;doi=10.3791/66255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0002">
<cite>

Agha, H. I.
, 
Endelman, J. B.
, 
Chitwood‐Brown, J.
, 
Clough, M.
, 
Coombs, J.
, 
De Jong, W. S.
, 
Douches, D. S.
, 
Higgins, C. R.
, 
Holm, D. G.
, 
Novy, R.
, 
Resende, M. F. R.
, 
Sathuvalli, V.
, 
Thompson, A. L.
, 
Yencho, G. C.
, 
Zotarelli, L.
, &amp; 
Shannon, L. M.
 (2024). Genotype‐by‐environment interactions and local adaptation shape selection in the US National Chip Processing Trial. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 137(5), Article 99. 10.1007/s00122-024-04610-3

</cite> [<a href="https://doi.org/10.1007/s00122-024-04610-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11006776/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38598016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Genotype%E2%80%90by%E2%80%90environment%20interactions%20and%20local%20adaptation%20shape%20selection%20in%20the%20US%20National%20Chip%20Processing%20Trial&amp;author=H.%20I.%20Agha&amp;author=J.%20B.%20Endelman&amp;author=J.%20Chitwood%E2%80%90Brown&amp;author=M.%20Clough&amp;author=J.%20Coombs&amp;volume=137&amp;issue=5&amp;publication_year=2024&amp;pages=99&amp;pmid=38598016&amp;doi=10.1007/s00122-024-04610-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0003">
<cite>

Alemu, A.
, 
Åstrand, J.
, 
Montesinos‐López, O. A.
, 
Isidro Y Sánchez, J.
, 
Fernández‐Gónzalez, J.
, 
Tadesse, W.
, 
Vetukuri, R. R.
, 
Carlsson, A. S.
, 
Ceplitis, A.
, 
Crossa, J.
, 
Ortiz, R.
, &amp; 
Chawade, A.
 (2024). Genomic selection in plant breeding: Key factors shaping two decades of progress. Molecular Plant, 17(4), 552–578. 10.1016/j.molp.2024.03.007

</cite> [<a href="https://doi.org/10.1016/j.molp.2024.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38475993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=Genomic%20selection%20in%20plant%20breeding:%20Key%20factors%20shaping%20two%20decades%20of%20progress&amp;author=A.%20Alemu&amp;author=J.%20%C3%85strand&amp;author=O.%20A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=J.%20Isidro%20Y%20S%C3%A1nchez&amp;author=J.%20Fern%C3%A1ndez%E2%80%90G%C3%B3nzalez&amp;volume=17&amp;issue=4&amp;publication_year=2024&amp;pages=552-578&amp;pmid=38475993&amp;doi=10.1016/j.molp.2024.03.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0004">
<cite>

Alkhaled, A.
, 
Townsend, P. A.
, &amp; 
Wang, Y.
 (2023). Remote sensing for monitoring potato nitrogen status. American Journal of Potato Research, 100(1), 1–14. 10.1007/s12230-022-09898-9
</cite> [<a href="https://doi.org/10.1007/s12230-022-09898-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Potato%20Research&amp;title=Remote%20sensing%20for%20monitoring%20potato%20nitrogen%20status&amp;author=A.%20Alkhaled&amp;author=P.%20A.%20Townsend&amp;author=Y.%20Wang&amp;volume=100&amp;issue=1&amp;publication_year=2023&amp;pages=1-14&amp;doi=10.1007/s12230-022-09898-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0005">
<cite>

Asano, K.
, &amp; 
Endelman, J. B.
 (2023). Development of KASP markers for the potato virus Y resistance gene <em>Rychc</em> using whole‐genome resequencing data. bioRxiv. 10.1101/2023.12.20.572658
</cite> [<a href="https://doi.org/10.1101/2023.12.20.572658" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Development%20of%20KASP%20markers%20for%20the%20potato%20virus%20Y%20resistance%20gene%20Rychc%20using%20whole%E2%80%90genome%20resequencing%20data&amp;author=K.%20Asano&amp;author=J.%20B.%20Endelman&amp;publication_year=2023&amp;doi=10.1101/2023.12.20.572658&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0006">
<cite>

Bradshaw, J. E.
 (2017). Review and analysis of limitations in ways to improve conventional potato breeding. Potato Research, 60(2), 171–193. 10.1007/s11540-017-9346-z
</cite> [<a href="https://doi.org/10.1007/s11540-017-9346-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Potato%20Research&amp;title=Review%20and%20analysis%20of%20limitations%20in%20ways%20to%20improve%20conventional%20potato%20breeding&amp;author=J.%20E.%20Bradshaw&amp;volume=60&amp;issue=2&amp;publication_year=2017&amp;pages=171-193&amp;doi=10.1007/s11540-017-9346-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0007">
<cite>

Caraza‐Harter, M. V.
, &amp; 
Endelman, J. B.
 (2020). Image‐based phenotyping and genetic analysis of potato skin set and color. Crop Science, 60(1), 202–210. 10.1002/csc2.20093
</cite> [<a href="https://doi.org/10.1002/csc2.20093" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Image%E2%80%90based%20phenotyping%20and%20genetic%20analysis%20of%20potato%20skin%20set%20and%20color&amp;author=M.%20V.%20Caraza%E2%80%90Harter&amp;author=J.%20B.%20Endelman&amp;volume=60&amp;issue=1&amp;publication_year=2020&amp;pages=202-210&amp;doi=10.1002/csc2.20093&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0008">
<cite>

Clot, C. R.
, 
Polzer, C.
, 
Prodhomme, C.
, 
Schuit, C.
, 
Engelen, C. J. M.
, 
Hutten, R. C. B.
, &amp; 
van Eck, H. J.
 (2020). The origin and widespread occurrence of <em>Sli</em>‐based self‐compatibility in potato. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 133, 2713–2728. 10.1007/s00122-020-03627-8

</cite> [<a href="https://doi.org/10.1007/s00122-020-03627-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7419354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32514711/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=The%20origin%20and%20widespread%20occurrence%20of%20Sli%E2%80%90based%20self%E2%80%90compatibility%20in%20potato&amp;author=C.%20R.%20Clot&amp;author=C.%20Polzer&amp;author=C.%20Prodhomme&amp;author=C.%20Schuit&amp;author=C.%20J.%20M.%20Engelen&amp;volume=133&amp;publication_year=2020&amp;pages=2713-2728&amp;pmid=32514711&amp;doi=10.1007/s00122-020-03627-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0009">
<cite>

Dallinger, H. G.
, 
Löschenberger, F.
, 
Bistrich, H.
, 
Ametz, C.
, 
Hetzendorfer, H.
, 
Morales, L.
, 
Michel, S.
, &amp; 
Buerstmayr, H.
 (2023). Predictor bias in genomic and phenomic selection. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 136(11), 235. 10.1007/s00122-023-04479-8

</cite> [<a href="https://doi.org/10.1007/s00122-023-04479-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10600307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37878079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Predictor%20bias%20in%20genomic%20and%20phenomic%20selection&amp;author=H.%20G.%20Dallinger&amp;author=F.%20L%C3%B6schenberger&amp;author=H.%20Bistrich&amp;author=C.%20Ametz&amp;author=H.%20Hetzendorfer&amp;volume=136&amp;issue=11&amp;publication_year=2023&amp;pages=235&amp;pmid=37878079&amp;doi=10.1007/s00122-023-04479-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0010">
<cite>

De Jong, W. S.
, 
Halseth, D. E.
, 
Plaisted, R. L.
, 
Wang, X.
, 
Perry, K. L.
, 
Qu, X.
, 
Paddock, K. M.
, 
Falise, M.
, 
Christ, B. J.
, &amp; 
Porter, G. A.
 (2017). Lamoka, a variety with excellent chip color out of cold storage and resistance to the golden cyst nematode. American Journal of Potato Research, 94(2), 148–152. 10.1007/s12230-016-9557-x
</cite> [<a href="https://doi.org/10.1007/s12230-016-9557-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Potato%20Research&amp;title=Lamoka,%20a%20variety%20with%20excellent%20chip%20color%20out%20of%20cold%20storage%20and%20resistance%20to%20the%20golden%20cyst%20nematode&amp;author=W.%20S.%20De%20Jong&amp;author=D.%20E.%20Halseth&amp;author=R.%20L.%20Plaisted&amp;author=X.%20Wang&amp;author=K.%20L.%20Perry&amp;volume=94&amp;issue=2&amp;publication_year=2017&amp;pages=148-152&amp;doi=10.1007/s12230-016-9557-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0011">
<cite>

DeSalvio, A. J.
, 
Adak, A.
, 
Murray, S. C.
, 
Jarquín, D.
, 
Winans, N. D.
, 
Crozier, D.
, &amp; 
Rooney, W. L.
 (2024). Near‐infrared reflectance spectroscopy phenomic prediction can perform similarly to genomic prediction of maize agronomic traits across environments. The Plant Genome, 17(2), e20454. 10.1002/tpg2.20454

</cite> [<a href="https://doi.org/10.1002/tpg2.20454" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38715204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Near%E2%80%90infrared%20reflectance%20spectroscopy%20phenomic%20prediction%20can%20perform%20similarly%20to%20genomic%20prediction%20of%20maize%20agronomic%20traits%20across%20environments&amp;author=A.%20J.%20DeSalvio&amp;author=A.%20Adak&amp;author=S.%20C.%20Murray&amp;author=D.%20Jarqu%C3%ADn&amp;author=N.%20D.%20Winans&amp;volume=17&amp;issue=2&amp;publication_year=2024&amp;pages=e20454&amp;pmid=38715204&amp;doi=10.1002/tpg2.20454&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0012">
<cite>

Douches, D. S.
, 
Maas, D.
, 
Jastrzebski, K.
, &amp; 
Chase, R. W.
 (1996). Assessment of potato breeding progress in the USA over the last century. Crop Science, 36(6), 1544–1552. 10.2135/cropsci1996.0011183X003600060024x
</cite> [<a href="https://doi.org/10.2135/cropsci1996.0011183X003600060024x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Assessment%20of%20potato%20breeding%20progress%20in%20the%20USA%20over%20the%20last%20century&amp;author=D.%20S.%20Douches&amp;author=D.%20Maas&amp;author=K.%20Jastrzebski&amp;author=R.%20W.%20Chase&amp;volume=36&amp;issue=6&amp;publication_year=1996&amp;pages=1544-1552&amp;doi=10.2135/cropsci1996.0011183X003600060024x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0013">
<cite>

Endelman, J. B.
 (2023). Fully efficient, two‐stage analysis of multi‐environment trials with directional dominance and multi‐trait genomic selection. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 136(4), Article 65. 10.1007/s00122-023-04298-x

</cite> [<a href="https://doi.org/10.1007/s00122-023-04298-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10033618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36949348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Fully%20efficient,%20two%E2%80%90stage%20analysis%20of%20multi%E2%80%90environment%20trials%20with%20directional%20dominance%20and%20multi%E2%80%90trait%20genomic%20selection&amp;author=J.%20B.%20Endelman&amp;volume=136&amp;issue=4&amp;publication_year=2023&amp;pages=65&amp;pmid=36949348&amp;doi=10.1007/s00122-023-04298-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0014">
<cite>

Endelman, J. B.
, 
Carley, C. A. S.
, 
Bethke, P. C.
, 
Coombs, J. J.
, 
Clough, M. E.
, 
Da Silva, W. L.
, 
De Jong, W. S.
, 
Douches, D. S.
, 
Frederick, C. M.
, 
Haynes, K. G.
, 
Holm, D. G.
, 
Miller, J. C.
, 
Muñoz, P. R.
, 
Navarro, F. M.
, 
Novy, R. G.
, 
Palta, J. P.
, 
Porter, G. A.
, 
Rak, K. T.
, 
Sathuvalli, V. R.
, … 
Yencho, G. C.
 (2018). Genetic variance partitioning and genome‐wide prediction with allele dosage information in autotetraploid potato. Genetics, 209(1), 77–87. 10.1534/genetics.118.300685

</cite> [<a href="https://doi.org/10.1534/genetics.118.300685" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5937173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29514860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genetic%20variance%20partitioning%20and%20genome%E2%80%90wide%20prediction%20with%20allele%20dosage%20information%20in%20autotetraploid%20potato&amp;author=J.%20B.%20Endelman&amp;author=C.%20A.%20S.%20Carley&amp;author=P.%20C.%20Bethke&amp;author=J.%20J.%20Coombs&amp;author=M.%20E.%20Clough&amp;volume=209&amp;issue=1&amp;publication_year=2018&amp;pages=77-87&amp;pmid=29514860&amp;doi=10.1534/genetics.118.300685&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0015">
<cite>

Endelman, J. B.
, 
Kante, M.
, 
Lindqvist‐Kreuze, H.
, 
Kilian, A.
, 
Shannon, L. M.
, 
Caraza‐Harter, M. V.
, 
Vaillancourt, B.
, 
Mailloux, K.
, 
Hamilton, J. P.
, &amp; 
Buell, C. R.
 (2024). Targeted genotyping‐by‐sequencing of potato and software for imputation. bioRxiv. 10.1101/2024.02.12.579978
</cite> [<a href="https://doi.org/10.1101/2024.02.12.579978" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38887158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Targeted%20genotyping%E2%80%90by%E2%80%90sequencing%20of%20potato%20and%20software%20for%20imputation&amp;author=J.%20B.%20Endelman&amp;author=M.%20Kante&amp;author=H.%20Lindqvist%E2%80%90Kreuze&amp;author=A.%20Kilian&amp;author=L.%20M.%20Shannon&amp;publication_year=2024&amp;pmid=38887158&amp;doi=10.1101/2024.02.12.579978&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0016"><cite>
FAO
. 
Crop and livestock products
. <a href="https://www.fao.org/faostat/en/#data/QCL" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fao.org/faostat/en/#data/QCL</a>
</cite></li>
<li id="tpg270082-bib-0017">
<cite>

Felcher, K. J.
, 
Coombs, J. J.
, 
Massa, A. N.
, 
Hansey, C. N.
, 
Hamilton, J. P.
, 
Veilleux, R. E.
, 
Buell, C. R.
, &amp; 
Douches, D. S.
 (2012). Integration of two diploid potato linkage maps with the potato genome sequence. PLoS ONE, 7(4), e36347. 10.1371/journal.pone.0036347

</cite> [<a href="https://doi.org/10.1371/journal.pone.0036347" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3338666/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22558443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Integration%20of%20two%20diploid%20potato%20linkage%20maps%20with%20the%20potato%20genome%20sequence&amp;author=K.%20J.%20Felcher&amp;author=J.%20J.%20Coombs&amp;author=A.%20N.%20Massa&amp;author=C.%20N.%20Hansey&amp;author=J.%20P.%20Hamilton&amp;volume=7&amp;issue=4&amp;publication_year=2012&amp;pages=e36347&amp;pmid=22558443&amp;doi=10.1371/journal.pone.0036347&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0018">
<cite>

Feldman, M. J.
, 
Park, J.
, 
Miller, N.
, 
Wakholi, C.
, 
Greene, K.
, 
Abbasi, A.
, 
Rippner, D. A.
, 
Navarre, D.
, 
Schmitz Carley, C.
, 
Shannon, L. M.
, &amp; 
Novy, R.
 (2024). A scalable, low‐cost phenotyping strategy to assess tuber size, shape, and the colorimetric features of tuber skin and flesh in potato breeding populations. The Plant Phenome Journal, 7(1), e20099. 10.1002/ppj2.20099
</cite> [<a href="https://doi.org/10.1002/ppj2.20099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Phenome%20Journal&amp;title=A%20scalable,%20low%E2%80%90cost%20phenotyping%20strategy%20to%20assess%20tuber%20size,%20shape,%20and%20the%20colorimetric%20features%20of%20tuber%20skin%20and%20flesh%20in%20potato%20breeding%20populations&amp;author=M.%20J.%20Feldman&amp;author=J.%20Park&amp;author=N.%20Miller&amp;author=C.%20Wakholi&amp;author=K.%20Greene&amp;volume=7&amp;issue=1&amp;publication_year=2024&amp;pages=e20099&amp;doi=10.1002/ppj2.20099&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0019">
<cite>

Friedman, J.
, 
Hastie, T.
, &amp; 
Tibshirani, R.
 (2010). Regularization paths for generalized linear models via coordinate descent. Journal of Statistical Software, 33(1), 1–22. 10.18637/jss.v033.i01

</cite> [<a href="https://doi.org/10.18637/jss.v033.i01" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2929880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20808728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=Regularization%20paths%20for%20generalized%20linear%20models%20via%20coordinate%20descent&amp;author=J.%20Friedman&amp;author=T.%20Hastie&amp;author=R.%20Tibshirani&amp;volume=33&amp;issue=1&amp;publication_year=2010&amp;pages=1-22&amp;pmid=20808728&amp;doi=10.18637/jss.v033.i01&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0020">
<cite>

Galán, R. J.
, 
Bernal‐Vasquez, A.‐M.
, 
Jebsen, C.
, 
Piepho, H.‐P.
, 
Thorwarth, P.
, 
Steffan, P.
, 
Gordillo, A.
, &amp; 
Miedaner, T.
 (2020). Integration of genotypic, hyperspectral, and phenotypic data to improve biomass yield prediction in hybrid rye. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 133(11), 3001–3015. 10.1007/s00122-020-03651-8

</cite> [<a href="https://doi.org/10.1007/s00122-020-03651-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7548001/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32681289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Integration%20of%20genotypic,%20hyperspectral,%20and%20phenotypic%20data%20to%20improve%20biomass%20yield%20prediction%20in%20hybrid%20rye&amp;author=R.%20J.%20Gal%C3%A1n&amp;author=A.%E2%80%90M.%20Bernal%E2%80%90Vasquez&amp;author=C.%20Jebsen&amp;author=H.%E2%80%90P.%20Piepho&amp;author=P.%20Thorwarth&amp;volume=133&amp;issue=11&amp;publication_year=2020&amp;pages=3001-3015&amp;pmid=32681289&amp;doi=10.1007/s00122-020-03651-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0021">
<cite>

Gemenet, D. C.
, 
Lindqvist‐Kreuze, H.
, 
De Boeck, B.
, 
Da Silva Pereira, G.
, 
Mollinari, M.
, 
Zeng, Z.‐B.
, 
Craig Yencho, G.
, &amp; 
Campos, H.
 (2020). Sequencing depth and genotype quality: Accuracy and breeding operation considerations for genomic selection applications in autopolyploid crops. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 133(12), 3345–3363. 10.1007/s00122-020-03673-2

</cite> [<a href="https://doi.org/10.1007/s00122-020-03673-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7567692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32876753/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Sequencing%20depth%20and%20genotype%20quality:%20Accuracy%20and%20breeding%20operation%20considerations%20for%20genomic%20selection%20applications%20in%20autopolyploid%20crops&amp;author=D.%20C.%20Gemenet&amp;author=H.%20Lindqvist%E2%80%90Kreuze&amp;author=B.%20De%20Boeck&amp;author=G.%20Da%20Silva%20Pereira&amp;author=M.%20Mollinari&amp;volume=133&amp;issue=12&amp;publication_year=2020&amp;pages=3345-3363&amp;pmid=32876753&amp;doi=10.1007/s00122-020-03673-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0079">
<cite>

Gitelson, A. A.
, &amp; 
Merzlyak, M. N.
 (1994a). In vivo optical properties of higher plant leaves. Plant Physiology, 104(3), 1019–1028. 10.1104/pp.104.3.1019
</cite> [<a href="https://doi.org/10.1104/pp.104.3.1019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=In%20vivo%20optical%20properties%20of%20higher%20plant%20leaves&amp;author=A.%20A.%20Gitelson&amp;author=M.%20N.%20Merzlyak&amp;volume=104&amp;issue=3&amp;publication_year=1994a&amp;pages=1019-1028&amp;doi=10.1104/pp.104.3.1019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0080">
<cite>

Gitelson, A. A.
, &amp; 
Merzlyak, M. N.
 (1994b). Remote sensing of chlorophyll a in higher plant leaves. Remote Sensing of Environment, 50(3), 246–252. 10.1016/0034-4257(94)90026-9
</cite> [<a href="https://doi.org/10.1016/0034-4257(94)90026-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Remote%20Sensing%20of%20Environment&amp;title=Remote%20sensing%20of%20chlorophyll%20a%20in%20higher%20plant%20leaves&amp;author=A.%20A.%20Gitelson&amp;author=M.%20N.%20Merzlyak&amp;volume=50&amp;issue=3&amp;publication_year=1994b&amp;pages=246-252&amp;doi=10.1016/0034-4257(94)90026-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0082">
<cite>

Gitelson, A. A.
, 
Gritz, S.
, &amp; 
Merzlyak, M. N.
 (2012). Chlorophyll content estimation in plant leaves using reflectance spectroscopy. Agronomy Journal, 104(3), 721–727. 10.2134/agronj2011.0370
</cite> [<a href="https://doi.org/10.2134/agronj2011.0370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy%20Journal&amp;title=Chlorophyll%20content%20estimation%20in%20plant%20leaves%20using%20reflectance%20spectroscopy&amp;author=A.%20A.%20Gitelson&amp;author=S.%20Gritz&amp;author=M.%20N.%20Merzlyak&amp;volume=104&amp;issue=3&amp;publication_year=2012&amp;pages=721-727&amp;doi=10.2134/agronj2011.0370&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0022">
<cite>

Gutaker, R. M.
, 
Weiß, C. L.
, 
Ellis, D.
, 
Anglin, N. L.
, 
Knapp, S.
, 
Luis Fernández‐Alonso, J.
, 
Prat, S.
, &amp; 
Burbano, H. A.
 (2019). The origins and adaptation of European potatoes reconstructed from historical genomes. Nature Ecology &amp; Evolution, 3(7), 1093–1101. 10.1038/s41559-019-0921-3

</cite> [<a href="https://doi.org/10.1038/s41559-019-0921-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31235927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Ecology%20&amp;%20Evolution&amp;title=The%20origins%20and%20adaptation%20of%20European%20potatoes%20reconstructed%20from%20historical%20genomes&amp;author=R.%20M.%20Gutaker&amp;author=C.%20L.%20Wei%C3%9F&amp;author=D.%20Ellis&amp;author=N.%20L.%20Anglin&amp;author=S.%20Knapp&amp;volume=3&amp;issue=7&amp;publication_year=2019&amp;pages=1093-1101&amp;pmid=31235927&amp;doi=10.1038/s41559-019-0921-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0083">
<cite>

Haboudane, D.
, 
Silva, C. A.
, 
Street, D.
, &amp; 
Zarco‐Tejada, P. J.
 (2010). Integrating remote sensing and crop growth models for irrigation management and yield forecasting. Field Crops Research, 118, 150–162. 10.1016/j.fcr.2010.06.007
</cite> [<a href="https://doi.org/10.1016/j.fcr.2010.06.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Field%20Crops%20Research&amp;title=Integrating%20remote%20sensing%20and%20crop%20growth%20models%20for%20irrigation%20management%20and%20yield%20forecasting&amp;author=D.%20Haboudane&amp;author=C.%20A.%20Silva&amp;author=D.%20Street&amp;author=P.%20J.%20Zarco%E2%80%90Tejada&amp;volume=118&amp;publication_year=2010&amp;pages=150-162&amp;doi=10.1016/j.fcr.2010.06.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0023">
<cite>

Habyarimana, E.
, 
Parisi, B.
, &amp; 
Mandolino, G.
 (2017). Genomic prediction for yields, processing and nutritional quality traits in cultivated potato (<em>Solanum tuberosum</em> L.). Plant Breeding, 136(2), 245–252. 10.1111/pbr.12461
</cite> [<a href="https://doi.org/10.1111/pbr.12461" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Breeding&amp;title=Genomic%20prediction%20for%20yields,%20processing%20and%20nutritional%20quality%20traits%20in%20cultivated%20potato%20(Solanum%20tuberosum%20L.)&amp;author=E.%20Habyarimana&amp;author=B.%20Parisi&amp;author=G.%20Mandolino&amp;volume=136&amp;issue=2&amp;publication_year=2017&amp;pages=245-252&amp;doi=10.1111/pbr.12461&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0024">
<cite>

Hardigan, M. A.
, 
Laimbeer, F. P. E.
, 
Newton, L.
, 
Crisovan, E.
, 
Hamilton, J. P.
, 
Vaillancourt, B.
, 
Wiegert‐Rininger, K.
, 
Wood, J. C.
, 
Douches, D. S.
, 
Farré, E. M.
, 
Veilleux, R. E.
, &amp; 
Buell, C. R.
 (2017). Genome diversity of tuber‐bearing <em>Solanum</em> uncovers complex evolutionary history and targets of domestication in the cultivated potato. PNAS, 114(46), E9999–E10008. 10.1073/pnas.1714380114

</cite> [<a href="https://doi.org/10.1073/pnas.1714380114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5699086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29087343/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Genome%20diversity%20of%20tuber%E2%80%90bearing%20Solanum%20uncovers%20complex%20evolutionary%20history%20and%20targets%20of%20domestication%20in%20the%20cultivated%20potato&amp;author=M.%20A.%20Hardigan&amp;author=F.%20P.%20E.%20Laimbeer&amp;author=L.%20Newton&amp;author=E.%20Crisovan&amp;author=J.%20P.%20Hamilton&amp;volume=114&amp;issue=46&amp;publication_year=2017&amp;pages=E9999-E10008&amp;pmid=29087343&amp;doi=10.1073/pnas.1714380114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0025">
<cite>

Herrera, M. D. R.
, 
Vidalon, L. J.
, 
Montenegro, J. D.
, 
Riccio, C.
, 
Guzman, F.
, 
Bartolini, I.
, &amp; 
Ghislain, M.
 (2018). Molecular and genetic characterization of the <em>Ryadg
</em> locus on chromosome XI from Andigena potatoes conferring extreme resistance to potato virus Y. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 131(9), 1925–1938. 10.1007/s00122-018-3123-5

</cite> [<a href="https://doi.org/10.1007/s00122-018-3123-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6096621/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29855674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Molecular%20and%20genetic%20characterization%20of%20the%20Ryadg%20locus%20on%20chromosome%20XI%20from%20Andigena%20potatoes%20conferring%20extreme%20resistance%20to%20potato%20virus%20Y&amp;author=M.%20D.%20R.%20Herrera&amp;author=L.%20J.%20Vidalon&amp;author=J.%20D.%20Montenegro&amp;author=C.%20Riccio&amp;author=F.%20Guzman&amp;volume=131&amp;issue=9&amp;publication_year=2018&amp;pages=1925-1938&amp;pmid=29855674&amp;doi=10.1007/s00122-018-3123-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0026">
<cite>

Hickey, L. T.
, 
Hafeez, A. N.
, 
Robinson, H.
, 
Jackson, S. A.
, 
Leal‐Bertioli, S. C. M.
, 
Tester, M.
, 
Gao, C.
, 
Godwin, I. D.
, 
Hayes, B. J.
, &amp; 
Wulff, B. B. H.
 (2019). Breeding crops to feed 10 billion. Nature Biotechnology, 37(7), 744–754. 10.1038/s41587-019-0152-9
</cite> [<a href="https://doi.org/10.1038/s41587-019-0152-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31209375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Breeding%20crops%20to%20feed%2010%20billion&amp;author=L.%20T.%20Hickey&amp;author=A.%20N.%20Hafeez&amp;author=H.%20Robinson&amp;author=S.%20A.%20Jackson&amp;author=S.%20C.%20M.%20Leal%E2%80%90Bertioli&amp;volume=37&amp;issue=7&amp;publication_year=2019&amp;pages=744-754&amp;pmid=31209375&amp;doi=10.1038/s41587-019-0152-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0027">
<cite>

Hoopes, G.
, 
Meng, X.
, 
Hamilton, J. P.
, 
Achakkagari, S. R.
, 
De Alvess Freitas Guesdes, F.
, 
Bolger, M. E.
, 
Coombs, J. J.
, 
Esselink, D.
, 
Kaiser, N. R.
, 
Kodde, L.
, 
Kyriakidou, M.
, 
Lavrijssen, B.
, 
Van Lieshout, N.
, 
Shereda, R.
, 
Tuttle, H. K.
, 
Vaillancourt, B.
, 
Wood, J. C.
, 
De Boer, J. M.
, 
Bornowski, N.
, … 
Finkers, R.
 (2022). Phased, chromosome‐scale genome assemblies of tetraploid potato reveal a complex genome, transcriptome, and predicted proteome landscape underpinning genetic diversity. Molecular Plant, 15(3), 520–536. 10.1016/j.molp.2022.01.003

</cite> [<a href="https://doi.org/10.1016/j.molp.2022.01.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35026436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=Phased,%20chromosome%E2%80%90scale%20genome%20assemblies%20of%20tetraploid%20potato%20reveal%20a%20complex%20genome,%20transcriptome,%20and%20predicted%20proteome%20landscape%20underpinning%20genetic%20diversity&amp;author=G.%20Hoopes&amp;author=X.%20Meng&amp;author=J.%20P.%20Hamilton&amp;author=S.%20R.%20Achakkagari&amp;author=F.%20De%20Alvess%20Freitas%20Guesdes&amp;volume=15&amp;issue=3&amp;publication_year=2022&amp;pages=520-536&amp;pmid=35026436&amp;doi=10.1016/j.molp.2022.01.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0028">
<cite>

Jansky, S.
 (2009). Breeding, genetics, and cultivar development. In Singh J. &amp; Kaur L. (Eds.), Advances in potato chemistry and technology (pp. 27–62). Academic Press.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20potato%20chemistry%20and%20technology&amp;author=S.%20Jansky&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0029">
<cite>

Jarquín, D.
, 
Crossa, J.
, 
Lacaze, X.
, 
Du Cheyron, P.
, 
Daucourt, J.
, 
Lorgeou, J.
, 
Piraux, F.
, 
Guerreiro, L.
, 
Pérez, P.
, 
Calus, M.
, 
Burgueño, J.
, &amp; 
de los Campos, G.
 (2014). A reaction norm model for genomic selection using high‐dimensional genomic and environmental data. Theoretical and Applied Genetics, 127, 595–607. 10.1007/s00122-013-2243-1

</cite> [<a href="https://doi.org/10.1007/s00122-013-2243-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3931944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24337101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=A%20reaction%20norm%20model%20for%20genomic%20selection%20using%20high%E2%80%90dimensional%20genomic%20and%20environmental%20data&amp;author=D.%20Jarqu%C3%ADn&amp;author=J.%20Crossa&amp;author=X.%20Lacaze&amp;author=P.%20Du%20Cheyron&amp;author=J.%20Daucourt&amp;volume=127&amp;publication_year=2014&amp;pages=595-607&amp;pmid=24337101&amp;doi=10.1007/s00122-013-2243-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0030">
<cite>

Jarquín, D.
, 
Lemes Da Silva, C.
, 
Gaynor, R. C.
, 
Poland, J.
, 
Fritz, A.
, 
Howard, R.
, 
Battenfield, S.
, &amp; 
Crossa, J.
 (2017). Increasing genomic‐enabled prediction accuracy by modeling genotype × environment interactions in Kansas wheat. The Plant Genome, 10(2), plantgenome2016.12.0130. 10.3835/plantgenome2016.12.0130
</cite> [<a href="https://doi.org/10.3835/plantgenome2016.12.0130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28724062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Increasing%20genomic%E2%80%90enabled%20prediction%20accuracy%20by%20modeling%20genotype%20%C3%97%20environment%20interactions%20in%20Kansas%20wheat&amp;author=D.%20Jarqu%C3%ADn&amp;author=C.%20Lemes%20Da%20Silva&amp;author=R.%20C.%20Gaynor&amp;author=J.%20Poland&amp;author=A.%20Fritz&amp;volume=10&amp;issue=2&amp;publication_year=2017&amp;pmid=28724062&amp;doi=10.3835/plantgenome2016.12.0130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0031">
<cite>

Johansen, R. H.
, 
Sandar, N.
, 
Hoyman, W. G.
, &amp; 
Lana, E. P.
 (1959). Norland a new red‐skinned potato variety with early maturity and moderate resistance to common scab. American Potato Journal, 36(1), 12–15. 10.1007/BF02877209
</cite> [<a href="https://doi.org/10.1007/BF02877209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Potato%20Journal&amp;title=Norland%20a%20new%20red%E2%80%90skinned%20potato%20variety%20with%20early%20maturity%20and%20moderate%20resistance%20to%20common%20scab&amp;author=R.%20H.%20Johansen&amp;author=N.%20Sandar&amp;author=W.%20G.%20Hoyman&amp;author=E.%20P.%20Lana&amp;volume=36&amp;issue=1&amp;publication_year=1959&amp;pages=12-15&amp;doi=10.1007/BF02877209&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0032">
<cite>

Johnston, G. R.
, &amp; 
Rowberry, R. G.
 (1981). Yukon Gold: A new yellow‐fleshed, medium‐early, high quality table and French‐fry cultivar. American Potato Journal, 58(5), 241–244. 10.1007/BF02853905
</cite> [<a href="https://doi.org/10.1007/BF02853905" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Potato%20Journal&amp;title=Yukon%20Gold:%20A%20new%20yellow%E2%80%90fleshed,%20medium%E2%80%90early,%20high%20quality%20table%20and%20French%E2%80%90fry%20cultivar&amp;author=G.%20R.%20Johnston&amp;author=R.%20G.%20Rowberry&amp;volume=58&amp;issue=5&amp;publication_year=1981&amp;pages=241-244&amp;doi=10.1007/BF02853905&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0033">
<cite>

Jones, C. R.
, 
Michaels, T. E.
, 
Schmitz Carley, C.
, 
Rosen, C. J.
, &amp; 
Shannon, L. M.
 (2021). Nitrogen uptake and utilization in advanced fresh‐market red potato breeding lines. Crop Science, 61(2), 878–895. 10.1002/csc2.20297
</cite> [<a href="https://doi.org/10.1002/csc2.20297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Nitrogen%20uptake%20and%20utilization%20in%20advanced%20fresh%E2%80%90market%20red%20potato%20breeding%20lines&amp;author=C.%20R.%20Jones&amp;author=T.%20E.%20Michaels&amp;author=C.%20Schmitz%20Carley&amp;author=C.%20J.%20Rosen&amp;author=L.%20M.%20Shannon&amp;volume=61&amp;issue=2&amp;publication_year=2021&amp;pages=878-895&amp;doi=10.1002/csc2.20297&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0034">
<cite>

Kaiser, N. R.
, 
Jansky, S.
, 
Coombs, J. J.
, 
Collins, P.
, 
Alsahlany, M.
, &amp; 
Douches, D. S.
 (2021). Assessing the contribution of <em>Sli</em> to self‐compatibility in North American diploid potato germplasm using KASP™ markers. American Journal of Potato Research, 98(2), 104–113. 10.1007/s12230-021-09821-8
</cite> [<a href="https://doi.org/10.1007/s12230-021-09821-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Potato%20Research&amp;title=Assessing%20the%20contribution%20of%20Sli%20to%20self%E2%80%90compatibility%20in%20North%20American%20diploid%20potato%20germplasm%20using%20KASP%E2%84%A2%20markers&amp;author=N.%20R.%20Kaiser&amp;author=S.%20Jansky&amp;author=J.%20J.%20Coombs&amp;author=P.%20Collins&amp;author=M.%20Alsahlany&amp;volume=98&amp;issue=2&amp;publication_year=2021&amp;pages=104-113&amp;doi=10.1007/s12230-021-09821-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0035">
<cite>

Kloosterman, B.
, 
Abelenda, J. A.
, 
Gomez, M. D. M. C.
, 
Oortwijn, M.
, 
De Boer, J. M.
, 
Kowitwanich, K.
, 
Horvath, B. M.
, 
Van Eck, H. J.
, 
Smaczniak, C.
, 
Prat, S.
, 
Visser, R. G. F.
, &amp; 
Bachem, C. W. B.
 (2013). Naturally occurring allele diversity allows potato cultivation in northern latitudes. Nature, 495(7440), 246–250. 10.1038/nature11912

</cite> [<a href="https://doi.org/10.1038/nature11912" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23467094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Naturally%20occurring%20allele%20diversity%20allows%20potato%20cultivation%20in%20northern%20latitudes&amp;author=B.%20Kloosterman&amp;author=J.%20A.%20Abelenda&amp;author=M.%20D.%20M.%20C.%20Gomez&amp;author=M.%20Oortwijn&amp;author=J.%20M.%20De%20Boer&amp;volume=495&amp;issue=7440&amp;publication_year=2013&amp;pages=246-250&amp;pmid=23467094&amp;doi=10.1038/nature11912&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0036">
<cite>

Krause, M. R.
, 
González‐Pérez, L.
, 
Crossa, J.
, 
Pérez‐Rodríguez, P.
, 
Montesinos‐López, O.
, 
Singh, R. P.
, 
Dreisigacker, S.
, 
Poland, J.
, 
Rutkoski, J.
, 
Sorrells, M.
, 
Gore, M. A.
, &amp; 
Mondal, S.
 (2019). Hyperspectral reflectance‐derived relationship matrices for genomic prediction of grain yield in wheat. G3 Genes|Genomes|Genetics, 9(4), 1231–1247. 10.1534/g3.118.200856

</cite> [<a href="https://doi.org/10.1534/g3.118.200856" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6469421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30796086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Hyperspectral%20reflectance%E2%80%90derived%20relationship%20matrices%20for%20genomic%20prediction%20of%20grain%20yield%20in%20wheat&amp;author=M.%20R.%20Krause&amp;author=L.%20Gonz%C3%A1lez%E2%80%90P%C3%A9rez&amp;author=J.%20Crossa&amp;author=P.%20P%C3%A9rez%E2%80%90Rodr%C3%ADguez&amp;author=O.%20Montesinos%E2%80%90L%C3%B3pez&amp;volume=9&amp;issue=4&amp;publication_year=2019&amp;pages=1231-1247&amp;pmid=30796086&amp;doi=10.1534/g3.118.200856&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0037">
<cite>

Lopez‐Cruz, M.
, 
Crossa, J.
, 
Bonnett, D.
, 
Dreisigacker, S.
, 
Poland, J.
, 
Jannink, J.‐L.
, 
Singh, R. P.
, 
Autrique, E.
, &amp; 
de los Campos, G.
 (2015). Increased prediction accuracy in wheat breeding trials using a marker × environment interaction genomic selection model. G3 Genes|Genomes|Genetics, 5(4), 569–582. 10.1534/g3.114.016097

</cite> [<a href="https://doi.org/10.1534/g3.114.016097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4390573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25660166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Increased%20prediction%20accuracy%20in%20wheat%20breeding%20trials%20using%20a%20marker%20%C3%97%20environment%20interaction%20genomic%20selection%20model&amp;author=M.%20Lopez%E2%80%90Cruz&amp;author=J.%20Crossa&amp;author=D.%20Bonnett&amp;author=S.%20Dreisigacker&amp;author=J.%20Poland&amp;volume=5&amp;issue=4&amp;publication_year=2015&amp;pages=569-582&amp;pmid=25660166&amp;doi=10.1534/g3.114.016097&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0038">
<cite>

Maggiorelli, A.
, 
Baig, N.
, 
Prigge, V.
, 
Bruckmüller, J.
, &amp; 
Stich, B.
 (2024). Using drone‐retrieved multispectral data for phenomic selection in potato breeding. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 137(3), Article 70. 10.1007/s00122-024-04567-3

</cite> [<a href="https://doi.org/10.1007/s00122-024-04567-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10917832/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38446220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Using%20drone%E2%80%90retrieved%20multispectral%20data%20for%20phenomic%20selection%20in%20potato%20breeding&amp;author=A.%20Maggiorelli&amp;author=N.%20Baig&amp;author=V.%20Prigge&amp;author=J.%20Bruckm%C3%BCller&amp;author=B.%20Stich&amp;volume=137&amp;issue=3&amp;publication_year=2024&amp;pages=70&amp;pmid=38446220&amp;doi=10.1007/s00122-024-04567-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0039">
<cite>

Martins, V. S.
, 
Andrade, M. H. M. L.
, 
Padua, L. N.
, 
Miguel, L. A.
, 
Fernandes Filho, C. C.
, 
Guedes, M. L.
, 
Nunes, J. A. R.
, 
Hoffmann, L., Jr.
, 
Zotarelli, L.
, 
Resende, M. F. R. D., Jr.
, 
Carneiro, P. C. S.
, &amp; 
Marçal, T. D. S.
 (2023). Evaluating the impact of modeling the family effect for clonal selection in potato‐breeding programs. Frontiers in Plant Science, 14, 1253706. 10.3389/fpls.2023.1253706

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1253706" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10642306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37965021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Evaluating%20the%20impact%20of%20modeling%20the%20family%20effect%20for%20clonal%20selection%20in%20potato%E2%80%90breeding%20programs&amp;author=V.%20S.%20Martins&amp;author=M.%20H.%20M.%20L.%20Andrade&amp;author=L.%20N.%20Padua&amp;author=L.%20A.%20Miguel&amp;author=C.%20C.%20Fernandes%20Filho&amp;volume=14&amp;publication_year=2023&amp;pages=1253706&amp;pmid=37965021&amp;doi=10.3389/fpls.2023.1253706&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0081">
<cite>

McFeeters, S. K.
 (1994). Estimation of wetlands and open water using Landsat Thematic Mapper data. Photogrammetric Engineering and Remote Sensing, 60(3), 281–289. 10.14358/PERS.60.3.281
</cite> [<a href="https://doi.org/10.14358/PERS.60.3.281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Photogrammetric%20Engineering%20and%20Remote%20Sensing&amp;title=Estimation%20of%20wetlands%20and%20open%20water%20using%20Landsat%20Thematic%20Mapper%20data&amp;author=S.%20K.%20McFeeters&amp;volume=60&amp;issue=3&amp;publication_year=1994&amp;pages=281-289&amp;doi=10.14358/PERS.60.3.281&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0040">
<cite>

Meade, F.
, 
Byrne, S.
, 
Griffin, D.
, 
Kennedy, C.
, 
Mesiti, F.
, &amp; 
Milbourne, D.
 (2020). Rapid development of KASP markers for disease resistance genes using pooled whole‐genome resequencing. Potato Research, 63(1), 57–73. 10.1007/s11540-019-09428-x
</cite> [<a href="https://doi.org/10.1007/s11540-019-09428-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Potato%20Research&amp;title=Rapid%20development%20of%20KASP%20markers%20for%20disease%20resistance%20genes%20using%20pooled%20whole%E2%80%90genome%20resequencing&amp;author=F.%20Meade&amp;author=S.%20Byrne&amp;author=D.%20Griffin&amp;author=C.%20Kennedy&amp;author=F.%20Mesiti&amp;volume=63&amp;issue=1&amp;publication_year=2020&amp;pages=57-73&amp;doi=10.1007/s11540-019-09428-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0041">
<cite>

Meuwissen, T. H. E.
, 
Hayes, B. J.
, &amp; 
Goddard, M. E.
 (2001). Prediction of total genetic value using genome‐wide dense marker maps. Genetics, 157(4), 1819–1829. 10.1093/genetics/157.4.1819

</cite> [<a href="https://doi.org/10.1093/genetics/157.4.1819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11290733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Prediction%20of%20total%20genetic%20value%20using%20genome%E2%80%90wide%20dense%20marker%20maps&amp;author=T.%20H.%20E.%20Meuwissen&amp;author=B.%20J.%20Hayes&amp;author=M.%20E.%20Goddard&amp;volume=157&amp;issue=4&amp;publication_year=2001&amp;pages=1819-1829&amp;pmid=11290733&amp;doi=10.1093/genetics/157.4.1819&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0042">
<cite>

Miller, M. D.
, 
Schmitz Carley, C. A.
, 
Figueroa, R. A.
, 
Feldman, M. J.
, 
Haagenson, D.
, &amp; 
Shannon, L. M.
 (2022). TubAR: An R package for quantifying tuber shape and skin traits from images. American Journal of Potato Research, 100(1), 52–62. 10.1007/s12230-022-09894-z
</cite> [<a href="https://doi.org/10.1007/s12230-022-09894-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Potato%20Research&amp;title=TubAR:%20An%20R%20package%20for%20quantifying%20tuber%20shape%20and%20skin%20traits%20from%20images&amp;author=M.%20D.%20Miller&amp;author=C.%20A.%20Schmitz%20Carley&amp;author=R.%20A.%20Figueroa&amp;author=M.%20J.%20Feldman&amp;author=D.%20Haagenson&amp;volume=100&amp;issue=1&amp;publication_year=2022&amp;pages=52-62&amp;doi=10.1007/s12230-022-09894-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0043">
<cite>

Montesinos‐López, O. A.
, 
Herr, A. W.
, 
Crossa, J.
, &amp; 
Carter, A. H.
 (2023). Genomics combined with UAS data enhances prediction of grain yield in winter wheat. Frontiers in Genetics, 14, 1124218. 10.3389/fgene.2023.1124218

</cite> [<a href="https://doi.org/10.3389/fgene.2023.1124218" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10090417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37065497/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Genomics%20combined%20with%20UAS%20data%20enhances%20prediction%20of%20grain%20yield%20in%20winter%20wheat&amp;author=O.%20A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=A.%20W.%20Herr&amp;author=J.%20Crossa&amp;author=A.%20H.%20Carter&amp;volume=14&amp;publication_year=2023&amp;pages=1124218&amp;pmid=37065497&amp;doi=10.3389/fgene.2023.1124218&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0044">
<cite>

Montesinos‐López, O. A.
, 
Montesinos‐López, A.
, 
Crossa, J.
, 
de los Campos, G.
, 
Alvarado, G.
, 
Suchismita, M.
, 
Rutkoski, J.
, 
González‐Pérez, L.
, &amp; 
Burgueño, J.
 (2017). Predicting grain yield using canopy hyperspectral reflectance in wheat breeding data. Plant Methods, 13(1), Article 4. 10.1186/s13007-016-0154-2

</cite> [<a href="https://doi.org/10.1186/s13007-016-0154-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5209864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28053649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Methods&amp;title=Predicting%20grain%20yield%20using%20canopy%20hyperspectral%20reflectance%20in%20wheat%20breeding%20data&amp;author=O.%20A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=J.%20Crossa&amp;author=G.%20de%20los%20Campos&amp;author=G.%20Alvarado&amp;volume=13&amp;issue=1&amp;publication_year=2017&amp;pages=4&amp;pmid=28053649&amp;doi=10.1186/s13007-016-0154-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0045">
<cite>

Mróz, T.
, 
Shafiee, S.
, 
Crossa, J.
, 
Montesinos‐Lopez, O. A.
, &amp; 
Lillemo, M.
 (2024). Multispectral‐derived genotypic similarities from budget cameras allow grain yield prediction and genomic selection augmentation in single and multi‐environment scenarios in spring wheat. Molecular Breeding: New strategies in plant improvement, 44(1), Article 5. 10.1007/s11032-024-01449-w

</cite> [<a href="https://doi.org/10.1007/s11032-024-01449-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10789716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38230361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding:%20New%20strategies%20in%20plant%20improvement&amp;title=Multispectral%E2%80%90derived%20genotypic%20similarities%20from%20budget%20cameras%20allow%20grain%20yield%20prediction%20and%20genomic%20selection%20augmentation%20in%20single%20and%20multi%E2%80%90environment%20scenarios%20in%20spring%20wheat&amp;author=T.%20Mr%C3%B3z&amp;author=S.%20Shafiee&amp;author=J.%20Crossa&amp;author=O.%20A.%20Montesinos%E2%80%90Lopez&amp;author=M.%20Lillemo&amp;volume=44&amp;issue=1&amp;publication_year=2024&amp;pages=5&amp;pmid=38230361&amp;doi=10.1007/s11032-024-01449-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0046">
<cite>

Neilson, J. A. D.
, 
Smith, A. M.
, 
Mesina, L.
, 
Vivian, R.
, 
Smienk, S.
, &amp; 
De Koyer, D.
 (2021). Potato tuber shape phenotyping using RGB imaging. Agronomy, 11(9), 1781. 10.3390/agronomy11091781
</cite> [<a href="https://doi.org/10.3390/agronomy11091781" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=Potato%20tuber%20shape%20phenotyping%20using%20RGB%20imaging&amp;author=J.%20A.%20D.%20Neilson&amp;author=A.%20M.%20Smith&amp;author=L.%20Mesina&amp;author=R.%20Vivian&amp;author=S.%20Smienk&amp;volume=11&amp;issue=9&amp;publication_year=2021&amp;pages=1781&amp;doi=10.3390/agronomy11091781&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0047">
<cite>

Nie, X.
, 
Sutherland, D.
, 
Dickison, V.
, 
Singh, M.
, 
Murphy, A. M.
, &amp; 
De Koeyer, D.
 (2016). Development and validation of high‐resolution melting markers derived from <em>Rysto
</em> STS markers for high‐throughput marker‐assisted selection of potato carrying <em>Rysto
</em>
. Phytopathology, 106(11), 1366–1375. 10.1094/PHYTO-05-16-0204-R

</cite> [<a href="https://doi.org/10.1094/PHYTO-05-16-0204-R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27442536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phytopathology&amp;title=Development%20and%20validation%20of%20high%E2%80%90resolution%20melting%20markers%20derived%20from%20Rysto%20STS%20markers%20for%20high%E2%80%90throughput%20marker%E2%80%90assisted%20selection%20of%20potato%20carrying%20Rysto&amp;author=X.%20Nie&amp;author=D.%20Sutherland&amp;author=V.%20Dickison&amp;author=M.%20Singh&amp;author=A.%20M.%20Murphy&amp;volume=106&amp;issue=11&amp;publication_year=2016&amp;pages=1366-1375&amp;pmid=27442536&amp;doi=10.1094/PHYTO-05-16-0204-R&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0048">
<cite>

Ortiz, R.
 (2020). Genomic‐led potato breeding for increasing genetic gains: Achievements and outlook. Crop Breeding, Genetics and Genomics, 2(2), e200010. 10.20900/cbgg20200010
</cite> [<a href="https://doi.org/10.20900/cbgg20200010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Breeding,%20Genetics%20and%20Genomics&amp;title=Genomic%E2%80%90led%20potato%20breeding%20for%20increasing%20genetic%20gains:%20Achievements%20and%20outlook&amp;author=R.%20Ortiz&amp;volume=2&amp;issue=2&amp;publication_year=2020&amp;pages=e200010&amp;doi=10.20900/cbgg20200010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0049">
<cite>

Perez, B. C.
, 
Bink, M. C. A. M.
, 
Svenson, K. L.
, 
Churchill, G. A.
, &amp; 
Calus, M. P. L.
 (2022). Adding gene transcripts into genomic prediction improves accuracy and reveals sampling time dependence. G3 Genes|Genomes|Genetics, 12(11), jkac258. 10.1093/g3journal/jkac258

</cite> [<a href="https://doi.org/10.1093/g3journal/jkac258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9635642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36161485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Adding%20gene%20transcripts%20into%20genomic%20prediction%20improves%20accuracy%20and%20reveals%20sampling%20time%20dependence&amp;author=B.%20C.%20Perez&amp;author=M.%20C.%20A.%20M.%20Bink&amp;author=K.%20L.%20Svenson&amp;author=G.%20A.%20Churchill&amp;author=M.%20P.%20L.%20Calus&amp;volume=12&amp;issue=11&amp;publication_year=2022&amp;pages=jkac258&amp;pmid=36161485&amp;doi=10.1093/g3journal/jkac258&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0050">
<cite>

Pérez, P.
, &amp; 
de los Campos, G.
 (2014). Genome‐wide regression and prediction with the BGLR statistical package. Genetics, 198(2), 483–495. 10.1534/genetics.114.164442

</cite> [<a href="https://doi.org/10.1534/genetics.114.164442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4196607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25009151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genome%E2%80%90wide%20regression%20and%20prediction%20with%20the%20BGLR%20statistical%20package&amp;author=P.%20P%C3%A9rez&amp;author=G.%20de%20los%20Campos&amp;volume=198&amp;issue=2&amp;publication_year=2014&amp;pages=483-495&amp;pmid=25009151&amp;doi=10.1534/genetics.114.164442&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0051">
<cite>

Prodhomme, C.
, 
Esselink, D.
, 
Borm, T.
, 
Visser, R. G. F.
, 
Van Eck, H. J.
, &amp; 
Vossen, J. H.
 (2019). Comparative subsequence sets analysis (CoSSA) is a robust approach to identify haplotype specific SNPs; mapping and pedigree analysis of a potato wart disease resistance gene <em>Sen3</em>
. Plant Methods, 15(1), Article 60. 10.1186/s13007-019-0445-5

</cite> [<a href="https://doi.org/10.1186/s13007-019-0445-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6540404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31160919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Methods&amp;title=Comparative%20subsequence%20sets%20analysis%20(CoSSA)%20is%20a%20robust%20approach%20to%20identify%20haplotype%20specific%20SNPs;%20mapping%20and%20pedigree%20analysis%20of%20a%20potato%20wart%20disease%20resistance%20gene%20Sen3&amp;author=C.%20Prodhomme&amp;author=D.%20Esselink&amp;author=T.%20Borm&amp;author=R.%20G.%20F.%20Visser&amp;author=H.%20J.%20Van%20Eck&amp;volume=15&amp;issue=1&amp;publication_year=2019&amp;pages=60&amp;pmid=31160919&amp;doi=10.1186/s13007-019-0445-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0052"><cite>
R Core Team
. (2022). 
CRAN mirrors
. CRAN. <a href="https://cran.r-project.org/mirrors.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r‐project.org/mirrors.html</a>
</cite></li>
<li id="tpg270082-bib-0053">
<cite>

Rieman, G. H.
 (1962). Superior: A new white, medium‐maturing, scabresistant potato variety with high chipping quality. American Potato Journal, 39(1), 19–28. 10.1007/BF02912628
</cite> [<a href="https://doi.org/10.1007/BF02912628" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Potato%20Journal&amp;title=Superior:%20A%20new%20white,%20medium%E2%80%90maturing,%20scabresistant%20potato%20variety%20with%20high%20chipping%20quality&amp;author=G.%20H.%20Rieman&amp;volume=39&amp;issue=1&amp;publication_year=1962&amp;pages=19-28&amp;doi=10.1007/BF02912628&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0054">
<cite>

Rodríguez‐Álvarez, M. X.
, 
Boer, M. P.
, 
Van Eeuwijk, F. A.
, &amp; 
Eilers, P. H. C.
 (2018). Correcting for spatial heterogeneity in plant breeding experiments with P‐splines. Spatial Statistics, 23, 52–71. 10.1016/j.spasta.2017.10.003
</cite> [<a href="https://doi.org/10.1016/j.spasta.2017.10.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Spatial%20Statistics&amp;title=Correcting%20for%20spatial%20heterogeneity%20in%20plant%20breeding%20experiments%20with%20P%E2%80%90splines&amp;author=M.%20X.%20Rodr%C3%ADguez%E2%80%90%C3%81lvarez&amp;author=M.%20P.%20Boer&amp;author=F.%20A.%20Van%20Eeuwijk&amp;author=P.%20H.%20C.%20Eilers&amp;volume=23&amp;publication_year=2018&amp;pages=52-71&amp;doi=10.1016/j.spasta.2017.10.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0078"><cite>
Rouse,  J. W., Haas,  R. H., Deering,  D. W., Schell,  J. A., &amp; Harlan,  J. C. (1974). Monitoring the vernal advancement and retrogradation (green wave effect) of natural vegetation.</cite></li>
<li id="tpg270082-bib-0055">
<cite>

Schmitz Carley, C. A.
, 
Coombs, J. J.
, 
Clough, M. E.
, 
De Jong, W. S.
, 
Douches, D. S.
, 
Haynes, K. G.
, 
Higgins, C. R.
, 
Holm, D. G.
, 
Miller, J. C.
, 
Navarro, F. M.
, 
Novy, R. G.
, 
Palta, J. P.
, 
Parish, D. L.
, 
Porter, G. A.
, 
Sathuvalli, V. R.
, 
Thompson, A. L.
, 
Yencho, G. C.
, 
Zotarelli, L.
, &amp; 
Endelman, J. B.
 (2019). Genetic covariance of environments in the Potato National Chip Processing Trial. Crop Science, 59(1), 107–114. 10.2135/cropsci2018.05.0314
</cite> [<a href="https://doi.org/10.2135/cropsci2018.05.0314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genetic%20covariance%20of%20environments%20in%20the%20Potato%20National%20Chip%20Processing%20Trial&amp;author=C.%20A.%20Schmitz%20Carley&amp;author=J.%20J.%20Coombs&amp;author=M.%20E.%20Clough&amp;author=W.%20S.%20De%20Jong&amp;author=D.%20S.%20Douches&amp;volume=59&amp;issue=1&amp;publication_year=2019&amp;pages=107-114&amp;doi=10.2135/cropsci2018.05.0314&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0056">
<cite>

Shakoor, N.
, 
Lee, S.
, &amp; 
Mockler, T. C.
 (2017). High throughput phenotyping to accelerate crop breeding and monitoring of diseases in the field. Current Opinion in Plant Biology, 38, 184–192. 10.1016/j.pbi.2017.05.006

</cite> [<a href="https://doi.org/10.1016/j.pbi.2017.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28738313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Plant%20Biology&amp;title=High%20throughput%20phenotyping%20to%20accelerate%20crop%20breeding%20and%20monitoring%20of%20diseases%20in%20the%20field&amp;author=N.%20Shakoor&amp;author=S.%20Lee&amp;author=T.%20C.%20Mockler&amp;volume=38&amp;publication_year=2017&amp;pages=184-192&amp;pmid=28738313&amp;doi=10.1016/j.pbi.2017.05.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0057">
<cite>

Silva, M. F. E.
, 
Maciel, G. M.
, 
Gallis, R. B.
, 
Barbosa, R. L.
, 
Carneiro, V. Q.
, 
Rezende, W. S.
, &amp; 
Siquieroli, A. C. S.
 (2022). High‐throughput phenotyping by RGB and multispectral imaging analysis of genotypes in sweet corn. Horticultura Brasileira, 40, 92–98. 10.1590/s0102-0536-2022012
</cite> [<a href="https://doi.org/10.1590/s0102-0536-2022012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticultura%20Brasileira&amp;title=High%E2%80%90throughput%20phenotyping%20by%20RGB%20and%20multispectral%20imaging%20analysis%20of%20genotypes%20in%20sweet%20corn&amp;author=M.%20F.%20E.%20Silva&amp;author=G.%20M.%20Maciel&amp;author=R.%20B.%20Gallis&amp;author=R.%20L.%20Barbosa&amp;author=V.%20Q.%20Carneiro&amp;volume=40&amp;publication_year=2022&amp;pages=92-98&amp;doi=10.1590/s0102-0536-2022012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0058">
<cite>

Singh, V.
, 
Krause, M.
, 
Sandhu, D.
, 
Sekhon, R. S.
, &amp; 
Kaundal, A.
 (2023). Salinity stress tolerance prediction for biomass‐related traits in maize (<em>Zea mays</em> L.) using genome‐wide markers. The Plant Genome, 16, e20385. 10.1002/tpg2.20385

</cite> [<a href="https://doi.org/10.1002/tpg2.20385" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37667417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Salinity%20stress%20tolerance%20prediction%20for%20biomass%E2%80%90related%20traits%20in%20maize%20(Zea%20mays%20L.)%20using%20genome%E2%80%90wide%20markers&amp;author=V.%20Singh&amp;author=M.%20Krause&amp;author=D.%20Sandhu&amp;author=R.%20S.%20Sekhon&amp;author=A.%20Kaundal&amp;volume=16&amp;publication_year=2023&amp;pages=e20385&amp;pmid=37667417&amp;doi=10.1002/tpg2.20385&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0059">
<cite>

Slater, A. T.
, 
Cogan, N. O. I.
, 
Forster, J. W.
, 
Hayes, B. J.
, &amp; 
Daetwyler, H. D.
 (2016). Improving genetic gain with genomic selection in autotetraploid potato. The Plant Genome, 9(3), plantgenome2016.02.0021. 10.3835/plantgenome2016.02.0021
</cite> [<a href="https://doi.org/10.3835/plantgenome2016.02.0021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27902807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Improving%20genetic%20gain%20with%20genomic%20selection%20in%20autotetraploid%20potato&amp;author=A.%20T.%20Slater&amp;author=N.%20O.%20I.%20Cogan&amp;author=J.%20W.%20Forster&amp;author=B.%20J.%20Hayes&amp;author=H.%20D.%20Daetwyler&amp;volume=9&amp;issue=3&amp;publication_year=2016&amp;pmid=27902807&amp;doi=10.3835/plantgenome2016.02.0021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0060">
<cite>

Song, P.
, 
Wang, J.
, 
Guo, X.
, 
Yang, W.
, &amp; 
Zhao, C.
 (2021). High‐throughput phenotyping: Breaking through the bottleneck in future crop breeding. The Crop Journal, 9(3), 633–645. 10.1016/j.cj.2021.03.015
</cite> [<a href="https://doi.org/10.1016/j.cj.2021.03.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Crop%20Journal&amp;title=High%E2%80%90throughput%20phenotyping:%20Breaking%20through%20the%20bottleneck%20in%20future%20crop%20breeding&amp;author=P.%20Song&amp;author=J.%20Wang&amp;author=X.%20Guo&amp;author=W.%20Yang&amp;author=C.%20Zhao&amp;volume=9&amp;issue=3&amp;publication_year=2021&amp;pages=633-645&amp;doi=10.1016/j.cj.2021.03.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0061">
<cite>

Sood, S.
, 
Bhardwaj, V.
, 
Kaushik, S. K.
, &amp; 
Sharma, S.
 (2020). Prediction based on estimated breeding values using genealogy for tuber yield and late blight resistance in auto‐tetraploid potato (<em>Solanum tuberosum</em> L.). Heliyon, 6(11), e05624. 10.1016/j.heliyon.2020.e05624

</cite> [<a href="https://doi.org/10.1016/j.heliyon.2020.e05624" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7710635/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33305041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heliyon&amp;title=Prediction%20based%20on%20estimated%20breeding%20values%20using%20genealogy%20for%20tuber%20yield%20and%20late%20blight%20resistance%20in%20auto%E2%80%90tetraploid%20potato%20(Solanum%20tuberosum%20L.)&amp;author=S.%20Sood&amp;author=V.%20Bhardwaj&amp;author=S.%20K.%20Kaushik&amp;author=S.%20Sharma&amp;volume=6&amp;issue=11&amp;publication_year=2020&amp;pages=e05624&amp;pmid=33305041&amp;doi=10.1016/j.heliyon.2020.e05624&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0062">
<cite>

Stefaniak, T. R.
, 
Fitzcollins, S.
, 
Figueroa, R.
, 
Thompson, A. L.
, 
Schmitz Carley, C.
, &amp; 
Shannon, L. M.
 (2021). Genotype and variable nitrogen effects on tuber yield and quality for red fresh market potatoes in Minnesota. Agronomy, 11(2), 255. 10.3390/agronomy11020255
</cite> [<a href="https://doi.org/10.3390/agronomy11020255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=Genotype%20and%20variable%20nitrogen%20effects%20on%20tuber%20yield%20and%20quality%20for%20red%20fresh%20market%20potatoes%20in%20Minnesota&amp;author=T.%20R.%20Stefaniak&amp;author=S.%20Fitzcollins&amp;author=R.%20Figueroa&amp;author=A.%20L.%20Thompson&amp;author=C.%20Schmitz%20Carley&amp;volume=11&amp;issue=2&amp;publication_year=2021&amp;pages=255&amp;doi=10.3390/agronomy11020255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0063">
<cite>

Stich, B.
, &amp; 
Van Inghelandt, D.
 (2018). Prospects and potential uses of genomic prediction of key performance traits in tetraploid potato. Frontiers in Plant Science, 9, 159. 10.3389/fpls.2018.00159

</cite> [<a href="https://doi.org/10.3389/fpls.2018.00159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5845909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29563919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Prospects%20and%20potential%20uses%20of%20genomic%20prediction%20of%20key%20performance%20traits%20in%20tetraploid%20potato&amp;author=B.%20Stich&amp;author=D.%20Van%20Inghelandt&amp;volume=9&amp;publication_year=2018&amp;pages=159&amp;pmid=29563919&amp;doi=10.3389/fpls.2018.00159&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0064">
<cite>

Su, W.‐H.
, &amp; 
Xue, H.
 (2021). Imaging spectroscopy and machine learning for intelligent determination of potato and sweet potato quality. Foods, 10(9), 2146. 10.3390/foods10092146

</cite> [<a href="https://doi.org/10.3390/foods10092146" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8472741/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34574253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Foods&amp;title=Imaging%20spectroscopy%20and%20machine%20learning%20for%20intelligent%20determination%20of%20potato%20and%20sweet%20potato%20quality&amp;author=W.%E2%80%90H.%20Su&amp;author=H.%20Xue&amp;volume=10&amp;issue=9&amp;publication_year=2021&amp;pages=2146&amp;pmid=34574253&amp;doi=10.3390/foods10092146&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0065">
<cite>

Tuttle, H. K.
, 
Del Rio, A. H.
, 
Bamberg, J. B.
, &amp; 
Shannon, L. M.
 (2024). Potato soup: Analysis of cultivated potato gene bank populations reveals high diversity and little structure. Frontiers in Plant Science, 15, 1429279. 10.3389/fpls.2024.1429279

</cite> [<a href="https://doi.org/10.3389/fpls.2024.1429279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11291250/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39091313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Potato%20soup:%20Analysis%20of%20cultivated%20potato%20gene%20bank%20populations%20reveals%20high%20diversity%20and%20little%20structure&amp;author=H.%20K.%20Tuttle&amp;author=A.%20H.%20Del%20Rio&amp;author=J.%20B.%20Bamberg&amp;author=L.%20M.%20Shannon&amp;volume=15&amp;publication_year=2024&amp;pages=1429279&amp;pmid=39091313&amp;doi=10.3389/fpls.2024.1429279&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0066">
<cite>

van Eck, H. J.
, 
Oortwijn, M. E. P.
, 
Terpstra, I. R.
, 
van Lieshout, N. H. M.
, 
van der Knaap, E.
, 
Willemsen, J. H.
, &amp; 
Bachem, C. W. B.
 (2022). Engineering of tuber shape in potato (<em>Solanum tuberosum</em>) with marker assisted breeding or genetic modification using StOFP20. <em>Research Square</em>. 10.21203/rs.3.rs-1807189/v1
</cite> [<a href="https://doi.org/10.21203/rs.3.rs-1807189/v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270082-bib-0067">
<cite>

VanRaden, P. M.
 (2008). Efficient methods to compute genomic predictions. Journal of Dairy Science, 91(11), 4414–4423. 10.3168/jds.2007-0980

</cite> [<a href="https://doi.org/10.3168/jds.2007-0980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18946147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Dairy%20Science&amp;title=Efficient%20methods%20to%20compute%20genomic%20predictions&amp;author=P.%20M.%20VanRaden&amp;volume=91&amp;issue=11&amp;publication_year=2008&amp;pages=4414-4423&amp;pmid=18946147&amp;doi=10.3168/jds.2007-0980&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0068">
<cite>

Vos, P. G.
, 
Uitdewilligen, J. G. A. M. L.
, 
Voorrips, R. E.
, 
Visser, R. G. F.
, &amp; 
van Eck, H. J.
 (2015). Development and analysis of a 20K SNP array for potato (<em>Solanum tuberosum</em>): An insight into the breeding history. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 128(12), 2387–2401. 10.1007/s00122-015-2593-y

</cite> [<a href="https://doi.org/10.1007/s00122-015-2593-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4636999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26263902/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Development%20and%20analysis%20of%20a%2020K%20SNP%20array%20for%20potato%20(Solanum%20tuberosum):%20An%20insight%20into%20the%20breeding%20history&amp;author=P.%20G.%20Vos&amp;author=J.%20G.%20A.%20M.%20L.%20Uitdewilligen&amp;author=R.%20E.%20Voorrips&amp;author=R.%20G.%20F.%20Visser&amp;author=H.%20J.%20van%20Eck&amp;volume=128&amp;issue=12&amp;publication_year=2015&amp;pages=2387-2401&amp;pmid=26263902&amp;doi=10.1007/s00122-015-2593-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0069">
<cite>

Webb, R. E.
, 
Wilson, D. R.
, 
Shumaker, J. R.
, 
Graves, B.
, 
Henninger, M. R.
, 
Watts, J.
, 
Frank, J. A.
, &amp; 
Murphy, H. J.
 (1978). Atlantic: A new potato variety with high solids, good processing quality, and resistance to pests. American Potato Journal, 55(3), 141–145. 10.1007/BF02852087
</cite> [<a href="https://doi.org/10.1007/BF02852087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Potato%20Journal&amp;title=Atlantic:%20A%20new%20potato%20variety%20with%20high%20solids,%20good%20processing%20quality,%20and%20resistance%20to%20pests&amp;author=R.%20E.%20Webb&amp;author=D.%20R.%20Wilson&amp;author=J.%20R.%20Shumaker&amp;author=B.%20Graves&amp;author=M.%20R.%20Henninger&amp;volume=55&amp;issue=3&amp;publication_year=1978&amp;pages=141-145&amp;doi=10.1007/BF02852087&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0070">
<cite>

Weigle, J. L.
, 
Kehr, A. E.
, 
Akeley, R. V.
, &amp; 
Horton, J. C.
 (1968). Chieftain: A red‐skinned potato with attractive appearance and broad adaptability. American Potato Journal, 45(8), 293–296. 10.1007/BF02850285
</cite> [<a href="https://doi.org/10.1007/BF02850285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Potato%20Journal&amp;title=Chieftain:%20A%20red%E2%80%90skinned%20potato%20with%20attractive%20appearance%20and%20broad%20adaptability&amp;author=J.%20L.%20Weigle&amp;author=A.%20E.%20Kehr&amp;author=R.%20V.%20Akeley&amp;author=J.%20C.%20Horton&amp;volume=45&amp;issue=8&amp;publication_year=1968&amp;pages=293-296&amp;doi=10.1007/BF02850285&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0071">
<cite>

Xu, S.
, 
Xu, X.
, 
Zhu, Q.
, 
Meng, Y.
, 
Yang, G.
, 
Feng, H.
, 
Yang, M.
, 
Zhu, Q.
, 
Xue, H.
, &amp; 
Wang, B.
 (2023). Monitoring leaf nitrogen content in rice based on information fusion of multi‐sensor imagery from UAV. Precision Agriculture, 24, 2327–2349. 10.1007/s11119-023-10042-8
</cite> [<a href="https://doi.org/10.1007/s11119-023-10042-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Precision%20Agriculture&amp;title=Monitoring%20leaf%20nitrogen%20content%20in%20rice%20based%20on%20information%20fusion%20of%20multi%E2%80%90sensor%20imagery%20from%20UAV&amp;author=S.%20Xu&amp;author=X.%20Xu&amp;author=Q.%20Zhu&amp;author=Y.%20Meng&amp;author=G.%20Yang&amp;volume=24&amp;publication_year=2023&amp;pages=2327-2349&amp;doi=10.1007/s11119-023-10042-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0072">
<cite>

Yang, H.
, 
Hu, Y.
, 
Zheng, Z.
, 
Qiao, Y.
, 
Zhang, K.
, 
Guo, T.
, &amp; 
Chen, J.
 (2022). Estimation of potato chlorophyll content from UAV multispectral images with stacking ensemble algorithm. Agronomy, 12(10), 2318. 10.3390/agronomy12102318
</cite> [<a href="https://doi.org/10.3390/agronomy12102318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=Estimation%20of%20potato%20chlorophyll%20content%20from%20UAV%20multispectral%20images%20with%20stacking%20ensemble%20algorithm&amp;author=H.%20Yang&amp;author=Y.%20Hu&amp;author=Z.%20Zheng&amp;author=Y.%20Qiao&amp;author=K.%20Zhang&amp;volume=12&amp;issue=10&amp;publication_year=2022&amp;pages=2318&amp;doi=10.3390/agronomy12102318&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0073">
<cite>

Ye, S.
, 
Li, J.
, &amp; 
Zhang, Z.
 (2020). Multi‐omics‐data‐assisted genomic feature markers preselection improves the accuracy of genomic prediction. Journal of Animal Science and Biotechnology, 11(1), Article 109. 10.1186/s40104-020-00515-5

</cite> [<a href="https://doi.org/10.1186/s40104-020-00515-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7708144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33292577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Animal%20Science%20and%20Biotechnology&amp;title=Multi%E2%80%90omics%E2%80%90data%E2%80%90assisted%20genomic%20feature%20markers%20preselection%20improves%20the%20accuracy%20of%20genomic%20prediction&amp;author=S.%20Ye&amp;author=J.%20Li&amp;author=Z.%20Zhang&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=109&amp;pmid=33292577&amp;doi=10.1186/s40104-020-00515-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0074">
<cite>

Yu, H.
, 
Jiang, S.
, &amp; 
Land, K. C.
 (2015). Multicollinearity in hierarchical linear models. Social Science Research, 53, 118–136. 10.1016/j.ssresearch.2015.04.008

</cite> [<a href="https://doi.org/10.1016/j.ssresearch.2015.04.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26188442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Social%20Science%20Research&amp;title=Multicollinearity%20in%20hierarchical%20linear%20models&amp;author=H.%20Yu&amp;author=S.%20Jiang&amp;author=K.%20C.%20Land&amp;volume=53&amp;publication_year=2015&amp;pages=118-136&amp;pmid=26188442&amp;doi=10.1016/j.ssresearch.2015.04.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0075">
<cite>

Yusuf, M.
, 
Miller, M. D.
, 
Stefaniak, T. R.
, 
Haagenson, D.
, 
Endelman, J. B.
, 
Thompson, A. L.
, &amp; 
Shannon, L. M.
 (2024). Genomic prediction for potato (<em>Solanum tuberosum</em>) quality traits improved through image analysis. The Plant Genome, 17(4), e20507. 10.1002/tpg2.20507

</cite> [<a href="https://doi.org/10.1002/tpg2.20507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11628938/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39256988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20prediction%20for%20potato%20(Solanum%20tuberosum)%20quality%20traits%20improved%20through%20image%20analysis&amp;author=M.%20Yusuf&amp;author=M.%20D.%20Miller&amp;author=T.%20R.%20Stefaniak&amp;author=D.%20Haagenson&amp;author=J.%20B.%20Endelman&amp;volume=17&amp;issue=4&amp;publication_year=2024&amp;pages=e20507&amp;pmid=39256988&amp;doi=10.1002/tpg2.20507&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0076">
<cite>

Zhou, Z.
, 
Jabloun, M.
, 
Plauborg, F.
, &amp; 
Andersen, M. N.
 (2018). Using ground‐based spectral reflectance sensors and photography to estimate shoot N concentration and dry matter of potato. Computers and Electronics in Agriculture, 144, 154–163. 10.1016/j.compag.2017.12.005
</cite> [<a href="https://doi.org/10.1016/j.compag.2017.12.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Computers%20and%20Electronics%20in%20Agriculture&amp;title=Using%20ground%E2%80%90based%20spectral%20reflectance%20sensors%20and%20photography%20to%20estimate%20shoot%20N%20concentration%20and%20dry%20matter%20of%20potato&amp;author=Z.%20Zhou&amp;author=M.%20Jabloun&amp;author=F.%20Plauborg&amp;author=M.%20N.%20Andersen&amp;volume=144&amp;publication_year=2018&amp;pages=154-163&amp;doi=10.1016/j.compag.2017.12.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270082-bib-0077">
<cite>

Zhu, X.
, 
Leiser, W. L.
, 
Hahn, V.
, &amp; 
Würschum, T.
 (2021). Phenomic selection is competitive with genomic selection for breeding of complex traits. Plant Phenome Journal, 4(1), e20027. 10.1002/ppj2.20027
</cite> [<a href="https://doi.org/10.1002/ppj2.20027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Phenome%20Journal&amp;title=Phenomic%20selection%20is%20competitive%20with%20genomic%20selection%20for%20breeding%20of%20complex%20traits&amp;author=X.%20Zhu&amp;author=W.%20L.%20Leiser&amp;author=V.%20Hahn&amp;author=T.%20W%C3%BCrschum&amp;volume=4&amp;issue=1&amp;publication_year=2021&amp;pages=e20027&amp;doi=10.1002/ppj2.20027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>
<strong>Table S1</strong>. Variable features based on combined stages versus selected.</p>
<p>
<strong>Figure S1</strong>. Broad‐sense heritability of multispectral bands and indices within the chips market.</p>
<p>
<strong>Figure S2</strong>. Broad‐sense heritability of multispectral bands and indices within the fresh market.</p>
<p>
<strong>Figure S3</strong>. Prediction ability for traits within the chips market clones across growth stages using multispectral bands and Indices.</p>
<p>
<strong>Figure S4</strong>. Prediction ability for traits within the fresh market clones across growth stages using multispectral bands and Indices.</p>
<p>
<strong>Figure S5</strong>. Prediction ability for 2022 using 2023 within the chips market clones across growth stages using multispectral bands and Indices.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12290324/bin/TPG2-18-e70082-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70082-s001.pdf</a><sup> (4MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data and scripts used for this work are available at: <a href="https://github.com/shannonlabumn/GS_multispectra_analysis.git" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/shannonlabumn/GS_multispectra_analysis.git</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70082"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70082.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12290324/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12290324/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12290324%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12290324/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12290324/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12290324/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40708258/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12290324/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40708258/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12290324/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12290324/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="9zQQ9YMG6scZfoElqnjiAN4jWu89MMZb1IISO20bISzNu3NsbeurmshoUjmooqOc">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
