
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            GGCRB: A Graph Neural Network Approach for Predicting CircRNA–RBP Interactions Using Structural and Sequence Features - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="67D7131F89B72E431C131F001FD3651C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsomega">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332793/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Omega">
<meta name="citation_title" content="GGCRB: A Graph Neural Network Approach for Predicting CircRNA–RBP Interactions Using Structural and Sequence Features">
<meta name="citation_author" content="Guangyi Tang">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Harbin University of Science and Technology, Harbin 150080, China">
<meta name="citation_author" content="Hongyuan Xing">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Harbin University of Science and Technology, Harbin 150080, China">
<meta name="citation_author" content="Dengju Yao">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Harbin University of Science and Technology, Harbin 150080, China">
<meta name="citation_author" content="Xiaojuan Zhan">
<meta name="citation_author_institution" content="College of Computer Science and Technology, Heilongjiang Institute of Technology, Harbin 150050, China">
<meta name="citation_author" content="Xiangkui Li">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Harbin University of Science and Technology, Harbin 150080, China">
<meta name="citation_publication_date" content="2025 Jul 22">
<meta name="citation_volume" content="10">
<meta name="citation_issue" content="30">
<meta name="citation_firstpage" content="33662">
<meta name="citation_doi" content="10.1021/acsomega.5c04524">
<meta name="citation_pmid" content="40787315">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332793/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332793/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332793/pdf/ao5c04524.pdf">
<meta name="description" content="The interaction between circular RNAs (circRNAs) and RNA-binding proteins (RBPs) plays a crucial role in gene regulation; however, experimental identification is costly and inefficient. Current computational methods often overlook the structural ...">
<meta name="og:title" content="GGCRB: A Graph Neural Network Approach for Predicting CircRNA–RBP Interactions Using Structural and Sequence Features">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The interaction between circular RNAs (circRNAs) and RNA-binding proteins (RBPs) plays a crucial role in gene regulation; however, experimental identification is costly and inefficient. Current computational methods often overlook the structural ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332793/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332793">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsomega.5c04524"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ao5c04524.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332793%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332793/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332793/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332793/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsomega.png" alt="ACS Omega logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Omega" title="Link to ACS Omega" shape="default" href="https://pubs.acs.org/journal/acsodf" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Omega</button></div>. 2025 Jul 22;10(30):33662–33674. doi: <a href="https://doi.org/10.1021/acsomega.5c04524" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsomega.5c04524</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Omega%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Omega%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>GGCRB: A Graph
Neural Network Approach for Predicting
CircRNA–RBP Interactions Using Structural and Sequence Features</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Guangyi Tang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Guangyi Tang</span></h3>
<div class="p">
<sup>1</sup>
School of
Computer Science and Technology, Harbin
University of Science and Technology, Harbin
150080, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Guangyi Tang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xing%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Hongyuan Xing</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Hongyuan Xing</span></h3>
<div class="p">
<sup>1</sup>
School of
Computer Science and Technology, Harbin
University of Science and Technology, Harbin
150080, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xing%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hongyuan Xing</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yao%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Dengju Yao</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Dengju Yao</span></h3>
<div class="p">
<sup>1</sup>
School of
Computer Science and Technology, Harbin
University of Science and Technology, Harbin
150080, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yao%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dengju Yao</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhan%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Xiaojuan Zhan</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Xiaojuan Zhan</span></h3>
<div class="p">
<sup>2</sup>
College of
Computer Science and Technology, Heilongjiang
Institute of Technology, Harbin
150050, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhan%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaojuan Zhan</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Xiangkui Li</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Xiangkui Li</span></h3>
<div class="p">
<sup>1</sup>
School of
Computer Science and Technology, Harbin
University of Science and Technology, Harbin
150080, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiangkui Li</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>
School of
Computer Science and Technology, Harbin
University of Science and Technology, Harbin
150080, China
</div>
<div id="aff2">
<sup>2</sup>
College of
Computer Science and Technology, Heilongjiang
Institute of Technology, Harbin
150050, China
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>ydkvictory@hrbust.edu.cn</span>.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 15; Accepted 2025 Jul 9; Revised 2025 Jun 20; Collection date 2025 Aug 5.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332793  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40787315/" class="usa-link">40787315</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The interaction between
circular RNAs (circRNAs) and
RNA-binding
proteins (RBPs) plays a crucial role in gene regulation; however,
experimental identification is costly and inefficient. Current computational
methods often overlook the structural features of circRNAs, thereby
limiting prediction accuracy. To address these challenges, we propose
GGCRB, a deep learning framework that integrates both sequence and
structural features for predicting circRNA–RBP binding sites.
Sequence features are captured through five encoding schemes (HFN,
ND, NCP, DPCP, and Doc2Vec), followed by convolutional layers for
local pattern extraction. Structural features are derived from base-pairing
adjacency matrices generated by RNAstructure and modeled using graph
convolutional networks and graph attention networks to learn topological
dependencies. The fused representations are further processed by bidirectional
LSTM and multihead attention modules to capture global interactions.
Final predictions are made through pooling and softmax layers. Extensive
experiments on 16 benchmark data sets demonstrate that GGCRB significantly
outperforms existing models. Ablation studies and motif analyses further
confirm its effectiveness, underscoring the importance of integrating
structural and sequence information for accurate prediction of circRNA–RBP
interactions.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/18ab/12332793/95aaf75ae4f2/ao5c04524_0009.jpg" loading="lazy" id="abs1" height="558" width="529" alt="graphic file with name ao5c04524_0009.jpg"></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/18ab/12332793/95aaf75ae4f2/ao5c04524_0007.jpg" loading="lazy" id="tgr1" height="558" width="529" alt="graphic file with name ao5c04524_0007.jpg"></p></section><section id="sec1"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>CircRNAs feature covalently
sealed circular architectures.<em>
,
</em> They possess disease-modulating
expression profiles and functionally
impact multiple disorders, especially cancers,<em>
−

</em> cardiovascular
diseases, neurological disorders, and autoimmune diseases. The unique structure of circRNAs enhances stability and protein
binding, influencing gene expression and cell processes. As key regulators of RNA metabolism, RBPs orchestrate
RNA maturation, cellular trafficking, subcellular positioning, and
translational control. Emerging evidence
highlights the regulatory supremacy of circRNA-associated proteins
in balancing circRNA biogenesis and catabolism. Furthermore, such
interactions exhibit pleiotropic effects, spanning from translational
regulation to the modulation of pathogenic cascades in various disorders. Although circRNAs have fewer binding sites for
RNA-binding proteins (RBPs) than linear mRNAs, strong evidence supports
their interactions. Initially regarded
as a splicing byproduct with limited importance, circRNA, a type of
noncoding RNA, has gained attention due to discoveries enabled by
high-throughput sequencing. These advances have promoted computational
research on circRNA–protein interactions, disease links, and
drug development. Consequently, precise
identification of circRNA–RBP binding sites serves dual purposes:
elucidating their uncharacterized biological roles while simultaneously
revealing novel molecular markers for clinical diagnostics and therapeutic
development.<em>
,
</em>
</p>
<p>The gold-standard experimental
pipelines for circRNA–RBP
interaction analysis typically involve antibody-based RNA enrichment
(RIP), cross-linking-coupled immunoprecipitation (such as CLIP variants), and nucleic acid–protein binding detection
via gel shift assays (e.g., EMSA). With their respective technical
merits, these experimental strategies have become indispensable for
elucidating the nature of RNA–protein associations and precisely
localizing binding domains. As an immunocapture-based approach, RIP
enables the systematic isolation and identification of naturally occurring
RNA–protein assemblies within cells.<em>
,
</em> EMSA represents a robust biochemical technique that exploits electrophoretic
separation principles to detect and quantify interactions between
proteins and nucleic acids. CLIP-Seq is a widely used technique. Despite their utility,
these experimental approaches frequently require substantial time
investment, intensive manual operations, and significant financial
resources, posing scalability limitations for high-throughput analyses.
Advancements in deep learning architectures are increasingly demonstrating
their viability for supplementing traditional experimental paradigms
in this research domain. Zhang et al. introduced the CRIP model, employing superimposed codon-based sequence
encoding coupled with a hybrid architecture integrating convolutional
and recurrent neural networks to extract and represent discriminative
features. Jia et al. developed the PASSION framework, employing sequential feature refinement with XGBoost-based
selection to identify the most discriminative feature combinations
from six distinct circRNA representation approaches, subsequently
enhancing predictive accuracy via an integrated deep learning architecture.
Yang et al. developed HCRNet, incorporating
a novel deep temporal convolution architecture (DTCN) for precise
localization of circRNA–RBP binding interfaces and interaction
pattern characterization. Niu et al. proposed CRBPDL, which employs an AdaBoost-based ensemble deep network integrating
five feature encoding methods alongside MSRN and BiGRU to effectively
capture both local and global sequence contexts for predicting circRNA–RBP
binding sites. Liu et al. introduced circ-FHN, employing quadruple feature encoding strategies for sequence
representation, coupled with a CNN-BiGRU fusion architecture designed
to capture both local patterns and extended sequential dependencies
through bidirectional gated mechanisms. Lasantha et al. developed
CRIECNN, a hybrid architecture combining
convolutional layers, bidirectional LSTM, and attention-based feature
optimization to predict protein binding patterns across complete circRNA
transcripts and their structural subunits.</p>
<p>Although computational
methods for identifying circRNA–protein
binding sites have made notable progress, several limitations remain
unresolved. Most existing approaches primarily rely on the nucleotide
sequence of circRNAs, and current feature encoding techniques often
neglect structural information, thereby constraining model performance.
In recent years, the development of next-generation RNA three-dimensional
(3D) conformation prediction methods has provided new opportunities
for incorporating structural features.<em>
−

</em> However, such methods
typically require substantial computational resources and manual intervention,
particularly when analyzing long-chain RNAs or molecules with complex
spatial folding. Due to challenges related
to sequence length, resolution, and cost, these 3D-based approaches
are difficult to scale for large-scale applications. Therefore, integrating more accessible secondary structure
information on circRNAs may help improve the accuracy of interaction
prediction.</p>
<p>To predict the binding sites of circular RNAs and
proteins, we
developed the GGCRB model, which effectively integrates structural
and sequence features through graph convolution and attention mechanisms,
enhancing the feature representation capability. This method combines
nucleotide sequence information with RNA secondary structure features
to achieve a comprehensive characterization of multiscale features
of circular RNAs. In sequence feature extraction, we introduced five
complementary encoding strategies: HNF, ND, NCP, DPCP, and Doc2Vec
to enhance the expressiveness of sequence information. Structural
features were extracted using the RNAstructure tool. To fully exploit
the advanced features of structural information, we designed a hybrid
learning framework combining graph convolutional networks (GCN) and
graph attention networks (GAT) and used convolutional neural networks
(CNN) to extract advanced features of sequences. The fused multiscale
features were further modeled by BiLSTM to capture sequence patterns
and context dependencies, and a multihead attention layer was used
for adaptive feature weighting to capture global dependencies. The
model was evaluated on 16 benchmark data sets using five-fold cross-validation,
achieving an overall AUC of 0.9634. The results demonstrate that GGCRB
outperforms existing methods in the task of circular RNA–protein
binding site prediction, proving its effectiveness.</p>
<p>In this
study, we propose a graph neural network framework that
integrates sequence and structural features for predicting RBP binding
sites on circRNA. The main contributions are summarized as follows:</p>
<ul class="list" style="list-style-type:none">
<li>
<span class="label">1.</span><p class="display-inline">We design a multiscale
feature fusion
approach that integrates both sequence and structural features of
circRNAs. This overcomes the limitations of single-modality representation
in previous methods and enhances the model’s ability to capture
complex biological information.</p>
</li>
<li>
<span class="label">2.</span><p class="display-inline">We construct diverse sequence feature
matrices using multiple encoding schemes, including physicochemical
properties, <em>k</em>-mer features, and pretrained embeddings.
In parallel, we extract structure feature matrices from RNA secondary
structures, considering base pairing and thermodynamic properties,
thus enriching the model input with both local and global structural
contexts.</p>
</li>
<li>
<span class="label">3.</span><p class="display-inline">We develop
a hybrid graph neural network
combining GCN and GAT to learn topological dependencies from circRNA
structure graphs. This enables effective modeling of intricate node–edge
relationships and improves the prediction accuracy of circRNA–RBP
interaction sites.</p>
</li>
</ul></section><section id="sec2"><h2 class="pmc_sec_title">2. Materials
and Methods</h2>
<p>GGCRB integrates
three specialized componentssequence feature
extraction, structure feature extraction, feature fusion and predictionas
detailed in <a href="#fig1" class="usa-link">Figure 
</a>. The model first extracts sequence features using multiple encoding
methods and simultaneously processes RNA secondary structure information
via GCN and GAT. These multiscale features are then fused and passed
through convolutional layers (Conv1D), bidirectional LSTM, and multihead
attention mechanisms to accurately predict circRNA–RBP binding
sites.</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332793_ao5c04524_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/18ab/12332793/d0fb6560a43d/ao5c04524_0001.jpg" loading="lazy" id="gr1" height="389" width="700" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Flowchart of the proposed method. First, circRNA sequences are
collected and divided into training and testing sets. Then, sequence
features are extracted using five encoding methods (HFN, ND, NCP,
DPCP, and Doc2Vec) and concatenated. At the same time, structure features
are obtained by transforming RNA secondary structures into base-pairing
adjacency matrices and processed through GCN and GAT networks. Finally,
the extracted sequence and structure features are fused and passed
through Conv1D, BiLSTM, and multihead attention layers for prediction
of circRNA–RBP binding sites.</p></figcaption></figure><section id="sec2.1"><h3 class="pmc_sec_title">2.1. Data Sets</h3>
<p>Sixteen circRNA–RBP
interaction data sets are collected from the circRNA–protein
interaction database to evaluate the
performance of GGCRB. CD-HIT is employed
to remove redundant sequences from the circRNA data set, ensuring
nonrepetitive data for downstream analysis. A 101-base pair genomic
segment is isolated, spanning 50 nucleotides in both directions from
the CLIP-seq peak summit. Binding-positive
examples are obtained from high-confidence CLIP-seq peaks, with binding-negative
examples generated through random selection from nonpeak circRNA segments.
A balanced 1:1 ratio between binding-positive and binding-negative
instances is maintained across all data sets to prevent class imbalance.
In the 16 data sets, there are 189,620 positive and negative samples,
with each data set containing up to 40,000 circRNA sequence samples.
Detailed information about the 16 data sets can be found in <a href="#tbl1" class="usa-link">Table 
</a>. All data sets are
partitioned into an 80–20 split, with the majority allocated
for model training and the reserved subset for performance evaluation.</p>
<section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">1. Characteristics of the 16 Benchmark
Data Sets.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">RBP</th>
<th align="center" colspan="1" rowspan="1">size</th>
<th align="center" colspan="1" rowspan="1">positive numbers</th>
<th align="center" colspan="1" rowspan="1">negative numbers</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">AUF1</td>
<td align="left" colspan="1" rowspan="1">5792</td>
<td align="left" colspan="1" rowspan="1">2896</td>
<td align="left" colspan="1" rowspan="1">2896</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DGCR8</td>
<td align="left" colspan="1" rowspan="1">40,000</td>
<td align="left" colspan="1" rowspan="1">20,000</td>
<td align="left" colspan="1" rowspan="1">20,000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">EWSR1</td>
<td align="left" colspan="1" rowspan="1">9390</td>
<td align="left" colspan="1" rowspan="1">4695</td>
<td align="left" colspan="1" rowspan="1">4695</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HNRNPC</td>
<td align="left" colspan="1" rowspan="1">28,448</td>
<td align="left" colspan="1" rowspan="1">14,224</td>
<td align="left" colspan="1" rowspan="1">14,224</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MOV10</td>
<td align="left" colspan="1" rowspan="1">11,778</td>
<td align="left" colspan="1" rowspan="1">5889</td>
<td align="left" colspan="1" rowspan="1">5889</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TDP43</td>
<td align="left" colspan="1" rowspan="1">10,968</td>
<td align="left" colspan="1" rowspan="1">5484</td>
<td align="left" colspan="1" rowspan="1">5484</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FMRP</td>
<td align="left" colspan="1" rowspan="1">40,000</td>
<td align="left" colspan="1" rowspan="1">20,000</td>
<td align="left" colspan="1" rowspan="1">20,000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HUR</td>
<td align="left" colspan="1" rowspan="1">40,000</td>
<td align="left" colspan="1" rowspan="1">20,000</td>
<td align="left" colspan="1" rowspan="1">20,000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IGF2BP1</td>
<td align="left" colspan="1" rowspan="1">40,000</td>
<td align="left" colspan="1" rowspan="1">20,000</td>
<td align="left" colspan="1" rowspan="1">20,000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PTB</td>
<td align="left" colspan="1" rowspan="1">40,000</td>
<td align="left" colspan="1" rowspan="1">20,000</td>
<td align="left" colspan="1" rowspan="1">20,000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IGF2BP3</td>
<td align="left" colspan="1" rowspan="1">40,000</td>
<td align="left" colspan="1" rowspan="1">20,000</td>
<td align="left" colspan="1" rowspan="1">20,000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">U2AF65</td>
<td align="left" colspan="1" rowspan="1">16,448</td>
<td align="left" colspan="1" rowspan="1">8224</td>
<td align="left" colspan="1" rowspan="1">8224</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PUM2</td>
<td align="left" colspan="1" rowspan="1">4658</td>
<td align="left" colspan="1" rowspan="1">2829</td>
<td align="left" colspan="1" rowspan="1">2829</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TAF15</td>
<td align="left" colspan="1" rowspan="1">2934</td>
<td align="left" colspan="1" rowspan="1">1467</td>
<td align="left" colspan="1" rowspan="1">1467</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FXR2</td>
<td align="left" colspan="1" rowspan="1">11,270</td>
<td align="left" colspan="1" rowspan="1">5635</td>
<td align="left" colspan="1" rowspan="1">5635</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LIN28A</td>
<td align="left" colspan="1" rowspan="1">36,554</td>
<td align="left" colspan="1" rowspan="1">18,277</td>
<td align="left" colspan="1" rowspan="1">18,277</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">sum</td>
<td align="left" colspan="1" rowspan="1">378,240</td>
<td align="left" colspan="1" rowspan="1">189,620</td>
<td align="left" colspan="1" rowspan="1">189,620</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec2.2"><h3 class="pmc_sec_title">2.2. Sequence
Feature Extraction</h3>
<p>HNF combines
nucleotide codon spatial superposition
encoding and <em>k</em>-mer encoding methods to comprehensively
capture the local contextual sequence information. HNF uses <em>k</em>-mer encoding for single nucleotides and dinucleotides
and codon stacking encoding for trinucleotides. <em>k</em>-Mer encoding (<em>k</em> = 1 or 2) extracts local sequence patterns by representing contiguous nucleotide
groups, thereby preserving neighborhood relationships within the sequence.
Consider a circRNA sequence <em>S</em> = (<em>c</em>
<sub>1</sub>, <em>c</em>
<sub>2</sub>, <em>c</em>
<sub>3</sub>, ... <em>c</em>
<sub>
<em>l</em>
</sub>) of
fixed length <em>l</em> = 101, where each <em>c</em>
<sub>
<em>i</em>
</sub> denotes a nucleotide position. The
sequence is encoded as 20-dimensional one-hot vectors (combining monomer
and dimer features). This procedure can be formally expressed by</p>
<table class="disp-formula p" id="eq1"><tr>
<td class="formula"><math id="d1e586" display="block"><msub><mi>X</mi><mi>k</mi></msub><mo>=</mo><mo stretchy="false">[</mo><msubsup><mi>x</mi><mn>1</mn><mi>k</mi></msubsup><mo>,</mo><msubsup><mi>x</mi><mrow><mn>2</mn><mo>,</mo></mrow><mi>k</mi></msubsup><mo>,</mo><mspace width="0.25em"></mspace><mo>.</mo><mo>.</mo><mo>.</mo><mo>,</mo><mspace width="0.25em"></mspace><msubsup><mi>x</mi><mi>n</mi><mi>k</mi></msubsup><mo stretchy="false">]</mo></math></td>
<td class="label">1</td>
</tr></table>
<p>Specifically, single-nucleotide
(one-mer) encoding generates a matrix of size <em>N</em>
<sub>1</sub> × 4, dinucleotide (two-mer) encoding generates a matrix
of size <em>N</em>
<sub>2</sub> × 16, and trinucleotide
(three-mer) encoding generates a matrix of size <em>N</em>
<sub>3</sub> × 21, where <em>N</em> is the unified sequence
length after preprocessing. Through feature matrix concatenation,
we generate an <em>N</em> × 41 integrated representation
that comprehensively encodes circRNA’s sequential properties.
The entries of the matrix satisfy</p>
<table class="disp-formula p" id="eq2"><tr>
<td class="formula"><math id="d1e643" display="block"><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><msub><mi>X</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>=</mo><mrow><mo stretchy="true">{</mo><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><mn>1</mn><mo>,</mo><mspace width="0.25em"></mspace><mi>if</mi><mspace width="0.25em"></mspace><msub><mi>c</mi><mi>i</mi></msub><mo>=</mo><msub><mi>j</mi><mrow><mi>t</mi><mi>h</mi></mrow></msub></mtd></mtr><mtr><mtd><mn>0</mn><mo>,</mo><mspace width="0.25em"></mspace><mi>else</mi></mtd></mtr></mtable></mstyle></mrow></mtd></mtr></mtable></mstyle></math></td>
<td class="label">2</td>
</tr></table>
<p>The subscript <em>i</em> specifies
the ordinal position
of nucleotides along the circRNA sequence <em>S</em>, and <em>j</em>
<sub>
<em>th</em>
</sub> is from the set {<em>A</em>, <em>G</em>, <em>C</em>, <em>U</em>} (mononucleotide), {<em>AA</em>, <em>AC</em>, ..., <em>UU</em>} (dinucleotide), and {<em>AAA</em>, <em>AAC</em>, ..., <em>UUU</em>} (trinucleotides). <em>X</em>
<sub>
<em>i</em>
</sub> captures nucleotide units of size 1–3
at site <em>c</em>
<sub>
<em>i</em>
</sub>. For the standard
101-nt circRNA, this produces a 41-dimensional feature vector per
position, resulting in a 101 × 41 matrix.</p>
<p>Nucleotide density
(ND) encoding is
a feature representation method that combines nucleotide position
information with frequency information to describe the distribution
characteristics of nucleotides in a circRNA sequence. Given a circRNA
sequence <em>S</em> = {<em>s</em>
<sub>1</sub>, <em>s</em>
<sub>2</sub>, <em>s</em>
<sub>3</sub>, ..., <em>s</em>
<sub>
<em>l</em>
</sub>}, where <em>s</em>
<sub>
<em>i</em>
</sub> ∈ {<em>A</em>, <em>G</em>, <em>C</em>, <em>U</em>} denotes the nucleotide
at position <em>i</em>, with <em>l</em> being the
sequence length, the core idea of this method is to calculate the
density value of each nucleotide at a specific position to reflect
its distribution weight within the entire sequence. The nucleotide
density <em>d</em>
<sub>
<em>i</em>
</sub> at position <em>i</em> is formally expressed by</p>
<table class="disp-formula p" id="eq3"><tr>
<td class="formula"><math id="d1e805" display="block"><msub><mi>d</mi><mi>i</mi></msub><mo>=</mo><mfrac><mn>1</mn><mrow><mo>∥</mo><msub><mi>S</mi><mi>i</mi></msub><mo>∥</mo></mrow></mfrac><munderover accent="false" accentunder="false"><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>p</mi></munderover><mi>f</mi><mo stretchy="false">(</mo><msub><mi>S</mi><mi>j</mi></msub><mo stretchy="false">)</mo></math></td>
<td class="label">3</td>
</tr></table>
<p>where ∥<em>S</em>
<sub>
<em>i</em>
</sub>∥ represents the length of the
subsequence from the beginning of the sequence (the first nucleotide)
to the current position (the <em>i</em>th nucleotide), which
is used to normalize the nucleotide density and ensure consistency
in the features. <em>p</em> represents the range of the cumulative
window, i.e., the cardinality of nucleotide positions encompassed
by the density metric. <em>f</em>(<em>S</em>
<sub>
<em>j</em>
</sub>) is an indicator function used to check whether
the nucleotide at position <em>j</em> is equal to the target
nucleotide <em>q</em>:</p>
<table class="disp-formula p" id="eq4"><tr>
<td class="formula"><math id="d1e867" display="block"><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><mi>f</mi><mo stretchy="false">(</mo><mi>q</mi><mo stretchy="false">)</mo><mo>=</mo><mrow><mo stretchy="true">{</mo><mrow><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><mn>1</mn><mo>,</mo><mspace width="0.25em"></mspace><mi>if</mi><mspace width="0.25em"></mspace><msub><mi>s</mi><mrow><mi>j</mi><mo>=</mo></mrow></msub><mi>q</mi></mtd></mtr><mtr><mtd><mn>0</mn><mo>,</mo><mspace width="0.25em"></mspace><mi>else</mi></mtd></mtr></mtable></mstyle><mspace width="0.25em"></mspace></mrow></mrow><mi>q</mi><mo>∈</mo><mo stretchy="false">{</mo><mi>A</mi><mo>,</mo><mi>G</mi><mo>,</mo><mi>C</mi><mo>,</mo><mi>U</mi><mo stretchy="false">}</mo><mspace width="0.25em"></mspace><mi>and</mi><mspace width="0.25em"></mspace><mi>j</mi><mo>∈</mo><mn>1</mn><mo>,</mo><mspace width="0.25em"></mspace><mn>2</mn><mo>,</mo><mspace width="0.25em"></mspace><mn>3</mn><mo>,</mo><mspace width="0.25em"></mspace><mo>.</mo><mo>.</mo><mo>.</mo><mo>,</mo><mspace width="0.25em"></mspace><mi>l</mi></mtd></mtr></mtable></mstyle></math></td>
<td class="label">4</td>
</tr></table>
<p>That is, if the current
nucleotide <em>s</em>
<sub>
<em>i</em>
</sub> equals
the target nucleotide <em>q</em>,
then <em>f</em>(<em>q</em>) = 1; otherwise, it is
0. Using the above formula, the density information for each nucleotide
at different positions can be computed. For example, a circRNA sequence <em>S</em> = <em>GAAUCUACU</em> with length <em>l</em> = 9 is considered. For the target nucleotide <em>q</em> = <em>A</em>, we calculate the density values <em>d</em>
<sub>
<em>i</em>
</sub> for each position. At position 3, the subsequence
is <em>GAA</em>, where the target nucleotide <em>A</em> appears two times and the subsequence length is 3, so <em>d</em>
<sub>3</sub> = 0.67. At position 6, the subsequence is <em>GAAUCU</em>, where the target nucleotide <em>A</em> appears two times
and the subsequence length is 6, so <em>d</em>
<sub>6</sub> = 0.33. By performing similar calculations for each position, we
can obtain the density matrix for the entire sequence. The same method
can be applied to calculate densities for other nucleotides (<em>G</em>, <em>C</em>, <em>U</em>), ultimately generating
a 9 × 4 density feature matrix.</p>
<p>The dinucleotide physicochemical
property (DPCP) is a method for characterizing
RNA sequences by extracting
the physical and chemical properties of dinucleotides. DPCP provides
an efficient way to gain insight into the structure and stability
of nucleic acid molecules by quantifying the spatial and energetic
properties of dinucleotides. It contains 11 key physicochemical properties, which specifically include rise, translation,
slide, tilt, stacking energy, roll, torsion, enthalpy change, entropy
change, hydrophilicity, and free energy. The values of DPCP<sub>
<em>i</em>
</sub>(<em>j</em>) are provided in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04524/suppl_file/ao5c04524_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supplementary Table S1</a>. The formula for DPCP
is given below:</p>
<table class="disp-formula p" id="eq5"><tr>
<td class="formula"><math id="d1e1032" display="block"><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><mi>DPCP</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo>=</mo><mi>freq</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo>×</mo><msub><mi>DPCP</mi><mi>i</mi></msub><mo stretchy="false">(</mo><mi>j</mi><mo stretchy="false">)</mo></mtd></mtr></mtable></mstyle></math></td>
<td class="label">5</td>
</tr></table>
<p>where freq­(<em>i</em>) represents
the frequency of the <em>i</em>th dinucleotide
pair, i.e., the frequency of the <em>i</em>th dinucleotide
pair appearing in the given sequence. DPCP<sub>
<em>i</em>
</sub>(<em>j</em>) represents the value of the physicochemical
property (e.g., stacking energy, rise, shift, etc.) associated with
the <em>i</em>th dinucleotide pair. This value corresponds
to a specific physicochemical property of the dinucleotide pair. DPCP­(<em>i</em>) represents the weighted physicochemical property value,
which is obtained by multiplying the frequency of the <em>i</em>th dinucleotide with its corresponding physicochemical property value.
DPCP results in a 101 × 11 feature matrix.</p>
<p>Structural variations
among circRNA nucleotides confer distinct
biochemical functionalities, influencing their biological roles. Based on the chemical properties of these nucleotides,
we have classified them into three major groups, each with unique
physical and chemical properties. These three groups are categorized
as follows:</p>
<table class="disp-formula p" id="eq6"><tr>
<td class="formula"><math id="d1e1095" display="block"><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><mi>ring</mi><mspace width="0.25em"></mspace><mi>structure</mi><mo>=</mo><mrow><mo stretchy="true">{</mo><mrow><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><mi>purine</mi><mo>,</mo><mi>A</mi><mo>,</mo><mi>G</mi></mtd></mtr><mtr><mtd><mi>pyrimidine</mi><mo>,</mo><mi>C</mi><mo>,</mo><mi>U</mi></mtd></mtr></mtable></mstyle><mrow></mrow></mrow></mrow></mtd></mtr><mtr><mtd><mi>functional</mi><mspace width="0.25em"></mspace><mi>group</mi><mo>=</mo><mrow><mo stretchy="true">{</mo><mrow><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><mi mathvariant="normal">a</mi><mi>min</mi><mi mathvariant="normal">o</mi><mo>,</mo><mi>A</mi><mo>,</mo><mi>C</mi></mtd></mtr><mtr><mtd><mi>keto</mi><mo>,</mo><mi>G</mi><mo>,</mo><mi>U</mi></mtd></mtr></mtable></mstyle><mrow></mrow></mrow></mrow></mtd></mtr><mtr><mtd><mi>hydrogen</mi><mspace width="0.25em"></mspace><mi>bond</mi><mo>=</mo><mrow><mo stretchy="true">{</mo><mrow><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><mi>weak</mi><mo>,</mo><mi>A</mi><mo>,</mo><mi>U</mi></mtd></mtr><mtr><mtd><mi>strong</mi><mo>,</mo><mi>G</mi><mo>,</mo><mi>C</mi></mtd></mtr></mtable></mstyle><mrow></mrow></mrow></mrow></mtd></mtr></mtable></mstyle></math></td>
<td class="label">6</td>
</tr></table>
<p>Based on the above
classification, we define a three-dimensional
coordinate system for each nucleotide to describe the chemical properties
in sequence analysis. Specifically, each nucleotide is encoded as
three binary attributes (<em>x</em>
<sub>
<em>i</em>
</sub>, <em>y</em>
<sub>
<em>i</em>
</sub>, <em>z</em>
<sub>
<em>i</em>
</sub>), where the values depend
on the category to which the nucleotide belongs. The encoding rules
are as follows:</p>
<table class="disp-formula p" id="eq7"><tr>
<td class="formula"><math id="d1e1207" display="block"><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><msub><mi>x</mi><mi>i</mi></msub><mo>=</mo><mrow><mo stretchy="true">{</mo><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><mn>1</mn><mo>,</mo><mspace width="0.25em"></mspace><mi>if</mi><mspace width="0.25em"></mspace><msub><mi>N</mi><mi>i</mi></msub><mo>∈</mo><mo stretchy="false">{</mo><mi>A</mi><mo>,</mo><mi>G</mi><mo stretchy="false">}</mo></mtd></mtr><mtr><mtd><mn>0</mn><mo>,</mo><mspace width="0.25em"></mspace><mi>if</mi><mspace width="0.25em"></mspace><msub><mi>N</mi><mi>i</mi></msub><mo>∈</mo><mo stretchy="false">{</mo><mi>C</mi><mo>,</mo><mi>U</mi><mo stretchy="false">}</mo></mtd></mtr></mtable></mstyle></mrow></mtd></mtr><mtr><mtd><msub><mi>y</mi><mi>i</mi></msub><mo>=</mo><mrow><mo stretchy="true">{</mo><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><mn>1</mn><mo>,</mo><mspace width="0.25em"></mspace><mi>if</mi><mspace width="0.25em"></mspace><msub><mi>N</mi><mi>i</mi></msub><mo>∈</mo><mo stretchy="false">{</mo><mi>A</mi><mo>,</mo><mi>C</mi><mo stretchy="false">}</mo></mtd></mtr><mtr><mtd><mn>0</mn><mo>,</mo><mspace width="0.25em"></mspace><mi>if</mi><mspace width="0.25em"></mspace><msub><mi>N</mi><mi>i</mi></msub><mo>∈</mo><mo stretchy="false">{</mo><mi>G</mi><mo>,</mo><mi>U</mi><mo stretchy="false">}</mo></mtd></mtr></mtable></mstyle></mrow></mtd></mtr><mtr><mtd><msub><mi>z</mi><mi>i</mi></msub><mo>=</mo><mrow><mo stretchy="true">{</mo><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><mn>1</mn><mo>,</mo><mspace width="0.25em"></mspace><mi>if</mi><mspace width="0.25em"></mspace><msub><mi>N</mi><mi>i</mi></msub><mo>∈</mo><mo stretchy="false">{</mo><mi>A</mi><mo>,</mo><mi>U</mi><mo stretchy="false">}</mo></mtd></mtr><mtr><mtd><mn>0</mn><mo>,</mo><mspace width="0.25em"></mspace><mi>if</mi><mspace width="0.25em"></mspace><msub><mi>N</mi><mi>i</mi></msub><mo>∈</mo><mo stretchy="false">{</mo><mi>C</mi><mo>,</mo><mi>G</mi><mo stretchy="false">}</mo></mtd></mtr></mtable></mstyle></mrow></mtd></mtr></mtable></mstyle></math></td>
<td class="label">7</td>
</tr></table>
<p>In this three-dimensional
coordinate system, nucleotides {<em>A</em>, <em>G</em>, <em>C</em>, <em>U</em>} are uniquely encoded as
three-bit vectors: (1,1,1), (1,0,0), (0,1,0),
and (0,0,1) respectively. Using this encoding scheme, we assign a
three-dimensional vector to each nucleotide, allowing us to quantify
its differences in various physicochemical properties. By applying
these encodings to each nucleotide in the circRNA sequence, we construct
a feature matrix of size (101 × 3), where each row corresponds
to a nucleotide in the sequence, and each column represents its binary
values across the three different categories.</p>
<p>Doc2Vec is a natural language processing
technique for converting text, sequences, or documents into fixed-length
dense vector representations. Doc2Vec is widely used for long sequence
feature extraction and representation learning. Compared with Word2Vec’s
distributed representation for words only, Doc2Vec is able to learn
semantic features of complete sequences, which is suitable for higher-level
feature analysis. In Doc2Vec, each sequence is treated as a separate
“document”. The algorithm captures local and global
semantic features by encoding the contextual information in the sequence
as numeric vectors and generates a fixed-dimensional representation.
Input sequences of variable lengths are transformed into output vectors
with uniform dimensionality, standardizing feature space for ML processing.
Doc2Vec adopts the process of partitioning a circRNA sequence into
fixed-length fragments (e.g., 101-nucleotide sequences). Each sequence
fragment is treated as a document, and the context of the nucleotide
fragment is used as the “word”. The distribution of
nucleotides in the sequence is characterized by Doc2Vec, which embeds
the sequence fragments as fixed-dimensional feature vectors. After
training, the model is used to transform all circRNA sequences into
vector representations that serve as inputs to downstream classification
or regression models. Compared with traditional sequence representation
methods, Doc2Vec generates length-agnostic sequence embeddings by
modeling global contextual relationships via distributed representations,
enabling robust feature extraction.</p></section><section id="sec2.3"><h3 class="pmc_sec_title">2.3. Structure
Feature Extraction</h3>
<p>In this
work, we employ RNAstructure to determine
circRNA secondary structures, generating a 101 × 101 feature
matrix that captures local and global RNA structural characteristics.
The 101-nt circRNA fragments contain 50-nt sequences on both sides
of the target binding region. This length setting ensures that the
contextual information surrounding the binding site is sufficiently
captured, while avoiding redundancy or computational overhead caused
by longer sequences. We preprocess the circRNA sequences and utilize
RNAstructure to predict their two-dimensional folding configurations.
The algorithm employs MFE as the optimization criterion to compute
nucleotide pairing probabilities and energy variations. The predicted
results are typically output as a CT (connectivity table) file.</p>
<p>The CT file describes the positions of nucleotides in the sequence
and the index of each nucleotide’s paired base (if there is
no pairing, the index is 0). For example, given a 101-nucleotide circRNA
sequence, the RNAstructure tool will output a data table with 101
rows, where each row records the specific pairing information on a
nucleotide. Given a circRNA sequence <em>S</em> = {<em>N</em>
<sub>1</sub>, <em>N</em>
<sub>2</sub>, ..., <em>N</em>
<sub>
<em>n</em>
</sub>} of length <em>n</em>, where nucleotides <em>N</em>
<sub>
<em>i</em>
</sub> take values from {<em>A</em>, <em>U</em>, <em>C</em>, <em>G</em>}, RNAstructure produces the pairing
set <em>P</em> = {(<em>i</em>, <em>j</em>)|where <em>N</em>
<sub>
<em>i</em>
</sub> pairs with <em>N</em>
<sub>
<em>j</em>
</sub>}. Following this, the adjacency matrix <em>A</em> can be formally defined:</p>
<table class="disp-formula p" id="eq8"><tr>
<td class="formula"><math id="d1e1454" display="block"><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><mi>A</mi><mo stretchy="false">[</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="false">]</mo><mo>=</mo><mrow><mo stretchy="true">{</mo><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><mn>1</mn><mo>,</mo><mspace width="0.25em"></mspace><mi>if</mi><mspace width="0.25em"></mspace><mo stretchy="false">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="false">)</mo><mo>∈</mo><mi>P</mi><mspace width="0.25em"></mspace><mi>or</mi><mspace width="0.25em"></mspace><mo stretchy="false">(</mo><mi>j</mi><mo>,</mo><mi>i</mi><mo stretchy="false">)</mo><mo>∈</mo><mi>P</mi></mtd></mtr><mtr><mtd><mn>0</mn><mo>,</mo><mspace width="0.25em"></mspace><mi>else</mi></mtd></mtr></mtable></mstyle></mrow></mtd></mtr></mtable></mstyle></math></td>
<td class="label">8</td>
</tr></table>
<p>The constructed adjacency
matrix <em>A</em> is an <em>n</em> × <em>n</em> sparse matrix. When <em>a</em>
<sub>
<em>ij</em>
</sub> = 1, this denotes a base-pairing
interaction between nucleotides at positions <em>i</em> and <em>j</em>.</p></section><section id="sec2.4"><h3 class="pmc_sec_title">2.4. Feature Fusion and Prediction</h3>
<section id="sec2.4.1"><h4 class="pmc_sec_title">2.4.1. Convolutional Neural Networks</h4>
<p>CNN
are a powerful tool widely used in deep learning, particularly well-suited
for autonomously extracting feature representations from data. Via convolutional operations, CNN hierarchically
extracts features from input data, enabling robust identification
of localized patterns and intricate nonlinear correlations. The CNN
in this study is mainly used for the extraction of sequence features,
where the sequence coding matrix of cyclic RNA (circRNA) is processed
by a one-dimensional convolution operation to extract potential feature
representations. The matrix input to the CNN is <em>H</em>, which has a size of <em>m</em> × <em>l</em>, where <em>m</em> denotes the total dimension of the features,
which are generated by multiple coding methods (e.g., HNF, DPCP, ND,
and NCP), and the dimensions of the features for each coding method
are 4141, 1111, 101, and 303, respectively, for a total of <em>m</em> = 56. The circRNA fragment has a length (<em>L</em>) of 101 nucleotides. As a key element in CNNs, the 1D convolution
operation facilitates the capture of local sequence patterns. If the
convolution kernel has a length of <em>h</em>
<sub>
<em>r</em>
</sub>, then it slides from <em>H</em>’s
first <em>l</em> columns, yielding an output feature per position.
The convolution formula is as follows:</p>
<table class="disp-formula p" id="eq9"><tr>
<td class="formula"><math id="d1e1578" display="block"><mstyle lineleading=".25ex" rowspacing="2ex"><mtable displaystyle="true"><mtr><mtd><msubsup><mi>y</mi><mi>i</mi><mi>j</mi></msubsup><mo>=</mo><mi>ReLU</mi><mo stretchy="false">(</mo><msub><mi>X</mi><mi>i</mi></msub><mo>×</mo><mi>W</mi><mo stretchy="false">[</mo><mo>:</mo><mo>,</mo><mo>:</mo><mo>,</mo><mi>j</mi><mo stretchy="false">]</mo><mo>+</mo><msub><mi>b</mi><mi>j</mi></msub><mo stretchy="false">)</mo></mtd></mtr></mtable></mstyle></math></td>
<td class="label">9</td>
</tr></table>
<p>where <em>X</em>
<sub>
<em>i</em>
</sub> is the submatrix consisting of columns <em>i</em> through <em>i</em> + <em>h</em>
<sub>
<em>r</em>
</sub> – 1 of the original input matrix <em>H</em>. The weight matrix <em>W</em> and bias term <em>b</em> are associated with each convolutional filter. Here, <em>j</em> indicates the filter index, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e1648" display="inline"><msubsup><mi>y</mi><mi>i</mi><mi>j</mi></msubsup></math>
</span> corresponds to the output generated by
the <em>j</em>th filter at location <em>i</em>.</p></section><section id="sec2.4.2"><h4 class="pmc_sec_title">2.4.2. Graph Convolutional Networks</h4>
<p>This
research employs GCN to derive circRNA’s topological characteristics
by analyzing the RNA adjacency matrix and identity matrix, enabling
the capture of nucleotide interactions and overall structural patterns
through graph-based feature aggregation. Matrix <em>A</em> captures base-pairing interactions within the RNA secondary structure,
configured as a 101 × 101 sparse matrix. The unit matrix <em>I</em> serves as the initial feature of each node and is used
to ensure that the node maintains self-information (i.e., the node’s
own features) during computation at each level, rather than relying
exclusively on the features of its neighbors. The matrix size is also
101 × 101. By adding self-loops, nodes can integrate both neighborhood
information and their intrinsic features during message passing. The
basic operation of the GCN is graph convolution, which is the updating
of each node’s features using the node’s local neighborhood
information. The formula for each layer is as follows:</p>
<table class="disp-formula p" id="eq10"><tr>
<td class="formula"><math id="d1e1675" display="block"><msup><mi>H</mi><mrow><mo stretchy="false">(</mo><mi>l</mi><mo>+</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></msup><mo>=</mo><mi>ReLU</mi><mo stretchy="false">(</mo><msup><mi>D̃</mi><mrow><mo>−</mo><mn>1</mn><mo>/</mo><mn>2</mn></mrow></msup><mi>Ã</mi><msup><mi>D̃</mi><mrow><mo>−</mo><mn>1</mn><mo>/</mo><mn>2</mn></mrow></msup><msup><mi>H</mi><mrow><mo stretchy="false">(</mo><mi>l</mi><mo stretchy="false">)</mo></mrow></msup><msup><mi>W</mi><mrow><mo stretchy="false">(</mo><mi>l</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo></math></td>
<td class="label">10</td>
</tr></table>
<p>where <em>H</em>
<sup>(<em>l</em>)</sup> denotes the
layer <em>l</em> node
feature matrix, initially set as the identity matrix <em>I</em>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e1743" display="inline"><mi>Ã</mi></math>
</span> = <em>A</em> + <em>I</em> joins the adjacency matrix of the self-loop, indicating that each
node can also use its own information. <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e1755" display="inline"><mi>D̃</mi></math>
</span> represents the degree matrix of nodes,
while <em>W</em>
<sup>(<em>l</em>)</sup> corresponds
to the trainable weight matrix at layer <em>l</em>.</p></section><section id="sec2.4.3"><h4 class="pmc_sec_title">2.4.3. Graph Attention Networks</h4>
<p>GAT employs
multihead attention mechanisms to dynamically compute varying importance
scores for neighboring nodes in graph-structured data. In this study, GAT is used to further extract
RNA structural features, addressing the limitation of uniform weights
in GCN and emphasizing the importance of specific structures. Unlike
GCN, which uses a fixed normalized adjacency matrix, GAT utilizes
attention-based computation to determine the significance of adjacent
nodes, allowing for a more flexible focus on relationships between
key bases. The GAT layer performs secondary processing on the initial
node features extracted by GCN, further enriching the node representations,
especially by assigning higher weights to critical regions of the
RNA (such as important pairing sites). GAT’s attention mechanism
computes varying attention coefficients for each node’s local
neighborhood, reflecting the relative significance of neighboring
features. The calculation formula for the weight of each edge is as
follows:</p>
<table class="disp-formula p" id="eq11"><tr>
<td class="formula"><math id="d1e1780" display="block"><msub><mi>a</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>exp</mi><mo stretchy="false">(</mo><mi>leaky</mi><mspace width="0.25em"></mspace><mi>ReLU</mi><mo stretchy="false">(</mo><msup><mi>a</mi><mi>T</mi></msup><mo stretchy="false">[</mo><mi>W</mi><msub><mi>h</mi><mi>i</mi></msub><mo>∥</mo><mi>W</mi><msub><mi>h</mi><mi>j</mi></msub><mo stretchy="false">]</mo><mo stretchy="false">)</mo><mo stretchy="false">)</mo></mrow><mrow><msub><mo>∑</mo><mrow><mi>k</mi><mo>∈</mo><msub><mi>N</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msub></mrow></msub><mi>exp</mi><mo stretchy="false">(</mo><mi>leaky</mi><mspace width="0.25em"></mspace><mi>ReLU</mi><mo stretchy="false">(</mo><msup><mi>a</mi><mi>T</mi></msup><mo stretchy="false">[</mo><mi>W</mi><msub><mi>h</mi><mi>i</mi></msub><mo>∥</mo><mi>W</mi><msub><mi>h</mi><mi>k</mi></msub><mo stretchy="false">]</mo><mo stretchy="false">)</mo><mo stretchy="false">)</mo></mrow></mfrac></math></td>
<td class="label">11</td>
</tr></table>
<p>
<em>h</em>
<sub>
<em>i</em>
</sub> and <em>h</em>
<sub>
<em>j</em>
</sub> correspond to the feature embeddings of
node <em>i</em> and node <em>j</em>, respectively. <em>W</em> serves as the learnable transformation matrix for node
features,
while <em>a</em> functions as the attention vector that computes
edge importance scores. ∥ denotes the vector concatenation
operation.</p>
<p>By applying attention weights, through feature aggregation,
the
central node’s embedding is updated based on its neighborhood
information, as expressed by the following formula:</p>
<table class="disp-formula p" id="eq12"><tr>
<td class="formula"><math id="d1e1893" display="block"><msubsup><mi>h</mi><mi>i</mi><mrow><mo stretchy="false">(</mo><mi>l</mi><mo>+</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></msubsup><mo>=</mo><mi>ReLU</mi><mrow><mo stretchy="true">(</mo><mrow><munder accentunder="false"><mo form="prefix">∑</mo><mrow><mi>j</mi><mo>∈</mo><msub><mi>N</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msub></mrow></munder><msub><mi>a</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mi>W</mi><msub><mi>h</mi><mi>j</mi></msub></mrow><mo stretchy="true">)</mo></mrow></math></td>
<td class="label">12</td>
</tr></table>
<p>The weight matrix <em>W</em> is a learnable component that
undergoes optimization during GAT training.</p>
<p>Enhancing model
stability, GAT employs multihead attention that
calculates several distinct attention patterns and combines their
outputs to produce more comprehensive features, as formulated below:</p>
<table class="disp-formula p" id="eq13"><tr>
<td class="formula"><math id="d1e1947" display="block"><msubsup><mi>h</mi><mi>i</mi><mrow><mo stretchy="false">(</mo><mi>l</mi><mo>+</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></msubsup><mo>=</mo><msubsup><mo>∥</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mi>K</mi></msubsup><mi>σ</mi><mrow><mo stretchy="true">(</mo><mrow><msub><mo>∑</mo><mrow><mi>j</mi><mo>∈</mo><msub><mi>N</mi><mrow><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msub></mrow></msub><msubsup><mi>a</mi><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mo stretchy="false">(</mo><mi>k</mi><mo stretchy="false">)</mo></mrow></msubsup><msup><mi>W</mi><mrow><mo stretchy="false">(</mo><mi>K</mi><mo stretchy="false">)</mo></mrow></msup><msubsup><mi>h</mi><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>l</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow><mo stretchy="true">)</mo></mrow></math></td>
<td class="label">13</td>
</tr></table>
<p>where <em>K</em> represents
the total count of attention heads, with the symbol ∥ indicating
feature vector concatenation.</p>
<p>The GCN and GAT hybrid architecture
effectively captures fundamental
RNA graph characteristics while dynamically weighting feature significance
via attention mechanisms, enabling comprehensive RNA structural representation.</p></section><section id="sec2.4.4"><h4 class="pmc_sec_title">2.4.4. BiLSTM</h4>
<p>BiLSTM is an important module
for further mining temporal dependencies and contextual information
from multiscale features. By capturing information from both the forward
and backward directions of the input features, BiLSTM can effectively
extract bidirectional dependencies within the features, providing
a more comprehensive contextual representation for downstream tasks.
BiLSTM considers information from both the preceding and succeeding
directions in a single time step to effectively learn the latent contextual
dependencies within circRNA sequences or feature matrices. Unlike
static feature extraction methods such as convolution, BiLSTM dynamically
captures the long-range dependencies within feature sequences, making
it suitable for biological sequences like RNA, which exhibit complex
dependency patterns. BiLSTM receives fused inputs from sequence features
(extracted by Conv1D) and structural features (extracted by GCN and
GAT), enabling further dynamic combination of these features.</p>
<p>The BiLSTM takes as input the multimodally fused sequence features
of dimensionality <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e2030" display="inline"><msup><mrow><mi mathvariant="double-struck">R</mi></mrow><mrow><mi>T</mi><mo>×</mo><mi>d</mi></mrow></msup></math>
</span>, with <em>T</em> corresponding
to the temporal steps and <em>d</em> denoting the feature
embedding size. Information is extracted through both the forward
and backward LSTMs.</p>
<p>In the forward direction, the LSTM unit
iteratively processes temporal
data starting from the initial to the terminal time step. The formula
is as follows:</p>
<table class="disp-formula p" id="eq14"><tr>
<td class="formula"><math id="d1e2051" display="block"><mover accent="true"><mrow><msub><mi>h</mi><mi>t</mi></msub></mrow><mo>→</mo></mover><mo>=</mo><mi>LSTM</mi><mo stretchy="false">(</mo><msub><mi>x</mi><mi>t</mi></msub><mo>,</mo><mover accent="true"><mrow><msub><mi>h</mi><mrow><mi>t</mi><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo>→</mo></mover><mo stretchy="false">)</mo></math></td>
<td class="label">14</td>
</tr></table>
<p>Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e2082" display="inline"><mover accent="true"><mrow><msub><mi>h</mi><mi>t</mi></msub></mrow><mo>→</mo></mover></math>
</span> represents the
current time step’s
hidden state, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e2093" display="inline"><mover accent="true"><mrow><msub><mi>h</mi><mrow><mi>t</mi><mo>−</mo><mn>1</mn></mrow></msub></mrow><mo>→</mo></mover></math>
</span> being its
immediate predecessor in the
sequence.</p>
<p>Operating in reverse chronological order, the backward
LSTM analyzes
the input sequence starting from the final time step back to the initial
one. The mathematical formulation is presented below:</p>
<table class="disp-formula p" id="eq15"><tr>
<td class="formula"><math id="d1e2109" display="block"><mover accent="true"><mrow><msub><mi>h</mi><mi>t</mi></msub></mrow><mo>←</mo></mover><mo>=</mo><mi>LSTM</mi><mo stretchy="false">(</mo><msub><mi>x</mi><mi>t</mi></msub><mo>,</mo><mover accent="true"><mrow><msub><mi>h</mi><mrow><mi>t</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow><mo>←</mo></mover><mo stretchy="false">)</mo></math></td>
<td class="label">15</td>
</tr></table>
<p>Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e2140" display="inline"><mover accent="true"><mrow><msub><mi>h</mi><mi>t</mi></msub></mrow><mo>←</mo></mover></math>
</span> captures the
system state at time step <em>t</em>, while <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e2154" display="inline"><mover accent="true"><mrow><msub><mi>h</mi><mrow><mi>t</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow><mo>←</mo></mover></math>
</span> represents its immediate successor in the
temporal sequence.</p>
<p>BiLSTM concatenates the forward and backward
hidden states to generate
a comprehensive bidirectional feature representation.</p>
<table class="disp-formula p" id="eq16"><tr>
<td class="formula"><math id="d1e2170" display="block"><msub><mi>h</mi><mi>t</mi></msub><mo>=</mo><mo stretchy="false">[</mo><mover accent="true"><mrow><msub><mi>h</mi><mi>t</mi></msub></mrow><mo>→</mo></mover><mi>;</mi><mover accent="true"><mrow><msub><mi>h</mi><mi>t</mi></msub></mrow><mo>←</mo></mover><mo stretchy="false">]</mo><mo>,</mo><mspace width="0.25em"></mspace><msub><mi>h</mi><mi>t</mi></msub><mo>∈</mo><msup><mi mathvariant="double-struck">R</mi><mrow><mn>2</mn><msub><mi>d</mi><mi>h</mi></msub></mrow></msup></math></td>
<td class="label">16</td>
</tr></table>
<p>Here, <em>h</em>
<sub>
<em>t</em>
</sub> represents the consolidated feature embedding
at time step <em>t</em>, encapsulating both local and global
contextual information.</p>
<p>The final output is a full sequence
of hidden state matrices:</p>
<table class="disp-formula p" id="eq17"><tr>
<td class="formula"><math id="d1e2223" display="block"><mi>H</mi><mo>=</mo><mo stretchy="false">[</mo><msub><mi>h</mi><mn>1</mn></msub><mo>,</mo><msub><mi>h</mi><mn>2</mn></msub><mo>,</mo><mo>.</mo><mo>.</mo><mo>.</mo><mo>,</mo><msub><mi>h</mi><mi>T</mi></msub><mo>,</mo><mo stretchy="false">]</mo><mo>,</mo><mspace width="0.25em"></mspace><mi>H</mi><mo>∈</mo><msup><mi mathvariant="double-struck">R</mi><mrow><mi>T</mi><mo>×</mo><mn>2</mn><msub><mi>d</mi><mi>h</mi></msub></mrow></msup></math></td>
<td class="label">17</td>
</tr></table>
<p>LSTM serves
as the
foundational component of BiLSTM. Given input
sequence <em>X</em> = [<em>x</em>
<sub>1</sub>, <em>x</em>
<sub>2</sub>, ..., <em>x</em>
<sub>
<em>n</em>
</sub>], the computation at time step <em>t</em> is defined
by</p>
<table class="disp-formula p" id="eq18"><tr>
<td class="formula"><math id="d1e2288" display="block"><msub><mi>f</mi><mi>t</mi></msub><mo>=</mo><mi>σ</mi><mo stretchy="false">(</mo><msub><mi>W</mi><mi>f</mi></msub><mo>·</mo><mo stretchy="false">[</mo><msub><mi>h</mi><mrow><mi>t</mi><mo>−</mo><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>x</mi><mi>t</mi></msub><mo stretchy="false">]</mo><mo>+</mo><msub><mi>b</mi><mi>f</mi></msub><mo stretchy="false">)</mo></math></td>
<td class="label">18</td>
</tr></table>
<table class="disp-formula p" id="eq19"><tr>
<td class="formula"><math id="d1e2325" display="block"><msub><mi>i</mi><mi>t</mi></msub><mo>=</mo><mi>σ</mi><mo stretchy="false">(</mo><msub><mi>W</mi><mi>i</mi></msub><mo>·</mo><mo stretchy="false">[</mo><msub><mi>h</mi><mrow><mi>t</mi><mo>−</mo><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>x</mi><mi>t</mi></msub><mo stretchy="false">]</mo><mo>+</mo><msub><mi>b</mi><mi>i</mi></msub><mo stretchy="false">)</mo></math></td>
<td class="label">19</td>
</tr></table>
<table class="disp-formula p" id="eq20"><tr>
<td class="formula"><math id="d1e2362" display="block"><msub><mi>C̃</mi><mi>t</mi></msub><mo>=</mo><mi>tanh</mi><mo stretchy="false">(</mo><msub><mi>W</mi><mi>C</mi></msub><mo>·</mo><mo stretchy="false">[</mo><msub><mi>h</mi><mrow><mi>t</mi><mo>−</mo><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>x</mi><mi>t</mi></msub><mo stretchy="false">]</mo><mo>+</mo><msub><mi>b</mi><mi>C</mi></msub><mo stretchy="false">)</mo></math></td>
<td class="label">20</td>
</tr></table>
<table class="disp-formula p" id="eq21"><tr>
<td class="formula"><math id="d1e2399" display="block"><msub><mi>C</mi><mi>t</mi></msub><mo>=</mo><msub><mi>f</mi><mi>t</mi></msub><mo>⊙</mo><msub><mi>C</mi><mrow><mi>t</mi><mo>−</mo><mn>1</mn></mrow></msub><mo>+</mo><msub><mi>i</mi><mi>t</mi></msub><mo>⊙</mo><msub><mi>C̃</mi><mi>t</mi></msub></math></td>
<td class="label">21</td>
</tr></table>
<table class="disp-formula p" id="eq22"><tr>
<td class="formula"><math id="d1e2426" display="block"><msub><mi>o</mi><mi>t</mi></msub><mo>=</mo><mi>σ</mi><mo stretchy="false">(</mo><msub><mi>W</mi><mi>o</mi></msub><mo>·</mo><mo stretchy="false">[</mo><msub><mi>h</mi><mrow><mi>t</mi><mo>−</mo><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>x</mi><mi>t</mi></msub><mo stretchy="false">]</mo><mo>+</mo><msub><mi>b</mi><mi>o</mi></msub><mo stretchy="false">)</mo></math></td>
<td class="label">22</td>
</tr></table>
<table class="disp-formula p" id="eq23"><tr>
<td class="formula"><math id="d1e2463" display="block"><msub><mi>h</mi><mi>t</mi></msub><mo>=</mo><msub><mi>o</mi><mi>t</mi></msub><mo>⊙</mo><mi>tanh</mi><mo stretchy="false">(</mo><msub><mi>C</mi><mi>t</mi></msub><mo stretchy="false">)</mo></math></td>
<td class="label">23</td>
</tr></table>
<p>Here, <em>f</em>
<sub>
<em>t</em>
</sub> represents
the forget gate’s activation
value, quantifying how much information from the prior time step is
preserved. <em>W</em>
<sub>
<em>f</em>
</sub> and <em>b</em>
<sub>
<em>f</em>
</sub> correspond to the learnable
parameters of the forget gate, specifically the transformation weights
and offset terms. <em>h</em>
<sub>
<em>t</em>–1</sub> represents the LSTM’s hidden state from the preceding time
step. <em>x</em>
<sub>
<em>t</em>
</sub> represents the
input feature vector at time step <em>t</em>. σ denotes
the activation function. <em>i</em>
<sub>
<em>t</em>
</sub> represents the input gate’s activation value, quantifying
how much new information is stored at time step <em>t</em>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e2530" display="inline"><msub><mi>C̃</mi><mi>t</mi></msub></math>
</span>represents the candidate
memory content
proposed for time step <em>t</em>, while <em>C</em>
<sub>
<em>t</em>
</sub> denotes the finalized memory state
at this time step. ⊙ denotes element-by-element multiplication. <em>o</em>
<sub>
<em>t</em>
</sub> represents the output gate’s
activation value, regulating the exposure degree of the cell state
at time step <em>t</em>. <em>h</em>
<sub>
<em>t</em>
</sub> represents the LSTM’s hidden state at time step <em>t</em>, demonstrating the network’s capability to model
distant temporal relationships.</p></section><section id="sec2.4.5"><h4 class="pmc_sec_title">2.4.5. Multihead
Attention</h4>
<p>This work employs
a multihead self-attention architecture to capture long-range interactions
in circRNA sequences. This approach enhances the model’s representational
power through parallel computation of attention distributions across
multiple projected feature subspaces. Specifically, the multiattention
mechanism receives the time step sequence feature output from BiLSTM
as input, computes the correlation between each time step and other
time steps, and thus generates a global sequence representation containing
contextual information. For a circRNA sequence of length 101, the
output of each attention head is spliced and linearly transformed
to generate new sequence features, providing richer feature representations
for the subsequent fully connected layer. Empirical evidence demonstrates
that the MHA architecture substantially strengthens the model’s
capacity for capturing distal sequence relationships, leading to superior
predictive outcomes.</p>
<p>The MHA architecture extends single-head
attention by first projecting input features <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e2573" display="inline"><mi>H</mi><mo>∈</mo><msup><mrow><mi mathvariant="double-struck">R</mi></mrow><mrow><mi>T</mi><mo>×</mo><mi>d</mi></mrow></msup></math>
</span> into triplets of query
(<em>Q</em>), key (<em>K</em>), and value (<em>V</em>) matrices.
Attention weights are derived through computing
similarity scores via <em>Q</em> and <em>K</em> dot
products, applying softmax normalization,
and parallel computation across <em>h</em> independent heads, each operating in distinct feature
subspaces. The mathematical formulation is</p>
<table class="disp-formula p" id="eq24"><tr>
<td class="formula"><math id="d1e2613" display="block"><mi>Q</mi><mo>=</mo><mi>H</mi><msup><mi>W</mi><mi>Q</mi></msup><mo>,</mo><mspace width="0.25em"></mspace><mi>K</mi><mo>=</mo><mi>H</mi><msup><mi>W</mi><mi>K</mi></msup><mo>,</mo><mspace width="0.25em"></mspace><mi>V</mi><mo>=</mo><mi>H</mi><msup><mi>W</mi><mi>V</mi></msup></math></td>
<td class="label">24</td>
</tr></table>
<table class="disp-formula p" id="eq25"><tr>
<td class="formula"><math id="d1e2642" display="block"><mi>attention</mi><mo stretchy="false">(</mo><mi>Q</mi><mo>,</mo><mi>K</mi><mo>,</mo><mi>V</mi><mo stretchy="false">)</mo><mo>=</mo><mi>soft</mi><mi>max</mi><mrow><mo stretchy="true">(</mo><mfrac><mrow><mi>Q</mi><msup><mi>K</mi><mi>T</mi></msup></mrow><msqrt><mrow><msub><mi>d</mi><mi>k</mi></msub></mrow></msqrt></mfrac><mo stretchy="true">)</mo></mrow><mi>V</mi></math></td>
<td class="label">25</td>
</tr></table>
<table class="disp-formula p" id="eq26"><tr>
<td class="formula"><math id="d1e2678" display="block"><mi>multihead</mi><mo stretchy="false">(</mo><mi>Q</mi><mo>,</mo><mi>K</mi><mo>,</mo><mi>V</mi><mo stretchy="false">)</mo><mo>=</mo><mi>concat</mi><mo stretchy="false">(</mo><msub><mi>head</mi><mn>1</mn></msub><mo>,</mo><msub><mi>head</mi><mn>2</mn></msub><mo>,</mo><mo>.</mo><mo>.</mo><mo>.</mo><mo>,</mo><msub><mi>head</mi><mi>h</mi></msub><mo stretchy="false">)</mo><msup><mi>W</mi><mi>O</mi></msup></math></td>
<td class="label">26</td>
</tr></table>
<table class="disp-formula p" id="eq27"><tr>
<td class="formula"><math id="d1e2718" display="block"><msub><mi>head</mi><mi>i</mi></msub><mo>=</mo><mi>attention</mi><mo stretchy="false">(</mo><mi>Q</mi><msubsup><mi>W</mi><mi>i</mi><mi>Q</mi></msubsup><mo>,</mo><mi>K</mi><msubsup><mi>W</mi><mi>i</mi><mi>K</mi></msubsup><mo>,</mo><mi>V</mi><msubsup><mi>W</mi><mi>i</mi><mi>V</mi></msubsup><mo stretchy="false">)</mo></math></td>
<td class="label">27</td>
</tr></table>
<p>where <em>T</em> is
the sequence length and <em>d</em> is the feature dimension. <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e2757" display="inline"><msup><mi>W</mi><mi>Q</mi></msup><mo>,</mo><mspace width="0.25em"></mspace><msup><mi>W</mi><mi>K</mi></msup><mo>,</mo><mspace width="0.25em"></mspace><mi>and</mi><mspace width="0.25em"></mspace><msup><mi>W</mi><mi>V</mi></msup><mo>∈</mo><msup><mi mathvariant="double-struck">R</mi><mrow><mi>d</mi><mo>×</mo><msub><mi>d</mi><mi>k</mi></msub></mrow></msup></math>
</span> are the learnable projection matrices,
and <em>d</em>
<sub>
<em>k</em>
</sub> is the feature
dimension of each attention head. <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e2794" display="inline"><mi>Q</mi><mo>,</mo><mspace width="0.25em"></mspace><mi>K</mi><mo>,</mo><mspace width="0.25em"></mspace><mi>and</mi><mspace width="0.25em"></mspace><mi>V</mi><mo>∈</mo><msup><mi mathvariant="double-struck">R</mi><mrow><mi>T</mi><mo>×</mo><mi>d</mi></mrow></msup></math>
</span> are the
query, key, and value matrices
obtained by linearly transforming the input sequence. The attention
weights are calculated through a scaled dot-product operation <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e2816" display="inline"><mfrac><mrow><mi>Q</mi><msup><mi>K</mi><mi>T</mi></msup></mrow><msqrt><mrow><msub><mi>d</mi><mi>k</mi></msub></mrow></msqrt></mfrac></math>
</span>, where <em>d</em>
<sub>
<em>k</em>
</sub> represents the key vector dimension.
This scaling prevents
gradient vanishing issues caused by large values. The architecture
employs <em>h</em> parallel attention heads, each with dedicated
projection matrices <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e2842" display="inline"><msubsup><mi>W</mi><mi>i</mi><mi>Q</mi></msubsup><mo>,</mo><mspace width="0.25em"></mspace><msubsup><mi>W</mi><mi>i</mi><mi>K</mi></msubsup><mo>,</mo><mspace width="0.25em"></mspace><mi>and</mi><mspace width="0.25em"></mspace><msubsup><mi>W</mi><mi>i</mi><mi>V</mi></msubsup><mo>∈</mo><msup><mi mathvariant="double-struck">R</mi><mrow><mi>d</mi><mo>×</mo><msub><mi>d</mi><mi>k</mi></msub></mrow></msup></math>
</span> and final output transformation <em>W</em>
<sup>
<em>O</em>
</sup>.</p></section><section id="sec2.4.6"><h4 class="pmc_sec_title">2.4.6. The
Fully Connected Layers</h4>
<p>Positioned
after the multihead attention module, the FC layer transforms high-dimensional
sequence features into lower-dimensional classification representations.
The FC layer projects these high-dimensional representations to a
reduced feature space while integrating global sequence information.
The FC layer processes global features <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e2886" display="inline"><mi>Z</mi><mo>∈</mo><msup><mi mathvariant="double-struck">R</mi><mrow><mi>T</mi><mo>×</mo><msub><mi>d</mi><mi>out</mi></msub></mrow></msup></math>
</span> generated
by the attention module, with <em>T</em> denoting sequence
length and <em>d</em>
<sub>out</sub> representing feature dimensionality.
These representations are subsequently
merged to form a unified global feature vector <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e2909" display="inline"><msub><mi>Z</mi><mi>global</mi></msub><mo>∈</mo><msup><mi mathvariant="double-struck">R</mi><mrow><msub><mi>d</mi><mi>out</mi></msub></mrow></msup></math>
</span>.
The fully connected layer consists of
two layers of neurons. The initial layer processes <em>Z</em>
<sub>global</sub> through an affine mapping succeeded by a nonlinearity,
producing <em>O</em>
<sub>
<em>fc</em>1</sub>.The output
of the first layer is further mapped to the classifier’s dimension
space, resulting in <em>O</em>
<sub>
<em>fc</em>2</sub>. Following processing by both FC layers, the softmax layer ultimately
transforms the output into class probabilities.</p>
<table class="disp-formula p" id="eq28"><tr>
<td class="formula"><math id="d1e2944" display="block"><msub><mi>O</mi><mrow><mi>f</mi><mi>c</mi><mn>1</mn></mrow></msub><mo>=</mo><mi>g</mi><mo stretchy="false">(</mo><msub><mi>W</mi><mrow><mi>f</mi><mi>c</mi><mn>1</mn></mrow></msub><mo>·</mo><msub><mi>Z</mi><mi>global</mi></msub><mo>+</mo><msub><mi>b</mi><mrow><mi>f</mi><mi>c</mi><mn>1</mn></mrow></msub><mo stretchy="false">)</mo></math></td>
<td class="label">28</td>
</tr></table>
<table class="disp-formula p" id="eq29"><tr>
<td class="formula"><math id="d1e2978" display="block"><msub><mi>O</mi><mrow><mi>f</mi><mi>c</mi><mn>2</mn></mrow></msub><mo>=</mo><mi>g</mi><mo stretchy="false">(</mo><msub><mi>W</mi><mrow><mi>f</mi><mi>c</mi><mn>2</mn></mrow></msub><mo>·</mo><msub><mi>O</mi><mrow><mi>f</mi><mi>c</mi><mn>1</mn></mrow></msub><mo>+</mo><msub><mi>b</mi><mrow><mi>f</mi><mi>c</mi><mn>2</mn></mrow></msub><mo stretchy="false">)</mo></math></td>
<td class="label">29</td>
</tr></table>
<table class="disp-formula p" id="eq30"><tr>
<td class="formula"><math id="d1e3015" display="block"><mi>P</mi><mo>=</mo><mi>soft</mi><mi>max</mi><mo stretchy="false">(</mo><msub><mi>O</mi><mrow><mi>f</mi><mi>c</mi><mn>2</mn></mrow></msub><mo stretchy="false">)</mo></math></td>
<td class="label">30</td>
</tr></table>
<p>The architecture comprises
two fully connected layers: the first layer applies a linear transformation
parametrized by weight matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e3036" display="inline"><msub><mi>W</mi><mrow><mi>f</mi><mi>c</mi><mn>1</mn></mrow></msub><mo>∈</mo><msup><mi mathvariant="double-struck">R</mi><mrow><msub><mi>d</mi><mrow><mi>f</mi><mi>c</mi><mn>1</mn></mrow></msub><mo>×</mo><msub><mi>d</mi><mi>out</mi></msub></mrow></msup></math>
</span> and bias vector <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e3062" display="inline"><msub><mrow><mi>b</mi></mrow><mrow><mi>f</mi><mi>c</mi><mn>1</mn></mrow></msub><mo>∈</mo><msup><mrow><mi mathvariant="double-struck">R</mi></mrow><mrow><msub><mrow><mi>d</mi></mrow><mrow><mi>f</mi><mi>c</mi><mn>1</mn></mrow></msub></mrow></msup></math>
</span>, followed by ReLU activation <em>g</em>. The subsequent layer processes features using parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e3090" display="inline"><msub><mrow><mi>W</mi></mrow><mrow><mi>f</mi><mi>c</mi><mn>2</mn></mrow></msub><mo>∈</mo><msup><mrow><mi mathvariant="double-struck">R</mi></mrow><mrow><msub><mrow><mi>d</mi></mrow><mrow><mi>f</mi><mi>c</mi><mn>2</mn></mrow></msub><mo>×</mo><msub><mrow><mi>d</mi></mrow><mrow><mi>f</mi><mi>c</mi><mn>1</mn></mrow></msub></mrow></msup></math>
</span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e3123" display="inline"><msub><mrow><mi>b</mi></mrow><mrow><mi>f</mi><mi>c</mi><mn>2</mn></mrow></msub><mo>∈</mo><msup><mrow><mi mathvariant="double-struck">R</mi></mrow><mrow><msub><mrow><mi>d</mi></mrow><mrow><mi>f</mi><mi>c</mi><mn>2</mn></mrow></msub></mrow></msup></math>
</span>, ultimately
producing class probability
distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e3149" display="inline"><mi>P</mi><mo>∈</mo><msup><mi mathvariant="double-struck">R</mi><mrow><msub><mi>n</mi><mi>class</mi></msub></mrow></msup></math>
</span> through the output layer.</p></section><section id="sec2.4.7"><h4 class="pmc_sec_title">2.4.7. Loss
Function</h4>
<p>The model is optimized
using a cross-entropy minimization criterion as the objective. The
loss function aims to reduce the discrepancy between model predictions
and ground truth labels, thereby enhancing circRNA–RBP interaction
classification performance. Its mathematical expression is</p>
<table class="disp-formula p" id="eq31"><tr>
<td class="formula"><math id="d1e3168" display="block"><mi>L</mi><mo>=</mo><mo form="prefix">−</mo><mfrac><mn>1</mn><mi>N</mi></mfrac><munderover accent="false" accentunder="false"><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>N</mi></munderover><mo stretchy="false">[</mo><msub><mi>y</mi><mi>i</mi></msub><mi>log</mi><mo stretchy="false">(</mo><msub><mover accent="true"><mi>y</mi><mo>^</mo></mover><mi>i</mi></msub><mo stretchy="false">)</mo><mo>+</mo><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><msub><mi>y</mi><mi>i</mi></msub><mo stretchy="false">)</mo><mi>log</mi><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><msub><mover accent="true"><mi>y</mi><mo>^</mo></mover><mi>i</mi></msub><mo stretchy="false">)</mo><mo stretchy="false">]</mo></math></td>
<td class="label">31</td>
</tr></table>
<p>where <em>ŷ</em>
<sub>
<em>i</em>
</sub> represents the predicted value of the <em>i</em>th instance and <em>y</em>
<sub>
<em>i</em>
</sub> corresponds
to the ground truth annotation, with <em>N</em> indicating
the batch size.</p></section><section id="sec2.4.8"><h4 class="pmc_sec_title">2.4.8. Experimental Settings</h4>
<p>On an Alibaba
Cloud server with 16 GB of memory and an NVIDIA V100 GPU, we trained
the proposed GGCRB using 16 circRNA data sets. The training process,
which incorporates early stopping to monitor validation loss, took
approximately 2.30 h in total. Due to differences in sample sizes
across data sets, the time required to train one epoch varies. The
shortest epoch time is 1.01 s for TAF15, and the longest is 9.89 s
for IGF2BP1. For details on the hyperparameter configuration, please
refer to <a href="#tbl2" class="usa-link">Table 
</a>. In
addition, to further evaluate the impact of key hyperparameters, we
conducted experiments on the number of heads in the multihead attention
mechanism and the dimensionality of the LSTM layer. The results of
these experiments are presented in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04524/suppl_file/ao5c04524_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S1 and S2</a>. Among them, the specific training times for one epoch
of each data set are shown in <a href="#tbl3" class="usa-link">Table 
</a>.</p>
<section class="tw xbox font-sm" id="tbl2"><h5 class="obj_head">2. Hyperparameter Settings of the GGCRB
Model.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">parameters</th>
<th align="center" colspan="1" rowspan="1">setting</th>
<th align="center" colspan="1" rowspan="1">description</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">learning rate</td>
<td align="left" colspan="1" rowspan="1">0.0005</td>
<td align="left" colspan="1" rowspan="1">step size for model optimization</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">batch
size</td>
<td align="left" colspan="1" rowspan="1">30</td>
<td align="left" colspan="1" rowspan="1">number of training samples per
batch</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">epoch</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">total number of training epochs</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">L</td>
<td align="left" colspan="1" rowspan="1">101</td>
<td align="left" colspan="1" rowspan="1">length of the circRNA fragment (in nucleotides)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CNN</td>
<td align="left" colspan="1" rowspan="1">128,3</td>
<td align="left" colspan="1" rowspan="1">128 filters
with kernel size 3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GCN\GAT</td>
<td align="left" colspan="1" rowspan="1">128</td>
<td align="left" colspan="1" rowspan="1">output dimension of graph convolution/attention
layers</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">multihead attention</td>
<td align="left" colspan="1" rowspan="1">eight heads</td>
<td align="left" colspan="1" rowspan="1">number of attention heads in the multihead
attention module</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LSTM</td>
<td align="left" colspan="1" rowspan="1">120 units</td>
<td align="left" colspan="1" rowspan="1">hidden size of the bidirectional LSTM (per direction)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="tbl3"><h5 class="obj_head">3. Mean Training
Duration per Epoch across
16 Benchmark Collections.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">RBP</th>
<th align="center" colspan="1" rowspan="1">time (s)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">AUF1</td>
<td align="left" colspan="1" rowspan="1">1.48</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DGCR8</td>
<td align="left" colspan="1" rowspan="1">7.56</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">EWSR1</td>
<td align="left" colspan="1" rowspan="1">2.23</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HNRNPC</td>
<td align="left" colspan="1" rowspan="1">6.73</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MOV10</td>
<td align="left" colspan="1" rowspan="1">2.70</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TDP43</td>
<td align="left" colspan="1" rowspan="1">2.50</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FMRP</td>
<td align="left" colspan="1" rowspan="1">7.43</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HUR</td>
<td align="left" colspan="1" rowspan="1">9.39</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IGF2BP1</td>
<td align="left" colspan="1" rowspan="1">9.89</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PTB</td>
<td align="left" colspan="1" rowspan="1">9.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IGF2BP3</td>
<td align="left" colspan="1" rowspan="1">9.67</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">U2AF65</td>
<td align="left" colspan="1" rowspan="1">3.49</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PUM2</td>
<td align="left" colspan="1" rowspan="1">1.32</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TAF15</td>
<td align="left" colspan="1" rowspan="1">1.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FXR2</td>
<td align="left" colspan="1" rowspan="1">2.61</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LIN28A</td>
<td align="left" colspan="1" rowspan="1">8.48</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">all</td>
<td align="left" colspan="1" rowspan="1">2.30 (h)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section></section></section><section id="sec3"><h2 class="pmc_sec_title">3. Results</h2>
<section id="sec3.1"><h3 class="pmc_sec_title">3.1. Performance Evaluation</h3>
<p>To holistically
assess the model’s forecasting capability, this research implements
quintuple cross-validation testing on 16 circular RNA collections and uses five commonly employed evaluation metrics:
AUC-ROC, predictive precision, true positive rate, <em>F</em>1-index, and overall classification accuracy. Among these, AUC is the primary evaluation metric to measure
the model’s discriminative ability, with a value range from
0.5 (random guessing) to 1 (perfect prediction). The mathematical
definitions of these metrics are as follows:</p>
<table class="disp-formula p" id="eq32"><tr>
<td class="formula"><math id="d1e3530" display="block"><mi>precision</mi><mo>=</mo><mfrac><mi>TP</mi><mrow><mi>TP</mi><mo>+</mo><mi>FP</mi></mrow></mfrac></math></td>
<td class="label">32</td>
</tr></table>
<table class="disp-formula p" id="eq33"><tr>
<td class="formula"><math id="d1e3543" display="block"><mi>recall</mi><mo>=</mo><mfrac><mi>TP</mi><mrow><mi>TP</mi><mo>+</mo><mi>FN</mi></mrow></mfrac></math></td>
<td class="label">33</td>
</tr></table>
<table class="disp-formula p" id="eq34"><tr>
<td class="formula"><math id="d1e3556" display="block"><mi>F</mi><mn>1</mn><mo>=</mo><mn>2</mn><mo>×</mo><mfrac><mrow><mi>precision</mi><mo>×</mo><mi>recall</mi></mrow><mrow><mi>precision</mi><mo>+</mo><mi>recall</mi></mrow></mfrac></math></td>
<td class="label">34</td>
</tr></table>
<table class="disp-formula p" id="eq35"><tr>
<td class="formula"><math id="d1e3575" display="block"><mi>ACC</mi><mo>=</mo><mfrac><mrow><mi>TP</mi><mo>+</mo><mi>TN</mi></mrow><mrow><mi>TP</mi><mo>+</mo><mi>TN</mi><mo>+</mo><mi>FP</mi><mo>+</mo><mi>FN</mi></mrow></mfrac></math></td>
<td class="label">35</td>
</tr></table>
<p>where TP represents correctly
identified interacting circRNA–RBP pairs, while TN denotes
properly recognized noninteracting pairs. FN indicates missed true
interactions, and FP corresponds to erroneously predicted nonexistent
interactions.</p>
<p>GGCRB’s forecasting efficacy was systematically
examined across 16 circRNA–protein interaction collections,
and the results are shown in <a href="#fig2" class="usa-link">Figure 
</a> and <a href="#tbl4" class="usa-link">Table 
</a>, demonstrating the model performance on the five evaluated
metrics. In all data sets, the AUC of GGCRB is more than 0.94 on 15
data sets; especially, on the AUF1, HNRNPC, PUM2, and TAF15 data sets,
the AUC is more than 0.99.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332793_ao5c04524_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/18ab/12332793/90a32a60c3b2/ao5c04524_0002.jpg" loading="lazy" id="gr2" height="445" width="764" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>GGCRB was evaluated using five assessment criteria.</p></figcaption></figure><section class="tw xbox font-sm" id="tbl4"><h4 class="obj_head">4. GGCRB’s Effectiveness Measured
Using Five Assessment Criteria across 16 Data Sets.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">Data sets</th>
<th align="center" colspan="1" rowspan="1">AUC</th>
<th align="center" colspan="1" rowspan="1">ACC</th>
<th align="center" colspan="1" rowspan="1">precision</th>
<th align="center" colspan="1" rowspan="1">recall</th>
<th align="center" colspan="1" rowspan="1">F1-score</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">AUF1</td>
<td align="center" colspan="1" rowspan="1">0.9903</td>
<td align="center" colspan="1" rowspan="1">0.9599</td>
<td align="center" colspan="1" rowspan="1">0.9544</td>
<td align="center" colspan="1" rowspan="1">0.9668</td>
<td align="center" colspan="1" rowspan="1">0.9605</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DGCR8</td>
<td align="center" colspan="1" rowspan="1">0.9634</td>
<td align="center" colspan="1" rowspan="1">0.9005</td>
<td align="center" colspan="1" rowspan="1">0.9014</td>
<td align="center" colspan="1" rowspan="1">0.8985</td>
<td align="center" colspan="1" rowspan="1">0.8998</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">EWSR1</td>
<td align="center" colspan="1" rowspan="1">0.9721</td>
<td align="center" colspan="1" rowspan="1">0.9099</td>
<td align="center" colspan="1" rowspan="1">0.8997</td>
<td align="center" colspan="1" rowspan="1">0.9240</td>
<td align="center" colspan="1" rowspan="1">0.9114</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HNRNPC</td>
<td align="center" colspan="1" rowspan="1">0.9902</td>
<td align="center" colspan="1" rowspan="1">0.9540</td>
<td align="center" colspan="1" rowspan="1">0.9486</td>
<td align="center" colspan="1" rowspan="1">0.9608</td>
<td align="center" colspan="1" rowspan="1">0.9546</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MOV10</td>
<td align="center" colspan="1" rowspan="1">0.9608</td>
<td align="center" colspan="1" rowspan="1">0.8968</td>
<td align="center" colspan="1" rowspan="1">0.8778</td>
<td align="center" colspan="1" rowspan="1">0.9149</td>
<td align="center" colspan="1" rowspan="1">0.8957</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TDP43</td>
<td align="center" colspan="1" rowspan="1">0.9525</td>
<td align="center" colspan="1" rowspan="1">0.8828</td>
<td align="center" colspan="1" rowspan="1">0.9004</td>
<td align="center" colspan="1" rowspan="1">0.8609</td>
<td align="center" colspan="1" rowspan="1">0.8802</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FMRP</td>
<td align="center" colspan="1" rowspan="1">0.9435</td>
<td align="center" colspan="1" rowspan="1">0.8676</td>
<td align="center" colspan="1" rowspan="1">0.8699</td>
<td align="center" colspan="1" rowspan="1">0.8629</td>
<td align="center" colspan="1" rowspan="1">0.8661</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HUR</td>
<td align="center" colspan="1" rowspan="1">0.9430</td>
<td align="center" colspan="1" rowspan="1">0.8689</td>
<td align="center" colspan="1" rowspan="1">0.8672</td>
<td align="center" colspan="1" rowspan="1">0.8696</td>
<td align="center" colspan="1" rowspan="1">0.8682</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IGF2BP1</td>
<td align="center" colspan="1" rowspan="1">0.9406</td>
<td align="center" colspan="1" rowspan="1">0.8623</td>
<td align="center" colspan="1" rowspan="1">0.8703</td>
<td align="center" colspan="1" rowspan="1">0.8534</td>
<td align="center" colspan="1" rowspan="1">0.8612</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PTB</td>
<td align="center" colspan="1" rowspan="1">0.9466</td>
<td align="center" colspan="1" rowspan="1">0.8722</td>
<td align="center" colspan="1" rowspan="1">0.8771</td>
<td align="center" colspan="1" rowspan="1">0.8643</td>
<td align="center" colspan="1" rowspan="1">0.8704</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IGF2BP3</td>
<td align="center" colspan="1" rowspan="1">0.9366</td>
<td align="center" colspan="1" rowspan="1">0.8601</td>
<td align="center" colspan="1" rowspan="1">0.8619</td>
<td align="center" colspan="1" rowspan="1">0.8567</td>
<td align="center" colspan="1" rowspan="1">0.8588</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">U2AF65</td>
<td align="center" colspan="1" rowspan="1">0.9496</td>
<td align="center" colspan="1" rowspan="1">0.8805</td>
<td align="center" colspan="1" rowspan="1">0.8793</td>
<td align="center" colspan="1" rowspan="1">0.8858</td>
<td align="center" colspan="1" rowspan="1">0.8823</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PUM2</td>
<td align="center" colspan="1" rowspan="1">0.9938</td>
<td align="center" colspan="1" rowspan="1">0.9622</td>
<td align="center" colspan="1" rowspan="1">0.9525</td>
<td align="center" colspan="1" rowspan="1">0.9702</td>
<td align="center" colspan="1" rowspan="1">0.9612</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TAF15</td>
<td align="center" colspan="1" rowspan="1">0.9985</td>
<td align="center" colspan="1" rowspan="1">0.9799</td>
<td align="center" colspan="1" rowspan="1">0.9711</td>
<td align="center" colspan="1" rowspan="1">0.9899</td>
<td align="center" colspan="1" rowspan="1">0.9804</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FXR2</td>
<td align="center" colspan="1" rowspan="1">0.9757</td>
<td align="center" colspan="1" rowspan="1">0.9160</td>
<td align="center" colspan="1" rowspan="1">0.9010</td>
<td align="center" colspan="1" rowspan="1">0.9254</td>
<td align="center" colspan="1" rowspan="1">0.9127</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LIN28A</td>
<td align="center" colspan="1" rowspan="1">0.9567</td>
<td align="center" colspan="1" rowspan="1">0.8883</td>
<td align="center" colspan="1" rowspan="1">0.9040</td>
<td align="center" colspan="1" rowspan="1">0.8705</td>
<td align="center" colspan="1" rowspan="1">0.8868</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">average</td>
<td align="center" colspan="1" rowspan="1">0.9634</td>
<td align="center" colspan="1" rowspan="1">0.9039</td>
<td align="center" colspan="1" rowspan="1">0.9023</td>
<td align="center" colspan="1" rowspan="1">0.9047</td>
<td align="center" colspan="1" rowspan="1">0.9031</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>In addition, all data set
ROC curves are graphically
represented
in <a href="#fig3" class="usa-link">Figure 
</a>. Regarding
overall efficacy, GGCRB achieves an AUC of 0.963 across the 16 data
sets, with mean values exceeding 0.90 for precision, classification
accuracy, sensitivity, and <em>F</em>-measure. These results
indicate that the proposed GGCRB exhibits an excellent performance
with a high generalization ability and application potential.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332793_ao5c04524_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/18ab/12332793/33dce84c9a39/ao5c04524_0003.jpg" loading="lazy" id="gr3" height="456" width="625" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Receiver operating
characteristic curves for GGCRB across 16 circular
RNA collections.</p></figcaption></figure></section><section id="sec3.2"><h3 class="pmc_sec_title">3.2. Benchmarking
against Existing Approaches</h3>
<p>The evaluation framework contrasted
GGCRB with five reference algorithms
in circRNA–RBP binding site identification (CRIP, PASSION, HCRNet, Circ-FHN, and CRIECNN).</p>
<p>Benchmarking was conducted across 16
circRNA data sets using a five-fold cross-validation scheme to evaluate
predictive performance. The results, detailed in <a href="#fig4" class="usa-link">Figure 
</a> and <a href="#tbl5" class="usa-link">Table 
</a>, demonstrate that GGCRB consistently outperforms
competing methods. Specifically, GGCRB achieves the highest AUC scores
on 14 out of the 16 data sets, with particularly notable improvements
on MOV10, IGF2BP1, IGF2BP3, LIN28A, and PTB data sets. For the remaining
two data sets, GGCRB’s AUC values remain competitive, comparable
to those of Circ-FHN and HCRNet. Moreover, GGCRB exhibits robust performance
on large-scale data sets containing up to 40,000 samples, such as
IGF2BP1, HUR, and PTB.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332793_ao5c04524_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/18ab/12332793/f2193eede16a/ao5c04524_0004.jpg" loading="lazy" id="gr4" height="392" width="645" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance comparison using AUC metrics conducted on
16 distinct
biological data sets.</p></figcaption></figure><section class="tw xbox font-sm" id="tbl5"><h4 class="obj_head">5. Area under
the Curve Measurements
for Six Competing Methods (CRIP, PASSION, HCRNet, Circ-FHN, CRIECNN,
and GGCRB) Evaluated Using 16 circRNA Collections.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">data sets</th>
<th align="center" colspan="1" rowspan="1">CRIP</th>
<th align="center" colspan="1" rowspan="1">PASSION</th>
<th align="center" colspan="1" rowspan="1">HCRNet</th>
<th align="center" colspan="1" rowspan="1">Circ-FHN</th>
<th align="center" colspan="1" rowspan="1">CRIECNN</th>
<th align="center" colspan="1" rowspan="1">GGCRB</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">AUF1</td>
<td align="center" colspan="1" rowspan="1">0.9838</td>
<td align="center" colspan="1" rowspan="1">0.9790</td>
<td align="center" colspan="1" rowspan="1">0.9800</td>
<td align="center" colspan="1" rowspan="1">0.9886</td>
<td align="center" colspan="1" rowspan="1">0.9887</td>
<td align="center" colspan="1" rowspan="1">0.9903</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DGCR8</td>
<td align="center" colspan="1" rowspan="1">0.9048</td>
<td align="center" colspan="1" rowspan="1">0.9170</td>
<td align="center" colspan="1" rowspan="1">0.9140</td>
<td align="center" colspan="1" rowspan="1">0.9189</td>
<td align="center" colspan="1" rowspan="1">0.9506</td>
<td align="center" colspan="1" rowspan="1">0.9634</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">EWSR1</td>
<td align="center" colspan="1" rowspan="1">0.9341</td>
<td align="center" colspan="1" rowspan="1">0.9380</td>
<td align="center" colspan="1" rowspan="1">0.9360</td>
<td align="center" colspan="1" rowspan="1">0.9441</td>
<td align="center" colspan="1" rowspan="1">0.9624</td>
<td align="center" colspan="1" rowspan="1">0.9721</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HNRNPC</td>
<td align="center" colspan="1" rowspan="1">0.9714</td>
<td align="center" colspan="1" rowspan="1">0.9760</td>
<td align="center" colspan="1" rowspan="1">0.9720</td>
<td align="center" colspan="1" rowspan="1">0.9782</td>
<td align="center" colspan="1" rowspan="1">0.9915</td>
<td align="center" colspan="1" rowspan="1">0.9902</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MOV10</td>
<td align="center" colspan="1" rowspan="1">0.8296</td>
<td align="center" colspan="1" rowspan="1">0.8450</td>
<td align="center" colspan="1" rowspan="1">0.8490</td>
<td align="center" colspan="1" rowspan="1">0.8475</td>
<td align="center" colspan="1" rowspan="1">0.9011</td>
<td align="center" colspan="1" rowspan="1">0.9608</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TDP43</td>
<td align="center" colspan="1" rowspan="1">0.9358</td>
<td align="center" colspan="1" rowspan="1">0.9340</td>
<td align="center" colspan="1" rowspan="1">0.9300</td>
<td align="center" colspan="1" rowspan="1">0.9437</td>
<td align="center" colspan="1" rowspan="1">0.9580</td>
<td align="center" colspan="1" rowspan="1">0.9525</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FMRP</td>
<td align="center" colspan="1" rowspan="1">0.8868</td>
<td align="center" colspan="1" rowspan="1">0.9000</td>
<td align="center" colspan="1" rowspan="1">0.8980</td>
<td align="center" colspan="1" rowspan="1">0.8995</td>
<td align="center" colspan="1" rowspan="1">0.9389</td>
<td align="center" colspan="1" rowspan="1">0.9435</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HUR</td>
<td align="center" colspan="1" rowspan="1">0.8628</td>
<td align="center" colspan="1" rowspan="1">0.8790</td>
<td align="center" colspan="1" rowspan="1">0.8740</td>
<td align="center" colspan="1" rowspan="1">0.8874</td>
<td align="center" colspan="1" rowspan="1">0.9210</td>
<td align="center" colspan="1" rowspan="1">0.9430</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IGF2BP1</td>
<td align="center" colspan="1" rowspan="1">0.8204</td>
<td align="center" colspan="1" rowspan="1">0.8450</td>
<td align="center" colspan="1" rowspan="1">0.8430</td>
<td align="center" colspan="1" rowspan="1">0.8537</td>
<td align="center" colspan="1" rowspan="1">0.9076</td>
<td align="center" colspan="1" rowspan="1">0.9406</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PTB</td>
<td align="center" colspan="1" rowspan="1">0.8078</td>
<td align="center" colspan="1" rowspan="1">0.8290</td>
<td align="center" colspan="1" rowspan="1">0.8260</td>
<td align="center" colspan="1" rowspan="1">0.8286</td>
<td align="center" colspan="1" rowspan="1">0.9171</td>
<td align="center" colspan="1" rowspan="1">0.9466</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IGF2BP3</td>
<td align="center" colspan="1" rowspan="1">0.7756</td>
<td align="center" colspan="1" rowspan="1">0.8310</td>
<td align="center" colspan="1" rowspan="1">0.8220</td>
<td align="center" colspan="1" rowspan="1">0.8240</td>
<td align="center" colspan="1" rowspan="1">0.9183</td>
<td align="center" colspan="1" rowspan="1">0.9366</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">U2AF65</td>
<td align="center" colspan="1" rowspan="1">0.9183</td>
<td align="center" colspan="1" rowspan="1">0.9300</td>
<td align="center" colspan="1" rowspan="1">0.9280</td>
<td align="center" colspan="1" rowspan="1">0.9377</td>
<td align="center" colspan="1" rowspan="1">0.9475</td>
<td align="center" colspan="1" rowspan="1">0.9496</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PUM2</td>
<td align="center" colspan="1" rowspan="1">0.9728</td>
<td align="center" colspan="1" rowspan="1">0.9520</td>
<td align="center" colspan="1" rowspan="1">0.9530</td>
<td align="center" colspan="1" rowspan="1">0.9669</td>
<td align="center" colspan="1" rowspan="1">0.9919</td>
<td align="center" colspan="1" rowspan="1">0.9938</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TAF15</td>
<td align="center" colspan="1" rowspan="1">0.9810</td>
<td align="center" colspan="1" rowspan="1">0.9670</td>
<td align="center" colspan="1" rowspan="1">0.9650</td>
<td align="center" colspan="1" rowspan="1">0.9838</td>
<td align="center" colspan="1" rowspan="1">0.9967</td>
<td align="center" colspan="1" rowspan="1">0.9985</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FXR2</td>
<td align="center" colspan="1" rowspan="1">0.9421</td>
<td align="center" colspan="1" rowspan="1">0.9410</td>
<td align="center" colspan="1" rowspan="1">0.9380</td>
<td align="center" colspan="1" rowspan="1">0.9432</td>
<td align="center" colspan="1" rowspan="1">0.9720</td>
<td align="center" colspan="1" rowspan="1">0.9757</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LIN28A</td>
<td align="center" colspan="1" rowspan="1">0.8587</td>
<td align="center" colspan="1" rowspan="1">0.8750</td>
<td align="center" colspan="1" rowspan="1">0.8650</td>
<td align="center" colspan="1" rowspan="1">0.8706</td>
<td align="center" colspan="1" rowspan="1">0.9124</td>
<td align="center" colspan="1" rowspan="1">0.9567</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">average</td>
<td align="center" colspan="1" rowspan="1">0.8909</td>
<td align="center" colspan="1" rowspan="1">0.9086</td>
<td align="center" colspan="1" rowspan="1">0.9058</td>
<td align="center" colspan="1" rowspan="1">0.9135</td>
<td align="center" colspan="1" rowspan="1">0.9485</td>
<td align="center" colspan="1" rowspan="1">0.9634</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>As shown in <a href="#tbl5" class="usa-link">Table 
</a>, GGCRB attains a mean AUC of 0.9634 across
all benchmark
data sets,
representing significant gains over existing approaches: 7.25% higher
than CRIP (0.8909), 5.48% higher than PASSION (0.9086), 4.99% higher
than Circ-FHN (0.9135), 5.76% higher than HCRNet (0.9058), 1.49% higher
than CRIECNN (0.9485), and 5.66% higher than other models on average.
These empirical results confirm GGCRB’s superior predictive
capability in circRNA–RBP interaction prediction.</p>
<p>These
results are primarily attributed to GGCRB’s deep integration
of multiscale sequence features and structural information, as well
as the effective synergy between graph neural networks and multihead
attention mechanisms, enabling the model to capture the complex biological
characteristics of circRNA–RBP interactions more comprehensively
and accurately.</p></section><section id="sec3.3"><h3 class="pmc_sec_title">3.3. Ablation Experiments</h3>
<p>To validate
the effectiveness of individual components in the GGCRB model, we
designed eight ablation experiments to systematically assess the impact
of different feature encoding strategies and structural modules on
model performance. GGCRB integrates five types of feature encoding
methods and incorporates GCN, GAT, and multihead attention mechanisms.
The specific combinations of modules are summarized in <a href="#tbl6" class="usa-link">Table 
</a>, and their corresponding results
are illustrated in the bar chart shown in <a href="#fig5" class="usa-link">Figure 
</a>. The configurations and motivations of each
variant are as follows:</p>
<ul class="list" style="list-style-type:none">
<li><p>M1 removes
NCP and DPCP encodings to evaluate model
performance under the absence of nucleotide physicochemical properties,
which represent low-level sequence features.</p></li>
<li><p>M2 eliminates HNF and ND encodings to assess the contribution
of higher-order <em>k</em>-mer frequency and nucleotide density
features.</p></li>
<li><p>M3 removes the Doc2Vec encoding
to investigate the role
of deep semantic information derived from a global sequence context.</p></li>
<li><p>M4 removes all structural features and disables
the
GCN and GAT modules, aiming to determine whether structural information
is critical to performance improvement.</p></li>
<li><p>M5 retains all feature encodings but replaces the graph-based
modules (GCN and GAT) with conventional CNNs, to compare the effectiveness
of graph neural networks versus convolutional approaches in structural
modeling.</p></li>
<li><p>M6 removes only the HNF encoding
to further isolate
and examine its individual contribution to sequence representation.</p></li>
<li><p>M7 preserves the full architecture except
for the removal
of the multihead attention mechanism, in order to evaluate its impact
on modeling global sequence dependencies.</p></li>
<li><p>M8 simplifies the original three-layer structure module
(“GCN → GAT → GCN”) to a two-layer configuration
(“GCN → GAT”) to assess the effect of structural
modeling depth.</p></li>
</ul>
<section class="tw xbox font-sm" id="tbl6"><h4 class="obj_head">6. Combinations
of Different Variant
Models.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1"> </th>
<th align="center" colspan="1" rowspan="1">HNF</th>
<th align="center" colspan="1" rowspan="1">ND</th>
<th align="center" colspan="1" rowspan="1">NCP + DPCP</th>
<th align="center" colspan="1" rowspan="1">Doc2Vec</th>
<th align="center" colspan="1" rowspan="1">GCN + GAT + GCN</th>
<th align="center" colspan="1" rowspan="1">GCN + GAT</th>
<th align="center" colspan="1" rowspan="1">RNAstructure</th>
<th align="center" colspan="1" rowspan="1">mutihead attention</th>
<th align="center" colspan="1" rowspan="1">AUC</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">M1</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="center" colspan="1" rowspan="1">0.9480</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">M2</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="center" colspan="1" rowspan="1">0.9455</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">M3</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="center" colspan="1" rowspan="1">0.9422</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">M4</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="center" colspan="1" rowspan="1">0.9137</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">M5</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="center" colspan="1" rowspan="1">0.9426</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">M6</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="center" colspan="1" rowspan="1">0.9449</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">M7</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">0.9520</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">M8</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="center" colspan="1" rowspan="1">0.9483</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GGCRB</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="left" colspan="1" rowspan="1">√</td>
<td align="center" colspan="1" rowspan="1">0.9634</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332793_ao5c04524_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/18ab/12332793/94a3d71af853/ao5c04524_0005.jpg" loading="lazy" id="gr5" height="539" width="700" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance assessment of GGCRB with eight
architectural variants
on 16 circRNA collections.</p></figcaption></figure><p>We evaluated the average AUC of the eight model
variants across
16 data sets, yielding the following results: M1 (0.9480), M2 (0.9455),
M3 (0.9422), M4 (0.9137), M5 (0.9426), M6 (0.9449), M7 (0.9520), and
M8 (0.9483).</p>
<p>These results demonstrate that the joint modeling
of both sequence
and structural features is essential for optimal predictive performance.
In particular, the substantial performance drop in M4 confirms the
critical role of structural information and the effectiveness of the
GCN and GAT hybrid framework. The lower performance of M5 compared
to the full model further suggests the superiority of graph-based
structural modeling over conventional convolutional methods.</p>
<p>Additionally, we conducted a comprehensive comparison of GGCRB,
CRIECNN, and their respective variants using five performance metrics:
AUC, accuracy, precision, recall, and <em>F</em>1-score. The
results, visualized in the radar chart shown in <a href="#fig6" class="usa-link">Figure 
</a>, further confirm the superior overall performance
of GGCRB and highlight the individual contributions of each component
to the model’s efficacy.</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/18ab/12332793/57af81c16775/ao5c04524_0006.jpg" loading="lazy" id="gr6" height="498" width="666" alt="6"></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Systematic comparison between GGCRB, eight
architectural variants,
and CRIECNN.</p></figcaption></figure></section></section><section id="sec4"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>The proposed GGCRB model
effectively integrates multiscale features
and shows promising performance in circRNA–RBP binding site
prediction. The combination of GCN and GAT enables effective learning
of RNA secondary structure topology, while multihead attention captures
long-range sequence dependencies, thereby enhancing feature representation.
However, GGCRB has limitations. The reliance on predicted RNA secondary
structures may not fully capture dynamic or tertiary conformations,
potentially limiting structural feature fidelity. Additionally, the
limited diversity and size of available circRNA–RBP data sets
constrain the model’s generalizability, especially for rare
interaction types. The model’s complexity also entails considerable
computational cost, which may hinder scalability. Our future work
will explore advanced architectures such as graph autoencoders and
transformers to better capture complex RNA features. Expanding and
diversifying data sets will improve robustness and applicability.
Finally, optimizing computational efficiency will facilitate broader
application in large-scale and real-time scenarios.</p></section><section id="sec5"><h2 class="pmc_sec_title">5. Conclusions</h2>
<p>The superiority of GGCRB
stems from its efficient fusion of multiscale
features. This model combines multiple sequence encoding methods and
utilizes a hybrid GCN and GAT framework to capture structural features,
thereby comprehensively capturing both local and global features of
circular RNAs. Overall, GGCRB offers a new and effective solution
for the prediction of circular RNA–RBP interactions and provides
valuable references for the optimization and improvement of future
models.</p></section><section id="sec21"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12332793/bin/ao5c04524_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c04524_si_001.pdf</a><sup> (315.9KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We greatly thank the anonymous reviewers
for their
careful reading of our manuscript and their many insightful comments
and suggestions.</p></section><section id="notes2"><p>The source code
of this study is available at https://github.com/ydkvictory/GGCRB.</p></section><section id="notes4"><p>The Supporting Information is
available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsomega.5c04524?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsomega.5c04524</a>.</p>
<ul class="list" style="list-style-type:none"><li><p>Table of values for 11 dinucleotide
physicochemical
properties (DPCP); experimental results of the number of heads in
the multihead attention mechanism; experimental results of dimensional
parameter settings for the LSTM network (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04524/suppl_file/ao5c04524_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes1"><p>G.T. designed
and supervised the research; H.X. conceptualized and implemented the
model, conducted experiments, and drafted the manuscript; D.Y. and
X.Z. critically reviewed the paper; X.L. participated in manuscript
revision. All authors contributed to the editing process and approved
the final version for publication.</p></section><section id="notes3"><p>This work is
supported by the Heilongjiang Provincial Natural Science Foundation
of China (No. PL2024F011).</p></section><section id="notes6"><p>No animals/humans were
used for studies that are the basis of this research.</p></section><section id="notes5"><p>The authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Zhou W. Y., Cai Z. R., Liu J., Wang D. S., Ju H. Q., Xu R. H.. Circular RNA: metabolism,
functions and interactions with proteins. Mol.
Cancer. 2020;19(1):172. doi: 10.1186/s12943-020-01286-3.</cite> [<a href="https://doi.org/10.1186/s12943-020-01286-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7734744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33317550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer&amp;title=Circular%20RNA:%20metabolism,%20functions%20and%20interactions%20with%20proteins&amp;author=W.%20Y.%20Zhou&amp;author=Z.%20R.%20Cai&amp;author=J.%20Liu&amp;author=D.%20S.%20Wang&amp;author=H.%20Q.%20Ju&amp;volume=19&amp;issue=1&amp;publication_year=2020&amp;pages=172&amp;pmid=33317550&amp;doi=10.1186/s12943-020-01286-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Yin Y., Long J., He Q., Li Y., Liao Y., He P., Zhu W.. Emerging roles
of circRNA in formation and progression
of cancer. J. Cancer. 2019;10(21):5015–5021. doi: 10.7150/jca.30828.</cite> [<a href="https://doi.org/10.7150/jca.30828" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6775606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31602252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Cancer&amp;title=Emerging%20roles%20of%20circRNA%20in%20formation%20and%20progression%20of%20cancer&amp;author=Y.%20Yin&amp;author=J.%20Long&amp;author=Q.%20He&amp;author=Y.%20Li&amp;author=Y.%20Liao&amp;volume=10&amp;issue=21&amp;publication_year=2019&amp;pages=5015-5021&amp;pmid=31602252&amp;doi=10.7150/jca.30828&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Vo J. N., Cieslik M., Zhang Y., Shukla S., Xiao L., Zhang Y., Wu Y. M., Dhanasekaran S. M., Engelke C. G., Cao X., Robinson D. R., Nesvizhskii A. I., Chinnaiyan A. M.. The Landscape of Circular RNA in
Cancer. Cell. 2019;176(4):869–881.e13. doi: 10.1016/j.cell.2018.12.021.</cite> [<a href="https://doi.org/10.1016/j.cell.2018.12.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6601354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30735636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The%20Landscape%20of%20Circular%20RNA%20in%20Cancer&amp;author=J.%20N.%20Vo&amp;author=M.%20Cieslik&amp;author=Y.%20Zhang&amp;author=S.%20Shukla&amp;author=L.%20Xiao&amp;volume=176&amp;issue=4&amp;publication_year=2019&amp;pages=869-881.e13&amp;pmid=30735636&amp;doi=10.1016/j.cell.2018.12.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Li J., Sun D., Pu W., Wang J., Peng Y.. Circular RNAs in Cancer:
Biogenesis, Function, and Clinical Significance. Trends Cancer. 2020;6(4):319–336. doi: 10.1016/j.trecan.2020.01.012.</cite> [<a href="https://doi.org/10.1016/j.trecan.2020.01.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32209446/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Cancer&amp;title=Circular%20RNAs%20in%20Cancer:%20Biogenesis,%20Function,%20and%20Clinical%20Significance&amp;author=J.%20Li&amp;author=D.%20Sun&amp;author=W.%20Pu&amp;author=J.%20Wang&amp;author=Y.%20Peng&amp;volume=6&amp;issue=4&amp;publication_year=2020&amp;pages=319-336&amp;pmid=32209446&amp;doi=10.1016/j.trecan.2020.01.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Ju H. Q., Zhao Q., Wang F., Lan P., Wang Z., Zuo Z. X., Wu Q. N., Fan X. J., Mo H. Y., Chen L., Li T., Ren C., Wan X. B., Chen G., Li Y. H., Jia W. H., Xu R. H.. A circRNA
signature predicts postoperative recurrence in stage II/III colon
cancer. EMBO Mol. Med. 2019;11(10):e10168.  doi: 10.15252/emmm.201810168.</cite> [<a href="https://doi.org/10.15252/emmm.201810168" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6783650/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31475771/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Mol.%20Med.&amp;title=A%20circRNA%20signature%20predicts%20postoperative%20recurrence%20in%20stage%20II/III%20colon%20cancer&amp;author=H.%20Q.%20Ju&amp;author=Q.%20Zhao&amp;author=F.%20Wang&amp;author=P.%20Lan&amp;author=Z.%20Wang&amp;volume=11&amp;issue=10&amp;publication_year=2019&amp;pages=e10168&amp;pmid=31475771&amp;doi=10.15252/emmm.201810168&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Aufiero S., Reckman Y. J., Pinto Y. M., Creemers E. E.. Circular RNAs open
a new chapter in cardiovascular biology. Nat.
Rev. Cardiol. 2019;16(8):503–14. doi: 10.1038/s41569-019-0185-2.</cite> [<a href="https://doi.org/10.1038/s41569-019-0185-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30952956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cardiol.&amp;title=Circular%20RNAs%20open%20a%20new%20chapter%20in%20cardiovascular%20biology&amp;author=S.%20Aufiero&amp;author=Y.%20J.%20Reckman&amp;author=Y.%20M.%20Pinto&amp;author=E.%20E.%20Creemers&amp;volume=16&amp;issue=8&amp;publication_year=2019&amp;pages=503-14&amp;pmid=30952956&amp;doi=10.1038/s41569-019-0185-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Mehta S. L., Dempsey R. J., Vemuganti R.. Role of circular
RNAs in brain development
and CNS diseases. Prog. Neurobiol. 2020;186:101746.  doi: 10.1016/j.pneurobio.2020.101746.</cite> [<a href="https://doi.org/10.1016/j.pneurobio.2020.101746" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7024016/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31931031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Prog.%20Neurobiol.&amp;title=Role%20of%20circular%20RNAs%20in%20brain%20development%20and%20CNS%20diseases&amp;author=S.%20L.%20Mehta&amp;author=R.%20J.%20Dempsey&amp;author=R.%20Vemuganti&amp;volume=186&amp;publication_year=2020&amp;pages=101746&amp;pmid=31931031&amp;doi=10.1016/j.pneurobio.2020.101746&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Zhou Z., Sun B., Huang S., Zhao L.. Roles of circular RNAs in immune
regulation and autoimmune diseases. Cell Death
Dis. 2019;10(7):503. doi: 10.1038/s41419-019-1744-5.</cite> [<a href="https://doi.org/10.1038/s41419-019-1744-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6594938/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31243263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Dis.&amp;title=Roles%20of%20circular%20RNAs%20in%20immune%20regulation%20and%20autoimmune%20diseases&amp;author=Z.%20Zhou&amp;author=B.%20Sun&amp;author=S.%20Huang&amp;author=L.%20Zhao&amp;volume=10&amp;issue=7&amp;publication_year=2019&amp;pages=503&amp;pmid=31243263&amp;doi=10.1038/s41419-019-1744-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Cao C., Wang C., Dai Q., Zou Q., Wang T.. CRBPSA: CircRNA-RBP
interaction sites identification using sequence structural attention
model. BMC Biol. 2024;22(1):260. doi: 10.1186/s12915-024-02055-0.</cite> [<a href="https://doi.org/10.1186/s12915-024-02055-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11566611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39543602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Biol.&amp;title=CRBPSA:%20CircRNA-RBP%20interaction%20sites%20identification%20using%20sequence%20structural%20attention%20model&amp;author=C.%20Cao&amp;author=C.%20Wang&amp;author=Q.%20Dai&amp;author=Q.%20Zou&amp;author=T.%20Wang&amp;volume=22&amp;issue=1&amp;publication_year=2024&amp;pages=260&amp;pmid=39543602&amp;doi=10.1186/s12915-024-02055-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Conlon E. G., Manley J. L.. RNA-binding proteins in neurodegeneration:
mechanisms
in aggregate. Genes Dev. 2017;31:1509–28. doi: 10.1101/gad.304055.117.</cite> [<a href="https://doi.org/10.1101/gad.304055.117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5630017/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28912172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=RNA-binding%20proteins%20in%20neurodegeneration:%20mechanisms%20in%20aggregate&amp;author=E.%20G.%20Conlon&amp;author=J.%20L.%20Manley&amp;volume=31&amp;publication_year=2017&amp;pages=1509-28&amp;pmid=28912172&amp;doi=10.1101/gad.304055.117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Du W. W., Zhang C., Yang W., Yong T., Awan F. M., Yang B. B.. Identifying and characterizing
circRNA-protein interaction. Theranostics. 2017;7:4183–91. doi: 10.7150/thno.21299.</cite> [<a href="https://doi.org/10.7150/thno.21299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5695005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29158818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Identifying%20and%20characterizing%20circRNA-protein%20interaction.%20Theranostics.&amp;author=W.%20W.%20Du&amp;author=C.%20Zhang&amp;author=W.%20Yang&amp;author=T.%20Yong&amp;author=F.%20M.%20Awan&amp;volume=7&amp;publication_year=2017&amp;pages=4183-91&amp;pmid=29158818&amp;doi=10.7150/thno.21299&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Niu M., Zou Q., Lin C.. CRBPDL: Identification
of circRNA-RBP interaction sites
using an ensemble neural network approach. PLoS
Comput. Biol. 2022;18(1):e1009798.  doi: 10.1371/journal.pcbi.1009798.</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1009798" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8806072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35051187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol.&amp;title=CRBPDL:%20Identification%20of%20circRNA-RBP%20interaction%20sites%20using%20an%20ensemble%20neural%20network%20approach&amp;author=M.%20Niu&amp;author=Q.%20Zou&amp;author=C.%20Lin&amp;volume=18&amp;issue=1&amp;publication_year=2022&amp;pages=e1009798&amp;pmid=35051187&amp;doi=10.1371/journal.pcbi.1009798&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Cao C., Li M., Wang C., Xu L., Zou Q., Wang Y., Han W.. DGCLCMI: a deep graph collaboration
learning method to predict circRNA-miRNA
interactions. BMC Biol. 2025;23(1):104. doi: 10.1186/s12915-025-02197-9.</cite> [<a href="https://doi.org/10.1186/s12915-025-02197-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12016396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40264118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Biol.&amp;title=DGCLCMI:%20a%20deep%20graph%20collaboration%20learning%20method%20to%20predict%20circRNA-miRNA%20interactions&amp;author=C.%20Cao&amp;author=M.%20Li&amp;author=C.%20Wang&amp;author=L.%20Xu&amp;author=Q.%20Zou&amp;volume=23&amp;issue=1&amp;publication_year=2025&amp;pages=104&amp;pmid=40264118&amp;doi=10.1186/s12915-025-02197-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Huang A., Zheng H., Wu Z., Chen M., Huang Y.. Circular RNA-protein
interactions: functions, mechanisms, and identification. Theranostics. 2020;10(8):3503–3517. doi: 10.7150/thno.42174.</cite> [<a href="https://doi.org/10.7150/thno.42174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7069073/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32206104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theranostics&amp;title=Circular%20RNA-protein%20interactions:%20functions,%20mechanisms,%20and%20identification&amp;author=A.%20Huang&amp;author=H.%20Zheng&amp;author=Z.%20Wu&amp;author=M.%20Chen&amp;author=Y.%20Huang&amp;volume=10&amp;issue=8&amp;publication_year=2020&amp;pages=3503-3517&amp;pmid=32206104&amp;doi=10.7150/thno.42174&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>He A. T., Liu J., Li F., Yang B. B.. Targeting circular RNAs as a therapeutic
approach: current strategies and challenges. Signal Transduct Target Ther. 2021;6(1):185. doi: 10.1038/s41392-021-00569-5.</cite> [<a href="https://doi.org/10.1038/s41392-021-00569-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8137869/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34016945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduct%20Target%20Ther&amp;title=Targeting%20circular%20RNAs%20as%20a%20therapeutic%20approach:%20current%20strategies%20and%20challenges&amp;author=A.%20T.%20He&amp;author=J.%20Liu&amp;author=F.%20Li&amp;author=B.%20B.%20Yang&amp;volume=6&amp;issue=1&amp;publication_year=2021&amp;pages=185&amp;pmid=34016945&amp;doi=10.1038/s41392-021-00569-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Van
Nostrand E. L., Pratt G. A., Shishkin A. A., Gelboin-Burkhart C., Fang M. Y., Sundararaman B., Blue S. M., Nguyen T. B., Surka C., Elkins K., Stanton R., Rigo F., Guttman M., Yeo G. W.. Robust transcriptome-wide discovery
of RNA-binding protein binding sites with enhanced CLIP (eCLIP) Nat. Methods. 2016;13(6):508–514. doi: 10.1038/nmeth.3810.</cite> [<a href="https://doi.org/10.1038/nmeth.3810" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4887338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27018577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Robust%20transcriptome-wide%20discovery%20of%20RNA-binding%20protein%20binding%20sites%20with%20enhanced%20CLIP%20(eCLIP)&amp;author=E.%20L.%20Van%20Nostrand&amp;author=G.%20A.%20Pratt&amp;author=A.%20A.%20Shishkin&amp;author=C.%20Gelboin-Burkhart&amp;author=M.%20Y.%20Fang&amp;volume=13&amp;issue=6&amp;publication_year=2016&amp;pages=508-514&amp;pmid=27018577&amp;doi=10.1038/nmeth.3810&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Gilbert C., Svejstrup J. Q.. RNA immunoprecipitation for determining RNA-protein
associations in vivo. Curr. Protoc. Mol. Biol. 2006:27.4.1. doi: 10.1002/0471142727.mb2704s75.</cite> [<a href="https://doi.org/10.1002/0471142727.mb2704s75" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18265380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Mol.%20Biol.&amp;title=RNA%20immunoprecipitation%20for%20determining%20RNA-protein%20associations%20in%20vivo&amp;author=C.%20Gilbert&amp;author=J.%20Q.%20Svejstrup&amp;publication_year=2006&amp;pages=27.4.1&amp;pmid=18265380&amp;doi=10.1002/0471142727.mb2704s75&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Selth L. A., Gilbert C., Svejstrup J. Q.. RNA immunoprecipitation
to determine
RNA-protein associations in vivo. Cold Spring
Harbor Protoc. 2009;2009(6):pdb.prot5234.  doi: 10.1101/pdb.prot5234.</cite> [<a href="https://doi.org/10.1101/pdb.prot5234" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20147192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Protoc&amp;title=RNA%20immunoprecipitation%20to%20determine%20RNA-protein%20associations%20in%20vivo&amp;author=L.%20A.%20Selth&amp;author=C.%20Gilbert&amp;author=J.%20Q.%20Svejstrup&amp;volume=2009&amp;issue=6&amp;publication_year=2009&amp;pages=pdb.prot5234&amp;pmid=20147192&amp;doi=10.1101/pdb.prot5234&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Hellman L. M., Fried M. G.. Electrophoretic mobility shift assay (EMSA) for detecting
protein-nucleic acid interactions. Nat. Protoc. 2007;2(8):1849–61. doi: 10.1038/nprot.2007.249.</cite> [<a href="https://doi.org/10.1038/nprot.2007.249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2757439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17703195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Electrophoretic%20mobility%20shift%20assay%20(EMSA)%20for%20detecting%20protein-nucleic%20acid%20interactions&amp;author=L.%20M.%20Hellman&amp;author=M.%20G.%20Fried&amp;volume=2&amp;issue=8&amp;publication_year=2007&amp;pages=1849-61&amp;pmid=17703195&amp;doi=10.1038/nprot.2007.249&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Uhl M., Houwaart T., Corrado G., Wright P. R., Backofen R.. Computational
analysis of CLIP-seq data. Methods. 2017;118–119:60–72. doi: 10.1016/j.ymeth.2017.02.006.</cite> [<a href="https://doi.org/10.1016/j.ymeth.2017.02.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28254606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Computational%20analysis%20of%20CLIP-seq%20data&amp;author=M.%20Uhl&amp;author=T.%20Houwaart&amp;author=G.%20Corrado&amp;author=P.%20R.%20Wright&amp;author=R.%20Backofen&amp;volume=118%E2%80%93119&amp;publication_year=2017&amp;pages=60-72&amp;pmid=28254606&amp;doi=10.1016/j.ymeth.2017.02.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Zhang K., Pan X., Yang Y., Shen H. B.. CRIP: predicting circRNA-RBP-binding
sites using a codon-based encoding and hybrid deep neural networks. RNA. 2019;25(12):1604–1615. doi: 10.1261/rna.070565.119.</cite> [<a href="https://doi.org/10.1261/rna.070565.119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6859861/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31537716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RNA&amp;title=CRIP:%20predicting%20circRNA-RBP-binding%20sites%20using%20a%20codon-based%20encoding%20and%20hybrid%20deep%20neural%20networks&amp;author=K.%20Zhang&amp;author=X.%20Pan&amp;author=Y.%20Yang&amp;author=H.%20B.%20Shen&amp;volume=25&amp;issue=12&amp;publication_year=2019&amp;pages=1604-1615&amp;pmid=31537716&amp;doi=10.1261/rna.070565.119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Jia C., Bi Y., Chen J., Leier A., Li F., Song J.. PASSION: an
ensemble neural network approach for identifying the binding sites
of RBPs on circRNAs. Bioinformatics. 2020;36(15):4276–4282. doi: 10.1093/bioinformatics/btaa522.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa522" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32426818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PASSION:%20an%20ensemble%20neural%20network%20approach%20for%20identifying%20the%20binding%20sites%20of%20RBPs%20on%20circRNAs&amp;author=C.%20Jia&amp;author=Y.%20Bi&amp;author=J.%20Chen&amp;author=A.%20Leier&amp;author=F.%20Li&amp;volume=36&amp;issue=15&amp;publication_year=2020&amp;pages=4276-4282&amp;pmid=32426818&amp;doi=10.1093/bioinformatics/btaa522&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Yang Y., Hou Z., Wang Y., Ma H., Sun P., Ma Z., Wong K. C., Li X.. HCRNet: high-throughput circRNA-binding
event identification from CLIP-seq data using deep temporal convolutional
network. Brief Bioinform. 2022;23(2):bbac027.  doi: 10.1093/bib/bbac027.</cite> [<a href="https://doi.org/10.1093/bib/bbac027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35189638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform&amp;title=HCRNet:%20high-throughput%20circRNA-binding%20event%20identification%20from%20CLIP-seq%20data%20using%20deep%20temporal%20convolutional%20network&amp;author=Y.%20Yang&amp;author=Z.%20Hou&amp;author=Y.%20Wang&amp;author=H.%20Ma&amp;author=P.%20Sun&amp;volume=23&amp;issue=2&amp;publication_year=2022&amp;pages=bbac027&amp;pmid=35189638&amp;doi=10.1093/bib/bbac027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Liu L., Wei Y., Tan Z., Zhang Q., Sun J., Zhao Q.. Predicting
circRNA-RBP Binding Sites Using a Hybrid Deep Neural Network. Interdiscip Sci. 2024;16(3):635–648. doi: 10.1007/s12539-024-00616-z.</cite> [<a href="https://doi.org/10.1007/s12539-024-00616-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38381315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Interdiscip%20Sci.&amp;title=Predicting%20circRNA-RBP%20Binding%20Sites%20Using%20a%20Hybrid%20Deep%20Neural%20Network&amp;author=L.%20Liu&amp;author=Y.%20Wei&amp;author=Z.%20Tan&amp;author=Q.%20Zhang&amp;author=J.%20Sun&amp;volume=16&amp;issue=3&amp;publication_year=2024&amp;pages=635-648&amp;pmid=38381315&amp;doi=10.1007/s12539-024-00616-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Lasantha D., Vidanagamachchi S., Nallaperuma S.. CRIECNN: Ensemble convolutional neural
network and advanced feature extraction methods for the precise forecasting
of circRNA-RBP binding sites. Comput. Biol.
Med. 2024;174:108466.  doi: 10.1016/j.compbiomed.2024.108466.</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2024.108466" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38615462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Biol.%20Med.&amp;title=CRIECNN:%20Ensemble%20convolutional%20neural%20network%20and%20advanced%20feature%20extraction%20methods%20for%20the%20precise%20forecasting%20of%20circRNA-RBP%20binding%20sites&amp;author=D.%20Lasantha&amp;author=S.%20Vidanagamachchi&amp;author=S.%20Nallaperuma&amp;volume=174&amp;publication_year=2024&amp;pages=108466&amp;pmid=38615462&amp;doi=10.1016/j.compbiomed.2024.108466&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Ou X., Zhang Y., Xiong Y., Xiao Y.. Advances in RNA 3D
Structure Prediction. J. Chem. Inf Model. 2022;62(23):5862–5874. doi: 10.1021/acs.jcim.2c00939.</cite> [<a href="https://doi.org/10.1021/acs.jcim.2c00939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36451090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Inf%20Model&amp;title=Advances%20in%20RNA%203D%20Structure%20Prediction&amp;author=X.%20Ou&amp;author=Y.%20Zhang&amp;author=Y.%20Xiong&amp;author=Y.%20Xiao&amp;volume=62&amp;issue=23&amp;publication_year=2022&amp;pages=5862-5874&amp;pmid=36451090&amp;doi=10.1021/acs.jcim.2c00939&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Laing C., Schlick T.. Computational approaches to 3D modeling of RNA. J. Phys.: Condens. Matter. 2010;22(28):283101. doi: 10.1088/0953-8984/22/28/283101.</cite> [<a href="https://doi.org/10.1088/0953-8984/22/28/283101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6286080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21399271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.:%20Condens.%20Matter&amp;title=Computational%20approaches%20to%203D%20modeling%20of%20RNA&amp;author=C.%20Laing&amp;author=T.%20Schlick&amp;volume=22&amp;issue=28&amp;publication_year=2010&amp;pages=283101&amp;pmid=21399271&amp;doi=10.1088/0953-8984/22/28/283101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Li J., Chen S. J.. RNA 3D Structure
Prediction Using Coarse-Grained Models. Front
Mol. Biosci. 2021;8:720937.  doi: 10.3389/fmolb.2021.720937.</cite> [<a href="https://doi.org/10.3389/fmolb.2021.720937" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8283274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34277713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Mol.%20Biosci&amp;title=RNA%203D%20Structure%20Prediction%20Using%20Coarse-Grained%20Models&amp;author=J.%20Li&amp;author=S.%20J.%20Chen&amp;volume=8&amp;publication_year=2021&amp;pages=720937&amp;pmid=34277713&amp;doi=10.3389/fmolb.2021.720937&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Schlick T., Pyle A. M.. Opportunities and Challenges in RNA Structural Modeling
and Design. Biophys. J. 2017;113(2):225–234. doi: 10.1016/j.bpj.2016.12.037.</cite> [<a href="https://doi.org/10.1016/j.bpj.2016.12.037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5529161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28162235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.%20J.&amp;title=Opportunities%20and%20Challenges%20in%20RNA%20Structural%20Modeling%20and%20Design&amp;author=T.%20Schlick&amp;author=A.%20M.%20Pyle&amp;volume=113&amp;issue=2&amp;publication_year=2017&amp;pages=225-234&amp;pmid=28162235&amp;doi=10.1016/j.bpj.2016.12.037&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Sato K., Kato Y.. Prediction of RNA secondary
structure including pseudoknots for long
sequences. Brief Bioinform. 2022;23(1):bbab395.  doi: 10.1093/bib/bbab395.</cite> [<a href="https://doi.org/10.1093/bib/bbab395" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8769711/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34601552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform&amp;title=Prediction%20of%20RNA%20secondary%20structure%20including%20pseudoknots%20for%20long%20sequences&amp;author=K.%20Sato&amp;author=Y.%20Kato&amp;volume=23&amp;issue=1&amp;publication_year=2022&amp;pages=bbab395&amp;pmid=34601552&amp;doi=10.1093/bib/bbab395&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Dudekula D. B., Panda A. C., Grammatikakis I.. et al. CircInteractome: a web
tool for exploring circular RNAs and their interacting proteins and
microRNAs. RNA Biol. 2016;13:34–42. doi: 10.1080/15476286.2015.1128065.</cite> [<a href="https://doi.org/10.1080/15476286.2015.1128065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4829301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26669964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RNA%20Biol.&amp;title=CircInteractome:%20a%20web%20tool%20for%20exploring%20circular%20RNAs%20and%20their%20interacting%20proteins%20and%20microRNAs&amp;author=D.%20B.%20Dudekula&amp;author=A.%20C.%20Panda&amp;author=I.%20Grammatikakis&amp;volume=13&amp;publication_year=2016&amp;pages=34-42&amp;pmid=26669964&amp;doi=10.1080/15476286.2015.1128065&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Fu L., Niu B., Zhu Z., Wu S., Li W.. CD-HIT: accelerated
for clustering the next-generation sequencing data. Bioinformatics. 2012;28(23):3150–2. doi: 10.1093/bioinformatics/bts565.</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3516142/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23060610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=CD-HIT:%20accelerated%20for%20clustering%20the%20next-generation%20sequencing%20data&amp;author=L.%20Fu&amp;author=B.%20Niu&amp;author=Z.%20Zhu&amp;author=S.%20Wu&amp;author=W.%20Li&amp;volume=28&amp;issue=23&amp;publication_year=2012&amp;pages=3150-2&amp;pmid=23060610&amp;doi=10.1093/bioinformatics/bts565&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Pan X., Shen H. B.. RNA-protein binding
motifs mining with a new hybrid
deep learning based cross-domain knowledge integration approach. BMC Bioinform. 2017;18:1–14. doi: 10.1186/s12859-017-1561-8.</cite> [<a href="https://doi.org/10.1186/s12859-017-1561-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5331642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28245811/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinform&amp;title=RNA-protein%20binding%20motifs%20mining%20with%20a%20new%20hybrid%20deep%20learning%20based%20cross-domain%20knowledge%20integration%20approach&amp;author=X.%20Pan&amp;author=H.%20B.%20Shen&amp;volume=18&amp;publication_year=2017&amp;pages=1-14&amp;pmid=28245811&amp;doi=10.1186/s12859-017-1561-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Chaabane M., Williams R. M., Stephens A. T., Park J. W.. circDeep:
deep learning
approach for circular RNA classification from other long non-coding
RNA. Bioinformatics. 2020;36(1):73–80. doi: 10.1093/bioinformatics/btz537.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz537" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6956777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31268128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=circDeep:%20deep%20learning%20approach%20for%20circular%20RNA%20classification%20from%20other%20long%20non-coding%20RNA&amp;author=M.%20Chaabane&amp;author=R.%20M.%20Williams&amp;author=A.%20T.%20Stephens&amp;author=J.%20W.%20Park&amp;volume=36&amp;issue=1&amp;publication_year=2020&amp;pages=73-80&amp;pmid=31268128&amp;doi=10.1093/bioinformatics/btz537&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>

Golam Bari, A. T. M.
; 
Reaz, R.
; 
Choi, H.-J.
; 
Jeong, B.-S.
. 
DNA encoding
for splice site prediction in large DNA sequence In: Database Systems for Advanced Applications: 18th International Conference,
DASFAA 2013, International Workshops: BDMA, SNSM, SeCoP, Springer Berlin Heidelberg: Wuhan,
China, 2013.  pp 46–58 doi: 10.1007/978-3-642-40270-8_4.</cite> [<a href="https://doi.org/10.1007/978-3-642-40270-8_4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Database%20Systems%20for%20Advanced%20Applications:%2018th%20International%20Conference,%20DASFAA%202013,%20International%20Workshops:%20BDMA,%20SNSM,%20SeCoP&amp;publication_year=2013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Friedel M., Nikolajewa S., Sühnel J., Wilhelm T.. DiProDB: a database
for dinucleotide properties. Nucleic Acids Res. 2009;37:D37–D40. doi: 10.1093/nar/gkn597.</cite> [<a href="https://doi.org/10.1093/nar/gkn597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2686603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18805906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=DiProDB:%20a%20database%20for%20dinucleotide%20properties&amp;author=M.%20Friedel&amp;author=S.%20Nikolajewa&amp;author=J.%20S%C3%BChnel&amp;author=T.%20Wilhelm&amp;volume=37&amp;publication_year=2009&amp;pages=D37-D40&amp;pmid=18805906&amp;doi=10.1093/nar/gkn597&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Jiang Z., Ren Y., Ye S.. Chromobacterium violaceum infections in children: two
case reports and literature review. Eur. J.
Clin Microbiol Infect Dis. 2024;43(12):2477–2483. doi: 10.1007/s10096-024-04949-x.</cite> [<a href="https://doi.org/10.1007/s10096-024-04949-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11608380/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39417917/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin%20Microbiol%20Infect%20Dis&amp;title=Chromobacterium%20violaceum%20infections%20in%20children:%20two%20case%20reports%20and%20literature%20review&amp;author=Z.%20Jiang&amp;author=Y.%20Ren&amp;author=S.%20Ye&amp;volume=43&amp;issue=12&amp;publication_year=2024&amp;pages=2477-2483&amp;pmid=39417917&amp;doi=10.1007/s10096-024-04949-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Le Q., Mikolov T.. Distributed representations
of sentences and documents. PMLR. 2014:1188–1196.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=PMLR&amp;title=Distributed%20representations%20of%20sentences%20and%20documents&amp;author=Q.%20Le&amp;author=T.%20Mikolov&amp;publication_year=2014&amp;pages=1188-1196&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Ali S. E., Mittal A., Mathews D. H.. RNA Secondary Structure Analysis
Using RNAstructure. Curr. Protoc. 2023;3(7):e846.  doi: 10.1002/cpz1.846.</cite> [<a href="https://doi.org/10.1002/cpz1.846" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11267465/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37487054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc&amp;title=RNA%20Secondary%20Structure%20Analysis%20Using%20RNAstructure&amp;author=S.%20E.%20Ali&amp;author=A.%20Mittal&amp;author=D.%20H.%20Mathews&amp;volume=3&amp;issue=7&amp;publication_year=2023&amp;pages=e846&amp;pmid=37487054&amp;doi=10.1002/cpz1.846&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Khan A., Sohail A., Zahoora U.. et al. A survey of the recent
architectures of deep convolutional neural networks. Artif Intell Rev. 2020;53:5455–5516. doi: 10.1007/s10462-020-09825-6.</cite> [<a href="https://doi.org/10.1007/s10462-020-09825-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Artif%20Intell%20Rev.&amp;title=A%20survey%20of%20the%20recent%20architectures%20of%20deep%20convolutional%20neural%20networks&amp;author=A.%20Khan&amp;author=A.%20Sohail&amp;author=U.%20Zahoora&amp;volume=53&amp;publication_year=2020&amp;pages=5455-5516&amp;doi=10.1007/s10462-020-09825-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Dai Q., Liu Z., Wang Z., Duan X., Guo M.. GraphCDA:
a hybrid
graph representation learning framework based on GCN and GAT for predicting
disease-associated circRNAs. Brief Bioinform. 2022;23(5):bbac379.  doi: 10.1093/bib/bbac379.</cite> [<a href="https://doi.org/10.1093/bib/bbac379" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36070619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform&amp;title=GraphCDA:%20a%20hybrid%20graph%20representation%20learning%20framework%20based%20on%20GCN%20and%20GAT%20for%20predicting%20disease-associated%20circRNAs&amp;author=Q.%20Dai&amp;author=Z.%20Liu&amp;author=Z.%20Wang&amp;author=X.%20Duan&amp;author=M.%20Guo&amp;volume=23&amp;issue=5&amp;publication_year=2022&amp;pages=bbac379&amp;pmid=36070619&amp;doi=10.1093/bib/bbac379&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Varma S., Simon R.. Bias in error estimation when using cross-validation for model selection. BMC Bioinformatics. 2006;7:91.  doi: 10.1186/1471-2105-7-91.</cite> [<a href="https://doi.org/10.1186/1471-2105-7-91" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1397873/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16504092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Bias%20in%20error%20estimation%20when%20using%20cross-validation%20for%20model%20selection&amp;author=S.%20Varma&amp;author=R.%20Simon&amp;volume=7&amp;publication_year=2006&amp;pages=91&amp;pmid=16504092&amp;doi=10.1186/1471-2105-7-91&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Bradley A. P.. The use
of the area under the ROC curve in the evaluation of machine learning
algorithms. Pattern Recogn. 1997;30:1145–1159. doi: 10.1016/S0031-3203(96)00142-2.</cite> [<a href="https://doi.org/10.1016/S0031-3203(96)00142-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pattern%20Recogn&amp;title=The%20use%20of%20the%20area%20under%20the%20ROC%20curve%20in%20the%20evaluation%20of%20machine%20learning%20algorithms&amp;author=A.%20P.%20Bradley&amp;volume=30&amp;publication_year=1997&amp;pages=1145-1159&amp;doi=10.1016/S0031-3203(96)00142-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12332793/bin/ao5c04524_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c04524_si_001.pdf</a><sup> (315.9KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The source code
of this study is available at https://github.com/ydkvictory/GGCRB.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Omega are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsomega.5c04524"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ao5c04524.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (9.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332793/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332793/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332793%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332793/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332793/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332793/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40787315/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332793/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40787315/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332793/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332793/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="0ue4FKphSWwkXeTOI2Y3kQLYWMR8qpE64sUqdrW0WKHBKJmERtTjTURjf4rL4clo">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
