
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Haplotype‐based insights into seminal root angle in barley - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="67D7125089B71C131C12500010E087A3.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332326/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Haplotype‐based insights into seminal root angle in barley">
<meta name="citation_author" content="Zachary Aldiss">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Yasmine Lam">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Silvina Baraibar">
<meta name="citation_author_institution" content="InterGrain Pty Ltd. Perth, Western Australia, Australia">
<meta name="citation_author" content="Sarah Van Der Meer">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Eric Dinglasan">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Karen Massel">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Peter Crisp">
<meta name="citation_author_institution" content="School of Agriculture and Food Sustainability, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Ian Godwin">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Andrew Borrell">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Warwick, Queensland, Australia">
<meta name="citation_author" content="David Moody">
<meta name="citation_author_institution" content="InterGrain Pty Ltd. Perth, Western Australia, Australia">
<meta name="citation_author" content="Lee Hickey">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Hannah Robinson">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author_institution" content="Department for Plant Breeding, Hochschule Geisenheim University, Geisenheim, Germany">
<meta name="citation_publication_date" content="2025 Aug 7">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70088">
<meta name="citation_doi" content="10.1002/tpg2.70088">
<meta name="citation_pmid" content="40776464">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332326/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332326/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332326/pdf/TPG2-18-e70088.pdf">
<meta name="description" content="Root system architecture (RSA) plays a crucial role in crop adaptation and yield stability, especially in the context of climate change and variable growing conditions. Despite this, the genetic basis of RSA remains poorly understood in barley ...">
<meta name="og:title" content="Haplotype‐based insights into seminal root angle in barley">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Root system architecture (RSA) plays a crucial role in crop adaptation and yield stability, especially in the context of climate change and variable growing conditions. Despite this, the genetic basis of RSA remains poorly understood in barley ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332326/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332326">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70088"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70088.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332326%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332326/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332326/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332326/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70088" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Aug 7;18(3):e70088. doi: <a href="https://doi.org/10.1002/tpg2.70088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70088</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Haplotype‐based insights into seminal root angle in barley</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aldiss%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zachary Aldiss</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zachary Aldiss</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aldiss%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zachary Aldiss</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lam%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yasmine Lam</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yasmine Lam</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lam%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yasmine Lam</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baraibar%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Silvina Baraibar</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Silvina Baraibar</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>InterGrain Pty Ltd. Perth, Western Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baraibar%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Silvina Baraibar</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20Der%20Meer%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Sarah Van Der Meer</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Sarah Van Der Meer</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20Der%20Meer%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sarah Van Der Meer</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dinglasan%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Eric Dinglasan</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Eric Dinglasan</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dinglasan%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Eric Dinglasan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Massel%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Karen Massel</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Karen Massel</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Massel%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Karen Massel</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Crisp%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Peter Crisp</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Peter Crisp</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>School of Agriculture and Food Sustainability, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Crisp%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peter Crisp</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Godwin%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Ian Godwin</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Ian Godwin</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Godwin%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ian Godwin</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Borrell%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Andrew Borrell</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Andrew Borrell</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Warwick, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Borrell%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew Borrell</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moody%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">David Moody</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">David Moody</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>InterGrain Pty Ltd. Perth, Western Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moody%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">David Moody</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hickey%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Lee Hickey</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Lee Hickey</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hickey%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lee Hickey</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Robinson%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Hannah Robinson</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Hannah Robinson</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>Department for Plant Breeding, Hochschule Geisenheim University, Geisenheim, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Robinson%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hannah Robinson</span></a>
</div>
</div>
<sup>1,</sup><sup>5,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270088-aff-0001">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div id="tpg270088-aff-0002">
<sup>
<sup>2</sup>
</sup>InterGrain Pty Ltd. Perth, Western Australia, Australia</div>
<div id="tpg270088-aff-0003">
<sup>
<sup>3</sup>
</sup>School of Agriculture and Food Sustainability, The University of Queensland, Brisbane, Queensland, Australia</div>
<div id="tpg270088-aff-0004">
<sup>
<sup>4</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Warwick, Queensland, Australia</div>
<div id="tpg270088-aff-0005">
<sup>
<sup>5</sup>
</sup>Department for Plant Breeding, Hochschule Geisenheim University, Geisenheim, Germany</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Hannah Robinson, Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia. Email: <span>h.robinson1@uq.edu.au</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 25; Received 2024 Dec 18; Accepted 2025 Jun 27; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332326  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40776464/" class="usa-link">40776464</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Root system architecture (RSA) plays a crucial role in crop adaptation and yield stability, especially in the context of climate change and variable growing conditions. Despite this, the genetic basis of RSA remains poorly understood in barley (<em>Hordeum vulgare</em> L.), necessitating the need for more research to better characterize this architecture and explore the potential of diverse germplasm for trait improvement. In this study, we aimed to dissect the genetic basis of seminal root angle (SRA) by examining natural variation within a diverse global collection of 816 barley accessions, including both landraces and modern cultivars. Using a haplotype‐based mapping approach, which reflects the recombination patterns considered in breeding programs, we identified chromosomal regions associated with SRA variation. Notably, two major genomic regions on chromosome 5H were identified as novel, while a previously reported region, RAQ1, was confirmed on chromosome 3H. Our analysis revealed significant genetic diversity for SRA within the global collection, with accessions from distinct geographic origins exhibiting unique haplotype combinations. This finding underscores the quantitative nature of the SRA trait and suggests the likelihood of inadvertent selection through polygenic traits related to canopy or yield in commercial breeding programs. To further explore the genetic potential of SRA, we employed a simulation approach to evaluate the feasibility of creating an “ultimate genotype” for narrow SRA. Our results highlight the challenges associated with significantly altering SRA through traditional breeding approaches due to its quantitative, polygenic nature. Consequently, we recommend the integration of predictive and precision breeding techniques, such as genomic selection and gene editing, to effectively capture genetic diversity and accelerate RSA improvement in barley.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270088-list-10001" class="list" style="list-style-type:disc">
<li><p>First haplotype‐based mapping study for seminal root angle in a diverse global collection of barley.</p></li>
<li><p>Two novel chromosomal regions on chromosome 5H were identified.</p></li>
<li><p>The previously reported region, RAQ1, was confirmed on chromosome 3H.</p></li>
<li><p>Substantial haplotype diversity already present within elite breeding populations.</p></li>
<li><p>Seminal root angle is highly quantitative; combining predictive and precision breeding techniques are likely key to barley improvement.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270088-dl-0001">
<dt>AGG</dt>
<dd><p>Australian Grains GeneBank</p></dd>
<dt>BLUEs</dt>
<dd><p>best linear unbiased estimates</p></dd>
<dt>BLUPs</dt>
<dd><p>best linear unbiased predictors</p></dd>
<dt>CRISPR</dt>
<dd><p>clustered regularly interspaced short palindromic repeats</p></dd>
<dt>DH</dt>
<dd><p>doubled haploid</p></dd>
<dt>DRO1</dt>
<dd><p>DEEPER ROOTING1</p></dd>
<dt>IGG</dt>
<dd><p>InterGrain Core Collection</p></dd>
<dt>LD</dt>
<dd><p>linkage disequilibrium</p></dd>
<dt>LMM</dt>
<dd><p>linear mixed model</p></dd>
<dt>local GEBV</dt>
<dd><p>local genomic estimated breeding value</p></dd>
<dt>NRB</dt>
<dd><p>Northern Region Barley</p></dd>
<dt>PC</dt>
<dd><p>principal component</p></dd>
<dt>PCA</dt>
<dd><p>principal component analysis</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus</p></dd>
<dt>REML</dt>
<dd><p>restricted maximum likelihood</p></dd>
<dt>RSA</dt>
<dd><p>root system architecture</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
</dl>
<section id="tpg270088-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Barley (<em>Hordeum vulgare</em> L.) is the fourth most important cereal crop cultivated globally (FAS, <a href="#tpg270088-bib-0012" class="usa-link" aria-describedby="tpg270088-bib-0012"><sup>2023</sup></a>; Gürel et al., <a href="#tpg270088-bib-0014" class="usa-link" aria-describedby="tpg270088-bib-0014"><sup>2016</sup></a>), and well regarded for its adaptability to high‐stress growing environments, an attribute of barley's high level of diversity and genomic plasticity (Cronin et al., <a href="#tpg270088-bib-0009" class="usa-link" aria-describedby="tpg270088-bib-0009">2007</a>; Vanhala et al., <a href="#tpg270088-bib-0043" class="usa-link" aria-describedby="tpg270088-bib-0043"><sup>2004</sup></a>). With the increasing globalization of developing countries, there is a higher demand for barley grain to be used for the malting and brewing industries. Additionally, rising meat consumption in these regions drives up the demand for barley as a source for animal feed (van Dijk et al., <a href="#tpg270088-bib-0050" class="usa-link" aria-describedby="tpg270088-bib-0050"><sup>2021</sup></a>).</p>
<p>A substantial body of research suggests that optimized root systems can improve water acquisition with minimal metabolic cost, which could improve crop yields while maintaining grain quality under both favorable and adverse growing conditions (Lynch, <a href="#tpg270088-bib-0028" class="usa-link" aria-describedby="tpg270088-bib-0028">2019</a>). The spatial arrangement of roots, known as root system architecture (RSA), is influenced by many growth mechanisms, including root elongation, branching, and curving (Rich &amp; Watt, <a href="#tpg270088-bib-0034" class="usa-link" aria-describedby="tpg270088-bib-0034">2013</a>). These factors collectively determine the distribution of roots through the soil over time, influencing the plant's anchorage and access to water and nutrients (Koevoets et al., <a href="#tpg270088-bib-0025" class="usa-link" aria-describedby="tpg270088-bib-0025">2016</a>). This was illustrated in wheat (<em>Triticum aestivum</em> L.) where drought‐adapted lines had a more compact root system, possessing a higher root surface area per soil volume uniformly and allocated more root growth to the deepest soil layers, leading to improvements in yield (Manschadi et al., <a href="#tpg270088-bib-0029" class="usa-link" aria-describedby="tpg270088-bib-0029">2006</a>). Similarly, in rice, the introduction of allelic variation in the <em>DEEPER ROOTING1</em> (<em>DRO1)</em> gene reduced root growth angle and led to a narrow and deep root system that increased yield under water‐limited conditions (Kitomi et al., <a href="#tpg270088-bib-0024" class="usa-link" aria-describedby="tpg270088-bib-0024">2020</a>). The latter examples in other cereal crops demonstrate the potential for selection of an optimal RSA in barley to improve adaptation and increase production in the face of increased climate variability.</p>
<p>Despite the potential for improved adaptation and yield stability, insights into RSA have been limited due to the phenotyping bottleneck. In‐field, belowground measurements are extremely labor‐intensive, expensive and are typically destructive in nature, and as such are a challenge for large scale phenotyping in breeding programs. As an alternative, glasshouse phenotyping strategies like rhizotrons have been used for RSA observation, but are often limited by glasshouse size and experimental design (Hendrick &amp; Pregitzer, <a href="#tpg270088-bib-0016" class="usa-link" aria-describedby="tpg270088-bib-0016">1996</a>). The clear pot method provides a high‐throughput glasshouse‐based alternative approach to phenotype small‐grained cereal crops for seminal root angle (SRA), which provides a reliable proxy trait for mature root system (Kang et al., <a href="#tpg270088-bib-0019" class="usa-link" aria-describedby="tpg270088-bib-0019">2024</a>; Ober et al., <a href="#tpg270088-bib-0033" class="usa-link" aria-describedby="tpg270088-bib-0033"><sup>2021</sup></a>; Richard et al., <a href="#tpg270088-bib-0035" class="usa-link" aria-describedby="tpg270088-bib-0035"><sup>2015</sup></a>). Proxy traits such as SRA can be used in conjunction with genome‐wide mapping approaches for quantification and exploration of the underlying genetics of RSA.</p>
<p>To date, a number of studies have explored genomic regions influencing RSA in barley across adapted and diverse germplasm. Most studies have revealed substantial phenotypic variation for seminal root traits. For example, in a population comprising doubled haploid (DH) lines from the ND24260 × Flagship population and Australian cultivars, SRA ranged from 13.5° to 82.2° (Robinson et al., <a href="#tpg270088-bib-0036" class="usa-link" aria-describedby="tpg270088-bib-0036">2016</a>). Quantitative trait locus (QTL) mapping on the DH population identified seven genomic regions associated with seminal root traits across chromosomes 1H, 3H, 4H, 5H, and 6H (Robinson et al., <a href="#tpg270088-bib-0036" class="usa-link" aria-describedby="tpg270088-bib-0036">2016</a>). A major QTL influencing both seminal root angle and number on chromosome 5H (i.e., <em>qRA‐5/RAQ2</em>) was also detected in a genome‐wide association study on an Australian barley breeding population (Robinson et al., <a href="#tpg270088-bib-0037" class="usa-link" aria-describedby="tpg270088-bib-0037">2018</a>). Notably, this genomic region is co‐located with <em>qSDND</em>, a major QTL for grain dormancy (Comadran et al., <a href="#tpg270088-bib-0008" class="usa-link" aria-describedby="tpg270088-bib-0008">2009</a>; Hickey et al., <a href="#tpg270088-bib-0017" class="usa-link" aria-describedby="tpg270088-bib-0017"><sup>2012</sup></a>). Beyond adapted germplasm, Arifuzzaman et al. (<a href="#tpg270088-bib-0051" class="usa-link" aria-describedby="tpg270088-bib-0051">2014</a>) detected 33 QTLs for root and shoot traits within a DH population derived from a cross between elite cultivar Scarlett and wild accession ISR42‐8, where almost half of the favorable alleles were donated by the wild parent. A more recent study identified 55 QTLs for RSA in a panel of 221 spring barley accessions, including 55 landraces, phenotyped under controlled conditions (Jia et al., <a href="#tpg270088-bib-0018" class="usa-link" aria-describedby="tpg270088-bib-0018">2019</a>). In a panel constituting only landraces, Khodaeiaminjan et al. (<a href="#tpg270088-bib-0052" class="usa-link" aria-describedby="tpg270088-bib-0052">2023</a>) identified five root‐specific seedling QTLs under osmotic stress conditions. Despite the high likelihood of overlap, the genomic regions detected across literature have yet to be directly compared. Furthermore, the value of introducing diverse allelic variation into adapted germplasm is yet to be explored.</p>
<p>In the previous QTL mapping and genome‐wide association studies, a single marker association approach has been the norm and typically a result of lower density genotyping platforms. While useful for broad genomic region discovery, this approach does not consider the relationship between neighboring alleles through linkage disequilibrium (LD) and thus has limitations when used in marker‐assisted selection strategy. An alternative haplotype‐based mapping approach counteracts these limitations, where markers are grouped together based on LD and their cumulative effect on the phenotype calculated, and thus representative of the future chromosomal block inheritance in breeding. The use of the local genomic estimated breeding values (local GEBVs) approach serves as an example of this haplotype‐based approach (Voss‐Fels et al., <a href="#tpg270088-bib-0047" class="usa-link" aria-describedby="tpg270088-bib-0047">2019</a>). To our knowledge, haplotype‐based mapping studies are lacking for RSA in barley.</p>
<p>In the present study, a collection of 816 barley accessions from landraces and Australian modern breeding lines was analyzed to dissect the genetic bases of SRA. The local GEBV haplotype‐based mapping approach was used to identify genomic regions of interest, with a focus on the comparison between global diversity and elite breeding material. Haplotypes with negative, angle‐reducing effects were “stacked” to represent the effects of intensive selection or introgression for the narrow SRA trait in a breeding program.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270088-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270088-list-0001" class="list" style="list-style-type:disc">
<li><p>First haplotype‐based mapping study for seminal root angle in a diverse global collection of barley.</p></li>
<li><p>Two novel chromosomal regions on chromosome 5H were identified.</p></li>
<li><p>The previously reported region, RAQ1, was confirmed on chromosome 3H.</p></li>
<li><p>Substantial haplotype diversity already present within elite breeding populations.</p></li>
<li><p>Seminal root angle is highly quantitative; combining predictive and precision breeding techniques are likely key to barley improvement.</p></li>
</ul></section></section></section><section id="tpg270088-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270088-sec-0040"><h3 class="pmc_sec_title">2.1. Plant materials</h3>
<p>A panel of 816 barley accessions, consisting of a mixture of spring and winter, and two‐ and six‐rowed lines, was characterized for SRA. Within the panel, 384 of the lines were a subset of the Australian Grains GeneBank (AGG) and the remaining 432 were a subset of the InterGrain Core Collection (IGG). The AGG collection represents the global diversity of barley, including wild and modern cultivars alongside landraces, a derivative of the international barley core collection first established in 1992 and brought to Australia in 2000 (van Hintum, <a href="#tpg270088-bib-0044" class="usa-link" aria-describedby="tpg270088-bib-0044"><sup>1994</sup></a>). Conversely, the IGG is a representative collection of the genetic diversity of Australian barley breeding across the last decade, consisting of two‐row spring breeding lines and commercial cultivars. All lines in the panel were previously genotyped with 12,562 unique markers using the Illumina Infinium 40K XT SNP bead chip (Keeble‐Gagnère et al., <a href="#tpg270088-bib-0020" class="usa-link" aria-describedby="tpg270088-bib-0020">2021</a>). Genotypes were encoded in numeric variant call format as diploid genotypes, with the reference allele followed by the most common alternate (A = 1, C = 2, G = 3, T = 4) (Danecek et al., <a href="#tpg270088-bib-0011" class="usa-link" aria-describedby="tpg270088-bib-0011">2011</a>). Missing marker values for each genotype of the AGG panel were imputed using a low‐rank approximation using nuclear‐norm regularization through the R studio package softImpute (Mazumder et al., <a href="#tpg270088-bib-0030" class="usa-link" aria-describedby="tpg270088-bib-0030">2010</a>). Imputation for the IGG panel was performed by the genotyping service provider using Beagle5.4 (Browning et al., <a href="#tpg270088-bib-0004" class="usa-link" aria-describedby="tpg270088-bib-0004">2018</a>). Alternating approaches were used due to the diversity in the AGG panel and the potential for bias using an unrelated reference population in the imputation.</p></section><section id="tpg270088-sec-0050"><h3 class="pmc_sec_title">2.2. Root angle phenotyping</h3>
<p>The “clear pot” method (Richard et al., <a href="#tpg270088-bib-0035" class="usa-link" aria-describedby="tpg270088-bib-0035">2015</a>) adapted to barley (Robinson et al., <a href="#tpg270088-bib-0036" class="usa-link" aria-describedby="tpg270088-bib-0036">2016</a>) was used to evaluate SRA, where SRA is defined as the average deviation angle of the first pair of seminal roots to the vertical plane (Christopher et al., <a href="#tpg270088-bib-0007" class="usa-link" aria-describedby="tpg270088-bib-0007">2013</a>; Richard et al., <a href="#tpg270088-bib-0035" class="usa-link" aria-describedby="tpg270088-bib-0035"><sup>2015</sup></a>; Robinson et al., <a href="#tpg270088-bib-0036" class="usa-link" aria-describedby="tpg270088-bib-0036"><sup>2016</sup></a>).</p>
<p>Due to the size of the panel and number of replicates, the experiment was subset into three sub‐experiments, where appropriate concurrence was maintained for downstream multi‐experiment statistical analysis. Each experiment had six replicates for each genotype across 96 pots, containing 24 seeds per pot using the panel's relatedness, as determined by the genomic relationship matrix, as weighting of samples in a resolvable incomplete block design. Each block consisted of 32 pots spread out in a 2D array of 12 columns × 8 rows, blocked in the column direction with two columns per replicate. Experiment design was generated under a linear mixed model (LMM) framework where lines were biased against genetic similarity using a genotype relationship matrix using the R packages Optimal Design and Pedicure, an extension of ASReml‐R version 4 (Butler et al., <a href="#tpg270088-bib-0006" class="usa-link" aria-describedby="tpg270088-bib-0006">2017</a>).</p>
<p>Seeds were sown in 4 L ANOVApots of 200 mm diameter and 190 mm in height. Seeds were sown at a depth of 2 cm at a 45° angle, with the embryonic axis of each seed oriented downward, every 2.5 cm along the transparent wall of the pot to result in 24 barley seeds per pot (Robinson et al., <a href="#tpg270088-bib-0036" class="usa-link" aria-describedby="tpg270088-bib-0036">2016</a>). UQ‐23 potting mix was used, containing 70% composted pine bark (0–5 mm) and 30% cocoa peat. Plants were housed in a temperature‐controlled glasshouse, with 22°C/17°C (day/night) temperature cycle and an 18 h/6 h (day/night) photoperiod with natural lighting and growth lights to supplement.</p>
<p>The pots were thoroughly watered the day before sowing and no additional water was provided, which ensured the orientation of seeds did not shift during germination. After sowing, each clear pot was placed inside a black pot of the same dimension to prevent light from influencing root development. Images of roots were captured at 5 days after sowing using a camera when roots reached approximately 5 cm in length (iPhone 13 Pro Max). Analysis for SRA was determined using ImageJ (Schneider et al., <a href="#tpg270088-bib-0039" class="usa-link" aria-describedby="tpg270088-bib-0039">2012</a>).</p></section><section id="tpg270088-sec-0060"><h3 class="pmc_sec_title">2.3. Statistical analysis</h3>
<p>A multi‐experiment analysis was performed where an LMM was fitted to account for spatial variation and maximize the genetic variance captured in each trial. The LMM was fitted using the following equation:
</p>
<table class="disp-formula p" id="tpg270088-disp-0001"><tr><td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><mi>y</mi><mo linebreak="badbreak">=</mo><mi>X</mi><mi mathvariant="normal">τ</mi><mo linebreak="goodbreak">+</mo><mi>Z</mi><mi>u</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">ε</mi></mrow></mrow></math></td></tr></table>
<p>where <em>X</em> and <em>Z</em> are design matrices associated with fixed (<em>τ</em>) and random (<em>𝑢</em>) effects and <em>𝜀</em> is the residual errors. The fixed and random effects deemed significant in single experiment analysis were fit in the multi‐experiment model to account for spatial variability. The genotype by experiment interaction term was then explored for the best variance structure fit using a restricted maximum likelihood (REML) approach. The goodness of fit was determined by the Wald test statistics (for fixed effect terms only), the highest REML log‐likelihood value, and the smallest Akaike information criterion and Bayesian information criterion (Akaike, <a href="#tpg270088-bib-0001" class="usa-link" aria-describedby="tpg270088-bib-0001">1974</a>; Kenward &amp; Roger, <a href="#tpg270088-bib-0022" class="usa-link" aria-describedby="tpg270088-bib-0022"><sup>1997</sup></a>) (Supplementary Table <a href="#tpg270088-supinfo-0001" class="usa-link">1</a>).</p>
<p>For SRA, an unstructured variance structure was used, and linear column and row effects were fitted as fixed effects, while column, row, and pot were included as random effects (Table <a href="#tpg270088-supinfo-0001" class="usa-link">S1</a>). The auto‐regressive residual variance structure in the order of one was fitted to the residuals. The LMM was fit using ASReml‐R in the R software environment (R Core Team, <a href="#tpg270088-bib-0053" class="usa-link" aria-describedby="tpg270088-bib-0053"><sup>2024</sup></a>). The best linear unbiased estimates (BLUEs) and best linear unbiased predictors (BLUPs) were calculated using genotype as a fixed and random effects, respectively. A generalized measure of broad sense heritability was calculated to quantify the proportion of phenotypic variance that is attributed to genetic effects. The Cullis method was used to account for the unbalanced population distribution across the three trials (Cullis et al., <a href="#tpg270088-bib-0010" class="usa-link" aria-describedby="tpg270088-bib-0010">2006</a>). This was determined using the following equation:</p>
<table class="disp-formula p" id="tpg270088-disp-0002"><tr><td class="formula"><math id="jats-math-2" display="block"><mrow><mrow><msubsup><mi>H</mi><mi>Cullis</mi><mn>2</mn></msubsup><mo linebreak="badbreak">=</mo><mn>1</mn><mo linebreak="goodbreak">−</mo><mfrac><msubsup><mover accent="true"><mi>v</mi><mo>¯</mo></mover><mi mathvariant="normal">Δ</mi><mi>BLUP</mi></msubsup><mrow><mn>2</mn><mo>×</mo><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup></mrow></mfrac></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup></mrow></math></span> is the genotypic variance and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><msubsup><mover accent="true"><mi>v</mi><mo>¯</mo></mover><mi mathvariant="normal">Δ</mi><mi>BLUP</mi></msubsup></mrow></math></span> is the mean standard error of the genotype BLUPs. Figures were created using the ggplot package in R (Wickham et al., <a href="#tpg270088-bib-0048" class="usa-link" aria-describedby="tpg270088-bib-0048">2016</a>). As seminal root angle follows a continuous distribution, a variety was classified as having a narrow angle if it was less than the mean minus 1.5 times the standard deviation and wide angle if it was greater than the mean plus 1.5 times the standard deviation, respectively.</p></section><section id="tpg270088-sec-0070"><h3 class="pmc_sec_title">2.4. Population structure</h3>
<p>Single nucleotide polymorphism (SNP) marker curation was performed, removing markers with a missingness and heterozygosity of greater than 10%. Due to the high diversity of the collection, a minor allele frequency of &gt;1% was used to capture rare alleles in the population. As a result, 11,646 high quality polymorphic SNP markers with good chromosome coverage (Figure <a href="#tpg270088-supinfo-0001" class="usa-link">S1</a>) were identified for downstream analysis.</p>
<p>Principal component analysis (PCA) of the SNP data was performed to explore population structure among the 816 barley accessions. Specifically, multidimensional scaling was performed based on genetic distance matrices calculated using Rogers Distance (Rogers, <a href="#tpg270088-bib-0038" class="usa-link" aria-describedby="tpg270088-bib-0038">1972</a>). The Ward 2 error sum of squares hierarchical clustering method was used to classify lines into clusters of relatedness (Murtagh &amp; Legendre, <a href="#tpg270088-bib-0032" class="usa-link" aria-describedby="tpg270088-bib-0032">2014</a>).</p></section><section id="tpg270088-sec-0080"><h3 class="pmc_sec_title">2.5. Haplotype‐based mapping of seminal root angle</h3>
<p>SRA BLUEs and the SNP marker profiles of the panel were used in a haplotype‐based approach to calculate local GEBVs. A haplotype mapping approach was selected as it provides more trackable and achievable targets for plant breeders than QTL mapping or genome‐wide association studies due to its incorporation of LD and its ability to more successfully capture weak signal QTLs that may be lost through genome‐wide association studies (Brunner et al., <a href="#tpg270088-bib-0005" class="usa-link" aria-describedby="tpg270088-bib-0005">2024</a>; Voss‐Fels et al., <a href="#tpg270088-bib-0047" class="usa-link" aria-describedby="tpg270088-bib-0047"><sup>2019</sup></a>). Prior to analysis, genome‐wide SNP markers were assigned to blocks of high LD based on pairwise <em>r</em>
<sup>2</sup> with a minimum LD threshold of 0.3. This threshold was selected due to the lower level of LD within the diverse population, where the genome‐wide rate of decay was 0.028 (Figure <a href="#tpg270088-supinfo-0001" class="usa-link">S2</a>). Markers within this threshold or above were considered to be in the same block, while markers below are considered new blocks. A marker tolerance of <em>t</em> = 3 was set per block to account for incorrectly positioned markers. As such, when at least <em>t</em> flanking markers had lower than 0.3 <em>r</em>
<sup>2</sup> LD threshold, the block is complete. This process was used to establish the blocks genome wide, resulting in 3347 haploblocks with an average of 3.5 markers per block.</p>
<p>To identify chromosomal regions with a large impact on SRA, the effect values were assigned to the LD blocks, termed haploblocks. Marker effects within each block were predicted using the ridge regression linear unbiased prediction model and summed to calculate the haplotype effect. (Meuwissen et al., <a href="#tpg270088-bib-0031" class="usa-link" aria-describedby="tpg270088-bib-0031">2001</a>). The variance in haplotype effect for each haploblock was calculated and correlated with the number of SNPs within each haploblock to ensure that there is no bias between the number of SNPs and variance for each haploblock (Figure <a href="#tpg270088-supinfo-0001" class="usa-link">S3</a>). Haploblock variances were plotted on a Manhattan plot to identify the blocks with the highest variance for block effects. Block variance was log scaled and plotted against the range in haplotype effect for each haploblock. For the top 0.5% of high variance blocks, the start and end physical positions were collected and positioned against known QTLs and genes regulating root traits. A physical map was generated using MapChart (Voorrips, <a href="#tpg270088-bib-0046" class="usa-link" aria-describedby="tpg270088-bib-0046">2002</a>).</p></section><section id="tpg270088-sec-0090"><h3 class="pmc_sec_title">2.6. Simulated haplotype analysis</h3>
<p>For downstream haplotype analysis, the top 0.5%, 10%, and 100% of blocks with the highest scaled variance were selected. To contrast the potential improvement in SRA across both germplasm pools, a simulation approach was employed. Using only the genetic information from the IGG commercial breeding population, the maximum negative effect, favoring a narrow SRA, for each haploblock was selected and stacked to explore its cumulative phenotypic effect. This process was then repeated with AGG and then the combined dataset, both IGG and AGG genetics, to consider the broader genetic diversity offered by the global diversity panel.</p></section></section><section id="tpg270088-sec-0100"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270088-sec-0110"><h3 class="pmc_sec_title">3.1. Wide variation for SRA across the AGG and IGG collections</h3>
<p>In total, 816 accessions were phenotyped for SRA with a broad‐sense heritability of 0.65 from the multi‐experiment analysis. The BLUEs across environments revealed 10% of accessions showed wide‐angled phenotypes (SRA &gt; 75°) with a maximum of 103.7°, 8% showed narrow root phenotypes (SRA &lt; 55°) with a minimum of 39.7°, and 72% displayed a moderate root angle phenotype with an SRA greater than 55° and less than 75° (Figure <a href="#tpg270088-fig-0001" class="usa-link">1</a>). Of those displaying narrow root angles, 66% were from the AGG collection, while for those with wide root angles, AGG represented only 22% of accessions. The accession with the most extreme narrow phenotype was from the IGG collection, while the accession with the widest root angle was from the AGG collection. The greatest variance came from the IGG population with a trial mean of 66.8° and variance of 36° compared to AGG with a trial average of 65.7° and a variance of 29°.</p>
<figure class="fig xbox font-sm" id="tpg270088-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/089f/12332326/c1d43ed2e2c6/TPG2-18-e70088-g001.jpg" loading="lazy" id="jats-graphic-1" height="322" width="400" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270088-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Seminal root angle variation across the panel of barley accessions. (A) Density plot showing a smoothed distribution of the seminal root angle (SRA) trait captured across three trials; predominately InterGrain Core Collection (IGG) (orange), Mix (gray), and predominately Australian Grains GeneBank (AGG) (green). The vertical line indicates mean SRA. (B) Summary table for adjusted results (best linear unbiased predictors [BLUPs]) of each trial following spatial analysis. From left to right, the columns indicate the population included for each experiment, the mean SRA, the maximum SRA, the minimum SRA, the median, and the standard deviation.</p></figcaption></figure></section><section id="tpg270088-sec-0120"><h3 class="pmc_sec_title">3.2. Distinct population clusters within IGG and across the entire collection</h3>
<p>A PCA was performed to explore population structure within and across groups. The first principal component (PC) captures 11.7% of the variance for the IGG collection (Figure <a href="#tpg270088-fig-0002" class="usa-link">2A</a>) compared to 24.4% of the IGG + AGG collection (Figure <a href="#tpg270088-fig-0002" class="usa-link">2B</a>). Thus, more population structure is found in the combined population as more variation is explained by the first PC.</p>
<figure class="fig xbox font-sm" id="tpg270088-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/089f/12332326/d35ed8f87abc/TPG2-18-e70088-g006.jpg" loading="lazy" id="jats-graphic-3" height="321" width="720" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270088-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Visualization of population structure (principal component analysis [PCA]) of the InterGrain Core Collection (IGG) and Australian Grains GeneBank (AGG) + IGG collections. (A) Visualizing the genotypic diversity of the IGG collection. Clustering into those of European origin, Australian Northern Region Breeding, Commander, and Hindmarsh varieties. PC1 and PC2 (where PC is principal component) explains 11.7% and 9.7%, respectively. (B) Genetic relationship of AGG and IGG accessions where PC1 (24.4% of variation) and PC2 (9.6% of variation). PCA of AGG and IGG collection visualizes the continent of origin for each accession.</p></figcaption></figure><p>The PCA highlights the distinct populations present within the collections, coinciding with the accessions geographic origin. Hierarchical clustering analysis of IGG revealed four clusters aligning with four subgroups: those of European origin, Australian Northern Region Breeding, Commander, and Hindmarsh origins. A similar clustering analysis on the combined population formed three major subgroups, with the accessions from Asia appearing the most distinct (Figure <a href="#tpg270088-supinfo-0001" class="usa-link">S4</a>). Meanwhile, the largest spread of variance was with the European varieties, clustering with the IGG collection. Interestingly, the Americas have a uniquely diverse collection of germplasm being present alongside and distant from European lines. This aligns with the distribution of row type into two major groupings.</p></section><section id="tpg270088-sec-0130"><h3 class="pmc_sec_title">3.3. Chromosome 5H contains the highest variance blocks for SRA</h3>
<p>The highest variance block was detected on chromosome 5H (b001908), with 258 SNPs within the haploblock. No correlation between the number of SNPs and the variance explained was detected (Figure <a href="#tpg270088-supinfo-0001" class="usa-link">S3</a>). At a threshold of 0.5%, the top 16 highest variance blocks were identified as the primary focus for further analysis. The second highest variance block was also found on 5H (b001923). b001908 exhibited the widest range of haplotype effects on GEBVs, with estimates spanning from −0.6° to 0.6°. The second largest block, b001923, also showed both negative and positive haplotype effects on GEBVs, skewing more toward larger positive effects with a range of −0.3° to 0.4°. Overall, a good range in variance for haploblock effects was detected in this analysis showing both positive and negative haplotypes.</p></section><section id="tpg270088-sec-0140"><h3 class="pmc_sec_title">3.4. High variance haploblock co‐located with known root angle QTL <em>RAQ1</em>
</h3>
<p>Across the diversity represented in both collections, the top 0.5% of blocks were aligned with previously identified QTLs in the literature (Figure <a href="#tpg270088-fig-0004" class="usa-link">4</a>; Table <a href="#tpg270088-supinfo-0001" class="usa-link">S2</a>). The highest variance blocks, b001908 and b001923 on 5H, were novel, positioned toward the center of the chromosome. Of the significant blocks, b001027, located on 3H and comprised of 115 SNP markers co‐located with known root angle QTL <em>RAQ1</em>. The largest effect block, b001908, aligned with the known root angle gene <em>EGT2</em> and contained 258 markers, spanning approximately 194.44 Mbp, with an overall variance of 0.14. While there were no markers directly positioned on EGT2 with the closest flanking markers being 1 Mbp from the gene, these are highly linked due to being within the linkage block (Figure <a href="#tpg270088-supinfo-0001" class="usa-link">S5</a>). All other significant blocks did not align with known QTLs and were considered novel. Interestingly, a number of the top blocks were identified in the centromeric region of the chromosomes (e.g., b000083, b000527), where previous studies have predominantly identified associations on the proximal and distal ends of the chromosomes. Additionally, candidate genes associated with root traits and development did not co‐locate with any major haploblocks (Figure <a href="#tpg270088-fig-0004" class="usa-link">4</a>) a full list of haploblocks and candidate genes can be found in Table <a href="#tpg270088-supinfo-0001" class="usa-link">S3</a>. Interestingly, b001538 on 4H collocated with vascular patterning gene <em>PIN1b</em> (HORVU4Hr1G026680), although effects on root architecture in barley are currently unknown.</p>
<figure class="fig xbox font-sm" id="tpg270088-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/089f/12332326/c7201ed7776e/TPG2-18-e70088-g005.jpg" loading="lazy" id="jats-graphic-7" height="810" width="800" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270088-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Alignment of discovered haploblocks with previously reported root trait quantitative trait loci (QTLs). Previously reported QTLs are listed on the left side of the chromosome with the corresponding physical position (Mbp) shown on the right. Highlighted in green are the significant haploblocks. Those that do not align with known root trait QTLs are considered novel. b001027 aligns with <em>RAQ1</em> and b001908 aligns with <em>EGT2</em>. All blue markers on the physical map are specifically associated with root angle or number. Pink highlights known genes regulating root or aboveground development. Full marker information is listed in Table <a href="#tpg270088-supinfo-0001" class="usa-link">S3</a> (Arifuzzaman et al., <a href="#tpg270088-bib-0051" class="usa-link" aria-describedby="tpg270088-bib-0051">2014</a>; Chen et al., <a href="#tpg270088-bib-0054" class="usa-link" aria-describedby="tpg270088-bib-0054"><sup>2009</sup></a>; Cheng et al., <a href="#tpg270088-bib-0055" class="usa-link" aria-describedby="tpg270088-bib-0055"><sup>2023</sup></a>; Chloupek et al., <a href="#tpg270088-bib-0056" class="usa-link" aria-describedby="tpg270088-bib-0056"><sup>2006</sup></a>; Diab et al., <a href="#tpg270088-bib-0057" class="usa-link" aria-describedby="tpg270088-bib-0057"><sup>2004</sup></a>; Siahsar &amp; Narouei, <a href="#tpg270088-bib-0058" class="usa-link" aria-describedby="tpg270088-bib-0058"><sup>2010</sup></a>; Fusi et al., <a href="#tpg270088-bib-0059" class="usa-link" aria-describedby="tpg270088-bib-0059"><sup>2022</sup></a>; Kirschner et al., <a href="#tpg270088-bib-0023" class="usa-link" aria-describedby="tpg270088-bib-0023"><sup>2021</sup></a>; Robinson et al., <a href="#tpg270088-bib-0036" class="usa-link" aria-describedby="tpg270088-bib-0036"><sup>2016</sup></a><sup>, </sup><a href="#tpg270088-bib-0037" class="usa-link" aria-describedby="tpg270088-bib-0037"><sup>2018</sup></a>; Siddiqui et al. <a href="#tpg270088-bib-0060" class="usa-link" aria-describedby="tpg270088-bib-0060"><sup>2024</sup></a>; Teulat et al., <a href="#tpg270088-bib-0061" class="usa-link" aria-describedby="tpg270088-bib-0061"><sup>2001</sup></a>; Uga et al., <a href="#tpg270088-bib-0062" class="usa-link" aria-describedby="tpg270088-bib-0062"><sup>2011</sup></a>; Voss‐Fels et al., <a href="#tpg270088-bib-0063" class="usa-link" aria-describedby="tpg270088-bib-0063"><sup>2018</sup></a>).</p></figcaption></figure></section><section id="tpg270088-sec-0150"><h3 class="pmc_sec_title">3.5. Block variance clustering reveals shared and population‐specific variation</h3>
<p>Contrasting the block variances of the IGG and AGG populations revealed the highest variant block, b001908, shared its high haploblock variance in both populations. Similarly, b001923, the second highest block overall, follows a similar trend with only slightly higher variance in the IGG collection (Figure <a href="#tpg270088-fig-0005" class="usa-link">5A</a>). To explore the variance of the smaller effect blocks, the variance was log‐scaled. Smaller effect blocks separated out into two subgroups, those with high variance in both the diverse AGG collection and the IGG breeding population, in the top cluster, and those exclusive to the AGG collection clustering at the bottom (Figure <a href="#tpg270088-fig-0005" class="usa-link">5B</a>). As indicated by color, the top cluster has a higher proportion of negative haplotype effect blocks (yellow), while the bottom cluster has a more even spread of positive (blue) and negative effects.</p>
<figure class="fig xbox font-sm" id="tpg270088-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/089f/12332326/a99cb18b3dc2/TPG2-18-e70088-g002.jpg" loading="lazy" id="jats-graphic-9" height="369" width="600" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270088-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparing haploblock variance between the diverse Australian Grains GeneBank (AGG) collection and the InterGrain Core Collection (IGG) breeding population. (A) The raw variance of each haploblock for both the IGG collection (<em>y</em>‐axis) and the AGG collection (<em>x</em>‐axis). The haploblock effect variance is on a linear scale, and higher values indicate a stronger influence of the haploblock on the GEBVS for seminal root angle (SRA). (B) Log‐scaled variance of the values from A, particularly to explore the lower variance blocks without the skew caused by the highest variance blocks. The point size represents block effect in both panels, with yellow representing negative effects and blue representing positive effects on GEBVs for SRA.</p></figcaption></figure></section><section id="tpg270088-sec-0160"><h3 class="pmc_sec_title">3.6. Haplotype effects cluster based on population structure</h3>
<p>The origins of the haplotypes for the highest variance block b00908 and the haploblock collocating with known root angle QTL <em>RAQ1</em> were explored through the population structure described by the PCA. For b001908, in the IGG population, the strongest negative haplotypes originate from the Hindmarsh background, while the most positive effects originate from the northern region cluster (Figure <a href="#tpg270088-fig-0006" class="usa-link">6A</a>). Haplogroups were also created based on flanking markers of EGT2 located within b001908.There were two major haplogroups, CC‐CC and CC‐TT, capturing 47% and 44.4% of the population, respectively (Figure <a href="#tpg270088-supinfo-0001" class="usa-link">S7A</a>). Haplogroup CC‐TT was significantly associated with GEBVs for SRA, showing an approximate 2° increase compared to the reference group CC‐CC (Figure <a href="#tpg270088-supinfo-0001" class="usa-link">S7B</a>). However, there was no significant association between haplogroup frequency and population. In the AGG global diversity, the strongest negative effect haplotypes originate from Europe with the remaining positive effect haplotypes appearing in European and Asian accessions (Figure <a href="#tpg270088-fig-0006" class="usa-link">6B</a>). <em>RAQ1</em> showed widespread presence of positive effect haplotypes in the IGG lines, while the strong negative effect haplotypes were rare but tended to appear more frequently in the European accessions (Figure <a href="#tpg270088-fig-0006" class="usa-link">6C</a>). In the AGG collection, <em>RAQ1</em> shows null effect haplotypes in both Asian and some European subgroups, while the strong negative effect haplotypes cluster in the Americas (Figure <a href="#tpg270088-fig-0006" class="usa-link">6D</a>).</p>
<figure class="fig xbox font-sm" id="tpg270088-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/089f/12332326/bded973ee3fb/TPG2-18-e70088-g004.jpg" loading="lazy" id="jats-graphic-11" height="659" width="640" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270088-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Principal component analysis (PCA) of highest variance and co‐located haploblocks shows haplotype effect with comparisons to accession origins. Each point represents a line from either the InterGrain Core Collection (IGG) panel (left) or the Australian Grains GeneBank (AGG) panel (right) and is colored by the effect size and direction of the line's individual haplotype within the haploblock. Positive effects (yellow), zero effect (green), and negative effect (blue) are plotted for b001908 (A and B), the highest variance haploblock located on 5H for IGG and AGG populations, and b001027 (C and D), which collocated with known root angle quantitative trait locus (QTL) <em>RAQ1</em> on 3H, haplotype effects colored on IGG and AGG population structures with origin circled.</p></figcaption></figure></section><section id="tpg270088-sec-0170"><h3 class="pmc_sec_title">3.7. Targeted haplotype stacking for “ultimate” narrow rooting genotype</h3>
<p>Of the 3347 haploblocks detected, 2815 had haplotype effects conferring a narrow SRA and reducing the mean estimated GEBV for SRA. Selection of the 0.5% of the highest variance blocks produced a mean shift in the GEBV for SRA of approximately −3° for the stacking of 16 blocks, with no difference in GEBV across the three genetic pools (Figure <a href="#tpg270088-fig-0007" class="usa-link">7A</a>). This outcome was similar following selection of the top 10% of blocks, producing a shift of −15° for all populations, with a slight difference in the GEBV for SRA beginning to appear between AGG and IGG at 280 blocks (Figure <a href="#tpg270088-fig-0007" class="usa-link">7B</a>). The ultimate narrow root angle genotype was simulated by incorporating all available highest effect haplotypes for both populations, shifting the mean GEBV for SRA by approximately 30° (Figure <a href="#tpg270088-fig-0007" class="usa-link">7C</a>). Using IGG genetics alone shows a maximum shift of 27° (Figure <a href="#tpg270088-fig-0007" class="usa-link">7C</a>). There was no difference between IGG + AGG and AGG alone, suggesting the additional 4° is coming from exclusively AGG genetics. The SRA haploblocks are highly quantitative with the difference in diversity between AGG and IGG beginning to show after stacking approximately 100 blocks, with the maximum difference in GEBV for SRA present after approximately 2000 blocks before reaching a plateau.</p>
<figure class="fig xbox font-sm" id="tpg270088-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/089f/12332326/857a59d2a99b/TPG2-18-e70088-g003.jpg" loading="lazy" id="jats-graphic-13" height="389" width="720" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270088-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Haploblock stacking analysis for narrow root angle. Haplotype blocks present in three conditions, utilizing only InterGrain Core Collection (IGG) germplasm (orange), only Australian Grains GeneBank (AGG) (green), and all available germplasm (AGG and IGG; gray), for the genomic region associated with seminal root angle (SRA) are selected in order of highest variance. The <em>x</em>‐axis represents the number of blocks fixed at the highest haplotype effect for that block. The <em>y</em>‐axis is the mean change in predicted SRA (GEBV) through cumulative addition of negative effect haplotypes. (A) Visualizing the mean change in GEBV through selecting the top 0.5% (top) and (B) 10% (bottom) of haploblocks sorted by variance. (C) The ultimate genotype produced through stacking all 2815 haploblocks with conferring narrow SRA GEBV.</p></figcaption></figure></section></section><section id="tpg270088-sec-0180"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>In a comparison of the two groups, the diverse AGG collection displayed a higher frequency of wide SRAs, while the IGG breeding collection was skewed toward narrower SRAs. This is not surprising, based on the broad global genetic diversity that the AGG collection captures compared to the IGG that clusters with the European germplasm within the AGG (Figure <a href="#tpg270088-fig-0002" class="usa-link">2B</a>). This is also a potential reflection of inadvertent selection and adaptation toward the dominant growing environments within Australia, where a narrow and deep RSA may be beneficial. However, it is important to note that although skewed toward a narrower SRA, wide variation was still observed for the IGG collection, where SRA ranged from 41.8° to 99.6° (Figure <a href="#tpg270088-fig-0001" class="usa-link">1B</a>).</p>
<p>Even narrower SRA phenotypes were observed in the previous studies examining Australian germplasm using the same phenotyping method. For example, in the ND24260 × Flagship DH population, SRA ranged from 16.4° to 70.5° (Robinson et al., <a href="#tpg270088-bib-0036" class="usa-link" aria-describedby="tpg270088-bib-0036">2016</a>). Similarly, SRA ranged from 12.0° to 89.4° in elite germplasm from the Northern Region Barley (NRB) breeding program (Robinson et al., <a href="#tpg270088-bib-0037" class="usa-link" aria-describedby="tpg270088-bib-0037">2018</a>). While the range of approximately 70° is consistent for both the breeding populations, the maximum and minimum are shifted by approximately 20°. Again, this could reflect inadvertent environmental adaptation whereby the IGG program typically breeds for broad adaptation across Australia, while the NRB program specifically targets the northern grain‐growing region. This region has a unique growing environment, characterized by high clay content soils with high water‐holding capacity, and high frequency of summer rains yet minimal growing‐season rainfall. As a result, crops are typically reliant on stored soil moisture throughout the growing season. This suggests that the NRB population may have narrower RSA to access deep stored soil moisture, similar to the previously described “Steep, Cheap, and Deep” ideotype (Lynch, <a href="#tpg270088-bib-0027" class="usa-link" aria-describedby="tpg270088-bib-0027">2013</a>). This also suggests the allelic variation driving this narrow RSA in the Australian NRB and DH population may not be captured in the present study and is likely originating from the North Dakota breeding germplasm ancestry common in both populations. Future research combining a selection of the NRB breeding lines with the panel examined in the current study could be beneficial to explore genomic regions influencing narrow SRA from a haplotype‐based mapping approach.</p>
<p>A comparison of the phenotypic diversity of the AGG collection, compared with a diverse spring barley collection composed of cultivars, landraces, and breeding lines from 51 countries, revealed similar phenotypic variation in SRA. Whereby, the AGG collection SRA ranges from 48.3° to 103.7° with a mean of 65.7°, and the spring barley panel ranges from 46.7° to 131.7° with a mean of 70.5° (Jia et al., <a href="#tpg270088-bib-0018" class="usa-link" aria-describedby="tpg270088-bib-0018">2019</a>). While the minimum and mean of the SRA for both populations are similar, the maximum observed is higher in the spring barley diversity panel. This difference may be due to variations in the phenotyping methodology, particularly the later date and greater depth at which measurements were taken. However, this may also reflect the fact that most accessions in the spring barley panel are from Europe, which has cooler temperatures and higher in‐season rainfall compared to the African and Australian varieties in the AGG collection. In comparison, commercial lines bred for Nordic climates were recently examined for SRA and found to have a mean of 95°, with a range from 43.2° to 151.3° (Smith et al., <a href="#tpg270088-bib-0041" class="usa-link" aria-describedby="tpg270088-bib-0041">2024</a>). It is important to note that while these trends align with previously reported ideotypes of wide RSA being beneficial for shallow light soils with high in‐season rainfall and narrow RSA in converse environmental conditions (Lynch, <a href="#tpg270088-bib-0027" class="usa-link" aria-describedby="tpg270088-bib-0027">2013</a>), the benefits of these ideotypes within these environment types are yet to be confirmed. Quantifying the value of different RSA in specific growing conditions for improved yield stability is critical for breeding. However, this remains underrepresented in the literature due to the challenges associated with reliably measuring below‐ground RSA.</p>
<p>Beyond commercial cultivars, landraces and domesticated relatives have been evaluated for centuries in local agricultural communities where they have been adapted to specific environmental conditions. This global pool of diversity, represented by the AGG collection, provides a significantly larger variation in genotypes for potential introgression into elite breeding material, represented by the IGG collection. The population structure supports this, with the majority of IGG material grouping with the AGG lines of European origin. This is consistent with the origins of barley cultivation in Australia, where at the turn of the 19th to 20th century, two‐rowed spring landrace selections from the UK were introduced (Seko, <a href="#tpg270088-bib-0040" class="usa-link" aria-describedby="tpg270088-bib-0040">1987</a>). This approach has continued, with donor material brought into Australia from Europe, such as cultivars Archer or Chevallier, as the germplasm is adapted to higher rainfall zones (Baumer &amp; Cais, <a href="#tpg270088-bib-0064" class="usa-link" aria-describedby="tpg270088-bib-0064">2000</a>).</p>
<p>Within the top 0.5% of the highest variance haploblocks, b001027 on 3H was found to collocate exactly with the previously reported QTL, <em>RAQ1</em> (Figure <a href="#tpg270088-fig-0004" class="usa-link">4</a>). Within the IGG population, one of the highest effect haplotypes for <em>RAQ1</em> is present within Flagship, the malting variety where <em>RAQ1</em> was originally detected (Robinson et al., <a href="#tpg270088-bib-0036" class="usa-link" aria-describedby="tpg270088-bib-0036">2016</a>). Additionally, many of the larger top blocks were in the centromeric region of the chromosomes (Figure <a href="#tpg270088-fig-0004" class="usa-link">4</a>), a result of the haplotype mapping incorporating LD, where recombination occurs at lower rates in the more central regions of the chromosomes, resulting in larger blocks. When exploring the effects across the entire IGG collection, the strongest negative effect trends toward the cluster with European origins, while sporadic in its distribution, the remaining accessions in the panel have strong positive effect haplotypes, demonstrating the negative effects are rare (Figure <a href="#tpg270088-fig-0006" class="usa-link">6B</a>). AGG has a higher proportion of null or negative effect blocks, with the positive effects clustering toward African and American origins while the negative and null effects are clustering in Europe, consistent with IGG and Asia (Figures <a href="#tpg270088-fig-0003" class="usa-link">3B</a> and <a href="#tpg270088-fig-0006" class="usa-link">6B</a>). Interestingly, while <em>RAQ1</em> explained 3.8% of phenotypic variation in the double haploid population, b001027 aligning with RAQ1 explains a significantly lower percentage, only 1.1%, likely reflecting the higher genetic diversity within the AGG and IGG collections (Table <a href="#tpg270088-supinfo-0001" class="usa-link">S2</a>).</p>
<figure class="fig xbox font-sm" id="tpg270088-fig-0003"><h3 class="obj_head">FIGURE 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/089f/12332326/ad4cd6015ef4/TPG2-18-e70088-g007.jpg" loading="lazy" id="jats-graphic-5" height="244" width="720" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270088-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Haploblocks with high variance for effect on GEBVs for seminal root angle. (A) Manhattan plot displaying the high variance for haploblock effects. Each point represents a haploblock, where the <em>x</em>‐axis is the genomic position on each chromosome and the <em>y</em>‐axis indicates haploblock variance where higher variance indicates a greater range of haplotype effects, thus greater genetic influence on the trait. (B) Scatter plot illustrating the relationship between haplotype effects and scaled block variance for each haploblock. The <em>x</em>‐axis represents the haplotype effect where each dot is an individual haplotype, while the <em>y</em>‐axis represents the scaled variance of each haploblock.</p></figcaption></figure><p>While the top two highest variance blocks were found on 5H, the chromosome that contains several previously detected seminal root trait QTLs, they do not align with these known regions and are considered novel (Robinson et al., <a href="#tpg270088-bib-0036" class="usa-link" aria-describedby="tpg270088-bib-0036">2016</a>, <a href="#tpg270088-bib-0037" class="usa-link" aria-describedby="tpg270088-bib-0037">2018</a>). This may be due to the significant differences in diversity of the collections, where previous studies examined a DH population and a breeding program targeting the Australian Northern Grain Growing region (Robinson et al., <a href="#tpg270088-bib-0036" class="usa-link" aria-describedby="tpg270088-bib-0036">2016</a>). While DH populations enhance the statistical power to detect rare alleles through an increase in allelic frequency throughout the population, they are limited to the genetic diversity present within the two parents.</p>
<p>Comparing the haploblock variances between IGG and AGG, the highest variance blocks, b001908 and b001923, have equal variance for marker effects in both populations, demonstrating a strong presence in both the IGG and the AGG (Figure <a href="#tpg270088-fig-0005" class="usa-link">5A</a>
). This suggests that although the IGG breeding program has not specifically targeted RSA, they may have likely unintentionally selected for RSA through the selection of aboveground polygenic traits crucial for adaptation, such as flowering time and yield. The presence of these high value haploblocks within the IGG collection is highly advantageous for breeding and simplifies selection, since there is no requirement for labor‐intensive introgression. While high variance blocks associated with SRA have equal variance in both collections, smaller effect blocks separate into two distinct clusters (Figure <a href="#tpg270088-fig-0005" class="usa-link">5B</a>). Interestingly, the bottom cluster consists of haploblocks that have low variance in the IGG collection yet high variance in the AGG panel, demonstrating its contribution of allelic diversity (Figure <a href="#tpg270088-fig-0005" class="usa-link">5B</a>). The reflected diversity of this cluster is a prime target for introgression into the IGG breeding collection; however, the value of such is unclear and likely minimal given the small effect sizes of the haplotypes.</p>
<p>Through a simulated approach, we can explore the effects of cumulatively staking the maximum effect haplotypes to develop the “ultimate genotype” (Hayes et al., <a href="#tpg270088-bib-0015" class="usa-link" aria-describedby="tpg270088-bib-0015">2024</a>; Kemper et al., <a href="#tpg270088-bib-0021" class="usa-link" aria-describedby="tpg270088-bib-0021"><sup>2012</sup></a>), where here we focus on developing a narrow RSA following the “Steep, Cheap, and Deep” ideotype. Using this approach, we demonstrate the quantitative nature of the trait, as it requires over 100 blocks to detect a difference between AGG and IGG collections (Figure <a href="#tpg270088-fig-0007" class="usa-link">7B</a>; Figure <a href="#tpg270088-supinfo-0001" class="usa-link">S5</a>). Considering the complexity of the trait and the genetic architecture of similar traits, the highly quantitative nature of SRA is not surprising, but it does make a traditional marker‐assisted selection approach unfeasible. A whole‐genome approach, such as genomic selection, is more efficient and would allow downstream cross‐prediction, where artificial intelligence is used to optimally select high value parents for breeding of optimal RSA (Villiers et al., <a href="#tpg270088-bib-0045" class="usa-link" aria-describedby="tpg270088-bib-0045">2024</a>). One such approach incorporates haplotype selection into a genetic simulation where parent selection can be specifically determined for breeding the ultimate genotype, in this case for a narrow RSA (Villiers et al., <a href="#tpg270088-bib-0045" class="usa-link" aria-describedby="tpg270088-bib-0045">2024</a>). However, the potential for modifying this trait is limited. Using all available genetics within the IGG collection, the maximum trait shift is −27°. When combining both IGG and AGG, the shift is −30°, demonstrating that even with global diversity, the SRA cannot be significantly altered beyond what is already found in nature.</p>
<p>To make large modifications to the RSA, the mechanistic pathway likely needs to be unveiled and then exploited. In this context, key genes in the root developmental pathway that may have minimal allelic variation in current barley germplasm need to be discovered. If they have a large effect, these genes could be targeted to create beneficial variation using biotechnological tools such as clustered regularly interspaced short palindromic repeats (CRISPR) genome editing. While there is still limited knowledge of the molecular pathways influencing SRA in cereals, knowledge of developmental genes in model species such as Arabidopsis could be leveraged to accelerate progress in barley (Kong et al., <a href="#tpg270088-bib-0026" class="usa-link" aria-describedby="tpg270088-bib-0026">2014</a>; Zhang et al., <a href="#tpg270088-bib-0049" class="usa-link" aria-describedby="tpg270088-bib-0049"><sup>2018</sup></a>). Recent examples include knockout of a single gene (<em>PIN‐FORMED 2</em>) in barley resulting in reduced gravitropic response and shallower root architecture with an increase in root angle by over 100% (Aldiss et al., <a href="#tpg270088-bib-0002" class="usa-link" aria-describedby="tpg270088-bib-0002">2024</a>). Alternatively, the reduced function of <em>EGT2</em> results in an increase in gravitropic response, causing a reduction in root angle by 50% (Kirschner et al., <a href="#tpg270088-bib-0023" class="usa-link" aria-describedby="tpg270088-bib-0023">2021</a>). A similar approach has been illustrated in rice, where introduction of allelic variation in the <em>DRO1</em> gene decreased root growth angle and led to deeper rooting plants and increased yield under water‐limited conditions (Kitomi et al., <a href="#tpg270088-bib-0024" class="usa-link" aria-describedby="tpg270088-bib-0024">2020</a>). With advancements in genomic selection and gene editing technologies, these predictive and precision breeding tools create a toolbox that can be exploited to enhance genomic breeding. This integration has the potential to fast‐track cereal crop improvement, enabling rapid adaptation to changing growing conditions (Feng et al., <a href="#tpg270088-bib-0013" class="usa-link" aria-describedby="tpg270088-bib-0013">2017</a>).</p></section><section id="tpg270088-sec-0190"><h2 class="pmc_sec_title">5. CONCLUSION</h2>
<p>We assessed the genetic basis of SRA in a large panel of barley accessions and compared the differences between an Australian commercial breeding collection and a diverse collection of global gene‐bank germplasm. Using a haplotype‐based mapping approach, the study successfully identified two novel major genomic regions on chromosome 5H and confirmed a previously identified QTL on chromosome 3H, <em>RAQ1</em>, influencing SRA. The findings highlight the quantitative nature of the SRA trait and the likelihood of inadvertent selection through related polygenic traits within commercial breeding. To achieve significant improvement in barley in RSA and enhance adaptation, it is essential to harness predictive and precision breeding techniques that combine popular technologies such as genomic selection and gene editing.</p></section><section id="tpg270088-sec-0200"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Zachary Aldiss</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; writing—original draft; writing—review and editing. <strong>Yasmine Lam</strong>: Methodology; writing—review and editing. <strong>Silvina Baraibar</strong>: Resources; validation; writing—review and editing. <strong>Sarah Van Der Meer</strong>: Investigation; methodology; writing—review and editing. <strong>Eric Dinglasan</strong>: Investigation; validation; writing—review and editing. <strong>Karen Massel</strong>: Supervision; writing—review and editing. <strong>Peter A. Crisp</strong>: Supervision; writing—review and editing. <strong>Ian Godwin</strong>: Supervision; writing—review and editing. <strong>Andrew Borrell</strong>: Supervision; writing—review and editing. <strong>David Moody</strong>: Resources; writing—review and editing. <strong>Lee Hickey</strong>: Conceptualization; project administration; supervision; writing—review and editing. <strong>Hannah Maria Robinson</strong>: Conceptualization; project administration; supervision; validation; writing—review and editing.</p></section><section id="tpg270088-sec-0220"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec22"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270088-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270088-supitem-0001"><div class="caption p">
<p>
<strong>Supplementary Figure 1</strong>. Marker distribution and density of curated SNP markers across chromosomes 1H through to 7H of the barley genome.</p>
<p>
<strong>Supplementary Figure 2</strong>. Pairwise marker LD‐matrices were calculated for each chromosome and LD‐decay was analysed showing an r2 of 0.2 with a sharp drop to 0.1 genome wide.</p>
<p>
<strong>Supplementary Figure 3</strong>. Relationship between haploblock SNP density and haplotype effect variance. The x‐axis shows the number of SNPs per haplotype block, and the y‐axis represents the variance of haplotype effects within each block. No clear linear trend is observed, suggesting that the number of SNPs in a block does not directly predict the magnitude of haplotype effect variance.</p>
<p>
<strong>Supplementary Figure 4</strong>. PCA of the AGG collection based on genome‐wide SNP variation. PC1 explains 29.5% and PC2 explains 13.9% of the total genetic variance. Individuals are coloured by passport information to reflect geographic origin: Africa (teal green), Americas (orange), Asia (purple), Europe (pink), Oceania (light green), United Kingdom (yellow), and individuals with no passport data (NA) are uncoloured.</p>
<p>
<strong>Supplemental Figure 5</strong>. Physical positioning of EGT2 and marker identification. <strong>(A)</strong> Positioned within significant haploblock b001908, EGT2 (HORVU5Hr1G027890) was co‐located. (B) The closest SNP markers flanking EGT2 were identified at approximately 162.5 M (chr5H_00023624_161513719‐161674224_3547) and 159.5 M (chr5H_00023425_159395855‐159466194_9375) base pairs respectively. Both markers are approximately 1 M base pairs from the EGT2 locus. These markers were used to classify EGT2 haplogroups for subsequent analysis.</p>
<p>
<strong>Supplemental Figure 6</strong>. EGT2 haplogroup frequency and association with SRA. <strong>(A)</strong> Haplogroups were created based on the marker information of the markers flanking EGT2 for each genotype. SNP combination CC‐CC was the largest frequency haplogroup followed by CC‐TT. (<strong>B)</strong> The highest frequency haplogroup was used as the reference for ANOVA to determine significant associations between haplogroups and SRA. CC‐TT was shown to have a significant association (p &lt; 0.05), with an increased SRA of two degrees.</p>
<p>
<strong>Supplementary Figure 7</strong>. Haploblock stacking for wide root phenotype. Each of the three panels displays the relationship between the number of beneficial haplotype blocks stacked (x‐axis) and the seminal root angle in degrees (y‐axis). The grey line represents the combined IGG and AGG populations, the orange line represents the IGG population alone, and the green line represents the AGG population alone. The plots illustrate the cumulative effect of stacking haplotypes associated with increased root angle, contributing to a wider root system architecture. The first panel considers the top 0.5% of haplotypes with the strongest positive effect on root angle, the second includes the top 10%, and the third includes all (100%) haplotypes.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12332326/bin/TPG2-18-e70088-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70088-s001.xlsx</a><sup> (45.2KB, xlsx) </sup>
</div></div></section></section><section id="tpg270088-sec-0210" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This research was funded by the Australian Research Council Industry Linkage Project ‘Digging Deeper to Improve Yield Stability’ (LP200200927) in collaboration with InterGrain Pty. Ltd. Australian barley breeding lines were also kindly provided by InterGrain. LTH was supported through an ARC Future Fellowship (FT220100350). The authors are grateful for the technical and breeding staff at InterGrain Pty Ltd for the supply of germplasm and genotype data utilized in this study.</p>
<p>Open access publishing facilitated by The University of Queensland, as part of the Wiley ‐ The University of Queensland agreement via the Council of Australian University Librarians.</p></section><section id="notes1"><p>


Aldiss, Z.
, 
Lam, Y.
, 
Baraibar, S.
, 
Van Der Meer, S.
, 
Dinglasan, E.
, 
Massel, K.
, 
Crisp, P.
, 
Godwin, I.
, 
Borrell, A.
, 
Moody, D.
, 
Hickey, L.
, &amp; 
Robinson, H.
 (2025). Haplotype‐based insights into seminal root angle in barley. The Plant Genome, 18, e70088. 10.1002/tpg2.70088

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270088-note-0001"><p>Assigned to Associate Editor Roberto Tuberosa.</p></div></div></section></section><section id="tpg270088-sec-0240"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The data that support the findings of this study are available from InterGrain Pty Ltd, but restrictions apply to the availability of these data, which were used under licence for the current study, and so are not publicly available. Data are however available from the authors upon reasonable request and with permission of InterGrain Pty Ltd.</p></section><section id="tpg270088-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270088-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270088-bib-0001">
<cite>

Akaike, H.
 (1974). A new look at the statistical model identification. IEEE Transactions on Automatic Control, 19, 716–723. 10.1109/TAC.1974.1100705
</cite> [<a href="https://doi.org/10.1109/TAC.1974.1100705" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Transactions%20on%20Automatic%20Control&amp;title=A%20new%20look%20at%20the%20statistical%20model%20identification&amp;author=H.%20Akaike&amp;volume=19&amp;publication_year=1974&amp;pages=716-723&amp;doi=10.1109/TAC.1974.1100705&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0002">
<cite>

Aldiss, Z.
, 
Robinson, H.
, 
Lam, Y.
, 
Dixon, R.
, 
Crisp, P.
, 
Godwin, I.
, 
Borrell, A.
, 
Hickey, L.
, &amp; 
Massel, K.
 (2024). The <em>PIN2</em> ortholog in barley modifies root gravitropism and architecture without impacting the shoot. <em>bioRxiv</em>. 10.1101/2024.03.28.587117
</cite> [<a href="https://doi.org/10.1101/2024.03.28.587117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12328839/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40770375/" class="usa-link">PubMed</a>]</li>
<li id="tpg270088-bib-0051">
<cite>

Arifuzzaman, M.
, 
Sayed, M. A.
, 
Muzammil, S.
, 
Pillen, K.
, 
Schumann, H.
, 
Naz, A. A.
, &amp; 
Léon, J.
 (2014). Detection and validation of novel QTL for shoot and root traits in barley (<em>Hordeum vulgare</em> L.). Molecular Breeding, 34(3), 1373–1387. 10.1007/s11032-014-0122-3
</cite> [<a href="https://doi.org/10.1007/s11032-014-0122-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Detection%20and%20validation%20of%20novel%20QTL%20for%20shoot%20and%20root%20traits%20in%20barley%20(Hordeum%20vulgare%20L.)&amp;author=M.%20Arifuzzaman&amp;author=M.%20A.%20Sayed&amp;author=S.%20Muzammil&amp;author=K.%20Pillen&amp;author=H.%20Schumann&amp;volume=34&amp;issue=3&amp;publication_year=2014&amp;pages=1373-1387&amp;doi=10.1007/s11032-014-0122-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0064"><cite>

Baumer, M.
, &amp; 
Cais, R.
 (2000). 
<em>Abstammung der Gerstensorten</em> [Pedigrees of barley cultivars]. Bayerische Landesanstalt für Bodenkultur und Pflanzenbau. <a href="http://www.stmelf.bayern.de/lbp/forsch/pz/gerstenstamm.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.stmelf.bayern.de/lbp/forsch/pz/gerstenstamm.pdf</a>
</cite></li>
<li id="tpg270088-bib-0004">
<cite>

Browning, B. L.
, 
Zhou, Y.
, &amp; 
Browning, S. R.
 (2018). A one‐penny imputed genome from next‐generation reference panels. American Journal of Human Genetics, 103, 338–348. 10.1016/j.ajhg.2018.07.015

</cite> [<a href="https://doi.org/10.1016/j.ajhg.2018.07.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6128308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30100085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=A%20one%E2%80%90penny%20imputed%20genome%20from%20next%E2%80%90generation%20reference%20panels&amp;author=B.%20L.%20Browning&amp;author=Y.%20Zhou&amp;author=S.%20R.%20Browning&amp;volume=103&amp;publication_year=2018&amp;pages=338-348&amp;pmid=30100085&amp;doi=10.1016/j.ajhg.2018.07.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0005">
<cite>

Brunner, S. M.
, 
Dinglasan, E.
, 
Baraibar, S.
, 
Alahmad, S.
, 
Katsikis, C.
, 
Van Der Meer, S.
, 
Godoy, J.
, 
Moody, D.
, 
Smith, M.
, 
Hickey, L.
, &amp; 
Robinson, H.
 (2024). Characterizing stay‐green in barley across diverse environments: Unveiling novel haplotypes. Theoretical and Applied Genetics, 137, Article 120. 10.1007/s00122-024-04612-1

</cite> [<a href="https://doi.org/10.1007/s00122-024-04612-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11074220/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38709310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Characterizing%20stay%E2%80%90green%20in%20barley%20across%20diverse%20environments:%20Unveiling%20novel%20haplotypes&amp;author=S.%20M.%20Brunner&amp;author=E.%20Dinglasan&amp;author=S.%20Baraibar&amp;author=S.%20Alahmad&amp;author=C.%20Katsikis&amp;volume=137&amp;publication_year=2024&amp;pages=120&amp;pmid=38709310&amp;doi=10.1007/s00122-024-04612-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0006">
<cite>

Butler, D.
, 
Cullis, B.
, 
Gilmour, A.
, 
Gogel, B.
, &amp; 
Thompson, R.
 (2017). ASReml‐R reference manual version 4. VSN International Ltd.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=ASReml%E2%80%90R%20reference%20manual%20version%204&amp;author=D.%20Butler&amp;author=B.%20Cullis&amp;author=A.%20Gilmour&amp;author=B.%20Gogel&amp;author=R.%20Thompson&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0054">
<cite>

Chen, G.
, 
Krugman, T.
, 
Fahima, T.
, 
Chen, K.
, 
Hu, Y.
, 
Röder, M.
, 
Nevo, E.
, &amp; 
Korol, A.
 (2009). Chromosomal regions controlling seedling drought resistance in Israeli wild barley, <em>Hordeum spontaneum</em> C. Koch. Genetic Resources and Crop Evolution, 57(1), 85–99. 10.1007/s10722-009-9453-z
</cite> [<a href="https://doi.org/10.1007/s10722-009-9453-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetic%20Resources%20and%20Crop%20Evolution&amp;title=Chromosomal%20regions%20controlling%20seedling%20drought%20resistance%20in%20Israeli%20wild%20barley,%20Hordeum%20spontaneum%20C.%20Koch&amp;author=G.%20Chen&amp;author=T.%20Krugman&amp;author=T.%20Fahima&amp;author=K.%20Chen&amp;author=Y.%20Hu&amp;volume=57&amp;issue=1&amp;publication_year=2009&amp;pages=85-99&amp;doi=10.1007/s10722-009-9453-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0055">
<cite>

Cheng, B.
, 
Gao, X.
, 
Luo, Y.
, 
Ding, Y.
, 
Chen, T.
, 
Cao, N.
, 
Xu, J.
, 
Xin, Z.
, &amp; 
Zhang, L.
 (2023). Utilization of wheat 55K SNP array for QTL mapping of plant height and flag leaf in a RIL population. Cereal Research Communications, 52(4), 1273–1286. 10.1007/s42976-023-00475-2
</cite> [<a href="https://doi.org/10.1007/s42976-023-00475-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cereal%20Research%20Communications&amp;title=Utilization%20of%20wheat%2055K%20SNP%20array%20for%20QTL%20mapping%20of%20plant%20height%20and%20flag%20leaf%20in%20a%20RIL%20population&amp;author=B.%20Cheng&amp;author=X.%20Gao&amp;author=Y.%20Luo&amp;author=Y.%20Ding&amp;author=T.%20Chen&amp;volume=52&amp;issue=4&amp;publication_year=2023&amp;pages=1273-1286&amp;doi=10.1007/s42976-023-00475-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0056">
<cite>

Chloupek, O.
, 
Forster, B. P.
, &amp; 
Thomas, W. T. B.
 (2006). The effect of semi‐dwarf genes on root system size in field‐grown barley. Theoretical and Applied Genetics, 112(5), 779–786. 10.1007/s00122-005-0147-4

</cite> [<a href="https://doi.org/10.1007/s00122-005-0147-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16425022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=The%20effect%20of%20semi%E2%80%90dwarf%20genes%20on%20root%20system%20size%20in%20field%E2%80%90grown%20barley&amp;author=O.%20Chloupek&amp;author=B.%20P.%20Forster&amp;author=W.%20T.%20B.%20Thomas&amp;volume=112&amp;issue=5&amp;publication_year=2006&amp;pages=779-786&amp;pmid=16425022&amp;doi=10.1007/s00122-005-0147-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0007">
<cite>

Christopher, J.
, 
Christopher, M.
, 
Jennings, R.
, 
Jones, S.
, 
Fletcher, S.
, 
Borrell, A.
, 
Manschadi, A. M.
, 
Jordan, D.
, 
Mace, E.
, &amp; 
Hammer, G.
 (2013). QTL for root angle and number in a population developed from bread wheats (<em>Triticum aestivum</em>) with contrasting adaptation to water‐limited environments. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 126, 1563–1574. 10.1007/s00122-013-2074-0

</cite> [<a href="https://doi.org/10.1007/s00122-013-2074-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23525632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=QTL%20for%20root%20angle%20and%20number%20in%20a%20population%20developed%20from%20bread%20wheats%20(Triticum%20aestivum)%20with%20contrasting%20adaptation%20to%20water%E2%80%90limited%20environments&amp;author=J.%20Christopher&amp;author=M.%20Christopher&amp;author=R.%20Jennings&amp;author=S.%20Jones&amp;author=S.%20Fletcher&amp;volume=126&amp;publication_year=2013&amp;pages=1563-1574&amp;pmid=23525632&amp;doi=10.1007/s00122-013-2074-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0008">
<cite>

Comadran, J.
, 
Thomas, W. T. B.
, 
Van Eeuwijk, F. Á.
, 
Ceccarelli, S.
, 
Grando, S.
, 
Stanca, A. M.
, 
Pecchioni, N.
, 
Akar, T.
, 
Al‐Yassin, A.
, 
Benbelkacem, A.
, 
Ouabbou, H.
, 
Bort, J.
, 
Romagosa, I.
, 
Hackett, C. A.
, &amp; 
Russell, J. R.
 (2009). Patterns of genetic diversity and linkage disequilibrium in a highly structured Hordeum vulgare association‐mapping population for the Mediterranean basin. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 119, 175–187. 10.1007/s00122-009-1027-0

</cite> [<a href="https://doi.org/10.1007/s00122-009-1027-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19415228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Patterns%20of%20genetic%20diversity%20and%20linkage%20disequilibrium%20in%20a%20highly%20structured%20Hordeum%20vulgare%20association%E2%80%90mapping%20population%20for%20the%20Mediterranean%20basin&amp;author=J.%20Comadran&amp;author=W.%20T.%20B.%20Thomas&amp;author=F.%20%C3%81.%20Van%20Eeuwijk&amp;author=S.%20Ceccarelli&amp;author=S.%20Grando&amp;volume=119&amp;publication_year=2009&amp;pages=175-187&amp;pmid=19415228&amp;doi=10.1007/s00122-009-1027-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0009">
<cite>

Cronin, J. K.
, 
Bundock, P. C.
, 
Henry, R. J.
, &amp; 
Nevo, E.
 (2007). Adaptive climatic molecular evolution in wild barley at the <em>Isa</em>defense locus. Proceedings of the National Academy of Sciences of the USA, 104, 2773–2778. 10.1073/pnas.0611226104

</cite> [<a href="https://doi.org/10.1073/pnas.0611226104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1815257/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17301230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20USA&amp;title=Adaptive%20climatic%20molecular%20evolution%20in%20wild%20barley%20at%20the%20Isadefense%20locus&amp;author=J.%20K.%20Cronin&amp;author=P.%20C.%20Bundock&amp;author=R.%20J.%20Henry&amp;author=E.%20Nevo&amp;volume=104&amp;publication_year=2007&amp;pages=2773-2778&amp;pmid=17301230&amp;doi=10.1073/pnas.0611226104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0010">
<cite>

Cullis, B. R.
, 
Smith, A. B.
, &amp; 
Coombes, N. E.
 (2006). On the design of early generation variety trials with correlated data. Journal of Agricultural, Biological, and Environmental Statistics, 11, 381–393. 10.1198/108571106X154443
</cite> [<a href="https://doi.org/10.1198/108571106X154443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Agricultural,%20Biological,%20and%20Environmental%20Statistics&amp;title=On%20the%20design%20of%20early%20generation%20variety%20trials%20with%20correlated%20data&amp;author=B.%20R.%20Cullis&amp;author=A.%20B.%20Smith&amp;author=N.%20E.%20Coombes&amp;volume=11&amp;publication_year=2006&amp;pages=381-393&amp;doi=10.1198/108571106X154443&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0011">
<cite>

Danecek, P.
, 
Auton, A.
, 
Abecasis, G.
, 
Albers, C. A.
, 
Banks, E.
, 
Depristo, M. A.
, 
Handsaker, R. E.
, 
Lunter, G.
, 
Marth, G. T.
, 
Sherry, S. T.
, 
Mcvean, G.
, &amp; 
Durbin, R.
 (2011). The variant call format and VCFtools. Bioinformatics, 27, 2156–2158. 10.1093/bioinformatics/btr330

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3137218/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21653522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The%20variant%20call%20format%20and%20VCFtools&amp;author=P.%20Danecek&amp;author=A.%20Auton&amp;author=G.%20Abecasis&amp;author=C.%20A.%20Albers&amp;author=E.%20Banks&amp;volume=27&amp;publication_year=2011&amp;pages=2156-2158&amp;pmid=21653522&amp;doi=10.1093/bioinformatics/btr330&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0057">
<cite>

Diab, A. A.
, 
Teulat‐Merah, B.
, 
This, D.
, 
Ozturk, N. Z.
, 
Benscher, D.
, &amp; 
Sorrells, M. E.
 (2004). Identification of drought‐inducible genes and differentially expressed sequence tags in barley. Theoretical and Applied Genetics, 109(7), 1417–1425. 10.1007/s00122-004-1755-0

</cite> [<a href="https://doi.org/10.1007/s00122-004-1755-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15517148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Identification%20of%20drought%E2%80%90inducible%20genes%20and%20differentially%20expressed%20sequence%20tags%20in%20barley&amp;author=A.%20A.%20Diab&amp;author=B.%20Teulat%E2%80%90Merah&amp;author=D.%20This&amp;author=N.%20Z.%20Ozturk&amp;author=D.%20Benscher&amp;volume=109&amp;issue=7&amp;publication_year=2004&amp;pages=1417-1425&amp;pmid=15517148&amp;doi=10.1007/s00122-004-1755-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0012">
<cite>
FAS
. (2023). World agricultural production. United States Department of Agriculture.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=World%20agricultural%20production&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0013">
<cite>

Feng, X.
, 
Peng, C.
, 
Chen, Y.
, 
Liu, X.
, 
Feng, X.
, &amp; 
He, Y.
 (2017). Discrimination of CRISPR/Cas9‐induced mutants of rice seeds using near‐infrared hyperspectral imaging. Scientific Reports, 7, Article 15934. 10.1038/s41598-017-16254-z

</cite> [<a href="https://doi.org/10.1038/s41598-017-16254-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5698449/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29162881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Discrimination%20of%20CRISPR/Cas9%E2%80%90induced%20mutants%20of%20rice%20seeds%20using%20near%E2%80%90infrared%20hyperspectral%20imaging&amp;author=X.%20Feng&amp;author=C.%20Peng&amp;author=Y.%20Chen&amp;author=X.%20Liu&amp;author=X.%20Feng&amp;volume=7&amp;publication_year=2017&amp;pages=15934&amp;pmid=29162881&amp;doi=10.1038/s41598-017-16254-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0059">
<cite>

Fusi, R.
, 
Rosignoli, S.
, 
Lou, H.
, 
Sangiorgi, G.
, 
Bovina, R.
, 
Pattem, J. K.
, 
Borkar, A. N.
, 
Lombardi, M.
, 
Forestan, C.
, 
Milner, S. G.
, 
Davis, J. L.
, 
Lale, A.
, 
Kirschner, G. K.
, 
Swarup, R.
, 
Tassinari, A.
, 
Pandey, B. K.
, 
York, L. M.
, 
Atkinson, B. S.
, 
Sturrock, C. J.
, … 
Salvi, S.
 (2022). Root angle is controlled byEGT1in cereal crops employing an antigravitropic mechanism. Proceedings of the National Academy of Sciences, 119(31). 10.1073/pnas.2201350119
</cite> [<a href="https://doi.org/10.1073/pnas.2201350119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9351459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35881796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Root%20angle%20is%20controlled%20byEGT1in%20cereal%20crops%20employing%20an%20antigravitropic%20mechanism&amp;author=R.%20Fusi&amp;author=S.%20Rosignoli&amp;author=H.%20Lou&amp;author=G.%20Sangiorgi&amp;author=R.%20Bovina&amp;volume=119&amp;issue=31&amp;publication_year=2022&amp;pmid=35881796&amp;doi=10.1073/pnas.2201350119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0014">
<cite>

Gürel, F.
, 
Öztürk, Z. N.
, 
Uçarlı, C.
, &amp; 
Rosellini, D.
 (2016). Barley genes as tools to confer abiotic stress tolerance in crops. Frontiers in Plant Science, 7, 1137.
</cite> [<a href="https://doi.org/10.3389/fpls.2016.01137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4971604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27536305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Barley%20genes%20as%20tools%20to%20confer%20abiotic%20stress%20tolerance%20in%20crops&amp;author=F.%20G%C3%BCrel&amp;author=Z.%20N.%20%C3%96zt%C3%BCrk&amp;author=C.%20U%C3%A7arl%C4%B1&amp;author=D.%20Rosellini&amp;volume=7&amp;publication_year=2016&amp;pages=1137&amp;pmid=27536305&amp;doi=10.3389/fpls.2016.01137&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0015">
<cite>

Hayes, B. J.
, 
Mahony, T. J.
, 
Villiers, K.
, 
Warburton, C.
, 
Kemper, K. E.
, 
Dinglasan, E.
, 
Robinson, H.
, 
Powell, O.
, 
Voss‐Fels, K.
, 
Godwin, I. D.
, &amp; 
Hickey, L. T.
 (2024). Potential approaches to create ultimate genotypes in crops and livestock. Nature Genetics, 56, 2310–2317. 10.1038/s41588-024-01942-0

</cite> [<a href="https://doi.org/10.1038/s41588-024-01942-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39402155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Potential%20approaches%20to%20create%20ultimate%20genotypes%20in%20crops%20and%20livestock&amp;author=B.%20J.%20Hayes&amp;author=T.%20J.%20Mahony&amp;author=K.%20Villiers&amp;author=C.%20Warburton&amp;author=K.%20E.%20Kemper&amp;volume=56&amp;publication_year=2024&amp;pages=2310-2317&amp;pmid=39402155&amp;doi=10.1038/s41588-024-01942-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0016">
<cite>

Hendrick, R. L.
, &amp; 
Pregitzer, K. S.
 (1996). Applications of minirhizotrons to understand root function in forests and other natural ecosystems. Plant and Soil, 185, 293–304. 10.1007/BF02257535
</cite> [<a href="https://doi.org/10.1007/BF02257535" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20and%20Soil&amp;title=Applications%20of%20minirhizotrons%20to%20understand%20root%20function%20in%20forests%20and%20other%20natural%20ecosystems&amp;author=R.%20L.%20Hendrick&amp;author=K.%20S.%20Pregitzer&amp;volume=185&amp;publication_year=1996&amp;pages=293-304&amp;doi=10.1007/BF02257535&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0017">
<cite>

Hickey, L. T.
, 
Lawson, W.
, 
Arief, V. N.
, 
Fox, G.
, 
Franckowiak, J.
, &amp; 
Dieters, M. J.
 (2012). Grain dormancy QTL identified in a doubled haploid barley population derived from two non‐dormant parents. Euphytica, 188, 113–122. 10.1007/s10681-011-0577-9
</cite> [<a href="https://doi.org/10.1007/s10681-011-0577-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Grain%20dormancy%20QTL%20identified%20in%20a%20doubled%20haploid%20barley%20population%20derived%20from%20two%20non%E2%80%90dormant%20parents&amp;author=L.%20T.%20Hickey&amp;author=W.%20Lawson&amp;author=V.%20N.%20Arief&amp;author=G.%20Fox&amp;author=J.%20Franckowiak&amp;volume=188&amp;publication_year=2012&amp;pages=113-122&amp;doi=10.1007/s10681-011-0577-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0018">
<cite>

Jia, Z.
, 
Liu, Y.
, 
Gruber, B. D.
, 
Neumann, K.
, 
Kilian, B.
, 
Graner, A.
, &amp; 
Von Wirén, N.
 (2019). Genetic dissection of root system architectural traits in spring barley. Frontiers in Plant Science, 10, 400. 10.3389/fpls.2019.00400

</cite> [<a href="https://doi.org/10.3389/fpls.2019.00400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6454135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31001309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genetic%20dissection%20of%20root%20system%20architectural%20traits%20in%20spring%20barley&amp;author=Z.%20Jia&amp;author=Y.%20Liu&amp;author=B.%20D.%20Gruber&amp;author=K.%20Neumann&amp;author=B.%20Kilian&amp;volume=10&amp;publication_year=2019&amp;pages=400&amp;pmid=31001309&amp;doi=10.3389/fpls.2019.00400&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0019">
<cite>

Kang, Y.
, 
Rambla, C.
, 
Haeften, S.
, &amp; 
Fu, B.
, 
Akinlade, O.
, 
Potgieter, A.
, 
Borrell, A.
, 
Mace, E.
, 
Jordan, D.
, 
Alahmad, S.
, 
Hickey, L.
 (2024). Seminal root angle is associated with root system architecture in durum wheat. <em>bioRxiv</em>. 10.1101/2024.02.27.582216
</cite> [<a href="https://doi.org/10.1101/2024.02.27.582216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270088-bib-0020">
<cite>

Keeble‐Gagnère, G.
, 
Pasam, R.
, 
Forrest, K. L.
, 
Wong, D.
, 
Robinson, H.
, 
Godoy, J.
, 
Rattey, A.
, 
Moody, D.
, 
Mullan, D.
, 
Walmsley, T.
, 
Daetwyler, H. D.
, 
Tibbits, J.
, &amp; 
Hayden, M. J.
 (2021). Novel design of imputation‐enabled SNP arrays for breeding and research applications supporting multi‐species hybridization. Frontiers in Plant Science, 12, 756877. 10.3389/fpls.2021.756877

</cite> [<a href="https://doi.org/10.3389/fpls.2021.756877" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728019/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35003156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Novel%20design%20of%20imputation%E2%80%90enabled%20SNP%20arrays%20for%20breeding%20and%20research%20applications%20supporting%20multi%E2%80%90species%20hybridization&amp;author=G.%20Keeble%E2%80%90Gagn%C3%A8re&amp;author=R.%20Pasam&amp;author=K.%20L.%20Forrest&amp;author=D.%20Wong&amp;author=H.%20Robinson&amp;volume=12&amp;publication_year=2021&amp;pages=756877&amp;pmid=35003156&amp;doi=10.3389/fpls.2021.756877&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0021">
<cite>

Kemper, K. E.
, 
Bowman, P. J.
, 
Pryce, J. E.
, 
Hayes, B. J.
, &amp; 
Goddard, M. E.
 (2012). Long‐term selection strategies for complex traits using high‐density genetic markers. Journal of Dairy Science, 95, 4646–4656. 10.3168/jds.2011-5289

</cite> [<a href="https://doi.org/10.3168/jds.2011-5289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22818479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Dairy%20Science&amp;title=Long%E2%80%90term%20selection%20strategies%20for%20complex%20traits%20using%20high%E2%80%90density%20genetic%20markers&amp;author=K.%20E.%20Kemper&amp;author=P.%20J.%20Bowman&amp;author=J.%20E.%20Pryce&amp;author=B.%20J.%20Hayes&amp;author=M.%20E.%20Goddard&amp;volume=95&amp;publication_year=2012&amp;pages=4646-4656&amp;pmid=22818479&amp;doi=10.3168/jds.2011-5289&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0022">
<cite>

Kenward, M. G.
, &amp; 
Roger, J. H.
 (1997). Small sample inference for fixed effects from restricted maximum likelihood. Biometrics, 53, 983–997. 10.2307/2533558

</cite> [<a href="https://doi.org/10.2307/2533558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9333350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Small%20sample%20inference%20for%20fixed%20effects%20from%20restricted%20maximum%20likelihood&amp;author=M.%20G.%20Kenward&amp;author=J.%20H.%20Roger&amp;volume=53&amp;publication_year=1997&amp;pages=983-997&amp;pmid=9333350&amp;doi=10.2307/2533558&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0052">
<cite>

Khodaeiaminjan, M.
, 
Knoch, D.
, 
Ndella Thiaw, M. R.
, 
Marchetti, C. F.
, 
Kořínková, N.
, 
Techer, A.
, 
Nguyen, T. D.
, 
Chu, J.
, 
Bertholomey, V.
, 
Doridant, I.
, 
Gantet, P.
, 
Graner, A.
, 
Neumann, K.
, &amp; 
Bergougnoux, V.
 (2023). Genome‐wide association study in two‐row spring barley landraces identifies QTL associated with plantlets root system architecture traits in well‐watered and osmotic stress conditions. Frontiers in Plant Science, 14, 1125672. 10.3389/fpls.2023.1125672

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1125672" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10106628/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37077626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%E2%80%90wide%20association%20study%20in%20two%E2%80%90row%20spring%20barley%20landraces%20identifies%20QTL%20associated%20with%20plantlets%20root%20system%20architecture%20traits%20in%20well%E2%80%90watered%20and%20osmotic%20stress%20conditions&amp;author=M.%20Khodaeiaminjan&amp;author=D.%20Knoch&amp;author=M.%20R.%20Ndella%20Thiaw&amp;author=C.%20F.%20Marchetti&amp;author=N.%20Ko%C5%99%C3%ADnkov%C3%A1&amp;volume=14&amp;publication_year=2023&amp;pages=1125672&amp;pmid=37077626&amp;doi=10.3389/fpls.2023.1125672&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0023">
<cite>

Kirschner, G. K.
, 
Rosignoli, S.
, 
Guo, L.
, 
Vardanega, I.
, 
Imani, J.
, 
Altmüller, J.
, 
Milner, S. G.
, 
Balzano, R.
, 
Nagel, K. A.
, 
Pflugfelder, D.
, 
Forestan, C.
, 
Bovina, R.
, 
Koller, R.
, 
Stöcker, T. G.
, 
Mascher, M.
, 
Simmonds, J.
, 
Uauy, C.
, 
Schoof, H.
, 
Tuberosa, R.
, … 
Hochholdinger, F.
 (2021). 
<em>ENHANCED GRAVITROPISM 2</em> encodes a STERILE ALPHA MOTIF‐containing protein that controls root growth angle in barley and wheat. Proceedings of the National Academy of Sciences of the USA, 118(35), e2101526118. 10.1073/pnas.2101526118

</cite> [<a href="https://doi.org/10.1073/pnas.2101526118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8536364/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34446550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20USA&amp;title=ENHANCED%20GRAVITROPISM%202%20encodes%20a%20STERILE%20ALPHA%20MOTIF%E2%80%90containing%20protein%20that%20controls%20root%20growth%20angle%20in%20barley%20and%20wheat&amp;author=G.%20K.%20Kirschner&amp;author=S.%20Rosignoli&amp;author=L.%20Guo&amp;author=I.%20Vardanega&amp;author=J.%20Imani&amp;volume=118&amp;issue=35&amp;publication_year=2021&amp;pages=e2101526118&amp;pmid=34446550&amp;doi=10.1073/pnas.2101526118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0024">
<cite>

Kitomi, Y.
, 
Hanzawa, E.
, 
Kuya, N.
, 
Inoue, H.
, 
Hara, N.
, 
Kawai, S.
, 
Kanno, N.
, 
Endo, M.
, 
Sugimoto, K.
, 
Yamazaki, T.
, 
Sakamoto, S.
, 
Sentoku, N.
, 
Wu, J.
, 
Kanno, H.
, 
Mitsuda, N.
, 
Toriyama, K.
, 
Sato, T.
, &amp; 
Uga, Y.
 (2020). Root angle modifications by the <em>DRO1</em> homolog improve rice yields in saline paddy fields. Proceedings of the National Academy of Sciences of the USA, 117, 21242–21250. 10.1073/pnas.2005911117

</cite> [<a href="https://doi.org/10.1073/pnas.2005911117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7474696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32817523/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20USA&amp;title=Root%20angle%20modifications%20by%20the%20DRO1%20homolog%20improve%20rice%20yields%20in%20saline%20paddy%20fields&amp;author=Y.%20Kitomi&amp;author=E.%20Hanzawa&amp;author=N.%20Kuya&amp;author=H.%20Inoue&amp;author=N.%20Hara&amp;volume=117&amp;publication_year=2020&amp;pages=21242-21250&amp;pmid=32817523&amp;doi=10.1073/pnas.2005911117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0025">
<cite>

Koevoets, I. T.
, 
Venema, J. H.
, 
Elzenga, J. T. M.
, &amp; 
Testerink, C.
 (2016). Roots withstanding their environment: Exploiting root system architecture responses to abiotic stress to improve crop tolerance. Frontiers in Plant Science, 7, 1335. 10.3389/fpls.2016.01335

</cite> [<a href="https://doi.org/10.3389/fpls.2016.01335" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5005332/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27630659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Roots%20withstanding%20their%20environment:%20Exploiting%20root%20system%20architecture%20responses%20to%20abiotic%20stress%20to%20improve%20crop%20tolerance&amp;author=I.%20T.%20Koevoets&amp;author=J.%20H.%20Venema&amp;author=J.%20T.%20M.%20Elzenga&amp;author=C.%20Testerink&amp;volume=7&amp;publication_year=2016&amp;pages=1335&amp;pmid=27630659&amp;doi=10.3389/fpls.2016.01335&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0026">
<cite>

Kong, X.
, 
Zhang, M.
, 
De Smet, I.
, &amp; 
Ding, Z.
 (2014). Designer crops: Optimal root system architecture for nutrient acquisition. Trends in Biotechnology, 32, 597–598. 10.1016/j.tibtech.2014.09.008

</cite> [<a href="https://doi.org/10.1016/j.tibtech.2014.09.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25450041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Biotechnology&amp;title=Designer%20crops:%20Optimal%20root%20system%20architecture%20for%20nutrient%20acquisition&amp;author=X.%20Kong&amp;author=M.%20Zhang&amp;author=I.%20De%20Smet&amp;author=Z.%20Ding&amp;volume=32&amp;publication_year=2014&amp;pages=597-598&amp;pmid=25450041&amp;doi=10.1016/j.tibtech.2014.09.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0027">
<cite>

Lynch, J. P.
 (2013). Steep, cheap and deep: An ideotype to optimize water and N acquisition by maize root systems. Annals of Botany, 112, 347–357. 10.1093/aob/mcs293

</cite> [<a href="https://doi.org/10.1093/aob/mcs293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3698384/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23328767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Botany&amp;title=Steep,%20cheap%20and%20deep:%20An%20ideotype%20to%20optimize%20water%20and%20N%20acquisition%20by%20maize%20root%20systems&amp;author=J.%20P.%20Lynch&amp;volume=112&amp;publication_year=2013&amp;pages=347-357&amp;pmid=23328767&amp;doi=10.1093/aob/mcs293&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0028">
<cite>

Lynch, J. P.
 (2019). Root phenotypes for improved nutrient capture: An underexploited opportunity for global agriculture. New Phytologist, 223, 548–564. 10.1111/nph.15738

</cite> [<a href="https://doi.org/10.1111/nph.15738" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30746704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Root%20phenotypes%20for%20improved%20nutrient%20capture:%20An%20underexploited%20opportunity%20for%20global%20agriculture&amp;author=J.%20P.%20Lynch&amp;volume=223&amp;publication_year=2019&amp;pages=548-564&amp;pmid=30746704&amp;doi=10.1111/nph.15738&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0029">
<cite>

Manschadi, A. M.
, 
Christopher, J.
, 
Devoil, P.
, &amp; 
Hammer, G. L.
 (2006). The role of root architectural traits in adaptation of wheat to water‐limited environments. Functional Plant Biology, 33, 823–837. 10.1071/FP06055

</cite> [<a href="https://doi.org/10.1071/FP06055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32689293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20Plant%20Biology&amp;title=The%20role%20of%20root%20architectural%20traits%20in%20adaptation%20of%20wheat%20to%20water%E2%80%90limited%20environments&amp;author=A.%20M.%20Manschadi&amp;author=J.%20Christopher&amp;author=P.%20Devoil&amp;author=G.%20L.%20Hammer&amp;volume=33&amp;publication_year=2006&amp;pages=823-837&amp;pmid=32689293&amp;doi=10.1071/FP06055&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0030">
<cite>

Mazumder, R.
, 
Hastie, T.
, &amp; 
Tibshirani, R.
 (2010). Spectral regularization algorithms for learning large incomplete matrices. Journal of Machine Learning Research, 11, 2287–2322.
</cite> [<a href="/articles/PMC3087301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21552465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Machine%20Learning%20Research&amp;title=Spectral%20regularization%20algorithms%20for%20learning%20large%20incomplete%20matrices&amp;author=R.%20Mazumder&amp;author=T.%20Hastie&amp;author=R.%20Tibshirani&amp;volume=11&amp;publication_year=2010&amp;pages=2287-2322&amp;pmid=21552465&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0031">
<cite>

Meuwissen, T. H. E.
, 
Hayes, B. J.
, &amp; 
Goddard, M. E.
 (2001). Prediction of total genetic value using genome‐wide dense marker maps. Genetics, 157, 1819–1829. 10.1093/genetics/157.4.1819

</cite> [<a href="https://doi.org/10.1093/genetics/157.4.1819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11290733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Prediction%20of%20total%20genetic%20value%20using%20genome%E2%80%90wide%20dense%20marker%20maps&amp;author=T.%20H.%20E.%20Meuwissen&amp;author=B.%20J.%20Hayes&amp;author=M.%20E.%20Goddard&amp;volume=157&amp;publication_year=2001&amp;pages=1819-1829&amp;pmid=11290733&amp;doi=10.1093/genetics/157.4.1819&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0032">
<cite>

Murtagh, F.
, &amp; 
Legendre, P.
 (2014). Ward's hierarchical agglomerative clustering method: Which algorithms implement Ward's criterion?
Journal of Classification, 31, 274–295. 10.1007/s00357-014-9161-z
</cite> [<a href="https://doi.org/10.1007/s00357-014-9161-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Classification&amp;title=Ward's%20hierarchical%20agglomerative%20clustering%20method:%20Which%20algorithms%20implement%20Ward's%20criterion?&amp;author=F.%20Murtagh&amp;author=P.%20Legendre&amp;volume=31&amp;publication_year=2014&amp;pages=274-295&amp;doi=10.1007/s00357-014-9161-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0033">
<cite>

Ober, E. S.
, 
Alahmad, S.
, 
Cockram, J.
, 
Forestan, C.
, 
Hickey, L. T.
, 
Kant, J.
, 
Maccaferri, M.
, 
Marr, E.
, 
Milner, M.
, 
Pinto, F.
, 
Rambla, C.
, 
Reynolds, M.
, 
Salvi, S.
, 
Sciara, G.
, 
Snowdon, R. J.
, 
Thomelin, P.
, 
Tuberosa, R.
, 
Uauy, C.
, 
Voss‐Fels, K. P.
, … 
Watt, M.
 (2021). Wheat root systems as a breeding target for climate resilience. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 134, 1645–1662. 10.1007/s00122-021-03819-w

</cite> [<a href="https://doi.org/10.1007/s00122-021-03819-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8206059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33900415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Wheat%20root%20systems%20as%20a%20breeding%20target%20for%20climate%20resilience&amp;author=E.%20S.%20Ober&amp;author=S.%20Alahmad&amp;author=J.%20Cockram&amp;author=C.%20Forestan&amp;author=L.%20T.%20Hickey&amp;volume=134&amp;publication_year=2021&amp;pages=1645-1662&amp;pmid=33900415&amp;doi=10.1007/s00122-021-03819-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0053">
<cite>
R Core Team
. (2024). R: A language and environment for statistical computing. R Foundation for Statistical Computing. <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R-project.org/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=R:%20A%20language%20and%20environment%20for%20statistical%20computing&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0034">
<cite>

Rich, S. M.
, &amp; 
Watt, M.
 (2013). Soil conditions and cereal root system architecture: Review and considerations for linking Darwin and Weaver. Journal of Experimental Botany, 64, 1193–1208. 10.1093/jxb/ert043

</cite> [<a href="https://doi.org/10.1093/jxb/ert043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23505309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Soil%20conditions%20and%20cereal%20root%20system%20architecture:%20Review%20and%20considerations%20for%20linking%20Darwin%20and%20Weaver&amp;author=S.%20M.%20Rich&amp;author=M.%20Watt&amp;volume=64&amp;publication_year=2013&amp;pages=1193-1208&amp;pmid=23505309&amp;doi=10.1093/jxb/ert043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0035">
<cite>

Richard, C.
, 
Hickey, L. T.
, 
Fletcher, S.
, 
Jennings, R.
, 
Chenu, K.
, &amp; 
Christopher, J. T.
 (2015). High‐throughput phenotyping of seminal root traits in wheat. Plant Methods, 11, Article 13. 10.1186/s13007-015-0055-9

</cite> [<a href="https://doi.org/10.1186/s13007-015-0055-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4351910/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25750658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Methods&amp;title=High%E2%80%90throughput%20phenotyping%20of%20seminal%20root%20traits%20in%20wheat&amp;author=C.%20Richard&amp;author=L.%20T.%20Hickey&amp;author=S.%20Fletcher&amp;author=R.%20Jennings&amp;author=K.%20Chenu&amp;volume=11&amp;publication_year=2015&amp;pages=13&amp;pmid=25750658&amp;doi=10.1186/s13007-015-0055-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0036">
<cite>

Robinson, H.
, 
Hickey, L.
, 
Richard, C.
, 
Mace, E.
, 
Kelly, A.
, 
Borrell, A.
, 
Franckowiak, J.
, &amp; 
Fox, G.
 (2016). Genomic regions influencing seminal root traits in barley. The Plant Genome, 9, plantgenome2015.03.0012. 10.3835/plantgenome2015.03.0012
</cite> [<a href="https://doi.org/10.3835/plantgenome2015.03.0012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27898766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20regions%20influencing%20seminal%20root%20traits%20in%20barley&amp;author=H.%20Robinson&amp;author=L.%20Hickey&amp;author=C.%20Richard&amp;author=E.%20Mace&amp;author=A.%20Kelly&amp;volume=9&amp;publication_year=2016&amp;pmid=27898766&amp;doi=10.3835/plantgenome2015.03.0012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0037">
<cite>

Robinson, H.
, 
Kelly, A.
, 
Fox, G.
, 
Franckowiak, J.
, 
Borrell, A.
, &amp; 
Hickey, L.
 (2018). Root architectural traits and yield: Exploring the relationship in barley breeding trials. Euphytica, 214, Article 151. 10.1007/s10681-018-2219-y
</cite> [<a href="https://doi.org/10.1007/s10681-018-2219-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Root%20architectural%20traits%20and%20yield:%20Exploring%20the%20relationship%20in%20barley%20breeding%20trials&amp;author=H.%20Robinson&amp;author=A.%20Kelly&amp;author=G.%20Fox&amp;author=J.%20Franckowiak&amp;author=A.%20Borrell&amp;volume=214&amp;publication_year=2018&amp;pages=151&amp;doi=10.1007/s10681-018-2219-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0038">
<cite>

Rogers, J.
 (1972). Measures of genetic similarity and genetic distance. In Studies in genetics VII (pp. 141–153, Publication 7213). University of Texas.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Studies%20in%20genetics%20VII&amp;author=J.%20Rogers&amp;publication_year=1972&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0039">
<cite>

Schneider, C. A.
, 
Rasband, W. S.
, &amp; 
Eliceiri, K. W.
 (2012). NIH Image to ImageJ: 25 years of image analysis. Nature Methods, 9, 671–675. 10.1038/nmeth.2089

</cite> [<a href="https://doi.org/10.1038/nmeth.2089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5554542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=NIH%20Image%20to%20ImageJ:%2025%20years%20of%20image%20analysis&amp;author=C.%20A.%20Schneider&amp;author=W.%20S.%20Rasband&amp;author=K.%20W.%20Eliceiri&amp;volume=9&amp;publication_year=2012&amp;pages=671-675&amp;pmid=22930834&amp;doi=10.1038/nmeth.2089&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0040">
<cite>

Seko, H.
 (1987). History of barley breeding in Japan. In 
Barley genetics V: Proceedings of the 5th barley genetics symposium
 (pp 915–922). Maruzen Co Ltd. Okayama Branch.</cite> [<a href="https://scholar.google.com/scholar_lookup?Seko,%20H.%20(1987).%20History%20of%20barley%20breeding%20in%20Japan.%20In%20Barley%20genetics%20V:%20Proceedings%20of%20the%205th%20barley%20genetics%20symposium%20(pp%20915%E2%80%93922).%20Maruzen%20Co%20Ltd.%20Okayama%20Branch." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0058">
<cite>

Siahsar, B. A.
, &amp; 
Narouei, M.
 (2010). Mapping QTLs of physiological traits associated with salt tolerance in ‘Steptoe’ × ‘Morex’ doubled haploid lines of barley at the seedling stage. Journal of Food, Agriculture &amp; Environment, 8, 751–759.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Food,%20Agriculture%20&amp;%20Environment&amp;title=Mapping%20QTLs%20of%20physiological%20traits%20associated%20with%20salt%20tolerance%20in%20%E2%80%98Steptoe%E2%80%99%20%C3%97%20%E2%80%98Morex%E2%80%99%20doubled%20haploid%20lines%20of%20barley%20at%20the%20seedling%20stage&amp;author=B.%20A.%20Siahsar&amp;author=M.%20Narouei&amp;volume=8&amp;publication_year=2010&amp;pages=751-759&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0060">
<cite>

Siddiqui, M. N.
, 
Jahiu, M.
, 
Kamruzzaman, M.
, 
Sanchez‐Garcia, M.
, 
Mason, A. S.
, 
Léon, J.
, &amp; 
Ballvora, A.
 (2024). Genetic control of root architectural traits under drought stress in spring barley (<em>Hordeum vulgare</em> L.). The Plant Genome, 17(2). 10.1002/tpg2.20463
</cite> [<a href="https://doi.org/10.1002/tpg2.20463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38764204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genetic%20control%20of%20root%20architectural%20traits%20under%20drought%20stress%20in%20spring%20barley%20(Hordeum%20vulgare%20L.)&amp;author=M.%20N.%20Siddiqui&amp;author=M.%20Jahiu&amp;author=M.%20Kamruzzaman&amp;author=M.%20Sanchez%E2%80%90Garcia&amp;author=A.%20S.%20Mason&amp;volume=17&amp;issue=2&amp;publication_year=2024&amp;pmid=38764204&amp;doi=10.1002/tpg2.20463&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0041">
<cite>

Smith, A. G.
, 
Malinowska, M.
, 
Janss, L.
, 
Ruud, A. K.
, 
Krusell, L.
, 
Jensen, J. D.
, &amp; 
Asp, T.
 (2024). Automated seminal root angle measurement with corrective annotation. <em>bioRxiv</em>. 10.1101/2024.03.13.584803
</cite> [<a href="https://doi.org/10.1101/2024.03.13.584803" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11512109/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39465185/" class="usa-link">PubMed</a>]</li>
<li id="tpg270088-bib-0061">
<cite>

Teulat, B.
, 
Borries, C.
, &amp; 
This, D.
 (2001). New QTLs identified for plant water status, water‐soluble carbohydrate and osmotic adjustment in a barley population grown in a growth‐chamber under two water regimes. Theoretical and Applied Genetics, 103(1), 161–170. 10.1007/s001220000503
</cite> [<a href="https://doi.org/10.1007/s001220000503" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=New%20QTLs%20identified%20for%20plant%20water%20status,%20water%E2%80%90soluble%20carbohydrate%20and%20osmotic%20adjustment%20in%20a%20barley%20population%20grown%20in%20a%20growth%E2%80%90chamber%20under%20two%20water%20regimes&amp;author=B.%20Teulat&amp;author=C.%20Borries&amp;author=D.%20This&amp;volume=103&amp;issue=1&amp;publication_year=2001&amp;pages=161-170&amp;doi=10.1007/s001220000503&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0062">
<cite>

Uga, Y.
, 
Okuno, K.
, &amp; 
Yano, M.
 (2011). Dro1, a major QTL involved in deep rooting of rice under upland field conditions. Journal of Experimental Botany, 62(8), 2485–2494. 10.1093/jxb/erq429

</cite> [<a href="https://doi.org/10.1093/jxb/erq429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21212298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Dro1,%20a%20major%20QTL%20involved%20in%20deep%20rooting%20of%20rice%20under%20upland%20field%20conditions&amp;author=Y.%20Uga&amp;author=K.%20Okuno&amp;author=M.%20Yano&amp;volume=62&amp;issue=8&amp;publication_year=2011&amp;pages=2485-2494&amp;pmid=21212298&amp;doi=10.1093/jxb/erq429&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0050">
<cite>

van Dijk, M.
, 
Morley, T.
, 
Rau, M. L.
, &amp; 
Saghai, Y.
 (2021). A meta‐analysis of projected global food demand and population at risk of hunger for the period 2010–2050. Nature Food, 2(7), 494–501. 10.1038/s43016-021-00322-9

</cite> [<a href="https://doi.org/10.1038/s43016-021-00322-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37117684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Food&amp;title=A%20meta%E2%80%90analysis%20of%20projected%20global%20food%20demand%20and%20population%20at%20risk%20of%20hunger%20for%20the%20period%202010%E2%80%932050&amp;author=M.%20van%20Dijk&amp;author=T.%20Morley&amp;author=M.%20L.%20Rau&amp;author=Y.%20Saghai&amp;volume=2&amp;issue=7&amp;publication_year=2021&amp;pages=494-501&amp;pmid=37117684&amp;doi=10.1038/s43016-021-00322-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0043">
<cite>

Vanhala, T. K.
, 
Van Rijn, C. P. E.
, 
Buntjer, J.
, 
Stam, P.
, 
Nevo, E.
, 
Poorter, H.
, &amp; 
Van Eeuwijk, F. A.
 (2004). Environmental, phenotypic and genetic variation of wild barley (<em>Hordeum spontaneum</em>) from Israel. Euphytica, 137, 297–309. 10.1023/B:EUPH.0000040450.91906.c5
</cite> [<a href="https://doi.org/10.1023/B:EUPH.0000040450.91906.c5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Environmental,%20phenotypic%20and%20genetic%20variation%20of%20wild%20barley%20(Hordeum%20spontaneum)%20from%20Israel&amp;author=T.%20K.%20Vanhala&amp;author=C.%20P.%20E.%20Van%20Rijn&amp;author=J.%20Buntjer&amp;author=P.%20Stam&amp;author=E.%20Nevo&amp;volume=137&amp;publication_year=2004&amp;pages=297-309&amp;doi=10.1023/B:EUPH.0000040450.91906.c5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0044">
<cite>

Van Hintum, T. J. L.
 (1994). Comparison of marker systems and construction of a core collection in a pedigree of European spring barley. Theoretical and Applied Genetics, 89, 991–997.
</cite> [<a href="https://doi.org/10.1007/BF00224529" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24178115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Comparison%20of%20marker%20systems%20and%20construction%20of%20a%20core%20collection%20in%20a%20pedigree%20of%20European%20spring%20barley&amp;author=T.%20J.%20L.%20Van%20Hintum&amp;volume=89&amp;publication_year=1994&amp;pages=991-997&amp;pmid=24178115&amp;doi=10.1007/BF00224529&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0045">
<cite>

Villiers, K.
, 
Voss‐Fels, K. P.
, 
Dinglasan, E.
, 
Jacobs, B.
, 
Hickey, L.
, &amp; 
Hayes, B. J.
 (2024). Evolutionary computing to assemble standing genetic diversity and achieve long‐term genetic gain. The Plant Genome, 17(2), e20467. 10.1002/tpg2.20467

</cite> [<a href="https://doi.org/10.1002/tpg2.20467" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38816340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Evolutionary%20computing%20to%20assemble%20standing%20genetic%20diversity%20and%20achieve%20long%E2%80%90term%20genetic%20gain&amp;author=K.%20Villiers&amp;author=K.%20P.%20Voss%E2%80%90Fels&amp;author=E.%20Dinglasan&amp;author=B.%20Jacobs&amp;author=L.%20Hickey&amp;volume=17&amp;issue=2&amp;publication_year=2024&amp;pages=e20467&amp;pmid=38816340&amp;doi=10.1002/tpg2.20467&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0046">
<cite>

Voorrips, R. E.
 (2002). MapChart: Software for the graphical presentation of linkage maps and QTLs. Journal of Heredity, 93, 77–78. 10.1093/jhered/93.1.77

</cite> [<a href="https://doi.org/10.1093/jhered/93.1.77" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12011185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Heredity&amp;title=MapChart:%20Software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.%20E.%20Voorrips&amp;volume=93&amp;publication_year=2002&amp;pages=77-78&amp;pmid=12011185&amp;doi=10.1093/jhered/93.1.77&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0063">
<cite>

Voss‐Fels, K. P.
, 
Robinson, H.
, 
Mudge, S. R.
, 
Richard, C.
, 
Newman, S.
, 
Wittkop, B.
, 
Stahl, A.
, 
Friedt, W.
, 
Frisch, M.
, 
Gabur, I.
, 
Miller‐Cooper, A.
, 
Campbell, B. C.
, 
Kelly, A.
, 
Fox, G.
, 
Christopher, J.
, 
Christopher, M.
, 
Chenu, K.
, 
Franckowiak, J.
, 
Mace, E. S.
, … 
Hickey, L. T.
 (2018). VERNALIZATION1 modulates root system architecture in wheat and barley. Molecular Plant, 11(1), 226–229. 10.1016/j.molp.2017.10.005

</cite> [<a href="https://doi.org/10.1016/j.molp.2017.10.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29056533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=VERNALIZATION1%20modulates%20root%20system%20architecture%20in%20wheat%20and%20barley&amp;author=K.%20P.%20Voss%E2%80%90Fels&amp;author=H.%20Robinson&amp;author=S.%20R.%20Mudge&amp;author=C.%20Richard&amp;author=S.%20Newman&amp;volume=11&amp;issue=1&amp;publication_year=2018&amp;pages=226-229&amp;pmid=29056533&amp;doi=10.1016/j.molp.2017.10.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0047">
<cite>

Voss‐Fels, K. P.
, 
Stahl, A.
, 
Wittkop, B.
, 
Lichthardt, C.
, 
Nagler, S.
, 
Rose, T.
, 
Chen, T.‐W.
, 
Zetzsche, H.
, 
Seddig, S.
, 
Majid Baig, M.
, 
Ballvora, A.
, 
Frisch, M.
, 
Ross, E.
, 
Hayes, B. J.
, 
Hayden, M. J.
, 
Ordon, F.
, 
Leon, J.
, 
Kage, H.
, 
Friedt, W.
, … 
Snowdon, R. J.
 (2019). Breeding improves wheat productivity under contrasting agrochemical input levels. Nature Plants, 5, 706–714. 10.1038/s41477-019-0445-5

</cite> [<a href="https://doi.org/10.1038/s41477-019-0445-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31209285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Plants&amp;title=Breeding%20improves%20wheat%20productivity%20under%20contrasting%20agrochemical%20input%20levels&amp;author=K.%20P.%20Voss%E2%80%90Fels&amp;author=A.%20Stahl&amp;author=B.%20Wittkop&amp;author=C.%20Lichthardt&amp;author=S.%20Nagler&amp;volume=5&amp;publication_year=2019&amp;pages=706-714&amp;pmid=31209285&amp;doi=10.1038/s41477-019-0445-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0048">
<cite>

Wickham, H.
, 
Chang, W.
, &amp; 
Wickham, M. H.
 (2016). Package ‘ggplot2’: Create elegant data visualisations using the grammar of graphics (Version 2) [Computer software]. Posit PBC. <a href="https://ggplot2.tidyverse.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ggplot2.tidyverse.org/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Wickham,%20H.%20,%20Chang,%20W.%20,%20&amp;%20Wickham,%20M.%20H.%20(2016).%20Package%20%E2%80%98ggplot2%E2%80%99:%20Create%20elegant%20data%20visualisations%20using%20the%20grammar%20of%20graphics%20(Version%202)%20%5BComputer%20software%5D.%20Posit%20PBC.%20https://ggplot2.tidyverse.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270088-bib-0049">
<cite>

Zhang, Y.
, 
Massel, K.
, 
Godwin, I. D.
, &amp; 
Gao, C.
 (2018). Applications and potential of genome editing in crop improvement. Genome Biology, 19, Article 210. 10.1186/s13059-018-1586-y

</cite> [<a href="https://doi.org/10.1186/s13059-018-1586-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6267055/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30501614/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Applications%20and%20potential%20of%20genome%20editing%20in%20crop%20improvement&amp;author=Y.%20Zhang&amp;author=K.%20Massel&amp;author=I.%20D.%20Godwin&amp;author=C.%20Gao&amp;volume=19&amp;publication_year=2018&amp;pages=210&amp;pmid=30501614&amp;doi=10.1186/s13059-018-1586-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>
<strong>Supplementary Figure 1</strong>. Marker distribution and density of curated SNP markers across chromosomes 1H through to 7H of the barley genome.</p>
<p>
<strong>Supplementary Figure 2</strong>. Pairwise marker LD‐matrices were calculated for each chromosome and LD‐decay was analysed showing an r2 of 0.2 with a sharp drop to 0.1 genome wide.</p>
<p>
<strong>Supplementary Figure 3</strong>. Relationship between haploblock SNP density and haplotype effect variance. The x‐axis shows the number of SNPs per haplotype block, and the y‐axis represents the variance of haplotype effects within each block. No clear linear trend is observed, suggesting that the number of SNPs in a block does not directly predict the magnitude of haplotype effect variance.</p>
<p>
<strong>Supplementary Figure 4</strong>. PCA of the AGG collection based on genome‐wide SNP variation. PC1 explains 29.5% and PC2 explains 13.9% of the total genetic variance. Individuals are coloured by passport information to reflect geographic origin: Africa (teal green), Americas (orange), Asia (purple), Europe (pink), Oceania (light green), United Kingdom (yellow), and individuals with no passport data (NA) are uncoloured.</p>
<p>
<strong>Supplemental Figure 5</strong>. Physical positioning of EGT2 and marker identification. <strong>(A)</strong> Positioned within significant haploblock b001908, EGT2 (HORVU5Hr1G027890) was co‐located. (B) The closest SNP markers flanking EGT2 were identified at approximately 162.5 M (chr5H_00023624_161513719‐161674224_3547) and 159.5 M (chr5H_00023425_159395855‐159466194_9375) base pairs respectively. Both markers are approximately 1 M base pairs from the EGT2 locus. These markers were used to classify EGT2 haplogroups for subsequent analysis.</p>
<p>
<strong>Supplemental Figure 6</strong>. EGT2 haplogroup frequency and association with SRA. <strong>(A)</strong> Haplogroups were created based on the marker information of the markers flanking EGT2 for each genotype. SNP combination CC‐CC was the largest frequency haplogroup followed by CC‐TT. (<strong>B)</strong> The highest frequency haplogroup was used as the reference for ANOVA to determine significant associations between haplogroups and SRA. CC‐TT was shown to have a significant association (p &lt; 0.05), with an increased SRA of two degrees.</p>
<p>
<strong>Supplementary Figure 7</strong>. Haploblock stacking for wide root phenotype. Each of the three panels displays the relationship between the number of beneficial haplotype blocks stacked (x‐axis) and the seminal root angle in degrees (y‐axis). The grey line represents the combined IGG and AGG populations, the orange line represents the IGG population alone, and the green line represents the AGG population alone. The plots illustrate the cumulative effect of stacking haplotypes associated with increased root angle, contributing to a wider root system architecture. The first panel considers the top 0.5% of haplotypes with the strongest positive effect on root angle, the second includes the top 10%, and the third includes all (100%) haplotypes.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12332326/bin/TPG2-18-e70088-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70088-s001.xlsx</a><sup> (45.2KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are available from InterGrain Pty Ltd, but restrictions apply to the availability of these data, which were used under licence for the current study, and so are not publicly available. Data are however available from the authors upon reasonable request and with permission of InterGrain Pty Ltd.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70088"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70088.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332326/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332326/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332326%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332326/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332326/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332326/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40776464/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332326/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40776464/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332326/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332326/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="wpEPI7v9SmfbLGqOKj05tYvZoAMC8GGQkOyjy8B3tQCH1t8Ucb7LxYCtzATHDrF9">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
