
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Non-native entanglement protein misfolding observed in all-atom simulations and supported by experimental structural ensembles - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="FF2082FE89B753831A82FE0030768981.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="sciadv">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333692/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Science Advances">
<meta name="citation_title" content="Non-native entanglement protein misfolding observed in all-atom simulations and supported by experimental structural ensembles">
<meta name="citation_author" content="Quyen V Vu">
<meta name="citation_author_institution" content="Institute of Physics, Polish Academy of Sciences, Al. Lotnikow 32/46, 02-668 Warsaw, Poland.">
<meta name="citation_author_institution" content="Department of Chemistry, Pennsylvania State University, University Park, PA 16802, USA.">
<meta name="citation_author" content="Ian Sitarik">
<meta name="citation_author_institution" content="Department of Chemistry, Pennsylvania State University, University Park, PA 16802, USA.">
<meta name="citation_author" content="Yang Jiang">
<meta name="citation_author_institution" content="Department of Chemistry, Pennsylvania State University, University Park, PA 16802, USA.">
<meta name="citation_author" content="Yingzi Xia">
<meta name="citation_author_institution" content="Department of Chemistry, Johns Hopkins University, Baltimore, MD 21218, USA.">
<meta name="citation_author" content="Piyoosh Sharma">
<meta name="citation_author_institution" content="Department of Chemistry, Johns Hopkins University, Baltimore, MD 21218, USA.">
<meta name="citation_author" content="Divya Yadav">
<meta name="citation_author_institution" content="Department of Chemistry, Johns Hopkins University, Baltimore, MD 21218, USA.">
<meta name="citation_author" content="Hyebin Song">
<meta name="citation_author_institution" content="Bioinformatics and Genomics Graduate Program, The Huck Institutes of the Life Sciences, Pennsylvania State University, University Park, PA 16802, USA.">
<meta name="citation_author_institution" content="Department of Statistics, Pennsylvania State University, University Park, PA 16802, USA.">
<meta name="citation_author" content="Mai Suan Li">
<meta name="citation_author_institution" content="Institute of Physics, Polish Academy of Sciences, Al. Lotnikow 32/46, 02-668 Warsaw, Poland.">
<meta name="citation_author_institution" content="Institute for Computational Sciences and Technology, Quang Trung Software City, Tan Chanh Hiep Ward, District 12, Ho Chi Minh City, Vietnam.">
<meta name="citation_author" content="Stephen D Fried">
<meta name="citation_author_institution" content="Department of Chemistry, Johns Hopkins University, Baltimore, MD 21218, USA.">
<meta name="citation_author_institution" content="T. C. Jenkins Department of Biophysics, Johns Hopkins University, Baltimore, MD 21218, USA.">
<meta name="citation_author" content="Edward P O’Brien">
<meta name="citation_author_institution" content="Department of Chemistry, Pennsylvania State University, University Park, PA 16802, USA.">
<meta name="citation_author_institution" content="Bioinformatics and Genomics Graduate Program, The Huck Institutes of the Life Sciences, Pennsylvania State University, University Park, PA 16802, USA.">
<meta name="citation_author_institution" content="Institute for Computational and Data Sciences, Pennsylvania State University, University Park, PA 16802, USA.">
<meta name="citation_publication_date" content="2025 Aug 8">
<meta name="citation_volume" content="11">
<meta name="citation_issue" content="32">
<meta name="citation_firstpage" content="eadt8974">
<meta name="citation_doi" content="10.1126/sciadv.adt8974">
<meta name="citation_pmid" content="40779622">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333692/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333692/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333692/pdf/sciadv.adt8974.pdf">
<meta name="description" content="Several mechanisms are known to cause monomeric protein misfolding. Coarse-grained simulations have predicted an additional mechanism exists involving off-pathway, noncovalent lasso entanglements, which are long-lived kinetic traps and structurally ...">
<meta name="og:title" content="Non-native entanglement protein misfolding observed in all-atom simulations and supported by experimental structural ensembles">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Several mechanisms are known to cause monomeric protein misfolding. Coarse-grained simulations have predicted an additional mechanism exists involving off-pathway, noncovalent lasso entanglements, which are long-lived kinetic traps and structurally ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333692/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12333692">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1126/sciadv.adt8974"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/sciadv.adt8974.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12333692%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12333692/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12333692/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333692/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-sciadv.gif" alt="Science Advances logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Science Advances" title="Link to Science Advances" shape="default" href="http://advances.sciencemag.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Adv</button></div>. 2025 Aug 8;11(32):eadt8974. doi: <a href="https://doi.org/10.1126/sciadv.adt8974" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1126/sciadv.adt8974</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Adv%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Adv%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Non-native entanglement protein misfolding observed in all-atom simulations and supported by experimental structural ensembles</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vu%20QV%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Quyen V Vu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Quyen V Vu</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Institute of Physics, Polish Academy of Sciences, Al. Lotnikow 32/46, 02-668 Warsaw, Poland.</div>
<div class="p">
<sup><sup>2</sup></sup>Department of Chemistry, Pennsylvania State University, University Park, PA 16802, USA.</div>
<div>Data curation, Formal analysis, Investigation, Methodology, Resources, Software, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vu%20QV%22%5BAuthor%5D" class="usa-link"><span class="name western">Quyen V Vu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sitarik%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ian Sitarik</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ian Sitarik</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Department of Chemistry, Pennsylvania State University, University Park, PA 16802, USA.</div>
<div>Formal analysis, Investigation, Methodology, Software, Validation, Visualization, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sitarik%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ian Sitarik</span></a>
</div>
</div>
<sup>2,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yang Jiang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yang Jiang</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Department of Chemistry, Pennsylvania State University, University Park, PA 16802, USA.</div>
<div>Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Resources, Software, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yang Jiang</span></a>
</div>
</div>
<sup>2,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xia%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yingzi Xia</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yingzi Xia</span></h3>
<div class="p">
<sup><sup>3</sup></sup>Department of Chemistry, Johns Hopkins University, Baltimore, MD 21218, USA.</div>
<div>Investigation, Methodology, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xia%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yingzi Xia</span></a>
</div>
</div>
<sup>3,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Piyoosh Sharma</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Piyoosh Sharma</span></h3>
<div class="p">
<sup><sup>3</sup></sup>Department of Chemistry, Johns Hopkins University, Baltimore, MD 21218, USA.</div>
<div>Investigation, Resources</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Piyoosh Sharma</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yadav%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Divya Yadav</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Divya Yadav</span></h3>
<div class="p">
<sup><sup>3</sup></sup>Department of Chemistry, Johns Hopkins University, Baltimore, MD 21218, USA.</div>
<div>Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Project administration, Resources, Validation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yadav%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Divya Yadav</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Hyebin Song</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Hyebin Song</span></h3>
<div class="p">
<sup><sup>4</sup></sup>Bioinformatics and Genomics Graduate Program, The Huck Institutes of the Life Sciences, Pennsylvania State University, University Park, PA 16802, USA.</div>
<div class="p">
<sup><sup>5</sup></sup>Department of Statistics, Pennsylvania State University, University Park, PA 16802, USA.</div>
<div>Methodology, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hyebin Song</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20MS%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Mai Suan Li</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Mai Suan Li</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Institute of Physics, Polish Academy of Sciences, Al. Lotnikow 32/46, 02-668 Warsaw, Poland.</div>
<div class="p">
<sup><sup>6</sup></sup>Institute for Computational Sciences and Technology, Quang Trung Software City, Tan Chanh Hiep Ward, District 12, Ho Chi Minh City, Vietnam.</div>
<div>Conceptualization, Formal analysis, Funding acquisition, Investigation, Methodology, Resources, Supervision, Validation, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20MS%22%5BAuthor%5D" class="usa-link"><span class="name western">Mai Suan Li</span></a>
</div>
</div>
<sup>1,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fried%20SD%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Stephen D Fried</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Stephen D Fried</span></h3>
<div class="p">
<sup><sup>3</sup></sup>Department of Chemistry, Johns Hopkins University, Baltimore, MD 21218, USA.</div>
<div class="p">
<sup><sup>7</sup></sup>T. C. Jenkins Department of Biophysics, Johns Hopkins University, Baltimore, MD 21218, USA.</div>
<div>Conceptualization, Funding acquisition, Investigation, Methodology, Project administration, Supervision, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fried%20SD%22%5BAuthor%5D" class="usa-link"><span class="name western">Stephen D Fried</span></a>
</div>
</div>
<sup>3,</sup><sup>7,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Brien%20EP%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Edward P O’Brien</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Edward P O’Brien</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Department of Chemistry, Pennsylvania State University, University Park, PA 16802, USA.</div>
<div class="p">
<sup><sup>4</sup></sup>Bioinformatics and Genomics Graduate Program, The Huck Institutes of the Life Sciences, Pennsylvania State University, University Park, PA 16802, USA.</div>
<div class="p">
<sup><sup>8</sup></sup>Institute for Computational and Data Sciences, Pennsylvania State University, University Park, PA 16802, USA.</div>
<div>Conceptualization, Funding acquisition, Investigation, Methodology, Project administration, Resources, Supervision, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Brien%20EP%22%5BAuthor%5D" class="usa-link"><span class="name western">Edward P O’Brien</span></a>
</div>
</div>
<sup>2,</sup><sup>4,</sup><sup>8,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>Institute of Physics, Polish Academy of Sciences, Al. Lotnikow 32/46, 02-668 Warsaw, Poland.</div>
<div id="aff2">
<sup><sup>2</sup></sup>Department of Chemistry, Pennsylvania State University, University Park, PA 16802, USA.</div>
<div id="aff3">
<sup><sup>3</sup></sup>Department of Chemistry, Johns Hopkins University, Baltimore, MD 21218, USA.</div>
<div id="aff4">
<sup><sup>4</sup></sup>Bioinformatics and Genomics Graduate Program, The Huck Institutes of the Life Sciences, Pennsylvania State University, University Park, PA 16802, USA.</div>
<div id="aff5">
<sup><sup>5</sup></sup>Department of Statistics, Pennsylvania State University, University Park, PA 16802, USA.</div>
<div id="aff6">
<sup><sup>6</sup></sup>Institute for Computational Sciences and Technology, Quang Trung Software City, Tan Chanh Hiep Ward, District 12, Ho Chi Minh City, Vietnam.</div>
<div id="aff7">
<sup><sup>7</sup></sup>T. C. Jenkins Department of Biophysics, Johns Hopkins University, Baltimore, MD 21218, USA.</div>
<div id="aff8">
<sup><sup>8</sup></sup>Institute for Computational and Data Sciences, Pennsylvania State University, University Park, PA 16802, USA.</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">Corresponding author. Email: <span>sdfried@jhu.edu</span> (S.D.F.); <span>epo2@psu.edu</span> (E.P.O.)</p>
</div>
<div class="fn" id="afn1">
<sup>†</sup><p class="display-inline">These authors contributed equally to this work.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Quyen V Vu</span></strong>: <span class="role">Data curation, Formal analysis, Investigation, Methodology, Resources, Software, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ian Sitarik</span></strong>: <span class="role">Formal analysis, Investigation, Methodology, Software, Validation, Visualization, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yang Jiang</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Resources, Software, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yingzi Xia</span></strong>: <span class="role">Investigation, Methodology, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Piyoosh Sharma</span></strong>: <span class="role">Investigation, Resources</span>
</div>
<div>
<strong class="contrib"><span class="name western">Divya Yadav</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Project administration, Resources, Validation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Hyebin Song</span></strong>: <span class="role">Methodology, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mai Suan Li</span></strong>: <span class="role">Conceptualization, Formal analysis, Funding acquisition, Investigation, Methodology, Resources, Supervision, Validation, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Stephen D Fried</span></strong>: <span class="role">Conceptualization, Funding acquisition, Investigation, Methodology, Project administration, Supervision, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Edward P O’Brien</span></strong>: <span class="role">Conceptualization, Funding acquisition, Investigation, Methodology, Project administration, Resources, Supervision, Writing - original draft, Writing - review &amp; editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 15; Accepted 2025 Jul 3; Collection date 2025 Aug 8.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © 2025 The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works. Distributed under a Creative Commons Attribution NonCommercial License 4.0 (CC BY-NC).</div>
<p>This is an open-access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by-nc/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution-NonCommercial license</a>, which permits use, distribution, and reproduction in any medium, so long as the resultant use is <strong>not</strong> for commercial advantage and provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12333692  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40779622/" class="usa-link">40779622</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Several mechanisms are known to cause monomeric protein misfolding. Coarse-grained simulations have predicted an additional mechanism exists involving off-pathway, noncovalent lasso entanglements, which are long-lived kinetic traps and structurally resemble the native state. Here, we examine whether such misfolded states occur in long-timescale, all-atom folding simulations of ubiquitin and λ-repressor. We find that these entangled misfolded states are populated in higher-resolution models. However, because of the small size of ubiquitin and λ-repressor, these states are short-lived. In contrast, coarse-grained simulations of a larger protein, IspE, predict that it populates long-lived misfolded states. Using an Arrhenius extrapolation applied to all-atom simulations, we estimate that these IspE misfolded states have lifetimes similar to the native state while remaining soluble. We further show that these misfolded states are consistent with the structural changes inferred from limited proteolysis and cross-linking mass spectrometry experiments. Our results indicate that misfolded states composed of non-native entanglements can persist for long timescales in both all-atom simulations and experiments.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>All-atom simulations populate a predicted class of misfolding, and ensembles are proposed based on structural mass spectrometry.</p></section><section id="sec1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p>Various factors are known to cause misfolding in monomeric proteins (<a href="#T1" class="usa-link">Table 1</a>). Recently, high-throughput, coarse-grained simulations of protein synthesis and folding of the <em>Escherichia coli</em> proteome have suggested that there exists an additional widespread mechanism of misfolding (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>–<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>): Proteins can populate off-pathway misfolded states that involve a change in noncovalent lasso entanglements (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>, <a href="#R2" class="usa-link" aria-describedby="R2"><em>2</em></a>). This type of entanglement is defined by the presence of two structural components: a loop formed by a protein backbone segment closed by a noncovalent native contact and another segment of the protein that is threaded through this loop (<a href="#R4" class="usa-link" aria-describedby="R4"><em>4</em></a>–<a href="#R6" class="usa-link" aria-describedby="R6"><em>6</em></a>) and, in some cases, wrapped around it multiple times (<a href="#F1" class="usa-link">Fig. 1, A and B</a>). Between 49 and 71% of native structures contain a noncovalent lasso entanglement depending on the organism (<a href="#R7" class="usa-link" aria-describedby="R7"><em>7</em></a>).</p>
<section class="tw xbox font-sm" id="T1"><h3 class="obj_head">Table 1. Mechanisms of protein misfolding.</h3>
<div class="caption p"><p>Protein misfolding can result from factors inherent to the amino acid sequence or from external conditions or processes, see references (<a href="#R56" class="usa-link" aria-describedby="R56"><em>56</em></a>–<a href="#R61" class="usa-link" aria-describedby="R61"><em>61</em></a>).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col width="%" span="1">
<col width="%" span="1">
<thead><tr>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Intrinsic to the primary structure</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Extrinsic to the primary structure</th>
</tr></thead>
<tbody>
<tr>
<td scope="row" valign="top" align="center" rowspan="1" colspan="1">Proline isomerization</td>
<td valign="top" align="center" rowspan="1" colspan="1">Transcription &amp; translation errors</td>
</tr>
<tr>
<td valign="top" align="center" scope="row" rowspan="1" colspan="1">Intrachain domain swapping</td>
<td valign="top" align="center" rowspan="1" colspan="1">mRNA splicing errors</td>
</tr>
<tr>
<td valign="top" align="center" scope="row" rowspan="1" colspan="1">Out-of-register β strands</td>
<td valign="top" align="center" rowspan="1" colspan="1">Perturbation of temperature, oxidative stress, pH, osmotic pressure, protein concentration, chaperones, processing enzymes</td>
</tr>
<tr>
<td valign="top" align="center" scope="row" rowspan="1" colspan="1">Incorrect helix packing</td>
<td valign="top" align="center" rowspan="1" colspan="1">Soluble &amp; insoluble aggregation</td>
</tr>
<tr>
<td valign="top" align="center" scope="row" rowspan="1" colspan="1">Mispacking of side chains</td>
<td valign="top" align="center" rowspan="1" colspan="1">Genetic mutations</td>
</tr>
<tr>
<td valign="top" align="center" scope="row" rowspan="1" colspan="1">Alternative secondary or tertiary structure</td>
<td valign="top" align="center" rowspan="1" colspan="1">Erroneous complex formation</td>
</tr>
<tr>
<td valign="top" align="center" scope="row" rowspan="1" colspan="1">Backtracking (frustration)</td>
<td valign="top" align="center" rowspan="1" colspan="1">Improper/incomplete degradation</td>
</tr>
<tr>
<td valign="top" align="center" scope="row" rowspan="1" colspan="1">Incorrect disulfide bond formation</td>
<td valign="top" align="center" rowspan="1" colspan="1">Cofactor and ligand concentrations</td>
</tr>
<tr>
<td scope="row" rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1">Posttranslational modifications</td>
</tr>
<tr>
<td scope="row" rowspan="1" colspan="1"></td>
<td valign="top" align="center" rowspan="1" colspan="1">Interchain domain swapping</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="F1"><h3 class="obj_head">Fig. 1. Visualizing changes of entanglement through various representations.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333692_sciadv.adt8974-f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e3e/12333692/ea23cd13b4c7/sciadv.adt8974-f1.jpg" loading="lazy" height="465" width="700" alt="Fig. 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Illustration of the two geometric elements that compose an entanglement: The closed loop is colored in red, and the threading segment is in blue. The loop is closed by a noncovalent contact between two residues (yellow). Such entanglements occur naturally in the native states of some proteins. In other cases, non-native formation of these entanglements can occur. (<strong>B</strong>) Three-dimensional (3D) structure of a misfolded entangled state from the protein <span class="font-variant-small-caps">d</span>-alanine—<span class="font-variant-small-caps">d</span>-alanine ligase B (DDLB; the closed loop and crossing section of the threading segment of their entangled regions are colored in red and blue, respectively) taken from previously reported coarse-grained simulations (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>). <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m1" display="inline" overflow="linebreak"><mrow><mi>Q</mi></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m2" display="inline" overflow="linebreak"><mrow><mi>G</mi></mrow></math></span> correspond respectively to the fraction of native contacts formed in the structure and the fraction of native contacts that exhibit a change in entanglement. (<strong>C</strong>) Flattened two-dimensional (2D) structure representation of the entangled state of DDLB. S, β sheet; A, α helix. (<strong>D</strong>) 3D structure of the crystal structure of DDLB. All native contacts are formed in the crystal structure ( <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m3" display="inline" overflow="linebreak"><mrow><mi>Q</mi></mrow></math></span> = 1), and there is no entanglement change (hence, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m4" display="inline" overflow="linebreak"><mrow><mi>G</mi><mo>=</mo><mn>0</mn></mrow></math></span> ). (<strong>E</strong>) Flattened 2D structure representation of the native state of DDLB. The coloring scheme of the relevant elements is the same in all panels.</p></figcaption></figure><p>The misfolded states observed in the coarse-grained simulations involved either the gain of a non-native entanglement (i.e., the formation of an entanglement that is not present in the native ensemble; table S1 and fig. S1) or the loss of a native entanglement (i.e., an entanglement present in the native state fails to form; table S1 and fig. S1) (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>–<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>). This predicted class of misfolding offers an explanation for the decades-old observations that nonfunctional (or less functional) protein molecules can persist for long timescales without aggregating and, in the presence of chaperones, neither get refolded nor degraded (<a href="#R8" class="usa-link" aria-describedby="R8"><em>8</em></a>–<a href="#R10" class="usa-link" aria-describedby="R10"><em>10</em></a>).</p>
<p>The coarse-grained simulations, in which individual residues are represented by single interaction sites, suggest that these misfolded states are often long-lived because, to correctly fold, they need to change their entanglement state: For example, an entangled state would need to disentangle, which is energetically costly as correctly folded portions of the protein would need to unfold (see folded gray segments in <a href="#F1" class="usa-link">Fig. 1, B and C</a>). Moreover, these states are more likely to bypass cellular quality control mechanisms and remain long-lived in vivo (<a href="#R8" class="usa-link" aria-describedby="R8"><em>8</em></a>–<a href="#R10" class="usa-link" aria-describedby="R10"><em>10</em></a>) because, in some cases, they are similar in size and structure to the native state and do not expose that much more hydrophobic surface (compare <a href="#F1" class="usa-link">Fig. 1, B and D</a>).</p>
<p>Two criticisms of these findings are that they are based on a model with limited spatial resolution and use force field approximations that have the potential to affect the results. It could be the case, for example, that these states are never populated in a more detailed, transferable all-atom model. Furthermore, the coarse-grained force field previously used was “structure-based,” meaning that the native state is encoded to be favored over other states (<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>). Both approximations have the potential to affect the accuracy of the earlier findings.</p>
<p>Here, we examine whether an all-atom model of the protein folding process—based on a transferable force field—also exhibits these self-entangled states, as observed previously in coarse-grained models; and if they do, we test whether those states are off-pathway long-lived traps with properties similar to the native state. We show that this type of misfolding occurs in all-atom simulations and that for typically sized proteins, these misfolded states can be very long-lived, soluble kinetically trapped states and then combine information from mass spectrometry and simulations to propose experimentally informed misfolded ensembles.</p></section><section id="sec2"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="sec3"><h3 class="pmc_sec_title">Ubiquitin and λ-repressor exhibit short-lived entangled states</h3>
<p>To address these questions, we analyze previously reported all-atom protein folding trajectories of ubiquitin (<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>) and the N-terminal domain (NTD) of λ-repressor (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>) starting from their native and unfolded states (<a href="#F2" class="usa-link">Fig. 2, A and B</a>). Ubiquitin is a small protein (76 residues) found in eukaryotic organisms and regulates a range of processes including protein degradation and the cell cycle. Ubiquitin folds on the millisecond timescale under typical conditions in vitro (<a href="#R14" class="usa-link" aria-describedby="R14"><em>14</em></a>) and in 3 ms in molecular dynamics simulations (<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>). The NTD of λ-repressor in the published trajectories (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>) is 80 residues in length, binds DNA, and folds on the microsecond timescale (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>, <a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>). Noncovalent lasso entanglements are not present in the native structure of either of these proteins.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Fig. 2. Misfolded gains of entanglement are observed in all-atom protein folding simulations.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333692_sciadv.adt8974-f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e3e/12333692/d9b97a9f19f3/sciadv.adt8974-f2.jpg" loading="lazy" height="1275" width="700" alt="Fig. 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Ubiquitin’s native structure and (<strong>B</strong>) λ-repressor’s native structure have no entanglement present. (<strong>C</strong>) Upper panel: ubiquitin’s fraction of native contacts <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m5" display="inline" overflow="linebreak"><mrow><mi>Q</mi></mrow></math></span> versus time from all-atom simulations. The blue dashed line represents <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m6" display="inline" overflow="linebreak"><mrow><mi>Q</mi><mo>=</mo><mn>0.6</mn></mrow></math></span> . Lower panel: ubiquitin’s degree of entanglement <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m7" display="inline" overflow="linebreak"><mrow><mi>G</mi></mrow></math></span> versus time for the same trajectory. (<strong>D</strong>) Same as (C) but for λ-repressor. (<strong>E</strong>) 3D structure of a ubiquitin entangled state observed at the time point labeled (i) in (C), with entanglement elements colored as in <a href="#F1" class="usa-link">Fig. 1A</a>. (<strong>F</strong>) Same as (E) but for λ-repressor and from point labeled (ii) in (D). (<strong>G</strong>) Flattened secondary structure representation for the entangled state in (E). (<strong>H</strong>) Same as (G) but for λ-repressor.</p></figcaption></figure><p>We analyzed whether non-native entanglements are populated during the folding trajectories of ubiquitin and λ-repressor totaling 8 ms and 643 μs of simulation time, respectively. To characterize the folding events in the simulations, we used the fraction of native contacts ( <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m8" display="inline" overflow="linebreak"><mrow><mi>Q</mi></mrow></math></span> ) and the order parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m9" display="inline" overflow="linebreak"><mrow><mi>G</mi></mrow></math></span> (defined in <a href="#E4" class="usa-link">Eq. 4</a>, which measures the fraction of native contacts that exhibit a change in entanglement). We observed several non-native entangled states (denoted by nonzero <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m10" display="inline" overflow="linebreak"><mrow><mi>G</mi></mrow></math></span> values, shown in <a href="#F2" class="usa-link">Fig. 2, C and D</a>) during folding transitions, which are indicated by shifts from low to high <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m11" display="inline" overflow="linebreak"><mrow><mi>Q</mi></mrow></math></span> values. Thus, protein self-entangled states are observed in all-atom models. Next, we asked whether these entangled states are off-pathway. We never observe in these trajectories a direct transition from entangled clusters (clusters 1 and 4 for ubiquitin and clusters 2 and 4 for λ-repressor from structural clustering; fig. S2) to the native cluster (clusters 6 and 5, respectively, for ubiquitin and λ-repressor). Hence, these misfolded states are off-pathway—they must unfold to reach the native state (as indicated in fig. S2, these entangled structures always transition to clusters with lower <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m12" display="inline" overflow="linebreak"><mrow><mi>Q</mi></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m13" display="inline" overflow="linebreak"><mrow><mi>G</mi></mrow></math></span> ). These entangled states clearly are short-lived, lasting just a few nanoseconds according to <a href="#F2" class="usa-link">Fig. 2 (C and D)</a>. This is seemingly at odds with the coarse-grained model prediction that these can be long-lived states (<a href="#R2" class="usa-link" aria-describedby="R2"><em>2</em></a>). However, these all-atom simulations were performed at high temperatures (390 K for ubiquitin and 350 K for λ-repressor), near their melting temperatures in the all-atom force field, and configurational transitions are accelerated. Therefore, these entangled states may be able to rapidly disentangle because of these high temperatures.</p>
<p>To test whether these entangled states are near-native, kinetic traps at physiological temperatures, we performed unrestrained molecular dynamics simulations at 310 K (37°C). We selected entangled structures that had at least 60% of their native contacts formed as starting structures, resulting in 21 and 12 structures, respectively, for ubiquitin and λ-repressor. Structurally, these entangled structures are qualitatively similar to each other (table S2); 20 of the 21 ubiquitin structures have closed loops located toward the C terminus, and the threading segment is composed of the N-terminal portion. For λ-repressor, the loop forms toward the C-terminal end and the N terminus threads through it in all 12 structures. Representative structures for these entangled states are shown in <a href="#F2" class="usa-link">Fig. 2 (E and F)</a>. Three independent trajectories were started from each of these conformations, and the simulations were run for 700 ns or until the entanglement was lost (i.e., a value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m14" display="inline" overflow="linebreak"><mrow><mi>G</mi><mo>=</mo><mn>0</mn></mrow></math></span> was reached).</p>
<p>For the 63 trajectories (= 21 × 3) of ubiquitin, 71% (45 of 63) of trajectories persist for 700 ns (an example in fig. S3A and detailed in table S2). On average then, the time to disentangle is ~2.1 μs (estimated using <a href="#E6" class="usa-link">Eq. 6</a>, setting <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m15" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo><mn>700</mn></mrow></math></span> ns). For λ-repressor, 17% of trajectories (6 of 36) persist in an entangled state for 700 ns (an example is shown in fig. S3B and table S2). The average time to disentangle for λ-repressor is ~390 ns, 50 times faster than for ubiquitin entanglements.</p>
<p>The radii of gyration of these entangled states are comparable to the native state (the differences are less than 10%; table S3), indicating that the entangled and native states are of similar sizes. These entangled states are estimated to have solubility similar to the native state (table S3) according to a model that accounts for the chemical properties of the exposed protein surface (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>). Next, we analyzed the secondary structure of proteins using Stride software (<a href="#R16" class="usa-link" aria-describedby="R16"><em>16</em></a>) and found that the non-native entangled states contain up to 83% of the secondary structure found in the native state (table S2). Thus, the size, solubility, and secondary structure analyses indicate that these misfolded states are similar to the native state.</p>
<p>Relative to their protein folding timescales, these entangled states are not long-lived kinetic traps even at 310 K. On the basis of the previous coarse-grained simulation results, this is to be expected (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>), because ubiquitin and the NTD of λ-repressor are small single-domain proteins. They are representative of the “model proteins” that have been traditionally studied by biophysicists and demonstrate the ability to rapidly and reversibly unfold and refold (<a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>) but are not representative of the complexity of proteomes (<a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>) [for example, the median protein length among eukaryotes is 361 residues and that among archaea is 247 residues (<a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>)]. In such small proteins, entanglements comprise a large proportion of the total protein structure present—and hence, it is easier to disentangle since most of the protein structure is misfolded and less stable. For example, for the entanglements reported in table S2, up to 78% of ubiquitin’s primary structure is involved in the entanglement (i.e., the minimum distance along the primary sequence required to maintain the entanglement; <a href="#F2" class="usa-link">Fig. 2, G and H</a>), and up to 84% of λ-repressor’s primary structure is involved in the entanglement. In contrast, when a protein of median length misfolds, a larger proportion of the already folded structure will need to unfold to permit disentanglement (<a href="#F1" class="usa-link">Fig. 1B</a>). Unfolding the correctly folded portions of these proteins is energetically more costly, and hence, misfolded states involving non-native entanglements in typically sized proteins are more likely to be long-lived kinetic traps (<a href="#R2" class="usa-link" aria-describedby="R2"><em>2</em></a>).</p></section><section id="sec4"><h3 class="pmc_sec_title">Misfolded states in a typically sized protein, IspE, are long-lived states in all-atom simulations</h3>
<p>Unrestrained all-atom simulations are incapable of folding such large proteins on tractable simulation timescales. Therefore, to test the aforementioned “size-effect” hypothesis, we calculated the lifetime of misfolded states of a larger protein, <em>E. coli</em>’s 4-diphosphocytidyl-2-<em>C</em>-methyl-<span class="font-variant-small-caps">d</span>-erythritol kinase (283 residues; gene <em>ispE</em>), within an aqueous, all-atom simulation model. This protein was chosen because it was previously identified to exhibit entangled misfolded states in coarse-grained simulations (<a href="#R2" class="usa-link" aria-describedby="R2"><em>2</em></a>, <a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>). To create models of misfolded IspE for all-atom simulations, we (i) performed protein folding simulations in a coarse-grained model; (ii) carried out structural clustering; (iii) scored the structures within each cluster for their consistency with limited proteolysis mass spectrometry (LiP-MS) and cross-linking mass spectrometry (XL-MS) data (described in the next section and Materials and Methods); (iv) chose, within each misfolded cluster, the single structure that had the highest score; and (v) backmapped those structures to an all-atom representation. These backmapped structures were then used as starting conformations for the aqueous, all-atom temperature jump simulations that were used to estimate their lifetimes at 298 K using an Arrhenius analysis. Five distinct misfolded clusters are observed in the simulations of this enzyme (step ii; <a href="#F3" class="usa-link">Fig. 3A</a>), and their backmapped all-atom representative structures (step v) are visualized in <a href="#F3" class="usa-link">Fig. 3B</a>. An Arrhenius analysis [i.e., performing unfolding simulations at high temperatures (600, 650, 700, 750, and 800 K) and extrapolating the unfolding time to 298 K] is necessary because all-atom simulations cannot reach the timescales associated with unfolding at low temperatures. The resulting unfolding rates exhibited super-Arrhenius behavior (figs. S5 and S6) (<a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>) and were fit to <a href="#E11" class="usa-link">Eq. 11</a> to estimate the unfolding rate at 298 K. The lifetimes were obtained as the inverse of the extrapolated unfolding rates.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Fig. 3. IspE misfolded states persist for long periods while remaining soluble.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333692_sciadv.adt8974-f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e3e/12333692/6f9001339218/sciadv.adt8974-f3.jpg" loading="lazy" height="582" width="800" alt="Fig. 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) −ln(<em>P</em>) surface and structural clusters identified from clustering analysis. (<strong>B</strong>) Structures selected from each cluster for all-atom temperature jump simulations. <em>Q</em> and <em>G</em> values of the structure are reported below the structure. Entanglement elements colored as in <a href="#F1" class="usa-link">Fig. 1A</a>. Interactive visualizations of these structures are available at <a href="https://obrien-lab.github.io/All_Atom_Misfolding_Exp_Ensemble/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://obrien-lab.github.io/All_Atom_Misfolding_Exp_Ensemble/</a>. Additional structures from each cluster can be found in data S2. (<strong>C</strong>) Apparent unfolding rates and estimated lifetimes of the structures in (B), derived from Arrhenius extrapolation. (<strong>D</strong>) Relative aggregation propensities of misfolded clusters compared to the native state. The aggregation propensity of cluster 4 is not significantly different from that of the native cluster, which is indicated by the blue circle. Error bars in (C) and (D) represent the 95% confidence intervals.</p></figcaption></figure><p>To determine whether these misfolded structures are long-lived, we compared their estimated lifetimes to the lifetime of the native cluster in the simulation model. (There are no experimentally reported lifetimes of native IspE; hence, we had to use the lifetime of native IspE within the model.) We find that the native cluster is estimated to have a lifetime of 3 hours (<a href="#F3" class="usa-link">Fig. 3C</a>), corresponding to an unfolding rate of 9.1 × 10<sup>−5</sup> s<sup>−1</sup>. In contrast, structures from clusters 2 to 4 persist for days to months (<a href="#F3" class="usa-link">Fig. 3, B and C</a>), while less structured clusters 0 and 1 last only seconds to minutes. We emphasize that because of the large uncertainties in these lifetime estimates (see 95% confidence intervals in <a href="#F3" class="usa-link">Fig. 3C</a>), only qualitative conclusions can be drawn that are relative to the estimated lifetime of the native cluster. We conclude that in all-atom simulations, misfolded clusters 2 to 4 are likely to be kinetic traps with lifetimes of a similar order of magnitude to the native cluster.</p></section><section id="sec5"><h3 class="pmc_sec_title">IspE misfolds for over an hour in refolding experiments</h3>
<p>We next experimentally tested whether IspE populates misfolded states. To address this, we performed LiP-MS (<a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>–<a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>) and XL-MS (<a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>–<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>) on IspE after denaturing and refolding, and compared it to a native IspE sample that was never denatured to detect structural changes such as those associated with changes in entanglement (<a href="#R2" class="usa-link" aria-describedby="R2"><em>2</em></a>, <a href="#R28" class="usa-link" aria-describedby="R28"><em>28</em></a>). Refolded samples were prepared by first chemically denaturing IspE and then refolding it by dilution from high to low denaturant concentration. Then, the sample was incubated at room temperature for 1 hour before being exposed either to proteinase K (PK) in the LiP-MS experiment or to the cross-linking agent in the XL-MS experiment. We then detect differential signal changes between native (untreated) samples and refolded (treated) samples. We define a signal as significantly different if it exhibits more than a twofold difference in the mean abundance between the refolded and native samples ( <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m16" display="inline" overflow="linebreak"><mrow><mrow><mo>∣</mo><mspace width="-.15em"></mspace><msub><mtext>log</mtext><mn>2</mn></msub><mtext>FC</mtext><mspace width="0.25em"></mspace><mrow><mo stretchy="true">(</mo><mtext>fold change</mtext><mo stretchy="true">)</mo></mrow><mspace width="-.15em"></mspace><mo>∣</mo></mrow><mo>≥</mo><mn>1</mn></mrow></math></span> ) and an adjusted <em>P</em> value, false discovery rate corrected—applied separately to the XL-MS and LiP-MS datasets—using the Benjamini-Hochberg method (<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>), of less than or equal to 0.05. We observe that there are changes in the proteolytic susceptibility and cross-linking patterns between chemically renatured IspE 1 hour after refolding conditions have been established and a native reference that was not denatured. Specifically, we identified 13 PK cleavage sites from LiP-MS (<a href="#T2" class="usa-link">Table 2</a> and data S1, sheet labeled “EXP-LiPMS”) and 20 cross-links from XL-MS (<a href="#T3" class="usa-link">Table 3</a> and data S1, sheet labeled “EXP-XLMS”) experiments that exhibited significant differences in abundance between refolded and native samples. These results suggest that in the case of LiP-MS, the solvent exposure of these regions along the primary structure of IspE was significantly altered in the refolded sample, and in the case of XL-MS, structural changes in the protein led to altered cross-linking propensity between various pairs of residues. We therefore conclude that IspE can populate soluble misfolded conformations over long timescales, qualitatively consistent with our simulation predictions.</p>
<section class="tw xbox font-sm" id="T2"><h4 class="obj_head">Table 2. PK cut sites exhibiting significant differences in abundance between refolded and native samples identified by LiP-MS.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<thead><tr>
<th valign="top" align="left" scope="col" rowspan="1" colspan="1">PK site</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="m17" display="inline" overflow="linebreak"><mrow><mrow><mo>∣</mo><msub><mtext mathvariant="bold">log</mtext><mn mathvariant="bold">2</mn></msub><mtext>FC</mtext><mo>∣</mo></mrow></mrow></math>
</span>
<sup>*</sup>
</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="m18" display="inline" overflow="linebreak"><mrow><mtext>Adj</mtext><mo mathvariant="bold">.</mo><mi mathvariant="bold-italic">P</mi><mo> </mo><mtext mathvariant="bold">value</mtext></mrow></math>
</span>
<sup>†</sup>
</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">PK site</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="m19" display="inline" overflow="linebreak"><mrow><mrow><mo>∣</mo><msub><mtext mathvariant="bold">log</mtext><mn mathvariant="bold">2</mn></msub><mtext>FC</mtext><mo>∣</mo></mrow></mrow></math>
</span>
<sup>*</sup>
</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="m20" display="inline" overflow="linebreak"><mrow><mtext>Adj</mtext><mo mathvariant="bold">.</mo><mi mathvariant="bold-italic">P</mi><mo> </mo><mtext mathvariant="bold">value</mtext></mrow></math>
</span>
<sup>†</sup>
</th>
</tr></thead>
<tbody>
<tr>
<td scope="row" valign="top" align="left" rowspan="1" colspan="1">
<strong>R69</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.78</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.028</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>F207</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.60</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.000</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>M75</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">4.14</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.007</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>A254</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.15</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.016</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>A78</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.17</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.000</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>R255</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.53</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.033</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>I159</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.14</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.002</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>L258</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.90</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.009</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>L202</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.46</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.007</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>Q260</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.47</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.018</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>K204</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.74</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.038</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>A261</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.67</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.004</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>E206</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.82</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.003</td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p22"><p>*FC: fold change defined as the ratio of the mean refold-sample abundance to native-sample abundance across five replicates each. †Adj. <em>P</em> value: Benjamini-Hochberg adjusted <em>P</em> value.</p></div></div></section><section class="tw xbox font-sm" id="T3"><h4 class="obj_head">Table 3. Cross-links exhibiting significant differences in abundance between refolded and native samples identified by XL-MS.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<thead><tr>
<th valign="top" align="left" scope="col" rowspan="1" colspan="1">Cross-link</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="m23" display="inline" overflow="linebreak"><mrow><mrow><mo>∣</mo><msub><mtext mathvariant="bold">log</mtext><mn mathvariant="bold">2</mn></msub><mtext>FC</mtext><mo>∣</mo></mrow></mrow></math>
</span>
<sup>*</sup>
</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="m24" display="inline" overflow="linebreak"><mrow><mtext>Adj</mtext><mo mathvariant="bold">.</mo><mi mathvariant="bold-italic">P</mi><mo> </mo><mtext mathvariant="bold">value</mtext></mrow></math>
</span>
<sup>†</sup>
</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Cross-link</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="m25" display="inline" overflow="linebreak"><mrow><mrow><mo>∣</mo><msub><mtext mathvariant="bold">log</mtext><mn mathvariant="bold">2</mn></msub><mtext>FC</mtext><mo>∣</mo></mrow></mrow></math>
</span>
<sup>*</sup>
</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="m26" display="inline" overflow="linebreak"><mrow><mtext>Adj</mtext><mo mathvariant="bold">.</mo><mi mathvariant="bold-italic">P</mi><mo> </mo><mtext mathvariant="bold">value</mtext></mrow></math>
</span>
<sup>†</sup>
</th>
</tr></thead>
<tbody>
<tr>
<td scope="row" valign="top" align="left" rowspan="1" colspan="1">
<strong>S7-Y16</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.54</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.050</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>K96-K196</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">5.33</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.000</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>K10-K196</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">7.24</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.003</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>K96-S208</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">5.33</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.001</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>Y16-K196</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.72</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.003</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>T161-K204</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.19</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.042</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>K76-T86</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">4.53</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.000</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>K196-S198</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.72</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.000</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>K76-T161</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">11.93</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.008</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>K196-K204</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">3.90</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.000</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>K76-K196</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.85</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.000</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>K196-S208</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">6.83</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.000</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>K76-S208</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">5.55</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.014</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>K196-K271</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.93</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.000</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>T77-S208</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">4.74</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.016</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>K196-S276</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.34</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.042</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>S81-T86</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.20</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.002</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>S198-S208</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">3.87</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.024</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>S81-K96</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.88</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.000</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>T201-S208</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.60</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.001</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p23"><p>*FC: fold change defined as the ratio of the mean refold-sample abundance to native-sample abundance across five replicates each. † Adj. <em>P</em> value: Benjamini-Hochberg adjusted <em>P</em> value.</p></div></div></section></section><section id="sec6"><h3 class="pmc_sec_title">Misfolded structural ensembles of IspE that are consistent with LiP-MS and XL-MS</h3>
<p>With 33 unique experimental signals reporting on changes in structure in IspE relative to the native state, we can examine which misfolded states seen in our simulations are consistent with these signals. To do this, we take a two-stage approach: First, we simulated the refolding of IspE in our coarse-grained model, and second, we scored the resulting misfolded subpopulations for their overall consistency with the mass spectrometric data (see Materials and Methods). From the simulations, we structurally clustered the resulting conformations and found one cluster corresponding to the native ensemble and five distinct misfolded clusters (<a href="#F3" class="usa-link">Fig. 3A</a>) that vary in their degree of native-structure formation (<em>Q</em>) and changes in entanglement status (<em>G</em>). Scoring each misfolded cluster in terms of its relative change in solvent accessibility for LiP-MS results and solvent-accessible surface distance (SASD) [as calculated with JWalk (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>)] for XL-MS results, we find that cluster 3 is the highest scoring with 22 of 33 experimental signals consistent with the structural changes observed in this cluster. (For LiP-MS and XL-MS, respectively, 7 of 13 and 15 of 20 are consistent; data S1 labeled “Comparison-LiPMS” and “Comparison-XLMS”). In comparison, cluster 0 is consistent with 14 of the signals, cluster 2 with 11, and cluster 1 with 9. Cluster 4, which is the most similar to the native cluster, is consistent with only four of the signals.</p>
<p>Across the five misfolded clusters, 29 unique experimental signals can be explained by the structural changes observed in this combination of clusters. One LiP-MS and three XL-MS signals cannot be explained by our methodology. Thus, 88% of the experimental signals can be explained by this ensemble of five misfolded clusters. For interactive visualization, we provide structures from each misfolded cluster in PDB format in data S2.</p></section><section id="sec7"><h3 class="pmc_sec_title">Most IspE misfolded states are likely to stay soluble</h3>
<p>Although most of these misfolded states are long-lived, we next examine whether they are likely to remain soluble or aggregate. To address this, we estimated the relative aggregation propensity of these misfolded clusters compared to the native cluster by incorporating information on the solvent accessibility of aggregation-prone regions (<a href="#E14" class="usa-link">Eq. 14</a>). We first used the AMYLPRED2 server (<a href="#R31" class="usa-link" aria-describedby="R31"><em>31</em></a>) to identify regions in IspE that are prone to aggregation when exposed to solvent. We then calculated the solvent-accessible surface area (SASA) of these regions from our simulated trajectories for each misfolded and native cluster. Our analysis indicates that the aggregation propensity of these misfolded clusters is very similar to that of the native cluster (<a href="#F3" class="usa-link">Fig. 3D</a>), with differences of less than 5%—except for cluster 3, which exhibits a 12.15% relative increase to the native state. Although these small differences are statistically significant (Benjamini-Hochberg adjusted <em>P</em> value ≤ 0.05) for all clusters except cluster 4, the small effect sizes (&lt;5% increase) suggest that most of these misfolded ensembles would remain soluble on timescales similar to the native ensemble.</p></section></section><section id="sec8"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p>Coarse-grained simulations predicted the existence of an additional widespread mechanism of protein misfolding. Motivated by the possibility this may be an artifact of those simulations’ resolution and structure-based force field, in this study, we examined whether protein folding modeled at all-atom resolution also exhibits this type of misfolding. We have demonstrated that ubiquitin and λ-repressor populate these entangled misfolded states in all-atom folding simulations. Thus, protein folding involving a change of entanglement status occurs regardless of model resolution, lending support to the validity of the earlier conclusions drawn from coarse-grained modeling.</p>
<p>Because of the small size of ubiquitin and λ-repressor, the entangled states they populate are short-lived, persisting for just a few microseconds. For larger proteins, all-atom simulations indicate that these misfolded states can persist for much longer periods—on timescales similar to the lifetime of the native state while remaining soluble. This relationship between protein size and misfolded lifetimes arises from the fact that a more stabilizing native structure can form around the localized misfolded elements in larger proteins as compared to smaller proteins, creating a larger energy barrier to unfold improperly folded portions.</p>
<p>Experimentally, we observed that IspE can populate misfolded structures that form after dilution from the denaturant and remain misfolded for over an hour. This result is consistent with our simulations of IspE (<a href="#F3" class="usa-link">Fig. 3, A to C</a>), giving us more confidence in our broader conclusions. When cross-referencing the misfolded signals from LiP-MS and XL-MS experiments with the misfolded clusters from the IspE coarse-grained simulations, we find that almost 90% of the experimental signals can be explained by the simulated structural changes. Thus, the structures we have identified in <a href="#F3" class="usa-link">Fig. 3B</a> and data S2 represent an experimentally informed misfolded ensemble involving soluble, long-lived states.</p>
<p>Misfolding involving changes of entanglement status can arise from either a gain of a non-native entanglement or a loss of a native entanglement. In ubiquitin and λ-repressor, which contain no noncovalent lasso entanglements in their native state, it is only possible to misfold through the formation of a non-native entanglement (gain of entanglement), which we observed during their folding. In contrast, IspE has two unique representative, noncovalent lasso entanglements present in its native structure (table S5). In this system, we observed misfolded structures involving either gains of entanglement, losses of entanglement, or combinations thereof. Both classes of misfolding are manifestations of topological frustration. In the one case, a protein backbone segment is improperly positioned outside a backbone loop, corresponding to a loss of entanglement; in the other case, it is improperly positioned within a backbone loop, corresponding to a gain of entanglement. The consequence of both of these types of topological traps is that they both must backtrack (<a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>) out of these states, meaning that already natively structured portions of the protein must unfold to allow proper folding. Small proteins that misfold in this way have additional folding pathways that bypass backtracking that include threading of the loop in cases in which the native entanglement failed to form or unthreading of a loop in misfolded states involving formation of non-native entanglements (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>, <a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>). However, for typically sized proteins, such as IspE, backtracking will often be the dominant pathway because substantial portions of the structure can still properly fold.</p>
<p>These changes of entanglement that lead to misfolded, long-lived kinetic traps expand the mechanisms of topological frustration that give rise to the kinetic partitioning mechanism for understanding biomolecular self-assembly in proteins and nucleic acids (<a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>–<a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>). In the kinetic partitioning model, frustration is inherent to the nature of biopolymers, because interactions at local and global length scales can be in energetic conflict, giving rise to different structural preferences. Particular packing arrangements, for example, of hydrophobic residues or secondary structural elements might be energetically favored locally but disfavored in the presence of tertiary interactions (<a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>). As a consequence, near-native misfolded states can be populated, especially for larger proteins, which will tend to exhibit greater frustration. The current study adds to the structural mechanisms by which such frustration can arise, specifically through the formation of native-like states that either fail to form their native entanglement or form a non-native entanglement.</p>
<p>Proteins containing entanglements in their native state are widespread across different organisms. The proportion of globular proteins with them ranges from 49% in humans to 71% in <em>E. coli</em> (<a href="#R7" class="usa-link" aria-describedby="R7"><em>7</em></a>). Perhaps expectedly then, this type of misfolding has been found to be common based on high-throughput coarse-grained simulations of the <em>E. coli</em> proteome (<a href="#R2" class="usa-link" aria-describedby="R2"><em>2</em></a>). Furthermore, because structure equals function, such misfolding has been implicated in influencing ensemble-averaged functioning of proteins like specific activity (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>) and, in some cases, in bypassing the chaperone machinery in cells depending on how native-like in structure are these misfolded states (<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>). We hypothesize that this understudied class of protein misfolding will offer insights into protein function and homeostasis, including new causal mechanisms for disease and aging that can open up new therapeutic targets.</p></section><section id="sec9"><h2 class="pmc_sec_title">MATERIALS AND METHODS</h2>
<section id="sec10"><h3 class="pmc_sec_title">All-atom simulations of ubiquitin and λ-repressor</h3>
<p>Initial entangled conformations of the protein were placed at the center of a rectangular periodic box with the minimum distance between the box edge and protein atoms of 1.5 nm, the system was solvated with the TIP3P water model (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>), and NaCl was added to neutralize and mimic relevant salt concentrations [ubiquitin: 0 mM; λ-repressor: 50 mM; the salt concentration was added according to original studies (<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>, <a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>)]. Energy minimization was then carried out using the steepest descent algorithm (<a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>, <a href="#R39" class="usa-link" aria-describedby="R39"><em>39</em></a>) to minimize steric conflict. The systems were then gradually heated from 2 to 310 K over 1 ns. After that, the system was equilibrated for 1 ns in the NVT ensemble and 1 ns in the NPT ensemble with all heavy atoms restrained using a harmonic potential with a force constant of 1000 kJ/mol per square nanometer before it went into production run in the NPT ensemble. The particle mesh Ewald method (<a href="#R40" class="usa-link" aria-describedby="R40"><em>40</em></a>) was used to calculate long-range electrostatic interactions beyond 12 Å, and the Lennard-Jones interactions were calculated with a cutoff distance of 12 Å and applied, smoothly switching the forces to zero between 10 and 12 Å. Temperature and pressure were maintained at 310 K and 1 atm using a Nose-Hoover thermostat (<a href="#R41" class="usa-link" aria-describedby="R41"><em>41</em></a>, <a href="#R42" class="usa-link" aria-describedby="R42"><em>42</em></a>) and Parrinello-Rahman barostat (<a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>), respectively. The LINCS algorithm (<a href="#R44" class="usa-link" aria-describedby="R44"><em>44</em></a>) was used to constrain all bonds involving hydrogen atoms, and the integration time step was set to 2 fs. Simulations were performed using GROMACS 2018 (<a href="#R45" class="usa-link" aria-describedby="R45"><em>45</em></a>) with the CHARMM36m force field (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). This procedure was used in all the new simulations carried out in this study.</p></section><section id="sec11"><h3 class="pmc_sec_title">Characterizing changes in entanglement</h3>
<p>Calculating changes in entanglement has been reported elsewhere (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>–<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>). Here, we briefly describe our procedure. To detect noncovalent lasso entanglements, we use linking numbers, which require at least one closed loop. We define this loop as being composed of the backbone trace connecting residues 𝑖 and 𝑗 that form a noncovalent contact in the given protein conformation. The looped portion (red segment in <a href="#F1" class="usa-link">Fig. 1A</a>) is identified if the C<sub>α</sub> coordinates of residues 𝑖 and 𝑗 (yellow spheres in <a href="#F1" class="usa-link">Fig. 1A</a>) are closer than 8 Å and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m30" display="inline" overflow="linebreak"><mrow><mrow><mo>∣</mo><mi>i</mi><mo>−</mo><mi>j</mi><mo>∣</mo></mrow><mo>≥</mo><mn>10</mn></mrow></math></span> residues. Outside this loop is an N-terminal segment composed of residues 1 through 𝑖 − 1, and there is a C-terminal segment composed of residues 𝑗 + 1 through 𝑁. These two segments represent open curves, whose entanglement through the closed loop was characterized by linking numbers denoted 𝑔<sub>N</sub> and 𝑔<sub>C</sub>. For a given structure of an <em>N</em>-residue protein, with a contact present at residues (𝑖, 𝑗), the average coordinates <strong>R</strong><sub>𝑙</sub> and the gradient <em>d</em><strong>R</strong><sub>𝑙</sub> of point 𝑙 on the curves were calculated as</p>
<table class="disp-formula p" id="E1"><tr>
<td class="formula"><math id="m31" display="block" overflow="linebreak"><mrow><mrow><mo stretchy="true">{</mo><mtable><mtr><mtd><maligngroup></maligngroup><msub><mi mathvariant="bold">R</mi><mi>l</mi></msub><mo>=</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mrow><mo stretchy="true">(</mo><msub><mi mathvariant="bold">r</mi><mi>l</mi></msub><mo>+</mo><msub><mi mathvariant="bold">r</mi><mrow><mi>l</mi><mo>+</mo><mn>1</mn></mrow></msub><mo stretchy="true">)</mo></mrow></mtd></mtr><mtr><mtd><maligngroup></maligngroup><mi>d</mi><msub><mi mathvariant="bold">R</mi><mi>l</mi></msub><mo>=</mo><msub><mi mathvariant="bold">r</mi><mrow><mi>l</mi><mo>+</mo><mn>1</mn></mrow></msub><mo>−</mo><msub><mi mathvariant="bold">r</mi><mi>l</mi></msub></mtd></mtr></mtable></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where <strong>r</strong><sub>𝑙</sub> is the coordinates of the C<sub>α</sub> atom in residue 𝑙. The linking numbers 𝑔<sub>N</sub>(𝑖, 𝑗) and 𝑔<sub>C</sub>(𝑖, 𝑗) of N and C termini, respectively, were calculated as</p>
<table class="disp-formula p" id="E2"><tr>
<td class="formula"><math id="m32" display="block" overflow="linebreak"><mtable><mtr><mtd columnalign="left"><mrow><mrow><mo stretchy="true">{</mo><mtable><mtr><mtd columnalign="left"><maligngroup></maligngroup><msub><mi mathvariant="bold">g</mi><mi>N</mi></msub><mrow><mo stretchy="true">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="true">)</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mrow><mn>4</mn><mi mathvariant="normal">π</mi></mrow></mfrac><mstyle displaystyle="true"><munderover><mo stretchy="true">∑</mo><mrow><mi>m</mi><mo>=</mo><mn>6</mn></mrow><mrow><mi>i</mi><mo>−</mo><mn>5</mn></mrow></munderover></mstyle><mstyle displaystyle="true"><munderover><mo stretchy="true">∑</mo><mrow><mi>n</mi><mo>=</mo><mi>i</mi></mrow><mrow><mi>j</mi><mo>−</mo><mn>1</mn></mrow></munderover></mstyle><mfrac><mrow><msub><mi mathvariant="bold">R</mi><mi>m</mi></msub><mo>−</mo><msub><mi mathvariant="bold">R</mi><mi>n</mi></msub></mrow><msup><mrow><mo>∣</mo><msub><mi mathvariant="bold">R</mi><mi>m</mi></msub><mo>−</mo><msub><mi mathvariant="bold">R</mi><mi>n</mi></msub><mspace width="-.15em"></mspace><mo>∣</mo></mrow><mn>3</mn></msup></mfrac><mrow><mo stretchy="true">(</mo><mi>d</mi><msub><mi mathvariant="bold">R</mi><mi>m</mi></msub><mo>×</mo><mi>d</mi><msub><mi mathvariant="bold">R</mi><mi>n</mi></msub><mo stretchy="true">)</mo></mrow></mtd></mtr><mtr><mtd><maligngroup></maligngroup><msub><mi mathvariant="bold">g</mi><mi mathvariant="normal">C</mi></msub><mrow><mo stretchy="true">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="true">)</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mrow><mn>4</mn><mi mathvariant="normal">π</mi></mrow></mfrac><mstyle displaystyle="true"><munderover><mo stretchy="true">∑</mo><mrow><mi>m</mi><mo>=</mo><mi>i</mi></mrow><mrow><mi>j</mi><mo>−</mo><mn>1</mn></mrow></munderover></mstyle><mstyle displaystyle="true"><munderover><mo stretchy="true">∑</mo><mrow><mi>n</mi><mo>=</mo><mi>j</mi><mo>+</mo><mn>4</mn></mrow><mrow><mi>N</mi><mo>−</mo><mn>6</mn></mrow></munderover></mstyle><mfrac><mrow><msub><mi mathvariant="bold">R</mi><mi>m</mi></msub><mo>−</mo><msub><mi mathvariant="bold-italic">R</mi><mi>n</mi></msub></mrow><msup><mrow><mo>∣</mo><msub><mi mathvariant="bold">R</mi><mi>m</mi></msub><mo>−</mo><msub><mi mathvariant="bold">R</mi><mi>n</mi></msub><mspace width="-.15em"></mspace><mo>∣</mo></mrow><mn>3</mn></msup></mfrac><mrow><mo stretchy="true">(</mo><mi>d</mi><msub><mi mathvariant="bold">R</mi><mi>m</mi></msub><mo>×</mo><mi>d</mi><msub><mi mathvariant="bold">R</mi><mi>n</mi></msub><mo stretchy="true">)</mo></mrow></mtd></mtr></mtable></mrow></mrow></mtd></mtr></mtable></math></td>
<td class="label">(2)</td>
</tr></table>
<p>where we excluded the first five residues on the N-terminal curve, the last five residues on the C-terminal curve, and four residues before and after the native contact to eliminate the error introduced by both the high flexibility and contiguity of the termini and trivial entanglements in the local structure. It is worth noting that partial linking values between 0.5 and 0.7 may not signify real entanglements and, thus, those have to be checked manually (<a href="#R47" class="usa-link" aria-describedby="R47"><em>47</em></a>). The above integrations yield two noninteger values, and the total linking number for a contact (𝑖, 𝑗) is estimated as the sum of N- and C-terminal linking numbers</p>
<table class="disp-formula p" id="E3"><tr>
<td class="formula"><math id="m33" display="block" overflow="linebreak"><mrow><mi>g</mi><mrow><mo stretchy="true">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="true">)</mo></mrow><mo>=</mo><mtext>round</mtext><mrow><mo stretchy="true">[</mo><msub><mi>g</mi><mi>N</mi></msub><mrow><mo stretchy="true">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="true">)</mo></mrow><mo stretchy="true">]</mo></mrow><mo>+</mo><mtext>round</mtext><mrow><mo stretchy="true">[</mo><msub><mi>g</mi><mi>C</mi></msub><mrow><mo stretchy="true">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="true">)</mo></mrow><mo stretchy="true">]</mo></mrow></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>The use of the round function in <a href="#E3" class="usa-link">Eq. 3</a> serves to discretize the computed linking numbers, which can result in noninteger values because of numerical integration between a closed loop and an open segment. By applying the round function, we map such values to discrete integers (for example, <em>g</em> = 0.6 → 1), effectively capturing the presence or absence of an entanglement in a robust and consistent way. This discretization allows us to perform meaningful comparisons between the entanglement states of different structures. For instance, even if both a conformation in the simulated trajectory and the native structure are entangled, their computed linking values may slightly differ (for example, <em>g</em> = 0.6 versus <em>g</em> = 0.7), yet both indicate the same qualitative topological feature (i.e., one event of terminal threading). The round function ensures that such topologies are treated identically in our analysis.</p>
<p>Comparing the absolute value of the total linking number for a contact (𝑖, 𝑗) at a given conformation to that of a reference state (i.e., native state) allows us to ascertain a gain or loss of linking between the backbone trace loop and the terminal open curves as well as any switches in chirality. There are six changes in linking cases we should consider (table S1 and fig. S1) when using this approach to quantify entanglement.</p>
<p>We defined the <em>G</em> metric, the degree of entanglement, as a time-dependent order parameter that reflects the extent of the topological entanglement changes in a given structure compared to the native structure and is calculated as</p>
<table class="disp-formula p" id="E4"><tr>
<td class="formula"><math id="m34" display="block" overflow="linebreak"><mrow><mi>G</mi><mrow><mo stretchy="true">(</mo><mi>t</mi><mo stretchy="true">)</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mi>M</mi></mfrac><munder><mo>∑</mo><mrow><mo stretchy="true">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="true">)</mo></mrow></munder><mi mathvariant="normal">Θ</mi><mrow><mo stretchy="true">[</mo><mrow><mo stretchy="true">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="true">)</mo></mrow><mo>∈</mo><mtext>NC</mtext><mo>⋂</mo><mi>g</mi><mrow><mo stretchy="true">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>t</mi><mo stretchy="true">)</mo></mrow><mo>≠</mo><msup><mi>g</mi><mtext>native</mtext></msup><mrow><mo stretchy="true">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="true">)</mo></mrow><mo stretchy="true">]</mo></mrow></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<p>where (𝑖, 𝑗) is one of the native contacts in the native crystal structure; NC is the set of native contacts formed in the current structure at time 𝑡; 𝑔(𝑖, 𝑗, 𝑡) and 𝑔<sup>native</sup>(𝑖, 𝑗) are, respectively, the total linking number of the contact (𝑖, 𝑗) at time 𝑡 and native structures estimated using <a href="#E3" class="usa-link">Eq. 3</a>; <em>M</em> is the total number of native contacts within the native structure; and the selection function Θ equals 1 when the condition is true and equals 0 when it is false. The larger <em>G</em> is, the greater the number of native contacts that have changed their entanglement status relative to the reference state.</p></section><section id="sec12"><h3 class="pmc_sec_title">Disentanglement time constant from unrestrained simulations at physiological temperature</h3>
<p>The disentangled time constant can be estimated through the connection with the portion of disentangled trajectories</p>
<table class="disp-formula p" id="E5"><tr>
<td class="formula"><math id="m35" display="block" overflow="linebreak"><mrow><msub><mi>P</mi><mtext>dis</mtext></msub><mrow><mo stretchy="true">(</mo><mi>t</mi><mo stretchy="true">)</mo></mrow><mo>=</mo><mn>1</mn><mo>−</mo><msup><mi>e</mi><mfrac><mrow><mo>−</mo><mi>t</mi></mrow><mi mathvariant="normal">τ</mi></mfrac></msup></mrow></math></td>
<td class="label">(5)</td>
</tr></table>
<table class="disp-formula p" id="E6"><tr>
<td class="formula"><math id="m36" display="block" overflow="linebreak"><mrow><mi mathvariant="normal">τ</mi><mo>=</mo><mfrac><mrow><mo>−</mo><mi>t</mi></mrow><mrow><mtext>ln</mtext><mrow><mo stretchy="true">[</mo><mn>1</mn><mo>−</mo><msub><mi>P</mi><mtext>dis</mtext></msub><mrow><mo stretchy="true">(</mo><mi>t</mi><mo stretchy="true">)</mo></mrow><mo stretchy="true">]</mo></mrow></mrow></mfrac></mrow></math></td>
<td class="label">(6)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m37" display="inline" overflow="linebreak"><mrow><msub><mi>P</mi><mtext>dis</mtext></msub><mrow><mo stretchy="true">(</mo><mi>t</mi><mo stretchy="true">)</mo></mrow></mrow></math></span> is the portion of disentangled trajectories at time <em>t</em>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m38" display="inline" overflow="linebreak"><mrow><mi mathvariant="normal">τ</mi></mrow></math></span> is a time constant that entanglement disappears. We use this equation to estimate the average lifetime of entangled states of ubiquitin and λ-repressor by setting <em>t</em> = 700 ns.</p></section><section id="sec13"><h3 class="pmc_sec_title">Secondary structure similarity definition</h3>
<p>The secondary structure similarity in table S2 is defined as the fraction of residues that are in the correct secondary element at the current structure compared to the native structure; we only count residues in the α helix or β sheet. The secondary structural elements of protein are assigned by the STRIDE program (<a href="#R16" class="usa-link" aria-describedby="R16"><em>16</em></a>).</p></section><section id="sec14"><h3 class="pmc_sec_title">Calculation of solubility of the entangled structures</h3>
<p>The solubility of the entangled structures in table S3 is defined as the percent soluble protein estimated via the insolubility propensities of structure 𝑖 ( <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m39" display="inline" overflow="linebreak"><mrow><msubsup><mi mathvariant="normal">χ</mi><mi>i</mi><mtext>sol</mtext></msubsup></mrow></math></span> ), a fully disordered structure, which has no secondary and tertiary structure elements ( <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m40" display="inline" overflow="linebreak"><mrow><msubsup><mi mathvariant="normal">χ</mi><mtext>disordered</mtext><mtext>insol</mtext></msubsup></mrow></math></span> ), and the minimum propensity values of all structures [ <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m41" display="inline" overflow="linebreak"><mrow><mtext>min</mtext><mrow><mo stretchy="true">(</mo><msubsup><mi mathvariant="normal">χ</mi><mi>i</mi><mtext>sol</mtext></msubsup><mo stretchy="true">)</mo></mrow></mrow></math></span>]</p>
<table class="disp-formula p" id="E7"><tr>
<td class="formula"><math id="m42" display="block" overflow="linebreak"><mrow><msubsup><mi>f</mi><mi>i</mi><mtext>sol</mtext></msubsup><mo>=</mo><mfrac><mrow><msubsup><mi mathvariant="normal">χ</mi><mtext>disordered</mtext><mtext>insol</mtext></msubsup><mo>−</mo><msubsup><mi mathvariant="normal">χ</mi><mi>i</mi><mtext>insol</mtext></msubsup></mrow><mrow><msubsup><mi mathvariant="normal">χ</mi><mtext>disordered</mtext><mtext>insol</mtext></msubsup><mo>−</mo><mtext>min</mtext><mrow><mo stretchy="true">(</mo><msubsup><mi mathvariant="normal">χ</mi><mi>i</mi><mtext>insol</mtext></msubsup><mo stretchy="true">)</mo></mrow></mrow></mfrac></mrow></math></td>
<td class="label">(7)</td>
</tr></table>
<p>The insolubility was estimated by considering the aggregation propensity ( <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m43" display="inline" overflow="linebreak"><mrow><msup><mi mathvariant="normal">χ</mi><mtext>agg</mtext></msup></mrow></math></span> ) and degradation propensity ( <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m44" display="inline" overflow="linebreak"><mrow><msup><mi mathvariant="normal">χ</mi><mtext>deg</mtext></msup></mrow></math></span> ) minus the 70-kDa heat shock protein (Hsp70) binding propensity ( <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m45" display="inline" overflow="linebreak"><mrow><msup><mi mathvariant="normal">χ</mi><mrow><mtext>Hsp</mtext><mn>70</mn></mrow></msup></mrow></math></span> ) that is considered to prevent the misfolding protein from aggregation</p>
<table class="disp-formula p" id="E8"><tr>
<td class="formula"><math id="m46" display="block" overflow="linebreak"><mrow><msup><mi mathvariant="normal">χ</mi><mtext>insol</mtext></msup><mo>=</mo><msup><mi mathvariant="normal">χ</mi><mtext>agg</mtext></msup><mo>+</mo><msup><mi mathvariant="normal">χ</mi><mtext>deg</mtext></msup><mo>−</mo><msup><mi mathvariant="normal">χ</mi><mrow><mtext>Hsp</mtext><mn>70</mn></mrow></msup></mrow></math></td>
<td class="label">(8)</td>
</tr></table>
<p>For a given entangled structure, the aggregation, degradation, and Hsp70 binding propensities were estimated as the relative change of the SASA of the aggregation-prone, degradation-prone, and Hsp70 binding regions of the entangled state 𝑖 against the SASA of the native structure</p>
<table class="disp-formula p" id="E9"><tr>
<td class="formula"><math id="m47" display="block" overflow="linebreak"><mrow><msubsup><mi mathvariant="normal">χ</mi><mi>i</mi><mi>T</mi></msubsup><mo>=</mo><mfrac><msubsup><mtext>SASA</mtext><mi>i</mi><mi>T</mi></msubsup><msubsup><mtext>SASA</mtext><mtext>native</mtext><mi>T</mi></msubsup></mfrac><mo>−</mo><mn>1</mn></mrow></math></td>
<td class="label">(9)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m48" display="inline" overflow="linebreak"><mrow><mi>T</mi></mrow></math></span> can be aggregation, degradation, or Hsp70 binding propensities. The aggregation-prone region was predicted by the AmylPred2 server (<a href="#R31" class="usa-link" aria-describedby="R31"><em>31</em></a>). The degradation-prone region was defined as the hydrophobic residues (Ile, Val, Phe, Cys, Met, Ala, Gly, and Trp). The Hsp70 binding region was predicted by using ChaperISM (<a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>).</p></section><section id="sec15"><h3 class="pmc_sec_title">Coarse-grained protein folding simulations of IspE protein and structural clustering</h3>
<p>We use a previously published Go̅-based coarse-grain model (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>) in which each amino acid is represented by a single interaction site placed at the C<sub>α</sub> atom with a specific van der Waals radius for each amino acid. Initial conformations for refolding simulations were generated by heating the native state of the protein to 1000 K for 15 ns. The final conformation from heating was then temperature quenched at 310 K to initialize refolding. All simulations were carried out using a Langevin thermostat at a temperature of 310 K, with a time step of 15 fs and a friction coefficient of 0.050 ps<sup>−1</sup>. All simulations were carried out using OpenMM version 7.7 (<a href="#R49" class="usa-link" aria-describedby="R49"><em>49</em></a>). Trajectories were saved after every 5000 steps; a total of 56 independent trajectories of 1.5 μs was performed. The last 500 ns from each trajectory was used for structural clustering on the basis of two order parameters that capture the nativeness of the structures (fraction of native contacts, <em>Q</em>) and the changes in self-entanglement of the protein (fraction of native contact change in entanglement, <em>G</em>). The fraction of native contacts at each conformation was defined as the number of native contacts formed divided by the total number of native contacts in the crystal structure. Two residues (𝑖, 𝑗) were considered to form a native contact if the distance between their C<sub>α</sub> atoms was less than 1.2 times their distance in the crystal structure, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m49" display="inline" overflow="linebreak"><mrow><mrow><mo>∣</mo><mi>i</mi><mo>−</mo><mi>j</mi><mo>∣</mo></mrow><mo>&gt;</mo><mn>3</mn></mrow></math></span> . The fraction of native contact change in entanglement (<em>G</em>) was calculated as described in <a href="#E4" class="usa-link">Eq. 4</a>. Using the K-mean++ clustering algorithm (<a href="#R50" class="usa-link" aria-describedby="R50"><em>50</em></a>), 100 microstates were grouped from the coarse-grained simulations. These microstates are then coarse grained into structural states using the PCCA+ algorithm (<a href="#R51" class="usa-link" aria-describedby="R51"><em>51</em></a>). To determine a reasonable number of clusters for the PCCA+ analysis, we used a timescale separation criterion based on identifying a large gap in the implied timescale (ITS) spectrum derived from eigenvalues of the transition probability matrix (<a href="#R52" class="usa-link" aria-describedby="R52"><em>52</em></a>). Specifically, we calculated the ITSs, sorted them in descending order, and computed the ITS ratio, defined by the ratio <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m50" display="inline" overflow="linebreak"><mrow><msub><mtext>ITS</mtext><mi>i</mi></msub><mo>/</mo><msub><mtext>ITS</mtext><mrow><mi>i</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow></math></span> . A large ratio indicates that the corresponding dynamical processes are kinetically well separated, while a ratio close to 1 suggests that the processes can be grouped together. All clustering analyses were performed by using the PyEmma package (<a href="#R53" class="usa-link" aria-describedby="R53"><em>53</em></a>).</p></section><section id="sec16"><h3 class="pmc_sec_title">Structure selection for Arrhenius analyses</h3>
<p>The last 500-ns simulation structures were labeled using the cluster ID and the experimental signals they produce. Here, we identify a simulated structure capable of generating an experimental signal only if it demonstrates a SASA greater than the 97.5th percentile or less than the 2.5th percentile of all structures in the native ensemble for a LiP-MS signal or exhibits a cross-linking propensity score outside the corresponding percentiles for an XL-MS signal. These groups represent the misfolded structural ensemble that can generate the observed LiP-MS and/or XL-MS signals.</p>
<p>For each of the clusters 0 to 4, we choose a single structure that can generate the greatest number of experimental signals. If there is a tie (multiple structures generate the same number of consistent experiment signals), preference is given to the structure with the highest microstate probability, followed by the highest <em>Q</em> and <em>G</em> values.</p></section><section id="sec17"><h3 class="pmc_sec_title">In silico temperature jump simulations and Arrhenius analysis of 4-diphosphocytidyl-2-<em>C</em>-methyl-<span class="font-variant-small-caps">d</span>-erythritol kinase</h3>
<p>To estimate the lifetime of the representative structure of each metastable state, we calculate the time it takes for this structure state to unfold. It is not feasible to trace the unfolding process of such large protein in conventional simulation; hence, we performed simulation at high temperatures, 600, 650, 700, 750, and 800 K, and then used Arrhenius analysis to extrapolate the unfolding at 298 K. The simulation protocol was similar to that used for ubiquitin and λ-repressor, with the exception that the salt concentration was set at 150 mM NaCl. For the production runs, at each temperature, we performed 50 independent trajectories in the NVT ensemble and then got the list of first-passage times that the structure is unfolded. To account for fluctuation, a trajectory was considered to reach the unfolded state if <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m51" display="inline" overflow="linebreak"><mrow><mi>Q</mi><mo>≤</mo><mn>0.3</mn></mrow></math></span> for 1 ns. The starting structures are highly native-like (<em>Q</em> &gt; 0.7; see <a href="#F3" class="usa-link">Fig. 3B</a> in the main text), so there is an initial delay time during which native contacts are partially lost, but the system has not yet reached the unfolding threshold. This results in a delay time in the survival probability curves, during which the fraction of unfolded conformations remains zero (fig. S4). Once sufficient disruption of native contacts occurs, the survival probability begins to decay exponentially. The survival probability was then fitted to a single-exponential function to find the fitting parameters (<em>t</em><sub>0</sub> and <em>k</em>)</p>
<table class="disp-formula p" id="E10"><tr>
<td class="formula"><math id="m52" display="block" overflow="linebreak"><mrow><msub><mi>S</mi><mi>U</mi></msub><mrow><mo stretchy="true">(</mo><mi>t</mi><mo stretchy="true">)</mo></mrow><mo>=</mo><mrow><mo stretchy="true">{</mo><mtable><mtr><mtd><maligngroup></maligngroup><mn>1</mn><mo>,</mo><mspace width="0.5em"></mspace><mn>0</mn><mo>≤</mo><mi>t</mi><mo>&lt;</mo><msub><mi>t</mi><mn>0</mn></msub></mtd></mtr><mtr><mtd><maligngroup></maligngroup><msup><mi>e</mi><mrow><mo>−</mo><mi>k</mi><mrow><mo stretchy="true">(</mo><mi>t</mi><mo>−</mo><msub><mi>t</mi><mn>0</mn></msub><mo stretchy="true">)</mo></mrow></mrow></msup><mo>,</mo><mspace width="0.5em"></mspace><mi>t</mi><mo>≥</mo><msub><mi>t</mi><mn>0</mn></msub></mtd></mtr></mtable></mrow></mrow></math></td>
<td class="label">(10)</td>
</tr></table>
<p>where <em>t</em><sub>0</sub> is the delay time of entanglement (fig. S4). The apparent unfolding rate at temperature <em>T</em> is <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m53" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mtext>app</mtext></msub><mrow><mo stretchy="true">(</mo><mi>T</mi><mo stretchy="true">)</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mrow><msub><mi>t</mi><mn>0</mn></msub><mo>+</mo><mfrac><mn>1</mn><mi>k</mi></mfrac></mrow></mfrac></mrow></math></span>.</p>
<p>The apparent unfolding rates are found to have a super-Arrhenius behavior (figs. S5 and S6) (<a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>, <a href="#R54" class="usa-link" aria-describedby="R54"><em>54</em></a>) and are then fitted as the function of <em>T</em></p>
<table class="disp-formula p" id="E11"><tr>
<td class="formula"><math id="m54" display="block" overflow="linebreak"><mrow><mtext>ln</mtext><mrow><mo stretchy="true">(</mo><msub><mi>k</mi><mtext>app</mtext></msub><mo stretchy="true">)</mo></mrow><mo>=</mo><mfrac><mi>a</mi><msup><mi>T</mi><mn>2</mn></msup></mfrac><mo>+</mo><mfrac><mi>b</mi><mi>T</mi></mfrac><mo>+</mo><mi>c</mi></mrow></math></td>
<td class="label">(11)</td>
</tr></table>
<p>Here <em>a</em>, <em>b</em>, and <em>c</em> are the free fitting parameters. We then extrapolated to obtain the apparent unfolding rate at the target temperature of 298 K. The extrapolated unfolding time in simulation is <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m55" display="inline" overflow="linebreak"><mrow><msub><mi mathvariant="normal">τ</mi><mtext>sim</mtext></msub><mo>=</mo><mfrac><mn>1</mn><msub><mi>k</mi><mtext>app</mtext></msub></mfrac></mrow></math></span>.</p>
<p>The uncertainties were estimated using bootstrapped resampling with 10<sup>4</sup> iterations. For each iteration, at each temperature, we randomly selected with replacement 50 samples from the list of mean first-passage times of the original data at that temperature and then repeated the fitting procedure on the resampled data. Resampled datasets were discarded if the coefficient of determination <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m56" display="inline" overflow="linebreak"><mrow><msup><mi>R</mi><mn>2</mn></msup><mo>&lt;</mo><mn>0.9</mn></mrow></math></span>.</p></section><section id="sec18"><h3 class="pmc_sec_title">Limited proteolysis experiments</h3>
<p>To perform limited proteolysis, 2 μl of PK (from Tritirachium album) stock solutions (prepared at a concentration of 0.1 mg/ml in native buffer with 20% glycerol and then stored at −20°C for future use) was added to either native or refolded samples, which were prepared on a 200-μl scale. The reaction mixtures were rapidly mixed using a pipette [the enzyme:substrate ratio is 1:100 (w/w)]. Samples were incubated for 1 min before being transferred to a preheated mineral oil bath at 105°C to quench the PK activity. The samples were left in the oil bath for 5 min. After quenching, the proteolyzed samples were transferred to a microcentrifuge tube containing 152 mg of urea to reach a final concentration of 8 M. The limited proteolysis experiments were conducted in five replicates for both native and refolded IspE samples, resulting in a total of 10 samples.</p></section><section id="sec19"><h3 class="pmc_sec_title">Cross-linking experiments</h3>
<p>Two hundred microliters of native samples was prepared, consisting of 20 mM Hepes, 100 mM NaCl, 2 mM MgCl<sub>2</sub>, 1 mM tris(2-carboxyethyl)phosphine (TCEP), and 0.1% glycerol at pH 7.4, with IspE protein (0.1 mg/ml). Cross-linking was initiated by adding 2 μl of DSBU-<em>d</em><sub>0</sub> (100 mM stock in dimethyl sulfoxide; Thermo Fisher Scientific, A35459) to a final concentration of 1 mM. Simultaneously, refolded IspE samples were cross-linked with 2 μl of DSBU-<em>d</em><sub>12</sub> [deuterated disuccinimidyl dibutyric urea (DSBU); 100 mM stock in dimethyl sulfoxide]. DSBU-<em>d</em><sub>12</sub> was prepared as described in a separate publication (<a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a>).</p>
<p>The samples were then mixed and incubated at room temperature for 1 hour using a Roto-Mini rotator (Alkali Scientific RS3024) at 10 rpm. The reactions were quenched by adding 1 M tris-HCl (pH 7.5) to a final concentration of 20 mM, followed by 30-min incubation at room temperature, rotating at 10 rpm on the Roto-Mini. After quenching, the native and refolded samples were combined and transferred to a new microfuge tube containing 304 mg of urea to achieve a final concentration of 8 M. Cross-linking experiments were conducted in five replicates for both native and refolded IspE samples; after mixing the pairs, a total of five replicate samples (400 μl) was prepared.</p></section><section id="sec20"><h3 class="pmc_sec_title">Modifying matched and nonaccessible cross-link score function</h3>
<p>To computationally estimate the cross-linking propensity between residues 𝑖 and 𝑗, we first use the JWalk algorithm (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>) to calculate the SASD between their C<sub>α</sub> atoms. Subsequently, the cross-linking propensity is assessed using a customized version of the matched and nonaccessible cross-link (MNXL) score function (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>). The original function was designed for pairs of lysines cross-linked by the dextran sulfate sodium linker (with a length of 11.4 Å); the MNXL score function needs adaptation for our XL-MS experiment where we use the DSBU cross-linker molecule (has a length of 12.5 Å) (<a href="#R26" class="usa-link" aria-describedby="R26"><em>26</em></a>) and permit nonlysine residues to participate in cross-linking. To account for the differences, we have adjusted the parameters of the MNXL scoring function</p>
<table class="disp-formula p" id="E12"><tr>
<td class="formula"><math id="m57" display="block" overflow="linebreak"><mtable><mtr><mtd columnalign="left"><mrow><mtext>MNXL score</mtext><mrow><mo stretchy="true">(</mo><msub><mtext>SASD</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo stretchy="true">)</mo></mrow></mrow><mo>=</mo></mtd></mtr><mtr><mtd columnalign="left"><mrow><mrow><mo stretchy="true">{</mo><mtable><mtr><mtd columnalign="left"><maligngroup></maligngroup><mi>N</mi><mrow><mo stretchy="true">(</mo><msub><mtext>SASD</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>,</mo><msub><mi mathvariant="normal">μ</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mn>2</mn></msubsup><mo stretchy="true">)</mo></mrow><mo>,</mo><mspace width="0.5em"></mspace><malignmark></malignmark><msub><mtext>SASD</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>≤</mo><msub><mtext>threshold</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mtd></mtr><mtr><mtd><maligngroup></maligngroup><mn>0</mn><mo>,</mo><mspace width="0.5em"></mspace><malignmark></malignmark><mtext>otherwise</mtext></mtd></mtr></mtable></mrow></mrow></mtd></mtr></mtable></math></td>
<td class="label">(12)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m58" display="inline" overflow="linebreak"><mrow><mi>N</mi><mrow><mo stretchy="true">(</mo><msub><mtext>SASD</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>,</mo><msub><mi mathvariant="normal">μ</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mn>2</mn></msubsup><mo stretchy="true">)</mo></mrow></mrow></math></span> represents the normal distribution</p>
<table class="disp-formula p" id="E13"><tr>
<td class="formula"><math id="m59" display="block" overflow="linebreak"><mrow><mi>N</mi><mrow><mo stretchy="true">(</mo><msub><mtext>SASD</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>,</mo><msub><mi mathvariant="normal">μ</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mn>2</mn></msubsup><mo stretchy="true">)</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mrow><msqrt><mrow><mn>2</mn><mi mathvariant="normal">π</mi></mrow></msqrt><msub><mi mathvariant="normal">σ</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow></mfrac><msup><mi>e</mi><mrow><mfrac><mrow><mo>−</mo><mn>1</mn></mrow><mn>2</mn></mfrac><msup><mrow><mo stretchy="true">(</mo><mfrac><mrow><msub><mtext>SASD</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>−</mo><msub><mi mathvariant="normal">μ</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow><msub><mi mathvariant="normal">σ</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mfrac><mo stretchy="true">)</mo></mrow><mn>2</mn></msup></mrow></msup></mrow></math></td>
<td class="label">(13)</td>
</tr></table>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m60" display="inline" overflow="linebreak"><mrow><msub><mi mathvariant="normal">μ</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow></math></span> , <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m61" display="inline" overflow="linebreak"><mrow><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mn>2</mn></msubsup></mrow></math></span> , and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m62" display="inline" overflow="linebreak"><mrow><msub><mtext>threshold</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow></math></span> denote the mean, variance, and the SASD cutoff threshold for a specific pair of residues <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m63" display="inline" overflow="linebreak"><mrow><mrow><mo stretchy="true">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="true">)</mo></mrow></mrow></math></span> involved in cross-linking, respectively. If the SASD of the pair of residues is less than the cutoff threshold, the propensity is determined by a normal distribution with the mean and variance specifically tailored for the pair of residues (table S6). If the SASD exceeds the threshold or if at least one residue in the pair is buried and inaccessible, then the propensity is assigned a value of 0.</p></section><section id="sec21"><h3 class="pmc_sec_title">Comparison of misfolded and native clusters for consistency between simulations and experimental data</h3>
<p>LiP-MS and XL-MS detected 13 and 20 structural signals, respectively, exhibiting significant differences between refolded and native samples. To assess the consistency between the misfolded ensembles from our simulations and experimental data, we examined whether the SASA of a region within five residues around the PK cut site identified from the experiment and the cross-linking propensity (MNXL score, <a href="#E12" class="usa-link">Eq. 12</a>) of pairs of residues from misfolded ensembles are statistically significantly different from the native ensemble from our simulations. The <em>P</em> values were computed using numerical permutation tests with 10<sup>5</sup> iterations with the null hypothesis stating that there is no difference in the ensemble mean of SASA or cross-linking propensity between misfolded clusters and the native cluster. The <em>P</em> values were then adjusted using the Benjamini-Hochberg method (<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>). If <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m64" display="inline" overflow="linebreak"><mrow><mtext>adj</mtext><mo>.</mo><mi>P</mi><mspace width="0.25em"></mspace><mtext>value</mtext><mo>≤</mo><mn>0.05</mn></mrow></math></span> , we reject the null hypothesis and accept the alternative hypothesis: The ensemble means of SASA or cross-linking propensity are significantly different between misfolded and native clusters. To mitigate the risk of underestimating the confidence interval of the ensemble mean (thus potentially overestimating the number of consistent signals) because of data correlation, we subsampled our data every 50 frames (as suggested by block averaging).</p></section><section id="sec22"><h3 class="pmc_sec_title">Calculation of the relative aggregation propensity of the IspE misfolded clusters</h3>
<p>The aggregation propensity of the misfolded clusters of IspE relative to the native cluster is quantified by the difference in the SASA of aggregation-prone regions between these two clusters. This propensity of misfolded cluster <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m65" display="inline" overflow="linebreak"><mrow><mi>i</mi></mrow></math></span> relative to the native state is calculated using the following equation</p>
<table class="disp-formula p" id="E14"><tr>
<td class="formula"><math id="m66" display="block" overflow="linebreak"><mrow><msub><mi mathvariant="normal">χ</mi><mi>i</mi></msub><mo>=</mo><mfrac><mrow><msub><mrow><mo stretchy="true">〈</mo><mtext>SASA</mtext><mo stretchy="true">〉</mo></mrow><mi>i</mi></msub><mo>−</mo><msub><mrow><mo stretchy="true">〈</mo><mtext>SASA</mtext><mo stretchy="true">〉</mo></mrow><mtext>native</mtext></msub></mrow><msub><mrow><mo stretchy="true">〈</mo><mtext>SASA</mtext><mo stretchy="true">〉</mo></mrow><mtext>native</mtext></msub></mfrac><mo>×</mo><mn>100</mn><mo>%</mo></mrow></math></td>
<td class="label">(14)</td>
</tr></table>
<p>The aggregation-prone region was predicted by the AmylPred2 server (<a href="#R31" class="usa-link" aria-describedby="R31"><em>31</em></a>). The uncertainties were estimated from bootstrap resampling with 10<sup>5</sup> iterations.</p></section><section id="sec23"><h3 class="pmc_sec_title">Uncertainty estimation</h3>
<p>The uncertainties, represented as 95% confidence intervals, were estimated using bootstrapped resampling with 10<sup>5</sup> iterations (except for the Arrhenius extrapolation, where 10<sup>4</sup> iterations were used because of computational expensive). For a sample of <em>N</em> elements, in each iteration, we randomly selected <em>N</em> values from the original data with replacement. The mean was calculated from these resampled values, and the distribution of these means was used to estimate the 95% confidence interval.</p></section><section id="sec24"><h3 class="pmc_sec_title">Permutation test</h3>
<p>To evaluate whether the sample means ( <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m67" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mi mathvariant="normal">A</mi></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m68" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mi mathvariant="normal">B</mi></msub></mrow></math></span> ) of two groups, A (with size <em>n</em><sub>A</sub>) and B (with size <em>n</em><sub>B</sub>), are significantly different, we performed a permutation test with 10<sup>5</sup> iterations. The null hypothesis states that there is no difference between the sample means of the two groups. First, we calculated the absolute observed mean difference: <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m69" display="inline" overflow="linebreak"><mrow><msub><mi>T</mi><mtext>obs</mtext></msub><mo>=</mo><mrow><mo>∣</mo><msub><mi>x</mi><mi mathvariant="normal">A</mi></msub><mo>−</mo><msub><mi>x</mi><mi mathvariant="normal">B</mi></msub><mspace width="-.15em"></mspace><mo>∣</mo></mrow></mrow></math></span> . Next, we pooled all the data from both groups and randomly selected, without replacement, <em>n</em><sub>A</sub> values for group A and <em>n</em><sub>B</sub> values for group B to form two new resampled groups. We then computed the absolute mean difference for these resampled groups. This process was repeated for 10<sup>5</sup> iterations, generating a distribution of mean differences under the null hypothesis.</p>
<p>The two-tailed <em>P</em> value was calculated as the proportion of times the absolute resampled mean difference was equal or more extreme than the absolute observed difference ( <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m70" display="inline" overflow="linebreak"><mrow><msub><mi>T</mi><mtext>obs</mtext></msub></mrow></math></span> ). For multiple test comparison, we applied Benjamini-Hochberg correction to <em>P</em> values. If the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m71" display="inline" overflow="linebreak"><mrow><mtext>adj</mtext><mo>.</mo><mi>P</mi><mspace width="0.25em"></mspace><mtext>value</mtext><mo>≤</mo><mn>0.05</mn></mrow></math></span> , we reject the null hypothesis and accept the alternative hypothesis suggesting a statistically significant difference between the sample means of the two groups.</p></section><section id="sec25"><h3 class="pmc_sec_title">Supporting methods for LiP-MS and XL-MS experiments</h3>
<p>Additional methods related to LiP-MS and XL-MS experiments can be found in the Supplementary Materials.</p></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank D. E. Shaw’s group for providing access to molecular dynamics trajectories for ubiquitin and λ-repressor. This research was supported in part by the TASK Supercomputer Center in Gdansk, Poland, and PLGrid Infrastructure in Poland (Grant ID: plgribliq), as well as Roar supercomputer in the US.</p>
<section id="sec26"><p><strong>Funding:</strong> This work was supported by the following: National Science Centre, Poland, grant 2019/35/B/ST4/02086 (to M.S.L.), National Science Foundation grant MCB-2045844 (to S.D.F.), National Institutes of Health grant DP2-GM140926 (to S.D.F.), National Science Foundation grant MCB-1553291 (to E.P.O.), and National Institutes of Health grant R35-GM124818 (to E.P.O.).</p></section><section id="sec27"><p><strong>Author contributions:</strong> Conceptualization: S.D.F. and E.P.O. Methodology: Q.V.V., I.S., Y.J., Y.X., H.S., M.S.L., S.D.F., and E.P.O. Investigation: Q.V.V., I.S., Y.J., Y.X., P.S., D.Y., M.S.L., S.D.F., and E.P.O. Resources: Q.V.V., Y.J., Y.X., P.S., M.S.L., S.D.F., and E.P.O. Data curation: Q.V.V., Y.J., and Y.X. Validation: Q.V.V., I.S., Y.J., and Y.X. Formal analysis: Q.V.V., I.S., Y.J., and Y.X. Software: Q.V.V., I.S., and Y.J. Visualization: Q.V.V., I.S., and Y.J. Funding acquisition: M.S.L., S.D.F., and E.P.O. Project administration: S.D.F. and E.P.O. Supervision: S.D.F. and E.P.O. Writing—original draft: Q.V.V. and E.P.O. Writing—review and editing: Q.V.V., I.S., Y.J., Y.X., P.S., D.Y., H.S., M.S.L., S.D.F., and E.P.O.</p></section><section id="sec28"><p><strong>Competing interests:</strong> The authors declare that they have no competing interests.</p></section><section id="sec29"><p><strong>Data and materials availability:</strong> All data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials. LiP-MS and XL-MS data and comparison between misfolded clusters and the native cluster from simulations based on experimental signals are provided in data S1. Misfolded structures of IspE are provided in data S2. The computer code for coarse-grained simulations and entanglement analysis is available at <a href="https://github.com/obrien-lab/cg_simtk_protein_folding" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/obrien-lab/cg_simtk_protein_folding</a>. Additional analyses and data are available at <a href="https://github.com/obrien-lab/All_Atom_Misfolding_Exp_Ensemble" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/obrien-lab/All_Atom_Misfolding_Exp_Ensemble</a> or <a href="https://zenodo.org/records/15412020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/15412020</a>. Interactive visualizations of structures in <a href="#F3" class="usa-link">Fig. 3B</a> are available at <a href="https://obrien-lab.github.io/All_Atom_Misfolding_Exp_Ensemble/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://obrien-lab.github.io/All_Atom_Misfolding_Exp_Ensemble/</a>. Software used in experimental data analysis is available at <a href="https://github.com/FriedLabJHU/Refoldability-Tools" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/FriedLabJHU/Refoldability-Tools</a>. The mass spectrometry data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifiers PXD055795 for LiP-MS and PXD066083 for XL-MS.</p></section></section><section id="sec30"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section id="sec31"><h3 class="pmc_sec_title">The PDF file includes:</h3>
<section class="sm xbox font-sm" id="supplementary-material1"><div class="caption p">
<p>Supplementary Text</p>
<p>Figs. S1 to S7</p>
<p>Tables S1 to S6</p>
<p>Legends for data S1 and S2</p>
<p>References</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333692/bin/sciadv.adt8974_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adt8974_sm.pdf</a><sup> (2.2MB, pdf) </sup>
</div></div></section></section><section id="sec32"><h3 class="pmc_sec_title">Other Supplementary Material for this manuscript includes the following:</h3>
<section class="sm xbox font-sm" id="supplementary-material2"><div class="caption p"><p>Data S1 and S2</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333692/bin/sciadv.adt8974_data_s1_and_s2.zip" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adt8974_data_s1_and_s2.zip</a><sup> (9.5MB, zip) </sup>
</div></div></section></section></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">REFERENCES AND NOTES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Jiang Y., Neti S. S., Sitarik I., Pradhan P., To P., Xia Y., Fried S. D., Booker S. J., O’Brien E. P., 
How synonymous mutations alter enzyme structure and function over long timescales. Nat. Chem.
15, 
308–318 (2023).
</cite> [<a href="https://doi.org/10.1038/s41557-022-01091-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11267483/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36471044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Chem.&amp;title=How%20synonymous%20mutations%20alter%20enzyme%20structure%20and%20function%20over%20long%20timescales&amp;volume=15&amp;publication_year=2023&amp;pages=308-318&amp;pmid=36471044&amp;doi=10.1038/s41557-022-01091-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Nissley D. A., Jiang Y., Trovato F., Sitarik I., Narayan K. B., To P., Xia Y., Fried S. D., O’Brien E. P., 
Universal protein misfolding intermediates can bypass the proteostasis network and remain soluble and less functional. Nat. Commun.
13, 
3081 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-30548-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9163053/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35654797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Universal%20protein%20misfolding%20intermediates%20can%20bypass%20the%20proteostasis%20network%20and%20remain%20soluble%20and%20less%20functional&amp;volume=13&amp;publication_year=2022&amp;pages=3081&amp;pmid=35654797&amp;doi=10.1038/s41467-022-30548-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Halder R., Nissley D. A., Sitarik I., Jiang Y., Rao Y., Vu Q. V., Li M. S., Pritchard J., O’Brien E. P., 
How soluble misfolded proteins bypass chaperones at the molecular level. Nat. Commun.
14, 
3689 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-38962-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10284856/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37344452/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=How%20soluble%20misfolded%20proteins%20bypass%20chaperones%20at%20the%20molecular%20level&amp;volume=14&amp;publication_year=2023&amp;pages=3689&amp;pmid=37344452&amp;doi=10.1038/s41467-023-38962-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Baiesi M., Orlandini E., Trovato A., Seno F., 
Linking in domain-swapped protein dimers. Sci. Rep.
6, 
33872 (2016).
</cite> [<a href="https://doi.org/10.1038/srep33872" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5034241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27659606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Linking%20in%20domain-swapped%20protein%20dimers&amp;volume=6&amp;publication_year=2016&amp;pages=33872&amp;pmid=27659606&amp;doi=10.1038/srep33872&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Baiesi M., Orlandini E., Seno F., Trovato A., 
Exploring the correlation between the folding rates of proteins and the entanglement of their native states. J. Phys. A. Math. Theor.
50, 
504001 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20A.%20Math.%20Theor.&amp;title=Exploring%20the%20correlation%20between%20the%20folding%20rates%20of%20proteins%20and%20the%20entanglement%20of%20their%20native%20states&amp;volume=50&amp;publication_year=2017&amp;pages=504001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Baiesi M., Orlandini E., Seno F., Trovato A., 
Sequence and structural patterns detected in entangled proteins reveal the importance of co-translational folding. Sci. Rep.
9, 
8426 (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-44928-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6557820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31182755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Sequence%20and%20structural%20patterns%20detected%20in%20entangled%20proteins%20reveal%20the%20importance%20of%20co-translational%20folding&amp;volume=9&amp;publication_year=2019&amp;pages=8426&amp;pmid=31182755&amp;doi=10.1038/s41598-019-44928-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>Rana V., Sitarik I., Petucci J., Jiang Y., Song H., O’Brien E. P., 
Non-covalent lasso entanglements in folded proteins: Prevalence, functional implications, and evolutionary significance. J. Mol. Biol.
436, 
168459 (2024).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2024.168459" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11265471/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38296158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Non-covalent%20lasso%20entanglements%20in%20folded%20proteins:%20Prevalence,%20functional%20implications,%20and%20evolutionary%20significance&amp;volume=436&amp;publication_year=2024&amp;pages=168459&amp;pmid=38296158&amp;doi=10.1016/j.jmb.2024.168459&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Komar A. A., Lesnik T., Reiss C., 
Synonymous codon substitutions affect ribosome traffic and protein folding during in vitro translation. FEBS Lett.
462, 
387–391 (1999).
</cite> [<a href="https://doi.org/10.1016/s0014-5793(99)01566-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10622731/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett.&amp;title=Synonymous%20codon%20substitutions%20affect%20ribosome%20traffic%20and%20protein%20folding%20during%20in%20vitro%20translation&amp;volume=462&amp;publication_year=1999&amp;pages=387-391&amp;pmid=10622731&amp;doi=10.1016/s0014-5793(99)01566-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Zhou M., Guo J., Cha J., Chae M., Chen S., Barral J. M., Sachs M. S., Liu Y., 
Non-optimal codon usage affects expression, structure and function of clock protein FRQ. Nature
494, 
111–115 (2013).
</cite> [<a href="https://doi.org/10.1038/nature11833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3629845/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23417067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Non-optimal%20codon%20usage%20affects%20expression,%20structure%20and%20function%20of%20clock%20protein%20FRQ&amp;volume=494&amp;publication_year=2013&amp;pages=111-115&amp;pmid=23417067&amp;doi=10.1038/nature11833&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Zhou M., Wang T., Fu J., Xiao G., Liu Y., 
Nonoptimal codon usage influences protein structure in intrinsically disordered regions. Mol. Microbiol.
97, 
974–987 (2015).
</cite> [<a href="https://doi.org/10.1111/mmi.13079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4636118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26032251/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&amp;title=Nonoptimal%20codon%20usage%20influences%20protein%20structure%20in%20intrinsically%20disordered%20regions&amp;volume=97&amp;publication_year=2015&amp;pages=974-987&amp;pmid=26032251&amp;doi=10.1111/mmi.13079&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Nissley D. A., Vu Q. V., Trovato F., Ahmed N., Jiang Y., Li M. S., O’Brien E. P., 
Electrostatic interactions govern extreme nascent protein ejection times from ribosomes and can delay ribosome recycling. J. Am. Chem. Soc.
142, 
6103–6110 (2020).
</cite> [<a href="https://doi.org/10.1021/jacs.9b12264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7312765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32138505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Electrostatic%20interactions%20govern%20extreme%20nascent%20protein%20ejection%20times%20from%20ribosomes%20and%20can%20delay%20ribosome%20recycling&amp;volume=142&amp;publication_year=2020&amp;pages=6103-6110&amp;pmid=32138505&amp;doi=10.1021/jacs.9b12264&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Piana S., Lindorff-Larsen K., Shaw D. E., 
Atomic-level description of ubiquitin folding. Proc. Natl. Acad. Sci. U.S.A.
110, 
5915–5920 (2013).
</cite> [<a href="https://doi.org/10.1073/pnas.1218321110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3625349/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23503848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Atomic-level%20description%20of%20ubiquitin%20folding&amp;volume=110&amp;publication_year=2013&amp;pages=5915-5920&amp;pmid=23503848&amp;doi=10.1073/pnas.1218321110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>Lindorff-Larsen K., Piana S., Dror R. O., Shaw D. E., 
How fast-folding proteins fold. Science
334, 
517–520 (2011).
</cite> [<a href="https://doi.org/10.1126/science.1208351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22034434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=How%20fast-folding%20proteins%20fold&amp;volume=334&amp;publication_year=2011&amp;pages=517-520&amp;pmid=22034434&amp;doi=10.1126/science.1208351&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Sivaraman T., Arrington C. B., Robertson A. D., 
Kinetics of unfolding and folding from amide hydrogen exchange in native ubiquitin. Nat. Struct. Biol.
8, 
331–333 (2001).
</cite> [<a href="https://doi.org/10.1038/86208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11276253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Struct.%20Biol.&amp;title=Kinetics%20of%20unfolding%20and%20folding%20from%20amide%20hydrogen%20exchange%20in%20native%20ubiquitin&amp;volume=8&amp;publication_year=2001&amp;pages=331-333&amp;pmid=11276253&amp;doi=10.1038/86208&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Yang W. Y., Gruebele M., 
Folding at the speed limit. Nature
423, 
193–197 (2003).
</cite> [<a href="https://doi.org/10.1038/nature01609" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12736690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Folding%20at%20the%20speed%20limit&amp;volume=423&amp;publication_year=2003&amp;pages=193-197&amp;pmid=12736690&amp;doi=10.1038/nature01609&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Frishman D., Argos P., 
Knowledge-based protein secondary structure assignment. Proteins Struc. Func. Genet.
23, 
566–579 (1995).</cite> [<a href="https://doi.org/10.1002/prot.340230412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8749853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins%20Struc.%20Func.%20Genet.&amp;title=Knowledge-based%20protein%20secondary%20structure%20assignment&amp;volume=23&amp;publication_year=1995&amp;pages=566-579&amp;pmid=8749853&amp;doi=10.1002/prot.340230412&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Maxwell K. L., Wildes D., Zarrine-Afsar A., De Los Rios M. A., Brown A. G., Friel C. T., Hedberg L., Horng J.-C., Bona D., Miller E. J., Vallée-Bélisle A., Main E. R. G., Bemporad F., Qiu L., Teilum K., Vu N.-D., Edwards A. M., Ruczinski I., Poulsen F. M., Kragelund B. B., Michnick S. W., Chiti F., Bai Y., Hagen S. J., Serrano L., Oliveberg M., Raleigh D. P., Wittung-Stafshede P., Radford S. E., Jackson S. E., Sosnick T. R., Marqusee S., Davidson A. R., Plaxco K. W., 
Protein folding: Defining a “standard” set of experimental conditions and a preliminary kinetic data set of two-state proteins. Protein Sci.
14, 
602–616 (2005).
</cite> [<a href="https://doi.org/10.1110/ps.041205405" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2279278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15689503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&amp;title=Protein%20folding:%20Defining%20a%20%E2%80%9Cstandard%E2%80%9D%20set%20of%20experimental%20conditions%20and%20a%20preliminary%20kinetic%20data%20set%20of%20two-state%20proteins&amp;volume=14&amp;publication_year=2005&amp;pages=602-616&amp;pmid=15689503&amp;doi=10.1110/ps.041205405&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>Braselmann E., Chaney J. L., Clark P. L., 
Folding the proteome. Trends Biochem. Sci.
38, 
337–344 (2013).
</cite> [<a href="https://doi.org/10.1016/j.tibs.2013.05.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3691291/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23764454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem.%20Sci.&amp;title=Folding%20the%20proteome&amp;volume=38&amp;publication_year=2013&amp;pages=337-344&amp;pmid=23764454&amp;doi=10.1016/j.tibs.2013.05.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Brocchieri L., Karlin S., 
Protein length in eukaryotic and prokaryotic proteomes. Nucleic Acids Res.
33, 
3390–3400 (2005).
</cite> [<a href="https://doi.org/10.1093/nar/gki615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1150220/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15951512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Protein%20length%20in%20eukaryotic%20and%20prokaryotic%20proteomes&amp;volume=33&amp;publication_year=2005&amp;pages=3390-3400&amp;pmid=15951512&amp;doi=10.1093/nar/gki615&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Sarkar D., Kang P., Nielsen S. O., Qin Z., 
Non-arrhenius reaction-diffusion kinetics for protein inactivation over a large temperature range. ACS Nano
13, 
8669–8679 (2019).
</cite> [<a href="https://doi.org/10.1021/acsnano.9b00068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7384293/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31268674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Nano&amp;title=Non-arrhenius%20reaction-diffusion%20kinetics%20for%20protein%20inactivation%20over%20a%20large%20temperature%20range&amp;volume=13&amp;publication_year=2019&amp;pages=8669-8679&amp;pmid=31268674&amp;doi=10.1021/acsnano.9b00068&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Feng Y., De Franceschi G., Kahraman A., Soste M., Melnik A., Boersema P. J., de Laureto P. P., Nikolaev Y., Oliveira A. P., Picotti P., 
Global analysis of protein structural changes in complex proteomes. Nat. Biotechnol.
32, 
1036–1044 (2014).
</cite> [<a href="https://doi.org/10.1038/nbt.2999" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25218519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;title=Global%20analysis%20of%20protein%20structural%20changes%20in%20complex%20proteomes&amp;volume=32&amp;publication_year=2014&amp;pages=1036-1044&amp;pmid=25218519&amp;doi=10.1038/nbt.2999&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>P. To, Whitehead B., Tarbox H. E., Fried S. D., 
Nonrefoldability is pervasive across the <em>E. coli</em> proteome. J. Am. Chem. Soc.
143, 
11435–11448 (2021).
</cite> [<a href="https://doi.org/10.1021/jacs.1c03270" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8650709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34308638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Nonrefoldability%20is%20pervasive%20across%20the%20E.%20coli%20proteome&amp;volume=143&amp;publication_year=2021&amp;pages=11435-11448&amp;pmid=34308638&amp;doi=10.1021/jacs.1c03270&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Schopper S., Kahraman A., Leuenberger P., Feng Y., Piazza I., Müller O., Boersema P. J., Picotti P., 
Measuring protein structural changes on a proteome-wide scale using limited proteolysis-coupled mass spectrometry. Nat. Protoc.
12, 
2391–2410 (2017).
</cite> [<a href="https://doi.org/10.1038/nprot.2017.100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29072706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Measuring%20protein%20structural%20changes%20on%20a%20proteome-wide%20scale%20using%20limited%20proteolysis-coupled%20mass%20spectrometry&amp;volume=12&amp;publication_year=2017&amp;pages=2391-2410&amp;pmid=29072706&amp;doi=10.1038/nprot.2017.100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Malinovska L., Cappelletti V., Kohler D., Piazza I., Tsai T.-H., Pepelnjak M., Stalder P., Dörig C., Sesterhenn F., Elsässer F., Kralickova L., Beaton N., Reiter L., de Souza N., Vitek O., Picotti P., 
Proteome-wide structural changes measured with limited proteolysis-mass spectrometry: An advanced protocol for high-throughput applications. Nat. Protoc.
18, 
659–682 (2023).
</cite> [<a href="https://doi.org/10.1038/s41596-022-00771-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36526727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Proteome-wide%20structural%20changes%20measured%20with%20limited%20proteolysis-mass%20spectrometry:%20An%20advanced%20protocol%20for%20high-throughput%20applications&amp;volume=18&amp;publication_year=2023&amp;pages=659-682&amp;pmid=36526727&amp;doi=10.1038/s41596-022-00771-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>O’Reilly F. J., Rappsilber J., 
Cross-linking mass spectrometry: Methods and applications in structural, molecular and systems biology. Nat. Struct. Mol. Biol.
25, 
1000–1008 (2018).
</cite> [<a href="https://doi.org/10.1038/s41594-018-0147-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30374081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Struct.%20Mol.%20Biol.&amp;title=Cross-linking%20mass%20spectrometry:%20Methods%20and%20applications%20in%20structural,%20molecular%20and%20systems%20biology&amp;volume=25&amp;publication_year=2018&amp;pages=1000-1008&amp;pmid=30374081&amp;doi=10.1038/s41594-018-0147-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>Pan D., Brockmeyer A., Mueller F., Musacchio A., Bange T., 
Simplified protocol for cross-linking mass spectrometry using the MS-cleavable cross-linker DSBU with efficient cross-link identification. Anal. Chem.
90, 
10990–10999 (2018).
</cite> [<a href="https://doi.org/10.1021/acs.analchem.8b02593" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30074391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=Simplified%20protocol%20for%20cross-linking%20mass%20spectrometry%20using%20the%20MS-cleavable%20cross-linker%20DSBU%20with%20efficient%20cross-link%20identification&amp;volume=90&amp;publication_year=2018&amp;pages=10990-10999&amp;pmid=30074391&amp;doi=10.1021/acs.analchem.8b02593&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Herzog F., Kahraman A., Boehringer D., Mak R., Bracher A., Walzthoeni T., Leitner A., Beck M., Hartl F.-U., Ban N., Malmström L., Aebersold R., 
Structural probing of a protein phosphatase 2A network by chemical cross-linking and mass spectrometry. Science
337, 
1348–1352 (2012).
</cite> [<a href="https://doi.org/10.1126/science.1221483" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22984071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Structural%20probing%20of%20a%20protein%20phosphatase%202A%20network%20by%20chemical%20cross-linking%20and%20mass%20spectrometry&amp;volume=337&amp;publication_year=2012&amp;pages=1348-1352&amp;pmid=22984071&amp;doi=10.1126/science.1221483&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>To P., Xia Y., Lee S. O., Devlin T., Fleming K. G., Fried S. D., 
A proteome-wide map of chaperone-assisted protein refolding in a cytosol-like milieu. Proc. Natl. Acad. Sci. U.S.A.
119, 
e2210536119 (2022).
</cite> [<a href="https://doi.org/10.1073/pnas.2210536119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9860312/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36417429/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=A%20proteome-wide%20map%20of%20chaperone-assisted%20protein%20refolding%20in%20a%20cytosol-like%20milieu&amp;volume=119&amp;publication_year=2022&amp;pages=e2210536119&amp;pmid=36417429&amp;doi=10.1073/pnas.2210536119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Benjamini Y., Hochberg Y., 
Controlling the false discovery rate: A practical and powerful approach to multiple testing. J. R. Stat. Soc. Series B Stat. Methodol.
57, 
289–300 (1995).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20R.%20Stat.%20Soc.%20Series%20B%20Stat.%20Methodol.&amp;title=Controlling%20the%20false%20discovery%20rate:%20A%20practical%20and%20powerful%20approach%20to%20multiple%20testing&amp;volume=57&amp;publication_year=1995&amp;pages=289-300&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>Bullock J. M. A., Schwab J., Thalassinos K., Topf M., 
The importance of non-accessible crosslinks and solvent accessible surface distance in modeling proteins with restraints from crosslinking mass spectrometry. Mol. Cell. Proteomics
15, 
2491–2500 (2016).
</cite> [<a href="https://doi.org/10.1074/mcp.M116.058560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4937519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27150526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=The%20importance%20of%20non-accessible%20crosslinks%20and%20solvent%20accessible%20surface%20distance%20in%20modeling%20proteins%20with%20restraints%20from%20crosslinking%20mass%20spectrometry&amp;volume=15&amp;publication_year=2016&amp;pages=2491-2500&amp;pmid=27150526&amp;doi=10.1074/mcp.M116.058560&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Tsolis A. C., Papandreou N. C., Iconomidou V. A., Hamodrakas S. J., 
A consensus method for the prediction of “Aggregation-Prone” peptides in globular proteins. PLOS ONE
8, 
e54175 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0054175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3542318/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23326595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&amp;title=A%20consensus%20method%20for%20the%20prediction%20of%20%E2%80%9CAggregation-Prone%E2%80%9D%20peptides%20in%20globular%20proteins&amp;volume=8&amp;publication_year=2013&amp;pages=e54175&amp;pmid=23326595&amp;doi=10.1371/journal.pone.0054175&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Sułkowska J. I., Noel J. K., Onuchic J. N., 
Energy landscape of knotted protein folding. Proc. Natl. Acad. Sci. U.S.A.
109, 
17783–17788 (2012).
</cite> [<a href="https://doi.org/10.1073/pnas.1201804109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3497829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22891304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Energy%20landscape%20of%20knotted%20protein%20folding&amp;volume=109&amp;publication_year=2012&amp;pages=17783-17788&amp;pmid=22891304&amp;doi=10.1073/pnas.1201804109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Salicari L., Baiesi M., Orlandini E., Trovato A., 
Folding kinetics of an entangled protein. PLOS Comput. Biol.
19, 
e1011107 (2023).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1011107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10681328/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37956216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Comput.%20Biol.&amp;title=Folding%20kinetics%20of%20an%20entangled%20protein&amp;volume=19&amp;publication_year=2023&amp;pages=e1011107&amp;pmid=37956216&amp;doi=10.1371/journal.pcbi.1011107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Guo Z., Thirumalai D., 
Kinetics of protein folding: Nucleation mechanism, time scales, and pathways. Biopolymers
36, 
83–102 (1995).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Biopolymers&amp;title=Kinetics%20of%20protein%20folding:%20Nucleation%20mechanism,%20time%20scales,%20and%20pathways&amp;volume=36&amp;publication_year=1995&amp;pages=83-102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Thirumalai D., Woodson S. A., 
Kinetics of folding of proteins and RNA. Acc. Chem. Res.
29, 
433–439 (1996).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Acc.%20Chem.%20Res.&amp;title=Kinetics%20of%20folding%20of%20proteins%20and%20RNA&amp;volume=29&amp;publication_year=1996&amp;pages=433-439&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>Thirumalai D., Klimov D. K., Woodson S. A., 
Kinetic partitioning mechanism as a unifying theme in the folding of biomolecules. Theor. Chem. Acta
96, 
14–22 (1997).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Theor.%20Chem.%20Acta&amp;title=Kinetic%20partitioning%20mechanism%20as%20a%20unifying%20theme%20in%20the%20folding%20of%20biomolecules&amp;volume=96&amp;publication_year=1997&amp;pages=14-22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Jorgensen W. L., Chandrasekhar J., Madura J. D., Impey R. W., Klein M. L., 
Comparison of simple potential functions for simulating liquid water. J. Chem. Phys.
79, 
926–935 (1983).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Phys.&amp;title=Comparison%20of%20simple%20potential%20functions%20for%20simulating%20liquid%20water&amp;volume=79&amp;publication_year=1983&amp;pages=926-935&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Bryson A. E., Denham W. F., 
A steepest-ascent method for solving optimum programming problems. J. Appl. Mech.
29, 
247–257 (1962).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Mech.&amp;title=A%20steepest-ascent%20method%20for%20solving%20optimum%20programming%20problems&amp;volume=29&amp;publication_year=1962&amp;pages=247-257&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Haug E. J., Arora J. S., Matsui K., 
A steepest-descent method for optimization of mechanical systems. J. Optim. Theory Appl.
19, 
401–424 (1976).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Optim.%20Theory%20Appl.&amp;title=A%20steepest-descent%20method%20for%20optimization%20of%20mechanical%20systems&amp;volume=19&amp;publication_year=1976&amp;pages=401-424&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Darden T., York D., Pedersen L., 
Particle mesh Ewald: An N·log(N) method for Ewald sums in large systems. J. Chem. Phys.
98, 
10089–10092 (1993).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Phys.&amp;title=Particle%20mesh%20Ewald:%20An%20N%C2%B7log(N)%20method%20for%20Ewald%20sums%20in%20large%20systems&amp;volume=98&amp;publication_year=1993&amp;pages=10089-10092&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Nosé S., 
A unified formulation of the constant temperature molecular dynamics methods. J. Chem. Phys.
81, 
511–519 (1984).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Phys.&amp;title=A%20unified%20formulation%20of%20the%20constant%20temperature%20molecular%20dynamics%20methods&amp;volume=81&amp;publication_year=1984&amp;pages=511-519&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Hoover W. G., 
Canonical dynamics: Equilibrium phase-space distributions. Phys. Rev. A. (Coll Park)
31, 
1695–1697 (1985).</cite> [<a href="https://doi.org/10.1103/physreva.31.1695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9895674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phys.%20Rev.%20A.%20(Coll%20Park)&amp;title=Canonical%20dynamics:%20Equilibrium%20phase-space%20distributions&amp;volume=31&amp;publication_year=1985&amp;pages=1695-1697&amp;pmid=9895674&amp;doi=10.1103/physreva.31.1695&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>Parrinello M., Rahman A., 
Polymorphic transitions in single crystals: A new molecular dynamics method. J. Appl. Phys.
52, 
7182–7190 (1981).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Phys.&amp;title=Polymorphic%20transitions%20in%20single%20crystals:%20A%20new%20molecular%20dynamics%20method&amp;volume=52&amp;publication_year=1981&amp;pages=7182-7190&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>Hess B., Bekker H., Berendsen H. J. C., Fraaije J. G. E. M., 
LINCS: A linear constraint solver for molecular simulations. J. Comput. Chem.
18, 
1463–1472 (1997).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=LINCS:%20A%20linear%20constraint%20solver%20for%20molecular%20simulations&amp;volume=18&amp;publication_year=1997&amp;pages=1463-1472&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>Abraham M. J., Murtola T., Schulz R., Páll S., Smith J. C., Hess B., Lindah E., 
Gromacs: High performance molecular simulations through multi-level parallelism from laptops to supercomputers. SoftwareX
1–2, 
19–25 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=SoftwareX&amp;title=Gromacs:%20High%20performance%20molecular%20simulations%20through%20multi-level%20parallelism%20from%20laptops%20to%20supercomputers&amp;volume=1%E2%80%932&amp;publication_year=2015&amp;pages=19-25&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Huang J., Rauscher S., Nawrocki G., Ran T., Feig M., De Groot B. L., Grubmüller H., MacKerell A. D., 
CHARMM36m: An improved force field for folded and intrinsically disordered proteins. Nat. Methods
14, 
71–73 (2016).
</cite> [<a href="https://doi.org/10.1038/nmeth.4067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5199616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27819658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=CHARMM36m:%20An%20improved%20force%20field%20for%20folded%20and%20intrinsically%20disordered%20proteins&amp;volume=14&amp;publication_year=2016&amp;pages=71-73&amp;pmid=27819658&amp;doi=10.1038/nmeth.4067&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Niemyska W., Millett K. C., Sulkowska J. I., 
GLN: A method to reveal unique properties of lasso type topology in proteins. Sci. Rep.
10, 
15186 (2020).
</cite> [<a href="https://doi.org/10.1038/s41598-020-71874-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7494857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32938999/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=GLN:%20A%20method%20to%20reveal%20unique%20properties%20of%20lasso%20type%20topology%20in%20proteins&amp;volume=10&amp;publication_year=2020&amp;pages=15186&amp;pmid=32938999&amp;doi=10.1038/s41598-020-71874-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Gutierres M. B. B., Bonorino C. B. C., Rigo M. M., 
ChaperISM: Improved chaperone binding prediction using position-independent scoring matrices. Bioinformatics
36, 
735–741 (2020).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31504177/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=ChaperISM:%20Improved%20chaperone%20binding%20prediction%20using%20position-independent%20scoring%20matrices&amp;volume=36&amp;publication_year=2020&amp;pages=735-741&amp;pmid=31504177&amp;doi=10.1093/bioinformatics/btz670&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Eastman P., Swails J., Chodera J. D., McGibbon R. T., Zhao Y., Beauchamp K. A., Wang L. P., Simmonett A. C., Harrigan M. P., Stern C. D., Wiewiora R. P., Brooks B. R., Pande V. S., 
OpenMM 7: Rapid development of high performance algorithms for molecular dynamics. PLOS Comput. Biol.
13, 
e1005659 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1005659" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5549999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28746339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Comput.%20Biol.&amp;title=OpenMM%207:%20Rapid%20development%20of%20high%20performance%20algorithms%20for%20molecular%20dynamics&amp;volume=13&amp;publication_year=2017&amp;pages=e1005659&amp;pmid=28746339&amp;doi=10.1371/journal.pcbi.1005659&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>D. Arthur, S. Vassilvitskii, “K-means++: The advantages of careful seeding,” in <em>Proceedings of the Annual ACM-SIAM Symposium on Discrete Algorithms</em> (Society for Industrial and Applied Mathematics, 2007), pp. 1027–1035.</cite>
</li>
<li id="R51">
<span class="label">51.</span><cite>Röblitz S., Weber M., 
Fuzzy spectral clustering by PCCA+: Application to Markov state models and data classification. Adv. Data Anal. Classif.
7, 
147–179 (2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Data%20Anal.%20Classif.&amp;title=Fuzzy%20spectral%20clustering%20by%20PCCA+:%20Application%20to%20Markov%20state%20models%20and%20data%20classification&amp;volume=7&amp;publication_year=2013&amp;pages=147-179&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>Prinz J.-H., Wu H., Sarich M., Keller B., Senne M., Held M., Chodera J. D., Schütte C., Noé F., 
Markov models of molecular kinetics: Generation and validation. J. Chem. Phys.
134, 
174105 (2011).
</cite> [<a href="https://doi.org/10.1063/1.3565032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21548671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Phys.&amp;title=Markov%20models%20of%20molecular%20kinetics:%20Generation%20and%20validation&amp;volume=134&amp;publication_year=2011&amp;pages=174105&amp;pmid=21548671&amp;doi=10.1063/1.3565032&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Scherer M. K., Trendelkamp-Schroer B., Paul F., Pérez-Hernández G., Hoffmann M., Plattner N., Wehmeyer C., Prinz J. H., Noé F., 
PyEMMA 2: A software package for estimation, validation, and analysis of markov models. J. Chem. Theory Comput.
11, 
5525–5542 (2015).
</cite> [<a href="https://doi.org/10.1021/acs.jctc.5b00743" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26574340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=PyEMMA%202:%20A%20software%20package%20for%20estimation,%20validation,%20and%20analysis%20of%20markov%20models&amp;volume=11&amp;publication_year=2015&amp;pages=5525-5542&amp;pmid=26574340&amp;doi=10.1021/acs.jctc.5b00743&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>Onuchic J. N., Luthey-Schulten Z., Wolynes P. G., 
THEORY OF PROTEIN FOLDING: The energy landscape perspective. Annu. Rev. Phys. Chem.
48, 
545–600 (1997).
</cite> [<a href="https://doi.org/10.1146/annurev.physchem.48.1.545" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9348663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Phys.%20Chem.&amp;title=THEORY%20OF%20PROTEIN%20FOLDING:%20The%20energy%20landscape%20perspective&amp;volume=48&amp;publication_year=1997&amp;pages=545-600&amp;pmid=9348663&amp;doi=10.1146/annurev.physchem.48.1.545&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R55">
<span class="label">55.</span><cite>Jiang Y., Xia Y., Sitarik I., Sharma P., Song H., Fried S. D., O’Brien E. P., 
Protein misfolding involving entanglements providesa structural explanation for the origin of stretched-exponential refolding kinetics. Sci. Adv.
11, 
eads7379 (2025).
</cite> [<a href="https://doi.org/10.1126/sciadv.ads7379" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11908495/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40085700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&amp;title=Protein%20misfolding%20involving%20entanglements%20providesa%20structural%20explanation%20for%20the%20origin%20of%20stretched-exponential%20refolding%20kinetics&amp;volume=11&amp;publication_year=2025&amp;pages=eads7379&amp;pmid=40085700&amp;doi=10.1126/sciadv.ads7379&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R56">
<span class="label">56.</span><cite>Gosavi S., Chavez L. L., Jennings P. A., Onuchic J. N., 
Topological frustration and the folding of interleukin-1β. J. Mol. Biol.
357, 
986–996 (2006).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2005.11.074" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16469330/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Topological%20frustration%20and%20the%20folding%20of%20interleukin-1%CE%B2&amp;volume=357&amp;publication_year=2006&amp;pages=986-996&amp;pmid=16469330&amp;doi=10.1016/j.jmb.2005.11.074&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R57">
<span class="label">57.</span><cite>Sherman M. Y., Qian S. B., 
Less is more: Improving proteostasis by translation slow down. Trends Biochem. Sci.
38, 
585–591 (2013).
</cite> [<a href="https://doi.org/10.1016/j.tibs.2013.09.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24126073/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem.%20Sci.&amp;title=Less%20is%20more:%20Improving%20proteostasis%20by%20translation%20slow%20down&amp;volume=38&amp;publication_year=2013&amp;pages=585-591&amp;pmid=24126073&amp;doi=10.1016/j.tibs.2013.09.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R58">
<span class="label">58.</span><cite>Kiefhaber T., Kohler H. H., Schmid F. X., 
Kinetic coupling between protein folding and prolyl isomerization. I. Theoretical models. J. Mol. Biol.
224, 
217–229 (1992).
</cite> [<a href="https://doi.org/10.1016/0022-2836(92)90585-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1548700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Kinetic%20coupling%20between%20protein%20folding%20and%20prolyl%20isomerization.%20I.%20Theoretical%20models&amp;volume=224&amp;publication_year=1992&amp;pages=217-229&amp;pmid=1548700&amp;doi=10.1016/0022-2836(92)90585-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R59">
<span class="label">59.</span><cite>E. A. Kikis, “The intrinsic and extrinsic factors that contribute to proteostasis decline and pathological protein misfolding” in <a href="https://www.sciencedirect.com/bookseries/advances-in-protein-chemistry-and-structural-biology" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer"><em>Advances in Protein Chemistry and Structural Biology</em></a> (Elsevier Ltd., 2019), vol. 118.</cite> [<a href="https://doi.org/10.1016/bs.apcsb.2019.07.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31928724/" class="usa-link">PubMed</a>]</li>
<li id="R60">
<span class="label">60.</span><cite>Salahuddin P., Siddiqi M. K., Khan S., Abdelhameed A. S., Khan R. H., 
Mechanisms of protein misfolding: Novel therapeutic approaches to protein-misfolding diseases. J. Mol. Struct.
1123, 
311–326 (2016).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Struct.&amp;title=Mechanisms%20of%20protein%20misfolding:%20Novel%20therapeutic%20approaches%20to%20protein-misfolding%20diseases&amp;volume=1123&amp;publication_year=2016&amp;pages=311-326&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R61">
<span class="label">61.</span><cite>Lafita A., Tian P., Best R. B., Bateman A., 
Tandem domain swapping: Determinants of multidomain protein misfolding. Curr. Opin. Struct. Biol.
58, 
97–104 (2019).
</cite> [<a href="https://doi.org/10.1016/j.sbi.2019.05.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6863430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31260947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Struct.%20Biol.&amp;title=Tandem%20domain%20swapping:%20Determinants%20of%20multidomain%20protein%20misfolding&amp;volume=58&amp;publication_year=2019&amp;pages=97-104&amp;pmid=31260947&amp;doi=10.1016/j.sbi.2019.05.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R62">
<span class="label">62.</span><cite>Yu F., Haynes S. E., Teo G. C., Avtonomov D. M., Polasky D. A., Nesvizhskii A. I., 
Fast quantitative analysis of timsTOF PASEF data with MSFragger and IonQuant. Mol. Cell. Proteomics
19, 
1575–1585 (2020).
</cite> [<a href="https://doi.org/10.1074/mcp.TIR120.002048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7996969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32616513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Fast%20quantitative%20analysis%20of%20timsTOF%20PASEF%20data%20with%20MSFragger%20and%20IonQuant&amp;volume=19&amp;publication_year=2020&amp;pages=1575-1585&amp;pmid=32616513&amp;doi=10.1074/mcp.TIR120.002048&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R63">
<span class="label">63.</span><cite>Manriquez-Sandoval E., Brewer J., Lule G., Lopez S., Fried S. D., 
FLiPPR: A processor for limited proteolysis (LiP) mass spectrometry data sets built on FragPipe. J. Proteome Res.
23, 
2332–2342 (2024).
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.3c00887" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11590172/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38787630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=FLiPPR:%20A%20processor%20for%20limited%20proteolysis%20(LiP)%20mass%20spectrometry%20data%20sets%20built%20on%20FragPipe&amp;volume=23&amp;publication_year=2024&amp;pages=2332-2342&amp;pmid=38787630&amp;doi=10.1021/acs.jproteome.3c00887&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R64">
<span class="label">64.</span><cite>Mendes M. L., Fischer L., Chen Z. A., Barbon M., O’Reilly F. J., Giese S. H., Bohlke-Schneider M., Belsom A., Dau T., Combe C. W., Graham M., Eisele M. R., Baumeister W., Speck C., Rappsilber J., 
An integrated workflow for crosslinking mass spectrometry. Mol. Syst. Biol.
15, 
e8994 (2019).
</cite> [<a href="https://doi.org/10.15252/msb.20198994" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6753376/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31556486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Syst.%20Biol.&amp;title=An%20integrated%20workflow%20for%20crosslinking%20mass%20spectrometry&amp;volume=15&amp;publication_year=2019&amp;pages=e8994&amp;pmid=31556486&amp;doi=10.15252/msb.20198994&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>Supplementary Text</p>
<p>Figs. S1 to S7</p>
<p>Tables S1 to S6</p>
<p>Legends for data S1 and S2</p>
<p>References</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333692/bin/sciadv.adt8974_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adt8974_sm.pdf</a><sup> (2.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Data S1 and S2</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333692/bin/sciadv.adt8974_data_s1_and_s2.zip" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adt8974_data_s1_and_s2.zip</a><sup> (9.5MB, zip) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Science Advances are provided here courtesy of <strong>American Association for the Advancement of Science</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1126/sciadv.adt8974"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/sciadv.adt8974.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12333692/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12333692/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12333692%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333692/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12333692/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12333692/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40779622/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12333692/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40779622/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12333692/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12333692/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="bRB8wDdwpslTLG1xA5WeHg3DrJ2vwQ8GldVP47QekddOzrWqaowH89tSj1vj06kK">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
