
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            BIGapp: A user‐friendly genomic tool kit identified quantitative trait loci for creeping rootedness in alfalfa (Medicago sativa L.) - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="4DDE446589B6CBA31B446500460DA22A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12245737/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="BIGapp: A user‐friendly genomic tool kit identified quantitative trait loci for creeping rootedness in alfalfa (Medicago sativa L.)">
<meta name="citation_author" content="Alexander M Sandercock">
<meta name="citation_author_institution" content="Breeding Insight, Cornell University, Ithaca, New York, USA">
<meta name="citation_author" content="Michael D Peel">
<meta name="citation_author_institution" content="USDA‐ARS Forage and Range Research Lab, Utah State University, Logan, Utah, USA">
<meta name="citation_author" content="Cristiane H Taniguti">
<meta name="citation_author_institution" content="Breeding Insight, Cornell University, Ithaca, New York, USA">
<meta name="citation_author" content="Josué Chinchilla‐Vargas">
<meta name="citation_author_institution" content="Breeding Insight, Cornell University, Ithaca, New York, USA">
<meta name="citation_author" content="Shufen Chen">
<meta name="citation_author_institution" content="Breeding Insight, Cornell University, Ithaca, New York, USA">
<meta name="citation_author" content="Manoj Sapkota">
<meta name="citation_author_institution" content="Breeding Insight, Cornell University, Ithaca, New York, USA">
<meta name="citation_author" content="Meng Lin">
<meta name="citation_author_institution" content="Breeding Insight, Cornell University, Ithaca, New York, USA">
<meta name="citation_author" content="Dongyan Zhao">
<meta name="citation_author_institution" content="Breeding Insight, Cornell University, Ithaca, New York, USA">
<meta name="citation_author" content="Arlyn J Ackerman">
<meta name="citation_author_institution" content="Breeding Insight, Cornell University, Ithaca, New York, USA">
<meta name="citation_author" content="Bhoja R Basnet">
<meta name="citation_author_institution" content="Breeding Insight, Cornell University, Ithaca, New York, USA">
<meta name="citation_author" content="Craig T Beil">
<meta name="citation_author_institution" content="Breeding Insight, Cornell University, Ithaca, New York, USA">
<meta name="citation_author" content="Moira J Sheehan">
<meta name="citation_author_institution" content="Breeding Insight, Cornell University, Ithaca, New York, USA">
<meta name="citation_publication_date" content="2025 Jul 10">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70067">
<meta name="citation_doi" content="10.1002/tpg2.70067">
<meta name="citation_pmid" content="40640114">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12245737/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12245737/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12245737/pdf/TPG2-18-e70067.pdf">
<meta name="description" content="Alfalfa (Medicago sativa L.) is a globally vital forage crop valued for its perennial growth and multiple annual harvests. A breeding effort is underway to improve the crop for productivity and persistence against biotic and abiotic stresses using ...">
<meta name="og:title" content="BIGapp: A user‐friendly genomic tool kit identified quantitative trait loci for creeping rootedness in alfalfa (Medicago sativa L.)">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Alfalfa (Medicago sativa L.) is a globally vital forage crop valued for its perennial growth and multiple annual harvests. A breeding effort is underway to improve the crop for productivity and persistence against biotic and abiotic stresses using ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12245737/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12245737">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70067"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70067.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12245737%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12245737/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12245737/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12245737/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70067" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Jul 10;18(3):e70067. doi: <a href="https://doi.org/10.1002/tpg2.70067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70067</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>BIGapp: A user‐friendly genomic tool kit identified quantitative trait loci for creeping rootedness in alfalfa (<em>Medicago sativa</em> L.)</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sandercock%20AM%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Alexander M Sandercock</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Alexander M Sandercock</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Breeding Insight, Cornell University, Ithaca, New York, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sandercock%20AM%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexander M Sandercock</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peel%20MD%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Michael D Peel</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Michael D Peel</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>USDA‐ARS Forage and Range Research Lab, Utah State University, Logan, Utah, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peel%20MD%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael D Peel</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Taniguti%20CH%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Cristiane H Taniguti</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Cristiane H Taniguti</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Breeding Insight, Cornell University, Ithaca, New York, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Taniguti%20CH%22%5BAuthor%5D" class="usa-link"><span class="name western">Cristiane H Taniguti</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chinchilla%E2%80%90Vargas%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Josué Chinchilla‐Vargas</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Josué Chinchilla‐Vargas</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Breeding Insight, Cornell University, Ithaca, New York, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chinchilla%E2%80%90Vargas%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Josué Chinchilla‐Vargas</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Shufen Chen</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Shufen Chen</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Breeding Insight, Cornell University, Ithaca, New York, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shufen Chen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sapkota%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Manoj Sapkota</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Manoj Sapkota</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Breeding Insight, Cornell University, Ithaca, New York, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sapkota%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Manoj Sapkota</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Meng Lin</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Meng Lin</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Breeding Insight, Cornell University, Ithaca, New York, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Meng Lin</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Dongyan Zhao</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Dongyan Zhao</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Breeding Insight, Cornell University, Ithaca, New York, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dongyan Zhao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ackerman%20AJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Arlyn J Ackerman</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Arlyn J Ackerman</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Breeding Insight, Cornell University, Ithaca, New York, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ackerman%20AJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Arlyn J Ackerman</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Basnet%20BR%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Bhoja R Basnet</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Bhoja R Basnet</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Breeding Insight, Cornell University, Ithaca, New York, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Basnet%20BR%22%5BAuthor%5D" class="usa-link"><span class="name western">Bhoja R Basnet</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Beil%20CT%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Craig T Beil</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Craig T Beil</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Breeding Insight, Cornell University, Ithaca, New York, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Beil%20CT%22%5BAuthor%5D" class="usa-link"><span class="name western">Craig T Beil</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sheehan%20MJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Moira J Sheehan</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Moira J Sheehan</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Breeding Insight, Cornell University, Ithaca, New York, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sheehan%20MJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Moira J Sheehan</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270067-aff-0001">
<sup>
<sup>1</sup>
</sup>Breeding Insight, Cornell University, Ithaca, New York, USA</div>
<div id="tpg270067-aff-0002">
<sup>
<sup>2</sup>
</sup>USDA‐ARS Forage and Range Research Lab, Utah State University, Logan, Utah, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Moira J. Sheehan, Breeding Insight, Cornell University, 525 Tower Rd. Ithaca, NY 14853, USA. Email: <span>moirasheehan@cornell.edu</span>.
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 May 23; Received 2025 Jan 24; Accepted 2025 May 28; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12245737  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40640114/" class="usa-link">40640114</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Alfalfa (<em>Medicago sativa</em> L.) is a globally vital forage crop valued for its perennial growth and multiple annual harvests. A breeding effort is underway to improve the crop for productivity and persistence against biotic and abiotic stresses using “creeping rootedness,” a trait where plants exhibit horizontal root growth, similar to rhizomes, with increased vegetative ground surface area. In this study, we genotyped a breeding population of 648 alfalfa lines segregating for creeping rootedness using the 3K DArTag marker panel to identify trait‐associated genomic loci and evaluate the feasibility of genomic prediction to accelerate breeding cycles. Using genome‐wide association studies (GWAS), we identified three quantitative trait loci (QTLs), with one major QTL located on chromosome 6.1 associated with this trait. Genomic prediction showed moderate predictive ability (<em>r</em> = 0.68) for creeping rootedness. A significant advancement in this study was the development and utilization of the Breeding Insight Genomics Application (BIGapp), an R Shiny application designed to streamline the processing of genomic data through an intuitive interface. This tool makes integrating genomics into existing breeding programs accessible, regardless of species ploidy or the researcher's coding proficiency. The identified QTL will be essential in future efforts to develop new alfalfa cultivars with the creeping rootedness trait and accelerate the breeding cycle, with BIGapp playing a pivotal role in these advancements.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270067-list-10001" class="list" style="list-style-type:disc">
<li><p>Breeding Insight Genomics Application (BIGapp) is a user‐friendly tool kit that simplifies genomic analyses.</p></li>
<li><p>Evaluated the genetic architecture for creeping rootedness in alfalfa with BIGapp.</p></li>
<li><p>Identified three quantitative trait loci associated with the alfalfa creeping rootedness trait.</p></li>
<li><p>Genomic prediction shows a moderate predictive ability for creeping rootedness.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270067-dl-0001">
<dt>BIGapp</dt>
<dd><p>Breeding Insight Genomics Application</p></dd>
<dt>BIGr</dt>
<dd><p>Breeding Insight Genomics R Package</p></dd>
<dt>BLUE</dt>
<dd><p>best linear unbiased estimate</p></dd>
<dt>DArT</dt>
<dd><p>diversity arrays technology</p></dd>
<dt>GS</dt>
<dd><p>genomic selection</p></dd>
<dt>GT</dt>
<dd><p>genotype</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association study</p></dd>
<dt>LD</dt>
<dd><p>linkage disequilibrium</p></dd>
<dt>MADC</dt>
<dd><p>missing allele discovery counts</p></dd>
<dt>MAF</dt>
<dd><p>minor allele frequency</p></dd>
<dt>MAS</dt>
<dd><p>marker‐assisted selection</p></dd>
<dt>PC</dt>
<dd><p>principal component dimension</p></dd>
<dt>PCA</dt>
<dd><p>principal component analysis</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus</p></dd>
<dt>RRID</dt>
<dd><p>research resource ID</p></dd>
<dt>SNP</dt>
<dd><p>single‐nucleotide polymorphism</p></dd>
<dt>USDA</dt>
<dd><p>united states department of agriculture</p></dd>
<dt>VCF</dt>
<dd><p>variant call file</p></dd>
</dl>
<section id="tpg270067-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Alfalfa (<em>Medicago sativa</em> L.) is one of the most extensively grown forage crops, with between 32 and 40 million ha under cultivation globally (Castonguay et al., <a href="#tpg270067-bib-0011" class="usa-link" aria-describedby="tpg270067-bib-0011">2006</a>; Luo et al., <a href="#tpg270067-bib-0036" class="usa-link" aria-describedby="tpg270067-bib-0036">2019</a>; Russelle, <a href="#tpg270067-bib-0052" class="usa-link" aria-describedby="tpg270067-bib-0052">2001</a>). It has tremendous agronomic importance due to its high nutritional value for cattle, perennial growth, and nitrogen‐fixing capabilities (Flajoulot et al., <a href="#tpg270067-bib-0020" class="usa-link" aria-describedby="tpg270067-bib-0020">2005</a>; Mårtensson &amp; Ljunggren, <a href="#tpg270067-bib-0038" class="usa-link" aria-describedby="tpg270067-bib-0038">1984</a>). In primarily grass pastures, forage production is most often limited by available soil N (Chinea &amp; Arévalo, <a href="#tpg270067-bib-0015" class="usa-link" aria-describedby="tpg270067-bib-0015">2014</a>; Solomon et al., <a href="#tpg270067-bib-0057" class="usa-link" aria-describedby="tpg270067-bib-0057">2011</a>). Supplemental N fertilizer is an added expense or not feasible for most rangeland situations. Low bioavailable soil N can be mitigated by including N‐fixing legumes into rangelands and grazing situations, which has been shown to increase productivity and more uniform distribution of seasonal forage production (Cox et al., <a href="#tpg270067-bib-0016" class="usa-link" aria-describedby="tpg270067-bib-0016">2017</a>; Sleugh et al., <a href="#tpg270067-bib-0056" class="usa-link" aria-describedby="tpg270067-bib-0056">2000</a>). More specifically, the value of alfalfa in grazing lands has been well‐documented for many years, as alfalfa inclusion boosts forage yield and subsequent increases in animal mass (Berdahl et al., <a href="#tpg270067-bib-0005" class="usa-link" aria-describedby="tpg270067-bib-0005">1989</a>; Cox et al., <a href="#tpg270067-bib-0016" class="usa-link" aria-describedby="tpg270067-bib-0016">2017</a>; Sleugh et al., <a href="#tpg270067-bib-0056" class="usa-link" aria-describedby="tpg270067-bib-0056">2000</a>; Waldron et al., <a href="#tpg270067-bib-0060" class="usa-link" aria-describedby="tpg270067-bib-0060">2020</a>). Despite the benefits alfalfa provides for forage animal nutrition, even the hardiest improved cultivars still succumb to decreases in plant stands under grazing, which is exacerbated and hastened on arid rangelands (Heinrichs, <a href="#tpg270067-bib-0025" class="usa-link" aria-describedby="tpg270067-bib-0025">1954</a>; Misar et al., <a href="#tpg270067-bib-0044" class="usa-link" aria-describedby="tpg270067-bib-0044">2015</a>; Van Keuren &amp; Matches, <a href="#tpg270067-bib-0058" class="usa-link" aria-describedby="tpg270067-bib-0058">1988</a>).</p>
<p>Typical hay‐type <em>M. sativa</em> alfalfa cultivars have a single deep taproot with narrow or protruding crowns, resulting in limited tolerance to animal trampling under grazing, as well as frost injury, which increases their susceptibility to biotic and abiotic stresses and consequently leads to poor stand persistence (Brummer &amp; Bouton, <a href="#tpg270067-bib-0008" class="usa-link" aria-describedby="tpg270067-bib-0008">1991</a>; Kehr et al., <a href="#tpg270067-bib-0029" class="usa-link" aria-describedby="tpg270067-bib-0029">1963</a>). Traits important to survival and persistence are variously expressed in different alfalfa cultivars and may include broad or deep crowns, spreading or creeping rootedness, and dormant types in drought‐prone areas. The use of creeping rootedness is supported by Heinrichs, who demonstrated that new shoots from horizontal roots, for example, the creeping root trait, protect the plants from mortality due to trampling during grazing as well as protection from winter injury (Heinrichs, <a href="#tpg270067-bib-0026" class="usa-link" aria-describedby="tpg270067-bib-0026">1975</a>). This was further demonstrated by the release of Travois, a very hardy dormant alfalfa with a proliferating root system (Rumbaugh et al., <a href="#tpg270067-bib-0051" class="usa-link" aria-describedby="tpg270067-bib-0051">1965</a>).</p>
<p>Commercially cultivated purple‐flowered alfalfa (<em>M. sativa</em> subsp. <em>sativa</em>) is a self‐incompatible cross‐pollinated autotetraploid (2<em>n</em> = 4<em>x</em> = 32) (McCoy &amp; Bingham, <a href="#tpg270067-bib-0039" class="usa-link" aria-describedby="tpg270067-bib-0039">1988</a>). There are both diploid (2<em>n</em> = 2<em>x</em> = 16) and tetraploid yellow‐flowered forms of <em>M. sativa</em>, which are classified as subsp. <em>Falcata</em>, while the purple‐flowered forms are classified as subsp. <em>sativa</em> (Brummer et al., <a href="#tpg270067-bib-0009" class="usa-link" aria-describedby="tpg270067-bib-0009">1999</a>). The <em>falcata</em>‐type alfalfa genetic background is not found in high proportions in commercial cultivars. However, it remains a valuable source of germplasm for improved persistence due to winter, drought, and grazing tolerance, as well as the spreading/creeping root phenotype (Berdahl et al., <a href="#tpg270067-bib-0006" class="usa-link" aria-describedby="tpg270067-bib-0006">1986</a>; Rumbaugh,<a href="#tpg270067-bib-0051" class="usa-link" aria-describedby="tpg270067-bib-0051">1965</a>).</p>
<p>The genetic basis of the creeping root phenotype is largely unknown and understudied (Heinrichs, <a href="#tpg270067-bib-0025" class="usa-link" aria-describedby="tpg270067-bib-0025">1954</a>; Polegri et al., <a href="#tpg270067-bib-0047" class="usa-link" aria-describedby="tpg270067-bib-0047">2011</a>). A lack of genetic studies for creeping rootedness has hindered its incorporation by marker‐assisted selection (MAS), genomic selection (GS), or other screening approaches aimed at expediting the breeding process (e.g., seedling screening and speed breeding). Genomic resources for tetraploid alfalfa include several chromosomal‐scale reference genomes (H. Chen et al., <a href="#tpg270067-bib-0013" class="usa-link" aria-describedby="tpg270067-bib-0013">2020</a>; Li et al., <a href="#tpg270067-bib-0031" class="usa-link" aria-describedby="tpg270067-bib-0031">2020</a>; Shen et al., <a href="#tpg270067-bib-0054" class="usa-link" aria-describedby="tpg270067-bib-0054">2020</a>), a 9K single‐nucleotide polymorphism (SNP) marker array (X. Li et al., <a href="#tpg270067-bib-0034" class="usa-link" aria-describedby="tpg270067-bib-0034">2014</a>), and a diversity arrays technology (DArT) 3K DArTag panel, which is a targeted amplicon genotyping platform (X. Zhao, Han, et al., <a href="#tpg270067-bib-0066" class="usa-link" aria-describedby="tpg270067-bib-0066">2023</a>; D. Zhao, Mejia‐Guerra, et al., <a href="#tpg270067-bib-0064" class="usa-link" aria-describedby="tpg270067-bib-0064">2023</a>). The 3K DArTag panel has already shown its power to identify quantitative trait loci (QTLs) and evaluate breeding populations (Medina, Heuschele, et al., <a href="#tpg270067-bib-0040" class="usa-link" aria-describedby="tpg270067-bib-0040">2024</a>; Medina, Zhao, et al., <a href="#tpg270067-bib-0042" class="usa-link" aria-describedby="tpg270067-bib-0042">2024</a>). This panel provides a rapid, affordable, and reproducible method to evaluate genome‐wide markers across different genotyping projects, and it has been shown to be effective in both diploid and tetraploid alfalfa species (D. Zhao et al., <a href="#tpg270067-bib-0065" class="usa-link" aria-describedby="tpg270067-bib-0065">2024</a>).</p>
<p>Despite the increasing number of genomic resources for alfalfa, many breeders face barriers to implementing genomic tools into their programs. The barriers include the cost and time to genotype (GT), uncertainty or incomplete adoption of routine genotyping into their breeding cycle effectively, and coding expertise needed to analyze the large and complex data formats of next‐generation sequencing technologies. When breeding programs lack in‐house expertise, they must rely on external collaborators, hire bioinformaticians, or purchase services from genotyping vendors. This can lead to added costs, delays in analyzing results for time‐sensitive breeding decisions, or vendors using improper bioinformatic tools designed for model diploid species. R Shiny applications exist to leverage the strengths of the R coding language while providing a more user‐friendly way (i.e., low‐ or no‐code) to view and analyze data in a web‐based environment (Chang et al., <a href="#tpg270067-bib-0012" class="usa-link" aria-describedby="tpg270067-bib-0012">2024</a>). Recent R Shiny applications exist for analyzing RNAseq data (Farrel et al., <a href="#tpg270067-bib-0019" class="usa-link" aria-describedby="tpg270067-bib-0019">2023</a>), visualizing genomic data (T. Chen et al., <a href="#tpg270067-bib-0014" class="usa-link" aria-describedby="tpg270067-bib-0014">2023</a>; Ramirez et al., <a href="#tpg270067-bib-0048" class="usa-link" aria-describedby="tpg270067-bib-0048">2020</a>), and analyzing genome‐wide identity‐by‐descent data (Mahmoudiandehkordi et al., <a href="#tpg270067-bib-0037" class="usa-link" aria-describedby="tpg270067-bib-0037">2024</a>). However, many of these applications primarily support diploid species, posing an additional challenge for researchers working with autopolyploids, which are more commonly encountered by specialty crop breeders. Thus, an R Shiny application that can provide the ability to preprocess and analyze genomic data, especially read depth data from targeted genotyping platforms, regardless of the ploidy of the target organism, would lower the barrier for breeders to efficiently and reproducibly integrate genomics into their existing breeding programs to assist in making breeding decisions.</p>
<p>Here, we introduce the Breeding Insight Genomics R Shiny application (BIGapp, research resource ID [RRID]: SCR_026676) and breeding insight genomics R package (BIGr) (RRID: SCR_026677, available on CRAN (<a href="https://CRAN.R-project.org/package=BIGr" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R‐project.org/package=BIGr</a>). The BIGapp is designed to offer accessible, web‐based genomic and bioinformatics tools tailored for analyzing both diploid and autopolyploid data. Using BIGapp, we analyzed 648 alfalfa samples that were sequenced with the alfalfa 3K DArTag panel to explore the genomic basis of creeping rootedness in a mapping population. Specifically, we (i) established a breeding population of alfalfa exhibiting the creeping‐rooted phenotype, characterized by lateral root spreading; (ii) identified QTLs associated with creeping rootedness; and (iii) evaluated the potential of GS for this trait. These findings aim to enhance alfalfa breeding programs through the identification of QTL related to creeping rootedness, and the BIGapp enables researchers to analyze and publish their genomic data with various ploidy levels regardless of their computer coding background.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270067-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270067-list-0001" class="list" style="list-style-type:disc">
<li><p>Breeding Insight Genomics Application (BIGapp) is a user‐friendly tool kit that simplifies genomic analyses.</p></li>
<li><p>Evaluated the genetic architecture for creeping rootedness in alfalfa with BIGapp.</p></li>
<li><p>Identified three quantitative trait loci associated with the alfalfa creeping rootedness trait.</p></li>
<li><p>Genomic prediction shows a moderate predictive ability for creeping rootedness.</p></li>
</ul></section></section></section><section id="tpg270067-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270067-sec-0040"><h3 class="pmc_sec_title">2.1. Breeding population design</h3>
<p>A spreading/creeping rooted, tetraploid alfalfa population developed at the United States Department of Agriculture (USDA) Agricultural Research Service Forage and Range Research unit in Logan, UT, was the parental source of spreading phenotype. This material was selected specifically for its spread and survival/persistence under dry‐range conditions typical of the Intermountain Region of the Western United States. It was selected from a broad genetic base of cultivars (released prior to 1982), experimental lines, and naturalized populations that included both <em>sativa</em> and <em>falcata</em> types exhibiting the creeping rooted phenotype. The <em>falcata</em> background was derived primarily from germplasm introduced by N.E. Hansen as described by Boe et al. (<a href="#tpg270067-bib-0007" class="usa-link" aria-describedby="tpg270067-bib-0007">2020</a>). Two plants from this population were used as the spreading parents and designated Entry 3 and Entry 8. These plants were chosen as parents for their strong expression of the creeping root/spreading phenotype (Figure <a href="#tpg270067-fig-0001" class="usa-link">1</a>). The two spreading plants were vegetatively cloned to generate additional plants for crossing. The plants with the spreading phenotype were manually crossed in a greenhouse with six plants from a non‐spreading commercial‐type <em>sativa</em> (ID = CT3‐08). Seed was collected from both the spreading (Entry 3 and Entry 8) and non‐spreading (CT3‐08) parents in each cross and used for testing.</p>
<figure class="fig xbox font-sm" id="tpg270067-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f650/12245737/cf45da6a1252/TPG2-18-e70067-g004.jpg" loading="lazy" id="jats-graphic-1" height="337" width="700" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270067-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The creeping rooted phenotype in alfalfa grown at the study site near Logan, UT. (A) Seeded rows of non‐spreading versus spreading alfalfa showing early spring growth. (B) Creeping root structure showing multiple shoots arising from the single creeping root. (C) A 9‐year‐old single alfalfa plant showing spread in comparison to a meter stick. (D) The descriptions for each of the two measured traits used to evaluate creeping rootedness.</p></figcaption></figure><p>The different crosses within the spreading root mapping population were assigned a color name. Each color corresponds to a specific cross between spreading and non‐spreading alfalfa parental lines (Figure <a href="#tpg270067-supinfo-0001" class="usa-link">S1</a>). Green designates progeny from reciprocal crosses between the two spreading parental lines (Entries 3 and 8), resulting in full‐sibling progenies. Blue, pink, and red are three separate cross events between the spreading parent (Entry 3) and the non‐spreading parent (CT3‐08), whereas orange, white, and yellow represent crosses between the spreading parent (Entry 8) and the non‐spreading parent (CT3‐08) (Figure <a href="#tpg270067-supinfo-0001" class="usa-link">S1</a>). All progeny within each of these crosses are full siblings.</p>
<p>Field plots were located at the Utah State University Evans Research Farm near Logan, UT (41°41′36″ N, 111°49′55″ W, elevation 1381 m). Soil at the Evans Farm is a Nibley silty clay loam (fine, mixed, mesic Aquic Argiustolls) where annual precipitation averages 47 cm (1981–2010) (<a href="http://www.usclimatedata.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.usclimatedata.com</a>, November 8, 2024). The Evans Farm site is in the Central Great Basin of the western United States, characterized by hot, dry summers where most of the annual precipitation typically occurs as snowfall.</p>
<p>Plant materials were established as transplanted spaced plants in 10‐plant plots in the spring of 2014. Plants were started in the greenhouse the second week of January in Ray Leach Cone‐tainers (Stewe and Sons) and transplanted to the field in the second week of May. Dead plants were replaced with plants from their respective cross 3 weeks after transplanting. Plants were established on 1‐m centers within rows with 1‐m spacing between rows. The 1‐m spacing between all plants allowed for easy measurement of spread from any individual plant. Prior to planting, mono‐ammonium phosphate (11N‐52P‐0K) was applied at 90 kg ha<sup>−1</sup> providing 9.9 and 46.6 kg ha<sup>−1</sup> each of N and P, respectively. The application was made to ensure plant P requirements were met for the duration of the study.</p>
<p>The field experiment was conducted using an augmented incomplete block design, with repeated checks and parental lines distributed across the experimental field, which was divided into four incomplete blocks. Individual plants were arranged at the intersections of rows and columns, each being treated as an independent observational unit.</p></section><section id="tpg270067-sec-0050"><h3 class="pmc_sec_title">2.2. Data collection</h3>
<p>Vegetative spread was assessed at two different time points, May 13, 2022 and May 17, 2023, during the study using a visual rating scale of 0–10, where 0 = no spread and 10 = spread up to the trial maximum, or in excess of, the midpoint between each individual plant (Figure <a href="#tpg270067-fig-0001" class="usa-link">1</a>). Spread from single plants was rarely uniform, as a single plant might spread excessively in one orientation but not in another. To accommodate this, the spread was measured between the two widest points on the same plant. In some instances, the determination of spread was subjective. As the plants matured, their crowns grew wider than is common in stands with low plant density, a condition intentionally created for this study. Therefore, it was possible to classify a plant as spreading/creeping rooted when it was simply a large crown. However, these cases would always appear at the low end of the spreading spectrum.</p></section><section id="tpg270067-sec-0060"><h3 class="pmc_sec_title">2.3. Generation of best linear unbiased estimates (BLUEs)</h3>
<p>Phenotypic data were analyzed with a two‐phase mixed effects model in ASReml‐R (Butler et al., <a href="#tpg270067-bib-0010" class="usa-link" aria-describedby="tpg270067-bib-0010">2018</a>). To ensure unbiased GT estimates suitable for genome‐wide association studies (GWAS), GT was modeled as a fixed effect to avoid over shrinkage (Holland &amp; Piepho, <a href="#tpg270067-bib-0028" class="usa-link" aria-describedby="tpg270067-bib-0028">2024</a>), and the two‐phase analysis was utilized to accommodate unbalanced, multi‐environment data (Endelman, <a href="#tpg270067-bib-0018" class="usa-link" aria-describedby="tpg270067-bib-0018">2023</a>). Experiments were conducted across two consecutive years (2022 and 2023), with each year being treated as a unique environment throughout the analysis.</p>
<p>In the first phase, GT effects were designated explicitly as fixed effects to derive BLUEs separately for each year and accurately quantify GT‐specific performance unbiased by shrinkage, through the following fixed effects model:</p>
<table class="disp-formula p" id="tpg270067-disp-0001"><tr><td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>G</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>S</mi><mrow><mi>j</mi><mi>k</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi></mrow></msub></mrow></mrow></math></td></tr></table>
<p>where <em>y<sub>ijk</sub>
</em> is the observed phenotype for the <em>i</em>th GT in the <em>j</em>th column and <em>k</em>th row, <em>μ</em> is the overall mean, <em>G<sub>i</sub>
</em> denotes the fixed effects of GTs, and <em>S<sub>jk</sub>
</em> is a random spatial effect associated with the plot at column <em>j</em> and row <em>k</em>, which is modeled as below:</p>
<p>
<em>S<sub>jk</sub>
</em> follows a normal distribution <em>N</em> (0, Σ<sub>spatial</sub>) where <em>Var</em>(<em>S<sub>jk</sub>
</em>) = Σ<sub>spatial</sub> = [(<em>σ</em>
<sup>2</sup>
<sub>c</sub> Σ<sub>c</sub>) ⊗ Σ<sub>r</sub>]. Here, <em>σ</em>
<sup>2</sup>
<sub>c</sub> is the column variance, Σ<sub>c</sub> is the first‐order autoregressive‐correlation matrix with elements {<em>σ</em>
<sub>mn</sub>} = <em>ρ</em>
<sup>|m‐n|</sup> for plots in columns m and n (in the same row), and Σ<sub>r</sub> is the first‐order autoregressive matrix for rows. The symbol ⊗ implies the Kronecker product of the two auto‐correlation matrices. <em>ρ</em> is the autocorrelation parameter for both row and column factors. <em>ε<sub>ijk</sub>
</em> is the residual error, which is independently and identically distributed with mean 0 and variance <em>σ</em>
<sup>2</sup>. BLUEs were estimated for each GT within each environment, and precision weights were applied for each estimate as follows:</p>
<table class="disp-formula p" id="tpg270067-disp-0002"><tr><td class="formula"><math id="jats-math-2" display="block"><mrow><mrow><msub><mi>w</mi><mrow><mi>i</mi><mi>l</mi></mrow></msub><mo linebreak="badbreak">=</mo><mfrac><mn>1</mn><mrow><mi>S</mi><msubsup><mi>E</mi><mrow><mi>i</mi><mi>l</mi></mrow><mn>2</mn></msubsup></mrow></mfrac></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline"><mrow><msub><mi>w</mi><mrow><mi>i</mi><mi>l</mi></mrow></msub></mrow></math></span> is the weight for the <em>i</em>th GT in the <em>l</em>th environment and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><mrow><mi>S</mi><msubsup><mi>E</mi><mrow><mi>i</mi><mi>l</mi></mrow><mn>2</mn></msubsup></mrow></mrow></math></span> is the corresponding standard error. BLUEs were carried into phase 2 as the response variable in the following precision‐weighted fixed effect model:</p>
<table class="disp-formula p" id="tpg270067-disp-0003"><tr><td class="formula"><math id="jats-math-5" display="block"><mrow><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>l</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>G</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>l</mi></mrow></msub></mrow></mrow></math></td></tr></table>
<p>where <em>y<sub>il</sub>
</em> is the BLUE for the <em>i</em>th GT in the <em>l</em>th environment, <em>μ</em> is the overall mean, <em>G<sub>i</sub>
</em> is the fixed effect of the <em>i</em>th GT, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>l</mi></mrow></msub></mrow></math></span> is the weighted residual error term, modeled with variance as follows:</p>
<table class="disp-formula p" id="tpg270067-disp-0004"><tr><td class="formula"><math id="jats-math-7" display="block"><mrow><mrow><mi>Var</mi><mspace width="0.28em"></mspace><mfenced separators="" open="(" close=")"><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>l</mi></mrow></msub></mfenced><mo linebreak="badbreak">=</mo><mfrac><msup><mi mathvariant="normal">σ</mi><mn>2</mn></msup><msub><mi>w</mi><mrow><mi>i</mi><mi>l</mi></mrow></msub></mfrac></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><mrow><mi>Var</mi><mo stretchy="false">(</mo><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>l</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mrow></math></span> is the variance of the weighted residual for the <em>i</em>th GT in the <em>l</em>th environment, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><msup><mi mathvariant="normal">σ</mi><mn>2</mn></msup></mrow></math></span> is fixed at one, implying that the precision weight (<em>w<sub>il</sub>
</em>) determines residual variance <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><mrow><mi>Var</mi><mo stretchy="false">(</mo><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>l</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mrow></math></span> (Supporting Information <a href="#tpg270067-supinfo-0001" class="usa-link">1</a>).</p>
<p>Broad‐sense heritability (<em>H</em>
<sup>2</sup>) for each of the two creeping root measurements was calculated independently using the getHeritability() function from the spatial analysis of field trials with splines R package (v1.0.19; Oakey et al., <a href="#tpg270067-bib-0046" class="usa-link" aria-describedby="tpg270067-bib-0046">2006</a>; Rodríguez‐Álvarez et al., <a href="#tpg270067-bib-0049" class="usa-link" aria-describedby="tpg270067-bib-0049">2018</a>) as implemented in the MrBean R Shiny application (v2.0.9; Aparicio et al., <a href="#tpg270067-bib-0004" class="usa-link" aria-describedby="tpg270067-bib-0004">2023</a>).</p></section><section id="tpg270067-sec-0070"><h3 class="pmc_sec_title">2.4. Sequencing and processing of genomic data</h3>
<p>A previously developed alfalfa 3000 DArTag marker panel (D. Zhao, Mejia‐Guerra, et al., <a href="#tpg270067-bib-0064" class="usa-link" aria-describedby="tpg270067-bib-0064">2023</a>) was used to GT the 648 alfalfa individuals. DArT provided a missing allele discovery counts (MADC) file that contains the 81 bp amplicon at each of the 3000 panel target loci and their multiallelic information (Supporting Information <a href="#tpg270067-supinfo-0001" class="usa-link">2</a>). From this, we extracted the read count information for the target 3000 SNPs and for the additional genetic variants (off‐target SNPs) contained within the 81 bp region using the BIGapp “Convert to VCF” tab (VCF is variant call file) (Table <a href="#tpg270067-tbl-0001" class="usa-link">1</a>). The “Convert to VCF” tab relies on the BIGr madc2vcf_all() function, which performs pairwise alignments between all of the unique sequence reads (microhaplotype) and the reference to obtain all the biallelic SNPs contained within each microhaplotype using the R/Rsamtools package (v2.22.0; Morgan et al., <a href="#tpg270067-bib-0045" class="usa-link" aria-describedby="tpg270067-bib-0045">2025</a>). In the process, sequences with low alignment score (&lt;40) and multiallelic SNPs are discarded. Alternatively, DArT can provide a VCF file of target and off‐target SNPs that is derived from DArT's internal SNP calling pipeline, upon request. The read count information for a total of 6172 target and off‐target SNPs was converted to a VCF file format and then used as the initial input for BIGapp “Dosage Calling” tab (Table <a href="#tpg270067-tbl-0001" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="tpg270067-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>The Breeding Insight Genomics Application (BIGapp) modules and the supporting breeding insight genomics R package (BIGr) functions were used to preprocess and analyze the alfalfa creeping root population.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Tool</th>
<th align="left" rowspan="1" colspan="1">Module/function</th>
<th align="left" rowspan="1" colspan="1">Description</th>
</tr></thead>
<tbody>
<tr><td colspan="3" align="left" rowspan="1">Genotype processing</td></tr>
<tr>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">BIGapp</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">Convert to VCF</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">Convert genotype data to the standard variant call format (VCF) file. This module includes the ability to extract biallelic SNPs from the DArTag MADC format.</td>
</tr>
<tr>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">BIGapp</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">Dosage calling</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">Use read count information in a VCF or MADC file format to generate genotype calls using the R/Updog package, output the information in VCF format.</td>
</tr>
<tr>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">BIGapp</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">VCF filtering</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">VCF files can be filtered to remove low quality SNPs and samples.</td>
</tr>
<tr>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">BIGr</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">madc2vcf_all()</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">Function to extract the read count information for the target and off‐target biallelic SNPs contained within the DArTag MADC file. The output is a VCF file.</td>
</tr>
<tr>
<td style="background-color:#D0E0E3" align="left" rowspan="1" colspan="1">BIGr</td>
<td style="background-color:#D0E0E3" align="left" rowspan="1" colspan="1">updog2vcf()</td>
<td style="background-color:#D0E0E3" align="left" rowspan="1" colspan="1">Function to convert the output from the R/Updog dosage calling <em>multidog()</em> function to a VCF format (Gerard, <a href="#tpg270067-bib-0021" class="usa-link" aria-describedby="tpg270067-bib-0021">2023</a>; Gerard &amp; Ferrão, <a href="#tpg270067-bib-0022" class="usa-link" aria-describedby="tpg270067-bib-0022">2020</a>).</td>
</tr>
<tr>
<td style="background-color:#D0E0E3" align="left" rowspan="1" colspan="1">BIGr</td>
<td style="background-color:#D0E0E3" align="left" rowspan="1" colspan="1">filterVCF()</td>
<td style="background-color:#D0E0E3" align="left" rowspan="1" colspan="1">Function to filter a VCF file for SNP and sample quality metrics.</td>
</tr>
<tr><td colspan="3" align="left" rowspan="1">Summary metrics</td></tr>
<tr>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">BIGapp</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">Genomic diversity</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">Calculate SNP minor allele frequency, sample observed heterozygosity, dosage call ratios, and visualize marker distribution.</td>
</tr>
<tr>
<td style="background-color:#D0E0E3" align="left" rowspan="1" colspan="1">BIGr</td>
<td style="background-color:#D0E0E3" align="left" rowspan="1" colspan="1">calculate_Het()</td>
<td style="background-color:#D0E0E3" align="left" rowspan="1" colspan="1">Calculate the observed heterozygosity for each sample.</td>
</tr>
<tr><td colspan="3" align="left" rowspan="1">Population structure</td></tr>
<tr>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">BIGapp</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">PCA</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">Evaluate genomic relationships between samples using a principal component analysis.</td>
</tr>
<tr><td colspan="3" align="left" rowspan="1">GWAS</td></tr>
<tr>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">BIGapp</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">GWASpoly</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">Perform genome wide association analyses to identify quantitative trait loci (QTLs) using the R/GWASpoly package (Rosyara et al., <a href="#tpg270067-bib-0050" class="usa-link" aria-describedby="tpg270067-bib-0050">2016</a>).</td>
</tr>
<tr><td colspan="3" align="left" rowspan="1">Genomic Prediction</td></tr>
<tr>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">BIGapp</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">Predictive ability</td>
<td style="background-color:#D9EAD3" align="left" rowspan="1" colspan="1">Estimate the predictive ability of a genomic prediction model using a fivefold cross validation. The estimated breeding values (EBVs) and predicted traits can be estimated for samples with and without phenotype information.</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270067-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270067-tbl1-note-0001"><p>
<em>Note</em>: All modules/functions listed here support diploid and polyploid data. Both the BIGapp and BIGr contain additional functions for processing genotypic and phenotypic breeding data not described here.</p></div>
<div class="fn" id="tpg270067-tbl1-note-0002"><p>Abbreviations: GWAS, genome‐wide association study; PCA, principal component analysis; SNP, single‐nucleotide polymorphism.</p></div>
</div></section><p>Dosage calls were estimated from the read count information in the BIGapp “Dosage Calling” tab. The read count information was first extracted from the VCF file with the vcfR package (v1.15.0) and converted to matrices (<em>m</em> × <em>n</em>), where <em>m</em> is equal to the number of samples and <em>n</em> is equal to the number of SNPs (Knaus &amp; Grünwald, <a href="#tpg270067-bib-0030" class="usa-link" aria-describedby="tpg270067-bib-0030">2017</a>). These read count matrices were used by the updog R package (v2.1.5) with the “norm” population model and ploidy level of four to estimate dosage calls for each of the alfalfa samples (Gerard, <a href="#tpg270067-bib-0021" class="usa-link" aria-describedby="tpg270067-bib-0021">2023</a>; Gerard &amp; Ferrão, <a href="#tpg270067-bib-0022" class="usa-link" aria-describedby="tpg270067-bib-0022">2020</a>; Gerard et al., <a href="#tpg270067-bib-0023" class="usa-link" aria-describedby="tpg270067-bib-0023">2018</a>). The resulting VCF file contained the estimated updog dosage calls, dosage calling statistics, and read count information.</p>
<p>The VCF file of raw dosage calls was then filtered in the BIGapp “VCF Filtering” tab. The R/updog genotyping quality metrics were first used to remove SNPs if they had an allelic bias (Bias) value below 0.2 or &gt;2, a locus‐specific estimated overdispersion parameter (OD) value &gt;0.05, posterior proportion of individuals GTd incorrectly for that SNP (Prop_mis) &gt;0.05, and a maximum posterior probability that the GT estimate for the SNP/sample is correct (maxpostprob) &lt;0.9 (BIGr v0.5.5; <a href="https://CRAN.R-project.org/package=BIGr" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R‐project.org/package=BIGr</a>). SNPs were removed with a minor allele frequency (MAF) threshold &lt;0.05. The dosage value was set to missing if the read depth for any marker‐sample combination was &lt;10. Finally, SNPs and samples were removed if they had ≥50% missing data. The filtered GT dosage file contained 2211 SNPs for 648 samples (Supporting Information <a href="#tpg270067-supinfo-0001" class="usa-link">3</a>).</p></section><section id="tpg270067-sec-0080"><h3 class="pmc_sec_title">2.5. Summary metrics and structure of GT data</h3>
<p>Summary metrics for the filtered alfalfa genotyping data were evaluated with the BIGapp “Genomic Diversity” tab. MAF was estimated for each SNP. Observed heterozygosity and the GT distributions were estimated per sample. The genomic relationship between each sample was visualized with a principal component analysis (PCA) in the “PCA” tab. The filtered VCF file was input, and the information from the GT field was extracted with vcfR (v1.15.0) and then converted to a matrix of numeric GT values of the alternate allele count (0 = homozygous reference, and 4 = homozygous alternate) (Knaus &amp; Grünwald, <a href="#tpg270067-bib-0030" class="usa-link" aria-describedby="tpg270067-bib-0030">2017</a>). This GT matrix was then converted to an additive genomic relationship matrix with the Gmatrix function from the AGHmatrix R package (v2.1.4) with the VanRaden parameter (Amadeu et al., <a href="#tpg270067-bib-0003" class="usa-link" aria-describedby="tpg270067-bib-0003">2023</a>). This relationship matrix was then used by prcomp() from base R to calculate the eigenvectors. The PCA was then visualized with R packages plotly (v4.10.4; Li &amp; Bilal, <a href="#tpg270067-bib-0032" class="usa-link" aria-describedby="tpg270067-bib-0032">2021</a>; Wickham, <a href="#tpg270067-bib-0061" class="usa-link" aria-describedby="tpg270067-bib-0061">2011</a>) for a three‐dimensional visualization and ggplot2 (v3.5.1; Li &amp; Bilal, <a href="#tpg270067-bib-0032" class="usa-link" aria-describedby="tpg270067-bib-0032">2021</a>; Wickham, <a href="#tpg270067-bib-0061" class="usa-link" aria-describedby="tpg270067-bib-0061">2011</a>) for a two‐dimensional figure.</p></section><section id="tpg270067-sec-0090"><h3 class="pmc_sec_title">2.6. QTL analysis</h3>
<p>A genome‐wide association study (GWAS) was performed to first identify SNPs associated with the creeping root phenotype using the BIGapp “GWASpoly” tab. We implemented the GWASpoly R package (Rosyara et al., <a href="#tpg270067-bib-0050" class="usa-link" aria-describedby="tpg270067-bib-0050">2016</a>) for its ability to use both polyploid and diploid species data. The input VCF file was first converted to the GWASpoly input format, where SNP location information is in the first two columns of a GT matrix. BLUEs for the creeping rootedness trait were used for the phenotype information. As outcrossing species may potentially be pollinated by contaminated pollen, 417 progenies (Supporting Information <a href="#tpg270067-supinfo-0001" class="usa-link">1</a>) with known pedigree and expected GT cluster patterns (Figure <a href="#tpg270067-supinfo-0001" class="usa-link">S2</a>) were selected for GWAS. The mixed linear model for GWAS included principal components (PCs) based on SNP GT data and a kinship matrix to control for variation in population stratification and unequal relatedness, respectively. PC and the kinship matrix were calculated using GWASpoly (v2.13; Rosyara et al., <a href="#tpg270067-bib-0050" class="usa-link" aria-describedby="tpg270067-bib-0050">2016</a>). The optimal model for GWAS was selected based on the Bayesian information criterion (Schwarz, <a href="#tpg270067-bib-0053" class="usa-link" aria-describedby="tpg270067-bib-0053">1978</a>) for the trait (Figure <a href="#tpg270067-supinfo-0001" class="usa-link">S3A</a>). For tetraploid alfalfa, different genetic models were tested, including additive, dominant (“1‐dom” and “2‐dom”), general, diplo‐additive, and diplo‐general. The threshold for significance was the “M.eff” method for its balance of computational efficiency and accuracy. The <em>p</em>‐value distribution from GWAS was visualized using a <em>Q</em>–<em>Q</em>‐plot (Figure <a href="#tpg270067-supinfo-0001" class="usa-link">S3B</a>) with code derived from the CMplot package (LiLin‐Yin, <a href="#tpg270067-bib-0035" class="usa-link" aria-describedby="tpg270067-bib-0035">2024</a>).</p>
<p>A genome‐wide linkage disequilibrium (LD) decay plot was used to determine the minimum distance (8.575 Mb) between genetically independent markers (<em>r</em>
<sup>2</sup> &lt; 0.05; Figure <a href="#tpg270067-supinfo-0001" class="usa-link">S4</a>). Based on the selected window size, the significant markers identified were further evaluated for their location within 8.575 Mb LD regions with the get.QTL function and percent of explainable variance for the trait of interest using the fit.QTL function from GWASpoly as implemented in BIGapp (Rosyara et al., <a href="#tpg270067-bib-0050" class="usa-link" aria-describedby="tpg270067-bib-0050">2016</a>). The final set of QTL from the GWASpoly additive model for each trait was then evaluated for their proximity to genes from the XinJiangDaYe <em>M</em>. <em>sativa</em> reference genome annotation (H. Chen et al., <a href="#tpg270067-bib-0013" class="usa-link" aria-describedby="tpg270067-bib-0013">2020</a>; D. Zhao, Mejia‐Guerra, et al., <a href="#tpg270067-bib-0064" class="usa-link" aria-describedby="tpg270067-bib-0064">2023</a>).</p></section><section id="tpg270067-sec-0100"><h3 class="pmc_sec_title">2.7. Genomic prediction</h3>
<p>Predictive ability of a genomic prediction model was estimated for the creeping rootedness trait in the BIGapp “Predictive Ability” tab. For the same set of 417 samples used in GWAS, GT information was extracted from the VCF files and converted into a GT matrix. The GT matrix was then normalized and scaled so that the dosage calls were within a −1 to 1 range for compliance with the A.mat() function. The A.mat() function from ridge regression best linear unbiased prediction (rrBLUP) (v4.6.3) was used to generate an additive relationship matrix, where missing data were imputed with the mean value of each SNP (Endelman, <a href="#tpg270067-bib-0017" class="usa-link" aria-describedby="tpg270067-bib-0017">2011</a>). A fivefold cross‐validation with the kin.blup() function was then performed for 10 iterations (Endelman, <a href="#tpg270067-bib-0017" class="usa-link" aria-describedby="tpg270067-bib-0017">2011</a>). The predictive abilities were the averaged Pearson correlation coefficients between the predicted breeding value and BLUEs of the creeping root phenotype across 10 iterations.</p></section></section><section id="tpg270067-sec-0110"><h2 class="pmc_sec_title">3. RESULTS AND DISCUSSION</h2>
<section id="tpg270067-sec-0120"><h3 class="pmc_sec_title">3.1. Phenotype and GT summary for the creeping root breeding population</h3>
<p>The mean values for the non‐spreading alfalfa parental line (CT3‐08) were consistently below the average across all samples for both measurements (Figure <a href="#tpg270067-fig-0002" class="usa-link">2</a>). Both spreading parental lines had above‐average measurements for each trait, with Entry 8 recording greater mean values than Entry 3 (Figure <a href="#tpg270067-fig-0002" class="usa-link">2</a>). BLUEs were generated from the raw trait datasets to account for spatial variance within the planting areas. There was a high correlation between the two spread traits, with a Pearson's correlation value of 0.8. Broad‐sense heritability was higher for the Spread 0–10 measurement taken on 5/17/23 (<em>H</em>
<sup>2</sup> = 0.68) compared to the measurement from May 13, 2022 (<em>H</em>
<sup>2</sup> = 0.51) (Figure <a href="#tpg270067-fig-0002" class="usa-link">2C</a>).</p>
<figure class="fig xbox font-sm" id="tpg270067-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f650/12245737/5caeecc2cdba/TPG2-18-e70067-g003.jpg" loading="lazy" id="jats-graphic-3" height="368" width="700" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270067-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Summary of two measurements for creeping rootedness across all phenotyped samples. (A) Distribution of the ratings for “Spread 0–10 (5/13/22).” (B) Distribution of the ratings for “Spread 0–10 (5/17/23).” (A–B) The mean value for the spreading parental lines (Entry 3 and Entry 8) and the non‐spreading parental line (CT3‐08) is labeled with vertical dashed lines. (C) Summary metrics for the two creeping root measurements.</p></figcaption></figure><p>A total of 2211 SNPs across 648 samples remained after filtering for GT data quality. Genome‐wide SNP detection was largely well distributed, apart from Chromosome 6.1, which showed lower SNP density (1.7 SNPs/Mb vs. genome‐wide mean = 3.24 SNPs/Mb) (Figure <a href="#tpg270067-supinfo-0001" class="usa-link">S5B</a>; Table <a href="#tpg270067-supinfo-0001" class="usa-link">S1</a>), which was also present in the raw sequencing data (Figure <a href="#tpg270067-supinfo-0001" class="usa-link">S5A</a>). Population summary metrics found that most SNPs contained at least one alternative allele, while homozygous reference alleles were the second most common GT. The average minor‐allele frequency across all markers was 0.249 (Figure <a href="#tpg270067-supinfo-0001" class="usa-link">S6A,B</a>). Consistent with the prevalence of heterozygous SNPs, the average observed heterozygosity for the alfalfa samples was 0.63 (Figure <a href="#tpg270067-supinfo-0001" class="usa-link">S6C</a>).</p></section><section id="tpg270067-sec-0130"><h3 class="pmc_sec_title">3.2. Structure of the creeping rooted breeding population</h3>
<p>Population structure within the breeding population was visualized with a PCA to evaluate the relationships between individuals depending on their parental crosses. Moderate clustering due to the parental cross shows three distinct clusters, with the Green progeny being one, the Orange, Yellow, and White progeny being another, and the final distinct cluster containing the Red, Pink, and Blue cross progeny (Figure <a href="#tpg270067-fig-0003" class="usa-link">3</a>). This clustering pattern matches the spreading parental GT (either Entry 3 or Entry 8), with a mixture of both Entry 3 and Entry 8 progeny in the Green cross cluster, and the parental GTs distributed between the clusters (Figure <a href="#tpg270067-fig-0003" class="usa-link">3A</a>). The Yellow, Orange, and White progeny separate into distinct clusters when comparing PC1 to PC3 (Figure <a href="#tpg270067-fig-0003" class="usa-link">3B</a>). The remaining samples did not belong to an identifiable cluster based on parental cross (Figure <a href="#tpg270067-fig-0003" class="usa-link">3</a>), which could be potentially caused by pollen contamination when making crosses.</p>
<figure class="fig xbox font-sm" id="tpg270067-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f650/12245737/e1f5cdd8975a/TPG2-18-e70067-g002.jpg" loading="lazy" id="jats-graphic-5" height="340" width="700" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270067-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Principal component analysis of the alfalfa creeping root breeding population (<em>n</em> = 648). (A) Genetic relations plotted for PC1 and PC2 dimensions. (B) Genetic relations plotted for PC1 and PC3 dimensions. Each point on the plot is an individual plant, and the colors/shapes are indicative of their parental cross. The labels denoted with “*” were the parental/check plants. The progeny derives from the following crosses: Orange, White, and Yellow (Entry 8 × CT3‐08); Blue, Pink, and Red (Entry 3 × CT3‐08); and Green (Entry 3 × Entry 8). (A) PC1 versus PC2 and (B) PC1 versus PC3.</p></figcaption></figure></section><section id="tpg270067-sec-0140"><h3 class="pmc_sec_title">3.3. QTL and candidate genes associated with creeping root</h3>
<p>A GWAS analysis was performed to identify genomic regions significantly associated with the creeping rooted trait. The filtered 2211 SNPs and 417 selected samples were used in the association analysis (Figure <a href="#tpg270067-supinfo-0001" class="usa-link">S2</a>). Using the phenotypic BLUE values, eight unique SNPs (Table <a href="#tpg270067-supinfo-0001" class="usa-link">S2</a>) were found to be associated with the creeping root trait using six genetic effect models. A basic local alignment search tool analysis of microhaplotypes (54–81 bp in length) harboring significant spread trait‐associated SNPs revealed that the sequence harboring chr6.1_016478433 had a single copy in each of the four sub‐genomes, but these copies were not located in syntenic regions. Additionally, chr6.1_016478433 showed moderate LD (<em>r</em>
<sup>2</sup> = 0.41) with the peak SNP (chr6.1_071635255) near the terminus of Chromosome 6.1, indicating potential misplacement of chr6.1_016478433. Therefore, chr6.1_016478433 was removed from the GT data, and the GWAS analysis was refined accordingly (Figure <a href="#tpg270067-fig-0004" class="usa-link">4</a>).</p>
<figure class="fig xbox font-sm" id="tpg270067-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f650/12245737/b8c2e65e9015/TPG2-18-e70067-g001.jpg" loading="lazy" id="jats-graphic-7" height="277" width="700" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270067-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Manhattan plot for the creeping root trait across all genome‐wide association study (GWAS) models. 2210 single‐nucleotide polymorphisms (SNPs) were evaluated with 417 progeny using GWASpoly. The M.eff significance threshold is the horizontal dashed line. SNPs above that line were significantly associated with the creeping root trait.</p></figcaption></figure><p>Application of an 8.575 Mb LD‐pruning window identified three distinct QTLs across five GWAS models (Table <a href="#tpg270067-tbl-0002" class="usa-link">2</a>). These include one major QTL located on Chromosome 6.1 and two minor QTLs: one on Chromosome 2.1 (1‐dom model) and another on Chromosome 3.1 (diplo‐general model) (Table <a href="#tpg270067-tbl-0002" class="usa-link">2</a>). The major QTL on Chromosome 6.1 was significant in all GWAS models except the 2‐dom model (Table <a href="#tpg270067-tbl-0002" class="usa-link">2</a>) and explained 9.09% of the phenotypic variance in the breeding population under the additive model. Progeny (<em>n</em> = 263) that were homozygous of the reference allele (GGGG) at the peak SNP (chr6.1_071635255) showed an average spreading root rating of 5.53, while progeny (<em>n</em> = 153) with one alternative allele (GGGA) showed an average rating of 8.12, indicating that the alternative allele is favorable for the creeping root trait. The peak SNP was located within a gene annotated as exonuclease 1 (MS.gene58914). Examination of the coding sequence revealed that the SNP occurs in the third exon, resulting in a synonymous mutation (CTG to TTG) that maintains leucine as the encoded amino acid.</p>
<section class="tw xbox font-sm" id="tpg270067-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Quantitative trait loci (QTL) associated with alfalfa creeping rootedness.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Marker</th>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">Threshold</th>
<th align="left" rowspan="1" colspan="1">Chrom</th>
<th align="left" rowspan="1" colspan="1">Position</th>
<th align="left" rowspan="1" colspan="1">Ref</th>
<th align="left" rowspan="1" colspan="1">Alt</th>
<th align="left" rowspan="1" colspan="1">Score</th>
<th align="left" rowspan="1" colspan="1">Effect</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">chr6.1_071635255</td>
<td align="left" rowspan="1" colspan="1">Additive</td>
<td align="left" rowspan="1" colspan="1">4.47</td>
<td align="left" rowspan="1" colspan="1">chr6.1</td>
<td align="left" rowspan="1" colspan="1">71,635,255</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">8.85</td>
<td align="left" rowspan="1" colspan="1">1.260</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr6.1_071635255</td>
<td align="left" rowspan="1" colspan="1">General</td>
<td align="left" rowspan="1" colspan="1">4.47</td>
<td align="left" rowspan="1" colspan="1">chr6.1</td>
<td align="left" rowspan="1" colspan="1">71,635,255</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">7.98</td>
<td align="left" rowspan="1" colspan="1">NA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr3.1_065262400</td>
<td align="left" rowspan="1" colspan="1">Diplo‐general</td>
<td align="left" rowspan="1" colspan="1">4.47</td>
<td align="left" rowspan="1" colspan="1">chr3.1</td>
<td align="left" rowspan="1" colspan="1">65,262,400</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">4.76</td>
<td align="left" rowspan="1" colspan="1">NA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr6.1_071635255</td>
<td align="left" rowspan="1" colspan="1">diplo‐general</td>
<td align="left" rowspan="1" colspan="1">4.47</td>
<td align="left" rowspan="1" colspan="1">chr6.1</td>
<td align="left" rowspan="1" colspan="1">71,635,255</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">8.84</td>
<td align="left" rowspan="1" colspan="1">NA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr6.1_071635255</td>
<td align="left" rowspan="1" colspan="1">diplo‐additive</td>
<td align="left" rowspan="1" colspan="1">4.47</td>
<td align="left" rowspan="1" colspan="1">chr6.1</td>
<td align="left" rowspan="1" colspan="1">71,635,255</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">8.84</td>
<td align="left" rowspan="1" colspan="1">1.277</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr6.1_071635255</td>
<td align="left" rowspan="1" colspan="1">1‐dom‐alt</td>
<td align="left" rowspan="1" colspan="1">4.4</td>
<td align="left" rowspan="1" colspan="1">chr6.1</td>
<td align="left" rowspan="1" colspan="1">71,635,255</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">8.84</td>
<td align="left" rowspan="1" colspan="1">1.277</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chr2.1_064883289</td>
<td align="left" rowspan="1" colspan="1">1‐dom‐ref</td>
<td align="left" rowspan="1" colspan="1">4.07</td>
<td align="left" rowspan="1" colspan="1">chr2.1</td>
<td align="left" rowspan="1" colspan="1">64,883,289</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">4.15</td>
<td align="left" rowspan="1" colspan="1">−1.077</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270067-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270067-tbl2-note-0001"><p>
<em>Note</em>: Significant markers from each of the evaluated traits from GWASpoly were filtered to remove linked markers within an 8.575 Mb window, and the most significant marker in each linkage block was retained. Markers were filtered for linkage disequilibrium (LD) separately within each GWASpoly genetic model.</p></div>
<div class="fn" id="tpg270067-tbl2-note-0002"><p>Abbreviations: Alt, alternative allele; Ref, reference allele.</p></div>
</div></section></section><section id="tpg270067-sec-0150"><h3 class="pmc_sec_title">3.4. GS with the creeping root traits</h3>
<p>The predictive ability of a fivefold cross‐validation selection model was estimated to determine the feasibility of using GS for creeping rootedness. We found that creeping rootedness had a moderate predictive ability with a mean value of 0.68 (Figure <a href="#tpg270067-supinfo-0001" class="usa-link">S7</a>).</p></section></section><section id="tpg270067-sec-0160"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>The goals of this study were to evaluate the creeping root trait in alfalfa to identify QTL and evaluate the performance of this trait for GS. To achieve this, we genotyped 648 alfalfa samples from a creeping root breeding population using the alfalfa 3K DArTag panel, resulting in 2211 high‐quality SNPs (including both target and off‐target SNPs). From the 648 genotyped alfalfa plants, 417 progeny were selected based on their clustering in a PCA for GWAS and GS analyses. We identified three QTLs that were significantly associated with creeping rootedness, with one major QTL that explained 9% of the phenotypic variance. GS showed a moderate predictive ability with a mean value of 0.68. To facilitate these analyses, we developed the Breeding Insight Genomics R Shiny Application (BIGapp) to provide a coding‐free way for breeders to evaluate genomic data and assist in making selection decisions.</p>
<section id="tpg270067-sec-0170"><h3 class="pmc_sec_title">4.1. Creeping root breeding population</h3>
<p>We found moderate levels of observed heterozygosity among the alfalfa samples and a medium average (0.249) MAF among the filtered markers. Compared to wild accessions, originating from Central Asia, northeastern Europe, Balkans‐Turkey‐Black Sea, and Siberia/Mongolia, within the USDA National Plant Germplasm System, the average heterozygosity of the creeping root breeding population was more than twice the average observed heterozygosity of each of these Eurasian alfalfas (Medina, Zhao, et al., <a href="#tpg270067-bib-0042" class="usa-link" aria-describedby="tpg270067-bib-0042">2024</a>). This is likely due to our sampled population intentionally maintaining higher diversity through use of both the <em>M. sativa</em> subsp. <em>sativa</em> and <em>M. sativa</em> subsp. <em>falcata</em> types. Marker distribution was genome‐wide with a deficiency of SNPs on Chromosome 6.1 in both the raw and filtered datasets. This is likely due to a known limitation of the panel design, as there is sparse marker coverage on the short arm near the centromere of Chromosome 6.1, and similar gaps are present in a previous alfalfa study that used the DArTag panel (D. Zhao et al., <a href="#tpg270067-bib-0065" class="usa-link" aria-describedby="tpg270067-bib-0065">2024</a>).</p>
<p>The PCA analysis showed moderate clustering based on the parental crosses, with the parental and check GTs distributed between the progeny clusters. Most of the progeny with shared parentage clustered within the same groups, suggesting that population structure is influenced by the genetic background of the parental lines. However, several outliers among the progeny deviated from the major progeny clusters based on putative pedigree and were subsequently excluded from the GWAS and GS analysis.</p></section><section id="tpg270067-sec-0180"><h3 class="pmc_sec_title">4.2. QTL and their associated creeping root genes</h3>
<p>Three QTLs were found to be significantly associated with creeping root in a GWAS using an intercross population originating from three parental lines. One major QTL was identified on Chromosome 6.1 with the alternative allele (A) of the peak SNP (chr6.1_071635255) as the favorable allele, showing elevated allele frequency in both spreading parents (Entry 3 and Entry 8). Initial GWAS analysis identified chr6.1_016478433 as a minor‐effect QTL in addition to the major QTL on Chromosome 6. However, LD analysis revealed a moderate correlation (<em>r</em>
<sup>2</sup> = 0.41) between chr6.1_016478433 and chr6.1_071635255, suggesting these two markers likely belong to the same LD block. The position of chr6.1_016478433 may be incorrectly placed due to genome assembly challenges in the first alfalfa sub‐genome. Given this positional uncertainty and our current resources, we decided to exclude chr6.1_016478433 in the second round of GWAS.</p>
<p>The peak SNP (chr6.1_071635255) of the major QTL is located within the putative creeping root gene, MS.gene58914, which was found to be expressed in root tissue in an alfalfa RNAseq study (Miller et al., <a href="#tpg270067-bib-0043" class="usa-link" aria-describedby="tpg270067-bib-0043">2022</a>), supporting that this gene could influence the differential growth in creeping root alfalfa. Exonucleases are essential enzymes that play critical roles in cellular metabolism and genome stability by cleaving DNA from free ends, supporting various cellular maintenance processes. Although no direct link between exonuclease 1 (MS.gene58914) and root growth has been established in the current literature, a reduction in this gene could increase mutation rates during DNA transcription. The SNP in exonuclease 1 does not alter the amino acid sequence, but the codon change from CTG to TTG could significantly impact translation efficiency. Studies in rice have shown a strong codon usage bias, with the CTG codon for leucine being six times more prevalent than TTG in actively translating ribosome complexes, implying that the TTG codon may have lower translation efficiency (Zhao et al., <a href="#tpg270067-bib-0063" class="usa-link" aria-describedby="tpg270067-bib-0063">2017</a>). While further investigation into this gene and its mutants would be necessary to determine its potential role in creeping rootedness in alfalfa populations, its association with the trait may still be valuable for breeding purposes and allele tracking in crosses.</p>
<p>In addition to identifying the gene within which the peak QTL was located, we also report the associated genes that were within the 8.575 Mb linkage window of the three QTLs (Supporting Information <a href="#tpg270067-supinfo-0001" class="usa-link">4</a>). However, exploration of all the associated genes within the QTL window is beyond the scope of this paper. As such, any errors in unannotated regions from the reference genome would lead to undiscovered SNP‐gene associations. It should be noted that while we identify candidate QTL and genes associated with creeping root in alfalfa, we may not have located the causal QTL or genes due to the reduced genome representation of the mid‐density DArTag panel. Rather, we have identified candidate regions for further exploration or used them for targeted breeding decisions to introgress the creeping rootedness trait. Future studies into the identified regions with a significant association with the creeping root trait should focus on (1) fine mapping of the major Chromosome 6.1 QTL to narrow down the region and identify the causal genes and (2) validation of the putatively causal gene through gene expression, overexpression, or knockout experiments. These follow‐up analyses can validate the regions that influence the creeping root trait, leading to a more targeted breeding effort and a better understanding of this phenotype.</p></section><section id="tpg270067-sec-0190"><h3 class="pmc_sec_title">4.3. Implications for creeping root in alfalfa breeding and genomic prediction</h3>
<p>The QTL we found related to creeping root could be used to inform future alfalfa breeding efforts and expedite breeding cycles. Our finding of a major QTL (and two additional minor QTLs) that contribute to the creeping root trait in this intercross population suggests that this trait can be incorporated into other breeding populations quickly. This integration of these QTLs into existing breeding programs allows for the simultaneous selection of the creeping root trait alongside other desirable traits, such as flowering date and yield (Adhikari et al., <a href="#tpg270067-bib-0001" class="usa-link" aria-describedby="tpg270067-bib-0001">2019</a>; He et al., <a href="#tpg270067-bib-0024" class="usa-link" aria-describedby="tpg270067-bib-0024">2020</a>; Zhang et al., <a href="#tpg270067-bib-0062" class="usa-link" aria-describedby="tpg270067-bib-0062">2019</a>), ultimately leading to the development of improved elite cultivars. Furthermore, the identification of these candidate QTLs enables MAS as a practical approach to improve selection decisions and screening tools in alfalfa (Li &amp; Brummer, <a href="#tpg270067-bib-0033" class="usa-link" aria-describedby="tpg270067-bib-0033">2012</a>).</p>
<p>In this work, we found that using a genomic best linear unbiased prediction GS model revealed a moderate predictive ability for the creeping root trait. We employed a fivefold cross‐validation method for estimating the predictive ability of the creeping root trait, and this could likely be optimized using alternative models, testing different marker filtering options, and weighting trait‐associated markers (Alemu et al., <a href="#tpg270067-bib-0002" class="usa-link" aria-describedby="tpg270067-bib-0002">2024</a>; Heslot et al., <a href="#tpg270067-bib-0027" class="usa-link" aria-describedby="tpg270067-bib-0027">2012</a>; Medina et al., <a href="#tpg270067-bib-0041" class="usa-link" aria-describedby="tpg270067-bib-0041">2021</a>). Our use of a mid‐density DArTag panel could have led to slightly conservative estimates for predictive ability, but is likely comparable to using higher density genotyping methods (Simeão Resende et al., <a href="#tpg270067-bib-0055" class="usa-link" aria-describedby="tpg270067-bib-0055">2014</a>; VanRaden, <a href="#tpg270067-bib-0059" class="usa-link" aria-describedby="tpg270067-bib-0059">2020</a>). We have demonstrated that GS is suitable for predicting the phenotype of alfalfa samples for the creeping root trait, particularly since it is likely polygenic. Future iterations of BIGapp will include support for additional GS methods. Our results demonstrate that the GS implementation in BIGapp currently can streamline the breeding process, ultimately contributing to the development of improved alfalfa cultivars with desirable traits.</p></section><section id="tpg270067-sec-0200"><h3 class="pmc_sec_title">4.4. Utility of BIGapp for data analysis</h3>
<p>BIGapp was developed to lower the barriers of integrating genomics into breeding programs and provide accessible genomic pipelines for breeding decisions. As shown in this study, its key feature is the support of both diploid and polyploid species, allowing breeders to preprocess, analyze, and visualize genomic data without requiring coding skills. Importantly, each analysis tab is independent, allowing users to utilize VCF files generated from various pipelines or platforms as long as they contain GT or read count information. Alternatively, breeders can process genomic information within BIGapp and export it as a VCF file for use in other pipelines. Many of the custom functions within BIGapp are also available in the BIGr R package, catering to experienced R users who want to incorporate these functions into their own bioinformatic pipelines. Comprehensive training materials, including step‐by‐step tutorials and in‐app guides, are provided to enhance user engagement and facilitate the interpretation of results. The development of BIGapp represents a significant advancement in making genomic data analysis accessible to non‐experts in coding across diverse species and ploidy levels. By packaging bioinformatic pipelines in a reproducible and user‐friendly interface, BIGapp promotes standardized approaches for analyzing diploid and polyploid breeding data in a centralized location.</p></section></section><section id="tpg270067-sec-0210"><h2 class="pmc_sec_title">5. CONCLUSION</h2>
<p>In summary, the alfalfa creeping root trait is crucial for alfalfa populations, as it enhances persistence over the long term when facing grazing pressures and thereby increases productivity. We have demonstrated that one major QTL predominantly influences the creeping root trait in this alfalfa intercross population and that its moderate predictive ability makes it feasible to use GS to augment alfalfa breeding progress. The markers used in this study were obtained from the alfalfa 3K DArTag panel and can be reproducibly sequenced in other alfalfa populations. Lastly, the BIGapp R Shiny application and BIGr R package lower the barriers for integrating genomics into breeding programs regardless of the species’ ploidy or breeder's coding expertise.</p></section><section id="tpg270067-sec-0220"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Alexander M. Sandercock</strong>: Conceptualization; data curation; formal analysis; methodology; software; visualization; writing—original draft; writing—review and editing. <strong>Michael D. Peel</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; resources; supervision; writing—original draft; writing—review and editing. <strong>Cristiane H. Taniguti</strong>: Data curation; formal analysis; methodology; software; validation; visualization; writing—review and editing. <strong>Josué Chinchilla‐Vargas</strong>: Formal analysis; investigation; software; validation; visualization. <strong>Shufen Chen</strong>: Data curation; formal analysis; methodology; software; validation; visualization. <strong>Manoj Sapkota</strong>: Data curation; formal analysis; methodology; resources; software; supervision; validation; visualization. <strong>Meng Lin</strong>: Formal analysis; investigation; project administration; software; supervision; validation; visualization. <strong>Dongyan Zhao</strong>: Investigation; methodology; project administration; software; supervision; validation; visualization; writing—review and editing. <strong>Arlyn J. Ackerman</strong>: Data curation; formal analysis; methodology; validation. <strong>Bhoja R. Basnet</strong>: Data curation; formal analysis; methodology; validation; visualization. <strong>Craig T. Beil</strong>: Project administration; resources; supervision; writing—review and editing. <strong>Moira J. Sheehan</strong>: Funding acquisition; project administration; resources; supervision; writing—review and editing.</p></section><section id="tpg270067-sec-0240"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec24"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270067-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270067-supitem-0001"><div class="caption p">
<p>
<strong>Supplemental Table 1</strong>: Summary of the distribution of the filtered 2,211 SNPs.</p>
<p>
<strong>Supplemental Table 2</strong>: Significant markers associated with alfalfa creeping rootedness.</p>
<p>
<strong>Supplemental Figure 1</strong>: Summary of the genetic backgrounds of 800 plants evaluated for creeping rootedness.</p>
<p>
<strong>Supplemental Figure 2</strong>: PCA with the progeny labelled for their inclusion in the GWAS and GS analysis.</p>
<p>
<strong>Supplemental Figure 3</strong>: GWAS diagnostic plots.</p>
<p>
<strong>Supplemental Figure 4</strong>: Linkage‐disequilibrium decay plot for the filtered markers.</p>
<p>
<strong>Supplemental Figure 5</strong>: Distribution of the alfalfa SNPs.</p>
<p>
<strong>Supplemental Figure 6</strong>: Genomic diversity of the alfalfa creeping root breeding population.</p>
<p>
<strong>Supplemental Figure 7</strong>: Box plot of the 10 iterations for the GS GBLUP model evaluation.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12245737/bin/TPG2-18-e70067-s003.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70067-s003.docx</a><sup> (1.7MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270067-supitem-0002"><div class="caption p"><p>
<strong>Supplemental File 1</strong>: Raw phenotypic and BLUE data for the alfalfa samples.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12245737/bin/TPG2-18-e70067-s001.zip" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70067-s001.zip</a><sup> (79.2KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270067-supitem-0003"><div class="caption p"><p>
<strong>Supplemental File 2</strong>: Raw DArTag MADC File.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12245737/bin/TPG2-18-e70067-s002.zip" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70067-s002.zip</a><sup> (6MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270067-supitem-0004"><div class="caption p"><p>
<strong>Supplemental File 3</strong>: Filtered VCF file of the 2,211 biallelic SNPs and 648 samples.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12245737/bin/TPG2-18-e70067-s005.zip" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70067-s005.zip</a><sup> (13.5MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270067-supitem-0005"><div class="caption p"><p>
<strong>Supplemental File 4</strong>: Genes associated with the three GWAS identified QTL based on a 8.575 Mb LD window.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12245737/bin/TPG2-18-e70067-s004.zip" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70067-s004.zip</a><sup> (41.5KB, zip) </sup>
</div></div></section></section><section id="tpg270067-sec-0230" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We thank Alexandra Casa for reviewing the phenotypic data, Kristen Lind for assisting with the development of the BIGapp training materials, and our colleagues at Breeding Insight for their review and suggestions for the BIGapp. We thank DArT for genotyping the alfalfa accessions. These materials are based upon efforts from Breeding Insight (RID: SCR_026645), a USDA‐ARS initiative hosted by Cornell University. The work is supported by the US Department of Agriculture under agreement numbers (<a href="http://8062-21000-043-004-A" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8062‐21000‐043‐004‐A</a>, <a href="http://8062-21000-052-002-A" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8062‐21000‐052‐002‐A</a>, and <a href="http://8062-21000-052-003-A" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8062‐21000‐052‐003‐A</a>). Any opinions, findings, conclusions, or recommendations expressed in this publication are those of the author(s) and do not necessarily reflect the views of the U.S. Department of Agriculture. In addition, any reference to specific brands or types of products or services does not constitute or imply an endorsement by the US Department of Agriculture for those products or services.</p></section><section id="notes1"><p>


Sandercock, A. M.
, 
Peel, M. D.
, 
Taniguti, C. H.
, 
Chinchilla‐Vargas, J.
, 
Chen, S.
, 
Sapkota, M.
, 
Lin, M.
, 
Zhao, D.
, 
Ackerman, A. J.
, 
Basnet, B. R.
, 
Beil, C. T.
, &amp; 
Sheehan, M. J.
 (2025). BIGapp: A user‐friendly genomic tool kit identified quantitative trait loci for creeping rootedness in alfalfa (<em>Medicago sativa</em> L.). The Plant Genome, 18, e70067. 10.1002/tpg2.70067


</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="tpg270067-note-0001"><p>Assigned to Associate Editor Nonoy Bandillo.</p></div>
<div class="fn p" id="tpg270067-note-0002"><p>Alexander M. Sandercock and Michael D. Peel contributed equally to this work.</p></div>
</div></section></section><section id="tpg270067-sec-0260"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>Genomic and phenotypic data files are included as Supporting Information: BLUE calculations: <a href="https://github.com/Breeding-Insight/alfalfa_BLUE" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Breeding‐Insight/alfalfa_BLUE</a>; BIGr (RRID: SCR_026677): <a href="https://CRAN.R-project.org/package=BIGr" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R‐project.org/package=BIGr</a>; BIGapp (RRID: SCR_026676): <a href="https://github.com/Breeding-Insight/BIGapp" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Breeding‐Insight/BIGapp</a>. BIGapp is available on Sci‐Net for USDA‐ARS scientists.</p></section><section id="tpg270067-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270067-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270067-bib-0001">
<cite>

Adhikari, L.
, 
Makaju, S. O.
, &amp; 
Missaoui, A. M.
 (2019). QTL mapping of flowering time and biomass yield in tetraploid alfalfa (<em>Medicago sativa</em> L.). BMC Plant Biology, 19, 359. 10.1186/s12870-019-1946-0

</cite> [<a href="https://doi.org/10.1186/s12870-019-1946-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6697951/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31419945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=QTL%20mapping%20of%20flowering%20time%20and%20biomass%20yield%20in%20tetraploid%20alfalfa%20(Medicago%20sativa%20L.)&amp;author=L.%20Adhikari&amp;author=S.%20O.%20Makaju&amp;author=A.%20M.%20Missaoui&amp;volume=19&amp;publication_year=2019&amp;pages=359&amp;pmid=31419945&amp;doi=10.1186/s12870-019-1946-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0002">
<cite>

Alemu, A.
, 
Åstrand, J.
, 
Montesinos‐López, O. A.
, 
Isidro Y Sánchez, J.
, 
Fernández‐Gónzalez, J.
, 
Tadesse, W.
, 
Vetukuri, R. R.
, 
Carlsson, A. S.
, 
Ceplitis, A.
, 
Crossa, J.
, 
Ortiz, R.
, &amp; 
Chawade, A.
 (2024). Genomic selection in plant breeding: Key factors shaping two decades of progress. Molecular Plant, 17, 552–578. 10.1016/j.molp.2024.03.007

</cite> [<a href="https://doi.org/10.1016/j.molp.2024.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38475993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=Genomic%20selection%20in%20plant%20breeding:%20Key%20factors%20shaping%20two%20decades%20of%20progress&amp;author=A.%20Alemu&amp;author=J.%20%C3%85strand&amp;author=O.%20A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=J.%20Isidro%20Y%20S%C3%A1nchez&amp;author=J.%20Fern%C3%A1ndez%E2%80%90G%C3%B3nzalez&amp;volume=17&amp;publication_year=2024&amp;pages=552-578&amp;pmid=38475993&amp;doi=10.1016/j.molp.2024.03.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0003">
<cite>

Amadeu, R. R.
, 
Garcia, A. A. F.
, &amp; 
Munoz, P. R.
 (2023). AGHmatrix: Genetic relationship matrices in R. Bioinformatics, 39(7), btad445.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btad445" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10371492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37471595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=AGHmatrix:%20Genetic%20relationship%20matrices%20in%20R&amp;author=R.%20R.%20Amadeu&amp;author=A.%20A.%20F.%20Garcia&amp;author=P.%20R.%20Munoz&amp;volume=39&amp;issue=7&amp;publication_year=2023&amp;pages=btad445&amp;pmid=37471595&amp;doi=10.1093/bioinformatics/btad445&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0004">
<cite>

Aparicio, J.
, 
Gezan, S. A.
, 
Ariza‐Suarez, D.
, 
Raatz, B.
, 
Diaz, S.
, 
Heilman‐Morales, A.
, &amp; 
Lobaton, J.
 (2023). Mr.Bean: A comprehensive statistical and visualization application for modeling agricultural field trials data. Frontiers in Plant Science, 14, 1290078. 10.3389/fpls.2023.1290078

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1290078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10792065/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38235208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Mr.Bean:%20A%20comprehensive%20statistical%20and%20visualization%20application%20for%20modeling%20agricultural%20field%20trials%20data&amp;author=J.%20Aparicio&amp;author=S.%20A.%20Gezan&amp;author=D.%20Ariza%E2%80%90Suarez&amp;author=B.%20Raatz&amp;author=S.%20Diaz&amp;volume=14&amp;publication_year=2023&amp;pages=1290078&amp;pmid=38235208&amp;doi=10.3389/fpls.2023.1290078&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0005">
<cite>

Berdahl, J. D.
, 
Wilton, A. C.
, &amp; 
Frank, A. B.
 (1989). Survival and agronomic performance of 25 alfalfa cultivars and strains interseeded into rangeland. Rangeland Ecology &amp; Management/Journal of Range Management Archives, 42, 312–316.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Rangeland%20Ecology%20&amp;%20Management/Journal%20of%20Range%20Management%20Archives&amp;title=Survival%20and%20agronomic%20performance%20of%2025%20alfalfa%20cultivars%20and%20strains%20interseeded%20into%20rangeland&amp;author=J.%20D.%20Berdahl&amp;author=A.%20C.%20Wilton&amp;author=A.%20B.%20Frank&amp;volume=42&amp;publication_year=1989&amp;pages=312-316&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0006">
<cite>

Berdahl, J. D.
, 
Wilton, A. C.
, 
Lorenz, R. J.
, &amp; 
Frank, A. B.
 (1986). Alfalfa survival and vigor in rangeland grazed by sheep. Rangeland Ecology &amp; Management/Journal of Range Management Archives, 39, 59–62.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Rangeland%20Ecology%20&amp;%20Management/Journal%20of%20Range%20Management%20Archives&amp;title=Alfalfa%20survival%20and%20vigor%20in%20rangeland%20grazed%20by%20sheep&amp;author=J.%20D.%20Berdahl&amp;author=A.%20C.%20Wilton&amp;author=R.%20J.%20Lorenz&amp;author=A.%20B.%20Frank&amp;volume=39&amp;publication_year=1986&amp;pages=59-62&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0007">
<cite>

Boe, A.
, 
Kephart, K. D.
, 
Berdahl, J. D.
, 
Peel, M. D.
, 
Brummer, E. C.
, 
Xu, L.
, &amp; 
Wu, Y.
 (2020). Breeding alfalfa for semiarid regions in the northern Great Plains: History and additional genetic evaluations of novel germplasm. Agronomy, 10, 1686. 10.3390/agronomy10111686
</cite> [<a href="https://doi.org/10.3390/agronomy10111686" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=Breeding%20alfalfa%20for%20semiarid%20regions%20in%20the%20northern%20Great%20Plains:%20History%20and%20additional%20genetic%20evaluations%20of%20novel%20germplasm&amp;author=A.%20Boe&amp;author=K.%20D.%20Kephart&amp;author=J.%20D.%20Berdahl&amp;author=M.%20D.%20Peel&amp;author=E.%20C.%20Brummer&amp;volume=10&amp;publication_year=2020&amp;pages=1686&amp;doi=10.3390/agronomy10111686&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0008">
<cite>

Brummer, E. C.
, &amp; 
Bouton, J. H.
 (1991). Plant traits associated with grazing‐tolerant alfalfa. Agronomy Journal, 83, 996–1000. 10.2134/agronj1991.00021962008300060014x
</cite> [<a href="https://doi.org/10.2134/agronj1991.00021962008300060014x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy%20Journal&amp;title=Plant%20traits%20associated%20with%20grazing%E2%80%90tolerant%20alfalfa&amp;author=E.%20C.%20Brummer&amp;author=J.%20H.%20Bouton&amp;volume=83&amp;publication_year=1991&amp;pages=996-1000&amp;doi=10.2134/agronj1991.00021962008300060014x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0009">
<cite>

Brummer, E. C.
, 
Cazcarro, P. M.
, &amp; 
Luth, D.
 (1999). Ploidy determination of alfalfa germplasm accessions using flow cytometry. Crop Science, 39, 1202–1207. 10.2135/cropsci1999.0011183X003900040041x
</cite> [<a href="https://doi.org/10.2135/cropsci1999.0011183X003900040041x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Ploidy%20determination%20of%20alfalfa%20germplasm%20accessions%20using%20flow%20cytometry&amp;author=E.%20C.%20Brummer&amp;author=P.%20M.%20Cazcarro&amp;author=D.%20Luth&amp;volume=39&amp;publication_year=1999&amp;pages=1202-1207&amp;doi=10.2135/cropsci1999.0011183X003900040041x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0010">
<cite>

Butler, D. G.
, 
Cullis, B. R.
, 
Gilmour, A. R.
, &amp; 
Gogel, B. J.
 (2018). 
ASReml estimates variance components under a general linear mixed model by residual maximum likelihood (REML)
. VSN International Ltd.</cite> [<a href="https://scholar.google.com/scholar_lookup?Butler,%20D.%20G.%20,%20Cullis,%20B.%20R.%20,%20Gilmour,%20A.%20R.%20,%20&amp;%20Gogel,%20B.%20J.%20(2018).%20ASReml%20estimates%20variance%20components%20under%20a%20general%20linear%20mixed%20model%20by%20residual%20maximum%20likelihood%20(REML)%20.%20VSN%20International%20Ltd." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0011">
<cite>

Castonguay, Y.
, 
Laberge, S.
, 
Brummer, E. C.
, &amp; 
Volenec, J. J.
 (2006). Alfalfa winter hardiness: A research retrospective and integrated perspective. In Frey K. J., Kamprath E. J., &amp; Alexander M. (Eds.), Advances in agronomy (pp. 203–265). Elsevier.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20agronomy&amp;author=Y.%20Castonguay&amp;author=S.%20Laberge&amp;author=E.%20C.%20Brummer&amp;author=J.%20J.%20Volenec&amp;publication_year=2006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0012"><cite>

Chang, W.
, 
Cheng, J.
, 
Allaire, J. J.
, 
Sievert, C.
, 
Schloerke, B.
, 
Xie, Y.
, 
Allen, J.
, 
McPherson, J.
, 
Dipert, A.
, &amp; 
Borges, B.
 (2024). 
shiny: Web application framework for R
. <a href="https://CRAN.R-project.org/package=shiny" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R‐project.org/package=shiny</a>
</cite></li>
<li id="tpg270067-bib-0013">
<cite>

Chen, H.
, 
Zeng, Y.
, 
Yang, Y.
, 
Huang, L.
, 
Tang, B.
, 
Zhang, H.
, 
Hao, F.
, 
Liu, W.
, 
Li, Y.
, 
Liu, Y.
, 
Zhang, X.
, 
Zhang, R.
, 
Zhang, Y.
, 
Li, Y.
, 
Wang, K.
, 
He, H.
, 
Wang, Z.
, 
Fan, G.
, 
Yang, H.
, … 
Qiu, Q.
 (2020). Allele‐aware chromosome‐level genome assembly and efficient transgene‐free genome editing for the autotetraploid cultivated alfalfa. Nature Communications, 11, 1–11. 10.1038/s41467-020-16338-x
</cite> [<a href="https://doi.org/10.1038/s41467-020-16338-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7237683/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32427850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Allele%E2%80%90aware%20chromosome%E2%80%90level%20genome%20assembly%20and%20efficient%20transgene%E2%80%90free%20genome%20editing%20for%20the%20autotetraploid%20cultivated%20alfalfa&amp;author=H.%20Chen&amp;author=Y.%20Zeng&amp;author=Y.%20Yang&amp;author=L.%20Huang&amp;author=B.%20Tang&amp;volume=11&amp;publication_year=2020&amp;pages=1-11&amp;pmid=32427850&amp;doi=10.1038/s41467-020-16338-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0014">
<cite>

Chen, T.
, 
Tang, C.
, 
Zheng, W.
, 
Qian, Y.
, 
Chen, M.
, 
Zou, Q.
, 
Jin, Y.
, 
Wang, K.
, 
Zhou, X.
, 
Gou, S.
, &amp; 
Lai, L.
 (2023). VCFshiny: An R/Shiny application for interactively analyzing and visualizing genetic variants. Bioinformatics Advances, 3, vbad107. 10.1093/bioadv/vbad107

</cite> [<a href="https://doi.org/10.1093/bioadv/vbad107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10493178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37701675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics%20Advances&amp;title=VCFshiny:%20An%20R/Shiny%20application%20for%20interactively%20analyzing%20and%20visualizing%20genetic%20variants&amp;author=T.%20Chen&amp;author=C.%20Tang&amp;author=W.%20Zheng&amp;author=Y.%20Qian&amp;author=M.%20Chen&amp;volume=3&amp;publication_year=2023&amp;pages=vbad107&amp;pmid=37701675&amp;doi=10.1093/bioadv/vbad107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0015">
<cite>

Chinea, E.
, &amp; 
Arévalo, J. R.
 (2014). Effects of fertilization management on pasture productivity and nutrient composition. Grass and Forage Science: The Journal of the British Grassland Society, 69, 415–424. 10.1111/gfs.12057
</cite> [<a href="https://doi.org/10.1111/gfs.12057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Grass%20and%20Forage%20Science:%20The%20Journal%20of%20the%20British%20Grassland%20Society&amp;title=Effects%20of%20fertilization%20management%20on%20pasture%20productivity%20and%20nutrient%20composition&amp;author=E.%20Chinea&amp;author=J.%20R.%20Ar%C3%A9valo&amp;volume=69&amp;publication_year=2014&amp;pages=415-424&amp;doi=10.1111/gfs.12057&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0016">
<cite>

Cox, S.
, 
Peel, M. D.
, 
Creech, J. E.
, 
Waldron, B. L.
, 
Eun, J.‐S.
, 
Zobell, D. R.
, 
Miller, R. L.
, &amp; 
Snyder, D. L.
 (2017). Forage production of grass–legume binary mixtures on intermountain western USA irrigated pastures. Crop Science, 57, 1742–1753. 10.2135/cropsci2016.04.0235
</cite> [<a href="https://doi.org/10.2135/cropsci2016.04.0235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Forage%20production%20of%20grass%E2%80%93legume%20binary%20mixtures%20on%20intermountain%20western%20USA%20irrigated%20pastures&amp;author=S.%20Cox&amp;author=M.%20D.%20Peel&amp;author=J.%20E.%20Creech&amp;author=B.%20L.%20Waldron&amp;author=J.%E2%80%90S.%20Eun&amp;volume=57&amp;publication_year=2017&amp;pages=1742-1753&amp;doi=10.2135/cropsci2016.04.0235&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0017">
<cite>

Endelman, J. B.
 (2011). Ridge regression and other kernels for genomic selection with R package rrBLUP. The Plant Genome, 4, 250–255. 10.3835/plantgenome2011.08.0024
</cite> [<a href="https://doi.org/10.3835/plantgenome2011.08.0024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Ridge%20regression%20and%20other%20kernels%20for%20genomic%20selection%20with%20R%20package%20rrBLUP&amp;author=J.%20B.%20Endelman&amp;volume=4&amp;publication_year=2011&amp;pages=250-255&amp;doi=10.3835/plantgenome2011.08.0024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0018">
<cite>

Endelman, J. B.
 (2023). Fully efficient, two‐stage analysis of multi‐environment trials with directional dominance and multi‐trait genomic selection. Theoretical and Applied Genetics, 136, 65. 10.1007/s00122-023-04298-x

</cite> [<a href="https://doi.org/10.1007/s00122-023-04298-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10033618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36949348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Fully%20efficient,%20two%E2%80%90stage%20analysis%20of%20multi%E2%80%90environment%20trials%20with%20directional%20dominance%20and%20multi%E2%80%90trait%20genomic%20selection&amp;author=J.%20B.%20Endelman&amp;volume=136&amp;publication_year=2023&amp;pages=65&amp;pmid=36949348&amp;doi=10.1007/s00122-023-04298-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0019">
<cite>

Farrel, A.
, 
Li, P.
, 
Veenbergen, S.
, 
Patel, K.
, 
Maris, J. M.
, &amp; 
Leonard, W. J.
 (2023). ROGUE: An R Shiny app for RNA sequencing analysis and biomarker discovery. BMC Bioinformatics, 24, Article 303. 10.1186/s12859-023-05420-y

</cite> [<a href="https://doi.org/10.1186/s12859-023-05420-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10386769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37516886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=ROGUE:%20An%20R%20Shiny%20app%20for%20RNA%20sequencing%20analysis%20and%20biomarker%20discovery&amp;author=A.%20Farrel&amp;author=P.%20Li&amp;author=S.%20Veenbergen&amp;author=K.%20Patel&amp;author=J.%20M.%20Maris&amp;volume=24&amp;publication_year=2023&amp;pages=303&amp;pmid=37516886&amp;doi=10.1186/s12859-023-05420-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0020">
<cite>

Flajoulot, S.
, 
Ronfort, J.
, 
Baudouin, P.
, 
Barre, P.
, 
Huguet, T.
, 
Huyghe, C.
, &amp; 
Julier, B.
 (2005). Genetic diversity among alfalfa (<em>Medicago sativa</em>) cultivars coming from a breeding program, using SSR markers. Theoretical and Applied Genetics, 111, 1420–1429. 10.1007/s00122-005-0074-4

</cite> [<a href="https://doi.org/10.1007/s00122-005-0074-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16151797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genetic%20diversity%20among%20alfalfa%20(Medicago%20sativa)%20cultivars%20coming%20from%20a%20breeding%20program,%20using%20SSR%20markers&amp;author=S.%20Flajoulot&amp;author=J.%20Ronfort&amp;author=P.%20Baudouin&amp;author=P.%20Barre&amp;author=T.%20Huguet&amp;volume=111&amp;publication_year=2005&amp;pages=1420-1429&amp;pmid=16151797&amp;doi=10.1007/s00122-005-0074-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0021">
<cite>

Gerard, D.
 (2023). 
Updog: Flexible genotyping for polyploids
. Oxford University Press.</cite> [<a href="https://scholar.google.com/scholar_lookup?Gerard,%20D.%20(2023).%20Updog:%20Flexible%20genotyping%20for%20polyploids%20.%20Oxford%20University%20Press." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0022">
<cite>

Gerard, D.
, &amp; 
Ferrão, L. F. V.
 (2020). Priors for genotyping polyploids. Bioinformatics, 36, 1795–1800. 10.1093/bioinformatics/btz852

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz852" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32176767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Priors%20for%20genotyping%20polyploids&amp;author=D.%20Gerard&amp;author=L.%20F.%20V.%20Ferr%C3%A3o&amp;volume=36&amp;publication_year=2020&amp;pages=1795-1800&amp;pmid=32176767&amp;doi=10.1093/bioinformatics/btz852&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0023">
<cite>

Gerard, D.
, 
Ferrão, L. F. V.
, 
Garcia, A. A. F.
, &amp; 
Stephens, M.
 (2018). Genotyping polyploids from messy sequencing data. Genetics, 210, 789–807. 10.1534/genetics.118.301468

</cite> [<a href="https://doi.org/10.1534/genetics.118.301468" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6218231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30185430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genotyping%20polyploids%20from%20messy%20sequencing%20data&amp;author=D.%20Gerard&amp;author=L.%20F.%20V.%20Ferr%C3%A3o&amp;author=A.%20A.%20F.%20Garcia&amp;author=M.%20Stephens&amp;volume=210&amp;publication_year=2018&amp;pages=789-807&amp;pmid=30185430&amp;doi=10.1534/genetics.118.301468&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0024">
<cite>

He, F.
, 
Long, R.
, 
Zhang, T.
, 
Zhang, F.
, 
Wang, Z.
, 
Yang, X.
, 
Jiang, X.
, 
Yang, C.
, 
Zhi, X.
, 
Li, M.
, 
Yu, L.
, 
Kang, J.
, &amp; 
Yang, Q.
 (2020). Quantitative trait locus mapping of yield and plant height in autotetraploid alfalfa (<em>Medicago sativa</em> L.). The Crop Journal, 8, 812–818. 10.1016/j.cj.2020.05.003
</cite> [<a href="https://doi.org/10.1016/j.cj.2020.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Crop%20Journal&amp;title=Quantitative%20trait%20locus%20mapping%20of%20yield%20and%20plant%20height%20in%20autotetraploid%20alfalfa%20(Medicago%20sativa%20L.)&amp;author=F.%20He&amp;author=R.%20Long&amp;author=T.%20Zhang&amp;author=F.%20Zhang&amp;author=Z.%20Wang&amp;volume=8&amp;publication_year=2020&amp;pages=812-818&amp;doi=10.1016/j.cj.2020.05.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0025">
<cite>

Heinrichs, D. H.
 (1954). Developing creeping‐rooted alfalfa for pasture. Canadian Journal of Agricultural Science, 34, 269–280. 10.4141/agsci-1954-0034
</cite> [<a href="https://doi.org/10.4141/agsci-1954-0034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Canadian%20Journal%20of%20Agricultural%20Science&amp;title=Developing%20creeping%E2%80%90rooted%20alfalfa%20for%20pasture&amp;author=D.%20H.%20Heinrichs&amp;volume=34&amp;publication_year=1954&amp;pages=269-280&amp;doi=10.4141/agsci-1954-0034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0026">
<cite>

Heinrichs, D. H.
 (1975). Potential of legumes for rangelands. In 
R. S. Campbell
 &amp; 
C. H. Herbel
 (Eds.), Improved range plants, range symposium series (pp. 50–61). Society for Range Management.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Improved%20range%20plants,%20range%20symposium%20series&amp;author=D.%20H.%20Heinrichs&amp;publication_year=1975&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0027">
<cite>

Heslot, N.
, 
Yang, H.‐P.
, 
Sorrells, M. E.
, &amp; 
Jannink, J.‐L.
 (2012). Genomic selection in plant breeding: A comparison of models. Crop Science, 52, 146–160. 10.2135/cropsci2011.06.0297
</cite> [<a href="https://doi.org/10.2135/cropsci2011.06.0297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genomic%20selection%20in%20plant%20breeding:%20A%20comparison%20of%20models&amp;author=N.%20Heslot&amp;author=H.%E2%80%90P.%20Yang&amp;author=M.%20E.%20Sorrells&amp;author=J.%E2%80%90L.%20Jannink&amp;volume=52&amp;publication_year=2012&amp;pages=146-160&amp;doi=10.2135/cropsci2011.06.0297&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0028">
<cite>

Holland, J. B.
, &amp; 
Piepho, H.‐P.
 (2024). Don't BLUP twice. G3 Genes|Genomes|Genetics, 14, jkae250. 10.1093/g3journal/jkae250

</cite> [<a href="https://doi.org/10.1093/g3journal/jkae250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11631391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39558791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Don't%20BLUP%20twice&amp;author=J.%20B.%20Holland&amp;author=H.%E2%80%90P.%20Piepho&amp;volume=14&amp;publication_year=2024&amp;pages=jkae250&amp;pmid=39558791&amp;doi=10.1093/g3journal/jkae250&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0029"><cite>

Kehr, W. R.
, 
Conard, E. C.
, 
Alexander, M. A.
, &amp; 
Owen, F. G.
 (1963). 
Performance of alfalfas under five management systems
. Bulletin of the Agricultural Experiment Station of Nebraska.</cite></li>
<li id="tpg270067-bib-0030">
<cite>

Knaus, B. J.
, &amp; 
Grünwald, N. J.
 (2017). vcfr: A package to manipulate and visualize variant call format data in R. Molecular Ecology Resources, 17, 44–53. 10.1111/1755-0998.12549

</cite> [<a href="https://doi.org/10.1111/1755-0998.12549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27401132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Resources&amp;title=vcfr:%20A%20package%20to%20manipulate%20and%20visualize%20variant%20call%20format%20data%20in%20R&amp;author=B.%20J.%20Knaus&amp;author=N.%20J.%20Gr%C3%BCnwald&amp;volume=17&amp;publication_year=2017&amp;pages=44-53&amp;pmid=27401132&amp;doi=10.1111/1755-0998.12549&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0031">
<cite>

Li, A.
, 
Liu, A.
, 
Du, X.
, 
Chen, J.‐Y.
, 
Yin, M.
, 
Hu, H.‐Y.
, 
Shrestha, N.
, 
Wu, S.‐D.
, 
Wang, H.‐Q.
, 
Dou, Q.‐W.
, 
Liu, Z.‐P.
, 
Liu, J.‐Q.
, 
Yang, Y.‐Z.
, &amp; 
Ren, G.‐P.
 (2020). A chromosome‐scale genome assembly of a diploid alfalfa, the progenitor of autotetraploid alfalfa. Horticulture Research, 7, 194–112. 10.1038/s41438-020-00417-7

</cite> [<a href="https://doi.org/10.1038/s41438-020-00417-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7705661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33328470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=A%20chromosome%E2%80%90scale%20genome%20assembly%20of%20a%20diploid%20alfalfa,%20the%20progenitor%20of%20autotetraploid%20alfalfa&amp;author=A.%20Li&amp;author=A.%20Liu&amp;author=X.%20Du&amp;author=J.%E2%80%90Y.%20Chen&amp;author=M.%20Yin&amp;volume=7&amp;publication_year=2020&amp;pages=194-112&amp;pmid=33328470&amp;doi=10.1038/s41438-020-00417-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0032">
<cite>

Li, R.
, &amp; 
Bilal, U.
 (2021). Interactive web‐based data visualization with R, plotly, and shiny (Carson Sievert). Biometrics, 77, 776–777. 10.1111/biom.13474
</cite> [<a href="https://doi.org/10.1111/biom.13474" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Interactive%20web%E2%80%90based%20data%20visualization%20with%20R,%20plotly,%20and%20shiny%20(Carson%20Sievert)&amp;author=R.%20Li&amp;author=U.%20Bilal&amp;volume=77&amp;publication_year=2021&amp;pages=776-777&amp;doi=10.1111/biom.13474&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0033">
<cite>

Li, X.
, &amp; 
Brummer, E.
 (2012). Applied genetics and genomics in alfalfa breeding. Agronomy, 2, 40–61. 10.3390/AGRONOMY2010040
</cite> [<a href="https://doi.org/10.3390/AGRONOMY2010040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=Applied%20genetics%20and%20genomics%20in%20alfalfa%20breeding&amp;author=X.%20Li&amp;author=E.%20Brummer&amp;volume=2&amp;publication_year=2012&amp;pages=40-61&amp;doi=10.3390/AGRONOMY2010040&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0034">
<cite>

Li, X.
, 
Han, Y.
, 
Wei, Y.
, 
Acharya, A.
, 
Farmer, A. D.
, 
Ho, J.
, 
Monteros, M. J.
, &amp; 
Brummer, E. C.
 (2014). Development of an alfalfa SNP array and its use to evaluate patterns of population structure and linkage disequilibrium. PLoS One, 9, e84329. 10.1371/journal.pone.0084329

</cite> [<a href="https://doi.org/10.1371/journal.pone.0084329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3887001/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24416217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Development%20of%20an%20alfalfa%20SNP%20array%20and%20its%20use%20to%20evaluate%20patterns%20of%20population%20structure%20and%20linkage%20disequilibrium&amp;author=X.%20Li&amp;author=Y.%20Han&amp;author=Y.%20Wei&amp;author=A.%20Acharya&amp;author=A.%20D.%20Farmer&amp;volume=9&amp;publication_year=2014&amp;pages=e84329&amp;pmid=24416217&amp;doi=10.1371/journal.pone.0084329&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0035"><cite>
LiLin‐Yin (2024). 
CMplot: Circle manhattan plot
. <a href="https://CRAN.R-project.org/package=CMplot" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R‐project.org/package=CMplot</a>
</cite></li>
<li id="tpg270067-bib-0036">
<cite>

Luo, D.
, 
Zhou, Q.
, 
Wu, Y.
, 
Chai, X.
, 
Liu, W.
, 
Wang, Y.
, 
Yang, Q.
, 
Wang, Z.
, &amp; 
Liu, Z.
 (2019). Full‐length transcript sequencing and comparative transcriptomic analysis to evaluate the contribution of osmotic and ionic stress components towards salinity tolerance in the roots of cultivated alfalfa (<em>Medicago sativa</em> L.). BMC Plant Biology, 19, Article 32. 10.1186/s12870-019-1630-4

</cite> [<a href="https://doi.org/10.1186/s12870-019-1630-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6341612/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30665358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Full%E2%80%90length%20transcript%20sequencing%20and%20comparative%20transcriptomic%20analysis%20to%20evaluate%20the%20contribution%20of%20osmotic%20and%20ionic%20stress%20components%20towards%20salinity%20tolerance%20in%20the%20roots%20of%20cultivated%20alfalfa%20(Medicago%20sativa%20L.)&amp;author=D.%20Luo&amp;author=Q.%20Zhou&amp;author=Y.%20Wu&amp;author=X.%20Chai&amp;author=W.%20Liu&amp;volume=19&amp;publication_year=2019&amp;pages=32&amp;pmid=30665358&amp;doi=10.1186/s12870-019-1630-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0037">
<cite>

Mahmoudiandehkordi, S.
, 
Maadooliat, M.
, &amp; 
Schrodi, S. J.
 (2024). gwid: An R package and Shiny application for genome‐wide analysis of IBD data. Bioinformatics Advances, 4, vbae115. 10.1093/bioadv/vbae115

</cite> [<a href="https://doi.org/10.1093/bioadv/vbae115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11379470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39246385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics%20Advances&amp;title=gwid:%20An%20R%20package%20and%20Shiny%20application%20for%20genome%E2%80%90wide%20analysis%20of%20IBD%20data&amp;author=S.%20Mahmoudiandehkordi&amp;author=M.%20Maadooliat&amp;author=S.%20J.%20Schrodi&amp;volume=4&amp;publication_year=2024&amp;pages=vbae115&amp;pmid=39246385&amp;doi=10.1093/bioadv/vbae115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0038">
<cite>

Mårtensson, A. M.
, &amp; 
Ljunggren, H. D.
 (1984). Nitrogen fixation in an establishing alfalfa (<em>Medicago sativa</em> L.) Ley in Sweden, estimated by three different methods. Applied and Environmental Microbiology, 48, 702–707. 10.1128/aem.48.4.702-707.1984

</cite> [<a href="https://doi.org/10.1128/aem.48.4.702-707.1984" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC241598/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16346637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Nitrogen%20fixation%20in%20an%20establishing%20alfalfa%20(Medicago%20sativa%20L.)%20Ley%20in%20Sweden,%20estimated%20by%20three%20different%20methods&amp;author=A.%20M.%20M%C3%A5rtensson&amp;author=H.%20D.%20Ljunggren&amp;volume=48&amp;publication_year=1984&amp;pages=702-707&amp;pmid=16346637&amp;doi=10.1128/aem.48.4.702-707.1984&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0039">
<cite>

McCoy, T. J.
, &amp; 
Bingham, E. T.
 (1988). Cytology and cytogenetics of alfalfa. In 
A. A. Hanson
, 
D. K. Barnes
, &amp; 
R. R. Hill
 (Eds.), Alfalfa and alfalfa improvement (pp. 737–776). John Wiley &amp; Sons, Ltd.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Alfalfa%20and%20alfalfa%20improvement&amp;author=T.%20J.%20McCoy&amp;author=E.%20T.%20Bingham&amp;publication_year=1988&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0040">
<cite>

Medina, C. A.
, 
Heuschele, D. J.
, 
Zhao, D.
, 
Lin, M.
, 
Beil, C. T.
, 
Sheehan, M. J.
, &amp; 
Xu, Z.
 (2024). Multi‐trait modeling and machine learning discover new markers associated with stem traits in alfalfa. Frontiers in Plant Science, 15, 1429976. 10.3389/fpls.2024.1429976

</cite> [<a href="https://doi.org/10.3389/fpls.2024.1429976" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11418689/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39315379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Multi%E2%80%90trait%20modeling%20and%20machine%20learning%20discover%20new%20markers%20associated%20with%20stem%20traits%20in%20alfalfa&amp;author=C.%20A.%20Medina&amp;author=D.%20J.%20Heuschele&amp;author=D.%20Zhao&amp;author=M.%20Lin&amp;author=C.%20T.%20Beil&amp;volume=15&amp;publication_year=2024&amp;pages=1429976&amp;pmid=39315379&amp;doi=10.3389/fpls.2024.1429976&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0041">
<cite>

Medina, C. A.
, 
Kaur, H.
, 
Ray, I.
, &amp; 
Yu, L.‐X.
 (2021). Strategies to increase prediction accuracy in genomic selection of complex traits in alfalfa (<em>Medicago sativa</em> L.). Cells, 10, 3372. 10.3390/cells10123372

</cite> [<a href="https://doi.org/10.3390/cells10123372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8699225/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34943880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=Strategies%20to%20increase%20prediction%20accuracy%20in%20genomic%20selection%20of%20complex%20traits%20in%20alfalfa%20(Medicago%20sativa%20L.)&amp;author=C.%20A.%20Medina&amp;author=H.%20Kaur&amp;author=I.%20Ray&amp;author=L.%E2%80%90X.%20Yu&amp;volume=10&amp;publication_year=2021&amp;pages=3372&amp;pmid=34943880&amp;doi=10.3390/cells10123372&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0042">
<cite>

Medina, C. A.
, 
Zhao, D.
, 
Lin, M.
, 
Sapkota, M.
, 
Sandercock, A. M.
, 
Beil, C. T.
, 
Sheehan, M. J.
, 
Irish, B. M.
, 
Yu, L.‐X.
, 
Poudel, H.
, 
Claessens, A.
, 
Moore, V.
, 
Crawford, J.
, 
Hansen, J.
, 
Viands, D.
, 
Peel, M. D.
, 
Tilhou, N.
, 
Riday, H.
, 
Brummer, E. C.
, &amp; 
Xu, Z.
 (2024). Pre‐breeding in alfalfa germplasm develops highly differentiated populations, as revealed by genome‐wide microhaplotype markers. Scientific Reports, 15(1), 1253. 10.1038/s41598-024-84262-x
</cite> [<a href="https://doi.org/10.1038/s41598-024-84262-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11711157/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39779777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Pre%E2%80%90breeding%20in%20alfalfa%20germplasm%20develops%20highly%20differentiated%20populations,%20as%20revealed%20by%20genome%E2%80%90wide%20microhaplotype%20markers&amp;author=C.%20A.%20Medina&amp;author=D.%20Zhao&amp;author=M.%20Lin&amp;author=M.%20Sapkota&amp;author=A.%20M.%20Sandercock&amp;volume=15&amp;issue=1&amp;publication_year=2024&amp;pages=1253&amp;pmid=39779777&amp;doi=10.1038/s41598-024-84262-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0043">
<cite>

Miller, S. S.
, 
Dornbusch, M. R.
, 
Farmer, A. D.
, 
Huertas, R.
, 
Gutierrez‐Gonzalez, J. J.
, 
Young, N. D.
, 
Samac, D. A.
, &amp; 
Curtin, S. J.
 (2022). Alfalfa (<em>Medicago sativa</em> L.) pho2 mutant plants hyperaccumulate phosphate. G3 Genes|Genomes|Genetics, 12, jkac096. 10.1093/g3journal/jkac096

</cite> [<a href="https://doi.org/10.1093/g3journal/jkac096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9157135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35471600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Alfalfa%20(Medicago%20sativa%20L.)%20pho2%20mutant%20plants%20hyperaccumulate%20phosphate&amp;author=S.%20S.%20Miller&amp;author=M.%20R.%20Dornbusch&amp;author=A.%20D.%20Farmer&amp;author=R.%20Huertas&amp;author=J.%20J.%20Gutierrez%E2%80%90Gonzalez&amp;volume=12&amp;publication_year=2022&amp;pages=jkac096&amp;pmid=35471600&amp;doi=10.1093/g3journal/jkac096&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0044">
<cite>

Misar, C. G.
, 
Xu, L.
, 
Gates, R. N.
, 
Boe, A.
, &amp; 
Johnson, P. S.
 (2015). Stand persistence and forage yield of 11 alfalfa (<em>Medicago sativa</em>) populations in semiarid rangeland. Rangeland Ecology &amp; Management, 68, 79–85. 10.1016/j.rama.2014.12.012
</cite> [<a href="https://doi.org/10.1016/j.rama.2014.12.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rangeland%20Ecology%20&amp;%20Management&amp;title=Stand%20persistence%20and%20forage%20yield%20of%2011%20alfalfa%20(Medicago%20sativa)%20populations%20in%20semiarid%20rangeland&amp;author=C.%20G.%20Misar&amp;author=L.%20Xu&amp;author=R.%20N.%20Gates&amp;author=A.%20Boe&amp;author=P.%20S.%20Johnson&amp;volume=68&amp;publication_year=2015&amp;pages=79-85&amp;doi=10.1016/j.rama.2014.12.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0045"><cite>

Morgan, M.
, 
Pagès, H.
, 
Obenchain, V.
, &amp; 
Hayden, N.
 (2025). 
Rsamtools: Binary alignment (BAM), FASTA, variant call (BCF), and tabix file import
. <a href="https://bioconductor.org/packages/Rsamtools" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bioconductor.org/packages/Rsamtools</a>
</cite></li>
<li id="tpg270067-bib-0046">
<cite>

Oakey, H.
, 
Verbyla, A.
, 
Pitchford, W.
, 
Cullis, B.
, &amp; 
Kuchel, H.
 (2006). Joint modeling of additive and non‐additive genetic line effects in single field trials. Theoretical and Applied Genetics, 113, 809–819. 10.1007/s00122-006-0333-z

</cite> [<a href="https://doi.org/10.1007/s00122-006-0333-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16896718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Joint%20modeling%20of%20additive%20and%20non%E2%80%90additive%20genetic%20line%20effects%20in%20single%20field%20trials&amp;author=H.%20Oakey&amp;author=A.%20Verbyla&amp;author=W.%20Pitchford&amp;author=B.%20Cullis&amp;author=H.%20Kuchel&amp;volume=113&amp;publication_year=2006&amp;pages=809-819&amp;pmid=16896718&amp;doi=10.1007/s00122-006-0333-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0047">
<cite>

Polegri, L.
, 
Pecetti, L.
, 
Piano, E.
, 
Arcioni, S.
, 
Guaragno, M.
, &amp; 
Pupilli, F.
 (2011). Identification of AFLPs co‐segregating with the creeping‐rootedness trait in lucerne (<em>Medicago sativa</em> L. complex). Molecular Breeding: New Strategies in Plant Improvement, 28, 91–103. 10.1007/s11032-010-9463-8
</cite> [<a href="https://doi.org/10.1007/s11032-010-9463-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding:%20New%20Strategies%20in%20Plant%20Improvement&amp;title=Identification%20of%20AFLPs%20co%E2%80%90segregating%20with%20the%20creeping%E2%80%90rootedness%20trait%20in%20lucerne%20(Medicago%20sativa%20L.%20complex)&amp;author=L.%20Polegri&amp;author=L.%20Pecetti&amp;author=E.%20Piano&amp;author=S.%20Arcioni&amp;author=M.%20Guaragno&amp;volume=28&amp;publication_year=2011&amp;pages=91-103&amp;doi=10.1007/s11032-010-9463-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0048">
<cite>

Ramirez, R. N.
, 
Bedirian, K.
, 
Gray, S. M.
, &amp; 
Diallo, A.
 (2020). DNA Rchitect: An R based visualizer for network analysis of chromatin interaction data. Bioinformatics, 36, 644–646. 10.1093/bioinformatics/btz608

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz608" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7867998/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31373608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=DNA%20Rchitect:%20An%20R%20based%20visualizer%20for%20network%20analysis%20of%20chromatin%20interaction%20data&amp;author=R.%20N.%20Ramirez&amp;author=K.%20Bedirian&amp;author=S.%20M.%20Gray&amp;author=A.%20Diallo&amp;volume=36&amp;publication_year=2020&amp;pages=644-646&amp;pmid=31373608&amp;doi=10.1093/bioinformatics/btz608&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0049">
<cite>

Rodríguez‐Álvarez, M. X.
, 
Boer, M. P.
, 
van Eeuwijk, F. A.
, &amp; 
Eilers, P. H. C.
 (2018). Correcting for spatial heterogeneity in plant breeding experiments with P‐splines. Spatial Statistics, 23, 52–71. 10.1016/j.spasta.2017.10.003
</cite> [<a href="https://doi.org/10.1016/j.spasta.2017.10.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Spatial%20Statistics&amp;title=Correcting%20for%20spatial%20heterogeneity%20in%20plant%20breeding%20experiments%20with%20P%E2%80%90splines&amp;author=M.%20X.%20Rodr%C3%ADguez%E2%80%90%C3%81lvarez&amp;author=M.%20P.%20Boer&amp;author=F.%20A.%20van%20Eeuwijk&amp;author=P.%20H.%20C.%20Eilers&amp;volume=23&amp;publication_year=2018&amp;pages=52-71&amp;doi=10.1016/j.spasta.2017.10.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0050">
<cite>

Rosyara, U. R.
, 
De Jong, W. S.
, 
Douches, D. S.
, &amp; 
Endelman, J. B.
 (2016). Software for genome‐wide association studies in autopolyploids and its application to potato. The Plant Genome, 9, plantgenome2015.08.0073. 10.3835/plantgenome2015.08.0073
</cite> [<a href="https://doi.org/10.3835/plantgenome2015.08.0073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27898814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Software%20for%20genome%E2%80%90wide%20association%20studies%20in%20autopolyploids%20and%20its%20application%20to%20potato&amp;author=U.%20R.%20Rosyara&amp;author=W.%20S.%20De%20Jong&amp;author=D.%20S.%20Douches&amp;author=J.%20B.%20Endelman&amp;volume=9&amp;publication_year=2016&amp;pmid=27898814&amp;doi=10.3835/plantgenome2015.08.0073&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0051"><cite>

Rumbaugh, M. D.
, 
Moore, S. R.
, &amp; 
Colburn, J. D.
 (1965). 
Travois: An alfalfa for grazing
. <a href="https://openprairie.sdstate.edu/cgi/viewcontent.cgi?article=1524&amp;context=agexperimentsta_bulletins" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://openprairie.sdstate.edu/cgi/viewcontent.cgi?article=1524&amp;context=agexperimentsta_bulletins</a>
</cite></li>
<li id="tpg270067-bib-0052">
<cite>

Russelle, M. P.
 (2001). Alfalfa: After an 8,000‐year journey, the “Queen of Forages” stands poised to enjoy renewed popularity. American Scientist, 89, 252–261. 10.1511/2001.3.252
</cite> [<a href="https://doi.org/10.1511/2001.3.252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Scientist&amp;title=Alfalfa:%20After%20an%208,000%E2%80%90year%20journey,%20the%20%E2%80%9CQueen%20of%20Forages%E2%80%9D%20stands%20poised%20to%20enjoy%20renewed%20popularity&amp;author=M.%20P.%20Russelle&amp;volume=89&amp;publication_year=2001&amp;pages=252-261&amp;doi=10.1511/2001.3.252&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0053">
<cite>

Schwarz, G.
 (1978). Estimating the dimension of a model. Annals of Statistics, 6, 461–464. 10.1214/aos/1176344136
</cite> [<a href="https://doi.org/10.1214/aos/1176344136" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Statistics&amp;title=Estimating%20the%20dimension%20of%20a%20model&amp;author=G.%20Schwarz&amp;volume=6&amp;publication_year=1978&amp;pages=461-464&amp;doi=10.1214/aos/1176344136&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0054">
<cite>

Shen, C.
, 
Du, H.
, 
Chen, Z.
, 
Lu, H.
, 
Zhu, F.
, 
Chen, H.
, 
Meng, X.
, 
Liu, Q.
, 
Liu, P.
, 
Zheng, L.
, 
Li, X.
, 
Dong, J.
, 
Liang, C.
, &amp; 
Wang, T.
 (2020). The chromosome‐level genome sequence of the autotetraploid alfalfa and resequencing of core germplasms provide genomic resources for alfalfa research. Molecular Plant, 13, 1250–1261. 10.1016/j.molp.2020.07.003

</cite> [<a href="https://doi.org/10.1016/j.molp.2020.07.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32673760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=The%20chromosome%E2%80%90level%20genome%20sequence%20of%20the%20autotetraploid%20alfalfa%20and%20resequencing%20of%20core%20germplasms%20provide%20genomic%20resources%20for%20alfalfa%20research&amp;author=C.%20Shen&amp;author=H.%20Du&amp;author=Z.%20Chen&amp;author=H.%20Lu&amp;author=F.%20Zhu&amp;volume=13&amp;publication_year=2020&amp;pages=1250-1261&amp;pmid=32673760&amp;doi=10.1016/j.molp.2020.07.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0055">
<cite>

Simeão Resende, R. M.
, 
Casler, M. D.
, &amp; 
de Resende, M. D. V.
 (2014). Genomic selection in forage breeding: Accuracy and methods. Crop Science, 54, 143–156. 10.2135/cropsci2013.05.0353
</cite> [<a href="https://doi.org/10.2135/cropsci2013.05.0353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genomic%20selection%20in%20forage%20breeding:%20Accuracy%20and%20methods&amp;author=R.%20M.%20Sime%C3%A3o%20Resende&amp;author=M.%20D.%20Casler&amp;author=M.%20D.%20V.%20de%20Resende&amp;volume=54&amp;publication_year=2014&amp;pages=143-156&amp;doi=10.2135/cropsci2013.05.0353&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0056">
<cite>

Sleugh, B.
, 
Moore, K. J.
, 
George, J. R.
, &amp; 
Brummer, E. C.
 (2000). Binary legume–grass mixtures improve forage yield, quality, and seasonal distribution. Agronomy Journal, 92, 24–29. 10.2134/agronj2000.92124x
</cite> [<a href="https://doi.org/10.2134/agronj2000.92124x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy%20Journal&amp;title=Binary%20legume%E2%80%93grass%20mixtures%20improve%20forage%20yield,%20quality,%20and%20seasonal%20distribution&amp;author=B.%20Sleugh&amp;author=K.%20J.%20Moore&amp;author=J.%20R.%20George&amp;author=E.%20C.%20Brummer&amp;volume=92&amp;publication_year=2000&amp;pages=24-29&amp;doi=10.2134/agronj2000.92124x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0057">
<cite>

Solomon, J. K. Q.
, 
Macoon, B.
, 
Lang, D. J.
, 
Parish, J. A.
, &amp; 
Vann, R. C.
 (2011). A novel approach to grass–legume management. Crop Science, 51, 1865–1876. 10.2135/cropsci2010.10.0598
</cite> [<a href="https://doi.org/10.2135/cropsci2010.10.0598" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=A%20novel%20approach%20to%20grass%E2%80%93legume%20management&amp;author=J.%20K.%20Q.%20Solomon&amp;author=B.%20Macoon&amp;author=D.%20J.%20Lang&amp;author=J.%20A.%20Parish&amp;author=R.%20C.%20Vann&amp;volume=51&amp;publication_year=2011&amp;pages=1865-1876&amp;doi=10.2135/cropsci2010.10.0598&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0058">
<cite>

Van Keuren, R. W.
, &amp; 
Matches, A. G.
 (1988). Pasture production and utilization. 
A. A. Hanson
, 
D. K. Barnes
, 
R. R. Hill Jr. (Eds.), Alfalfa and alfalfa improvement (pp. 515–538). ASA, CSSA, SSSA.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Alfalfa%20and%20alfalfa%20improvement&amp;author=R.%20W.%20Van%20Keuren&amp;author=A.%20G.%20Matches&amp;publication_year=1988&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0059">
<cite>

VanRaden, P. M.
 (2020). Symposium review: How to implement genomic selection. Journal of Dairy Science, 103, 5291–5301. 10.3168/jds.2019-17684

</cite> [<a href="https://doi.org/10.3168/jds.2019-17684" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32331884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Dairy%20Science&amp;title=Symposium%20review:%20How%20to%20implement%20genomic%20selection&amp;author=P.%20M.%20VanRaden&amp;volume=103&amp;publication_year=2020&amp;pages=5291-5301&amp;pmid=32331884&amp;doi=10.3168/jds.2019-17684&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0060">
<cite>

Waldron, B. L.
, 
Bingham, T. J.
, 
Creech, J. E.
, 
Peel, M. D.
, 
Miller, R.
, 
Jensen, K. B.
, 
ZoBell, D. R.
, 
Eun, J.
, 
Heaton, K.
, &amp; 
Snyder, D. L.
 (2020). Binary mixtures of alfalfa and birdsfoot trefoil with tall fescue: Herbage traits associated with the improved growth performance of beef steers. Grassland Science, 66, 74–87. 10.1111/grs.12257
</cite> [<a href="https://doi.org/10.1111/grs.12257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Grassland%20Science&amp;title=Binary%20mixtures%20of%20alfalfa%20and%20birdsfoot%20trefoil%20with%20tall%20fescue:%20Herbage%20traits%20associated%20with%20the%20improved%20growth%20performance%20of%20beef%20steers&amp;author=B.%20L.%20Waldron&amp;author=T.%20J.%20Bingham&amp;author=J.%20E.%20Creech&amp;author=M.%20D.%20Peel&amp;author=R.%20Miller&amp;volume=66&amp;publication_year=2020&amp;pages=74-87&amp;doi=10.1111/grs.12257&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0061">
<cite>

Wickham, H.
 (2011). Ggplot2: Ggplot2. Wiley Interdisciplinary Reviews. Computational Statistics, 3, 180–185. 10.1002/wics.147
</cite> [<a href="https://doi.org/10.1002/wics.147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Wiley%20Interdisciplinary%20Reviews.%20Computational%20Statistics&amp;title=Ggplot2:%20Ggplot2&amp;author=H.%20Wickham&amp;volume=3&amp;publication_year=2011&amp;pages=180-185&amp;doi=10.1002/wics.147&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0062">
<cite>

Zhang, F.
, 
Kang, J.
, 
Long, R.
, 
Yu, L.‐X.
, 
Wang, Z.
, 
Zhao, Z.
, 
Zhang, T.
, &amp; 
Yang, Q.
 (2019). High‐density linkage map construction and mapping QTL for yield and yield components in autotetraploid alfalfa using RAD‐seq. BMC pl#ant Biology, 19, Article 165. 10.1186/s12870-019-1770-6
</cite> [<a href="https://doi.org/10.1186/s12870-019-1770-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6487053/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31029106/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20pl#ant%20Biology&amp;title=High%E2%80%90density%20linkage%20map%20construction%20and%20mapping%20QTL%20for%20yield%20and%20yield%20components%20in%20autotetraploid%20alfalfa%20using%20RAD%E2%80%90seq&amp;author=F.%20Zhang&amp;author=J.%20Kang&amp;author=R.%20Long&amp;author=L.%E2%80%90X.%20Yu&amp;author=Z.%20Wang&amp;volume=19&amp;publication_year=2019&amp;pages=165&amp;pmid=31029106&amp;doi=10.1186/s12870-019-1770-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0063">
<cite>

Zhao, D.
, 
Hamilton, J. P.
, 
Hardigan, M.
, 
Yin, D.
, 
He, T.
, 
Vaillancourt, B.
, 
Reynoso, M.
, 
Pauluzzi, G.
, 
Funkhouser, S.
, 
Cui, Y.
, 
Bailey‐Serres, J.
, 
Jiang, J.
, 
Buell, C. R.
, &amp; 
Jiang, N.
 (2017). Analysis of ribosome‐associated mRNAs in rice reveals the importance of transcript size and GC content in translation. G3 Genes|Genomes|Genetics, 7, 203–219. 10.1534/g3.116.036020

</cite> [<a href="https://doi.org/10.1534/g3.116.036020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5217110/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27852012/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Analysis%20of%20ribosome%E2%80%90associated%20mRNAs%20in%20rice%20reveals%20the%20importance%20of%20transcript%20size%20and%20GC%20content%20in%20translation&amp;author=D.%20Zhao&amp;author=J.%20P.%20Hamilton&amp;author=M.%20Hardigan&amp;author=D.%20Yin&amp;author=T.%20He&amp;volume=7&amp;publication_year=2017&amp;pages=203-219&amp;pmid=27852012&amp;doi=10.1534/g3.116.036020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0064">
<cite>

Zhao, D.
, 
Mejia‐Guerra, K. M.
, 
Mollinari, M.
, 
Samac, D.
, 
Irish, B.
, 
Heller‐Uszynska, K.
, 
Beil, C. T.
, &amp; 
Sheehan, M. J.
 (2023). A public mid‐density genotyping platform for alfalfa (<em>Medicago sativa</em> L.). Genetic Resources, 4, 55–63. 10.46265/genresj.emor6509
</cite> [<a href="https://doi.org/10.46265/genresj.emor6509" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetic%20Resources&amp;title=A%20public%20mid%E2%80%90density%20genotyping%20platform%20for%20alfalfa%20(Medicago%20sativa%20L.)&amp;author=D.%20Zhao&amp;author=K.%20M.%20Mejia%E2%80%90Guerra&amp;author=M.%20Mollinari&amp;author=D.%20Samac&amp;author=B.%20Irish&amp;volume=4&amp;publication_year=2023&amp;pages=55-63&amp;doi=10.46265/genresj.emor6509&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0065">
<cite>

Zhao, D.
, 
Sapkota, M.
, 
Lin, M.
, 
Beil, C.
, 
Sheehan, M.
, 
Greene, S.
, &amp; 
Irish, B. M.
 (2024). Genetic diversity, population structure, and taxonomic confirmation in annual medic (<em>Medicago</em> spp.) collections from Crimea, Ukraine. Frontiers in Plant Science, 15, 1339298. 10.3389/fpls.2024.1339298

</cite> [<a href="https://doi.org/10.3389/fpls.2024.1339298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11021755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38633467/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genetic%20diversity,%20population%20structure,%20and%20taxonomic%20confirmation%20in%20annual%20medic%20(Medicago%20spp.)%20collections%20from%20Crimea,%20Ukraine&amp;author=D.%20Zhao&amp;author=M.%20Sapkota&amp;author=M.%20Lin&amp;author=C.%20Beil&amp;author=M.%20Sheehan&amp;volume=15&amp;publication_year=2024&amp;pages=1339298&amp;pmid=38633467&amp;doi=10.3389/fpls.2024.1339298&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270067-bib-0066">
<cite>

Zhao, X.
, 
Han, X.
, 
Lu, X.
, 
Yang, H.
, 
Wang, Z.‐Y.
, &amp; 
Chai, M.
 (2023). Genome‐wide identification and characterization of the Msr gene family in alfalfa under abiotic stress. International Journal of Molecular Sciences, 24, 9638. 10.3390/ijms24119638

</cite> [<a href="https://doi.org/10.3390/ijms24119638" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10253413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37298589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Genome%E2%80%90wide%20identification%20and%20characterization%20of%20the%20Msr%20gene%20family%20in%20alfalfa%20under%20abiotic%20stress&amp;author=X.%20Zhao&amp;author=X.%20Han&amp;author=X.%20Lu&amp;author=H.%20Yang&amp;author=Z.%E2%80%90Y.%20Wang&amp;volume=24&amp;publication_year=2023&amp;pages=9638&amp;pmid=37298589&amp;doi=10.3390/ijms24119638&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>
<strong>Supplemental Table 1</strong>: Summary of the distribution of the filtered 2,211 SNPs.</p>
<p>
<strong>Supplemental Table 2</strong>: Significant markers associated with alfalfa creeping rootedness.</p>
<p>
<strong>Supplemental Figure 1</strong>: Summary of the genetic backgrounds of 800 plants evaluated for creeping rootedness.</p>
<p>
<strong>Supplemental Figure 2</strong>: PCA with the progeny labelled for their inclusion in the GWAS and GS analysis.</p>
<p>
<strong>Supplemental Figure 3</strong>: GWAS diagnostic plots.</p>
<p>
<strong>Supplemental Figure 4</strong>: Linkage‐disequilibrium decay plot for the filtered markers.</p>
<p>
<strong>Supplemental Figure 5</strong>: Distribution of the alfalfa SNPs.</p>
<p>
<strong>Supplemental Figure 6</strong>: Genomic diversity of the alfalfa creeping root breeding population.</p>
<p>
<strong>Supplemental Figure 7</strong>: Box plot of the 10 iterations for the GS GBLUP model evaluation.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12245737/bin/TPG2-18-e70067-s003.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70067-s003.docx</a><sup> (1.7MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>
<strong>Supplemental File 1</strong>: Raw phenotypic and BLUE data for the alfalfa samples.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12245737/bin/TPG2-18-e70067-s001.zip" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70067-s001.zip</a><sup> (79.2KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><p>
<strong>Supplemental File 2</strong>: Raw DArTag MADC File.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12245737/bin/TPG2-18-e70067-s002.zip" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70067-s002.zip</a><sup> (6MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p"><p>
<strong>Supplemental File 3</strong>: Filtered VCF file of the 2,211 biallelic SNPs and 648 samples.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12245737/bin/TPG2-18-e70067-s005.zip" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70067-s005.zip</a><sup> (13.5MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p"><p>
<strong>Supplemental File 4</strong>: Genes associated with the three GWAS identified QTL based on a 8.575 Mb LD window.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12245737/bin/TPG2-18-e70067-s004.zip" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70067-s004.zip</a><sup> (41.5KB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Genomic and phenotypic data files are included as Supporting Information: BLUE calculations: <a href="https://github.com/Breeding-Insight/alfalfa_BLUE" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Breeding‐Insight/alfalfa_BLUE</a>; BIGr (RRID: SCR_026677): <a href="https://CRAN.R-project.org/package=BIGr" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R‐project.org/package=BIGr</a>; BIGapp (RRID: SCR_026676): <a href="https://github.com/Breeding-Insight/BIGapp" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Breeding‐Insight/BIGapp</a>. BIGapp is available on Sci‐Net for USDA‐ARS scientists.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70067"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70067.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12245737/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12245737/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12245737%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12245737/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12245737/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12245737/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40640114/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12245737/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40640114/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12245737/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12245737/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Y72sqANvaa2zWa2fHvWhgmNpoRd1edIpQq5nW1pTsOWDPwQfRDIaKNvm5tuNoFHH">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
