
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Production of Envelope Protein-Based Virus-like Particles of DENV‑2 in and Investigation of Their Binding Interactions with Gallic Acid Derivatives of Glucose as Potential Drug Candidates - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="67D7128F89B720F31C128F003305CB3B.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsomega">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332551/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Omega">
<meta name="citation_title" content="Production of Envelope Protein-Based Virus-like Particles of DENV‑2 in and Investigation of Their Binding Interactions with Gallic Acid Derivatives of Glucose as Potential Drug Candidates">
<meta name="citation_author" content="Anjana Sharma">
<meta name="citation_author_institution" content="Department of Biotechnology, School of Life Sciences, Central University of Rajasthan, NH-8, Bandarsindri, Ajmer, Rajasthan 305817, India">
<meta name="citation_author" content="Surendra Nimesh">
<meta name="citation_author_institution" content="Department of Biotechnology, School of Life Sciences, Central University of Rajasthan, NH-8, Bandarsindri, Ajmer, Rajasthan 305817, India">
<meta name="citation_author" content="Suman Tapryal">
<meta name="citation_author_institution" content="Department of Biophysics, University of Delhi South Campus, Benito Juarez Road, South Moti Bagh, New Delhi 110021, India">
<meta name="citation_author_institution" content="Department of Biotechnology, School of Life Sciences, Central University of Rajasthan, NH-8, Bandarsindri, Ajmer, Rajasthan 305817, India">
<meta name="citation_publication_date" content="2025 Jul 21">
<meta name="citation_volume" content="10">
<meta name="citation_issue" content="30">
<meta name="citation_firstpage" content="33090">
<meta name="citation_doi" content="10.1021/acsomega.5c02548">
<meta name="citation_pmid" content="40787407">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332551/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332551/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332551/pdf/ao5c02548.pdf">
<meta name="description" content="Perennial endemics of mosquito-borne dengue infection affect millions of lives across tropical and subtropical regions every year. Factors like the Earth’s rising temperature, climate change, and lack of dengue-specific drugs and vaccines have led ...">
<meta name="og:title" content="Production of Envelope Protein-Based Virus-like Particles of DENV‑2 in and Investigation of Their Binding Interactions with Gallic Acid Derivatives of Glucose as Potential Drug Candidates">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Perennial endemics of mosquito-borne dengue infection affect millions of lives across tropical and subtropical regions every year. Factors like the Earth’s rising temperature, climate change, and lack of dengue-specific drugs and vaccines have led ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332551/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332551">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsomega.5c02548"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ao5c02548.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332551%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332551/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332551/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332551/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsomega.png" alt="ACS Omega logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Omega" title="Link to ACS Omega" shape="default" href="https://pubs.acs.org/journal/acsodf" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Omega</button></div>. 2025 Jul 21;10(30):33090–33107. doi: <a href="https://doi.org/10.1021/acsomega.5c02548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsomega.5c02548</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Omega%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Omega%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Production of Envelope
Protein-Based Virus-like Particles
of DENV‑2 in and Investigation
of Their Binding Interactions with Gallic Acid Derivatives of Glucose
as Potential Drug Candidates</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Anjana Sharma</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Anjana Sharma</span></h3>
<div class="p">
<sup>‡</sup>
Department
of Biotechnology, School of Life Sciences, Central University of Rajasthan, NH-8, Bandarsindri, Ajmer, Rajasthan
305817, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anjana Sharma</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nimesh%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Surendra Nimesh</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Surendra Nimesh</span></h3>
<div class="p">
<sup>‡</sup>
Department
of Biotechnology, School of Life Sciences, Central University of Rajasthan, NH-8, Bandarsindri, Ajmer, Rajasthan
305817, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nimesh%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Surendra Nimesh</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tapryal%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Suman Tapryal</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Suman Tapryal</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biophysics, University of Delhi South
Campus, Benito Juarez Road, South Moti Bagh, New Delhi
110021, India
</div>
<div class="p">
<sup>‡</sup>
Department
of Biotechnology, School of Life Sciences, Central University of Rajasthan, NH-8, Bandarsindri, Ajmer, Rajasthan
305817, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tapryal%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Suman Tapryal</span></a>
</div>
</div>
<sup>†,</sup><sup>‡,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Department
of Biophysics, University of Delhi South
Campus, Benito Juarez Road, South Moti Bagh, New Delhi
110021, India
</div>
<div id="aff2">
<sup>‡</sup>
Department
of Biotechnology, School of Life Sciences, Central University of Rajasthan, NH-8, Bandarsindri, Ajmer, Rajasthan
305817, India
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>stapryal@south.du.ac.in</span>.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 19; Accepted 2025 Jul 4; Revised 2025 Jun 27; Collection date 2025 Aug 5.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332551  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40787407/" class="usa-link">40787407</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Perennial endemics
of mosquito-borne dengue infection affect millions
of lives across tropical and subtropical regions every year. Factors
like the Earth’s rising temperature, climate change, and lack
of dengue-specific drugs and vaccines have led to a significant surge
in these infections, warranting urgent efforts to develop new antidengue
agents. The dengue virus (DENV) envelope exposes approximately 90
E (envelope) protein dimers on its surface, making it a potent target
for drug discovery and vaccine development. E protein-based virus-like
particles (VLPs) may serve as an ideal system to test the binding
activity of compounds <em>in vitro</em>. Such VLPs have not
been defined for DENV serotype 2 (DENV-2). The current study demonstrates
the expression of the E protein-ectodomain in and VLP generation. Previously, chebulinic acid
(CA), a tannin, was demonstrated as a strong DENV-2 inhibitor in Vero
cells. However, the structural complexity of CA made its synthesis
a challenging task, whereas toxicity at higher doses limited its therapeutic
use. To investigate binding to E protein <em>in silico</em>, simple analogues of CA (mono, di, tri, tetra, and pentagalloylglucose
(PGG)) were selected and evaluated using molecular docking, molecular
dynamics (MD) simulations, and electrostatic complementarity analysis.
Herein, PGG–E-protein complex demonstrated higher docking scores,
stable MD simulation patterns and electrostatics, equivalent to that
of positive controls, CA, and epigallocatechin-3-gallate. Furthermore,
the PGG interaction with VLP molecules was analyzed <em>in vitro</em> using biolayer interferometry, where nM-scale equilibrium constants
were recorded. Therefore, a strong <em>in silico</em> and <em>in vitro</em> E protein-binding profile of PGG supports its potential
as a DENV-2 inhibitor, which may be developed as an anti-DENV-2 therapeutic.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/211f/12332551/fe103c8879fc/ao5c02548_0011.jpg" loading="lazy" id="abs1" height="372" width="633" alt="graphic file with name ao5c02548_0011.jpg"></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/211f/12332551/fe103c8879fc/ao5c02548_0009.jpg" loading="lazy" id="tgr1" height="372" width="633" alt="graphic file with name ao5c02548_0009.jpg"></p></section><section id="sec1"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Dengue virus (DENV) is
transmitted through a mosquito vector and
is responsible for the onset of dengue fever. DENV has been classified into four serotypes (DENV-1, DENV-2, DENV-3,
and DENV-4), and is known to be transmitted by <em>Aedes</em> mosquitoes to humans. Another serotype
of this virus, the fifth serotype (DENV-5), has been discovered in
Sarawak state, Malaysia, with the help of isolation and genetic sequence
analysis. Dengue fever has been marked
as a global health concern by the World Health Organization (WHO)
for being the leading cause of the arthropod-borne viral disease, which is significantly linked with a huge social
and economic burden worldwide. In both
tropical and subtropical regions, about 50 million people get dengue
infections each year.</p>
<p>The infection caused by various DENV serotypes
could result in
asymptomatic disease in most cases, or it may show a wide spectrum
of clinical symptoms, starting from mild flu-like syndrome, also known
as dengue fever, to severe forms of the disease. The severe disease
forms are characterized by permeability, coagulopathy, and increased
vascular fragility, which later could lead to hypovolemic shock and
death. Despite the known severity and
clinical significance of this disease, fewer treatment regimens exist,
and the therapeutic strategy is mostly dependent on supportive care. A vaccine has been developed against the dengue
virus earlier and is in use in a limited number of countries; it has
not been used widely due to the associated risk factors of severe
dengue. Therefore, there exists an immediate
need for the development of efficient vaccines and therapeutics for
the efficient reduction in dengue morbidities and the burden of infection
in the susceptible population.
</p>
<p>DENV
has a single-stranded RNA genome, which has a positive polarity
that translates in the cytoplasm as a single polyprotein. The cellular
and viral proteases process the polyprotein and release 3 structural
and 7 nonstructural proteins. The structural proteins that play a
key role in host entry are the capsid protein (C), membrane precursor
protein (prM), and the glycosylated envelope protein (E). The envelope protein consists of 3 domains: the
E protein domain-I (EDI), also known as the central domain, the dimerization
domain II (EDII), which harbors the conserved fusion loop, and the
putative receptor-binding domain III (EDIII). E protein has two N-linked glycosylation sites, Asn-67, which is
specific to DENV serotypes, and Asn-153, which is conserved in most
of the flaviviruses. The E protein is
the most significant factor in DENV structure as it occupies the major
area of the viral surface and is considered the primary target of
the humoral immune system. Subsequently,
anti-E protein antibodies have been shown to inhibit the binding of
the DENV to its receptor molecules, thereby inhibiting the infection.
</p>
<p>Mammalian cells harbor several surface
molecules that act as receptors
for DENV. These include lectins that recognize carbohydrates, like
Dendritic cell-specific ICAM-3-grabbing nonintegrin (DC-SIGN), glycosphingolipid
(GSL), laminin-binding proteins, sulfated glycosaminoglycans (GAGs),
proteins with chaperone activity, and some other uncharacterized proteins
found in the cells and organs of mosquitoes
</p>
<p>An earlier study demonstrated the antiviral potential of chebulinic
acid (CA), which is a plant-based complex biomolecule, as an inhibitor
of DENV-2 infection. The structural complexity
and large size of CA make its chemical synthesis difficult, and it
has not yet been synthesized in a lab setup. Additionally, the toxic
side effects of CA have also been observed at moderate to high concentrations
in Vero cells, making its natural form
somewhat unsafe as a potential DENV inhibitor. CA contains multiple
gallic acid moieties and is generally involved in interacting with
the receptor molecules through them. Therefore, natural molecules
harboring similar gallic acid units were screened, and five CA analogues
or gallic acid derivatives, the mono, di, tri, tetra, and pentagalloylglucose
(PGG), were selected for investigating their propensity for binding
the E protein of DENV-2 using molecular docking and MD simulation
studies. Based on the structural and chemical similarity between CA
and the gallic acid derivatives of glucose, two of the three previously
defined CA binding pockets on E protein were chosen for docking experiments. The most potent binder was
then used to bind the E protein <em>in vitro</em>. To perform
this study, the viral envelope protein was cloned, expressed in ,
and <em>in vitro</em> refolded into higher-order oligomeric
assemblies, virus-like particles (VLPs). Various types of VLPs of
DENV serotypes have been demonstrated earlier in different host systems,
mammalian cells,<em>
−

</em> insect cells, insects,<em>
,
</em> plants,
<em>Pichia pastoris</em>,<em>
−



</em> and .<em>
,
</em> Most of these VLPs are lipid-based particles wherein a combination
of DENV proteins (E, prM, and C) are embedded in the lipid bilayer,
exposing the constituent protein-ectodomains on the surface. E protein-based
virus-sized particles have been generated in for DENV-1, whereas, in the case of
DENV-2, EDIII-HBcAg (Hepatitis B virus core antigen) fusion-protein-based
VLPs have been reported. However, full-length
E-ectodomain-based VLP of DENV-2 has not been demonstrated so far.
Such protein-based VLPs, in addition to serving as a vaccine platform,
are more desirable for anti-DENV-2 drug discovery, wherein compounds
may be screened through their E protein-binding ability <em>in
vitro</em>. Therefore, the current study also demonstrates an
elaborate, optimized method to generate proteinaceous VLPs of the
DENV-2 E-protein and its characterization.</p></section><section id="sec2"><h2 class="pmc_sec_title">2. Results</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">2.1. E Protein Expression, Purification, and <em>In Vitro</em> Refolding into Oligomers</h3>
<p>The recombinant
plasmid pET22b-E-DENV-2 was subjected to restriction digestion using <em>Xho</em>I and <em>Nde</em>I enzymes and PCR amplification
using T7 forward and T7 reverse primers to confirm the presence of
the recombinant gene. The products were analyzed further on an agarose
gel, wherein the appearance of a fallout and an amplified DNA band,
corresponding to the size of the DENV-2 envelope ectodomain gene,
suggested its cloning (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02548/suppl_file/ao5c02548_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Figure S1</a>). The identification of the cloned gene was thereafter confirmed
by Sanger sequencing.</p>
<p>The culture of BL21 (DE3) cells, transformed
with pET22b-E-DENV-2, was induced with IPTG and monitored for 5 h
postinduction for the assessment of the E protein expression. The
heterologous E protein was expressed as a fusion of a 6 × His-tag
at its C-terminus. The induced and uninduced culture samples and segregated
pellet and supernatant fractions of the lysed 5 h culture sample were
analyzed on SDS-PAGE. As shown in <a href="#fig1" class="usa-link">Figure 
</a>A, an induced protein band of ∼ 44
kDa was observed migrating with relative bands of the molecular weight
marker in all the samples. The appearance of a prominent induced protein
band in the pellet fraction suggested accumulation of the recombinant
protein in inclusion bodies. Thereafter, the pellet fraction, solubilized
by using 8 M urea, was subjected to purification using Ni-NTA-His-tag
affinity chromatography under denaturing conditions. The purified
recombinant protein samples were pooled and assessed on SDS-PAGE.
As shown in <a href="#fig1" class="usa-link">Figure 
</a>B, the recombinant protein was obtained with ∼ 95% purity.</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332551_ao5c02548_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/211f/12332551/a8551f08b61e/ao5c02548_0001.jpg" loading="lazy" id="gr1" height="604" width="777" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SDS-PAGE
and Western blot analyses of E protein expression, purification,
and <em>in vitro</em> refolding. (A) The recombinant DENV-2
E protein was expressed in BL21
(DE3) cells. The culture was induced after attaining an optical density
(OD<sub>600</sub>) of 0.8; thereafter, samples were drawn every hour
until 5 h. The 5 h induced culture was lysed using ultrasonication,
and pellet (P) and supernatant (S) fractions were separated by centrifugation.
The normalized whole cell lysate samples of uninduced (UI) and IPTG-induced
(1–5 h) cultures, and the pellet and supernatant fraction samples
were analyzed with reference to a molecular weight marker (M) on a
12% SDS-PAGE. An induced protein band of approximately 44 kDa, corresponding
to the molecular weight of the E protein, was observed in the whole
cell lysate samples and the P sample, indicating the accumulation
of the recombinant protein in inclusion bodies (IBs). (B) The recombinant
IBs in the insoluble fraction were washed using 1% sodium deoxycholate
and 2% Triton X-100 and solubilized using denaturing buffer A (50
mM Tris-Cl, 8 M Urea, 20 mM imidazole pH 7.4). The solubilized E protein
sample (D1) was purified via Ni-NTA affinity chromatography, and analyzed
with the flowthrough (D2), and the eluted sample (D3) carrying purified
E protein, on 12% SDS-PAGE. (C) The <em>in vitro</em> refolded
DENV-2 E protein samples were mixed with reducing (R1) and nonreducing
(R2) Laemmli buffers and analyzed using SDS-PAGE. Under reducing conditions
(R1), the monomeric protein migrated at ∼ 44 kDa. In contrast,
under nonreducing conditions (R2), high-molecular-weight species that
were retained at the interface of the stacking-resolving gel were
observed, suggesting disulfide bond-mediated multimerization of the
refolded protein. (D) The identity of the E protein was checked with
Western blotting analysis using an anti-E protein antibody directed
against EDIII. The appearance of the developed dark-colored band on
the nitrocellulose membrane confirmed the identity of the target protein
as a DENV-2 envelope protein at ∼ 44 kDa. In addition to the
predominant monomeric form, higher-order multimeric species were also
faintly detected, further validating the presence of structurally
relevant oligomeric forms.</p></figcaption></figure><p>Subsequently, the purified protein sample was subjected
to refolding
by employing the gradual and differential removal of the chaotropic
and oxidizing reagents. This method led to a 78% recovery yield of
the refolded recombinant protein (<a href="#tbl1" class="usa-link">Table 
</a>). The identity of the E protein was also
confirmed by Western blotting performed using an anti-DENV E protein
antibody directed against domain III. As shown in <a href="#fig1" class="usa-link">Figure 
</a>C, a prominent band of the
refolded E protein was highlighted on the blot, suggesting it to be
the recombinant E protein. The oligomerization of E protein into higher-order
assemblies was investigated using SDS-PAGE, employing the presence
and absence of a reducing agent. As shown in <a href="#fig1" class="usa-link">Figure 
</a>D, in the absence of a reducing agent, the
majority of the E protein was retained at the interface of the stacking-resolving
gel, while only a small portion was observed to enter the resolving
gel, indicating a higher molecular-sized assembly of the E protein.</p>
<section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">1. Presentation of the Key Parameters
Associated with the Expression, Purification, and Refolding of DENV-2
Recombinant Envelope Protein<sup></sup>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr><th colspan="2" align="center" rowspan="1">protein purification
and refolding parameters</th></tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">cell
biomass wet weight (per 1000 mL culture)</td>
<td align="left" colspan="1" rowspan="1">C</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">wet weight of washed inclusion bodies (per 1000 mL culture)</td>
<td align="left" colspan="1" rowspan="1">0.562 g</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">denatured E protein purification yield</td>
<td align="left" colspan="1" rowspan="1">95%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">purified E protein before refolding</td>
<td align="left" colspan="1" rowspan="1">19 mg</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E protein after refolding</td>
<td align="left" colspan="1" rowspan="1">15 mg</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E protein refolding yield</td>
<td align="left" colspan="1" rowspan="1">78%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E protein VLP molecular weight</td>
<td align="left" colspan="1" rowspan="1">876 kDa</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E protein monomer molecular weight (calculated)</td>
<td align="left" colspan="1" rowspan="1">43.8 kDa</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E protein VLP stoichiometry of oligomerization</td>
<td align="left" colspan="1" rowspan="1">20</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
</colgroup>
<thead><tr><th colspan="2" align="center" rowspan="1">dynamic light scattering
analysis of E protein VLPs</th></tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">hydrodynamic radius before refolding</td>
<td align="center" colspan="1" rowspan="1">606 nm</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">polydispersity index
before refolding</td>
<td align="center" colspan="1" rowspan="1">0.538</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">hydrodynamic
radius after refolding</td>
<td align="center" colspan="1" rowspan="1">249.2 nm</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">polydispersity index after refolding</td>
<td align="center" colspan="1" rowspan="1">0.440</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
</colgroup>
<thead><tr><th colspan="2" align="center" rowspan="1">E protein secondary-structure
composition</th></tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">α-helix
content</td>
<td align="center" colspan="1" rowspan="1">14.4%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">β-sheet
content</td>
<td align="center" colspan="1" rowspan="1">31.30%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">β-turns
content</td>
<td align="center" colspan="1" rowspan="1">23.3%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">random coils</td>
<td align="center" colspan="1" rowspan="1">31%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t1fn1">
<sup>a</sup><p class="display-inline">The data provide information about
the target protein’s quality and yields at various purification
and refolding stages.</p>
</div></div></section></section><section id="sec2.2"><h3 class="pmc_sec_title">2.2. Biophysical Analysis of E Protein Oligomers</h3>
<p>The SDS-PAGE
analysis of the E protein refolded samples suggested
their higher-order assembly into oligomers. The structural change
observed during refolding was further validated by the dynamic light
scattering (DLS) technique, performed with the E protein (purified)
samples before and after the refolding. As shown in <a href="#fig2" class="usa-link">Figure 
</a>A, the average particle size
of the unfolded protein sample was recorded with a diameter of 1212
nm [hydrodynamic radius (Rh) of 606 nm] with a polydispersity index
(PdI) of 0.538. On the other hand, the average particle size of the
refolded protein sample was recorded with a diameter of 499.4 nm (Rh
of 249.7 nm) with a PdI of 0.440. The observed particle size of the
E protein VLPs fell within the range of the reported sizes of virus
particles (∼20–500 nm diameter). The PdI, a parameter
indicative of the distribution of different-sized moieties within
a sample, was anticipated to be higher, as observed, due to the expected
presence of a mixed population of different-sized entities in the
refolded sample.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332551_ao5c02548_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/211f/12332551/f7824f98b3a3/ao5c02548_0002.jpg" loading="lazy" id="gr2" height="437" width="696" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Biophysical analyses of the refolded DENV-2 E protein.
(A) The
dynamic light scattering (DLS) patterns of the unfolded and refolded
E protein samples (purified), were generated. The graphs depict the
particle sizes (diameter in nm) and polydispersity indices for respective
samples. (B) The size exclusion chromatography (SEC) was performed
with an <em>in vitro</em> refolded sample of E protein to
segregate the oligomeric and other low molecular weight species and
to analyze the number and the distribution of such species. (C) Analysis
of the secondary-structure content of E protein oligomers using Fourier
Transform Infrared (FTIR) spectroscopy. (D) Far-UV CD-spectrum profile
of E protein oligomers, measured from 190 to 300 nm.</p></figcaption></figure><p>To investigate the presence of different oligomeric
entities
within
the refolded sample, as indicated by the DLS study, the sample was
subjected to size exclusion chromatography (SEC). As shown in <a href="#fig2" class="usa-link">Figure 
</a>B, a single major
elution peak was observed at 46.73 mL of elution volume, in addition
to 3 other comparatively low-intensity peaks, corresponding to small-sized
assemblies, monomeric forms of the E protein, and impurities, respectively.
The size of the first major protein peak, corresponding to the largest
oligomeric assemblies, was calculated to be 876 kDa. The other peaks
that were observed at 69.48 and 78.64 mL elution volumes correlated
with the dimeric and monomeric forms of the refolded E protein, respectively.
These fractions were not followed up on thereafter. The calculated
molecular weight of the E protein is 43.8 kDa, and therefore, the
stoichiometry of assembly of the E protein oligomer was estimated
to be 20 (<a href="#tbl1" class="usa-link">Table 
</a>).
Afterward, the secondary-structure elements of the purified oligomers
were studied using FTIR spectroscopy to assess the proper folding
of the E protein to its natural three-dimensional state. The prominent
peaks at 3341.24, 1636.00, 1217.37, and 1736.27 cm<sup>–1</sup> (<a href="#fig2" class="usa-link">Figure 
</a>C), representing
the amine group, the β-sheet, the carboxylic group, and the
CO stretch, respectively, demonstrated the presence of secondary-structure
elements, further supporting the proper refolding of E protein to
its native form. Furthermore, circular dichroism (CD) spectroscopy
was performed with the purified oligomers in the 180–300 nm
UV range (<a href="#fig2" class="usa-link">Figure 
</a>D).
The data was assessed using the software Spectra Manager, which revealed
the composition of the secondary-structure elements to be 14.4% α
helix, 31.3% β-sheet, 23.3% β-turns, and 31% random coil.
The observed structural description was found in agreement with the
earlier defined crystal structure of the E protein (<a href="https://www.rcsb.org/annotations/1OAN" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rcsb.org/annotations/1OAN</a>), further supporting the proper refolding of the recombinant E-protein
to its native conformation. The analysis of E protein structure in
its VLP form using FTIR and CD, reflected the structural conformation
of E protein molecules while organized as a VLP, validating their
structural integrity and suitability for use in <em>in vitro</em> binding assays.</p></section><section id="sec2.3"><h3 class="pmc_sec_title">2.3. Molecular Docking and MM-GBSA
dG Binding Score
Analysis</h3>
<p>Chebulinic acid (CA), a plant tannin, has shown
inhibitory activity against DENV-2 in virus neutralization assays.
The mechanism of its antiviral activity was proposed based on its
interaction with the viral envelope protein. The information about
the residues of various host receptors involved in making contact
with the viral E protein and correlation of those with the CA binding
residues of E protein (<em>in silico</em>) was used to define
two binding pockets (grids 1 and 2) in the E protein dimer, which
were used in the current study. As shown in <a href="#fig3" class="usa-link">Figure 
</a>, for grid 1, the constituent residues were
contributed by the two subunits of the homodimer, the distal end of
domain II of chain A and domains I and II of chain-B. This binding
pocket harbored a K1 β-hairpin loop that was shown to play a
role in virus-host membrane fusion. Grid
1 residues also include 7 residues (T70, E71, G102, N103, A245, K246,
K247) out of 32 residues that have been shown to interact through
an interface between the E protein and host receptor DC-SIGN, and also include residues G102 and N103 of the
fusion peptide. Therefore, this region was defined as a site for receptor
binding and initiation of the fusion process through the K1 β-hairpin
loop and the fusion peptide. On the other hand, the grid 2 constituent
residues were contributed predominantly by EDIII, the associated hinge
(between EDIII and EDI), and the EDI of chain A. Herein, the binding
pocket harbored an essential residue, K295, which is conserved across
the four serotypes of DENV and has been shown to directly bind to
the GAG receptor, a common viral receptor on various host cells. Grid
2 also has an exposed sugar moiety (Fuc1399), covalently attached
to the glycosylation residue N153, which is conserved in all four
serotypes of DENV. This glycosyl moiety has been shown to participate
in binding mannose receptors, found on the host cells, and mediate
viral entry.<em>
,
</em> Based on the above-cited literature,
the sites selected for docking studies represent suitable binding
pockets for testing the antiviral inhibitory potential of test molecules.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332551_ao5c02548_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/211f/12332551/f8bf1ddc0b68/ao5c02548_0003.jpg" loading="lazy" id="gr3" height="510" width="750" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Depiction
of binding pockets, grids 1 and 2 on E protein, selected
for docking experiments. (A) The table provides a list of the residues
contributed by chains A and/or B of the E protein homodimer that were
involved in making contact with the CA molecule. The CA binding region, constructed by residues at the distal
end of EDII of chain A and EDI and EDII of chain-B was selected as
grid 1, and the one constructed by residues of EDIII of chain A was
selected as grid 2 for docking studies. (B) Cartoon diagram of the
DENV-2 envelope protein showing encircled grid 1 and grid 2 positions,
occupied with docked ligand molecules (galloylglucose molecules).
The subunit chains A and B are depicted with yellow and green colors,
respectively, and the ligand molecules are represented by colored
sticks.</p></figcaption></figure><p>Protein and ligand preparation
were conducted as described in <a href="#sec5" class="usa-link">Section 
</a>. Following the
preparation, two grid boxes were generated for the E protein, as shown
in <a href="#fig3" class="usa-link">Figure 
</a>. Subsequently,
separate molecular docking experiments were performed using the optimized
3D structures of five ligand molecules, namely, 1-galloylglucose,
1,6-digalloylglucose, 1,3,6-trigalloylglucose, 1,2,4,6-tetragalloylglucose,
and 1,2,3,4,6-pentagalloylglucose, at grids 1 and 2, defined within
the E protein dimer, adopted from the crystal structure (PDB ID: <a href="https://www.rcsb.org/structure/1OAN" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1OAN</a>). The results of docking analysis, thus generated for these
five protein–ligand complexes, are presented in <a href="#fig4" class="usa-link">Figure 
</a> and compiled in <a href="#tbl2" class="usa-link">Table 
</a>. In this study, CA and epigallocatechin-3-gallate
(EGCG) molecules were used as positive controls for reference, as
both had demonstrated anti-DENV activity in previous studies.<em>
,
</em> All ligands showed significant docking scores for binding to both
pockets, equivalent to those obtained with the positive control molecules
(CA and EGCG).</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332551_ao5c02548_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/211f/12332551/9e3fd9aac0c7/ao5c02548_0004.jpg" loading="lazy" id="gr4" height="1308" width="750" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Molecular docking analysis of the mono, di, tri, tetra,
and pentagalloylglucose
and control molecules with the DENV-2 envelope protein-ectodomain
dimer. The molecular docking of the E protein with respective ligands
was performed using the <em>Glide</em> module from the Schrodinger
2019–4 suite Maestro. Each subset figure represents the best
docking pose of the ligand within grids 1 and 2 and the corresponding
2D protein–ligand interaction diagram. These images illustrate
important interactions such as hydrogen bonds, π–π
stacking, hydrophobic contacts, and other noncovalent interactions
crucial for protein–ligand binding. The epigallocatechin-3-gallate
(EGCG) and chebulinic acid (CA) were used as control molecules, which
have shown anti-DENV activity in the past.<em>
,
</em>.</p></figcaption></figure><section class="tw xbox font-sm" id="tbl2"><h4 class="obj_head">2. Presentation of the Docking and MM-GBSA
Binding Free Energy Scores Calculated for Docking of Mono, Di, Tri,
Tetra, and Pentagalloylglucose, Epigallocatechin-3-gallate (EGCG),
and Chebulinic Acid (CA) with DENV-2 Envelope Protein Dimer (1OAN),
at Two Sites (grid 1 and grid 2), Using Schrodinger Software <em>Glide</em>
<sup></sup>
.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332551_ao5c02548_0008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/211f/12332551/0d0d6c7782d4/ao5c02548_0008.jpg" loading="lazy" id="fx1" height="337" width="696" alt="2.3."></a></p>
<div class="p text-right font-secondary"><a href="table/tbl2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t2fn1">
<sup>a</sup><p class="display-inline">The residues of
the DENV-2 E protein
interacting with the respective compounds have also been listed. Each
residue is labeled by its three-letter code:position/number on the
polypeptide chain and the corresponding chain identifiers (A or B).
The E dimer residues, commonly engaged by different ligand molecules,
at grids 1 and 2 are depicted by the same color</p>
</div></div></section><p>The formation of a considerable
number of hydrogen bonds and interactions,
like halogen bonds, metal coordination, Pi-Pi stacking, Pi cation,
and salt bridge, at the ligand-protein interfaces (<a href="#tbl2" class="usa-link">Table 
</a> and <a href="#fig4" class="usa-link">Figure 
</a>), indicated specific interactions. Based
on this analysis, PGG recorded the lowest docking scores, equivalent
to CA, followed by trigalloylglucose, digalloylglucose, tetragalloylglucose,
and monogalloylglucose. The molecular docking data were further validated
by the MM-GBSA (Molecular Mechanics-Generalized Born Surface Area)
binding free energy (ΔG) score assessment, with lower values
suggesting a more stable binding conformation. As shown in <a href="#tbl2" class="usa-link">Table 
</a>, all five test ligand
molecules exhibited significant binding scores, wherein the trigalloylglucose
was found to be the most stable at grid 2 followed by PGG, whereas
at grid 1, PGG and digalloylglucose showed equivalent and the lowest
values among the test molecules. Collectively, the docking and MM-GBSA
scores of the top two molecules, PGG and trigalloylglucose, were falling
in between those shown by EGCG and CA, highlighting their potential
as antivirals.</p>
<p>In addition to the high-resolution crystal structure
of the soluble
ectodomain of E from dengue virus type 2 (1OAN), the cryo-electron
microscopic structure of DENV-2 virion, defining the atomic structures
of membrane proteins E and M, at 3.5-Å resolution (PDB ID: <a href="https://www.rcsb.org/structure/3J27" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">3J27</a>) has also been reported. Later, Wen-Fan Shen and group reconstructed
a cryo-EM structure of a mature VLP of DENV-2, produced in COS-1 cells
(mD2VLP), by fitting the atomic E and M surface proteins, using PDB: <a href="https://www.rcsb.org/structure/3J27" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">3J27</a>, into the VLP cryo-EM
density map (mD2VLP_3J27). The 3J27 and
mD2VLP_3J27, being the cryo-EM structures of the DENV-2 virion and
VLP, respectively, although at low resolution, offer an E dimer arrangement
closer to that of the native, live virion. Therefore, to validate
the docking results obtained with the crystal structure 1OAN, docking
studies were repeated with cryo-EM structures 3J27 and mD2VLP_3J27.
The data thus generated have been compiled and presented in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02548/suppl_file/ao5c02548_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Tables S1 and S2 and Figures S2 and S3</a>. Furthermore, the docking and MM-GBSA binding Δ<em>G</em> scores from the three docking experiments were compared and analyzed
for deviations in the binding patterns. As depicted in <a href="#tbl3" class="usa-link">Table 
</a>, the docking scores of test
and control compounds for binding E dimers originating from three
different structures showed mild variation. The binding scores remained
low enough to suggest a biologically relevant interaction for all,
except for EGCG, which in one of the three docking setups reported
higher docking scores with the mD2VLP_3J27 E dimer (<a href="#tbl3" class="usa-link">Table 
</a>). Among all the compounds tested,
PGG showed the most consistent, lower docking and MM-GBSA binding
ΔG scores across the docking studies, especially the one conducted
with crystal structure 1OAM and cryo-EM structure mD2VLP_3J27. The
docking studies with cryo-EM structure 3J27 also produced equivalent
scores. These studies, therefore, support the suitability of the crystal
structure 1OAN for conducting docking studies, screening anti-DENV
molecules, and suggest that PGG may bind the E protein on the DENV-2
virion surface and exhibit anti-DENV activity.</p>
<section class="tw xbox font-sm" id="tbl3"><h4 class="obj_head">3. Presentation of the Docking and MM-GBSA
Binding Free Energy (Δ<em>G</em>) Scores Calculated
for Complexes of Mono-, Di-, Tri-, Tetra-, and Pentagalloylglucose,
Epigallocatechin-3-gallate (EGCG), Chebulinic Acid (CA), and E Dimer
of Dengue Virus Serotype 2 (DENV-2)<sup></sup>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
</colgroup>
<thead>
<tr>
<th rowspan="2" align="center" colspan="1"> </th>
<th colspan="4" align="center" rowspan="1">crystal
structure 1OAN<hr>
</th>
<th colspan="4" align="center" rowspan="1">cryo-EM
structure 3J27<hr>
</th>
<th colspan="4" align="center" rowspan="1">cryo-EM
structure mD2VLP 3J27<hr>
</th>
</tr>
<tr>
<th colspan="2" align="center" rowspan="1">docking score</th>
<th colspan="2" align="center" rowspan="1">MMCBSA binding
Δ<em>G</em> score</th>
<th colspan="2" align="center" rowspan="1">docking score</th>
<th colspan="2" align="center" rowspan="1">MM-GBSA binding
Δ<em>G</em> score</th>
<th colspan="2" align="center" rowspan="1">docking score</th>
<th colspan="2" align="center" rowspan="1">MMCBSA binding
Δ<em>G</em> score</th>
</tr>
<tr>
<th align="center" colspan="1" rowspan="1">compound name</th>
<th align="center" colspan="1" rowspan="1">grid
1</th>
<th align="center" colspan="1" rowspan="1">grid 2</th>
<th align="center" colspan="1" rowspan="1">grid 1</th>
<th align="center" colspan="1" rowspan="1">grid 2</th>
<th align="center" colspan="1" rowspan="1">grid 1</th>
<th align="center" colspan="1" rowspan="1">grid 2</th>
<th align="center" colspan="1" rowspan="1">grid 1</th>
<th align="center" colspan="1" rowspan="1">grid 2</th>
<th align="center" colspan="1" rowspan="1">grid 1</th>
<th align="center" colspan="1" rowspan="1">grid 2</th>
<th align="center" colspan="1" rowspan="1">grid 1</th>
<th align="center" colspan="1" rowspan="1">grid 2</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">1-galloylglucose</td>
<td align="center" colspan="1" rowspan="1">–9.11 8</td>
<td align="center" colspan="1" rowspan="1">–8.85</td>
<td align="center" colspan="1" rowspan="1">–56.5</td>
<td align="center" colspan="1" rowspan="1">–36.72</td>
<td align="center" colspan="1" rowspan="1">–6.391</td>
<td align="center" colspan="1" rowspan="1">–8.09</td>
<td align="center" colspan="1" rowspan="1">–39.41</td>
<td align="center" colspan="1" rowspan="1">–39.8</td>
<td align="center" colspan="1" rowspan="1">–6.631</td>
<td align="center" colspan="1" rowspan="1">–7.234</td>
<td align="center" colspan="1" rowspan="1">–41.5</td>
<td align="center" colspan="1" rowspan="1">–40</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1,6-digalloylglucose</td>
<td align="center" colspan="1" rowspan="1">–11.6</td>
<td align="center" colspan="1" rowspan="1">–10.78</td>
<td align="center" colspan="1" rowspan="1">–64.9</td>
<td align="center" colspan="1" rowspan="1">–62.6</td>
<td align="center" colspan="1" rowspan="1">0.66</td>
<td align="center" colspan="1" rowspan="1">–6.277</td>
<td align="center" colspan="1" rowspan="1">–56.7</td>
<td align="center" colspan="1" rowspan="1">–59.4</td>
<td align="center" colspan="1" rowspan="1">–10.05</td>
<td align="center" colspan="1" rowspan="1">–6.263</td>
<td align="center" colspan="1" rowspan="1">–58.2</td>
<td align="center" colspan="1" rowspan="1">–48.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1,3,6-trigalloylglucose</td>
<td align="center" colspan="1" rowspan="1">–12.51</td>
<td align="center" colspan="1" rowspan="1">–9.897</td>
<td align="center" colspan="1" rowspan="1">–49.7</td>
<td align="center" colspan="1" rowspan="1">–68.56</td>
<td align="center" colspan="1" rowspan="1">–9.962</td>
<td align="center" colspan="1" rowspan="1">–8.867</td>
<td align="center" colspan="1" rowspan="1">–75.7</td>
<td align="center" colspan="1" rowspan="1">–70.4</td>
<td align="center" colspan="1" rowspan="1">–7.959</td>
<td align="center" colspan="1" rowspan="1">–4.242</td>
<td align="center" colspan="1" rowspan="1">–53</td>
<td align="center" colspan="1" rowspan="1">–50.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1,2,4,6-tetragalloylglucose</td>
<td align="center" colspan="1" rowspan="1">–10.37</td>
<td align="center" colspan="1" rowspan="1">–8.405</td>
<td align="center" colspan="1" rowspan="1">–49</td>
<td align="center" colspan="1" rowspan="1">–47.52</td>
<td align="center" colspan="1" rowspan="1">–12.88</td>
<td align="center" colspan="1" rowspan="1">–11.68</td>
<td align="center" colspan="1" rowspan="1">–81.8</td>
<td align="center" colspan="1" rowspan="1">–69.4</td>
<td align="center" colspan="1" rowspan="1">–9.508</td>
<td align="center" colspan="1" rowspan="1">–9.76</td>
<td align="center" colspan="1" rowspan="1">–74.1</td>
<td align="center" colspan="1" rowspan="1">–52.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>1,2,3,4,6-pentagalloylglucose</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>–13.78</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>–7.638</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>–63.1</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>–62.68</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>–11.89</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>–8.781</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>–85</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>–99.5</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>–12.37</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>–7.04</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>–62.2</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>–63.8</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">chebulinic acid</td>
<td align="center" colspan="1" rowspan="1">–13.08</td>
<td align="center" colspan="1" rowspan="1">–11.13</td>
<td align="center" colspan="1" rowspan="1">–91.9</td>
<td align="center" colspan="1" rowspan="1">–92.64</td>
<td align="center" colspan="1" rowspan="1">–10.66</td>
<td align="center" colspan="1" rowspan="1">–8.229</td>
<td align="center" colspan="1" rowspan="1">–38.2</td>
<td align="center" colspan="1" rowspan="1">–50.8</td>
<td align="center" colspan="1" rowspan="1">–8.502</td>
<td align="center" colspan="1" rowspan="1">–10.48</td>
<td align="center" colspan="1" rowspan="1">–29.2</td>
<td align="center" colspan="1" rowspan="1">–54.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">epigallocatechin-3-gallate</td>
<td align="center" colspan="1" rowspan="1">–6.683</td>
<td align="center" colspan="1" rowspan="1">–9.615</td>
<td align="center" colspan="1" rowspan="1">–39.7</td>
<td align="center" colspan="1" rowspan="1">–70.23</td>
<td align="center" colspan="1" rowspan="1">–8.606</td>
<td align="center" colspan="1" rowspan="1">–8.4</td>
<td align="center" colspan="1" rowspan="1">–73.1</td>
<td align="center" colspan="1" rowspan="1">–30.9</td>
<td align="center" colspan="1" rowspan="1">–2.753</td>
<td align="center" colspan="1" rowspan="1">–1.996</td>
<td align="center" colspan="1" rowspan="1">–47.4</td>
<td align="center" colspan="1" rowspan="1">–39.7</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t3fn1">
<sup>a</sup><p class="display-inline">The three E protein structures used
in the study were adopted from the crystal structure, PDB ID: <a href="https://www.rcsb.org/structure/1OAN" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1OAN</a>, cryo-EM structure
PDB ID: <a href="https://www.rcsb.org/structure/3J27" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">3J27</a>, and cryo-EM structure mD2VLP_3J27. Docking was performed at two
binding sites (grid 1 and grid 2) using Schrodinger software Glide.</p>
</div></div></section></section><section id="sec2.4"><h3 class="pmc_sec_title">2.4. Analysis
of ADMET (Absorption, Distribution,
Metabolism, Excretion, and Toxicity) Properties</h3>
<p>ADMET study
helps determine a compound’s potential and efficacy as a drug
since it assesses its physicochemical characteristics, biological
activities, and drug-likeness. These characteristics of a compound
may be assessed through several descriptors, such as CNS activity,
molecular weight, solvent accessibility, hydrophobic and polar surface
areas, volume, molecular interactions, solubility, permeability, potential
metabolic reactions, binding affinity to human serum albumin, oral
absorption, etc. The ADMET parameters were predicted for all five
test compounds using QikProp module of the Schrodinger suite, which are detailed in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02548/suppl_file/ao5c02548_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Table S3</a>. The data suggest that as the number of galloyl moieties,
the functional groups on the glucose molecule, increases, parameters
linked to bioavailability and permeability decrease. On the other
hand, the hydrophobicity and solubility scores of all the compounds
were satisfactory, indicating their high solubility in an aqueous
environment, which is the most desirable parameter for their function.
The test compounds showed moderate to poor pharmacokinetic properties
that favor conventional drug-like properties. These parameters are
primarily designed for small-sized drug compounds, especially those
required to cross the cellular membranes to gain access to target
molecules or sites. The test compounds used in the study are neither
small molecules nor required to cross the plasma membranes for their
activity. Since the test compound’s activity of binding the
viral particles warrants their presence in extracellular spaces, the
observed ADMET data support their potential as host-entry inhibitors.</p></section><section id="sec2.5"><h3 class="pmc_sec_title">2.5. Molecular Dynamics Simulation and Electrostatic
Complementarity Studies</h3>
<p>The molecular dynamics simulations
provide detailed information about the stability and structural dynamics
of the protein–ligand interactions. Herein, the protein ‘Root-Mean
Square Deviation’ (RMSD) values indicate the structural deviation
of the protein backbone from its initial conformation; the ligand
RMSD values reflect the movement and stability of the ligand within
the binding pocket; the protein ‘Root-Mean Square Fluctuation’
(RMSF) data captures the per-residue flexibility of the protein; the
hydrogen bonding between protein and ligand determines the overall
strength of the interactions, and the radius of gyration (Rg) measures
the overall compactness of the protein structure during the simulation,
indicating the structure unfolding. Initial 100 ns simulations revealed
that grid 2 consistently offered greater stability than grid 1, based
on comparative analyses of protein–ligand RMSD, protein RMSF,
hydrogen bonding, and Rg values (<a href="#tbl4" class="usa-link">Table 
</a> and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02548/suppl_file/ao5c02548_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S4–S7</a>). Protein RMSD trends mostly stayed within acceptable ranges (&lt;5
Å), confirming the structural integrity of the protein during
100 ns simulations (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02548/suppl_file/ao5c02548_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4</a>). However,
the ligand RMSD shows diverse patterns from stable retention to partial
or complete dissociation, with notably higher stable values at grid
2 for tetragalloylglucose, PGG, and CA (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02548/suppl_file/ao5c02548_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4</a>). Protein RMSF profiles indicated moderate flexibility across
most protein regions, with notably higher fluctuations at grid 1,
suggesting localized instability (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02548/suppl_file/ao5c02548_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S5</a>). Persistent hydrogen bonding was observed at grid 2 except for
trigalloylglucose, while grid 1 displayed delayed or weaker interactions
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02548/suppl_file/ao5c02548_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S6</a>). Additionally, Rg analysis
showed grid 1 often approaching ∼ 60 Å (indicative of
partial unfolding), whereas grid 2 remained compact (Rg &lt; 35 Å)
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02548/suppl_file/ao5c02548_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S7</a>). The key MD simulation parameters
defining the stability of protein–ligand complexes, including
protein RMSD, ligand RMSD, protein RMSF, number of hydrogen bonds,
and Rg, estimated over a 100 ns simulation, are summarized in <a href="#tbl4" class="usa-link">Table 
</a>. Among all galloylglucose
ligands, PGG displayed the most consistent and balanced stability
pattern over the 100-ns time frame at grid 2.</p>
<section class="tw xbox font-sm" id="tbl4"><h4 class="obj_head">4. Presentation
of the Key Molecular
Dynamics Simulation Parameters Defining the Stability of E Protein-Ligand
Complexes, Including Protein RMSD, Ligand RMSD, Protein RMSF, Number
of Hydrogen Bonds, and Rg, Estimated Over 100 ns at Grids 1 and 2<sup></sup>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
</colgroup>
<thead>
<tr>
<th align="center" colspan="1" rowspan="1"> </th>
<th align="center" colspan="1" rowspan="1"> </th>
<th colspan="3" align="center" rowspan="1">protein
RMSD (Å)<hr>
</th>
<th colspan="3" align="center" rowspan="1">ligand
RMSD (Å)<hr>
</th>
<th colspan="3" align="center" rowspan="1">protein
RMSF (Å)<hr>
</th>
<th colspan="2" align="center" rowspan="1">protein–ligand
H-bonds<hr>
</th>
<th colspan="3" align="center" rowspan="1">radius
of gyration (Å)<hr>
</th>
</tr>
<tr>
<th align="center" colspan="1" rowspan="1">compound name</th>
<th align="center" colspan="1" rowspan="1">grid</th>
<th align="center" colspan="1" rowspan="1">initial value (Å)</th>
<th align="center" colspan="1" rowspan="1">peak (Å)</th>
<th align="center" colspan="1" rowspan="1">interaction
stabilization pattern at 100 ns
(Å)</th>
<th align="center" colspan="1" rowspan="1">initial value (Å)</th>
<th align="center" colspan="1" rowspan="1">peak (Å)</th>
<th align="center" colspan="1" rowspan="1">interaction
stabilization pattern at 100 ns
(Å)</th>
<th align="center" colspan="1" rowspan="1">peak (Å)</th>
<th align="center" colspan="1" rowspan="1">range (Å)</th>
<th align="center" colspan="1" rowspan="1">interaction
stabilization pattern at 100 ns
(Å)</th>
<th align="center" colspan="1" rowspan="1">ieak H-bonds</th>
<th align="center" colspan="1" rowspan="1">average H-bonds</th>
<th align="center" colspan="1" rowspan="1">peak (Å)</th>
<th align="center" colspan="1" rowspan="1">range (Å)</th>
<th align="center" colspan="1" rowspan="1">interaction pattern</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">monogalloylglucose</td>
<td align="left" colspan="1" rowspan="1">grid 1</td>
<td align="center" colspan="1" rowspan="1">2.4</td>
<td align="center" colspan="1" rowspan="1">6.4</td>
<td align="center" colspan="1" rowspan="1">2.4–3.5</td>
<td align="center" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">75</td>
<td align="center" colspan="1" rowspan="1">43–46</td>
<td align="center" colspan="1" rowspan="1">4.8</td>
<td align="center" colspan="1" rowspan="1">1–4.2</td>
<td align="center" colspan="1" rowspan="1">l.8–2.8</td>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">60</td>
<td align="center" colspan="1" rowspan="1">40–60</td>
<td align="left" colspan="1" rowspan="1">stabilization at 35–40 Å, 80 ns onward</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">grid 2</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">7</td>
<td align="center" colspan="1" rowspan="1">4–5</td>
<td align="center" colspan="1" rowspan="1">2.5</td>
<td align="center" colspan="1" rowspan="1">20</td>
<td align="center" colspan="1" rowspan="1">14–20</td>
<td align="center" colspan="1" rowspan="1">4.2</td>
<td align="center" colspan="1" rowspan="1">1.2–3.2</td>
<td align="center" colspan="1" rowspan="1">1.2–2.5</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">40</td>
<td align="center" colspan="1" rowspan="1">26–40</td>
<td align="left" colspan="1" rowspan="1">stabilization at 26–30
Å, 50 ns onward</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">digalloylglucose</td>
<td align="left" colspan="1" rowspan="1">grid 1</td>
<td align="center" colspan="1" rowspan="1">2–4</td>
<td align="center" colspan="1" rowspan="1">6.4</td>
<td align="center" colspan="1" rowspan="1">3.2–4.8</td>
<td align="center" colspan="1" rowspan="1">2.5</td>
<td align="center" colspan="1" rowspan="1">20</td>
<td align="center" colspan="1" rowspan="1">12–15</td>
<td align="center" colspan="1" rowspan="1">4.8</td>
<td align="center" colspan="1" rowspan="1">1–4</td>
<td align="center" colspan="1" rowspan="1">1.5–3</td>
<td align="left" colspan="1" rowspan="1">NA</td>
<td align="left" colspan="1" rowspan="1">NA</td>
<td align="center" colspan="1" rowspan="1">60</td>
<td align="center" colspan="1" rowspan="1">45–60</td>
<td align="left" colspan="1" rowspan="1">stabilization at 35–45
Å</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">grid 2</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">2.5–3.5</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">6–8</td>
<td align="center" colspan="1" rowspan="1">4.8</td>
<td align="center" colspan="1" rowspan="1">1–4</td>
<td align="center" colspan="1" rowspan="1">1.6–2.5</td>
<td align="left" colspan="1" rowspan="1">NA</td>
<td align="left" colspan="1" rowspan="1">NA</td>
<td align="center" colspan="1" rowspan="1">60</td>
<td align="center" colspan="1" rowspan="1">45–60</td>
<td align="left" colspan="1" rowspan="1">stabilization at 45–60
Å</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trigalloylglucose</td>
<td align="left" colspan="1" rowspan="1">grid
1</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">9</td>
<td align="center" colspan="1" rowspan="1">4–6</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">18</td>
<td align="center" colspan="1" rowspan="1">14–17</td>
<td align="center" colspan="1" rowspan="1">4.5</td>
<td align="center" colspan="1" rowspan="1">1.2–4.5</td>
<td align="center" colspan="1" rowspan="1">2.4–3.6</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">60</td>
<td align="center" colspan="1" rowspan="1">45–60</td>
<td align="left" colspan="1" rowspan="1">stabilization at 45 Å,
90 ns onward</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">grid 2</td>
<td align="center" colspan="1" rowspan="1">2.4</td>
<td align="center" colspan="1" rowspan="1">6.4</td>
<td align="center" colspan="1" rowspan="1">3.5–4.5</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">105</td>
<td align="center" colspan="1" rowspan="1">60–90</td>
<td align="center" colspan="1" rowspan="1">6</td>
<td align="center" colspan="1" rowspan="1">0.8–4</td>
<td align="center" colspan="1" rowspan="1">0.8–3.2</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="center" colspan="1" rowspan="1">34</td>
<td align="center" colspan="1" rowspan="1">26–34</td>
<td align="left" colspan="1" rowspan="1">stabilization at 26–30
Å, 70 ns onward</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">tetragalloylglucose</td>
<td align="left" colspan="1" rowspan="1">grid 1</td>
<td align="center" colspan="1" rowspan="1">2.4</td>
<td align="center" colspan="1" rowspan="1">5.6</td>
<td align="center" colspan="1" rowspan="1">3.2–3.8</td>
<td align="center" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">10</td>
<td align="center" colspan="1" rowspan="1">7–9</td>
<td align="center" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">0 8–4</td>
<td align="center" colspan="1" rowspan="1">0.8–2.5</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">60</td>
<td align="center" colspan="1" rowspan="1">35–40</td>
<td align="left" colspan="1" rowspan="1">stabilization at 32–37 Å, 50 ns onward</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">grid 2</td>
<td align="center" colspan="1" rowspan="1">2.5</td>
<td align="center" colspan="1" rowspan="1">9</td>
<td align="center" colspan="1" rowspan="1">3–4.5</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">9</td>
<td align="center" colspan="1" rowspan="1">3–4.8</td>
<td align="center" colspan="1" rowspan="1">4.8</td>
<td align="center" colspan="1" rowspan="1">0.8–4</td>
<td align="center" colspan="1" rowspan="1">0.8–2.5</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="center" colspan="1" rowspan="1">38</td>
<td align="center" colspan="1" rowspan="1">28–38</td>
<td align="left" colspan="1" rowspan="1">stabilization at 28–30
Å, 90 ns onward</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">pentagalloylglucose</td>
<td align="left" colspan="1" rowspan="1">grid 1</td>
<td align="center" colspan="1" rowspan="1">2.4</td>
<td align="center" colspan="1" rowspan="1">7 2</td>
<td align="center" colspan="1" rowspan="1">2.6–3.2</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">64</td>
<td align="center" colspan="1" rowspan="1">28–30</td>
<td align="center" colspan="1" rowspan="1">4.5</td>
<td align="center" colspan="1" rowspan="1">1.2–4.5</td>
<td align="center" colspan="1" rowspan="1">1.8–4</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">60</td>
<td align="center" colspan="1" rowspan="1">35–45</td>
<td align="left" colspan="1" rowspan="1">stabilization at 35–38 Å, 60 ns onward</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">grid 2</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">6</td>
<td align="center" colspan="1" rowspan="1">3–4.5</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">6</td>
<td align="center" colspan="1" rowspan="1">3–5</td>
<td align="center" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">1.2–3</td>
<td align="center" colspan="1" rowspan="1">1.8–3</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="center" colspan="1" rowspan="1">50</td>
<td align="center" colspan="1" rowspan="1">30–40</td>
<td align="left" colspan="1" rowspan="1">stabilization at 25–30
Å, 70 ns onward</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">chebulinic acid</td>
<td align="left" colspan="1" rowspan="1">grid 1</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">5.4</td>
<td align="center" colspan="1" rowspan="1">3–3.6</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">51</td>
<td align="center" colspan="1" rowspan="1">49–51</td>
<td align="center" colspan="1" rowspan="1">5.4</td>
<td align="center" colspan="1" rowspan="1">1.2–4</td>
<td align="center" colspan="1" rowspan="1">1.2–3</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">60</td>
<td align="center" colspan="1" rowspan="1">37–42</td>
<td align="left" colspan="1" rowspan="1">stabilization at 35–40 Å, 30 ns onward</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">grid 2</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">9</td>
<td align="center" colspan="1" rowspan="1">5–6.5</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">3–4.5</td>
<td align="center" colspan="1" rowspan="1">8.5</td>
<td align="center" colspan="1" rowspan="1">1–3</td>
<td align="center" colspan="1" rowspan="1">1.2–2.8</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">36</td>
<td align="center" colspan="1" rowspan="1">25–36</td>
<td align="left" colspan="1" rowspan="1">stabilization at 28–30
Å, 80 ns onward</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t4fn1">
<sup>a</sup><p class="display-inline">Overall, complexes
formed at grid
2 demonstrate improved values across all simulation parameters, indicating
more stable protein–ligand conformations compared to those
at grid 1. (RMSD- Root-Mean Square Deviation, RMSF- Root-Mean Square
Fluctuation, Rg- Radius of gyration, NA-Not available).</p>
</div></div></section><p>To analyze the stability of the
ligand-protein complexes, extended
period simulations were performed at grid 2, binding to which ligands
had recorded lower RMSD values in the previous analysis. Herein, PGG
and EGCG were analyzed by using a 500 ns simulation, whereas the rest
of the test molecules and CA were analyzed by using a 200 ns simulation.
These extended simulations (200 and 500 ns) at grid 2 provided a better
understanding of protein–ligand complex stability. As shown
in the RMSD plots (<a href="#fig5" class="usa-link">Figures 
</a> and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02548/suppl_file/ao5c02548_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S8</a>), protein RMSD values remained
within a typical range (1–6 Å), indicating reasonable
conformational flexibility, especially for PGG, EGCG, and CA (<a href="#fig5" class="usa-link">Figure 
</a>) and monogalloylglucose,
digalloylglucose (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02548/suppl_file/ao5c02548_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S8</a>).</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332551_ao5c02548_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/211f/12332551/8f8268483092/ao5c02548_0005.jpg" loading="lazy" id="gr5" height="198" width="696" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Molecular dynamics
simulation analysis of E protein dimer, in complex
with three ligands, pentagalloylglucose, epigallocatechin gallate,
and chebulinic acid, at extended periods (500 and 200 ns). The graphs
represent protein and ligand root-mean-square deviation (RMSD) plots
wherein the blue graph represents the protein RMSD (left <em>Y</em>-axis), and the red graph depicts the ligand RMSD (right <em>Y</em>-axis).</p></figcaption></figure><p>As shown in <a href="#fig5" class="usa-link">Figure 
</a>, the ligand RMSD
values showed variability; PGG shows unbinding
events in the initial and later stages, while EGCG and CA show late-stage
unbinding events, implying possible ligand dissociation or reorientation
within the binding pocket. In the case of other test compounds, di-
and trigalloylglucose showed stable binding as reflected by their
ligand RMSD values (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02548/suppl_file/ao5c02548_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S8</a>). Notably,
CA and EGCG were included as positive controls in this study, as both
compounds have demonstrated significant inhibitory activity against
dengue virus both <em>in silico</em> and <em>in vitro</em>,<em>
,
</em> and serve as positive references to compare and assess
the binding behavior of the candidate ligands (galloylglucose molecules).
Among all candidates, PGG remained a more stable and promising ligand,
particularly at grid 2, showing favorable long-term interaction dynamics
equivalent to those of CA and EGCG.</p>
<p>Furthermore, the electrostatic
complementarity between PGG and
the binding sites (grids 1 and 2), the electrostatic potential of
the ligand, and the binding pocket were analyzed using electrostatic
surface potential (ESP) maps, for PGG, CA, and EGCG. These maps highlight
the positional orientation of the ligand molecule within the binding
pocket and also reveal regions of surface electrostatic complementarity,
where opposite charges align between the ligand and protein, suggesting
favorable electrostatic interactions. The ESP maps were generated
using Schrodinger Maestro 14.2., illustrating the charge distribution
across the ligand-binding interface, wherein regions colored in red
denote electronegative potential, while blue regions indicate electropositive
potential, providing visual hints for electrostatic compatibility
between the ligand and protein surfaces. ESP analyses were performed
for key complexes involving PGG, EGCG, and CA at both binding grids
(<a href="#fig6" class="usa-link">Figure 
</a>).</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332551_ao5c02548_0006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/211f/12332551/6eb8782cd688/ao5c02548_0006.jpg" loading="lazy" id="gr6" height="400" width="696" alt="6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Electrostatic
complementarity analysis of ligands pentagalloylglucose,
epigallocatechin gallate, and chebulinic acid for binding pockets
(grids 1 and 2) presented on DENV-2 E protein dimer. The electrostatic
surface potential maps of the DENV-2 envelope protein (PDB ID: <a href="https://www.rcsb.org/structure/1OAN" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1OAN</a>) complexes with
pentagalloylglucose, epigallocatechin gallate, and chebulinic acid
were generated using the Schrödinger Maestro. The protein is
displayed in cartoon representation, and the ligands are shown as
sticks. The surface potential is colored on a scale from red (electronegative)
to blue (electropositive), illustrating the charge distribution around
the protein–ligand-binding interface. The electrostatic complementarity
(EC) scores for each complex were estimated using the Schrodinger
suite, BioLuminate. The score values
range from +1 to −1, depicting perfect complementarity to electrostatic
clash. A zero EC score indicates no correlation.</p></figcaption></figure><p>The electrostatic complementarity was assessed
using EC scores,
calculated using the Schrodinger suite, BioLuminate. The EC scores range from +1 to 0 to −1 values, referring
to perfect electrostatic complementarity, to no correlation, and to
electrostatic clash, respectively. As shown in <a href="#fig6" class="usa-link">Figure 
</a>, the EC scores revealed values above 0.5
at grid 1 for all three ligands, including PGG, indicating strong
electrostatic surface complementarity. In contrast, grid 2 exhibited
EC scores of approximately 0.4, reflecting a moderate electrostatic
compatibility. These results further corroborate the docking and MD
simulation findings about PGG-E-protein complex stability, underlining
the contribution of both sites toward effective binding. When compared,
the docking scores, MD simulation data, and EC scores of PGG were
found to be equivalent to those of CA and EGCG, for some parameters,
even performing better than EGCG. These results therefore support
the anti-DENV-2 characteristics of PGG, identifying it as a potential
antidengue therapeutic agent.</p></section><section id="sec2.6"><h3 class="pmc_sec_title">2.6. 
<em>In Vitro</em> Protein–Ligand
Binding Assay</h3>
<p>Through docking and molecular dynamics simulation
studies, PGG emerged as the most stable and high-affinity binder of
the E protein. The E protein-binding kinetics of PGG was investigated <em>in vitro</em> using a biolayer interferometry-based assay. The
assay was performed at four different concentrations of ligands, ranging
from 6.25 to 100 μM. Herein, PGG showed high affinity, dose-dependent
binding to the E protein VLPs, which were immobilized on Ni-NTA biosensors
using 6 × His-tag mediated chemistry. Subsequently, in agreement
with the molecular docking studies that were performed at two different
pockets, a 1:2 heterotypic ligand-binding model was adopted to plot
and analyze the PGG-VLPs binding data. The binding model provided
two equilibrium dissociation constants for multiple-site binding,
a <em>K</em>
<sub>D</sub>1 of 1.07 nM and a second <em>K</em>
<sub>D</sub>2 of 25 nM (<a href="#fig7" class="usa-link">Figure 
</a>A), wherein a correlation value (<em>R</em>
<sup>2</sup>) of 0.8982 suggested a good correlation between
the concentration and the observed binding of the compound.</p>
<figure class="fig xbox font-sm" id="fig7"><h4 class="obj_head">7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332551_ao5c02548_0007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/211f/12332551/94da0c1b7ae3/ao5c02548_0007.jpg" loading="lazy" id="gr7" height="485" width="696" alt="7"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Binding affinity
analysis of pentagalloylglucose (PGG) and control
molecule epigallocatechin-3-gallate (EGCG) for E protein VLPs, using
biolayer interferometry. The graphs depict the association and dissociation
curves of ligands PGG (A) and EGCG (B) upon binding to and dissociating
from the VLPs of DENV-2 E protein immobilized onto the biosensors.
Four curves represent four different concentrations of the respective
ligands used in the range of 6.25–100 μM. A 1:2 heterotypic
ligand-binding model was adopted to process the association and dissociation
curves, which provided two equilibrium dissociation constants for
PGG/EGCG binding to two distinct pockets on the E protein VLP with
high correlation (<em>R</em>
<sup>2</sup>) values.</p></figcaption></figure><p>A similar binding assay, performed with EGCG (positive
control)
under the same experimental setup, produced equivalent binding affinity
and kinetics data, as presented in <a href="#fig7" class="usa-link">Figure 
</a>B. As per the binding kinetics, the two equilibrium
dissociation constants, <em>K</em>
<sub>D</sub>1 and 2, were
observed to be 0.06 nM and 4.5 μM, respectively, with a correlation
(<em>R</em>
<sup>2</sup>) of 0.9505. PGG showed high-affinity
binding at both sites, whereas EGCG showed high affinity for only
one site, signifying the limited use of available sites. The high-affinity
interaction between PGG and E protein, observed during <em>in
vitro</em> binding assay, substantiates the observations made
during the docking and MD simulation studies, signifying the antiviral
potential of PGG.</p></section></section><section id="sec3"><h2 class="pmc_sec_title">3. Discussion</h2>
<p>In this
study, two objectives were aligned to generate a system
that serves synchronously towards the anti-dengue drug discovery and
development. In this quest, a method of generating E protein-based
VLPs of DENV-2 using a cost- and time-effective bacterial system was
developed. The adopted method led to the successful expression, purification,
and refolding of the full-length ectodomain of dengue virus serotype
2 (DENV-2) envelope E protein in BL21 cells. In the past decade, many studies focused on generating
either lipid-based or protein VLPs of DENV, as summarized in <a href="#tbl5" class="usa-link">Table 
</a>. They explored their
ability to elicit a strong immune response as vaccine candidates.
These VLPs, especially lipid-based, mimic virus particles and the
arrangement of proteins like E, prM, and C, depending on the combination
of the proteins used in VLP formation, and generate neutralizing immune
response. Often, the presence of more than one protein constituent,
prM/C, etc., in addition to E protein, renders these VLPs unsuitable
for screening E protein inhibitors, as nonenvelope protein molecules
may serve as off-targets and produce false-positive hits.</p>
<section class="tw xbox font-sm" id="tbl5"><h3 class="obj_head">5. Summary of Previous Studies Reporting
the Production of Dengue Virus-like Particles (VLPs)<sup></sup>
.</h3>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">s. no.</th>
<th align="center" colspan="1" rowspan="1">host expression
system</th>
<th align="center" colspan="1" rowspan="1">VLP format</th>
<th align="center" colspan="1" rowspan="1">DENV serotype</th>
<th align="center" colspan="1" rowspan="1">protein
composition</th>
<th align="center" colspan="1" rowspan="1">VLP size</th>
<th align="center" colspan="1" rowspan="1">references</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">mammalian cells</td>
<td align="left" colspan="1" rowspan="1">lipid-based</td>
<td align="left" colspan="1" rowspan="1">DENV-1–4</td>
<td align="left" colspan="1" rowspan="1">prM, E (FLAG-tagged)</td>
<td align="left" colspan="1" rowspan="1">30–50 nm</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">mammalian cells</td>
<td align="left" colspan="1" rowspan="1">lipid-based</td>
<td align="left" colspan="1" rowspan="1">DENV-2</td>
<td align="left" colspan="1" rowspan="1">prM, E (engineered)</td>
<td align="left" colspan="1" rowspan="1">28–31 nm</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">mammalian
cells</td>
<td align="left" colspan="1" rowspan="1">lipid-based</td>
<td align="left" colspan="1" rowspan="1">Tetravalent</td>
<td align="left" colspan="1" rowspan="1">prM, E and a Japanese encephalitis virus (JEV) signal
peptide</td>
<td align="left" colspan="1" rowspan="1">45–55 nm</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">insect cells</td>
<td align="left" colspan="1" rowspan="1">lipid-based</td>
<td align="left" colspan="1" rowspan="1">DENV-2</td>
<td align="left" colspan="1" rowspan="1">defensin A signal peptide, prM and E, and a JEV-derived stem-anchor
protein</td>
<td align="left" colspan="1" rowspan="1">20–38 nm</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">silkworm larvae (insect)
and Bm5 cells (insect cells)</td>
<td align="left" colspan="1" rowspan="1">lipid-based</td>
<td align="left" colspan="1" rowspan="1">DENV-3</td>
<td align="left" colspan="1" rowspan="1">C, prM and E
(PA tagged)</td>
<td align="left" colspan="1" rowspan="1">30–55 nm</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">silkworm larvae (insect)</td>
<td align="left" colspan="1" rowspan="1">lipid-based</td>
<td align="left" colspan="1" rowspan="1">DENV-2</td>
<td align="left" colspan="1" rowspan="1">C, prM and E</td>
<td align="left" colspan="1" rowspan="1">30–35 nm</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">lettuce
chloroplast</td>
<td align="left" colspan="1" rowspan="1">protein-based</td>
<td align="left" colspan="1" rowspan="1">DENV-3</td>
<td align="left" colspan="1" rowspan="1">prM, F. and partial C</td>
<td align="left" colspan="1" rowspan="1">15–20 nm</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">protein-based</td>
<td align="left" colspan="1" rowspan="1">DENV-1–1</td>
<td align="left" colspan="1" rowspan="1">E ectodomains (tetravalent)</td>
<td align="left" colspan="1" rowspan="1">30–55 nm</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">protein-based</td>
<td align="left" colspan="1" rowspan="1">DENV-1</td>
<td align="left" colspan="1" rowspan="1">E ectodomain + prM signal peptide</td>
<td align="left" colspan="1" rowspan="1">35–38 nm</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">protein-based</td>
<td align="left" colspan="1" rowspan="1">DENV-1–4</td>
<td align="left" colspan="1" rowspan="1">prM and E</td>
<td align="left" colspan="1" rowspan="1">25–30 nm</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">protein-based</td>
<td align="left" colspan="1" rowspan="1">DENV-2</td>
<td align="left" colspan="1" rowspan="1">E ectodomain + prM signal peptide</td>
<td align="left" colspan="1" rowspan="1">30–50 nm</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">protein-based</td>
<td align="left" colspan="1" rowspan="1">DENV-2</td>
<td align="left" colspan="1" rowspan="1">C, prM and E</td>
<td align="left" colspan="1" rowspan="1">not specified</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">13</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">protein-based</td>
<td align="left" colspan="1" rowspan="1">DENV-1</td>
<td align="left" colspan="1" rowspan="1">E ectodomain</td>
<td align="left" colspan="1" rowspan="1">20–100 nm</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">14</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">protein-based</td>
<td align="left" colspan="1" rowspan="1">DENV-2</td>
<td align="left" colspan="1" rowspan="1">DENV-2 EDIII-HBcAg
(Hepatitis B virus core antigen) fusion
protein</td>
<td align="left" colspan="1" rowspan="1">35–40 nm</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t5fn1">
<sup>a</sup><p class="display-inline">The table details the expression
format, DENV serotype, protein composition, and size of various VLPs,
produced using various expression hosts.</p>
</div></div></section><p>Additionally, the higher cost of production and relatively
high
instability of lipid-based VLPs are factors of concern. The <em>in vitro</em> screening assays, being high-throughput, demand
a continuous supply of VLPs, and therefore, their cost of production
has a direct impact on the overall cost of the drug discovery process.
Herein, an E protein VLP platform, produced using an inexpensive expression
system such as , offers an easy-to-handle
and scalable production process, which is more suitable and desirable.
Earlier attempts to use and hosts for E protein expression reported
significantly lower purification yields.<em>
−

</em> A later study
demonstrated the production of dengue virus serotype 1 (DENV-1) E
protein in , achieving a postrefolding
yield of approximately 15–20 mg/L. However, the tendency of E proteins to form VLPs was not explored.
In an attempt to generate protein-only VLPs of DENV-2, domain III
of the E protein was used and expressed in fusion with HBcAg to produce chimeric VLPs. A recent study demonstrated
the generation of E protein-based particles of DENV-1 in . The study, however, revealed the formation
of virus-sized particulate assemblies, which reportedly lack the characteristic
morphology of VLPs.
</p>
<p>In the current
study, the DENV-2 E protein expression in , and the accumulation of recombinant protein
in the form of IBs turned out to be beneficial in terms of higher
protein yields, resulting in high-level recovery of functional protein
post-<em>in vitro</em> refolding (78%). Oligomerization of
E protein subunits into VLPs, with a stoichiometry as high as 20,
was achieved through refolding of the denatured E protein using a
protocol described earlier.<em>
,
</em> The E protein dimers,
formed through the association of domain II of the two subunits, further
interact closely with other E protein dimers on the surface of the
viral envelope.<em>
,
</em> Therefore, the use of the full-length
ectodomain of E protein, which tends to be associated with other subunits
within the viral envelope, and the process used for refolding may
have facilitated VLP formation. To comprehensively characterize the
refolded protein VLPs, various biophysical techniques were employed,
including SDS-PAGE, size exclusion chromatography (SEC), Fourier transform
infrared (FTIR) spectroscopy, circular dichroism (CD), and dynamic
light scattering (DLS). SDS-PAGE (reducing/nonreducing) predicted
the oligomerization of E protein within the refolded sample, which
was then subjected to DLS analysis, which further validated the VLP
formation by specifying the size (∼500 nm diameter) of the
particles. Simultaneously, SEC analysis provided supporting data for
the presence of differently sized oligomeric forms in the refolded
sample, wherein the data showed one major peak corresponding to the
large-sized oligomer and three distinct smaller peaks corresponding
to lower assemblies, including the monomeric form and impurities.
SEC also provided the purified VLPs that were further subjected to
structural analyses. The FTIR and CD spectroscopic studies, conducted
with purified VLPs, demonstrated the presence of anticipated secondary-structure
elements within the refolded proteins, supporting the proper refolding
and attainment of native structural organization by the E protein
subunits. These VLPs, if used for high-throughput screening of host-entry
inhibitor molecules during a preliminary analysis, may provide vital
information predicting their performance in an <em>in vitro</em> setup and help select suitable candidates for follow-up studies.
The E protein VLPs may also be developed as an effective vaccine,
as the particle-like nature of antigen molecules is known to stimulate
the host immune system at a broader level, leading to a strong anti-antigen
humoral as well as cellular immune response.<em>
,
</em>
</p>
<p>Additionally, the study also explored new natural compounds
as
potential antidengue candidates for managing dengue infections. The
focus was primarily on the analogues of CA, which have previously
been shown to inhibit DENV-2 entry in Vero cells. Since the study was designed to be conducted with the CA
analogues, the crystal structure (PDB ID– 1OAN), used earlier for CA docking studies, was employed
for the current studies. In addition to the crystal structure, a cryo-EM
structure of a mature DENV-2 virion (PDB ID – 3J27) has also
been solved at 3.5-Å resolution. Later, Wen-Fan Shen et al., reconstructed the atomic model of a
mature DENV-2 virus-like particle (mD2VLP) by fitting the atomic coordinates
of E and M proteins of 3J27 in the cryo-EM density map of mD2VLP,
representing the VLP-specific arrangement of E and M proteins (mD2VLP_3J27). In general, the docking suites perform a reliable
assessment of molecular interactions with crystal structures of high
resolution, as they are more complete structures, having better resolved
side chains and binding sites, and may underperform with structures
of low resolution. Therefore, the primary docking studies were conducted
with the crystal structure (1OAN) resolved at a higher resolution
(2.75 Å) than the cryo-EM structures solved at lower (3.5 Å)
resolution. The dynamism of surface E and M proteins as a function
of pH and temperature has been demonstrated earlier,<em>
,
</em> suggesting differential spatial arrangements of the E subunits in
a crystal lattice, a protein virus-like particle, a live virion, and
a lipid bilayer-based virus-like particle. Therefore, although solved at lower resolution, the cryo-EM structures
of DENV-2 virion and mD2VLP offered an opportunity to test the binding
activity of the test compounds with these structures <em>in silico</em> and compare the results with those produced by using a crystal structure.
Therefore, the docking studies were repeated with the cryo-EM structures,
which produced equivalent binding scores in the majority of the cases,
except in the case of EGCG, which showed high docking scores with
E dimers of the mD2VLP_3J27 structure. In comparison to the entire
set of compounds tested in these docking experiments, PGG again demonstrated
the most consistent, lowest docking and MM-GBSA binding free energy
scores, validating its selection as the most stable and high-affinity
binder of E protein.</p>
<p>To conduct the docking studies with CA
analogues (gallic acid derivatives
of glucose), two out of three, previously defined CA binding sites
on E protein, where maximum binding scores were achieved, were chosen
for docking experiments and named as grid 1 and grid 2 (<a href="#fig3" class="usa-link">Figure 
</a>). The grid 1 encompassed E
protein regions carrying the K1 β-hairpin loop, involved in
the fusion process; receptor DC-SIGN
binding site (partial), and residues
of the fusion peptide. This site played a vital role in virus-host
membrane fusion, which is common to all serotypes, and therefore,
this site was a prime site for drug screening. On the other hand,
grid 2 harbored binding sites for host receptors like GAG and mannose
receptors.<em>
,
</em> Additionally, grid 2 contains
two residues, K295, which is involved in host receptor binding, and
a glycosyl moiety, Fuc1399, attached to residue 153; both residues
are conserved in all four serotypes of DENV. Therefore, E protein regions selected in grids 1 and 2, as discussed
above, are suggested to mediate host attachment and entry activities,
and therefore, if these sites are bound by drug molecules with high
affinity, individually or simultaneously, they may inhibit viral infection.
The <em>in silico</em> docking studies indicated the same,
that the gallic acid derivatives may serve as DENV-2 entry inhibitors,
mediating their effects through binding to the E protein at grids
1 and 2. Among the five test molecules, PGG generated the highest
binding scores at grid 1, whereas moderate yet significant binding
scores were observed at grid 2. Additionally, PGG demonstrated a comparatively
balanced stability of the protein–ligand complex during molecular
dynamics simulation studies. Herein, 100 and 500 ns MD simulation
data of PGG demonstrated a pattern equivalent to CA. The other small
compound used as a positive control, EGCG, which otherwise has significant
structural disparity with PGG and CA, showed equivalent and rather
lower docking scores. During MD simulation studies, the lower protein
RMSD indicated a stable conformation; however, the ligand RMSD was
consistently in an unstable range throughout. To further validate
the stability and biological significance of the PGG-E protein interaction,
the surface electrostatic complementarity of the docked complexes
of PGG, CA, and EGCG with the E dimer was analyzed through electrostatic
surface potential (ESP) maps. Herein, all three complexes showed similar
scores, with PGG and CA performing better than EGCG, further validating
the potential of PGG as a possible anti-DENV-2 inhibitor. Therefore,
PGG was selected for the subsequent <em>in vitro</em> binding
assay, where it exhibited high-affinity kinetics for binding E protein
VLPs.</p>
<p>PGG, a naturally occurring, hydrolyzable plant tannin,
which is
associated with a wide range of biological activities such as antiviral,
anticancer, and anti-inflammatory properties, makes it a promising therapeutic candidate. Recently, PGG has been
demonstrated to block SARS-CoV-2′s papain-like protease (PLpro),
essential for viral replication, highlighting its antiviral properties,
particularly against viruses like SARS-CoV-2. Studies have shown a favorable safety profile of PGG, notably in
mice, with manageable dosages.<em>
,
</em> However, PGG has also
exhibited cytotoxicity in several cell lines, necessitating further
investigations to optimize its therapeutic window. Additionally, PGG has been studied in animal models, including
mice and prairie dogs, which demonstrate good tolerance profiles.
These trials suggest that with more focused research, PGG could be
repurposed as a medication for treating viral infections, particularly
in combination with other drugs. To assess
the drug-likeness of the test compounds, ADMET analysis was conducted,
which did not predict favorable values for most of the parameters,
like CNS activity, molecular weight, volume, molecular interactions,
solubility, permeability, potential metabolic reactions, binding affinity
to human serum albumin, oral absorption, etc. The only parameters
working in favor of the molecules were hydrophobic and polar surface
areas, solvent accessibility, solubility, etc., indicating mostly
the extracellular confinement of the compounds. However, since the
molecules are required to bind the free viral particles present in
the extracellular regions, the conventional ADMET parameters may not
directly correlate with the drug-likeness of the test compounds. The
adverse data obtained for most of the parameters in the ADMET analysis
are desired for optimal activity of the compound(s). Using <em>in silico</em> molecular docking, molecular dynamics simulations,
and <em>in vitro</em> experiments, it was demonstrated that
PGG strongly binds to the DENV-2 E protein and may potentially block
the host entry. The <em>in silico</em> binding predictions
were validated by biolayer interferometry (BLI) based protein–ligand-binding
analysis, revealing PGG’s high affinity for the E protein VLPs,
as depicted in its low K<sub>D</sub> values, calculated for two independent
sites. The observed <em>K</em>
<sub>D</sub> values may correspond
to the two sites described and used in the <em>in silico</em> docking experiments. Nonetheless, they do confirm high-affinity,
multiple-site binding of E protein by the test compounds, supporting
their possible inhibitory function. While PGG demonstrated promising
results, other analogues, including monogalloylglucose, digalloylglucose,
trigalloylglucose, and tetragalloylglucose, also exhibited <em>in silico</em> binding with significant affinities. To confirm
the effectiveness and safety of PGG and other analogues, additional
preclinical and clinical assessments are required in the future. Research
into new antiviral treatments remains essential to mitigate the impact
of this debilitating illness, as dengue continues to be a serious
health concern globally.</p></section><section id="sec4"><h2 class="pmc_sec_title">4. Conclusions</h2>
<p>The
successful expression, purification, and refolding of DENV-2
envelope protein was achieved in , yielding high-quality, well-characterized VLPs for subsequent protein–ligand
binding assays. The <em>in silico</em> docking and MD simulation
studies demonstrated strong binding affinities of galloylglucose molecules
(mono, di, tri, tetra, and pentagalloylglucose) for the DENV-2 envelope
protein. These findings not only provide valuable insights into identifying
viable dengue antiviral candidates but also highlight the influence
of the nature and the number of galloyl groups on their possible antiviral
activity. Among the five tested galloylglucose molecules, pentagalloylglucose
(PGG) emerged as the most promising candidate, demonstrating lower
binding scores in molecular docking studies and comparatively stable
interactions within the E protein-PGG complex as revealed in MD simulation
studies performed at 100 and 500 ns. In addition to that, PGG showed
strong and moderate electrostatic complementarity for E protein sites
at grids 1 and 2, respectively. The estimated EC scores of PGG matched
those of CA and were slightly better than epigallocatechin-3-gallate
(EGCG). Based on these results, PGG was selected for further <em>in vitro</em> protein-binding studies. The <em>in vitro</em> binding assay, performed using biolayer interferometry (BLI), demonstrated
strong binding of E protein VLPs by PGG, further supporting its potential
as a host-entry inhibitor. Furthermore, PGG has reportedly demonstrated
diverse pharmacological properties, including antiviral, antibacterial,
anti-inflammatory, anticarcinogenic, antidiabetic, and antioxidant
activities. PGG has shown good tolerance profiles in mice and prairie
dogs, further supporting its use as a repurposed medication for the
treatment of viral infections. In addition to PGG, trigalloylglucose
also emerges as a promising alternative. Its smaller size than that
of PGG and fewer galloyl groups may impact antiviral efficacy while
offering advantages such as improved solubility, reduced toxicity,
and enhanced pharmacokinetics. However, further experimental and clinical
investigations are necessary to fully characterize and validate the
antiviral potential of trigalloylglucose.</p></section><section id="sec5"><h2 class="pmc_sec_title">5. Materials
and Methods</h2>
<section id="sec5.1"><h3 class="pmc_sec_title">5.1. Protein and Gene Sequences for <em>In
Silico</em> and <em>In Vitro</em> Experiments</h3>
<p>The protein PDB ID 1OAN was used for the retrieval of the protein
sequence of the Envelope protein of Dengue virus serotype 2. The selected
amino acid sequence was reverse-translated into a coding gene sequence
with codon bias using an online
tool (<a href="https://www.bioinformatics.org/sms2/rev_trans.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.bioinformatics.org/sms2/rev_trans.html</a>). The gene
DNA was commercially synthesized and cloned (Biotech Desk Pvt Ltd.)
in plasmid pET22b between <em>Nde</em>I and <em>Xho</em>I restriction sites, producing a recombinant plasmid pET22b-E-DENV-2.</p></section><section id="sec5.2"><h3 class="pmc_sec_title">5.2. Plasmid and Bacterial Strains</h3>
<p>The
plasmid pET22b­(+) vector (Novagen), was used for cloning and expression
of the vaccine candidate proteins in . The vector and recombinant plasmid were propagated in the DH5α
strain of (Invitrogen), whereas
the recombinant proteins were expressed in the BL21 (DE3) strain of (Invitrogen).</p></section><section id="sec5.3"><h3 class="pmc_sec_title">5.3. Protein
Expression and Purification</h3>
<p>The recombinant envelope protein
of DENV-2 was produced in BL21
(DE3) cells. In brief, the competent BL21 (DE3) cells were transformed with recombinant
plasmid pET22b-E-DENV-2. The transformed single colony was used to
inoculate 5 mL of LB medium supplemented with 100 μg/mL ampicillin,
and the cells were grown overnight at 37 °C with continuous shaking
at 150 rpm. The next day, 400 mL terrific broth (TB) medium, supplemented
with ampicillin, was inoculated with 1% overnight-grown culture of cells and allowed to grow under the same
conditions as for the primary culture. At an optical density (OD)
of 0.8, the culture was induced by adding 0.8 mM IPTG and incubated
further for 5 h with regular shaking at 200 rpm. After 5 h, the cells
were harvested by centrifugation, and the cell pellet was resuspended
in lysis buffer (50 mM Tris-Cl, 5 mM EDTA, pH 8.0) supplemented with
1 mM PMSF. The cells were lysed by sonication using an ultrasonic
processor (Cole-Parmer) through 2 cycles of 5 min each with a program
of 20 s ON/20 s OFF pulse at 50% amplitude. The cell lysate was centrifuged
at 9500 rpm for 30 min. The resulting pellet containing inclusion
bodies was washed twice with washing buffer (1% sodium deoxycholate,
2% Triton X-100, 50 mM Tris-Cl, and 5 mM EDTA) and finally washed
with MiliQ water. The insoluble IB pellet was resuspended in solubilizing
buffer A (50 mM Tris-Cl, 8 M Urea, pH 7.4) and centrifuged at 9500
rpm for 30 min. The supernatant was collected and supplemented with
20 mM imidazole to prevent nonspecific binding, then loaded onto a
Ni-NTA agarose column pre-equilibrated with the same solubilizing
buffer. The unbound protein fraction was collected, and the column
was washed three times with washing buffer (50 mM Tris-Cl, 8 M Urea,
40 mM Imidazole, 300 mM NaCl, pH 7.4) to remove nonspecifically bound
proteins. Finally, the target protein was eluted using an elution
buffer (50 mM Tris-Cl, 8 M Urea, 100 mM Imidazole, pH 7.4). The eluted
protein fractions were analyzed by 12% SDS-PAGE.</p></section><section id="sec5.4"><h3 class="pmc_sec_title">5.4. 
<em>In Vitro</em> Protein Refolding</h3>
<p>The denatured
recombinant protein was <em>in vitro</em> refolded, following
a modified version of the protocol described
earlier. Briefly, all eluted protein
fractions were pooled and diluted to a final concentration of 0.4
mg/mL, followed by reduction with 10 mM dithiothreitol (DTT) at room
temperature for 1 h. The reduced protein was then subjected to sequential
dialysis against denaturing buffer (50 mM Tris-Cl, 50 mM NaCl, 8 M
urea, 1 mM EDTA, pH 8.0) under continuous stirring at 4 °C for
6–8 h. To facilitate stepwise renaturation, the urea concentration
was systematically reduced (8 → 6 → 4 → 2 M)
via successive dialysis steps. Subsequently, the protein solution
was transferred to an oxidizing refolding buffer (50 mM Tris-Cl, 50
mM NaCl, 1 M urea, 375 μM oxidized glutathione, and 400 mM <span class="font-variant-small-caps">l</span>-arginine, pH 8.0) to promote proper oxidative folding. The
urea concentration was progressively decreased (1 → 0.5 →
0.25 → 0 M), keeping the other components of the buffer the
same, followed by the gradual removal of oxidizing agents in four
additional dialysis steps. In the final step, the refolded protein
was dialyzed using a buffer (50 mM Tris-Cl and 50 mM NaCl, pH 8.0).
Each dialysis step was performed under constant slow stirring for
6–8 h to ensure efficient protein refolding. The monomeric
and oligomeric states of the refolded E protein were assessed via
SDS-PAGE under reducing and nonreducing conditions, respectively.
The refolding percentage of the protein was calculated using the following
formula</p>
<table class="disp-formula p" id="ueq1"><tr><td class="formula"><math id="d1e2444" display="block"><mi mathvariant="normal">r</mi><mi>efolding percentage</mi><mo>=</mo><mfrac><mrow><mi mathvariant="normal">t</mi><mi>otal refolded protein</mi><mrow><mo stretchy="false">(</mo><mi>mg</mi><mo stretchy="false">)</mo></mrow></mrow><mrow><mi mathvariant="normal">i</mi><mi>nitial protein used for refolding</mi><mrow><mo stretchy="false">(</mo><mi>mg</mi><mo stretchy="false">)</mo></mrow></mrow></mfrac><mo>×</mo><mn>100</mn></math></td></tr></table></section><section id="sec5.5"><h3 class="pmc_sec_title">5.5. Western
Blotting</h3>
<p>To confirm the identity
of the purified refolded recombinant protein, Western blotting was
performed by using the anti-DENV E protein antibody directed against
domain III. The refolded protein sample was resolved on a 12% acrylamide
gel by SDS-PAGE and transferred onto the nitrocellulose membrane.
The membrane was blocked with 5% BSA prepared in Tris-Buffer saline
(TBS) for 1 h at room temperature. The membrane was then washed with
TBS containing 0.05% Tween-20 (TBST). Afterward, the membrane was
treated with mouse anti-DENV EDIII antibody (Invitrogen MA5–30000)
at a dilution of 1:5000 (Ab:TBST) and incubated overnight at 4 °C
with mild rocking. The next day, the membrane was removed and washed
thrice with TBST. The membrane was further incubated with a goat antimouse
HRP conjugated antibody (Thermo-Scientific 31430) at a dilution of
1:10000 for 1 h, followed by three washes of TBST. The blot was developed
by ECL (Amersham) HRP substrate solution on a blot scanner (LI-COR)
and analyzed by Image Studio Digits Ver 4.0.</p></section><section id="sec5.6"><h3 class="pmc_sec_title">5.6. Size
Exclusion Chromatography</h3>
<p>The <em>in vitro</em> refolded
E protein was self-assembled into oligomeric
forms, which were further purified based on their molecular weights
using size exclusion chromatography (SEC). The purification was performed
on an KTA Pure system (GE Healthcare) utilizing a 16/600 Superdex
200 pg column with a total column volume of 120 mL and a flow rate
of 0.8–1.0 mL/min. Protein fractions were continuously monitored
by ultraviolet (UV) detection at 280 nm, enabling their collection
in multiple fractions. Data acquisition and processing were carried
out using UNICORN 7.0 software, while SEC chromatographic profiles
were plotted using GraphPad Prism 8.0.1.</p></section><section id="sec5.7"><h3 class="pmc_sec_title">5.7. Characterization
of <em>In Vitro</em> Refolded Protein Using FTIR, CD, and
DLS</h3>
<p>Fourier transform
infrared (FTIR) spectroscopy, circular dichroism (CD) spectroscopy,
and dynamic light scattering (DLS) analyses were performed to characterize
the refolded protein. FTIR spectroscopy was used to identify functional
groups in the refolded protein sample, ensuring proper folding. The
protein sample in the buffer was mixed with KBr solution, and the
spectrum was recorded using an FTIR Spectrum-2 (PerkinElmer) within
the infrared range of 4000–400 cm<sup>–1</sup>. CD spectroscopy
was conducted to further analyze the secondary-structure elements
of the refolded E protein. The measurements were taken for a sample
at a protein concentration of 0.1 mg/mL in the UV range 180–300
nm, using a JASCO J-815 CD spectrometer. Spectral data was processed
with JASCO spectra manager software as per Yang’s reference.
Thereafter, DLS was employed to determine the particle size, polydispersity
index (PDI), and size distribution of the refolded protein samples.
DLS measurements, based on the diffraction characteristics of monochromatic
or laser light scattered through the sample, analyzed scattered light
intensities and wavelengths to calculate the molecular size and shape
within the range 10–1000 nm using a Malvern Zetasizer Nano
system. Data collected for all three techniques were plotted by using
GraphPad Prism 8.0.1.</p></section><section id="sec5.8"><h3 class="pmc_sec_title">5.8. 
<em>In Silico</em> Studies</h3>
<section id="sec5.8.1"><h4 class="pmc_sec_title">5.8.1. Protein Preparation, Ligand Preparation,
and Grid Selection</h4>
<p>The two of the structures of envelope
glycoprotein (E) of DENV-2 (PDB ID: <a href="https://www.rcsb.org/structure/1OAN" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1OAN</a> and 3J27), were retrieved from the RCSB
Protein Data Bank (<a href="https://www.rcsb.org/structure/1oan" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rcsb.org/structure/1oan</a>), whereas, the atomic model
(PDB: mD2VLP_3J27) was provided by Dr. Shang-Rung Wu, Associate Professor,
School of Dentistry &amp; Institute of Oral Medicine, National Cheng
Kung University, Taiwan. All three structures
of E dimer (1OAN, 3J27, and mD2VLP_3J27) were prepared for molecular
docking using the <em>Protein Preparation Wizard</em> in Maestro
Schrödinger. The preparation process
included the optimization of missing side chains and energy minimization
to ensure structural stability. Ligand structures corresponding to
seven compounds [PubChem IDs: 65238 (Pentagalloylglucose), 14464350
(Tetragalloylglucose), 452707 (Trigalloylglucose), 440221 (Digalloylglucose),
4628122 (monogalloylglucose), 72284 (Chebulinic acid) and 65064 (Epigallocatechin-3-gallate)],
were obtained from the PubChem database and converted into 3D structures
using the <em>LigPrep</em> module in Maestro Schrödinger.
<em>LigPrep</em> was employed to refine
protonation and ionization states, assign proper bond orders, and
generate relevant tautomeric and ionization states for each ligand.
Based on prior <em>in silico</em> studies on the DENV-2 E
protein binding with chebulinic acid, two potential binding sites
were selected as key targets for inhibition. A grid box was generated at these sites using the receptor grid
generation tool in <em>Glide</em> (Maestro Schrödinger),
with an illustration of the grid positions provided in <a href="#fig2" class="usa-link">Figure 
</a>.</p></section><section id="sec5.8.2"><h4 class="pmc_sec_title">5.8.2. Molecular
Docking</h4>
<p>The <em>Glide</em> module from the Schrodinger
2019–4 suite Maestro (<a href="https://www.schrodinger.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.schrodinger.com</a>) was used for molecular docking
experiments. During the process of docking,
the protein was fixed, while
the ligands were flexible. The molecular mechanics-generalized born
surface area (MM-GBSA) method in <em>Prime</em> was used for
rescoring the docked pose of the ligand.
</p></section><section id="sec5.8.3"><h4 class="pmc_sec_title">5.8.3. Absorption, Distribution, Metabolism, Excretion
(ADMET), and Toxicity Prediction/Qik Properties</h4>
<p>ADMET (absorption,
distribution, metabolism, excretion, and toxicity) properties were
predicted using <em>QikProp</em>. ADMET study is essential for determining a compound’s potential
efficacy as a drug since it assesses a compound’s physicochemical
characteristics, biological activities, and drug-likeness.</p></section><section id="sec5.8.4"><h4 class="pmc_sec_title">5.8.4. Molecular Dynamics (MD) Simulation Studies</h4>
<p>Molecular
dynamics (MD) simulations were performed to evaluate
the stability and reliability of protein–ligand interactions
by observing atomic-level movements over time, providing insights
into protein conformational changes, folding behavior, and ligand-binding
dynamics. Based on docking scores, initially
100 ns MD simulations were carried out using Desmond for six selected complexes (1OAN-14464350, 1OAN-4628122,
1OAN-440221, 1OAN-452707, 1OAN-72284, and 1OAN-65238) at both grid
positions to assess early conformational stability. Following these
results, extended simulations were conducted for the same six complexes,
along with epigallocatechin gallate (EGCG), exclusively at grid position
2 due to its superior stability observed in the 100 ns runs. A 200
ns simulation was performed for five complexes (1OAN-14464350, 1OAN-4628122,
1OAN-440221, 1OAN-452707, and 1OAN-72284), while 500 ns simulations
were carried out for two complexes (1OAN-65238 and 1OAN-65064). All
systems were solvated using the TIP3P water model within an orthorhombic
simulation box with a 10 Å buffer on all sides. A physiological
ionic strength of 0.15 M NaCl was maintained, and additional counterions
were added to neutralize the system’s net charge. Energy minimization
was conducted to relieve steric clashes, followed by equilibration
steps with positional restraints. MD simulations were executed under
the NPT ensemble at a constant temperature of 300 K using the Nose
Hoover chain thermostat and a pressure of 1.01325 bar maintained by
the Langevin barostat. A time step of 2 fs and a nonbonded interaction
cutoff of 9 Å were applied, following Schrödinger Maestro’s
default settings. The OPLS_2005 force field was employed throughout,
with energy values recorded every 1.2 ps during the simulation run.</p>
<section id="sec5.8.4.1"><h5 class="pmc_sec_title">5.8.4.1. Electrostatic Surface Potential (ESP)
Analysis</h5>
<p>In addition to docking studies, electrostatic surface
potential (ESP) maps were generated to visualize the charge distribution
across the protein–ligand-binding interfaces and to evaluate
electrostatic compatibility. ESP maps were prepared in Schrodinger
Maestro 14.2. using the electrostatic potential color scheme. Molecular
surfaces were created using the surface tool available in the style
toolbox. The generated surfaces were then managed through the manage
surfaces panel, and display settings were adjusted by selecting display
options, where the color scheme was set to property, followed by selecting
electrostatic potential. This approach utilizes the partial atomic
charges to compute and visualize the electrostatic potential distribution
on the molecular surface, facilitating the interpretation of electrostatic
complementarity between the ligands and the binding site. For quantitative
evaluation, electrostatic complementarity (EC) scores were calculated
using BioLuminate (Schrodinger suite). The analysis was conducted for three protein–ligand complexes
(1OAN-65238, 1OAN-72284, and 1OAN-65064) at both grid positions.</p></section></section></section><section id="sec5.9"><h3 class="pmc_sec_title">5.9. 
<em>In Vitro</em> DENV-2 E Protein–Ligand-Binding
Analysis Using Biolayer Interferometry (BLI)</h3>
<p>Biolayer Interferometry
is one of the most widely used techniques to analyze biomolecular
interactions for assessing kinetic parameters. It is utilized to analyze the binding affinity of molecules
under <em>in vitro</em> conditions. The knowledge regarding
molecular interactions is the primary part of the field of drug discovery,
and <em>in vitro</em> binding studies play a crucial role
in predicting the candidate drug’s pharmacological activity. This analysis detects the pattern of interference
of the white light, which gets reflected from the layer of protein
that has been immobilized on the biosensor’s tip. This pattern
is then compared with the pattern of the internal reference layer.
With this data, valuable and real-time information on the amount of
bound protein on the biosensor tip could be obtained.
</p>
<p>In this study, this technique has been utilized to
study the biomolecular interaction between the DENV-2 E protein VLPs
and compound Penta-<em>O</em>-galloyl-β-<span class="font-variant-small-caps">d</span>-glucose
(PGG). Herein, the PGG hydrate (Sigma-Aldrich) samples were prepared
at 4 different concentrations (6.25, 12.5, 25, and 50 μM) in
buffer (50 mM Tris, 50 mM NaCl, pH 7.4). After the PGG solutions were
prepared, the purified DENV-2 E protein VLPs were immobilized on a
Ni-NTA-coated biosensor (Octet, Sartorius). Following this, the protein-coated
biosensors were washed with Tris-NaCl buffer containing 0.1% DMSO,
and the baseline was set for 60 s. Afterward, the tips of the biosensors
were immersed completely in the wells of the black ELISA plate containing
various drug concentrations as mentioned above, allowing ligand association
for 600 s. Right after the completion of this step, the dissociation
step was followed for another 600 s, after which the <em>K</em>
<sub>D</sub> values were calculated by using the Data Analysis Software,
considering a 1:2 binding model with <em>k</em>
<sub>dis</sub> as the dissociation constant (s<sup>–1</sup>) and <em>k</em>
<sub>a</sub> as the association constant (1/Ms). <em>K</em>
<sub>D</sub> is calculated by using the <em>k</em>
<sub>dis</sub>/<em>k</em>
<sub>a</sub> formula. The experiment
was also performed with another flavonoid, EGCG (Sigma-Aldrich), as
a control, at 4 different concentrations (12.5, 25, 50, and 100 μM),
under a similar experimental setup as used in the case of the PGG
binding assay. The EGCG, being a reported antidengue inhibitor, was
used as a positive control in the current study.
</p></section></section><section id="sec26"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12332551/bin/ao5c02548_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c02548_si_001.pdf</a><sup> (31.7MB, pdf) </sup>
</div></div></section></section><section id="ACK-d14e1897-autogenerated" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors
would like to acknowledge the Department of Biotechnology,
School of Life Sciences, Central University of Rajasthan, NH-8, Bandarsindri,
Ajmer, Rajasthan, 305817, India, for providing the necessary lab infrastructure
and other necessary resources. The authors would like to thank the
Department of Biophysics, University of Delhi South Campus, Benito
Juarez Road, New Delhi-110021, for providing CD spectroscopy and Sanger
sequencing instrumentation facilities. The authors would also like
to acknowledge the Malaviya National Institute of Technology, Jaipur,
Jawahar Lal Nehru Marg, Jaipur 302017, (Rajasthan), India, for the
FTIR and DLS instrumentation facility. The authors appreciate and
thank Dr. Shang-Rung Wu, Associate Professor, School of Dentistry
&amp; Institute of Oral Medicine, National Cheng Kung University,
Taiwan, for providing the cryo-EM structure file: mD2VLP_3J27. The
authors gratefully acknowledge the funding agencies, namely, ANRF
(earlier SERB), India (SPG/2021/004521), and ICMR, India (IIRP-2023-6434/F1),
for financial support. Anjana Sharma is supported by the Department
of Biotechnology (DBT) JRF fellowship (DBT/2020/CUR/1354), funded
by DBT, Govt. of India.</p></section><section id="notes-1"><p>The Supporting Information
is available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsomega.5c02548?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsomega.5c02548</a>.</p>
<ul id="silist" class="list" style="list-style-type:none"><li><p>Description of the recombinant
plasmid pET22b-E-DENV-2,
analysis of cloning; molecular docking data analysis; molecular dynamics
simulation analysis of E protein complexes of galloylglucose molecules,
chebulinic acid, and epigallocatechin gallate (EGCG); root-mean-square
deviation (RMSD) plots, root-mean-square fluctuation (RMSF) plots,
hydrogen bond graphs, radius of gyration (Rg) plots; ADMET profiling
of the mono, di, tri, tetra, and pentagalloylglucose and EGCG (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02548/suppl_file/ao5c02548_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes-3"><p>A.S., S.T.,
and S.N. contributed to manuscript writing and editing. The study
was conceptualized and experimentally designed by S.T. and A.S. <em>In silico</em> and <em>in vitro</em> experiments were
performed by A.S. and evaluated by A.S. and S.T. The manuscript editing
was performed by S.N.</p></section><section id="notes-2"><p>The authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>

Schaefer, T. J.
; 
Panda, P. K.
; 
Wolford, R. W.
. 
Dengue Fever; StatPearls Publishing; 
2025.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/28613483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Dengue%20Fever&amp;publication_year=2025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Martina B. E. E., Koraka P., Osterhaus A. D. M. E.. Dengue
Virus Pathogenesis: An Integrated
View. Clin. Microbiol. Rev. 2009;22(4):564–581. doi: 10.1128/CMR.00035-09.</cite> [<a href="https://doi.org/10.1128/CMR.00035-09" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2772360/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19822889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Microbiol.%20Rev.&amp;title=Dengue%20Virus%20Pathogenesis:%20An%20Integrated%20View&amp;author=B.%20E.%20E.%20Martina&amp;author=P.%20Koraka&amp;author=A.%20D.%20M.%20E.%20Osterhaus&amp;volume=22&amp;issue=4&amp;publication_year=2009&amp;pages=564-581&amp;pmid=19822889&amp;doi=10.1128/CMR.00035-09&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Bhatt P., Sabeena S. P., Varma M., Arunkumar G.. Current Understanding
of the Pathogenesis of Dengue Virus Infection. Curr. Microbiol. 2021;78(1):17–32. doi: 10.1007/s00284-020-02284-w.</cite> [<a href="https://doi.org/10.1007/s00284-020-02284-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7815537/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33231723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Microbiol.&amp;title=Current%20Understanding%20of%20the%20Pathogenesis%20of%20Dengue%20Virus%20Infection&amp;author=P.%20Bhatt&amp;author=S.%20P.%20Sabeena&amp;author=M.%20Varma&amp;author=G.%20Arunkumar&amp;volume=78&amp;issue=1&amp;publication_year=2021&amp;pages=17-32&amp;pmid=33231723&amp;doi=10.1007/s00284-020-02284-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Mondotte J. A., Lozach P.-Y., Amara A., Gamarnik A. V.. Essential
Role of
Dengue Virus Envelope Protein N Glycosylation at Asparagine-67 during
Viral Propagation. J. Virol. 2007;81(13):7136–7148. doi: 10.1128/JVI.00116-07.</cite> [<a href="https://doi.org/10.1128/JVI.00116-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1933273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17459925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Virol.&amp;title=Essential%20Role%20of%20Dengue%20Virus%20Envelope%20Protein%20N%20Glycosylation%20at%20Asparagine-67%20during%20Viral%20Propagation&amp;author=J.%20A.%20Mondotte&amp;author=P.-Y.%20Lozach&amp;author=A.%20Amara&amp;author=A.%20V.%20Gamarnik&amp;volume=81&amp;issue=13&amp;publication_year=2007&amp;pages=7136-7148&amp;pmid=17459925&amp;doi=10.1128/JVI.00116-07&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Kala M. P., St John A. L., Rathore A. P. S.. Dengue:
Update on Clinically Relevant
Therapeutic Strategies and Vaccines. Curr. Treat.
Options Infect. Dis. 2023;15(2):27–52. doi: 10.1007/s40506-023-00263-w.</cite> [<a href="https://doi.org/10.1007/s40506-023-00263-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10111087/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37124673/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Treat.%20Options%20Infect.%20Dis.&amp;title=Dengue:%20Update%20on%20Clinically%20Relevant%20Therapeutic%20Strategies%20and%20Vaccines&amp;author=M.%20P.%20Kala&amp;author=A.%20L.%20St%20John&amp;author=A.%20P.%20S.%20Rathore&amp;volume=15&amp;issue=2&amp;publication_year=2023&amp;pages=27-52&amp;pmid=37124673&amp;doi=10.1007/s40506-023-00263-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Obi J. O., Gutiérrez-Barbosa H., Chua J. V., Deredge D. J.. Current
Trends and Limitations in Dengue Antiviral Research. Trop. Med. Infect. Dis. 2021;6(4):180.  doi: 10.3390/tropicalmed6040180.</cite> [<a href="https://doi.org/10.3390/tropicalmed6040180" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8544673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34698303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trop.%20Med.%20Infect.%20Dis.&amp;title=Current%20Trends%20and%20Limitations%20in%20Dengue%20Antiviral%20Research&amp;author=J.%20O.%20Obi&amp;author=H.%20Guti%C3%A9rrez-Barbosa&amp;author=J.%20V.%20Chua&amp;author=D.%20J.%20Deredge&amp;volume=6&amp;issue=4&amp;publication_year=2021&amp;pages=180&amp;pmid=34698303&amp;doi=10.3390/tropicalmed6040180&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Pitcher T. J., Sarathy V. V., Matsui K., Gromowski G. D., Huang C. Y.-H., Barrett A. D. T.. Functional Analysis
of Dengue Virus
(DENV) Type 2 Envelope Protein Domain 3 Type-Specific and DENV Complex-Reactive
Critical Epitope Residues. J. Gen. Virol. 2015;96(2):288–293. doi: 10.1099/vir.0.070813-0.</cite> [<a href="https://doi.org/10.1099/vir.0.070813-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4298678/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25351518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Gen.%20Virol.&amp;title=Functional%20Analysis%20of%20Dengue%20Virus%20(DENV)%20Type%202%20Envelope%20Protein%20Domain%203%20Type-Specific%20and%20DENV%20Complex-Reactive%20Critical%20Epitope%20Residues&amp;author=T.%20J.%20Pitcher&amp;author=V.%20V.%20Sarathy&amp;author=K.%20Matsui&amp;author=G.%20D.%20Gromowski&amp;author=C.%20Y.-H.%20Huang&amp;volume=96&amp;issue=2&amp;publication_year=2015&amp;pages=288-293&amp;pmid=25351518&amp;doi=10.1099/vir.0.070813-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Hidari K. I. P. J., Suzuki T.. Dengue Virus Receptor. Trop Med.
Health. 2011;39(4):S37–S43. doi: 10.2149/tmh.2011-S03.</cite> [<a href="https://doi.org/10.2149/tmh.2011-S03" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3317600/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22500135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trop%20Med.%20Health&amp;title=Dengue%20Virus%20Receptor&amp;author=K.%20I.%20P.%20J.%20Hidari&amp;author=T.%20Suzuki&amp;volume=39&amp;issue=4&amp;publication_year=2011&amp;pages=S37-S43&amp;pmid=22500135&amp;doi=10.2149/tmh.2011-S03&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Thomas N., Patil P., Sharma A., Kumar S., Singh V. K., Alagarasu K., Parashar D., Tapryal S.. Studies on the Antiviral
Activity of Chebulinic Acid against Dengue and Chikungunya Viruses
and in Silico Investigation of Its Mechanism of Inhibition. Sci. Rep. 2022;12(1):10397.  doi: 10.1038/s41598-022-13923-6.</cite> [<a href="https://doi.org/10.1038/s41598-022-13923-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9213501/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35729191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Studies%20on%20the%20Antiviral%20Activity%20of%20Chebulinic%20Acid%20against%20Dengue%20and%20Chikungunya%20Viruses%20and%20in%20Silico%20Investigation%20of%20Its%20Mechanism%20of%20Inhibition&amp;author=N.%20Thomas&amp;author=P.%20Patil&amp;author=A.%20Sharma&amp;author=S.%20Kumar&amp;author=V.%20K.%20Singh&amp;volume=12&amp;issue=1&amp;publication_year=2022&amp;pages=10397&amp;pmid=35729191&amp;doi=10.1038/s41598-022-13923-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Imagawa T., Ito M., Matsuda M., Nakashima K., Tokunaga Y., Ohta I., Li T.-C., Suzuki R., Suzuki T.. Virus-like Particles
with FLAG-Tagged Envelope Protein as a Tetravalent Dengue Vaccine
Candidate. Sci. Rep. 2021;11(1):17542.  doi: 10.1038/s41598-021-97038-4.</cite> [<a href="https://doi.org/10.1038/s41598-021-97038-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8413300/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34475493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Virus-like%20Particles%20with%20FLAG-Tagged%20Envelope%20Protein%20as%20a%20Tetravalent%20Dengue%20Vaccine%20Candidate&amp;author=T.%20Imagawa&amp;author=M.%20Ito&amp;author=M.%20Matsuda&amp;author=K.%20Nakashima&amp;author=Y.%20Tokunaga&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pages=17542&amp;pmid=34475493&amp;doi=10.1038/s41598-021-97038-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Shen W.-F., Galula J. U., Liu J.-H., Liao M.-Y., Huang C.-H., Wang Y.-C., Wu H.-C., Liang J.-J., Lin Y.-L., Whitney M. T., Chang G.-J. J., Chen S.-R., Wu S.-R., Chao D.-Y.. Epitope Resurfacing
on Dengue Virus-like Particle Vaccine
Preparation to Induce Broad Neutralizing Antibody. eLife. 2018;7:e38970.  doi: 10.7554/eLife.38970.</cite> [<a href="https://doi.org/10.7554/eLife.38970" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6234032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30334522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Epitope%20Resurfacing%20on%20Dengue%20Virus-like%20Particle%20Vaccine%20Preparation%20to%20Induce%20Broad%20Neutralizing%20Antibody&amp;author=W.-F.%20Shen&amp;author=J.%20U.%20Galula&amp;author=J.-H.%20Liu&amp;author=M.-Y.%20Liao&amp;author=C.-H.%20Huang&amp;volume=7&amp;publication_year=2018&amp;pages=e38970&amp;pmid=30334522&amp;doi=10.7554/eLife.38970&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Zhang S., Liang M., Gu W., Li C., Miao F., Wang X., Jin C., Zhang L., Zhang F., Zhang Q., Jiang L., Li M., Li D.. Vaccination with Dengue Virus-like Particles Induces
Humoral and Cellular Immune Responses in Mice. Virol J. 2011;8(1):333.  doi: 10.1186/1743-422X-8-333.</cite> [<a href="https://doi.org/10.1186/1743-422X-8-333" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3144018/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21714940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Virol%20J.&amp;title=Vaccination%20with%20Dengue%20Virus-like%20Particles%20Induces%20Humoral%20and%20Cellular%20Immune%20Responses%20in%20Mice&amp;author=S.%20Zhang&amp;author=M.%20Liang&amp;author=W.%20Gu&amp;author=C.%20Li&amp;author=F.%20Miao&amp;volume=8&amp;issue=1&amp;publication_year=2011&amp;pages=333&amp;pmid=21714940&amp;doi=10.1186/1743-422X-8-333&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Charoensri N., Suphatrakul A., Sriburi R., Yasanga T., Junjhon J., Keelapang P., Utaipat U., Puttikhunt C., Kasinrerk W., Malasit P., Sittisombut N.. An Optimized
Expression Vector for Improving the Yield of Dengue Virus-like Particles
from Transfected Insect Cells. J. Virol Methods. 2014;205:116–123. doi: 10.1016/j.jviromet.2014.04.019.</cite> [<a href="https://doi.org/10.1016/j.jviromet.2014.04.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24814967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Virol%20Methods&amp;title=An%20Optimized%20Expression%20Vector%20for%20Improving%20the%20Yield%20of%20Dengue%20Virus-like%20Particles%20from%20Transfected%20Insect%20Cells&amp;author=N.%20Charoensri&amp;author=A.%20Suphatrakul&amp;author=R.%20Sriburi&amp;author=T.%20Yasanga&amp;author=J.%20Junjhon&amp;volume=205&amp;publication_year=2014&amp;pages=116-123&amp;pmid=24814967&amp;doi=10.1016/j.jviromet.2014.04.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Utomo D. I. S., Pambudi S., Sjatha F., Kato T., Park E. Y.. Production
of Dengue Virus-like Particles Serotype-3 in Silkworm Larvae and Their
Ability to Elicit a Humoral Immune Response in Mice. AMB Express. 2020;10(1):147.  doi: 10.1186/s13568-020-01087-3.</cite> [<a href="https://doi.org/10.1186/s13568-020-01087-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7431508/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32804287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=AMB%20Express&amp;title=Production%20of%20Dengue%20Virus-like%20Particles%20Serotype-3%20in%20Silkworm%20Larvae%20and%20Their%20Ability%20to%20Elicit%20a%20Humoral%20Immune%20Response%20in%20Mice&amp;author=D.%20I.%20S.%20Utomo&amp;author=S.%20Pambudi&amp;author=F.%20Sjatha&amp;author=T.%20Kato&amp;author=E.%20Y.%20Park&amp;volume=10&amp;issue=1&amp;publication_year=2020&amp;pages=147&amp;pmid=32804287&amp;doi=10.1186/s13568-020-01087-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Utomo D. I. S., Hirono I., Kato T., Park E. Y.. Formation
of Virus-Like
Particles of the Dengue Virus Serotype 2 Expressed in Silkworm Larvae. Mol. Biotechnol. 2019;61(11):852–859. doi: 10.1007/s12033-019-00210-5.</cite> [<a href="https://doi.org/10.1007/s12033-019-00210-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31473916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biotechnol.&amp;title=Formation%20of%20Virus-Like%20Particles%20of%20the%20Dengue%20Virus%20Serotype%202%20Expressed%20in%20Silkworm%20Larvae&amp;author=D.%20I.%20S.%20Utomo&amp;author=I.%20Hirono&amp;author=T.%20Kato&amp;author=E.%20Y.%20Park&amp;volume=61&amp;issue=11&amp;publication_year=2019&amp;pages=852-859&amp;pmid=31473916&amp;doi=10.1007/s12033-019-00210-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Kanagaraj A. P., Verma D., Daniell H.. Expression
of Dengue-3 Premembrane
and Envelope Polyprotein in Lettuce Chloroplasts. Plant Mol. Biol. 2011;76(3–5):323–333. doi: 10.1007/s11103-011-9766-0.</cite> [<a href="https://doi.org/10.1007/s11103-011-9766-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3468899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21431782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Mol.%20Biol.&amp;title=Expression%20of%20Dengue-3%20Premembrane%20and%20Envelope%20Polyprotein%20in%20Lettuce%20Chloroplasts&amp;author=A.%20P.%20Kanagaraj&amp;author=D.%20Verma&amp;author=H.%20Daniell&amp;volume=76&amp;issue=3%E2%80%935&amp;publication_year=2011&amp;pages=323-333&amp;pmid=21431782&amp;doi=10.1007/s11103-011-9766-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Rajpoot R. K., Shukla R., Arora U., Swaminathan S., Khanna N.. Dengue Envelope-Based ‘Four-in-One’
Virus-like
Particles Produced Using Pichia Pastoris Induce Enhancement-Lacking,
Domain III-Directed Tetravalent Neutralising Antibodies in Mice. Sci. Rep. 2018;8(1):8643.  doi: 10.1038/s41598-018-26904-5.</cite> [<a href="https://doi.org/10.1038/s41598-018-26904-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5988708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29872153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Dengue%20Envelope-Based%20%E2%80%98Four-in-One%E2%80%99%20Virus-like%20Particles%20Produced%20Using%20Pichia%20Pastoris%20Induce%20Enhancement-Lacking,%20Domain%20III-Directed%20Tetravalent%20Neutralising%20Antibodies%20in%20Mice&amp;author=R.%20K.%20Rajpoot&amp;author=R.%20Shukla&amp;author=U.%20Arora&amp;author=S.%20Swaminathan&amp;author=N.%20Khanna&amp;volume=8&amp;issue=1&amp;publication_year=2018&amp;pages=8643&amp;pmid=29872153&amp;doi=10.1038/s41598-018-26904-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Poddar A., Ramasamy V., Shukla R., Rajpoot R. K., Arora U., Jain S. K., Swaminathan S., Khanna N.. Virus-like Particles
Derived from Pichia Pastoris-Expressed Dengue Virus Type 1 Glycoprotein
Elicit Homotypic Virus-Neutralizing Envelope Domain III-Directed Antibodies. BMC Biotechnol. 2016;16(1):50.  doi: 10.1186/s12896-016-0280-y.</cite> [<a href="https://doi.org/10.1186/s12896-016-0280-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4908714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27301568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Biotechnol.&amp;title=Virus-like%20Particles%20Derived%20from%20Pichia%20Pastoris-Expressed%20Dengue%20Virus%20Type%201%20Glycoprotein%20Elicit%20Homotypic%20Virus-Neutralizing%20Envelope%20Domain%20III-Directed%20Antibodies&amp;author=A.%20Poddar&amp;author=V.%20Ramasamy&amp;author=R.%20Shukla&amp;author=R.%20K.%20Rajpoot&amp;author=U.%20Arora&amp;volume=16&amp;issue=1&amp;publication_year=2016&amp;pages=50&amp;pmid=27301568&amp;doi=10.1186/s12896-016-0280-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Liu Y., Zhou J., Yu Z., Fang D., Fu C., Zhu X., He Z., Yan H., Jiang L.. Tetravalent Recombinant
Dengue Virus-like Particles as Potential Vaccine Candidates: Immunological
Properties. BMC Microbiol. 2014;14(1):233.  doi: 10.1186/s12866-014-0233-3.</cite> [<a href="https://doi.org/10.1186/s12866-014-0233-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4396058/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25520151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol.&amp;title=Tetravalent%20Recombinant%20Dengue%20Virus-like%20Particles%20as%20Potential%20Vaccine%20Candidates:%20Immunological%20Properties&amp;author=Y.%20Liu&amp;author=J.%20Zhou&amp;author=Z.%20Yu&amp;author=D.%20Fang&amp;author=C.%20Fu&amp;volume=14&amp;issue=1&amp;publication_year=2014&amp;pages=233&amp;pmid=25520151&amp;doi=10.1186/s12866-014-0233-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Mani S., Tripathi L., Raut R., Tyagi P., Arora U., Barman T., Sood R., Galav A., Wahala W., de Silva A., Swaminathan S., Khanna N.. Pichia Pastoris-Expressed
Dengue 2 Envelope Forms Virus-Like Particles without Pre-Membrane
Protein and Induces High Titer Neutralizing Antibodies. PLoS One. 2013;8(5):e64595.  doi: 10.1371/journal.pone.0064595.</cite> [<a href="https://doi.org/10.1371/journal.pone.0064595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3662778/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23717637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Pichia%20Pastoris-Expressed%20Dengue%202%20Envelope%20Forms%20Virus-Like%20Particles%20without%20Pre-Membrane%20Protein%20and%20Induces%20High%20Titer%20Neutralizing%20Antibodies&amp;author=S.%20Mani&amp;author=L.%20Tripathi&amp;author=R.%20Raut&amp;author=P.%20Tyagi&amp;author=U.%20Arora&amp;volume=8&amp;issue=5&amp;publication_year=2013&amp;pages=e64595&amp;pmid=23717637&amp;doi=10.1371/journal.pone.0064595&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>

Tan, B.-H.
; 
Fu, J.
L.
; 
Sugrue, R.
J.
. 
Characterization
of the Dengue Virus Envelope Glycoprotein Expressed in . In Methods
in Molecular Biology; Springer, 2007; Vol. 379, pp 163–176.
</cite> [<a href="https://doi.org/10.1007/978-1-59745-393-6_12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17502678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Molecular%20Biology&amp;publication_year=2007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Hirsch J., Faber B. W., Crowe J. E., Verstrepen B., Cornelissen G..  Production Process
Yields Stable Dengue 1 Virus-Sized Particles (VSPs) Vaccine. 2020;38(17):3305–3312. doi: 10.1016/j.vaccine.2020.03.003.</cite> [<a href="https://doi.org/10.1016/j.vaccine.2020.03.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32197924/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Production%20Process%20Yields%20Stable%20Dengue%201%20Virus-Sized%20Particles%20(VSPs)&amp;author=J.%20Hirsch&amp;author=B.%20W.%20Faber&amp;author=J.%20E.%20Crowe&amp;author=B.%20Verstrepen&amp;author=G.%20Cornelissen&amp;volume=38&amp;issue=17&amp;publication_year=2020&amp;pages=3305-3312&amp;pmid=32197924&amp;doi=10.1016/j.vaccine.2020.03.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Arora U., Tyagi P., Swaminathan S., Khanna N.. Chimeric Hepatitis
B Core Antigen Virus-like Particles Displaying the Envelope Domain
III of Dengue Virus Type 2. J. Nanobiotechnol. 2012;10(1):30.  doi: 10.1186/1477-3155-10-30.</cite> [<a href="https://doi.org/10.1186/1477-3155-10-30" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3411447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22794664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Nanobiotechnol.&amp;title=Chimeric%20Hepatitis%20B%20Core%20Antigen%20Virus-like%20Particles%20Displaying%20the%20Envelope%20Domain%20III%20of%20Dengue%20Virus%20Type%202&amp;author=U.%20Arora&amp;author=P.%20Tyagi&amp;author=S.%20Swaminathan&amp;author=N.%20Khanna&amp;volume=10&amp;issue=1&amp;publication_year=2012&amp;pages=30&amp;pmid=22794664&amp;doi=10.1186/1477-3155-10-30&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Modis Y., Ogata S., Clements D., Harrison S. C.. A Ligand-Binding
Pocket in the Dengue Virus Envelope Glycoprotein. Proc. Natl. Acad. Sci. U.S.A. 2003;100(12):6986–6991. doi: 10.1073/pnas.0832193100.</cite> [<a href="https://doi.org/10.1073/pnas.0832193100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC165817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12759475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=A%20Ligand-Binding%20Pocket%20in%20the%20Dengue%20Virus%20Envelope%20Glycoprotein&amp;author=Y.%20Modis&amp;author=S.%20Ogata&amp;author=D.%20Clements&amp;author=S.%20C.%20Harrison&amp;volume=100&amp;issue=12&amp;publication_year=2003&amp;pages=6986-6991&amp;pmid=12759475&amp;doi=10.1073/pnas.0832193100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Shah M., Wadood A., Rahman Z., Husnain T.. Interaction and Inhibition
of Dengue Envelope Glycoprotein with Mammalian Receptor DC-Sign, an
In-Silico Approach. PLoS One. 2013;8(3):e59211.  doi: 10.1371/journal.pone.0059211.</cite> [<a href="https://doi.org/10.1371/journal.pone.0059211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3601059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23527139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Interaction%20and%20Inhibition%20of%20Dengue%20Envelope%20Glycoprotein%20with%20Mammalian%20Receptor%20DC-Sign,%20an%20In-Silico%20Approach&amp;author=M.%20Shah&amp;author=A.%20Wadood&amp;author=Z.%20Rahman&amp;author=T.%20Husnain&amp;volume=8&amp;issue=3&amp;publication_year=2013&amp;pages=e59211&amp;pmid=23527139&amp;doi=10.1371/journal.pone.0059211&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Cruz-Oliveira C., Freire J. M., Conceição T. M., Higa L. M., Castanho M. A. R. B., Da Poian A. T.. Receptors and Routes
of Dengue Virus
Entry into the Host Cells. FEMS Microbiol. Rev. 2015;39(2):155–170. doi: 10.1093/femsre/fuu004.</cite> [<a href="https://doi.org/10.1093/femsre/fuu004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25725010/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Rev.&amp;title=Receptors%20and%20Routes%20of%20Dengue%20Virus%20Entry%20into%20the%20Host%20Cells&amp;author=C.%20Cruz-Oliveira&amp;author=J.%20M.%20Freire&amp;author=T.%20M.%20Concei%C3%A7%C3%A3o&amp;author=L.%20M.%20Higa&amp;author=M.%20A.%20R.%20B.%20Castanho&amp;volume=39&amp;issue=2&amp;publication_year=2015&amp;pages=155-170&amp;pmid=25725010&amp;doi=10.1093/femsre/fuu004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Raekiansyah M., Buerano C. C., Luz M. A. D., Morita K.. Inhibitory Effect of
the Green Tea Molecule EGCG against Dengue Virus Infection. Arch. Virol. 2018;163(6):1649–1655. doi: 10.1007/s00705-018-3769-y.</cite> [<a href="https://doi.org/10.1007/s00705-018-3769-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29429035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arch.%20Virol.&amp;title=Inhibitory%20Effect%20of%20the%20Green%20Tea%20Molecule%20EGCG%20against%20Dengue%20Virus%20Infection&amp;author=M.%20Raekiansyah&amp;author=C.%20C.%20Buerano&amp;author=M.%20A.%20D.%20Luz&amp;author=K.%20Morita&amp;volume=163&amp;issue=6&amp;publication_year=2018&amp;pages=1649-1655&amp;pmid=29429035&amp;doi=10.1007/s00705-018-3769-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Zhang X., Ge P., Yu X., Brannan J. M., Bi G., Zhang Q., Schein S., Zhou Z. H.. Cryo-EM Structure of the Mature Dengue
Virus at 3.5-Å Resolution. Nat. Struct
Mol. Biol. 2013;20(1):105–110. doi: 10.1038/nsmb.2463.</cite> [<a href="https://doi.org/10.1038/nsmb.2463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3593067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23241927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Struct%20Mol.%20Biol.&amp;title=Cryo-EM%20Structure%20of%20the%20Mature%20Dengue%20Virus%20at%203.5-%C3%85%20Resolution&amp;author=X.%20Zhang&amp;author=P.%20Ge&amp;author=X.%20Yu&amp;author=J.%20M.%20Brannan&amp;author=G.%20Bi&amp;volume=20&amp;issue=1&amp;publication_year=2013&amp;pages=105-110&amp;pmid=23241927&amp;doi=10.1038/nsmb.2463&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>
Schrödinger Release 2019–4: QikProp, Schrödinger; LLC: New York,
NY, 2019.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Schr%C3%B6dinger%20Release%202019%E2%80%934:%20QikProp,%20Schr%C3%B6dinger&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Sankar K., Trainor K., Blazer L. L., Adams J. J., Sidhu S. S., Day T., Meiering E., Maier J. K. X.. A Descriptor Set for Quantitative
Structure-property Relationship Prediction in Biologics. Mol. Inf. 2022;41(9):2100240.  doi: 10.1002/minf.202100240.</cite> [<a href="https://doi.org/10.1002/minf.202100240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35277930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Inf.&amp;title=A%20Descriptor%20Set%20for%20Quantitative%20Structure-property%20Relationship%20Prediction%20in%20Biologics&amp;author=K.%20Sankar&amp;author=K.%20Trainor&amp;author=L.%20L.%20Blazer&amp;author=J.%20J.%20Adams&amp;author=S.%20S.%20Sidhu&amp;volume=41&amp;issue=9&amp;publication_year=2022&amp;pages=2100240&amp;pmid=35277930&amp;doi=10.1002/minf.202100240&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Sugrue R. J., Cui T., Xu Q., Fu J., Cheong Chan Y.. The Production
of Recombinant Dengue Virus E Protein Using and Pichia Pastoris. J.
Virol Methods. 1997;69(1–2):159–169. doi: 10.1016/S0166-0934(97)00151-1.</cite> [<a href="https://doi.org/10.1016/S0166-0934(97)00151-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9504761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Virol%20Methods&amp;title=The%20Production%20of%20Recombinant%20Dengue%20Virus%20E%20Protein%20Using%20and%20Pichia%20Pastoris&amp;author=R.%20J.%20Sugrue&amp;author=T.%20Cui&amp;author=Q.%20Xu&amp;author=J.%20Fu&amp;author=Y.%20Cheong%20Chan&amp;volume=69&amp;issue=1%E2%80%932&amp;publication_year=1997&amp;pages=159-169&amp;pmid=9504761&amp;doi=10.1016/S0166-0934(97)00151-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Staropoli I.. Affinity-Purified
Dengue-2 Virus Envelope Glycoprotein Induces Neutralizing Antibodies
and Protective Immunity in Mice. Vaccine. 1997;15(17–18):1946–1954. doi: 10.1016/S0264-410X(97)00128-X.</cite> [<a href="https://doi.org/10.1016/S0264-410X(97)00128-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9413106/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Affinity-Purified%20Dengue-2%20Virus%20Envelope%20Glycoprotein%20Induces%20Neutralizing%20Antibodies%20and%20Protective%20Immunity%20in%20Mice&amp;author=I.%20Staropoli&amp;volume=15&amp;issue=17%E2%80%9318&amp;publication_year=1997&amp;pages=1946-1954&amp;pmid=9413106&amp;doi=10.1016/S0264-410X(97)00128-X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Jaiswal S., Khanna N., Swaminathan S.. High-Level
Expression and One-Step
Purification of Recombinant Dengue Virus Type 2 Envelope Domain III
Protein in . Protein Expression Purif. 2004;33(1):80–91. doi: 10.1016/j.pep.2003.09.009.</cite> [<a href="https://doi.org/10.1016/j.pep.2003.09.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14680965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Expression%20Purif.&amp;title=High-Level%20Expression%20and%20One-Step%20Purification%20of%20Recombinant%20Dengue%20Virus%20Type%202%20Envelope%20Domain%20III%20Protein%20in&amp;author=S.%20Jaiswal&amp;author=N.%20Khanna&amp;author=S.%20Swaminathan&amp;volume=33&amp;issue=1&amp;publication_year=2004&amp;pages=80-91&amp;pmid=14680965&amp;doi=10.1016/j.pep.2003.09.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Ganguly A., Malabadi R. B., Das D., Suresh M. R., Sunwoo H. H.. Enhanced
Prokaryotic Expression of Dengue Virus Envelope Protein. J. Pharm. Pharm. Sci. 2013;16(4):609–621. doi: 10.18433/J3PC80.</cite> [<a href="https://doi.org/10.18433/J3PC80" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24210067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Pharm.%20Sci.&amp;title=Enhanced%20Prokaryotic%20Expression%20of%20Dengue%20Virus%20Envelope%20Protein&amp;author=A.%20Ganguly&amp;author=R.%20B.%20Malabadi&amp;author=D.%20Das&amp;author=M.%20R.%20Suresh&amp;author=H.%20H.%20Sunwoo&amp;volume=16&amp;issue=4&amp;publication_year=2013&amp;pages=609-621&amp;pmid=24210067&amp;doi=10.18433/J3PC80&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Hirsch J., Faber B. W., Crowe J. E., Verstrepen B., Cornelissen G. E.. Coli Production Process Yields Stable
Dengue 1 Virus-Sized
Particles (VSPs) Vaccine. 2020;38(17):3305–3312. doi: 10.1016/j.vaccine.2020.03.003.</cite> [<a href="https://doi.org/10.1016/j.vaccine.2020.03.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32197924/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Coli%20Production%20Process%20Yields%20Stable%20Dengue%201%20Virus-Sized%20Particles%20(VSPs)&amp;author=J.%20Hirsch&amp;author=B.%20W.%20Faber&amp;author=J.%20E.%20Crowe&amp;author=B.%20Verstrepen&amp;author=G.%20E.%20Cornelissen&amp;volume=38&amp;issue=17&amp;publication_year=2020&amp;pages=3305-3312&amp;pmid=32197924&amp;doi=10.1016/j.vaccine.2020.03.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Singh V. K., Kumar S., Dhaked R. K., Ansari A. S., Lohiya N. K., Tapryal S.. Generation of Oligomers
of Subunit Vaccine Candidate
Glycoprotein D of Herpes Simplex Virus-2 Expressed in Fusion with
IgM Fc Domain(s) in : A Strategy to Enhance the Immunogenicity of the Antigen. 3 Biotech. 2020;10(11):463.  doi: 10.1007/s13205-020-02452-6.</cite> [<a href="https://doi.org/10.1007/s13205-020-02452-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7541101/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33047090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=3%20Biotech&amp;title=Generation%20of%20Oligomers%20of%20Subunit%20Vaccine%20Candidate%20Glycoprotein%20D%20of%20Herpes%20Simplex%20Virus-2%20Expressed%20in%20Fusion%20with%20IgM%20Fc%20Domain(s)%20in%20:%20A%20Strategy%20to%20Enhance%20the%20Immunogenicity%20of%20the%20Antigen&amp;author=V.%20K.%20Singh&amp;author=S.%20Kumar&amp;author=R.%20K.%20Dhaked&amp;author=A.%20S.%20Ansari&amp;author=N.%20K.%20Lohiya&amp;volume=10&amp;issue=11&amp;publication_year=2020&amp;pages=463&amp;pmid=33047090&amp;doi=10.1007/s13205-020-02452-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Tapryal S., Krishnan L., Batra J. K., Kaur K. J., Salunke D. M.. Cloning,
Expression and Efficient Refolding of Carbohydrate–Peptide
Mimicry Recognizing Single Chain Antibody 2D10. Protein Expression Purif. 2010;72(2):162–168. doi: 10.1016/j.pep.2010.03.024.</cite> [<a href="https://doi.org/10.1016/j.pep.2010.03.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20363331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Expression%20Purif.&amp;title=Cloning,%20Expression%20and%20Efficient%20Refolding%20of%20Carbohydrate%E2%80%93Peptide%20Mimicry%20Recognizing%20Single%20Chain%20Antibody%202D10&amp;author=S.%20Tapryal&amp;author=L.%20Krishnan&amp;author=J.%20K.%20Batra&amp;author=K.%20J.%20Kaur&amp;author=D.%20M.%20Salunke&amp;volume=72&amp;issue=2&amp;publication_year=2010&amp;pages=162-168&amp;pmid=20363331&amp;doi=10.1016/j.pep.2010.03.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Modis Y., Ogata S., Clements D., Harrison S. C.. A Ligand-Binding
Pocket in the Dengue Virus Envelope Glycoprotein. Proc. Natl. Acad. Sci. U.S.A. 2003;100(12):6986–6991. doi: 10.1073/pnas.0832193100.</cite> [<a href="https://doi.org/10.1073/pnas.0832193100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC165817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12759475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=A%20Ligand-Binding%20Pocket%20in%20the%20Dengue%20Virus%20Envelope%20Glycoprotein&amp;author=Y.%20Modis&amp;author=S.%20Ogata&amp;author=D.%20Clements&amp;author=S.%20C.%20Harrison&amp;volume=100&amp;issue=12&amp;publication_year=2003&amp;pages=6986-6991&amp;pmid=12759475&amp;doi=10.1073/pnas.0832193100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Kuhn R. J., Zhang W., Rossmann M. G., Pletnev S. V., Corver J., Lenches E., Jones C. T., Mukhopadhyay S., Chipman P. R., Strauss E. G., Baker T. S., Strauss J. H.. Structure
of Dengue Virus. Cell. 2002;108(5):717–725. doi: 10.1016/S0092-8674(02)00660-8.</cite> [<a href="https://doi.org/10.1016/S0092-8674(02)00660-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4152842/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11893341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Structure%20of%20Dengue%20Virus&amp;author=R.%20J.%20Kuhn&amp;author=W.%20Zhang&amp;author=M.%20G.%20Rossmann&amp;author=S.%20V.%20Pletnev&amp;author=J.%20Corver&amp;volume=108&amp;issue=5&amp;publication_year=2002&amp;pages=717-725&amp;pmid=11893341&amp;doi=10.1016/S0092-8674(02)00660-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Kumar S., Singh V. K., Vasam M., Patil P. S., Dhaked R. K., Ansari A. S., Lohiya N. K., Parashar D., Tapryal S.. An in Vitro
Refolding Method to Produce Oligomers of Anti-CHIKV, E2-IgM Fc Fusion
Subunit Vaccine Candidates Expressed in . J. Immunol. Methods. 2020;487:112869.  doi: 10.1016/j.jim.2020.112869.</cite> [<a href="https://doi.org/10.1016/j.jim.2020.112869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32971119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.%20Methods&amp;title=An%20in%20Vitro%20Refolding%20Method%20to%20Produce%20Oligomers%20of%20Anti-CHIKV,%20E2-IgM%20Fc%20Fusion%20Subunit%20Vaccine%20Candidates%20Expressed%20in&amp;author=S.%20Kumar&amp;author=V.%20K.%20Singh&amp;author=M.%20Vasam&amp;author=P.%20S.%20Patil&amp;author=R.%20K.%20Dhaked&amp;volume=487&amp;publication_year=2020&amp;pages=112869&amp;pmid=32971119&amp;doi=10.1016/j.jim.2020.112869&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Zhang X., Sheng J., Plevka P., Kuhn R. J., Diamond M. S., Rossmann M. G.. Dengue Structure
Differs at the Temperatures of Its
Human and Mosquito Hosts. Proc. Natl. Acad.
Sci. U.S.A. 2013;110(17):6795–6799. doi: 10.1073/pnas.1304300110.</cite> [<a href="https://doi.org/10.1073/pnas.1304300110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3637732/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23569243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Dengue%20Structure%20Differs%20at%20the%20Temperatures%20of%20Its%20Human%20and%20Mosquito%20Hosts&amp;author=X.%20Zhang&amp;author=J.%20Sheng&amp;author=P.%20Plevka&amp;author=R.%20J.%20Kuhn&amp;author=M.%20S.%20Diamond&amp;volume=110&amp;issue=17&amp;publication_year=2013&amp;pages=6795-6799&amp;pmid=23569243&amp;doi=10.1073/pnas.1304300110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Miller J. L., deWet B. J. M., Martinez-Pomares L., Radcliffe C. M., Dwek R. A., Rudd P. M., Gordon S.. The Mannose Receptor
Mediates Dengue Virus Infection of Macrophages. PLoS Pathog. 2008;4(2):e17.  doi: 10.1371/journal.ppat.0040017.</cite> [<a href="https://doi.org/10.1371/journal.ppat.0040017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2233670/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18266465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&amp;title=The%20Mannose%20Receptor%20Mediates%20Dengue%20Virus%20Infection%20of%20Macrophages&amp;author=J.%20L.%20Miller&amp;author=B.%20J.%20M.%20deWet&amp;author=L.%20Martinez-Pomares&amp;author=C.%20M.%20Radcliffe&amp;author=R.%20A.%20Dwek&amp;volume=4&amp;issue=2&amp;publication_year=2008&amp;pages=e17&amp;pmid=18266465&amp;doi=10.1371/journal.ppat.0040017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Wen C., Dechsupa N., Yu Z., Zhang X., Liang S., Lei X., Xu T., Gao X., Hu Q., Innuan P., Kantapan J., Lü M.. Pentagalloyl Glucose: A Review of
Anticancer Properties, Molecular Targets, Mechanisms of Action, Pharmacokinetics,
and Safety Profile. Molecules. 2023;28(12):4856.  doi: 10.3390/molecules28124856.</cite> [<a href="https://doi.org/10.3390/molecules28124856" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10300743/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37375411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Pentagalloyl%20Glucose:%20A%20Review%20of%20Anticancer%20Properties,%20Molecular%20Targets,%20Mechanisms%20of%20Action,%20Pharmacokinetics,%20and%20Safety%20Profile&amp;author=C.%20Wen&amp;author=N.%20Dechsupa&amp;author=Z.%20Yu&amp;author=X.%20Zhang&amp;author=S.%20Liang&amp;volume=28&amp;issue=12&amp;publication_year=2023&amp;pages=4856&amp;pmid=37375411&amp;doi=10.3390/molecules28124856&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>

Tan, H.
; 
Ma, C.
; 
Wang, J.
. 
Invalidation of Dieckol and 1,2,3,4,6-Pentagalloylglucose
(PGG) as SARS-CoV-2 Main Protease Inhibitors and the Discovery of
PGG as a Papain-like Protease Inhibitor. 2022. 
10.21203/rs.3.rs-1490282/v1.</cite> [<a href="https://doi.org/10.21203/rs.3.rs-1490282/v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9095416/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35578732/" class="usa-link">PubMed</a>]</li>
<li id="ref45">
<cite>Cao Y., Himmeldirk K. B., Qian Y., Ren Y., Malki A., Chen X.. Biological and Biomedical Functions of Penta-O-Galloyl-d-Glucose
and Its Derivatives. J. Nat. Med. 2014;68(3):465–472. doi: 10.1007/s11418-014-0823-2.</cite> [<a href="https://doi.org/10.1007/s11418-014-0823-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24532420/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Nat.%20Med.&amp;title=Biological%20and%20Biomedical%20Functions%20of%20Penta-O-Galloyl-d-Glucose%20and%20Its%20Derivatives&amp;author=Y.%20Cao&amp;author=K.%20B.%20Himmeldirk&amp;author=Y.%20Qian&amp;author=Y.%20Ren&amp;author=A.%20Malki&amp;volume=68&amp;issue=3&amp;publication_year=2014&amp;pages=465-472&amp;pmid=24532420&amp;doi=10.1007/s11418-014-0823-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Patnaik S. S., Simionescu D. T., Goergen C. J., Hoyt K., Sirsi S., Finol E. A.. Pentagalloyl
Glucose and Its Functional Role in Vascular
Health: Biomechanics and Drug-Delivery Characteristics. Ann. Biomed. Eng. 2019;47(1):39–59. doi: 10.1007/s10439-018-02145-5.</cite> [<a href="https://doi.org/10.1007/s10439-018-02145-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6318003/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30298373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann.%20Biomed.%20Eng.&amp;title=Pentagalloyl%20Glucose%20and%20Its%20Functional%20Role%20in%20Vascular%20Health:%20Biomechanics%20and%20Drug-Delivery%20Characteristics&amp;author=S.%20S.%20Patnaik&amp;author=D.%20T.%20Simionescu&amp;author=C.%20J.%20Goergen&amp;author=K.%20Hoyt&amp;author=S.%20Sirsi&amp;volume=47&amp;issue=1&amp;publication_year=2019&amp;pages=39-59&amp;pmid=30298373&amp;doi=10.1007/s10439-018-02145-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>
Schrödinger
Release 2019–4: Protein Preparation
Wizard; Schrödinger.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Schr%C3%B6dinger%20Release%202019%E2%80%934:%20Protein%20Preparation%20Wizard&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>
Schrödinger
Release 2019–4: LigPrep, Schrödinger; LLC: New York,
NY, 2019.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Schr%C3%B6dinger%20Release%202019%E2%80%934:%20LigPrep,%20Schr%C3%B6dinger&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>Friesner R. A., Murphy R. B., Repasky M. P., Frye L. L., Greenwood J. R., Halgren T. A., Sanschagrin P. C., Mainz D. T.. Extra Precision
Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure
for Protein–Ligand Complexes. J. Med.
Chem. 2006;49(21):6177–6196. doi: 10.1021/jm051256o.</cite> [<a href="https://doi.org/10.1021/jm051256o" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17034125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=Extra%20Precision%20Glide:%20Docking%20and%20Scoring%20Incorporating%20a%20Model%20of%20Hydrophobic%20Enclosure%20for%20Protein%E2%80%93Ligand%20Complexes&amp;author=R.%20A.%20Friesner&amp;author=R.%20B.%20Murphy&amp;author=M.%20P.%20Repasky&amp;author=L.%20L.%20Frye&amp;author=J.%20R.%20Greenwood&amp;volume=49&amp;issue=21&amp;publication_year=2006&amp;pages=6177-6196&amp;pmid=17034125&amp;doi=10.1021/jm051256o&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>Jacobson M. P., Pincus D. L., Rapp C. S., Day T. J. F., Honig B., Shaw D. E., Friesner R. A.. A Hierarchical Approach to All-atom
Protein Loop Prediction. Proteins:Struct., Funct.,
Bioinf. 2004;55(2):351–367. doi: 10.1002/prot.10613.</cite> [<a href="https://doi.org/10.1002/prot.10613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15048827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins:Struct.,%20Funct.,%20Bioinf.&amp;title=A%20Hierarchical%20Approach%20to%20All-atom%20Protein%20Loop%20Prediction&amp;author=M.%20P.%20Jacobson&amp;author=D.%20L.%20Pincus&amp;author=C.%20S.%20Rapp&amp;author=T.%20J.%20F.%20Day&amp;author=B.%20Honig&amp;volume=55&amp;issue=2&amp;publication_year=2004&amp;pages=351-367&amp;pmid=15048827&amp;doi=10.1002/prot.10613&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Hollingsworth S. A., Dror R. O.. Molecular Dynamics Simulation for All. Neuron. 2018;99(6):1129–1143. doi: 10.1016/j.neuron.2018.08.011.</cite> [<a href="https://doi.org/10.1016/j.neuron.2018.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6209097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30236283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Molecular%20Dynamics%20Simulation%20for%20All&amp;author=S.%20A.%20Hollingsworth&amp;author=R.%20O.%20Dror&amp;volume=99&amp;issue=6&amp;publication_year=2018&amp;pages=1129-1143&amp;pmid=30236283&amp;doi=10.1016/j.neuron.2018.08.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>
Schrödinger
Release 2019–4: Desmond Molecular
Dynamics System, Maestro-Desmond Interoperability Tools; D. E. Shaw Research: New York, NY, 2019.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Schr%C3%B6dinger%20Release%202019%E2%80%934:%20Desmond%20Molecular%20Dynamics%20System,%20Maestro-Desmond%20Interoperability%20Tools&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12332551/bin/ao5c02548_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c02548_si_001.pdf</a><sup> (31.7MB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Omega are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsomega.5c02548"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ao5c02548.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332551/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332551/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332551%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332551/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332551/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332551/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40787407/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332551/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40787407/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332551/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332551/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="ZKYNY5Kwut0PmTKTd6a8ojf7ZSoM6aYXpu0YG8OmgOxoL4mwWWUjhU5N0R1HBBcr">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
