
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Optimizing genomic prediction with transfer learning under a ridge regression framework - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="FF207FF889B712431A7FF80026FBD3F3.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12271683/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Optimizing genomic prediction with transfer learning under a ridge regression framework">
<meta name="citation_author" content="Osval A Montesinos‐López">
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, Colima, México">
<meta name="citation_author" content="Eduardo A Barajas‐Ramirez">
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, Colima, México">
<meta name="citation_author" content="Josafhat Salinas‐Ruiz">
<meta name="citation_author_institution" content="Colegio de Postgraduados Campus Córdoba, Veracruz, Mexico">
<meta name="citation_author" content="Abelardo Montesinos‐López">
<meta name="citation_author_institution" content="Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, Guadalajara, México">
<meta name="citation_author" content="Guillermo Gerard">
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Edo. de México, México">
<meta name="citation_author" content="Paolo Vitale">
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Edo. de México, México">
<meta name="citation_author" content="Susanne Dreisigacker">
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Edo. de México, México">
<meta name="citation_author" content="Carolina Saint Pierre">
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Edo. de México, México">
<meta name="citation_author" content="José Crossa">
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Edo. de México, México">
<meta name="citation_author_institution" content="Colegio de Postgraduados, Edo. de México, México">
<meta name="citation_publication_date" content="2025 Jul 17">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70049">
<meta name="citation_doi" content="10.1002/tpg2.70049">
<meta name="citation_pmid" content="40676776">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12271683/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12271683/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12271683/pdf/TPG2-18-e70049.pdf">
<meta name="description" content="Genomic selection (GS) is a predictive plant and animal methodology that allows the selection of plants and animals based on predictions without the need to measure the phenotype. However, its practical application requires challenging prediction ...">
<meta name="og:title" content="Optimizing genomic prediction with transfer learning under a ridge regression framework">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Genomic selection (GS) is a predictive plant and animal methodology that allows the selection of plants and animals based on predictions without the need to measure the phenotype. However, its practical application requires challenging prediction ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12271683/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12271683">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70049"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70049.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12271683%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12271683/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12271683/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12271683/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70049" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Jul 17;18(3):e70049. doi: <a href="https://doi.org/10.1002/tpg2.70049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70049</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Optimizing genomic prediction with transfer learning under a ridge regression framework</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Montesinos%E2%80%90L%C3%B3pez%20OA%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Osval A Montesinos‐López</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Osval A Montesinos‐López</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Facultad de Telemática, Universidad de Colima, Colima, México</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Montesinos%E2%80%90L%C3%B3pez%20OA%22%5BAuthor%5D" class="usa-link"><span class="name western">Osval A Montesinos‐López</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Barajas%E2%80%90Ramirez%20EA%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Eduardo A Barajas‐Ramirez</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Eduardo A Barajas‐Ramirez</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Facultad de Telemática, Universidad de Colima, Colima, México</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Barajas%E2%80%90Ramirez%20EA%22%5BAuthor%5D" class="usa-link"><span class="name western">Eduardo A Barajas‐Ramirez</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Salinas%E2%80%90Ruiz%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Josafhat Salinas‐Ruiz</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Josafhat Salinas‐Ruiz</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Colegio de Postgraduados Campus Córdoba, Veracruz, Mexico</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Salinas%E2%80%90Ruiz%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Josafhat Salinas‐Ruiz</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Montesinos%E2%80%90L%C3%B3pez%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Abelardo Montesinos‐López</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Abelardo Montesinos‐López</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, Guadalajara, México</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Montesinos%E2%80%90L%C3%B3pez%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Abelardo Montesinos‐López</span></a>
</div>
</div>
<sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gerard%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Guillermo Gerard</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Guillermo Gerard</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>International Maize and Wheat Improvement Center (CIMMYT), Edo. de México, México</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gerard%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Guillermo Gerard</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vitale%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Paolo Vitale</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Paolo Vitale</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>International Maize and Wheat Improvement Center (CIMMYT), Edo. de México, México</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vitale%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Paolo Vitale</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dreisigacker%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Susanne Dreisigacker</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Susanne Dreisigacker</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>International Maize and Wheat Improvement Center (CIMMYT), Edo. de México, México</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dreisigacker%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Susanne Dreisigacker</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pierre%20CS%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Carolina Saint Pierre</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Carolina Saint Pierre</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>International Maize and Wheat Improvement Center (CIMMYT), Edo. de México, México</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pierre%20CS%22%5BAuthor%5D" class="usa-link"><span class="name western">Carolina Saint Pierre</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Crossa%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">José Crossa</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">José Crossa</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>International Maize and Wheat Improvement Center (CIMMYT), Edo. de México, México</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>Colegio de Postgraduados, Edo. de México, México</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Crossa%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">José Crossa</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270049-aff-0001">
<sup>
<sup>1</sup>
</sup>Facultad de Telemática, Universidad de Colima, Colima, México</div>
<div id="tpg270049-aff-0002">
<sup>
<sup>2</sup>
</sup>Colegio de Postgraduados Campus Córdoba, Veracruz, Mexico</div>
<div id="tpg270049-aff-0003">
<sup>
<sup>3</sup>
</sup>Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, Guadalajara, México</div>
<div id="tpg270049-aff-0004">
<sup>
<sup>4</sup>
</sup>International Maize and Wheat Improvement Center (CIMMYT), Edo. de México, México</div>
<div id="tpg270049-aff-0005">
<sup>
<sup>5</sup>
</sup>Colegio de Postgraduados, Edo. de México, México</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Abelardo Montesinos‐López, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, Guadalajara, 44430, Jalisco, México. Email: <span>abelardo.montesinos@academicos.udg.mx</span>
, José Crossa, International Maize and Wheat Improvement Center (CIMMYT), Km 45, Carretera México‐Veracruz, CP 52640, Edo. de México, México. Email: <span>j.crossa@cgiar.org</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Mar 8; Received 2024 Oct 16; Accepted 2025 Apr 12; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 International Maize and Wheat Improvement Center(CIMMYT). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12271683  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40676776/" class="usa-link">40676776</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Genomic selection (GS) is a predictive plant and animal methodology that allows the selection of plants and animals based on predictions without the need to measure the phenotype. However, its practical application requires challenging prediction accuracy due to the noise observations collected in experiments in these areas. Many strategies and approaches have been proposed to improve the prediction accuracy of this methodology. This paper explores the use of transfer learning in the context of GS. Transfer learning with (1) ridge regression (RR) (Transfer RR) and (2) analytic RR (ARR) (Transfer ARR) were applied from cultivars in the proxy environment to predict those cultivars in the goal environments. Also, we compared the performance of models RR and ARR without transfer learning. We used 11 real multi‐environment datasets (wheat and rice) and evaluated them in terms of Pearson's correlation (Cor) and normalized root mean square error (NRMSE). Our study shows empirical evidence that the Transfer RR or Transfer ARR approaches significantly enhanced predictive performance. Across the datasets, Transfer RR (or Transfer ARR) method improved Cor by 22.962% and NRMSE by 5.757%, in comparison to models RR and ARR. These results underscore the potential of Transfer RR (or Transfer ARR) when enhancing predictive accuracy in this context.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270049-list-10001" class="list" style="list-style-type:disc">
<li><p>Genomic selection is transforming plant breeding but still faces challenges in achieving high prediction accuracy with limited data.</p></li>
<li><p>Transfer learning adapts models trained on one task to improve performance on a related task, enabling efficient learning with limited data.</p></li>
<li><p>Transfer learning leverages knowledge from a well‐studied domain to enhance prediction in a related, data‐scarce domain.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270049-dl-0001">
<dt>ARR</dt>
<dd><p>analytic ridge regression</p></dd>
<dt>Cor</dt>
<dd><p>correlation</p></dd>
<dt>EYT</dt>
<dd><p>elite yield trial</p></dd>
<dt>GS</dt>
<dd><p>genomic selection</p></dd>
<dt>IR</dt>
<dd><p>irrigation</p></dd>
<dt>NRMSE</dt>
<dd><p>normalized root mean square error</p></dd>
<dt>RR</dt>
<dd><p>ridge regression</p></dd>
</dl>
<section id="tpg270049-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Genomic selection (GS) has revolutionized plant breeding by significantly lowering the expenses associated with phenotyping. This approach leverages machine learning models that are trained on a reference population with both phenotypic and genotypic information. Once trained, these models can predict phenotypic traits for a new population using only genotypic data (Montesinos‐López et al., <a href="#tpg270049-bib-0021" class="usa-link" aria-describedby="tpg270049-bib-0021">2022</a>). As a result, GS is increasingly implemented in breeding programs, allowing for the early identification of promising candidate lines without the need for extensive phenotypic evaluation.</p>
<p>GS has made significant progress in a wide array of crop breeding programs, driving notable improvements (Heffner et al., <a href="#tpg270049-bib-0010" class="usa-link" aria-describedby="tpg270049-bib-0010">2009</a>). In maize, GS has played a critical role in increasing yields, bolstering resistance to diseases, and enhancing drought tolerance (Crossa et al., <a href="#tpg270049-bib-0006" class="usa-link" aria-describedby="tpg270049-bib-0006">2013</a>). For wheat, this technique has led to advancements in grain yield, disease resistance, and quality traits (Crossa et al., <a href="#tpg270049-bib-0006" class="usa-link" aria-describedby="tpg270049-bib-0006">2013</a>; Montesinos‐López et al., <a href="#tpg270049-bib-0018" class="usa-link" aria-describedby="tpg270049-bib-0018">2016</a>). Rice breeding programs have also benefited from GS, which has contributed to better yields, improved resistance to pests and diseases, and greater adaptability to environmental conditions (Bartholomé et al., <a href="#tpg270049-bib-0003" class="usa-link" aria-describedby="tpg270049-bib-0003">2022</a>; Spindel et al., <a href="#tpg270049-bib-0026" class="usa-link" aria-describedby="tpg270049-bib-0026">2015</a>). In soybean, GS has optimized yield, disease resistance, and oil and protein content (Jean et al., <a href="#tpg270049-bib-0013" class="usa-link" aria-describedby="tpg270049-bib-0013">2021</a>; J. Zhang et al., <a href="#tpg270049-bib-0035" class="usa-link" aria-describedby="tpg270049-bib-0035">2016</a>). Barley has seen improved yield, malting quality, and disease resistance (Nielsen et al., <a href="#tpg270049-bib-0023" class="usa-link" aria-describedby="tpg270049-bib-0023">2016</a>), while GS in cassava has boosted yield, disease resistance, and nutritional value (Long et al., <a href="#tpg270049-bib-0016" class="usa-link" aria-describedby="tpg270049-bib-0016">2023</a>; Yonis et al., <a href="#tpg270049-bib-0033" class="usa-link" aria-describedby="tpg270049-bib-0033">2020</a>). Potato breeding has similarly benefited, with increases in yield, tuber quality, and disease resistance (Adams et al., <a href="#tpg270049-bib-0001" class="usa-link" aria-describedby="tpg270049-bib-0001">2023</a>; Habyarimana et al., <a href="#tpg270049-bib-0009" class="usa-link" aria-describedby="tpg270049-bib-0009">2017</a>). These examples highlight GS's widespread impact in advancing essential traits across diverse crops.</p>
<p>However, GS is not yet fully optimized for all plant breeding programs due to several factors that influence its prediction accuracy (Montesinos López et al., <a href="#tpg270049-bib-0021" class="usa-link" aria-describedby="tpg270049-bib-0021">2022</a>). Key challenges include the level of relatedness between training and testing populations, the quality and density of genetic markers, population size and structure, the heritability of the traits being predicted, and the choice of prediction models (Alemu et al., <a href="#tpg270049-bib-0002" class="usa-link" aria-describedby="tpg270049-bib-0002">2024</a>). Additionally, the goals of prediction—whether focusing on tested lines in tested environments, untested lines in tested environments, or even untested lines in untested environments—can also impact accuracy. Factors such as genetic architecture further complicate the achievement of consistent results (Alemu et al., <a href="#tpg270049-bib-0002" class="usa-link" aria-describedby="tpg270049-bib-0002">2024</a>).</p>
<p>To further improve the practical utility of GS, research is continuously aimed at enhancing its predictive accuracy. A diverse range of statistical and machine learning techniques has been investigated, including linear models, mixed models, random forests, support vector machines, Bayesian approaches, and deep learning (Montesinos‐López et al., <a href="#tpg270049-bib-0021" class="usa-link" aria-describedby="tpg270049-bib-0021">2022</a>). Each one of these methods brings distinct advantages, from the simplicity and interpretability of linear and mixed models to the flexibility and power of more complex algorithms such as random forests and deep learning.</p>
<p>Despite these advancements, the inherent noise and complexity in the data collected through plant breeding programs continue to represent significant challenges. The presence of environmental variability, genotype‐by‐environment interactions, and incomplete or imprecise phenotypic data introduces uncertainty that often limits the accuracy of predictions (Crossa et al., <a href="#tpg270049-bib-0006" class="usa-link" aria-describedby="tpg270049-bib-0006">2013</a>). This issue is particularly pronounced when the models are applied to untested lines or environments, where the lack of direct phenotypic information complicates prediction efforts.</p>
<p>As a result, there is a growing need for novel approaches that can better handle these complexities and reduce prediction errors. These may include hybrid models that combine the strengths of multiple techniques, more sophisticated feature selection methods, and advanced algorithms that are better equipped to manage noise and uncertainty in large, complex datasets. Some types of hybrid models are ensembles and transfer learning. Recently authors have demonstrated that combining predictions from different models using simple ensemble approaches results in better prediction accuracy than using any one of the models separately from the others (Kick &amp; Washburn, <a href="#tpg270049-bib-0015" class="usa-link" aria-describedby="tpg270049-bib-0015">2023</a>). Tomura et al. (<a href="#tpg270049-bib-0028" class="usa-link" aria-describedby="tpg270049-bib-0028">2023</a>) applied the concept of an ensemble to predict several traits in crop breeding. Authors found that the prediction accuracy in the ensemble approach outperformed other models in most cases. While in the context of genomic predictions Washburn et al. (<a href="#tpg270049-bib-0031" class="usa-link" aria-describedby="tpg270049-bib-0031">2021</a>) showed that transfer learning methods can enhance genomic prediction accuracy.</p>
<p>Furthermore, integrating multi‐omics data—such as genomic, transcriptomic, and metabolomic information—into GS models is promising in terms of capturing a more complete picture of trait inheritance and expression. Such innovations are essential to push the boundaries of GS and make it a more reliable and robust tool for plant breeding programs.</p>
<p>Transfer learning is a statistical machine learning technique where a model developed for one task is adapted to a different, but related, task (Pan &amp; Yang, <a href="#tpg270049-bib-0024" class="usa-link" aria-describedby="tpg270049-bib-0024">2010</a>). The key principle behind transfer learning is the reuse of knowledge, in the form of learned features or parameters, from a source domain or dataset to improve learning efficiency and performance in a target domain. This is particularly useful in situations in which the target domain has limited data, since the model can leverage patterns or insights gained from a larger, related dataset (Pan &amp; Yang, <a href="#tpg270049-bib-0024" class="usa-link" aria-describedby="tpg270049-bib-0024">2010</a>).</p>
<p>Transfer learning is often used in scenarios such as natural language processing, image recognition, and regression tasks. The technique typically involves pre‐training a model on a broad dataset, followed by fine‐tuning or adjustment of the model parameters to suit the specific requirements of the target task. By initializing the model with pre‐learned knowledge, transfer learning can reduce training time, enhance model accuracy, and help overcome challenges related to data scarcity or computational constraints (Weiss et al., <a href="#tpg270049-bib-0032" class="usa-link" aria-describedby="tpg270049-bib-0032">2016</a>).</p>
<p>Although transfer learning has been applied in genomic prediction to leverage information from related datasets or populations, its use has not been as extensively explored or rigorously implemented as required to fully harness its potential (Washburn et al., <a href="#tpg270049-bib-0036" class="usa-link" aria-describedby="tpg270049-bib-0036">2020</a>). This limitation highlights the need for more systematic studies and innovative approaches to adapt transfer learning methodologies for genomic prediction tasks effectively (Montesinos‐López, Montesinos‐López, Pérez‐Rodríguez, et al., <a href="#tpg270049-bib-0020" class="usa-link" aria-describedby="tpg270049-bib-0020">2021</a>; Montesinos‐López, Montesinos‐López, Hernandez‐Suarez, et al., <a href="#tpg270049-bib-0019" class="usa-link" aria-describedby="tpg270049-bib-0019">2021</a>).</p>
<p>In regression contexts, transfer learning can involve the transfer of parameter estimates from previous models, which can act as starting points for new models. However, when complete parameter estimates are unavailable or incompatible, techniques like penalized estimation are used to refine these initial estimates, improving the model's performance in the target domain (van Wieringen &amp; Binder, <a href="#tpg270049-bib-0029" class="usa-link" aria-describedby="tpg270049-bib-0029">2020</a>).</p>
<p>Ridge regression (RR) provides a robust framework for genomic prediction and could integrate effectively into transfer learning strategies. Transfer learning leverages information from related datasets or domains to improve performance, especially when training data are limited. This is particularly useful in GS, where environmental or population‐specific data are heterogeneous. The regularization feature of RR prevents overfitting to specific datasets, enabling the development of models that generalize well across domains. For instance, in plant breeding, a model trained on genomic data from one environment can be adapted to predict traits in another environment. Studies like Dar and Baraniuk (<a href="#tpg270049-bib-0007" class="usa-link" aria-describedby="tpg270049-bib-0007">2022</a>) and Chen et al. (<a href="#tpg270049-bib-0004" class="usa-link" aria-describedby="tpg270049-bib-0004">2015</a>) demonstrate the effectiveness of RR and similar linear models in transfer learning,</p>
<p>Furthermore, the study of Tang and de Sa (<a href="#tpg270049-bib-0027" class="usa-link" aria-describedby="tpg270049-bib-0027">2020</a>) addresses the computational challenges of fine‐tuning deep neural networks for transfer learning by leveraging the low‐rank properties of feature vectors produced by these networks. The authors demonstrate success in both supervised and semi‐supervised transfer learning tasks. Moreover, transfer learning with random coefficient RR was analyzed by H. Zhang and Li (<a href="#tpg270049-bib-0034" class="usa-link" aria-describedby="tpg270049-bib-0034">2023</a>), which proposes estimators that combine ridge estimates from both target and source models, with weights determined by minimizing empirical estimation or prediction risks. Their approach is applied to predicting polygenic risk scores for lipid traits, demonstrating improved prediction errors compared to single sample or Lasso‐based transfer learning methods.</p>
<p>For the reasons outlined above, this study investigated the potential of transfer learning in GS within a RR framework. This evaluation was conducted using 11 multi‐environment real datasets. In this context, we applied transfer learning by leveraging information from one environment (proxy) to improve predictions in another environment (target). The results of transfer learning were compared to those obtained without applying the transfer process, using cross‐validation to assess predictive performance. The evaluation of the prediction models was conducted using two metrics: the average Pearson's correlation (Cor) and the average normalized root mean square error (NRMSE).</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270049-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270049-list-0001" class="list" style="list-style-type:disc">
<li><p>Genomic selection is transforming plant breeding but still faces challenges in achieving high prediction accuracy with limited data.</p></li>
<li><p>Transfer learning adapts models trained on one task to improve performance on a related task, enabling efficient learning with limited data.</p></li>
<li><p>Transfer learning leverages knowledge from a well‐studied domain to enhance prediction in a related, data‐scarce domain.</p></li>
</ul></section></section></section><section id="tpg270049-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270049-sec-0040"><h3 class="pmc_sec_title">2.1. Datasets</h3>
<p>Table <a href="#tpg270049-tbl-0001" class="usa-link">1</a> outlines the general information of the 11 multi‐environment datasets used in this study. A total of nine datasets are from wheat trials and two belong to rice sets. Also, it is important to point out that the 11 datasets are divided into two subgroups. Subgroup G1 contains all those datasets that share the same lines in all its environments (Prop. shared lines = 1) while subgroup G2 are those datasets that share less than 100% of lines between environments (Prop. shared lines &lt; 1). For this reason, the environments of subgroup G1 are more related than those of subgroup G2. Note that Indica and Japonica datasets pertain to rice, whereas the remaining datasets are related to wheat. Detailed information about the Indica and Japonica datasets can be found in Monteverde et al. (<a href="#tpg270049-bib-0022" class="usa-link" aria-describedby="tpg270049-bib-0022">2019</a>).</p>
<section class="tw xbox font-sm" id="tpg270049-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Description of the datasets.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Dataset number</th>
<th align="left" rowspan="1" colspan="1">
<p>Dataset</p>
<p>(crop and number)</p>
</th>
<th align="left" rowspan="1" colspan="1">
<p>Cultivars</p>
<p>(number)</p>
</th>
<th align="left" rowspan="1" colspan="1">
<p>Markers</p>
<p>(number)</p>
</th>
<th align="left" rowspan="1" colspan="1">
<p>Environments</p>
<p>(number)</p>
</th>
<th align="left" rowspan="1" colspan="1">
<p>Traits</p>
<p>(number)</p>
</th>
<th align="left" rowspan="1" colspan="1">Prop. shared lines</th>
<th align="left" rowspan="1" colspan="1">Subgroup</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Dataset 1</td>
<td align="left" rowspan="1" colspan="1">EYT_1</td>
<td align="left" rowspan="1" colspan="1">766</td>
<td align="left" rowspan="1" colspan="1">2038</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">G1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Dataset 2</td>
<td align="left" rowspan="1" colspan="1">EYT_2</td>
<td align="left" rowspan="1" colspan="1">775</td>
<td align="left" rowspan="1" colspan="1">2038</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">G1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Dataset 3</td>
<td align="left" rowspan="1" colspan="1">EYT_3</td>
<td align="left" rowspan="1" colspan="1">964</td>
<td align="left" rowspan="1" colspan="1">2038</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">G1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Dataset 4</td>
<td align="left" rowspan="1" colspan="1">Indica</td>
<td align="left" rowspan="1" colspan="1">327</td>
<td align="left" rowspan="1" colspan="1">16,383</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">G1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Dataset 5</td>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">320</td>
<td align="left" rowspan="1" colspan="1">16,383</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">0.510</td>
<td align="left" rowspan="1" colspan="1">G2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Dataset 6</td>
<td align="left" rowspan="1" colspan="1">Wheat_1</td>
<td align="left" rowspan="1" colspan="1">1301</td>
<td align="left" rowspan="1" colspan="1">5741</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">0.363</td>
<td align="left" rowspan="1" colspan="1">G2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Dataset 7</td>
<td align="left" rowspan="1" colspan="1">Wheat_2</td>
<td align="left" rowspan="1" colspan="1">1403</td>
<td align="left" rowspan="1" colspan="1">5741</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">0.378</td>
<td align="left" rowspan="1" colspan="1">G2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Dataset 8</td>
<td align="left" rowspan="1" colspan="1">Wheat_3</td>
<td align="left" rowspan="1" colspan="1">1275</td>
<td align="left" rowspan="1" colspan="1">5741</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">0.347</td>
<td align="left" rowspan="1" colspan="1">G2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Dataset 9</td>
<td align="left" rowspan="1" colspan="1">Wheat_4</td>
<td align="left" rowspan="1" colspan="1">1388</td>
<td align="left" rowspan="1" colspan="1">5741</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">0.397</td>
<td align="left" rowspan="1" colspan="1">G2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Dataset 10</td>
<td align="left" rowspan="1" colspan="1">Wheat_5</td>
<td align="left" rowspan="1" colspan="1">1398</td>
<td align="left" rowspan="1" colspan="1">5741</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">0.356</td>
<td align="left" rowspan="1" colspan="1">G2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Dataset 11</td>
<td align="left" rowspan="1" colspan="1">Wheat_6</td>
<td align="left" rowspan="1" colspan="1">1277</td>
<td align="left" rowspan="1" colspan="1">5741</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">0.096</td>
<td align="left" rowspan="1" colspan="1">G2</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270049-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270049-tbl1-note-0001"><p>
<em>Note</em>: Prop. shared lines denotes the average proportion of lines shared between environments. Subgroup G1 are those datasets that share the same lines (100% sharing) in all environments within a year, whereas in Subgroup G2, the proportion of shared lines is less than 1 (or 100%). For Datasets 1–3 (EYT_1, EYT_2, EYT_3), data are obtained from elite variety trials from CIMMYT from the past wheat breeding program where large trials were established each year under several environments (3–5) under bed planting, irrigated, drought, semi‐drought, late planting, and early planting. Trials in environments within a year combination are the same but different between years. Traits usually collected are days to maturity, days to heading, plant height, and grain yield). Wheat_1 to Wheat_6 are wheat trials from CIMMYT to assess bread wheat quality and it comprises trials across 2 years under irrigated conditions with a varying number of wheat cultivar repeated across years.</p></div></div></section><section id="tpg270049-sec-0050"><h4 class="pmc_sec_title">2.1.1. Plant materials for Elite Yield Trials (EYTs) and wheat</h4>
<p>Data from EYTs conducted by International Maize and Wheat Improvement Center (CIMMYT) are established annually across three to five different environments, encompassing diverse conditions such as varied planting methods, irrigation (IR), drought, semi‐drought, late planting, and early planting. The experimental designs for all trials follow an alpha‐lattice structure with three replicates, and spatial analysis is performed to control for plot‐to‐plot variability in each trial. Traits such as grain yield, heading, lodging, and maturity are typically collected and analyzed. All wheat cultivars included in an EYT (conducted within a single year) are tested across all environments for that year, but none are included in EYTs conducted in subsequent years. EYT includes multiple EYTs, such as EYT_1, EYT_2, and EYT_3 (Juliana et al., <a href="#tpg270049-bib-0014" class="usa-link" aria-describedby="tpg270049-bib-0014">2018</a>).</p>
<p>The datasets Wheat_1 through Wheat_6 represent trials conducted to assess the bread‐making quality of CIMMYT wheat varieties. These trials include some varieties tested under irrigated conditions across different years, with only a subset being repeated between two consecutive years. While the original study by Ibba et al. (<a href="#tpg270049-bib-0012" class="usa-link" aria-describedby="tpg270049-bib-0012">2020</a>) analyzed five datasets, this study reorganized some of the cultivars measured across 2 years, expanding the analysis to six datasets. Each dataset focuses on a single trait, with varying numbers of cultivars tested in 2 years (environments).</p>
<p>The statistical analysis of each EYT and wheat is carried out in two stages. In the first stage, the experimental design is accounted for, and spatial analyses are performed to recover inter‐block information and improve capturing plot‐to‐plot variability, respectively. Best linear unbiased estimates (BLUEs) are calculated for each wheat cultivar during this stage.</p>
<p>In the second stage, the BLUEs are used for further analysis, where transfer learning with RR is applied. Similar procedure is applied to Wheat_1 throughout Wheat_6 trials but only under irrigated environment in several years.</p></section></section><section id="tpg270049-sec-0060"><h3 class="pmc_sec_title">2.2. RR model</h3>
<p>In a general framework, let <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-1" display="inline"><mrow><mrow><msub><mi mathvariant="bold-italic">x</mi><mi>i</mi></msub><mo>=</mo><msup><mrow><mo stretchy="false">(</mo><msub><mi>x</mi><mrow><mi>i</mi><mn>1</mn></mrow></msub><mo>,</mo><mtext>…</mtext><mo>,</mo><msub><mi>x</mi><mi>ip</mi></msub><mo stretchy="false">)</mo></mrow><mi>T</mi></msup></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline"><mrow><mrow><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mtext>…</mtext><mo>,</mo><mi>n</mi></mrow></mrow></math></span>, represents a covariate vector (marker data in the context of our application). Our objective is to use this information to predict or explain the impact of these variables on a continuous response <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline"><mrow><msub><mi>y</mi><mi>i</mi></msub></mrow></math></span> that in our context are traits of phenotypic values. The multiple linear regression model posits a relationship between the covariates and the response is given by the following equation:</p>
<table class="disp-formula p" id="tpg270049-disp-0001"><tr>
<td class="formula"><math id="jats-math-4" display="block"><mrow><mrow><msub><mi>y</mi><mi>i</mi></msub><mo linebreak="badbreak">=</mo><msub><mi mathvariant="normal">β</mi><mn>0</mn></msub><mo linebreak="goodbreak">+</mo><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>p</mi></munderover><msub><mi>x</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><msub><mi mathvariant="normal">β</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi mathvariant="normal">ε</mi><mi>i</mi></msub></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-5" display="inline"><mrow><msub><mi mathvariant="normal">ε</mi><mi>i</mi></msub></mrow></math></span> represents a random error term with mean 0, <em>E</em>(<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><msub><mi mathvariant="normal">ε</mi><mi>i</mi></msub></mrow></math></span>) = 0, and is assumed to be independent of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><msub><mi>x</mi><mi>i</mi></msub></mrow></math></span>. This error term accounts for measurement errors and the influence of unobserved explanatory variables that may also contribute to explaining the response. The conditional mean of the model is given by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><mrow><mi>E</mi><mspace width="0.28em"></mspace><mrow><mo stretchy="false">(</mo><mrow><msub><mi>y</mi><mi>i</mi></msub><mrow><mo stretchy="false">|</mo></mrow><msub><mi>x</mi><mi>i</mi></msub></mrow><mo stretchy="false">)</mo></mrow><mo>=</mo><msub><mi mathvariant="normal">β</mi><mn>0</mn></msub><mo>+</mo><msubsup><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>p</mi></msubsup><msub><mi>x</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><msub><mi mathvariant="normal">β</mi><mi>j</mi></msub></mrow></mrow></math></span>, where the conditional distribution of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><msub><mi>y</mi><mi>i</mi></msub></mrow></math></span>, given <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><msub><mi>x</mi><mi>i</mi></msub></mrow></math></span>, is determined solely by the information contained in <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline"><mrow><msub><mi>x</mi><mi>i</mi></msub></mrow></math></span>.</p>
<p>To estimate the parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-12" display="inline"><mrow><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo>=</mo><msup><mrow><mo stretchy="false">(</mo><msub><mi mathvariant="normal">β</mi><mn>0</mn></msub><mo>,</mo><msub><mi mathvariant="normal">β</mi><mn>1</mn></msub><mo>,</mo><mi>⋯</mi><mo>,</mo><msub><mi mathvariant="normal">β</mi><mi>p</mi></msub><mo stretchy="false">)</mo></mrow><mi>T</mi></msup><mspace width="2.79999pt"></mspace></mrow></mrow></math></span>, we typically rely on a dataset <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline"><mrow><mrow><mo stretchy="false">(</mo><mrow><msubsup><mi>x</mi><mi>i</mi><mi>T</mi></msubsup><mo>,</mo><msub><mi>y</mi><mi>i</mi></msub></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-14" display="inline"><mrow><mrow><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mtext>…</mtext><mo>,</mo><mi>n</mi></mrow></mrow></math></span>, often referred to as the training data. In this dataset, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-15" display="inline"><mrow><msub><mi>x</mi><mi>i</mi></msub></mrow></math></span> represents a vector of feature measurements and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-16" display="inline"><mrow><msub><mi>y</mi><mi>i</mi></msub></mrow></math></span> is the corresponding response for the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-17" display="inline"><mrow><mi>i</mi></mrow></math></span>th individual. When dealing with a large number of features (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-18" display="inline"><mrow><mi>p</mi></mrow></math></span>) and a small sample size (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-19" display="inline"><mrow><mi>n</mi></mrow></math></span>), RR is one of the methods most commonly used to estimate <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-20" display="inline"><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></mrow></math></span>. This method finds the value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-21" display="inline"><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></mrow></math></span> that minimizes the penalized residual sum of squares (Hoerl &amp; Kennard, <a href="#tpg270049-bib-0011" class="usa-link" aria-describedby="tpg270049-bib-0011">1970</a>; Montesinos‐López et al., <a href="#tpg270049-bib-0021" class="usa-link" aria-describedby="tpg270049-bib-0021">2022</a>), defined as follows:</p>
<table class="disp-formula p" id="tpg270049-disp-0002"><tr>
<td class="formula"><math id="jats-math-22" display="block"><mrow><mrow><mi>PRS</mi><msub><mi mathvariant="normal">S</mi><mi mathvariant="normal">λ</mi></msub><mspace width="2.79999pt"></mspace><mfenced open="(" close=")"><mi mathvariant="normal">β</mi></mfenced><mo linebreak="badbreak">=</mo><msup><mfenced separators="" open="(" close=")"><mi mathvariant="bold-italic">y</mi><mspace width="2.79999pt"></mspace><mo>−</mo><mspace width="2.79999pt"></mspace><mi mathvariant="bold-italic">X</mi><mrow><mi mathvariant="bold-italic">β</mi></mrow></mfenced><mi>T</mi></msup><mspace width="2.79999pt"></mspace><mfenced separators="" open="(" close=")"><mi mathvariant="bold-italic">y</mi><mspace width="2.79999pt"></mspace><mo>−</mo><mspace width="2.79999pt"></mspace><mi mathvariant="bold-italic">X</mi><mrow><mi mathvariant="bold-italic">β</mi></mrow></mfenced><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">λ</mi><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>p</mi></munderover><msubsup><mi mathvariant="normal">β</mi><mi>j</mi><mn>2</mn></msubsup></mrow></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p> Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-23" display="inline"><mrow><mrow><mi mathvariant="normal">λ</mi><mspace width="0.28em"></mspace><mo>≥</mo><mspace width="0.28em"></mspace><mn>0</mn><mspace width="0.28em"></mspace></mrow></mrow></math></span> represents the regularization parameter, which controls the extent to which the beta coefficients shrunk toward zero. When <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-24" display="inline"><mrow><mrow><mi mathvariant="normal">λ</mi><mo>=</mo><mn>0</mn></mrow></mrow></math></span>, the solution for the beta coefficients corresponds to the ordinary least squares (OLS) method. However, as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-25" display="inline"><mrow><mi mathvariant="normal">λ</mi></mrow></math></span> increases, the penalized residual sum of squares <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-26" display="inline"><mrow><mrow><mi>PRS</mi><msub><mi mathvariant="normal">S</mi><mi mathvariant="normal">λ</mi></msub><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo stretchy="false">)</mo></mrow><mspace width="2.79999pt"></mspace></mrow></mrow></math></span> becomes dominated by the penalty term, forcing the penalized solution to shrink toward zero (Christensen, <a href="#tpg270049-bib-0005" class="usa-link" aria-describedby="tpg270049-bib-0005">2011</a>). Generally, when the number of parameters exceeds the number of observations, the OLS estimator becomes invalid. A way to address this issue is by constraining the sum of squares of the beta coefficients, thereby regularizing the estimation process (Wakefield, <a href="#tpg270049-bib-0030" class="usa-link" aria-describedby="tpg270049-bib-0030">2013</a>). When the optimization of Equation (<a href="#tpg270049-disp-0002" class="usa-link">2</a>) was carried out using the library glmnet (Friedman et al., <a href="#tpg270049-bib-0008" class="usa-link" aria-describedby="tpg270049-bib-0008">2010</a>), this method was denoted as the standard RR model, whereas when the optimization of Equation (<a href="#tpg270049-disp-0002" class="usa-link">2</a>) was performed manually (writing our own R code) estimating the beta coefficients as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-27" display="inline"><mrow><mrow><mover accent="true"><mi mathvariant="bold-italic">β</mi><mo>^</mo></mover><mo>=</mo><msup><mrow><mo stretchy="false">(</mo><msup><mi mathvariant="bold-italic">X</mi><mi>T</mi></msup><mi mathvariant="bold-italic">X</mi><mo>+</mo><mi mathvariant="normal">λ</mi><mi mathvariant="bold-italic">I</mi><mo stretchy="false">)</mo></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup><mspace width="2.79999pt"></mspace><msup><mi mathvariant="bold-italic">X</mi><mi>T</mi></msup><mi mathvariant="bold-italic">y</mi></mrow></mrow></math></span>, this method was called the analytic RR (ARR) model.</p></section><section id="tpg270049-sec-0070"><h3 class="pmc_sec_title">2.3. Transfer RR methods</h3>
<p>In the context of <em>transfer learning</em> applied to variety trials in environments, the <em>goal environments</em> refer to the specific environmental conditions or sites where improved predictions or insights are desired. For example, if the breeding program focuses on improving performance under <em>drought‐prone environments</em>, these drought‐prone conditions would be the <em>goal</em> environments, whereas data from <em>well‐irrigated or semi‐drought environments</em> might serve as <em>proxy</em> environments for transfer learning. The aim is to predict cultivar performance in the <em>goal</em> environments with higher accuracy, even if direct trials in these environments are sparse or absent.</p>
<p>To implement the Transfer RR, we assume that we have a proxy dataset <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-28" display="inline"><mrow><mrow><mo stretchy="false">(</mo><mrow><msubsup><mi>x</mi><mrow><mi>p</mi><mi>i</mi></mrow><mi>T</mi></msubsup><mo>,</mo><msub><mi>y</mi><mrow><mi>p</mi><mi>i</mi></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-29" display="inline"><mrow><mrow><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mtext>…</mtext><mo>,</mo><msub><mi>n</mi><mi>p</mi></msub></mrow></mrow></math></span>, and a goal dataset <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-30" display="inline"><mrow><mrow><mo stretchy="false">(</mo><mrow><msubsup><mi>x</mi><mrow><mi>g</mi><mi>i</mi></mrow><mi>T</mi></msubsup><mo>,</mo><msub><mi>y</mi><mrow><mi>g</mi><mi>i</mi></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-31" display="inline"><mrow><mrow><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mtext>…</mtext><mo>,</mo><msub><mi>n</mi><mi>g</mi></msub></mrow></mrow></math></span>. In our context the proxy dataset is the information of an environment that is quite related but different to the goal environment. It is assumed that the same markers were measured in both environments, but of course, due to the nature of both environments, the response variables observed in them are different. Regarding the lines measured, in the proxy environment we assume that for all lines (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-32" display="inline"><mrow><mrow><msub><mi>n</mi><mi>p</mi></msub><mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span> markers and phentotypic values are available, whereas for the goal environments, markers are available for all lines (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-33" display="inline"><mrow><mrow><msub><mi>n</mi><mi>g</mi></msub><mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>, but only a portion of the lines contain phenotypic values (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-34" display="inline"><mrow><mrow><msub><mi>n</mi><mi>p</mi></msub><mo>&lt;</mo><msub><mi>n</mi><mi>g</mi></msub></mrow></mrow></math></span>). This means that for the proxy dataset, all individuals contain phenotypic data and markers, whereas for the goal, only a subset of the whole lines contains phenotypic and markers data. The implementation of the transfer learning model involves the following steps:
</p>
<ul id="tpg270049-list-0002" class="list" style="list-style-type:none">
<li><p>Step 1. Using Equation (<a href="#tpg270049-disp-0002" class="usa-link">2</a>) and the complete data of the proxy environments, we compute a <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-35" display="inline"><mrow><msub><mover accent="true"><mi mathvariant="bold-italic">β</mi><mo>^</mo></mover><mi>p</mi></msub></mrow></math></span> that denotes the beta coefficients of the proxy data.</p></li>
<li><p>Step 2. Then, to the response variable (trait of interest with phenotypic values) of the goal data, we subtract the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-36" display="inline"><mrow><mrow><msub><mi>X</mi><mi>g</mi></msub><msub><mover accent="true"><mi mathvariant="bold-italic">β</mi><mo>^</mo></mover><mi>p</mi></msub></mrow></mrow></math></span> and denote this modified response variable as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-37" display="inline"><mrow><mrow><mspace width="2.79999pt"></mspace><msub><mi mathvariant="bold-italic">y</mi><mo>*</mo></msub><mo>=</mo><msub><mi mathvariant="bold-italic">y</mi><mi>g</mi></msub><mspace width="2.79999pt"></mspace><mo>−</mo><msub><mi>X</mi><mi>g</mi></msub><msub><mover accent="true"><mi mathvariant="bold-italic">β</mi><mo>^</mo></mover><mi>p</mi></msub></mrow></mrow></math></span>.</p></li>
<li><div class="p">Step 3. Next, we minimize the following penalized residual sum of squares (Hoerl &amp; Kennard, <a href="#tpg270049-bib-0011" class="usa-link" aria-describedby="tpg270049-bib-0011">1970</a>; Montesinos‐López et al., <a href="#tpg270049-bib-0021" class="usa-link" aria-describedby="tpg270049-bib-0021">2022</a>), with the modified response variable obtained in Step 2 as follows:<table class="disp-formula p" id="tpg270049-disp-0003"><tr>
<td class="formula"><math id="jats-math-38" display="block"><mrow><mrow><mi>PRS</mi><msub><mi mathvariant="normal">S</mi><mi mathvariant="normal">λ</mi></msub><mspace width="2.79999pt"></mspace><mfenced separators="" open="(" close=")"><mrow><mi mathvariant="bold-italic">γ</mi></mrow></mfenced><mo linebreak="badbreak">=</mo><msup><mfenced separators="" open="(" close=")"><msub><mi mathvariant="bold-italic">y</mi><mo>*</mo></msub><mo>−</mo><msub><mi mathvariant="bold-italic">X</mi><mi>g</mi></msub><mrow><mi mathvariant="bold-italic">γ</mi></mrow></mfenced><mi>T</mi></msup><mspace width="2.79999pt"></mspace><mfenced separators="" open="(" close=")"><msub><mi>y</mi><mo>*</mo></msub><mo>−</mo><msub><mi mathvariant="bold-italic">X</mi><mi>g</mi></msub><mrow><mi mathvariant="bold-italic">γ</mi></mrow></mfenced><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">λ</mi><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>p</mi></munderover><msubsup><mi mathvariant="normal">γ</mi><mi>j</mi><mn>2</mn></msubsup></mrow></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
</div></li>
<li><p>Step 4. Next, we compute the transfer learning beta coefficients as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-39" display="inline"><mrow><mrow><msub><mover accent="true"><mi mathvariant="bold-italic">β</mi><mo>^</mo></mover><mi>g</mi></msub><mo>=</mo><msub><mover accent="true"><mi mathvariant="bold-italic">β</mi><mo>^</mo></mover><mi>p</mi></msub><mo>+</mo><mover accent="true"><mi mathvariant="bold-italic">γ</mi><mo>^</mo></mover><mo>.</mo></mrow></mrow></math></span>
</p></li>
<li><div class="p">Step 5. Finally, for a testing set <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-40" display="inline"><mrow><msub><mi mathvariant="bold-italic">X</mi><mi>tst</mi></msub></mrow></math></span> from the goal environment, we make predictions using the transfer learning beta coefficients as follows:<table class="disp-formula p" id="tpg270049-disp-0004"><tr><td class="formula"><math id="jats-math-41" display="block"><mrow><mrow><msub><mi>y</mi><mi mathvariant="bold">tst</mi></msub><mo linebreak="badbreak">=</mo><msub><mi>X</mi><mi>tst</mi></msub><mspace width="0.28em"></mspace><msub><mover accent="true"><mi mathvariant="bold-italic">β</mi><mo>^</mo></mover><mi>g</mi></msub></mrow></mrow></math></td></tr></table>
</div></li>
</ul>
<p>When the optimization of Equation (<a href="#tpg270049-disp-0003" class="usa-link">3</a>) was performed with the library glmnet (Friedman et al., <a href="#tpg270049-bib-0008" class="usa-link" aria-describedby="tpg270049-bib-0008">2010</a>), we called this method Transfer RR, whereas when the optimization of Equation (<a href="#tpg270049-disp-0003" class="usa-link">3</a>) was performed manually (writing our own R code) for estimating the gamma coefficients as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-42" display="inline"><mrow><mrow><mover accent="true"><mi mathvariant="bold-italic">γ</mi><mo>^</mo></mover><mo>=</mo><msup><mrow><mo stretchy="false">(</mo><msubsup><mi mathvariant="bold-italic">X</mi><mi>g</mi><mi>T</mi></msubsup><msub><mi mathvariant="bold-italic">X</mi><mi>g</mi></msub><mo>+</mo><mi mathvariant="normal">λ</mi><mi mathvariant="bold-italic">I</mi><mo stretchy="false">)</mo></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup><mspace width="2.79999pt"></mspace><msubsup><mi mathvariant="bold-italic">X</mi><mi>g</mi><mi>T</mi></msubsup><msub><mi mathvariant="bold-italic">y</mi><mo>*</mo></msub></mrow></mrow></math></span>, this method was named <em>analytic transfer RR (Transfer ARR)</em>. The four methods evaluated were implemented in the R statistical software (R Core Team, <a href="#tpg270049-bib-0025" class="usa-link" aria-describedby="tpg270049-bib-0025">2024</a>).</p></section><section id="tpg270049-sec-0080"><h3 class="pmc_sec_title">2.4. Training the RR, ARR, Transfer RR, and the Transfer ARR models</h3>
<p>The proposed transfer learning methods require information of two environments, one served as the goal (target) environment and the other as the proxy environment. All possible combinations of environment pairs were considered to implement the transfer methods (Transfer RR and Transfer ARR), the results for each dataset reflect the performance across all such combinations. During training, conventional methods and transfer learning methods utilized the same training subset of the goal environment, but the transfer methods were enriched with beta coefficients computed from the entire proxy environment during its training process. In contrast, the RR methods were implemented only using the corresponding training subset of the goal environment without the enrichment of its beta coefficients.</p>
<section id="tpg270049-sec-0090"><h4 class="pmc_sec_title">2.4.1. Cross‐validation strategy</h4>
<p>The study employed a dual cross‐validation approach to evaluate and optimize models.</p>
<section id="tpg270049-sec-0100"><h5 class="pmc_sec_title">Inner cross‐validation (tuning)</h5>
<p>Inner cross‐validation was used within the training data of the outer cross‐validation to optimize the regularization parameter (<em>λ</em>). Each outer training set was further divided into an inner training set (80%) and a validation set (20%) using 10 random partitions. Hyperparameters were selected based on average mean square error (MSE) since this is the default in glmnet library. It is important to point out that for selecting the hyperparameter, only the inner training was used for the training process for selecting the optimal hyperparameter.</p></section><section id="tpg270049-sec-0110"><h5 class="pmc_sec_title">Outer cross‐validation (evaluation)</h5>
<p>Outer cross‐validation assessed the genomic prediction accuracy. A 10‐random partition strategy was used, where each partition allocated 80% of the data for training and 20% for testing. Models were trained on for each partition with 80% of the data and the remaining 20% were used as testing set, and this process was done with each of the 10 partitions. Prediction accuracy was reported as the average Cor and NRMSE across the 10 partitions (Montesinos‐López et al., <a href="#tpg270049-bib-0021" class="usa-link" aria-describedby="tpg270049-bib-0021">2022</a>). In the outer cross‐validation the whole training set is used in the training process after the inner training process was used to select the optimal hyperparameter, and this training process used the optimal hyperparameter estimated in the inner training process.</p>
<p>This approach ensured a clear distinction between tuning hyperparameters (inner cross‐validation) and evaluating model performance on independent data (outer cross‐validation). We reported average Cor and NRMSE as performance metrics in the outer cross‐validation because they are widely used in genomic prediction and because they are not scale dependent, and while other metrics are available, these two were considered sufficient and appropriate for the objectives of this study (Montesinos‐López et al., <a href="#tpg270049-bib-0021" class="usa-link" aria-describedby="tpg270049-bib-0021">2022</a>). However, for inner cross‐validations, we used MSE since this is the default metric for glmnet in which the RR and Transfer RR methods were computed using the observed and predicted values.</p>
<p>Note that the size of the training sets used in all models were identical and belong only to a subset of the goal environment. Consequently, both inner and outer cross‐validation processes were conducted under the same conditions. However, when training the transfer models, an adjusted response variable was used instead of the original response variable. This adjusted variable incorporates the beta coefficients computed using the complete data from the proxy environments. For further details, refer to Steps 1 through 5 of the Transfer RR and Transfer ARR models.</p></section></section></section><section id="tpg270049-sec-0120"><h3 class="pmc_sec_title">2.5. Comparing results of genomic prediction accuracy of RR, ARR, Transfer RR, and Transfer ARR</h3>
<p>To facilitate the comparisons, we computed the average Cor values of the conventional (RR and ARR) models [AvgCor(RR, ARR)] without the transfer methods and compared them with the average Cor values of the transfer methods [AvgCor(Transfer_RR, Transfer_ARR)] via the relative efficiency computed as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-43" display="inline"><mrow><mrow><mo stretchy="false">[</mo><mfrac><mrow><mo stretchy="false">(</mo><mi>AvgCor</mi><mo stretchy="false">{</mo><mi>Transfer</mi><mo>_</mo><mi>RR</mi><mo>,</mo><mi>Transfer</mi><mo>_</mo><mi>ARR</mi><mo stretchy="false">}</mo><mo stretchy="false">)</mo></mrow><mrow><mo stretchy="false">(</mo><mi>AvgCor</mi><mo stretchy="false">{</mo><mi>RR</mi><mo>,</mo><mspace width="2.79999pt"></mspace><mi>ARR</mi><mo stretchy="false">}</mo><mo stretchy="false">)</mo></mrow></mfrac><mo>−</mo><mn>1</mn><mo stretchy="false">]</mo><mo>×</mo><mrow></mrow><mn>100</mn></mrow></mrow></math></span>. Similarly, we compared the average NRMSE values of the conventional (RR and ARR) models [AvgNRMSE(RR, ARR)] with the average NRMSE values of the transfer models [AvgNRMSE(Transfer_RR, Transfer_ARR)] via the computation of the relative efficiency as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-44" display="inline"><mrow><mrow><mo stretchy="false">[</mo><mfrac><mrow><mo stretchy="false">(</mo><mi>AvgNRMSE</mi><mo stretchy="false">{</mo><mi>RR</mi><mo>,</mo><mspace width="2.79999pt"></mspace><mi>ARR</mi><mo stretchy="false">}</mo><mo stretchy="false">)</mo></mrow><mrow><mo stretchy="false">(</mo><mi>AvgNRMSE</mi><mo stretchy="false">{</mo><mi>Transfer</mi><mo>_</mo><mi>RR</mi><mo>,</mo><mspace width="2.79999pt"></mspace><mi>Transfer</mi><mo>_</mo><mi>ARR</mi><mo stretchy="false">}</mo><mo stretchy="false">)</mo></mrow></mfrac><mo>−</mo><mn>1</mn><mo stretchy="false">]</mo><mo>×</mo><mrow></mrow><mn>100</mn></mrow></mrow></math></span>.</p></section></section><section id="tpg270049-sec-0130"><h2 class="pmc_sec_title">3. RESULTS</h2>
<p>We divided the results displayed into five sections which contain the results of <em>EYT_1</em>, <em>EYT_2</em>, <em>Wheat_2</em>, <em>Wheat_6</em>, <em>and across all the datasets</em>, presented in Figures <a href="#tpg270049-fig-0001" class="usa-link">1</a> and <a href="#tpg270049-fig-0002" class="usa-link">2</a> and Tables <a href="#tpg270049-tbl-0002" class="usa-link">2</a>, <a href="#tpg270049-tbl-0003" class="usa-link">3</a>, <a href="#tpg270049-tbl-0004" class="usa-link">4</a>, <a href="#tpg270049-tbl-0005" class="usa-link">5</a>, <a href="#tpg270049-tbl-0006" class="usa-link">6</a>. The results of the remaining datasets (<em>EYT_3</em>, <em>Japonica</em>, <em>Indica</em>, <em>Wheat_1</em>, <em>Wheat_3</em>, <em>Wheat_4</em>, <em>and Wheat_5</em>) are presented in the <a href="#tpg270049-app-0001" class="usa-link">Appendix</a>, which contains Tables <a href="#tpg270049-tbl-0007" class="usa-link">A1</a>, <a href="#tpg270049-tbl-0008" class="usa-link">A2</a>, <a href="#tpg270049-tbl-0009" class="usa-link">A3</a>, <a href="#tpg270049-tbl-0010" class="usa-link">A4</a>, <a href="#tpg270049-tbl-0011" class="usa-link">A5</a>, <a href="#tpg270049-tbl-0012" class="usa-link">A6</a>, <a href="#tpg270049-tbl-0013" class="usa-link">A7</a> and Figures <a href="#tpg270049-fig-0003" class="usa-link">A1</a>, <a href="#tpg270049-fig-0004" class="usa-link">A2</a>, <a href="#tpg270049-fig-0005" class="usa-link">A3</a>, <a href="#tpg270049-fig-0006" class="usa-link">A4</a>, <a href="#tpg270049-fig-0007" class="usa-link">A5</a>, <a href="#tpg270049-fig-0008" class="usa-link">A6</a>, <a href="#tpg270049-fig-0009" class="usa-link">A7</a>. In each section, we compare the performance of the traditional ridge models (RR and ARR) with the performance of the transfer models (Transfer_RR and Transfer_ARR) in terms of two metrics Cor and NRMSE. The results presented in this section were selected as a representative sample of the results of the datasets, the first two datasets (EYT_1 and EYT_2) with good performance, the third (Wheat_2) with regular performance, and the last one (Wheat_6) with the worst performance. For this reason, we provided the across datasets that offer a general view on the performance of the proposed models across all datasets evaluated.</p>
<figure class="fig xbox font-sm" id="tpg270049-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d9c0/12271683/499c1ac4cc5a/TPG2-18-e70049-g006.jpg" loading="lazy" id="jats-graphic-1" height="560" width="700" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270049-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Prediction performance across traits and environments in terms of Pearson's correlation (Cor; A) and normalized root mean square error (NRMSE; B) for the EYT_1, EYT_2, Wheat_2, and Wheat_6 datasets. The models evaluated were ridge regression (RR), analytic ridge regression (ARR), Transfer_RR, and Transfer_ARR.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270049-fig-0002"><h3 class="obj_head">FIGURE 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d9c0/12271683/6b307bc0b5cb/TPG2-18-e70049-g008.jpg" loading="lazy" id="jats-graphic-3" height="564" width="703" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270049-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Prediction performance across traits, environments, and datasets in terms of Pearson's correlation (Cor; A) and normalized root mean square error (NRMSE; B) for each model (ridge regression [RR], analytic ridge regression [ARR], Transfer_RR, and Transfer_ARR). The <em>x</em>‐axis shows the prediction performance for each subgroup. Across_All are the results across the 11 datasets, Across_G1 are the results across datasets EYT_1, EYT_2, EYT_3, and Indica, and Across_G2 are the results across the remaining datasets that are not included in Across_G1.</p></figcaption></figure><section class="tw xbox font-sm" id="tpg270049-tbl-0002"><h3 class="obj_head">TABLE 2.</h3>
<div class="caption p"><p>Prediction performance across traits and environments in terms of normalized root mean square error (NRMSE) and Pearson's correlation (Cor) for the EYT_1 dataset.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Dataset</th>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">NRMSE</th>
<th align="left" rowspan="1" colspan="1">NRMSE_SD</th>
<th align="left" rowspan="1" colspan="1">Cor</th>
<th align="left" rowspan="1" colspan="1">Cor_SD</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_1</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">0.064</td>
<td align="left" rowspan="1" colspan="1">0.025</td>
<td align="left" rowspan="1" colspan="1">0.433</td>
<td align="left" rowspan="1" colspan="1">0.085</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_1</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">0.065</td>
<td align="left" rowspan="1" colspan="1">0.026</td>
<td align="left" rowspan="1" colspan="1">0.425</td>
<td align="left" rowspan="1" colspan="1">0.087</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_1</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">0.060</td>
<td align="left" rowspan="1" colspan="1">0.025</td>
<td align="left" rowspan="1" colspan="1">0.548</td>
<td align="left" rowspan="1" colspan="1">0.108</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_1</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">0.060</td>
<td align="left" rowspan="1" colspan="1">0.026</td>
<td align="left" rowspan="1" colspan="1">0.543</td>
<td align="left" rowspan="1" colspan="1">0.122</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270049-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270049-tbl2-note-0001"><p>
<em>Note</em>: NRMSE_SD denotes the standard deviation of NRMSE and Cor_SD denotes the standard deviation of Cor.</p></div>
<div class="fn" id="tpg270049-tbl2-note-0002"><p>Abbreviations: ARR, analytic ridge regression; RR, ridge regression.</p></div>
</div></section><section class="tw xbox font-sm" id="tpg270049-tbl-0003"><h3 class="obj_head">TABLE 3.</h3>
<div class="caption p"><p>Prediction performance across traits and environments in terms of normalized root mean square error (NRMSE) and Pearson's correlation (Cor) for the EYT_2 dataset.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Dataset</th>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">NRMSE</th>
<th align="left" rowspan="1" colspan="1">NRMSE_SD</th>
<th align="left" rowspan="1" colspan="1">Cor</th>
<th align="left" rowspan="1" colspan="1">Cor_SD</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_2</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">0.052</td>
<td align="left" rowspan="1" colspan="1">0.019</td>
<td align="left" rowspan="1" colspan="1">0.459</td>
<td align="left" rowspan="1" colspan="1">0.073</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_2</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">0.052</td>
<td align="left" rowspan="1" colspan="1">0.020</td>
<td align="left" rowspan="1" colspan="1">0.451</td>
<td align="left" rowspan="1" colspan="1">0.077</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_2</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">0.048</td>
<td align="left" rowspan="1" colspan="1">0.019</td>
<td align="left" rowspan="1" colspan="1">0.579</td>
<td align="left" rowspan="1" colspan="1">0.097</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_2</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">0.052</td>
<td align="left" rowspan="1" colspan="1">0.022</td>
<td align="left" rowspan="1" colspan="1">0.606</td>
<td align="left" rowspan="1" colspan="1">0.168</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270049-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270049-tbl3-note-0001"><p>
<em>Note</em>: NRMSE_SD denotes the standard deviation of NRMSE and Cor_SD denotes the standard deviation of Cor.</p></div>
<div class="fn" id="tpg270049-tbl3-note-0002"><p>Abbreviations: ARR, analytic ridge regression; RR, ridge regression.</p></div>
</div></section><section class="tw xbox font-sm" id="tpg270049-tbl-0004"><h3 class="obj_head">TABLE 4.</h3>
<div class="caption p"><p>Prediction performance across traits and environments in terms of normalized root mean square error (NRMSE) and Pearson's correlation (Cor) for the Wheat_2 dataset.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Dataset</th>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">NRMSE</th>
<th align="left" rowspan="1" colspan="1">NRMSE_SD</th>
<th align="left" rowspan="1" colspan="1">Cor</th>
<th align="left" rowspan="1" colspan="1">Cor_SD</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_2</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">0.048</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">0.248</td>
<td align="left" rowspan="1" colspan="1">0.071</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_2</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">0.049</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">0.149</td>
<td align="left" rowspan="1" colspan="1">0.089</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_2</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">0.047</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">0.346</td>
<td align="left" rowspan="1" colspan="1">0.085</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_2</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">0.047</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">0.254</td>
<td align="left" rowspan="1" colspan="1">0.069</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270049-tbl-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270049-tbl4-note-0001"><p>
<em>Note</em>: NRMSE_SD denotes the standard deviation of NRMSE and Cor_SD denotes the standard deviation of Cor.</p></div>
<div class="fn" id="tpg270049-tbl4-note-0002"><p>Abbreviations: ARR, analytic ridge regression; RR, ridge regression.</p></div>
</div></section><section class="tw xbox font-sm" id="tpg270049-tbl-0005"><h3 class="obj_head">TABLE 5.</h3>
<div class="caption p"><p>Prediction performance across traits and environments in terms of normalized root mean square error (NRMSE) and Pearson's correlation (Cor) for the Wheat_6 dataset.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Dataset</th>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">NRMSE</th>
<th align="left" rowspan="1" colspan="1">NRMSE_SD</th>
<th align="left" rowspan="1" colspan="1">Cor</th>
<th align="left" rowspan="1" colspan="1">Cor_SD</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_6</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">0.048</td>
<td align="left" rowspan="1" colspan="1">0.007</td>
<td align="left" rowspan="1" colspan="1">0.413</td>
<td align="left" rowspan="1" colspan="1">0.144</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_6</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">0.049</td>
<td align="left" rowspan="1" colspan="1">0.007</td>
<td align="left" rowspan="1" colspan="1">0.430</td>
<td align="left" rowspan="1" colspan="1">0.151</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_6</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">0.048</td>
<td align="left" rowspan="1" colspan="1">0.007</td>
<td align="left" rowspan="1" colspan="1">0.411</td>
<td align="left" rowspan="1" colspan="1">0.179</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_6</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">0.048</td>
<td align="left" rowspan="1" colspan="1">0.007</td>
<td align="left" rowspan="1" colspan="1">0.420</td>
<td align="left" rowspan="1" colspan="1">0.160</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270049-tbl-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270049-tbl5-note-0001"><p>
<em>Note</em>: NRMSE_SD denotes the standard deviation of NRMSE and Cor_SD denotes the standard deviation of Cor.</p></div>
<div class="fn" id="tpg270049-tbl5-note-0002"><p>Abbreviations: ARR, analytic ridge regression; RR, ridge regression.</p></div>
</div></section><section class="tw xbox font-sm" id="tpg270049-tbl-0006"><h3 class="obj_head">TABLE 6.</h3>
<div class="caption p"><p>Prediction performance across traits; environments; and across subgroups G1, G2, and across all datasets for the proposed transfer learning methods and conventional ridge regression (RR) approaches in terms of normalized root mean square error (NRMSE) and Pearson's correlation (Cor) across the 11 datasets.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">Subgroup</th>
<th align="left" rowspan="1" colspan="1">NRMSE</th>
<th align="left" rowspan="1" colspan="1">NRMSE_SD</th>
<th align="left" rowspan="1" colspan="1">Cor</th>
<th align="left" rowspan="1" colspan="1">Cor_SD</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">Across_G1</td>
<td align="left" rowspan="1" colspan="1">0.085</td>
<td align="left" rowspan="1" colspan="1">0.065</td>
<td align="left" rowspan="1" colspan="1">0.442</td>
<td align="left" rowspan="1" colspan="1">0.053</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">Across_G2</td>
<td align="left" rowspan="1" colspan="1">0.074</td>
<td align="left" rowspan="1" colspan="1">0.045</td>
<td align="left" rowspan="1" colspan="1">0.418</td>
<td align="left" rowspan="1" colspan="1">0.097</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">Across_All</td>
<td align="left" rowspan="1" colspan="1">0.080</td>
<td align="left" rowspan="1" colspan="1">0.055</td>
<td align="left" rowspan="1" colspan="1">0.430</td>
<td align="left" rowspan="1" colspan="1">0.075</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">Across_G1</td>
<td align="left" rowspan="1" colspan="1">0.082</td>
<td align="left" rowspan="1" colspan="1">0.064</td>
<td align="left" rowspan="1" colspan="1">0.432</td>
<td align="left" rowspan="1" colspan="1">0.048</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">Across_G2</td>
<td align="left" rowspan="1" colspan="1">0.080</td>
<td align="left" rowspan="1" colspan="1">0.052</td>
<td align="left" rowspan="1" colspan="1">0.357</td>
<td align="left" rowspan="1" colspan="1">0.120</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">Across_All</td>
<td align="left" rowspan="1" colspan="1">0.081</td>
<td align="left" rowspan="1" colspan="1">0.058</td>
<td align="left" rowspan="1" colspan="1">0.395</td>
<td align="left" rowspan="1" colspan="1">0.084</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">Across_G1</td>
<td align="left" rowspan="1" colspan="1">0.081</td>
<td align="left" rowspan="1" colspan="1">0.066</td>
<td align="left" rowspan="1" colspan="1">0.546</td>
<td align="left" rowspan="1" colspan="1">0.048</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">Across_G2</td>
<td align="left" rowspan="1" colspan="1">0.072</td>
<td align="left" rowspan="1" colspan="1">0.043</td>
<td align="left" rowspan="1" colspan="1">0.489</td>
<td align="left" rowspan="1" colspan="1">0.092</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">Across_All</td>
<td align="left" rowspan="1" colspan="1">0.076</td>
<td align="left" rowspan="1" colspan="1">0.055</td>
<td align="left" rowspan="1" colspan="1">0.517</td>
<td align="left" rowspan="1" colspan="1">0.070</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">Across_G1</td>
<td align="left" rowspan="1" colspan="1">0.083</td>
<td align="left" rowspan="1" colspan="1">0.061</td>
<td align="left" rowspan="1" colspan="1">0.564</td>
<td align="left" rowspan="1" colspan="1">0.056</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">Across_G2</td>
<td align="left" rowspan="1" colspan="1">0.073</td>
<td align="left" rowspan="1" colspan="1">0.044</td>
<td align="left" rowspan="1" colspan="1">0.433</td>
<td align="left" rowspan="1" colspan="1">0.105</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">Across_All</td>
<td align="left" rowspan="1" colspan="1">0.078</td>
<td align="left" rowspan="1" colspan="1">0.052</td>
<td align="left" rowspan="1" colspan="1">0.498</td>
<td align="left" rowspan="1" colspan="1">0.080</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270049-tbl-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270049-tbl6-note-0001"><p>
<em>Note</em>: NRMSE_SD denotes the standard deviation of NRMSE and Cor_SD denotes the standard deviation of Cor.</p></div>
<div class="fn" id="tpg270049-tbl6-note-0002"><p>Abbreviation: ARR, analytic ridge regression.</p></div>
</div></section><figure class="fig xbox font-sm" id="tpg270049-fig-0003"><h3 class="obj_head">FIGURE A1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d9c0/12271683/f048bacbb39f/TPG2-18-e70049-g009.jpg" loading="lazy" id="jats-graphic-5" height="617" width="640" alt="FIGURE A1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270049-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Box plots of Pearson's correlation (Cor) and normalized root mean square error (NRMSE) for the “<em>EYT_3</em>″ <em>dataset</em> are presented in the top and bottom graphs, respectively. Each blue dot represents a data point per partition and trait. The <em>x</em>‐axis shows each model evaluated (ridge regression [RR], analytic ridge regression [ARR], Transfer_RR, and Transfer_ARR). One box plot is presented for each transfer with the label “Env1‐ Env2” to represent the fact that Env1 is the environment used as a proxy and Env2 is the environment used as goal. (a) Box plot correlation of the transfer from environment Bed2IR to Bed5IR. (b) Box plot correlation of the transfer from environment Bed5IR to Flat5IR. (c) Box plot correlation of the transfer from environment Flat5IR to FlatDrip. (d) Box plot NRMSE of the transfer from environment Bed2IR to Bed5IR. (e) Box plot NRMSE of the transfer from environment Bed5IR to Flat5IR. (f) Box plot NRMSE of the transfer from environment Flat5IR to FlatDrip.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270049-fig-0004"><h3 class="obj_head">FIGURE A2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d9c0/12271683/b12d344be066/TPG2-18-e70049-g003.jpg" loading="lazy" id="jats-graphic-7" height="498" width="400" alt="FIGURE A2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270049-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Box plots of Pearson's correlation (Cor) and normalized root mean square error (NRMSE) for the “<em>Indica</em>” <em>dataset</em> are presented in the top and bottom graphs, respectively. Each blue dot represents a data point per partition and trait. The <em>x</em>‐axis shows each model evaluated (ridge regression [RR], analytic ridge regression [ARR], Transfer_RR, and Transfer_ARR). One box plot is presented for each transfer with the label “Env1‐ Env2” to represent the fact that Env1 is the environment used as a proxy and Env2 is the environment used as goal. (a) Box plot correlation of the transfer from environment 2010 to 2011. (b) Box plot correlation of the transfer from environment 2011 to 2012. (c) Box plot NRMSE of the transfer from environment 2010 to 2011. (d) Box plot NRMSE of the transfer from environment 2011 to 2012.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270049-fig-0005"><h3 class="obj_head">FIGURE A3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d9c0/12271683/7bd18e581bc7/TPG2-18-e70049-g004.jpg" loading="lazy" id="jats-graphic-9" height="719" width="800" alt="FIGURE A3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270049-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Box plots of Pearson's correlation (Cor) and normalized root mean square error (NRMSE) for the “<em>Japonica</em>” <em>dataset</em> are presented in the top and bottom graphs, respectively. Each blue dot represents a data point per partition and trait. The <em>x</em>‐axis shows each model evaluated (ridge regression [RR], analytic ridge regression [ARR], Transfer_RR, and Transfer_ARR). One box plot is presented for each transfer with the label “Env1‐ Env2” to represent the fact that Env1 is the environment used as proxy and Env2 is the environment used as goal. (a) Box plot correlation of the transfer from environment 2009 to 2010. (b) Box plot correlation of the transfer from environment 2010 to 2011. (c) Box plot correlation of the transfer from environment 2011 to 2012. (d) Box plot correlation of the transfer from environment 2012 to 2013. (e) Box plot NRMSE of the transfer from environment 2009 to 2010. (f) Box plot NRMSE of the transfer from environment 2010 to 2011. (g) Box plot NRMSE of the transfer from environment 2011 to 2012. (h) Box plot NRMSE of the transfer from environment 2012 to 2013.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270049-fig-0006"><h3 class="obj_head">FIGURE A4.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d9c0/12271683/1e57c7603e74/TPG2-18-e70049-g001.jpg" loading="lazy" id="jats-graphic-11" height="539" width="400" alt="FIGURE A4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270049-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Box plots of Pearson's correlation (Cor) and normalized root mean square error (NRMSE) for the “<em>Wheat_1</em>” <em>dataset</em> are presented in the top and bottom graphs, respectively. Each blue dot represents a data point per partition and trait. The <em>x</em>‐axis shows each model evaluated (ridge regression [RR], analytic ridge regression [ARR], Transfer_RR, and Transfer_ARR). One box plot is presented for each transfer with the label “Env1‐ Env2” to represent the fact that Env1 is the environment used as a proxy and Env2 is the environment used as goal. (a) Box plot correlation of the transfer from environment YT_13_14 to YT_14_15. (b) Box plot correlation of the transfer from environment YT_14_15 to YT_13_14. (c) Box plot NRMSE of the transfer from environment YT_13_14 to YT_14_15. (d) Box plot NRMSE of the transfer from environment YT_14_15 to YT_13_14.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270049-fig-0007"><h3 class="obj_head">FIGURE A5.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d9c0/12271683/c603333920ae/TPG2-18-e70049-g002.jpg" loading="lazy" id="jats-graphic-13" height="553" width="400" alt="FIGURE A5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270049-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Box plots of Pearson's correlation (Cor) and normalized root mean square error (NRMSE) for the “<em>Wheat_3</em>” <em>dataset</em> are presented in the top and bottom graphs, respectively. Each blue dot represents a data point per partition and trait. The <em>x</em>‐axis shows each model evaluated (ridge regression [RR], analytic ridge regression [ARR], Transfer_RR, and Transfer_ARR). One box plot is presented for each transfer with the label “Env1‐ Env2” to represent the fact that Env1 is the environment used as a proxy and Env2 is the environment used as goal. (a) Box plot correlation of the transfer from environment YT_15_16 to YT_16_17. (b) Box plot correlation of the transfer from environment YT_16_17 to YT_15_16. (c) Box plot NRMSE of the transfer from environment YT_15_16 to YT_16_17. (d) Box plot NRMSE of the transfer from environment YT_16_17 to YT_15_16.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270049-fig-0008"><h3 class="obj_head">FIGURE A6.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d9c0/12271683/c7f86577fb3e/TPG2-18-e70049-g005.jpg" loading="lazy" id="jats-graphic-15" height="524" width="400" alt="FIGURE A6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270049-fig-0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Box plots of Pearson's correlation (Cor) and normalized root mean square error (NRMSE) for the “<em>Wheat_4</em>” dataset are presented in the top and bottom graphs, respectively. Each blue dot represents a data point per partition and trait. The <em>x</em>‐axis shows each model evaluated (ridge regression [RR], analytic ridge regression [ARR], Transfer_RR, and Transfer_ARR). One box plot is presented for each transfer with the label “Env1‐ Env2” to represent the fact that Env1 is the environment used as a proxy and Env2 is the environment used as goal. (a) Box plot correlation of the transfer from environment YT_16_17 to YT_17_18. (b) Box plot correlation of the transfer from environment YT_17_18 to YT_16_17. (c) Box plot NRMSE of the transfer from environment YT_16_17 to YT_17_18. (d) Box plot NRMSE of the transfer from environment YT_17_18 to YT_16_17.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270049-fig-0009"><h3 class="obj_head">FIGURE A7.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d9c0/12271683/41ad1cfe5581/TPG2-18-e70049-g007.jpg" loading="lazy" id="jats-graphic-17" height="602" width="400" alt="FIGURE A7"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270049-fig-0009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Box plots of Pearson's correlation (Cor) and normalized root mean square error (NRMSE) for the “<em>Wheat_5</em>” dataset are presented in the top and bottom graphs, respectively. Each blue dot represents a data point per partition and trait. The <em>x</em>‐axis shows each model evaluated (ridge regression [RR], analytic ridge regression [ARR], Transfer_RR, and Transfer_ARR). One box plot is presented for each transfer with the label “Env1‐ Env2” to represent the fact that Env1 is the environment used as a proxy and Env2 is the environment used as goal. (a) Box plot correlation of the transfer from environment YT_17_18 to YT_18_19. (b) Box plot correlation of the transfer from environment YT_18_19 to YT_17_18. (c) Box plot NRMSE of the transfer from environment YT_17_18 to YT_18_19. (d) Box plot NRMSE of the transfer from environment YT_18_19 to YT_17_18.</p></figcaption></figure><section id="tpg270049-sec-0140"><h3 class="pmc_sec_title">3.1. Dataset EYT_1</h3>
<p>Figure <a href="#tpg270049-fig-0001" class="usa-link">1</a> illustrates the prediction performance across traits and environments for the EYT_1 dataset, evaluated using the Pearson's Cor and the NRMSE. In terms of Cor, the conventional methods achieved an average value of 0.429, which was lower than the average Cor of the transfer methods (Transfer_RR and Transfer_ARR) at 0.546. This represents a 27.2% improvement in prediction accuracy for the transfer methods (Figure <a href="#tpg270049-fig-0001" class="usa-link">1</a>). Detailed results for each method are provided in Table <a href="#tpg270049-tbl-0002" class="usa-link">2</a>.</p>
<p>Regarding NRMSE, the conventional methods reported an average value of 0.064, whereas the transfer methods achieved an average value of 0.060, reflecting a 6.9% improvement (Figure <a href="#tpg270049-fig-0001" class="usa-link">1</a>). Individual results for each method are also detailed in Table <a href="#tpg270049-tbl-0002" class="usa-link">2</a>.</p></section><section id="tpg270049-sec-0150"><h3 class="pmc_sec_title">3.2. Dataset EYT_2</h3>
<p>Figure <a href="#tpg270049-fig-0001" class="usa-link">1</a> depicts the predictive performance across traits and environments for the EYT_2 dataset, assessed using the Pearson's Cor coefficient and the NRMSE.</p>
<p>In terms of Cor, the conventional methods yielded an average value of 0.455, which was inferior to the mean Cor of the transfer methods (Transfer_RR and Transfer_ARR) at 0.593. This signifies a 30.341% enhancement in predictive accuracy for the transfer approaches (Figure <a href="#tpg270049-fig-0001" class="usa-link">1</a>). Comprehensive results for each method are presented in Table <a href="#tpg270049-tbl-0003" class="usa-link">3</a>.</p>
<p>For NRMSE, the methods exhibited an average value of 0.052, whereas the transfer methods achieved a slightly lower average value of 0.050, representing a 4.3% improvement (Figure <a href="#tpg270049-fig-0001" class="usa-link">1</a>).</p></section><section id="tpg270049-sec-0160"><h3 class="pmc_sec_title">3.3. Dataset Wheat_2</h3>
<p>Regarding Cor, the conventional methods produced an average Cor of 0.198, which was lower than the average Cor of the transfer methods (Transfer_RR and Transfer_ARR) at 0.300. This corresponds to a 51.082% increase in predictive accuracy for the transfer approaches (Figure <a href="#tpg270049-fig-0001" class="usa-link">1</a>). Detailed results for each method are provided in Table <a href="#tpg270049-tbl-0004" class="usa-link">4</a>.</p>
<p>For NRMSE, the conventional methods reported an average value of 0.048, whereas the transfer methods achieved a marginally lower value of 0.047, indicating a 2.7% improvement (Figure <a href="#tpg270049-fig-0001" class="usa-link">1</a>). Additional detailed results for each method are available in Table <a href="#tpg270049-tbl-0004" class="usa-link">4</a>.</p></section><section id="tpg270049-sec-0170"><h3 class="pmc_sec_title">3.4. Dataset Wheat_6</h3>
<p>The comparison for this dataset employed a methodology consistent with that applied to previous datasets. In terms of Cor, the conventional methods achieved an average Cor of 0.421, slightly exceeding the average Cor of the transfer methods (Transfer_RR and Transfer_ARR), which was 0.416. This represents a 1.323% decrease in predictive accuracy for the transfer approaches compared to the conventional methods (Figure <a href="#tpg270049-fig-0001" class="usa-link">1</a>). Results for each method are presented in Table <a href="#tpg270049-tbl-0005" class="usa-link">5</a>.</p>
<p>With respect to NRMSE, the conventional methods yielded an average value of 0.049, while the transfer methods reported a slightly lower value of 0.048, reflecting a 1.6% improvement (Figure <a href="#tpg270049-fig-0001" class="usa-link">1</a>). Further detailed outcomes for each method can be found in Table <a href="#tpg270049-tbl-0005" class="usa-link">5</a>.</p></section><section id="tpg270049-sec-0180"><h3 class="pmc_sec_title">3.5. Across data</h3>
<p>Across all datasets presented in Figure <a href="#tpg270049-fig-0002" class="usa-link">2</a>, it is evident that the conventional models (RR and ARR), with an average Cor of 0.413, were outperformed by the transfer models (Transfer_RR and Transfer_ARR), which achieved an average Cor of 0.508, representing a 23.03% improvement (Table <a href="#tpg270049-tbl-0006" class="usa-link">6</a>).</p>
<p>Additionally, as shown in Table <a href="#tpg270049-tbl-0006" class="usa-link">6</a> and Figure <a href="#tpg270049-fig-0002" class="usa-link">2</a>, the conventional models reported an average NRMSE of 0.080, which was higher than the average NRMSE of 0.077 achieved by the transfer models, reflecting a 4.5% improvement. Comprehensive individual results for each method are detailed in Table <a href="#tpg270049-tbl-0006" class="usa-link">6</a>.</p>
<p>Table <a href="#tpg270049-tbl-0006" class="usa-link">6</a> reveals that, in terms of Pearson Cor, subgroup G1, which exhibits the highest degree of relatedness between environments (comprising datasets EYT_1, EYT_2, EYT_3, and Indica), achieved the most substantial improvement, with a 27.00% increase when comparing transfer learning models to conventional methods. In contrast, subgroup G2 demonstrated a more modest gain of 18.960%. Notably, the poorest performance was observed in the Wheat_6 dataset, which has the lowest degree of relatedness among all datasets analyzed.</p>
<p>The results across the datasets demonstrate that the proposed transfer learning methods generally outperform conventional methods. Notably, this pattern of superiority is evident both across datasets and within individual datasets, underscoring the robustness and effectiveness of the proposed method. Despite the difficulty in separating dataset size from the proportion of shared lines—since datasets with more shared lines (G1) were also the smaller datasets, whereas those with fewer shared lines (G2) were the larger datasets—the degree of relatedness between environments emerges as a critical factor. When the relatedness between the proxy and goal environments is negligible, the transfer learning method shows no significant improvement over conventional methods. However, as the relatedness between the proxy and target environments strengthens (genetic similarity, environmental similarity, etc.) the accuracy gains achieved through the transfer learning approaches become increasingly pronounced. These findings highlight the importance of leveraging environmental similarity to maximizing the advantages of transfer learning in genomic predictive modeling.</p></section></section><section id="tpg270049-sec-0190"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>GS has emerged as a transformative predictive methodology in plant breeding, offering the potential to accelerate genetic gains and improve crop traits (Alemu et al., <a href="#tpg270049-bib-0002" class="usa-link" aria-describedby="tpg270049-bib-0002">2024</a>). Despite its promise, its practical application remains challenging due to the complexity of plant breeding experiments, where numerous environmental and genetic factors are difficult to control and can significantly affect the accuracy of GS predictions, making it harder to implement the method reliably across different contexts. To address this issue, researchers have explored a range of approaches aimed at enhancing the prediction accuracy of GS models. These include integrating advanced statistical techniques, incorporating environmental covariates, and leveraging multi‐trait and multi‐environmental data to improve robustness and reliability in real‐world applications (Alemu et al., <a href="#tpg270049-bib-0002" class="usa-link" aria-describedby="tpg270049-bib-0002">2024</a>).</p>
<p>For this reason, our research focuses on exploring the application of transfer learning to enhance the prediction accuracy of the GS methodology. Transfer learning, which involves leveraging knowledge gained from one context to improve performance in another, has shown great promise in fields such as computer vision and natural language processing (Weiss et al., <a href="#tpg270049-bib-0032" class="usa-link" aria-describedby="tpg270049-bib-0032">2016</a>). Applying this approach to GS allows us to take advantage of models trained on well‐characterized datasets to improve predictions in less studied or more challenging environments. By transferring learned patterns from related crops, traits, or environments, transfer learning can help overcome the limitations posed by small or incomplete datasets, as well as variability in environmental conditions. This strategy not only improves model generalization but also opens new opportunities to streamline breeding programs, especially in regions where experimental data are scarce or highly variable.</p>
<p>In this research, our results across the 11 datasets provide strong empirical evidence that the transfer learning approach significantly improves prediction performance. Specifically, in terms of Cor, the prediction accuracy increased by 23.03% (see Table <a href="#tpg270049-tbl-0006" class="usa-link">6</a>) compared to the conventional methods. Similarly, for the NRMSE, the transfer learning methods achieved a gain of 4.5% over the conventional approaches. These findings underscore the effectiveness of transfer learning in enhancing prediction accuracy, particularly by leveraging additional knowledge from related datasets, which helps mitigate the limitations of conventional methods when faced with complex or variable environments.</p>
<p>However, it is crucial to highlight that the variability observed in the results in the 11 different datasets evaluated using the proposed transfer regression methods is partly attributable to the degree of relatedness between the proxy and goal environments. The proposed methods perform optimally when there is a strong Cor between these environments (as observed in those environments classified as group G1); conversely, a lower degree of relatedness leads to poorer prediction performance (as observed in those environments that were classified as group G2). This represents a significant limitation of the proposed methods, as a lack of relatedness can result in negative transfer as was observed in Wheat_6 dataset. Negative transfer learning occurs when knowledge transferred from a proxy environment adversely affects performance in the goal domain, often due to fundamental differences that lead to inaccurate or misleading information being applied. As a result, predictions may be worse than if no transfer had been utilized at all.</p>
<p>Also, it is important to clarify that both the transfer models and conventional models were trained using the same dataset from the goal environment. However, a key distinction lies in the transfer learning process: The transfer models incorporated beta coefficients learned from proxy environments, which were estimated in a separate, independent training process. Thus, while both models technically rely on the same training set from the goal environment, the transfer models benefit from an additional source of information, the precomputed beta coefficients from the proxy environment. This distinction is crucial when evaluating their performance, as the observed advantage of the transfer models depends not only on the training data itself but also on the structural knowledge transferred from the proxy environment.</p>
<p>Based on these promising results, further exploration of transfer learning in GS is warranted. The substantial gains in prediction accuracy, as proven by improvements in both Cor and NRMSE, highlight the potential of this approach to address key limitations in conventional GS methods. Given that plant breeding often involves working with incomplete or limited datasets, particularly in challenging environments or less‐studied crops, transfer learning provides a means to overcome these barriers by leveraging existing knowledge from related datasets. Additionally, the complexity of genotype × environment interactions require more robust models, capable of generalizing across diverse conditions. Transfer learning not only enhances the accuracy of predictions but also holds the promise of making GS more widely applicable and efficient in real‐world breeding programs. By continuing to explore and refine transfer learning methodologies, we can unlock new possibilities for the acceleration of genetic gains, especially in regions with scarce data or highly variable environmental conditions.</p>
<p>Given that transfer learning has shown significant improvements in prediction accuracy across various fields of science and technology (Pan &amp; Yang, <a href="#tpg270049-bib-0024" class="usa-link" aria-describedby="tpg270049-bib-0024">2010</a>), our application of this approach within the context of GS represents a novel contribution. While this research used a transfer learning model based on a penalized RR framework, we recognize that this is only one potential approach. There is substantial room for further exploration of transfer learning using alternative methods, including Bayesian approaches, advanced machine learning algorithms, and deep learning techniques. Each of these methods could offer unique advantages in terms of model flexibility, the ability to capture complex genotype–environment interactions, and improved generalization across different breeding environments.</p>
<section id="tpg270049-sec-0200"><h3 class="pmc_sec_title">4.1. Comparing transfer learning methods with alternative machine learning methods</h3>
<p>While other methods like neural networks or ensemble approaches might capture complex nonlinear relationships, they often require large datasets and intensive tuning, which may not always be feasible in genomic contexts. Transfer RR strikes a balance by being simpler, faster, and effective for linear relationships. Furthermore, RR's ability to handle high‐dimensional genomic data, its computational efficiency, and its effectiveness in preventing overfitting make it an excellent choice for transfer learning in genomic prediction. It also aligns well with the linear structure of many genomic datasets, ensuring both practical applicability and scientific interpretability.</p>
<p>Moreover, beyond the Transfer RR framework, there is potential to explore more sophisticated strategies to enrich goal (target) environment data with proxy environment information. These could include methods that incorporate environmental and other omics data, multi‐trait data, or ensemble techniques, all of which could enhance the power of GS. Such advancements would help address the inherent limitations of current GS methods, particularly in scenarios involving incomplete or sparse data.</p>
<p>The results of this study demonstrate that RR combined transfer learning provides a robust method of genomic prediction and integration. Some previous studies (Dar &amp; Baraniuk, <a href="#tpg270049-bib-0007" class="usa-link" aria-describedby="tpg270049-bib-0007">2022</a>; Chen et al., <a href="#tpg270049-bib-0004" class="usa-link" aria-describedby="tpg270049-bib-0004">2015</a>; H. Zhang &amp; Li, <a href="#tpg270049-bib-0034" class="usa-link" aria-describedby="tpg270049-bib-0034">2023</a>; Tang &amp; de Sa, <a href="#tpg270049-bib-0027" class="usa-link" aria-describedby="tpg270049-bib-0027">2020</a>) demonstrate the effectiveness of RR and similar linear models in transfer learning.</p>
<p>In practical terms, goal (target) environments might lack sufficient direct experimental data, making it essential to use data from other environments (proxy environments) to make informed predictions. These goal environments are critical for evaluating the performance of wheat cultivars because they might align with the target population of environments (TPE) for which new varieties are being developed using a sparse testing scheme where not all the environments in the TPE (as well as the cultivars) were sampled or not all varieties were sampled in all of the environments in the TPE. Goal environments often represent the regions or conditions or TPE where breeders aim to introduce improved varieties to maximize yield, stress tolerance, or other desired traits.</p></section><section id="tpg270049-sec-0210"><h3 class="pmc_sec_title">4.2. Discussion of appendix results</h3>
<p>The results presented in the <a href="#tpg270049-app-0001" class="usa-link">Appendix</a> strongly reinforce the utility of transfer learning in genomic prediction across both crop types and environmental conditions. Notably, in the EYT_3 dataset (Table <a href="#tpg270049-tbl-0007" class="usa-link">A1</a>; Figure <a href="#tpg270049-fig-0003" class="usa-link">A1</a>), which involves similar IR systems (Flat5IR→FlatDrip), Transfer_ARR significantly improved the Cor (from 0.517 to 0.629) and slightly reduced NRMSE (from 0.081 to 0.078). This reflects the high effectiveness of knowledge transfer under similar agronomic management practices, especially within the same breeding cycle.</p>
<p>In Indica rice (Table <a href="#tpg270049-tbl-0008" class="usa-link">A2</a>; Figure <a href="#tpg270049-fig-0004" class="usa-link">A2</a>), a transfer from 2011 to 2012 increased Cor from 0.487 (ARR) to 0.564 (Transfer_ARR) and reduced NRMSE from 0.205 to 0.191. Despite inter‐annual variability, the consistency of genetic evaluation environments enabled successful transfer. Similarly, in Japonica rice (Table <a href="#tpg270049-tbl-0009" class="usa-link">A3</a>; Figure <a href="#tpg270049-fig-0005" class="usa-link">A3</a>), though the Cor gain was more modest (from 0.588 to 0.619), the maintenance of NRMSE at 0.131 suggests that while the signal strength may saturate, transfer learning still stabilizes prediction performance across years.</p>
<p>In the wheat datasets, similar patterns emerge. For Wheat_1 (Table <a href="#tpg270049-tbl-0010" class="usa-link">A4</a>; Figure <a href="#tpg270049-fig-0006" class="usa-link">A4</a>), Transfer_ARR yielded a substantial Cor increase (from 0.518 to 0.614) and NRMSE reduction (from 0.068 to 0.063), affirming transfer learning's benefits in irrigated wheat systems. Wheat_3 (Table <a href="#tpg270049-tbl-0011" class="usa-link">A5</a>; Figure <a href="#tpg270049-fig-0007" class="usa-link">A5</a>) showed moderate yet consistent improvement (Cor from 0.415 to 0.458; NRMSE from 0.070 to 0.068), even under potentially less correlated environmental conditions. In Wheat_4 (Table <a href="#tpg270049-tbl-0012" class="usa-link">A6</a>; Figure <a href="#tpg270049-fig-0008" class="usa-link">A6</a>), the Cor increase from 0.412 to 0.480 with a corresponding NRMSE drop (0.044 to 0.043) supports the idea that transfer learning remains advantageous even when environments are only moderately related.</p>
<p>The most striking result came from Wheat_5 (Table <a href="#tpg270049-tbl-0013" class="usa-link">A7</a>; Figure <a href="#tpg270049-fig-0009" class="usa-link">A7</a>), where Transfer_ARR outperformed ARR with a Cor gain of 0.135 and an NRMSE reduction from 0.059 to 0.055. This scenario likely represents an optimal case where the proxy and target environments are strongly aligned in terms of genetic, agronomic, and climatic conditions, allowing Transfer_ARR to capture consistent genotype performance patterns more effectively.</p>
<p>Overall, these findings support several conclusions:
</p>
<ol id="tpg270049-list-0003" class="list" style="list-style-type:decimal">
<li><p>Transfer learning offers consistent improvement across crops (wheat, Indica, Japonica rice) and years.</p></li>
<li><p>Gains in predictive accuracy were observed not only for yield but also for other traits like protein content.</p></li>
<li><p>The Transfer_ARR method generally outperformed Transfer_RR, likely due to improved numerical stability, especially when data dimensionality is high or when marker matrices have collinearity.</p></li>
</ol>
<p>These results align with recent studies, demonstrating the benefit of model reuse or adaptation across time, location, or management systems in GS. Importantly, the variability in improvement magnitude across datasets also reflects the importance of environmental relatedness, suggesting that transfer learning works best when the proxy and target conditions share similar environmental signal profiles. This study contributes empirical support for including transfer learning as a regular practice in GS pipelines, particularly when phenotypic data collection is resource limited.</p></section></section><section id="tpg270049-sec-0220"><h2 class="pmc_sec_title">5. CONCLUSIONS</h2>
<p>In this research, we applied transfer learning within the context of GS. Specifically, we transferred knowledge learned from a proxy environment to improve predictions in a target (goal) environment when limited information exist. This transfer process involved learning the beta coefficients from the proxy environment using a RR framework, which were then used to enrich the incomplete information in the goal environment. Our results prove that transfer learning has a significant potential to enhance the prediction accuracy of the GS methodology in goal environments. Across the 11 datasets evaluated, we observed a 22.962% improvement in Pearson's Cor and a 5.757% improvement in NRMSE in comparison to traditional methods. Despite these promising results, we advocate for further research to fully explore the potential of transfer learning in plant breeding, as it holds great promise in terms of addressing the challenges posed by limited or incomplete data in GS applications.</p></section><section id="tpg270049-sec-0230"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Osval A. Montesinos‐López</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; writing—original draft; writing—review and editing. <strong>Eduardo A. Barajas‐Ramírez</strong>: Validation; writing—original draft; writing—review and editing. <strong>Josafhat Salinas‐Ruiz</strong>: Data curation; validation; writing—original draft; writing—review and editing. <strong>Abelardo Montesinos‐López</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; software; writing—original draft; writing—review and editing. <strong>Guillermo Gerard</strong>: Methodology; writing—original draft; writing—review and editing. <strong>Paolo Vitale</strong>: Conceptualization; investigation; writing—original draft; writing—review and editing. <strong>Susanne Dreisigacker</strong>: Investigation; methodology; writing—original draft; writing—review and editing. <strong>Carolina Saint Pierre</strong>: Conceptualization; funding acquisition; writing—original draft; writing—review and editing. <strong>José Crossa</strong>: Investigation; methodology; writing—original draft; writing—review and editing.</p></section><section id="tpg270049-sec-0250"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="tpg270049-sec-0290"><h2>6.</h2></section><section id="tpg270049-sec-0240" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>Open Access fees were received from the Bill &amp; Melinda Gates Foundation. We acknowledge the financial support provided by the Bill &amp; Melinda Gates Foundation (INV‐003439 BMGF/FCDO Accelerating Genetic Gains in Maize and Wheat for Improved Livelihoods [AGG]) as well as the USAID projects (Amend. No. 9 MTO 069033, USAID‐CIMMYT Wheat/AGGMW, Genes 2023, 14, 927 14 of 18 AGG‐Maize Supplementary Project, AGG [Stress Tolerant Maize for Africa]), which generated the CIMMYT data analyzed in this study. We are also thankful for the financial support provided by the Foundation for Research Levy on Agricultural Products (FFL) and the Agricultural Agreement Research Fund (JA) through the Research Council of Norway for grants 301835 (Sustainable Management of Rust Diseases in Wheat) and 320090 (Phenotyping for Healthier and more Productive Wheat Crops). We acknowledge the support of the Window 1 and 2 funders to the Accelerated Breeding Initiative (ABI).</p></section><section id="tpg270049-app-0001" class="app"><section id="tpg270049-sec-0280"><h3>1.</h3>
<section class="tw xbox font-sm" id="tpg270049-tbl-0007"><h4 class="obj_head">TABLE A1.</h4>
<div class="caption p"><p>Prediction performance in terms of normalized root mean square error (NRMSE) and Pearson's correlation (Cor) for the EYT_3 dataset.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Dataset</th>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">Transfer</th>
<th align="left" rowspan="1" colspan="1">NRMSE</th>
<th align="left" rowspan="1" colspan="1">NRMSE_SD</th>
<th align="left" rowspan="1" colspan="1">Cor</th>
<th align="left" rowspan="1" colspan="1">Cor_SD</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_3</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">Bed2IR‐Bed5IR</td>
<td align="left" rowspan="1" colspan="1">0.038</td>
<td align="left" rowspan="1" colspan="1">0.011</td>
<td align="left" rowspan="1" colspan="1">0.424</td>
<td align="left" rowspan="1" colspan="1">0.089</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_3</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">Bed2IR‐Bed5IR</td>
<td align="left" rowspan="1" colspan="1">0.037</td>
<td align="left" rowspan="1" colspan="1">0.010</td>
<td align="left" rowspan="1" colspan="1">0.452</td>
<td align="left" rowspan="1" colspan="1">0.063</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_3</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">Bed2IR‐Bed5IR</td>
<td align="left" rowspan="1" colspan="1">0.047</td>
<td align="left" rowspan="1" colspan="1">0.027</td>
<td align="left" rowspan="1" colspan="1">0.563</td>
<td align="left" rowspan="1" colspan="1">0.235</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_3</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">Bed2IR‐Bed5IR</td>
<td align="left" rowspan="1" colspan="1">0.035</td>
<td align="left" rowspan="1" colspan="1">0.011</td>
<td align="left" rowspan="1" colspan="1">0.557</td>
<td align="left" rowspan="1" colspan="1">0.103</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_3</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">Bed5IR‐Flat5IR</td>
<td align="left" rowspan="1" colspan="1">0.045</td>
<td align="left" rowspan="1" colspan="1">0.018</td>
<td align="left" rowspan="1" colspan="1">0.391</td>
<td align="left" rowspan="1" colspan="1">0.057</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_3</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">Bed5IR‐Flat5IR</td>
<td align="left" rowspan="1" colspan="1">0.045</td>
<td align="left" rowspan="1" colspan="1">0.017</td>
<td align="left" rowspan="1" colspan="1">0.398</td>
<td align="left" rowspan="1" colspan="1">0.058</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_3</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">Bed5IR‐Flat5IR</td>
<td align="left" rowspan="1" colspan="1">0.044</td>
<td align="left" rowspan="1" colspan="1">0.021</td>
<td align="left" rowspan="1" colspan="1">0.564</td>
<td align="left" rowspan="1" colspan="1">0.161</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_3</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">Bed5IR‐Flat5IR</td>
<td align="left" rowspan="1" colspan="1">0.042</td>
<td align="left" rowspan="1" colspan="1">0.018</td>
<td align="left" rowspan="1" colspan="1">0.52</td>
<td align="left" rowspan="1" colspan="1">0.111</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_3</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">Flat5IR‐FlatDrip</td>
<td align="left" rowspan="1" colspan="1">0.084</td>
<td align="left" rowspan="1" colspan="1">0.056</td>
<td align="left" rowspan="1" colspan="1">0.497</td>
<td align="left" rowspan="1" colspan="1">0.062</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_3</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">Flat5IR‐FlatDrip</td>
<td align="left" rowspan="1" colspan="1">0.081</td>
<td align="left" rowspan="1" colspan="1">0.052</td>
<td align="left" rowspan="1" colspan="1">0.517</td>
<td align="left" rowspan="1" colspan="1">0.060</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_3</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">Flat5IR‐FlatDrip</td>
<td align="left" rowspan="1" colspan="1">0.078</td>
<td align="left" rowspan="1" colspan="1">0.059</td>
<td align="left" rowspan="1" colspan="1">0.629</td>
<td align="left" rowspan="1" colspan="1">0.121</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EYT_3</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">Bed2IR‐Bed5IR</td>
<td align="left" rowspan="1" colspan="1">0.038</td>
<td align="left" rowspan="1" colspan="1">0.011</td>
<td align="left" rowspan="1" colspan="1">0.424</td>
<td align="left" rowspan="1" colspan="1">0.089</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270049-tbl-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270049-tbl7-note-0001"><p>
<em>Note</em>: NRMSE_SD denotes the standard deviation of NRMSE and Cor_SD denotes the standard deviation of Cor. In the transfer column, the notation “Env1‐Env2” denotes that Env1 is the environment used as a proxy and Env2 is the environment used as goal.</p></div>
<div class="fn" id="tpg270049-tbl7-note-0002"><p>Abbreviations: ARR, analytic ridge regression; RR, ridge regression.</p></div>
</div></section><section class="tw xbox font-sm" id="tpg270049-tbl-0008"><h4 class="obj_head">TABLE A2.</h4>
<div class="caption p"><p>Prediction performance in terms of normalized root mean square error (NRMSE) and Pearson's correlation (Cor) for the Indica dataset.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Dataset</th>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">Transfer</th>
<th align="left" rowspan="1" colspan="1">NRMSE</th>
<th align="left" rowspan="1" colspan="1">NRMSE_SD</th>
<th align="left" rowspan="1" colspan="1">Cor</th>
<th align="left" rowspan="1" colspan="1">Cor_SD</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Indica</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">2010–2011</td>
<td align="left" rowspan="1" colspan="1">0.223</td>
<td align="left" rowspan="1" colspan="1">0.297</td>
<td align="left" rowspan="1" colspan="1">0.335</td>
<td align="left" rowspan="1" colspan="1">0.175</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Indica</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">2010–2011</td>
<td align="left" rowspan="1" colspan="1">0.223</td>
<td align="left" rowspan="1" colspan="1">0.298</td>
<td align="left" rowspan="1" colspan="1">0.336</td>
<td align="left" rowspan="1" colspan="1">0.174</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Indica</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">2010–2011</td>
<td align="left" rowspan="1" colspan="1">0.217</td>
<td align="left" rowspan="1" colspan="1">0.292</td>
<td align="left" rowspan="1" colspan="1">0.431</td>
<td align="left" rowspan="1" colspan="1">0.129</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Indica</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">2010–2011</td>
<td align="left" rowspan="1" colspan="1">0.217</td>
<td align="left" rowspan="1" colspan="1">0.290</td>
<td align="left" rowspan="1" colspan="1">0.432</td>
<td align="left" rowspan="1" colspan="1">0.125</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Indica</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">2011–2012</td>
<td align="left" rowspan="1" colspan="1">0.206</td>
<td align="left" rowspan="1" colspan="1">0.261</td>
<td align="left" rowspan="1" colspan="1">0.490</td>
<td align="left" rowspan="1" colspan="1">0.231</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Indica</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">2011–2012</td>
<td align="left" rowspan="1" colspan="1">0.205</td>
<td align="left" rowspan="1" colspan="1">0.260</td>
<td align="left" rowspan="1" colspan="1">0.487</td>
<td align="left" rowspan="1" colspan="1">0.233</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Indica</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">2011–2012</td>
<td align="left" rowspan="1" colspan="1">0.191</td>
<td align="left" rowspan="1" colspan="1">0.234</td>
<td align="left" rowspan="1" colspan="1">0.567</td>
<td align="left" rowspan="1" colspan="1">0.178</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Indica</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">2011–2012</td>
<td align="left" rowspan="1" colspan="1">0.191</td>
<td align="left" rowspan="1" colspan="1">0.234</td>
<td align="left" rowspan="1" colspan="1">0.564</td>
<td align="left" rowspan="1" colspan="1">0.181</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270049-tbl-0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270049-tbl8-note-0001"><p>
<em>Note</em>: NRMSE_SD denotes the standard deviation of NRMSE and Cor_SD denotes the standard deviation of Cor. In the transfer column, the notation “Env1‐Env2” denotes that Env1 is the environment used as a proxy and Env2 is the environment used as goal.</p></div>
<div class="fn" id="tpg270049-tbl8-note-0002"><p>Abbreviations: ARR, analytic ridge regression; RR, ridge regression.</p></div>
</div></section><section class="tw xbox font-sm" id="tpg270049-tbl-0009"><h4 class="obj_head">TABLE A3.</h4>
<div class="caption p"><p>Prediction performance in terms of normalized root mean square error (NRMSE) and Pearson's correlation (Cor) for the Japonica dataset.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Dataset</th>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">Transfer</th>
<th align="left" rowspan="1" colspan="1">NRMSE</th>
<th align="left" rowspan="1" colspan="1">NRMSE_SD</th>
<th align="left" rowspan="1" colspan="1">Cor</th>
<th align="left" rowspan="1" colspan="1">Cor_SD</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">2009–2010</td>
<td align="left" rowspan="1" colspan="1">0.205</td>
<td align="left" rowspan="1" colspan="1">0.262</td>
<td align="left" rowspan="1" colspan="1">0.455</td>
<td align="left" rowspan="1" colspan="1">0.255</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">2009–2010</td>
<td align="left" rowspan="1" colspan="1">0.182</td>
<td align="left" rowspan="1" colspan="1">0.225</td>
<td align="left" rowspan="1" colspan="1">0.463</td>
<td align="left" rowspan="1" colspan="1">0.258</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">2009–2010</td>
<td align="left" rowspan="1" colspan="1">0.178</td>
<td align="left" rowspan="1" colspan="1">0.225</td>
<td align="left" rowspan="1" colspan="1">0.510</td>
<td align="left" rowspan="1" colspan="1">0.288</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">2009–2010</td>
<td align="left" rowspan="1" colspan="1">0.171</td>
<td align="left" rowspan="1" colspan="1">0.216</td>
<td align="left" rowspan="1" colspan="1">0.544</td>
<td align="left" rowspan="1" colspan="1">0.294</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">2010–2011</td>
<td align="left" rowspan="1" colspan="1">0.185</td>
<td align="left" rowspan="1" colspan="1">0.212</td>
<td align="left" rowspan="1" colspan="1">0.504</td>
<td align="left" rowspan="1" colspan="1">0.084</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">2010–2011</td>
<td align="left" rowspan="1" colspan="1">0.155</td>
<td align="left" rowspan="1" colspan="1">0.163</td>
<td align="left" rowspan="1" colspan="1">0.535</td>
<td align="left" rowspan="1" colspan="1">0.100</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">2010–2011</td>
<td align="left" rowspan="1" colspan="1">0.153</td>
<td align="left" rowspan="1" colspan="1">0.164</td>
<td align="left" rowspan="1" colspan="1">0.569</td>
<td align="left" rowspan="1" colspan="1">0.118</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">2010–2011</td>
<td align="left" rowspan="1" colspan="1">0.151</td>
<td align="left" rowspan="1" colspan="1">0.161</td>
<td align="left" rowspan="1" colspan="1">0.590</td>
<td align="left" rowspan="1" colspan="1">0.119</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">2011–2012</td>
<td align="left" rowspan="1" colspan="1">0.104</td>
<td align="left" rowspan="1" colspan="1">0.081</td>
<td align="left" rowspan="1" colspan="1">0.308</td>
<td align="left" rowspan="1" colspan="1">0.177</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">2011–2012</td>
<td align="left" rowspan="1" colspan="1">0.105</td>
<td align="left" rowspan="1" colspan="1">0.082</td>
<td align="left" rowspan="1" colspan="1">0.330</td>
<td align="left" rowspan="1" colspan="1">0.155</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">2011–2012</td>
<td align="left" rowspan="1" colspan="1">0.104</td>
<td align="left" rowspan="1" colspan="1">0.084</td>
<td align="left" rowspan="1" colspan="1">0.364</td>
<td align="left" rowspan="1" colspan="1">0.197</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">2011–2012</td>
<td align="left" rowspan="1" colspan="1">0.107</td>
<td align="left" rowspan="1" colspan="1">0.091</td>
<td align="left" rowspan="1" colspan="1">0.427</td>
<td align="left" rowspan="1" colspan="1">0.191</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">2012–2013</td>
<td align="left" rowspan="1" colspan="1">0.141</td>
<td align="left" rowspan="1" colspan="1">0.161</td>
<td align="left" rowspan="1" colspan="1">0.577</td>
<td align="left" rowspan="1" colspan="1">0.116</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">2012–2013</td>
<td align="left" rowspan="1" colspan="1">0.131</td>
<td align="left" rowspan="1" colspan="1">0.145</td>
<td align="left" rowspan="1" colspan="1">0.588</td>
<td align="left" rowspan="1" colspan="1">0.121</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">2012–2013</td>
<td align="left" rowspan="1" colspan="1">0.127</td>
<td align="left" rowspan="1" colspan="1">0.147</td>
<td align="left" rowspan="1" colspan="1">0.632</td>
<td align="left" rowspan="1" colspan="1">0.137</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Japonica</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">2012–2013</td>
<td align="left" rowspan="1" colspan="1">0.131</td>
<td align="left" rowspan="1" colspan="1">0.161</td>
<td align="left" rowspan="1" colspan="1">0.619</td>
<td align="left" rowspan="1" colspan="1">0.151</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270049-tbl-0009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270049-tbl9-note-0001"><p>
<em>Note</em>: NRMSE_SD denotes the standard deviation of NRMSE and Cor_SD denotes the standard deviation of Cor. In the transfer column, the notation “Env1‐Env2” denotes that Env1 is the environment used as a proxy and Env2 is the environment used as goal.</p></div>
<div class="fn" id="tpg270049-tbl9-note-0002"><p>Abbreviations: ARR, analytic ridge regression; RR, ridge regression.</p></div>
</div></section><section class="tw xbox font-sm" id="tpg270049-tbl-0010"><h4 class="obj_head">TABLE A4.</h4>
<div class="caption p"><p>Prediction performance in terms of normalized root mean square error (NRMSE) and Pearson's correlation (Cor) for the Wheat_1 dataset.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Dataset</th>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">Transfer</th>
<th align="left" rowspan="1" colspan="1">NRMSE</th>
<th align="left" rowspan="1" colspan="1">NRMSE_SD</th>
<th align="left" rowspan="1" colspan="1">Cor</th>
<th align="left" rowspan="1" colspan="1">Cor_SD</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_1</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">YT_13_14‐YT_14_15</td>
<td align="left" rowspan="1" colspan="1">0.076</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">0.414</td>
<td align="left" rowspan="1" colspan="1">0.047</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_1</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">YT_13_14‐YT_14_15</td>
<td align="left" rowspan="1" colspan="1">0.068</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">0.518</td>
<td align="left" rowspan="1" colspan="1">0.050</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_1</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">YT_13_14‐YT_14_15</td>
<td align="left" rowspan="1" colspan="1">0.068</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">0.523</td>
<td align="left" rowspan="1" colspan="1">0.047</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_1</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">YT_13_14‐YT_14_15</td>
<td align="left" rowspan="1" colspan="1">0.063</td>
<td align="left" rowspan="1" colspan="1">0.003</td>
<td align="left" rowspan="1" colspan="1">0.614</td>
<td align="left" rowspan="1" colspan="1">0.028</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_1</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">YT_14_15‐YT_13_14</td>
<td align="left" rowspan="1" colspan="1">0.058</td>
<td align="left" rowspan="1" colspan="1">0.003</td>
<td align="left" rowspan="1" colspan="1">0.384</td>
<td align="left" rowspan="1" colspan="1">0.090</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_1</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">YT_14_15‐YT_13_14</td>
<td align="left" rowspan="1" colspan="1">0.052</td>
<td align="left" rowspan="1" colspan="1">0.003</td>
<td align="left" rowspan="1" colspan="1">0.486</td>
<td align="left" rowspan="1" colspan="1">0.099</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_1</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">YT_14_15‐YT_13_14</td>
<td align="left" rowspan="1" colspan="1">0.052</td>
<td align="left" rowspan="1" colspan="1">0.003</td>
<td align="left" rowspan="1" colspan="1">0.495</td>
<td align="left" rowspan="1" colspan="1">0.098</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_1</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">YT_14_15‐YT_13_14</td>
<td align="left" rowspan="1" colspan="1">0.049</td>
<td align="left" rowspan="1" colspan="1">0.003</td>
<td align="left" rowspan="1" colspan="1">0.575</td>
<td align="left" rowspan="1" colspan="1">0.074</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270049-tbl-0010/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270049-tbl10-note-0001"><p>
<em>Note</em>: NRMSE_SD denotes the standard deviation of NRMSE and Cor_SD denotes the standard deviation of Cor. In the transfer column, the notation “Env1‐Env2” denotes that Env1 is the environment used as a proxy and Env2 is the environment used as goal.</p></div>
<div class="fn" id="tpg270049-tbl10-note-0002"><p>Abbreviations: ARR, analytic ridge regression; RR, ridge regression.</p></div>
</div></section><section class="tw xbox font-sm" id="tpg270049-tbl-0011"><h4 class="obj_head">TABLE A5.</h4>
<div class="caption p"><p>Prediction performance in terms of normalized root mean square error (NRMSE) and Pearson's correlation (Cor) for the Wheat_3 dataset.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Dataset</th>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">Transfer</th>
<th align="left" rowspan="1" colspan="1">NRMSE</th>
<th align="left" rowspan="1" colspan="1">NRMSE_SD</th>
<th align="left" rowspan="1" colspan="1">Cor</th>
<th align="left" rowspan="1" colspan="1">Cor_SD</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_3</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">YT_15_16‐YT_16_17</td>
<td align="left" rowspan="1" colspan="1">0.074</td>
<td align="left" rowspan="1" colspan="1">0.005</td>
<td align="left" rowspan="1" colspan="1">0.317</td>
<td align="left" rowspan="1" colspan="1">0.080</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_3</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">YT_15_16‐YT_16_17</td>
<td align="left" rowspan="1" colspan="1">0.070</td>
<td align="left" rowspan="1" colspan="1">0.005</td>
<td align="left" rowspan="1" colspan="1">0.415</td>
<td align="left" rowspan="1" colspan="1">0.079</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_3</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">YT_15_16‐YT_16_17</td>
<td align="left" rowspan="1" colspan="1">0.070</td>
<td align="left" rowspan="1" colspan="1">0.005</td>
<td align="left" rowspan="1" colspan="1">0.416</td>
<td align="left" rowspan="1" colspan="1">0.079</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_3</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">YT_15_16‐YT_16_17</td>
<td align="left" rowspan="1" colspan="1">0.068</td>
<td align="left" rowspan="1" colspan="1">0.005</td>
<td align="left" rowspan="1" colspan="1">0.458</td>
<td align="left" rowspan="1" colspan="1">0.075</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_3</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">YT_16_17‐YT_15_16</td>
<td align="left" rowspan="1" colspan="1">0.038</td>
<td align="left" rowspan="1" colspan="1">0.002</td>
<td align="left" rowspan="1" colspan="1">0.372</td>
<td align="left" rowspan="1" colspan="1">0.083</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_3</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">YT_16_17‐YT_15_16</td>
<td align="left" rowspan="1" colspan="1">0.035</td>
<td align="left" rowspan="1" colspan="1">0.002</td>
<td align="left" rowspan="1" colspan="1">0.424</td>
<td align="left" rowspan="1" colspan="1">0.092</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_3</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">YT_16_17‐YT_15_16</td>
<td align="left" rowspan="1" colspan="1">0.035</td>
<td align="left" rowspan="1" colspan="1">0.002</td>
<td align="left" rowspan="1" colspan="1">0.438</td>
<td align="left" rowspan="1" colspan="1">0.087</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_3</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">YT_16_17‐YT_15_16</td>
<td align="left" rowspan="1" colspan="1">0.036</td>
<td align="left" rowspan="1" colspan="1">0.002</td>
<td align="left" rowspan="1" colspan="1">0.487</td>
<td align="left" rowspan="1" colspan="1">0.053</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270049-tbl-0011/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270049-tbl11-note-0001"><p>
<em>Note</em>: NRMSE_SD denotes the standard deviation of NRMSE and Cor_SD denotes the standard deviation of Cor. In the transfer column, the notation “Env1‐Env2” denotes that Env1 is the environment used as a proxy and Env2 is the environment used as goal.</p></div>
<div class="fn" id="tpg270049-tbl11-note-0002"><p>Abbreviations: ARR, analytic ridge regression; RR, ridge regression.</p></div>
</div></section><section class="tw xbox font-sm" id="tpg270049-tbl-0012"><h4 class="obj_head">TABLE A6.</h4>
<div class="caption p"><p>Prediction performance in terms of normalized root mean square error (NRMSE) and Pearson's correlation (Cor) for the Wheat_4 dataset.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Dataset</th>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">Transfer</th>
<th align="left" rowspan="1" colspan="1">NRMSE</th>
<th align="left" rowspan="1" colspan="1">NRMSE_SD</th>
<th align="left" rowspan="1" colspan="1">Cor</th>
<th align="left" rowspan="1" colspan="1">Cor_SD</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_4</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">YT_16_17‐YT_17_18</td>
<td align="left" rowspan="1" colspan="1">0.055</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">0.196</td>
<td align="left" rowspan="1" colspan="1">0.074</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_4</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">YT_16_17‐YT_17_18</td>
<td align="left" rowspan="1" colspan="1">0.052</td>
<td align="left" rowspan="1" colspan="1">0.003</td>
<td align="left" rowspan="1" colspan="1">0.345</td>
<td align="left" rowspan="1" colspan="1">0.056</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_4</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">YT_16_17‐YT_17_18</td>
<td align="left" rowspan="1" colspan="1">0.052</td>
<td align="left" rowspan="1" colspan="1">0.003</td>
<td align="left" rowspan="1" colspan="1">0.348</td>
<td align="left" rowspan="1" colspan="1">0.054</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_4</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">YT_16_17‐YT_17_18</td>
<td align="left" rowspan="1" colspan="1">0.050</td>
<td align="left" rowspan="1" colspan="1">0.003</td>
<td align="left" rowspan="1" colspan="1">0.419</td>
<td align="left" rowspan="1" colspan="1">0.061</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_4</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">YT_17_18‐YT_16_17</td>
<td align="left" rowspan="1" colspan="1">0.048</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">0.334</td>
<td align="left" rowspan="1" colspan="1">0.099</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_4</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">YT_17_18‐YT_16_17</td>
<td align="left" rowspan="1" colspan="1">0.044</td>
<td align="left" rowspan="1" colspan="1">0.003</td>
<td align="left" rowspan="1" colspan="1">0.412</td>
<td align="left" rowspan="1" colspan="1">0.104</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_4</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">YT_17_18‐YT_16_17</td>
<td align="left" rowspan="1" colspan="1">0.044</td>
<td align="left" rowspan="1" colspan="1">0.003</td>
<td align="left" rowspan="1" colspan="1">0.417</td>
<td align="left" rowspan="1" colspan="1">0.102</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_4</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">YT_17_18‐YT_16_17</td>
<td align="left" rowspan="1" colspan="1">0.043</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">0.480</td>
<td align="left" rowspan="1" colspan="1">0.087</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270049-tbl-0012/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270049-tbl12-note-0001"><p>
<em>Note</em>: NRMSE_SD denotes the standard deviation of NRMSE and Cor_SD denotes the standard deviation of Cor. In the transfer column, the notation “Env1‐Env2” denotes that Env1 is the environment used as a proxy and Env2 is the environment used as goal.</p></div>
<div class="fn" id="tpg270049-tbl12-note-0002"><p>Abbreviations: ARR, analytic ridge regression; RR, ridge regression.</p></div>
</div></section><section class="tw xbox font-sm" id="tpg270049-tbl-0013"><h4 class="obj_head">TABLE A7.</h4>
<div class="caption p"><p>Prediction performance in terms of normalized root mean square error (NRMSE) and Pearson's correlation (Cor) for the Wheat_5 dataset.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Dataset</th>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">Transfer</th>
<th align="left" rowspan="1" colspan="1">NRMSE</th>
<th align="left" rowspan="1" colspan="1">NRMSE_SD</th>
<th align="left" rowspan="1" colspan="1">Cor</th>
<th align="left" rowspan="1" colspan="1">Cor_SD</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_5</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">YT_17_18‐YT_18_19</td>
<td align="left" rowspan="1" colspan="1">0.064</td>
<td align="left" rowspan="1" colspan="1">0.005</td>
<td align="left" rowspan="1" colspan="1">0.300</td>
<td align="left" rowspan="1" colspan="1">0.055</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_5</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">YT_17_18‐YT_18_19</td>
<td align="left" rowspan="1" colspan="1">0.059</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">0.427</td>
<td align="left" rowspan="1" colspan="1">0.074</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_5</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">YT_17_18‐YT_18_19</td>
<td align="left" rowspan="1" colspan="1">0.059</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">0.430</td>
<td align="left" rowspan="1" colspan="1">0.073</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_5</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">YT_17_18‐YT_18_19</td>
<td align="left" rowspan="1" colspan="1">0.055</td>
<td align="left" rowspan="1" colspan="1">0.003</td>
<td align="left" rowspan="1" colspan="1">0.529</td>
<td align="left" rowspan="1" colspan="1">0.068</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_5</td>
<td align="left" rowspan="1" colspan="1">RR</td>
<td align="left" rowspan="1" colspan="1">YT_18_19‐YT_17_18</td>
<td align="left" rowspan="1" colspan="1">0.042</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">0.394</td>
<td align="left" rowspan="1" colspan="1">0.081</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_5</td>
<td align="left" rowspan="1" colspan="1">ARR</td>
<td align="left" rowspan="1" colspan="1">YT_18_19‐YT_17_18</td>
<td align="left" rowspan="1" colspan="1">0.040</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">0.421</td>
<td align="left" rowspan="1" colspan="1">0.056</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_5</td>
<td align="left" rowspan="1" colspan="1">Transfer_RR</td>
<td align="left" rowspan="1" colspan="1">YT_18_19‐YT_17_18</td>
<td align="left" rowspan="1" colspan="1">0.040</td>
<td align="left" rowspan="1" colspan="1">0.003</td>
<td align="left" rowspan="1" colspan="1">0.434</td>
<td align="left" rowspan="1" colspan="1">0.053</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Wheat_5</td>
<td align="left" rowspan="1" colspan="1">Transfer_ARR</td>
<td align="left" rowspan="1" colspan="1">YT_18_19‐YT_17_18</td>
<td align="left" rowspan="1" colspan="1">0.037</td>
<td align="left" rowspan="1" colspan="1">0.003</td>
<td align="left" rowspan="1" colspan="1">0.562</td>
<td align="left" rowspan="1" colspan="1">0.044</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270049-tbl-0013/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270049-tbl13-note-0001"><p>
<em>Note</em>: NRMSE_SD denotes the standard deviation of NRMSE and Cor_SD denotes the standard deviation of Cor. In the transfer column, the notation “Env1‐Env2” denotes the fact that Env1 is the environment used as a proxy and Env2 is the environment used as goal.</p></div>
<div class="fn" id="tpg270049-tbl13-note-0002"><p>Abbreviations: ARR, analytic ridge regression; RR, ridge regression.</p></div>
</div></section></section></section><section id="notes1"><p>


Montesinos‐López, O. A.
, 
Barajas‐Ramirez, E. A.
, 
Salinas‐Ruiz, J.
, 
Montesinos‐López, A.
, 
Gerard, G.
, 
Vitale, P.
, 
Dreisigacker, S.
, 
Pierre, C. S.
, &amp; 
Crossa, J.
 (2025). Optimizing genomic prediction with transfer learning under a ridge regression framework. The Plant Genome, 18, e70049. 10.1002/tpg2.70049

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270049-note-0001"><p>Assigned to Associate Editor Jianming Yu.</p></div></div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Abelardo Montesinos‐López, Email: abelardo.montesinos@academicos.udg.mx.</p>
<p>José Crossa, Email: j.crossa@cgiar.org.</p></section><section id="tpg270049-sec-0270"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The link <a href="https://github.com/alexa11235/TransferOnGenoData" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/alexa11235/TransferOnGenoData</a> contains the phenotypic and genomic data of the 11 datasets used in this study plus the codes for the transfer learning.</p></section><section id="tpg270049-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270049-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270049-bib-0001">
<cite>

Adams, J.
, 
de Vries, M.
, &amp; 
van Eeuwijk, F.
 (2023). Efficient genomic prediction of yield and dry matter in hybrid potato. Plants, 12(14), 2617. 10.3390/plants12142617

</cite> [<a href="https://doi.org/10.3390/plants12142617" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10385487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37514232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Efficient%20genomic%20prediction%20of%20yield%20and%20dry%20matter%20in%20hybrid%20potato&amp;author=J.%20Adams&amp;author=M.%20de%20Vries&amp;author=F.%20van%20Eeuwijk&amp;volume=12&amp;issue=14&amp;publication_year=2023&amp;pages=2617&amp;pmid=37514232&amp;doi=10.3390/plants12142617&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0002">
<cite>

Alemu, A.
, 
Åstrand, J.
, 
Montesinos‐López, O. A.
, 
Isidro, Y.
, 
Sánchez, J.
, 
Fernández‐Gónzalez, J.
, 
Tadesse, W.
, 
Vetukuri, R. R.
, 
Carlsson, A. S.
, 
Ceplitis, A.
, 
Crossa, J.
, 
Ortiz, R.
, &amp; 
Chawade, A.
 (2024). Genomic selection in plant breeding: Key factors shaping two decades of progress. Molecular Plant, 17(4), 552–578. 10.1016/j.molp.2024.03.007

</cite> [<a href="https://doi.org/10.1016/j.molp.2024.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38475993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=Genomic%20selection%20in%20plant%20breeding:%20Key%20factors%20shaping%20two%20decades%20of%20progress&amp;author=A.%20Alemu&amp;author=J.%20%C3%85strand&amp;author=O.%20A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=Y.%20Isidro&amp;author=J.%20S%C3%A1nchez&amp;volume=17&amp;issue=4&amp;publication_year=2024&amp;pages=552-578&amp;pmid=38475993&amp;doi=10.1016/j.molp.2024.03.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0003">
<cite>

Bartholomé, J.
, 
Prakash, P. T.
, &amp; 
Cobb, J. N.
 (2022). Genomic prediction: Progress and perspectives for rice improvement. Genomic Prediction of Complex Traits: Methods and Protocols, 2467, 569–617.</cite> [<a href="https://doi.org/10.1007/978-1-0716-2205-6_21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35451791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomic%20Prediction%20of%20Complex%20Traits:%20Methods%20and%20Protocols&amp;title=Genomic%20prediction:%20Progress%20and%20perspectives%20for%20rice%20improvement&amp;author=J.%20Bartholom%C3%A9&amp;author=P.%20T.%20Prakash&amp;author=J.%20N.%20Cobb&amp;volume=2467&amp;publication_year=2022&amp;pages=569-617&amp;pmid=35451791&amp;doi=10.1007/978-1-0716-2205-6_21&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0004">
<cite>

Chen, S. X.
, 
Owen, A. B.
, &amp; 
Shi, M.
 (2015). Data enriched linear regression. Electronic Journal of Statistics, 9(2), 2716–2749. 10.1214/15-EJS1027
</cite> [<a href="https://doi.org/10.1214/15-EJS1027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Electronic%20Journal%20of%20Statistics&amp;title=Data%20enriched%20linear%20regression&amp;author=S.%20X.%20Chen&amp;author=A.%20B.%20Owen&amp;author=M.%20Shi&amp;volume=9&amp;issue=2&amp;publication_year=2015&amp;pages=2716-2749&amp;doi=10.1214/15-EJS1027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0005">
<cite>

Christensen, P.
 (2011). Plane answers to complex questions: The theory of linear models. Springer Science+Business Media.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Plane%20answers%20to%20complex%20questions:%20The%20theory%20of%20linear%20models&amp;author=P.%20Christensen&amp;publication_year=2011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0006">
<cite>

Crossa, J.
, 
Beyene, Y.
, 
Kassa, S.
, 
Pérez, P.
, 
Hickey, J. M.
, 
Chen, C.
, 
de los Campos, G.
, 
Burgueño, J.
, 
Windhausen, V. S.
, 
Buckler, E.
, 
Jannink, J.‐L.
, 
Lopez Cruz, M. A.
, &amp; 
Babu, R.
 (2013). Genomic prediction in maize breeding populations with genotyping‐by‐sequencing. G3: Genes, genomes, genetics, 3(11), 1903–1926. 10.1534/g3.113.008227

</cite> [<a href="https://doi.org/10.1534/g3.113.008227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3815055/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24022750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20genomes,%20genetics&amp;title=Genomic%20prediction%20in%20maize%20breeding%20populations%20with%20genotyping%E2%80%90by%E2%80%90sequencing&amp;author=J.%20Crossa&amp;author=Y.%20Beyene&amp;author=S.%20Kassa&amp;author=P.%20P%C3%A9rez&amp;author=J.%20M.%20Hickey&amp;volume=3&amp;issue=11&amp;publication_year=2013&amp;pages=1903-1926&amp;pmid=24022750&amp;doi=10.1534/g3.113.008227&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0007">
<cite>

Dar, S.
, &amp; 
Baraniuk, R. G.
 (2022). A universal framework for transfer learning in linear models. SIAM Journal on Mathematics of Data Science, 4(4), 1347–1379. 10.1137/22M1469559
</cite> [<a href="https://doi.org/10.1137/22M1469559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=SIAM%20Journal%20on%20Mathematics%20of%20Data%20Science&amp;title=A%20universal%20framework%20for%20transfer%20learning%20in%20linear%20models&amp;author=S.%20Dar&amp;author=R.%20G.%20Baraniuk&amp;volume=4&amp;issue=4&amp;publication_year=2022&amp;pages=1347-1379&amp;doi=10.1137/22M1469559&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0008">
<cite>

Friedman, J.
, 
Hastie, T.
, &amp; 
Tibshirani, R.
 (2010). Regularization paths for generalized linear models via coordinate descent. Journal of Statistical Software, 33(1), 1–22. 10.18637/jss.v033.i01

</cite> [<a href="https://doi.org/10.18637/jss.v033.i01" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2929880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20808728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=Regularization%20paths%20for%20generalized%20linear%20models%20via%20coordinate%20descent&amp;author=J.%20Friedman&amp;author=T.%20Hastie&amp;author=R.%20Tibshirani&amp;volume=33&amp;issue=1&amp;publication_year=2010&amp;pages=1-22&amp;pmid=20808728&amp;doi=10.18637/jss.v033.i01&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0009">
<cite>

Habyarimana, E.
, 
Parisi, B.
, &amp; 
Mandolino, G.
 (2017). Genomic prediction for yields, processing and nutritional quality traits in cultivated potato (<em>Solanum tuberosum</em> L.). Plant Breeding, 136(2), 245–252. 10.1111/pbr.12461
</cite> [<a href="https://doi.org/10.1111/pbr.12461" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Breeding&amp;title=Genomic%20prediction%20for%20yields,%20processing%20and%20nutritional%20quality%20traits%20in%20cultivated%20potato%20(Solanum%20tuberosum%20L.)&amp;author=E.%20Habyarimana&amp;author=B.%20Parisi&amp;author=G.%20Mandolino&amp;volume=136&amp;issue=2&amp;publication_year=2017&amp;pages=245-252&amp;doi=10.1111/pbr.12461&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0010">
<cite>

Heffner, E. L.
, 
Sorrells, M. E.
, &amp; 
Jannink, J. L.
 (2009). Genomic selection for crop improvement. Crop Science, 49(1), 1–12. 10.2135/cropsci2008.08.0512
</cite> [<a href="https://doi.org/10.2135/cropsci2008.08.0512" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genomic%20selection%20for%20crop%20improvement&amp;author=E.%20L.%20Heffner&amp;author=M.%20E.%20Sorrells&amp;author=J.%20L.%20Jannink&amp;volume=49&amp;issue=1&amp;publication_year=2009&amp;pages=1-12&amp;doi=10.2135/cropsci2008.08.0512&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0011">
<cite>

Hoerl, A. E.
, &amp; 
Kennard, R. W.
 (1970). Ridge regression: Applications to nonorthogonal problems. Technometrics, 12(1), 69–82. 10.1080/00401706.1970.10488635
</cite> [<a href="https://doi.org/10.1080/00401706.1970.10488635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Technometrics&amp;title=Ridge%20regression:%20Applications%20to%20nonorthogonal%20problems&amp;author=A.%20E.%20Hoerl&amp;author=R.%20W.%20Kennard&amp;volume=12&amp;issue=1&amp;publication_year=1970&amp;pages=69-82&amp;doi=10.1080/00401706.1970.10488635&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0012">
<cite>

Ibba, M. I.
, 
Crossa, J.
, 
MontesinosLópez, O. A.
, 
MontesinosLópez, A.
, 
Juliana, P.
, &amp; 
Guzman, C.
 (2020). Genome‐based prediction of multiple wheat quality traits in multiple years. The Plant Genome, 13(3), e20034. 10.1002/tpg2.20034

</cite> [<a href="https://doi.org/10.1002/tpg2.20034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33217204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genome%E2%80%90based%20prediction%20of%20multiple%20wheat%20quality%20traits%20in%20multiple%20years&amp;author=M.%20I.%20Ibba&amp;author=J.%20Crossa&amp;author=O.%20A.%20MontesinosL%C3%B3pez&amp;author=A.%20MontesinosL%C3%B3pez&amp;author=P.%20Juliana&amp;volume=13&amp;issue=3&amp;publication_year=2020&amp;pages=e20034&amp;pmid=33217204&amp;doi=10.1002/tpg2.20034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0013">
<cite>

Jean, M.
, 
Cober, E.
, 
O'Donoughue, L.
, 
Rajcan, I.
, &amp; 
Belzile, F.
 (2021). Improvement of key agronomical traits in soybean through genomic prediction of superior crosses. Crop Science, 61(6), 3908–3918. 10.1002/csc2.20583
</cite> [<a href="https://doi.org/10.1002/csc2.20583" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Improvement%20of%20key%20agronomical%20traits%20in%20soybean%20through%20genomic%20prediction%20of%20superior%20crosses&amp;author=M.%20Jean&amp;author=E.%20Cober&amp;author=L.%20O'Donoughue&amp;author=I.%20Rajcan&amp;author=F.%20Belzile&amp;volume=61&amp;issue=6&amp;publication_year=2021&amp;pages=3908-3918&amp;doi=10.1002/csc2.20583&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0014">
<cite>

Juliana, P.
, 
Singh, R. P.
, 
Poland, J.
, 
Mondal, S.
, 
Crossa, J.
, 
Montesinos‐López, O. A.
, 
Dreisigacker, S.
, 
Pérez‐Rodríguez, P.
, 
Huerta‐Espino, J.
, 
Crespo‐Herrera, L.
, &amp; 
Govindan, V.
 (2018). Prospects and challenges of applied genomic selection—A new paradigm in breeding for grain yield in bread wheat. The Plant Genome, 11(3), 180017. 10.3835/plantgenome2018.03.0017
</cite> [<a href="https://doi.org/10.3835/plantgenome2018.03.0017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7822054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30512048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Prospects%20and%20challenges%20of%20applied%20genomic%20selection%E2%80%94A%20new%20paradigm%20in%20breeding%20for%20grain%20yield%20in%20bread%20wheat&amp;author=P.%20Juliana&amp;author=R.%20P.%20Singh&amp;author=J.%20Poland&amp;author=S.%20Mondal&amp;author=J.%20Crossa&amp;volume=11&amp;issue=3&amp;publication_year=2018&amp;pages=180017&amp;pmid=30512048&amp;doi=10.3835/plantgenome2018.03.0017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0015">
<cite>

Kick, D.
, &amp; 
Washburn, J. D.
 (2023). Ensemble of best linear unbiased predictor, machine learning and deep learning models predict maize yield better than each model alone. In Silico Plants, 5(2), diad015. 10.1093/insilicoplants/diad015
</cite> [<a href="https://doi.org/10.1093/insilicoplants/diad015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=In%20Silico%20Plants&amp;title=Ensemble%20of%20best%20linear%20unbiased%20predictor,%20machine%20learning%20and%20deep%20learning%20models%20predict%20maize%20yield%20better%20than%20each%20model%20alone&amp;author=D.%20Kick&amp;author=J.%20D.%20Washburn&amp;volume=5&amp;issue=2&amp;publication_year=2023&amp;pages=diad015&amp;doi=10.1093/insilicoplants/diad015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0016">
<cite>

Long, E. M.
, 
Romay, M. C.
, 
Ramstein, G.
, 
Buckler, E. S.
, &amp; 
Robbins, K. R.
 (2023). Utilizing evolutionary conservation to detect deleterious mutations and improve genomic prediction in cassava. Frontiers in Plant Science, 13, 1041925. 10.3389/fpls.2022.1041925

</cite> [<a href="https://doi.org/10.3389/fpls.2022.1041925" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10112518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37082510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Utilizing%20evolutionary%20conservation%20to%20detect%20deleterious%20mutations%20and%20improve%20genomic%20prediction%20in%20cassava&amp;author=E.%20M.%20Long&amp;author=M.%20C.%20Romay&amp;author=G.%20Ramstein&amp;author=E.%20S.%20Buckler&amp;author=K.%20R.%20Robbins&amp;volume=13&amp;publication_year=2023&amp;pages=1041925&amp;pmid=37082510&amp;doi=10.3389/fpls.2022.1041925&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0018">
<cite>

Montesinos‐López, O. A.
, 
Montesinos‐López, O. A.
, 
Crossa, J.
, 
Toledo, F. H.
, 
Pérez‐Hernández, O.
, 
Eskridge, K. M.
, &amp; 
Rutkoski, J.
 (2016). A genomic Bayesian multi‐trait and multi‐environment model. Genes, Genomes and Genetics, 6(9), 2725–44. 10.1534/g3.116.032359
</cite> [<a href="https://doi.org/10.1534/g3.116.032359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5015931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27342738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes,%20Genomes%20and%20Genetics&amp;title=A%20genomic%20Bayesian%20multi%E2%80%90trait%20and%20multi%E2%80%90environment%20model&amp;author=A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=O.%20A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=J.%20Crossa&amp;author=F.%20H.%20Toledo&amp;author=O.%20P%C3%A9rez%E2%80%90Hern%C3%A1ndez&amp;volume=6&amp;issue=9&amp;publication_year=2016&amp;pages=2725-44&amp;pmid=27342738&amp;doi=10.1534/g3.116.032359&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0019">
<cite>

Montesinos‐López, O. A.
, 
Montesinos‐López, A.
, 
Hernandez‐Suarez, C. M.
, 
Barrón‐López, J. A.
, &amp; 
Rossa, J.
 (2021). Deep‐learning power and perspectives for genomic selection. The Plant Genome, 14(3), e20122. 10.1002/tpg2.20122

</cite> [<a href="https://doi.org/10.1002/tpg2.20122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34309215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Deep%E2%80%90learning%20power%20and%20perspectives%20for%20genomic%20selection&amp;author=O.%20A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=C.%20M.%20Hernandez%E2%80%90Suarez&amp;author=J.%20A.%20Barr%C3%B3n%E2%80%90L%C3%B3pez&amp;author=J.%20Rossa&amp;volume=14&amp;issue=3&amp;publication_year=2021&amp;pages=e20122&amp;pmid=34309215&amp;doi=10.1002/tpg2.20122&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0020">
<cite>

Montesinos‐López, O. A.
, 
Montesinos‐López, A.
, 
Pérez‐Rodríguez, P.
, 
Barrón‐López, J. A.
, 
Martini, J. W. R.
, 
Fajardo‐Flores, S. B.
, 
Gaytan‐Lugo, L. S.
, 
Santana‐Mancilla, P. C.
, &amp; 
Rossa, J.
 (2021). A review of deep learning applications for genomic selection. BMC Genomics, 22(1), 19. 10.1186/s12864-020-07319-x

</cite> [<a href="https://doi.org/10.1186/s12864-020-07319-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7789712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33407114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=A%20review%20of%20deep%20learning%20applications%20for%20genomic%20selection&amp;author=O.%20A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=P.%20P%C3%A9rez%E2%80%90Rodr%C3%ADguez&amp;author=J.%20A.%20Barr%C3%B3n%E2%80%90L%C3%B3pez&amp;author=J.%20W.%20R.%20Martini&amp;volume=22&amp;issue=1&amp;publication_year=2021&amp;pages=19&amp;pmid=33407114&amp;doi=10.1186/s12864-020-07319-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0021">
<cite>

Montesinos López, O. A.
, 
Montesinos‐López, A.
, &amp; 
Rossa, J.
 (2022). Multivariate statistical machine learning methods for genomic prediction. Springer. 10.1007/978-3-030-89010-0
</cite> [<a href="https://doi.org/10.1007/978-3-030-89010-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36103587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Multivariate%20statistical%20machine%20learning%20methods%20for%20genomic%20prediction&amp;author=O.%20A.%20Montesinos%20L%C3%B3pez&amp;author=A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=J.%20Rossa&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0022">
<cite>

Monteverde, E.
, 
Gutierrez, L.
, 
Blanco, P.
, 
Pérez de Vida, F.
, 
Rosas, J. E.
, 
Bonnecarrère, V.
, 
Quero, G.
, &amp; 
McCouch, S.
 (2019). Integrating molecular markers and environmental covariates to interpret genotype by environment interaction in rice (<em>Oryza sativa</em> L.) grown in subtropical areas. G3 Genes|Genomes|Genetics, 9(5), 1519–1531. 10.1534/g3.119.400064

</cite> [<a href="https://doi.org/10.1534/g3.119.400064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6505146/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30877079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Integrating%20molecular%20markers%20and%20environmental%20covariates%20to%20interpret%20genotype%20by%20environment%20interaction%20in%20rice%20(Oryza%20sativa%20L.)%20grown%20in%20subtropical%20areas&amp;author=E.%20Monteverde&amp;author=L.%20Gutierrez&amp;author=P.%20Blanco&amp;author=F.%20P%C3%A9rez%20de%20Vida&amp;author=J.%20E.%20Rosas&amp;volume=9&amp;issue=5&amp;publication_year=2019&amp;pages=1519-1531&amp;pmid=30877079&amp;doi=10.1534/g3.119.400064&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0023">
<cite>

Nielsen, N. H.
, 
Jahoor, A.
, 
Jensen, J. D.
, 
Orabi, J.
, 
Cericola, F.
, 
Edriss, V.
, &amp; 
Jensen, J.
 (2016). Genomic prediction of seed quality traits using advanced barley breeding lines. PLoS ONE, 11(10), e0164494. 10.1371/journal.pone.0164494

</cite> [<a href="https://doi.org/10.1371/journal.pone.0164494" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5082657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27783639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Genomic%20prediction%20of%20seed%20quality%20traits%20using%20advanced%20barley%20breeding%20lines&amp;author=N.%20H.%20Nielsen&amp;author=A.%20Jahoor&amp;author=J.%20D.%20Jensen&amp;author=J.%20Orabi&amp;author=F.%20Cericola&amp;volume=11&amp;issue=10&amp;publication_year=2016&amp;pages=e0164494&amp;pmid=27783639&amp;doi=10.1371/journal.pone.0164494&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0024">
<cite>

Pan, S. J.
, &amp; 
Yang, Q.
 (2010). A survey on transfer learning. IEEE Transactions on Knowledge and Data Engineering, 22(10), 1345–1359. 10.1109/TKDE.2009.191
</cite> [<a href="https://doi.org/10.1109/TKDE.2009.191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Transactions%20on%20Knowledge%20and%20Data%20Engineering&amp;title=A%20survey%20on%20transfer%20learning&amp;author=S.%20J.%20Pan&amp;author=Q.%20Yang&amp;volume=22&amp;issue=10&amp;publication_year=2010&amp;pages=1345-1359&amp;doi=10.1109/TKDE.2009.191&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0025">
<cite>
R Core Team
. (2024). R: A language and environment for statistical computing. R Foundation for Statistical Computing. <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R‐project.org/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=R:%20A%20language%20and%20environment%20for%20statistical%20computing&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0026">
<cite>

Spindel, J.
, 
Begum, H.
, 
Akdemir, D.
, 
Virk, P.
, 
Collard, B.
, 
Redona, E.
, 
Atlin, G.
, 
Jannink, J.‐L.
, &amp; 
McCouch, S. R.
 (2015). Genomic selection and association mapping in rice (<em>Oryza sativa</em>): Effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding lines. PLoS Genetics, 11(2), e1004982. 10.1371/journal.pgen.1004982

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1004982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4334555/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25689273/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&amp;title=Genomic%20selection%20and%20association%20mapping%20in%20rice%20(Oryza%20sativa):%20Effect%20of%20trait%20genetic%20architecture,%20training%20population%20composition,%20marker%20number%20and%20statistical%20model%20on%20accuracy%20of%20rice%20genomic%20selection%20in%20elite,%20tropical%20rice%20breeding%20lines&amp;author=J.%20Spindel&amp;author=H.%20Begum&amp;author=D.%20Akdemir&amp;author=P.%20Virk&amp;author=B.%20Collard&amp;volume=11&amp;issue=2&amp;publication_year=2015&amp;pages=e1004982&amp;pmid=25689273&amp;doi=10.1371/journal.pgen.1004982&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0027">
<cite>

Tang, S.
, &amp; 
de Sa, V. R.
 (2020). Deep transfer learning with ridge regression. <em>arXiv</em>. 10.48550/arXiv.2006.06791
</cite> [<a href="https://doi.org/10.48550/arXiv.2006.06791" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270049-bib-0028">
<cite>

Tomura, S.
, 
Wilkinson, M.
, 
Cooper, M.
, &amp; 
Powell, O.
 (2023). Improvements in prediction performance of ensemble approaches for genomic prediction in crop breeding. <em>bioXiv</em>. 10.1101/2024.09.06.611589
</cite> [<a href="https://doi.org/10.1101/2024.09.06.611589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270049-bib-0029">
<cite>

van Wieringen, W. N.
, &amp; 
Binder, H.
 (2020). Transfer learning of regression models from a sequence of datasets by penalized estimation. <em>arXiv</em>. 10.48550/arXiv.2007.02117
</cite> [<a href="https://doi.org/10.48550/arXiv.2007.02117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270049-bib-0030">
<cite>

Wakefield, J.
 (2013). Bayesian and frequentist regression methods. Springer Science+Business Media.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Bayesian%20and%20frequentist%20regression%20methods&amp;author=J.%20Wakefield&amp;publication_year=2013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0036">
<cite>

Washburn, J. D.
, 
Burch, M. B.
, &amp; 
Franco, J. A. V.
 (2020). Predictive breeding for maize: Making use of molecular phenotypes, machine learning, and physiological crop models. Crop Science, 60(2), 622–638. 10.1002/csc2.20052
</cite> [<a href="https://doi.org/10.1002/csc2.20052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Predictive%20breeding%20for%20maize:%20Making%20use%20of%20molecular%20phenotypes,%20machine%20learning,%20and%20physiological%20crop%20models&amp;author=J.%20D.%20Washburn&amp;author=M.%20B.%20Burch&amp;author=J.%20A.%20V.%20Franco&amp;volume=60&amp;issue=2&amp;publication_year=2020&amp;pages=622-638&amp;doi=10.1002/csc2.20052&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0031">
<cite>

Washburn, J. D.
, 
Cimen, E.
, 
Ramstein, G.
, 
Reeves, T.
, 
O'Briant, P.
, 
McLean, G.
, 
Cooper, M.
, 
Hammer, G.
, &amp; 
Buckler, E. S.
 (2021). Predicting phenotypes from genetic, environment, management, and historical data using CNNs. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 134, 3997–4011. 10.1007/s00122-021-03943-7

</cite> [<a href="https://doi.org/10.1007/s00122-021-03943-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34448888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Predicting%20phenotypes%20from%20genetic,%20environment,%20management,%20and%20historical%20data%20using%20CNNs&amp;author=J.%20D.%20Washburn&amp;author=E.%20Cimen&amp;author=G.%20Ramstein&amp;author=T.%20Reeves&amp;author=P.%20O'Briant&amp;volume=134&amp;publication_year=2021&amp;pages=3997-4011&amp;pmid=34448888&amp;doi=10.1007/s00122-021-03943-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0032">
<cite>

Weiss, K.
, 
Khoshgoftaar, T. M.
, &amp; 
Wang, D.
 (2016). A survey of transfer learning. Journal of Big Data, 3(1), 1–40. 10.1186/s40537-016-0043-6
</cite> [<a href="https://doi.org/10.1186/s40537-016-0043-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Big%20Data&amp;title=A%20survey%20of%20transfer%20learning&amp;author=K.%20Weiss&amp;author=T.%20M.%20Khoshgoftaar&amp;author=D.%20Wang&amp;volume=3&amp;issue=1&amp;publication_year=2016&amp;pages=1-40&amp;doi=10.1186/s40537-016-0043-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0033">
<cite>

Yonis, B. O.
, 
Pino del Carpio, D.
, 
Wolfe, M.
, 
Jannink, J. L.
, 
Kulakow, P.
, &amp; 
Rabbi, I.
 (2020). Improving root characterisation for genomic prediction in cassava. Scientific Reports, 10(1), 8003. 10.1038/s41598-020-64963-9

</cite> [<a href="https://doi.org/10.1038/s41598-020-64963-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7224197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32409788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Improving%20root%20characterisation%20for%20genomic%20prediction%20in%20cassava&amp;author=B.%20O.%20Yonis&amp;author=D.%20Pino%20del%20Carpio&amp;author=M.%20Wolfe&amp;author=J.%20L.%20Jannink&amp;author=P.%20Kulakow&amp;volume=10&amp;issue=1&amp;publication_year=2020&amp;pages=8003&amp;pmid=32409788&amp;doi=10.1038/s41598-020-64963-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270049-bib-0034">
<cite>

Zhang, H.
, &amp; 
Li, H.
 (2023). Transfer learning with random coefficient ridge regression. <em>arXiv</em>. 10.48550/arXiv.2306.15915
</cite> [<a href="https://doi.org/10.48550/arXiv.2306.15915" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270049-bib-0035">
<cite>

Zhang, J.
, 
Song, Q.
, 
Cregan, P. B.
, &amp; 
Jiang, G. L.
 (2016). Genome‐wide association study, genomic prediction and marker‐assisted selection for seed weight in soybean (<em>Glycine max</em>). Theoretical and Applied Genetics, 129, 117–130. 10.1007/s00122-015-2614-x

</cite> [<a href="https://doi.org/10.1007/s00122-015-2614-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4703630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26518570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genome%E2%80%90wide%20association%20study,%20genomic%20prediction%20and%20marker%E2%80%90assisted%20selection%20for%20seed%20weight%20in%20soybean%20(Glycine%20max)&amp;author=J.%20Zhang&amp;author=Q.%20Song&amp;author=P.%20B.%20Cregan&amp;author=G.%20L.%20Jiang&amp;volume=129&amp;publication_year=2016&amp;pages=117-130&amp;pmid=26518570&amp;doi=10.1007/s00122-015-2614-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The link <a href="https://github.com/alexa11235/TransferOnGenoData" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/alexa11235/TransferOnGenoData</a> contains the phenotypic and genomic data of the 11 datasets used in this study plus the codes for the transfer learning.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70049"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70049.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12271683/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12271683/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12271683%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12271683/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12271683/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12271683/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40676776/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12271683/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40676776/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12271683/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12271683/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="dBiMGBc4p0qqzZwVTkE3pidxDgMJM2bbNXmB2YvhN1L7ljlxMhj6CqkJcXaELT5t">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
