
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Spider venom peptides with unique fold selectively block Shaker-type potassium channels - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="67D7146689B748931C1466002FFB0B5A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="cmls">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334782/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Cellular and Molecular Life Sciences: CMLS">
<meta name="citation_title" content="Spider venom peptides with unique fold selectively block Shaker-type potassium channels">
<meta name="citation_author" content="Alexey I Kuzmenkov">
<meta name="citation_author_institution" content="Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia">
<meta name="citation_author" content="Valentina A Iunusova">
<meta name="citation_author_institution" content="Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia">
<meta name="citation_author" content="Vladislav A Lushpa">
<meta name="citation_author_institution" content="Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia">
<meta name="citation_author" content="Yakov A Deyev">
<meta name="citation_author_institution" content="Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia">
<meta name="citation_author" content="Vladislav V Babenko">
<meta name="citation_author_institution" content="Lopukhin Federal Research and Clinical Centre of Physical-Chemical Medicine of Federal Medical Biological Agency, Moscow, 119435 Russia">
<meta name="citation_author" content="Daniil V Osipov">
<meta name="citation_author_institution" content="Entomology Department, Moscow Zoo, Moscow, 123557 Russia">
<meta name="citation_author" content="Antonina A Berkut">
<meta name="citation_author_institution" content="Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia">
<meta name="citation_author" content="Jan Tytgat">
<meta name="citation_author_institution" content="Toxicology and Pharmacology, KU Leuven, Leuven, B-3000 Belgium">
<meta name="citation_author" content="Eduard V Bocharov">
<meta name="citation_author_institution" content="Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia">
<meta name="citation_author" content="David J Adams">
<meta name="citation_author_institution" content="Faculty of Science, Medicine and Health, University of Wollongong, Wollongong, NSW 2522 Australia">
<meta name="citation_author_institution" content="Molecular Horizons, University of Wollongong, Wollongong, Australia">
<meta name="citation_author" content="Rocio K Finol-Urdaneta">
<meta name="citation_author_institution" content="Faculty of Science, Medicine and Health, University of Wollongong, Wollongong, NSW 2522 Australia">
<meta name="citation_author_institution" content="Molecular Horizons, University of Wollongong, Wollongong, Australia">
<meta name="citation_author" content="Alexander A Vassilevski">
<meta name="citation_author_institution" content="Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia">
<meta name="citation_author_institution" content="Moscow Institute of Physics and Technology, National Research University, Dolgoprudny, 141701 Moscow Region Russia">
<meta name="citation_publication_date" content="2025 Aug 8">
<meta name="citation_volume" content="82">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="300">
<meta name="citation_doi" content="10.1007/s00018-025-05778-7">
<meta name="citation_pmid" content="40778948">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334782/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334782/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334782/pdf/18_2025_Article_5778.pdf">
<meta name="description" content="Natural toxins are highly effective at targeting ion channels with high selectivity and potency. To date, all identified spider venom peptide toxins that modulate voltage-gated potassium (KV) channels inhibit Shab (KV2) or Shal-related isoforms ...">
<meta name="og:title" content="Spider venom peptides with unique fold selectively block Shaker-type potassium channels">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Natural toxins are highly effective at targeting ion channels with high selectivity and potency. To date, all identified spider venom peptide toxins that modulate voltage-gated potassium (KV) channels inhibit Shab (KV2) or Shal-related isoforms ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334782/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12334782">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1007/s00018-025-05778-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/18_2025_Article_5778.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12334782%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12334782/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12334782/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334782/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-cmls.png" alt="Cellular and Molecular Life Sciences: CMLS logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Cellular and Molecular Life Sciences: CMLS" title="Link to Cellular and Molecular Life Sciences: CMLS" shape="default" href="https://doi.org/10.1007/s00018-025-05778-7" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Cell Mol Life Sci</button></div>. 2025 Aug 8;82(1):300. doi: <a href="https://doi.org/10.1007/s00018-025-05778-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1007/s00018-025-05778-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Cell%20Mol%20Life%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cell%20Mol%20Life%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Cell%20Mol%20Life%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Cell%20Mol%20Life%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Spider venom peptides with unique fold selectively block Shaker-type potassium channels</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kuzmenkov%20AI%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Alexey I Kuzmenkov</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Alexey I Kuzmenkov</span></h3>
<div class="p">
<sup>1</sup>Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kuzmenkov%20AI%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexey I Kuzmenkov</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Iunusova%20VA%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Valentina A Iunusova</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Valentina A Iunusova</span></h3>
<div class="p">
<sup>1</sup>Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Iunusova%20VA%22%5BAuthor%5D" class="usa-link"><span class="name western">Valentina A Iunusova</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lushpa%20VA%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Vladislav A Lushpa</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Vladislav A Lushpa</span></h3>
<div class="p">
<sup>1</sup>Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lushpa%20VA%22%5BAuthor%5D" class="usa-link"><span class="name western">Vladislav A Lushpa</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deyev%20YA%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yakov A Deyev</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yakov A Deyev</span></h3>
<div class="p">
<sup>1</sup>Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deyev%20YA%22%5BAuthor%5D" class="usa-link"><span class="name western">Yakov A Deyev</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Babenko%20VV%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Vladislav V Babenko</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Vladislav V Babenko</span></h3>
<div class="p">
<sup>2</sup>Lopukhin Federal Research and Clinical Centre of Physical-Chemical Medicine of Federal Medical Biological Agency, Moscow, 119435 Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Babenko%20VV%22%5BAuthor%5D" class="usa-link"><span class="name western">Vladislav V Babenko</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Osipov%20DV%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Daniil V Osipov</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Daniil V Osipov</span></h3>
<div class="p">
<sup>3</sup>Entomology Department, Moscow Zoo, Moscow, 123557 Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Osipov%20DV%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniil V Osipov</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Berkut%20AA%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Antonina A Berkut</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Antonina A Berkut</span></h3>
<div class="p">
<sup>1</sup>Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Berkut%20AA%22%5BAuthor%5D" class="usa-link"><span class="name western">Antonina A Berkut</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tytgat%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Jan Tytgat</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jan Tytgat</span></h3>
<div class="p">
<sup>4</sup>Toxicology and Pharmacology, KU Leuven, Leuven, B-3000 Belgium </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tytgat%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jan Tytgat</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bocharov%20EV%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Eduard V Bocharov</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Eduard V Bocharov</span></h3>
<div class="p">
<sup>1</sup>Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bocharov%20EV%22%5BAuthor%5D" class="usa-link"><span class="name western">Eduard V Bocharov</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Adams%20DJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">David J Adams</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">David J Adams</span></h3>
<div class="p">
<sup>5</sup>Faculty of Science, Medicine and Health, University of Wollongong, Wollongong, NSW 2522 Australia </div>
<div class="p">
<sup>6</sup>Molecular Horizons, University of Wollongong, Wollongong, Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Adams%20DJ%22%5BAuthor%5D" class="usa-link"><span class="name western">David J Adams</span></a>
</div>
</div>
<sup>5,</sup><sup>6,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Finol-Urdaneta%20RK%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Rocio K Finol-Urdaneta</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Rocio K Finol-Urdaneta</span></h3>
<div class="p">
<sup>5</sup>Faculty of Science, Medicine and Health, University of Wollongong, Wollongong, NSW 2522 Australia </div>
<div class="p">
<sup>6</sup>Molecular Horizons, University of Wollongong, Wollongong, Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Finol-Urdaneta%20RK%22%5BAuthor%5D" class="usa-link"><span class="name western">Rocio K Finol-Urdaneta</span></a>
</div>
</div>
<sup>5,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vassilevski%20AA%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Alexander A Vassilevski</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Alexander A Vassilevski</span></h3>
<div class="p">
<sup>1</sup>Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia </div>
<div class="p">
<sup>7</sup>Moscow Institute of Physics and Technology, National Research University, Dolgoprudny, 141701 Moscow Region Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vassilevski%20AA%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexander A Vassilevski</span></a>
</div>
</div>
<sup>1,</sup><sup>7,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Moscow, 117997 Russia </div>
<div id="Aff2">
<sup>2</sup>Lopukhin Federal Research and Clinical Centre of Physical-Chemical Medicine of Federal Medical Biological Agency, Moscow, 119435 Russia </div>
<div id="Aff3">
<sup>3</sup>Entomology Department, Moscow Zoo, Moscow, 123557 Russia </div>
<div id="Aff4">
<sup>4</sup>Toxicology and Pharmacology, KU Leuven, Leuven, B-3000 Belgium </div>
<div id="Aff5">
<sup>5</sup>Faculty of Science, Medicine and Health, University of Wollongong, Wollongong, NSW 2522 Australia </div>
<div id="Aff6">
<sup>6</sup>Molecular Horizons, University of Wollongong, Wollongong, Australia </div>
<div id="Aff7">
<sup>7</sup>Moscow Institute of Physics and Technology, National Research University, Dolgoprudny, 141701 Moscow Region Russia </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 7; Revised 2025 May 29; Accepted 2025 May 30; Collection date 2025 Dec.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12334782  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40778948/" class="usa-link">40778948</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Natural toxins are highly effective at targeting ion channels with high selectivity and potency. To date, all identified spider venom peptide toxins that modulate voltage-gated potassium (K<sub>V</sub>) channels inhibit Shab (K<sub>V</sub>2) or Shal-related isoforms (K<sub>V</sub>4) by interacting with their voltage-sensing domains. In this study, we report novel spider-derived pore-blocking toxins that selectively target Shaker-type (K<sub>V</sub>1) channels with nanomolar potency. We isolated murinotoxins MnTx-1 and MnTx-2 from the orange baboon tarantula <em>Pterinochilus murinus</em> and sequenced them using a combination of Edman degradation, mass spectrometry, and venom gland nanopore transcriptomics. MnTx-1 was produced recombinantly, and its NMR solution structure was determined. Although MnTx-1 shares sequence motifs common to spider toxins, it displays a distinctly different three-dimensional structure, featuring an alternative disulfide linkage, which we have termed the Disulfide-Reined Hairpin (DRH). We attribute the unique pharmacology of MnTx-1 to its unusual spatial structure. The DRH motif represents a promising new miniature scaffold for future bioengineering applications.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1007/s00018-025-05778-7.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Ion channel, Neurotoxin, Blocker, Spatial structure, Inhibitor cystine knot, Disulfide</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">A clear trend in drug discovery is the growing use of peptide substances for clinical applications [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Compared to other pharmaceuticals, peptides offer several advantages, including high potency, unique selectivity, and low toxicity [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Furthermore, ongoing advances in biochemistry and pharmacology are addressing the limitations of peptides, including low oral bioavailability, poor plasma stability, and short circulation times. Currently, approximately 80 peptide drugs are available on the global market, with more than 150 candidates undergoing clinical trials and an additional 400–600 peptides in preclinical studies [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>].</p>
<p id="Par3">Animal venoms are a promising and abundant source of pharmacologically active peptides, representing natural cocktails of evolutionary selected, specific, potent, and stable compounds [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Several venom-derived peptides have already been approved by the U.S. Food and Drug Administration, including an antidiabetic peptide from the lizard <em>Heloderma suspectum</em> (Gila monster) and an analgesic peptide derived from the cone snail <em>Conus magus</em>. Additionally, several venom-derived lead compounds are in clinical or preclinical development, including a monomeric insulin from a cone snail [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>], a sea anemone peptide for treating autoimmune diseases [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>], a tumor-targeting peptide from scorpion venom [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>], a spider toxin with potential applications in heart transplantation and neuroprotection after stroke [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>, <a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>], and several toxins from various sources being explored as analgesics [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>].</p>
<p id="Par4">Many venom-derived peptides target ion channels, which are pore-forming transmembrane proteins that regulate ion flux across cell membranes [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. In humans, ion channels are the third most numerous group of proteins involved in signal transduction [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>] and the second largest target for marketed drugs [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Interestingly, more than a quarter of all known ion channel isoforms in the human genome belong to the potassium (K<sup>+</sup>) channel superfamily [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>], which represents significant therapeutic targets [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>].</p>
<p id="Par5">According to the Kalium database (<a href="https://kaliumdb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://kaliumdb.org/</a>), over 400 natural polypeptides target K<sup>+</sup> channels, with nearly all acting as inhibitors [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]; only a few toxins function as channel activators [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Inhibition of K<sup>+</sup> channels by peptide ligands occurs through two distinct mechanisms [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Pore-blocking toxins from snakes, scorpions, sea anemones, cone snails, and other sources bind to the outer vestibule of the channels, obstructing the passage of K<sup>+</sup> ions through the permeation pathway or pore [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Alternatively, gating-modifier toxins from spiders stabilize non-conducting states of K<sup>+</sup> channels by interacting with their voltage-sensing domain [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>, <a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Notably, spider toxins tend to preferentially modulate the gating of Shab (K<sub>V</sub>2) and Shal-related isoforms (K<sub>V</sub>4), whereas most other animal venom peptides block multiple K<sup>+</sup> channel subtypes, with a strong preference for Shaker type (K<sub>V</sub>1) channels [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>, <a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>].</p>
<p id="Par6">Through function-based screening of spider venom fractions, we identified an unexpected activity in the venom of the orange baboon tarantula <em>Pterinochilus murinus</em> Pocock, 1897 (Fig. <a href="#Fig1" class="usa-link">1</a>A) that inhibited Shaker-type K<sup>+</sup> channels, an unprecedented finding in spider venom. This tarantula, commonly known as OBT among pet keepers, inhabits Central, Eastern, and Southern Africa and has several color morphs, some of which were previously classified as distinct species [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>, <a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Bites from these spiders have been reported to cause “more violent pain than the sting of a scorpion”, and often lead to cold sweats and fainting, possibly contributing to arachnophobia in people, both at present and in the past [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>, <a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. We identified the active compound responsible for K<sup>+</sup> channel block as a peptide with a novel folding pattern, a feature more commonly associated with conotoxins and sea anemone peptides. Unlike the conserved structural motifs typically found in venom from snakes, scorpions, and spiders, this discovery highlights an alternative venom-derived peptide structure with unique pharmacology, offering novel insights into protein science and venomics.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334782_18_2025_5778_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/860a/12334782/9a56647d793b/18_2025_5778_Fig1_HTML.jpg" loading="lazy" id="MO1" height="237" width="685" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Purification of murinotoxins (MnTx). <strong>A</strong> Female specimen of <em>P. murinus</em> and the process of venom extraction (“milking”). <strong>B</strong> RP-HPLC separation of crude venom (5 µL) using a Jupiter C<sub>5</sub> column with a linear gradient of acetonitrile concentration. <strong>C</strong> and <strong>D</strong> Second round of RP-HPLC of MnTx-1 and MnTx-2 using a Luna C<sub>18</sub> column. <strong>E</strong> and <strong>F</strong> Reflectron-mode MALDI mass spectra with an isotopic resolution of the purified MnTx-1 and MnTx-2, respectively. The measured monoisotopic molecular mass [M + H]<sup>+</sup> is indicated</p></figcaption></figure></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Purification of active compounds</h3>
<p id="Par8">Crude venom of <em>P. murinus</em> was initially fractionated using reverse-phase (RP) HPLC, and its components were pooled into six major fractions (I–VI; Fig. <a href="#Fig1" class="usa-link">1</a>B). Electrophysiological measurements indicated predominant activity against Shaker-related channels in fraction I (Table <a href="#Tab1" class="usa-link">1</a>). Components of this fraction were then tested individually, revealing the activity in two subfractions. Each step of electrophysiological measurements was conducted using material derived from the separation of 2.5 µL of crude venom (or its dry equivalent of ≈ 0.5 mg). To isolate individual active compounds, these subfractions were subjected to a second round of separation by RP-HPLC (Fig. <a href="#Fig1" class="usa-link">1</a>C, D). This process yielded single symmetrical Gaussian peaks corresponding to &gt; 98% purity for both substances, which were designated murinotoxins (MnTx-1 or κ<sub>1</sub>-theraphotoxin-Pmu3a and MnTx-2 or κ<sub>1</sub>-theraphotoxin-Pmu3b). Further analysis using MALDI mass spectrometry (MS) showed monoisotopic molecular masses of 3851.7 Da and 3913.7 Da (Fig. <a href="#Fig1" class="usa-link">1</a>E, F) for MnTx-1 and MnTx-2, respectively.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Two-electrode voltage-clamp functional profiling of six fractions of <em>P. murinus</em> venom, obtained after RP-HPLC, was performed on <em>Xenopus laevis</em> oocytes to assess their effects on several potassium channels</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334782_18_2025_5778_Tab1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/860a/12334782/277f2174435f/18_2025_5778_Tab1_HTML.jpg" loading="lazy" id="MO10" height="303" width="766" alt="graphic file with name 18_2025_5778_Tab1_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p12"><p>The values represent the percentage of K<sup>+</sup> current inhibition following the application of the respective fractions obtained after separation of crude venom (≈ 0.5 mg)</p></div></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">Sequencing murinotoxins</h3>
<p id="Par9">Alkylation of non-reduced MnTx-1 and MnTx-2 with 4-vinylpyridine revealed no free thiol groups in the native peptides, as confirmed by MS analysis. However, the mass differences (636.6 Da) between the native and reduced-alkylated peptides indicated that both MnTx-1 and MnTx-2 contain six half-cystine residues. Direct Edman degradation provided partial N-terminal sequences consisting of 33 amino acid residues (Fig. <a href="#Fig2" class="usa-link">2</a>A, Fig. <a href="#MOESM1" class="usa-link">S1</a>E). Differences of measured and calculated molecular masses indicated the full-length sequences of both toxins consist of 35 amino acids, leaving the two C-terminal residues undetermined. To complete the primary structure, we constructed and sequenced a cDNA library from <em>P. murinus</em> venom glands for further analysis.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334782_18_2025_5778_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/860a/12334782/973d6ab7d4eb/18_2025_5778_Fig2_HTML.jpg" loading="lazy" id="MO3" height="506" width="685" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Structure of murinotoxins. <strong>A</strong> Alignment of MnTx-1 precursor sequence deduced from the cDNA of <em>P. murinus</em> venom glands with similar known sequences. Signal peptides and propeptides are shown, with cysteine residues highlighted in yellow. Differences in sequences of mature chains are shaded. The processing quadruplet motif (PQM) is enclosed in a rectangle. <strong>B</strong> Amino acid sequence of MnTx-1 is colored-coded based on chemical properties: positively charged and hydrophilic uncharged residues are shown in blue, negatively charged in red, hydrophobic in black, glycine is in light gray, and cysteines in yellow. Disulfide bridges are represented by lines and cysteines are numbered using Roman numerals. <strong>C</strong> Topology diagram of MnTx fold compared to the conventional ICK motif. Cysteines are labeled with Roman numerals, and corresponding disulfide bonds are represented by lines. In the disulfide-reined hairpin (DRH) motif, the two disulfide bridges connecting the C-terminal loop to the N-terminal cysteine-rich core are colored red. For the ICK motif, the ring and penetrating disulfide bond are highlighted in blue and red, respectively. <strong>D</strong> Set of 10 NMR structures of MnTx-1 with the fewest restraint violations (PDB ID: 9JGG). Disulfide bridges are colored in yellow and labeled accordingly. <strong>E</strong> The spatial structure of MnTx-1 is depicted as a ribbon representation, with disulfide bridges shown as yellow sticks. <strong>F</strong> Molecular surface representation of the lowest-energy MnTx-1 structure highlighting the amphipathic nature of the peptide. The negatively charged residue is shown in red, other hydrophilic residues in blue, and hydrophobic residues in gray</p></figcaption></figure><p id="Par10">We obtained sequencing data from a single venom gland sample using two technologies: 12,527,935 paired-end reads of Illumina (2 × 125 bp) and 17,890,347 long-reads of Oxford Nanopore following the direct cDNA sequencing protocol. On the first step, we assembled the <em>de novo</em> transcriptome from short-reads, resulting in 45,721 contig sequences (transcripts). To identify target transcripts, we performed a local search using BlastX and the toxin reference sequences established by Edman degradation. This process resulted in the selection of several potentially suitable sequences. Despite the high accuracy of short-reads, during the reconstruction (assembly) of transcripts, the occurrence of assembly artifacts cannot be ruled out. To verify the results, we utilized full-length Nanopore cDNA reads. The archived data have been deposited in the NCBI Sequence Read Archive under BioProject accession number PRJNA1252972. Initially, nanopore cDNA reads were clustered based on sequence similarity using the RATTLE program [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. This algorithm allows the assembly of consensus sequences corresponding to transcripts without requiring a reference in the form of a genome. By searching for correspondences of translated sequences, we identified two transcripts containing complete coding frames for MnTx-1 and 2 (Fig. <a href="#MOESM1" class="usa-link">S1</a>G). The deduced amino acid sequence of the full precursor protein for MnTx-1 is shown in Fig. <a href="#Fig2" class="usa-link">2</a>A. Mature MnTx peptides differ by three residues (Fig. <a href="#MOESM1" class="usa-link">S1</a>E; UniProt IDs: C0HMD2 and C0HMD3).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Production of recombinant MnTx-1</h3>
<p id="Par11">Recombinant MnTx-1 was produced following our standard approach, using <em>E. coli</em> SHuffle T7 Express as the host strain [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. The gene encoding the peptide was amplified by PCR (Table <a href="#MOESM1" class="usa-link">S1</a>) and subcloned into pET-32b(+) expression vector. MnTx-1 was fused with thioredoxin (Trx) as a partner to ensure a high yield of the disulfide-containing peptide (Fig. <a href="#MOESM1" class="usa-link">S1</a>A). The chimeric protein was then digested with CNBr, followed by separation using RP-HPLC and identification by MALDI MS (Fig. <a href="#MOESM1" class="usa-link">S1</a>B). Due to a sizable portion of the toxin being misfolded, we performed a refolding procedure (Fig. <a href="#MOESM1" class="usa-link">S1</a>C, D). A final round of RP-HPLC purification on an analytical column yielded the target peptide in an individual state. The measured molecular mass of the purified recombinant MnTx-1 matched the calculated value. Co-elution of natural MnTx-1 and its recombinant analogue confirmed their identity (Fig. <a href="#MOESM1" class="usa-link">S1</a>F). Furthermore, <sup>1</sup>H NMR spectra recorded for both the native and recombinant MnTx-1 additionally support their structural identity (Fig. <a href="#MOESM1" class="usa-link">S2</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">Spatial structure of MnTx-1</h3>
<p id="Par12">The 3D structure of the peptide in aqueous solution was solved using NMR spectroscopy. A total of 100 structures were calculated using the torsion angle restraints, upper NOE-based distance restraints, and hydrogen and disulfide bond restraints (Fig. <a href="#MOESM1" class="usa-link">S3</a>). Detailed data are provided in Table <a href="#MOESM1" class="usa-link">S2</a>. The final set of 10 NMR structures showed low RMSD values for backbone atoms (0.07 ± 0.03 Å) and heavy atoms (0.56 ± 0.06 Å), with minimal restraint violations, indicating that the structure is well defined by the experimental data (Fig. <a href="#Fig2" class="usa-link">2</a>D). The conformation of MnTx-1 is stabilized by three disulfide bridges (C3–C17, C10–C32, and C16–C22; Fig. <a href="#Fig2" class="usa-link">2</a>B, E) and 11 hydrogen bonds. The surface of the molecule is predominantly positively charged, except for a hydrophobic cluster composed of I4, W9, P11, A14, and P15 side chains (Fig. <a href="#Fig2" class="usa-link">2</a>F).</p></section><section id="Sec7"><h3 class="pmc_sec_title">MnTx-1 inhibits K<sub>V</sub>1-mediated potassium currents</h3>
<p id="Par13">Whole-cell patch-clamp electrophysiology was used to assess the effects of MnTx-1 on voltage-gated potassium (K<sub>V</sub>) and sodium (Na<sub>V</sub>) channels. The peptide inhibited K<sup>+</sup> currents mediated by K<sub>V</sub>1.1, K<sub>V</sub>1.2, K<sub>V</sub>1.3, K<sub>V</sub>1.6, and K<sub>V</sub>1.7 channels (Fig. <a href="#Fig3" class="usa-link">3</a>, Fig. <a href="#MOESM1" class="usa-link">S4</a>), whereas currents from the K<sub>V</sub>2–4 subfamilies, as well as those mediated by Na<sub>V</sub>1.7 and Na<sub>V</sub>1.8 channels, were resistant to micromolar concentrations of MnTx-1 (Fig. <a href="#Fig3" class="usa-link">3</a>A, Table <a href="#MOESM1" class="usa-link">S3</a>). Representative current traces for K<sub>V</sub>1.1 and K<sub>V</sub>1.6 exposed to increasing MnTx-1 concentrations, along with corresponding steady-state concentration-response curves, are shown in Fig. <a href="#Fig3" class="usa-link">3</a>B. The IC<sub>50</sub> values for MnTx-1 inhibition of K<sub>V</sub>1.1 and K<sub>V</sub>1.6-mediated currents were 1.1 ± 0.6 nM (<em>n</em> = 5) and 4.4 ± 1.0 nM (<em>n</em> = 6), respectively (h = 0.7), indicating high potency. K<sub>V</sub>1.2 was moderately sensitive, with an IC<sub>50</sub> of 171.1 ± 16.9 nM (<em>n</em> = 8). K<sub>V</sub>1.3 exhibited lower sensitivity, IC<sub>50</sub> = 381 ± 40 nM (<em>n</em> = 5). Thus, among the K<sub>V</sub>1 channels assessed, K<sub>V</sub>1.1 and K<sub>V</sub>1.6 exhibited the greatest sensitivity to MnTx-1.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334782_18_2025_5778_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/860a/12334782/9840f2afefd9/18_2025_5778_Fig3_HTML.jpg" loading="lazy" id="MO4" height="698" width="685" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MnTx-1 selectively inhibits currents mediated by recombinant K<sub>V</sub>1 channels. <strong>A</strong> Representative whole-cell currents mediated by K<sub>V</sub>1–4 channels are shown in control (black) and in the presence of MnTx-1 (green). <strong>B</strong> Current traces of K<sub>V</sub>1.1 and K<sub>V</sub>1.6 channels are displayed with increasing concentrations of MnTx-1, along with the corresponding concentration-response relationships for peak current inhibition. Data are presented as mean ± standard error of the mean (SEM), with <em>n</em> = 4–8 for each concentration</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">MnTx-1 interacts with the pore of K<sub>V</sub>1 channels</h3>
<p id="Par15">Pore blockers inhibit K<sub>V</sub> channel function by physically obstructing the ion conduction pathway, typically through binding within the channel pore, preventing the passage of potassium ions and effectively blocking channel activity. MnTx-1 rapidly inhibited K<sub>V</sub>1 currents upon application, with the inhibition being readily reversible after washout (Fig. <a href="#Fig4" class="usa-link">4</a>A), suggestive of a direct interaction with the pore domain of K<sub>V</sub>1 channels. The half-activation potential of K<sub>V</sub>1.1-mediated currents in the absence and presence of MnTx-1 (10 nM) was indistinguishable (two-tailed paired t-test <em>p</em> = 0.50) (Fig. <a href="#Fig4" class="usa-link">4</a>B), indicating that MnTx-1 does not affect the voltage-dependent activation properties of K<sub>V</sub>1.1. The apparent increased MnTx-1 block observed at more negative potentials aligns with the voltage range where K<sub>V</sub>1.1 transitions between resting to activated states, suggesting possible state-dependent toxin binding.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334782_18_2025_5778_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/860a/12334782/2762a8a828a2/18_2025_5778_Fig4_HTML.jpg" loading="lazy" id="MO5" height="614" width="685" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MnTx-1 interacts with the pore of K<sub>V</sub>1 channels. <strong>A</strong> Time course and reversibility of MnTx-1 block of K<sub>V</sub>1.1 channels. Whole-cell peak currents recorded from a cell expressing K<sub>V</sub>1.1 channels during the application of MnTx-1 (10 nM, green bar) and upon washout with extracellular solution. Stimuli: 50 ms pulses to 20 mV from a holding potential of − 100 mV at 0.1 Hz. <strong>B</strong> K<sub>V</sub>1.1 normalized conductance (left axis) in the absence (●) and presence of MnTx-1 (10 nM, <img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/860a/12334782/80253d01a956/18_2025_5778_Figa_HTML.gif" loading="lazy" id="d33e840" alt="Inline graphic">), along with the percent inhibition (right axis, <img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/860a/12334782/040426a9207c/18_2025_5778_Figb_HTML.gif" loading="lazy" id="d33e843" alt="Inline graphic">), plotted as a function of test potential. <strong>С</strong> Pore sequence alignment of K<sub>V</sub>1.5 and 1.2 shows that residue V381 in the toxin-sensitive K<sub>V</sub>1.2 channel is replaced by R451 in the insensitive K<sub>V</sub>1.5. The pore residues are also highlighted (left) in the top view of K<sub>V</sub>1.2 channel structure. <strong>D</strong> K<sub>V</sub>1.5-mediated currents are not blocked by 3 µM MnTx-1 (left), whereas 300 nM MnTx-1 blocks ≈ 50% of K<sub>V</sub>1.2-mediated currents. V381R substitution reduces the sensitivity of K<sub>V</sub>1.2 channels to MnTx-1</p></figcaption></figure><p id="Par16">Compared to other K<sub>V</sub>1 family members, differences in the amino acid residues lining the pore and vestibule of K<sub>V</sub>1.5 make it less susceptible to pore-blocking peptide toxins [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Consistent with this, MnTx-1 failed to inhibit K<sub>V</sub>1.5-mediated currents, even at micromolar concentrations (Fig. <a href="#Fig4" class="usa-link">4</a>D). Sequence alignment of the pore loops of K<sub>V</sub>1.2 and K<sub>V</sub>1.5 highlights key differences between the toxin-sensitive and toxin-insensitive isoforms (Fig. <a href="#Fig4" class="usa-link">4</a>C). Substituting V381 with the corresponding K<sub>V</sub>1.5 residue (R381) nearly eliminated MnTx-1 sensitivity of K<sub>V</sub>1.2 (Fig. <a href="#Fig4" class="usa-link">4</a>D), emphasizing the critical role of specific pore residues in MnTx-1 binding.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Murinotoxins represent a new class of spider toxins</h3>
<p id="Par17">We have identified two novel spider venom peptides with unique structural and functional features, likely representing a new class of toxins. While previous studies hinted at similar sequences in tarantulas, these were based on transcriptomic data without peptide verification or on proteomic data with incomplete sequencing (Fig. <a href="#Fig2" class="usa-link">2</a>A). For example, two related toxins were detected in the venom of the Chinese earth tiger tarantula <em>Chilobrachys guangxiensis</em>, which were only partially sequenced and later fully reconstructed from the transcriptome [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. Similarly, analysis of expressed sequence tags from the venom glands of the Chinese bird spider <em>Cyriopagopus schmidti</em> revealed a transcript with high similarity to MnTx [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. The highest similarity score was observed for peptides with partially determined primary structures from the venom of the Salem ornamental tarantula <em>Poecilotheria formosa</em> [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. However, those peptides were not purified in sufficient quantities to enable thorough functional characterization. In the present study, we purified and determined the complete primary structures of two MnTx (MnTx-1 and MnTx-2), along with their precursor sequences (Fig. <a href="#Fig2" class="usa-link">2</a>A). These precursors exhibit the typical architecture of spider venom-derived peptides, consisting of signal peptides, propeptides with well-defined processing quadruplet motifs (PQM), and mature chains [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>, <a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>].</p>
<p id="Par18">MnTx, along with related peptides from other tarantula species, share the inhibitor cystine knot (ICK) cysteine framework [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. They also contain an unusually high number of proline residues (up to five) and a high proportion of positively charged amino acids, resulting in a net charge of + 7 to + 9 at neutral pH.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Unanticipated pharmacology for spider toxins</h3>
<p id="Par19">Spider toxins that affect K<sup>+</sup> channels are well-known tools to target Shab (K<sub>V</sub>2) and Shal-related isoforms (K<sub>V</sub>4), as was first demonstrated by Swartz and MacKinnon, who discovered hanatoxins in the venom of the Chilean rose tarantula <em>Grammostola rosea</em> [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>, <a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. While some spider venom-derived polypeptide ligands possess selective activity against specific ion channels, target promiscuity is also a common phenomenon [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. As a result, the pharmacological profile of these toxins can extend to other K<sup>+</sup> channels, such as hERG (K<sub>V</sub>11) [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>], and even to other ion channels, including voltage-gated calcium (Ca<sub>V</sub>) and sodium (Na<sub>V</sub>) channels [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>–<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. Minor examples of spider toxins include inhibitors of EAG (K<sub>V</sub>10) [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>] and insect Ca<sup>2+</sup>-activated large conductance K<sup>+</sup> channels [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>, <a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. There is also evidence of weak activity of hanatoxin-1 against Shaker-IR [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>] and two Kunitz-type huwentoxins against K<sub>V</sub>1.1 [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>, <a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>], though only at micromolar concentrations. To date, high-affinity inhibitors of K<sub>V</sub>1 have not been found from spider venom.</p>
<p id="Par20">Earlier studies have reported two pterinotoxins, β-theraphotoxin-Pmu1a and β/κ-theraphotoxin-Pmu2a, isolated from the venom of <em>P. murinus</em>, which affect several Na<sub>V</sub> channels and K<sub>V</sub>11.1 [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. Another toxin, µ/ω-theraphotoxin-Pmu1a, exhibits dual inhibitory activity on Na<sub>V</sub> and Ca<sub>V</sub>3 channels [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. Although these substances have not been assessed against a broader range of K<sup>+</sup> channels, potential promiscuity is not unexpected given their mode of action (see below). In contrast to these polypeptides, MnTx selectively block Shaker-type K<sup>+</sup> channels with strong preference for K<sub>V</sub>1.1 and K<sub>V</sub>1.6, whereas K<sub>V</sub>2–4 channels are insensitive to these toxins (Fig. <a href="#Fig3" class="usa-link">3</a>). Furthermore, the venom fraction containing MnTx showed no effect on K<sub>V</sub>7.1, K<sub>V</sub>10.1, K<sub>V</sub>11.1 (Table <a href="#Tab1" class="usa-link">1</a>), or several human Na<sub>V</sub> channels. This molecular pharmacology is highly unusual for spider toxins and is more reminiscent of compounds found in other venomous animals such as snakes, scorpions, sea anemones, and cone snails.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Pore blockage vs. voltage sensor immobilization</h3>
<p id="Par21">Before the discovery of hanatoxins [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>], the only known mechanism by which toxins inhibited K<sup>+</sup> channels involved binding to the outer pore region and physical blockage of K<sup>+</sup> conduction. This mode of action was studied extensively through mutagenesis, initially using scorpion toxins, and later with peptides from other species. The hallmark of this mechanism is the projection of a key lysine residue from the toxin into the pore of the channel, where it interacts with potassium ions within the selectivity filter [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>, <a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. This ligand-channel stabilization can be also reinforced by an aromatic amino acid from the toxin, which together with the key lysine residue, forms a ‘functional dyad’ [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. Further structural studies confirmed this mechanism through the examination of complexes between: (i) charybdotoxin from scorpion venom and K<sub>V</sub>1.2–K<sub>V</sub>2.1 paddle chimera [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]; (ii) Fab-modified sea anemone toxin ShK and K<sub>V</sub>1.3 [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]; and (iii) snake α-dendrotoxin and K<sub>V</sub>1.2 [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>].The first evidence that peptide toxins can inhibit K<sup>+</sup> channels through a mode other than pore occlusion came from experiments with tarantula toxins [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>, <a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. As with pore blockers, mutagenesis was used to identify specific amino acid residues involved in channel-toxin recognition. A solvent-exposed hydrophobic surface, along with several surrounding basic residues, constitute the active face of the toxins [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>], which bind to the S3–S4 region of the voltage-sensing domain [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>, <a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. This mechanism, often referred to as “membrane-access” model [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>], is believed to proceed through the partitioning of toxins into the lipid membrane followed by lateral association with the voltage-sensing domains of the channels [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>, <a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>].</p>
<p id="Par22">To investigate the mechanism of action of MnTx, we performed electrophysiological measurements using a mutant K<sub>V</sub>1.2 channel. K<sub>V</sub>1.2 was selected for mutational analysis due to the availability of high-resolution structural data, which offer a robust framework for interpreting toxin-channel interactions. Owing to its high sequence similarity to K<sub>V</sub>1.1 and K<sub>V</sub>1.6, K<sub>V</sub>1.2 is widely accepted as an appropriate structural surrogate for these isoforms. In this context, we introduced a single-point mutation in the pore region, substituting valine with arginine (V381R), to assess its impact on MnTx-1 sensitivity. This site was selected based on the observation that MnTx-1 inhibits K<sub>V</sub>1.2 but has no measurable effect on K<sub>V</sub>1.5, making K<sub>V</sub>1.2-V381R a suitable model for identifying key toxin-pore interaction sites. The substitution of valine with a bulky, positively charged arginine at the pore entrance (Fig. <a href="#Fig4" class="usa-link">4</a>C) was expected to disrupt binding through steric and electrostatic repulsion. As shown in Fig. <a href="#Fig4" class="usa-link">4</a>D, this mutation significantly reduced MnTx-1-mediated current inhibition, supporting its action as a pore blocker. In light of prior work on Urotoxin [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>], which demonstrated that a single-point mutation can confer sensitivity to otherwise resistant K<sub>V</sub> channels, it would be of interest to examine the reciprocal substitution in K<sub>V</sub>1.5 (R487V) to determine whether a similar gain in MnTx-1 susceptibility can be achieved. We speculate that MnTx-1 functions similarly to the well-characterized toxins from scorpions, sea anemones, and cone snails, which employ a key lysine residue to obstruct the channel pore. Given that both MnTx-1 and MnTx-2 are active against K<sub>V</sub>1 channels (data not shown), it is possible that channel blockade occurs through the use of one of their eight identical lysine residues.</p>
<p id="Par23">Only a few studies have provided evidence supporting the pore-blocking mechanism for spider venom peptides acting on K<sup>+</sup> channels [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>, <a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. Consequently, most spider-derived peptides that target K<sup>+</sup> channels became synonymous with “voltage-sensor toxins” or “gating modifiers” [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. Our current study identifies a novel class of K<sup>+</sup> channel inhibitors from tarantula venom that occlude the channel pore, rather than alter gating. This type of pore-blocking toxin is likely to be just as significant a component of spider venom as voltage-sensor toxins. Future research may uncover additional members of this class in other tarantula species.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Unconventional structural motif</h3>
<p id="Par24">In contrast to marine invertebrates such as sea anemones and cone snails, terrestrial venomous animals tend to exploit conserved folds to build their toxin arsenal. For example, the highly populated scaffolds utilized by snakes, scorpions, and spiders are three-finger toxins (3FTх), cysteine-stabilized α-helix/β-sheet (CSα/β), and inhibitor cystine knot (ICK), respectively [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. While the ICK motif [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>] remains the predominant structure in spider venom, several other minor motifs have been identified in disulfide-rich peptides, including the disulfide-directed β-hairpin (DDH) [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>], Kunitz-type [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>], prokineticin/colipase [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>], and helical arthropod-neuropeptide-derived motif (HAND) [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>]. Additionally, a recent structural venomics study revealed several potential non-ICK toxins, such as peptides featuring the disulfide-stabilized antiparallel β-hairpin stack motif (DABS) [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>].</p>
<p id="Par25">The cysteine framework in MnTx is C<sup>I</sup>X<sub>6</sub>C<sup>II</sup>X<sub>5</sub>C<sup>III</sup>X<sub>0</sub>C<sup>IV</sup>X<sub>5</sub>C<sup>V</sup>X<sub>9</sub>C<sup>VI</sup> (where X represents any amino acid; Fig. <a href="#Fig2" class="usa-link">2</a>A), which closely resembles the framework found in ICK-containing peptides [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. This structural motif usually contains an antiparallel β-hairpin, stabilized by disulfide bridges with the following connectivity: C<sup>I</sup>–C<sup>IV</sup>, C<sup>II</sup>–C<sup>V</sup>, and C<sup>III</sup>–C<sup>VI</sup>. A key feature of the ICK is that two of the disulfide bridges (C<sup>I</sup>–C<sup>IV</sup> and C<sup>II</sup>–C<sup>V</sup>) and intervening stretches of backbone segments form a ring that is penetrated by the third disulfide bridge (C<sup>III</sup>–C<sup>VI</sup>), creating a pseudoknot structure (77). Unexpectedly, the NMR study of MnTx-1 revealed an alternative fold, previously uncharacterized for spider-derived peptides (Fig. <a href="#Fig2" class="usa-link">2</a>C–F). Instead of the canonical ICK disulfide connectivity, we observed the following disulfide bond arrangement: C<sup>I</sup>–C<sup>IV</sup>, C<sup>II</sup>–C<sup>VI</sup>, and C<sup>III</sup>–C<sup>V</sup>. This new fold lacks the penetrated ring structure, and the two corresponding cystines anchor the C-terminal loop to the N-terminal cysteine-rich core without any crossover. Additionally, the typical antiparallel β-hairpin is absent. Given these unique structural features, we propose naming this new structural motif the Disulfide-Reined Hairpin (DRH).</p>
<p id="Par26">Refolding experiments (Fig. <a href="#MOESM1" class="usa-link">S1</a>B–D) suggest that the DRH structure is energetically favored in MnTx compared to other potential folds, including the ICK motif. This implies that the DRH scaffold may also be the preferred structure for a broader group of peptides that share similarities with MnTx (Fig. <a href="#Fig2" class="usa-link">2</a>A). It is possible that a subset of spider toxins, previously thought to adopt the ICK motif, may in fact assume the DRH fold. Regardless, this newly identified minifold presents an exciting opportunity for future bioengineering efforts.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par27">The venom of <em>P. murinus</em> contains peptide toxins with unprecedented structural and pharmacological properties. MnTx-1 and MnTx-2, described here, are the founding members of a new class of spider toxins that adopt the novel DRH fold with unique disulfide connectivity. These toxins act as high-affinity, selective pore blockers of Shaker-type (K<sub>V</sub>1) channels, a function not previously recognized in spider venoms. Similar peptides from other tarantula species are likely to be identified and classified within this emerging family of K<sub>V</sub>1 channel blockers. Our findings not only advance the understanding of protein structure-function relationships but also provide new tools for biotechnological scaffold engineering applications.</p></section></section><section id="Sec15"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec16"><h3 class="pmc_sec_title">Ethics</h3>
<p id="Par28">All procedures involving animals were performed following the ARRIVE 2.0 guidelines (Animal Research: Reporting of In Vivo Experiments) [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>] and the “European Convention for the Protection of Vertebrate Animals Used for Experimental and Other Scientific Purposes” (Strasbourg, 18.III.1986). The use of the frogs was in accordance with the license number LA1210239 to Toxicology and Pharmacology, KU Leuven, and was approved by the Ethical Committee for animal experiments of KU Leuven (P186/2019). Maximum efforts were made to minimize animal suffering and the number of animals used in the experiments.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Venom collection</h3>
<p id="Par29">Female <em>P. murinus</em> tarantulas were obtained from private collections of spider keepers and housed in a controlled laboratory environment. Species identification was confirmed based on external morphology in vivo and molt <em>exuviae</em>, using appropriate references according to the World Spider Catalog (Fig. <a href="#Fig1" class="usa-link">1</a>A) [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Crude venom was “milked” by electrically stimulating the fangs of sedated spiders using a Model 2100 Isolated pulse stimulator (A-M Systems). To sedate spiders, they were placed in small cages within a closed box, which was then filled with carbon dioxide. After 15–20 min in the CO<sub>2</sub> atmosphere, spiders lost activity, were removed from the box, and positioned dorsal side down on a table. The chelicerae fangs were placed at the edge of a Petri dish. One electrode was placed on the cephalothorax between the second and third pairs of legs, while the other electrode was attached to the spider’s chelicerae or pedipalps. A brief electric shock (1–3 s, 70–80 V) led to the release of venom into the Petri dish. From each spider, 10–20 µL of venom was collected at a time. To minimize harm to the spiders, venom from a single individual was collected no more than once every 2–4 weeks and a maximum of 5 times within a year. Freshly milked venom was immediately stored in a freezer (–20 °C) and subsequently freeze-dried to optimize preservation.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Bioactivity-guided fractionation of venom</h3>
<p id="Par30"><em>P. murinus</em> venom (5 µL for each separation) was fractionated using RP-HPLC with a Jupiter C<sub>5</sub> column (4.6 × 250 mm; particle size, 5 μm; pore size, 300 Å; Phenomenex). The separation utilized a linear gradient of acetonitrile concentration (0–60% over 60 min) in the presence of 0.1% trifluoroacetic acid (TFA) at a flow rate of 1 mL·min<sup>–1</sup>. Fractions were collected based on absorbance at two wavelengths, 210 and 280 nm, then freeze-dried and stored at −20 °C. Immediately before the experiments, fractions were reconstituted in 10 µL of ND96 solution containing (in mM): 96 NaCl, 2 KCl, 1.8 CaCl<sub>2</sub>, 2 MgCl<sub>2</sub>, and 5 HEPES, pH 7.4.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Peptide sequencing</h3>
<p id="Par31">To perform direct protein sequencing, the purified toxins were first reduced and alkylated as described previously [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>]. The modified peptides were then separated using RP-HPLC with a Jupiter C<sub>5</sub> column. The N-terminal amino acid sequences of the modified peptides were determined by automated Edman degradation using a PPSQ-33 A protein sequencer (Shimadzu Biotech), following the manufacturer’s recommended protocols.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Venom glands isolation</h3>
<p id="Par32">Thirty-six hours after successful milking via electrostimulation, the test subject was exposed to carbon dioxide in a closed container for 1 h. Subsequently, the spider’s abdomen was severed from the cephalothorax to prevent resuscitation, as the abdomen contains the heart and lungs. The chelicerae were separated from the cephalothorax along the articular membrane with scissors. Each chelicera was then cut lengthwise on both sides, and the cuticle was removed from one side. The venom glands, which in tarantulas are fully enclosed within the chelicerae, and do not extend into the cavity of cephalothorax, were separated from muscular tissue with needles. The gland duct was cut at the exit in the fang of chelicerae, and the extracted glands were placed in tubes containing RNA stabilizer (IntactRNA; Evrogen, Russia) and stored at −80 °C.</p></section><section id="Sec21"><h3 class="pmc_sec_title">cDNA library construction and nanopore sequencing</h3>
<p id="Par33">Frozen spider venom glands, stored at −80 °C, were homogenized using a mortar and pestle in liquid nitrogen. Total RNA was extracted from the tissue fragments using the ExtractRNA reagent (Evrogen). The concentration and quality of the total RNA were assessed by the Quant-it RiboGreen RNA Assay Kit (Thermo Fisher Scientific) and the RNA 6000 Pico Kit for 2100 Bioanalyzer Systems (Agilent Technologies), respectively. Starting material (200 ng total RNA) was used to prepare the library according to the published protocol (10.17504/protocols.io.yxmvmkpxng3p/v1). The sequencing library for Oxford Nanopore PromethION was created using the Direct cDNA Sequencing Kit SQK-DCS109 (amplification-free), according to the manufacturer’s recommendations, and run in a FLO-PRO002 (R9.4.1) flow cell. Raw data were basecalled using Guppy v6.5.7. (guppy basecaller) in high-accuracy (HAC) mode with a qscore cutoff of 7. The RNA library was prepared using the NEBNext Poly(A) mRNA Magnetic Isolation Module and the NEBNext Ultra II Directional RNA Library Prep Kit (New England Biolabs), following the manufacturer’s protocol. The final library was cleaned up using the Agencourt AMPure XP system (Beckman Coulter). The libraries size distribution and quality were assessed using a high-sensitivity DNA chip (Agilent Technologies), and they were subsequently quantified with the Quant-iT dsDNA High-Sensitivity Assay Kit (Thermo Fisher Scientific). Equimolar quantities of all libraries (10 pmol) were sequenced in a high-throughput run on the Illumina HiSeq 2500 platform, using 2 × 100 bp paired-end reads and a 1% PhiX spike-in control. Quality control of the raw sequence data was performed with FastQC. Trimmomatic was used to trim the adapter sequences used for cDNA synthesis.</p>
<p id="Par34">The short-read transcriptome was <em>de novo</em> reconstructed using the Trinity assembler [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>] with default parameters. TransDecoder (<a href="https://github.com/TransDecoder/TransDecoder" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/TransDecoder/TransDecoder</a>) was used to identify putative ORFs with a minimum length of 30 amino acids. Long-reads sequencing was used to identify and validate full-length transcripts and potential splice variants. ReorientExpress [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>] was applied to perform reference-free orientation of transcriptomic long sequencing reads. Nanopore cDNA reads clustering was performed using RATTLE [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. For the identification of target peptide toxins, two complementary approaches were used: homology searches using BlastP/BlastX against the non-redundant (NR) database with an e-value cutoff of 0.001, and Cys-pattern searches using EMBOSS fuzzpro [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>].</p></section><section id="Sec22"><h3 class="pmc_sec_title">Production of recombinant toxin</h3>
<p id="Par35">To produce a recombinant analogue of MnTx-1, we followed our standard approach for disulfide-rich polypeptide production [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. Briefly, we used the pET-32b(+) vector (Novagen), which allows the production of target polypeptides as His-tagged fusions with Trx. Since MnTx-1 lacks methionine residues, we introduced a methionine site immediately before MnTx-1 to facilitate CNBr cleavage. Additionally, we used a mutated variant of the pET-32b vector (pET-32b<sup>M/L</sup>), in which all internal Met residues in Trx were replaced with Leu. The gene encoding MnTx-1 was synthesized using a two-step PCR with synthetic oligonucleotides (Table <a href="#MOESM1" class="usa-link">S1</a>) and inserted into pET-32b through <em>Kpn</em>I and <em>Bam</em>HI restriction, followed by ligation. The resulting vector, pET-32b<sup>M/L</sup>-MnTx-1, was then used to transform <em>Escherichia coli</em> SHuffle T7 Express cells (New England Biolabs), optimized for the expression of transgenes with T7 promoter and enhanced capacity to correctly fold proteins containing multiple disulfide bonds within the cytoplasm.</p>
<p id="Par36">The transformed bacteria were cultured in LB medium at 37 °C with vigorous stirring to the mid-log phase (A<sub>600</sub> ≈ 0.5). Expression was induced by adding 0.5 mM IPTG, and the bacteria were cultured at 24 °C for 16 h. Subsequently, the bacteria were harvested by centrifugation, and the pellet was resuspended in the start buffer for immobilized metal ion affinity chromatography (IMAC). The cells were lysed by ultrasonication, and the lysate was centrifuged. The soluble fraction was applied to a HisPur Cobalt Resin (Thermo Fisher Scientific), and the His-tagged fusion protein was purified by IMAC following the manufacturer’s protocol using imidazole for elution. Protein expression was verified by SDS-PAGE using 12% Mini-PROTEAN TGX gels (Bio-Rad). CNBr cleavage reaction was performed for Trx-MnTx-1 digestion [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>]. The target toxin was purified from the reaction solution via RP-HPLC using a Jupiter C<sub>5</sub> column (10 × 250 mm; particle size, 10 μm; pore size, 300 Å; Phenomenex) with a 60-min linear gradient of acetonitrile concentration (0–60%) in 0.1% TFA at a flow rate of 5 mL·min<sup>–1</sup>. A second RP-HPLC step was performed using a Luna C<sub>18</sub> column (4.6 × 250 mm; particle size, 5 μm; pore size, 100 Å; Phenomenex) applying a similar acetonitrile gradient (0–60% over 60 min) at a flow rate of 1 mL·min<sup>–1</sup>. The purity of the recombinant polypeptide was assessed by analytical HPLC and confirmed by MS. Concentrations of chimeric protein and the target toxin were calculated based on UV absorbance of aromatic amino acid residues at 280 nm.</p>
<p id="Par37">To increase the yield of target peptides, a refolding procedure was performed as described by Logashina et al. [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>]. For this, oxidized (GSSG) and reduced (GSH) glutathione (Sigma-Aldrich) were added at a concentration of 1 mM and 4 mM, respectively, to 1 mg of the chimeric protein purified immediately after IMAC. Refolding was conducted for three days at 16 °C, followed by digestion and purification as described above.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Mass spectrometry</h3>
<p id="Par38">Molecular mass measurements were performed using MALDI on an Ultraflex TOF-TOF (Bruker Daltonik) spectrometer, as previously described in detail [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>]. 2,5-Dihydroxybenzoic acid (Sigma-Aldrich) was used as the matrix. Measurements were performed in both linear and reflectron modes. Calibration was performed using the ProteoMass Peptide and Protein MALDI-MS Calibration Kit (Sigma-Aldrich). Mass spectra were analyzed using Data Analysis 4.3 (Bruker) and TOPPView [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>].</p></section><section id="Sec24"><h3 class="pmc_sec_title">NMR spectroscopy</h3>
<p id="Par39">All NMR experiments were conducted using the Avance II 700 MHz spectrometer (Bruker Biospin, Germany) at 30 °C. The protein sample was dissolved in H<sub>2</sub>O/D<sub>2</sub>O (19:1), and pH was adjusted to 7.4. <sup>1</sup>H, <sup>13</sup>C assignments were obtained via the standard procedure [<a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>], using 2D TOCSY, 2D NOESY (with 40 ms and 80 ms mixing times), <sup>13</sup>C-HSQC, and <sup>15</sup>N-HSQC spectra. After recording the initial set of spectra, the MnTx-1 sample was lyophilized and redissolved in 100% D<sub>2</sub>O (Acros Organics) to measure the proton-deuterium exchange and to record 2D NOESY (80 ms mixing time) and DQF-COSY spectra.</p>
<p id="Par40">The spatial structure was calculated using the simulated annealing/molecular dynamics protocol implemented in the CYANA software package (version 3.98.13) [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>]. Upper interproton distance restraints were determined by 1/r<sup>6</sup> calibration of NOESY cross-peaks. Torsion angle restraints and stereospecific assignments were obtained from J-couplings and NOE intensities. Hydrogen bonds and disulfide bonds were incorporated during the final stage of calculations, with disulfide linkages introduced based on NOE cross-peaks. Visual analysis of the calculated structures and figure drawings was performed using PyMOL software (Schrödinger). The structure was confirmed and deposited in the Protein Data Bank under the ID: 9JGG.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Two-electrode voltage-clamp</h3>
<p id="Par41">To screen the activity of venom fractions after separation, we expressed genes encoding the following voltage-gated potassium channels in <em>Xenopus laevis</em> oocytes: rK<sub>V</sub>1.1 (GenBank accession number: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_173095" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_173095</a>), rK<sub>V</sub>1.2 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_012970" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_012970</a>), hK<sub>V</sub>1.3 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_002232" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_002232</a>), rK<sub>V</sub>1.4 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_012971" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_012971</a>), rK<sub>V</sub>1.5 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_012972" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_012972</a>), rK<sub>V</sub>1.6 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_023954" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_023954</a>), rK<sub>V</sub>2.1 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_004975" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_004975</a>), hK<sub>V</sub>7.1 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_000218" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_000218</a>), hK<sub>V</sub>10.1 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_172362" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_172362</a>), hK<sub>V</sub>11.1 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_000238" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_000238</a>), and Shaker-IR from <em>Drosophila melanogaster</em> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_167595" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_167595</a>; residues 6–46 removed) as described [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>]. Linearized plasmids encoding the corresponding potassium channel genes were transcribed using the mMESSAGE mMACHINE SP6 or T7 Transcription Kits (Ambion) to prepare the respective cRNA. Oocytes were injected with 50 nL of cRNA at a concentration of 1 g·L<sup>–1</sup> using a micro-injector (Drummond Scientific). The oocytes were incubated at 16 °C in ND96 solution supplemented with 50 mg·L<sup>–1</sup> gentamicin sulfate.</p>
<p id="Par42">Electrophysiological recordings were conducted at room temperature (18–22 °C) using a GeneClamp 500B amplifier (Molecular Devices), controlled by a pClamp data acquisition system (Axon Instruments). Whole-cell currents from oocytes were recorded 1–4 days after cRNA injection. The bath solution composition was ND96. Voltage and current electrodes were filled with 3 M KCl, and electrode resistances were maintained at 0.7–1.5 MΩ. Depolarization-activated currents were sampled at 1 kHz and filtered at 0.5 kHz for K<sup>+</sup> currents, using a four-pole low-pass Bessel filter. Leak subtraction was performed using a −P/4 protocol.</p>
<p id="Par43">Currents were evoked by 500 ms step depolarization to the voltage corresponding to the maximal activation of the channels under control conditions, from a holding potential of −90 mV. Peak current amplitude was measured both before and after incubation with the sample. The time interval between depolarizing pulses was 8 s, allowing sufficient time for the channels to recover from inactivation. Eight pulses, totaling 1 min, were applied to achieve equilibrium block.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Automated patch-clamp electrophysiology</h3>
<p id="Par44">In-house generated stable HEK293 and CHO-K1 cell lines expressing the human orthologues of K<sub>V</sub>1.1 (GenBank accession number: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_000217" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_000217</a>), K<sub>V</sub>1.2 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_004974" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_004974</a>), K<sub>V</sub>1.3 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_002232" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_002232</a>), K<sub>V</sub>1.5 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_002234" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_002234</a>), K<sub>V</sub>1.6 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_172523" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_172523</a>), K<sub>V</sub>1.7 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_031886" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_031886</a>), K<sub>V</sub>2.1 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_004975" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_004975</a>), K<sub>V</sub>3.4 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_001039574.3" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_001039574.3</a>), and K<sub>V</sub>4.3 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_004980" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_004980</a>) were cultured at 37 °C with 5% CO<sub>2</sub> in DMEM and DMEM/F12 (Invitrogen, Life Technologies), respectively. CHO-K1 and HEK293T cells stably expressing human Na<sub>V</sub>1.7 (ChanClone) and Na<sub>V</sub>1.8 isoform (Scottish Biomedical) were cultured according to the manufacturers’ instructions. The culture medium was supplemented with 10% fetal bovine serum (Bovigen), 2 mM GlutaMAX, 100 U·mL<sup>–1</sup> penicillin, 10 mg·L<sup>–1</sup> streptomycin (Invitrogen), and/or the selection antibiotics blasticidin, G418 (Gibco) and hygromycin (Invitrogen). Cells were detached using TrypLE, resuspended in cold HBSS at a concentration of 100,000 cells·mL<sup>–1</sup>, and maintained in suspension in a Cell Hotel reservoir at 10 °C with shaking at 200 rpm.</p>
<p id="Par45">Concentration-response relationships were determined using automated, high throughput patch-clamp electrophysiology on a SyncroPatch 384PE (Nanion Technologies, Germany) integrated with a Biomek FX pipetting robot (Beckman Coulter). High resistance, four-hole-per-well borosilicate glass patch clamp chips (Nanion Technologies) were used. Whole-cell patch-clamp experiments were conducted at room temperature (21–23 °C) with full compensation for chip and whole-cell capacitance and 70% series resistance compensation. Peak K<sup>+</sup> currents (I<sub>K</sub>) were elicited by either 50 ms or 150 ms test pulses to 20 mV from a holding potential of −100 mV at 0.1 Hz.</p>
<p id="Par46">The intracellular solution consisted of 70 mM KCl, 60 mM KF, 10 mM NaCl, 10 mM EGTA, and 10 mM HEPES (pH 7.2, adjusted with KOH, 280 ± 3 mOsm). The extracellular solution contained 140 mM NaCl, 4 mM KCl, 2 mM CaCl<sub>2</sub>, 1 mM MgCl<sub>2</sub>, 5 mM glucose, and 10 mM HEPES (pH 7.4, adjusted with NaOH, 298 ± 3 mOsm). Seals were facilitated by brief exposure to a seal enhancer composed of 60 mM NMDG, 40 mM NaCl, 4 mM KCl, 35 mM CaCl<sub>2</sub>, 1 mM MgCl<sub>2</sub>, 5 mM glucose, and 10 mM HEPES (pH 7.3, adjusted with NaOH, 298 ± 3 mOsm), followed by thorough washing with the standard extracellular solution. All solutions were filtered using a 0.22 μm membrane, and lyophilized peptides were dissolved in extracellular solution on the day of experiments and diluted to final test concentrations.</p>
<p id="Par47">Data acquisition and analysis were conducted using PatchControl 384 and DataControl 384 software (Nanion Technologies). Only cells with a seal resistance greater than 300 MΩ were included. Leak currents were subtracted using a −P/4 protocol.</p>
<p id="Par48">Voltage dependence of block and reversibility was assessed using manual patch-clamp, employing the same recording solutions as used in the automated patch-clamp experiments. The K<sub>V</sub>1.2<sup>V381R</sup> pore mutant construct was described previously [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. Manual patch-clamp recordings were performed using a MultiClamp 700B amplifier, digitized with a Digidata 1550, and controlled via Clampex 11.1 software (Molecular Devices). Patch pipettes were pulled from borosilicate glass (GC150F-15, Harvard Apparatus), and were fire-polished to a resistance of 1−3 MΩ. To minimize inactivation or rundown, a holding potential of −100 mV was used, with currents evoked by 25−150 ms depolarizing pulses to +20 mV at 0.1 Hz. Whole-cell currents were sampled at 100 kHz, filtered at 10 kHz, and series resistance was compensated by 60−80%.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Data analysis</h3>
<p id="Par49">Offline analysis was performed using DataControl 384 and Igor Pro 6.37 software (WaveMetrics). K<sub>V</sub> peak currents measured in the presence of increasing MnTx-1 concentrations (I<sub>MnTx1</sub>) were divided by control currents (I<sub>Ctr</sub>) to generate a concentration-response curve, which was fitted with a Hill equation:</p>
<table class="disp-formula p" id="Equa"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/860a/12334782/5d13494d9e52/18_2025_5778_Article_Equa.gif" loading="lazy" height="48" width="192" alt="graphic file with name 18_2025_5778_Article_Equa.gif"></td></tr></table>
<p> where IC<sub>50</sub> is the half-maximal inhibitory concentration, and h is the Hill coefficient (n<sub>H</sub>).</p>
<p id="Par50">Instantaneous current-voltage relationships were obtained from tail currents measured upon returning to −20 mV following 250 ms depolarizing pre-pulses from −80 to 50 mV in 10-mV steps (holding potential: −80 mV; 0.1 Hz). In the presence and absence of 10 nM MnTx-1, activation curves were generated by plotting normalized tail currents against pre-pulse potentials and fitting them to a Boltzmann function. Fractional inhibition (I<sub>MnTx1</sub>/I<sub>Ctr</sub>) was calculated from peak currents recorded using the same I–V protocol.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Statistics</h3>
<p id="Par51">Data and statistical analysis comply with recommendations on experimental design and analysis in pharmacology [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>]. All data points in the peptide concentration-response curves on channels are mean ± standard error of the mean (SEM), with <em>n</em> = 4–8 for each concentration. IC<sub>50</sub>s are reported with 95% confidence intervals (CI). A value of <em>p</em> &lt; 0.05 was considered statistically significant.</p></section></section><section id="Sec29"><h2 class="pmc_sec_title">Electronic supplementary material</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12334782/bin/18_2025_5778_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (1.2MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12334782/bin/18_2025_5778_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (478.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12334782/bin/18_2025_5778_MOESM3_ESM.fasta" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (1.4KB, fasta) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Dr Steve Peigneur (KU Leuven, Belgium) for his guidance during the initial testing of <em>P. murinus</em> venom and its fractions. We are grateful to Dr Eugene Rogozhin (Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry) for Edman sequencing. NGS sequencing for this work was performed using core facilities of the Lopukhin FRCC PCM “Genomics, proteomics, metabolomics”.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>A.I.K. and A.A.V. designed this research, A.I.K., V.A.I., V.A.L., Y.A.D., V.V.B., D.V.O., A.A.B., and R.K.F-U. performed the experiments, A.I.K., V.A.L., V.V.B., A.A.B., and R.K.F-U. analyzed the data, A.I.K., V.A.L., V.V.B., D.V.O., D.J.A., R.K.F-U., and A.A.V. wrote the manuscript. A.I.K., J.T., E.V.B., D.J.A., and A.A.V. supervised the project. The manuscript was finalized by all authors.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study was supported by the Russian Science Foundation (grant no. 22-74-10028) and Australian Research Council LIEF (LE180100066) and Discovery Grants (DP210102405) to D.J.A.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The data that support the findings of this study are available from the corresponding authors, upon request.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval</h3>
<p id="Par52">All procedures involving animals were performed following the ARRIVE 2.0 guidelines (Animal Research: Reporting of In Vivo Experiments) and the “European Convention for the Protection of Vertebrate Animals Used for Experimental and Other Scientific Purposes” (Strasbourg, 18.III.1986). The use of the frogs was in accordance with the license number LA1210239 to Toxicology and Pharmacology, KU Leuven, and was approved by the Ethical Committee for animal experiments of KU Leuven (P186/2019). Maximum efforts were made to minimize animal suffering and the number of animals used in the experiments.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par53">The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Significance statement</h3>
<p id="Par54">Since the discovery of hanatoxin, spider toxins have been widely recognized as voltage sensor modulators of potassium channels. In our research, we identified unique toxins (murinotoxins or MnTx) that inhibit the K<sub>V</sub> channel pore directly, rather than modulating voltage sensors. Remarkably, these molecules adopt a novel type of fold, distinct from the canonical inhibitor cystine knot (ICK) motif characteristic of spider toxins. The unique pharmacological and structural features of MnTx highlight their significance for both fundamental research and practical applications. These toxins offer exciting potential as innovative tools for probing membrane transport mechanisms and as templates for the development of novel drug candidates.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Alexey I. Kuzmenkov, Email: aleksey.kuzmenkov@gmail.com</p>
<p>David J. Adams, Email: djadams@uow.edu.au</p>
<p>Alexander A. Vassilevski, Email: avas@ibch.ru</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Muttenthaler M, King GF, Adams DJ, Alewood PF (2021) Trends in peptide drug discovery. Nat Rev Drug Discov 20:309–325
</cite> [<a href="https://doi.org/10.1038/s41573-020-00135-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33536635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Muttenthaler%20M,%20King%20GF,%20Adams%20DJ,%20Alewood%20PF%20(2021)%20Trends%20in%20peptide%20drug%20discovery.%20Nat%20Rev%20Drug%20Discov%2020:309%E2%80%93325" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Bruno BJ, Miller GD, Lim CS (2013) Basics and recent advances in peptide and protein drug delivery. Ther Deliv 4:1443–1467
</cite> [<a href="https://doi.org/10.4155/tde.13.104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3956587/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24228993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bruno%20BJ,%20Miller%20GD,%20Lim%20CS%20(2013)%20Basics%20and%20recent%20advances%20in%20peptide%20and%20protein%20drug%20delivery.%20Ther%20Deliv%204:1443%E2%80%931467" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Mitragotri S, Burke PA, Langer R (2014) Overcoming the challenges in administering biopharmaceuticals: formulation and delivery strategies. Nat Rev Drug Discov 13:655–672
</cite> [<a href="https://doi.org/10.1038/nrd4363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4455970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25103255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mitragotri%20S,%20Burke%20PA,%20Langer%20R%20(2014)%20Overcoming%20the%20challenges%20in%20administering%20biopharmaceuticals:%20formulation%20and%20delivery%20strategies.%20Nat%20Rev%20Drug%20Discov%2013:655%E2%80%93672" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Fetse J, Kandel S, Mamani UF, Cheng K (2023) Recent advances in the development of therapeutic peptides. Trends Pharmacol Sci 44:425–441
</cite> [<a href="https://doi.org/10.1016/j.tips.2023.04.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10330351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37246037/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fetse%20J,%20Kandel%20S,%20Mamani%20UF,%20Cheng%20K%20(2023)%20Recent%20advances%20in%20the%20development%20of%20therapeutic%20peptides.%20Trends%20Pharmacol%20Sci%2044:425%E2%80%93441" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Lau JL, Dunn MK (2018) Therapeutic peptides: historical perspectives, current development trends, and future directions. Bioorg Med Chem 26:2700–2707
</cite> [<a href="https://doi.org/10.1016/j.bmc.2017.06.052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28720325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lau%20JL,%20Dunn%20MK%20(2018)%20Therapeutic%20peptides:%20historical%20perspectives,%20current%20development%20trends,%20and%20future%20directions.%20Bioorg%20Med%20Chem%2026:2700%E2%80%932707" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Lewis RJ, Garcia ML (2003) Therapeutic potential of venom peptides. Nat Rev Drug Discov 2:790–802
</cite> [<a href="https://doi.org/10.1038/nrd1197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14526382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lewis%20RJ,%20Garcia%20ML%20(2003)%20Therapeutic%20potential%20of%20venom%20peptides.%20Nat%20Rev%20Drug%20Discov%202:790%E2%80%93802" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Holford M, Daly M, King GF, Norton RS (2018) Venoms to the rescue. Sci (1979) 361:842–843</cite> [<a href="https://doi.org/10.1126/science.aau7761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30166472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Holford%20M,%20Daly%20M,%20King%20GF,%20Norton%20RS%20(2018)%20Venoms%20to%20the%20rescue.%20Sci%20(1979)%20361:842%E2%80%93843" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Menting JG et al (2016) A minimized human insulin-receptor-binding motif revealed in a Conus geographus venom insulin. Nat Struct Mol Biol 23:916–920
</cite> [<a href="https://doi.org/10.1038/nsmb.3292" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27617429/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Menting%20JG%20et%20al%20(2016)%20A%20minimized%20human%20insulin-receptor-binding%20motif%20revealed%20in%20a%20Conus%20geographus%20venom%20insulin.%20Nat%20Struct%20Mol%20Biol%2023:916%E2%80%93920" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Beeton C, Pennington MW, Norton RS (2011) Analogs of the sea anemone potassium channel blocker ShK for the treatment of autoimmune diseases. Inflamm Allergy Drug Targets 10:313–321
</cite> [<a href="https://doi.org/10.2174/187152811797200641" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3398462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21824083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beeton%20C,%20Pennington%20MW,%20Norton%20RS%20(2011)%20Analogs%20of%20the%20sea%20anemone%20potassium%20channel%20blocker%20ShK%20for%20the%20treatment%20of%20autoimmune%20diseases.%20Inflamm%20Allergy%20Drug%20Targets%2010:313%E2%80%93321" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Mamelak AN et al (2006) Phase I single-dose study of intracavitary-administered iodine-131-TM-601 in adults with recurrent high-grade glioma. J Clin Oncol 24:3644–3650
</cite> [<a href="https://doi.org/10.1200/JCO.2005.05.4569" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16877732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mamelak%20AN%20et%20al%20(2006)%20Phase%20I%20single-dose%20study%20of%20intracavitary-administered%20iodine-131-TM-601%20in%20adults%20with%20recurrent%20high-grade%20glioma.%20J%20Clin%20Oncol%2024:3644%E2%80%933650" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Redd MA et al (2021) Therapeutic inhibition of acid-sensing ion channel 1a recovers heart function after ischemia-reperfusion injury. Circulation 144:947–960
</cite> [<a href="https://doi.org/10.1161/CIRCULATIONAHA.121.054360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34264749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Redd%20MA%20et%20al%20(2021)%20Therapeutic%20inhibition%20of%20acid-sensing%20ion%20channel%201a%20recovers%20heart%20function%20after%20ischemia-reperfusion%20injury.%20Circulation%20144:947%E2%80%93960" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Chassagnon IR et al (2017) Potent neuroprotection after stroke afforded by a double-knot spider-venom peptide that inhibits acid-sensing ion channel 1a. Proc Natl Acad Sci U S A 114:3750–3755
</cite> [<a href="https://doi.org/10.1073/pnas.1614728114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5389327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28320941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chassagnon%20IR%20et%20al%20(2017)%20Potent%20neuroprotection%20after%20stroke%20afforded%20by%20a%20double-knot%20spider-venom%20peptide%20that%20inhibits%20acid-sensing%20ion%20channel%201a.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20114:3750%E2%80%933755" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Grishin EV et al (2010) Novel peptide from spider venom inhibits P2X3 receptors and inflammatory pain. Ann Neurol 67:680–683
</cite> [<a href="https://doi.org/10.1002/ana.21949" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20437566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grishin%20EV%20et%20al%20(2010)%20Novel%20peptide%20from%20spider%20venom%20inhibits%20P2X3%20receptors%20and%20inflammatory%20pain.%20Ann%20Neurol%2067:680%E2%80%93683" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Reimers C et al (2017) Identification of a cono-RFamide from the venom of Conus textile that targets ASIC3 and enhances muscle pain. Proc Natl Acad Sci U S A 114:E3507–E3515
</cite> [<a href="https://doi.org/10.1073/pnas.1616232114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5410773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28396446/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reimers%20C%20et%20al%20(2017)%20Identification%20of%20a%20cono-RFamide%20from%20the%20venom%20of%20Conus%20textile%20that%20targets%20ASIC3%20and%20enhances%20muscle%20pain.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20114:E3507%E2%80%93E3515" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Yang S et al (2013) Discovery of a selective NaV1.7 inhibitor from centipede venom with analgesic efficacy exceeding morphine in rodent pain models. Proc Natl Acad Sci U S A 110:17534–17539
</cite> [<a href="https://doi.org/10.1073/pnas.1306285110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3808613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24082113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20S%20et%20al%20(2013)%20Discovery%20of%20a%20selective%20NaV1.7%20inhibitor%20from%20centipede%20venom%20with%20analgesic%20efficacy%20exceeding%20morphine%20in%20rodent%20pain%20models.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20110:17534%E2%80%9317539" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Murray JK et al (2015) Engineering potent and selective analogues of GpTx-1, a tarantula venom peptide antagonist of the Na(V)1.7 sodium channel. J Med Chem 58:2299–2314
</cite> [<a href="https://doi.org/10.1021/jm501765v" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25658507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murray%20JK%20et%20al%20(2015)%20Engineering%20potent%20and%20selective%20analogues%20of%20GpTx-1,%20a%20tarantula%20venom%20peptide%20antagonist%20of%20the%20Na(V)1.7%20sodium%20channel.%20J%20Med%20Chem%2058:2299%E2%80%932314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Kalia J et al (2015) From foe to friend: using animal toxins to investigate ion channel function. J Mol Biol 427:158–175
</cite> [<a href="https://doi.org/10.1016/j.jmb.2014.07.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4277912/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25088688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kalia%20J%20et%20al%20(2015)%20From%20foe%20to%20friend:%20using%20animal%20toxins%20to%20investigate%20ion%20channel%20function.%20J%20Mol%20Biol%20427:158%E2%80%93175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Hille B (2001) Ion channels of excitable membranes, 3rd edn. Sinauer Associates Inc Sunderland</cite>
</li>
<li id="CR19">
<span class="label">19.</span><cite>Harding SD et al (2024) The IUPHAR/BPS guide to PHARMACOLOGY in 2024. Nucleic Acids Res 52:D1438–D1449
</cite> [<a href="https://doi.org/10.1093/nar/gkad944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37897341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harding%20SD%20et%20al%20(2024)%20The%20IUPHAR/BPS%20guide%20to%20PHARMACOLOGY%20in%202024.%20Nucleic%20Acids%20Res%2052:D1438%E2%80%93D1449" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Santos R et al (2017) A comprehensive map of molecular drug targets. Nat Rev Drug Discov 16:19–34
</cite> [<a href="https://doi.org/10.1038/nrd.2016.230" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6314433/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27910877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Santos%20R%20et%20al%20(2017)%20A%20comprehensive%20map%20of%20molecular%20drug%20targets.%20Nat%20Rev%20Drug%20Discov%2016:19%E2%80%9334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Wulff H, Christophersen P, Colussi P, Chandy KG, Yarov-Yarovoy V (2019) Antibodies and venom peptides: new modalities for ion channels. Nat Rev Drug Discov 18:339–357
</cite> [<a href="https://doi.org/10.1038/s41573-019-0013-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6499689/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30728472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wulff%20H,%20Christophersen%20P,%20Colussi%20P,%20Chandy%20KG,%20Yarov-Yarovoy%20V%20(2019)%20Antibodies%20and%20venom%20peptides:%20new%20modalities%20for%20ion%20channels.%20Nat%20Rev%20Drug%20Discov%2018:339%E2%80%93357" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Alexander SP et al (2021) THE CONCISE GUIDE TO PHARMACOLOGY 2021/22: ion channels. Br J Pharmacol 178(Suppl 1):S157–S245
</cite> [<a href="https://doi.org/10.1111/bph.15539" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34529831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alexander%20SP%20et%20al%20(2021)%20THE%20CONCISE%20GUIDE%20TO%20PHARMACOLOGY%202021/22:%20ion%20channels.%20Br%20J%20Pharmacol%20178(Suppl%201):S157%E2%80%93S245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Ashcroft FM (2006) From molecule to malady. Nature 440:440–447
</cite> [<a href="https://doi.org/10.1038/nature04707" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16554803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ashcroft%20FM%20(2006)%20From%20molecule%20to%20malady.%20Nature%20440:440%E2%80%93447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Wulff H, Castle NA, Pardo LA (2009) Voltage-gated potassium channels as therapeutic targets. Nat Rev Drug Discov 8:982–1001
</cite> [<a href="https://doi.org/10.1038/nrd2983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2790170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19949402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wulff%20H,%20Castle%20NA,%20Pardo%20LA%20(2009)%20Voltage-gated%20potassium%20channels%20as%20therapeutic%20targets.%20Nat%20Rev%20Drug%20Discov%208:982%E2%80%931001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Krylov NA, Tabakmakher VM, Yureva DA, Vassilevski AA, Kuzmenkov AI (2023) Kalium 3.0 is a comprehensive depository of natural, artificial, and labeled polypeptides acting on potassium channels. Protein Sci 32:e4776
</cite> [<a href="https://doi.org/10.1002/pro.4776" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10578113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37682529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krylov%20NA,%20Tabakmakher%20VM,%20Yureva%20DA,%20Vassilevski%20AA,%20Kuzmenkov%20AI%20(2023)%20Kalium%203.0%20is%20a%20comprehensive%20depository%20of%20natural,%20artificial,%20and%20labeled%20polypeptides%20acting%20on%20potassium%20channels.%20Protein%20Sci%2032:e4776" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Paesen GC et al (2009) An ion-channel modulator from the saliva of the brown ear tick has a highly modified Kunitz/BPTI structure. J Mol Biol 389:734–747
</cite> [<a href="https://doi.org/10.1016/j.jmb.2009.04.045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19394347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paesen%20GC%20et%20al%20(2009)%20An%20ion-channel%20modulator%20from%20the%20saliva%20of%20the%20brown%20ear%20tick%20has%20a%20highly%20modified%20Kunitz/BPTI%20structure.%20J%20Mol%20Biol%20389:734%E2%80%93747" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Jin J et al (2020) Weaponisation on the fly: convergent recruitment of Knottin and defensin peptide scaffolds into the venom of predatory assassin flies. Insect Biochem Mol Biol 118:103310
</cite> [<a href="https://doi.org/10.1016/j.ibmb.2019.103310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31870846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin%20J%20et%20al%20(2020)%20Weaponisation%20on%20the%20fly:%20convergent%20recruitment%20of%20Knottin%20and%20defensin%20peptide%20scaffolds%20into%20the%20venom%20of%20predatory%20assassin%20flies.%20Insect%20Biochem%20Mol%20Biol%20118:103310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Kuzmenkov AI, Grishin EV, Vassilevski AA (2015) Diversity of potassium channel ligands: focus on scorpion toxins. Biochem (Moscow) 80:1764–1799</cite> [<a href="https://doi.org/10.1134/S0006297915130118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26878580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuzmenkov%20AI,%20Grishin%20EV,%20Vassilevski%20AA%20(2015)%20Diversity%20of%20potassium%20channel%20ligands:%20focus%20on%20scorpion%20toxins.%20Biochem%20(Moscow)%2080:1764%E2%80%931799" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Finol-Urdaneta RK et al (2020) Marine toxins targeting Kv1 channels: pharmacological tools and therapeutic scaffolds. Mar Drugs 18(3):173. 10.3390/md18030173</cite> [<a href="https://doi.org/10.3390/md18030173" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7143316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32245015/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Miller C (1988) Competition for block of a Ca2(+)-activated K + channel by charybdotoxin and tetraethylammonium. Neuron 1:1003–1006
</cite> [<a href="https://doi.org/10.1016/0896-6273(88)90157-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2483092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miller%20C%20(1988)%20Competition%20for%20block%20of%20a%20Ca2(+)-activated%20K%E2%80%89+%E2%80%89channel%20by%20charybdotoxin%20and%20tetraethylammonium.%20Neuron%201:1003%E2%80%931006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>MacKinnon R, Miller C (1989) Mutant potassium channels with altered binding of charybdotoxin, a pore-blocking peptide inhibitor. Sci (1979) 245:1382–1385</cite> [<a href="https://doi.org/10.1126/science.2476850" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2476850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?MacKinnon%20R,%20Miller%20C%20(1989)%20Mutant%20potassium%20channels%20with%20altered%20binding%20of%20charybdotoxin,%20a%20pore-blocking%20peptide%20inhibitor.%20Sci%20(1979)%20245:1382%E2%80%931385" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Swartz KJ, MacKinnon R (1995) An inhibitor of the Kv2.1 potassium channel isolated from the venom of a Chilean tarantula. Neuron 15:941–949
</cite> [<a href="https://doi.org/10.1016/0896-6273(95)90184-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7576642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Swartz%20KJ,%20MacKinnon%20R%20(1995)%20An%20inhibitor%20of%20the%20Kv2.1%20potassium%20channel%20isolated%20from%20the%20venom%20of%20a%20Chilean%20tarantula.%20Neuron%2015:941%E2%80%93949" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Swartz KJ, MacKinnon R (1997) Hanatoxin modifies the gating of a voltage-dependent K + channel through multiple binding sites. Neuron 18:665–673
</cite> [<a href="https://doi.org/10.1016/s0896-6273(00)80306-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9136774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Swartz%20KJ,%20MacKinnon%20R%20(1997)%20Hanatoxin%20modifies%20the%20gating%20of%20a%20voltage-dependent%20K%E2%80%89+%E2%80%89channel%20through%20multiple%20binding%20sites.%20Neuron%2018:665%E2%80%93673" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Norton RS, Chandy KG (2017) Venom-derived peptide inhibitors of voltage-gated potassium channels. Neuropharmacology 127:124–138
</cite> [<a href="https://doi.org/10.1016/j.neuropharm.2017.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28689025/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Norton%20RS,%20Chandy%20KG%20(2017)%20Venom-derived%20peptide%20inhibitors%20of%20voltage-gated%20potassium%20channels.%20Neuropharmacology%20127:124%E2%80%93138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Tabakmakher VM, Krylov NA, Kuzmenkov AI, Efremov RG, Vassilevski AA (2019) Kalium 2.0, a comprehensive database of polypeptide ligands of potassium channels. Sci Data 6:73
</cite> [<a href="https://doi.org/10.1038/s41597-019-0074-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6536513/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31133708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tabakmakher%20VM,%20Krylov%20NA,%20Kuzmenkov%20AI,%20Efremov%20RG,%20Vassilevski%20AA%20(2019)%20Kalium%202.0,%20a%20comprehensive%20database%20of%20polypeptide%20ligands%20of%20potassium%20channels.%20Sci%20Data%206:73" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>World Spider Catalog (2024) Version 25.5. Natural History Museum Bern, online at <a href="http://wsc.nmbe.ch" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://wsc.nmbe.ch</a></cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Charpentier P (1993) Pterinochilus murinus: morphology - biology - behaviour. Exothermae 1:1–72</cite> [<a href="https://scholar.google.com/scholar_lookup?Charpentier%20P%20(1993)%20Pterinochilus%20murinus:%20morphology%20-%20biology%20-%20behaviour.%20Exothermae%201:1%E2%80%9372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Smith AM (1990) Baboon spiders: tarantulas of Africa and the Middle East. Fitzgerald Publishing, London</cite> [<a href="https://scholar.google.com/scholar_lookup?Smith%20AM%20(1990)%20Baboon%20spiders:%20tarantulas%20of%20Africa%20and%20the%20Middle%20East.%20Fitzgerald%20Publishing,%20London" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Ahmed N, Pinkham M, Warrell DA (2009) Symptom in search of a toxin: muscle spasms following bites by old world tarantula spiders (Lampropelma nigerrimum, Pterinochilus murinus, Poecilotheria regalis) with review. QJM 102:851–857
</cite> [<a href="https://doi.org/10.1093/qjmed/hcp128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19776152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahmed%20N,%20Pinkham%20M,%20Warrell%20DA%20(2009)%20Symptom%20in%20search%20of%20a%20toxin:%20muscle%20spasms%20following%20bites%20by%20old%20world%20tarantula%20spiders%20(Lampropelma%20nigerrimum,%20Pterinochilus%20murinus,%20Poecilotheria%20regalis)%20with%20review.%20QJM%20102:851%E2%80%93857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>de la Rubia I et al (2022) RATTLE: reference-free reconstruction and quantification of transcriptomes from nanopore sequencing. Genome Biol 23:153. 10.1186/s13059-022-02715-w</cite> [<a href="https://doi.org/10.1186/s13059-022-02715-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9264490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35804393/" class="usa-link">PubMed</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Gigolaev AM et al (2020) Tuning scorpion toxin selectivity: switching from KV1.1 to KV1.3. Front Pharmacol 11:1010
</cite> [<a href="https://doi.org/10.3389/fphar.2020.01010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7358528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32733247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gigolaev%20AM%20et%20al%20(2020)%20Tuning%20scorpion%20toxin%20selectivity:%20switching%20from%20KV1.1%20to%20KV1.3.%20Front%20Pharmacol%2011:1010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Luna-Ramirez K et al (2020) Structural basis of the potency and selectivity of urotoxin, a potent Kv1 blocker from scorpion venom. Biochem Pharmacol 174:113782
</cite> [<a href="https://doi.org/10.1016/j.bcp.2019.113782" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31881193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luna-Ramirez%20K%20et%20al%20(2020)%20Structural%20basis%20of%20the%20potency%20and%20selectivity%20of%20urotoxin,%20a%20potent%20Kv1%20blocker%20from%20scorpion%20venom.%20Biochem%20Pharmacol%20174:113782" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Cordeiro S et al (2019) Conotoxin κM-RIIIJ, a tool targeting asymmetric heteromeric Kv1 channels. Proc Natl Acad Sci U S A 116:1059–1064
</cite> [<a href="https://doi.org/10.1073/pnas.1813161116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6338859/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30593566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cordeiro%20S%20et%20al%20(2019)%20Conotoxin%20%CE%BAM-RIIIJ,%20a%20tool%20targeting%20asymmetric%20heteromeric%20Kv1%20channels.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20116:1059%E2%80%931064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Liao Z et al (2007) Proteomic and peptidomic analysis of the venom from Chinese tarantula Chilobrachys jingzhao. Proteomics 7:1892–1907
</cite> [<a href="https://doi.org/10.1002/pmic.200600785" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17476710/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liao%20Z%20et%20al%20(2007)%20Proteomic%20and%20peptidomic%20analysis%20of%20the%20venom%20from%20Chinese%20tarantula%20Chilobrachys%20jingzhao.%20Proteomics%207:1892%E2%80%931907" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Chen J et al (2008) Molecular diversity and evolution of cystine knot toxins of the tarantula Chilobrachys jingzhao. Cell Mol Life Sci 65:2431–2444
</cite> [<a href="https://doi.org/10.1007/s00018-008-8135-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11131836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18581053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20J%20et%20al%20(2008)%20Molecular%20diversity%20and%20evolution%20of%20cystine%20knot%20toxins%20of%20the%20tarantula%20Chilobrachys%20jingzhao.%20Cell%20Mol%20Life%20Sci%2065:2431%E2%80%932444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Jiang L et al (2008) Molecular diversification based on analysis of expressed sequence tags from the venom glands of the Chinese bird spider Ornithoctonus huwena. Toxicon 51:1479–1489
</cite> [<a href="https://doi.org/10.1016/j.toxicon.2008.03.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18482741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20L%20et%20al%20(2008)%20Molecular%20diversification%20based%20on%20analysis%20of%20expressed%20sequence%20tags%20from%20the%20venom%20glands%20of%20the%20Chinese%20bird%20spider%20Ornithoctonus%20huwena.%20Toxicon%2051:1479%E2%80%931489" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Oldrati V et al (2017) Peptidomic and transcriptomic profiling of four distinct spider venoms. PLoS ONE 12:e0172966
</cite> [<a href="https://doi.org/10.1371/journal.pone.0172966" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5357004/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28306751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oldrati%20V%20et%20al%20(2017)%20Peptidomic%20and%20transcriptomic%20profiling%20of%20four%20distinct%20spider%20venoms.%20PLoS%20ONE%2012:e0172966" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Kozlov S et al (2005) A novel strategy for the identification of toxinlike structures in spider venom. Proteins 59:131–140
</cite> [<a href="https://doi.org/10.1002/prot.20390" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15688451/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kozlov%20S%20et%20al%20(2005)%20A%20novel%20strategy%20for%20the%20identification%20of%20toxinlike%20structures%20in%20spider%20venom.%20Proteins%2059:131%E2%80%93140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Kozlov SA, Grishin EV (2007) The universal algorithm of maturation for secretory and excretory protein precursors. Toxicon 49:721–726
</cite> [<a href="https://doi.org/10.1016/j.toxicon.2006.11.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17196238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kozlov%20SA,%20Grishin%20EV%20(2007)%20The%20universal%20algorithm%20of%20maturation%20for%20secretory%20and%20excretory%20protein%20precursors.%20Toxicon%2049:721%E2%80%93726" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Pallaghy PK, Norton RS, Nielsen KJ, Craik DJ (1994) A common structural motif incorporating a cystine knot and a triple-stranded beta-sheet in toxic and inhibitory polypeptides. Protein Sci 3:1833–1839
</cite> [<a href="https://doi.org/10.1002/pro.5560031022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2142598/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7849598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pallaghy%20PK,%20Norton%20RS,%20Nielsen%20KJ,%20Craik%20DJ%20(1994)%20A%20common%20structural%20motif%20incorporating%20a%20cystine%20knot%20and%20a%20triple-stranded%20beta-sheet%20in%20toxic%20and%20inhibitory%20polypeptides.%20Protein%20Sci%203:1833%E2%80%931839" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Swartz KJ, MacKinnon R (1997) Mapping the receptor site for hanatoxin, a gating modifier of voltage-dependent K + channels. Neuron 18:675–682
</cite> [<a href="https://doi.org/10.1016/s0896-6273(00)80307-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9136775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Swartz%20KJ,%20MacKinnon%20R%20(1997)%20Mapping%20the%20receptor%20site%20for%20hanatoxin,%20a%20gating%20modifier%20of%20voltage-dependent%20K%E2%80%89+%E2%80%89channels.%20Neuron%2018:675%E2%80%93682" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Wang Y et al (2021) Effects and mechanism of gating modifier spider toxins on the hERG channel. Toxicon 189:56–64
</cite> [<a href="https://doi.org/10.1016/j.toxicon.2020.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33212100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Y%20et%20al%20(2021)%20Effects%20and%20mechanism%20of%20gating%20modifier%20spider%20toxins%20on%20the%20hERG%20channel.%20Toxicon%20189:56%E2%80%9364" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Li-Smerin Y, Swartz KJ (1998) Gating modifier toxins reveal a conserved structural motif in voltage-gated Ca2 + and K + channels. Proc Natl Acad Sci U S A 95:8585–8589
</cite> [<a href="https://doi.org/10.1073/pnas.95.15.8585" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC21119/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9671721/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li-Smerin%20Y,%20Swartz%20KJ%20(1998)%20Gating%20modifier%20toxins%20reveal%20a%20conserved%20structural%20motif%20in%20voltage-gated%20Ca2%E2%80%89+%E2%80%89and%20K%E2%80%89+%E2%80%89channels.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%2095:8585%E2%80%938589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Middleton RE et al (2002) Two tarantula peptides inhibit activation of multiple sodium channels. Biochemistry 41:14734–14747
</cite> [<a href="https://doi.org/10.1021/bi026546a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12475222/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Middleton%20RE%20et%20al%20(2002)%20Two%20tarantula%20peptides%20inhibit%20activation%20of%20multiple%20sodium%20channels.%20Biochemistry%2041:14734%E2%80%9314747" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Redaelli E et al (2010) Target promiscuity and heterogeneous effects of tarantula venom peptides affecting Na + and K + ion channels. J Biol Chem 285:4130–4142
</cite> [<a href="https://doi.org/10.1074/jbc.M109.054718" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2823553/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19955179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Redaelli%20E%20et%20al%20(2010)%20Target%20promiscuity%20and%20heterogeneous%20effects%20of%20tarantula%20venom%20peptides%20affecting%20Na%E2%80%89+%E2%80%89and%20K%E2%80%89+%E2%80%89ion%20channels.%20J%20Biol%20Chem%20285:4130%E2%80%934142" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Finol-Urdaneta RK et al (2022) Multitarget nociceptor sensitization by a promiscuous peptide from the venom of the King baboon spider. Proc Natl Acad Sci U S A 119:e2110932119
</cite> [<a href="https://doi.org/10.1073/pnas.2110932119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8812547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35074873/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Finol-Urdaneta%20RK%20et%20al%20(2022)%20Multitarget%20nociceptor%20sensitization%20by%20a%20promiscuous%20peptide%20from%20the%20venom%20of%20the%20King%20baboon%20spider.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20119:e2110932119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Ma L et al (2018) Novel venom-derived inhibitors of the human EAG channel, a putative antiepileptic drug target. Biochem Pharmacol 158:60–72
</cite> [<a href="https://doi.org/10.1016/j.bcp.2018.08.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30149017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma%20L%20et%20al%20(2018)%20Novel%20venom-derived%20inhibitors%20of%20the%20human%20EAG%20channel,%20a%20putative%20antiepileptic%20drug%20target.%20Biochem%20Pharmacol%20158:60%E2%80%9372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Gunning SJ et al (2008) The Janus-faced atracotoxins are specific blockers of invertebrate K(Ca) channels. FEBS J 275:4045–4059
</cite> [<a href="https://doi.org/10.1111/j.1742-4658.2008.06545.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18625007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gunning%20SJ%20et%20al%20(2008)%20The%20Janus-faced%20atracotoxins%20are%20specific%20blockers%20of%20invertebrate%20K(Ca)%20channels.%20FEBS%20J%20275:4045%E2%80%934059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Windley MJ, Escoubas P, Valenzuela SM, Nicholson GM (2011) A novel family of insect-selective peptide neurotoxins targeting insect large-conductance calcium-activated K + channels isolated from the venom of the theraphosid spider Eucratoscelus constrictus. Mol Pharmacol 80:1–13
</cite> [<a href="https://doi.org/10.1124/mol.110.070540" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21447641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Windley%20MJ,%20Escoubas%20P,%20Valenzuela%20SM,%20Nicholson%20GM%20(2011)%20A%20novel%20family%20of%20insect-selective%20peptide%20neurotoxins%20targeting%20insect%20large-conductance%20calcium-activated%20K%E2%80%89+%E2%80%89channels%20isolated%20from%20the%20venom%20of%20the%20theraphosid%20spider%20Eucratoscelus%20constrictus.%20Mol%20Pharmacol%2080:1%E2%80%9313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Yuan CH et al (2008) Discovery of a distinct superfamily of Kunitz-type toxin (KTT) from tarantulas. PLoS ONE 3:e3414
</cite> [<a href="https://doi.org/10.1371/journal.pone.0003414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2561067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18923708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan%20CH%20et%20al%20(2008)%20Discovery%20of%20a%20distinct%20superfamily%20of%20Kunitz-type%20toxin%20(KTT)%20from%20tarantulas.%20PLoS%20ONE%203:e3414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Jiang L, Deng M, Duan Z, Tang X, Liang S (2014) Molecular cloning, bioinformatics analysis and functional characterization of HWTX-XI toxin superfamily from the spider Ornithoctonus huwena. Peptides (N Y) 54:9–18</cite> [<a href="https://doi.org/10.1016/j.peptides.2014.01.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24418069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20L,%20Deng%20M,%20Duan%20Z,%20Tang%20X,%20Liang%20S%20(2014)%20Molecular%20cloning,%20bioinformatics%20analysis%20and%20functional%20characterization%20of%20HWTX-XI%20toxin%20superfamily%20from%20the%20spider%20Ornithoctonus%20huwena.%20Peptides%20(N%20Y)%2054:9%E2%80%9318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Paessler S, Veljkovic V (2023) Potent and selective inhibitors of Nav1.7. United States patent US20230374084A1</cite>
</li>
<li id="CR63">
<span class="label">63.</span><cite>Giribaldi J et al (2023) Pmu1a, a novel spider toxin with dual inhibitory activity at pain targets hNaV 1.7 and hCaV 3 voltage-gated channels. FEBS J 290:3688–3702
</cite> [<a href="https://doi.org/10.1111/febs.16773" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36912793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Giribaldi%20J%20et%20al%20(2023)%20Pmu1a,%20a%20novel%20spider%20toxin%20with%20dual%20inhibitory%20activity%20at%20pain%20targets%20hNaV%201.7%20and%20hCaV%203%20voltage-gated%20channels.%20FEBS%20J%20290:3688%E2%80%933702" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>MacKinnon R, Miller C (1988) Mechanism of charybdotoxin block of the high-conductance, Ca2+-activated K + channel. J Gen Physiol 91:335–349
</cite> [<a href="https://doi.org/10.1085/jgp.91.3.335" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2216138/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2454283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?MacKinnon%20R,%20Miller%20C%20(1988)%20Mechanism%20of%20charybdotoxin%20block%20of%20the%20high-conductance,%20Ca2+-activated%20K%E2%80%89+%E2%80%89channel.%20J%20Gen%20Physiol%2091:335%E2%80%93349" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Hidalgo P, MacKinnon R (1995) Revealing the architecture of a K + channel pore through mutant cycles with a peptide inhibitor. Sci (1979) 268:307–310</cite> [<a href="https://doi.org/10.1126/science.7716527" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7716527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hidalgo%20P,%20MacKinnon%20R%20(1995)%20Revealing%20the%20architecture%20of%20a%20K%E2%80%89+%E2%80%89channel%20pore%20through%20mutant%20cycles%20with%20a%20peptide%20inhibitor.%20Sci%20(1979)%20268:307%E2%80%93310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Dauplais M et al (1997) On the convergent evolution of animal toxins. Conservation of a diad of functional residues in potassium channel-blocking toxins with unrelated structures. J Biol Chem 272:4302–4309
</cite> [<a href="https://doi.org/10.1074/jbc.272.7.4302" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9020148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dauplais%20M%20et%20al%20(1997)%20On%20the%20convergent%20evolution%20of%20animal%20toxins.%20Conservation%20of%20a%20diad%20of%20functional%20residues%20in%20potassium%20channel-blocking%20toxins%20with%20unrelated%20structures.%20J%20Biol%20Chem%20272:4302%E2%80%934309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Banerjee A, Lee A, Campbell E, Mackinnon R (2013) Structure of a pore-blocking toxin in complex with a eukaryotic voltage-dependent K(+) channel. Elife 2:e00594
</cite> [<a href="https://doi.org/10.7554/eLife.00594" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3660741/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23705070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banerjee%20A,%20Lee%20A,%20Campbell%20E,%20Mackinnon%20R%20(2013)%20Structure%20of%20a%20pore-blocking%20toxin%20in%20complex%20with%20a%20eukaryotic%20voltage-dependent%20K(+)%20channel.%20Elife%202:e00594" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Selvakumar P et al (2022) Structures of the T cell potassium channel Kv1.3 with immunoglobulin modulators. Nat Commun 13:3854
</cite> [<a href="https://doi.org/10.1038/s41467-022-31285-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9253088/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35788586/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Selvakumar%20P%20et%20al%20(2022)%20Structures%20of%20the%20T%20cell%20potassium%20channel%20Kv1.3%20with%20immunoglobulin%20modulators.%20Nat%20Commun%2013:3854" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Wu Y et al (2024) Cryo-EM structures of Kv1.2 potassium channels, conducting and non-conducting. Elife 12:RP89459. 10.7554/eLife.89459</cite> [<a href="https://doi.org/10.7554/eLife.89459" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11825129/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39945513/" class="usa-link">PubMed</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Wang JM et al (2004) Molecular surface of tarantula toxins interacting with voltage sensors in K(v) channels. J Gen Physiol 123:455–467
</cite> [<a href="https://doi.org/10.1085/jgp.200309005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2217462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15051809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20JM%20et%20al%20(2004)%20Molecular%20surface%20of%20tarantula%20toxins%20interacting%20with%20voltage%20sensors%20in%20K(v)%20channels.%20J%20Gen%20Physiol%20123:455%E2%80%93467" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Li-Smerin Y, Swartz KJ (2000) Localization and molecular determinants of the hanatoxin receptors on the voltage-sensing domains of a K(+) channel. J Gen Physiol 115:673–684
</cite> [<a href="https://doi.org/10.1085/jgp.115.6.673" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2232886/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10828242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li-Smerin%20Y,%20Swartz%20KJ%20(2000)%20Localization%20and%20molecular%20determinants%20of%20the%20hanatoxin%20receptors%20on%20the%20voltage-sensing%20domains%20of%20a%20K(+)%20channel.%20J%20Gen%20Physiol%20115:673%E2%80%93684" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Lee S-Y, MacKinnon R (2004) A membrane-access mechanism of ion channel Inhibition by voltage sensor toxins from spider venom. Nature 430:232–235
</cite> [<a href="https://doi.org/10.1038/nature02632" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15241419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20S-Y,%20MacKinnon%20R%20(2004)%20A%20membrane-access%20mechanism%20of%20ion%20channel%20Inhibition%20by%20voltage%20sensor%20toxins%20from%20spider%20venom.%20Nature%20430:232%E2%80%93235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Lee HC, Wang JM, Swartz KJ (2003) Interaction between extracellular hanatoxin and the resting conformation of the voltage-sensor paddle in Kv channels. Neuron 40:527–536
</cite> [<a href="https://doi.org/10.1016/s0896-6273(03)00636-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14642277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20HC,%20Wang%20JM,%20Swartz%20KJ%20(2003)%20Interaction%20between%20extracellular%20hanatoxin%20and%20the%20resting%20conformation%20of%20the%20voltage-sensor%20paddle%20in%20Kv%20channels.%20Neuron%2040:527%E2%80%93536" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Phillips LR et al (2005) Voltage-sensor activation with a tarantula toxin as cargo. Nature 436:857–860
</cite> [<a href="https://doi.org/10.1038/nature03873" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16094370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Phillips%20LR%20et%20al%20(2005)%20Voltage-sensor%20activation%20with%20a%20tarantula%20toxin%20as%20cargo.%20Nature%20436:857%E2%80%93860" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Swartz KJ (2007) Tarantula toxins interacting with voltage sensors in potassium channels. Toxicon 49:213–230
</cite> [<a href="https://doi.org/10.1016/j.toxicon.2006.09.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1839852/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17097703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Swartz%20KJ%20(2007)%20Tarantula%20toxins%20interacting%20with%20voltage%20sensors%20in%20potassium%20channels.%20Toxicon%2049:213%E2%80%93230" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Mouhat S, Jouirou B, Mosbah A, De Waard M, Sabatier J-M (2004) Diversity of folds in animal toxins acting on ion channels. Biochem J 378:717–726
</cite> [<a href="https://doi.org/10.1042/BJ20031860" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1224033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14674883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mouhat%20S,%20Jouirou%20B,%20Mosbah%20A,%20De%20Waard%20M,%20Sabatier%20J-M%20(2004)%20Diversity%20of%20folds%20in%20animal%20toxins%20acting%20on%20ion%20channels.%20Biochem%20J%20378:717%E2%80%93726" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Norton RS, Pallaghy PK (1998) The cystine knot structure of ion channel toxins and related polypeptides. Toxicon 36:1573–1583
</cite> [<a href="https://doi.org/10.1016/s0041-0101(98)00149-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9792173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Norton%20RS,%20Pallaghy%20PK%20(1998)%20The%20cystine%20knot%20structure%20of%20ion%20channel%20toxins%20and%20related%20polypeptides.%20Toxicon%2036:1573%E2%80%931583" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Wang X et al (2000) Discovery and characterization of a family of insecticidal neurotoxins with a rare vicinal disulfide bridge. Nat Struct Biol 7:505–513
</cite> [<a href="https://doi.org/10.1038/75921" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10881200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20X%20et%20al%20(2000)%20Discovery%20and%20characterization%20of%20a%20family%20of%20insecticidal%20neurotoxins%20with%20a%20rare%20vicinal%20disulfide%20bridge.%20Nat%20Struct%20Biol%207:505%E2%80%93513" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Szeto TH et al (2000) Isolation of a funnel-web spider polypeptide with homology to mamba intestinal toxin 1 and the embryonic head inducer Dickkopf-1. Toxicon 38:429–442
</cite> [<a href="https://doi.org/10.1016/s0041-0101(99)00174-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10669030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Szeto%20TH%20et%20al%20(2000)%20Isolation%20of%20a%20funnel-web%20spider%20polypeptide%20with%20homology%20to%20mamba%20intestinal%20toxin%201%20and%20the%20embryonic%20head%20inducer%20Dickkopf-1.%20Toxicon%2038:429%E2%80%93442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Kiyatkin N, Dulubova I, Chekhovskaya I, Lipkin A, Grishin E (1992) Structure of the low molecular weight protein copurified with alpha-latrotoxin. Toxicon 30:771–774
</cite> [<a href="https://doi.org/10.1016/0041-0101(92)90012-t" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1509496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kiyatkin%20N,%20Dulubova%20I,%20Chekhovskaya%20I,%20Lipkin%20A,%20Grishin%20E%20(1992)%20Structure%20of%20the%20low%20molecular%20weight%20protein%20copurified%20with%20alpha-latrotoxin.%20Toxicon%2030:771%E2%80%93774" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Pineda SS et al (2020) Structural venomics reveals evolution of a complex venom by duplication and diversification of an ancient peptide-encoding gene. Proc Natl Acad Sci U S A 117:11399–11408
</cite> [<a href="https://doi.org/10.1073/pnas.1914536117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7260951/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32398368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pineda%20SS%20et%20al%20(2020)%20Structural%20venomics%20reveals%20evolution%20of%20a%20complex%20venom%20by%20duplication%20and%20diversification%20of%20an%20ancient%20peptide-encoding%20gene.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20117:11399%E2%80%9311408" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Percie du N, Sert et al (2020) The ARRIVE guidelines 2.0: updated guidelines for reporting animal research. BMC Vet Res 16:242
</cite> [<a href="https://doi.org/10.1186/s12917-020-02451-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7359286/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32660541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Percie%20du%20N,%20Sert%20et%20al%20(2020)%20The%20ARRIVE%20guidelines%202.0:%20updated%20guidelines%20for%20reporting%20animal%20research.%20BMC%20Vet%20Res%2016:242" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Vassilevski AA, Kozlov SA, Egorov TA, Grishin EV (2010) Purification and characterization of biologically active peptides from spider venoms. Methods Mol Biol 615:87–100
</cite> [<a href="https://doi.org/10.1007/978-1-60761-535-4_7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20013202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vassilevski%20AA,%20Kozlov%20SA,%20Egorov%20TA,%20Grishin%20EV%20(2010)%20Purification%20and%20characterization%20of%20biologically%20active%20peptides%20from%20spider%20venoms.%20Methods%20Mol%20Biol%20615:87%E2%80%93100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Haas BJ et al (2013) De novo transcript sequence reconstruction from RNA-seq using the trinity platform for reference generation and analysis. Nat Protoc 8:1494–1512
</cite> [<a href="https://doi.org/10.1038/nprot.2013.084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3875132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23845962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haas%20BJ%20et%20al%20(2013)%20De%20novo%20transcript%20sequence%20reconstruction%20from%20RNA-seq%20using%20the%20trinity%20platform%20for%20reference%20generation%20and%20analysis.%20Nat%20Protoc%208:1494%E2%80%931512" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Ruiz-Reche A, Srivastava A, Indi JA, De La Rubia I, Eyras E (2019) ReorientExpress: reference-free orientation of nanopore cDNA reads with deep learning. Genome Biol 20:260
</cite> [<a href="https://doi.org/10.1186/s13059-019-1884-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6883653/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31783882/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ruiz-Reche%20A,%20Srivastava%20A,%20Indi%20JA,%20De%20La%20Rubia%20I,%20Eyras%20E%20(2019)%20ReorientExpress:%20reference-free%20orientation%20of%20nanopore%20cDNA%20reads%20with%20deep%20learning.%20Genome%20Biol%2020:260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Rice P, Longden L, Bleasby A (2000) EMBOSS: the European molecular biology open software suite. Trends Genet 16:276–277
</cite> [<a href="https://doi.org/10.1016/s0168-9525(00)02024-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10827456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rice%20P,%20Longden%20L,%20Bleasby%20A%20(2000)%20EMBOSS:%20the%20European%20molecular%20biology%20open%20software%20suite.%20Trends%20Genet%2016:276%E2%80%93277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Andreev YA, Kozlov SA, Vassilevski AA, Grishin EV (2010) Cyanogen bromide cleavage of proteins in salt and buffer solutions. Anal Biochem 407:144–146
</cite> [<a href="https://doi.org/10.1016/j.ab.2010.07.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20670609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andreev%20YA,%20Kozlov%20SA,%20Vassilevski%20AA,%20Grishin%20EV%20(2010)%20Cyanogen%20bromide%20cleavage%20of%20proteins%20in%20salt%20and%20buffer%20solutions.%20Anal%20Biochem%20407:144%E2%80%93146" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Logashina YA et al (2020) Refolding of disulfide containing peptides in fusion with thioredoxin. Mendeleev Commun 30:214–216</cite> [<a href="https://scholar.google.com/scholar_lookup?Logashina%20YA%20et%20al%20(2020)%20Refolding%20of%20disulfide%20containing%20peptides%20in%20fusion%20with%20thioredoxin.%20Mendeleev%20Commun%2030:214%E2%80%93216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Kuzmenkov AI, Sachkova MY, Kovalchuk SI, Grishin EV, Vassilevski AA (2016) Lachesana tarabaevi, an expert in membrane-active toxins. Biochem J 473:2495–2506
</cite> [<a href="https://doi.org/10.1042/BCJ20160436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27287558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuzmenkov%20AI,%20Sachkova%20MY,%20Kovalchuk%20SI,%20Grishin%20EV,%20Vassilevski%20AA%20(2016)%20Lachesana%20tarabaevi,%20an%20expert%20in%20membrane-active%20toxins.%20Biochem%20J%20473:2495%E2%80%932506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Kohlbacher O et al (2007) TOPP–the openms proteomics pipeline. Bioinformatics 23:e191–e197
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btl299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17237091/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kohlbacher%20O%20et%20al%20(2007)%20TOPP%E2%80%93the%20openms%20proteomics%20pipeline.%20Bioinformatics%2023:e191%E2%80%93e197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Wüthrich K (1986) NMR of proteins and nucleic acids. Wiley-Interscience</cite>
</li>
<li id="CR92">
<span class="label">92.</span><cite>Güntert P (2004) Automated NMR structure calculation with CYANA. Methods Mol Biol 278:353–378
</cite> [<a href="https://doi.org/10.1385/1-59259-809-9:353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15318003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?G%C3%BCntert%20P%20(2004)%20Automated%20NMR%20structure%20calculation%20with%20CYANA.%20Methods%20Mol%20Biol%20278:353%E2%80%93378" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Kuzmenkov AI et al (2022) Apamin structure and pharmacology revisited. Front Pharmacol 13:977440
</cite> [<a href="https://doi.org/10.3389/fphar.2022.977440" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9523135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36188602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuzmenkov%20AI%20et%20al%20(2022)%20Apamin%20structure%20and%20pharmacology%20revisited.%20Front%20Pharmacol%2013:977440" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Curtis MJ et al (2018) Experimental design and analysis and their reporting II: updated and simplified guidance for authors and peer reviewers. Br J Pharmacol 175:987–993
</cite> [<a href="https://doi.org/10.1111/bph.14153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5843711/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29520785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Curtis%20MJ%20et%20al%20(2018)%20Experimental%20design%20and%20analysis%20and%20their%20reporting%20II:%20updated%20and%20simplified%20guidance%20for%20authors%20and%20peer%20reviewers.%20Br%20J%20Pharmacol%20175:987%E2%80%93993" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12334782/bin/18_2025_5778_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (1.2MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12334782/bin/18_2025_5778_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (478.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12334782/bin/18_2025_5778_MOESM3_ESM.fasta" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (1.4KB, fasta) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are available from the corresponding authors, upon request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Cellular and Molecular Life Sciences: CMLS are provided here courtesy of <strong>Springer</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1007/s00018-025-05778-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/18_2025_Article_5778.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12334782/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12334782/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12334782%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334782/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12334782/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12334782/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40778948/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12334782/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40778948/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12334782/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12334782/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="IuCG5VLVwIlqTDwosJE6OVF4UEuDVCYZ0JalKbayPVxOzoZ7bdtjTc5C26KtF4J1">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
