
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Investigation of growth traits in Turkish Merino lambs using multi-locus GWAS approaches: Karacabey Merino - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="67D7146689B748931C146600348390D4.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcvetres">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333283/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Veterinary Research">
<meta name="citation_title" content="Investigation of growth traits in Turkish Merino lambs using multi-locus GWAS approaches: Karacabey Merino">
<meta name="citation_author" content="Yalçın Yaman">
<meta name="citation_author_institution" content="Department of Genetics, Faculty of Veterinary Medicine, Siirt University, Siirt, 56000 Turkey">
<meta name="citation_author" content="Ramazan Aymaz">
<meta name="citation_author_institution" content="Sheep Research and Breeding Institute, Bandırma, Balıkesir, Turkey">
<meta name="citation_author" content="Murat Keleş">
<meta name="citation_author_institution" content="Sheep Research and Breeding Institute, Bandırma, Balıkesir, Turkey">
<meta name="citation_author" content="Yiğit Emir Kişi">
<meta name="citation_author_institution" content="Sheep Research and Breeding Institute, Bandırma, Balıkesir, Turkey">
<meta name="citation_author" content="Ecem Hatipoğlu">
<meta name="citation_author_institution" content="International Center for Livestock Research and Training, Ankara, Turkey">
<meta name="citation_author" content="Arzu Özdemir">
<meta name="citation_author_institution" content="International Center for Livestock Research and Training, Ankara, Turkey">
<meta name="citation_author" content="Elif Çetinkaya">
<meta name="citation_author_institution" content="Fertilizer and Water Resources Central Research Institute, Ankara, Turkey">
<meta name="citation_publication_date" content="2025 Aug 8">
<meta name="citation_volume" content="21">
<meta name="citation_firstpage" content="511">
<meta name="citation_doi" content="10.1186/s12917-025-04957-9">
<meta name="citation_pmid" content="40775720">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333283/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333283/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333283/pdf/12917_2025_Article_4957.pdf">
<meta name="description" content="Sheep have demonstrated remarkable adaptability to diverse and unproductive pastures, making them highly advantageous in the context of sustainable farming practices in the globally warming world. Despite their adaptation skills, local sheep breeds ...">
<meta name="og:title" content="Investigation of growth traits in Turkish Merino lambs using multi-locus GWAS approaches: Karacabey Merino">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Sheep have demonstrated remarkable adaptability to diverse and unproductive pastures, making them highly advantageous in the context of sustainable farming practices in the globally warming world. Despite their adaptation skills, local sheep breeds ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333283/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12333283">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12917-025-04957-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12917_2025_Article_4957.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12333283%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12333283/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12333283/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333283/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcvetres.png" alt="BMC Veterinary Research logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Veterinary Research" title="Link to BMC Veterinary Research" shape="default" href="https://bmcvetres.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Vet Res</button></div>. 2025 Aug 8;21:511. doi: <a href="https://doi.org/10.1186/s12917-025-04957-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12917-025-04957-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Vet%20Res%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Vet%20Res%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Vet%20Res%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Vet%20Res%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Investigation of growth traits in Turkish Merino lambs using multi-locus GWAS approaches: Karacabey Merino</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yaman%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yalçın Yaman</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yalçın Yaman</span></h3>
<div class="p">
<sup>1</sup>Department of Genetics, Faculty of Veterinary Medicine, Siirt University, Siirt, 56000 Turkey </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yaman%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yalçın Yaman</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aymaz%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ramazan Aymaz</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ramazan Aymaz</span></h3>
<div class="p">
<sup>2</sup>Sheep Research and Breeding Institute, Bandırma, Balıkesir, Turkey </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aymaz%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ramazan Aymaz</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kele%C5%9F%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Murat Keleş</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Murat Keleş</span></h3>
<div class="p">
<sup>2</sup>Sheep Research and Breeding Institute, Bandırma, Balıkesir, Turkey </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kele%C5%9F%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Murat Keleş</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ki%C5%9Fi%20YE%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yiğit Emir Kişi</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yiğit Emir Kişi</span></h3>
<div class="p">
<sup>2</sup>Sheep Research and Breeding Institute, Bandırma, Balıkesir, Turkey </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ki%C5%9Fi%20YE%22%5BAuthor%5D" class="usa-link"><span class="name western">Yiğit Emir Kişi</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hatipo%C4%9Flu%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ecem Hatipoğlu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ecem Hatipoğlu</span></h3>
<div class="p">
<sup>3</sup>International Center for Livestock Research and Training, Ankara, Turkey </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hatipo%C4%9Flu%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Ecem Hatipoğlu</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22%C3%96zdemir%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Arzu Özdemir</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Arzu Özdemir</span></h3>
<div class="p">
<sup>3</sup>International Center for Livestock Research and Training, Ankara, Turkey </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22%C3%96zdemir%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Arzu Özdemir</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22%C3%87etinkaya%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Elif Çetinkaya</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Elif Çetinkaya</span></h3>
<div class="p">
<sup>4</sup>Fertilizer and Water Resources Central Research Institute, Ankara, Turkey </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22%C3%87etinkaya%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Elif Çetinkaya</span></a>
</div>
</div>
<sup>4</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Genetics, Faculty of Veterinary Medicine, Siirt University, Siirt, 56000 Turkey </div>
<div id="Aff2">
<sup>2</sup>Sheep Research and Breeding Institute, Bandırma, Balıkesir, Turkey </div>
<div id="Aff3">
<sup>3</sup>International Center for Livestock Research and Training, Ankara, Turkey </div>
<div id="Aff4">
<sup>4</sup>Fertilizer and Water Resources Central Research Institute, Ankara, Turkey </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Sep 21; Accepted 2025 Jul 22; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12333283  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40775720/" class="usa-link">40775720</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Sheep have demonstrated remarkable adaptability to diverse and unproductive pastures, making them highly advantageous in the context of sustainable farming practices in the globally warming world. Despite their adaptation skills, local sheep breeds generally exhibit low performance, highlighting the need to develop desired traits. Traditional Mixed Linear Model (MLM)-based single-locus Genome-Wide Association (GWA) studies may fall short in identifying multiple loci influencing traits due to their linear genome scanning approach, rendering them less effective for detecting polygenic effects.</p></section><section id="sec2"><h3 class="pmc_sec_title">Objectives</h3>
<p id="Par2">To identify genetic variants associated with birth weight (BW) and weaning weight (WW) in Karacabey Merino lambs using multi-locus genome-wide association study (GWAS) approaches, and to propose candidate genes for these economically important growth traits.</p></section><section id="sec3"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par3">Five multi-locus approaches were employed, including MrMLM, FastMrMLM, ISIS EM-BLASSO, FASTmrEMMA, pLARmEB, and pKWmEB. These methods utilize a two-stage process to detect associated markers, first screening with a single locus approach at a less strict significance threshold, followed by collective evaluation using multi-locus GWA models. Gene annotation was performed to identify candidate genes based on SNP locations within intron regions or within ± 100 Kb proximity of associated SNPs.</p></section><section id="sec4"><h3 class="pmc_sec_title">Results</h3>
<p id="Par4">Using the five multi-locus approaches, 11 SNPs were detected with significant effects on birth weight and seven SNPs on weaning weight in Karacabey merino lambs. Gene annotation revealed most associated SNPs were within or very close to protein-coding genes, suggesting a functional role in trait influence. The genes <em>GNAQ</em>, <em>CDKL4</em>, <em>PIP</em>, <em>SLC7A1</em>, <em>PBRM1</em>, <em>SORCS3</em>, and <em>NFATC1</em> were identified as candidate genes for birth weight, while <em>BABAM2</em>, <em>LALBA</em>, <em>NOP14</em>, <em>FAM110B</em>, <em>SKAP1</em>, <em>SVIL</em>, and <em>ATXN1</em> were proposed as candidate genes for weaning weight.</p></section><section id="sec5"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par5">These insights contribute to a better understanding of the genetic makeup of birth weight and weaning weight traits, supporting efforts to refine breeding programs for improved growth performance in Karacabey merino sheep.</p></section><section id="sec6"><h3 class="pmc_sec_title">Graphical Abstract</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333283_12917_2025_4957_Figa_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc27/12333283/d0fceeeec34e/12917_2025_4957_Figa_HTML.jpg" loading="lazy" id="MO1" height="594" width="669" alt="graphic file with name 12917_2025_4957_Figa_HTML.jpg"></a></p></section><section id="sec7"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12917-025-04957-9.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Karacabey merino, Multi-locus GWAS, Growth traits, Birth weight, Weaning weight</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par7">As one of the earliest domesticated animals, domestic sheep have spread globally, encompassing around 1,000 breeds thriving in various climates, from scorching heat to freezing cold. Their incredible adaptability to diverse environments enables them to efficiently graze on less productive pastures, making them incredibly valuable in sustainable farming. Research highlights the selective breeding of sheep for specific environments, underlining the importance of these adaptation skills for their survival and productivity across different regions [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>].</p>
<p id="Par8">In contrast, industrial cattle breeding often faces inbreeding issues [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>], leading to susceptibility to heat stress [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>], diseases, and decreased survival rates [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. On the other hand, sheep, with their wide genetic diversity, exhibit resilience to diseases and adeptness in coping with harsh climates. Their adaptability not only helps them thrive in challenging conditions but also ensures the sustainability of livestock populations overall [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>].</p>
<p id="Par9">While industrial farming relies heavily on intensive rearing through concentrated feeding, small ruminant farming, including sheep, stands out for its ability to utilize natural resources that are not usable by cattle. This positions small ruminants as a safeguard for sustainable animal production in our warming world.</p>
<p id="Par10">The Karacabey Merino breed, which is reared at the Sheep Research and Breeding Institute (SRBI) and is widespread in the Marmara region of Northwest Turkey, was improved during the 1940 s by crossing German Mutton Merino with native Kivircik sheep. Since then, it has remained closed to crossbreeding for at least the last three decades. The thin-tailed Kivircik sheep, which constitutes the maternal line of backcrossing efforts, is one of Turkey’s ancient sheep breeds, renowned for its delicious meat and resilience to environmental conditions. Due to over 80 years of backcrossing and classical breeding efforts, as well as its widespread presence in the northern regions of Turkey, the Karacabey Merino has been officially recognized as Turkey’s native sheep breed.</p>
<p id="Par11">Genome-Wide Association (GWA) studies typically utilize the Mixed Linear Model (MLM) as the primary statistical method, known for its efficacy in controlling population stratification and polygenic background inflation [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. However, MLM-based single marker GWA studies may lack the capability to effectively identify multiple loci simultaneously influencing the traits of interest due to the linear scanning of the genome. Hence, MLM is often deemed unsuitable for detecting polygenic effects. Moreover, MLM-based GWA studies encounter challenges in correcting for multiple tests to establish statistical significance. The Bonferroni test, commonly employed for this purpose, tends to be overly conservative, potentially failing to identify all relevant SNPs. Consequently, the corrected “<em>p</em>” value for many crucial markers might not surpass the significance threshold [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>].</p>
<p id="Par12">Various approaches have been developed in multi-locus methods to tackle these issues, including Multi-Locus Random-SNP-Effect Mixed Linear Model (MrMLM) [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>], Fast Multi-Locus Random-SNP-Effect Mixed Linear Model (FastMrMLM) [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>], Iterative sure independence screening EM-Bayesian LASSO (ISIS EM-BLASSO) [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>], fast multi-locus random-SNP-effect EMMA (FASTmrEMMA) [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>], polygenic-background-control-based least angle regression plus empirical Bayes (pLARmEB) [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>], and polygenic-background-control-based Kruskal-Wallis test plus empirical Bayes (pKWmEB) [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>].</p>
<p id="Par13">These multi-locus approaches employ multidimensional scanning and follow a two-stage process to detect associated markers. Initially, markers meeting quality control criteria undergo screening using a single locus approach with a less strict significance threshold, typically set at <em>p</em> &lt; 0.005 or 0.001. The selected markers from this screening are then subjected to further analysis. In the second stage, all chosen markers are collectively evaluated using multi-locus GWA models. Due to their multi-locus nature, these approaches eliminate the need for the Bonferroni test for multiple test correction [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>].</p>
<p id="Par14">Thus, in the present study, these five multi-locus methods were chosen because comparative analyses through both simulation studies and real data applications have demonstrated their enhanced capabilities in SNP detection power and effect estimation accuracy compared to conventional approaches such as EMMA, GEMMA, SUPER, CMLM, ECMLM, SCAD, and FarmCPU. These methods are effective for polygenic background control and have less bias. FASTmrEMMA and FASTmrMLM exhibit improved computational efficiency while maintaining high statistical power. MrMLM shows superior performance over RMLM and EMMA, while ISIS EM-BLASSO demonstrates the highest empirical power and accuracy compared to EMMA, SCAD, and FarmCPU. Similarly, pLARmEB exhibits enhanced performance over Bayesian hierarchical generalized linear model and EMMA. Additionally, all these methods are integrated into a single software package (mrMLM), providing a comprehensive and efficient analytical framework for multi-locus GWAS studies [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>].</p>
<p id="Par15">In addition to the advantages of local sheep breeds’ adaptation to various geographic areas, ecosystems, and management practices, and their superior adaptability to various stressors, including feed shortages and extreme climatic conditions, their low performance in desired traits can be seen as a significant disadvantage. Therefore, there is a need to improve the productivity of local sheep breeds.</p>
<p id="Par16">The study reported here aims to identify the genetic background of the growth traits of Karacabey Merino lambs. Due to the exhibition of quantitative traits under polygenic control, recently developed multi-locus GWAS methods, which can detect polygenic effects through multidimensional scanning, have been employed. Thus, using five multi-locus GWAS methods—mrMLM, FASTmrMLM, FASTmrEMMA, pLARmEB, and ISIS EM-BLASSO—potentially influential markers for growth performance were investigated.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Animals</h3>
<p id="Par17">The animal material used in this study, Karacabey Merino lambs, belongs to the Sheep Breeding Research Institute, an official institution under the Turkish Ministry of Agriculture and Forestry, and all necessary official permissions were obtained from the relevant institution prior to the study. A total of 734 lambs, including 338 males and 396 females, were included in the study. All lambs were weighed with a precision of 100 g within 24 h of birth, ear-tagged, and recorded. From 10 to 15 days after birth, in addition to nursing from their mothers, the lambs were allowed free access to concentrate feed, alfalfa hay, and clean water. They were fed <em>ad libitum</em> until weaning, with all lambs receiving the same concentrate and roughage feed. The management practices were the same for all the lambs, and they were weaned at an average of 90 days. At the time of weaning, the lambs were weighed again and recorded. The weaning weights were adjusted to 90 days based on birth weight and the number of days from birth to the weaning.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Genotyping and quality controls</h3>
<p id="Par18">The lambs were genotyped using the OVINE 50 K BeadChip on the ILLUMINA platform with a total genotyping rate of 0.995. The sex chromosomes were excluded from the analysis. Quality control procedures were carried out using Plink v1.07 [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. The Quality Control (QC) parameters applied were: Minor Allele Frequency (MAF) ≥ 0.05, animal missing genotype rates (mind) ≤ 0.1, SNP Call rate ≥ 0.95, and Hardy-Weinberg equilibrium threshold ≥ 10<sup>−5</sup>. After performing the QC, a total of 35,886 SNPs and 734 lambs (338 males and 396 females) remained for further analysis.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Association study</h3>
<p id="Par19">Before commencing the analyses, both an outlier detection test and a normality test were performed on each dataset. These tests were conducted using Minitab<sup>®</sup> version 21.4 software to ensure the data met the assumptions required for subsequent statistical analyses.</p>
<p id="Par20">The SNP-based heritability (h²), coefficient of variation (CV), and genetic correlation (rG) were computed using GCTA software [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Multi-locus GWA analysis was conducted utilizing the default settings of mrMLM, an R package developed by Zhang et al. [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Kinship matrices were created using the same package and integrated into all five multi-locus models to manage cryptic relatedness. To control possible population stratification, Principal Component Analysis (PCA) was performed using TASSEL software [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>], incorporating the first five principal components into the all statistical model. Due to the strong effect of gender on the BW and WW (exact <em>p-</em>value; 0.001 and 1.76E-24, respectively) and birth type (single, twin, or triplet) on the same traits (exact <em>p-</em>value; 2.22E-30 and 5.23E-11, respectively), these two important factors were included in all models as covariates. In multi-locus GWAS methods, a commonly acknowledged threshold for the Logarithm of the Odds (LOD) score is set at LOD ≥ 3 [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Traditionally, a LOD score of 3 means that the likelihood of the loci being linked is 1,000 times higher than the likelihood of them being unlinked [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. To ensure the most reliable association findings, we adopted the LOD ≥ 5 threshold for statistical significance. Such a significance threshold indicates that the probability of association is 100,000 times higher than the probability of no association. Manhattan plots were created to illustrate significant SNPs (LOD ≥ 5) across chromosomes. Quantile-Quantile (QQ) plots were employed to identify any inflation or deflation of test statistics resulting from systematic biases under the null model, such as unnoticed population stratification or cryptic relatedness.</p>
<p id="Par21">A post hoc statistical power analysis was conducted using the MAF and <em>r</em>² values of SNPs that have a significant relationship with the BW or WW. Since multiple SNPs were identified for each phenotype, the power analysis was conducted using the SNPs with the lowest and highest <em>r</em>² values that were associated with the BW or WW. The power analysis was performed using the “pwr” package in the R platform.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Genomic annotation, gene ontology, and pathway analysis</h3>
<p id="Par22">Gene annotations were conducted using the ARS-UI_Ramb_v2.0 assembly by Ensembl (<a href="https://www.ensembl.org/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ensembl.org/index.html</a>), Genome Data Viewer (<a href="https://www.ncbi.nlm.nih.gov/gdv" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/gdv</a>), and Genome Browser (<a href="https://genome.ucsc.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://genome.ucsc.edu/</a>).</p>
<p id="Par23">Further analysis was conducted on the genes situated within ± 100 Kb proximity of each associated SNP to identify the pathways in which they participate. Gene ontology and pathway analysis were performed using the widely-used online tool ShinyGO [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] with the Ovis aries database, employing a false discovery rate (FDR) cutoff of 0.05 to determine statistical significance. Pathway charts were illustrated using the same tool.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Promoter and intron splicing prediction</h3>
<p id="Par24">
To investigate potential promoter region associations of SNPs linked to the BW and WW that are situated in intergenic regions or upstream/downstream of genes, we employed Promoter 2.0 [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>], one of the limited available online tools applicable to non-human and non-model organisms. The analysis involved examining sequences of approximately 480–500 base pairs in total length, comprising 240–250 base pairs flanking each side of these SNPs, through the Promoter 2.0 platform to identify potential promoter region overlaps.</p>
<p id="Par25">
Additionally, we investigated whether the BW and WW-associated SNPs located within intronic regions were involved in splicing processes. This analysis was conducted using Spliceator [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>], an Artificial Intelligence-based online tool, examining sequences of approximately 60 base pairs in total length, comprising 30 base pairs flanking each side of these intronic SNPs.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec9"><h3 class="pmc_sec_title">Descriptive statistics</h3>
<p id="Par26">Prior to the genotype–phenotype association analyses, phenotypic data were examined through a two-step process based on a significance threshold of <em>p</em> &lt; 0.05: first, an outlier test was performed, followed by a normality test. According to the outlier test results, no outliers were detected in any of the datasets (<em>p</em> = 1.00). The normality test revealed that the WW data did not follow a normal distribution (<em>p</em> = 0.048). Consequently, a Box-Cox transformation was applied, after which the WW data showed a good fit to the normal distribution (<em>p</em> = 0.348). Outlier test results for BW, raw WW, and Box-Cox transformed WW are presented in Fig. <a href="#Fig1" class="usa-link">1</a>A, while the results of the normality test are visualised through QQ plots (Fig. <a href="#Fig1" class="usa-link">1</a>B) and histograms (Fig. <a href="#Fig1" class="usa-link">1</a>C).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333283_12917_2025_4957_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc27/12333283/c1f9b4c1892d/12917_2025_4957_Fig1_HTML.jpg" loading="lazy" id="MO2" height="492" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Visualisation of outlier and normality test results. Panel (<strong>A</strong>) shows the outlier test results, while panels (<strong>B</strong>) and (<strong>C</strong>) display the QQ plot and histogram, respectively, for the normality test of the BW, raw WW, and Box-Cox transformed WW datasets</p></figcaption></figure><p id="Par28">For both traits, the BW and WW, the SNP-based heritability was calculated. The heritability for the BW was found to be 0.282, indicating that approximately 28.2% of the total phenotypic variation in the BW can be attributed to genetic factors. For the WW, the heritability was 0.164, meaning that 16.4% of the variation in the WW is due to genetic influences. The coefficient of variation (CV%) for the BW was determined to be 17.63%, which reflects the ratio of the standard deviation to the mean, expressed as a percentage, and indicates the relative variability of the BW. Similarly, the CV for the WW was 13.80%, showing the relative variability of this trait. Genetic correlations between the BW and WW were observed at a high level, with a correlation coefficient of 0.857. This suggests a strong positive genetic relationship between the two traits. Descriptive statistics are provided in Table <a href="#Tab1" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Descriptive statistics for the BW and WW</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Traits</th>
<th align="left" colspan="1" rowspan="1">
<em>n</em>
</th>
<th align="left" colspan="1" rowspan="1">Mean</th>
<th align="left" colspan="1" rowspan="1">Min</th>
<th align="left" colspan="1" rowspan="1">Max</th>
<th align="left" colspan="1" rowspan="1">SD</th>
<th align="left" colspan="1" rowspan="1">SE</th>
<th align="left" colspan="1" rowspan="1">CV %</th>
<th align="left" colspan="1" rowspan="1">h<sup>2</sup>
</th>
<th align="left" colspan="1" rowspan="1">h<sup>2</sup> SE</th>
<th align="left" colspan="1" rowspan="1">h<sup>2</sup><em>p</em>-value</th>
<th align="left" colspan="1" rowspan="1">rG</th>
<th align="left" colspan="1" rowspan="1">rG SE</th>
<th align="left" colspan="1" rowspan="1">rG <em>p</em>-value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">BW</td>
<td align="left" colspan="1" rowspan="1">734</td>
<td align="left" colspan="1" rowspan="1">4.74</td>
<td align="left" colspan="1" rowspan="1">2.30</td>
<td align="left" colspan="1" rowspan="1">7.15</td>
<td align="left" colspan="1" rowspan="1">0.84</td>
<td align="left" colspan="1" rowspan="1">0.03</td>
<td align="left" colspan="1" rowspan="1">17.63</td>
<td align="left" colspan="1" rowspan="1">0.282</td>
<td align="left" colspan="1" rowspan="1">0.083</td>
<td align="left" colspan="1" rowspan="1">5.64E-06</td>
<td align="left" rowspan="2" colspan="1">0.857</td>
<td align="left" rowspan="2" colspan="1">0.05</td>
<td align="left" rowspan="2" colspan="1">&lt; 1.00E-6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WW</td>
<td align="left" colspan="1" rowspan="1">734</td>
<td align="left" colspan="1" rowspan="1">31.31</td>
<td align="left" colspan="1" rowspan="1">19.97</td>
<td align="left" colspan="1" rowspan="1">42.74</td>
<td align="left" colspan="1" rowspan="1">4.32</td>
<td align="left" colspan="1" rowspan="1">0.16</td>
<td align="left" colspan="1" rowspan="1">13.80</td>
<td align="left" colspan="1" rowspan="1">0.164</td>
<td align="left" colspan="1" rowspan="1">0.071</td>
<td align="left" colspan="1" rowspan="1">6.43E-04</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p33"><p><em>SD</em> standard deviation; <em>SE</em> standard error; <em>CV</em> Coefficient of variation (%); <em>h</em><sup>2</sup> heritability; <em>rG</em>genetic correlation</p></div></div></section></section><section id="Sec10"><h3 class="pmc_sec_title">Genotype-phenotype association analysis</h3>
<p id="Par30">Using multi-locus approaches, 11 SNPs were detected that met the LOD threshold of ≥ 5, indicating a significant effect on the BW (Table <a href="#Tab2" class="usa-link">2</a>; Fig. <a href="#Fig2" class="usa-link">2</a>A). However, when utilizing the ISIS EM-BLASSO method, no SNPs were found to be significantly associated with the BW.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Identified significant SNPs associated with the BW</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Method</th>
<th align="left" colspan="1" rowspan="1">Marker</th>
<th align="left" colspan="1" rowspan="1">Chr</th>
<th align="left" colspan="1" rowspan="1">SNP Effect</th>
<th align="left" colspan="1" rowspan="1">LOD score</th>
<th align="left" colspan="1" rowspan="1">-log10(<em>P</em>)</th>
<th align="left" colspan="1" rowspan="1">
<em>r</em><sup>2</sup> (%)<sup>1</sup>
</th>
<th align="left" colspan="1" rowspan="1">MAF</th>
<th align="left" colspan="1" rowspan="1">Allele</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">mrMLM</td>
<td align="left" rowspan="2" colspan="1">oar3_OAR2_58511524</td>
<td align="left" rowspan="2" colspan="1">2</td>
<td align="left" colspan="1" rowspan="1">−0.219</td>
<td align="left" colspan="1" rowspan="1">6.998</td>
<td align="left" colspan="1" rowspan="1">7.862</td>
<td align="left" colspan="1" rowspan="1">0.339</td>
<td align="left" rowspan="2" colspan="1">0.078</td>
<td align="left" rowspan="2" colspan="1">C</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FASTmrMLM</td>
<td align="left" colspan="1" rowspan="1">−0.222</td>
<td align="left" colspan="1" rowspan="1">5.593</td>
<td align="left" colspan="1" rowspan="1">6.411</td>
<td align="left" colspan="1" rowspan="1">0.348</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FASTmrEMMA</td>
<td align="left" colspan="1" rowspan="1">OAR2_16538077.1</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">−0.41</td>
<td align="left" colspan="1" rowspan="1">6.127</td>
<td align="left" colspan="1" rowspan="1">6.965</td>
<td align="left" colspan="1" rowspan="1">1.844</td>
<td align="left" colspan="1" rowspan="1">0.138</td>
<td align="left" colspan="1" rowspan="1">G</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mrMLM</td>
<td align="left" rowspan="3" colspan="1">Chr3:84999096</td>
<td align="left" rowspan="3" colspan="1">3</td>
<td align="left" colspan="1" rowspan="1">−0.179</td>
<td align="left" colspan="1" rowspan="1">7.523</td>
<td align="left" colspan="1" rowspan="1">8.402</td>
<td align="left" colspan="1" rowspan="1">0.572</td>
<td align="left" rowspan="3" colspan="1">0.153</td>
<td align="left" rowspan="3" colspan="1">T</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FASTmrMLM</td>
<td align="left" colspan="1" rowspan="1">−0.188</td>
<td align="left" colspan="1" rowspan="1">6.437</td>
<td align="left" colspan="1" rowspan="1">7.285</td>
<td align="left" colspan="1" rowspan="1">0.631</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FASTmrEMMA</td>
<td align="left" colspan="1" rowspan="1">−0.422</td>
<td align="left" colspan="1" rowspan="1">6.255</td>
<td align="left" colspan="1" rowspan="1">7.096</td>
<td align="left" colspan="1" rowspan="1">2.168</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mrMLM</td>
<td align="left" colspan="1" rowspan="1">oar3_OAR4_105954111</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">0.172</td>
<td align="left" colspan="1" rowspan="1">5.577</td>
<td align="left" colspan="1" rowspan="1">6.395</td>
<td align="left" colspan="1" rowspan="1">2.715</td>
<td align="left" colspan="1" rowspan="1">0.342</td>
<td align="left" colspan="1" rowspan="1">G</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mrMLM</td>
<td align="left" rowspan="2" colspan="1">OAR4_56152760.1</td>
<td align="left" rowspan="2" colspan="1">4</td>
<td align="left" colspan="1" rowspan="1">0.161</td>
<td align="left" colspan="1" rowspan="1">7.338</td>
<td align="left" colspan="1" rowspan="1">8.213</td>
<td align="left" colspan="1" rowspan="1">2.357</td>
<td align="left" rowspan="2" colspan="1">0.37</td>
<td align="left" rowspan="2" colspan="1">C</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FASTmrMLM</td>
<td align="left" colspan="1" rowspan="1">0.168</td>
<td align="left" colspan="1" rowspan="1">8.44</td>
<td align="left" colspan="1" rowspan="1">9.343</td>
<td align="left" colspan="1" rowspan="1">2.58</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">pLARmEB</td>
<td align="left" colspan="1" rowspan="1">OAR5_92790718.1</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">0.164</td>
<td align="left" colspan="1" rowspan="1">7.375</td>
<td align="left" colspan="1" rowspan="1">8.25</td>
<td align="left" colspan="1" rowspan="1">3.6</td>
<td align="left" colspan="1" rowspan="1">0.477</td>
<td align="left" colspan="1" rowspan="1">G</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mrMLM</td>
<td align="left" colspan="1" rowspan="1">OAR10_31255390.1</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">0.14</td>
<td align="left" colspan="1" rowspan="1">5.431</td>
<td align="left" colspan="1" rowspan="1">6.244</td>
<td align="left" colspan="1" rowspan="1">1.657</td>
<td align="left" colspan="1" rowspan="1">0.365</td>
<td align="left" colspan="1" rowspan="1">G</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mrMLM</td>
<td align="left" colspan="1" rowspan="1">Chr19:48330391</td>
<td align="left" colspan="1" rowspan="1">19</td>
<td align="left" colspan="1" rowspan="1">−0.225</td>
<td align="left" colspan="1" rowspan="1">5.811</td>
<td align="left" colspan="1" rowspan="1">6.638</td>
<td align="left" colspan="1" rowspan="1">1.434</td>
<td align="left" colspan="1" rowspan="1">0.258</td>
<td align="left" colspan="1" rowspan="1">G</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FASTmrEMMA</td>
<td align="left" colspan="1" rowspan="1">oar3_OAR22_24309813</td>
<td align="left" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1">−0.657</td>
<td align="left" colspan="1" rowspan="1">5.077</td>
<td align="left" colspan="1" rowspan="1">5.876</td>
<td align="left" colspan="1" rowspan="1">2.042</td>
<td align="left" colspan="1" rowspan="1">0.054</td>
<td align="left" colspan="1" rowspan="1">G</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FASTmrEMMA</td>
<td align="left" colspan="1" rowspan="1">Chr23:960746</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">−0.419</td>
<td align="left" colspan="1" rowspan="1">5.093</td>
<td align="left" colspan="1" rowspan="1">5.893</td>
<td align="left" colspan="1" rowspan="1">1.362</td>
<td align="left" colspan="1" rowspan="1">0.09</td>
<td align="left" colspan="1" rowspan="1">A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">pLARmEB</td>
<td align="left" colspan="1" rowspan="1">oar3_OAR24_30511307</td>
<td align="left" colspan="1" rowspan="1">24</td>
<td align="left" colspan="1" rowspan="1">0.149</td>
<td align="left" colspan="1" rowspan="1">5.261</td>
<td align="left" colspan="1" rowspan="1">6.067</td>
<td align="left" colspan="1" rowspan="1">0.913</td>
<td align="left" colspan="1" rowspan="1">0.26</td>
<td align="left" colspan="1" rowspan="1">T</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p36"><p><sup>1</sup>The proportion of phenotypic variance explained by significant SNP(%)</p></div></div></section><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333283_12917_2025_4957_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc27/12333283/f03569263bfa/12917_2025_4957_Fig2_HTML.jpg" loading="lazy" id="MO3" height="440" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Manhattan plots for illustrating significant SNPs associated with the BW (<strong>A</strong>) and WW (<strong>B</strong>). Manhattan plots showing -log10(<em>p</em>-values) and LOD score for SNPs across chromosomes. The horizontal gray line indicates the significance threshold (LOD ≥ 5). Pink points represent SNPs detected by multiple methods</p></figcaption></figure><p id="Par31">For the WW, a total of seven SNPs that met the LOD threshold of ≥ 5 were identified using four different methods: mrMLM, FASTmrMLM, ISIS EM-BLASSO, and FASTmrEMMA. These methods were effective in pinpointing SNPs that significantly affect the WW. However, the pLARmEB algorithm did not identify any SNPs meeting the critical threshold for the WW (Table <a href="#Tab3" class="usa-link">3</a>; Fig. <a href="#Fig2" class="usa-link">2</a>B).</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Identified significant SNPs associated with the WW</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Method</th>
<th align="left" colspan="1" rowspan="1">Marker</th>
<th align="left" colspan="1" rowspan="1">Chr</th>
<th align="left" colspan="1" rowspan="1">SNP Effect</th>
<th align="left" colspan="1" rowspan="1">LOD score</th>
<th align="left" colspan="1" rowspan="1">-log10(<em>P</em>)</th>
<th align="left" colspan="1" rowspan="1">
<em>r</em><sup>2</sup> (%)<sup>1</sup>
</th>
<th align="left" colspan="1" rowspan="1">MAF</th>
<th align="left" colspan="1" rowspan="1">Allele</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">mrMLM</td>
<td align="left" rowspan="3" colspan="1">s09049.1</td>
<td align="left" rowspan="3" colspan="1">3</td>
<td align="left" colspan="1" rowspan="1">−0.179</td>
<td align="left" colspan="1" rowspan="1">5.629</td>
<td align="left" colspan="1" rowspan="1">6.449</td>
<td align="left" colspan="1" rowspan="1">2.022</td>
<td align="left" rowspan="3" colspan="1">0.394</td>
<td align="left" rowspan="3" colspan="1">A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FASTmrMLM</td>
<td align="left" colspan="1" rowspan="1">−0.169</td>
<td align="left" colspan="1" rowspan="1">5.877</td>
<td align="left" colspan="1" rowspan="1">6.706</td>
<td align="left" colspan="1" rowspan="1">1.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISIS EM-BLASSO</td>
<td align="left" colspan="1" rowspan="1">−0.155</td>
<td align="left" colspan="1" rowspan="1">5.237</td>
<td align="left" colspan="1" rowspan="1">6.042</td>
<td align="left" colspan="1" rowspan="1">1.529</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FASTmrEMMA</td>
<td align="left" colspan="1" rowspan="1">OAR3_146832060.1</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">−0.546</td>
<td align="left" colspan="1" rowspan="1">5.028</td>
<td align="left" colspan="1" rowspan="1">5.825</td>
<td align="left" colspan="1" rowspan="1">2.168</td>
<td align="left" colspan="1" rowspan="1">0.119</td>
<td align="left" colspan="1" rowspan="1">A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mrMLM</td>
<td align="left" rowspan="3" colspan="1">OAR6_116658909.1</td>
<td align="left" rowspan="3" colspan="1">6</td>
<td align="left" colspan="1" rowspan="1">−0.254</td>
<td align="left" colspan="1" rowspan="1">7.088</td>
<td align="left" colspan="1" rowspan="1">7.955</td>
<td align="left" colspan="1" rowspan="1">1.54</td>
<td align="left" rowspan="3" colspan="1">0.25</td>
<td align="left" rowspan="3" colspan="1">A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FASTmrMLM</td>
<td align="left" colspan="1" rowspan="1">−0.229</td>
<td align="left" colspan="1" rowspan="1">8.246</td>
<td align="left" colspan="1" rowspan="1">9.144</td>
<td align="left" colspan="1" rowspan="1">1.257</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISIS EM-BLASSO</td>
<td align="left" colspan="1" rowspan="1">−0.205</td>
<td align="left" colspan="1" rowspan="1">5.497</td>
<td align="left" colspan="1" rowspan="1">6.312</td>
<td align="left" colspan="1" rowspan="1">1.005</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mrMLM</td>
<td align="left" rowspan="3" colspan="1">OAR9_39173859.1</td>
<td align="left" rowspan="3" colspan="1">9</td>
<td align="left" colspan="1" rowspan="1">0.185</td>
<td align="left" colspan="1" rowspan="1">6.746</td>
<td align="left" colspan="1" rowspan="1">7.602</td>
<td align="left" colspan="1" rowspan="1">1.915</td>
<td align="left" rowspan="3" colspan="1">0.356</td>
<td align="left" rowspan="3" colspan="1">C</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FASTmrMLM</td>
<td align="left" colspan="1" rowspan="1">0.186</td>
<td align="left" colspan="1" rowspan="1">7.363</td>
<td align="left" colspan="1" rowspan="1">8.237</td>
<td align="left" colspan="1" rowspan="1">1.941</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISIS EM-BLASSO</td>
<td align="left" colspan="1" rowspan="1">0.192</td>
<td align="left" colspan="1" rowspan="1">6.824</td>
<td align="left" colspan="1" rowspan="1">7.684</td>
<td align="left" colspan="1" rowspan="1">2.081</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mrMLM</td>
<td align="left" colspan="1" rowspan="1">s68294.1</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">−0.12</td>
<td align="left" colspan="1" rowspan="1">6.151</td>
<td align="left" colspan="1" rowspan="1">6.989</td>
<td align="left" colspan="1" rowspan="1">1.224</td>
<td align="left" colspan="1" rowspan="1">0.451</td>
<td align="left" colspan="1" rowspan="1">T</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mrMLM</td>
<td align="left" rowspan="2" colspan="1">oar3_OAR13_33751699</td>
<td align="left" rowspan="2" colspan="1">13</td>
<td align="left" colspan="1" rowspan="1">−0.184</td>
<td align="left" colspan="1" rowspan="1">5.472</td>
<td align="left" colspan="1" rowspan="1">6.286</td>
<td align="left" colspan="1" rowspan="1">2.054</td>
<td align="left" rowspan="2" colspan="1">0.367</td>
<td align="left" rowspan="2" colspan="1">A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISIS EM-BLASSO</td>
<td align="left" colspan="1" rowspan="1">−0.18</td>
<td align="left" colspan="1" rowspan="1">5.137</td>
<td align="left" colspan="1" rowspan="1">5.938</td>
<td align="left" colspan="1" rowspan="1">1.98</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISIS EM-BLASSO</td>
<td align="left" colspan="1" rowspan="1">s63779.1</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">−0.207</td>
<td align="left" colspan="1" rowspan="1">6.971</td>
<td align="left" colspan="1" rowspan="1">7.834</td>
<td align="left" colspan="1" rowspan="1">2.699</td>
<td align="left" colspan="1" rowspan="1">0.394</td>
<td align="left" colspan="1" rowspan="1">G</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p40"><p><sup>1</sup>The proportion of phenotypic variance explained by significant SNP(%)</p></div></div></section><p id="Par32">The calculated <em>r</em><sup>2</sup> values indicate that the proportion of phenotypic variance explained by significant SNPs for the BW ranged from 0.348 to 3.6%. The cumulative r<sup>2</sup> of all eleven SNPs reached 20.66%, accounting for one-fifth of the total phenotypic variance. The individual SNP effects ranged from − 0.657 to 0.172. Markers oar3_OAR4_105954111, OAR4_56152760.1, OAR5_92790718.1, OAR10_31255390.1, and oar3_OAR24_30511307 were shown to have a positive effect on the BW, while the remaining six markers exhibited a negative effect (Table <a href="#Tab2" class="usa-link">2</a>).</p>
<p id="Par33">For the WW, <em>r</em><sup>2</sup> values ranged from 1.122 to 2.699%, and the cumulative r<sup>2</sup> of the seven WW-associated SNPs was calculated at 13.79%, explaining one-seventh of the total phenotypic variance. Individual SNP effects for the WW ranged from − 0.546 to 0.192. The only positively effective marker on the WW was OAR9_39173859.1 (Table <a href="#Tab3" class="usa-link">3</a>). For both traits, when a SNP was detected as significant by multiple models, the direction of its effect (positive or negative) was consistent across all models.</p>
<p id="Par34">The quantile-quantile (QQ) plots indicated that there were no notable signs of inflation or deflation in the test statistics, suggesting that the observed <em>p</em>-values closely follow the expected distribution under the null hypothesis. The genomic inflation factor (λ) was calculated to further assess the presence of any systematic bias. For the BW, the λ value was 1.006 (Fig. <a href="#Fig3" class="usa-link">3</a>A), and for the WW, the λ value was 1.008 (Fig. <a href="#Fig3" class="usa-link">3</a>B). These values are very close to 1, which confirms that there was effective management of potential population stratification and/or cryptic relatedness during the multi-locus GWAS.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333283_12917_2025_4957_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc27/12333283/296f8c88f39a/12917_2025_4957_Fig3_HTML.jpg" loading="lazy" id="MO4" height="366" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>QQ plots for detecting inflation or deflation of the multi-locus GWAS results for the BW (<strong>A</strong>) and WW (<strong>B</strong>) traits. QQ plots comparing observed versus expected -log10(<em>P</em>-values). The diagonal line represents the null distribution. Points following the diagonal line indicate that the observed <em>p</em>-values follow the expected distribution under the null hypothesis</p></figcaption></figure><p id="Par36">In the association analysis, multiple detections of significant SNPs were observed. For instance, during the analysis for the BW, the SNPs oar3_OAR2_58511524 and OAR4_56152760.1 were co-detected by the two methods (mrMLM and FASTmrMLM), whereas the SNP Chr3:84999096 was detected three times by mrMLM, FASTmrMLM, and FASTmrEMMA. The ISIS EM-BLASSO has not been able to identify any SNP meeting the critical threshold for the BW. The remaining SNPs were identified by any of the other four methods (Fig. <a href="#Fig4" class="usa-link">4</a>A). In the WW analysis, the SNPs s09049.1, OAR6_116658909.1, and OAR9_39173859.1 were identified three times by mrMLM, FASTmrMLM, and ISIS EM-BLASSO, whereas oar3_OAR13_33751699 was co-detected by the mrMLM and ISIS EM-BLASSO. Other SNPs were identified by any of the other methods except pLARmEB (Fig. <a href="#Fig4" class="usa-link">4</a>B).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333283_12917_2025_4957_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc27/12333283/a6897e370e77/12917_2025_4957_Fig4_HTML.jpg" loading="lazy" id="MO5" height="280" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Venn diagrams illustrating co-detected markers for the BW (<strong>A</strong>) and WW (<strong>B</strong>)</p></figcaption></figure><p id="Par38">The post hoc power analysis results were calculated to be as follows: for the BW trait, the SNP with the lowest <em>r</em>² value (OAR5_92790718.1; <em>r</em>² = 0.913; MAF = 0.26) and the SNP with the highest <em>r</em>² value (oar3_OAR24_30511307; <em>r</em>² = 3.6; MAF = 0.477) yielded statistical powers of 0.75 and 0.99, respectively. For the WW trait, the SNP with the lowest <em>r</em>² value (s68294.1; r² = 1.224; MAF = 0.451) and the SNP with the highest <em>r</em>² value (s63779.1; <em>r</em>² = 2.699; MAF = 0.394) yielded statistical powers of 0.86 and 0.99, respectively (Fig. <a href="#Fig5" class="usa-link">5</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333283_12917_2025_4957_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc27/12333283/2e8d4bc931a7/12917_2025_4957_Fig5_HTML.jpg" loading="lazy" id="MO6" height="578" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Statistical power analyses for the BW and WW traits. Power analysis plots showing the relationship between sample size (x-axis) and statistical power (y-axis) for the BW and WW traits. The red dashed line indicates the acceptable minimum statistical power threshold (0.8). The intersection point between the ascending curve and red dashed line determines the required sample size to achieve 80% statistical power</p></figcaption></figure></section><section id="Sec11"><h3 class="pmc_sec_title">Gene annotation analysis</h3>
<p id="Par40">Gene annotation analysis revealed that the majority of the associated SNPs were located either within a ± 100 Kb proximity to protein-coding genes or within the intron regions of these genes. Specifically, out of the total 18 SNPs that were associated with either the BW or WW, the majority were closely linked to genes, enhancing the likelihood that these SNPs play a functional role in influencing the traits. Only four of these SNPs were located at a distance greater than 100 Kb from the nearest gene (Table <a href="#Tab4" class="usa-link">4</a>).</p>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Gene annotations for the BW and WW associated SNPs</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Marker</th>
<th align="left" colspan="1" rowspan="1">Chr</th>
<th align="left" colspan="1" rowspan="1">rs</th>
<th align="left" colspan="1" rowspan="1">Bp</th>
<th align="left" colspan="1" rowspan="1">Localization</th>
<th align="left" colspan="1" rowspan="1">Nearest gene</th>
<th align="left" colspan="1" rowspan="1">Distance (Kb)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">oar3_OAR2_58511524</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">rs415620599</td>
<td align="left" colspan="1" rowspan="1">58,511,524</td>
<td align="left" colspan="1" rowspan="1">Intron</td>
<td align="left" colspan="1" rowspan="1">
<em>GNAQ</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">OAR2_16538077.1</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">rs401147797</td>
<td align="left" colspan="1" rowspan="1">16,538,077</td>
<td align="left" colspan="1" rowspan="1">Intergenic</td>
<td align="left" colspan="1" rowspan="1">
<em>TMEM38B</em>
</td>
<td align="left" colspan="1" rowspan="1">273</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Chr3:84999096</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">
<em>na</em>
</td>
<td align="left" colspan="1" rowspan="1">84,999,096</td>
<td align="left" colspan="1" rowspan="1">Intron</td>
<td align="left" colspan="1" rowspan="1">
<em>CDKL4</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">oar3_OAR4_105954111</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">rs429073088</td>
<td align="left" colspan="1" rowspan="1">105,954,111</td>
<td align="left" colspan="1" rowspan="1">Intron</td>
<td align="left" colspan="1" rowspan="1">
<em>PIP</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">OAR4_56152760.1</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">rs401526652</td>
<td align="left" colspan="1" rowspan="1">56,152,760</td>
<td align="left" colspan="1" rowspan="1">Intergenic</td>
<td align="left" colspan="1" rowspan="1">
<em>MDFIC</em>
</td>
<td align="left" colspan="1" rowspan="1">417</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">OAR5_92790718.1</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">rs399038638</td>
<td align="left" colspan="1" rowspan="1">84,591,689</td>
<td align="left" colspan="1" rowspan="1">Intergenic</td>
<td align="left" colspan="1" rowspan="1">
<em>CCNH</em>
</td>
<td align="left" colspan="1" rowspan="1">196</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">OAR10_31255390.1</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">
<em>na</em>
</td>
<td align="left" colspan="1" rowspan="1">31,255,390</td>
<td align="left" colspan="1" rowspan="1">Intron</td>
<td align="left" colspan="1" rowspan="1">
<em>SLC7A1</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Chr19:48330391</td>
<td align="left" colspan="1" rowspan="1">19</td>
<td align="left" colspan="1" rowspan="1">
<em>na</em>
</td>
<td align="left" colspan="1" rowspan="1">48,330,391</td>
<td align="left" colspan="1" rowspan="1">Intron</td>
<td align="left" colspan="1" rowspan="1">
<em>PBRM1</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">oar3_OAR22_24309813</td>
<td align="left" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1">rs413210283</td>
<td align="left" colspan="1" rowspan="1">24,309,813</td>
<td align="left" colspan="1" rowspan="1">Intron</td>
<td align="left" colspan="1" rowspan="1">
<em>SORCS3</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Chr23:960746</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">
<em>na</em>
</td>
<td align="left" colspan="1" rowspan="1">960,746</td>
<td align="left" colspan="1" rowspan="1">Upstream</td>
<td align="left" colspan="1" rowspan="1">
<em>NFATC1</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">oar3_OAR24_30511307</td>
<td align="left" colspan="1" rowspan="1">24</td>
<td align="left" colspan="1" rowspan="1">rs410817330</td>
<td align="left" colspan="1" rowspan="1">30,511,307</td>
<td align="left" colspan="1" rowspan="1">Intergenic</td>
<td align="left" colspan="1" rowspan="1">
<em>ENSOARG00000010825</em>
</td>
<td align="left" colspan="1" rowspan="1">109</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Marker</td>
<td align="left" colspan="1" rowspan="1">Chr</td>
<td align="left" colspan="1" rowspan="1">rs</td>
<td align="left" colspan="1" rowspan="1">Bp</td>
<td align="left" colspan="1" rowspan="1">Localization</td>
<td align="left" colspan="1" rowspan="1">Nearest gene</td>
<td align="left" colspan="1" rowspan="1">Distance (Kb)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">s09049.1</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">rs415341962</td>
<td align="left" colspan="1" rowspan="1">34,939,097</td>
<td align="left" colspan="1" rowspan="1">Intron</td>
<td align="left" colspan="1" rowspan="1">
<em>BABAM2</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">OAR3_146832060.1</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">rs420551229</td>
<td align="left" colspan="1" rowspan="1">146,832,060</td>
<td align="left" colspan="1" rowspan="1">Intergenic</td>
<td align="left" colspan="1" rowspan="1">
<em>LALBA</em>
</td>
<td align="left" colspan="1" rowspan="1">1.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">OAR6_116658909.1</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">rs401509021</td>
<td align="left" colspan="1" rowspan="1">116,658,909</td>
<td align="left" colspan="1" rowspan="1">Intron</td>
<td align="left" colspan="1" rowspan="1">
<em>NOP14</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">OAR9_39173859.1</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">rs416943304</td>
<td align="left" colspan="1" rowspan="1">39,173,859</td>
<td align="left" colspan="1" rowspan="1">Intergenic</td>
<td align="left" colspan="1" rowspan="1">
<em>FAM110B</em>
</td>
<td align="left" colspan="1" rowspan="1">39</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">s68294.1</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">
<em>na</em>
</td>
<td align="left" colspan="1" rowspan="1">37,997,787</td>
<td align="left" colspan="1" rowspan="1">Intron</td>
<td align="left" colspan="1" rowspan="1">
<em>SKAP1</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">oar3_OAR13_33751699</td>
<td align="left" colspan="1" rowspan="1">13</td>
<td align="left" colspan="1" rowspan="1">rs402915340</td>
<td align="left" colspan="1" rowspan="1">33,751,699</td>
<td align="left" colspan="1" rowspan="1">Intron</td>
<td align="left" colspan="1" rowspan="1">
<em>SVIL</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">s63779.1</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">
<em>na</em>
</td>
<td align="left" colspan="1" rowspan="1">39,799,799</td>
<td align="left" colspan="1" rowspan="1">Intergenic</td>
<td align="left" colspan="1" rowspan="1">
<em>ATXN1</em>
</td>
<td align="left" colspan="1" rowspan="1">80</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec12"><h3 class="pmc_sec_title">Gene enrichment and pathway analysis</h3>
<p id="Par42">Pathway analysis demonstrated that the candidate genes located within a ± 100 Kb proximity of the associated SNPs are involved in a diverse array of biological pathways. For both traits, these candidate genes participate in numerous pathways that may play a role in growth and development (Fig. <a href="#Fig6" class="usa-link">6</a>A and B). This comprehensive involvement in multiple pathways underscores the complex genetic architecture underlying both the BW and WW traits, highlighting the multifaceted nature of genetic regulation and its impact on these phenotypes.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333283_12917_2025_4957_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc27/12333283/d4f984c73ebe/12917_2025_4957_Fig6_HTML.jpg" loading="lazy" id="MO7" height="661" width="669" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Pathways involving candidate genes for the BW (<strong>A</strong>) and WW (<strong>B</strong>) traits located within a ± 100 Kb range of associated markers. Bar length indicates fold enrichment magnitude, while color intensity (purple-pink) represents statistical significance (-log10(FDR)). Darker colors indicate higher significance levels</p></figcaption></figure></section><section id="Sec13"><h3 class="pmc_sec_title">Promoter and intron splicing prediction</h3>
<p id="Par44">
Our analysis demonstrated that none of the SNPs located in intergenic regions or upstream/downstream of genes were found to be localized within promoter regions. Additionally, investigation of SNPs situated within intronic regions revealed no evidence of their involvement in intron splicing processes (Supplemental Table <a href="#MOESM1" class="usa-link">S1</a>).</p></section></section><section id="Sec14"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par45">SRBI is a breeding enterprise that, in addition to being a research institute specialized in small ruminants under the Ministry of Agriculture and Forestry of Turkey, provides high-quality Karacabey Merino breeding rams to breeders. The Karacabey Merino was first developed at SRBI and distributed from there throughout Turkey, especially in the northwest. Currently, the nucleus Karacabey Merino flock of Turkey has been reared in the SRBI. In the SRBI herds, after closing to back-crossing, classical breeding efforts have been implemented for at least three decades. Thus, it is meaningful that the low heritability for the BW and WW traits, 0.282 and 0.164, respectively, indicates that most of the variations originates from the environment as a result of these ongoing classical breeding efforts. Additionally, the computed coefficient of variation indicates moderate variability, showing that both traits are influenced by multiple genetic and environmental factors. These factors, which lie beyond our ability to predict, may stem from numerous sources, such as the mother’s nurturing ability or maternal instincts, variations in the quantity and nutrient content of maternal milk, undiagnosed infections in the offspring, or individual epigenetic mechanisms that may respond differently to environmental conditions. However, despite the low heritability, there is still an opportunity for genetic improvement of these traits. Because SRBI is an official breeding enterprise that regularly provides breeding rams to farmers, it is evident that further genetic advancements in SRBI flocks will have a multiplicative effect on the productivity of farmers’ herds. Furthermore, given the genetic correlation of 0.857 between the two traits, it can be concluded that the BW is a strong predictor of the WW in Karacabey Merino lambs.</p>
<p id="Par46">Statistical power analysis, which included SNPs with the lowest and highest <em>r</em><sup>2</sup> values associated with both the BW and WW, has shown that: the total sample size of 736 barely meets the 0.8 statistical power threshold for the SNP with the lowest <em>r</em><sup>2</sup> value associated with the BW, while the mentioned sample size easily meets the &gt; 0.8 statistical power threshold for the SNP with the highest <em>r</em><sup>2</sup> value associated with the BW and for the SNPs with the lowest and highest <em>r</em><sup>2</sup> values for the WW (Fig. <a href="#Fig5" class="usa-link">5</a>).</p>
<p id="Par47">To enhance the reliability of association analysis results and obtain more robust findings, we implemented a stringent significance threshold of LOD ≥ 5 instead of the conventional LOD ≥ 3, representing a 100-fold more stringent adjustment. This elevated threshold was observed to influence the co-detection rate. As expected, different algorithms detected associated SNPs at varying significance levels. For instance, under the current LOD ≥ 5 threshold for the WW phenotype, among the seven significant SNPs identified, three SNPs were detected by one model, one SNP by two models, and three SNPs by three models. In contrast, when applying the LOD ≥ 3 threshold, six SNPs were detected by three models and one SNP by four models for the same trait, resulting in a more satisfactory parallel detection rate. As a notable observation, while pLARmEB was able to detect SNPs associated with the WW trait under LOD ≥ 3 threshold conditions, ISIS EM-BLASSO failed to identify any SNPs associated with the BW trait even at the same LOD threshold. The underlying basis for this differential detection capability remains unclear. To facilitate independent evaluation by readers, the multi-locus GWAS results under LOD ≥ 3 conditions are presented in Supplemental Table <a href="#MOESM1" class="usa-link">S1</a>.</p>
<p id="Par48">Among the SNPs associated with the BW, oar3_OAR2_58511524, Chr3:84999096, oar3_OAR4_105954111, OAR10_31255390.1, Chr19:48330391, and oar3_OAR22_24309813 were located in the intron regions of the <em>GNAQ</em>, <em>CDKL4</em>, <em>PIP</em>, <em>SLC7A1</em>, <em>PBRM1</em>, and <em>SORCS3</em> genes, respectively, while Chr23:960746 was located upstream of the <em>NFATC1</em> gene. Similarly, four of the SNPs associated with the WW, s09049.1, OAR6_116658909.1, s68294.1, and oar3_OAR13_33751699, were found in the introns of the <em>BABAM2</em>, <em>NOP14</em>, <em>SKAP1</em>, and <em>SVIL</em> genes, respectively. These findings highlight the importance of sequencing the coding regions of these associated genes to identify potential causal mutations. Other annotated genes for both traits were located in the intergenic regions (Table <a href="#Tab4" class="usa-link">4</a>).</p>
<p id="Par49">Some of the annotated genes are involved in critical processes, i.e., the regulation of signaling pathways, cell cycle, cellular growth, proliferation, and skeletal development, suggesting that they may be directly or indirectly related to growth performance.</p>
<p id="Par50">The <em>GNAQ</em> gene is known to have overlapping functions with its paralog <em>GNA11</em> in melanocytes, where they upregulate the MAP kinase pathway when constitutively active [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Mutations in <em>GNAQ</em> have been shown to activate the RAF/MEK/ERK pathway, mimicking growth factor signaling and leading to the transcriptional activation of cell cycle genes [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Moreover, <em>GNAQ</em> mutations have been linked to differential activation of the mTOR pathway in human-transformed cells [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. While mTOR’s role as a regulator of growth and protein translation was previously established, contemporary understanding emphasizes its pivotal function as a central orchestrator of organismal metabolism, elicited by environmental and hormonal signals [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>].</p>
<p id="Par51">The <em>TMEM38B</em> gene encodes the trimeric intracellular cation-B (TRIC-B) protein, which plays a crucial role in various biological processes, particularly in early skeletal development and bone cell differentiation. Studies have demonstrated that <em>TMEM38B</em> is essential for normal skeletal development in zebrafish, as <em>TMEM38B</em> knockout has been shown to result in skeletal abnormalities [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. In humans, <em>TMEM38B</em> has been reported as a candidate gene for pubertal growth [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Furthermore, mutations in <em>TMEM38B</em> have been associated with rare autosomal recessive osteogenesis imperfecta, further emphasizing its significance in growth properties [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Additionally, <em>TMEM38B</em> has been linked to ion channel activity and calcium release, which may have implications for cellular growth and signaling processes [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>].</p>
<p id="Par52"><em>CDKL4</em> belongs to the cyclin-dependent protein kinase family and is responsible for cell cycle progression, including transferase activity, transferring phosphorous-containing groups, and protein tyrosine kinase activity [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Also, in their study, Meckes et al. [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] suggested that the <em>CDKL4</em> gene could be a candidate gene for behavioral responsiveness to acute stress.</p>
<p id="Par53">Prolactin-induced protein (<em>PIP</em>) is a small secreted glycoprotein, equipped with multiple N-linked carbohydrate chains. Evidence indicates that <em>PIP</em> possesses immunomodulatory capabilities and is integral to cell-mediated adaptive immunity. Its interaction with bacteria across various genera implies potential involvement in innate immunity, augmenting host defenses against microbial threats [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. It is estimated that the PIP gene has been involved in cell cycle regulation, DNA replication, recombination, and repair, particularly in cellular growth and proliferation [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>].</p>
<p id="Par54">Huang et al. [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>] identified two target genes, <em>AKT</em>1 and <em>MDFIC</em>, regulated by miR-146b-3p, and observed that miR-146b-3p has the ability to suppress the PI3K/AKT pathway. The results revealed that both <em>AKT1</em> and <em>MDFIC</em> enhance myoblast proliferation and differentiation while reducing apoptosis in chickens, illustrating that miR-146b-3p plays a direct role in modulating the proliferation, differentiation, and apoptosis of chicken myoblasts by targeting the PI3K/AKT pathway and <em>MDFIC</em>. Furthermore, findings of a GWA study indicate that the specific variants at the MyoD family inhibitor domain containing <em>MDFIC</em> gene could potentially influence an individual’s birth weight in pigs [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>].</p>
<p id="Par55">The transporter <em>SLC7A1</em>, also known as <em>CAT1</em>, plays a crucial role in facilitating the uptake of cationic amino acids such as arginine, lysine, and ornithine, which are vital for cellular growth [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. The <em>CAT-1</em> gene is integral to numerous biological processes and is essential for cell survival under stress conditions, enabling cells to resume growth promptly upon the availability of amino acids [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Additionally, this gene is implicated in cell susceptibility to viral infections, serving as a receptor for specific viruses [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. Furthermore, the expression of <em>SLC7A1</em> is induced by stress, indicating its involvement in stress-induced gene regulation [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>].</p>
<p id="Par56">The molecular functions of <em>PBRM1</em> extend to its involvement in stress response mechanisms in epithelial cells, impacting pathways related to cell adhesion, neuronal processes, apoptosis, and ER stress or proteolysis [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. Re-expression of <em>PBRM1</em> has been shown to lead to the upregulation of genes involved in cellular adhesion, carbohydrate metabolism, apoptotic processes, and response to hypoxia, while downregulating genes associated with different stages of cell division. Furthermore, <em>PBRM1</em> controls genes influencing cell growth and mobility, highlighting its significance in these processes [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>].</p>
<p id="Par57">The <em>NFATC1</em> gene, a member of the NFAT transcription factor family, plays crucial roles in various cellular processes, including cell and muscle growth. <em>NFATC1</em> has been identified as a key regulator of skeletal muscle fiber type, with its transcriptional activation of slow-fiber-specific genes mediated by a combinatorial mechanism involving proteins of the NFAT and MEF2 families [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. <em>NFATC1</em> has been implicated in bone homeostasis, where it is essential for the differentiation of osteoclasts in the skeletal system [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. Additionally, <em>NFATC1</em> has been associated with promoting muscle fiber type transformation through CaN/NFATc1 signaling in myoblasts, indicating its involvement in regulating muscle fiber-related gene expression [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. <em>NFATC1</em> has also been shown to control the growth of multinucleated muscle cells through an NFATC2-dependent pathway, suggesting its role in regulating muscle cell fusion and growth [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>].</p>
<p id="Par58">The <em>BABAM2</em> gene has been implicated in regulating cell cycle progression, pluripotency, anti-apoptosis, and radioresistance in various cellular contexts. <em>BABAM2</em> has been shown to maintain cell cycle regulation and pluripotency in mouse embryonic stem cells in response to induced DNA damage, indicating its role in maintaining cellular homeostasis [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. Additionally, <em>BABAM2</em> has been reported to mediate the functions of Pancreatic Progenitor Cell Differentiation And Proliferation Factor gene in anti-apoptosis and radioresistance, suggesting its involvement in cellular survival mechanisms [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>].</p>
<p id="Par59">The lactalbumin protein encoded by the <em>LALBA</em> gene plays a pivotal role in the biosynthesis of lactose within the mammary gland. Its function primarily involves augmenting the substrate affinity of beta-1,4-galactosyltransferase (B4GALT1), an enzyme responsible for catalyzing the conversion of glucose and uridine diphosphate galactose into lactose. The role of the <em>LALBA</em> gene in lactation and milk production is well-established [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. Because lactalbumin is a key protein in the lactose synthase complex, which is crucial for lactose production in mammary glands during lactation, one can speculate that relevant <em>LALBA</em> variants inherited from the mother may contribute to better lamb feeding of ewes and increased growth performance of lambs.</p>
<p id="Par60"><em>SKAP1</em> has been identified as a key regulator in T cell motility and interactions, where it regulates RapL-Rap1 complex formation and binding to lymphocyte function-associated antigen-1 (LFA-1) for clustering. Additionally, <em>SKAP1</em> acts as a scaffold for Polo-like kinase 1, facilitating optimal cell cycling necessary for T-cell clonal expansion in response to antigenic activation [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>].</p>
<p id="Par61">The <em>SVIL</em> gene encodes the supervillin protein, which plays a significant role in cellular and molecular functions related to muscle growth processes. Supervillin is known to bind both myosin II and filamentous actin, interacting with various cytoskeletal proteins [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. This interaction is possibly crucial for the structural integrity and function of skeletal muscle, indicating <em>SVIL</em>’s involvement in muscle development and maintenance.</p>
<p id="Par62">The intricate interplay between genetic background and environmental influences contributes to the complexity observed in growth traits. This complexity manifests in the detection of a multitude of pathways associated with the trait under investigation. Specifically, when examining genes located within ± 100 Kb proximity to the BW-associated SNPs, we find their involvement in an extensive network of 19 distinct pathways, each contributing to various aspects of growth regulation and development. Similarly, genes situated in the vicinity of the WW-associated SNPs demonstrate their participation in 17 different pathways, underscoring the diverse molecular mechanisms underlying this trait’s expression and regulation (Fig. <a href="#Fig6" class="usa-link">6</a>).</p>
<p id="Par63">Transcription factors (TFs) are fundamental regulatory proteins that modulate gene expression through specific binding interactions with gene promoter regions. These molecular interactions are crucial for transcription initiation and the modulation of gene expression levels in response to diverse cellular signals. The architectural complexity of promoter regions, characterized by multiple transcription factor-binding sites, facilitates precise regulation of gene activity, thereby orchestrating various biological processes including development, cellular differentiation, and environmental response mechanisms [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>–<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>].</p>
<p id="Par64">Intron splicing represents a pivotal mechanism in eukaryotic gene expression, involving the precise excision of intronic sequences from pre-mRNA transcripts and the subsequent ligation of exons to generate mature mRNA. This highly coordinated process, which frequently occurs co-transcriptionally, is essential for the synthesis of functional proteins. The efficiency and accuracy of intron splicing are governed by multiple factors, including the recognition of specific splice sites, the coordinated assembly of splicing factors, and the broader transcriptional context [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>, <a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>].</p>
<p id="Par65">In our study, we explored the functional roles of trait-associated SNPs in promoter regions or intron splicing mechanisms but found no SNPs within these regulatory regions. This lack of regulatory SNPs suggests that these are not causal variants influencing the BW and WW traits and could instead be indirect markers. It is also possible that phenotypic variations in the BW and WW are mainly controlled by coding region variants affecting protein structure or function. Future studies should therefore consider a broader approach that includes multiple layers of genomic and protein function analysis.</p>
<section id="Sec15"><h3 class="pmc_sec_title">Limitations of the study</h3>
<p id="Par66">In the SRBI flocks, while classical breeding value estimation methods implemented over the past 20–25 years have yielded satisfactory improvements in growth performance, they have also led to an inherent limitation: increased genetic uniformity within the sheep population. This homogenization has manifested in relatively low heritability values and coefficients of variation for both the BW and WW traits. Nevertheless, the observed heritability values of 0.282 and 0.164, alongside coefficients of variation of 17.63% and 13.80% for the BW and WW respectively, suggest that there are still potential opportunities for Marker-assisted Selection (MAS) applications.</p>
<p id="Par67">The statistical power analysis revealed that the statistical power for some SNPs was just below or close to the acceptable threshold (≥ 0.8). Although the significance threshold was set at LOD ≥ 5 instead of LOD ≥ 3 to achieve more robust statistical results, these SNPs and their relevant annotated genes should be carefully considered. Another limitation of the study is that it was conducted within a single nucleus flock. Therefore, these results need to be tested and validated, particularly in crossbred and purebred Karacabey Merino flocks raised by local farmers. Similarly, the findings of this study do not represent all sheep breeds. Before applying these results to MAS studies in other breeds, further validation in larger populations is necessary.</p>
<p id="Par68">Furthermore, for the SNPs associated with both traits, the connections between the annotated genes and the molecular pathways in which they are involved are based on in silico calculations, making it impossible to assert definitive functional relationships. Additional studies are needed to explore possible direct and indirect functional connections of these genes.</p></section></section><section id="Sec16"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par69">The low heritability coefficients of both traits suggest significant environmental influences on trait variations. Despite the substantial impact of environmental factors, there remains notable potential for genetic improvement in these traits. Association analysis identified multiple significant SNPs consistently linked to the BW and WW traits across various methods, indicating their reliability. The complex relationship between genetic and environmental factors is evident in the involvement of numerous molecular pathways regulating growth, development, and metabolism. Understanding the functions of annotated genes and related pathways is crucial for devising targeted breeding and management strategies to enhance growth performance. This highlights the intricate interplay between genetics and the environment in shaping growth traits, emphasizing the need for comprehensive approaches to achieve genetic advancements and improve productivity in livestock farming.</p>
<p id="Par70">If the findings of this study are validated in different populations, it may be possible to screen the identified variants using faster and more cost-effective genotyping methods such as Restriction Fragment Length Polymorphism (RFLP), Sanger sequencing, Single-nucleotide Primer Extension (SNuPE), or TaqMan probes, and to utilise them as markers in breeding selection. In particular, implementing marker-assisted selection (MAS) in nucleus breeding flocks at the initial stage has the potential to create a multiplier effect in village-based flocks through the dissemination of improved genetics.</p></section><section id="Sec17"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12333283/bin/12917_2025_4957_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (33.8KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors express their gratitude to the General Directorate of Agricultural Research and Policies (TAGEM) for invaluable support. Language editing of this manuscript was assisted by the AI-based large language models (ChatGPT 4o and Claude 3.5 sonnet).</p></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>Y.Y. conceptualized and designed the study and performed the GWAS analysis. R.A., M.K., and Y.E.K. conducted laboratory work, E.H., A.Ö., and E.Ç. conducted fieldwork and data collection.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Funding for this research was provided by the Republic of Turkey Ministry of Agriculture and Forestry, General Directorate of Agricultural Research and Policies (TAGEM) (Project no: TAGEM/HAYSÜD/E/20/A4/P2/2141).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The datasets generated and/or analysed during the current study are available in the https://figshare.com/repository, https://figshare.com/articles/dataset/Karacabey_merino_50K_genotype_data/29184098?file=54931109.</p></section><section id="notes4"><h2 class="pmc_sec_title"> Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par71">This study was conducted in compliance with the guidelines of the Animal Experiments Local Ethics Committee and with an experimental protocol approved by the Ethics Committee for the Use of Animals in Research and Experimentation at the Sheep Breeding and Research Institute, Turkey (Approval no: 04.10.2021/049) and the authors complied with the ARRIVE guidelines. Informed consent was obtained from the SRBI administration prior to the study.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par72">Publication consent for the article was obtained from the SRBI administration.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par73">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Lv F-H, Agha S, Kantanen J, Colli L, Stucki S, Kijas JW, Joost S, Li M-H, Marsan PA. Adaptations to climate-mediated selective pressures in sheep. Mol Biol Evol. 2014;31:3324–43. 10.1093/molbev/msu264.
</cite> [<a href="https://doi.org/10.1093/molbev/msu264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4245822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25249477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lv%20F-H,%20Agha%20S,%20Kantanen%20J,%20Colli%20L,%20Stucki%20S,%20Kijas%20JW,%20Joost%20S,%20Li%20M-H,%20Marsan%20PA.%20Adaptations%20to%20climate-mediated%20selective%20pressures%20in%20sheep.%20Mol%20Biol%20Evol.%202014;31:3324%E2%80%9343.%2010.1093/molbev/msu264." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Mastrangelo S, Moioli B, Ahbara A, Latairish S, Portolano B, Pilla F, Ciani E. Genome-wide scan of fat-tail sheep identifies signals of selection for fat deposition and adaptation. Anim Prod Sci. 2019;59:835. 10.1071/an17753.</cite> [<a href="https://scholar.google.com/scholar_lookup?Mastrangelo%20S,%20Moioli%20B,%20Ahbara%20A,%20Latairish%20S,%20Portolano%20B,%20Pilla%20F,%20Ciani%20E.%20Genome-wide%20scan%20of%20fat-tail%20sheep%20identifies%20signals%20of%20selection%20for%20fat%20deposition%20and%20adaptation.%20Anim%20Prod%20Sci.%202019;59:835.%2010.1071/an17753." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>-Abied A, Bagadi A, Bordbar F, Pu Y, Augustino SMA, Xue X, Xing F, Gebreselassie G, Mwacharo J-LHJM, Ma Y, et al. Genomic diversity, population structure, and signature of selection in five Chinese native sheep breeds adapted to extreme environments. Genes. 2020;11:494. 10.3390/genes11050494.
</cite> [<a href="https://doi.org/10.3390/genes11050494" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7290715/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32365888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Abied%20A,%20Bagadi%20A,%20Bordbar%20F,%20Pu%20Y,%20Augustino%20SMA,%20Xue%20X,%20Xing%20F,%20Gebreselassie%20G,%20Mwacharo%20J-LHJM,%20Ma%20Y,%20et%20al.%20Genomic%20diversity,%20population%20structure,%20and%20signature%20of%20selection%20in%20five%20Chinese%20native%20sheep%20breeds%20adapted%20to%20extreme%20environments.%20Genes.%202020;11:494.%2010.3390/genes11050494." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Quan K, Li J, Han H, Wei H, Zhao J, Si H, Zhang X, Zhang D. Review of Huang-huai sheep, a new multiparous mutton sheep breed first identified in China. Trop Anim Health Prod. 2020. 10.1007/s11250-020-02453-w.
</cite> [<a href="https://doi.org/10.1007/s11250-020-02453-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7683451/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33230646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quan%20K,%20Li%20J,%20Han%20H,%20Wei%20H,%20Zhao%20J,%20Si%20H,%20Zhang%20X,%20Zhang%20D.%20Review%20of%20Huang-huai%20sheep,%20a%20new%20multiparous%20mutton%20sheep%20breed%20first%20identified%20in%20China.%20Trop%20Anim%20Health%20Prod.%202020.%2010.1007/s11250-020-02453-w." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Gutiérrez-Reinoso MA, Aponte PM, Cabezas J, Rodriguez-Alvarez L, Garcia-Herreros M. Genomic evaluation of primiparous high-producing dairy cows: inbreeding effects on genotypic and phenotypic production–reproductive traits. Animals. 2020;10: 1704. 10.3390/ani10091704.
</cite> [<a href="https://doi.org/10.3390/ani10091704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7552765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32967074/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guti%C3%A9rrez-Reinoso%20MA,%20Aponte%20PM,%20Cabezas%20J,%20Rodriguez-Alvarez%20L,%20Garcia-Herreros%20M.%20Genomic%20evaluation%20of%20primiparous%20high-producing%20dairy%20cows:%20inbreeding%20effects%20on%20genotypic%20and%20phenotypic%20production%E2%80%93reproductive%20traits.%20Animals.%202020;10:%201704.%2010.3390/ani10091704." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>-Sigdel A, Abdollahi-Arpanahi R, Aguilar I, Peñagaricano F. Whole genome mapping reveals novel genes and pathways involved in milk production under heat stress in US Holstein cows. Front Genet. 2019;10. 10.3389/fgene.2019.00928.</cite> [<a href="https://doi.org/10.3389/fgene.2019.00928" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6788456/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31636656/" class="usa-link">PubMed</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Gantner V, Bobić T, Potočnik K, Gregić M, Kučević D. Persistence of heat stress Eff Ect in dairy cows. Mljekarstvo/Mljekarstvo Com. 2019;69:30–41. 10.15567/mljekarstvo.2019.0103.</cite> [<a href="https://scholar.google.com/scholar_lookup?Gantner%20V,%20Bobi%C4%87%20T,%20Poto%C4%8Dnik%20K,%20Gregi%C4%87%20M,%20Ku%C4%8Devi%C4%87%20D.%20Persistence%20of%20heat%20stress%20Eff%20Ect%20in%20dairy%20cows.%20Mljekarstvo/Mljekarstvo%20Com.%202019;69:30%E2%80%9341.%2010.15567/mljekarstvo.2019.0103." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>-Knob DA, Neto ATT, Schweizer H, Weigand AC, Kappes R, Scholz AM. Energy balance indicators during the transition period and early lactation of purebred Holstein and Simmental cows and their crosses. Animals. 2021;11:309. 10.3390/ani11020309.
</cite> [<a href="https://doi.org/10.3390/ani11020309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7912011/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33530414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Knob%20DA,%20Neto%20ATT,%20Schweizer%20H,%20Weigand%20AC,%20Kappes%20R,%20Scholz%20AM.%20Energy%20balance%20indicators%20during%20the%20transition%20period%20and%20early%20lactation%20of%20purebred%20Holstein%20and%20Simmental%20cows%20and%20their%20crosses.%20Animals.%202021;11:309.%2010.3390/ani11020309." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>-Amundson LA, Rowson AD, Crump PM, Prichard AP, Cheng AA, Wimmler CE, Klister M, Weaver SR, Bascom SS, Nuzback DE, et al. Effect of induced hypocalcemia in nonlactating, nonpregnant Holstein cows fed negative DCAD with low, medium, or high concentrations of calcium. J Anim Science/Journal Anim Sci ASAS Ref Compendium. 2018;96:5010–23. 10.1093/jas/sky371.</cite> [<a href="https://doi.org/10.1093/jas/sky371" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6276567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30321366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Amundson%20LA,%20Rowson%20AD,%20Crump%20PM,%20Prichard%20AP,%20Cheng%20AA,%20Wimmler%20CE,%20Klister%20M,%20Weaver%20SR,%20Bascom%20SS,%20Nuzback%20DE,%20et%20al.%20Effect%20of%20induced%20hypocalcemia%20in%20nonlactating,%20nonpregnant%20Holstein%20cows%20fed%20negative%20DCAD%20with%20low,%20medium,%20or%20high%20concentrations%20of%20calcium.%20J%20Anim%20Science/Journal%20Anim%20Sci%20ASAS%20Ref%20Compendium.%202018;96:5010%E2%80%9323.%2010.1093/jas/sky371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>-Evans RD, Dillon P, Buckley F, Berry D, Wallace M, Ducrocq V, Garrick DJ. Trends in milk production, calving rate and survival of cows in 14 Irish dairy herds as a result of the introgression of Holstein-Friesian genes. Anim Science/Animal Sci. 2006;82:423–33. 10.1079/asc200660.</cite> [<a href="https://scholar.google.com/scholar_lookup?-Evans%20RD,%20Dillon%20P,%20Buckley%20F,%20Berry%20D,%20Wallace%20M,%20Ducrocq%20V,%20Garrick%20DJ.%20Trends%20in%20milk%20production,%20calving%20rate%20and%20survival%20of%20cows%20in%2014%20Irish%20dairy%20herds%20as%20a%20result%20of%20the%20introgression%20of%20Holstein-Friesian%20genes.%20Anim%20Science/Animal%20Sci.%202006;82:423%E2%80%9333.%2010.1079/asc200660." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>-Zhang J, Zhang C-L, Tuersuntuohe M, Liu S. Population structure and selective signature of sheep around Tarim basin. Front Ecol Evol. 2023;11. 10.3389/fevo.2023.1146561.</cite>
</li>
<li id="CR12">
<span class="label">12.</span><cite>Signer-Hasler H, Burren A, Ammann P, Drögemüller C, Flury C. Runs of homozygosity and signatures of selection: a comparison among eight local Swiss sheep breeds. Anim Genet. 2019;50:512–25. 10.1111/age.12828.
</cite> [<a href="https://doi.org/10.1111/age.12828" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31365135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Signer-Hasler%20H,%20Burren%20A,%20Ammann%20P,%20Dr%C3%B6gem%C3%BCller%20C,%20Flury%20C.%20Runs%20of%20homozygosity%20and%20signatures%20of%20selection:%20a%20comparison%20among%20eight%20local%20Swiss%20sheep%20breeds.%20Anim%20Genet.%202019;50:512%E2%80%9325.%2010.1111/age.12828." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Kijas JW, Porto-Neto L, Dominik S, Reverter A, Bunch R, McCulloch R, Hayes BJ, Brauning R, McEwan J. Linkage disequilibrium over short physical distances measured in sheep using a high‐density SNP chip. Anim Genet. 2014;45:754–7. 10.1111/age.12197.
</cite> [<a href="https://doi.org/10.1111/age.12197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25040320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kijas%20JW,%20Porto-Neto%20L,%20Dominik%20S,%20Reverter%20A,%20Bunch%20R,%20McCulloch%20R,%20Hayes%20BJ,%20Brauning%20R,%20McEwan%20J.%20Linkage%20disequilibrium%20over%20short%20physical%20distances%20measured%20in%20sheep%20using%20a%20high%E2%80%90density%20SNP%20chip.%20Anim%20Genet.%202014;45:754%E2%80%937.%2010.1111/age.12197." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Wen Y-J, Zhang H, Ni Y-L, Huang B, Zhang J, Feng J-Y, Wang S-B, Dunwell JM, Zhang Y-M, Wu R. Methodological implementation of mixed linear models in multi-locus genome-wide association studies. Brief Bioinform. 2017;19:700–12. 10.1093/bib/bbw145.</cite> [<a href="https://doi.org/10.1093/bib/bbw145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6054291/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28158525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wen%20Y-J,%20Zhang%20H,%20Ni%20Y-L,%20Huang%20B,%20Zhang%20J,%20Feng%20J-Y,%20Wang%20S-B,%20Dunwell%20JM,%20Zhang%20Y-M,%20Wu%20R.%20Methodological%20implementation%20of%20mixed%20linear%20models%20in%20multi-locus%20genome-wide%20association%20studies.%20Brief%20Bioinform.%202017;19:700%E2%80%9312.%2010.1093/bib/bbw145." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Wang S-B, Feng J-Y, Ren W-L, Huang B, Zhou L, Wen Y-J, Zhang J, Dunwell JM, Xu S, Zhang Y-M. Improving power and accuracy of genome-wide association studies via a multi-locus mixed linear model methodology. Sci Rep. 2016. 10.1038/srep19444.
</cite> [<a href="https://doi.org/10.1038/srep19444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4726296/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26787347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20S-B,%20Feng%20J-Y,%20Ren%20W-L,%20Huang%20B,%20Zhou%20L,%20Wen%20Y-J,%20Zhang%20J,%20Dunwell%20JM,%20Xu%20S,%20Zhang%20Y-M.%20Improving%20power%20and%20accuracy%20of%20genome-wide%20association%20studies%20via%20a%20multi-locus%20mixed%20linear%20model%20methodology.%20Sci%20Rep.%202016.%2010.1038/srep19444." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Cui Y, Zhang F, Zhou Y. The application of multi-locus GWAS for the detection of salt-tolerance loci in rice. Front Plant Sci. 2018. 10.3389/fpls.2018.01464.
</cite> [<a href="https://doi.org/10.3389/fpls.2018.01464" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6180169/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30337936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cui%20Y,%20Zhang%20F,%20Zhou%20Y.%20The%20application%20of%20multi-locus%20GWAS%20for%20the%20detection%20of%20salt-tolerance%20loci%20in%20rice.%20Front%20Plant%20Sci.%202018.%2010.3389/fpls.2018.01464." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>-Tamba CL, Ni Y-L, Zhang Y-M. Iterative sure independence screening EM-Bayesian LASSO algorithm for multi-locus genome-wide association studies. PLOS Comput Biology/PLoS Comput Biology. 2017;13:e1005357. 10.1371/journal.pcbi.1005357.</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1005357" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5308866/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28141824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Tamba%20CL,%20Ni%20Y-L,%20Zhang%20Y-M.%20Iterative%20sure%20independence%20screening%20EM-Bayesian%20LASSO%20algorithm%20for%20multi-locus%20genome-wide%20association%20studies.%20PLOS%20Comput%20Biology/PLoS%20Comput%20Biology.%202017;13:e1005357.%2010.1371/journal.pcbi.1005357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>-Zhang J, Feng J, Ni Y, Wen Y, Niu Y, Tamba CL, Yue C, Song Q, Zhang Y. pLARmEB: integration of least angle regression with empirical Bayes for multilocus genome-wide association studies. Heredity. 2017;118:517–24. 10.1038/hdy.2017.8.
</cite> [<a href="https://doi.org/10.1038/hdy.2017.8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5436030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28295030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Zhang%20J,%20Feng%20J,%20Ni%20Y,%20Wen%20Y,%20Niu%20Y,%20Tamba%20CL,%20Yue%20C,%20Song%20Q,%20Zhang%20Y.%20pLARmEB:%20integration%20of%20least%20angle%20regression%20with%20empirical%20Bayes%20for%20multilocus%20genome-wide%20association%20studies.%20Heredity.%202017;118:517%E2%80%9324.%2010.1038/hdy.2017.8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Ren W-L, Wen Y-J, Dunwell JM, Zhang Y-M. pKWmEB: integration of Kruskal–Wallis test with empirical Bayes under polygenic background control for multi-locus genome-wide association study. Heredity. 2017;120:208–18. 10.1038/s41437-017-0007-4.
</cite> [<a href="https://doi.org/10.1038/s41437-017-0007-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5836593/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29234158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ren%20W-L,%20Wen%20Y-J,%20Dunwell%20JM,%20Zhang%20Y-M.%20pKWmEB:%20integration%20of%20Kruskal%E2%80%93Wallis%20test%20with%20empirical%20Bayes%20under%20polygenic%20background%20control%20for%20multi-locus%20genome-wide%20association%20study.%20Heredity.%202017;120:208%E2%80%9318.%2010.1038/s41437-017-0007-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>-Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MAR, Bender D, Maller J, Sklar P, De Bakker PIW, Daly MJ, et al. PLINK: a tool set for Whole-Genome association and Population-Based linkage analyses. Am J Hum Genet. 2007;81:559–75. 10.1086/519795.
</cite> [<a href="https://doi.org/10.1086/519795" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1950838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Purcell%20S,%20Neale%20B,%20Todd-Brown%20K,%20Thomas%20L,%20Ferreira%20MAR,%20Bender%20D,%20Maller%20J,%20Sklar%20P,%20De%20Bakker%20PIW,%20Daly%20MJ,%20et%20al.%20PLINK:%20a%20tool%20set%20for%20Whole-Genome%20association%20and%20Population-Based%20linkage%20analyses.%20Am%20J%20Hum%20Genet.%202007;81:559%E2%80%9375.%2010.1086/519795." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: a tool for genome-wide complex trait analysis. Am J Hum Genet. 2011;88:76–82. 10.1016/j.ajhg.2010.11.011.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2010.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3014363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21167468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20J,%20Lee%20SH,%20Goddard%20ME,%20Visscher%20PM.%20GCTA:%20a%20tool%20for%20genome-wide%20complex%20trait%20analysis.%20Am%20J%20Hum%20Genet.%202011;88:76%E2%80%9382.%2010.1016/j.ajhg.2010.11.011." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>-Zhang Y-W, Tamba CL, Wen Y-J, Li P, Ren W-L, Ni Y-L, Gao J, Zhang Y-M. MrMLM V4.0.2: an R platform for multi-locus genome-wide association studies. Genomics Proteom Bioinformatics/Genomics Proteom Bioinf. 2020;18:481–7. 10.1016/j.gpb.2020.06.006.</cite> [<a href="https://doi.org/10.1016/j.gpb.2020.06.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8242264/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33346083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Zhang%20Y-W,%20Tamba%20CL,%20Wen%20Y-J,%20Li%20P,%20Ren%20W-L,%20Ni%20Y-L,%20Gao%20J,%20Zhang%20Y-M.%20MrMLM%20V4.0.2:%20an%20R%20platform%20for%20multi-locus%20genome-wide%20association%20studies.%20Genomics%20Proteom%20Bioinformatics/Genomics%20Proteom%20Bioinf.%202020;18:481%E2%80%937.%2010.1016/j.gpb.2020.06.006." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Bradbury PJ, Zhang Z, Kroon DE, Casstevens TM, Ramdoss Y, Buckler ES. TASSEL: software for association mapping of complex traits in diverse samples. Bioinformatics. 2007;23:2633–5. 10.1093/bioinformatics/btm308.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btm308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17586829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bradbury%20PJ,%20Zhang%20Z,%20Kroon%20DE,%20Casstevens%20TM,%20Ramdoss%20Y,%20Buckler%20ES.%20TASSEL:%20software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples.%20Bioinformatics.%202007;23:2633%E2%80%935.%2010.1093/bioinformatics/btm308." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>-Nyholt DR. All LODs are not created equal. Am J Hum Genet. 2000;67:282–8. 10.1086/303029.
</cite> [<a href="https://doi.org/10.1086/303029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1287176/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10884360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Nyholt%20DR.%20All%20LODs%20are%20not%20created%20equal.%20Am%20J%20Hum%20Genet.%202000;67:282%E2%80%938.%2010.1086/303029." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Ge SX, Jung D, Yao R. ShinyGO: a graphical gene-set enrichment tool for animals and plants. Bioinformatics. 2019;36:2628–9. 10.1093/bioinformatics/btz931.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7178415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31882993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ge%20SX,%20Jung%20D,%20Yao%20R.%20ShinyGO:%20a%20graphical%20gene-set%20enrichment%20tool%20for%20animals%20and%20plants.%20Bioinformatics.%202019;36:2628%E2%80%939.%2010.1093/bioinformatics/btz931." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Knudsen S. Promoter2.0: for the recognition of polii promoter sequences. Bioinformatics. 1999;15(5):356–61. 10.1093/bioinformatics/15.5.356.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/15.5.356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10366655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Knudsen%20S.%20Promoter2.0:%20for%20the%20recognition%20of%20polii%20promoter%20sequences.%20Bioinformatics.%201999;15(5):356%E2%80%9361.%2010.1093/bioinformatics/15.5.356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>-Scalzitti N, Kress A, Orhand R, Weber T, Moulinier L, Jeannin-Girardon A, Collet P, Poch O, Thompson JD. Spliceator: multi-species splice site prediction using convolutional neural networks. BMC Bioinformatics. 2021;22(1). 10.1186/s12859-021-04471-3.</cite> [<a href="https://doi.org/10.1186/s12859-021-04471-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8609763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34814826/" class="usa-link">PubMed</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Van Raamsdonk CD, Griewank KG, Crosby MB, Garrido MC, Vemula S, Wiesner T, Obenauf AC, Wackernagel W, Green G, Bouvier N, et al. Mutations in GNA11in uveal melanoma. N Engl J Med. 2010;363:2191–9. 10.1056/nejmoa1000584.
</cite> [<a href="https://doi.org/10.1056/NEJMoa1000584" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3107972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21083380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20Raamsdonk%20CD,%20Griewank%20KG,%20Crosby%20MB,%20Garrido%20MC,%20Vemula%20S,%20Wiesner%20T,%20Obenauf%20AC,%20Wackernagel%20W,%20Green%20G,%20Bouvier%20N,%20et%20al.%20Mutations%20in%20GNA11in%20uveal%20melanoma.%20N%20Engl%20J%20Med.%202010;363:2191%E2%80%939.%2010.1056/nejmoa1000584." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>-Onken MD, Worley LA, Long MD, Duan S, Council ML, Bowcock AM, Harbour JW. Oncogenic mutations InGNAQOccur early in uveal melanoma. Investig Ophthalmol Vis Sci. 2008;49:5230. 10.1167/iovs.08-2145.
</cite> [<a href="https://doi.org/10.1167/iovs.08-2145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2634606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18719078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Onken%20MD,%20Worley%20LA,%20Long%20MD,%20Duan%20S,%20Council%20ML,%20Bowcock%20AM,%20Harbour%20JW.%20Oncogenic%20mutations%20InGNAQOccur%20early%20in%20uveal%20melanoma.%20Investig%20Ophthalmol%20Vis%20Sci.%202008;49:5230.%2010.1167/iovs.08-2145." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Pópulo H, Tavares S, Faustino A, Nunes JB, Lopes JM, Soares P. GNAQ and BRAF mutations show differential activation of the mTOR pathway in human transformed cells. PeerJ. 2013;1: e104. 10.7717/peerj.104.
</cite> [<a href="https://doi.org/10.7717/peerj.104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3728761/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23904987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?P%C3%B3pulo%20H,%20Tavares%20S,%20Faustino%20A,%20Nunes%20JB,%20Lopes%20JM,%20Soares%20P.%20GNAQ%20and%20BRAF%20mutations%20show%20differential%20activation%20of%20the%20mTOR%20pathway%20in%20human%20transformed%20cells.%20PeerJ.%202013;1:%20e104.%2010.7717/peerj.104." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>-Kennedy BK, Lamming DW. The mechanistic target of rapamycin: the grand conductor of metabolism and aging. Cell Metabol. 2016;23:990–1003. 10.1016/j.cmet.2016.05.009.</cite> [<a href="https://doi.org/10.1016/j.cmet.2016.05.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4910876/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27304501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Kennedy%20BK,%20Lamming%20DW.%20The%20mechanistic%20target%20of%20rapamycin:%20the%20grand%20conductor%20of%20metabolism%20and%20aging.%20Cell%20Metabol.%202016;23:990%E2%80%931003.%2010.1016/j.cmet.2016.05.009." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>-Tonelli F, Leoni L, Daponte V, Gioia R, Cotti S, Fiedler IaK, Larianova D, Willaert A, Coucke PJ, Villani S, et al. Zebrafish Tric-b is required for skeletal development and bone cells differentiation. Front Endocrinol. 2023;14. 10.3389/fendo.2023.1002914.</cite> [<a href="https://doi.org/10.3389/fendo.2023.1002914" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9899828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36755921/" class="usa-link">PubMed</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>-Cousminer DL, Stergiakouli E, Berry DJ, Ang W, Groen-Blokhuis MM, Körner A, Siitonen N, Ntalla I, Marinelli M, Perry JRB, et al. Genome-wide association study of sexual maturation in males and females highlights a role for body mass and menarche loci in male puberty. Hum Mol Genet Online/Human Mol Genet. 2014;23:4452–64. 10.1093/hmg/ddu150.</cite> [<a href="https://doi.org/10.1093/hmg/ddu150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4168307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24770850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Cousminer%20DL,%20Stergiakouli%20E,%20Berry%20DJ,%20Ang%20W,%20Groen-Blokhuis%20MM,%20K%C3%B6rner%20A,%20Siitonen%20N,%20Ntalla%20I,%20Marinelli%20M,%20Perry%20JRB,%20et%20al.%20Genome-wide%20association%20study%20of%20sexual%20maturation%20in%20males%20and%20females%20highlights%20a%20role%20for%20body%20mass%20and%20menarche%20loci%20in%20male%20puberty.%20Hum%20Mol%20Genet%20Online/Human%20Mol%20Genet.%202014;23:4452%E2%80%9364.%2010.1093/hmg/ddu150." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>-Lv F, Xu X-J, Wang J-Y, Liu Y, Asan N, Wang J-W, Song L-J, Song Y-W, Jiang Y, Wang O, et al. Two novel mutations in TMEM38B result in rare autosomal recessive osteogenesis imperfecta. J Hum Genet. 2016;61:539–45. 10.1038/jhg.2016.11.
</cite> [<a href="https://doi.org/10.1038/jhg.2016.11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26911354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Lv%20F,%20Xu%20X-J,%20Wang%20J-Y,%20Liu%20Y,%20Asan%20N,%20Wang%20J-W,%20Song%20L-J,%20Song%20Y-W,%20Jiang%20Y,%20Wang%20O,%20et%20al.%20Two%20novel%20mutations%20in%20TMEM38B%20result%20in%20rare%20autosomal%20recessive%20osteogenesis%20imperfecta.%20J%20Hum%20Genet.%202016;61:539%E2%80%9345.%2010.1038/jhg.2016.11." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>-Fink EE, Moparthy S, Bagati A, Bianchi-Smiraglia A, Lipchick BC, Wolff DW, Roll MV, Wang J, Liu S, Bakin AV, et al. XBP1-KLF9 axis acts as a molecular rheostat to control the transition from adaptive to cytotoxic unfolded protein response. Cell Rep. 2018;25:212–e2234. 10.1016/j.celrep.2018.09.013.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2018.09.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6251307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30282030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Fink%20EE,%20Moparthy%20S,%20Bagati%20A,%20Bianchi-Smiraglia%20A,%20Lipchick%20BC,%20Wolff%20DW,%20Roll%20MV,%20Wang%20J,%20Liu%20S,%20Bakin%20AV,%20et%20al.%20XBP1-KLF9%20axis%20acts%20as%20a%20molecular%20rheostat%20to%20control%20the%20transition%20from%20adaptive%20to%20cytotoxic%20unfolded%20protein%20response.%20Cell%20Rep.%202018;25:212%E2%80%93e2234.%2010.1016/j.celrep.2018.09.013." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Lin M, Zhang Y, Li A, Tang E, Peng J, Tang W, Zhang Y, Lu L, Xiao Y, Wei Q, et al. High-throughput RNAi screening of human kinases identifies predictors of clinical outcome in colorectal cancer patients treated with oxaliplatin. Oncotarget. 2015;6:16774–85. 10.18632/oncotarget.3736.
</cite> [<a href="https://doi.org/10.18632/oncotarget.3736" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4599307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25904054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20M,%20Zhang%20Y,%20Li%20A,%20Tang%20E,%20Peng%20J,%20Tang%20W,%20Zhang%20Y,%20Lu%20L,%20Xiao%20Y,%20Wei%20Q,%20et%20al.%20High-throughput%20RNAi%20screening%20of%20human%20kinases%20identifies%20predictors%20of%20clinical%20outcome%20in%20colorectal%20cancer%20patients%20treated%20with%20oxaliplatin.%20Oncotarget.%202015;6:16774%E2%80%9385.%2010.18632/oncotarget.3736." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Meckes JK, Lim PH, Wert SL, Luo W, Gacek SA, Platt D, Jankord R, Saar K, Redei EE. Brain region-specific expression of genes mapped within quantitative trait loci for behavioral responsiveness to acute stress in Fisher 344 and Wistar Kyoto male rats. PLoS One. 2018;13: e0194293. 10.1371/journal.pone.0194293.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0194293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5847310/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29529077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meckes%20JK,%20Lim%20PH,%20Wert%20SL,%20Luo%20W,%20Gacek%20SA,%20Platt%20D,%20Jankord%20R,%20Saar%20K,%20Redei%20EE.%20Brain%20region-specific%20expression%20of%20genes%20mapped%20within%20quantitative%20trait%20loci%20for%20behavioral%20responsiveness%20to%20acute%20stress%20in%20Fisher%20344%20and%20Wistar%20Kyoto%20male%20rats.%20PLoS%20One.%202018;13:%20e0194293.%2010.1371/journal.pone.0194293." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>-Urbaniak A, Jablonska K, Podhorska-Okolow M, Ugorski M, Dziegiel P. Prolactin-induced protein (PIP)-characterization and role in breast cancer progression. <em>PubMed Central (PMC)</em> (2018) Available at: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6291655/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6291655/</a></cite> [<a href="/articles/PMC6291655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30555735/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Baniwal SK, Chimge NO, Jordan VC, Tripathy D, Frenkel B. Prolactin-induced protein (PIP) regulates proliferation of luminal A type breast cancer cells in an estrogen-independent manner. PLoS One. 2013;8: e62361. 10.1371/journal.pone.0062361.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0062361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3670933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23755096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baniwal%20SK,%20Chimge%20NO,%20Jordan%20VC,%20Tripathy%20D,%20Frenkel%20B.%20Prolactin-induced%20protein%20(PIP)%20regulates%20proliferation%20of%20luminal%20A%20type%20breast%20cancer%20cells%20in%20an%20estrogen-independent%20manner.%20PLoS%20One.%202013;8:%20e62361.%2010.1371/journal.pone.0062361." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Huang W, Guo L, Zhao M, Zhang D, Xu H, Nie Q. The inhibition on MDFIC and PI3K/AKT pathway caused by MIR-146B-3P triggers suppression of myoblast proliferation and differentiation and promotion of apoptosis. Cells. 2019;8: 656. 10.3390/cells8070656.
</cite> [<a href="https://doi.org/10.3390/cells8070656" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6678156/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31261950/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20W,%20Guo%20L,%20Zhao%20M,%20Zhang%20D,%20Xu%20H,%20Nie%20Q.%20The%20inhibition%20on%20MDFIC%20and%20PI3K/AKT%20pathway%20caused%20by%20MIR-146B-3P%20triggers%20suppression%20of%20myoblast%20proliferation%20and%20differentiation%20and%20promotion%20of%20apoptosis.%20Cells.%202019;8:%20656.%2010.3390/cells8070656." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Zhang L, Zhou X, Michal JJ, Ding B, Li R, Jiang Z. Genome wide screening of candidate genes for improving piglet birth weight using high and low estimated breeding value populations. Int J Biol Sci. 2014;10:236–44. 10.7150/ijbs.7744.
</cite> [<a href="https://doi.org/10.7150/ijbs.7744" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3957079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24644423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20L,%20Zhou%20X,%20Michal%20JJ,%20Ding%20B,%20Li%20R,%20Jiang%20Z.%20Genome%20wide%20screening%20of%20candidate%20genes%20for%20improving%20piglet%20birth%20weight%20using%20high%20and%20low%20estimated%20breeding%20value%20populations.%20Int%20J%20Biol%20Sci.%202014;10:236%E2%80%9344.%2010.7150/ijbs.7744." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Okita K, Hara Y, Okura H, Hayashi H, Sasaki Y, Masuko S, Kitadai E, Masuko K, Yoshimoto S, Hayashi N, et al. Antitumor effects of novel mAbs against cationic amino acid transporter 1 (CAT1) on human CRC with amplified CAT1 gene. Cancer Sci. 2020;112:563–74. 10.1111/cas.14741.
</cite> [<a href="https://doi.org/10.1111/cas.14741" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7894011/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33211385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Okita%20K,%20Hara%20Y,%20Okura%20H,%20Hayashi%20H,%20Sasaki%20Y,%20Masuko%20S,%20Kitadai%20E,%20Masuko%20K,%20Yoshimoto%20S,%20Hayashi%20N,%20et%20al.%20Antitumor%20effects%20of%20novel%20mAbs%20against%20cationic%20amino%20acid%20transporter%201%20(CAT1)%20on%20human%20CRC%20with%20amplified%20CAT1%20gene.%20Cancer%20Sci.%202020;112:563%E2%80%9374.%2010.1111/cas.14741." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Hatzoglou M, Fernandez J, Yaman I, Closs E. Regulation of cationic amino acid, transport. The story of the CAT-1 transporter. Annu Rev Nutr. 2004;24:377–99. 10.1146/annurev.nutr.23.011702.073120.
</cite> [<a href="https://doi.org/10.1146/annurev.nutr.23.011702.073120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15459982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hatzoglou%20M,%20Fernandez%20J,%20Yaman%20I,%20Closs%20E.%20Regulation%20of%20cationic%20amino%20acid,%20transport.%20The%20story%20of%20the%20CAT-1%20transporter.%20Annu%20Rev%20Nutr.%202004;24:377%E2%80%9399.%2010.1146/annurev.nutr.23.011702.073120." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Sato H, Bai L, Borjigin L, Aida Y. Overexpression of bovine leukemia virus receptor SLC7A1/CAT1 enhances cellular susceptibility to BLV infection on luminescence syncytium induction assay (LuSIA). Virol J. 2020. 10.1186/s12985-020-01324-y.
</cite> [<a href="https://doi.org/10.1186/s12985-020-01324-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7178733/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32321527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sato%20H,%20Bai%20L,%20Borjigin%20L,%20Aida%20Y.%20Overexpression%20of%20bovine%20leukemia%20virus%20receptor%20SLC7A1/CAT1%20enhances%20cellular%20susceptibility%20to%20BLV%20infection%20on%20luminescence%20syncytium%20induction%20assay%20(LuSIA).%20Virol%20J.%202020.%2010.1186/s12985-020-01324-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>-Juliana CA, Yang J, Cannon CE, Good AL, Haemmerle MW, Stoffers DA. A PDX1-ATF transcriptional complex governs β cell survival during stress. Mol Metabolism. 2018;17:39–48. 10.1016/j.molmet.2018.07.007.</cite> [<a href="https://doi.org/10.1016/j.molmet.2018.07.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6197747/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30174228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Juliana%20CA,%20Yang%20J,%20Cannon%20CE,%20Good%20AL,%20Haemmerle%20MW,%20Stoffers%20DA.%20A%20PDX1-ATF%20transcriptional%20complex%20governs%20%CE%B2%20cell%20survival%20during%20stress.%20Mol%20Metabolism.%202018;17:39%E2%80%9348.%2010.1016/j.molmet.2018.07.007." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Porter EG, Dhiman A, Chowdhury B, Carter BC, Lin H, Stewart JC, Kazemian M, Wendt MK, Dykhuizen EC. PBRM1 regulates stress response in epithelial cells. iScience. 2019;15:196–210. 10.1016/j.isci.2019.04.027.
</cite> [<a href="https://doi.org/10.1016/j.isci.2019.04.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6514269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31077944/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Porter%20EG,%20Dhiman%20A,%20Chowdhury%20B,%20Carter%20BC,%20Lin%20H,%20Stewart%20JC,%20Kazemian%20M,%20Wendt%20MK,%20Dykhuizen%20EC.%20PBRM1%20regulates%20stress%20response%20in%20epithelial%20cells.%20iScience.%202019;15:196%E2%80%93210.%2010.1016/j.isci.2019.04.027." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Chowdhury B, Porter EG, Stewart JC, Ferreira CR, Schipma MJ, Dykhuizen EC. PBRM1 regulates the expression of genes involved in metabolism and cell adhesion in renal clear cell carcinoma. PLoS One. 2016;11: e0153718. 10.1371/journal.pone.0153718.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0153718" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4839679/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27100670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chowdhury%20B,%20Porter%20EG,%20Stewart%20JC,%20Ferreira%20CR,%20Schipma%20MJ,%20Dykhuizen%20EC.%20PBRM1%20regulates%20the%20expression%20of%20genes%20involved%20in%20metabolism%20and%20cell%20adhesion%20in%20renal%20clear%20cell%20carcinoma.%20PLoS%20One.%202016;11:%20e0153718.%2010.1371/journal.pone.0153718." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Chin ER, Olson EN, Richardson JA, Yang Q, Humphries C, Shelton JM, Wu H, Zhu W, Bassel-Duby R, Williams RS. A calcineurin-dependent transcriptional pathway controls skeletal muscle fiber type. Genes Dev. 1998;12:2499–509. 10.1101/gad.12.16.2499.
</cite> [<a href="https://doi.org/10.1101/gad.12.16.2499" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC317085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9716403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chin%20ER,%20Olson%20EN,%20Richardson%20JA,%20Yang%20Q,%20Humphries%20C,%20Shelton%20JM,%20Wu%20H,%20Zhu%20W,%20Bassel-Duby%20R,%20Williams%20RS.%20A%20calcineurin-dependent%20transcriptional%20pathway%20controls%20skeletal%20muscle%20fiber%20type.%20Genes%20Dev.%201998;12:2499%E2%80%93509.%2010.1101/gad.12.16.2499." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Asagiri M, Sato K, Usami T, Ochi S, Nishina H, Yoshida H, Morita I, Wagner EF, Mak TW, Serfling E, et al. Autoamplification of NFATc1 expression determines its essential role in bone homeostasis. J Exp Med. 2005;202:1261–9. 10.1084/jem.20051150.
</cite> [<a href="https://doi.org/10.1084/jem.20051150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2213228/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16275763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Asagiri%20M,%20Sato%20K,%20Usami%20T,%20Ochi%20S,%20Nishina%20H,%20Yoshida%20H,%20Morita%20I,%20Wagner%20EF,%20Mak%20TW,%20Serfling%20E,%20et%20al.%20Autoamplification%20of%20NFATc1%20expression%20determines%20its%20essential%20role%20in%20bone%20homeostasis.%20J%20Exp%20Med.%202005;202:1261%E2%80%939.%2010.1084/jem.20051150." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Liu Y, Shen T, Randall WR, Schneider MF. Signaling pathways in activity-dependent fiber type plasticity in adult skeletal muscle. J Muscle Res Cell Motil. 2005;26:13–21. 10.1007/s10974-005-9002-0.
</cite> [<a href="https://doi.org/10.1007/s10974-005-9002-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16096682/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20Y,%20Shen%20T,%20Randall%20WR,%20Schneider%20MF.%20Signaling%20pathways%20in%20activity-dependent%20fiber%20type%20plasticity%20in%20adult%20skeletal%20muscle.%20J%20Muscle%20Res%20Cell%20Motil.%202005;26:13%E2%80%9321.%2010.1007/s10974-005-9002-0." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Horsley V, Friday BB, Matteson S, Kegley KM, Gephart J, Pavlath GK. Regulation of the growth of multinucleated muscle cells by an NFATC2-dependent pathway. J Cell Biol. 2001;153:329–38. 10.1083/jcb.153.2.329.
</cite> [<a href="https://doi.org/10.1083/jcb.153.2.329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2169453/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11309414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Horsley%20V,%20Friday%20BB,%20Matteson%20S,%20Kegley%20KM,%20Gephart%20J,%20Pavlath%20GK.%20Regulation%20of%20the%20growth%20of%20multinucleated%20muscle%20cells%20by%20an%20NFATC2-dependent%20pathway.%20J%20Cell%20Biol.%202001;153:329%E2%80%9338.%2010.1083/jcb.153.2.329." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Chung CYT, Lo PHY, Lee KKH. BABAM2 regulates cell cycle progression and pluripotency in mouse embryonic stem cells as revealed by induced DNA damage. Biomedicines. 2020;8: 397. 10.3390/biomedicines8100397.
</cite> [<a href="https://doi.org/10.3390/biomedicines8100397" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7600899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33050379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chung%20CYT,%20Lo%20PHY,%20Lee%20KKH.%20BABAM2%20regulates%20cell%20cycle%20progression%20and%20pluripotency%20in%20mouse%20embryonic%20stem%20cells%20as%20revealed%20by%20induced%20DNA%20damage.%20Biomedicines.%202020;8:%20397.%2010.3390/biomedicines8100397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>-Yun M, Yingzi L, Jie G, Guanxin L, Zimei Z, Zhen C, Zhi L, Yingjie N, Lunquan S, Tao C, et al. PPDPF promotes the progression and acts as an antiapoptotic protein in Non-Small cell lung cancer. Int J Biol Sci. 2022;18:214–28. 10.7150/ijbs.65654.
</cite> [<a href="https://doi.org/10.7150/ijbs.65654" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8692159/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34975328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Yun%20M,%20Yingzi%20L,%20Jie%20G,%20Guanxin%20L,%20Zimei%20Z,%20Zhen%20C,%20Zhi%20L,%20Yingjie%20N,%20Lunquan%20S,%20Tao%20C,%20et%20al.%20PPDPF%20promotes%20the%20progression%20and%20acts%20as%20an%20antiapoptotic%20protein%20in%20Non-Small%20cell%20lung%20cancer.%20Int%20J%20Biol%20Sci.%202022;18:214%E2%80%9328.%2010.7150/ijbs.65654." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>-Ostrowska M, Zwierzchowski L, Brzozowska P, Kawecka-Grochocka E, Żelazowska B, Bagnicka E. The effect of single-nucleotide polymorphism in the promoter region of bovinealpha-lactalbumin(LALBA) gene onlalbaexpression in milk cells and milk traits of cows. J Anim Science/Journal Anim Sci ASAS Ref Compendium. 2021;99. 10.1093/jas/skab169.</cite> [<a href="https://doi.org/10.1093/jas/skab169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8281099/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34032850/" class="usa-link">PubMed</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Raab M, Wang H, Lu Y, Smith X, Wu Z, Strebhardt K, Ladbury JE, Rudd CE. T cell receptor inside-out pathway via signaling module SKAP1-RAPL regulates T cell motility and interactions in lymph nodes. Immunity. 2010;32:541–56. 10.1016/j.immuni.2010.03.007.
</cite> [<a href="https://doi.org/10.1016/j.immuni.2010.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3812847/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20346707/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raab%20M,%20Wang%20H,%20Lu%20Y,%20Smith%20X,%20Wu%20Z,%20Strebhardt%20K,%20Ladbury%20JE,%20Rudd%20CE.%20T%20cell%20receptor%20inside-out%20pathway%20via%20signaling%20module%20SKAP1-RAPL%20regulates%20T%20cell%20motility%20and%20interactions%20in%20lymph%20nodes.%20Immunity.%202010;32:541%E2%80%9356.%2010.1016/j.immuni.2010.03.007." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Shin JJ, Kim B, Kang J, Choi J, Moon BJ, Ryu DS, Yoon SH, Chin DK, Lee J-K, Kim KN, et al. Clinical, radiographic, and genetic analyses in a population-based cohort of adult spinal deformity in the older population. Neurospine. 2021;18:608–17. 10.14245/ns.2142544.272.
</cite> [<a href="https://doi.org/10.14245/ns.2142544.272" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8497247/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34610692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shin%20JJ,%20Kim%20B,%20Kang%20J,%20Choi%20J,%20Moon%20BJ,%20Ryu%20DS,%20Yoon%20SH,%20Chin%20DK,%20Lee%20J-K,%20Kim%20KN,%20et%20al.%20Clinical,%20radiographic,%20and%20genetic%20analyses%20in%20a%20population-based%20cohort%20of%20adult%20spinal%20deformity%20in%20the%20older%20population.%20Neurospine.%202021;18:608%E2%80%9317.%2010.14245/ns.2142544.272." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>-Liu S, Zheng H, Jiang H, He J, Yu Y, Qu Y, Yue L, Zhang Y, Li Y. Identification and characterization of a novel gene, c1orf109, encoding a CK2 substrate that is involved in cancer cell proliferation. J Biomed Sci. 2012;19(1):49. 10.1186/1423-0127-19-49.
</cite> [<a href="https://doi.org/10.1186/1423-0127-19-49" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3546425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22548824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Liu%20S,%20Zheng%20H,%20Jiang%20H,%20He%20J,%20Yu%20Y,%20Qu%20Y,%20Yue%20L,%20Zhang%20Y,%20Li%20Y.%20Identification%20and%20characterization%20of%20a%20novel%20gene,%20c1orf109,%20encoding%20a%20CK2%20substrate%20that%20is%20involved%20in%20cancer%20cell%20proliferation.%20J%20Biomed%20Sci.%202012;19(1):49.%2010.1186/1423-0127-19-49." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Mata-Rocha M, Alvarado-Cuevas E, Hernández-Sánchez J, Cerecedo D, Felix R, Hernández-Reyes A, Tesoro-Cruz E, Oviedo N. Molecular cloning and analysis of the Catsper1 gene promoter. Mol Hum Reprod. 2013;19(5):336–47. 10.1093/molehr/gat003.
</cite> [<a href="https://doi.org/10.1093/molehr/gat003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23313885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mata-Rocha%20M,%20Alvarado-Cuevas%20E,%20Hern%C3%A1ndez-S%C3%A1nchez%20J,%20Cerecedo%20D,%20Felix%20R,%20Hern%C3%A1ndez-Reyes%20A,%20Tesoro-Cruz%20E,%20Oviedo%20N.%20Molecular%20cloning%20and%20analysis%20of%20the%20Catsper1%20gene%20promoter.%20Mol%20Hum%20Reprod.%202013;19(5):336%E2%80%9347.%2010.1093/molehr/gat003." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Bao G, Ashraf U, Li L, Qiao J, Wang C, Zheng Y. Transcription factor OSBZIP60-like regulating OSP5CS1 gene and 2-Acetyl-1-pyrroline (2-AP) biosynthesis in aromatic rice. Plants. 2023;13(1): 49. 10.3390/plants13010049.
</cite> [<a href="https://doi.org/10.3390/plants13010049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10780308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38202357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bao%20G,%20Ashraf%20U,%20Li%20L,%20Qiao%20J,%20Wang%20C,%20Zheng%20Y.%20Transcription%20factor%20OSBZIP60-like%20regulating%20OSP5CS1%20gene%20and%202-Acetyl-1-pyrroline%20(2-AP)%20biosynthesis%20in%20aromatic%20rice.%20Plants.%202023;13(1):%2049.%2010.3390/plants13010049." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>-Kallabi F, Rhouma BB, Baklouti S, Ghorbel R, Felhi R, Keskes L, Kamoun H. Splicing defects in the AAAS gene leading to both exon skipping and partial intron retention in a Tunisian patient with Allgrove syndrome. Hormone Res Paediatrics. 2016;86(2):90–3. 10.1159/000446539.</cite> [<a href="https://doi.org/10.1159/000446539" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27414811/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?-Kallabi%20F,%20Rhouma%20BB,%20Baklouti%20S,%20Ghorbel%20R,%20Felhi%20R,%20Keskes%20L,%20Kamoun%20H.%20Splicing%20defects%20in%20the%20AAAS%20gene%20leading%20to%20both%20exon%20skipping%20and%20partial%20intron%20retention%20in%20a%20Tunisian%20patient%20with%20Allgrove%20syndrome.%20Hormone%20Res%20Paediatrics.%202016;86(2):90%E2%80%933.%2010.1159/000446539." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Li M. Calculating the most likely intron splicing orders in S. pombe, fruit fly, <em>Arabidopsis thaliana</em>, and humans. BMC Bioinformatics. 2020. 10.1186/s12859-020-03818-6.
</cite> [<a href="https://doi.org/10.1186/s12859-020-03818-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7585206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33099301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20M.%20Calculating%20the%20most%20likely%20intron%20splicing%20orders%20in%20S.%20pombe,%20fruit%20fly,%20Arabidopsis%20thaliana,%20and%20humans.%20BMC%20Bioinformatics.%202020.%2010.1186/s12859-020-03818-6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12333283/bin/12917_2025_4957_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (33.8KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated and/or analysed during the current study are available in the https://figshare.com/repository, https://figshare.com/articles/dataset/Karacabey_merino_50K_genotype_data/29184098?file=54931109.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Veterinary Research are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12917-025-04957-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12917_2025_Article_4957.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12333283/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12333283/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12333283%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333283/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12333283/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12333283/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40775720/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12333283/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40775720/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12333283/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12333283/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="JOmcaGJbK9OhHv3oeoRay9f6zYcBT3djtqzdZ7CngflDtg57nqpgGo6Wl2gaZHv2">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
