
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            The transmission ability in a population of elite tetraploid potatoes - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="FF207D1889B6C8F31A7D180043BA142A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12207579/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="The transmission ability in a population of elite tetraploid potatoes">
<meta name="citation_author" content="Trine Aalborg">
<meta name="citation_author_institution" content="Department of Chemistry and Bioscience, Aalborg University, Aalborg, Denmark">
<meta name="citation_author" content="Hélène Romé">
<meta name="citation_author_institution" content="Center for Quantitative Genetics and Genomics, Aarhus University, Aarhus, Denmark">
<meta name="citation_author" content="Christina Ranzau">
<meta name="citation_author_institution" content="Danespo A/S, Give, Denmark">
<meta name="citation_author" content="Merethe Bagge">
<meta name="citation_author_institution" content="Danespo A/S, Give, Denmark">
<meta name="citation_author" content="Just Jensen">
<meta name="citation_author_institution" content="Center for Quantitative Genetics and Genomics, Aarhus University, Aarhus, Denmark">
<meta name="citation_author" content="Kåre Lehmann Nielsen">
<meta name="citation_author_institution" content="Department of Chemistry and Bioscience, Aalborg University, Aalborg, Denmark">
<meta name="citation_author_institution" content="KMC Amba, Denmark, Brande, Denmark">
<meta name="citation_publication_date" content="2025 Jun 30">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70066">
<meta name="citation_doi" content="10.1002/tpg2.70066">
<meta name="citation_pmid" content="40583826">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12207579/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12207579/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12207579/pdf/TPG2-18-e70066.pdf">
<meta name="description" content="Crop potato, Solanum tuberosum L., has great genetic improvement potential, but vegetative propagation, tetrasomic inheritance, and inbreeding depression have limited genetic gain in the elite gene pool. To elucidate the transmission ability of ...">
<meta name="og:title" content="The transmission ability in a population of elite tetraploid potatoes">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Crop potato, Solanum tuberosum L., has great genetic improvement potential, but vegetative propagation, tetrasomic inheritance, and inbreeding depression have limited genetic gain in the elite gene pool. To elucidate the transmission ability of ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12207579/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12207579">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70066"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70066.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12207579%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12207579/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12207579/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12207579/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70066" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Jun 30;18(3):e70066. doi: <a href="https://doi.org/10.1002/tpg2.70066" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70066</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>The transmission ability in a population of elite tetraploid potatoes</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aalborg%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Trine Aalborg</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Trine Aalborg</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry and Bioscience, Aalborg University, Aalborg, Denmark</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aalborg%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Trine Aalborg</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rom%C3%A9%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Hélène Romé</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Hélène Romé</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Center for Quantitative Genetics and Genomics, Aarhus University, Aarhus, Denmark</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rom%C3%A9%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hélène Romé</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ranzau%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Christina Ranzau</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Christina Ranzau</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Danespo A/S, Give, Denmark</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ranzau%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christina Ranzau</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bagge%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Merethe Bagge</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Merethe Bagge</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Danespo A/S, Give, Denmark</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bagge%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Merethe Bagge</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jensen%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Just Jensen</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Just Jensen</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Center for Quantitative Genetics and Genomics, Aarhus University, Aarhus, Denmark</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jensen%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Just Jensen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nielsen%20KL%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Kåre Lehmann Nielsen</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Kåre Lehmann Nielsen</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry and Bioscience, Aalborg University, Aalborg, Denmark</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>KMC Amba, Denmark, Brande, Denmark</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nielsen%20KL%22%5BAuthor%5D" class="usa-link"><span class="name western">Kåre Lehmann Nielsen</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270066-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Chemistry and Bioscience, Aalborg University, Aalborg, Denmark</div>
<div id="tpg270066-aff-0002">
<sup>
<sup>2</sup>
</sup>Center for Quantitative Genetics and Genomics, Aarhus University, Aarhus, Denmark</div>
<div id="tpg270066-aff-0003">
<sup>
<sup>3</sup>
</sup>Danespo A/S, Give, Denmark</div>
<div id="tpg270066-aff-0004">
<sup>
<sup>4</sup>
</sup>KMC Amba, Denmark, Brande, Denmark</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Trine Aalborg, Department of Chemistry and Bioscience, Aalborg University, Aalborg, Denmark. Email: <span>traa@bio.aau.dk</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 May 19; Received 2025 Apr 10; Accepted 2025 May 20; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12207579  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40583826/" class="usa-link">40583826</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Crop potato, <em>Solanum tuberosum</em> L., has great genetic improvement potential, but vegetative propagation, tetrasomic inheritance, and inbreeding depression have limited genetic gain in the elite gene pool. To elucidate the transmission ability of breeding material in the current elite gene pool, we estimated the additive genetic variance and specific combining abilities among 18 parents for 10 key agronomic traits and evaluated the impact of genotypic information in capturing trait inheritance patterns using genomic prediction. Analysis of 5013 clones from an F<sub>1</sub> population of an 18‐parent incomplete diallel cross showed that within the population, some traits had greater additive genetic effects, while others had greater nonadditive genetic contributions. There were clear differences in the breeding values between the evaluated parents. The effect of cytoplasmic genome type on phenotype was assessed and compared to its impact on fertility. These statistics were used to discuss the parental potential of the diallel parents. Including genotyping‐by‐sequencing information (available for 15% of the population) in a single‐step approach improved the prediction accuracy of genetic breeding values, and, surprisingly, the greatest effect was observed for the genomic prediction of high‐heritability traits. Increased prediction accuracy across multiple traits is crucial to support the breeder's decisions when selecting parents. The phenotypic and genetic correlations between traits were determined using pairwise bivariate models. Significant genetic correlations between several traits were identified, indicating that single‐trait selection in breeding programs will result in the indirect selection of correlated traits—possibly with opposing additive contributions to different breeding goal traits by the same progenitor.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270066-list-10001" class="list" style="list-style-type:disc">
<li><p>Cytoplasmic genome type can impact clone fertility and some trait phenotypes and should influence parent selection.</p></li>
<li><p>All traits had significant additive genetic contributions, indicating potential for genetic gain through breeding.</p></li>
<li><p>It was trait dependent whether additive versus nonadditive genetic effects were more important.</p></li>
<li><p>Prediction accuracy of parent estimated breeding values improves with the incorporation of genomic and pedigree information in ssGBLUP.</p></li>
<li><p>Significant pairwise trait genetic correlations indicate extensive indirect trait selection during breeding.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270066-dl-0001">
<dt>BLUP</dt>
<dd><p>best linear unbiased prediction</p></dd>
<dt>EBV</dt>
<dd><p>estimated breeding value</p></dd>
<dt>GCA</dt>
<dd><p>general combining ability</p></dd>
<dt>GBLUP</dt>
<dd><p>genomic BLUP</p></dd>
<dt>GBS</dt>
<dd><p>genotyping‐by‐sequencing</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association study</p></dd>
<dt>PA</dt>
<dd><p>prediction accuracy</p></dd>
<dt>PBLUP</dt>
<dd><p>pedigree BLUP</p></dd>
<dt>PEV</dt>
<dd><p>prediction error variance</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus</p></dd>
<dt>SCA</dt>
<dd><p>specific combining ability</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
<dt>ssGBLUP</dt>
<dd><p>single‐step GBLUP</p></dd>
</dl>
<section id="tpg270066-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Global food security is facing immediate challenges due to climate change, environmental concerns, increased urbanization, and the need for biodiversity conservation areas, leading to a current exhaustion and expected future loss of arable farmlands (Lenaerts et al., <a href="#tpg270066-bib-0034" class="usa-link" aria-describedby="tpg270066-bib-0034">2019</a>; W. Wu, Yu, et al., <a href="#tpg270066-bib-0078" class="usa-link" aria-describedby="tpg270066-bib-0078">2018</a>). Concurrently, an increase in global food demand of ∼56% from 2010 to 2050 is predicted (Van Dijk et al., <a href="#tpg270066-bib-0071" class="usa-link" aria-describedby="tpg270066-bib-0071">2021</a>) and this necessitates the advancement of agricultural systems to generate increased yields of high‐quality crops while decreasing resource consumption (Devaux et al., <a href="#tpg270066-bib-0015" class="usa-link" aria-describedby="tpg270066-bib-0015">2019</a>). One strategy to attain this is crop improvement by accelerating the breeding of superior crop cultivars (Lenaerts et al., <a href="#tpg270066-bib-0034" class="usa-link" aria-describedby="tpg270066-bib-0034">2019</a>). The potato crop, <em>Solanum tuberosum</em> L., is an attractive candidate for breeding optimization strategies since it has a high nutritional value, excellent yield, and global cultivation as a staple food (Devaux et al., <a href="#tpg270066-bib-0016" class="usa-link" aria-describedby="tpg270066-bib-0016">2014</a>). It is currently the world's fourth most important food crop and was in 2022 globally cultivated on 17.8 million ha of cropland, yielding a total production of ∼375 Mt (FAOSTAT, <a href="#tpg270066-bib-0019" class="usa-link" aria-describedby="tpg270066-bib-0019">2024</a>).</p>
<p>Furthermore, the potato is likely to harbor great potential for genetic improvement. The domestic tetraploid potato cultivars, delineated to Andean origin from landrace progenitors (Gutaker et al., <a href="#tpg270066-bib-0025" class="usa-link" aria-describedby="tpg270066-bib-0025">2019</a>; Spooner et al., <a href="#tpg270066-bib-0063" class="usa-link" aria-describedby="tpg270066-bib-0063">2005</a>), constitute a highly heterozygous gene pool of breeding material (Gebhardt, <a href="#tpg270066-bib-0021" class="usa-link" aria-describedby="tpg270066-bib-0021">2016</a>). The overall aim of potato breeding programs is the development of new cultivars for marketing and breeding clones for introduction as new parental lines by crossing breeding material to make seeds with new allele combinations superior to those in either progenitor (Hickey et al., <a href="#tpg270066-bib-0028" class="usa-link" aria-describedby="tpg270066-bib-0028">2017</a>). However, potatoes are vegetatively propagated, which causes varieties to exist indefinitely, and the breeding history of domesticated tetraploid elite potato has many recurrent parents at different levels in the pedigree. Furthermore, there have been relatively few sexual generations during the period of modern breeding starting around 1850s, following its post‐Colombian introduction into Europe around the 1600s (Brown, <a href="#tpg270066-bib-0010" class="usa-link" aria-describedby="tpg270066-bib-0010">1993</a>). Together with the tetrasomic inheritance of potatoes, this has produced only minimal changes to the allele frequencies of the elite gene pool, causing the genetic gain of potato breeding to be low (Ortiz, <a href="#tpg270066-bib-0045" class="usa-link" aria-describedby="tpg270066-bib-0045">2020</a>; Slater et al., <a href="#tpg270066-bib-0060" class="usa-link" aria-describedby="tpg270066-bib-0060">2016</a>).</p>
<p>Breeding programs in potatoes have traditionally relied on phenotypic selection and, more recently, partially on pedigree‐based selection. Also, the advent of affordable genome sequencing technology and high‐throughput genotyping has allowed the incorporation of genetic variants into selection models, termed genomic selection (Meuwissen et al., <a href="#tpg270066-bib-0042" class="usa-link" aria-describedby="tpg270066-bib-0042">2001</a>). Genomic selection is becoming widely employed in potato breeding to increase the genetic gain of key agronomic traits and accelerate breeding by predicting genomic estimated breeding values (genomic EBVs) (Ortiz, <a href="#tpg270066-bib-0045" class="usa-link" aria-describedby="tpg270066-bib-0045">2020</a>), using a variety of statistical and machine‐learning methods (Ortiz, <a href="#tpg270066-bib-0045" class="usa-link" aria-describedby="tpg270066-bib-0045">2020</a>; Ortiz et al., <a href="#tpg270066-bib-0047" class="usa-link" aria-describedby="tpg270066-bib-0047">2023</a>; Sverrisdóttir et al., <a href="#tpg270066-bib-0065" class="usa-link" aria-describedby="tpg270066-bib-0065">2017</a>; Wilson et al., <a href="#tpg270066-bib-0076" class="usa-link" aria-describedby="tpg270066-bib-0076">2021</a>), including the standard additive genomic BLUP (GBLUP, where BLUP is best linear unbiased prediction) model (Meuwissen et al., <a href="#tpg270066-bib-0042" class="usa-link" aria-describedby="tpg270066-bib-0042">2001</a>). Nonadditive genetic effects are lost over sexual generations due to recombination (epistatic effects) and independent assortment of chromosomes (dominance effects). However, they may still be very relevant to select variants for the market if the ratio of nonadditive compared to additive genetic variation is significant because the nonadditive genetic effects are maintained during vegetative propagation. This may be particularly valuable for lowheritability traits where nonadditive effects are generally more important (Endelman et al., <a href="#tpg270066-bib-0018" class="usa-link" aria-describedby="tpg270066-bib-0018">2018</a>; Meuwissen et al., <a href="#tpg270066-bib-0042" class="usa-link" aria-describedby="tpg270066-bib-0042">2001</a>; Wilson et al., <a href="#tpg270066-bib-0076" class="usa-link" aria-describedby="tpg270066-bib-0076">2021</a>). For such traits, traditional phenotype‐directed breeding has been largely ineffective in autotetraploid potato (Wilson et al., <a href="#tpg270066-bib-0076" class="usa-link" aria-describedby="tpg270066-bib-0076">2021</a>). It is thus relevant to elucidate whether this results from additive but complex polygenic inheritance patterns or whether nonadditive mechanisms of inheritance are not accounted for. The estimation of additive genetic effects and specific combining abilities (SCAs) for key agronomic traits can, therefore, impact the design and performance of genomic prediction models. The SCAs are effects that are due to the combined effect of specific parents, whereas the parent general combining ability (GCA) is the general effect of a parent in all potential matings (Gallais, <a href="#tpg270066-bib-0020" class="usa-link" aria-describedby="tpg270066-bib-0020">2003</a>). Additionally, if the SCA is significant, it can impact the cultivars selected for the commercial market, as they can be selected based on total genetic value, whereas parents in the breeding program should be selected for the total additive genetic value because it is preserved across recombination in future generations.</p>
<p>Historically, combining ability studies have been applied in potato research since the 1960s (Plaisted et al., <a href="#tpg270066-bib-0051" class="usa-link" aria-describedby="tpg270066-bib-0051">1962</a>) to evaluate the agronomic trait transmission ability of parental lines, usually using a diallel cross of a selection of clones in the breeding population (Griffing, <a href="#tpg270066-bib-0024" class="usa-link" aria-describedby="tpg270066-bib-0024">1956</a>). Results are specific to the breeding material from which they are estimated (Bradshaw, <a href="#tpg270066-bib-0008" class="usa-link" aria-describedby="tpg270066-bib-0008">2022</a>)—some previous studies have found greater GCA variance compared to SCA for a host of traits, including yield, dry matter content, emergence, maturity, fry color, and tuber count (Bradshaw et al., <a href="#tpg270066-bib-0009" class="usa-link" aria-describedby="tpg270066-bib-0009">2000</a>; Darabad et al., <a href="#tpg270066-bib-0013" class="usa-link" aria-describedby="tpg270066-bib-0013">2020</a>; Ruiz de Arcaute et al., <a href="#tpg270066-bib-0056" class="usa-link" aria-describedby="tpg270066-bib-0056">2022</a>; Terres et al., <a href="#tpg270066-bib-0067" class="usa-link" aria-describedby="tpg270066-bib-0067">2017</a>), while others have found nonadditive contributions to have ascendancy in traits such as tuber shape, count, and weight with fluctuations across generations (Gopal, <a href="#tpg270066-bib-0023" class="usa-link" aria-describedby="tpg270066-bib-0023">1998</a>; Terres et al., <a href="#tpg270066-bib-0067" class="usa-link" aria-describedby="tpg270066-bib-0067">2017</a>). In addition, GCA values are highly differentiated between parents (Ortiz et al., <a href="#tpg270066-bib-0046" class="usa-link" aria-describedby="tpg270066-bib-0046">1988</a>). General combining abilities can be a key parameter in parent selection and are superior compared to traditional phenotypic selection based on mean phenotypic values by accounting for both the efficiently transmitted additive genetic effects and distinguishing this from the nonadditive contribution. However, their widespread implementation by breeders has been limited (Bradshaw, <a href="#tpg270066-bib-0008" class="usa-link" aria-describedby="tpg270066-bib-0008">2022</a>) as accurate selection of parents based on additive genetic value requires the separation of additive and nonadditive effects. However, the separation of the nonadditive effects is only available for existing offspring. Hence, predictors for novel crosses require complex use of genomic information for unbiased results (Chu &amp; Jensen, <a href="#tpg270066-bib-0012" class="usa-link" aria-describedby="tpg270066-bib-0012">2025</a>; Kristensen et al., <a href="#tpg270066-bib-0032" class="usa-link" aria-describedby="tpg270066-bib-0032">2023</a>).</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270066-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270066-list-0001" class="list" style="list-style-type:disc">
<li><p>Cytoplasmic genome type can impact clone fertility and some trait phenotypes and should influence parent selection.</p></li>
<li><p>All traits had significant additive genetic contributions, indicating potential for genetic gain through breeding.</p></li>
<li><p>It was trait dependent whether additive versus nonadditive genetic effects were more important.</p></li>
<li><p>Prediction accuracy of parent estimated breeding values improves with the incorporation of genomic and pedigree information in ssGBLUP.</p></li>
<li><p>Significant pairwise trait genetic correlations indicate extensive indirect trait selection during breeding.</p></li>
</ul></section></section><p>Our interest in revisiting additive genetic variance in potato is to estimate its importance for genomic prediction models. We further explore the differences between parents and segregation within full‐ and half‐sib families using a training population of a diallel cross between parents derived from a representative sample of the germplasms in the elite gene pool of the Danish potato breeding company Danespo A/S. That is, determining the fraction of the total genetic trait variation that can be captured in additive prediction models by estimating the proportion of the variation that is attributable to additive versus nonadditive effects (the SCA/GCA ratio), where the SCA variance is defined as the sum of dominance and epistatic variation due to interaction between parental genomes, and the GCA variance is one‐fourth of the additive genetic variance (Gallais, <a href="#tpg270066-bib-0020" class="usa-link" aria-describedby="tpg270066-bib-0020">2003</a>). Besides the additive and nonadditive genetic contributions, the cytoplasmic genome type (i.e., the genetic variance found in the mitochondrial and plastid genomes) has also been found to contribute to the phenotypic variance of some agronomic traits, for example, tuber shape, vine maturity, starch content, and yield (Sanetomo &amp; Gebhardt, <a href="#tpg270066-bib-0057" class="usa-link" aria-describedby="tpg270066-bib-0057">2015</a>). However, due to the limited study of the effects of cytoplasmic genome type on phenotypes, the cytoplasmic genome effects are only known for a limited array of traits and have not been reproduced in populations including non‐selected clones. Six overall types of potato cytoplasmic genomes have been identified, that is, T, D, W, A, M, and P, some with respective α‐, β‐, and γ‐ mitochondrial subtypes (Lössl et al., <a href="#tpg270066-bib-0038" class="usa-link" aria-describedby="tpg270066-bib-0038">2000</a>; Sanetomo &amp; Gebhardt, <a href="#tpg270066-bib-0057" class="usa-link" aria-describedby="tpg270066-bib-0057">2015</a>). In the European germplasm pool, the types A, M, and P are relatively rare, while T is predominant (Sanetomo &amp; Gebhardt, <a href="#tpg270066-bib-0057" class="usa-link" aria-describedby="tpg270066-bib-0057">2015</a>). Besides contributing to the total genetic variation of cultivars, some of the cytoplasmic genome types can also confer cytoplasmic male sterility, that is, types D and subtype W/γ (Hosaka &amp; Sanetomo, <a href="#tpg270066-bib-0029" class="usa-link" aria-describedby="tpg270066-bib-0029">2012</a>; Lössl et al., <a href="#tpg270066-bib-0038" class="usa-link" aria-describedby="tpg270066-bib-0038">2000</a>; Sanetomo &amp; Gebhardt, <a href="#tpg270066-bib-0057" class="usa-link" aria-describedby="tpg270066-bib-0057">2015</a>). A specific open reading frame encoded in the mitochondrial genome, ORF137, was identified by comparative mitochondrial genome analysis, and genetic variation in the open reading frame is thought to be associated with cytoplasmic male sterility in <em>S. tuberosum</em> (Lian et al., <a href="#tpg270066-bib-0037" class="usa-link" aria-describedby="tpg270066-bib-0037">2024</a>). Through their unintentional propagation in the European gene pool, the cytoplasmic male sterility now constitutes a major genetic bottleneck in modern potato breeding programs (Provan et al., <a href="#tpg270066-bib-0053" class="usa-link" aria-describedby="tpg270066-bib-0053">1999</a>). In modern breeding programs, it is relevant to consider the phenotypic contribution of the cytoplasmic genome type in concert with its possible effect on fertility to make informed selection—while commercial clones may possibly benefit from the phenotypic contribution of a male sterile cytoplasmic genome type, retaining fertility is paramount in parent selection.</p>
<p>Another property relevant to evaluating the power of genomic prediction in breeding programs arises from its contextual application. While genomic selection models are commonly optimized for single traits (Aalborg et al., <a href="#tpg270066-bib-0002" class="usa-link" aria-describedby="tpg270066-bib-0002">2024</a>; Stich &amp; Van Inghelandt, <a href="#tpg270066-bib-0064" class="usa-link" aria-describedby="tpg270066-bib-0064">2018</a>; Sverrisdóttir et al., <a href="#tpg270066-bib-0065" class="usa-link" aria-describedby="tpg270066-bib-0065">2017</a>, <a href="#tpg270066-bib-0066" class="usa-link" aria-describedby="tpg270066-bib-0066">2018</a>; Wilson et al., <a href="#tpg270066-bib-0076" class="usa-link" aria-describedby="tpg270066-bib-0076">2021</a>), breeding programs involve the concurrent selection of multiple agronomic performance characteristics, for example, for optimizing the same trait in disparate environments (Ortiz et al., <a href="#tpg270066-bib-0047" class="usa-link" aria-describedby="tpg270066-bib-0047">2023</a>; Pandey et al., <a href="#tpg270066-bib-0048" class="usa-link" aria-describedby="tpg270066-bib-0048">2023</a>) or co‐selecting for several traits relevant for tuber quality, all requiring the application of multi‐environment/multi‐trait prediction models that can account for genetic correlations between the traits and between traits’ expressions in different environments. To guide parental selection on multiple traits, both individual traits’ genetic variance and the genetic and environmental correlations between traits need to be accounted for to estimate a direct selection response in selected traits, as well as indirect responses in correlated traits not under direct selection (Neyhart et al., <a href="#tpg270066-bib-0044" class="usa-link" aria-describedby="tpg270066-bib-0044">2019</a>).</p>
<p>This study aimed to estimate population parameters of 10 key agronomic traits relative to transmission to progeny. We used the MASPOT experimental population, consisting of 5013 clones of an F<sub>1</sub> population generated from an incomplete diallel cross of 18 elite cultivar/breeding material parents, to investigate the transmitting ability of the 18 parental cultivars included in the experiment. The traits include dry matter content, yield, eye depth, flesh color, skin finish, senescence, tubers/plant, length, diameter, and length/width ratio. The MASPOT experimental population has previously been used in genomic prediction studies of various agronomic traits (Sverrisdóttir et al., <a href="#tpg270066-bib-0065" class="usa-link" aria-describedby="tpg270066-bib-0065">2017</a>), to evaluate the effect of expanding the training population in genomic prediction models (Sverrisdóttir et al., <a href="#tpg270066-bib-0066" class="usa-link" aria-describedby="tpg270066-bib-0066">2018</a>), and to investigate the effect of marker type, ploidy, and density on genomic prediction and genome‐wide association study (GWAS) of tetraploid potatoes (Aalborg &amp; Nielsen, <a href="#tpg270066-bib-0001" class="usa-link" aria-describedby="tpg270066-bib-0001">2024</a>; Aalborg et al., <a href="#tpg270066-bib-0002" class="usa-link" aria-describedby="tpg270066-bib-0002">2024</a>). Here, we used phenotypic data of the MASPOT population and of the F<sub>0</sub> parents to estimate the amount of total genetic variation attributable to additive genetic effects and SCA. It was also investigated whether the relative amount of additive genetic variance and SCA variance is trait‐dependent and related to trait heritability in the defined population. The effect of the maternally inherited cytoplasmic genome type on trait phenotypes was also assessed relative to the fertility phenotype, also controlled by these genetic elements. Furthermore, we estimated the phenotypic and additive genetic correlations between the traits analyzed.</p></section><section id="tpg270066-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270066-sec-0040"><h3 class="pmc_sec_title">2.1. Plant material</h3>
<p>The MASPOT population, consisting of 5013 F<sub>1</sub> clones from an incomplete diallel cross of 18 parents (Figure <a href="#tpg270066-fig-0001" class="usa-link">1</a>), was created by Danespo A/S. The 18 elite parents, either established cultivars or advanced breeding clones from the Danespo A/S breeding program, were selected to be relatively unrelated through pedigree (maximum 12.5%) (Figure <a href="#tpg270066-supinfo-0001" class="usa-link">S1</a>). The MASPOT parents belonged to different market segments, that is, fresh market, starch, and processing (French fry and crisps). The population was generated by systematic cross‐pollination of all parent combinations, and all vegetative progeny plants capable of producing at least three tubers/plant were propagated and phenotyped for 10 agronomic traits. The diallel crossing design was limited by the male sterility of five parent cultivars and the low fertility of specific crosses, either failing to generate successful seeds or producing seeds that germinated into degenerate and/or non‐tuberizing plants. The pedigree inbreeding coefficient of the full MASPOT population was found to be 0.00021, and the genomic inbreeding coefficient for a representative panel of 755 genotyped clones was 0.033.</p>
<figure class="fig xbox font-sm" id="tpg270066-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6b9c/12207579/114892133401/TPG2-18-e70066-g003.jpg" loading="lazy" id="jats-graphic-1" height="526" width="800" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270066-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Crossing scheme of the MASPOT incomplete diallel cross showing the number of full‐sibs generated within each family in the F<sub>1</sub> generation in the full population of 5013 clones.</p></figcaption></figure><p>The F<sub>1</sub> clones were planted in field trials in Vandel, Denmark, in 2013 and 2014, as previously described (Aalborg et al., <a href="#tpg270066-bib-0002" class="usa-link" aria-describedby="tpg270066-bib-0002">2024</a>; Sverrisdóttir et al., <a href="#tpg270066-bib-0065" class="usa-link" aria-describedby="tpg270066-bib-0065">2017</a>). Tuber seedlings were planted in 24‐parcel blocks (no replicates and no checks) in 2013 and in randomized 28‐parcel blocks (two replicates, using a single check) in 2014. The parents were also planted in replicates in 2014 and functioned as checks. Plants were grown with a density of approximately 40,000 plants/ha, with 30 cm between plants and 75 cm between rows. In 2014, the plants were grouped by senescence to minimize “neighbor vigor effects” in the trials. Propagation was conducted concurrently with trials, and hence, the number of tubers available in 2013 was lower compared to 2014.</p></section><section id="tpg270066-sec-0050"><h3 class="pmc_sec_title">2.2. Cytoplasm genome type</h3>
<p>The effect of cytoplasm type on trait phenotype was included in all models. Cytoplasm types were assigned based on trace‐back of the maternal pedigree, using the Danespo A/S breeding database of pedigree on the breeding materials or the WUR Potato Pedigree Database (Hutten &amp; Van Berloo, <a href="#tpg270066-bib-0030" class="usa-link" aria-describedby="tpg270066-bib-0030">2001</a>; Van Berloo et al., <a href="#tpg270066-bib-0070" class="usa-link" aria-describedby="tpg270066-bib-0070">2007</a>) for established cultivars, until a maternal progenitor was encountered with cytoplasm type determined by Sanetomo and Gebhardt (<a href="#tpg270066-bib-0057" class="usa-link" aria-describedby="tpg270066-bib-0057">2015</a>). For cultivars of non‐European maternal descent, cytoplasm types determined by Danespo A/S were used.</p></section><section id="tpg270066-sec-0060"><h3 class="pmc_sec_title">2.3. Phenotypes</h3>
<p>A total of 10 traits were phenotyped, namely dry matter content, yield, eye depth, flesh color, skin finish, senescence, tubers/plant, length, diameter, and length/width ratio.</p>
<section id="tpg270066-sec-0070"><h4 class="pmc_sec_title">2.3.1. Dry matter content</h4>
<p>Dry matter content (%) was determined for clones harvested in 2013 (one replicate) and 2014 (two replicates). Tubers were washed, and a basket holding 1.5–10 kg of tubers was weighed above and below water shortly after harvest. Dry matter content was computed using an empirical equation (Equation <a href="#tpg270066-disp-0001" class="usa-link">1</a>) modified from the Haase (<a href="#tpg270066-bib-0026" class="usa-link" aria-describedby="tpg270066-bib-0026">2003</a>) equation for dry matter content estimation from specific gravity based on best linear fit with Danespo A/S data:
</p>
<table class="disp-formula p" id="tpg270066-disp-0001"><tr>
<td class="formula"><math id="jats-math-1" display="block"><mrow><mtable displaystyle="true"><mtr><mtd columnalign="right"><mrow><mrow><mi>DM</mi><mspace width="0.28em"></mspace></mrow><mfenced open="(" close=")"><mo>%</mo></mfenced></mrow></mtd><mtd><mo>=</mo></mtd><mtd columnalign="left"><mrow><mn>214</mn><mo linebreak="badbreak">×</mo><mo stretchy="true">(</mo><mfenced separators="" open="(" close=")"><mfrac><mrow><mi>Weight</mi><mspace width="0.28em"></mspace><mi>in</mi><mspace width="0.28em"></mspace><mi>air</mi></mrow><mrow><mfenced separators="" open="(" close=")"><mrow><mi>Weight</mi><mspace width="0.28em"></mspace><mi>in</mi><mspace width="0.28em"></mspace><mi>air</mi></mrow></mfenced><mo>−</mo><mfenced separators="" open="(" close=")"><mrow><mi>Weight</mi><mspace width="0.28em"></mspace><mi>in</mi><mspace width="0.28em"></mspace><mi>water</mi></mrow></mfenced></mrow></mfrac></mfenced></mrow></mtd></mtr><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mrow><mo linebreak="badbreak">−</mo><mspace width="0.16em"></mspace><mn>0.988</mn><mo stretchy="true">)</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where DM is dry matter content.</p></section><section id="tpg270066-sec-0080"><h4 class="pmc_sec_title">2.3.2. Yield and tuber count</h4>
<p>The yield was measured for the two replicates harvested in 2014 as the total weight of five tubers from each clone in two replicates, that is, 2 × 5 tubers for each clone. The weight was converted into hkg/ha, assuming 40,000 plants/ha. The total number of tubers per plant was counted for each clone.</p></section><section id="tpg270066-sec-0090"><h4 class="pmc_sec_title">2.3.3. Eye depth</h4>
<p>Eye depth was scored manually on a 1–9 scale from deep eyes to no eyes only in 2013. Due to limited application of the 1–4 end of the scale, the phenotype data were recoded to a 1–6 scale by pooling all observations ≤4 and resetting the scale to deep = 1.</p></section><section id="tpg270066-sec-0100"><h4 class="pmc_sec_title">2.3.4. Flesh color</h4>
<p>Flesh color was scored manually on a 1–9 scale from white to orange flesh for the 2014 harvested clones.</p></section><section id="tpg270066-sec-0110"><h4 class="pmc_sec_title">2.3.5. Skin finish</h4>
<p>Skin finish was scored manually on a 1–9 scale from rough to smooth for the 2014 harvested clones. Due to the limited application of the 1–4 end of the scale, the phenotype data were recoded to a 1–6 scale by pooling all observations ≤4 and resetting the scale to rough = 1.</p></section><section id="tpg270066-sec-0120"><h4 class="pmc_sec_title">2.3.6. Senescence</h4>
<p>Senescence was scored manually on a 1–9 scale from no senescence to late senescence for both the 2013 and 2014 harvested clones. The 2013 measures were used to group the clones according to senescence in the 2014 field layout. The scoring was performed at three temporal points, splitting the scale accordingly: first scoring (when the first cultivars begin dying off) used the upper end of the scale (9‐8‐7), second scoring used the middle of the scale (6‐5‐4), and the third scoring (capturing late senescence cultivars) used the low end of the scale (3‐2‐1). Each clone was only scored once; that is, different plants were scored at each temporal point.</p></section><section id="tpg270066-sec-0130"><h4 class="pmc_sec_title">2.3.7. Tuber size and shape</h4>
<p>A SCOUT camera (Nextec A/S, No.: 0213) was used to measure whole‐tuber length and diameter for the clones harvested in 2014. Tuber length was defined as the longest measure (mm), and the diameter (width) (mm) as the measure perpendicular to this. These measures did not consider tuber anatomy, where length is defined as the distance from the rose (apex) to the heel (attachment of stolon). Only irregular tubers will violate the true definition, which are relatively rare and assumed to not significantly affect downstream analyses. Outliers were identified on the length or diameter parameters relative to the nearest neighbor and removed following Dixon's <em>Q</em>‐test (Dean &amp; Dixon, <a href="#tpg270066-bib-0014" class="usa-link" aria-describedby="tpg270066-bib-0014">1951</a>) before calculation of a length/width ratio, where the critical confidence level, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline"><mrow><msub><mi>Q</mi><mi>crit</mi></msub></mrow></math></span>, was estimated for batches of up to 200 tubers by regression and used for a two‐tailed test as outlined by Rorabacher (<a href="#tpg270066-bib-0055" class="usa-link" aria-describedby="tpg270066-bib-0055">1991</a>).</p></section></section><section id="tpg270066-sec-0140"><h3 class="pmc_sec_title">2.4. Statistical modeling</h3>
<p>All models were fitted using the DMU software package (v6, release 5.6) ( Madsen et al., <a href="#tpg270066-bib-0080" class="usa-link" aria-describedby="tpg270066-bib-0080">2014</a>). Data analysis and other statistical analyses were performed in R Statistical Software (v4.2.2) (R Core Team, <a href="#tpg270066-bib-0054" class="usa-link" aria-describedby="tpg270066-bib-0054">2023</a>) in RStudio (v2024.04.0+735) (Posit team, <a href="#tpg270066-bib-0052" class="usa-link" aria-describedby="tpg270066-bib-0052">2023</a>). All graphics were generated in R using base R or the ggplot2 package (v3.4.2) or (v3.5.1) (Wickham, <a href="#tpg270066-bib-0075" class="usa-link" aria-describedby="tpg270066-bib-0075">2016</a>), unless otherwise stated.</p></section><section id="tpg270066-sec-0150"><h3 class="pmc_sec_title">2.5. Estimation of variance components and prediction of breeding values</h3>
<p>Additive and nonadditive genetic variance components were estimated, and breeding values were predicted for single traits using four models, each following the same general linear mixed model (Equation <a href="#tpg270066-disp-0002" class="usa-link">2</a>):
</p>
<table class="disp-formula p" id="tpg270066-disp-0002"><tr>
<td class="formula"><math id="jats-math-3" display="block"><mrow><mrow><mi>y</mi><mo linebreak="badbreak">=</mo><mi>X</mi><mi>b</mi><mo linebreak="goodbreak">+</mo><msub><mi>Z</mi><mi>a</mi></msub><mi>a</mi><mo linebreak="goodbreak">+</mo><msub><mi>Z</mi><mi>a</mi></msub><mi>l</mi><mo linebreak="goodbreak">+</mo><msub><mi>Z</mi><mi>f</mi></msub><mi>f</mi><mo linebreak="goodbreak">+</mo><mi>e</mi><mo>,</mo></mrow></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>where <strong>
<em>y</em>
</strong> is a vector of phenotypes, <strong>
<em>X</em>
</strong> is the design matrix of the fixed effects, and <strong>
<em>b</em>
</strong> is a vector of fixed effects, including block‐by‐year, to correct for soil heterogeneity and across‐year environmental effects for multi‐year traits, and cytoplasm type; <strong>
<em>a</em>
</strong> is a vector of additive genetic effects (breeding values) with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><mrow><mi>a</mi><mspace width="0.28em"></mspace><mo>∼</mo><mspace width="0.28em"></mspace><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><mi>M</mi><msubsup><mi mathvariant="normal">σ</mi><mi>a</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, where <strong>
<em>M</em>
</strong> is the relationship matrix and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-5" display="inline"><mrow><msub><mi>Z</mi><mi>a</mi></msub></mrow></math></span> is the incidence matrix relating observations to clones, that is, both additive genetic effects as well as residual line effects. Different assumptions were made depending on the relationship matrix used for the model. <strong>
<em>l</em>
</strong> is a vector of line effects with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><mrow><mi>l</mi><mspace width="0.28em"></mspace><mo>∼</mo><mspace width="0.28em"></mspace><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msub><mi>I</mi><mi>l</mi></msub><msubsup><mi mathvariant="normal">σ</mi><mi>l</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, where <strong>
<em>I<sub>l</sub>
</em>
</strong> is an identity matrix with the number of lines as dimensions; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><msub><mi>Z</mi><mi>f</mi></msub></mrow></math></span> is the design matrix for the full‐sib family (nonadditive) genetic effects due to interactions between parental genomes, indicating sire‐dam subgroups; <strong>
<em>f</em>
</strong> is a vector of the family genetic effects due to interaction between sire and dam contributions with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><mrow><mi>f</mi><mspace width="0.28em"></mspace><mo>∼</mo><mspace width="0.28em"></mspace><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msub><mi>I</mi><mi>f</mi></msub><mspace width="0.28em"></mspace><msubsup><mi mathvariant="normal">σ</mi><mi>f</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, where <strong>
<em>I<sub>f</sub>
</em>
</strong> is an identity matrix with the number of families as dimensions; <strong>
<em>e</em>
</strong> is a vector of residuals with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><mrow><mi>e</mi><mspace width="0.28em"></mspace><mo>∼</mo><mspace width="0.28em"></mspace><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><mi>I</mi><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span> with identity matrix <strong>
<em>I</em>
</strong> of dimension number of observations. We estimate the total additive genetic effect of a parental clone (the EBV) and not the classical GCA of parents, which is only half the additive genetic value (EBV = 2GCA). The use of full relationship matrices among all clones allows the exploitation of all relationships in estimating additive genetic variance and in the prediction of breeding values. Only dry matter content relied on phenotype recordings across two years for model fitting, and for this trait the line effect was nested within year of test. The line effect was only included for traits where the clones had replicates; that is, it was excluded for models on eye depth. The variance due to line effects will include nonadditive genetic variances due to interactions between loci inherited from individual parents as well as line‐related effects due to common source and storage of the seed potatoes for each clone. Due to this ambiguity, we do not estimate broad‐sense heritability in this analysis. The SCA/GCA ratio was estimated as the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><mfrac><msubsup><mi mathvariant="normal">σ</mi><mi>f</mi><mn>2</mn></msubsup><mrow><mfrac><mn>1</mn><mn>4</mn></mfrac><msubsup><mi mathvariant="normal">σ</mi><mi>a</mi><mn>2</mn></msubsup></mrow></mfrac></mrow></math></span>, as the additive value that is transmitted from parent to offspring is half the additive parent value, and the GCA variance equals <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline"><mrow><mrow><mfrac><mn>1</mn><mn>4</mn></mfrac><msubsup><mi mathvariant="normal">σ</mi><mi>a</mi><mn>2</mn></msubsup></mrow></mrow></math></span>. The incomplete nature of the diallel cross precluded estimation of separate maternal and paternal nonadditive genetic variance components. Instead, the nonadditive genetic variation was estimated as the family variance.</p>
<p>Variance components were estimated for additive genetic variance and SCA (nonadditive genetic variance due to interaction between parental genomes) using four models, namely, pedigree BLUP (PBLUP) on the full MASPOT population (PBLUP<sub>pop</sub>) and on a genotyped subset of 755 clones, called the MASPOT panel (PBLUP<sub>panel</sub>), genomic BLUP on the genotyped panel (GBLUP), and single‐step BLUP (ssGBLUP) on the full MASPOT population. Distinct relationship matrices were used for each model to describe the variance–covariance structure.</p>
<section id="tpg270066-sec-0160"><h4 class="pmc_sec_title">2.5.1. PBLUP</h4>
<p>A pedigree relationship (<strong>
<em>A</em>
</strong>) matrix (Figure <a href="#tpg270066-supinfo-0001" class="usa-link">S2</a>) was constructed from the pedigree information of the MASPOT F<sub>1</sub> offspring, the MASPOT parents, and the MASPOT grandparent generation for PBLUP for both the full MASPOT population and the genotyped MASPOT panel. The AGHmatrix package in R (v.2.1.4) (Amadeu et al., <a href="#tpg270066-bib-0003" class="usa-link" aria-describedby="tpg270066-bib-0003">2023</a>) with a ploidy of 4 and no double reduction was used to compute the matrices. A fixed level of double reduction was not included, despite it being an established inheritance mechanism in polysomic species (Bourke et al., <a href="#tpg270066-bib-0007" class="usa-link" aria-describedby="tpg270066-bib-0007">2018</a>; Gerard, <a href="#tpg270066-bib-0022" class="usa-link" aria-describedby="tpg270066-bib-0022">2023</a>; Hardy, <a href="#tpg270066-bib-0027" class="usa-link" aria-describedby="tpg270066-bib-0027">2016</a>; Parisod et al., <a href="#tpg270066-bib-0050" class="usa-link" aria-describedby="tpg270066-bib-0050">2010</a>), as simulations have shown low rates of double reduction, with variable rates relative to the distance to the centromeres in tetraploid potatoes (Bourke et al., <a href="#tpg270066-bib-0006" class="usa-link" aria-describedby="tpg270066-bib-0006">2015</a>). Analysis of model fit measured by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-12" display="inline"><mrow><mrow><mo>−</mo><mn>2</mn><mi>log</mi><mo stretchy="false">(</mo><mi>L</mi><mo stretchy="false">)</mo></mrow></mrow></math></span>, where <em>L</em> is the likelihood of the model, showed a local minimum with 0% double reduction (Figure <a href="#tpg270066-supinfo-0001" class="usa-link">S3</a>), which therefore was implemented in all models. For the models based on the full and reduced pedigree relationship matrices (PBLUP<sub>pop</sub> and PBLUP<sub>panel</sub>, respectively), the vector of additive genetic effects is defined as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline"><mrow><mrow><mi>a</mi><mspace width="0.28em"></mspace><mo>∼</mo><mspace width="0.28em"></mspace><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><mi>A</mi><msubsup><mi mathvariant="normal">σ</mi><mi>a</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>. The pedigree inbreeding coefficient was calculated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-14" display="inline"><mrow><mfrac><mrow><mi>mean</mi><mspace width="0.28em"></mspace><mo stretchy="false">(</mo><mrow><mi>diag</mi><mo stretchy="false">(</mo><mi>A</mi><mo stretchy="false">)</mo></mrow><mo stretchy="false">)</mo><mspace width="0.28em"></mspace><mo>−</mo><mspace width="0.28em"></mspace><mn>1</mn></mrow><mrow><mi>ploidy</mi><mspace width="0.28em"></mspace><mo>−</mo><mspace width="0.28em"></mspace><mn>1</mn></mrow></mfrac></mrow></math></span>, where diag(<strong>
<em>A</em>
</strong>) is the diagonal elements of <strong>
<em>A</em>
</strong> for the full MASPOT population (Gallais, <a href="#tpg270066-bib-0020" class="usa-link" aria-describedby="tpg270066-bib-0020">2003</a>).</p></section><section id="tpg270066-sec-0170"><h4 class="pmc_sec_title">2.5.2. GBLUP</h4>
<p>For GBLUP, genotyping‐by‐sequencing (GBS) allele frequency genotypic data for 755 MASPOT offspring and of the 18 parents were used (reduced crossing scheme in Aalborg et al. [<a href="#tpg270066-bib-0002" class="usa-link" aria-describedby="tpg270066-bib-0002">2024</a>]). A previously published 93,170 SNP (single nucleotide polymorphism) dataset filtered to minor allele frequency &gt; 1%, read depth &gt; 5 and &lt; 60, and missing rate &lt; 50% of approximately equal parts non‐synonymous, synonymous, and non‐coding biallelic variants (Aalborg et al., <a href="#tpg270066-bib-0002" class="usa-link" aria-describedby="tpg270066-bib-0002">2024</a>) was used. A genomic relationship (<strong>
<em>G</em>
</strong>) matrix (Figure <a href="#tpg270066-supinfo-0001" class="usa-link">S4</a>) was computed using the AGHmatrix package using the VanRaden method for non‐integer dosage genotypes with adjustment for tetraploidy (VanRaden, <a href="#tpg270066-bib-0074" class="usa-link" aria-describedby="tpg270066-bib-0074">2008</a>) and using mean imputation of missing SNP information (17.06% of all markers). For the GBLUP model, the vector of additive genetic effects is defined as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-15" display="inline"><mrow><mrow><mi>a</mi><mspace width="0.28em"></mspace><mo>∼</mo><mspace width="0.28em"></mspace><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><mi>G</mi><msubsup><mi mathvariant="normal">σ</mi><mi>a</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>. The genomic inbreeding coefficient was calculated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-16" display="inline"><mrow><mfrac><mrow><mi>mean</mi><mo stretchy="false">(</mo><mrow><mi>diag</mi><mo stretchy="false">(</mo><mi>G</mi><mo stretchy="false">)</mo></mrow><mo stretchy="false">)</mo><mo>−</mo><mn>1</mn></mrow><mrow><mrow><mi>ploidy</mi><mspace width="0.28em"></mspace></mrow><mo>−</mo><mn>1</mn></mrow></mfrac></mrow></math></span>, where diag(<strong>
<em>G</em>
</strong>) is the diagonal elements of <strong>
<em>G</em>
</strong> (Gallais, <a href="#tpg270066-bib-0020" class="usa-link" aria-describedby="tpg270066-bib-0020">2003</a>).</p></section><section id="tpg270066-sec-0180"><h4 class="pmc_sec_title">2.5.3. ssGBLUP</h4>
<p>For ssGBLUP, a single‐step covariance (<strong>
<em>H</em>
</strong>) matrix (Figure <a href="#tpg270066-supinfo-0001" class="usa-link">S5</a>) was constructed by combining the <strong>
<em>A</em>
</strong> and <strong>
<em>G</em>
</strong> matrices using the AGHmatrix package. The scale of the <strong>
<em>G</em>
</strong> matrix was adjusted to the scale of the pedigree relationship matrix of the genotyped clones (<strong>
<em>A</em>
<sub>22</sub>
</strong>) prior to the combination using Equation (<a href="#tpg270066-disp-0003" class="usa-link">3</a>):
</p>
<table class="disp-formula p" id="tpg270066-disp-0003"><tr>
<td class="formula"><math id="jats-math-17" display="block"><mrow><mrow><msub><mi>G</mi><mi>adj</mi></msub><mo linebreak="badbreak">=</mo><mfrac><mi>G</mi><mrow><mi>mean</mi><mfenced separators="" open="(" close=")"><mrow><mi>diag</mi><mfenced open="(" close=")"><mi>G</mi></mfenced></mrow></mfenced></mrow></mfrac><mo linebreak="goodbreak">×</mo><mi>m</mi><mi>e</mi><mi>a</mi><mi>n</mi><mfenced separators="" open="(" close=")"><mrow><mi>diag</mi><mfenced separators="" open="(" close=")"><mrow><msub><mi>A</mi><mn>22</mn></msub><mspace width="0.28em"></mspace></mrow></mfenced></mrow></mfenced><mo>.</mo></mrow></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>Following adjustment, the matrices were combined using the Martini method (Martini et al., <a href="#tpg270066-bib-0041" class="usa-link" aria-describedby="tpg270066-bib-0041">2018</a>) with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-18" display="inline"><mrow><mrow><mi mathvariant="normal">τ</mi><mo>=</mo><mi>ω</mi><mo>=</mo><mn>1</mn></mrow></mrow></math></span>, corresponding to the scaling in Legarra et al. (<a href="#tpg270066-bib-0033" class="usa-link" aria-describedby="tpg270066-bib-0033">2009</a>) and Christensen and Lund (<a href="#tpg270066-bib-0011" class="usa-link" aria-describedby="tpg270066-bib-0011">2010</a>). For the ssGBLUP model, the vector of additive genetic effects is defined as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-19" display="inline"><mrow><mrow><mi>a</mi><mspace width="0.28em"></mspace><mo>∼</mo><mspace width="0.28em"></mspace><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><mi>H</mi><msubsup><mi mathvariant="normal">σ</mi><mi>a</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>.</p></section><section id="tpg270066-sec-0190"><h4 class="pmc_sec_title">2.5.4. Heritability</h4>
<p>The narrow‐sense additive genetic heritability was computed as the ratio of additive genetic (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-20" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>a</mi><mn>2</mn></msubsup></mrow></math></span>) to total phenotypic variance (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-21" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>p</mi><mn>2</mn></msubsup></mrow></math></span>) based on the PBLUP<sub>pop</sub> linear mixed model following Equation (<a href="#tpg270066-disp-0004" class="usa-link">4</a>):
</p>
<table class="disp-formula p" id="tpg270066-disp-0004"><tr>
<td class="formula"><math id="jats-math-22" display="block"><mrow><mtable displaystyle="true"><mtr><mtd columnalign="right"><msup><mi>h</mi><mn>2</mn></msup></mtd><mtd><mo>=</mo></mtd><mtd columnalign="left"><mfrac><mrow><mo stretchy="false">(</mo><msubsup><mi mathvariant="normal">σ</mi><mi>a</mi><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow><mrow><mo stretchy="false">(</mo><msubsup><mi mathvariant="normal">σ</mi><mi>p</mi><mn>2</mn></msubsup><mo stretchy="false">)</mo></mrow></mfrac></mtd></mtr><mtr><mtd columnalign="right"><msubsup><mi mathvariant="normal">σ</mi><mi>p</mi><mn>2</mn></msubsup></mtd><mtd><mo>=</mo></mtd><mtd columnalign="left"><mrow><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>a</mi><mn>2</mn></msubsup><mo linebreak="badbreak">+</mo><msubsup><mi mathvariant="normal">σ</mi><mi>f</mi><mn>2</mn></msubsup><mo linebreak="goodbreak">+</mo><msubsup><mi mathvariant="normal">σ</mi><mi>l</mi><mn>2</mn></msubsup><mo linebreak="goodbreak">+</mo><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<p>that is, the heritability of an individual plot record.</p></section><section id="tpg270066-sec-0200"><h4 class="pmc_sec_title">2.5.5. Cross‐validation</h4>
<p>To validate models, breeding values were predicted using 30 repeats of eightfold random cross‐validation, splitting the lines into folds of equal size. Variance components were estimated in the full dataset using the DMU AIREML software and used as priors for predicting breeding values with DMU4 software, using each fold in each repeat in turn as the validation set and the remaining folds as a reduced training set. Prediction correlation of one repeat was determined as the Pearson correlation between the predicted breeding value of each line in the validation set and the mean corrected phenotype for each line (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-23" display="inline"><mrow><mover><msub><mi>y</mi><mi>c</mi></msub><mo>¯</mo></mover></mrow></math></span>) (Equation <a href="#tpg270066-disp-0005" class="usa-link">5</a>). The mean corrected phenotype was calculated by adjusting observed phenotypic values for fixed effects and subsequently calculating the average corrected phenotypic value for each entry. This was done for each of the applied models, using the estimated fixed effects for the respective model in which correlation was determined.</p>
<table class="disp-formula p" id="tpg270066-disp-0005"><tr>
<td class="formula"><math id="jats-math-24" display="block"><mrow><mrow><mi>r</mi><mfenced separators="" open="(" close=")"><mrow><mi>EBV</mi><mo>:</mo><mover><msub><mi>y</mi><mi>c</mi></msub><mo>¯</mo></mover></mrow></mfenced><mo>.</mo></mrow></mrow></math></td>
<td class="label">(5)</td>
</tr></table>
<p>Prediction accuracy (PA) of breeding values was determined as the prediction correlation divided by the square root of the design heritability (Equation <a href="#tpg270066-disp-0006" class="usa-link">6</a>), where no. rep. designates the number of replicates of each clone:
</p>
<table class="disp-formula p" id="tpg270066-disp-0006"><tr>
<td class="formula"><math id="jats-math-25" display="block"><mrow><mtable displaystyle="true"><mtr><mtd columnalign="right"><mi>PA</mi></mtd><mtd><mo>=</mo></mtd><mtd columnalign="left"><mrow><mfrac><mrow><mi>r</mi><mfenced separators="" open="(" close=")"><mrow><mi>EBV</mi><mo>,</mo><mover><msub><mi>y</mi><mi>c</mi></msub><mo>¯</mo></mover></mrow></mfenced></mrow><msqrt><mrow><msubsup><mi>h</mi><mover><msub><mi>y</mi><mi>c</mi></msub><mo>¯</mo></mover><mn>2</mn></msubsup><mspace width="0.28em"></mspace></mrow></msqrt></mfrac><mspace width="0.28em"></mspace><mo>,</mo></mrow></mtd></mtr><mtr><mtd columnalign="right"><msubsup><mi>h</mi><mover><msub><mi>y</mi><mi>c</mi></msub><mo>¯</mo></mover><mn>2</mn></msubsup></mtd><mtd><mo>=</mo></mtd><mtd columnalign="left"><mrow><mfrac><msubsup><mi mathvariant="normal">σ</mi><mi>a</mi><mn>2</mn></msubsup><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>+</mo><msubsup><mi mathvariant="normal">σ</mi><mi>f</mi><mn>2</mn></msubsup><mo>+</mo><msubsup><mi mathvariant="normal">σ</mi><mi>l</mi><mn>2</mn></msubsup><mo>+</mo><mfrac><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup><mrow><mi>no</mi><mrow><mo>.</mo><mspace width="0.28em"></mspace><mi>rep</mi></mrow><mo>.</mo></mrow></mfrac></mrow></mfrac><mo>.</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">(6)</td>
</tr></table>
<p>The design heritability, accounting for the specific experimental design, was used and calculated based on the variance components of the respective model used to calculate the breeding values.</p>
<p>Dispersion bias was determined as the linear regression coefficient between the mean predicted breeding values (EBVs) and the mean corrected phenotype (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-26" display="inline"><mrow><mover><msub><mi>y</mi><mi>c</mi></msub><mo>¯</mo></mover></mrow></math></span>). A slope (<em>β</em>) = 1 indicates no over‐ or under‐dispersion, while <em>β</em> &gt; 1 indicates that extreme EBVs are deflated, and conversely for <em>β</em> &lt; 1 (Luan et al., <a href="#tpg270066-bib-0039" class="usa-link" aria-describedby="tpg270066-bib-0039">2009</a>).</p></section><section id="tpg270066-sec-0210"><h4 class="pmc_sec_title">2.5.6. Phenotypic and additive genetic correlations</h4>
<p>Pairwise phenotypic and additive genetic correlations between traits were estimated using bivariate linear mixed models, using the following model structure for PBLUP<sub>pop</sub> (Equation <a href="#tpg270066-disp-0007" class="usa-link">7</a>):
</p>
<table class="disp-formula p" id="tpg270066-disp-0007"><tr>
<td class="formula"><math id="jats-math-27" display="block"><mrow><mtable displaystyle="true"><mtr><mtd columnalign="right"><mfenced separators="" open="(" close=")"><mtable><mtr><mtd><msub><mi>y</mi><mn>1</mn></msub></mtd></mtr><mtr><mtd><msub><mi>y</mi><mn>2</mn></msub></mtd></mtr></mtable></mfenced></mtd><mtd><mo>=</mo></mtd><mtd columnalign="left"><mrow><mfenced separators="" open="(" close=")"><mtable><mtr><mtd><msub><mi>X</mi><mn>1</mn></msub></mtd><mtd><mn>0</mn></mtd></mtr><mtr><mtd><mn>0</mn></mtd><mtd><msub><mi>X</mi><mn>2</mn></msub></mtd></mtr></mtable></mfenced><mfenced separators="" open="(" close=")"><mtable><mtr><mtd><msub><mi>b</mi><mn>1</mn></msub></mtd></mtr><mtr><mtd><msub><mi>b</mi><mn>2</mn></msub></mtd></mtr></mtable></mfenced><mo linebreak="badbreak">+</mo><mfenced separators="" open="(" close=")"><mtable><mtr><mtd><msub><mi>Z</mi><mrow><mi>a</mi><mn>1</mn></mrow></msub></mtd><mtd><mn>0</mn></mtd></mtr><mtr><mtd><mn>0</mn></mtd><mtd><msub><mi>Z</mi><mrow><mi>a</mi><mn>2</mn></mrow></msub></mtd></mtr></mtable></mfenced><mfenced separators="" open="(" close=")"><mtable><mtr><mtd><msub><mi>a</mi><mn>1</mn></msub></mtd></mtr><mtr><mtd><msub><mi>a</mi><mn>2</mn></msub></mtd></mtr></mtable></mfenced></mrow></mtd></mtr><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mrow><mo linebreak="badbreak">+</mo><mfenced separators="" open="(" close=")"><mtable><mtr><mtd><msub><mi>Z</mi><mrow><mi>a</mi><mn>1</mn></mrow></msub></mtd><mtd><mn>0</mn></mtd></mtr><mtr><mtd><mn>0</mn></mtd><mtd><msub><mi>Z</mi><mrow><mi>a</mi><mn>2</mn></mrow></msub></mtd></mtr></mtable></mfenced><mfenced separators="" open="(" close=")"><mtable><mtr><mtd><msub><mi>l</mi><mn>1</mn></msub></mtd></mtr><mtr><mtd><msub><mi>l</mi><mn>2</mn></msub></mtd></mtr></mtable></mfenced><mo linebreak="goodbreak">+</mo><mfenced separators="" open="(" close=")"><mtable><mtr><mtd><msub><mi>Z</mi><mrow><mi>f</mi><mn>1</mn></mrow></msub></mtd><mtd><mn>0</mn></mtd></mtr><mtr><mtd><mn>0</mn></mtd><mtd><msub><mi>Z</mi><mrow><mi>f</mi><mn>2</mn></mrow></msub></mtd></mtr></mtable></mfenced><mfenced separators="" open="(" close=")"><mtable><mtr><mtd><msub><mi>f</mi><mn>1</mn></msub></mtd></mtr><mtr><mtd><msub><mi>f</mi><mn>2</mn></msub></mtd></mtr></mtable></mfenced></mrow></mtd></mtr><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mrow><mo linebreak="badbreak">+</mo><mfenced separators="" open="(" close=")"><mtable><mtr><mtd><msub><mi>e</mi><mn>1</mn></msub></mtd></mtr><mtr><mtd><msub><mi>e</mi><mn>2</mn></msub></mtd></mtr></mtable></mfenced><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">(7)</td>
</tr></table>
<p>where <strong>
<em>y</em>
<sub>1</sub>
</strong> and <strong>
<em>y</em>
<sub>2</sub>
</strong> are the trait 1 and 2 phenotype vectors, <strong>
<em>b</em>
<sub>1</sub>
</strong> and <strong>
<em>b</em>
<sub>2</sub>
</strong> are the fixed effect vectors for traits 1 and 2 with respective design matrices <strong>
<em>X</em>
<sub>1</sub>
</strong> and <strong>
<em>X</em>
<sub>2</sub>
</strong>, and <strong>
<em>Z</em>
<sub>a1</sub>
</strong> and <strong>
<em>Z</em>
<sub>a2</sub>
</strong> are the incidence matrices relating observations to clones. The vector of additive effects <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-28" display="inline"><mrow><mrow><mo stretchy="false">(</mo><mtable><mtr><mtd><msub><mi>a</mi><mn>1</mn></msub></mtd></mtr><mtr><mtd><msub><mi>a</mi><mn>2</mn></msub></mtd></mtr></mtable><mo stretchy="false">)</mo></mrow></mrow></math></span> follows a distribution of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-29" display="inline"><mrow><mrow><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><mi>A</mi><mo>⊗</mo><mi>C</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, where <strong>
<em>A</em>
</strong> is the <strong>
<em>A</em>
</strong>‐matrix and <strong>
<em>C</em>
</strong> is the variance and covariance matrix of the breeding values of traits 1 and 2. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-30" display="inline"><mrow><mo>⊗</mo></mrow></math></span> is the Kronecker product, so the resulting matrix will have dimensions of the number of individuals in the pedigree by the number of traits in the analysis, that is, 2; <strong>
<em>l</em>
<sub>1</sub>
</strong> and <strong>
<em>l</em>
<sub>2</sub>
</strong> are the line effects; <strong>
<em>Z</em>
<sub>f1</sub>
</strong> and <strong>
<em>Z</em>
<sub>f2</sub>
</strong> are the design matrices of the family effect relating <strong>
<em>f</em>
<sub>1</sub>
</strong> and <strong>
<em>f</em>
<sub>2</sub>
</strong>, the vectors of family effects, to the observations; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-31" display="inline"><mrow><mrow><mo stretchy="false">(</mo><mtable><mtr><mtd><msub><mi>e</mi><mn>1</mn></msub></mtd></mtr><mtr><mtd><msub><mi>e</mi><mn>2</mn></msub></mtd></mtr></mtable><mo stretchy="false">)</mo></mrow></mrow></math></span> is the vector of residuals for both traits, following distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-32" display="inline"><mrow><mrow><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><mi>I</mi><mo>⊗</mo><mi>R</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, where <strong>
<em>I</em>
</strong> is an identity matrix and <strong>
<em>R</em>
</strong> is the residual variance and covariance matrix of the traits. While the additive effect incidence matrices were identical for both traits in all pairings, dry matter content again included nested line effects. The line effect was only included for traits with replicate phenotypic observations.</p>
<p>The phenotypic correlation between trait pairs was computed based on the phenotypic covariance matrix for the random effects and residuals, following Equation (<a href="#tpg270066-disp-0008" class="usa-link">8</a>):
</p>
<table class="disp-formula p" id="tpg270066-disp-0008"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6b9c/12207579/549fa002a55c/TPG2-18-e70066-e042.jpg" loading="lazy" alt="2.5.6."></td>
<td class="label">(8)</td>
</tr></table>
<p>Note that due to eye depth being recorded in 2013 only, and the remaining traits except dry matter content in 2014 only, a residual covariance could not be established for those combinations including eye depth, and hence a phenotypic correlation is not included for those pairs.</p>
<p>The additive genetic correlation between two traits was determined based on the genetic covariance matrix of the genomic breeding values of the two traits (Equation <a href="#tpg270066-disp-0009" class="usa-link">9</a>):
</p>
<table class="disp-formula p" id="tpg270066-disp-0009"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6b9c/12207579/d47fc222bea7/TPG2-18-e70066-e019.jpg" loading="lazy" alt="2.5.6."></td>
<td class="label">(9)</td>
</tr></table></section></section></section><section id="tpg270066-sec-0220"><h2 class="pmc_sec_title">3. RESULTS AND DISCUSSION</h2>
<section id="tpg270066-sec-0230"><h3 class="pmc_sec_title">3.1. Phenotypes</h3>
<p>Phenotypes were recorded for a total of 10 agronomic traits: dry matter content, yield, eye depth, flesh color, skin finish, senescence, tuber count (tubers/plant), tuber length, diameter, and length/width ratio (Figure <a href="#tpg270066-fig-0002" class="usa-link">2</a>, Table <a href="#tpg270066-tbl-0001" class="usa-link">1</a>). Phenotyping was performed across two consecutive years for dry matter content and senescence. The initial senescence gradings of 2013 were used to group the plants into three groups according to senescence in the 2014 field layout. Only the 2014 phenotypes were used for modeling senescence because neighbors were not conserved across the years 2013 and 2014, and the full grading scale was not applied in 2013 (Figure <a href="#tpg270066-fig-0002" class="usa-link">2f</a>). In 2013, extreme dry matter contents (&gt;40%) were measured for a small number of cultivars of diverse familial backgrounds and sizes. Some of these cultivars had non‐extreme dry matter content measurements in 2014, while others had been removed from the population due to degeneration (minimum criterion of three tubers/plant each year). Therefore, the extreme 2013 measures were likely a result of measuring error, but in lieu of specific observations that would merit the removal of individual outliers, they were retained in the analysis for dataset completeness.</p>
<figure class="fig xbox font-sm" id="tpg270066-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6b9c/12207579/164450bb1dd3/TPG2-18-e70066-g001.jpg" loading="lazy" id="jats-graphic-3" height="268" width="534" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270066-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of phenotypes in the MASPOT population. (a) Dry matter content, (b) yield, (c) eye depth, (d) flesh color, (e) skin finish, (f) senescence, (g) tubers/plant, (h) length, (i) diameter, and (j) length/width ratio.</p></figcaption></figure><section class="tw xbox font-sm" id="tpg270066-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Statistics describing phenotypic data across years: range, mean ± standard error (SE), phenotypic variance, coefficient of variance (CV), number of observations, and number of replicates (descriptive statistics of the mean corrected phenotypes across years are presented in Table <a href="#tpg270066-supinfo-0001" class="usa-link">S1</a>).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Phenotype</th>
<th align="left" rowspan="1" colspan="1">Range</th>
<th align="left" rowspan="1" colspan="1">Mean (± SE)</th>
<th align="left" rowspan="1" colspan="1">Variance</th>
<th align="left" rowspan="1" colspan="1">CV</th>
<th align="left" rowspan="1" colspan="1">No. of observation</th>
<th align="left" rowspan="1" colspan="1">No. of replications</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Dry matter content (%)</td>
<td align="left" rowspan="1" colspan="1">6.00–65.30</td>
<td align="left" rowspan="1" colspan="1">20.26 ± 0.03</td>
<td align="left" rowspan="1" colspan="1">9.46</td>
<td align="left" rowspan="1" colspan="1">0.15</td>
<td align="left" rowspan="1" colspan="1">13,481</td>
<td align="left" rowspan="1" colspan="1">3</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Yield (hkg/ha)</td>
<td align="left" rowspan="1" colspan="1">17.00–2136.00</td>
<td align="left" rowspan="1" colspan="1">616.71 ± 2.10</td>
<td align="left" rowspan="1" colspan="1">37,619.65</td>
<td align="left" rowspan="1" colspan="1">0.31</td>
<td align="left" rowspan="1" colspan="1">8562</td>
<td align="left" rowspan="1" colspan="1">2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Eye depth (1–6)</td>
<td align="left" rowspan="1" colspan="1">1–5</td>
<td align="left" rowspan="1" colspan="1">2.68 ± 0.01</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
<td align="left" rowspan="1" colspan="1">0.30</td>
<td align="left" rowspan="1" colspan="1">4914</td>
<td align="left" rowspan="1" colspan="1">1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Flesh color (1–9)</td>
<td align="left" rowspan="1" colspan="1">1–8</td>
<td align="left" rowspan="1" colspan="1">5.17 ± 0.02</td>
<td align="left" rowspan="1" colspan="1">2.11</td>
<td align="left" rowspan="1" colspan="1">0.28</td>
<td align="left" rowspan="1" colspan="1">8551</td>
<td align="left" rowspan="1" colspan="1">2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Skin finish (1–6)</td>
<td align="left" rowspan="1" colspan="1">1–5</td>
<td align="left" rowspan="1" colspan="1">2.66 ± 0.01</td>
<td align="left" rowspan="1" colspan="1">0.62</td>
<td align="left" rowspan="1" colspan="1">0.30</td>
<td align="left" rowspan="1" colspan="1">8560</td>
<td align="left" rowspan="1" colspan="1">2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Senescence (1–9)</td>
<td align="left" rowspan="1" colspan="1">1–9</td>
<td align="left" rowspan="1" colspan="1">6.01 ± 0.02</td>
<td align="left" rowspan="1" colspan="1">2.21</td>
<td align="left" rowspan="1" colspan="1">0.25</td>
<td align="left" rowspan="1" colspan="1">8570</td>
<td align="left" rowspan="1" colspan="1">2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Tubers/plant</td>
<td align="left" rowspan="1" colspan="1">1.00–66.00</td>
<td align="left" rowspan="1" colspan="1">16.20 ± 0.06</td>
<td align="left" rowspan="1" colspan="1">33.65</td>
<td align="left" rowspan="1" colspan="1">0.36</td>
<td align="left" rowspan="1" colspan="1">8524</td>
<td align="left" rowspan="1" colspan="1">2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Length (mm)</td>
<td align="left" rowspan="1" colspan="1">27.67–136.80</td>
<td align="left" rowspan="1" colspan="1">65.63 ± 0.12</td>
<td align="left" rowspan="1" colspan="1">119.57</td>
<td align="left" rowspan="1" colspan="1">0.17</td>
<td align="left" rowspan="1" colspan="1">8433</td>
<td align="left" rowspan="1" colspan="1">2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Diameter (mm)</td>
<td align="left" rowspan="1" colspan="1">14.22–74.58</td>
<td align="left" rowspan="1" colspan="1">47.23 ± 0.08</td>
<td align="left" rowspan="1" colspan="1">49.42</td>
<td align="left" rowspan="1" colspan="1">0.15</td>
<td align="left" rowspan="1" colspan="1">8433</td>
<td align="left" rowspan="1" colspan="1">2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Length/width ratio</td>
<td align="left" rowspan="1" colspan="1">1.13–3.76</td>
<td align="left" rowspan="1" colspan="1">1.44 ± 0.00</td>
<td align="left" rowspan="1" colspan="1">0.04</td>
<td align="left" rowspan="1" colspan="1">0.14</td>
<td align="left" rowspan="1" colspan="1">8433</td>
<td align="left" rowspan="1" colspan="1">2</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270066-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270066-tbl1-note-0001"><p>
<em>Note</em>: Eye depth: low = deep, high = shallow; Flesh color: low = white, high = orange; Skin finish: low = rough, high = smooth; Senescence: low = early, high = late.</p></div></div></section><p>The collection of phenotypes includes one ratio trait, length/width ratio, based on two approximately normally distributed traits (length and diameter) (Díaz‐Francés &amp; Rubio, <a href="#tpg270066-bib-0017" class="usa-link" aria-describedby="tpg270066-bib-0017">2013</a>), which was modeled using a normal distribution. Also, for one count trait, tubers/plant, a normal approximation was used to approximate the Poisson distribution due to the high observed lambda parameter (estimated from the mean: <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-35" display="inline"><mrow><mrow><mi mathvariant="normal">λ</mi><mo>≅</mo><mn>16.2</mn></mrow></mrow></math></span>) (Bland, <a href="#tpg270066-bib-0005" class="usa-link" aria-describedby="tpg270066-bib-0005">2000</a>). Additionally, four categorical traits based on either 1–6 or 1–9 manual grading scales (eye depth and skin finish of 1–6 and flesh color and senescence of 1–9, respectively) were also approximated using a normal rather than an ordinal distribution due to the high number of observations. Though some trait distributions were modeled with rather coarse approximations (Figure <a href="#tpg270066-supinfo-0001" class="usa-link">S6</a>), the collection of traits represents a realistic, diverse set of phenotype distributions in applied breeding programs.</p></section><section id="tpg270066-sec-0240"><h3 class="pmc_sec_title">3.2. Fertility and the effect of cytoplasm type on agronomic traits</h3>
<p>While potatoes are hermaphroditic, in principle allowing for bidirectional crosses with each parent, a widespread lack of fertility in several lines is a signature problem in tetraploid potato breeding, and both male and female sterility are issues (Bethke &amp; Jansky, <a href="#tpg270066-bib-0004" class="usa-link" aria-describedby="tpg270066-bib-0004">2021</a>). Infecundity results from myriad causes, most commonly male sterility, for example, due to cytoplasmic male sterility conferred by cytoplasm types W/γ (Lössl et al., <a href="#tpg270066-bib-0038" class="usa-link" aria-describedby="tpg270066-bib-0038">2000</a>) and D (Sanetomo &amp; Gebhardt, <a href="#tpg270066-bib-0057" class="usa-link" aria-describedby="tpg270066-bib-0057">2015</a>) or meiotic irregularities. Female sterility is an issue to a lesser extent (Bethke &amp; Jansky, <a href="#tpg270066-bib-0004" class="usa-link" aria-describedby="tpg270066-bib-0004">2021</a>). Of the 18 MASPOT parents, five were male sterile, and none were completely female sterile. However, a vast portion (∼50%) of specific crosses were infertile, produced degenerate plants, or failed to generate more than three tubers per plant, which is a practical prerequisite for propagation of the model population. The two W/γ cytoplasm‐type cultivars, Kuras and 07‐LJE‐1 were, expectedly, male sterile, but interestingly, none of the five D‐type parent lines were completely male sterile, despite reports of cytoplasm type D conferring functional male sterility (Hosaka &amp; Sanetomo, <a href="#tpg270066-bib-0029" class="usa-link" aria-describedby="tpg270066-bib-0029">2012</a>; Sanetomo &amp; Gebhardt, <a href="#tpg270066-bib-0057" class="usa-link" aria-describedby="tpg270066-bib-0057">2015</a>). This discrepancy has previously been seen empirically (Sanetomo &amp; Gebhardt, <a href="#tpg270066-bib-0057" class="usa-link" aria-describedby="tpg270066-bib-0057">2015</a>), but the current results constitute a systematic validation of the rule. Indeed, Santayana et al. (<a href="#tpg270066-bib-0058" class="usa-link" aria-describedby="tpg270066-bib-0058">2022</a>) showed that combining measures of flowering degree, pollen quantity and viability, and floral abnormalities, rather than solely regarding pollen viability, as is the standard practice, yields a more accurate profile of male fertility. They found that the D cytoplasm genome type performed comparatively better than types T/β and W/γ, with type T/β only being superior in quantities of berries and seeds per berry. This is in accordance with results from our crossing scheme of successful offspring generation in the male cross direction.</p>
<p>The differences between the three cytoplasm types included in the collection of maternal germplasms on trait phenotypes were analyzed by incorporating cytoplasm type as a fixed effect in the prediction models. There could be some confounding between the effect of cytoplasm type and the line effect; however, as the additive and nonadditive genetic contributions are accounted for in the model and the population only includes two reciprocal crosses between parents 04‐GIV‐03 × Rywal and 04‐GIV‐03 × 96‐BYM‐8 (Figure <a href="#tpg270066-fig-0001" class="usa-link">1</a>), we expect the bias to be minimal. Unfortunately, it was not possible to estimate the amount of variation due to interaction between the nuclear and cytoplasmic genomes due to insufficient statistical power.</p>
<p>Significant differences were found for dry matter content, skin finish, and senescence, while other traits were not significantly influenced by cytoplasm type (Table <a href="#tpg270066-tbl-0002" class="usa-link">2</a>). Cytoplasmic gene effects contributed to a 1.45 and 2.24 percentage point increase of dry matter content in W/γ cytoplasm type clones compared to D and T/β types, respectively, corresponding to 7.16 and 11.05% dry matter content reductions in those cytoplasm types relative to the population mean. Skin finish is increased by ∼0.3 points (i.e., smoother tubers) on the 1–6 scale in D and T/β relative to W/γ (10.08%–10.88% change relative to the mean). Only the Chilean <em>S. tuberosum</em>‐derived T/β cytoplasm type (Sanetomo &amp; Gebhardt, <a href="#tpg270066-bib-0057" class="usa-link" aria-describedby="tpg270066-bib-0057">2015</a>) is associated with later senescence (6.60% relative to the mean) compared to the <em>Solanum demissum</em> D‐type and wild species W/γ‐type; however, the 0.4‐point effect on the 1–9 scale trait is associated with a large standard error. The W/γ cytoplasm type has previously been correlated with increased tuber starch content (Lössl et al., <a href="#tpg270066-bib-0038" class="usa-link" aria-describedby="tpg270066-bib-0038">2000</a>) as well as yield, late maturity, and more round tubers (Sanetomo &amp; Gebhardt, <a href="#tpg270066-bib-0057" class="usa-link" aria-describedby="tpg270066-bib-0057">2015</a>). Notably, W/γ shows significantly earlier senescence compared to type T/β, in opposition to previous results (Sanetomo &amp; Gebhardt, <a href="#tpg270066-bib-0057" class="usa-link" aria-describedby="tpg270066-bib-0057">2015</a>). However, due to considerable variation of the senescence phenotypes based on manual grading, conclusions relating to senescence are subject to uncertainty. The previously identified association between cytoplasm type and the yield and tuber shape traits could not be reproduced for either of the tuber yield (total yield, tubers/plant) or tuber size (length, diameter, length/width ratio) traits analyzed here. Previous results were based on a population of established cultivars and breeding clones, which is a potential reason why the W/γ cytoplasm type associations that were not reproducible in this study could be due to the frequency of the cytoplasm types, as having only 18 mothers in the diallel cross limits the power of cytoplasm type effect detection, which can also be seen from the substantial standard errors of some of the fixed effect estimates (Table <a href="#tpg270066-tbl-0002" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="tpg270066-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Estimated contrasts between cytoplasm types on phenotype (± standard error) from the PBLUP<sub>pop</sub> model.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="." span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<thead>
<tr>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" colspan="5" style="border-bottom:solid 1px #000000" rowspan="1">Cytoplasm type</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" colspan="2" style="border-bottom:solid 1px #000000" rowspan="1">T/β (frequency 0.56)</th>
<th align="left" colspan="2" style="border-bottom:solid 1px #000000" rowspan="1">D (frequency 0.25)</th>
<th style="border-bottom:solid 1px #000000" align="left" rowspan="1" colspan="1">W/γ (frequency 0.19)</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">Fixed effect</th>
<th align="left" rowspan="1" colspan="1">% Diff. rel. to mean</th>
<th align="left" rowspan="1" colspan="1">Fixed effect</th>
<th align="left" rowspan="1" colspan="1">% Diff. rel. to mean</th>
<th align="left" rowspan="1" colspan="1">Fixed effect</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Dry matter content</td>
<td align="center" rowspan="1" colspan="1">−2.24 ± 0.62 (*)</td>
<td align="center" rowspan="1" colspan="1">11.05</td>
<td align="center" rowspan="1" colspan="1">−1.45 ± 0.64 (*)</td>
<td align="center" rowspan="1" colspan="1">7.16</td>
<td align="left" rowspan="1" colspan="1">0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Yield</td>
<td align="center" rowspan="1" colspan="1">6.48 ± 29.42</td>
<td align="center" rowspan="1" colspan="1">1.05</td>
<td align="center" rowspan="1" colspan="1">−1.88 ± 31.14</td>
<td align="center" rowspan="1" colspan="1">0.31</td>
<td align="left" rowspan="1" colspan="1">0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Eye depth</td>
<td align="center" rowspan="1" colspan="1">0.10 ± 0.24</td>
<td align="center" rowspan="1" colspan="1">3.56</td>
<td align="center" rowspan="1" colspan="1">−0.26 ± 0.29</td>
<td align="center" rowspan="1" colspan="1">9.85</td>
<td align="left" rowspan="1" colspan="1">0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Flesh color</td>
<td align="center" rowspan="1" colspan="1">0.54 ± 0.39</td>
<td align="center" rowspan="1" colspan="1">10.42</td>
<td align="center" rowspan="1" colspan="1">0.55 ± 0.39</td>
<td align="center" rowspan="1" colspan="1">10.60</td>
<td align="left" rowspan="1" colspan="1">0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Skin finish</td>
<td align="center" rowspan="1" colspan="1">0.29 ± 0.12 (*)</td>
<td align="center" rowspan="1" colspan="1">10.88</td>
<td align="center" rowspan="1" colspan="1">0.27 ± 0.12 (*)</td>
<td align="center" rowspan="1" colspan="1">10.08</td>
<td align="left" rowspan="1" colspan="1">0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Senescence</td>
<td align="center" rowspan="1" colspan="1">0.40 ± 0.18 (*)</td>
<td align="center" rowspan="1" colspan="1">6.60</td>
<td align="center" rowspan="1" colspan="1">0.16 ± 0.19</td>
<td align="center" rowspan="1" colspan="1">2.61</td>
<td align="left" rowspan="1" colspan="1">0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Tubers/plant</td>
<td align="center" rowspan="1" colspan="1">−0.68 ± 1.12</td>
<td align="center" rowspan="1" colspan="1">4.21</td>
<td align="center" rowspan="1" colspan="1">0.25 ± 1.17</td>
<td align="center" rowspan="1" colspan="1">1.54</td>
<td align="left" rowspan="1" colspan="1">0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Length</td>
<td align="center" rowspan="1" colspan="1">1.37 ± 2.93</td>
<td align="center" rowspan="1" colspan="1">2.09</td>
<td align="center" rowspan="1" colspan="1">−0.16 ± 2.97</td>
<td align="center" rowspan="1" colspan="1">0.25</td>
<td align="left" rowspan="1" colspan="1">0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Diameter</td>
<td align="center" rowspan="1" colspan="1">−0.57 ± 1.36</td>
<td align="center" rowspan="1" colspan="1">1.21</td>
<td align="center" rowspan="1" colspan="1">−0.96 ± 1.41</td>
<td align="center" rowspan="1" colspan="1">2.03</td>
<td align="left" rowspan="1" colspan="1">0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Length/width ratio</td>
<td align="center" rowspan="1" colspan="1">0.06 ± 0.05</td>
<td align="center" rowspan="1" colspan="1">4.36</td>
<td align="center" rowspan="1" colspan="1">0.03 ± 0.05</td>
<td align="center" rowspan="1" colspan="1">1.81</td>
<td align="left" rowspan="1" colspan="1">0</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270066-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270066-tbl2-note-0001"><p>
<em>Note</em>: The contrasts are differences relative to the W/γ cytoplasm type. The percentage change relative to the population mean phenotype is shown.</p></div>
<div class="fn" id="tpg270066-tbl2-note-0002"><p>Abbreviations: Diff rel. to mean, difference relative to mean; PBLUP<sub>pop</sub>, pedigree BLUP on the full MASPOT population.</p></div>
<div class="fn" id="tpg270066-tbl2-note-0003"><p>*95% confidence intervals, not including 0.</p></div>
</div></section><p>Despite the desirable effects of the W/γ on the dry matter content phenotype, increased use of W/γ parental lines in breeding programs would increase male sterility levels, thereby constraining future breeding strategies (Provan et al., <a href="#tpg270066-bib-0053" class="usa-link" aria-describedby="tpg270066-bib-0053">1999</a>; Sanetomo &amp; Gebhardt, <a href="#tpg270066-bib-0057" class="usa-link" aria-describedby="tpg270066-bib-0057">2015</a>). In contrast, purging the breeding pool of the table‐potato market segment of roughness‐promoting W/γ cytoplasm types could contribute to improvement in skin smoothness for the European consumer segment, with this skin texture preference (Pandey et al., <a href="#tpg270066-bib-0049" class="usa-link" aria-describedby="tpg270066-bib-0049">2022</a>). The cytoplasm genomic effects on dry matter content and skin finish could be correlated, as a negative additive genetic correlation between the traits indicates that the genomic variation underpinning the traits is pleiotropic, with increasing dry matter contents being associated with rougher‐skinned tubers (see later results).</p>
<p>Due to the incompleteness of the diallel cross, nonadditive contributions inherent to maternal and paternal crossing direction could not be estimated. Gamete‐specific effects could, however, have illuminated whether and how cross‐direction should be prioritized in breeding. The cytoplasm genomic effects indicate a significant maternally associated effect on some traits.</p></section><section id="tpg270066-sec-0250"><h3 class="pmc_sec_title">3.3. Additive genetic effects and specific combining abilities</h3>
<p>Significant additive genetic variance and variance due to SCAs were found for all traits (Table <a href="#tpg270066-tbl-0003" class="usa-link">3</a>). The breeding value of a clone (=2 GCA) includes at most one‐third of the digenic dominance (Gallais, <a href="#tpg270066-bib-0020" class="usa-link" aria-describedby="tpg270066-bib-0020">2003</a>). In this study, we estimate the additive variance based on all additive genetic relationships. Hence, this effect is expected to be reduced, and the contribution of the digenic dominance is expected to be insignificant. As we are accounting for all other relations in addition to half‐siblings, the digenic dominance can, in practice, contribute a maximum of one‐ninth of the digenic dominance variance to the GCA variance, which will be smaller for the employed dataset.</p>
<section class="tw xbox font-sm" id="tpg270066-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>The additive genetic variance (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-36" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>a</mi><mn>2</mn></msubsup></mrow></math></span>) ± standard error (SE), family effect (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-37" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>f</mi><mn>2</mn></msubsup></mrow></math></span>) ± SE, line effects (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-38" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>l</mi><mn>2</mn></msubsup></mrow></math></span>) ± SE, the percentages of phenotypic variance explained by each (VE [%]) for each component, and the SCA/GCA ratio of all phenotypes (*<em>p</em> &lt; 0.05) calculated by PBLUP<sub>pop</sub>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Phenotype</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-39_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-39" display="inline"><mrow><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>±</mo><mi>SE</mi></mrow></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">VE (%) (h<sup>2</sup>)</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-40_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-40" display="inline"><mrow><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>f</mi><mn>2</mn></msubsup><mo>±</mo><mi>SE</mi></mrow></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">VE (%)</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-41_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-41" display="inline"><mrow><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>l</mi><mn>2</mn></msubsup><mo>±</mo><mi>SE</mi></mrow></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">VE (%)</th>
<th align="left" rowspan="1" colspan="1">SCA/GCA</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Dry matter content<a href="#tpg270066-tbl3-note-0003" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="left" rowspan="1" colspan="1">2.42 ± 0.19*</td>
<td align="left" rowspan="1" colspan="1">29.83</td>
<td align="left" rowspan="1" colspan="1">0.38 ± 0.08*</td>
<td align="left" rowspan="1" colspan="1">4.67</td>
<td align="left" rowspan="1" colspan="1">4.61 ± 0.11*</td>
<td align="left" rowspan="1" colspan="1">56.93</td>
<td align="left" rowspan="1" colspan="1">0.63</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Yield</td>
<td align="left" rowspan="1" colspan="1">4712.16 ± 2076.41*</td>
<td align="left" rowspan="1" colspan="1">12.64</td>
<td align="left" rowspan="1" colspan="1">1918.28 ± 380.78*</td>
<td align="left" rowspan="1" colspan="1">5.15</td>
<td align="left" rowspan="1" colspan="1">15,473.31 ± 1208.25*</td>
<td align="left" rowspan="1" colspan="1">41.51</td>
<td align="left" rowspan="1" colspan="1">1.63</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Eye depth</td>
<td align="left" rowspan="1" colspan="1">0.08 ± 0.06</td>
<td align="left" rowspan="1" colspan="1">13.26</td>
<td align="left" rowspan="1" colspan="1">0.05 ± 0.02*</td>
<td align="left" rowspan="1" colspan="1">7.95</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">2.40</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Flesh color</td>
<td align="left" rowspan="1" colspan="1">1.01 ± 0.37*</td>
<td align="left" rowspan="1" colspan="1">49.11</td>
<td align="left" rowspan="1" colspan="1">0.06 ± 0.01*</td>
<td align="left" rowspan="1" colspan="1">2.94</td>
<td align="left" rowspan="1" colspan="1">0.37 ± 0.19</td>
<td align="left" rowspan="1" colspan="1">17.83</td>
<td align="left" rowspan="1" colspan="1">0.24</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Skin finish</td>
<td align="left" rowspan="1" colspan="1">0.09 ± 0.03*</td>
<td align="left" rowspan="1" colspan="1">15.09</td>
<td align="left" rowspan="1" colspan="1">0.01 ± 0.004*</td>
<td align="left" rowspan="1" colspan="1">2.58</td>
<td align="left" rowspan="1" colspan="1">0.18 ± 0.02</td>
<td align="left" rowspan="1" colspan="1">31.22</td>
<td align="left" rowspan="1" colspan="1">0.68</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Senescence</td>
<td align="left" rowspan="1" colspan="1">0.19 ± 0.08*</td>
<td align="left" rowspan="1" colspan="1">9.12</td>
<td align="left" rowspan="1" colspan="1">0.06 ± 0.01*</td>
<td align="left" rowspan="1" colspan="1">2.76</td>
<td align="left" rowspan="1" colspan="1">1.12 ± 0.05*</td>
<td align="left" rowspan="1" colspan="1">54.66</td>
<td align="left" rowspan="1" colspan="1">1.22</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Tubers/plant</td>
<td align="left" rowspan="1" colspan="1">7.47 ± 3.03*</td>
<td align="left" rowspan="1" colspan="1">22.00</td>
<td align="left" rowspan="1" colspan="1">1.94 ± 0.37*</td>
<td align="left" rowspan="1" colspan="1">5.71</td>
<td align="left" rowspan="1" colspan="1">14.21 ± 1.61*</td>
<td align="left" rowspan="1" colspan="1">41.85</td>
<td align="left" rowspan="1" colspan="1">1.04</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Length</td>
<td align="left" rowspan="1" colspan="1">58.41 ± 21.78*</td>
<td align="left" rowspan="1" colspan="1">54.07</td>
<td align="left" rowspan="1" colspan="1">2.47 ± 0.60*</td>
<td align="left" rowspan="1" colspan="1">2.28</td>
<td align="left" rowspan="1" colspan="1">21.09 ± 10.99</td>
<td align="left" rowspan="1" colspan="1">19.52</td>
<td align="left" rowspan="1" colspan="1">0.17</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Diameter</td>
<td align="left" rowspan="1" colspan="1">11.12 ± 4.56*</td>
<td align="left" rowspan="1" colspan="1">23.10</td>
<td align="left" rowspan="1" colspan="1">2.64 ± 0.51*</td>
<td align="left" rowspan="1" colspan="1">5.49</td>
<td align="left" rowspan="1" colspan="1">22.24 ± 2.40*</td>
<td align="left" rowspan="1" colspan="1">46.22</td>
<td align="left" rowspan="1" colspan="1">0.95</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Length/width ratio</td>
<td align="left" rowspan="1" colspan="1">0.013 ± 0.005*</td>
<td align="left" rowspan="1" colspan="1">33.58</td>
<td align="left" rowspan="1" colspan="1">0.003 ± 0.0005*</td>
<td align="left" rowspan="1" colspan="1">7.50</td>
<td align="left" rowspan="1" colspan="1">0.016 ± 0.003*</td>
<td align="left" rowspan="1" colspan="1">41.90</td>
<td align="left" rowspan="1" colspan="1">0.89</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270066-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270066-tbl3-note-0001"><p>
<em>Note</em>: For the additive variance explained, the VE was computed as the product of the mean diagonal elements of the covariance matrix (1.001) and the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-42" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>a</mi><mn>2</mn></msubsup></mrow></math></span> to account for the scaling of the covariance matrix. The percentage of explained phenotypic variance corresponds to the plot narrow‐sense heritability (<em>h</em>
<sup>2</sup>).</p></div>
<div class="fn" id="tpg270066-tbl3-note-0002"><p>Abbreviations: GCA, general combining ability; PBLUP<sub>pop</sub>, pedigree BLUP on the full MASPOT population; SCA, specific combining ability.</p></div>
<div class="fn" id="tpg270066-tbl3-note-0003">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Line effect nested within year of test.</p>
</div>
</div></section><p>The proportion of total genetic variation attributable to additive genetic effects and SCA effects varied across traits, and the predominance of additive versus nonadditive effects was trait‐dependent. Additionally, line effects were considerable for most traits, explaining 18%–57% of the total phenotypic variance. The line effects describe the variance associated with multiple observations of each MASPOT clone. It captures, for example, genetic interactions not captured by the additive and family‐based nonadditive components, as well as line‐specific nonadditive genetic variance. Like the family‐based SCA, the line‐specific nonadditive contributions are non‐transmissible over sexual generations due to recombination and the independent assortment of chromosomes. Particularly, the traits of dry matter content and senescence seem to be associated with strong line‐specific effects of 57% and 55%, respectively. However, a proportion of this variation captured in the line effect is likely also related to the extreme dry matter recordings of a few clones in 2013; that is, some experimental variability is captured in this component. Similarly, the manually graded senescence phenotypes are also prone to inconsistency in the application of the grading scale between graders (Danespo A/S, personal communication, 2024). Overall, the low number of replicates hampers the rigorous estimation of the line effect for individual clones.</p>
<p>High‐heritability traits, such as tuber length and flesh color (Table <a href="#tpg270066-tbl-0003" class="usa-link">3</a>), were found. Interestingly, dry matter content, a known high‐heritability trait (Ortiz et al., <a href="#tpg270066-bib-0047" class="usa-link" aria-describedby="tpg270066-bib-0047">2023</a>), has a narrow‐sense design heritability in the low range (0.31). The narrow‐sense heritability estimated from the regression between the mean corrected phenotype and the mid‐parent phenotype results in a 0.88 ± 0.02 dry matter heritability, consistent with the high heritability expectation for dry matter content. This deflation of the genomic heritability compared to the regression‐based heritability for dry matter content has been encountered in prior studies on the MASPOT panel using standard additive GBLUP (Aalborg et al., <a href="#tpg270066-bib-0002" class="usa-link" aria-describedby="tpg270066-bib-0002">2024</a>; Sverrisdóttir et al., <a href="#tpg270066-bib-0065" class="usa-link" aria-describedby="tpg270066-bib-0065">2017</a>), indicating that some genetic variance is not captured in the additive variance component. However, the specific reason remains obscure. Based on our models, a great portion of the phenotypic variance is captured in the nested line effect, constituting 57% of the total phenotypic variance. This is the highest recorded line effect. While the reason for the heritability‐related discrepancy is obscure, prior genomic prediction models still performed well relative to the regression‐based narrow‐sense heritability and presented with high prediction correlation coefficients (Aalborg et al., <a href="#tpg270066-bib-0002" class="usa-link" aria-describedby="tpg270066-bib-0002">2024</a>; Sverrisdóttir et al., <a href="#tpg270066-bib-0065" class="usa-link" aria-describedby="tpg270066-bib-0065">2017</a>), indicating that EBVs still correlate well with observed performance—the same is also observed in this study (Figure <a href="#tpg270066-fig-0003" class="usa-link">3</a>). Furthermore, the obtained prediction correlation coefficient of 0.62 using PBLUP<sub>pop</sub> exceeds the variance explained of the clone means of 0.56 (determined as the square root of the observed design heritability of 0.31). Overall, the deflated additive genetic variance compared to expectations does not seem to affect the EBVs. Dry matter content still presented with greater additive variance (30%) than yield (13%), a twofold difference also found in a previous diallel study (Bradshaw et al., <a href="#tpg270066-bib-0009" class="usa-link" aria-describedby="tpg270066-bib-0009">2000</a>). However, other results of a high specific gravity‐selected population of breeding clones indicated that additive genetic variance of yield exceeded that of specific gravity (closely correlated with dry matter content) (Endelman et al., <a href="#tpg270066-bib-0018" class="usa-link" aria-describedby="tpg270066-bib-0018">2018</a>), highlighting the population dependence of SCA/GCA estimates (Bradshaw, <a href="#tpg270066-bib-0008" class="usa-link" aria-describedby="tpg270066-bib-0008">2022</a>).</p>
<figure class="fig xbox font-sm" id="tpg270066-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6b9c/12207579/2dbce5bd913d/TPG2-18-e70066-g002.jpg" loading="lazy" id="jats-graphic-5" height="593" width="800" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270066-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Additive genetic breeding values of MASPOT parents in the unit of the respective phenotypes (with 95% confidence intervals shown as black whiskers) based on single‐step GBLUP (ssGBLUP). EBV, estimated breeding value; GBLUP, best linear unbiased prediction.</p></figcaption></figure><p>Senescence is influenced by a major quantitative trait locus (QTL), as alleles of the chromosome 5 <em>StCDF1</em> gene encoding truncated protein variants result in early maturity by affecting mediation between the circadian clock and the tuberization signal (Kloosterman et al., <a href="#tpg270066-bib-0031" class="usa-link" aria-describedby="tpg270066-bib-0031">2013</a>). Estimated additive genetic variance and SCA genetic variance only constituted &lt;15% of the total phenotypic variance. This, despite previous additive models on the genotyped MASPOT panel aptly capturing the <em>StCDF1</em> QTL in genome‐wide association studies (Aalborg et al., <a href="#tpg270066-bib-0002" class="usa-link" aria-describedby="tpg270066-bib-0002">2024</a>). Fifty‐five percent of the total variation is attributable to line effects. This possibly reflects the phenotyping method being subject to some inconsistency in scoring, affecting trait data quality. However, the results indicate that all replicates are affected. Eye depth, another major QTL trait, performs similarly, with only 21% variation captured in variance due to additive genetic effects and SCA, and had the highest recorded SCA/GCA ratio of 2.40, comparatively greater than the 1.63 ratio of the complex, polygenic trait yield. Moreover, the additive genetic variance of eye depth is not statistically significant. The known major eye depth QTL, <em>Eyd</em>, is located on chromosome 10 in close linkage with the tuber shape <em>R</em>
<sub>0</sub> QTL (4 cM), deep eyes being dominant (X. Q. Li et al., <a href="#tpg270066-bib-0036" class="usa-link" aria-describedby="tpg270066-bib-0036">2005</a>; Śliwka et al., <a href="#tpg270066-bib-0061" class="usa-link" aria-describedby="tpg270066-bib-0061">2008</a>). Skin finish shows an additive component (15%), but a sizable proportion of the total variation was captured in the residuals (&gt;50%). Generally, the manually graded phenotypes assessed for the population, apart from flesh color, present substantial line effects and variation not separable from the residuals using the models defined.</p>
<p>Our results indicate that several agronomic traits are mainly governed by additive genetic variance based on SCA/GCA ratios and show good potential for genetic change in the population by breeding. However, we estimate substantial line effects that will likely also include some dominance and epistatic effects not associated with the specific parents, which could indicate that the SCA contribution of some traits is underestimated. Traits relevant to total production quantities, that is, yield, tuber count, and senescence (as a longer growth period is generally associated with increased output), showed higher SCA variance than GCA variance (additive parental contributions), suggesting that transmission of these traits is most influenced by nonadditive effects. However, based on the total additive genetic variance, rather than the GCA variance, all traits were characterized by higher additive genetic variance compared to nonadditive genetic variance, with reservation for possible additional nonadditive variance captured in the line effect. As potato breeding is based on the crossing of selected parent combinations within the breeding population, the total additive genetic variance in the breeding population is important, as it affects the possible genetic gain attainable in progeny generations.</p>
<p>The vegetative propagation of clones that facilitates infinite lifespan entails that the clone is fixed after the first tuber is recovered from sowing the seed. Therefore, the predictions can be used for two purposes: (1) selection of new clones directly for the markets and (2) selection of parents, which can be used for further breeding with very intense selection if we can predict breeding values much earlier in the potato breeding programs. The parent additive genetic value (EBV = 2 GCA) should mainly be used for parental selection, even though some dominance effects are transmittable to progeny (Endelman et al., <a href="#tpg270066-bib-0018" class="usa-link" aria-describedby="tpg270066-bib-0018">2018</a>), and total genetic value can be used for direct selection of new market clones. The total genetic value of a clone will be the sum of the additive and nonadditive genetic effects. Our results regarding the predictions of breeding values of the MASPOT parents across 10 agronomic traits reveal that the additive transmission potential varies greatly across cultivars, both in terms of the number and nature of traits that are effectively transmitted by each parent as well as the transmission ability of each parent (Figure <a href="#tpg270066-fig-0003" class="usa-link">3</a>, Figure <a href="#tpg270066-supinfo-0001" class="usa-link">S7</a>). In addition, variation is also considerable in this experiment because the population is defined by sampling parents from different market segments.</p>
<p>While only some traits were influenced most by nonadditive genetic effects based on SCA/GCA ratio, all traits relied on significant nonadditive contributions to the ultimate phenotypic profile. Nonadditive effects are not effectively transmitted to future generations due to recombination and independent assortment of chromosomes. The SCAs of each family (Figures <a href="#tpg270066-supinfo-0001" class="usa-link">S8–S17</a>) show that only a small subset of families successfully transmits this (ranging between 1 and 12 out of 119 families, depending on the trait). Profiling of SCA is hence a valuable tool in breeding for characterizing cultivar potential. While commercial selection of cultivars for market segments relies on total genetic effects as the vegetative propagation of commercial clones allows a line with an attractive total genetic value to have indefinite life, breeding selection should rely solely on the EBV for parent selection. This distinction is vital, as erroneously attributing trait genetic variation to additive genetic effects rather than SCA results in a notable overestimation of parent potential, and ignoring nonadditive genetic effects might also bias the estimates of additive effects. Therefore, it is highly important to separate the additive and nonadditive genetic effects. As SCA is not accurately proxied by any direct measure, this profiling constitutes an important breeding tool, despite only applying to existing crosses. Testing for market development (clone total genetic value) would be based on additive genetic contribution and SCA effects and would also allow the identification of clones with positive deviations in the genetic part of the line effect, which could contribute to improved total genetic value for a trait relative to the market segment breeding goal. While accurate estimation of total genetic value is important for market clone selection, the overall trait performance of a clone should not be used in parent selection. This should be based on predicted breeding value (additive genetic contribution). A case example of this is the dry matter transmission ability of the starch segment cultivar Kuras. In Denmark, it has historically been a highly popular starch cultivar and consequently has been used in multiple crosses in breeding programs to promote starch content in the offspring. However, our results indicate that the superior starch content in Kuras is not rooted in additive effects but rather nonadditive effects. This is reflected in the low 0.05% dry matter content breeding value of Kuras (prediction error variance [PEV] of 1.31 of the PBLUP<sub>pop</sub> model for dry matter content). Based on our results, Kuras does not have merit as a dry matter content‐transmitting parent, compared to, for example, Aventra (T/β) with an EBV of 3.44% (PEV of 0.31), besides the increased dry matter content conferred by the undesirable W/γ cytoplasm type. This hypothesis can be validated using historic phenotyping data of, for example, Kuras progeny and Aventra progeny to evaluate dry matter content genetic gain in either parental lineage.</p></section><section id="tpg270066-sec-0260"><h3 class="pmc_sec_title">3.4. The value of using genomic information in genomic prediction</h3>
<p>Generating prediction models trained on phenotypic data alone allows for their implementation in other cultivars in the breeding program based on historical data but requires full phenotyping of each generation before selection. Using pedigree and/or genomic information and subsequent prediction allows for early selection without phenotyping, except for a subset of the breeding pool for model retraining, allowing improved genetic gain in a reduced number of breeding cycles (Lenaerts et al., <a href="#tpg270066-bib-0034" class="usa-link" aria-describedby="tpg270066-bib-0034">2019</a>; Naeem et al., <a href="#tpg270066-bib-0043" class="usa-link" aria-describedby="tpg270066-bib-0043">2021</a>; Ortiz, <a href="#tpg270066-bib-0045" class="usa-link" aria-describedby="tpg270066-bib-0045">2020</a>). We wished to estimate the value of including genotypic data compared to pedigree data on prediction model performance.</p>
<p>All models were validated by 30 repeats of random eightfold cross‐validation, and their performances were evaluated (Figure <a href="#tpg270066-fig-0004" class="usa-link">4</a>, Table <a href="#tpg270066-tbl-0004" class="usa-link">4</a>, Figure <a href="#tpg270066-supinfo-0001" class="usa-link">S18</a>, Tables <a href="#tpg270066-supinfo-0001" class="usa-link">S2–S4</a>). As prediction correlation varied by trait, the value of genomic information for improving model performance also varied. In general, increasing the population size by more than a factor of six from PBLUP<sub>panel</sub> to PBLUP<sub>pop</sub> improved the accuracy of PBLUP and, for some traits, significantly improved prediction, for example, for yield, skin finish, senescence, length, and length/width ratio. Including genotypes of the MASPOT panel, a 15% subset of the full population, in an ssGBLUP model resulted in minor variations in correlation coefficients compared to PBLUP<sub>pop</sub>, indicating that pedigree information governs performance at this proportion of genotyped clones. Comparing the performance based on the genotyped versus the non‐genotyped clones in ssGBLUP alludes to the value of adding genotypes. The genotyped clones are associated with greater variation in correlation coefficients between repeats, likely due to fewer individuals in the genotyped panel, but for dry matter content, eye depth, flesh color, tubers/plant, diameter, and length/width ratio, adding genotypes notably improves prediction correlation coefficients. In comparison, using standard additive GBLUP without pedigree information, only on the genotyped MASPOT panel, gave superior models for some traits, particularly for the high heritability trait flesh color and the expected high heritability trait dry matter content, but also for the low (observed) heritability traits of eye depth, senescence, and diameter—this is unexpected. Genomic information can capture cryptic relationships in a population and is generally most advantageous for low‐heritability traits. Furthermore, in this particular population, the genetic structure is very well‐defined, and the effect of each parent is well predicted due to the large progeny groups. This should allow a good prediction of the true breeding value of the parents. However, our results would indicate that the genomic data contributes additional information, and genotyping a higher proportion of the population might contribute to further improvement of prediction accuracies of these high‐heritability traits. Analyzing ssGBLUP models with a stepwise increase of the proportion of individuals genotyped indeed shows that for dry matter content prediction, correlation coefficients improve with the proportion genotyped, and based on the observed trend, a plateau in performance is not yet reached at the currently applied 15% genotyped (Figure <a href="#tpg270066-supinfo-0001" class="usa-link">S19</a>). Comparatively, an effect of the proportion genotyped is not observed for yield, which is consistent with the similar performance of the PBLUP<sub>pop</sub> and GBLUP models for this trait. Incorporating genetic information ostensibly improves prediction of the major QTL traits such as eye depth, senescence, and flesh color by including the genetic variation underpinning the bulk of the trait phenotypic variance. A similar effect is also observed for the tuber size and shape measures.</p>
<figure class="fig xbox font-sm" id="tpg270066-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6b9c/12207579/49d1717daf8e/TPG2-18-e70066-g005.jpg" loading="lazy" id="jats-graphic-7" height="387" width="800" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270066-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Boxplots of Pearson prediction correlation coefficients [<em>r</em>(<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-43" display="inline"><mrow><mover><msub><mi>y</mi><mi>c</mi></msub><mo>¯</mo></mover></mrow></math></span>,<em>â<sub>R</sub>
</em>)] between mean corrected phenotypes (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-44" display="inline"><mrow><mover><msub><mi>y</mi><mi>c</mi></msub><mo>¯</mo></mover></mrow></math></span>) and estimated breeding values (<em>â<sub>R</sub>
</em>) for 30 repeats of random eightfold cross‐validation for the four screened models: pedigree BLUP for the full MASPOT population (PBLUP<sub>pop</sub>), pedigree BLUP for the MASPOT panel (PBLUP<sub>panel</sub>), genomic BLUP (GBLUP), and single‐step GBLUP (ssGBLUP). Correlation coefficients of the ssGBLUP were also calculated based on only the genotyped clones (panel) and only the non‐genotyped clones (ped) to illustrate the contribution of genomic information to prediction power. BLUP, best linear unbiased prediction.</p></figcaption></figure><section class="tw xbox font-sm" id="tpg270066-tbl-0004"><h4 class="obj_head">TABLE 4.</h4>
<div class="caption p"><p>Mean Pearson prediction correlation coefficient between EBV (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-45" display="inline"><mrow><mover accent="true"><msub><mi>a</mi><mi>R</mi></msub><mo>^</mo></mover></mrow></math></span>) in the test set and the mean corrected phenotype (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-46" display="inline"><mrow><mover><msub><mi>y</mi><mi>c</mi></msub><mo>¯</mo></mover></mrow></math></span>) of 30 repeats of random eightfold cross‐validation ± standard deviation (SD), dispersion bias, plot narrow‐sense heritability (<em>h</em>
<sup>2</sup>), design narrow‐sense heritability (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-47" display="inline"><mrow><msubsup><mi>h</mi><mover><msub><mi>y</mi><mi>c</mi></msub><mo>¯</mo></mover><mn>2</mn></msubsup></mrow></math></span>), and prediction accuracy based on the ssGBLUP model.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">
<em>r</em>(<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-48" display="inline"><mrow><mover><msub><mi>y</mi><mi>c</mi></msub><mo>¯</mo></mover></mrow></math></span>
<em>, â<sub>R</sub>
</em>) ± SD</th>
<th align="left" rowspan="1" colspan="1">Dispersion (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-49" display="inline"><mrow><mover><msub><mi mathvariant="normal">y</mi><mi mathvariant="normal">c</mi></msub><mo>¯</mo></mover></mrow></math></span>, <em>â<sub>R</sub>
</em>)</th>
<th align="left" rowspan="1" colspan="1">
<em>h</em>
<sup>2</sup>
</th>
<th align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-50_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-50" display="inline"><mrow><msubsup><mi>h</mi><mover><msub><mi>y</mi><mi>c</mi></msub><mo>¯</mo></mover><mn>2</mn></msubsup></mrow></math></span></td></tr></table>
</th>
<th align="left" rowspan="1" colspan="1">Prediction accuracy <em>r</em>(<em>a â<sub>R</sub>
</em>)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Dry matter content</td>
<td align="left" rowspan="1" colspan="1">0.61 ± 0.005</td>
<td align="left" rowspan="1" colspan="1">1.15</td>
<td align="left" rowspan="1" colspan="1">0.29</td>
<td align="left" rowspan="1" colspan="1">0.31</td>
<td align="left" rowspan="1" colspan="1">1.09</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Yield</td>
<td align="left" rowspan="1" colspan="1">0.24 ± 0.002</td>
<td align="left" rowspan="1" colspan="1">1.35</td>
<td align="left" rowspan="1" colspan="1">0.15</td>
<td align="left" rowspan="1" colspan="1">0.19</td>
<td align="left" rowspan="1" colspan="1">0.56</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Eye depth</td>
<td align="left" rowspan="1" colspan="1">0.18 ± 0.005</td>
<td align="left" rowspan="1" colspan="1">1.43</td>
<td align="left" rowspan="1" colspan="1">0.18</td>
<td align="left" rowspan="1" colspan="1">0.18</td>
<td align="left" rowspan="1" colspan="1">0.43</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Flesh color</td>
<td align="left" rowspan="1" colspan="1">0.52 ± 0.001</td>
<td align="left" rowspan="1" colspan="1">1.07</td>
<td align="left" rowspan="1" colspan="1">0.58</td>
<td align="left" rowspan="1" colspan="1">0.68</td>
<td align="left" rowspan="1" colspan="1">0.63</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Skin finish</td>
<td align="left" rowspan="1" colspan="1">0.29 ± 0.003</td>
<td align="left" rowspan="1" colspan="1">1.11</td>
<td align="left" rowspan="1" colspan="1">0.21</td>
<td align="left" rowspan="1" colspan="1">0.28</td>
<td align="left" rowspan="1" colspan="1">0.55</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Senescence</td>
<td align="left" rowspan="1" colspan="1">0.23 ± 0.003</td>
<td align="left" rowspan="1" colspan="1">1.15</td>
<td align="left" rowspan="1" colspan="1">0.16</td>
<td align="left" rowspan="1" colspan="1">0.19</td>
<td align="left" rowspan="1" colspan="1">0.52</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Tubers/plant</td>
<td align="left" rowspan="1" colspan="1">0.33 ± 0.002</td>
<td align="left" rowspan="1" colspan="1">1.16</td>
<td align="left" rowspan="1" colspan="1">0.28</td>
<td align="left" rowspan="1" colspan="1">0.33</td>
<td align="left" rowspan="1" colspan="1">0.57</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Length</td>
<td align="left" rowspan="1" colspan="1">0.49 ± 0.002</td>
<td align="left" rowspan="1" colspan="1">1.07</td>
<td align="left" rowspan="1" colspan="1">0.47</td>
<td align="left" rowspan="1" colspan="1">0.54</td>
<td align="left" rowspan="1" colspan="1">0.67</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Diameter</td>
<td align="left" rowspan="1" colspan="1">0.33 ± 0.002</td>
<td align="left" rowspan="1" colspan="1">1.18</td>
<td align="left" rowspan="1" colspan="1">0.34</td>
<td align="left" rowspan="1" colspan="1">0.38</td>
<td align="left" rowspan="1" colspan="1">0.53</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Length/width ratio</td>
<td align="left" rowspan="1" colspan="1">0.42 ± 0.002</td>
<td align="left" rowspan="1" colspan="1">1.16</td>
<td align="left" rowspan="1" colspan="1">0.45</td>
<td align="left" rowspan="1" colspan="1">0.49</td>
<td align="left" rowspan="1" colspan="1">0.60</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270066-tbl-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270066-tbl4-note-0001"><p>
<em>Note</em>: The prediction accuracy is the correlation between the true breeding value (a) and the EBV.</p></div>
<div class="fn" id="tpg270066-tbl4-note-0002"><p>Abbreviations: EBV, estimated breeding value; GBLUP, best linear unbiased prediction; ssGBLUP, single‐step GBLUP.</p></div>
</div></section><p>Several traits were poorly predicted using either model, namely yield, skin finish, and the major QTL traits senescence and eye depth, with prediction correlations below or close to 0.3 and well below the within‐population estimation of the variance explained based on the square root of the observed heritability. Yield is generally recognized as a complex, highly polygenic trait (van Eck, <a href="#tpg270066-bib-0072" class="usa-link" aria-describedby="tpg270066-bib-0072">2007</a>), and the observed accuracy is consistent with several previous studies (Aalborg et al., <a href="#tpg270066-bib-0002" class="usa-link" aria-describedby="tpg270066-bib-0002">2024</a>; Ortiz, <a href="#tpg270066-bib-0045" class="usa-link" aria-describedby="tpg270066-bib-0045">2020</a>). The PA of each single‐trait ssGBLUP model (Table <a href="#tpg270066-tbl-0004" class="usa-link">4</a>), that is, the ability to predict the breeding value of future clones before they are phenotyped, shows that generally across most traits, prediction accuracies of over 0.52 are achieved, particularly flesh color and length perform well with prediction accuracies above 0.63. An exception to this is eye depth with a lower accuracy of 0.43. Also, the unexpectedly low heritability of dry matter content compared to the high prediction correlation of 0.61 also results in an unrealistic PA of 1.09. While tuber diameter presents with a PA of 0.53, this reflects a relatively low prediction correlation relative to an intermediate heritability. Tuber diameter likely increases to some extent as a function of tuber length, controlled by the dominant <em>R</em>
<sub>0</sub> locus, but these results indicate that the genetic inheritance of tuber width is more complexly governed.</p>
<p>The poor prediction correlation of tuber count might reflect the mortality and tuberization selection applied in the MASPOT diallel cross. Solely disqualifying plants based on mortality and tuberization (three per plant) leaves degenerate plants in the population, where tuber growth might be inhibited by overall plant vitality rather than genetic potential. In this scenario, trait performance instead reflects an overall bottleneck. This could be underlying the low plot heritability of tuber count of 0.28 with the ssGBLUP model. Yield could be expected to be subject to similar effects. This is an important consideration of genomic prediction in breeding programs, where selection increases progressively with breeding cycles, and models should be retrained accordingly, so that early models are trained based on populations that include degenerates, and late models are specialized to higher performing individuals with extensive prior selection.</p>
<p>Reviewing EBVs of dry matter content for the 18 MASPOT parent lines (Figure <a href="#tpg270066-fig-0005" class="usa-link">5</a>), the predicted estimates are subject to greater uncertainty in the GBLUP and PBLUP<sub>panel</sub> models with fewer clones due to greater inaccuracy as a result of fewer progeny sampled per parental line. Increasing the population size to ∼5000 clones mildly influenced pedigree EBV values. Also, the inclusion of genomic information in ssGBLUP reduced the standard error of prediction, generating more confident and generally more conservative EBV predictions. These overall trends are observed for all analyzed traits (Figures <a href="#tpg270066-supinfo-0001" class="usa-link">S20–S28</a>). The more conservative EBV predictions for ssGBLUP are unexpected, as adding genomic information normally amplifies EBV predictions, but the response observed is likely a result of the specific diallel experimental design.</p>
<figure class="fig xbox font-sm" id="tpg270066-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6b9c/12207579/b0aaf9966373/TPG2-18-e70066-g004.jpg" loading="lazy" id="jats-graphic-9" height="193" width="800" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270066-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Additive genetic breeding values of MASPOT parents for dry matter content (%) (with 95% confidence interval shown as error bars) predicted with genomic BLUP (GBLUP), PBLUP<sub>panel</sub>, PBLUP<sub>pop</sub>, and single‐step GBLUP (ssGBLUP). BLUP, best linear unbiased prediction; EBV, estimated breeding value; PBLUP, pedigree BLUP.</p></figcaption></figure><p>Our results show that including genotype information in ssGBLUP models has the potential to enhance PA compared to traditional PBLUP, but that the proportion of genotyped clones in the population (15%) was too small to influence overall model performance. Increasing the fraction of individuals genotyped is expected to increase ssGBLUP prediction accuracies (Song et al., <a href="#tpg270066-bib-0062" class="usa-link" aria-describedby="tpg270066-bib-0062">2019</a>), and a similar influence of population size is expected for GBLUP, as this would increase the portion of total genetic variance explained by genotype. Accounting for more individual variation by increasing genotyped fraction will likely also generate more accurate, and hypothetically also more conservative, predictions of breeding values when using ssGBLUP. To extrapolate upon this, we argue that genotyping efforts should be aimed at increasing the size of the genotyped panels to improve model performance. Additionally, extending the panel diversity will also improve the predictive ability for future extra‐population clones (Sverrisdóttir et al., <a href="#tpg270066-bib-0066" class="usa-link" aria-describedby="tpg270066-bib-0066">2018</a>). Comparatively, the value of intensifying or maintaining phenotyping efforts beyond what is needed for generating a retraining dataset for genomic prediction models relative to the cost and additional time required before possible selection is doubtful. Based on our results, the additive genetic variance was sufficiently captured with a 755‐clone population in PBLUP, as EBV predictions changed only slightly with a factor of more than a sixfold increase in phenotyped population, indicating that the genetic variance contained in the pedigree of this 18‐parent diallel is widely accounted for with a smaller population. However, for application in a general breeding program, where expanding the population size involves increasing the number of parent combinations, a comparatively larger population will likely also be required to capture the additive genetic variance. By increasing population size, a more precise estimate of the genetic variance can be obtained, and adding genetic information about new families of the diallel crossing design in addition to those already covered in the incomplete diallel would be further advantageous.</p>
<p>In conclusion, the value of adding genotypic data is context dependent, as can be observed from the prediction of EBVs in GBLUP compared to PBLUP and ssGBLUP. The mean additive contribution of a parent to a population of offspring is proxied by the mid‐parent value and contained in the pedigree, whereas genotypes are useful in describing the individual clone's deviation from the mean parental additive genetic value. The differing levels of information carried are apparent from the large uncertainty associated with the GBLUP additive genetic breeding value predictions of the parents compared to the pedigree‐based predictions. This is contrasted by the performance boost of the panel‐wide prediction correlation of some traits using a GBLUP compared to the PBLUP model.</p></section><section id="tpg270066-sec-0270"><h3 class="pmc_sec_title">3.5. Genetic correlations</h3>
<p>The phenotypic and additive genetic correlations between all pairwise trait combinations were evaluated by fitting bivariate PBLUP<sub>pop</sub> models (Figure <a href="#tpg270066-fig-0006" class="usa-link">6</a>). Overall, while few trait pairs showed strong phenotypic correlations, the statistically significant genetically correlated traits presented with greater absolute values, but consistently same direction, correlations as compared to the phenotypic. The higher the heritability of the traits involved in the correlation, the more the genetic parts also influence the phenotypic correlation. Dry matter content and eye depth were genetically correlated with most other traits analyzed. The <em>Eyd</em> locus for eye depth is genetically linked to the tuber shape <em>R</em>
<sub>0</sub> locus, controlled by the <em>StOFP20</em> gene (S. Wu, Zhang, et al., <a href="#tpg270066-bib-0077" class="usa-link" aria-describedby="tpg270066-bib-0077">2018</a>), with a distance of only 4 cM; rounder tubers being associated with deeper eyes (X. Q. Li et al., <a href="#tpg270066-bib-0036" class="usa-link" aria-describedby="tpg270066-bib-0036">2005</a>; Śliwka et al., <a href="#tpg270066-bib-0061" class="usa-link" aria-describedby="tpg270066-bib-0061">2008</a>), which was replicated here (0.42). In relation to this, the increased tuber count is also correlated with shallow eyes (0.38). As tuber count is inversely correlated with tuber size measures, both length (−0.84) and diameter (−0.55), this shows a propensity toward a tuberization dichotomy of generating either few, large tubers or a manifold of small tubers to store the assimilated carbon. The shallow eyes of tuber‐rich clones are likely a result of the small tubers (pleiotropy).</p>
<figure class="fig xbox font-sm" id="tpg270066-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6b9c/12207579/507a70f155b6/TPG2-18-e70066-g006.jpg" loading="lazy" id="jats-graphic-11" height="360" width="800" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270066-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Pairwise (a) phenotypic and (b) additive genetic correlations between traits with 95% confidence intervals based on PBLUP<sub>pop</sub>. (*denotes confidence intervals not crossing 0, and correlations with confidence intervals including 0 are shown in gray). Phenotypic correlations could not be computed for trait combinations including eye depth other than dry matter content, as eye depth was recorded without replicates in 2014, and a residual covariance could not be computed.</p></figcaption></figure><p>Similar results were also found for dry matter, with genetic correlations indicating increased dry matter content in smaller, oblong tubers. Oblong tuber shape has previously been linked to starch content (Urbany et al., <a href="#tpg270066-bib-0069" class="usa-link" aria-describedby="tpg270066-bib-0069">2011</a>), possibly a result of several starch metabolism genes coinhabiting the chromosome 10 South arm along with the <em>R</em>
<sub>0</sub> locus (Schreiber et al., <a href="#tpg270066-bib-0079" class="usa-link" aria-describedby="tpg270066-bib-0079">2014</a>). Dry matter content had a strong negative genetic correlation with total yield (−0.49), despite not presenting with a notable phenotypic correlation between the traits (−0.08). Other studies have also found negative genetic correlations of −0.18 and −0.24 between starch content and tuber yield (L. Li et al., <a href="#tpg270066-bib-0035" class="usa-link" aria-describedby="tpg270066-bib-0035">2013</a>; Urbany et al., <a href="#tpg270066-bib-0069" class="usa-link" aria-describedby="tpg270066-bib-0069">2011</a>), which are comparably weaker than the −0.49 reported here. This might reflect the inclusion of both starch and protein content in the joint dry matter content phenotype. As an enhancement of yield entails a reduction in dry matter content, the gained tuber weight likely arises from surged water contents. Genetic variation in the starch metabolism genes could result in disparate enzymatic profiles that catalyze the formation of distinct starch structures in low‐ and high‐starch tubers, where more heavily phosphorylated polymers and/or increased amylose/amylopectin ratio result in granule swelling by inflated water‐binding capacity in the amorphous lamella in low‐starch clones, compared to the starch structures in high‐dry matter content cultivars, where water retention capacity in the tuber storage polymer is inhibited. Indeed, the amylose/amylopectin ratio and crystallinity (negatively correlated with amylose content) are genotype‐dependent and impact starch granule size, with larger granules presenting with higher amylose contents (Schirmer et al., <a href="#tpg270066-bib-0059" class="usa-link" aria-describedby="tpg270066-bib-0059">2013</a>; Tong et al., <a href="#tpg270066-bib-0068" class="usa-link" aria-describedby="tpg270066-bib-0068">2023</a>). Dry matter content is also negatively genetically correlated with tuber size measures (diameter [−0.6] and length [−0.38]) and positively with tuber count (0.34), which could further indicate that low swelling power results in the formation of smaller tubers in higher numbers.</p>
<p>In previous findings, both high yield and dry matter contents have been phenotypically correlated with late maturity, as the reduced vegetation period of early senescing plants compared to late‐maturing cultivars results in reduced yield and dry matter content, muting genetic influences on these traits; that is, the maturity type is determinant of the phenotypic outputs of these traits (Urbany et al., <a href="#tpg270066-bib-0069" class="usa-link" aria-describedby="tpg270066-bib-0069">2011</a>; van Eck, <a href="#tpg270066-bib-0072" class="usa-link" aria-describedby="tpg270066-bib-0072">2007</a>). We did not find any genetic correlations between senescence and yield or dry matter content underpinning this effect. This might, however, be due to the low quality of senescence phenotypes.</p>
<p>Additive genetic correlations relating to tuber shape traits, that is, length, diameter, and length/width ratio, are consonant with expectations based on phenotypic correlations, with length/width ratio being positively correlated with length (0.78), and length also being positively correlated with diameter (0.68). This indicates a genetic control of both radial and apical tuber extension. Smoother skin and yellow flesh color correlated genetically with shallow eye depth (genetic correlations of 0.5 and 0.31, respectively), a genetic feature that is relevant in selection for processing and table potato markets, mentioning that preferences are highly variable across countries. In addition, skin finish is negatively genetically correlated with dry matter content (−0.4). The starch segment is largely unselected for smooth skin finish, and cultivars generally generate rough‐skinned tubers. Hence, this association could be population dependent, having arisen in the Danespo A/S breeding population, which we sampled through the 18 MASPOT parents, rather than being a true genetic correlation. Similarly, long tubers correlate with a smoother skin finish (0.54), a trait combination generally selected for in the French fry processing market (Van Eck et al., <a href="#tpg270066-bib-0073" class="usa-link" aria-describedby="tpg270066-bib-0073">1994</a>). However, these results would also indicate a genetic mode of inheritance linking these traits due to pleiotropic effects; for example, a pleiotropic effect of the <em>R</em>
<sub>0</sub> locus, where tuber elongation entails stretching of the skin to form a smoother surface. This effect could, however, also be specific to the assessed population due to two genetically linked loci.</p>
<p>Overall, the weak (0.1) to strong (−0.84) genetic correlations observed between pairwise trait combinations for the MASPOT population indicate that selection for any one trait during breeding will entail the indirect selection of up to several other important traits. Current selection in breeding programs is routinely based on multiple traits concurrently, but selection schemes typically employ an approach of sequential, single‐trait disqualification criteria, which has the obvious disadvantage of diminishing selection indices of later traits. We do not expect bivariate models or multivariate models to be much more accurate predictors for any single trait compared to single‐trait models. However, for breeding purposes, it is important to discern whether genetic optimization of one trait results in indirect selection of other traits due to underlying genetic correlations. Multi‐trait models are hence more suitable to optimize prediction models implemented to support selection (Neyhart et al., <a href="#tpg270066-bib-0044" class="usa-link" aria-describedby="tpg270066-bib-0044">2019</a>; Ortiz et al., <a href="#tpg270066-bib-0047" class="usa-link" aria-describedby="tpg270066-bib-0047">2023</a>), as they better capture correlated genetic features (Song et al., <a href="#tpg270066-bib-0062" class="usa-link" aria-describedby="tpg270066-bib-0062">2019</a>). Therefore, an interesting perspective is to quantify the increased genetic gain of employing multi‐trait models. In principle, the MASPOT panel data would be suitable for such a study, though using an actual breeding population compared to an experimental population will likely produce more accurate estimates of increased genetic gain. Software is also available for the simulation of an autotetraploid potato breeding population that could be used in such a study (Chu &amp; Jensen, <a href="#tpg270066-bib-0012" class="usa-link" aria-describedby="tpg270066-bib-0012">2025</a>).</p></section></section><section id="tpg270066-sec-0280"><h2 class="pmc_sec_title">4. CONCLUSIONS</h2>
<p>In this study, we explored the transmission ability of a collection of advanced breeding clones for a collection of agronomic traits using an unselected F<sub>1</sub> training population to explore means of trait genetic improvement through breeding. Expectedly, traits differed in their partitioning of genetic variance between predominantly additive or nonadditive, but all traits possessed significant additive genetic effects that have potential for genetic change in the population through breeding. Significant contributions to some trait phenotypes were found to be conferred by cytoplasmic genome types, indicating that the parent cytoplasmic genome type is an important consideration during crossing designs. Overall, the assessed parental cultivars displayed highly diverse transmission abilities across traits, and a network of pairwise trait genetic correlations indicates that single‐trait selection is associated with substantial indirect trait selection. It could be relevant to explore multi‐trait selection models to account for this co‐selection in the future of potato breeding. The inclusion of GBS information (available for 15% of the experimental population) in addition to phenotypic information in a single‐step approach improved the accuracy of cultivar EBV prediction, but for genomic prediction models, the optimal overall PA was obtained using GBLUP. Our results contribute to the mapping of transmission ability for the specific cultivars analyzed but also highlight the complex genetic inheritance and prediction of the total genetic value of clones in breeding.</p></section><section id="tpg270066-sec-0290"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Trine Aalborg</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; visualization; writing—original draft; writing—review and editing. <strong>Hélène Romé</strong>: Formal analysis; methodology; writing—review and editing. <strong>Christina Ranzau</strong>: Data curation; writing—review and editing. <strong>Merethe Bagge</strong>: Data curation; writing—review and editing. <strong>Just Jensen</strong>: Conceptualization; formal analysis; methodology; supervision; writing—review and editing. <strong>Kåre Lehmann Nielsen</strong>: Conceptualization; funding acquisition; methodology; supervision; writing—review and editing.</p></section><section id="tpg270066-sec-0300"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>Christina Ranzau and Merethe Bagge are employed by Danespo A/S, Denmark. Kåre Lehmann Nielsen is partially employed by KMC Amba, Denmark. Hélène Romé is currently affiliated with SEGES Innovation P/S, Denmark. The remaining authors declare that they have no conflicts of interest.</p></section><section id="sec31"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270066-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270066-supitem-0001"><div class="caption p"><p>Additional supporting information can be found online in the Supplemental Material section at the end of this article. These include supplementary figures (<strong>Supplemental Material S1</strong>) and supplementary tables (<strong>Supplemental Material S2</strong>).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12207579/bin/TPG2-18-e70066-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70066-s001.docx</a><sup> (2.4MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270066-supitem-0002"><div class="caption p"><p>Supplemental Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12207579/bin/TPG2-18-e70066-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70066-s002.docx</a><sup> (31.7KB, docx) </sup>
</div></div></section></section><section id="notes1"><p>


Aalborg, T.
, 
Romé, H.
, 
Ranzau, C.
, 
Bagge, M.
, 
Jensen, J.
, &amp; 
Nielsen, K. L.
 (2025). The transmission ability in a population of elite tetraploid potatoes. The Plant Genome, 18, e70066. 10.1002/tpg2.70066

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="tpg270066-note-0001"><p>Assigned to Associate Editor Martina Stromvik.</p></div>
<div class="fn p" id="tpg270066-note-0002">
<p>
<strong>Present address</strong>
</p>
<p>Hélène Romé, SEGES Innovation P/S, Aarhus, Denmark</p>
</div>
</div></section></section><section id="tpg270066-sec-0320"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The datasets needed to reproduce the results of this study are available in the Zenodo repositor: <a href="https://doi.org/10.5281/zenodo.11916143" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.11916143</a>. These include that marker and pedigree data of the population, as well as recorded phenotypic and metadata.</p></section><section id="tpg270066-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERNCES</h2>
<section id="tpg270066-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270066-bib-0001">
<cite>

Aalborg, T.
, &amp; 
Nielsen, K. L.
 (2024). To be or not to be tetraploid—The impact of marker ploidy on genomic prediction and GWAS of potato. Frontiers in Plant Science, 15, 1386837. 10.3389/fpls.2024.1386837

</cite> [<a href="https://doi.org/10.3389/fpls.2024.1386837" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11319270/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39139728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=To%20be%20or%20not%20to%20be%20tetraploid%E2%80%94The%20impact%20of%20marker%20ploidy%20on%20genomic%20prediction%20and%20GWAS%20of%20potato&amp;author=T.%20Aalborg&amp;author=K.%20L.%20Nielsen&amp;volume=15&amp;publication_year=2024&amp;pages=1386837&amp;pmid=39139728&amp;doi=10.3389/fpls.2024.1386837&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0002">
<cite>

Aalborg, T.
, 
Sverrisdóttir, E.
, 
Kristensen, H. T.
, &amp; 
Nielsen, K. L.
 (2024). The effect of marker types and density on genomic prediction and GWAS of key performance traits in tetraploid potato. Frontiers in Plant Science, 15, 1340189. 10.3389/fpls.2024.1340189

</cite> [<a href="https://doi.org/10.3389/fpls.2024.1340189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10957621/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38525152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=The%20effect%20of%20marker%20types%20and%20density%20on%20genomic%20prediction%20and%20GWAS%20of%20key%20performance%20traits%20in%20tetraploid%20potato&amp;author=T.%20Aalborg&amp;author=E.%20Sverrisd%C3%B3ttir&amp;author=H.%20T.%20Kristensen&amp;author=K.%20L.%20Nielsen&amp;volume=15&amp;publication_year=2024&amp;pages=1340189&amp;pmid=38525152&amp;doi=10.3389/fpls.2024.1340189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0003">
<cite>

Amadeu, R. R.
, 
Garcia, A. A. F.
, 
Munoz, P. R.
, &amp; 
Ferrão, L. F. V.
 (2023). AGHmatrix: Genetic relationship matrices in R. Bioinformatics, 39, btad445. 10.1093/BIOINFORMATICS/BTAD445

</cite> [<a href="https://doi.org/10.1093/BIOINFORMATICS/BTAD445" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10371492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37471595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=AGHmatrix:%20Genetic%20relationship%20matrices%20in%20R&amp;author=R.%20R.%20Amadeu&amp;author=A.%20A.%20F.%20Garcia&amp;author=P.%20R.%20Munoz&amp;author=L.%20F.%20V.%20Ferr%C3%A3o&amp;volume=39&amp;publication_year=2023&amp;pages=btad445&amp;pmid=37471595&amp;doi=10.1093/BIOINFORMATICS/BTAD445&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0004">
<cite>

Bethke, P. C.
, &amp; 
Jansky, S. H.
 (2021). Genetic and environmental factors contributing to reproductive success and failure in potato. American Journal of Potato Research, 98, 24–41. 10.1007/s12230-020-09810-3
</cite> [<a href="https://doi.org/10.1007/s12230-020-09810-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Potato%20Research&amp;title=Genetic%20and%20environmental%20factors%20contributing%20to%20reproductive%20success%20and%20failure%20in%20potato&amp;author=P.%20C.%20Bethke&amp;author=S.%20H.%20Jansky&amp;volume=98&amp;publication_year=2021&amp;pages=24-41&amp;doi=10.1007/s12230-020-09810-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0005">
<cite>

Bland, M.
 (2000). An introduction to medical statistics (3rd ed.). Oxford University Press.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=An%20introduction%20to%20medical%20statistics&amp;author=M.%20Bland&amp;publication_year=2000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0006">
<cite>

Bourke, P. M.
, 
Voorrips, R. E.
, 
Visser, R. G. F.
, &amp; 
Maliepaard, C.
 (2015). The double‐reduction landscape in tetraploid potato as revealed by a high‐density linkage map. Genetics, 201, 853–863. 10.1534/GENETICS.115.181008/-/DC1

</cite> [<a href="https://doi.org/10.1534/GENETICS.115.181008/-/DC1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4649655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26377683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=The%20double%E2%80%90reduction%20landscape%20in%20tetraploid%20potato%20as%20revealed%20by%20a%20high%E2%80%90density%20linkage%20map&amp;author=P.%20M.%20Bourke&amp;author=R.%20E.%20Voorrips&amp;author=R.%20G.%20F.%20Visser&amp;author=C.%20Maliepaard&amp;volume=201&amp;publication_year=2015&amp;pages=853-863&amp;pmid=26377683&amp;doi=10.1534/GENETICS.115.181008/-/DC1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0007">
<cite>

Bourke, P. M.
, 
Voorrips, R. E.
, 
Visser, R. G. F.
, &amp; 
Maliepaard, C.
 (2018). Tools for genetic studies in experimental populations of polyploids. Frontiers in Plant Science, 9, Article 513. 10.3389/FPLS.2018.00513

</cite> [<a href="https://doi.org/10.3389/FPLS.2018.00513" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5915555/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29720992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Tools%20for%20genetic%20studies%20in%20experimental%20populations%20of%20polyploids&amp;author=P.%20M.%20Bourke&amp;author=R.%20E.%20Voorrips&amp;author=R.%20G.%20F.%20Visser&amp;author=C.%20Maliepaard&amp;volume=9&amp;publication_year=2018&amp;pages=513&amp;pmid=29720992&amp;doi=10.3389/FPLS.2018.00513&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0008">
<cite>

Bradshaw, J. E.
 (2022). A brief history of the impact of potato genetics on the breeding of tetraploid potato cultivars for tuber propagation. Potato Research, 65, 461–501. 10.1007/s11540-021-09517-w
</cite> [<a href="https://doi.org/10.1007/s11540-021-09517-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Potato%20Research&amp;title=A%20brief%20history%20of%20the%20impact%20of%20potato%20genetics%20on%20the%20breeding%20of%20tetraploid%20potato%20cultivars%20for%20tuber%20propagation&amp;author=J.%20E.%20Bradshaw&amp;volume=65&amp;publication_year=2022&amp;pages=461-501&amp;doi=10.1007/s11540-021-09517-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0009">
<cite>

Bradshaw, J. E.
, 
Todd, D.
, &amp; 
Wilson, R. N.
 (2000). Use of tuber progeny tests for genetical studies as part of potato (<em>Solanum tuberosum</em> subsp. <em>tuberosum</em>) breeding programme. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 100, 772–781. 10.1007/s001220051351
</cite> [<a href="https://doi.org/10.1007/s001220051351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24172936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=Use%20of%20tuber%20progeny%20tests%20for%20genetical%20studies%20as%20part%20of%20potato%20(Solanum%20tuberosum%20subsp.%20tuberosum)%20breeding%20programme&amp;author=J.%20E.%20Bradshaw&amp;author=D.%20Todd&amp;author=R.%20N.%20Wilson&amp;volume=100&amp;publication_year=2000&amp;pages=772-781&amp;pmid=24172936&amp;doi=10.1007/s001220051351&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0010">
<cite>

Brown, C. R.
 (1993). Origin and history of the potato. American Potato Journal, 70, 363–373. 10.1007/BF02849117
</cite> [<a href="https://doi.org/10.1007/BF02849117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Potato%20Journal&amp;title=Origin%20and%20history%20of%20the%20potato&amp;author=C.%20R.%20Brown&amp;volume=70&amp;publication_year=1993&amp;pages=363-373&amp;doi=10.1007/BF02849117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0011">
<cite>

Christensen, O. F.
, &amp; 
Lund, M. S.
 (2010). Genomic prediction when some animals are not genotyped. Genetics Selection Evolution, 42, Article 2. 10.1186/1297-9686-42-2
</cite> [<a href="https://doi.org/10.1186/1297-9686-42-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2834608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20105297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Selection%20Evolution&amp;title=Genomic%20prediction%20when%20some%20animals%20are%20not%20genotyped&amp;author=O.%20F.%20Christensen&amp;author=M.%20S.%20Lund&amp;volume=42&amp;publication_year=2010&amp;pages=2&amp;pmid=20105297&amp;doi=10.1186/1297-9686-42-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0012">
<cite>

Chu, T. T.
, &amp; 
Jensen, J.
 (2025). ADAM‐multi: Software to simulate complex breeding programs for animals and plants with different ploidy levels and generalized genotypic effect models to account for multiple alleles. Frontiers in Genetics, 16, 1513615. 10.3389/FGENE.2025.1513615

</cite> [<a href="https://doi.org/10.3389/FGENE.2025.1513615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11847855/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39995464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=ADAM%E2%80%90multi:%20Software%20to%20simulate%20complex%20breeding%20programs%20for%20animals%20and%20plants%20with%20different%20ploidy%20levels%20and%20generalized%20genotypic%20effect%20models%20to%20account%20for%20multiple%20alleles&amp;author=T.%20T.%20Chu&amp;author=J.%20Jensen&amp;volume=16&amp;publication_year=2025&amp;pages=1513615&amp;pmid=39995464&amp;doi=10.3389/FGENE.2025.1513615&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0013">
<cite>

Darabad, G. R.
, 
Hassandokht, M. R.
, 
Hassanpanah, D.
, &amp; 
Mousavi, A.
 (2020). Diallel cross in potato cultivars (<em>Solanum tuberosum</em> L.) and evaluation of their progenies under deficit water stress. Acta Agrobotanica, 73, 1–9. 10.5586/AA.7325
</cite> [<a href="https://doi.org/10.5586/AA.7325" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Agrobotanica&amp;title=Diallel%20cross%20in%20potato%20cultivars%20(Solanum%20tuberosum%20L.)%20and%20evaluation%20of%20their%20progenies%20under%20deficit%20water%20stress&amp;author=G.%20R.%20Darabad&amp;author=M.%20R.%20Hassandokht&amp;author=D.%20Hassanpanah&amp;author=A.%20Mousavi&amp;volume=73&amp;publication_year=2020&amp;pages=1-9&amp;doi=10.5586/AA.7325&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0014">
<cite>

Dean, R. B.
, &amp; 
Dixon, W. J.
 (1951). Simplified statistics for small numbers of observations. Analytical Chemistry, 23, 636–638. 10.1021/ac60052a025
</cite> [<a href="https://doi.org/10.1021/ac60052a025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Simplified%20statistics%20for%20small%20numbers%20of%20observations&amp;author=R.%20B.%20Dean&amp;author=W.%20J.%20Dixon&amp;volume=23&amp;publication_year=1951&amp;pages=636-638&amp;doi=10.1021/ac60052a025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0015">
<cite>

Devaux, A.
, 
Goffart, J.‐P.
, 
Petsakos, A.
, 
Kromann, P.
, 
Gatto, M.
, 
Okello, J.
, 
Suarez, V.
, &amp; 
Hareau, G.
 (2019). Global food security, contributions from sustainable potato agri‐food systems. In Campos H.&amp; Ortiz O. (Eds.), The potato crop: Its agricultural, nutritional and social contribution to humankind (pp. 3–36). Springer International Publishing.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20potato%20crop:%20Its%20agricultural,%20nutritional%20and%20social%20contribution%20to%20humankind&amp;author=A.%20Devaux&amp;author=J.%E2%80%90P.%20Goffart&amp;author=A.%20Petsakos&amp;author=P.%20Kromann&amp;author=M.%20Gatto&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0016">
<cite>

Devaux, A.
, 
Kromann, P.
, &amp; 
Ortiz, O.
 (2014). Potatoes for sustainable global food security. Potato Research, 57, 185–199. 10.1007/S11540-014-9265-1
</cite> [<a href="https://doi.org/10.1007/S11540-014-9265-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Potato%20Research&amp;title=Potatoes%20for%20sustainable%20global%20food%20security&amp;author=A.%20Devaux&amp;author=P.%20Kromann&amp;author=O.%20Ortiz&amp;volume=57&amp;publication_year=2014&amp;pages=185-199&amp;doi=10.1007/S11540-014-9265-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0017">
<cite>

Díaz‐Francés, E.
, &amp; 
Rubio, F. J.
 (2013). On the existence of a normal approximation to the distribution of the ratio of two independent normal random variables. Statistical Papers, 54, 309–323. 10.1007/S00362-012-0429-2/METRICS
</cite> [<a href="https://doi.org/10.1007/S00362-012-0429-2/METRICS" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Statistical%20Papers&amp;title=On%20the%20existence%20of%20a%20normal%20approximation%20to%20the%20distribution%20of%20the%20ratio%20of%20two%20independent%20normal%20random%20variables&amp;author=E.%20D%C3%ADaz%E2%80%90Franc%C3%A9s&amp;author=F.%20J.%20Rubio&amp;volume=54&amp;publication_year=2013&amp;pages=309-323&amp;doi=10.1007/S00362-012-0429-2/METRICS&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0018">
<cite>

Endelman, J. B.
, 
Carley, C. A. S.
, 
Bethke, P. C.
, 
Coombs, J. J.
, 
Clough, M. E.
, 
Da Silva, W. L.
, 
De Jong, W. S.
, 
Douches, D. S.
, 
Frederick, C. M.
, 
Haynes, K. G.
, 
Holm, D. G.
, 
Miller, J. C.
, 
Muñoz, P. R.
, 
Navarro, F. M.
, 
Novy, R. G.
, 
Palta, J. P.
, 
Porter, G. A.
, 
Rak, K. T.
, 
Sathuvalli, V. R.
, … 
Yencho, G. C.
 (2018). Genetic variance partitioning and genome‐wide prediction with allele dosage information in autotetraploid potato. Genetics, 209, 77–87. 10.1534/GENETICS.118.300685

</cite> [<a href="https://doi.org/10.1534/GENETICS.118.300685" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5937173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29514860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genetic%20variance%20partitioning%20and%20genome%E2%80%90wide%20prediction%20with%20allele%20dosage%20information%20in%20autotetraploid%20potato&amp;author=J.%20B.%20Endelman&amp;author=C.%20A.%20S.%20Carley&amp;author=P.%20C.%20Bethke&amp;author=J.%20J.%20Coombs&amp;author=M.%20E.%20Clough&amp;volume=209&amp;publication_year=2018&amp;pages=77-87&amp;pmid=29514860&amp;doi=10.1534/GENETICS.118.300685&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0019">
<cite>
FAOSTAT
. (2024). Crops and livestock products. Food and Agriculture Organization of the United Nations Statistics Division. <a href="https://www.fao.org/faostat/en/#data/QCL" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fao.org/faostat/en/#data/QCL</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Crops%20and%20livestock%20products&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0020">
<cite>

Gallais, A.
 (2003). Quantitative genetics and breeding methods in autopolyploid plants (1st ed.). INRA.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20genetics%20and%20breeding%20methods%20in%20autopolyploid%20plants&amp;author=A.%20Gallais&amp;publication_year=2003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0021">
<cite>

Gebhardt, C.
 (2016). The historical role of species from the Solanaceae plant family in genetic research. Theoretical and Applied Genetics, 129, 2281–2294. 10.1007/s00122-016-2804-1

</cite> [<a href="https://doi.org/10.1007/s00122-016-2804-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5121179/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27744490/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=The%20historical%20role%20of%20species%20from%20the%20Solanaceae%20plant%20family%20in%20genetic%20research&amp;author=C.%20Gebhardt&amp;volume=129&amp;publication_year=2016&amp;pages=2281-2294&amp;pmid=27744490&amp;doi=10.1007/s00122-016-2804-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0022">
<cite>

Gerard, D.
 (2023). Double reduction estimation and equilibrium tests in natural autopolyploid populations. Biometrics, 79, 2143–2156. 10.1111/biom.13722

</cite> [<a href="https://doi.org/10.1111/biom.13722" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35848417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Double%20reduction%20estimation%20and%20equilibrium%20tests%20in%20natural%20autopolyploid%20populations&amp;author=D.%20Gerard&amp;volume=79&amp;publication_year=2023&amp;pages=2143-2156&amp;pmid=35848417&amp;doi=10.1111/biom.13722&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0023">
<cite>

Gopal, J.
 (1998). General combining ability and its repeatability in early generations of potato breeding programmes. Potato Research, 41, 21–28. 10.1007/BF02360258
</cite> [<a href="https://doi.org/10.1007/BF02360258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Potato%20Research&amp;title=General%20combining%20ability%20and%20its%20repeatability%20in%20early%20generations%20of%20potato%20breeding%20programmes&amp;author=J.%20Gopal&amp;volume=41&amp;publication_year=1998&amp;pages=21-28&amp;doi=10.1007/BF02360258&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0024">
<cite>

Griffing, B.
 (1956). Concept of general and specific combining ability in relation to diallel crossing systems. Australian Journal of Biological Sciences, 9, 463–493. 10.1071/BI9560463
</cite> [<a href="https://doi.org/10.1071/BI9560463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Australian%20Journal%20of%20Biological%20Sciences&amp;title=Concept%20of%20general%20and%20specific%20combining%20ability%20in%20relation%20to%20diallel%20crossing%20systems&amp;author=B.%20Griffing&amp;volume=9&amp;publication_year=1956&amp;pages=463-493&amp;doi=10.1071/BI9560463&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0025">
<cite>

Gutaker, R. M.
, 
Weiß, C. L.
, 
Ellis, D.
, 
Anglin, N. L.
, 
Knapp, S.
, 
Luis Fernández‐Alonso, J.
, 
Prat, S.
, &amp; 
Burbano, H. A.
 (2019). The origins and adaptation of European potatoes reconstructed from historical genomes. Nature Ecology &amp; Evolution, 3(7), 1093–1101. 10.1038/s41559-019-0921-3

</cite> [<a href="https://doi.org/10.1038/s41559-019-0921-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31235927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Ecology%20&amp;%20Evolution&amp;title=The%20origins%20and%20adaptation%20of%20European%20potatoes%20reconstructed%20from%20historical%20genomes&amp;author=R.%20M.%20Gutaker&amp;author=C.%20L.%20Wei%C3%9F&amp;author=D.%20Ellis&amp;author=N.%20L.%20Anglin&amp;author=S.%20Knapp&amp;volume=3&amp;issue=7&amp;publication_year=2019&amp;pages=1093-1101&amp;pmid=31235927&amp;doi=10.1038/s41559-019-0921-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0026">
<cite>

Haase, N. U.
 (2003). Estimation of dry matter and starch concentration in potatoes by determination of under‐water weight and near infrared spectroscopy. Potato Research, 46, 117–127. 10.1007/BF02736081
</cite> [<a href="https://doi.org/10.1007/BF02736081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Potato%20Research&amp;title=Estimation%20of%20dry%20matter%20and%20starch%20concentration%20in%20potatoes%20by%20determination%20of%20under%E2%80%90water%20weight%20and%20near%20infrared%20spectroscopy&amp;author=N.%20U.%20Haase&amp;volume=46&amp;publication_year=2003&amp;pages=117-127&amp;doi=10.1007/BF02736081&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0027">
<cite>

Hardy, O. J.
 (2016). Population genetics of autopolyploids under a mixed mating model and the estimation of selfing rate. Molecular Ecology Resources, 16, 103–117. 10.1111/1755-0998.12431

</cite> [<a href="https://doi.org/10.1111/1755-0998.12431" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25981126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Resources&amp;title=Population%20genetics%20of%20autopolyploids%20under%20a%20mixed%20mating%20model%20and%20the%20estimation%20of%20selfing%20rate&amp;author=O.%20J.%20Hardy&amp;volume=16&amp;publication_year=2016&amp;pages=103-117&amp;pmid=25981126&amp;doi=10.1111/1755-0998.12431&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0028">
<cite>

Hickey, J. M.
, 
Chiurugwi, T.
, 
Mackay, I.
, &amp; 
Powell, W.
 (2017). Genomic prediction unifies animal and plant breeding programs to form platforms for biological discovery. Nature Genetics, 49, 1297–1303. 10.1038/NG.3920

</cite> [<a href="https://doi.org/10.1038/NG.3920" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28854179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Genomic%20prediction%20unifies%20animal%20and%20plant%20breeding%20programs%20to%20form%20platforms%20for%20biological%20discovery&amp;author=J.%20M.%20Hickey&amp;author=T.%20Chiurugwi&amp;author=I.%20Mackay&amp;author=W.%20Powell&amp;volume=49&amp;publication_year=2017&amp;pages=1297-1303&amp;pmid=28854179&amp;doi=10.1038/NG.3920&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0029">
<cite>

Hosaka, K.
, &amp; 
Sanetomo, R.
 (2012). Development of a rapid identification method for potato cytoplasm and its use for evaluating Japanese collections. Theoretical and Applied Genetics, 125, 1237–1251. 10.1007/S00122-012-1909-4/FIGURES/4

</cite> [<a href="https://doi.org/10.1007/S00122-012-1909-4/FIGURES/4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22696007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Development%20of%20a%20rapid%20identification%20method%20for%20potato%20cytoplasm%20and%20its%20use%20for%20evaluating%20Japanese%20collections&amp;author=K.%20Hosaka&amp;author=R.%20Sanetomo&amp;volume=125&amp;publication_year=2012&amp;pages=1237-1251&amp;pmid=22696007&amp;doi=10.1007/S00122-012-1909-4/FIGURES/4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0030">
<cite>

Hutten, R.
, &amp; 
van Berloo, R.
 (2001). 
An online potato pedigree database
. Wageningen University &amp; Research. <a href="http://www.plantbreeding.wur.nl/PotatoPedigree/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.plantbreeding.wur.nl/PotatoPedigree/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Hutten,%20R.%20,%20&amp;%20van%20Berloo,%20R.%20(2001).%20An%20online%20potato%20pedigree%20database%20.%20Wageningen%20University%20&amp;%20Research.%20http://www.plantbreeding.wur.nl/PotatoPedigree/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0031">
<cite>

Kloosterman, B.
, 
Abelenda, J. A.
, 
Gomez, M. D. M. C.
, 
Oortwijn, M.
, 
De Boer, J. M.
, 
Kowitwanich, K.
, 
Horvath, B. M.
, 
Van Eck, H. J.
, 
Smaczniak, C.
, 
Prat, S.
, 
Visser, R. G. F.
, &amp; 
Bachem, C. W. B.
 (2013). Naturally occurring allele diversity allows potato cultivation in northern latitudes. Nature, 495(7440), 246–250. 10.1038/nature11912

</cite> [<a href="https://doi.org/10.1038/nature11912" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23467094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Naturally%20occurring%20allele%20diversity%20allows%20potato%20cultivation%20in%20northern%20latitudes&amp;author=B.%20Kloosterman&amp;author=J.%20A.%20Abelenda&amp;author=M.%20D.%20M.%20C.%20Gomez&amp;author=M.%20Oortwijn&amp;author=J.%20M.%20De%20Boer&amp;volume=495&amp;issue=7440&amp;publication_year=2013&amp;pages=246-250&amp;pmid=23467094&amp;doi=10.1038/nature11912&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0032">
<cite>

Kristensen, P. S.
, 
Sarup, P.
, 
Fé, D.
, 
Orabi, J.
, 
Snell, P.
, 
Ripa, L.
, 
Mohlfeld, M.
, 
Chu, T. T.
, 
Herrström, J.
, 
Jahoor, A.
, &amp; 
Jensen, J.
 (2023). Prediction of additive, epistatic, and dominance effects using models accounting for incomplete inbreeding in parental lines of hybrid rye and sugar beet. Frontiers in Plant Science, 14, 1193433. 10.3389/FPLS.2023.1193433

</cite> [<a href="https://doi.org/10.3389/FPLS.2023.1193433" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10756082/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38162304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Prediction%20of%20additive,%20epistatic,%20and%20dominance%20effects%20using%20models%20accounting%20for%20incomplete%20inbreeding%20in%20parental%20lines%20of%20hybrid%20rye%20and%20sugar%20beet&amp;author=P.%20S.%20Kristensen&amp;author=P.%20Sarup&amp;author=D.%20F%C3%A9&amp;author=J.%20Orabi&amp;author=P.%20Snell&amp;volume=14&amp;publication_year=2023&amp;pages=1193433&amp;pmid=38162304&amp;doi=10.3389/FPLS.2023.1193433&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0033">
<cite>

Legarra, A.
, 
Aguilar, I.
, &amp; 
Misztal, I.
 (2009). A relationship matrix including full pedigree and genomic information. Journal of Dairy Science, 92, 4656–4663. 10.3168/jds.2009-2061

</cite> [<a href="https://doi.org/10.3168/jds.2009-2061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19700729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Dairy%20Science&amp;title=A%20relationship%20matrix%20including%20full%20pedigree%20and%20genomic%20information&amp;author=A.%20Legarra&amp;author=I.%20Aguilar&amp;author=I.%20Misztal&amp;volume=92&amp;publication_year=2009&amp;pages=4656-4663&amp;pmid=19700729&amp;doi=10.3168/jds.2009-2061&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0034">
<cite>

Lenaerts, B.
, 
Collard, B. C. Y.
, &amp; 
Demont, M.
 (2019). Review: Improving global food security through accelerated plant breeding. Plant Science, 287, 110207. 10.1016/j.plantsci.2019.110207

</cite> [<a href="https://doi.org/10.1016/j.plantsci.2019.110207" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6745619/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31481198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Science&amp;title=Review:%20Improving%20global%20food%20security%20through%20accelerated%20plant%20breeding&amp;author=B.%20Lenaerts&amp;author=B.%20C.%20Y.%20Collard&amp;author=M.%20Demont&amp;volume=287&amp;publication_year=2019&amp;pages=110207&amp;pmid=31481198&amp;doi=10.1016/j.plantsci.2019.110207&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0035">
<cite>

Li, L.
, 
Tacke, E.
, 
Hofferbert, H.‐R.
, 
Lübeck, J.
, 
Strahwald, J.
, 
Draffehn, A. M.
, 
Walkemeier, B.
, &amp; 
Gebhardt, C.
 (2013). Validation of candidate gene markers for marker‐assisted selection of potato cultivars with improved tuber quality. Theoretical and Applied Genetics, 126, 1039–1052. 10.1007/S00122-012-2035-Z

</cite> [<a href="https://doi.org/10.1007/S00122-012-2035-Z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3607734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23299900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Validation%20of%20candidate%20gene%20markers%20for%20marker%E2%80%90assisted%20selection%20of%20potato%20cultivars%20with%20improved%20tuber%20quality&amp;author=L.%20Li&amp;author=E.%20Tacke&amp;author=H.%E2%80%90R.%20Hofferbert&amp;author=J.%20L%C3%BCbeck&amp;author=J.%20Strahwald&amp;volume=126&amp;publication_year=2013&amp;pages=1039-1052&amp;pmid=23299900&amp;doi=10.1007/S00122-012-2035-Z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0036">
<cite>

Li, X. Q.
, 
De Jong, H.
, 
De Jong, D. M.
, &amp; 
De Jong, W. S.
 (2005). Inheritance and genetic mapping of tuber eye depth in cultivated diploid potatoes. Theoretical and Applied Genetics, 110, 1068–1073. 10.1007/s00122-005-1927-6

</cite> [<a href="https://doi.org/10.1007/s00122-005-1927-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15719211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Inheritance%20and%20genetic%20mapping%20of%20tuber%20eye%20depth%20in%20cultivated%20diploid%20potatoes&amp;author=X.%20Q.%20Li&amp;author=H.%20De%20Jong&amp;author=D.%20M.%20De%20Jong&amp;author=W.%20S.%20De%20Jong&amp;volume=110&amp;publication_year=2005&amp;pages=1068-1073&amp;pmid=15719211&amp;doi=10.1007/s00122-005-1927-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0037">
<cite>

Lian, Q.
, 
Zhang, S.
, 
Wu, Z.
, 
Zhang, C.
, &amp; 
Negrão, S.
 (2024). Assembly and comparative analysis of the mitochondrial genome in diploid potatoes. Plant Cell Reports, 43, Article 49. 10.1007/S00299-024-03326-4

</cite> [<a href="https://doi.org/10.1007/S00299-024-03326-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39358565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Reports&amp;title=Assembly%20and%20comparative%20analysis%20of%20the%20mitochondrial%20genome%20in%20diploid%20potatoes&amp;author=Q.%20Lian&amp;author=S.%20Zhang&amp;author=Z.%20Wu&amp;author=C.%20Zhang&amp;author=S.%20Negr%C3%A3o&amp;volume=43&amp;publication_year=2024&amp;pages=49&amp;pmid=39358565&amp;doi=10.1007/S00299-024-03326-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0038">
<cite>

Lössl, A.
, 
Götz, M.
, 
Braun, A.
, &amp; 
Wenzel, G.
 (2000). Molecular markers for cytoplasm in potato: Male sterility and contribution of different plastid‐mitochondrial configurations to starch production. Euphytica, 116, 221–230. 10.1023/A:1004039320227
</cite> [<a href="https://doi.org/10.1023/A:1004039320227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Molecular%20markers%20for%20cytoplasm%20in%20potato:%20Male%20sterility%20and%20contribution%20of%20different%20plastid%E2%80%90mitochondrial%20configurations%20to%20starch%20production&amp;author=A.%20L%C3%B6ssl&amp;author=M.%20G%C3%B6tz&amp;author=A.%20Braun&amp;author=G.%20Wenzel&amp;volume=116&amp;publication_year=2000&amp;pages=221-230&amp;doi=10.1023/A:1004039320227&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0039">
<cite>

Luan, T.
, 
Woolliams, J. A.
, 
Lien, S.
, 
Kent, M.
, 
Svendsen, M.
, &amp; 
Meuwissen, T. H. E.
 (2009). The accuracy of genomic selection in Norwegian red cattle assessed by cross‐validation. Genetics, 183, 1119–1126. 10.1534/GENETICS.109.107391

</cite> [<a href="https://doi.org/10.1534/GENETICS.109.107391" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2778964/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19704013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=The%20accuracy%20of%20genomic%20selection%20in%20Norwegian%20red%20cattle%20assessed%20by%20cross%E2%80%90validation&amp;author=T.%20Luan&amp;author=J.%20A.%20Woolliams&amp;author=S.%20Lien&amp;author=M.%20Kent&amp;author=M.%20Svendsen&amp;volume=183&amp;publication_year=2009&amp;pages=1119-1126&amp;pmid=19704013&amp;doi=10.1534/GENETICS.109.107391&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0080">
<cite>

Madsen, P.
, 
Jensen, J.
, 
Labouriau, R.
, 
Christensen, O. F.
, &amp; 
Sahana, G.
 (2014). DMU ‐ A package for analyzing multivariate mixed models in quantitative genetics and genomics. In Proceedings, 10th World Congress of Genetics Applied to Livestock Production. American Society of Animal Science.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Proceedings,%2010th%20World%20Congress%20of%20Genetics%20Applied%20to%20Livestock%20Production&amp;author=P.%20Madsen&amp;author=J.%20Jensen&amp;author=R.%20Labouriau&amp;author=O.%20F.%20Christensen&amp;author=G.%20Sahana&amp;publication_year=2014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0041">
<cite>

Martini, J. W. R.
, 
Schrauf, M. F.
, 
Garcia‐Baccino, C. A.
, 
Pimentel, E. C. G.
, 
Munilla, S.
, 
Rogberg‐Muñoz, A.
, 
Cantet, R. J. C.
, 
Reimer, C.
, 
Gao, N.
, 
Wimmer, V.
, &amp; 
Simianer, H.
 (2018). The effect of the H<sup>−1</sup> scaling factors <em>τ</em> and <em>ω</em> on the structure of H in the single‐step procedure. Genetics Selection Evolution, 50, Article 16. 10.1186/S12711-018-0386-X/FIGURES/2
</cite> [<a href="https://doi.org/10.1186/S12711-018-0386-X/FIGURES/2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5899415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29653506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Selection%20Evolution&amp;title=The%20effect%20of%20the%20H%E2%88%921%20scaling%20factors%20%CF%84%20and%20%CF%89%20on%20the%20structure%20of%20H%20in%20the%20single%E2%80%90step%20procedure&amp;author=J.%20W.%20R.%20Martini&amp;author=M.%20F.%20Schrauf&amp;author=C.%20A.%20Garcia%E2%80%90Baccino&amp;author=E.%20C.%20G.%20Pimentel&amp;author=S.%20Munilla&amp;volume=50&amp;publication_year=2018&amp;pages=16&amp;pmid=29653506&amp;doi=10.1186/S12711-018-0386-X/FIGURES/2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0042">
<cite>

Meuwissen, T. H. E.
, 
Hayes, B. J.
, &amp; 
Goddard, M. E.
 (2001). Prediction of total genetic value using genome‐wide dense marker maps. Genetics, 157, 1819–1829. 10.1093/GENETICS/157.4.1819

</cite> [<a href="https://doi.org/10.1093/GENETICS/157.4.1819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11290733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Prediction%20of%20total%20genetic%20value%20using%20genome%E2%80%90wide%20dense%20marker%20maps&amp;author=T.%20H.%20E.%20Meuwissen&amp;author=B.%20J.%20Hayes&amp;author=M.%20E.%20Goddard&amp;volume=157&amp;publication_year=2001&amp;pages=1819-1829&amp;pmid=11290733&amp;doi=10.1093/GENETICS/157.4.1819&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0043">
<cite>

Naeem, M.
, 
Demirel, U.
, 
Yousaf, M. F.
, 
Caliskan, S.
, &amp; 
Caliskan, M. E.
 (2021). Overview on domestication, breeding, genetic gain and improvement of tuber quality traits of potato using fast forwarding technique (GWAS): A review. Plant Breeding, 140, 519–542. 10.1111/PBR.12927
</cite> [<a href="https://doi.org/10.1111/PBR.12927" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Breeding&amp;title=Overview%20on%20domestication,%20breeding,%20genetic%20gain%20and%20improvement%20of%20tuber%20quality%20traits%20of%20potato%20using%20fast%20forwarding%20technique%20(GWAS):%20A%20review&amp;author=M.%20Naeem&amp;author=U.%20Demirel&amp;author=M.%20F.%20Yousaf&amp;author=S.%20Caliskan&amp;author=M.%20E.%20Caliskan&amp;volume=140&amp;publication_year=2021&amp;pages=519-542&amp;doi=10.1111/PBR.12927&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0044">
<cite>

Neyhart, J. L.
, 
Lorenz, A. J.
, &amp; 
Smith, K. P.
 (2019). Multi‐trait improvement by predicting genetic correlations in breeding crosses. G3: Genes, Genomes, Genetics, 9, 3153–3165. 10.1534/g3.119.400406

</cite> [<a href="https://doi.org/10.1534/g3.119.400406" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6778794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31358561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=Multi%E2%80%90trait%20improvement%20by%20predicting%20genetic%20correlations%20in%20breeding%20crosses&amp;author=J.%20L.%20Neyhart&amp;author=A.%20J.%20Lorenz&amp;author=K.%20P.%20Smith&amp;volume=9&amp;publication_year=2019&amp;pages=3153-3165&amp;pmid=31358561&amp;doi=10.1534/g3.119.400406&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0045">
<cite>

Ortiz, R.
 (2020). Genomic‐led potato breeding for increasing genetic gains: Achievements and outlook. Crop Breeding, Genetics and Genomics, 2, e200010. 10.20900/CBGG20200010
</cite> [<a href="https://doi.org/10.20900/CBGG20200010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Breeding,%20Genetics%20and%20Genomics&amp;title=Genomic%E2%80%90led%20potato%20breeding%20for%20increasing%20genetic%20gains:%20Achievements%20and%20outlook&amp;author=R.%20Ortiz&amp;volume=2&amp;publication_year=2020&amp;pages=e200010&amp;doi=10.20900/CBGG20200010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0046">
<cite>

Ortiz, R.
, 
Iwanaga, M.
, &amp; 
Mendoza, H. A.
 (1988). Combining ability and parental effects in 4x‐2x crosses for potato breeding. Potato Research, 31, 643–650. 10.1007/BF02361857
</cite> [<a href="https://doi.org/10.1007/BF02361857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Potato%20Research&amp;title=Combining%20ability%20and%20parental%20effects%20in%204x%E2%80%902x%20crosses%20for%20potato%20breeding&amp;author=R.%20Ortiz&amp;author=M.%20Iwanaga&amp;author=H.%20A.%20Mendoza&amp;volume=31&amp;publication_year=1988&amp;pages=643-650&amp;doi=10.1007/BF02361857&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0047">
<cite>

Ortiz, R.
, 
Reslow, F.
, 
Montesinos‐López, A.
, 
Huicho, J.
, 
Pérez‐Rodríguez, P.
, 
Montesinos‐López, O. A.
, &amp; 
Crossa, J.
 (2023). Partial least squares enhance multi‐trait genomic prediction of potato cultivars in new environments. Scientific Reports, 13(1), Article 9947. 10.1038/s41598-023-37169-y

</cite> [<a href="https://doi.org/10.1038/s41598-023-37169-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10279678/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37336933/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Partial%20least%20squares%20enhance%20multi%E2%80%90trait%20genomic%20prediction%20of%20potato%20cultivars%20in%20new%20environments&amp;author=R.%20Ortiz&amp;author=F.%20Reslow&amp;author=A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=J.%20Huicho&amp;author=P.%20P%C3%A9rez%E2%80%90Rodr%C3%ADguez&amp;volume=13&amp;issue=1&amp;publication_year=2023&amp;pages=9947&amp;pmid=37336933&amp;doi=10.1038/s41598-023-37169-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0048">
<cite>

Pandey, J.
, 
Scheuring, D. C.
, 
Koym, J. W.
, 
Endelman, J. B.
, &amp; 
Vales, M. I.
 (2023). Genomic selection and genome‐wide association studies in tetraploid chipping potatoes. Plant Genome, 16, e20297. 10.1002/tpg2.20297

</cite> [<a href="https://doi.org/10.1002/tpg2.20297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36651146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Genome&amp;title=Genomic%20selection%20and%20genome%E2%80%90wide%20association%20studies%20in%20tetraploid%20chipping%20potatoes&amp;author=J.%20Pandey&amp;author=D.%20C.%20Scheuring&amp;author=J.%20W.%20Koym&amp;author=J.%20B.%20Endelman&amp;author=M.%20I.%20Vales&amp;volume=16&amp;publication_year=2023&amp;pages=e20297&amp;pmid=36651146&amp;doi=10.1002/tpg2.20297&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0049">
<cite>

Pandey, J.
, 
Scheuring, D. C.
, 
Koym, J. W.
, &amp; 
Vales, M. I.
 (2022). Genomic regions associated with tuber traits in tetraploid potatoes and identification of superior clones for breeding purposes. Frontiers in Plant Science, 13, 952263. 10.3389/fpls.2022.952263

</cite> [<a href="https://doi.org/10.3389/fpls.2022.952263" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9354404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35937326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genomic%20regions%20associated%20with%20tuber%20traits%20in%20tetraploid%20potatoes%20and%20identification%20of%20superior%20clones%20for%20breeding%20purposes&amp;author=J.%20Pandey&amp;author=D.%20C.%20Scheuring&amp;author=J.%20W.%20Koym&amp;author=M.%20I.%20Vales&amp;volume=13&amp;publication_year=2022&amp;pages=952263&amp;pmid=35937326&amp;doi=10.3389/fpls.2022.952263&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0050">
<cite>

Parisod, C.
, 
Holderegger, R.
, &amp; 
Brochmann, C.
 (2010). Evolutionary consequences of autopolyploidy. The New Phytologist, 186, 5–17. 10.1111/j.1469-8137.2009.03142.x

</cite> [<a href="https://doi.org/10.1111/j.1469-8137.2009.03142.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20070540/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20New%20Phytologist&amp;title=Evolutionary%20consequences%20of%20autopolyploidy&amp;author=C.%20Parisod&amp;author=R.%20Holderegger&amp;author=C.%20Brochmann&amp;volume=186&amp;publication_year=2010&amp;pages=5-17&amp;pmid=20070540&amp;doi=10.1111/j.1469-8137.2009.03142.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0051">
<cite>

Plaisted, R. L.
, 
Sanford, L.
, 
Federer, W. T.
, 
Kehr, A. E.
, &amp; 
Peterson, L. C.
 (1962). Specific and general combining ability specific and general combining ability for yield in potatoes. American Potato Journal, 39, 185–197. 10.1007/BF02871402
</cite> [<a href="https://doi.org/10.1007/BF02871402" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Potato%20Journal&amp;title=Specific%20and%20general%20combining%20ability%20specific%20and%20general%20combining%20ability%20for%20yield%20in%20potatoes&amp;author=R.%20L.%20Plaisted&amp;author=L.%20Sanford&amp;author=W.%20T.%20Federer&amp;author=A.%20E.%20Kehr&amp;author=L.%20C.%20Peterson&amp;volume=39&amp;publication_year=1962&amp;pages=185-197&amp;doi=10.1007/BF02871402&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0052"><cite>
Posit Team
. (2023). 
RStudio: Integrated development environment for R
.</cite></li>
<li id="tpg270066-bib-0053">
<cite>

Provan, J.
, 
Powell, W.
, 
Dewar, H.
, 
Bryan, G.
, 
Machray, G. C.
, &amp; 
Waugh, R.
 (1999). An extreme cytoplasmic bottleneck in the modern European cultivated potato (<em>Solanum tuberosum</em>) is not reflected in decreased levels of nuclear diversity. Proceedings of the Royal Society of London. Series B: Biological Sciences, 266, 633–639. 10.1098/rspb.1999.0683
</cite> [<a href="https://doi.org/10.1098/rspb.1999.0683" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20Royal%20Society%20of%20London.%20Series%20B:%20Biological%20Sciences&amp;title=An%20extreme%20cytoplasmic%20bottleneck%20in%20the%20modern%20European%20cultivated%20potato%20(Solanum%20tuberosum)%20is%20not%20reflected%20in%20decreased%20levels%20of%20nuclear%20diversity&amp;author=J.%20Provan&amp;author=W.%20Powell&amp;author=H.%20Dewar&amp;author=G.%20Bryan&amp;author=G.%20C.%20Machray&amp;volume=266&amp;publication_year=1999&amp;pages=633-639&amp;doi=10.1098/rspb.1999.0683&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0054"><cite>
R Core Team
. (2023). 
R: A language and environment for statistical computing
.</cite></li>
<li id="tpg270066-bib-0055">
<cite>

Rorabacher, D. B.
 (1991). Statistical treatment for rejection of deviant values: Critical value of Dixon's “Q” parameter and related subrange ratios at the 95% confidence level. Analytical Chemistry, 63, 139–146.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Statistical%20treatment%20for%20rejection%20of%20deviant%20values:%20Critical%20value%20of%20Dixon's%20%E2%80%9CQ%E2%80%9D%20parameter%20and%20related%20subrange%20ratios%20at%20the%2095%%20confidence%20level&amp;author=D.%20B.%20Rorabacher&amp;volume=63&amp;publication_year=1991&amp;pages=139-146&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0056">
<cite>

Ruiz De Arcaute, R.
, 
Carrasco, A.
, 
Ortega, F.
, 
Rodriguez‐Quijano, M.
, &amp; 
Carrillo, J. M.
 (2022). Evaluation of genetic resources in a potato breeding program for chip quality. Agronomy, 12(5), 1142. 10.3390/agronomy12051142
</cite> [<a href="https://doi.org/10.3390/agronomy12051142" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=Evaluation%20of%20genetic%20resources%20in%20a%20potato%20breeding%20program%20for%20chip%20quality&amp;author=R.%20Ruiz%20De%20Arcaute&amp;author=A.%20Carrasco&amp;author=F.%20Ortega&amp;author=M.%20Rodriguez%E2%80%90Quijano&amp;author=J.%20M.%20Carrillo&amp;volume=12&amp;issue=5&amp;publication_year=2022&amp;pages=1142&amp;doi=10.3390/agronomy12051142&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0057">
<cite>

Sanetomo, R.
, &amp; 
Gebhardt, C.
 (2015). Cytoplasmic genome types of European potatoes and their effects on complex agronomic traits. BMC Plant Biology, 15, Article 162. 10.1186/S12870-015-0545-Y

</cite> [<a href="https://doi.org/10.1186/S12870-015-0545-Y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4480903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26112802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Cytoplasmic%20genome%20types%20of%20European%20potatoes%20and%20their%20effects%20on%20complex%20agronomic%20traits&amp;author=R.%20Sanetomo&amp;author=C.%20Gebhardt&amp;volume=15&amp;publication_year=2015&amp;pages=162&amp;pmid=26112802&amp;doi=10.1186/S12870-015-0545-Y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0058">
<cite>

Santayana, M.
, 
Aponte, M.
, 
Kante, M.
, 
Eyzaguirre, R.
, 
Gastelo, M.
, &amp; 
Lindqvist‐Kreuze, H.
 (2022). Cytoplasmic male sterility incidence in potato breeding populations with late blight resistance and identification of breeding lines with a potential fertility restorer mechanism. Plants, 11(22), 3093. 10.3390/PLANTS11223093

</cite> [<a href="https://doi.org/10.3390/PLANTS11223093" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9696232/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36432822/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Cytoplasmic%20male%20sterility%20incidence%20in%20potato%20breeding%20populations%20with%20late%20blight%20resistance%20and%20identification%20of%20breeding%20lines%20with%20a%20potential%20fertility%20restorer%20mechanism&amp;author=M.%20Santayana&amp;author=M.%20Aponte&amp;author=M.%20Kante&amp;author=R.%20Eyzaguirre&amp;author=M.%20Gastelo&amp;volume=11&amp;issue=22&amp;publication_year=2022&amp;pages=3093&amp;pmid=36432822&amp;doi=10.3390/PLANTS11223093&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0059">
<cite>

Schirmer, M.
, 
Höchstötter, A.
, 
Jekle, M.
, 
Arendt, E.
, &amp; 
Becker, T.
 (2013). Physicochemical and morphological characterization of different starches with variable amylose/amylopectin ratio. Food Hydrocolloids, 32, 52–63. 10.1016/j.foodhyd.2012.11.032
</cite> [<a href="https://doi.org/10.1016/j.foodhyd.2012.11.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Hydrocolloids&amp;title=Physicochemical%20and%20morphological%20characterization%20of%20different%20starches%20with%20variable%20amylose/amylopectin%20ratio&amp;author=M.%20Schirmer&amp;author=A.%20H%C3%B6chst%C3%B6tter&amp;author=M.%20Jekle&amp;author=E.%20Arendt&amp;author=T.%20Becker&amp;volume=32&amp;publication_year=2013&amp;pages=52-63&amp;doi=10.1016/j.foodhyd.2012.11.032&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0079">
<cite>

Schreiber, L.
, 
Nader‐Nieto, A. C.
, 
Schönhals, E. M.
, 
Walkemeier, B.
, &amp; 
Gebhardt, C.
 (2014). SNPs in genes functional in starch‐sugar interconversion associate with natural variation of tuber starch and sugar content of potato (<em>Solanum tuberosum</em> L.). G3 Genes|Genomes|Genetics, 4(10), 1797–1811. 10.1534/g3.114.012377

</cite> [<a href="https://doi.org/10.1534/g3.114.012377" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4199688/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25081979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=SNPs%20in%20genes%20functional%20in%20starch%E2%80%90sugar%20interconversion%20associate%20with%20natural%20variation%20of%20tuber%20starch%20and%20sugar%20content%20of%20potato%20(Solanum%20tuberosum%20L.)&amp;author=L.%20Schreiber&amp;author=A.%20C.%20Nader%E2%80%90Nieto&amp;author=E.%20M.%20Sch%C3%B6nhals&amp;author=B.%20Walkemeier&amp;author=C.%20Gebhardt&amp;volume=4&amp;issue=10&amp;publication_year=2014&amp;pages=1797-1811&amp;pmid=25081979&amp;doi=10.1534/g3.114.012377&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0060">
<cite>

Slater, A. T.
, 
Cogan, N. O. I.
, 
Forster, J. W.
, 
Hayes, B. J.
, &amp; 
Daetwyler, H. D.
 (2016). Improving genetic gain with genomic selection in autotetraploid potato. Plant Genome, 9, plantgenome2016.02.0021. 10.3835/PLANTGENOME2016.02.0021
</cite> [<a href="https://doi.org/10.3835/PLANTGENOME2016.02.0021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27902807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Genome&amp;title=Improving%20genetic%20gain%20with%20genomic%20selection%20in%20autotetraploid%20potato&amp;author=A.%20T.%20Slater&amp;author=N.%20O.%20I.%20Cogan&amp;author=J.%20W.%20Forster&amp;author=B.%20J.%20Hayes&amp;author=H.%20D.%20Daetwyler&amp;volume=9&amp;publication_year=2016&amp;pmid=27902807&amp;doi=10.3835/PLANTGENOME2016.02.0021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0061">
<cite>

Śliwka, J.
, 
Wasilewicz‐Flis, I.
, 
Jakuczun, H.
, &amp; 
Gebhardt, C.
 (2008). Tagging quantitative trait loci for dormancy, tuber shape, regularity of tuber shape, eye depth and flesh colour in diploid potato originated from six <em>Solanum</em> species. Plant Breeding, 127, 49–55. 10.1111/j.1439-0523.2008.01420.x
</cite> [<a href="https://doi.org/10.1111/j.1439-0523.2008.01420.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Breeding&amp;title=Tagging%20quantitative%20trait%20loci%20for%20dormancy,%20tuber%20shape,%20regularity%20of%20tuber%20shape,%20eye%20depth%20and%20flesh%20colour%20in%20diploid%20potato%20originated%20from%20six%20Solanum%20species&amp;author=J.%20%C5%9Aliwka&amp;author=I.%20Wasilewicz%E2%80%90Flis&amp;author=H.%20Jakuczun&amp;author=C.%20Gebhardt&amp;volume=127&amp;publication_year=2008&amp;pages=49-55&amp;doi=10.1111/j.1439-0523.2008.01420.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0062">
<cite>

Song, H.
, 
Zhang, J.
, 
Zhang, Q.
, &amp; 
Ding, X.
 (2019). Using different single‐step strategies to improve the efficiency of genomic prediction on body measurement traits in pig. Frontiers in Genetics, 10. 10.3389/fgene.2018.00730
</cite> [<a href="https://doi.org/10.3389/fgene.2018.00730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6340005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30693018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Using%20different%20single%E2%80%90step%20strategies%20to%20improve%20the%20efficiency%20of%20genomic%20prediction%20on%20body%20measurement%20traits%20in%20pig&amp;author=H.%20Song&amp;author=J.%20Zhang&amp;author=Q.%20Zhang&amp;author=X.%20Ding&amp;volume=10&amp;publication_year=2019&amp;pmid=30693018&amp;doi=10.3389/fgene.2018.00730&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0063">
<cite>

Spooner, D. M.
, 
Mclean, K.
, 
Ramsay, G.
, 
Waugh, R.
, &amp; 
Bryan, G. J.
 (2005). A single domestication for potato based on multilocus amplified fragment length polymorphism genotyping. Proceedings of the National Academy of Sciences, 102, 14694–14699. 10.1073/PNAS.0507400102
</cite> [<a href="https://doi.org/10.1073/PNAS.0507400102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1253605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16203994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=A%20single%20domestication%20for%20potato%20based%20on%20multilocus%20amplified%20fragment%20length%20polymorphism%20genotyping&amp;author=D.%20M.%20Spooner&amp;author=K.%20Mclean&amp;author=G.%20Ramsay&amp;author=R.%20Waugh&amp;author=G.%20J.%20Bryan&amp;volume=102&amp;publication_year=2005&amp;pages=14694-14699&amp;pmid=16203994&amp;doi=10.1073/PNAS.0507400102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0064">
<cite>

Stich, B.
, &amp; 
Van Inghelandt, D.
 (2018). Prospects and potential uses of genomic prediction of key performance traits in tetraploid potato. Frontiers in Plant Science, 9, Article 159. 10.3389/fpls.2018.00159

</cite> [<a href="https://doi.org/10.3389/fpls.2018.00159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5845909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29563919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Prospects%20and%20potential%20uses%20of%20genomic%20prediction%20of%20key%20performance%20traits%20in%20tetraploid%20potato&amp;author=B.%20Stich&amp;author=D.%20Van%20Inghelandt&amp;volume=9&amp;publication_year=2018&amp;pages=159&amp;pmid=29563919&amp;doi=10.3389/fpls.2018.00159&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0065">
<cite>

Sverrisdóttir, E.
, 
Byrne, S.
, 
Sundmark, E. H. R.
, 
Johnsen, H. Ø.
, 
Kirk, H. G.
, 
Asp, T.
, 
Janss, L.
, &amp; 
Nielsen, K. L.
 (2017). Genomic prediction of starch content and chipping quality in tetraploid potato using genotyping‐by‐sequencing. Theoretical and Applied Genetics, 130, 2091–2108. 10.1007/s00122-017-2944-y

</cite> [<a href="https://doi.org/10.1007/s00122-017-2944-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5606954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28707250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genomic%20prediction%20of%20starch%20content%20and%20chipping%20quality%20in%20tetraploid%20potato%20using%20genotyping%E2%80%90by%E2%80%90sequencing&amp;author=E.%20Sverrisd%C3%B3ttir&amp;author=S.%20Byrne&amp;author=E.%20H.%20R.%20Sundmark&amp;author=H.%20%C3%98.%20Johnsen&amp;author=H.%20G.%20Kirk&amp;volume=130&amp;publication_year=2017&amp;pages=2091-2108&amp;pmid=28707250&amp;doi=10.1007/s00122-017-2944-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0066">
<cite>

Sverrisdóttir, E.
, 
Sundmark, E. H. R.
, 
Johnsen, H. Ø.
, 
Kirk, H. G.
, 
Asp, T.
, 
Janss, L.
, 
Bryan, G.
, &amp; 
Nielsen, K. L.
 (2018). The value of expanding the training population to improve genomic selection models in tetraploid potato. Frontiers in Plant Science, 9, Article 1118. 10.3389/fpls.2018.01118

</cite> [<a href="https://doi.org/10.3389/fpls.2018.01118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6090097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30131817/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=The%20value%20of%20expanding%20the%20training%20population%20to%20improve%20genomic%20selection%20models%20in%20tetraploid%20potato&amp;author=E.%20Sverrisd%C3%B3ttir&amp;author=E.%20H.%20R.%20Sundmark&amp;author=H.%20%C3%98.%20Johnsen&amp;author=H.%20G.%20Kirk&amp;author=T.%20Asp&amp;volume=9&amp;publication_year=2018&amp;pages=1118&amp;pmid=30131817&amp;doi=10.3389/fpls.2018.01118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0067">
<cite>

Terres, L. R.
, 
Lenz, E. A.
, 
Rocha, D.
, 
Cerioli, M.
, &amp; 
Pereira, A. D. S.
 (2017). Combining ability of potato parents for tuber appearance and tuber yield component traits. Crop Breeding and Applied Biotechnology, 17, 99–106. 10.1590/1984-70332017V17N2A16
</cite> [<a href="https://doi.org/10.1590/1984-70332017V17N2A16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Breeding%20and%20Applied%20Biotechnology&amp;title=Combining%20ability%20of%20potato%20parents%20for%20tuber%20appearance%20and%20tuber%20yield%20component%20traits&amp;author=L.%20R.%20Terres&amp;author=E.%20A.%20Lenz&amp;author=D.%20Rocha&amp;author=M.%20Cerioli&amp;author=A.%20D.%20S.%20Pereira&amp;volume=17&amp;publication_year=2017&amp;pages=99-106&amp;doi=10.1590/1984-70332017V17N2A16&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0068">
<cite>

Tong, C.
, 
Ma, Z.
, 
Chen, H.
, &amp; 
Gao, H.
 (2023). Toward an understanding of potato starch structure, function, biosynthesis, and applications. Food Frontiers, 4, 980–1000. 10.1002/fft2.223
</cite> [<a href="https://doi.org/10.1002/fft2.223" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Frontiers&amp;title=Toward%20an%20understanding%20of%20potato%20starch%20structure,%20function,%20biosynthesis,%20and%20applications&amp;author=C.%20Tong&amp;author=Z.%20Ma&amp;author=H.%20Chen&amp;author=H.%20Gao&amp;volume=4&amp;publication_year=2023&amp;pages=980-1000&amp;doi=10.1002/fft2.223&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0069">
<cite>

Urbany, C.
, 
Stich, B.
, 
Schmidt, L.
, 
Simon, L.
, 
Berding, H.
, 
Junghans, H.
, 
Niehoff, K.‐H.
, 
Braun, A.
, 
Tacke, E.
, 
Hofferbert, H.‐R.
, 
Lübeck, J.
, 
Strahwald, J.
, &amp; 
Gebhardt, C.
 (2011). Association genetics in <em>Solanum tuberosum</em> provides new insights into potato tuber bruising and enzymatic tissue discoloration. BMC Genomics, 12, Article 7. 10.1186/1471-2164-12-7

</cite> [<a href="https://doi.org/10.1186/1471-2164-12-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3023753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21208436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Association%20genetics%20in%20Solanum%20tuberosum%20provides%20new%20insights%20into%20potato%20tuber%20bruising%20and%20enzymatic%20tissue%20discoloration&amp;author=C.%20Urbany&amp;author=B.%20Stich&amp;author=L.%20Schmidt&amp;author=L.%20Simon&amp;author=H.%20Berding&amp;volume=12&amp;publication_year=2011&amp;pages=7&amp;pmid=21208436&amp;doi=10.1186/1471-2164-12-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0070">
<cite>

Van Berloo, R.
, 
Hutten, R. C. B.
, 
Van Eck, H. J.
, &amp; 
Visser, R. G. F.
 (2007). An online potato pedigree database resource. Potato Research, 50, 45–57. 10.1007/S11540-007-9028-3
</cite> [<a href="https://doi.org/10.1007/S11540-007-9028-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Potato%20Research&amp;title=An%20online%20potato%20pedigree%20database%20resource&amp;author=R.%20Van%20Berloo&amp;author=R.%20C.%20B.%20Hutten&amp;author=H.%20J.%20Van%20Eck&amp;author=R.%20G.%20F.%20Visser&amp;volume=50&amp;publication_year=2007&amp;pages=45-57&amp;doi=10.1007/S11540-007-9028-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0071">
<cite>

Van Dijk, M.
, 
Morley, T.
, 
Rau, M. L.
, &amp; 
Saghai, Y.
 (2021). A meta‐analysis of projected global food demand and population at risk of hunger for the period 2010–2050. Nature Food, 2, 494–501. 10.1038/s43016-021-00322-9

</cite> [<a href="https://doi.org/10.1038/s43016-021-00322-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37117684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Food&amp;title=A%20meta%E2%80%90analysis%20of%20projected%20global%20food%20demand%20and%20population%20at%20risk%20of%20hunger%20for%20the%20period%202010%E2%80%932050&amp;author=M.%20Van%20Dijk&amp;author=T.%20Morley&amp;author=M.%20L.%20Rau&amp;author=Y.%20Saghai&amp;volume=2&amp;publication_year=2021&amp;pages=494-501&amp;pmid=37117684&amp;doi=10.1038/s43016-021-00322-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0072">
<cite>

van Eck, H. J.
 (2007). Genetics of morphological and tuber traits. In Vreugdenhil D. (Ed.), Potato biology and biotechnology: Advances and perspectives (1st ed., pp. 91–115). Elsevier.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Potato%20biology%20and%20biotechnology:%20Advances%20and%20perspectives&amp;author=H.%20J.%20van%20Eck&amp;publication_year=2007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0073">
<cite>

Van Eck, H. J.
, 
Jacobs, J. M.
, 
Stam, P.
, 
Ton, J.
, 
Stiekema, W. J.
, &amp; 
Jacobsen, E.
 (1994). Multiple alleles for tuber shape in diploid potato detected by qualitative and quantitative genetic analysis using RFLPs. Genetics, 137, 303–309. 10.1093/GENETICS/137.1.303

</cite> [<a href="https://doi.org/10.1093/GENETICS/137.1.303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1205946/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7914504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Multiple%20alleles%20for%20tuber%20shape%20in%20diploid%20potato%20detected%20by%20qualitative%20and%20quantitative%20genetic%20analysis%20using%20RFLPs&amp;author=H.%20J.%20Van%20Eck&amp;author=J.%20M.%20Jacobs&amp;author=P.%20Stam&amp;author=J.%20Ton&amp;author=W.%20J.%20Stiekema&amp;volume=137&amp;publication_year=1994&amp;pages=303-309&amp;pmid=7914504&amp;doi=10.1093/GENETICS/137.1.303&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0074">
<cite>

VanRaden, P. M.
 (2008). Efficient methods to compute genomic predictions. Journal of Dairy Science, 91, 4414–4423. 10.3168/JDS.2007-0980

</cite> [<a href="https://doi.org/10.3168/JDS.2007-0980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18946147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Dairy%20Science&amp;title=Efficient%20methods%20to%20compute%20genomic%20predictions&amp;author=P.%20M.%20VanRaden&amp;volume=91&amp;publication_year=2008&amp;pages=4414-4423&amp;pmid=18946147&amp;doi=10.3168/JDS.2007-0980&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0075">
<cite>

Wickham, H.
 (2016). ggplot2: Elegant graphics for data analysis. Springer‐Verlag.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=ggplot2:%20Elegant%20graphics%20for%20data%20analysis&amp;author=H.%20Wickham&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0076">
<cite>

Wilson, S.
, 
Zheng, C.
, 
Maliepaard, C.
, 
Mulder, H. A.
, 
Visser, R. G. F.
, 
Van Der Burgt, A.
, &amp; 
Van Eeuwijk, F.
 (2021). Understanding the effectiveness of genomic prediction in tetraploid potato. Frontiers in Plant Science, 12, Article 672417. 10.3389/fpls.2021.672417

</cite> [<a href="https://doi.org/10.3389/fpls.2021.672417" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8381724/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34434201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Understanding%20the%20effectiveness%20of%20genomic%20prediction%20in%20tetraploid%20potato&amp;author=S.%20Wilson&amp;author=C.%20Zheng&amp;author=C.%20Maliepaard&amp;author=H.%20A.%20Mulder&amp;author=R.%20G.%20F.%20Visser&amp;volume=12&amp;publication_year=2021&amp;pages=672417&amp;pmid=34434201&amp;doi=10.3389/fpls.2021.672417&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0077">
<cite>

Wu, S.
, 
Zhang, B.
, 
Keyhaninejad, N.
, 
Rodríguez, G. R.
, 
Kim, H. J.
, 
Chakrabarti, M.
, 
Illa‐Berenguer, E.
, 
Taitano, N. K.
, 
Gonzalo, M. J.
, 
Díaz, A.
, 
Pan, Y.
, 
Leisner, C. P.
, 
Halterman, D.
, 
Buell, C. R.
, 
Weng, Y.
, 
Jansky, S. H.
, 
Van Eck, H.
, 
Willemsen, J.
, 
Monforte, A. J.
, … 
Van Der Knaap, E.
 (2018). A common genetic mechanism underlies morphological diversity in fruits and other plant organs. Nature Communications, 9(1), Article 4734. 10.1038/s41467-018-07216-8
</cite> [<a href="https://doi.org/10.1038/s41467-018-07216-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6226536/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30413711/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=A%20common%20genetic%20mechanism%20underlies%20morphological%20diversity%20in%20fruits%20and%20other%20plant%20organs&amp;author=S.%20Wu&amp;author=B.%20Zhang&amp;author=N.%20Keyhaninejad&amp;author=G.%20R.%20Rodr%C3%ADguez&amp;author=H.%20J.%20Kim&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pages=4734&amp;pmid=30413711&amp;doi=10.1038/s41467-018-07216-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270066-bib-0078">
<cite>

Wu, W.
, 
Yu, Q.
, 
You, L.
, 
Chen, K.
, 
Tang, H.
, &amp; 
Liu, J.
 (2018). Global cropping intensity gaps: Increasing food production without cropland expansion. Land Use Policy, 76, 515–525. 10.1016/j.landusepol.2018.02.032
</cite> [<a href="https://doi.org/10.1016/j.landusepol.2018.02.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Land%20Use%20Policy&amp;title=Global%20cropping%20intensity%20gaps:%20Increasing%20food%20production%20without%20cropland%20expansion&amp;author=W.%20Wu&amp;author=Q.%20Yu&amp;author=L.%20You&amp;author=K.%20Chen&amp;author=H.%20Tang&amp;volume=76&amp;publication_year=2018&amp;pages=515-525&amp;doi=10.1016/j.landusepol.2018.02.032&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Additional supporting information can be found online in the Supplemental Material section at the end of this article. These include supplementary figures (<strong>Supplemental Material S1</strong>) and supplementary tables (<strong>Supplemental Material S2</strong>).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12207579/bin/TPG2-18-e70066-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70066-s001.docx</a><sup> (2.4MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>Supplemental Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12207579/bin/TPG2-18-e70066-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70066-s002.docx</a><sup> (31.7KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets needed to reproduce the results of this study are available in the Zenodo repositor: <a href="https://doi.org/10.5281/zenodo.11916143" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.11916143</a>. These include that marker and pedigree data of the population, as well as recorded phenotypic and metadata.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70066"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70066.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12207579/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12207579/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12207579%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12207579/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12207579/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12207579/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40583826/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12207579/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40583826/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12207579/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12207579/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="qK9YAvMQEhK9Q4Xu9C4vZczNYfB7zVMSO9uFVi9F1JDmY11yUEQrPHhMKmkraPmE">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
