
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Machine learning‐driven GWAS uncovers novel candidate genes for resistance to frosty pod rot and witches&#x27; broom disease in cacao - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="FF20802B89B714031A802B001F9F16A3.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12245736/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Machine learning‐driven GWAS uncovers novel candidate genes for resistance to frosty pod rot and witches&#x27; broom disease in cacao">
<meta name="citation_author" content="Ezekiel Ahn">
<meta name="citation_author_institution" content="USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA">
<meta name="citation_author" content="Sunchung Park">
<meta name="citation_author_institution" content="USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA">
<meta name="citation_author" content="Insuck Baek">
<meta name="citation_author_institution" content="USDA‐ARS, Agriculture, Environmental Microbial and Food Safety Laboratory, Beltsville, Maryland, USA">
<meta name="citation_author" content="Dongho Lee">
<meta name="citation_author_institution" content="USDA‐ARS, Environmental Microbial and Food Safety Laboratory, Beltsville, Maryland, USA">
<meta name="citation_author" content="Jishnu Bhatt">
<meta name="citation_author_institution" content="USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA">
<meta name="citation_author" content="Seunghyun Lim">
<meta name="citation_author_institution" content="USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA">
<meta name="citation_author" content="Jae Hee Jang">
<meta name="citation_author_institution" content="USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA">
<meta name="citation_author" content="Dapeng Zhang">
<meta name="citation_author_institution" content="USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA">
<meta name="citation_author" content="Moon S Kim">
<meta name="citation_author_institution" content="USDA‐ARS, Agriculture, Environmental Microbial and Food Safety Laboratory, Beltsville, Maryland, USA">
<meta name="citation_author" content="Lyndel W Meinhardt">
<meta name="citation_author_institution" content="USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA">
<meta name="citation_publication_date" content="2025 Jul 10">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70069">
<meta name="citation_doi" content="10.1002/tpg2.70069">
<meta name="citation_pmid" content="40640111">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12245736/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12245736/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12245736/pdf/TPG2-18-e70069.pdf">
<meta name="description" content="Cacao (Theobroma cacao), the source of chocolate, is threatened by devastating diseases like frosty pod rot (FPR) and witches&#x27; broom disease (WBD), impacting global production and farmer livelihoods. Here, we employ a machine learning‐driven ...">
<meta name="og:title" content="Machine learning‐driven GWAS uncovers novel candidate genes for resistance to frosty pod rot and witches&#x27; broom disease in cacao">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Cacao (Theobroma cacao), the source of chocolate, is threatened by devastating diseases like frosty pod rot (FPR) and witches&#x27; broom disease (WBD), impacting global production and farmer livelihoods. Here, we employ a machine learning‐driven ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12245736/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12245736">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70069"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70069.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12245736%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12245736/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12245736/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12245736/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70069" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Jul 10;18(3):e70069. doi: <a href="https://doi.org/10.1002/tpg2.70069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70069</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Machine learning‐driven GWAS uncovers novel candidate genes for resistance to frosty pod rot and witches' broom disease in cacao</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahn%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Ezekiel Ahn</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Ezekiel Ahn</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahn%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Ezekiel Ahn</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Sunchung Park</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Sunchung Park</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sunchung Park</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baek%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Insuck Baek</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Insuck Baek</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>USDA‐ARS, Agriculture, Environmental Microbial and Food Safety Laboratory, Beltsville, Maryland, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baek%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Insuck Baek</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Dongho Lee</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Dongho Lee</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>USDA‐ARS, Environmental Microbial and Food Safety Laboratory, Beltsville, Maryland, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dongho Lee</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhatt%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Jishnu Bhatt</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jishnu Bhatt</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhatt%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jishnu Bhatt</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lim%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Seunghyun Lim</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Seunghyun Lim</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lim%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Seunghyun Lim</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jang%20JH%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jae Hee Jang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jae Hee Jang</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jang%20JH%22%5BAuthor%5D" class="usa-link"><span class="name western">Jae Hee Jang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Dapeng Zhang</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Dapeng Zhang</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dapeng Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20MS%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Moon S Kim</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Moon S Kim</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>USDA‐ARS, Agriculture, Environmental Microbial and Food Safety Laboratory, Beltsville, Maryland, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20MS%22%5BAuthor%5D" class="usa-link"><span class="name western">Moon S Kim</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meinhardt%20LW%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Lyndel W Meinhardt</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Lyndel W Meinhardt</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meinhardt%20LW%22%5BAuthor%5D" class="usa-link"><span class="name western">Lyndel W Meinhardt</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270069-aff-0001">
<sup>
<sup>1</sup>
</sup>USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, Maryland, USA</div>
<div id="tpg270069-aff-0002">
<sup>
<sup>2</sup>
</sup>USDA‐ARS, Agriculture, Environmental Microbial and Food Safety Laboratory, Beltsville, Maryland, USA</div>
<div id="tpg270069-aff-0003">
<sup>
<sup>3</sup>
</sup>USDA‐ARS, Environmental Microbial and Food Safety Laboratory, Beltsville, Maryland, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Ezekiel Ahn, USDA‐ARS, Sustainable Perennial Crops Laboratory, Beltsville, MD 20705, USA. Email: <span>ezekiel.ahn@usda.gov</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 May 5; Received 2025 Jan 30; Accepted 2025 Jun 2; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12245736  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40640111/" class="usa-link">40640111</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Cacao (<em>Theobroma cacao</em>), the source of chocolate, is threatened by devastating diseases like frosty pod rot (FPR) and witches' broom disease (WBD), impacting global production and farmer livelihoods. Here, we employ a machine learning‐driven genome‐wide association study to dissect the genetic architecture of disease resistance and productivity in cacao. Upon analyzing phenotypic data for healthy pod rate along with FPR and WBD resistance across 102 diverse accessions, coupled with single nucleotide polymorphism (SNP) markers mapped to the Criollo and Matina reference genomes, we identified numerous novel candidate genes. These genes are implicated in various biological processes, including cell wall modification, stress response signaling, and defense‐related mechanisms. Notably, associations varied between the reference genomes, highlighting the genomic complexity of these traits. Our analyses, using Bootstrap Forest and Boosted Tree models, uncovered loci not previously reported, demonstrating the power of machine learning in uncovering complex genetic interactions. This study offers important insights into the polygenic nature of disease resistance in cacao and presents a genomic roadmap for developing disease‐resistant varieties.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270069-list-10001" class="list" style="list-style-type:disc">
<li><p>Machine learning genome‐wide association studies (GWAS) identified novel genes for frosty pod rot (FPR) and witches' broom disease (WBD) resistance in cacao.</p></li>
<li><p>Candidate genes implicate cell wall, stress response, and defense in polygenic disease resistance.
</p></li>
<li><p>Varied associations in Criollo and Matina genomes portray complexity of resistance traits.</p></li>
<li><p>This study provides genomic roadmap to breed cacao for resistance to diseases like frosty pod rot (FPR) and WBD.</p></li>
<li><p>Combined GWAS results demonstrate diverse processes in cacao productivity and disease resistance.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270069-dl-0001">
<dt>AUDPC</dt>
<dd><p>area under the disease progress curve</p></dd>
<dt>BF</dt>
<dd><p>Bootstrap Forest</p></dd>
<dt>BT</dt>
<dd><p>Boosted Tree</p></dd>
<dt>FPR</dt>
<dd><p>frosty pod rot</p></dd>
<dt>GBS</dt>
<dd><p>genotyping‐by‐sequencing</p></dd>
<dt>GO</dt>
<dd><p>Gene Ontology</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association study</p></dd>
<dt>IBS</dt>
<dd><p>identity by state</p></dd>
<dt>MAS</dt>
<dd><p>marker‐assisted selection</p></dd>
<dt>ML</dt>
<dd><p>machine learning</p></dd>
<dt>PC1</dt>
<dd><p>first principal component</p></dd>
<dt>PCA</dt>
<dd><p>principal component analysis</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
<dt>t‐SNE</dt>
<dd><p>t‐distributed stochastic neighbor embedding</p></dd>
<dt>WBD</dt>
<dd><p>witches' broom disease</p></dd>
</dl>
<section id="tpg270069-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>As the source of cocoa, the globally significant perennial tree crop <em>Theobroma cacao</em> L. underpins the multi‐billion dollar chocolate industry and supports the livelihoods of millions of smallholder farmers in developing nations, significantly contributing to their national economies (Díaz‐Montenegro, <a href="#tpg270069-bib-0014" class="usa-link" aria-describedby="tpg270069-bib-0014">2019</a>; Hall, <a href="#tpg270069-bib-0020" class="usa-link" aria-describedby="tpg270069-bib-0020">2022</a>). However, many small‐scale producers face economic challenges that threaten the long‐term sustainability of their operations (Manrique et al., <a href="#tpg270069-bib-0029" class="usa-link" aria-describedby="tpg270069-bib-0029">2024</a>). Achieving a competitive advantage in the global market is crucial for the long‐term viability of the cacao agro‐industry, particularly in regions like Southeast Sulawesi, Indonesia, where factors such as market orientation and supply chain flexibility play a significant role (Hatani et al., <a href="#tpg270069-bib-0022" class="usa-link" aria-describedby="tpg270069-bib-0022">2016</a>). Beyond its economic importance, sustainable cacao cultivation contributes to biodiversity conservation and carbon sequestration (Obeng &amp; Aguilar, <a href="#tpg270069-bib-0035" class="usa-link" aria-describedby="tpg270069-bib-0035">2015</a>). The specialty cacao industry, in particular, has been recognized for its potential to enhance farmer welfare and environmental resource conservation through premium pricing and direct trade models (Cadby &amp; Araki, <a href="#tpg270069-bib-0008" class="usa-link" aria-describedby="tpg270069-bib-0008">2021</a>). However, the future of cacao production faces mounting challenges, including the impacts of climate change (Ceccarelli et al., <a href="#tpg270069-bib-0009" class="usa-link" aria-describedby="tpg270069-bib-0009">2021</a>), volatile market prices (Villacis et al., <a href="#tpg270069-bib-0044" class="usa-link" aria-describedby="tpg270069-bib-0044">2022</a>), and the persistent threat of devastating diseases (Evans, <a href="#tpg270069-bib-0015" class="usa-link" aria-describedby="tpg270069-bib-0015">2007</a>). Addressing these challenges is paramount, not only for ensuring the long‐term viability of the chocolate industry but also for safeguarding the livelihoods of farmers and promoting sustainable development in cacao‐growing regions.</p>
<p>Among biotic constraints, diseases exert significant pressure on cacao productivity worldwide. Two of the most economically impactful diseases are frosty pod rot (FPR), caused by the hemibiotrophic fungus <em>Moniliophthora roreri</em>, and witches' broom disease (WBD), caused by <em>Moniliophthora perniciosa</em> (Phillips‐Mora et al., <a href="#tpg270069-bib-0037" class="usa-link" aria-describedby="tpg270069-bib-0037">2007</a>; Purdy &amp; Schmidt, <a href="#tpg270069-bib-0039" class="usa-link" aria-describedby="tpg270069-bib-0039">1996</a>). FPR is characterized by its destructive pod rot and has been reported to cause yield losses of up to 80% in some regions, making it a major threat to cacao production in Latin America (Phillips‐Mora et al., <a href="#tpg270069-bib-0037" class="usa-link" aria-describedby="tpg270069-bib-0037">2007</a>). WBD, a systemic disease affecting various plant tissues, causes severe yield decline and, in extreme cases, tree mortality. WBD, first described in the late 1700s, has a long history of impacting cacao production in South America and presents ongoing management challenges (Purdy &amp; Schmidt, <a href="#tpg270069-bib-0039" class="usa-link" aria-describedby="tpg270069-bib-0039">1996</a>). Since its initial description in the late 18th century, WBD has caused dramatic shifts in cacao production, including Brazil's fall from a leading exporter to a net importer following a severe outbreak in the late 1980s (Meinhardt et al., <a href="#tpg270069-bib-0031" class="usa-link" aria-describedby="tpg270069-bib-0031">2008</a>).</p>
<p>Traditional disease management approaches rely heavily on cultural practices, sanitation, and occasional fungicide use; unfortunately, these methods often prove insufficient or too costly to implement, particularly for resource‐constrained smallholder farmers (Purdy &amp; Schmidt, <a href="#tpg270069-bib-0039" class="usa-link" aria-describedby="tpg270069-bib-0039">1996</a>). Therefore, the development and deployment of disease‐resistant cacao varieties are increasingly recognized as a critical and sustainable solution for mitigating the impact of FPR and WBD. Harnessing genetic resistance offers a powerful and environmentally sound strategy for managing cacao diseases (Poland et al., <a href="#tpg270069-bib-0038" class="usa-link" aria-describedby="tpg270069-bib-0038">2012</a>). While sources of resistance to both FPR and WBD have been identified within the vast and largely untapped genetic diversity of cacao germplasm, the introgression of these traits into elite, high‐yielding varieties through conventional breeding remains a slow and arduous process that often takes decades (Bartley, <a href="#tpg270069-bib-0004" class="usa-link" aria-describedby="tpg270069-bib-0004">2005</a>; Gutiérrez et al., <a href="#tpg270069-bib-0019" class="usa-link" aria-describedby="tpg270069-bib-0019">2021</a>; McElroy et al., <a href="#tpg270069-bib-0030" class="usa-link" aria-describedby="tpg270069-bib-0030">2018</a>; Osorio‐Guarín et al., <a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">2020</a>; Romero Navarro et al., <a href="#tpg270069-bib-0040" class="usa-link" aria-describedby="tpg270069-bib-0040">2017</a>). Additionally, understanding the genetic diversity and biogeography of pathogens like <em>M. roreri</em> is crucial for developing effective and durable resistance strategies (Phillips‐Mora et al., <a href="#tpg270069-bib-0037" class="usa-link" aria-describedby="tpg270069-bib-0037">2007</a>).</p>
<p>The emergence of genomics, however, has revolutionized plant breeding, providing unprecedented opportunities to accelerate the development of improved varieties (Lahive et al., <a href="#tpg270069-bib-0026" class="usa-link" aria-describedby="tpg270069-bib-0026">2019</a>). The release of the first cacao reference genome (Criollo) in 2011 (Argout et al., <a href="#tpg270069-bib-0002" class="usa-link" aria-describedby="tpg270069-bib-0002">2017</a>), followed by a second (Matina) in 2013 (Motamayor et al., <a href="#tpg270069-bib-0032" class="usa-link" aria-describedby="tpg270069-bib-0032">2013</a>), has established fundamental genomic resources for cacao research, paving the way for marker‐assisted selection (MAS) and other genomic‐assisted breeding approaches. While MAS holds immense potential for enhancing the efficiency and precision of cacao breeding programs, challenges to its effective implementation remain, particularly in managing QTL stability and integrating MAS into traditional breeding practices (Hospital, <a href="#tpg270069-bib-0024" class="usa-link" aria-describedby="tpg270069-bib-0024">2009</a>).</p>
<p>Genome‐wide association studies (GWAS) have become a cornerstone of modern plant genetics, providing a powerful means to dissect the complex genetic architecture of polygenic traits (Brachi et al., <a href="#tpg270069-bib-0005" class="usa-link" aria-describedby="tpg270069-bib-0005">2011</a>). By leveraging high‐density single nucleotide polymorphism (SNP) markers distributed across the entire genome, GWAS can identify genomic regions and, ultimately, candidate genes that contribute to phenotypic variation (Brachi et al., <a href="#tpg270069-bib-0005" class="usa-link" aria-describedby="tpg270069-bib-0005">2011</a>). In cacao, GWAS has been applied to identify loci associated with yield components, bean quality, aroma, and resistance to specific diseases (Colonges et al., <a href="#tpg270069-bib-0013" class="usa-link" aria-describedby="tpg270069-bib-0013">2022</a>; Wickramasuriya &amp; Dunwell, <a href="#tpg270069-bib-0045" class="usa-link" aria-describedby="tpg270069-bib-0045">2018</a>). The availability of the cacao genome sequences has facilitated these efforts, enabling the development of high‐density SNP arrays and the identification of candidate genes. However, previous studies have often been constrained by limited scope, focusing primarily on single diseases or narrow germplasm collections.</p>
<p>A more holistic approach is crucial for a comprehensive understanding of the genetic groundwork of these complex traits. This should encompass a broader spectrum of genetic diversity and a simultaneous examination of multiple traits related to productivity and stress resistance, including both abiotic (e.g., drought and heat) and biotic stresses (e.g., pathogen‐triggered stress) (Lahive et al., <a href="#tpg270069-bib-0026" class="usa-link" aria-describedby="tpg270069-bib-0026">2019</a>). Recent research has highlighted the polygenic nature of traits such as WBD resistance, identifying multiple QTLs with minor effects and revealing the intricate host‐pathogen interactions involved (Chia Wong et al., <a href="#tpg270069-bib-0012" class="usa-link" aria-describedby="tpg270069-bib-0012">2022</a>). For instance, Romero Navarro et al. (<a href="#tpg270069-bib-0040" class="usa-link" aria-describedby="tpg270069-bib-0040">2017</a>) applied GWAS and genomic prediction to a population of improved cacao clones, locating markers associated with resistance against FPR and black pod rot as well as yield‐related traits. Similarly, Osorio‐Guarín et al. (<a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">2020</a>) conducted a GWAS on a diverse collection of 102 cacao accessions to ascertain candidate genes for productivity and disease resistance using traditional statistical methods.</p>
<p>Building on this foundational work, our study employed advanced machine learning (ML) techniques, including Bootstrap Forest (BF) (Random Forest) and Boosted Tree (BT) models (Breiman, <a href="#tpg270069-bib-0007" class="usa-link" aria-describedby="tpg270069-bib-0007">2001</a>; Chen, <a href="#tpg270069-bib-0011" class="usa-link" aria-describedby="tpg270069-bib-0011">2014</a>), to re‐analyze the phenotypic and genotypic data generated by Osorio‐Guarín et al. (<a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">2020</a>). These methods are particularly suited to handle high‐dimensional data and capturing complex interactions between genes and environmental factors. We further conducted a comparative analysis using both the Criollo and Matina reference genomes to assess the influence of reference genome choice on SNP discovery and association mapping. We hypothesized that the genetic architecture of productivity and disease resistance in cacao involves numerous loci with small‐to‐moderate effects and that the choice of reference genome can significantly impact SNP‐trait associations. By integrating GWAS with advanced ML techniques, we aimed to provide a deeper understanding of the genetic basis of healthy pod rate and resistance to FPR and WBD. The results of this comprehensive study provide valuable new insights for cacao breeding programs and serve as a valuable resource for future functional validation studies.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270069-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270069-list-0001" class="list" style="list-style-type:disc">
<li><p>Machine learning genome‐wide association studies (GWAS) identified novel genes for frosty pod rot (FPR) and witches' broom disease (WBD) resistance in cacao.</p></li>
<li><p>Candidate genes implicate cell wall, stress response, and defense in polygenic disease resistance.
</p></li>
<li><p>Varied associations in Criollo and Matina genomes portray complexity of resistance traits.</p></li>
<li><p>This study provides genomic roadmap to breed cacao for resistance to diseases like frosty pod rot (FPR) and WBD.</p></li>
<li><p>Combined GWAS results demonstrate diverse processes in cacao productivity and disease resistance.</p></li>
</ul></section></section></section><section id="tpg270069-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270069-sec-0040"><h3 class="pmc_sec_title">2.1. Genetic distance and phenotypic correlation analysis</h3>
<p>To investigate the relationship between genetic relatedness and phenotypic variation among the 229 cacao accessions, we performed a correlation analysis using genetic distances and phenotypic data. Genetic distances were estimated using the identity by state (IBS) method implemented in the TASSEL 5 software (Bradbury et al., <a href="#tpg270069-bib-0006" class="usa-link" aria-describedby="tpg270069-bib-0006">2007</a>), which calculated the proportion of alleles shared between each pair of individuals across all marker loci, providing a measure of genetic similarity. Two sets of genome‐wide SNP markers were used for this analysis: 9003 SNPs mapped to the Criollo reference genome (Argout et al., <a href="#tpg270069-bib-0002" class="usa-link" aria-describedby="tpg270069-bib-0002">2017</a>) and 8131 SNPs mapped to the Matina reference genome (Motamayor et al., <a href="#tpg270069-bib-0032" class="usa-link" aria-describedby="tpg270069-bib-0032">2013</a>). These SNP datasets were obtained from the genotyping‐by‐sequencing (GBS) data previously generated and described by Osorio‐Guarín et al. (<a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">2020</a>). As reported for this dataset mapped to the Criollo genome, the average distance between markers was 39.8 kb, ranging from 31.1 to 43.3 kb depending on the chromosome (Osorio‐Guarín et al., <a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">2020</a>). The IBS calculations resulted in two separate distance matrices, denoted as “IBS‐Matina” and “IBS‐Criollo,” corresponding to the reference genome used for SNP mapping.</p>
<p>Phenotypic data for four traits of 102 accessions related to cacao productivity and disease resistance were also obtained from Osorio‐Guarín et al. (<a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">2020</a>). These traits included: (1) healthy pod rate, calculated as the number of healthy pods divided by the total number of pods for each accession; (2) pods with FPR, quantified using the area under the disease progress curve (AUDPC) to reflect the severity of FPR caused by <em>M. roreri</em>; (3) flower with cushion WBD, representing the AUDPC of flower cushions infected with WBD caused by <em>M. perniciosa</em>; and (4) branches with WBD, representing the AUDPC of branches exhibiting WBD symptoms. The AUDPC was calculated as described by Shaner and Finney (<a href="#tpg270069-bib-0043" class="usa-link" aria-describedby="tpg270069-bib-0043">1977</a>).</p>
<p>Pearson's correlation coefficients were computed between each genetic distance matrix (IBS‐Matina and IBS‐Criollo) and each of the four phenotypic traits using the JMP Pro 17 software (SAS Institute Inc.; Klimberg, <a href="#tpg270069-bib-0025" class="usa-link" aria-describedby="tpg270069-bib-0025">2023</a>). This analysis assessed the degree to which genetic similarity, as measured by IBS, was associated with phenotypic similarity for each trait.</p></section><section id="tpg270069-sec-0050"><h3 class="pmc_sec_title">2.2. Hierarchical clustering analysis</h3>
<p>To explore the structure within the genotypic and phenotypic data, we performed hierarchical clustering analyses using JMP Pro 17. Two distinct clustering approaches were employed. First, we conducted genotype‐based clustering separately for each reference genome (Criollo and Matina) using the SNP datasets described previously. For each reference genome, a distance matrix was calculated based on the SNP data using the Ward method (Murtagh &amp; Legendre, <a href="#tpg270069-bib-0034" class="usa-link" aria-describedby="tpg270069-bib-0034">2014</a>), an agglomerative hierarchical clustering algorithm implemented in JMP Pro 17. Second, we performed clustering based on a combination of genotype and phenotype data. The phenotypic data for the four traits (healthy pod rate, FPR, flower cushions with WBD, and branches with WBD) were combined with the SNP data for each reference genome (Criollo and Matina). A distance matrix was calculated for each combined dataset (Criollo SNPs + phenotypes, Matina SNPs + phenotypes) using the Ward method in JMP Pro 17. The Ward method (Murtagh &amp; Legendre, <a href="#tpg270069-bib-0034" class="usa-link" aria-describedby="tpg270069-bib-0034">2014</a>) was then applied to cluster the accessions based on their combined genotypic and phenotypic profiles.</p></section><section id="tpg270069-sec-0060"><h3 class="pmc_sec_title">2.3. t‐Distributed stochastic neighbor embedding analysis</h3>
<p>To further explore and visualize the population structure within the genotypic data, we performed t‐distributed stochastic neighbor embedding (t‐SNE) analyses using JMP Pro 17. t‐SNE is a nonlinear dimensionality reduction technique that aims to preserve local neighborhood relationships in high‐dimensional data when reducing it to a lower dimensional representation, typically two or three dimensions (Hamid &amp; Sugumaran, <a href="#tpg270069-bib-0021" class="usa-link" aria-describedby="tpg270069-bib-0021">2020</a>). Separate t‐SNE analyses were conducted for each reference genome (Criollo and Matina) using the corresponding SNP data. Analyses were performed using JMP Pro 17′s default settings, which include a perplexity value of 30, a learning rate (epsilon) of 10, and a maximum of 1,000 iterations.</p></section><section id="tpg270069-sec-0070"><h3 class="pmc_sec_title">2.4. Genome‐wide association study using ML</h3>
<p>To identify genomic regions associated with variation in four phenotypic traits (healthy pod rate, pods exhibiting FPR, flower cushions with WBD, and branches affected by WBD), GWAS were conducted using two ML algorithms: BF and BT, implemented in JMP Pro 17. These tree‐based ensemble methods were chosen for their potential ability to capture nonlinear relationships and complex interactions (epistasis) between SNPs, aspects of genetic architecture that may not be fully accounted for by standard additive linear models often used in GWAS. These analyses were carried out separately for each trait based on SNP data mapped to both the Criollo and Matina reference genomes. Population structure, such as principal components or an estimated Q‐matrix, was not explicitly included as a covariate in these models. For the BF method, we used the following settings: number of trees in the forest = 100, number of terms sampled per split = 1 (this is the square root of the number of predictors, rounded down, and is the default setting), bootstrap sample rate = 1, minimum splits per tree = 10, maximum splits per tree = 2000, and minimum size split = 5. For the BT method, we employed the following settings: learning rate = 0.1, minimum size split = 5, maximum layers = 6, number of layers to estimate = 20, and sampling rate = 1. The importance of each SNP in predicting the trait of interest was assessed using the “Portion” statistic, representing the relative contribution of each SNP to the overall model precision, with higher values indicating greater importance. The results of these analyses are visualized in Manhattan plots, which display the logarithm of the exponentiated Portion scores (Log‐Exp(Importance score)) for each SNP along the respective chromosomes/scaffolds. For identifying candidate regions associated with each trait and model combination, SNPs were ranked based on their importance scores (“Portion”). The top five SNPs with the highest importance scores were selected for further investigation and candidate gene identification (Tables <a href="#tpg270069-tbl-0001" class="usa-link">1</a>, <a href="#tpg270069-tbl-0002" class="usa-link">2</a>, <a href="#tpg270069-tbl-0003" class="usa-link">3</a> and Table <a href="#tpg270069-supinfo-0001" class="usa-link">S1</a>).</p>
<section class="tw xbox font-sm" id="tpg270069-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Candidate genes associated with cacao productivity and disease resistance identified by top single nucleotide polymorphisms (SNPs) from Bootstrap Forest genome‐wide association studies (GWAS) using the Criollo reference genome.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">Chr</th>
<th align="left" rowspan="1" colspan="1">SNP position</th>
<th align="left" rowspan="1" colspan="1">Nearest genes and function</th>
<th align="left" rowspan="1" colspan="1">GO annotation</th>
<th align="left" rowspan="1" colspan="1">Importance score (Portion)</th>
</tr></thead>
<tbody>
<tr><td colspan="6" align="left" rowspan="1">Bootstrap Forest</td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">Healthy pod rate</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">7754541, 7754552</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc05v2_g008540</em>
</p>
<p>Secoisolariciresinol dehydrogenase (7730282–7731554)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0317</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">3271174</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc03v2_g003210</em>
</p>
<p>Uncharacterized protein isoform 2 (3301477–3307744)</p>
</td>
<td align="left" rowspan="1" colspan="1">Cellular component, membrane</td>
<td align="left" rowspan="1" colspan="1">0.0252</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">13135179</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc08v2_g013610</em>
</p>
<p>Beta‐fructofuranosidase, insoluble isoenzyme 1 (12892301–12893596)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.019</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">10063324</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc05v2_g009360</em>
</p>
<p>Uncharacterized protein</p>
<p>(10072093–10084247)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0147</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">20378993</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc05v2_g010930</em>
</p>
<p>Gag protease polyprotein‐like protein (20381149–20381871)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0129</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Pods with FPR</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">25139428</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc03v2_g009640</em>
</p>
<p>Probable galactinol–sucrose galactosyltransferase 2 (25135345–25139665)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0643</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">23881358</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc09v2_g017680</em>
</p>
<p>DEAD‐box ATP‐dependent RNA helicase 37 (23889018–23895419)</p>
</td>
<td align="left" rowspan="1" colspan="1">Hydrolase activity, mRNA binding, ATP binding</td>
<td align="left" rowspan="1" colspan="1">0.0278</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">18355124, 18285303, 18354976</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc01v2_g019210</em>
</p>
<p>Uncharacterized protein (18311296–18312102)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0201</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">36944112</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc01v2_g034430</em>
</p>
<p>Uncharacterized protein (36937551–36945789)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0158</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">823888</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc06v2_g000820</em>
</p>
<p>Putative lectin (814417–818887)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0156</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Flower with cushion WBD</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">770436</td>
<td align="left" rowspan="1" colspan="1">
<p>
<strong>
<em>Tc02v2_g001090</em>
</strong>
</p>
<p>Probable xyloglucan galactosyltransferase GT17 (768191–770560)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.1528</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1064432</td>
<td align="left" rowspan="1" colspan="1">
<p>
<strong>
<em>Tc02v2_g001650</em>
</strong>
</p>
<p>Ion channel DMI1 (1060675–1066653)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.142</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">21174545, 21174462</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc01v2_g019770</em>
</p>
<p>Ferrochelatase‐2, chloroplastic (21204419–21219868)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.061</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">31472241</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc01v2_g025400</em>
</p>
<p>Putative exopolygalacturonase clone GBGE184 (31467814–31469699)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0498</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">4523324</td>
<td align="left" rowspan="1" colspan="1">
<p>
<strong>
<em>Tc02v2_g007330</em>
</strong>
</p>
<p>MLO‐like protein 4 (4521274–4527984)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0272</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Branches with WBD</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">10714823</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc06v2_g005450</em>
</p>
<p>Putative NAC domain‐containing protein 35 (10785856–10788539)</p>
</td>
<td align="left" rowspan="1" colspan="1">Transcription regulation, DNA binding</td>
<td align="left" rowspan="1" colspan="1">0.0361</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">1133400</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc09v2_g001870</em>
</p>
<p>Topless‐related protein 1 (1136104–1143084)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0237</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">2544954</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc10v2_g004160</em>
</p>
<p>Cysteine/Histidine‐rich C1 domain family protein, putative (2556402–2564757)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0204</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">3689811, 3689626</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc07v2_g005490</em>
</p>
<p>G‐type lectin S‐receptor‐like serine/threonine‐protein kinase At4g27290 (3679662–3684095)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0167</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">19646083</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc10v2_g012370</em>
</p>
<p>Putative cytosolic sulfotransferase 15 (19634059–19635764)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0128</td>
</tr>
<tr><td colspan="6" align="left" rowspan="1">Boosted Tree</td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">Healthy pod rate</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">7754541</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc05v2_g008540</em>
</p>
<p>Secoisolariciresinol dehydrogenase (7730282–7731554)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.1168</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">13135179</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc08v2_g013610</em>
</p>
<p>Beta‐fructofuranosidase, insoluble isoenzyme 1 (12892301–12893596)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0943</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">9516723</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc05v2_g009250</em>
</p>
<p>Uncharacterized protein (9552088–9552591)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0725</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">10954840</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc07v2_g012560</em>
</p>
<p>Uncharacterized protein (10937178–10937777)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0709</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">3271174</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc03v2_g003210</em>
</p>
<p>Uncharacterized protein isoform 2 (3301477–3307744)</p>
</td>
<td align="left" rowspan="1" colspan="1">Cellular Component, membrane</td>
<td align="left" rowspan="1" colspan="1">0.059</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Pods with FPR</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">23881358</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc09v2_g017680</em>
</p>
<p>DEAD‐box ATP‐dependent RNA helicase 37 (23889018–23895419)</p>
</td>
<td align="left" rowspan="1" colspan="1">Hydrolase activity, mRNA binding, ATP binding</td>
<td align="left" rowspan="1" colspan="1">0.1818</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">6853962</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc06v2_g004520</em>
</p>
<p>Long chain base biosynthesis protein 1 (6864200–6873051)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.1615</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">24715162</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc01v2_g020390</em>
</p>
<p>Uncharacterized protein (24734066–24734969)</p>
</td>
<td align="left" rowspan="1" colspan="1">Cellular component, membrane</td>
<td align="left" rowspan="1" colspan="1">0.0821</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">25139428</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc03v2_g009640</em>
</p>
<p>Probable galactinol–sucrose galactosyltransferase 2 (25135345–25139665)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0744</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">36944112</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc01v2_g034430</em>
</p>
<p>Uncharacterized protein (36937551–36945789)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.038</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Flower with cushion WBD</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">770436</td>
<td align="left" rowspan="1" colspan="1">
<p>
<strong>
<em>Tc02v2_g001090</em>
</strong>
</p>
<p>Probable xyloglucan galactosyltransferase GT17 (768191–770560)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.2104</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1064432</td>
<td align="left" rowspan="1" colspan="1">
<p>
<strong>
<em>Tc02v2_g001650</em>
</strong>
</p>
<p>Ion channel DMI1 (1060675–1066653)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.2054</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">21174462</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc01v2_g019770</em>
</p>
<p>Ferrochelatase‐2, chloroplastic (21204419–21219868)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.1364</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">23747515</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc01v2_g020200</em>
</p>
<p>Uncharacterized protein (23778724–23781550)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0681</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">31472241</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc01v2_g025400</em>
</p>
<p>Putative exopolygalacturonase clone GBGE184 (31467814–31469699)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0479</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Branches with WBD</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">10714823</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc06v2_g005450</em>
</p>
<p>Putative NAC domain‐containing protein 35 (10785856–10788539)</p>
</td>
<td align="left" rowspan="1" colspan="1">Transcription regulation, DNA binding</td>
<td align="left" rowspan="1" colspan="1">0.1549</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">32199021</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc01v2_g026530</em>
</p>
<p>RNA‐directed DNA polymerase (reverse transcriptase), ribonuclease H, putative (32199109–32200980)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.1119</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">1133400</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc09v2_g001870</em>
</p>
<p>Topless‐related protein 1 (1136104–1143084)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0899</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">11181343</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc01v2_g014750</em>
</p>
<p>Carbohydrate‐binding X8 domain superfamily protein, putative (11171940–11173636)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0547</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">6697943</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc04v2_g005150</em>
</p>
<p>Bifunctional protein FolD 1, mitochondrial (6714249–6720336)</p>
</td>
<td align="left" rowspan="1" colspan="1">Methenyltetrahydrofolate cyclohydrolase activity, methylenetetrahydrofolate dehydrogenase, tetrahydrofolate interconversion</td>
<td align="left" rowspan="1" colspan="1">0.0428</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270069-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270069-tbl1-note-0001"><p>
<em>Note</em>: SNPs are ranked by importance score (Portion). The table includes SNP identifier, chromosome (Chr), position, nearest gene (with genomic location), putative gene function, Gene Ontology (GO) annotation (if available), and importance score. Traits include healthy pod rate, pods with frosty pod rot (FPR), and flowers or branches with witches' broom disease (WBD). Genes in bold were also detected in the original GWAS conducted by Osorio‐Guarín et al.(<a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">2020</a>) on the same dataset.</p></div></div></section><section class="tw xbox font-sm" id="tpg270069-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Top single nucleotide polymorphisms (SNPs) from Bootstrap Forest and Boosted Tree genome‐wide association studies (GWAS) of cacao productivity and disease resistance using the Matina reference genome.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">Scaffold</th>
<th align="left" rowspan="1" colspan="1">SNP position</th>
<th align="left" rowspan="1" colspan="1">Nearest genes and function</th>
<th align="left" rowspan="1" colspan="1">GO annotation</th>
<th align="left" rowspan="1" colspan="1">Importance score (Portion)</th>
</tr></thead>
<tbody>
<tr><td colspan="6" align="left" rowspan="1">Bootstrap Forest</td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">Healthy pod rate</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">12534250</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG023287</em>
</p>
<p>PEBP (phosphatidylethanolamine‐binding protein) family protein (12598359–12601737)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0459</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">3505073</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG012520</em>
</p>
<p>Uncharacterized protein DUF599 (3535034–3541320)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0348</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">10101385</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG023050</em>
</p>
<p>RING/U‐box superfamily protein (10076217–10092402)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0225</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">21539308</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG024023</em>
</p>
<p>Uncharacterized protein, chromo (CHRromatin Organisation MOdifier) domain containing (21306383–21649104)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0132</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">11327144</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG023171</em>
</p>
<p>Uncharacterized protein (11271058–11294483)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0129</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Pods with FPR</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">22872047</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG014511</em>
</p>
<p>Hydrolase, Galactinol–sucrose galactosyltransferase / raffinose synthase (22867979–22872961)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0615</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">38575461</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG005862</em>
</p>
<p>Uncharacterized protein, PTHR12444//PTHR12444:SF0 ‐ uncharacterized // protein EFR3 homolog CMP44E (38570037–38577182)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0333</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">18890898</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG003073</em>
</p>
<p>Uncharacterized protein (18887461–18892499)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0264</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">27162270</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG003843</em>
</p>
<p>Gag‐protease‐integrase‐RT‐RNaseH polyprotein (27173246–27453604)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0262</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">3947337</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG000880</em>
</p>
<p>Uncharacterized protein (3946411–3948266)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.021</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Flower with cushion WBD</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">794296</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG006100</em>
</p>
<p>Leucine‐rich repeat‐containing protein (794098–800964)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.1478</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1107779</td>
<td align="left" rowspan="1" colspan="1">
<p>
<strong>
<em>Thecc1EG006180</em>
</strong>
</p>
<p>Uncharacterized protein, castor and pollux, part of voltage‐gated ion channel (1104002–1111419)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.1402</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">33051561</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG004701</em>
</p>
<p>Uncharacterized protein, reverse transcriptase‐like (RVT_3) // zinc‐binding in reverse transcriptase (zf‐RVT) (33049209–33052664)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0673</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">4567399</td>
<td align="left" rowspan="1" colspan="1">
<p>
<strong>
<em>Thecc1EG006939</em>
</strong>
</p>
<p>Seven transmembrane MLO family protein (4565183–4572400)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0594</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">2210089</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG030945</em>
</p>
<p>RNA‐binding CRS1 / YhbY domain‐containing protein (2204788–2210633)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.043</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Branches with WBD</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">1122735</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG036963</em>
</p>
<p>TOPLESS‐related 1 (1118542–1132915)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0355</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">8932900</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG027932</em>
</p>
<p>Uncharacterized protein (8920672–8943551)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0325</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">2638347</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG042851</em>
</p>
<p>Uncharacterized protein</p>
<p>(2585353–2812295)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0285</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">3661030</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG046864</em>
</p>
<p>S‐locus lectin protein kinase family protein (3663242–3667965)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0254</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">20317281</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG039538</em>
</p>
<p>Uncharacterized protein</p>
<p>(20318482–20318929)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0206</td>
</tr>
<tr><td colspan="6" align="left" rowspan="1">Boosted Tree</td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">Healthy pod rate</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">11327144</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG023171</em>
</p>
<p>Uncharacterized protein (11271058–11294483)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.1182</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">10101385</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG023050</em>
</p>
<p>RING/U‐box superfamily protein (10076217–10092402)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.1006</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">3505073</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG012520</em>
</p>
<p>Uncharacterized protein DUF599 (3535034–3541320)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0968</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">7438454</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG001487</em>
</p>
<p>Ankyrin repeat‐containing protein (7436699–7439430)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0673</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">8343325</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG017880</em>
</p>
<p>Uncharacterized protein, PF00385 ‐ Chromo (CHRromatin Organisation MOdifier) domain (Chromo) (8340704–9011110)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0288</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Pods with FPR</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">7186038</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG027776</em>
</p>
<p>Uncharacterized protein, troponin coil‐coiled subunits containing (7189328–7189674)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.1886</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">22872047</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG014511</em>
</p>
<p>Hydrolase, Galactinol–sucrose galactosyltransferase / raffinose synthase (22867979–22872961)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.1796</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">9466348</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG035323</em>
</p>
<p>Dead box ATP‐dependent RNA helicase (8092198–10317647)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0471</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">4938189</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG012671</em>
</p>
<p>21 kDa seed protein, trypsin and protease inhibitor (4976844–4993230)</p>
</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">0.0326</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">23097582</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG024191</em>
</p>
<p>DegP protease 10, serine protease family S1C HTRA‐related (23061719–23081514)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0321</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Flower with cushion WBD</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">33051561</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG004701</em>
</p>
<p>Uncharacterized protein, reverse transcriptase‐like (RVT_3) // zinc‐binding in reverse transcriptase (zf‐RVT) (33049209–33052664)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.2189</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1107779</td>
<td align="left" rowspan="1" colspan="1">
<p>
<strong>
<em>Thecc1EG006180</em>
</strong>
</p>
<p>Uncharacterized protein, castor and pollux, part of voltage‐gated ion channel (1104002–1111419)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.2071</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">794296</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG006100</em>
</p>
<p>Leucine‐rich repeat‐containing protein (794098–800964)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.127</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">6787809</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG022711</em>
</p>
<p>Uncharacterized protein (6796784–6802454)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0804</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">23054194</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG019515</em>
</p>
<p>Retrotransposon gag protein (23051547–23053027)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0452</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Branches with WBD</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">28258940</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG040241</em>
</p>
<p>RNA‐dependent RNA polymerase 2 (28257011–28264572)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.1129</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">8932900</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG027932</em>
</p>
<p>Uncharacterized protein (8920672–8943551)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.1047</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">1122735</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG036963</em>
</p>
<p>TOPLESS‐related 1 (1118542–1132915)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0962</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">12670234</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG023288</em>
</p>
<p>Uncharacterized protein (2604989–12617304)</p>
</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">0.0907</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">20317281</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG039538</em>
</p>
<p>Uncharacterized protein</p>
<p>(20318482–20318929)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.0631</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270069-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270069-tbl2-note-0001"><p>
<em>Note</em>: The table includes SNP identifier, scaffold, position, nearest gene (with genomic location), putative gene function, and importance score. Traits analyzed include healthy pod rate, pods with FPR, flowers with cushion WBD, and branches with WBD. GO annotation was not available for any of the listed genes. Genes in bold were also detected in the original GWAS conducted by Osorio‐Guarín et al.(<a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">2020</a>) on the same dataset.</p></div></div></section><section class="tw xbox font-sm" id="tpg270069-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Concordant candidate gene associations for cacao disease resistance and productivity traits identified by both ML genome‐wide association studies (GWAS) (current study) and traditional GWAS (Osorio‐Guarín et al., <a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">2020</a>).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">Chr</th>
<th align="left" rowspan="1" colspan="1">SNP position</th>
<th align="left" rowspan="1" colspan="1">Nearest genes and function</th>
<th align="left" rowspan="1" colspan="1">GO annotation</th>
<th align="left" rowspan="1" colspan="1">Reference genome</th>
<th align="left" rowspan="1" colspan="1">Model</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<p>Flower with cushion WBD</p>
<p>All traits combined</p>
<p>PC1</p>
</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1064432</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc02v2_g001650</em>
</p>
<p>Ion channel DMI1 (1060675–1066653)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">Criollo</td>
<td align="left" rowspan="1" colspan="1">BT, BF</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<p>Flower with cushion WBD</p>
<p>All traits combined</p>
<p>PC1</p>
</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">770436</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc02v2_g001090</em>
</p>
<p>Probable xyloglucan galactosyltransferase GT17 (768191–770560)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">Criollo</td>
<td align="left" rowspan="1" colspan="1">BT, BF</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<p>Flower with cushion WBD</p>
<p>All traits combined</p>
</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">4523324</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Tc02v2_g007330</em>
</p>
<p>MLO‐like protein 4 (4521274–4527984)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">Criollo</td>
<td align="left" rowspan="1" colspan="1">BF</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<p>Flower with cushion WBD</p>
<p>All traits combined</p>
<p>PC1</p>
</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1107779</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG006180</em>
</p>
<p>Uncharacterized protein, castor and pollux, part of voltage‐gated ion channel (1104002–1111419)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">Matina</td>
<td align="left" rowspan="1" colspan="1">BT, BF</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<p>Flower with cushion WBD</p>
<p>All traits combined</p>
</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">4567399</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG006939</em>
</p>
<p>Seven transmembrane MLO family protein (4565183–4572400)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">Matina</td>
<td align="left" rowspan="1" colspan="1">BT, BF</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Pods with FPR</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">9466348</td>
<td align="left" rowspan="1" colspan="1">
<p>
<em>Thecc1EG035323</em>
</p>
<p>Dead box ATP‐dependent RNA helicase (8092198–10317647)</p>
</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">Matina</td>
<td align="left" rowspan="1" colspan="1">BT</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270069-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270069-tbl3-note-0001"><p>Abbreviations: BF, Bootstrap Forest; BT, Boosted Tree; FPR, frosty pod rot; ML, machine learning; PC1, first principal component; WBD, witches' broom disease.</p></div></div></section><p>To evaluate the potential predictive performance and stability of different modeling approaches given the dataset characteristics (102 accessions, &gt; 8000 SNPs), a preliminary <em>K</em> = 5 cross‐validation was performed using the “Model Screening” platform in JMP Pro 17. This analysis compared K‐nearest neighbors, BF, support vector machines (RBF kernel), BT, and Decision Tree algorithms using default settings. The results indicated that while BF and BT models often achieved high explanatory power on the training sets (average training R‐squared across the 5 folds frequently exceeding 0.80 and sometimes over 0.90, depending on the trait and reference genome, e.g., 0.88 for BF and 0.93 for BT for Flower Cushion WBD), performance on the corresponding validation sets was highly variable and generally low (validation R‐squared values ranging from negative values up to over 0.5 [Decision tree and BF for flower with cushion WBD in Matina genome], but mostly around 0.20 or less). This instability in validation performance was attributed to the limited sample size relative to the number of predictors, making robust estimation on hold‐out data unreliable for assessing predictive accuracy or SNP importance. Therefore, to maximize the stability of SNP importance estimation for identifying potentially relevant loci within this specific dataset, the final BF and BT models used for generating the importance rankings reported in this study were trained using the entire available dataset.</p>
<p>To investigate the combined genetic effects on the four traits, we conducted two additional GWAS analyses using composite phenotypic measures derived from the original traits. First, we created a new variable representing the product of the four traits for each accession, calculated by multiplying the values of healthy pod rate, pods with FPR, flower cushions with WBD, and branches with WBD. Second, we performed a principal component analysis (PCA) on the four traits using JMP Pro 17 to extract the first principal component (PC1). As PC1 captures the largest proportion of shared variance among the measured traits, we utilized it as a composite phenotype in an exploratory GWAS aimed at identifying loci associated with the primary axis of phenotypic variation, potentially reflecting overall plant performance across these productivity and disease resistance metrics or underlying pleiotropic genetic effects. Subsequently, GWAS was carried out using both the product variable and PC1 as the response variables, following the BF and BT procedures, utilizing both the Criollo (<a href="https://cocoa-genome-hub.southgreen.fr/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cocoa‐genome‐hub.southgreen.fr/</a>) and Matina (<a href="https://phytozome-next.jgi.doe.gov/info/Tcacao_v1_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phytozome‐next.jgi.doe.gov/info/Tcacao_v1_1</a>) datasets. Gene annotations, including Gene Ontology (GO) terms and putative functions, were obtained from UniProt Knowledgebase (UniProtKB; <a href="https://www.uniprot.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.uniprot.org/</a>).</p></section><section id="tpg270069-sec-0080"><h3 class="pmc_sec_title">2.5. Correlation analysis of SNP importance scores</h3>
<p>To assess the consistency of SNP associations across different traits and ML models, we performed a correlation analysis using the SNP importance scores derived from the GWAS. For each reference genome (Criollo and Matina), we extracted the importance score (Portion) for each SNP from each of the eight analyses (4 traits × 2 models). This resulted in a matrix of SNP importance scores, where each row represented an SNP and each column represented a specific trait and model combination (e.g., Healthy pod rate—BF, Pods with FPR—BT).</p>
<p>Pearson's correlation coefficients were then calculated between all pairs of columns in the importance score matrix using the “Multivariate” platform within JMP Pro 17. The resulting correlation coefficients, which range from −1 to +1, reflect the degree and direction of the linear relationship between the SNP importance scores for different traits and models. A positive correlation indicates that SNPs with high importance for one trait/model tend to have high importance for the other trait/model, while a negative correlation indicates the opposite.</p></section></section><section id="tpg270069-sec-0090"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270069-sec-0100"><h3 class="pmc_sec_title">3.1. Weak correlations between genetic distance and phenotypic traits in cacao</h3>
<p>To investigate the relationship between genetic relatedness and phenotypic variation in our cacao collection, we calculated Pearson's correlation coefficients between genetic distances and four key traits: healthy pod rate, pods with FPR, flower cushions with WBD, and branches with WBD (Figure <a href="#tpg270069-fig-0001" class="usa-link">1</a>). Genetic distances were computed using the IBS method based on SNP markers mapped to two different reference genomes, Matina (IBS‐Matina) and Criollo (IBS‐Criollo). Overall, the correlations between genetic distance and the four traits were generally weak (Figure <a href="#tpg270069-fig-0001" class="usa-link">1</a>). The two genetic distance measures, IBS‐Matina and IBS‐Criollo, were highly correlated with each other (<em>r</em> = 0.98, <em>p</em> &lt; 0.0001), indicating that the choice of reference genome had a minimal impact on the overall assessment of genetic relationships among these accessions. Healthy pod rate showed weak, nonsignificant positive correlations with both IBS‐Matina (<em>r</em> = 0.11, <em>p</em> &gt; 0.05) and IBS‐Criollo (<em>r</em> = 0.13, <em>p</em> &gt; 0.05). This suggests that genetic similarity, as measured by IBS, is not a strong predictor of healthy pod production in this collection.</p>
<figure class="fig xbox font-sm" id="tpg270069-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bd2/12245736/a23563c170d3/TPG2-18-e70069-g003.jpg" loading="lazy" id="jats-graphic-1" height="371" width="600" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270069-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Correlation between genetic distance and phenotypic traits in cacao. Heatmap of Pearson's correlation coefficients between genetic distances and four phenotypic traits in cacao. Genetic distances were calculated using the IBS method based on SNP markers mapped to the Matina (IBS‐Matina) and Criollo (IBS‐Criollo) genomes. Phenotypic traits include healthy pod rate, AUDPC for pods with FPR, AUDPC for flower cushions with WBD, and AUDPC for branches with WBD. Red indicates a positive correlation, blue indicates a negative correlation, and color intensity represents the magnitude of the correlation. Asterisks (*) denote statistically significant correlations (<em>p</em> &lt; 0.05). FPRD, frosty pod rot disease; IBS, identity by state; WBD, witches' broom disease.</p></figcaption></figure><p>For disease resistance traits, the correlations were also generally weak. Pods with FPR exhibited a weak, nonsignificant positive correlation with IBS‐Matina (<em>r </em>= 0.11, <em>p</em> &gt; 0.05) and a weak, nonsignificant positive correlation with IBS‐Criollo (<em>r</em> = 0.15, <em>p </em>&gt; 0.05). A statistically significant (<em>p</em> &lt; 0.05) moderate negative correlation (<em>r</em> = ‐0.47) was observed between healthy pod rate and pods with FPR, indicating that accessions with higher healthy pod rates tended to have lower susceptibility to FPR as described by <a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">Osorio‐Guarín et al.(2020</a>). Similarly, a significant positive correlation was observed between flower cushions with WBD and branches with WBD (<em>r</em> = 0.53, <em>p</em> &lt; 0.05), as expected since they are both measures of susceptibility to the same disease.</p></section><section id="tpg270069-sec-0110"><h3 class="pmc_sec_title">3.2. Genetic structure and trait associations in cacao accessions</h3>
<p>Hierarchical clustering based on SNP data revealed complex relationships among the accessions, with no single clustering solution perfectly capturing the diversity within the collection (Figure <a href="#tpg270069-fig-0002" class="usa-link">2a,b</a>). The dendrograms for both Criollo and Matina showed that the accessions formed 14 distinct clusters, as determined by considering the underlying genetic structure. However, several noteworthy patterns emerged. The heatmaps, representing the SNP genotypes, visually illustrated the genetic similarities and differences among accessions and clusters. While some clusters appeared relatively homogeneous in terms of their SNP profiles, others exhibited greater within‐cluster diversity. The specific patterns of clustering and the composition of the clusters differed somewhat between the analyses based on the Criollo and Matina genomes, suggesting that the choice of reference genome can exert a slight influence on the resolution of population structure, although there was clear congruence between the two analyses, showing nearly identical heatmaps.</p>
<figure class="fig xbox font-sm" id="tpg270069-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bd2/12245736/14cd69da8512/TPG2-18-e70069-g006.jpg" loading="lazy" id="jats-graphic-3" height="488" width="555" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270069-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genomic and phenotypic diversity in cacao. Visualization of genetic and phenotypic diversity among 229 cacao accessions. (a and b) Hierarchical clustering based on single nucleotide polymorphism (SNP) data mapped to either the (a) Criollo or (b) Matina reference genomes. (c and d) t‐distributed stochastic neighbor embedding (t‐SNE) plots based on SNP and four phenotype data for (c) Criollo and (d) Matina, revealing three or four primary clusters within the collection.</p></figcaption></figure><p>Complementing the hierarchical clustering, t‐SNE, a nonlinear dimensionality reduction technique, provided an alternative visualization of the genetic structure, incorporating both SNP and phenotypic data (Figure <a href="#tpg270069-fig-0002" class="usa-link">2c,d</a>). When applied to the combined dataset, t‐SNE generated two‐dimensional plots that revealed distinct groupings for both Criollo and Matina. The t‐SNE plots effectively captured the relationships between genetic variation and phenotypic traits, with clusters appearing relatively well‐separated in the reduced‐dimensional space. Notably, the t‐SNE plots also revealed a degree of overlap between the clusters, particularly in the Matina‐based analysis, suggesting the presence of admixture or intermediate genotypes, consistent with the known complex history of cacao domestication and breeding. The specific arrangements of accessions within and between clusters differed somewhat between the Criollo and Matina‐based t‐SNE analyses, further emphasizing the influence of the reference genome on the resolution of population structure and trait associations.</p></section><section id="tpg270069-sec-0120"><h3 class="pmc_sec_title">3.3. Identification of candidate genes associated with disease resistance through BF and BT‐based GWAS in the Criollo genome</h3>
<p>To pinpoint genomic regions associated with cacao productivity (healthy pod rate) and disease resistance, we conducted a GWAS using both BF and BT models. We analyzed four key traits: healthy pod rate, pods with FPR, flowers with cushion WBD, and branches with WBD, using SNP markers mapped to the Criollo reference genome.</p>
<p>Our analyses identified numerous SNPs significantly associated with productivity and disease resistance traits in cacao. These associations, detailed in Table <a href="#tpg270069-tbl-0001" class="usa-link">1</a> and visualized in Figure <a href="#tpg270069-fig-0003" class="usa-link">3</a>, highlight a diverse set of candidate genes identified by both the BF and BT models. These genes are implicated in a range of biological processes, notably cell wall biosynthesis and modification, response to biotic and abiotic stress, and defense‐related mechanisms, which include potential roles in RNA silencing and ion transport. For example, we found strong associations with genes encoding a putative xyloglucan galactosyltransferase, a DEAD‐box ATP‐dependent RNA helicase and an ion channel protein (Table <a href="#tpg270069-tbl-0001" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="tpg270069-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bd2/12245736/7dff388e0065/TPG2-18-e70069-g005.jpg" loading="lazy" id="jats-graphic-5" height="776" width="697" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270069-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome‐wide association study of cacao disease resistance and healthy pod rate using single nucleotide polymorphism (SNP) markers mapped to the Criollo genome. Manhattan plots showing the results of a genome‐wide association studies (GWAS) for four traits in cacao using SNP markers mapped to the Criollo reference genome with Bootstrap Forest (left) and Boosted Tree (right) models. (a) Healthy pod rate, (b) pods with frosty pod rot (FPR), (c) flower with cushion WBD, and (d) branches with witches' broom disease (WBD). The <em>x</em>‐axis represents the chromosomal location of each SNP along the 10 cacao chromosomes. The <em>y</em>‐axis represents the Log‐Exp(Importance score) for each SNP, a measure of its association with the trait.</p></figcaption></figure></section><section id="tpg270069-sec-0130"><h3 class="pmc_sec_title">3.4. Identification of candidate genes associated with productivity and disease resistance through BF and BT‐based GWAS in the Matina genome</h3>
<p>To further investigate the genetic basis of cacao pod health and disease resistance, we conducted additional GWAS using SNP markers mapped to the Matina reference genome. Again, both BF and BT models were used to analyze the same four key traits: healthy pod rate, pods with FPR, flower cushions with WBD, and branches with WBD. Our analyses revealed several SNPs with significant associations with these traits in the Matina genome. The top SNPs and their associated candidate genes, identified using both models, are detailed in Table <a href="#tpg270069-tbl-0002" class="usa-link">2</a> and visualized in Figure <a href="#tpg270069-fig-0004" class="usa-link">4</a>. Similar to the Criollo genome analysis, we found associations with genes implicated in diverse biological processes, including potential roles in cell wall modification, stress response, and defense mechanisms. We identified candidate genes with putative functions in defense response, such as <em>Thecc1EG006100</em> encoding a Leucine‐rich repeat‐containing protein and <em>Thecc1EG040241</em> encoding an RNA polymerase 2, as well as <em>Thecc1EG036963</em>, a TOPLESS‐related 1 protein involved in transcriptional regulation. Notably, the specific genes and genomic regions associated with each trait differed between the Matina and Criollo analyses, indicating the influence of the reference genome on GWAS results and the presence of a complex genetic architecture underlying these traits.</p>
<figure class="fig xbox font-sm" id="tpg270069-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bd2/12245736/7ad19191fbd5/TPG2-18-e70069-g008.jpg" loading="lazy" id="jats-graphic-7" height="738" width="700" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270069-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Manhattan plots from genome‐wide association studies (GWAS) of productivity and disease resistance in cacao using the Matina reference genome. The results are shown for four traits: (a) healthy pod rate, (b) pods with frosty pod rot (FPR), (c) flower cushions with WBD, and (d) branches with witches' broom disease (WBD). Bootstrap Forest (left) and Boosted Tree (right) models were used to assess single nucleotide polymorphism (SNP)–trait associations. The <em>x</em>‐axis represents the physical position of each SNP along the 10 cacao chromosomes/scaffolds. The <em>y</em>‐axis displays the Log‐Exp(Importance score), reflecting the strength of its association with the trait.</p></figcaption></figure></section><section id="tpg270069-sec-0140"><h3 class="pmc_sec_title">3.5. Analysis of combined traits using multiplicative index and principal component analysis</h3>
<p>To explore the genetic architecture of cacao productivity and disease resistance through a combined trait approach, we performed GWAS using two composite measures: a multiplicative index of four traits and their first principal component. These analyses were conducted using both BF and BT models, with SNP markers mapped to either the Criollo or Matina reference genomes. The results, detailed in Table <a href="#tpg270069-supinfo-0001" class="usa-link">S1</a> and visualized in Figure <a href="#tpg270069-fig-0005" class="usa-link">5</a>, revealed several significant SNP associations with both composite measures. When using the Criollo genome, several SNPs were found to be associated with both the multiplicative index and PC1, including SNPs near genes encoding for a probable xyloglucan galactosyltransferase GT17 (<em>Tc02v2_g001090</em>) and an ion channel DMI1 (<em>Tc02v2_g001650</em>). For the Matina genome, we identified associated SNPs near genes encoding for a Leucine‐rich repeat‐containing protein (<em>Thecc1EG006100</em>) and an uncharacterized protein with a castor and pollux, part of voltage‐gated ion channel (<em>Thecc1EG006180</em>). The specific SNPs and candidate genes identified differed depending on the composite measure used and the reference genome, corroborating our other findings that confirm the complexity of these traits and the influence of the reference genome on GWAS results.</p>
<figure class="fig xbox font-sm" id="tpg270069-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bd2/12245736/f91dd69ac759/TPG2-18-e70069-g009.jpg" loading="lazy" id="jats-graphic-9" height="784" width="695" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270069-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome‐wide association studies (GWAS) of combined cacao traits using a multiplicative index and first principal component (PC1). Manhattan plots show results for (a) multiplicative index of traits using the Criollo genome, (b) PC1 using the Criollo genome, (c) multiplicative index of traits using the Matina genome, And (d) PC1 using the Matina genome. <em>X</em>‐axis: physical position of each single nucleotide polymorphism (SNP) along the 10 cacao chromosomes/scaffolds. <em>Y</em>‐axis: Log‐Exp(Importance score).</p></figcaption></figure><p>To comprehensively visualize the genomic regions associated with cacao productivity and disease resistance, we integrated the results of our GWAS across all four traits, incorporating both ML models as well as both reference genomes, and displayed them in Manhattan plots (Figure <a href="#tpg270069-fig-0006" class="usa-link">6</a>). This combined visualization revealed several prominent peaks, representing SNPs with strong composite association signals across traits, models, and genomes. Notably, several candidate genes identified in our ML‐based analyses were also reported in the original study by Osorio‐Guarín et al. (<a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">2020</a>) using traditional GWAS methods, highlighting concordance between the approaches; these overlapping findings are summarized in Table <a href="#tpg270069-tbl-0003" class="usa-link">3</a>. In the Criollo analysis, prominent peaks were located on chromosome 1 near genes encoding ferrochelatase‐2 and a putative exopolygalacturonase (Figure <a href="#tpg270069-fig-0006" class="usa-link">6a</a>, peaks A and B), on chromosome 2 near a gene encoding a probable xyloglucan galactosyltransferase GT17 (Figure <a href="#tpg270069-fig-0006" class="usa-link">6a</a>, peak C), and on chromosome 5 near a gene encoding a flavonol sulfotransferase‐like protein (Figure <a href="#tpg270069-fig-0006" class="usa-link">6a</a>, peak D). In the Matina analysis, notable peaks were found on several scaffolds, including those near genes encoding a leucine‐rich repeat‐containing protein and a galactinol‐sucrose galactosyltransferase/raffinose synthase (Figure <a href="#tpg270069-fig-0006" class="usa-link">6b</a>, peaks F and G). These findings highlight the potential involvement of diverse biological processes (e.g., iron metabolism, cell wall modification, carbohydrate metabolism, stress response) in determining the overall well‐being and performance of cacao plants, ultimately influencing productivity and disease resistance. A complete list of highlighted peaks and their associated candidate genes is provided in the caption of Figure <a href="#tpg270069-fig-0006" class="usa-link">6</a>.</p>
<figure class="fig xbox font-sm" id="tpg270069-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bd2/12245736/547f478e0e93/TPG2-18-e70069-g007.jpg" loading="lazy" id="jats-graphic-11" height="437" width="400" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270069-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Integrative genome‐wide association studies (GWAS) of cacao productivity and disease resistance using Bootstrap Forest and Boosted Tree models and two reference genomes. Manhattan plots show combined association results for healthy pod rate and resistance to three diseases: pods with frosty pod rot (FPR), flower cushions with witches' broom disease (WBD), and branches with WBD. (a) Criollo genome. (b) Matina genome. Each point represents a single nucleotide polymorphism (SNP). X‐axis: physical position along the ten chromosomes/scaffolds. <em>Y</em>‐axis: Log‐Exp(Importance score), a combined measure of association strength from both models. Highlighted peaks (A–I) correspond to notable SNP–trait associations and their potential candidate genes: A: Chr1_21174462 (<em>Tc01v2_g019770</em>, ferrochelatase‐2, chloroplastic); B: Chr1_31472241 (<em>Tc01v2_g025400</em>, putative exopolygalacturonase); C: Chr2_770436 (<em>Tc02v2_g001090</em>, probable xyloglucan galactosyltransferase GT17); D: Chr5_29556644 (<em>Tc05v2_g015380</em>, flavonol sulfotransferase‐like protein); E: Scaffold_1_33051561 (<em>Thecc1EG004701</em>, uncharacterized protein with reverse transcriptase‐like and zinc‐binding domains); F: Scaffold_2_794296 (<em>Thecc1EG006100</em>, leucine‐rich repeat‐containing protein); G: Scaffold_3_22872047 (<em>Thecc1EG014511</em>, galactinol‐sucrose galactosyltransferase/raffinose synthase); H: Scaffold_5_11327144 (<em>Thecc1EG023171</em>, uncharacterized protein); I: Scaffold_8_2342743 (<em>Thecc1EG034244</em>, clathrin adaptor complex medium subunit family protein).</p></figcaption></figure></section><section id="tpg270069-sec-0150"><h3 class="pmc_sec_title">3.6. Correlation of SNP importance scores across traits and models</h3>
<p>To assess the consistency of SNP associations across different traits and ML models, we calculated Pearson's correlation coefficients between SNP importance scores derived from the GWAS analyses (Figure <a href="#tpg270069-fig-0007" class="usa-link">7</a>). These analyses were performed separately using SNP data mapped to the Criollo (Figure <a href="#tpg270069-fig-0007" class="usa-link">7a</a>) and Matina (Figure <a href="#tpg270069-fig-0007" class="usa-link">7b</a>) reference genomes. In the Criollo genome analysis, the highest correlation (<em>r</em> = 0.91, <em>p</em> &lt; 0.0001) was observed between the two WBD traits (Cushion WBD‐BF and Cushion WBD‐BT), indicating strong agreement between models for this trait (Figure <a href="#tpg270069-fig-0007" class="usa-link">7a</a>). Similarly, in the Matina genome analysis, the two Cushion WBD traits also showed the highest correlation (<em>r</em> = 0.82, <em>p</em> &lt; 0.05) (Figure <a href="#tpg270069-fig-0007" class="usa-link">7b</a>). For both the Criollo and Matina genomes, correlations between other trait–model combinations were generally weaker or nonsignificant, suggesting that different sets of SNPs may influence each trait, and that model choice can impact the identification of these associations. However, in the Matina genome, a moderate positive correlation was observed between the two models for FPR (<em>r </em>= 0.51, <em>p</em> &lt; 0.05) (Figure <a href="#tpg270069-fig-0007" class="usa-link">7b</a>). The overall pattern of correlations suggests that while there is some consistency in SNP importance across models for certain traits, particularly WBD, the genetic architecture of these traits is complex and may involve distinct sets of genes.</p>
<figure class="fig xbox font-sm" id="tpg270069-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bd2/12245736/dd3adbf8f417/TPG2-18-e70069-g001.jpg" loading="lazy" id="jats-graphic-13" height="322" width="600" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270069-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Consistency of single nucleotide polymorphism (SNP) importance scores across traits and models in cacao genome‐wide association studies (GWAS). Heatmaps show Pearson's correlations between SNP importance scores from Bootstrap Forest (BF) and Boosted Tree (BT) models for four traits: healthy pod rate, pods with frosty pod rot (FPR), flower cushions with witches' broom disease (WBD), and branches with WBD. (a) Criollo genome. (b) Matina genome. Cell color intensity indicates correlation strength (red: positive, blue: negative); asterisks (*) denote significance (<em>p</em> &lt; 0.05). Only significant correlation coefficients considered to be practically meaningful (<em>r</em> &gt; 0.1) are displayed numerically.</p></figcaption></figure></section><section id="tpg270069-sec-0160"><h3 class="pmc_sec_title">3.7. Overlap of top SNP associations between models and traits</h3>
<p>To further investigate the relationships between the top SNPs identified by the different models and for different traits, the overlap among the top SNPs (up to 100 with non‐zero importance scores) was visualized using Venn diagrams for the Criollo (Figure <a href="#tpg270069-fig-0008" class="usa-link">8</a>) and Matina (Figure <a href="#tpg270069-fig-0009" class="usa-link">9</a>) reference genomes.</p>
<figure class="fig xbox font-sm" id="tpg270069-fig-0008"><h4 class="obj_head">FIGURE 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bd2/12245736/93a2dc50396f/TPG2-18-e70069-g004.jpg" loading="lazy" id="jats-graphic-15" height="566" width="682" alt="FIGURE 8"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270069-fig-0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Venn diagrams illustrating the overlap among the top single nucleotide polymorphisms (SNPs) identified by Bootstrap Forest (BF) and Boosted Tree (BT) models for cacao disease resistance and productivity traits using the Criollo reference genome. Sets represent the up to top 100 SNPs based on importance score for: healthy pod rate (HP), frosty pod rot disease (FPRD), flower cushion witches' broom disease (C_WBD), and branch witches' broom disease (B_WBD) (note: in cases where fewer than 100 SNPs had a non‐zero importance score for a given trait/model combination, the set includes all SNPs with a non‐zero score). Overlapping regions indicate SNPs identified as important for multiple traits and/or by different models. (a) Overlap among the top SNPs for the four traits identified using the BF model. (b) Overlap among the top SNPs for the four traits identified using the BT model. (c) Overlap among the top SNPs for selected trait‐model combinations: HP‐BF, FPRD‐BF, FPRD‐BT, and C_WBD‐BT. (d) Overlap among the top SNPs for WBD traits (C_WBD and B_WBD) identified by both BF and BT models.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270069-fig-0009"><h4 class="obj_head">FIGURE 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bd2/12245736/30dd58b3a331/TPG2-18-e70069-g002.jpg" loading="lazy" id="jats-graphic-17" height="581" width="702" alt="FIGURE 9"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270069-fig-0009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Venn diagrams illustrating the overlap among the top single nucleotide polymorphisms (SNPs) identified by Bootstrap Forest (BF) and Boosted Tree (BT) models for cacao disease resistance and productivity traits using the Matina reference genome. Sets represent the up to top 100 SNPs based on importance score for: healthy pod rate (HP), frosty pod rot disease (FPRD), flower cushion witches' broom disease (C_WBD), and branch witches' broom disease (B_WBD) (note: in cases where fewer than 100 SNPs had a non‐zero importance score for a given trait/model combination, the set includes all SNPs with a non‐zero score). Overlapping regions indicate SNPs identified as important for multiple traits and/or by different models. (a) Overlap among the top SNPs for the four traits identified using the BF model. (b) Overlap among the top SNPs for the four traits identified using the BT model. (c) Overlap among the top SNPs for HP and FPRD traits identified by both BF and BT models. (d) Overlap among the top SNPs for WBD traits (C_WBD and B_WBD) identified by both BF and BT models.</p></figcaption></figure><p>Consistent with the correlation analysis (Figure <a href="#tpg270069-fig-0007" class="usa-link">7</a>), the diagrams generally show limited overlap between the top SNPs identified for distinct traits, particularly when considering SNPs found uniquely by only one model (Figures <a href="#tpg270069-fig-0008" class="usa-link">8a,b</a> and <a href="#tpg270069-fig-0009" class="usa-link">9a,b</a>). For instance, using the BF model on the Criollo genome, only one SNP (chr5_29556644, near the flavonol sulfotransferase‐like gene <em>Tc05v2_g015380</em>) was found among the top SNPs for all four traits simultaneously (Figure <a href="#tpg270069-fig-0008" class="usa-link">8a</a>), suggesting that largely different sets of SNPs are prioritized for each specific trait.</p>
<p>However, comparing the findings for the same trait across the two different models (BF vs. BT) reveals greater concordance for some traits, especially WBD. For Flower Cushion WBD (C_WBD), 19 top SNPs were common to both BF and BT models in the Criollo analysis (Figure <a href="#tpg270069-fig-0008" class="usa-link">8d</a>) and 16 SNPs were common in the Matina analysis (Figure <a href="#tpg270069-fig-0009" class="usa-link">9d</a>). Similarly, for Branch WBD (B_WBD), 19 SNPs were shared between models for Criollo (Figure <a href="#tpg270069-fig-0008" class="usa-link">8d</a>) and 18 for Matina (Figure <a href="#tpg270069-fig-0009" class="usa-link">9d</a>). This concordance lends further confidence to these shared associations. Notably, the single SNP (chr5_29556644) identified as important for all four traits by the BF model in Criollo was also identified by both models for both WBD traits (central overlap in Figure <a href="#tpg270069-fig-0008" class="usa-link">8d</a>). Similarly, in the Matina analysis, one SNP (SCAFFOLD_2_9415422, near a protein kinase/lectin gene <em>Thecc1EG007853</em>) was identified by both models for both WBD traits (central overlap in Figure <a href="#tpg270069-fig-0009" class="usa-link">9d</a>), indicating these may represent particularly robust candidate loci for broad WBD responses.</p>
<p>Overlap between models for Healthy Pod rate and frosty pod rot disease (FPRD) traits appeared less pronounced overall in the top SNP sets visualized, although some robust SNPs were identified, such as SCAFFOLD_3_22872047 (near the galactinol‐sucrose synthase gene <em>Thecc1EG014511</em>), which was found in the top SNPs for both HP and FPRD by both BF and BT models in the Matina analysis (central overlap in Figure <a href="#tpg270069-fig-0009" class="usa-link">9c</a>).</p></section></section><section id="tpg270069-sec-0170"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>This study utilized a ML‐based GWAS to investigate the complex genetic architecture of productivity and disease resistance in a diverse cacao collection. We identified numerous candidate genes associated with healthy pod rate, resistance to FPR, and resistance to WBD (Figures <a href="#tpg270069-fig-0003" class="usa-link">3</a>, <a href="#tpg270069-fig-0004" class="usa-link">4</a>, <a href="#tpg270069-fig-0005" class="usa-link">5</a>, <a href="#tpg270069-fig-0006" class="usa-link">6</a>, Tables <a href="#tpg270069-tbl-0001" class="usa-link">1</a>, <a href="#tpg270069-tbl-0002" class="usa-link">2</a>, <a href="#tpg270069-tbl-0003" class="usa-link">3</a>, and Table <a href="#tpg270069-supinfo-0001" class="usa-link">S1</a>), revealing a complex interplay of loci influencing these traits. Analysis of population structure using hierarchical clustering and t‐SNE (Figure <a href="#tpg270069-fig-0002" class="usa-link">2</a>) indicated a high degree of admixture within the collection, reflecting cacao's history of domestication and breeding (Argout et al., <a href="#tpg270069-bib-0003" class="usa-link" aria-describedby="tpg270069-bib-0003">2011</a>). The observed weak correlations between genetic distance, as measured by IBS, and the phenotypic traits examined suggest that overall genetic similarity is not a strong predictor of productivity or disease resistance in this cacao collection. This finding, consistent with findings in other complex traits in plants (Lindhout, <a href="#tpg270069-bib-0028" class="usa-link" aria-describedby="tpg270069-bib-0028">2002</a>), implies that these traits are likely governed by a complex polygenic architecture involving numerous genes, each with a relatively small effect. The high degree of admixture suggested by the t‐SNE analysis (Figure <a href="#tpg270069-fig-0002" class="usa-link">2</a>), which revealed overlapping groupings rather than distinct clusters, likely contributes to these weak correlations. This admixture reflects the complex history of cacao domestication, hybridization, and breeding, resulting in a mosaic of genetic diversity. While the population size of 102 accessions used in this study is smaller than ideal for detecting numerous small‐effect loci typical of highly polygenic traits, the significant diversity within this panel (Figure <a href="#tpg270069-fig-0002" class="usa-link">2</a>) and the application of ML methods allowed for the identification of several genomic regions with potentially larger effects or complex interactions contributing to disease resistance and productivity. Future studies incorporating larger panels will be beneficial for increasing statistical power and detecting loci with smaller effects.</p>
<p>Our GWAS analyses, employing both BF and BT models, identified a suite of candidate genes associated with each of the four traits examined. This complexity highlights the potential utility of employing ML‐based GWAS, which may complement findings from traditional GWAS methods by potentially capturing complex effects (such as epistasis) or prioritizing loci differently, especially given the traits' polygenic nature. For instance, SNPs in proximity to a gene encoding a probable xyloglucan galactosyltransferase GT17 (<em>Tc02v2_g001090</em>, Figure <a href="#tpg270069-fig-0006" class="usa-link">6</a>, peak C, associated with all traits and flower cushion WBD) on chromosome 2 showed strong associations with flower cushion WBD resistance, PC1, and the trait product. Xyloglucans are major components of plant cell walls, and their modification can influence cell wall integrity and potentially impact pathogen penetration (Hématy et al., <a href="#tpg270069-bib-0023" class="usa-link" aria-describedby="tpg270069-bib-0023">2009</a>). Another compelling novel candidate gene association emerging from our ML analysis involves galactinol‐sucrose galactosyltransferase/raffinose synthase (<em>Thecc1EG014511</em>, Figure <a href="#tpg270069-fig-0006" class="usa-link">6</a>, peak G, associated with pods with FPR), identified in the Matina dataset and associated with resistance to FPR, is known to accumulate during stress responses, implying a role in defense signaling (Chang et al., <a href="#tpg270069-bib-0010" class="usa-link" aria-describedby="tpg270069-bib-0010">2023</a>). This aligns with the findings of Chang et al. (<a href="#tpg270069-bib-0010" class="usa-link" aria-describedby="tpg270069-bib-0010">2023</a>), who demonstrated that the raffinose synthase GhRFS6 in cotton interacts with GhOPR9, a 12‐oxo‐phytodienoic acid reductase involved in jasmonic acid (JA) biosynthesis, showing that GhRFS6 is upregulated during <em>Verticillium dahliae</em> infection and contributes to resistance by modulating JA signaling and cell wall properties.</p>
<p>We also identified a strong novel peak on chromosome 1 associated with a gene encoding ferrochelatase‐2 (<em>Tc01v2_g019770</em>, Figure <a href="#tpg270069-fig-0006" class="usa-link">6</a>, peak A, associated with all traits and flower cushion WBD), an enzyme involved in heme biosynthesis. Heme plays crucial roles in various cellular processes such as defense signaling, pointing to a potential role this gene may play in cacao's response to pathogens. This finding has parallels with recent work by Xu et al. (<a href="#tpg270069-bib-0046" class="usa-link" aria-describedby="tpg270069-bib-0046">2024</a>), who demonstrated that the RxLR effector Pi22922 from <em>Phytophthora infestans</em> targets potato StFC‐II, leading to increased susceptibility. While the specific effector may differ in our pathosystem, the involvement of FC‐II in modulating disease resistance appears to be conserved (Xu et al., <a href="#tpg270069-bib-0046" class="usa-link" aria-describedby="tpg270069-bib-0046">2024</a>). Furthermore, the novel association of a gene encoding a putative exopolygalacturonase (<em>Tc01v2_g025400</em>, Figure <a href="#tpg270069-fig-0006" class="usa-link">6</a>, peak B, associated with pods with FPR) with multiple traits hints at the importance of cell wall degradation pathways in both productivity and disease resistance. This finding is particularly interesting in light of the work by Zhu et al. (<a href="#tpg270069-bib-0047" class="usa-link" aria-describedby="tpg270069-bib-0047">2021</a>) on <em>Verticillium dahliae</em>, a hemibiotrophic pathogen causing vascular wilt in a wide range of dicot hosts. They found that the deletion of a specific ExoPG gene in <em>V. dahliae</em> did not affect the pathogen's virulence on potato, despite the fact that this gene displayed increased expression in a highly aggressive isolate, proposing that the role of individual cell wall degrading enzymes (CWDEs) in pathogenicity can be complex and may involve compensatory mechanisms (Zhu et al., <a href="#tpg270069-bib-0047" class="usa-link" aria-describedby="tpg270069-bib-0047">2021</a>). In our study, the association of an exopolygalacturonase with multiple traits hints at a broader role for this enzyme in cacao, possibly related to cell wall remodeling during growth and development, as well as in defense responses (Fry, <a href="#tpg270069-bib-0017" class="usa-link" aria-describedby="tpg270069-bib-0017">1995</a>).</p>
<p>Additionally, the Matina‐based analysis using the trait product highlighted a novel candidate gene encoding an uncharacterized protein with a reverse transcriptase‐like domain (<em>Thecc1EG004701</em>, Figure <a href="#tpg270069-fig-0006" class="usa-link">6</a>, peak E, associated with all traits and flower cushion WBD). This association suggests a potential role for retroviral defense mechanisms or transposable element (TE) regulation in shaping overall plant performance, including disease resistance. As reviewed by Bennetzen and Wang (2014), TEs are key players in genomic novelty, influencing genome size, gene regulation, and creating new genes or pseudogenes. Additionally, a novel association near a leucine‐rich repeat (LRR)‐containing protein (<em>Thecc1EG006100</em>, Figure <a href="#tpg270069-fig-0006" class="usa-link">6</a>, peak F, associated with all traits and flower cushion WBD), often involved in pathogen recognition, was identified in the Matina dataset. This finding is consistent with the emphasis on the accurate identification of LRR proteins by Gottin et al. (<a href="#tpg270069-bib-0018" class="usa-link" aria-describedby="tpg270069-bib-0018">2021</a>) and further accentuates the potential importance of defense‐related pathways in influencing the composite traits. These results demonstrate how composite trait analyses can provide a more holistic perspective on the complex genetic architecture of cacao productivity, disease resistance, and overall well‐being; such interconnected pathways may be missed when analyzing individual traits separately.</p>
<p>The correlation analysis of SNP importance scores revealed varying degrees of consistency between the BF and BT models across different traits and genomes (Figure <a href="#tpg270069-fig-0007" class="usa-link">7</a>). Notably, the models exhibited strong concordance for the flower cushion WBD trait in both the Criollo (<em>r </em>= 0.91) and Matina (<em>r </em>= 0.82) genomes, a finding supported by the Venn diagram analysis (Figures <a href="#tpg270069-fig-0008" class="usa-link">8d</a> and <a href="#tpg270069-fig-0009" class="usa-link">9d</a>), which showed considerable overlap (16‐19 SNPs) in the top SNPs identified by both models for both cushion and branch WBD traits. For healthy pod rate, high correlation was observed in Criollo (<em>r</em> = 0.72), but it was moderate in Matina (<em>r </em>= 0.49). These findings, particularly the strong agreement both in correlation and top SNP overlap for WBD traits, bolster our confidence in the identified SNP‐trait associations for WBD, such as the robustly identified SNP chr5_29556644 found important for WBD by both models in Criollo (Figure <a href="#tpg270069-fig-0008" class="usa-link">8d</a>). However, the generally weaker correlations and limited overlap across different traits observed in the Venn diagrams (Figures <a href="#tpg270069-fig-0008" class="usa-link">8a,b</a> and <a href="#tpg270069-fig-0009" class="usa-link">9a,b</a>) suggest that distinct genetic architectures may underlie each trait, with limited evidence for pleiotropy within the scope of this study, although a few SNPs like chr5_29556644 (Criollo) and SCAFFOLD_3_22872047 (Matina) were highly ranked across multiple traits and models (Figures <a href="#tpg270069-fig-0008" class="usa-link">8a</a> and <a href="#tpg270069-fig-0009" class="usa-link">9c</a>).</p>
<p>Furthermore, the choice of reference genome influenced the results, as evidenced by the differences in SNP associations, candidate genes, and model correlations (e.g., healthy pod rate) identified when using the Criollo versus Matina genomes. These discrepancies likely stem from structural variations between the two genomes, differences in the SNP sets identified during mapping, or biases inherent in SNP discovery and the models themselves (Saxena et al., <a href="#tpg270069-bib-0042" class="usa-link" aria-describedby="tpg270069-bib-0042">2014</a>). Future research employing a unified pangenome reference, such as the recently constructed <em>T. cacao</em> pangenome (Argout et al., <a href="#tpg270069-bib-0001" class="usa-link" aria-describedby="tpg270069-bib-0001">2023</a>), which incorporates the genomic diversity of multiple cacao varieties, could help to resolve these inconsistencies and provide a more comprehensive understanding of the genetic basis of these important traits. Furthermore, benchmarking against known resistance loci revealed that several associations identified by our ML models occur on chromosomes previously implicated in resistance; for example, WBD associations were found on chromosomes 1, 4, 6, and 9, and FPR associations on chromosomes 1, 5, 6, 8, and 9 (Faleiro et al., <a href="#tpg270069-bib-0016" class="usa-link" aria-describedby="tpg270069-bib-0016">2006</a>; Lanaud et al., <a href="#tpg270069-bib-0027" class="usa-link" aria-describedby="tpg270069-bib-0027">2009</a>; McElroy et al., <a href="#tpg270069-bib-0030" class="usa-link" aria-describedby="tpg270069-bib-0030">2018</a>; Motilal et al., <a href="#tpg270069-bib-0033" class="usa-link" aria-describedby="tpg270069-bib-0033">2016</a>; Romero Navarro et al., <a href="#tpg270069-bib-0040" class="usa-link" aria-describedby="tpg270069-bib-0040">2017</a>; Royaert et al., <a href="#tpg270069-bib-0041" class="usa-link" aria-describedby="tpg270069-bib-0041">2016</a>), providing external support for our approach's ability to detect relevant genomic regions, although direct overlap with specific major QTLs reported in some studies was not observed.</p>
<p>Methodological evaluation using <em>K</em> = 5 cross‐validation revealed a common challenge in genomic studies with high dimensionality and limited sample size. While the BF and BT models demonstrated high explanatory power on training data (average R‐squared often &gt; 0.70), their performance on validation sets was low and unstable (validation R‐squared typically &lt; 0.20 and highly variable). This discrepancy highlights that while the models could fit the training data well, likely capturing complex relationships within this specific cohort, their ability to generalize and make accurate predictions on unseen data was limited by the sample size. This instability in validation underscores the rationale for utilizing the full dataset to obtain the most stable estimates of SNP importance for candidate gene discovery in this context, rather than relying on potentially misleading validation metrics. It is also important to acknowledge limitations inherent to using ML importance scores for association mapping. Unlike <em>p</em>‐values from linear models, the statistical properties of these importance scores, such as their distribution under the null hypothesis of no association, are less well characterized, making formal statistical threshold setting for controlling false positives challenging beyond pragmatic approaches like selecting the top N SNPs. Furthermore, because population structure was not explicitly modeled as a covariate, there is a potential risk that high importance scores could be partly driven by SNPs differentiating subpopulations that also happen to differ in mean phenotypic values, rather than solely reflecting causal association within the population. Furthermore, it is important to contextualize the specific candidate gene identifications. These genes were identified based on proximity to the SNPs with the highest importance scores. Given the SNP density in our study (average 39.8 kb) (Osorio‐Guarín et al., <a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">2020</a>), the identified SNP may be in linkage disequilibrium with the actual causal variant, which could reside within the identified gene or a nearby gene. Therefore, while the identified SNPs show strong association signals within this dataset, a direct comparison with results from traditional linear models (e.g., MLM, FarmCPU) on this specific dataset was beyond the scope of the current work but represents an important avenue for future research to fully delineate the advantages of each approach. Furthermore, both the predictive utility of these SNPs for MAS and the specific roles of the nearest candidate genes require experimental validation (e.g., expression analysis, functional studies), potentially through fine‐mapping in larger or independent populations. Addressing potential confounding due to population structure and developing robust statistical thresholds for ML‐based GWAS importance scores remain active areas of research.</p>
<p>In conclusion, this ML‐based GWAS has significantly expanded our understanding of the genetic architecture of productivity and disease resistance in cacao. We identified numerous candidate genes associated with these economically important traits, adding to the foundation laid by the original GWAS on this same collection by Osorio‐Guarín et al. (<a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">2020</a>). Our analysis confirmed the potential importance of several genes identified in the previous study (Table <a href="#tpg270069-tbl-0003" class="usa-link">3</a>) and brought to the forefront additional novel candidate genes, showcasing the potential of ML to reveal different facets of complex trait genetics. Practically, while ML‐GWAS approaches like the ones used here can potentially capture complex nonlinear or epistatic effects that may be missed by conventional linear models, they currently lack the standardized statistical validation frameworks (e.g., <em>p</em>‐value thresholds, FDR) and easily interpretable effect size estimates of the latter. Conventional GWAS remains crucial for identifying loci with strong additive effects, whereas ML‐GWAS serves as a valuable complementary tool for exploring complex genetic architectures and generating novel hypotheses. These findings provide a rich set of targets for future research within our program, including prioritizing candidate genes for functional validation through expression studies during pathogen infection and developing robust molecular markers from the most promising SNP associations for potential integration into MAS strategies in cacao breeding. Ultimately, combining diverse genomic approaches offers a powerful pathway to accelerate the development of improved, disease‐resistant cacao varieties.</p></section><section id="tpg270069-sec-0180"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Ezekiel Ahn</strong>: Conceptualization; formal analysis; funding acquisition; investigation; methodology; project administration; software; supervision; validation; visualization; writing—original draft. <strong>Sunchung Park</strong>: Funding acquisition; investigation; methodology; validation; visualization; writing—review and editing. <strong>Insuck Baek</strong>: Funding acquisition; investigation; methodology; visualization; writing—review and editing. <strong>Dongho Lee</strong>: Validation; writing—review and editing. <strong>Jishnu Bhatt</strong>: Investigation; visualization; writing—review and editing. <strong>Seunghyun Lim</strong>: Investigation; methodology; writing—review and editing. <strong>Jaehee Jang</strong>: Investigation; methodology; writing—review and editing. <strong>Dapeng Zhang</strong>: Methodology; visualization; writing—review and editing. <strong>Moon Kim</strong>: Funding acquisition; methodology; resources; writing—review and editing. <strong>Lyndel Meinhardt</strong>: Funding acquisition; methodology; resources; writing—review and editing.</p></section><section id="tpg270069-sec-0200"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec20"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270069-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270069-supitem-0001"><div class="caption p"><p>Supplementary Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12245736/bin/TPG2-18-e70069-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70069-s002.docx</a><sup> (13.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270069-supitem-0002"><div class="caption p"><p>Supplementary Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12245736/bin/TPG2-18-e70069-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70069-s001.xlsx</a><sup> (662.3KB, xlsx) </sup>
</div></div></section></section><section id="tpg270069-sec-0190" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We are also grateful to the reviewers for their constructive feedback. This work is supported by the U.S. Department of Agriculture, Agricultural Research Service, In‐House Projects No. 8042‐21220‐258‐000‐D and 8042‐21000‐303‐000‐D. Mention of any trade names or commercial products in this article is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the U. S. Department of Agriculture. USDA is an equal opportunity provider and employer, and all agency services are available without discrimination.</p></section><section id="notes1"><p>


Ahn, E.
, 
Park, S.
, 
Baek, I.
, 
Lee, D.
, 
Bhatt, J.
, 
Lim, S.
, 
Jang, J. H.
, 
Zhang, D.
, 
Kim, M. S.
, &amp; 
Meinhardt, L. W.
 (2025). Machine learning‐driven GWAS uncovers novel candidate genes for resistance to frosty pod rot and witches' broom disease in cacao. The Plant Genome, 18, e70069. 10.1002/tpg2.70069


</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270069-note-0001"><p>Assigned to Associate Editor Diego Jarquin.</p></div></div></section></section><section id="tpg270069-sec-0220"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The phenotypic and genotypic datasets used in this study are available in the supplementary material of Osorio‐Guarín et al. (<a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">2020</a>) in the journal <em>G3</em> (<a href="https://doi.org/10.1534/g3.120.401153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1534/g3.120.401153</a>). The results of our machine learning‐based genome‐wide association study are provided in the Supporting Information of this article. The file include tables with the SNP position in the context of both the Criollo and Matina reference genomes, and the SNP importance (portion) score for each trait and each machine learning model (Bootstrap Forest and BT). As our analysis employed machine learning algorithms, traditional <em>p</em>‐values were not generated. The SNP importance (portion) score serves as the measure of association strength.</p></section><section id="tpg270069-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270069-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270069-bib-0001">
<cite>

Argout, X.
, 
Droc, G.
, 
Fouet, O.
, 
Rouard, M.
, 
Labadie, K.
, 
Rhoné, B.
, 
Rey Loor, G.
, &amp; 
Lanaud, C.
 (2023). Pangenomic exploration of <em>Theobroma cacao</em>: New insights into gene content diversity and selection during domestication. <em>BioRxiv</em>. 10.1101/2023.11.03.565324
</cite> [<a href="https://doi.org/10.1101/2023.11.03.565324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270069-bib-0002">
<cite>

Argout, X.
, 
Martin, G.
, 
Droc, G.
, 
Fouet, O.
, 
Labadie, K.
, 
Rivals, E.
, 
Aury, J.‐M.
, &amp; 
Lanaud, C.
 (2017). The cacao Criollo genome v2. 0: An improved version of the genome for genetic and functional genomic studies. BMC Genomics, 18, 1–9. 10.1186/s12864-017-4120-9

</cite> [<a href="https://doi.org/10.1186/s12864-017-4120-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5603072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28915793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=The%20cacao%20Criollo%20genome%20v2.%200:%20An%20improved%20version%20of%20the%20genome%20for%20genetic%20and%20functional%20genomic%20studies&amp;author=X.%20Argout&amp;author=G.%20Martin&amp;author=G.%20Droc&amp;author=O.%20Fouet&amp;author=K.%20Labadie&amp;volume=18&amp;publication_year=2017&amp;pages=1-9&amp;pmid=28915793&amp;doi=10.1186/s12864-017-4120-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0003">
<cite>

Argout, X.
, 
Salse, J.
, 
Aury, J.‐M.
, 
Guiltinan, M. J.
, 
Droc, G.
, 
Gouzy, J.
, 
Allegre, M.
, 
Chaparro, C.
, 
Legavre, T.
, &amp; 
Maximova, S. N.
 (2011). The genome of Theobroma cacao. Nature Genetics, 43, 101–108. 10.1038/ng.736

</cite> [<a href="https://doi.org/10.1038/ng.736" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21186351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=The%20genome%20of%20Theobroma%20cacao&amp;author=X.%20Argout&amp;author=J.%20Salse&amp;author=J.%E2%80%90M.%20Aury&amp;author=M.%20J.%20Guiltinan&amp;author=G.%20Droc&amp;volume=43&amp;publication_year=2011&amp;pages=101-108&amp;pmid=21186351&amp;doi=10.1038/ng.736&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0004">
<cite>

Bartley, B. G.
 (2005). 
The genetic diversity of cacao and its utilization
. CABI Digital Library.</cite> [<a href="https://scholar.google.com/scholar_lookup?Bartley,%20B.%20G.%20(2005).%20The%20genetic%20diversity%20of%20cacao%20and%20its%20utilization%20.%20CABI%20Digital%20Library." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0005">
<cite>

Brachi, B.
, 
Morris, G. P.
, &amp; 
Borevitz, J. O.
 (2011). Genome‐wide association studies in plants: The missing heritability is in the field. Genome Biology, 12, 1–8. 10.1186/gb-2011-12-10-232
</cite> [<a href="https://doi.org/10.1186/gb-2011-12-10-232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3333769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22035733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Genome%E2%80%90wide%20association%20studies%20in%20plants:%20The%20missing%20heritability%20is%20in%20the%20field&amp;author=B.%20Brachi&amp;author=G.%20P.%20Morris&amp;author=J.%20O.%20Borevitz&amp;volume=12&amp;publication_year=2011&amp;pages=1-8&amp;pmid=22035733&amp;doi=10.1186/gb-2011-12-10-232&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0006">
<cite>

Bradbury, P. J.
, 
Zhang, Z.
, 
Kroon, D. E.
, 
Casstevens, T. M.
, 
Ramdoss, Y.
, &amp; 
Buckler, E. S.
 (2007). TASSEL: Software for association mapping of complex traits in diverse samples. Bioinformatics, 23, 2633–2635. 10.1093/bioinformatics/btm308

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btm308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17586829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=TASSEL:%20Software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples&amp;author=P.%20J.%20Bradbury&amp;author=Z.%20Zhang&amp;author=D.%20E.%20Kroon&amp;author=T.%20M.%20Casstevens&amp;author=Y.%20Ramdoss&amp;volume=23&amp;publication_year=2007&amp;pages=2633-2635&amp;pmid=17586829&amp;doi=10.1093/bioinformatics/btm308&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0007">
<cite>

Breiman, L.
 (2001). Random forests. Machine learning, 45, 5–32. 10.1023/A:1010933404324
</cite> [<a href="https://doi.org/10.1023/A:1010933404324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Machine%20learning&amp;title=Random%20forests&amp;author=L.%20Breiman&amp;volume=45&amp;publication_year=2001&amp;pages=5-32&amp;doi=10.1023/A:1010933404324&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0008">
<cite>

Cadby, J.
, &amp; 
Araki, T.
 (2021). Towards ethical chocolate: Multicriterial identifiers, pricing structures, and the role of the specialty cacao industry in sustainable development. SN Business &amp; Economics, 1, 44.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=SN%20Business%20&amp;%20Economics&amp;title=Towards%20ethical%20chocolate:%20Multicriterial%20identifiers,%20pricing%20structures,%20and%20the%20role%20of%20the%20specialty%20cacao%20industry%20in%20sustainable%20development&amp;author=J.%20Cadby&amp;author=T.%20Araki&amp;volume=1&amp;publication_year=2021&amp;pages=44&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0009">
<cite>

Ceccarelli, V.
, 
Fremout, T.
, 
Zavaleta, D.
, 
Lastra, S.
, 
Imán Correa, S.
, 
Arévalo‐Gardini, E.
, 
Rodriguez, C. A.
, 
Cruz Hilacondo, W.
, &amp; 
Thomas, E.
 (2021). Climate change impact on cultivated and wild cacao in Peru and the search of climate change‐tolerant genotypes. Diversity and Distributions, 27, 1462–1476. 10.1111/ddi.13294
</cite> [<a href="https://doi.org/10.1111/ddi.13294" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diversity%20and%20Distributions&amp;title=Climate%20change%20impact%20on%20cultivated%20and%20wild%20cacao%20in%20Peru%20and%20the%20search%20of%20climate%20change%E2%80%90tolerant%20genotypes&amp;author=V.%20Ceccarelli&amp;author=T.%20Fremout&amp;author=D.%20Zavaleta&amp;author=S.%20Lastra&amp;author=S.%20Im%C3%A1n%20Correa&amp;volume=27&amp;publication_year=2021&amp;pages=1462-1476&amp;doi=10.1111/ddi.13294&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0010">
<cite>

Chang, B.
, 
Zhao, L.
, 
Feng, Z.
, 
Wei, F.
, 
Zhang, Y.
, 
Zhang, Y.
, 
Huo, P.
, 
Cheng, Y.
, 
Zhou, J.
, &amp; 
Feng, H.
 (2023). Galactosyltransferase GhRFS6 interacting with GhOPR9 involved in defense against Verticillium wilt in cotton. Plant Science, 328, 111582. 10.1016/j.plantsci.2022.111582

</cite> [<a href="https://doi.org/10.1016/j.plantsci.2022.111582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36632889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Science&amp;title=Galactosyltransferase%20GhRFS6%20interacting%20with%20GhOPR9%20involved%20in%20defense%20against%20Verticillium%20wilt%20in%20cotton&amp;author=B.%20Chang&amp;author=L.%20Zhao&amp;author=Z.%20Feng&amp;author=F.%20Wei&amp;author=Y.%20Zhang&amp;volume=328&amp;publication_year=2023&amp;pages=111582&amp;pmid=36632889&amp;doi=10.1016/j.plantsci.2022.111582&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0011">
<cite>

Chen, T.
 (2014). Introduction to boosted trees. University of Washington Computer Science, 22, 14–40.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=University%20of%20Washington%20Computer%20Science&amp;title=Introduction%20to%20boosted%20trees&amp;author=T.%20Chen&amp;volume=22&amp;publication_year=2014&amp;pages=14-40&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0012">
<cite>

Chia Wong, J. A.
, 
Clement, D. P. L.
, 
Mournet, P.
, 
dos Santos Nascimento, A.
, 
Solís Bonilla, J. L.
, 
Lopes, U. V.
, 
Pires, J. L.
, &amp; 
Gramacho, K. P.
 (2022). A high‐density genetic map from a cacao F2 progeny and QTL detection for resistance to witches’ broom disease. Tree Genetics &amp; Genomes, 18, 31.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Tree%20Genetics%20&amp;%20Genomes&amp;title=A%20high%E2%80%90density%20genetic%20map%20from%20a%20cacao%20F2%20progeny%20and%20QTL%20detection%20for%20resistance%20to%20witches%E2%80%99%20broom%20disease&amp;author=J.%20A.%20Chia%20Wong&amp;author=D.%20P.%20L.%20Clement&amp;author=P.%20Mournet&amp;author=A.%20dos%20Santos%20Nascimento&amp;author=J.%20L.%20Sol%C3%ADs%20Bonilla&amp;volume=18&amp;publication_year=2022&amp;pages=31&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0013">
<cite>

Colonges, K.
, 
Jimenez, J.‐C.
, 
Saltos, A.
, 
Seguine, E.
, 
Solorzano, R. G. L.
, 
Fouet, O.
, 
Argout, X.
, 
Assemat, S.
, 
Davrieux, F.
, &amp; 
Cros, E.
 (2022). Integration of GWAS, metabolomics, and sensorial analyses to reveal novel metabolic pathways involved in cocoa fruity aroma GWAS of fruity aroma in Theobroma cacao. Plant Physiology and Biochemistry, 171, 213–225. 10.1016/j.plaphy.2021.11.006

</cite> [<a href="https://doi.org/10.1016/j.plaphy.2021.11.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34863583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology%20and%20Biochemistry&amp;title=Integration%20of%20GWAS,%20metabolomics,%20and%20sensorial%20analyses%20to%20reveal%20novel%20metabolic%20pathways%20involved%20in%20cocoa%20fruity%20aroma%20GWAS%20of%20fruity%20aroma%20in%20Theobroma%20cacao&amp;author=K.%20Colonges&amp;author=J.%E2%80%90C.%20Jimenez&amp;author=A.%20Saltos&amp;author=E.%20Seguine&amp;author=R.%20G.%20L.%20Solorzano&amp;volume=171&amp;publication_year=2022&amp;pages=213-225&amp;pmid=34863583&amp;doi=10.1016/j.plaphy.2021.11.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0014">
<cite>

Díaz‐Montenegro, J.
 (2019). Livelihood strategies and risk behavior of cacao producers in Ecuador: effects of national policies to support cacao farmers and specialty cacao landraces. Journal of Rural Studies, 63, 141–156.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Rural%20Studies&amp;title=Livelihood%20strategies%20and%20risk%20behavior%20of%20cacao%20producers%20in%20Ecuador:%20effects%20of%20national%20policies%20to%20support%20cacao%20farmers%20and%20specialty%20cacao%20landraces&amp;author=J.%20D%C3%ADaz%E2%80%90Montenegro&amp;volume=63&amp;publication_year=2019&amp;pages=141-156&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0015">
<cite>

Evans, H. C.
 (2007). Cacao diseases—The trilogy revisited. Phytopathology, 97, 1640–1643. 10.1094/PHYTO-97-12-1640

</cite> [<a href="https://doi.org/10.1094/PHYTO-97-12-1640" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18943725/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phytopathology&amp;title=Cacao%20diseases%E2%80%94The%20trilogy%20revisited&amp;author=H.%20C.%20Evans&amp;volume=97&amp;publication_year=2007&amp;pages=1640-1643&amp;pmid=18943725&amp;doi=10.1094/PHYTO-97-12-1640&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0016">
<cite>

Faleiro, F. G.
, 
Queiroz, V. T.
, 
Lopes, U. V.
, 
Guimarães, C. T.
, 
Pires, J. L.
, 
Yamada, M. M.
, 
Araújo, I. S.
, 
Pereira, M. G.
, 
Schnell, R.
, 
de Filho, G. A. S.
, 
Ferreira, C. F.
, 
Barros, E. G.
, &amp; 
Moreira, M. A.
 (2006). Mapping QTLs for witches’ broom (<em>Crinipellis Perniciosa</em>) resistance in cacao (<em>Theobroma Cacao</em> L.). Euphytica, 149, 227–235. 10.1007/s10681-005-9070-7
</cite> [<a href="https://doi.org/10.1007/s10681-005-9070-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Mapping%20QTLs%20for%20witches%E2%80%99%20broom%20(Crinipellis%20Perniciosa)%20resistance%20in%20cacao%20(Theobroma%20Cacao%20L.)&amp;author=F.%20G.%20Faleiro&amp;author=V.%20T.%20Queiroz&amp;author=U.%20V.%20Lopes&amp;author=C.%20T.%20Guimar%C3%A3es&amp;author=J.%20L.%20Pires&amp;volume=149&amp;publication_year=2006&amp;pages=227-235&amp;doi=10.1007/s10681-005-9070-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0017">
<cite>

Fry, S. C.
 (1995). Polysaccharide‐modifying enzymes in the plant cell wall. Annual Review of Plant Biology, 46, 497–520. 10.1146/annurev.pp.46.060195.002433
</cite> [<a href="https://doi.org/10.1146/annurev.pp.46.060195.002433" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Plant%20Biology&amp;title=Polysaccharide%E2%80%90modifying%20enzymes%20in%20the%20plant%20cell%20wall&amp;author=S.%20C.%20Fry&amp;volume=46&amp;publication_year=1995&amp;pages=497-520&amp;doi=10.1146/annurev.pp.46.060195.002433&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0018">
<cite>

Gottin, C.
, 
Dievart, A.
, 
Summo, M.
, 
Droc, G.
, 
Périn, C.
, 
Ranwez, V.
, &amp; 
Chantret, N.
 (2021). A new comprehensive annotation of leucine‐rich repeat‐containing receptors in rice. The Plant Journal, 108, 492–508. 10.1111/tpj.15456

</cite> [<a href="https://doi.org/10.1111/tpj.15456" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9292849/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34382706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=A%20new%20comprehensive%20annotation%20of%20leucine%E2%80%90rich%20repeat%E2%80%90containing%20receptors%20in%20rice&amp;author=C.%20Gottin&amp;author=A.%20Dievart&amp;author=M.%20Summo&amp;author=G.%20Droc&amp;author=C.%20P%C3%A9rin&amp;volume=108&amp;publication_year=2021&amp;pages=492-508&amp;pmid=34382706&amp;doi=10.1111/tpj.15456&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0019">
<cite>

Gutiérrez, O. A.
, 
Puig, A. S.
, 
Phillips‐Mora, W.
, 
Bailey, B. A.
, 
Ali, S. S.
, 
Mockaitis, K.
, 
Schnell, R. J.
, 
Livingstone, D.
, 
Mustiga, G.
, &amp; 
Royaert, S.
 (2021). SNP markers associated with resistance to frosty pod and black pod rot diseases in an F1 population of <em>Theobroma cacao</em> L.. Tree Genetics &amp; Genomes, 17, 28.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Tree%20Genetics%20&amp;%20Genomes&amp;title=SNP%20markers%20associated%20with%20resistance%20to%20frosty%20pod%20and%20black%20pod%20rot%20diseases%20in%20an%20F1%20population%20of%20Theobroma%20cacao%20L.&amp;author=O.%20A.%20Guti%C3%A9rrez&amp;author=A.%20S.%20Puig&amp;author=W.%20Phillips%E2%80%90Mora&amp;author=B.%20A.%20Bailey&amp;author=S.%20S.%20Ali&amp;volume=17&amp;publication_year=2021&amp;pages=28&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0020">
<cite>

Hall, J. N.
 (2022). 
Applying a One Health approach to study the livelihoods of cocoa farming communities in Bougainville
 (Doctoral thesis). The University of Sydney. <a href="https://hdl.handle.net/2123/29544" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://hdl.handle.net/2123/29544</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Hall,%20J.%20N.%20(2022).%20Applying%20a%20One%20Health%20approach%20to%20study%20the%20livelihoods%20of%20cocoa%20farming%20communities%20in%20Bougainville%20(Doctoral%20thesis).%20The%20University%20of%20Sydney.%20https://hdl.handle.net/2123/29544" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0021">
<cite>

Hamid, Y.
, &amp; 
Sugumaran, M.
 (2020). A t‐SNE based non linear dimension reduction for network intrusion detection. International Journal of Information Technology, 12, 125–134. 10.1007/s41870-019-00323-9
</cite> [<a href="https://doi.org/10.1007/s41870-019-00323-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Information%20Technology&amp;title=A%20t%E2%80%90SNE%20based%20non%20linear%20dimension%20reduction%20for%20network%20intrusion%20detection&amp;author=Y.%20Hamid&amp;author=M.%20Sugumaran&amp;volume=12&amp;publication_year=2020&amp;pages=125-134&amp;doi=10.1007/s41870-019-00323-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0022">
<cite>

Hatani, L.
, 
Bua, H.
, 
Mukhtar, S. D.
, &amp; 
Geo, L. O.
 (2016). Development model of cacao agro‐industry with sectoral competitive advantage based in Southeast Sulawesi, Indonesia. Global Journal of Flexible Systems Management, 17, 229–246. 10.1007/s40171-015-0119-8
</cite> [<a href="https://doi.org/10.1007/s40171-015-0119-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Global%20Journal%20of%20Flexible%20Systems%20Management&amp;title=Development%20model%20of%20cacao%20agro%E2%80%90industry%20with%20sectoral%20competitive%20advantage%20based%20in%20Southeast%20Sulawesi,%20Indonesia&amp;author=L.%20Hatani&amp;author=H.%20Bua&amp;author=S.%20D.%20Mukhtar&amp;author=L.%20O.%20Geo&amp;volume=17&amp;publication_year=2016&amp;pages=229-246&amp;doi=10.1007/s40171-015-0119-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0023">
<cite>

Hématy, K.
, 
Cherk, C.
, &amp; 
Somerville, S.
 (2009). Host–pathogen warfare at the plant cell wall. Current Opinion in Plant Biology, 12, 406–413. 10.1016/j.pbi.2009.06.007

</cite> [<a href="https://doi.org/10.1016/j.pbi.2009.06.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19616468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Plant%20Biology&amp;title=Host%E2%80%93pathogen%20warfare%20at%20the%20plant%20cell%20wall&amp;author=K.%20H%C3%A9maty&amp;author=C.%20Cherk&amp;author=S.%20Somerville&amp;volume=12&amp;publication_year=2009&amp;pages=406-413&amp;pmid=19616468&amp;doi=10.1016/j.pbi.2009.06.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0024">
<cite>

Hospital, F.
 (2009). Challenges for effective marker‐assisted selection in plants. Genetica, 136, 303–310. 10.1007/s10709-008-9307-1

</cite> [<a href="https://doi.org/10.1007/s10709-008-9307-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18695989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetica&amp;title=Challenges%20for%20effective%20marker%E2%80%90assisted%20selection%20in%20plants&amp;author=F.%20Hospital&amp;volume=136&amp;publication_year=2009&amp;pages=303-310&amp;pmid=18695989&amp;doi=10.1007/s10709-008-9307-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0025">
<cite>

Klimberg, R.
 (2023). Fundamentals of predictive analytics with JMP. Sas institute.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Fundamentals%20of%20predictive%20analytics%20with%20JMP&amp;author=R.%20Klimberg&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0026">
<cite>

Lahive, F.
, 
Hadley, P.
, &amp; 
Daymond, A. J.
 (2019). The physiological responses of cacao to the environment and the implications for climate change resilience. A review. Agronomy for Sustainable Development, 39, 5. 10.1007/s13593-018-0552-0
</cite> [<a href="https://doi.org/10.1007/s13593-018-0552-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy%20for%20Sustainable%20Development&amp;title=The%20physiological%20responses%20of%20cacao%20to%20the%20environment%20and%20the%20implications%20for%20climate%20change%20resilience.%20A%20review&amp;author=F.%20Lahive&amp;author=P.%20Hadley&amp;author=A.%20J.%20Daymond&amp;volume=39&amp;publication_year=2019&amp;pages=5&amp;doi=10.1007/s13593-018-0552-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0027">
<cite>

Lanaud, C.
, 
Fouet, O.
, 
Clément, D.
, 
Boccara, M.
, 
Risterucci, A. M.
, 
Surujdeo‐Maharaj, S.
, 
Legavre, T.
, &amp; 
Argout, X.
 (2009). A meta–QTL analysis of disease resistance traits of <em>Theobroma cacao</em> L.
Molecular Breeding, 24, 361–374. 10.1007/s11032-009-9297-4
</cite> [<a href="https://doi.org/10.1007/s11032-009-9297-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=A%20meta%E2%80%93QTL%20analysis%20of%20disease%20resistance%20traits%20of%20Theobroma%20cacao%20L.&amp;author=C.%20Lanaud&amp;author=O.%20Fouet&amp;author=D.%20Cl%C3%A9ment&amp;author=M.%20Boccara&amp;author=A.%20M.%20Risterucci&amp;volume=24&amp;publication_year=2009&amp;pages=361-374&amp;doi=10.1007/s11032-009-9297-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0028">
<cite>

Lindhout, P.
 (2002). The perspectives of polygenic resistance in breeding for durable disease resistance. Euphytica, 124, 217–226. 10.1023/A:1015686601404
</cite> [<a href="https://doi.org/10.1023/A:1015686601404" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=The%20perspectives%20of%20polygenic%20resistance%20in%20breeding%20for%20durable%20disease%20resistance&amp;author=P.%20Lindhout&amp;volume=124&amp;publication_year=2002&amp;pages=217-226&amp;doi=10.1023/A:1015686601404&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0029">
<cite>

Manrique, A. D.
, 
Gomez, N. A. N.
, &amp; 
Medina, A. M.
 (2024). Model for managing economic sustainability in small‐scale cacao producers. TEM Journal, 13, 2988. 10.18421/TEM134-33
</cite> [<a href="https://doi.org/10.18421/TEM134-33" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=TEM%20Journal&amp;title=Model%20for%20managing%20economic%20sustainability%20in%20small%E2%80%90scale%20cacao%20producers&amp;author=A.%20D.%20Manrique&amp;author=N.%20A.%20N.%20Gomez&amp;author=A.%20M.%20Medina&amp;volume=13&amp;publication_year=2024&amp;pages=2988&amp;doi=10.18421/TEM134-33&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0030">
<cite>

McElroy, M. S.
, 
Navarro, A. J.
, 
Mustiga, G.
, 
Stack, C.
, 
Gezan, S.
, 
Peña, G.
, 
Sarabia, W.
, 
Saquicela, D.
, 
Sotomayor, I.
, &amp; 
Douglas, G. M.
 (2018). Prediction of cacao (<em>Theobroma cacao</em>) resistance to <em>Moniliophthora</em> spp. diseases via genome‐wide association analysis and genomic selection. Frontiers in Plant Science, 9, 343. 10.3389/fpls.2018.00343

</cite> [<a href="https://doi.org/10.3389/fpls.2018.00343" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5890178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29662497/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Prediction%20of%20cacao%20(Theobroma%20cacao)%20resistance%20to%20Moniliophthora%20spp.%20diseases%20via%20genome%E2%80%90wide%20association%20analysis%20and%20genomic%20selection&amp;author=M.%20S.%20McElroy&amp;author=A.%20J.%20Navarro&amp;author=G.%20Mustiga&amp;author=C.%20Stack&amp;author=S.%20Gezan&amp;volume=9&amp;publication_year=2018&amp;pages=343&amp;pmid=29662497&amp;doi=10.3389/fpls.2018.00343&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0031">
<cite>

Meinhardt, L. W.
, 
Rincones, J.
, 
Bailey, B. A.
, 
Aime, M. C.
, 
Griffith, G. W.
, 
Zhang, D.
, &amp; 
Pereira, G. A.
 (2008). 
<em>Moniliophthora perniciosa</em>, the causal agent of witches’ broom disease of cacao: What's new from this old foe?
Molecular Plant Pathology, 9, 577–588. 10.1111/j.1364-3703.2008.00496.x

</cite> [<a href="https://doi.org/10.1111/j.1364-3703.2008.00496.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6640444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19018989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant%20Pathology&amp;title=Moniliophthora%20perniciosa,%20the%20causal%20agent%20of%20witches%E2%80%99%20broom%20disease%20of%20cacao:%20What's%20new%20from%20this%20old%20foe?&amp;author=L.%20W.%20Meinhardt&amp;author=J.%20Rincones&amp;author=B.%20A.%20Bailey&amp;author=M.%20C.%20Aime&amp;author=G.%20W.%20Griffith&amp;volume=9&amp;publication_year=2008&amp;pages=577-588&amp;pmid=19018989&amp;doi=10.1111/j.1364-3703.2008.00496.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0032">
<cite>

Motamayor, J. C.
, 
Mockaitis, K.
, 
Schmutz, J.
, 
Haiminen, N.
, 
Iii, D. L.
, 
Cornejo, O.
, 
Findley, S. D.
, 
Zheng, P.
, 
Utro, F.
, &amp; 
Royaert, S.
 (2013). The genome sequence of the most widely cultivated cacao type and its use to identify candidate genes regulating pod color. Genome Biology, 14, 1–25. 10.1186/gb-2013-14-6-r53
</cite> [<a href="https://doi.org/10.1186/gb-2013-14-6-r53" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4053823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23731509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=The%20genome%20sequence%20of%20the%20most%20widely%20cultivated%20cacao%20type%20and%20its%20use%20to%20identify%20candidate%20genes%20regulating%20pod%20color&amp;author=J.%20C.%20Motamayor&amp;author=K.%20Mockaitis&amp;author=J.%20Schmutz&amp;author=N.%20Haiminen&amp;author=D.%20L.%20Iii&amp;volume=14&amp;publication_year=2013&amp;pages=1-25&amp;pmid=23731509&amp;doi=10.1186/gb-2013-14-6-r53&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0033">
<cite>

Motilal, L. A.
, 
Zhang, D.
, 
Mischke, S.
, 
Meinhardt, L. W.
, 
Boccara, M.
, 
Fouet, O.
, 
Lanaud, C.
, &amp; 
Umaharan, P.
 (2016). Association mapping of seed and disease resistance traits in <em>Theobroma cacao</em> L.
Planta, 244, 1265–1276. 10.1007/s00425-016-2582-7

</cite> [<a href="https://doi.org/10.1007/s00425-016-2582-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27534964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Association%20mapping%20of%20seed%20and%20disease%20resistance%20traits%20in%20Theobroma%20cacao%20L.&amp;author=L.%20A.%20Motilal&amp;author=D.%20Zhang&amp;author=S.%20Mischke&amp;author=L.%20W.%20Meinhardt&amp;author=M.%20Boccara&amp;volume=244&amp;publication_year=2016&amp;pages=1265-1276&amp;pmid=27534964&amp;doi=10.1007/s00425-016-2582-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0034">
<cite>

Murtagh, F.
, &amp; 
Legendre, P.
 (2014). Ward's hierarchical agglomerative clustering method: Which algorithms implement Ward's criterion?. Journal of Classification, 31, 274–295. 10.1007/s00357-014-9161-z
</cite> [<a href="https://doi.org/10.1007/s00357-014-9161-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Classification&amp;title=Ward's%20hierarchical%20agglomerative%20clustering%20method:%20Which%20algorithms%20implement%20Ward's%20criterion?&amp;author=F.%20Murtagh&amp;author=P.%20Legendre&amp;volume=31&amp;publication_year=2014&amp;pages=274-295&amp;doi=10.1007/s00357-014-9161-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0035">
<cite>

Obeng, E. A.
, &amp; 
Aguilar, F. X.
 (2015). Marginal effects on biodiversity, carbon sequestration and nutrient cycling of transitions from tropical forests to cacao farming systems. Agroforestry Systems, 89, 19–35. 10.1007/s10457-014-9739-9
</cite> [<a href="https://doi.org/10.1007/s10457-014-9739-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agroforestry%20Systems&amp;title=Marginal%20effects%20on%20biodiversity,%20carbon%20sequestration%20and%20nutrient%20cycling%20of%20transitions%20from%20tropical%20forests%20to%20cacao%20farming%20systems&amp;author=E.%20A.%20Obeng&amp;author=F.%20X.%20Aguilar&amp;volume=89&amp;publication_year=2015&amp;pages=19-35&amp;doi=10.1007/s10457-014-9739-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0036">
<cite>

Osorio‐Guarín, J. A.
, 
Berdugo‐Cely, J. A.
, 
Coronado‐Silva, R. A.
, 
Baez, E.
, 
Jaimes, Y.
, &amp; 
Yockteng, R.
 (2020). Genome‐wide association study reveals novel candidate genes associated with productivity and disease resistance to Moniliophthora spp. in cacao (<em>Theobroma cacao</em> L.). G3: Genes, Genomes, Genetics, 10, 1713–1725. 10.1534/g3.120.401153

</cite> [<a href="https://doi.org/10.1534/g3.120.401153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7202020/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32169867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=Genome%E2%80%90wide%20association%20study%20reveals%20novel%20candidate%20genes%20associated%20with%20productivity%20and%20disease%20resistance%20to%20Moniliophthora%20spp.%20in%20cacao%20(Theobroma%20cacao%20L.)&amp;author=J.%20A.%20Osorio%E2%80%90Guar%C3%ADn&amp;author=J.%20A.%20Berdugo%E2%80%90Cely&amp;author=R.%20A.%20Coronado%E2%80%90Silva&amp;author=E.%20Baez&amp;author=Y.%20Jaimes&amp;volume=10&amp;publication_year=2020&amp;pages=1713-1725&amp;pmid=32169867&amp;doi=10.1534/g3.120.401153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0037">
<cite>

Phillips‐Mora, W.
, 
Aime, M.
, &amp; 
Wilkinson, M.
 (2007). Biodiversity and biogeography of the cacao (<em>Theobroma cacao</em>) pathogen Moniliophthora roreri in tropical America. Plant Pathology, 56, 911–922. 10.1111/j.1365-3059.2007.01646.x
</cite> [<a href="https://doi.org/10.1111/j.1365-3059.2007.01646.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Pathology&amp;title=Biodiversity%20and%20biogeography%20of%20the%20cacao%20(Theobroma%20cacao)%20pathogen%20Moniliophthora%20roreri%20in%20tropical%20America&amp;author=W.%20Phillips%E2%80%90Mora&amp;author=M.%20Aime&amp;author=M.%20Wilkinson&amp;volume=56&amp;publication_year=2007&amp;pages=911-922&amp;doi=10.1111/j.1365-3059.2007.01646.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0038">
<cite>

Poland, J.
, 
Endelman, J.
, 
Dawson, J.
, 
Rutkoski, J.
, 
Wu, S.
, 
Manes, Y.
, 
Dreisigacker, S.
, 
Crossa, J.
, 
Sánchez‐Villeda, H.
, &amp; 
Sorrells, M.
 (2012). Genomic selection in wheat breeding using genotyping‐by‐sequencing. The Plant Genome, 5, 103–113.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20selection%20in%20wheat%20breeding%20using%20genotyping%E2%80%90by%E2%80%90sequencing&amp;author=J.%20Poland&amp;author=J.%20Endelman&amp;author=J.%20Dawson&amp;author=J.%20Rutkoski&amp;author=S.%20Wu&amp;volume=5&amp;publication_year=2012&amp;pages=103-113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0039">
<cite>

Purdy, L.
, &amp; 
Schmidt, R.
 (1996). STATUS OF CACAO WITCHES' BROOM: Biology, epidemiology, and management. Annual Review of Phytopathology, 34, 573–594. 10.1146/annurev.phyto.34.1.573
</cite> [<a href="https://doi.org/10.1146/annurev.phyto.34.1.573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15012557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Phytopathology&amp;title=STATUS%20OF%20CACAO%20WITCHES'%20BROOM:%20Biology,%20epidemiology,%20and%20management&amp;author=L.%20Purdy&amp;author=R.%20Schmidt&amp;volume=34&amp;publication_year=1996&amp;pages=573-594&amp;pmid=15012557&amp;doi=10.1146/annurev.phyto.34.1.573&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0040">
<cite>

Romero Navarro, J. A.
, 
Phillips‐Mora, W.
, 
Arciniegas‐Leal, A.
, 
Mata‐Quirós, A.
, 
Haiminen, N.
, 
Mustiga, G.
, 
Livingstone, D.
,  III, 
Van Bakel, H.
, 
Kuhn, D. N.
, &amp; 
Parida, L.
 (2017). Application of genome wide association and genomic prediction for improvement of cacao productivity and resistance to black and frosty pod diseases. Frontiers in Plant Science, 8, 1905. 10.3389/fpls.2017.01905

</cite> [<a href="https://doi.org/10.3389/fpls.2017.01905" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5694496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29184558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Application%20of%20genome%20wide%20association%20and%20genomic%20prediction%20for%20improvement%20of%20cacao%20productivity%20and%20resistance%20to%20black%20and%20frosty%20pod%20diseases&amp;author=J.%20A.%20Romero%20Navarro&amp;author=W.%20Phillips%E2%80%90Mora&amp;author=A.%20Arciniegas%E2%80%90Leal&amp;author=A.%20Mata%E2%80%90Quir%C3%B3s&amp;author=N.%20Haiminen&amp;volume=8&amp;publication_year=2017&amp;pages=1905&amp;pmid=29184558&amp;doi=10.3389/fpls.2017.01905&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0041">
<cite>

Royaert, S.
, 
Jansen, J.
, 
da Silva, D. V.
, 
de Jesus Branco, S. M.
, 
Livingstone, D. S.
, 
Mustiga, G.
, 
Marelli, J.‐P.
, 
Araújo, I. S.
, 
Corrêa, R. X.
, &amp; 
Motamayor, J. C.
 (2016). Identification of candidate genes involved in Witches’ broom disease resistance in a segregating mapping population of <em>Theobroma cacao</em> L. in Brazil. BMC Genomics, 17, 107. 10.1186/s12864-016-2415-x

</cite> [<a href="https://doi.org/10.1186/s12864-016-2415-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4750280/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26865216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Identification%20of%20candidate%20genes%20involved%20in%20Witches%E2%80%99%20broom%20disease%20resistance%20in%20a%20segregating%20mapping%20population%20of%20Theobroma%20cacao%20L.%20in%20Brazil&amp;author=S.%20Royaert&amp;author=J.%20Jansen&amp;author=D.%20V.%20da%20Silva&amp;author=S.%20M.%20de%20Jesus%20Branco&amp;author=D.%20S.%20Livingstone&amp;volume=17&amp;publication_year=2016&amp;pages=107&amp;pmid=26865216&amp;doi=10.1186/s12864-016-2415-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0042">
<cite>

Saxena, R. K.
, 
Edwards, D.
, &amp; 
Varshney, R. K.
 (2014). Structural variations in plant genomes. Briefings in Functional Genomics, 13, 296–307. 10.1093/bfgp/elu016

</cite> [<a href="https://doi.org/10.1093/bfgp/elu016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4110416/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24907366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Briefings%20in%20Functional%20Genomics&amp;title=Structural%20variations%20in%20plant%20genomes&amp;author=R.%20K.%20Saxena&amp;author=D.%20Edwards&amp;author=R.%20K.%20Varshney&amp;volume=13&amp;publication_year=2014&amp;pages=296-307&amp;pmid=24907366&amp;doi=10.1093/bfgp/elu016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0043">
<cite>

Shaner, G.
, &amp; 
Finney, R. E.
 (1977). The effect of nitrogen fertilization on the expression of slow‐mildewing resistance in Knox wheat. Phytopathology, 67, 1051–1056. 10.1094/Phyto-67-1051
</cite> [<a href="https://doi.org/10.1094/Phyto-67-1051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phytopathology&amp;title=The%20effect%20of%20nitrogen%20fertilization%20on%20the%20expression%20of%20slow%E2%80%90mildewing%20resistance%20in%20Knox%20wheat&amp;author=G.%20Shaner&amp;author=R.%20E.%20Finney&amp;volume=67&amp;publication_year=1977&amp;pages=1051-1056&amp;doi=10.1094/Phyto-67-1051&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0044">
<cite>

Villacis, A. H.
, 
Alwang, J. R.
, 
Barrera, V.
, &amp; 
Dominguez, J.
 (2022). Prices, specialty varieties, and postharvest practices: Insights from cacao value chains in Ecuador. Agribusiness, 38, 426–458. 10.1002/agr.21730
</cite> [<a href="https://doi.org/10.1002/agr.21730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agribusiness&amp;title=Prices,%20specialty%20varieties,%20and%20postharvest%20practices:%20Insights%20from%20cacao%20value%20chains%20in%20Ecuador&amp;author=A.%20H.%20Villacis&amp;author=J.%20R.%20Alwang&amp;author=V.%20Barrera&amp;author=J.%20Dominguez&amp;volume=38&amp;publication_year=2022&amp;pages=426-458&amp;doi=10.1002/agr.21730&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0045">
<cite>

Wickramasuriya, A. M.
, &amp; 
Dunwell, J. M.
 (2018). Cacao biotechnology: Current status and future prospects. Plant Biotechnology Journal, 16, 4–17. 10.1111/pbi.12848

</cite> [<a href="https://doi.org/10.1111/pbi.12848" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5785363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28985014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=Cacao%20biotechnology:%20Current%20status%20and%20future%20prospects&amp;author=A.%20M.%20Wickramasuriya&amp;author=J.%20M.%20Dunwell&amp;volume=16&amp;publication_year=2018&amp;pages=4-17&amp;pmid=28985014&amp;doi=10.1111/pbi.12848&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0046">
<cite>

Xu, M.
, 
Sun, X.
, 
Wu, X.
, 
Qi, Y.
, 
Li, H.
, 
Nie, J.
, 
Yang, Z.
, &amp; 
Tian, Z.
 (2024). Chloroplast protein StFC‐II was manipulated by a Phytophthora effector to enhance host susceptibility. Horticulture Research, 11, uhae149. 10.1093/hr/uhae149

</cite> [<a href="https://doi.org/10.1093/hr/uhae149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11237190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38994450/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=Chloroplast%20protein%20StFC%E2%80%90II%20was%20manipulated%20by%20a%20Phytophthora%20effector%20to%20enhance%20host%20susceptibility&amp;author=M.%20Xu&amp;author=X.%20Sun&amp;author=X.%20Wu&amp;author=Y.%20Qi&amp;author=H.%20Li&amp;volume=11&amp;publication_year=2024&amp;pages=uhae149&amp;pmid=38994450&amp;doi=10.1093/hr/uhae149&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270069-bib-0047">
<cite>

Zhu, X.
, 
Sayari, M.
, &amp; 
Daayf, F.
 (2021). Role of exopolygalacturonase‐related genes in potato‐<em>Verticillium dahliae</em> interaction. Pathogens, 10, 642. 10.3390/pathogens10060642

</cite> [<a href="https://doi.org/10.3390/pathogens10060642" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8224799/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34071016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pathogens&amp;title=Role%20of%20exopolygalacturonase%E2%80%90related%20genes%20in%20potato%E2%80%90Verticillium%20dahliae%20interaction&amp;author=X.%20Zhu&amp;author=M.%20Sayari&amp;author=F.%20Daayf&amp;volume=10&amp;publication_year=2021&amp;pages=642&amp;pmid=34071016&amp;doi=10.3390/pathogens10060642&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Supplementary Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12245736/bin/TPG2-18-e70069-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70069-s002.docx</a><sup> (13.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>Supplementary Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12245736/bin/TPG2-18-e70069-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70069-s001.xlsx</a><sup> (662.3KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The phenotypic and genotypic datasets used in this study are available in the supplementary material of Osorio‐Guarín et al. (<a href="#tpg270069-bib-0036" class="usa-link" aria-describedby="tpg270069-bib-0036">2020</a>) in the journal <em>G3</em> (<a href="https://doi.org/10.1534/g3.120.401153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1534/g3.120.401153</a>). The results of our machine learning‐based genome‐wide association study are provided in the Supporting Information of this article. The file include tables with the SNP position in the context of both the Criollo and Matina reference genomes, and the SNP importance (portion) score for each trait and each machine learning model (Bootstrap Forest and BT). As our analysis employed machine learning algorithms, traditional <em>p</em>‐values were not generated. The SNP importance (portion) score serves as the measure of association strength.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70069"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70069.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12245736/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12245736/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12245736%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12245736/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12245736/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12245736/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40640111/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12245736/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40640111/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12245736/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12245736/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="5h74YfQ8ACefzDbUKNLvGhny6YANUkIrCarWulVXjwQgx3B8U2iuNWl5gNbODc9i">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
