
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Organellar Genomes of Three Globally Important Nanoplanktonic Diatoms Refine Their Taxon‐Specific Distribution and Succession Patterns in the Northwest Atlantic - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="FF207FF889B712431A7FF80022017A31.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326293/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Journal of Eukaryotic Microbiology">
<meta name="citation_title" content="Organellar Genomes of Three Globally Important Nanoplanktonic Diatoms Refine Their Taxon‐Specific Distribution and Succession Patterns in the Northwest Atlantic">
<meta name="citation_author" content="Rebecca Stevens‐Green">
<meta name="citation_author_institution" content="Department of Biology, Dalhousie University, Halifax, Nova Scotia, Canada">
<meta name="citation_author_institution" content="Institute for Comparative Genomics, Dalhousie University, Halifax, Nova Scotia, Canada">
<meta name="citation_author" content="Caroline Chénard">
<meta name="citation_author_institution" content="Aquatic and Crop Resource Development, National Research Council Canada, Halifax, Nova Scotia, Canada">
<meta name="citation_author" content="Solenn Mordret">
<meta name="citation_author_institution" content="Aquatic and Crop Resource Development, National Research Council Canada, Halifax, Nova Scotia, Canada">
<meta name="citation_author" content="Jenna MacKinnon">
<meta name="citation_author_institution" content="Aquatic and Crop Resource Development, National Research Council Canada, Halifax, Nova Scotia, Canada">
<meta name="citation_author" content="Brent M Robicheau">
<meta name="citation_author_institution" content="Biology Department, Clark University, Worcester, Massachusetts, USA">
<meta name="citation_author" content="Julie LaRoche">
<meta name="citation_author_institution" content="Department of Biology, Dalhousie University, Halifax, Nova Scotia, Canada">
<meta name="citation_author_institution" content="Institute for Comparative Genomics, Dalhousie University, Halifax, Nova Scotia, Canada">
<meta name="citation_publication_date" content="2025 Aug 6">
<meta name="citation_volume" content="72">
<meta name="citation_issue" content="5">
<meta name="citation_firstpage" content="e70033">
<meta name="citation_doi" content="10.1111/jeu.70033">
<meta name="citation_pmid" content="40765478">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326293/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326293/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326293/pdf/JEU-72-e70033.pdf">
<meta name="description" content="Nanoplanktonic diatoms (2–20 μm) are a significant yet historically understudied component of marine ecosystems. We investigated three recently isolated nanoplanktonic diatoms from the Northwest Atlantic Ocean (NWA): Minidiscus spinulatus, ...">
<meta name="og:title" content="Organellar Genomes of Three Globally Important Nanoplanktonic Diatoms Refine Their Taxon‐Specific Distribution and Succession Patterns in the Northwest Atlantic">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Nanoplanktonic diatoms (2–20 μm) are a significant yet historically understudied component of marine ecosystems. We investigated three recently isolated nanoplanktonic diatoms from the Northwest Atlantic Ocean (NWA): Minidiscus spinulatus, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326293/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12326293">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1111/jeu.70033"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/JEU-72-e70033.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12326293%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12326293/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12326293/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326293/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1111/jeu.70033" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">J Eukaryot Microbiol</button></div>. 2025 Aug 6;72(5):e70033. doi: <a href="https://doi.org/10.1111/jeu.70033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1111/jeu.70033</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22J%20Eukaryot%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%20Eukaryot%20Microbiol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22J%20Eukaryot%20Microbiol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22J%20Eukaryot%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Organellar Genomes of Three Globally Important Nanoplanktonic Diatoms Refine Their Taxon‐Specific Distribution and Succession Patterns in the Northwest Atlantic</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stevens%E2%80%90Green%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Rebecca Stevens‐Green</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Rebecca Stevens‐Green</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Biology, Dalhousie University, Halifax, Nova Scotia, Canada</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Institute for Comparative Genomics, Dalhousie University, Halifax, Nova Scotia, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stevens%E2%80%90Green%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rebecca Stevens‐Green</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ch%C3%A9nard%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Caroline Chénard</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Caroline Chénard</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Aquatic and Crop Resource Development, National Research Council Canada, Halifax, Nova Scotia, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ch%C3%A9nard%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Caroline Chénard</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mordret%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Solenn Mordret</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Solenn Mordret</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Aquatic and Crop Resource Development, National Research Council Canada, Halifax, Nova Scotia, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mordret%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Solenn Mordret</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22MacKinnon%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jenna MacKinnon</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jenna MacKinnon</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Aquatic and Crop Resource Development, National Research Council Canada, Halifax, Nova Scotia, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22MacKinnon%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jenna MacKinnon</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Robicheau%20BM%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Brent M Robicheau</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Brent M Robicheau</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Biology Department, Clark University, Worcester, Massachusetts, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Robicheau%20BM%22%5BAuthor%5D" class="usa-link"><span class="name western">Brent M Robicheau</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22LaRoche%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Julie LaRoche</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Julie LaRoche</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Biology, Dalhousie University, Halifax, Nova Scotia, Canada</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Institute for Comparative Genomics, Dalhousie University, Halifax, Nova Scotia, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22LaRoche%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Julie LaRoche</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="jeu70033-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Biology, Dalhousie University, Halifax, Nova Scotia, Canada</div>
<div id="jeu70033-aff-0002">
<sup>
<sup>2</sup>
</sup>Institute for Comparative Genomics, Dalhousie University, Halifax, Nova Scotia, Canada</div>
<div id="jeu70033-aff-0003">
<sup>
<sup>3</sup>
</sup>Aquatic and Crop Resource Development, National Research Council Canada, Halifax, Nova Scotia, Canada</div>
<div id="jeu70033-aff-0004">
<sup>
<sup>4</sup>
</sup>Biology Department, Clark University, Worcester, Massachusetts, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Rebecca Stevens‐Green (<span>r.stevens-green@dal.ca</span>), 
Julie LaRoche (<span>julie.laroche@dal.ca</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jul 7; Received 2025 May 23; Accepted 2025 Jul 11; Issue date 2025 Sep-Oct.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Journal of Eukaryotic Microbiology</em> published by Wiley Periodicals LLC on behalf of International Society of Protistologists.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited and is not used for commercial purposes.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12326293  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40765478/" class="usa-link">40765478</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Nanoplanktonic diatoms (2–20 μm) are a significant yet historically understudied component of marine ecosystems. We investigated three recently isolated nanoplanktonic diatoms from the Northwest Atlantic Ocean (NWA): <em>Minidiscus spinulatus</em>, <em>Mediolabrus comicus</em>, and <em>Minidiscus trioculatus</em>. Using Oxford Nanopore sequencing, we assembled and annotated their complete chloroplast and mitochondrial genomes. Pangenome analyses revealed that <em>Minidiscus</em> species consistently clustered more closely with select <em>Thalassiosira</em> species, whereas <em>M. comicus</em> formed a sister clade with <em>Skeletonema</em>. Circularized chloroplast genomes allowed us to characterize the full‐length <em>16S</em> ribosomal RNAs for each isolate, thereby leading to higher resolution of these taxa in preexisting <em>16S</em> metabarcoding data. During our study, 
<em>M. spinulatus</em>
 was primarily restricted to the Bedford Basin. In contrast, both <em>M. trioculatus</em> and <em>M. comicus</em> had larger geographic ranges extending to the Labrador Sea, and in the case of <em>M. comicus</em>, to the Canadian Arctic Gateway. Weekly metabarcoding from the coastal Bedford Basin, N.S., Canada (2014–2022), revealed a seasonal succession of nanoplanktonic taxa, with <em>Minidiscus trioculatus</em> dominating in the early months, followed by <em>M. comicus</em> and 
<em>M. spinulatus</em>
. Our results highlight the critical value of phytoplankton isolations and organelle genomics for expanding our understanding of the diversity and biogeography of nanoplanktonic diatoms.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> Arctic, chloroplast genomes, eDNA, Labrador Sea, marine microbiology, <em>Mediolabrus</em>, microbial ecology, <em>Minidiscus</em>, mitochondrial genomes, scotia shelf</p></section></section><section id="jeu70033-sec-0001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Phytoplankton are vital contributors to biogeochemical cycles, including ocean carbon export, due to their role as primary producers. Diatoms are unicellular phytoplankton with silica shells that occur in marine environments worldwide, contributing around 20% of global oxygen production (Nelson et al. <a href="#jeu70033-bib-0083" class="usa-link" aria-describedby="jeu70033-bib-0083">1995</a>; Falkowski et al. <a href="#jeu70033-bib-0028" class="usa-link" aria-describedby="jeu70033-bib-0028">1998</a>). While conservative estimates suggest that ~12,000 identified species of diatoms exist with another 8,000 yet to be discovered, other estimates suggest there could be as many as 200,000 species (Guiry <a href="#jeu70033-bib-0039" class="usa-link" aria-describedby="jeu70033-bib-0039">2012</a>). Diatom size classes include picoplankton (&lt; 2 μm), nanoplankton (2–20 μm), and microplankton (&gt; 20 μm) (Guiry <a href="#jeu70033-bib-0039" class="usa-link" aria-describedby="jeu70033-bib-0039">2012</a>; Leblanc et al. <a href="#jeu70033-bib-0062" class="usa-link" aria-describedby="jeu70033-bib-0062">2018</a>). The ecological significance of nanoplanktonic diatoms is underscored by recent <em>Tara Oceans</em> metabarcoding data, which show that diatoms were most abundant in the 3–20 μm size class (Malviya et al. <a href="#jeu70033-bib-0076" class="usa-link" aria-describedby="jeu70033-bib-0076">2016</a>; Leblanc et al. <a href="#jeu70033-bib-0062" class="usa-link" aria-describedby="jeu70033-bib-0062">2018</a>; Pierella Karlusich et al. <a href="#jeu70033-bib-0093" class="usa-link" aria-describedby="jeu70033-bib-0093">2025</a>).</p>
<p>Thalassiosiraceae is a family of centric diatoms composed of 26 different genera, including <em>Thalassiosira</em>, <em>Conticriba</em>, <em>Porosira</em>, <em>Minidiscus</em>, and <em>Mediolabrus</em>. First discovered in 1973 by Hasle (Round et al. <a href="#jeu70033-bib-0101" class="usa-link" aria-describedby="jeu70033-bib-0101">1990</a>; Guiry and Guiry <a href="#jeu70033-bib-0042" class="usa-link" aria-describedby="jeu70033-bib-0042">2024</a>; Hasle <a href="#jeu70033-bib-0045" class="usa-link" aria-describedby="jeu70033-bib-0045">1973</a>), <em>Minidiscus</em> and <em>Mediolabrus</em> spp. are the smallest known centric diatoms, ranging from &lt; 2–8 μm, and are known to exist as either solitary cells, in aggregations, or sometimes in chains formed from the central fultoportulae (Arsenieff et al. <a href="#jeu70033-bib-0005" class="usa-link" aria-describedby="jeu70033-bib-0005">2020</a>; Kaczmarska et al. <a href="#jeu70033-bib-0053" class="usa-link" aria-describedby="jeu70033-bib-0053">2009</a>; Leblanc et al. <a href="#jeu70033-bib-0062" class="usa-link" aria-describedby="jeu70033-bib-0062">2018</a>). Importantly, recent taxonomic reclassification has placed some <em>Minidiscus</em> species into the new genus <em>Mediolabrus</em> (Li et al. <a href="#jeu70033-bib-0068" class="usa-link" aria-describedby="jeu70033-bib-0068">2020</a>) based on morphological and phylogenetic differences. When reclassifying some <em>Minidiscus</em> as <em>Mediolabrus</em> based on morphology, it was found that <em>Minidiscus</em> has at least one fultoportula in the valve center close to a rimoportula, whereas <em>Mediolabrus</em> only has a single rimoportula concentrating processes in the valve center, with no fultoportula near the rimoportula (Li et al. <a href="#jeu70033-bib-0068" class="usa-link" aria-describedby="jeu70033-bib-0068">2020</a>). Phylogenetic trees further demonstrated that <em>Minidiscus</em> is not a monophyletic group; the <em>Minidiscus</em> are closely related to <em>Thalassiosira</em> species, while the <em>Mediolabrus</em> are a sister clade to the <em>Skeletonema</em> genus. According to <em>Algaebase</em> [as of April 2025], there are nine known <em>Minidiscus</em> species and three known <em>Mediolabrus</em> species (Guiry and Guiry <a href="#jeu70033-bib-0041" class="usa-link" aria-describedby="jeu70033-bib-0041">2022</a>, <a href="#jeu70033-bib-0042" class="usa-link" aria-describedby="jeu70033-bib-0042">2024</a>).</p>
<p>
<em>Minidiscus</em> and <em>Mediolabrus</em> have been observed using metabarcoding in various locations across mid‐ and high‐latitude sites, as well as well‐mixed environments [e.g., within the Northwest Atlantic (NWA)] (Leblanc et al. <a href="#jeu70033-bib-0062" class="usa-link" aria-describedby="jeu70033-bib-0062">2018</a>; Bolaños et al. <a href="#jeu70033-bib-0009" class="usa-link" aria-describedby="jeu70033-bib-0009">2020</a>; Robicheau et al. <a href="#jeu70033-bib-0099" class="usa-link" aria-describedby="jeu70033-bib-0099">2022</a>; Daniels et al. <a href="#jeu70033-bib-0022" class="usa-link" aria-describedby="jeu70033-bib-0022">2015</a>; Aké‐Castillo et al. <a href="#jeu70033-bib-0002" class="usa-link" aria-describedby="jeu70033-bib-0002">2001</a>; Pesant et al. <a href="#jeu70033-bib-0092" class="usa-link" aria-describedby="jeu70033-bib-0092">2015</a>; Fernandes and Correr‐Da‐Silva <a href="#jeu70033-bib-0029" class="usa-link" aria-describedby="jeu70033-bib-0029">2020</a>; Malviya et al. <a href="#jeu70033-bib-0076" class="usa-link" aria-describedby="jeu70033-bib-0076">2016</a>). According to the metabarcoding of the V4 <em>18S</em> ribosomal RNA (rRNA) completed by the <em>Tara Oceans</em> expedition, <em>Minidiscus</em> (encompassing both <em>Minidiscus</em> and <em>Mediolabrus</em>) is the 10th most abundant diatom genus in the global ocean (Malviya et al. <a href="#jeu70033-bib-0076" class="usa-link" aria-describedby="jeu70033-bib-0076">2016</a>; Leblanc et al. <a href="#jeu70033-bib-0062" class="usa-link" aria-describedby="jeu70033-bib-0062">2018</a>; Pierella Karlusich et al. <a href="#jeu70033-bib-0093" class="usa-link" aria-describedby="jeu70033-bib-0093">2025</a>). More locally within the NWA, genetic markers targeting the highly variable regions (V4V5 and V6V8) of <em>16S</em> rRNA have identified that amplicon sequence variants (ASVs) in the <em>Minidiscus</em> genus rank among the top ten most common diatom genera in the Bedford Basin (BB) and the Scotian Shelf (SS) from 2014 to 2017 (Robicheau et al. <a href="#jeu70033-bib-0099" class="usa-link" aria-describedby="jeu70033-bib-0099">2022</a>).</p>
<p>Due to their size, it was previously assumed that nanoplanktonic diatoms were primarily remineralized in the microbial loop; however, recent observations have revealed that they can form large blooms and contribute to carbon export with high sinking rates (Bolaños et al. <a href="#jeu70033-bib-0009" class="usa-link" aria-describedby="jeu70033-bib-0009">2020</a>; Leblanc et al. <a href="#jeu70033-bib-0062" class="usa-link" aria-describedby="jeu70033-bib-0062">2018</a>; Malviya et al. <a href="#jeu70033-bib-0076" class="usa-link" aria-describedby="jeu70033-bib-0076">2016</a>; Arsenieff et al. <a href="#jeu70033-bib-0005" class="usa-link" aria-describedby="jeu70033-bib-0005">2020</a>). Carbon export is further supported through the identification of frustules in Scanning Electron Microscopy (SEM) images of sediment traps following a large diatom bloom dominated by <em>Mediolabrus comicus</em> and <em>Minidiscus trioculatus</em> in the Mediterranean Sea (Leblanc et al. <a href="#jeu70033-bib-0062" class="usa-link" aria-describedby="jeu70033-bib-0062">2018</a>). These findings suggest that, unlike previous assumptions, nanoplanktonic diatoms may play a significant role in carbon export that has been historically underestimated due to size‐limited sampling methods (Leblanc et al. <a href="#jeu70033-bib-0062" class="usa-link" aria-describedby="jeu70033-bib-0062">2018</a>). The higher proportion of unassigned reads in smaller size fractions (3/5–20 μm) reflects the limited representation of nanoplanktonic diatoms in genetic reference databases, in contrast to diatoms that have been identified in larger size fractions (Pierella Karlusich et al. <a href="#jeu70033-bib-0093" class="usa-link" aria-describedby="jeu70033-bib-0093">2025</a>). It is therefore critical that we investigate nanoplanktonic diatoms and discover more about their distribution, phylogeny, and physiology.</p>
<p>Distinguishing between centric nanoplanktonic diatoms is difficult using conventional light microscopy, as demonstrated by the recent reclassification of <em>Minidiscus</em> species. It is clear that molecular information in conjunction with enhanced morphological investigations (i.e., using SEM) are needed to more accurately identify these microbes (Li et al. <a href="#jeu70033-bib-0068" class="usa-link" aria-describedby="jeu70033-bib-0068">2020</a>). Oftentimes, smaller barcodes of DNA are unable to distinguish organisms down to the species‐level. Full mitochondrial (mt) and chloroplast (cp) genomes have been proposed as potential ‘super‐barcodes’ that can be used to effectively distinguish between organisms (Wu et al. <a href="#jeu70033-bib-0123" class="usa-link" aria-describedby="jeu70033-bib-0123">2021</a>). As the cost decreases due to the increasing efficiency of molecular methods, using longer nuclear or plastid molecular markers to examine microbial communities is becoming feasible (Wu et al. <a href="#jeu70033-bib-0123" class="usa-link" aria-describedby="jeu70033-bib-0123">2021</a>).</p>
<p>This work explores three nanoplanktonic diatoms in the Thalassiosiraceae family that were isolated from water collected in the BB and the SS, both from within the NWA: <em>Minidiscus spinulatus</em> (BB), <em>Minidiscus trioculatus</em> (SS), and <em>Mediolabrus comicus</em> (SS) (the third species previously being known as <em>Minidiscus comicus</em>). Prior work by Mordret et al. (<a href="#jeu70033-bib-0081" class="usa-link" aria-describedby="jeu70033-bib-0081">2024</a>) identified strains AGSB‐0017 (NCBI Accession: <a href="https://www.ncbi.nlm.nih.gov/nuccore/OR922670" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OR922670</a>) and AGSB‐0018 (NCBI Accession: <a href="https://www.ncbi.nlm.nih.gov/nuccore/OR922671" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OR922671</a>) as <em>Mediolabrus comicus</em> and <em>Minidiscus trioculatus</em>, respectively, using long rRNA fragments that included <em>18S</em>, <em>ITS</em>, and <em>28S</em> rRNA gene regions. We report the complete organellar genomes from these three Northwest Atlantic isolates, revealing differences in genome organization compared to previously sequenced organellar genomes in the Thalassiosirales order. Additionally, phylogenomic trees completed with concatenated alignments of protein‐coding genes re‐emphasize the classification of <em>M. comicus</em> as more closely related to the <em>Skeletonema</em> genus and the classification of <em>Minidiscus</em> species as closer to select <em>Thalassiosira</em>. The complete cp sequences also allowed us to revisit the distribution of these three species using highly resolved (to the species‐level) environmental <em>16S</em> rRNA data from the BB, SS, Labrador Sea (LS), and Canadian Arctic Gateway (CAG). Overall, this study broadens our understanding of the phylogeny, evolution, and distribution of nanoplanktonic centric diatoms in the Northwest Atlantic Ocean.</p></section><section id="jeu70033-sec-0002"><h2 class="pmc_sec_title">2. Materials and Methods</h2>
<section id="jeu70033-sec-0003"><h3 class="pmc_sec_title">2.1. Sampling, Isolation, and Culture Maintenance</h3>
<p>Diatoms were isolated from NWA coastal seawater in the BB and SS (Figure <a href="#jeu70033-supitem-0001" class="usa-link">S1</a>). <em>Minidiscus spinulatus</em> (AGSB‐0477) was isolated from seawater collected at the Compass Buoy Station sampled through the Fisheries and Oceans (DFO) Bedford Basin Monitoring Program (BBMP; <a href="https://www.bio.gc.ca/science/monitoring-monitorage/bbmp-pobb/bbmp-pobb-en.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Bedford Basin Monitoring Program</a>) [44°41′37″ N, 63°38′25″ W] on July 25, 2018 (NS, Canada) (Li et al. <a href="#jeu70033-bib-0067" class="usa-link" aria-describedby="jeu70033-bib-0067">1998</a>). For this isolate, 1 m seawater was plated on Artificial Seawater (ASW) media with 0.2 μM NaNO<sub>3</sub> plus 1.2% w/v agarose and was incubated at 15°C ± 1°C via a 12 h:12 h light:dark cycle at an irradiance of 100 μmol/m<sup>2</sup>/s. <em>Minidiscus spinulatus</em> was maintained for several years on the above medium by transferring it monthly. The <em>M. trioculatus</em> (AGSB‐0018) and <em>M. comicus</em> (AGSB‐0017) strains were isolated from seawater samples collected with a CTD‐Rosette at chlorophyll maximum depths (28 m) on August 8, 2021, during the COOGER‐2021 research cruise [45°30′37.70″ N, 60°39′5.47″ W] (Mordret et al. <a href="#jeu70033-bib-0081" class="usa-link" aria-describedby="jeu70033-bib-0081">2024</a>). Both strains were isolated by dilution series with L1 medium (NCMA/Bigelow Laboratory for Ocean Sciences, USA) and maintained in liquid culture at a 16:8 h light–dark cycle under 30 μmol of light (Mordret et al. <a href="#jeu70033-bib-0081" class="usa-link" aria-describedby="jeu70033-bib-0081">2024</a>).</p>
<p>In preparation for genomics work, the three strains were individually grown in 50 mL liquid cultures of L1 made with 0.2 μm filtered seawater [at ~34 ppt salinity] (NCMA/Bigelow Laboratory for Ocean Sciences, USA). Phytoplankton isolates were treated with an antibiotic mixture at a final concentration of 100 μg/mL penicillin G, 25 μg/mL dihydrostreptomycin sulfate, and 25 μg/mL gentamycin sulfate, minimizing bacterial contamination. Each strain was transferred every 2–3 weeks and maintained at 16°C ± 1°C. All three cultures are available in the Algal Genomics and Synthetic Biology (AGSB) culture collection maintained by the National Research Council of Canada (NRC) in Halifax, Nova Scotia.</p></section><section id="jeu70033-sec-0004"><h3 class="pmc_sec_title">2.2. Nucleic Acid Extraction and Genomic Sequencing</h3>
<p>Isolates were grown to 1–2 × 10<sup>9</sup> cells L<sup>−1</sup> and syringe‐filtered onto 3 μm nucleopore filters (Millipore, Germany). DNA was extracted from the cells using the enhanced lysis procedure described by Zorz et al. (<a href="#jeu70033-bib-0127" class="usa-link" aria-describedby="jeu70033-bib-0127">2019</a>) and the DNeasy Plant Extraction Kit (Qiagen, Germany). DNA was further concentrated using the Genomic DNA Clean &amp; Concentrator‐10 kit D4011 (Zymo Research, USA). Sample concentration and purity were assessed using a NanoDrop 2000 (Thermo Scientific, USA) and a Qubit 4 fluorometer via a High‐Sensitivity dsDNA Quantification Assay kit (Thermo Scientific, USA). Extracted genomic DNA was then sequenced using a Nanopore Ligation Sequencing Kit V14 (SQK‐LSK114) and a flow cell (R10.4.1) for each isolate according to the manufacturer's instructions (Noakes et al. <a href="#jeu70033-bib-0084" class="usa-link" aria-describedby="jeu70033-bib-0084">2019</a>; Wang et al. <a href="#jeu70033-bib-0116" class="usa-link" aria-describedby="jeu70033-bib-0116">2021</a>). Nanopore sequencing started with 1402–1491 pores available and ran for 32–48 h depending on the isolate, with runs resulting in 10.99–15.59 Gb and 2.84–5.56 M reads of data (Data <a href="#jeu70033-supitem-0001" class="usa-link">S1</a>).</p>
<p>Sequences were base‐called from FAST5 files and converted to FASTQ files. Porechop (v0.2.4) was used to trim adapters, filtlong (v0.2.1) to filter out low‐quality sequences, and then raven (v6.10.0) and raven‐assembler (v1.6.1) were used to assemble long reads into contigs with default parameters (Wick et al. <a href="#jeu70033-bib-0117" class="usa-link" aria-describedby="jeu70033-bib-0117">2017</a>; Vaser and Šikić <a href="#jeu70033-bib-0110" class="usa-link" aria-describedby="jeu70033-bib-0110">2020</a>, <a href="#jeu70033-bib-0111" class="usa-link" aria-describedby="jeu70033-bib-0111">2021</a>). Once assembled, the long‐read polishing programs pepper (v8.0) and medaka (v1.11.3) were used to polish the contigs (Shafin et al. <a href="#jeu70033-bib-0105" class="usa-link" aria-describedby="jeu70033-bib-0105">2021</a>; Lee et al. <a href="#jeu70033-bib-0063" class="usa-link" aria-describedby="jeu70033-bib-0063">2021</a>). Putative homopolymer‐associated frameshifts, commonly observed with nanopore sequencing, were manually inspected and corrected in Geneious Prime 2024.0.7. In Geneious, contigs with low GC‐content (29.1%–31.3%) were identified as mt and cp sequences, some of which were completed; their circularity was confirmed using the program GetOrganelle (v1.7.7.1) (Jin et al. <a href="#jeu70033-bib-0052" class="usa-link" aria-describedby="jeu70033-bib-0052">2020</a>). The cp genomes of <em>M. trioculatus</em> and <em>M. comicus</em> contained self‐overlapping regions with multiple inverted repeats, which were manually resolved in Geneious to produce circular organellar genomes.</p></section><section id="jeu70033-sec-0005"><h3 class="pmc_sec_title">2.3. Phylogenetic Tree of Related Species From the 
<em>18S</em> rRNA Gene</h3>
<p>A phylogenetic tree of the <em>18S</em> rRNA gene was constructed using 92 related species to determine the evolutionary relationships of the cultured isolates and to classify the AGSB‐0477 culture, which had not been identified elsewhere. The <em>18S</em> rRNA sequences were aligned using MUSCLE with default parameters, employing the PPP algorithm (Edgar <a href="#jeu70033-bib-0026" class="usa-link" aria-describedby="jeu70033-bib-0026">2004</a>). To determine the best‐fit model of nucleotide substitution, we used ModelFinder implemented in IQ‐TREE2 (v2.3.6), which identified GTR + F + I + R3 as the optimal model based on the Bayesian Information Criterion (BIC) (Minh et al. <a href="#jeu70033-bib-0080" class="usa-link" aria-describedby="jeu70033-bib-0080">2020</a>; Kalyaanamoorthy et al. <a href="#jeu70033-bib-0054" class="usa-link" aria-describedby="jeu70033-bib-0054">2017</a>). Maximum likelihood analysis was first performed using IQ‐TREE2 with 1000 ultrafast bootstrap replicates and model selection restricted to models compatible with MrBayes for later comparison. Additional maximum likelihood analysis was carried out using PHYML (v. 3.0) with 1000 bootstrap replicates under a GTR substitution model (Guindon et al. <a href="#jeu70033-bib-0038" class="usa-link" aria-describedby="jeu70033-bib-0038">2005</a>, <a href="#jeu70033-bib-0037" class="usa-link" aria-describedby="jeu70033-bib-0037">2010</a>). Bayesian inference was performed in MrBayes (v. 3.2.6) under the GTR + G model, with four Markov chains run for 1,100,000 generations, sampling every 200 generations (Huelsenbeck and Ronquist <a href="#jeu70033-bib-0049" class="usa-link" aria-describedby="jeu70033-bib-0049">2001</a>; Guindon et al. <a href="#jeu70033-bib-0038" class="usa-link" aria-describedby="jeu70033-bib-0038">2005</a>, <a href="#jeu70033-bib-0037" class="usa-link" aria-describedby="jeu70033-bib-0037">2010</a>). The IQ‐TREE2 software was used as the basis of the phylogenetic tree, and the PHYML bootstraps and Bayesian inferences were added as additional supports using Inkscape (v. 1.1.2). Biomass preparation and DNA extraction for this analysis were the same as that described below for genomic sequencing. The <em>18S</em> rRNA genes were amplified and sequenced using the methods described by Mordret et al. (<a href="#jeu70033-bib-0081" class="usa-link" aria-describedby="jeu70033-bib-0081">2024</a>).</p></section><section id="jeu70033-sec-0006"><h3 class="pmc_sec_title">2.4. Organellar Genome Annotation and Comparison</h3>
<p>Organellar genomes were first annotated using MFannot with the codon table for “Mold, Protozoan, Coelenterate Mitochondrial and Mycoplasma/Spiroplasma Code” selected for mt and the codon table “Bacterial, Archaeal, and Plant Plastid Code” selected for cps (Lang et al. <a href="#jeu70033-bib-0059" class="usa-link" aria-describedby="jeu70033-bib-0059">2023</a>). The MFannot output was converted to GenBank format using the AGAT v1.4.2 toolkit to convert the files to GFF3 format and the EMBOSS server to convert GFF3 files to GenBank format (Rice et al. <a href="#jeu70033-bib-0097" class="usa-link" aria-describedby="jeu70033-bib-0097">2000</a>). RNAweasel and MFannot were used to check tRNAs, introns (group I and group II), rnpB, rrn5, and rns (Greiner et al. <a href="#jeu70033-bib-0034" class="usa-link" aria-describedby="jeu70033-bib-0034">2019</a>; Tillich et al. <a href="#jeu70033-bib-0109" class="usa-link" aria-describedby="jeu70033-bib-0109">2017</a>). The annotations were subsequently checked using a variety of tools, including tRNAscan‐SE (v2.0.7) software and ARAGORN to identify introns in the mt (maximum length = 3000 bp) and Basic Local Alignment Search Tool (BLAST) against diatom reference genomes (Laslett and Canback <a href="#jeu70033-bib-0060" class="usa-link" aria-describedby="jeu70033-bib-0060">2004</a>; Chan and Lowe <a href="#jeu70033-bib-0014" class="usa-link" aria-describedby="jeu70033-bib-0014">2019</a>; Tillich et al. <a href="#jeu70033-bib-0109" class="usa-link" aria-describedby="jeu70033-bib-0109">2017</a>). For cps, references included <em>Minidiscus spinulatus</em>, <em>Conticribra weissflogii</em>, <em>Roundia cardiophora</em>, <em>Thalassiosira oceanica</em> CCMP1005 and 
<em>Thalassiosira pseudonana</em>
 and for mt, this included <em>Skeletonema marinoi</em> and 
<em>Thalassiosira pseudonana</em>
 (Tillich et al. <a href="#jeu70033-bib-0109" class="usa-link" aria-describedby="jeu70033-bib-0109">2017</a>). Annotations were visualized on circular genome maps using OGDRAW v1.3.1 (Figure <a href="#jeu70033-supitem-0001" class="usa-link">S2</a>) and with the software program Proksee (v2.0) (Figure <a href="#jeu70033-fig-0002" class="usa-link">2</a>) (Greiner et al. <a href="#jeu70033-bib-0034" class="usa-link" aria-describedby="jeu70033-bib-0034">2019</a>; Grant et al. <a href="#jeu70033-bib-0033" class="usa-link" aria-describedby="jeu70033-bib-0033">2023</a>). A BLAST comparison of each organellar genome to the two other isolates was conducted using the “BLAST” feature of the Proksee software (Grant et al. <a href="#jeu70033-bib-0033" class="usa-link" aria-describedby="jeu70033-bib-0033">2023</a>).</p>
<figure class="fig xbox font-sm" id="jeu70033-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2928/12326293/e8b634ac42f5/JEU-72-e70033-g006.jpg" loading="lazy" id="jats-graphic-3" height="640" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/jeu70033-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>BLAST comparison of the nanoplanktonic diatom mitochondrial genomes using Proksee (Grant et al. <a href="#jeu70033-bib-0033" class="usa-link" aria-describedby="jeu70033-bib-0033">2023</a>). Mitochondrial genomes were BLASTed against the other two isolates to highlight conserved genes. For ease of viewing, introns are highlighted. Innermost ring = GC‐content. The first two inner rings are annotations of key mt components on the forward and reverse strands. The outer two rings represent the BLAST alignment to the other two mt genomes; white = no alignment, and gradient = percent identity (%) for matches.</p></figcaption></figure></section><section id="jeu70033-sec-0007"><h3 class="pmc_sec_title">2.5. Comparison of Organellar Genomes Through Concatenated Phylogenomic and Pangenomic Analyses</h3>
<p>All available complete mt and cp genomes from the Thalassiosirales taxonomic order were downloaded from Genbank (Clark et al. <a href="#jeu70033-bib-0016" class="usa-link" aria-describedby="jeu70033-bib-0016">2016</a>). For the mt genomes, this resulted in 16 mt genomes, including the three provided in this study (Table <a href="#jeu70033-supitem-0001" class="usa-link">S2</a>) (Clark et al. <a href="#jeu70033-bib-0016" class="usa-link" aria-describedby="jeu70033-bib-0016">2016</a>). For these 16 genomes, we constructed a concatenated phylogenetic tree of 28 protein‐coding mt genes (see Table <a href="#jeu70033-supitem-0001" class="usa-link">S4</a> for gene list) (Cox et al. <a href="#jeu70033-bib-0020" class="usa-link" aria-describedby="jeu70033-bib-0020">2024</a>). For the cp genomes, identical sequences of the same species were removed, which resulted in 37 complete cp genomes obtained from Genbank—34 previously available and the three obtained herein (Table <a href="#jeu70033-supitem-0001" class="usa-link">S3</a>) (Clark et al. <a href="#jeu70033-bib-0016" class="usa-link" aria-describedby="jeu70033-bib-0016">2016</a>). Among these genomes is the previously mentioned <em>Minidiscus spinulatus</em>, isolated from Jiaozhou Bay, China (Nordberg et al. <a href="#jeu70033-bib-0085" class="usa-link" aria-describedby="jeu70033-bib-0085">2014</a>; Wang and Chen <a href="#jeu70033-bib-0114" class="usa-link" aria-describedby="jeu70033-bib-0114">2022</a>). There were 97 protein‐coding genes selected for the concatenated alignment and phylogenetic tree (see Table <a href="#jeu70033-supitem-0001" class="usa-link">S5</a> for the gene list) (Cox et al. <a href="#jeu70033-bib-0020" class="usa-link" aria-describedby="jeu70033-bib-0020">2024</a>).</p>
<p>In the case of the phylogenetic trees, there were two organisms from the <em>Lithodesmiales</em> order used as an outgroup: <em>Helicotheca tamesis</em> and 
<em>Lithodesmium undulatum</em>
. Organellar genomes were converted to contigs databases using the Anvi'o function ‘anvi‐gen‐contigs‐database’ (Eren et al. <a href="#jeu70033-bib-0027" class="usa-link" aria-describedby="jeu70033-bib-0027">2015</a>). Each genome database was then annotated with a custom HMM profile created from select protein‐coding genes in the cp (97) or mt (25) (Tables <a href="#jeu70033-supitem-0001" class="usa-link">S4</a> and <a href="#jeu70033-supitem-0001" class="usa-link">S5</a>). The function ‘anvi‐run‐hmms’ was run on each of the contig databases, and then the concatenated alignment was retrieved using ‘anvi‐get‐sequences‐for‐hmm‐hits’ and the MUSCLE alignment algorithm (Eren et al. <a href="#jeu70033-bib-0027" class="usa-link" aria-describedby="jeu70033-bib-0027">2015</a>; Edgar <a href="#jeu70033-bib-0026" class="usa-link" aria-describedby="jeu70033-bib-0026">2004</a>). The model for each of the partitions was determined using ModelFinder in IQ‐TREE2 (v2.3.6), which applied different best‐fit models (e.g., TIM3 + F + I + R3, GTR + F + R4) to each partitioned gene set based on the Bayesian Information Criterion (BIC) (Minh et al. <a href="#jeu70033-bib-0080" class="usa-link" aria-describedby="jeu70033-bib-0080">2020</a>; Kalyaanamoorthy et al. <a href="#jeu70033-bib-0054" class="usa-link" aria-describedby="jeu70033-bib-0054">2017</a>). In both the mt and cp trees, the maximum likelihood analysis was conducted using IQ‐TREE2 with 1000 ultrafast bootstrap replicates (Minh et al. <a href="#jeu70033-bib-0080" class="usa-link" aria-describedby="jeu70033-bib-0080">2020</a>). To further compare the similarity of the organisms, the same contigs databases were used in the Anvi'o pangenome pipeline (Eren et al. <a href="#jeu70033-bib-0027" class="usa-link" aria-describedby="jeu70033-bib-0027">2015</a>). Additionally, the average nucleotide identity (ANI) was calculated using the fastANI algorithm (Hernández‐Salmerón and Moreno‐Hagelsieb <a href="#jeu70033-bib-0047" class="usa-link" aria-describedby="jeu70033-bib-0047">2022</a>).</p></section><section id="jeu70033-sec-0008"><h3 class="pmc_sec_title">2.6. Sample Collection for Environmental Metabarcoding Analyses</h3>
<p>The distribution of nanoplanktonic diatom isolates in the NWA was examined using metabarcoding datasets and related environmental data compiled over multiple years from various regions, thanks to the work of many research groups. These datasets included weekly sampling through the BBMP and multiple expeditions to the SS through the DFO Atlantic Zone Monitoring Program (AZMP; <a href="https://www.dfo-mpo.gc.ca/science/data-donnees/azmp-pmza/index-eng.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Atlantic Zone Monitoring Program (AZMP</a>)), LS through the DFO Atlantic Zone Off‐Shelf Monitoring Program (AZOMP; <a href="https://www.bio.gc.ca/science/monitoring-monitorage/azomp-pmzao/azomp-pmzao-en.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Atlantic Zone Off‐Shelf Monitoring Program (AZOMP</a>)) and the Ocean Frontier Institute's Northwest Atlantic Biological Carbon Pump project (NWA‐BCP; <a href="https://www.ofi.ca/programs/safe-and-sustainable/project/the-northwest-atlantic-biological-carbon-pump" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">The Northwest Atlantic Biological Carbon Pump</a>) and the CAG through the GEOTRACES program (<a href="https://www.geotraces.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GEOTRACES</a>) (Table <a href="#jeu70033-tbl-0001" class="usa-link">1</a>) (Li et al. <a href="#jeu70033-bib-0067" class="usa-link" aria-describedby="jeu70033-bib-0067">1998</a>; Therriault et al. <a href="#jeu70033-bib-0108" class="usa-link" aria-describedby="jeu70033-bib-0108">1998</a>; Casault et al. <a href="#jeu70033-bib-0013" class="usa-link" aria-describedby="jeu70033-bib-0013">2024</a>; Yashayaev et al. <a href="#jeu70033-bib-0124" class="usa-link" aria-describedby="jeu70033-bib-0124">2021</a>). Due to the various sampling expeditions, some discrepancies in biomass sampling methods are important to take into account. The biomass samples for the SS, LS, and CAG expeditions were all size‐fractionated through large 3 μm (L) and small 0.2 μm (S) filters. In the case of the NWA‐BCP CE expedition to the LS, there were also marine snow catcher deployments at various sites with corresponding eDNA data. In the case of the BBMP, the filtering methods to collect biomass for DNA extraction have varied over that time. For some years (2014–2017, 2022), there was no size fractionation of the water sample, and therefore the sample was filtered through one 0.2 μm filter (W). However, from 2018 to 2021, the samples were size‐fractionated with the large 3 μm (L) and small 0.2 μm filters (S). To have comparable data across the years 2014–2022, the relative abundance of the sample was reconstituted using DNA concentration as a proxy for biomass (SL) (Haider et al. <a href="#jeu70033-bib-0043" class="usa-link" aria-describedby="jeu70033-bib-0043">2025</a>).</p>
<section class="tw xbox font-sm" id="jeu70033-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Environmental data collected from various research cruises from 2014 to 2022.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Year</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Mission ID</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Dates</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Research vessel</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Code</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Region</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2014</td>
<td align="center" valign="top" rowspan="1" colspan="1">HUD2014‐004</td>
<td align="center" valign="top" rowspan="1" colspan="1">Apr 4–23</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>CCGS Hudson</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">AS14</td>
<td align="center" valign="top" rowspan="1" colspan="1">Scotian Shelf</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2014</td>
<td align="center" valign="top" rowspan="1" colspan="1">HUD2014‐030</td>
<td align="center" valign="top" rowspan="1" colspan="1">Sep 19–Oct 8</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>CCGS Hudson</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">AF14</td>
<td align="center" valign="top" rowspan="1" colspan="1">Scotian Shelf</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2015</td>
<td align="center" valign="top" rowspan="1" colspan="1">GEOTRACES15</td>
<td align="center" valign="top" rowspan="1" colspan="1">July 10–Aug 20</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>CCGS Amundsen</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">Arctic15</td>
<td align="center" valign="top" rowspan="1" colspan="1">Arctic</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2016</td>
<td align="center" valign="top" rowspan="1" colspan="1">HUD2016‐003</td>
<td align="center" valign="top" rowspan="1" colspan="1">Apr 17–27</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>CCGS Hudson</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">AS16</td>
<td align="center" valign="top" rowspan="1" colspan="1">Scotian Shelf</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2016</td>
<td align="center" valign="top" rowspan="1" colspan="1">HUD2016‐027</td>
<td align="center" valign="top" rowspan="1" colspan="1">Sep 15–Oct 6</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>CCGS Hudson</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">AF16</td>
<td align="center" valign="top" rowspan="1" colspan="1">Scotian Shelf</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2017</td>
<td align="center" valign="top" rowspan="1" colspan="1">COR2017‐001</td>
<td align="center" valign="top" rowspan="1" colspan="1">Apr 17–27</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>CCGS Coriolis II</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">AS17</td>
<td align="center" valign="top" rowspan="1" colspan="1">Scotian Shelf</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2017</td>
<td align="center" valign="top" rowspan="1" colspan="1">EN2017‐606</td>
<td align="center" valign="top" rowspan="1" colspan="1">Nov 26–Dec 16</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>RV Endeavor</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">AF17</td>
<td align="center" valign="top" rowspan="1" colspan="1">Scotian Shelf</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2018</td>
<td align="center" valign="top" rowspan="1" colspan="1">HUD2018‐004</td>
<td align="center" valign="top" rowspan="1" colspan="1">Apr 8–23</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>CCGS Hudson</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">AS18</td>
<td align="center" valign="top" rowspan="1" colspan="1">Scotian Shelf</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2019</td>
<td align="center" valign="top" rowspan="1" colspan="1">COR2019‐001</td>
<td align="center" valign="top" rowspan="1" colspan="1">Apr 7–25</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>CCGS Coriolis II</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">AS19</td>
<td align="center" valign="top" rowspan="1" colspan="1">Scotian Shelf</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2019</td>
<td align="center" valign="top" rowspan="1" colspan="1">AMU2019‐001</td>
<td align="center" valign="top" rowspan="1" colspan="1">June 9–21</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>CCGS Amundsen</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">AZOMP19</td>
<td align="center" valign="top" rowspan="1" colspan="1">Labrador Sea</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2022</td>
<td align="center" valign="top" rowspan="1" colspan="1">AT4805</td>
<td align="center" valign="top" rowspan="1" colspan="1">May 10–20</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>RV Atlantis</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">AZOMP22</td>
<td align="center" valign="top" rowspan="1" colspan="1">Labrador Sea</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2022</td>
<td align="center" valign="top" rowspan="1" colspan="1">CE22009</td>
<td align="center" valign="top" rowspan="1" colspan="1">May 11–June 2</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>RV Celtic Explorer</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">BELAS‐1</td>
<td align="center" valign="top" rowspan="1" colspan="1">Labrador Sea</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2014–2022</td>
<td align="center" valign="top" rowspan="1" colspan="1">BBMP</td>
<td align="center" valign="top" rowspan="1" colspan="1">Weekly Jan‐Dec</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>CCGS Sigma T</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">BBMP</td>
<td align="center" valign="top" rowspan="1" colspan="1">Bedford Basin</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/jeu70033-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="jeu70033-sec-0009"><h3 class="pmc_sec_title">2.7. Corresponding Oceanographic Data and Flow Cytometry Measurements</h3>
<p>The BBMP is a long‐term weekly time series that has characterized microbial communities using flow cytometry since the 1990s (Li and Dickie <a href="#jeu70033-bib-0066" class="usa-link" aria-describedby="jeu70033-bib-0066">2001</a>; Li <a href="#jeu70033-bib-0065" class="usa-link" aria-describedby="jeu70033-bib-0065">1998</a>; Cullen et al. <a href="#jeu70033-bib-0021" class="usa-link" aria-describedby="jeu70033-bib-0021">2007</a>). In 2014, environmental DNA (eDNA) metabarcoding was incorporated into the monitoring efforts to further investigate microbial diversity. In this work, we used BBMP oceanographic measurements and flow cytometry measurements to make ecological predictions related to the prevalence of our three isolates of interest. Samples for the BBMP were collected at depths of 1, 5, 10, and 60 m. Environmental parameters, including temperature, chlorophyll <em>a</em>, and nutrient concentrations, were obtained from the publicly available dataset maintained by the Bedford Institute of Oceanography (BIO) (<a href="http://www.bio.gc.ca/science/monitoring-monitorage/bbmp-pobb/bbmp-pobb-en.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Bedford Basin Monitoring Program</a>). For consistency, flow cytometry cell counts (&gt; 35 μm) for 2014–2017 were obtained from Robicheau et al. (<a href="#jeu70033-bib-0099" class="usa-link" aria-describedby="jeu70033-bib-0099">2022</a>), while counts for 2018–2022 were conducted in the LaRoche lab using a BD Accuri flow cytometer (2018–2021) and a Novocyte flow cytometer (2021–2022) from Agilent Technologies. In both cases, a threshold of &lt; 450 was set for Chlorophyll‐H (proxy for chlorophyll), and approximate size gates of &lt; 3, 3–10, and &gt; 10 μm were established for forward scatter via Novocyte QC particles with a known size and fluorescence profile (Agilent Technologies).</p></section><section id="jeu70033-sec-0010"><h3 class="pmc_sec_title">2.8. Data Analyses for the Distribution of the Three Isolates</h3>
<p>To investigate the prevalence of the three isolates in environmental genomic data, cp genomes were aligned to the <em>16S</em> rRNA sequence available from 
<em>M. spinulatus</em>
 (Wang and Chen <a href="#jeu70033-bib-0114" class="usa-link" aria-describedby="jeu70033-bib-0114">2022</a>). Identified <em>16S</em> rRNA sequences were then aligned to ASVs for the V4V5 and V6V8 <em>16S</em> rRNA variable regions from environmental metabarcoding datasets (Data <a href="#jeu70033-supitem-0001" class="usa-link">S2</a>). Datasets from all of the expeditions were analyzed with the same pipeline as in Robicheau et al. (<a href="#jeu70033-bib-0099" class="usa-link" aria-describedby="jeu70033-bib-0099">2022</a>), with the only difference being that the ASV data was not rarefied (Robicheau et al. <a href="#jeu70033-bib-0099" class="usa-link" aria-describedby="jeu70033-bib-0099">2022</a>; Comeau et al. <a href="#jeu70033-bib-0018" class="usa-link" aria-describedby="jeu70033-bib-0018">2017</a>, <a href="#jeu70033-bib-0019" class="usa-link" aria-describedby="jeu70033-bib-0019">2011</a>; Willis et al. <a href="#jeu70033-bib-0120" class="usa-link" aria-describedby="jeu70033-bib-0120">2019</a>; Bolyen et al. <a href="#jeu70033-bib-0010" class="usa-link" aria-describedby="jeu70033-bib-0010">2019</a>; Decelle et al. <a href="#jeu70033-bib-0023" class="usa-link" aria-describedby="jeu70033-bib-0023">2015</a>; Quast et al. <a href="#jeu70033-bib-0095" class="usa-link" aria-describedby="jeu70033-bib-0095">2013</a>; Walters et al. <a href="#jeu70033-bib-0113" class="usa-link" aria-describedby="jeu70033-bib-0113">2016</a>; Parada et al. <a href="#jeu70033-bib-0088" class="usa-link" aria-describedby="jeu70033-bib-0088">2016</a>; Amir et al. <a href="#jeu70033-bib-0003" class="usa-link" aria-describedby="jeu70033-bib-0003">2017</a>). In determining the presence of these organisms at all geographic locations, a single identification of the nanoplanktonic diatom <em>16S</em> barcode within the 1–20 m depth range at a set of geographic coordinates in a given year was considered a detection of the species at that site. To quantify the prevalence of the three diatoms in the photosynthetic community within a water sample, relative abundances were calculated as the percentage of total cp <em>16S</em> reads in that sample.</p>
<p>Downstream data analysis and visualization were conducted using R (v4.4.2) and RStudio (v2024.09.1 + 394) with various packages (R Core Team <a href="#jeu70033-bib-0096" class="usa-link" aria-describedby="jeu70033-bib-0096">2021</a>; RStudio Team <a href="#jeu70033-bib-0102" class="usa-link" aria-describedby="jeu70033-bib-0102">2020</a>). Tidyverse (v2.2.0), tidyr (v1.3.1), and dplyr (v1.1.4) packages were used for initial data cleaning and filtering, and the microbiome (v1.28.0) and phyloseq (v2.12.1) packages were used to handle the microbial data (Wickham et al. <a href="#jeu70033-bib-0118" class="usa-link" aria-describedby="jeu70033-bib-0118">2019</a>, <a href="#jeu70033-bib-0119" class="usa-link" aria-describedby="jeu70033-bib-0119">2023</a>; Sudarshan <a href="#jeu70033-bib-0107" class="usa-link" aria-describedby="jeu70033-bib-0107">2012–2019</a>; McMurdie and Holmes <a href="#jeu70033-bib-0079" class="usa-link" aria-describedby="jeu70033-bib-0079">2013</a>; Boehmke <a href="#jeu70033-bib-0008" class="usa-link" aria-describedby="jeu70033-bib-0008">2016</a>). The redundancy analysis (RDA) using the environmental variables in the BBMP was performed using vegan (v2.6‐10) and microbiome (v1.28.0) packages (Sudarshan <a href="#jeu70033-bib-0107" class="usa-link" aria-describedby="jeu70033-bib-0107">2012–2019</a>; Dixon <a href="#jeu70033-bib-0024" class="usa-link" aria-describedby="jeu70033-bib-0024">2003</a>). ASV feature tables were first CLR transformed with the ‘microbiome::transform()’ function (Sudarshan <a href="#jeu70033-bib-0107" class="usa-link" aria-describedby="jeu70033-bib-0107">2012–2019</a>). All numerical environmental variables with sufficient data were initially retained (1530 out of 1736 samples). The environmental variables were checked for correlation using heatmaps, and variables were selected based on interest; highly correlated variables (|&gt; 0.7|) were then removed (Figure <a href="#jeu70033-supitem-0001" class="usa-link">S11</a>). Standardization of the numerical values was performed using ‘deconstand()’ which scaled the data to a mean of zero, and then the RDA was conducted using the ‘rda()’ function (Dixon <a href="#jeu70033-bib-0024" class="usa-link" aria-describedby="jeu70033-bib-0024">2003</a>). The significance of the RDA and each environmental variable was done using the ‘anova()’ function with 999 permutations. Variables deemed insignificant were discarded, resulting in a reduced RDA that focused on select oceanographic measurements (salinity, temperature, oxygen), nutrients (nitrate, ammonia), chlorophyll <em>a</em> and flow cytometry measurements (nanoeukaryotes and <em>Synechococcus</em>).</p></section></section><section id="jeu70033-sec-0011"><h2 class="pmc_sec_title">3. Results</h2>
<section id="jeu70033-sec-0012"><h3 class="pmc_sec_title">3.1. 
NWA Isolates Are Small Nanoplanktonic Diatoms Belonging to Thalassiosirales</h3>
<p>Isolates were initially identified as small centric diatoms based on cell morphology (Figure <a href="#jeu70033-fig-0001" class="usa-link">1</a>). A phylogenetic analysis of <em>18S</em> rRNA sequences again confirmed the taxonomic placement of <em>Minidiscus trioculatus</em> [AGSB‐0018; OR22670] and <em>Mediolabrus comicus</em> [AGSB‐0017; OR22671] (Mordret et al. <a href="#jeu70033-bib-0081" class="usa-link" aria-describedby="jeu70033-bib-0081">2024</a>) and further showed that the BB isolate was <em>Minidiscus spinulatus</em> [AGSB‐0477; <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV640934" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV640934</a>] (Figure <a href="#jeu70033-fig-0001" class="usa-link">1</a>). Specifically, AGSB‐0477 clustered closely with <a href="https://www.ncbi.nlm.nih.gov/nuccore/MN528624" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MN528624</a>‐<em>Minidiscus spinulatus</em> (100% alignment, 99% bootstrap support) and <a href="https://www.ncbi.nlm.nih.gov/nuccore/KY912618" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KY912618</a>‐<em>Minidiscus proschkinae</em> (previously <em>Thalassiosira proschkinae</em>; 100% alignment, 99 bootstraps), AGSB‐0017 clustered with <a href="https://www.ncbi.nlm.nih.gov/nuccore/MN528623" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MN528623</a>‐<em>Mediolabrus comicus</em> (99.9% alignment, 100% bootstrap support), and AGSB‐0018 clustered with <a href="https://www.ncbi.nlm.nih.gov/nuccore/FJ590768" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">FJ590768</a>‐<em>Minidiscus trioculatus</em> (100% alignment, 100% bootstrap support) (Li et al. <a href="#jeu70033-bib-0068" class="usa-link" aria-describedby="jeu70033-bib-0068">2020</a>). <em>Minidiscus spinulatus</em> and <em>Minidiscus proschkinae</em> have identical <em>18S</em> regions, and the AGSB‐0477 isolate was further characterized as 
<em>M. spinulatus</em>
 through additional phylogenetic trees, including its cp genome (presented later in Figure <a href="#jeu70033-fig-0005" class="usa-link">5</a>). <em>Mediolabrus</em> appeared as a sister clade of <em>Skeletonema</em>, and <em>Minidiscus</em> as a sister clade to a group containing <em>Cyclostephanos</em>, <em>Stephanodiscus</em>, <em>Pantocsekiella</em>, <em>Discostella</em>, and <em>Thalassiosira hispida</em>. In general, the <em>Minidiscus</em> isolates fell within a much broader clade containing many <em>Thalassiosira</em> species. The taxa in the <em>Thalassiosira</em> genus appear to form two different clusters in the phylogeny, one of which clusters more closely with the <em>Minidiscus</em> clade and one that forms its own cluster, dissimilar to both the <em>Mediolabrus</em> and <em>Minidiscus</em> clades.</p>
<figure class="fig xbox font-sm" id="jeu70033-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2928/12326293/aceb96b008af/JEU-72-e70033-g004.jpg" loading="lazy" id="jats-graphic-1" height="888" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/jeu70033-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>IQ‐TREE phylogenies of <em>18S</em> rRNA genes. Phylogenies generated with the GTR + F + I + R3 model in IQ‐TREE using a 1051 bp alignment and 89 closely related taxa. Species in <em>Lithodesmium</em>, <em>Helicotheca</em>, and <em>Bellerochea</em> were used as a monophyletic outgroup. Values reported at nodes indicate IQ‐TREE bootstrap support (%), PHYML bootstrap support (%), and Bayesian inferences using MrBayes, respectively (Huelsenbeck and Ronquist <a href="#jeu70033-bib-0049" class="usa-link" aria-describedby="jeu70033-bib-0049">2001</a>; Guindon et al. <a href="#jeu70033-bib-0038" class="usa-link" aria-describedby="jeu70033-bib-0038">2005</a>, <a href="#jeu70033-bib-0037" class="usa-link" aria-describedby="jeu70033-bib-0037">2010</a>). Bootstraps &gt; 50% and Bayesian inferences &gt; 0.9 are displayed, with “‐‐” indicating lower values and “**” indicating differing tree topologies. Scale bar indicates substitutions per site. Isolates are indicated with a star (★). Microscopy images show cell morphologies under oil immersion at 1000× magnification (using Olympus BX63). The associated cell diameter in micrometers (μm) for each organism was identified and is noted in Table <a href="#jeu70033-supitem-0001" class="usa-link">S1</a>.</p></figcaption></figure><figure class="fig xbox font-sm" id="jeu70033-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2928/12326293/827bef09e026/JEU-72-e70033-g002.jpg" loading="lazy" id="jats-graphic-9" height="284" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/jeu70033-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>IQ‐TREE phylogenies comparing concatenated gene trees using either (A) chloroplast or (B) mitochondrial data. The chloroplast tree uses 97 protein‐coding genes (79,470 bp total length), and the mt tree uses 25 protein‐coding genes (31,100 bp total length). All currently available cps (<em>n</em> = 35) and mt (<em>n</em> = 13) genomes in the order Thalassiosirales were included. Two species in the order <em>Lithodesmiales</em>: 
<em>Lithodesmium undulatum</em>
 and <em>Helicotheca tamesis</em>, serve as an outgroup in both cases. Best‐fit models for each of the partitioned gene sets were first selected using ModelFinder implemented in IQ‐TREE, then phylogenies were calculated with 1000 bootstraps (v2.3.6) (Eren et al. <a href="#jeu70033-bib-0027" class="usa-link" aria-describedby="jeu70033-bib-0027">2015</a>; Edgar <a href="#jeu70033-bib-0026" class="usa-link" aria-describedby="jeu70033-bib-0026">2004</a>; Minh et al. <a href="#jeu70033-bib-0080" class="usa-link" aria-describedby="jeu70033-bib-0080">2020</a>; Kalyaanamoorthy et al. <a href="#jeu70033-bib-0054" class="usa-link" aria-describedby="jeu70033-bib-0054">2017</a>). The scale bar indicates substitutions per site.</p></figcaption></figure></section><section id="jeu70033-sec-0013"><h3 class="pmc_sec_title">3.2. Distinct Mitochondrial Genome Characteristics Among the Three Isolates</h3>
<p>Mitochondrial (mt) genomes of <em>Minidiscus spinulatus</em>, <em>Minidiscus trioculatus</em>, and <em>Mediolabrus comicus</em> exhibited similar GC‐content, at 29.1%, 31.0%, and 30.5% GC, respectively. Their mt genomes, however, varied in size from 49,654 bp, 53,164 bp, to 79,303 bp for 
<em>M. spinulatus</em>
, <em>M. comicus</em>, and <em>M. trioculatus</em>, respectively (Table <a href="#jeu70033-tbl-0002" class="usa-link">2</a>). The regions of sequence contributing to the <em>M. trioculatus</em>' longer 79,303 bp mt genome include a non‐coding region (13% of the mt genome), which is primarily composed of repeats in addition to seven introns in its <em>cox1</em> gene (Figure <a href="#jeu70033-fig-0002" class="usa-link">2</a>). The three isolates shared a relatively conserved gene synteny, as is typical for species within the same order (Wang et al. <a href="#jeu70033-bib-0115" class="usa-link" aria-describedby="jeu70033-bib-0115">2022</a>). In all three, the <em>nad11</em> gene was not split into <em>nad11a</em> and <em>nad11b</em>, which is typical of species in the Order <em>Mediophyceae</em> (Wang et al. <a href="#jeu70033-bib-0115" class="usa-link" aria-describedby="jeu70033-bib-0115">2022</a>). Notably, the <em>tatA</em> gene was present only in <em>M. comicus</em> and absent in both <em>M. trioculatus</em> and 
<em>M. spinulatus</em>
.</p>
<section class="tw xbox font-sm" id="jeu70033-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Mitochondrial genome features of the NWA isolates.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom">
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="2" valign="bottom" colspan="1">Feature</th>
<th align="center" colspan="3" valign="bottom" rowspan="1">Species</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>Minidiscus spinulatus</em>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>Minidiscus trioculatus</em>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>Mediolabrus comicus</em>
</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Strain name</td>
<td align="center" valign="top" rowspan="1" colspan="1">AGSB‐0477</td>
<td align="center" valign="top" rowspan="1" colspan="1">AGSB‐0018</td>
<td align="center" valign="top" rowspan="1" colspan="1">AGSB‐0017</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GenBank ID</td>
<td align="center" valign="top" rowspan="1" colspan="1">PV641038</td>
<td align="center" valign="top" rowspan="1" colspan="1">PV641040</td>
<td align="center" valign="top" rowspan="1" colspan="1">PV641039</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Total size (bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">49,654</td>
<td align="center" valign="top" rowspan="1" colspan="1">79,303</td>
<td align="center" valign="top" rowspan="1" colspan="1">53,164</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">% GC</td>
<td align="center" valign="top" rowspan="1" colspan="1">29.1%</td>
<td align="center" valign="top" rowspan="1" colspan="1">31.0%</td>
<td align="center" valign="top" rowspan="1" colspan="1">30.5%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">% CDS</td>
<td align="center" valign="top" rowspan="1" colspan="1">53.7%</td>
<td align="center" valign="top" rowspan="1" colspan="1">35.8%</td>
<td align="center" valign="top" rowspan="1" colspan="1">57.9%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Total number of genes<a href="#jeu70033-note-0002" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">66</td>
<td align="center" valign="top" rowspan="1" colspan="1">67</td>
<td align="center" valign="top" rowspan="1" colspan="1">68</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Protein‐coding genes<a href="#jeu70033-note-0002" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">35</td>
<td align="center" valign="top" rowspan="1" colspan="1">34</td>
<td align="center" valign="top" rowspan="1" colspan="1">35</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">ORFs</td>
<td align="center" valign="top" rowspan="1" colspan="1">0</td>
<td align="center" valign="top" rowspan="1" colspan="1">4</td>
<td align="center" valign="top" rowspan="1" colspan="1">3</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">tRNA genes<a href="#jeu70033-note-0003" class="usa-link">
<sup>
b
</sup>
</a>, <a href="#jeu70033-note-0004" class="usa-link">
<sup>
c
</sup>
</a>, <a href="#jeu70033-note-0005" class="usa-link">
<sup>
d
</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">29</td>
<td align="center" valign="top" rowspan="1" colspan="1">28</td>
<td align="center" valign="top" rowspan="1" colspan="1">27</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">rRNA genes<a href="#jeu70033-note-0002" class="usa-link">
<sup>
a
</sup>
</a>, <a href="#jeu70033-note-0004" class="usa-link">
<sup>
c
</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">2</td>
<td align="center" valign="top" rowspan="1" colspan="1">2</td>
<td align="center" valign="top" rowspan="1" colspan="1">2</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Type II introns<a href="#jeu70033-note-0002" class="usa-link">
<sup>
a
</sup>
</a>, <a href="#jeu70033-note-0004" class="usa-link">
<sup>
c
</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">2 in <em>rnl</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">7 in <em>cox1</em>, 2 in <em>rnl</em>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">2 in <em>cox1</em>, 2 in <em>rnl</em>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Tandem repeats<a href="#jeu70033-note-0005" class="usa-link">
<sup>d</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">3</td>
<td align="center" valign="top" rowspan="1" colspan="1">19</td>
<td align="center" valign="top" rowspan="1" colspan="1">4</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/jeu70033-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="jeu70033-note-0002">
<sup>
<sup>a</sup>
</sup><p class="display-inline">MFannot (Lang et al. <a href="#jeu70033-bib-0059" class="usa-link" aria-describedby="jeu70033-bib-0059">2023</a>).</p>
</div>
<div class="fn" id="jeu70033-note-0003">
<sup>
<sup>b</sup>
</sup><p class="display-inline">tRNAscan‐SE v2.0.7 (Chan and Lowe <a href="#jeu70033-bib-0014" class="usa-link" aria-describedby="jeu70033-bib-0014">2019</a>).</p>
</div>
<div class="fn" id="jeu70033-note-0004">
<sup>
<sup>c</sup>
</sup><p class="display-inline">RNAweasel (Lang et al. <a href="#jeu70033-bib-0059" class="usa-link" aria-describedby="jeu70033-bib-0059">2023</a>).</p>
</div>
<div class="fn" id="jeu70033-note-0005">
<sup>
<sup>d</sup>
</sup><p class="display-inline">Tandem Repeats Finder (Benson <a href="#jeu70033-bib-0007" class="usa-link" aria-describedby="jeu70033-bib-0007">1999</a>).</p>
</div>
</div></section><p>Both 
<em>M. spinulatus</em>
 and <em>M. trioculatus</em> have long non‐coding regions in their mt genomes (Figure <a href="#jeu70033-fig-0002" class="usa-link">2</a>). Specifically, located in positions 66,692–77,002 in <em>M. trioculatus</em> and 40,021–47,556 in 
<em>M. spinulatus</em>
 are the non‐coding regions of the genomes, which are primarily composed of repeats. In comparison, the <em>M. comicus</em> mt genome is instead compact with minimal non‐coding regions. The <em>cox1</em> gene in <em>M. trioculatus</em> was ~21 kb in length from positions 23,248–44,209 due to seven introns that range in length from 2528 to 3263 bp. In comparison, <em>M. comicus</em> had two <em>cox1</em> introns, which were 2459 and 2693 bp, while there were no <em>cox1</em> introns found in <em>M. spinulatus</em>. Two introns were also annotated in each of the <em>rnl</em> genes in each species by aligning the mt to related species of <em>Thalassiosira</em> with and without introns (Table <a href="#jeu70033-tbl-0002" class="usa-link">2</a>; Figure <a href="#jeu70033-fig-0002" class="usa-link">2</a>). The <em>rnl</em> gene in 
<em>M. spinulatus</em>
 is 4613 bp (two introns of 731 and 852 bp), while <em>M. comicus</em> and <em>M. trioculatus</em> have genes that are 7932 bp (two introns of 2506 and 2411 bp) and 7733 bp (two introns of 2451 and 2333 bp), respectively. Comparison of the three isolates via local alignments (using BLAST) revealed that the two <em>Minidiscus</em> species had higher pairwise similarity than the <em>Mediolabrus</em> with either of the <em>Minidiscus</em> species (Figure <a href="#jeu70033-fig-0002" class="usa-link">2</a>). <em>Mediolabrus comicus</em> also had a closer overall similarity to 
<em>M. spinulatus</em>
 (Figure <a href="#jeu70033-fig-0002" class="usa-link">2</a>). The exons of the <em>rnl</em> gene were the most conserved regions across all the mt genomes, whereas introns and other non‐coding sequences were generally the least conserved (Figure <a href="#jeu70033-fig-0002" class="usa-link">2</a>).</p></section><section id="jeu70033-sec-0014"><h3 class="pmc_sec_title">3.3. 
NWA Isolates Have Unique Features in Their Chloroplast Genomic Repertoire Compared to Other Sequenced Thalassiosirales</h3>
<p>Chloroplast genome size was 126,734, 135,507, and 138,031 bp for <em>M. comicus</em>, <em>M. trioculatus</em>, and 
<em>M. spinulatus</em>
, respectively (Figure <a href="#jeu70033-fig-0003" class="usa-link">3</a>, Table <a href="#jeu70033-tbl-0003" class="usa-link">3</a>). Sequenced chloroplast genomes further exhibited similar GC content, with 
<em>M. spinulatus</em>
 and <em>M. comicus</em> both at 30.8% GC, while <em>M. trioculatus</em> was slightly higher at 31.1% GC. The large single‐copy (LSC) region varied in size among the species, with 
<em>M. spinulatus</em>
 having the largest at 70,509 bp, followed by <em>M. trioculatus</em> at 66,399 bp and <em>M. comicus</em> at 65,377 bp (Table <a href="#jeu70033-tbl-0003" class="usa-link">3</a>). The small single‐copy (SSC) region showed notable differences, with <em>M. trioculatus</em> possessing the largest at 45,515 bp, compared to 31,152 bp in 
<em>M. spinulatus</em>
 and 26,726 bp in <em>M. comicus</em>. The inverted repeat (IR) regions also differed, with 
<em>M. spinulatus</em>
 and <em>M. comicus</em> having similar lengths of 18,185 and 17,315 bp, respectively, whereas <em>M. trioculatus</em> had a substantially smaller IR region at 11,795 bp. The percentage of each genome attributed to coding sequences varied, with <em>M. trioculatus</em> at 68.7%, 
<em>M. spinulatus</em>
 at 71.9%, and <em>M. comicus</em> at 76.4%. All three species contained the overlapping gene pairs <em>atpF‐atpD</em>, <em>psbC‐psbD</em>, <em>rpl23‐rpl4</em>, and <em>sufC‐sufB</em>.</p>
<figure class="fig xbox font-sm" id="jeu70033-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2928/12326293/82bb26771e98/JEU-72-e70033-g005.jpg" loading="lazy" id="jats-graphic-5" height="678" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/jeu70033-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>BLAST comparison of the nanoplanktonic diatom cp genomes using Proksee (Grant et al. <a href="#jeu70033-bib-0033" class="usa-link" aria-describedby="jeu70033-bib-0033">2023</a>). Chloroplast genomes were BLASTed against the other two isolates to highlight conserved genes. Innermost ring = GC‐content. First two inner rings = annotations of key cp genome components on forward and reverse strands. Outer two rings = percent identity (%) of the genome of interest in comparison with the other two isolates.</p></figcaption></figure><section class="tw xbox font-sm" id="jeu70033-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Chloroplast genome features of the NWA isolates.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom">
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="2" valign="bottom" colspan="1">Feature</th>
<th align="center" colspan="3" valign="bottom" rowspan="1">Species</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>Minidiscus spinulatus</em>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>Minidiscus trioculatus</em>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>Mediolabrus comicus</em>
</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Genbank ID</td>
<td align="center" valign="top" rowspan="1" colspan="1">PV631330</td>
<td align="center" valign="top" rowspan="1" colspan="1">PV631332</td>
<td align="center" valign="top" rowspan="1" colspan="1">PV631331</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Total size (bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">138,031</td>
<td align="center" valign="top" rowspan="1" colspan="1">135,507</td>
<td align="center" valign="top" rowspan="1" colspan="1">126,734</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">% GC</td>
<td align="center" valign="top" rowspan="1" colspan="1">30.8%</td>
<td align="center" valign="top" rowspan="1" colspan="1">31.1%</td>
<td align="center" valign="top" rowspan="1" colspan="1">30.8%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">% CDS</td>
<td align="center" valign="top" rowspan="1" colspan="1">71.9%</td>
<td align="center" valign="top" rowspan="1" colspan="1">68.7%</td>
<td align="center" valign="top" rowspan="1" colspan="1">76.4%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Large single‐copy (LSC) region (bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">70,509</td>
<td align="center" valign="top" rowspan="1" colspan="1">66,399</td>
<td align="center" valign="top" rowspan="1" colspan="1">65,377</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Small single‐copy (SSC) region (bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">31,152</td>
<td align="center" valign="top" rowspan="1" colspan="1">45,515</td>
<td align="center" valign="top" rowspan="1" colspan="1">26,726</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Inverted repeat (IR) region (bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">18,185</td>
<td align="center" valign="top" rowspan="1" colspan="1">11,795</td>
<td align="center" valign="top" rowspan="1" colspan="1">17,315</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Total number of genes<a href="#jeu70033-note-0006" class="usa-link">
<sup>
a
</sup>
</a>, <a href="#jeu70033-note-0007" class="usa-link">
<sup>
b
</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">160</td>
<td align="center" valign="top" rowspan="1" colspan="1">158</td>
<td align="center" valign="top" rowspan="1" colspan="1">161</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Protein‐coding genes (PCGs)<a href="#jeu70033-note-0007" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">127</td>
<td align="center" valign="top" rowspan="1" colspan="1">127</td>
<td align="center" valign="top" rowspan="1" colspan="1">128</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Open reading frames without known annotation (ORFs)<a href="#jeu70033-note-0007" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">2</td>
<td align="center" valign="top" rowspan="1" colspan="1">0</td>
<td align="center" valign="top" rowspan="1" colspan="1">3</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">tRNA genes<a href="#jeu70033-note-0008" class="usa-link">
<sup>
c
</sup>
</a>, <a href="#jeu70033-note-0009" class="usa-link">
<sup>
d
</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">28</td>
<td align="center" valign="top" rowspan="1" colspan="1">28</td>
<td align="center" valign="top" rowspan="1" colspan="1">28</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">rRNA genes<a href="#jeu70033-note-0007" class="usa-link">
<sup>
b
</sup>
</a>, <a href="#jeu70033-note-0009" class="usa-link">
<sup>
d
</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">3</td>
<td align="center" valign="top" rowspan="1" colspan="1">3</td>
<td align="center" valign="top" rowspan="1" colspan="1">2</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Overlapping genes<a href="#jeu70033-note-0007" class="usa-link">
<sup>
b
</sup>
</a>, <a href="#jeu70033-note-0010" class="usa-link">
<sup>
e
</sup>
</a>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>atpF_atpD</em> (4 bp), <em>psbC_psbD</em> (53 bp), <em>rpl23_rpl4</em> (8 bp), <em>sufB_sufC</em> (1 bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>atpF_atpD</em> (4 bp), <em>psbC_psbD</em> (53 bp), <em>rpl23_rpl4</em> (8 bp), <em>sufB_sufC</em> (1 bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<em>atpF_atpD</em> (4 bp), <em>psbC_psbD</em> (53 bp), <em>rpl23_rpl4</em> (8 bp), <em>sufB_sufC</em> (1 bp)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/jeu70033-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="jeu70033-note-0006">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Genes duplicated in the IR region are only counted once.</p>
</div>
<div class="fn" id="jeu70033-note-0007">
<sup>
<sup>b</sup>
</sup><p class="display-inline">MFannot (Lang et al. <a href="#jeu70033-bib-0059" class="usa-link" aria-describedby="jeu70033-bib-0059">2023</a>).</p>
</div>
<div class="fn" id="jeu70033-note-0008">
<sup>
<sup>c</sup>
</sup><p class="display-inline">tRNAscan‐SE v2.0.7 (Chan and Lowe <a href="#jeu70033-bib-0014" class="usa-link" aria-describedby="jeu70033-bib-0014">2019</a>).</p>
</div>
<div class="fn" id="jeu70033-note-0009">
<sup>
<sup>d</sup>
</sup><p class="display-inline">RNAweasel (Lang et al. <a href="#jeu70033-bib-0059" class="usa-link" aria-describedby="jeu70033-bib-0059">2023</a>).</p>
</div>
<div class="fn" id="jeu70033-note-0010">
<sup>
<sup>e</sup>
</sup><p class="display-inline">Geneious Prime 2024.0.7.</p>
</div>
</div></section><p>BLAST comparison of the three isolates revealed the same pattern as the mt genomes, where the <em>Minidiscus</em> genera were more closely related and <em>M. comicus</em> exhibited a higher BLAST similarity to 
<em>M. spinulatus</em>
 than <em>M. trioculatus</em>. Unsurprisingly, the most highly conserved regions within the cp genomes were the <em>rRNA</em> genes (Figure <a href="#jeu70033-fig-0003" class="usa-link">3</a>). Liu, Chen, et al. (<a href="#jeu70033-bib-0070" class="usa-link" aria-describedby="jeu70033-bib-0070">2021</a>) analyzed gene gain and loss across 61 diatom cp genomes. Using their dataset as a reference, we found our three isolates showed identical presence/absence patterns across all 44 protein‐coding genes (Figure <a href="#jeu70033-supitem-0001" class="usa-link">S13</a>). This pattern aligns closely with previously sequenced cp genomes within the order Thalassiosirales (Liu, Chen, et al. <a href="#jeu70033-bib-0070" class="usa-link" aria-describedby="jeu70033-bib-0070">2021</a>). While all three retain <em>petF</em>, they notably lack <em>acpP</em>, <em>syfB</em>, and <em>ycf91</em> (Liu, Chen, et al. <a href="#jeu70033-bib-0070" class="usa-link" aria-describedby="jeu70033-bib-0070">2021</a>; Lommer et al. <a href="#jeu70033-bib-0075" class="usa-link" aria-describedby="jeu70033-bib-0075">2010</a>).</p></section><section id="jeu70033-sec-0015"><h3 class="pmc_sec_title">3.4. Pan‐Organellar Genome Analyses of Thalassiosirales Diatoms</h3>
<p>Further pangenomic analyses of Thalassiosirales mt genomes identified 112 gene clusters, of which 24 (21%) were present in all 16 genomes, forming the core mt genome, while the remaining 88 (79%) belonged to the accessory genome (Figure <a href="#jeu70033-fig-0004" class="usa-link">4A</a>; Data <a href="#jeu70033-supitem-0001" class="usa-link">S3</a>). In the accessory genome, 62 were unique to only one organism (70% of the accessory genome), and of these, 28 (45%) were singleton gene clusters belonging to <em>M. trioculatus</em>. The 79,302 bp genome of <em>M. trioculatus</em> is 1.79 times larger than the average of the 16 currently sequenced Thalassiosirales genomes at 44,285 [+/− 9709]. <em>Minidiscus trioculatus</em> also had the highest number of gene clusters (62, average = 40.6) and the highest GC‐content (31.0% GC, average = 29.7% GC).</p>
<figure class="fig xbox font-sm" id="jeu70033-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2928/12326293/f95c68cb7b06/JEU-72-e70033-g001.jpg" loading="lazy" id="jats-graphic-7" height="1176" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/jeu70033-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Pangenome analysis of all available mitochondrial (A) and chloroplast (B) genomes in the Thalassiosiraceae family. All sequences are unique, even though some species have multiple genomes available. This Anvi'o schematic shows the presence or absence of gene clusters [left] and average nucleotide identities [right] (Hernández‐Salmerón and Moreno‐Hagelsieb <a href="#jeu70033-bib-0047" class="usa-link" aria-describedby="jeu70033-bib-0047">2022</a>). Organellar genomes were ordered based on ANI similarity.</p></figcaption></figure><p>In the case of the cp pangenome for the 37 Thalassiosirales genomes, 189 gene clusters were found, of which 112 (59%) were a part of the core genome and 77 (41%) were identified as the accessory genome; that latter includes 47/77 (61%) that were singleton gene clusters (Figure <a href="#jeu70033-fig-0004" class="usa-link">4B</a>; Data <a href="#jeu70033-supitem-0001" class="usa-link">S3</a>). <em>Minidiscus trioculatus</em> had six singleton gene clusters, <em>M. comicus</em> had zero, and the NWA 
<em>M. spinulatus</em>
 isolate had one. The <a href="https://www.ncbi.nlm.nih.gov/nuccore/OL855995" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OL855995</a> 
<em>M. spinulatus</em>
 had two singleton gene clusters, showing regional‐specific diversity between the two isolates (Wang and Chen <a href="#jeu70033-bib-0114" class="usa-link" aria-describedby="jeu70033-bib-0114">2022</a>). Measuring the cp genomes' average nucleotide identities (ANI), <em>Minidiscus spinulatus</em> has 99.7% ANI to <a href="https://www.ncbi.nlm.nih.gov/nuccore/OL855995" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OL855995</a> 
<em>M. spinulatus</em>
 and 91.5% ANI to <em>M. trioculatus</em>. Interestingly, <em>M. trioculatus</em> had a slightly higher ANI when compared to the NWA 
<em>M. spinulatus</em>
 (91.6%) than to the <a href="https://www.ncbi.nlm.nih.gov/nuccore/OL855995" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OL855995</a> 
<em>M. spinulatus</em>
 (91.2%).</p>
<p>In both organellar pangenomes (Figure <a href="#jeu70033-fig-0004" class="usa-link">4</a>), as expected, isolates of the same species had highly similar ANIs (Data <a href="#jeu70033-supitem-0001" class="usa-link">S4</a>). The ANIs of the cluster including <em>Skeletonema</em> and <em>Mediolabrus</em> had overall higher similarity than the cluster of <em>Minidiscus</em> and <em>Thalassiosira</em>. Overall, the cp genomes had closer similarity than the mt genomes, ranging from 83.1% to ~100% ANI with isolates of the same species in comparison to 80.9% to ~100% with isolates of the same species (Data <a href="#jeu70033-supitem-0001" class="usa-link">S4</a>). In both cases, Cluster 1, comprising <em>Skeletonema</em> species and <em>Mediolabrus comicus</em>, exhibited a closer genetic relationship within the cluster compared to Cluster 2, which included the genera <em>Thalassiosira</em> and <em>Minidiscus</em> (Figure <a href="#jeu70033-fig-0004" class="usa-link">4</a>). This indicates that the genomes in Cluster 1 are more highly conserved than those in Cluster 2 in terms of ANI.</p></section><section id="jeu70033-sec-0016"><h3 class="pmc_sec_title">3.5. Phylogenomic Analyses of Thalassiosirales Organellar Genome DNA Re‐Emphasizes Prior Classification</h3>
<p>Concatenated alignments and trees for a suite of protein‐coding (PC) genes were generated for currently available data to further explore the phylogenomic relationships of the three new isolates within the Thalassiosirales order (one analysis using mt PC genes, the other cp PC genes). Both trees further emphasized the characterization of these three isolates as <em>Minidiscus</em> and <em>Mediolabrus</em> species, respectively (Figure <a href="#jeu70033-fig-0005" class="usa-link">5</a>). The <em>Minidiscus</em> species clustered closely together (at 99%–100% bootstrap support), and the previously found clade separation was demonstrated as the <em>M. comicus</em> isolate formed a sister clade with the <em>Skeletonema</em> genus (at 100% bootstrap support), and the <em>Minidiscus</em> species were either within the same clade or a sister clade to members of the <em>Thalassiosira</em> genus. Thus, the concatenated alignments from the mt and cp genomes (i.e., the ‘super‐barcodes’) re‐emphasize the clade separation of <em>Mediolabrus</em> and <em>Minidiscus</em> (Li et al. <a href="#jeu70033-bib-0068" class="usa-link" aria-describedby="jeu70033-bib-0068">2020</a>).</p></section><section id="jeu70033-sec-0017"><h3 class="pmc_sec_title">3.6. Thalassiosirales Isolates Exhibit Differing Biogeography and Succession in the NWA
</h3>
<p>Seagoing expeditions in the NWA (AZMP, AZOMP, NWA‐BCP CE, and GEOTRACES) characterized the distribution of the three isolates using <em>16S</em> rRNA metabarcoding. The presence and absence of the ASVs pertaining to the isolates in the surface photosynthetic community (1–20 m) were determined by subsetting the dataset to include only samples collected within that depth range (Figure <a href="#jeu70033-fig-0006" class="usa-link">6</a>). When considering all samples across all depths and sampling methods, the ASVs were detected at more sites (Figure <a href="#jeu70033-supitem-0001" class="usa-link">S4</a>). Additionally, the proportions of the surface ASVs (1–20 m) to the rest of the photosynthetic community were determined and plotted (Figure <a href="#jeu70033-supitem-0001" class="usa-link">S5</a>). Of all these off‐shore samples, 
<em>M. spinulatus</em>
 was only detected at one location off the SS in the Fall of 2017 (Figure <a href="#jeu70033-fig-0006" class="usa-link">6</a>). In the sample at this location, it accounted for approximately 0.9% of the total photosynthetic community. In contrast, both <em>M. trioculatus</em> and <em>M. comicus</em> were commonly found at various locations and seasons. It is important to note that <em>M. trioculatus</em> was not detected in samples collected at any depth during the BELAS‐1 expedition (LS) and in any surface samples (1–20 m) in the AZOMP22 (LS) mission; hence, in 2022 this species had limited presence in the Central Labrador Basin and Greenland shelf (Figure <a href="#jeu70033-fig-0006" class="usa-link">6</a>; Figure <a href="#jeu70033-supitem-0001" class="usa-link">S4</a>). <em>Mediolabrus comicus</em> was detected at the surface in all expeditions except during AZOMP22 in 2022 (Figure <a href="#jeu70033-supitem-0001" class="usa-link">S3</a>; Figure <a href="#jeu70033-fig-0006" class="usa-link">6</a>). In the 2015 cruise to the CAG, <em>M. trioculatus</em> was also only found at the Southern site in the Central LS, while <em>M. comicus</em> was more frequently detected at northern sites, specifically within Baffin Bay and the Canadian Arctic Archipelago (Figure <a href="#jeu70033-fig-0006" class="usa-link">6</a>). This demonstrates that <em>M. comicus</em> potentially has a more northern geographic range than <em>M. trioculatus</em>. When comparing the relative proportions of phytoplankton communities on the SS in the fall versus spring, <em>M. trioculatus</em> was generally more abundant in the spring, while <em>M. comicus</em> appeared in greater proportions in the fall (Figure <a href="#jeu70033-supitem-0001" class="usa-link">S5</a>).</p>
<figure class="fig xbox font-sm" id="jeu70033-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2928/12326293/5b22118a35aa/JEU-72-e70033-g007.jpg" loading="lazy" id="jats-graphic-11" height="754" width="709" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/jeu70033-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Nanoplanktonic diatom isolates tracked across the NWA and Canadian Arctic using <em>16S</em> metabarcoding of eDNA from the upper photic zone during various expeditions (1–20 m). Abbreviations: AF, Atlantic Zone Monitoring Program (AZMP) Fall (usually Sept–Oct, but Nov–Dec in 2017); AS, AZMP Spring (Apr–Jun); AZOMP, Atlantic Zone Off‐shore Monitoring Program (Apr–Jun); BELAS‐1, Biological Carbon Export in the Labrador Sea (May–Jun); Arctic15, GEOTRACES mission to CAG (Jul–Aug). Numbers at the end of mission codes indicate the year (see Table <a href="#jeu70033-tbl-0001" class="usa-link">1</a> for more information). For a more complete figure including limited sampling in AS18 and AS19, see Figure <a href="#jeu70033-supitem-0001" class="usa-link">S3</a>.</p></figcaption></figure><p>While the expeditions above provide insight on the geographic range of the taxa in the NWA, in the BB (Halifax, N.S.), weekly <em>16S</em> rRNA data provide greater resolution of when the taxa were present in‐shore between 2014 and 2022. When observed, these nanoplankton occupied a higher proportion of the photosynthetic community at 1 m than at 60 m (Figure <a href="#jeu70033-fig-0007" class="usa-link">7A</a>). However, all three isolates were found throughout the water column at the four depths sampled in the BB (Figure <a href="#jeu70033-supitem-0001" class="usa-link">S6</a>). For size‐fractionated years (&gt; 3 μM versus 0.2–3 μM), the nanoplankton tended to occupy a larger proportion of the community in the &gt; 3 μM fraction (Figure <a href="#jeu70033-supitem-0001" class="usa-link">S7</a>).</p>
<figure class="fig xbox font-sm" id="jeu70033-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2928/12326293/f01611237599/JEU-72-e70033-g003.jpg" loading="lazy" id="jats-graphic-13" height="635" width="709" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/jeu70033-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Dynamics of the isolated nanoplanktonic diatoms relative to environmental conditions in the Bedford Basin (Halifax, N.S.) observed through a weekly time series over 8 years (2014–2022). Relative abundances are calculated as the percentage of cp <em>16S</em> rRNA V4V5 reads either on &gt; 0.2‐μm filters or reconstituted &gt; 3 and 0.2–3 μm filters. Missing data is due to COVID‐19 restrictions on sampling in 2020. Panels show (A) weekly relative abundances (Rel. Abun.) for each taxa out of the total phytoplankton community in the 1 and 60 m depths, (B) environmental variables including surface [1 m] temperature (temp), chlorophyll <em>a</em> (chl <em>a</em>), oxygen (O<sub>2</sub>), and nitrate (NO<sub>3</sub>), (C) flow cytometry‐derived cell counts (&lt; 35 μm) for nanoeukaryotes (nano), picoeukaryotes (pico), and <em>Synechococcus</em> (<em>Syn</em>), (D) box plots of the relative abundances of each isolate by each month irrespective of the year and depth (including 1, 5, 10, and 60 m), (E) box plots of relative abundances observed for each isolate according to temperature and irrespective of time of year and depth [species absences were excluded from the plot]. (F) redundancy analysis (RDA) showing relationships between each of the nanoplankton and the environmental variables, where RDA1 accounts for the largest fraction of variance in species abundance explained by the environmental variables (21.0%). The green star on the 1 m plot in panel A indicates when the BB isolate of <em>Minidiscus spinulatus</em> was originally isolated. Additional environmental parameters were plotted and can be seen in Figure <a href="#jeu70033-supitem-0001" class="usa-link">S8</a>.</p></figcaption></figure><p>Using the BBMP data, we found that the three nanoplanktonic diatoms each had a unique temporal pattern, with individual species tending to dominate different times of the year (Figure <a href="#jeu70033-fig-0007" class="usa-link">7A</a>). Although 
<em>M. spinulatus</em>
 was only detected in one off‐shore sample (Figure <a href="#jeu70033-fig-0006" class="usa-link">6</a>), it was observed consistently in the BB, occupying a large relative proportion of the phytoplankton community in the second half of the year (generally Jun–Dec). In comparison, <em>M. trioculatus</em> was consistently present mainly in the first half of the year (Jan–Jun) and was then followed by <em>M. comicus</em> and 
<em>M. spinulatus</em>
, with the former of the two taxa appearing slightly earlier than the other (Figure <a href="#jeu70033-fig-0007" class="usa-link">7D</a>). <em>Minidiscus spinulatus</em> and <em>M. trioculatus</em> rarely dominated the photosynthetic community simultaneously, indicating that the two species may be adapted to conditions and/or dynamics of opposite seasons at the in‐shore BBMP site. <em>Mediolabrus comicus</em> appears more frequently in the second half of the year but also coexists with <em>M. trioculatus</em> in winter and spring, preceding the dominance of 
<em>M. spinulatus</em>
 (Figure <a href="#jeu70033-fig-0007" class="usa-link">7D</a>). A dot plot of the relative abundance of each isolate by month is visualized in Figure <a href="#jeu70033-supitem-0001" class="usa-link">S9</a> to demonstrate sample‐level variability. The seasonal patterns observed in the BBMP environmental data suggest that temperature (via seasonal change) plays a key role in driving succession across the three studied taxa; <em>M. trioculatus</em> dominates at lower temperatures (between −1°C and 5°C), and 
<em>M. spinulatus</em>
 dominates at higher temperatures, overlapping with <em>M. comicus</em> (at &gt; 8°C; Figure <a href="#jeu70033-fig-0007" class="usa-link">7E</a>). To further investigate this, the temperature ranges were plotted for each isolate. The individual species are shown with distinct distributions, and the pairwise Wilcoxon rank‐sum tests (with Benjamini–Hochberg correction) indicate that each species' temperature range is significantly different, as evidenced by the low <em>p</em>‐values (<em>p</em> &lt; 8.1e‐09) (Figure <a href="#jeu70033-supitem-0001" class="usa-link">S10</a>).</p>
<p>To further explore the relationship between environmental variables and the nanoplanktonic diatoms, we performed an RDA using metadata from the BBMP (Figure <a href="#jeu70033-fig-0007" class="usa-link">7F</a>). The RDA model was statistically significant (ANOVA, 1000 permutations, <em>p</em> &lt; 0.001), with all selected environmental variables also showing significance (<em>p</em> &lt; 0.01). The first axis (RDA1) explained 21.0% of the total variance and captured 90.2% of the constrained variance within the model, while RDA2 explained 1.6% of the total variance and 6.8% of the constrained variance. Temperature (9.365 variance, <em>F</em> = 312.50, <em>p</em> &lt; 0.001) was the most influential variable, explaining the largest proportion of variance in the dataset, followed by ammonia (1.565 variance, <em>F</em> = 52.23, <em>p</em> &lt; 0.001) and nitrate (1.076 variance, <em>F</em> = 35.92, <em>p</em> &lt; 0.001). The impact of temperature on the RDA model re‐emphasizes our succession and temperature results (Figure <a href="#jeu70033-fig-0007" class="usa-link">7A–E</a>). <em>Minidiscus spinulatus</em> and <em>M. comicus</em> had higher RDA1 values; separation on this axis also correlates with warmer temperatures toward positive RDA1, whereas <em>M. trioculatus</em>, with its negative RDA1 value, exhibited the opposite trend, suggesting a preference for cooler conditions. In contrast, the RDA2 axis was most influenced by stronger correlations with chlorophyll a, followed by nitrate, oxygen, and <em>Synechococcus</em>, suggesting nutrient availability and bloom conditions as potential controls for RDA2. <em>Mediolabrus comicus</em> was more positively associated with RDA2 (1.94) in contrast to 
<em>M. spinulatus</em>
 (0.06), suggesting a potential preference for <em>M. comicus</em> for higher bioavailable nitrogen and less biomass as indicated by chlorophyll a (Figure <a href="#jeu70033-fig-0007" class="usa-link">7F</a>).</p></section></section><section id="jeu70033-sec-0018"><h2 class="pmc_sec_title">4. Discussion</h2>
<section id="jeu70033-sec-0019"><h3 class="pmc_sec_title">4.1. Contribution of New Organellar Genomes for Three Nanoplanktonic Diatoms</h3>
<p>Our work contributes new genomic information to the current repertoire of diatom organellar genomes. There are currently 266 species in the <em>Thalassiosiraceae</em> family, and only 19 cp genomes (16 species) and 8 mt genomes are available for this diverse diatom family (Guiry and Guiry <a href="#jeu70033-bib-0040" class="usa-link" aria-describedby="jeu70033-bib-0040">2015</a>). Here we present the complete cp and mt genomes for 
<em>M. spinulatus</em>
, <em>M. trioculatus</em>, and <em>M. comicus</em> isolates from the NWA. Currently, a complete cp genome (<a href="https://www.ncbi.nlm.nih.gov/nuccore/OL855995" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OL855995</a>) is available for an 
<em>M. spinulatus</em>
 strain from Jiaozhou Bay (China), while only a partial cp genome is available for <em>M. trioculatus</em> (strain CCMP495) (Nordberg et al. <a href="#jeu70033-bib-0085" class="usa-link" aria-describedby="jeu70033-bib-0085">2014</a>; Wang and Chen <a href="#jeu70033-bib-0114" class="usa-link" aria-describedby="jeu70033-bib-0114">2022</a>). Despite being from two distinct oceanographic regions, the <em>Minidiscus spinulatus</em> strains from the NWA and Jiaozhou Bay have highly similar cp genomes (ANI = 99.7%) (Figure <a href="#jeu70033-fig-0004" class="usa-link">4</a>, Figure <a href="#jeu70033-supitem-0001" class="usa-link">S12</a>). Recently established based on phylogenetic and morphological data (Li et al. <a href="#jeu70033-bib-0068" class="usa-link" aria-describedby="jeu70033-bib-0068">2020</a>), the genus <em>Mediolabrus</em> remains under investigation, and our work advances understanding of its genetic and ecological distinctions from <em>Minidiscus</em>.</p></section><section id="jeu70033-sec-0020"><h3 class="pmc_sec_title">4.2. Phylogenomic Classification of Three Isolates</h3>
<p>Recent studies have emphasized the power of organelle genomes in resolving fine‐scale phylogenetic relationships and uncovering unexpected evolutionary patterns within diatoms. For example, Jeong and Lee (<a href="#jeu70033-bib-0051" class="usa-link" aria-describedby="jeu70033-bib-0051">2024</a>) demonstrated that cp and mt phylogenies often conflict with traditional morphological classifications. In our case, using cp and mt genomes as super‐barcodes reaffirmed the two distinct clades of <em>Minidiscus</em> and <em>Mediolabrus</em> previously identified by Li et al. (<a href="#jeu70033-bib-0068" class="usa-link" aria-describedby="jeu70033-bib-0068">2020</a>) with more robust support. Multiple methods, including the partial <em>18S</em> rRNA phylogeny (1051 bp), separate concatenated gene trees of protein‐coding regions from both the mt genomes and cp genomes, and pangenome analyses, consistently showed that <em>Mediolabrus comicus</em> clustered with <em>Skeletonema</em> species. In contrast, in the <em>18S</em> rRNA tree, <em>Minidiscus</em> clustered closer to a group containing <em>Cyclostephanos</em>, <em>Stephanodiscus</em>, <em>Pantocsekiella</em>, <em>Discostella</em>, and <em>Thalassiosira</em> species (Li et al. <a href="#jeu70033-bib-0068" class="usa-link" aria-describedby="jeu70033-bib-0068">2020</a>). The same species were not available for the concatenated tree of the Thalassiosirales organellar genomes; however, <em>Minidiscus</em> still clustered closer to related <em>Thalassiosira</em> than <em>Skeletonema</em>. With additional sequencing of Thalassiosirales species' organellar genomes, the phylogenetic relationship of <em>Minidiscus</em> to other genera will be further revealed.</p>
<p>Cell size is a crucial factor that shapes various aspects of phytoplankton physiology, ecology, and evolution, influencing traits such as nutrient acquisition, photophysiology, and grazing pressures (Finkel et al. <a href="#jeu70033-bib-0030" class="usa-link" aria-describedby="jeu70033-bib-0030">2010</a>). For further context, we compiled a supplemental dataset of cell sizes for each species included in the three phylogenetic trees from various sources (Tables <a href="#jeu70033-supitem-0001" class="usa-link">S1–S3</a>). Within the <em>18S</em> rRNA tree, cell sizes varied across different taxonomic groupings, highlighting that smaller cell size is presumably not a monophyletic trait within the Thalassiosirales. For example, the sister clade to <em>Minidiscus</em> exhibited a range of 2.5–54 μm, with an average of 15.7 μm, while the sister clade to <em>Mediolabrus</em>, <em>Skeletonema menzelli</em>, showed a range of 2–10 μm, with the closely related <em>Skeletonema</em> species averaging 7.6 μm (range 2–21 μm). Additionally, more distantly related clades outside the nanoplanktonic diatoms generally had larger cell sizes, with average cell size (within the clades) ranging from 13.1 to 55.8 μm. However, there were also several species with small cell sizes that did not cluster closely with the nanoplankton (e.g., 
<em>Thalassiosira profunda</em>
 with a size range of 2–5 μm).</p></section><section id="jeu70033-sec-0021"><h3 class="pmc_sec_title">4.3. Larger Mitogenome in <em>M. trioculatus</em> is due to Nucleotide Repeats and <em>cox1</em> Introns</h3>
<p>Organellar genome comparisons for members of the Order Thalassiosirales revealed that <em>M. trioculatus</em> has a much larger mt genome than its close relatives. Within Thalassiosiraceae, the currently observed mt genome lengths range from 39,089 bp in 
<em>Thalassiosira pseudonana</em>
 to 51,728 bp in 
<em>Thalassiosira rotula</em>
. Interestingly, <em>M. trioculatus</em> has the largest currently reported mt genome in the Thalassiosiraceae family and is more similar to two pennate diatoms, 
<em>Phaeodactylum tricornutum</em>
 and <em>Halamphora calidilacuna</em>, which are 77,356 and 103,605 bp, respectively (Oudot‐Le Secq and Green <a href="#jeu70033-bib-0087" class="usa-link" aria-describedby="jeu70033-bib-0087">2011</a>; Pogoda et al. <a href="#jeu70033-bib-0094" class="usa-link" aria-describedby="jeu70033-bib-0094">2019</a>). The extended length of the <em>M. trioculatus</em> mt genome is due to two main features: the long non‐coding region and seven introns located within the <em>cox1</em> gene.</p>
<p>Both <em>M. trioculatus</em> and 
<em>M. spinulatus</em>
 possess non‐coding regions, measuring 10,301 and 7490 bp, respectively. While non‐coding regions are uncommon in diatom mt genomes, researchers have previously reported them. For example, 
<em>P. tricornutum</em>
 has a non‐coding region of 35,454 bp that is composed of directed repeats, and 
<em>T. pseudonana</em>
 has a 4970 bp repeat region (Oudot‐Le Secq and Green <a href="#jeu70033-bib-0087" class="usa-link" aria-describedby="jeu70033-bib-0087">2011</a>). In 
<em>P. tricornutum</em>
, repeats account for 46% of the mt genome, whereas in <em>M. trioculatus</em> and 
<em>M. spinulatus</em>
, repeats make up 13%–15% of the mt genome. In contrast to 
<em>P. tricornutum</em>
, <em>M. trioculatus</em> also has seven introns in the <em>cox1</em> gene, which account for 24.5% (19,435 bp) of the genome. <em>Mediolabrus comicus</em> similarly has two introns in the <em>cox1</em> gene, while 
<em>M. spinulatus</em>
 has none. The presence of group II introns within the <em>cox1</em> gene has been observed in other diatoms, such as <em>H. calidilacuna</em> and <em>H. coffeaeformis</em> (Pogoda et al. <a href="#jeu70033-bib-0094" class="usa-link" aria-describedby="jeu70033-bib-0094">2019</a>). The <em>cox1</em> gene is the most commonly used marker for animals and is often referred to as the ‘barcode of life’ (Nakazato and Jinbo <a href="#jeu70033-bib-0082" class="usa-link" aria-describedby="jeu70033-bib-0082">2022</a>). Our data supports Pogoda et al. (<a href="#jeu70033-bib-0094" class="usa-link" aria-describedby="jeu70033-bib-0094">2019</a>), who argue that the introns found in the <em>cox1</em> gene may partially explain the varying success in amplifying the <em>cox1</em> gene in diatoms. The lack of non‐coding regions in <em>M. comicus</em> mirrors the compact mitochondrial genomes of <em>Skeletonema</em> species, but unlike <em>Skeletonema</em>, <em>M. comicus</em> possesses four introns. Differences in non‐coding region lengths and intron numbers among closely related species' mt genomes highlight the need for broader organellar genome sequencing in diatoms to better understand the distribution of mobile genetic elements and assess the reliability of potential biomarkers (i.e., <em>cox1</em>) across taxonomic groups.</p></section><section id="jeu70033-sec-0022"><h3 class="pmc_sec_title">4.4. Patterns in Gene Retention and Loss Across the Three Isolates</h3>
<p>Results from this study contribute to our collective knowledge on gene retention and loss in diatoms. In the mt genomes, the <em>tatA</em> gene was present in <em>M. comicus</em> but absent in both of the <em>Minidiscus</em> isolates. The <em>tatA</em> gene is involved in the twin‐arginine translocation pathway that translocates proteins across membranes (Cline and Theg <a href="#jeu70033-bib-0017" class="usa-link" aria-describedby="jeu70033-bib-0017">2007</a>; Bennewitz et al. <a href="#jeu70033-bib-0006" class="usa-link" aria-describedby="jeu70033-bib-0006">2020</a>). Loss of <em>tatA</em> has previously been found in two Coscinodiscophyceae species, <em>Guinardia striata</em> and <em>Actinocyclus</em> sp., making the two <em>Minidiscus</em> species the first diatoms in the Mediophyceae class to lack the <em>tatA</em> gene (Wang et al. <a href="#jeu70033-bib-0115" class="usa-link" aria-describedby="jeu70033-bib-0115">2022</a>). Moreover, in all three mt genomes, the <em>nad11</em> gene was present as a single, uninterrupted sequence, consistent with most other mt genomes within the class Mediophyceae. The only currently known exception in Mediophyceae is 
<em>Odontella aurita</em>
 var. <em>minima</em>, in which <em>nad11</em> is split into two segments (Wang et al. <a href="#jeu70033-bib-0115" class="usa-link" aria-describedby="jeu70033-bib-0115">2022</a>).</p>
<p>The cp genomes contained a total of 31 tRNAs, comprising 28 distinct tRNAs when accounting for duplicates in the inverted repeat (IR) regions. All three species shared tRNA‐Ala and tRNA‐Ile in their IR regions. However, while 
<em>M. spinulatus</em>
 and <em>M. comicus</em> also contained tRNA‐Leu, <em>Minidiscus trioculatus</em> differed by featuring a duplicated tRNA‐Phe. The total tRNAs match what has been previously found in the <em>Skeletonema</em> genus and are close to the 30 total tRNAs in most currently known <em>Thalassiosira</em> cp genomes (Liu, Chen, et al. <a href="#jeu70033-bib-0070" class="usa-link" aria-describedby="jeu70033-bib-0070">2021</a>; Liu, Xu, et al. <a href="#jeu70033-bib-0073" class="usa-link" aria-describedby="jeu70033-bib-0073">2021</a>). Regarding overlapping gene pairs, the three isolates exhibited the same four gene overlaps that have been consistently observed across all <em>Thalassiosira</em> species (Table <a href="#jeu70033-tbl-0003" class="usa-link">3</a>) (Liu, Chen, et al. <a href="#jeu70033-bib-0070" class="usa-link" aria-describedby="jeu70033-bib-0070">2021</a>). The <em>petF</em> (ferredoxin) gene plays a role in electron transport during photosynthesis and has been transferred from the cp to the nuclear genome in select species, including some <em>Thalassiosira</em> species, likely enabling them to better tolerate iron‐limited environments (Lommer et al. <a href="#jeu70033-bib-0075" class="usa-link" aria-describedby="jeu70033-bib-0075">2010</a>; Liu, Chen, et al. <a href="#jeu70033-bib-0070" class="usa-link" aria-describedby="jeu70033-bib-0070">2021</a>). However, this endosymbiotic gene transfer has not occurred in any of the three isolates herein.</p></section><section id="jeu70033-sec-0023"><h3 class="pmc_sec_title">4.5. Organellar Genome Sequencing Refines Distributions of Isolated Thalassiosirales Nanoplanktonic Diatoms in the NWA That Are Derived by eDNA Metabarcoding</h3>
<p>A major benefit of sequencing chloroplast genomes from isolated cultures is the detection of full‐length <em>16S</em> rRNA gene sequences, which enables the strain‐specific identification of eDNA metabarcoding ASVs. Previously, Robicheau et al. (<a href="#jeu70033-bib-0099" class="usa-link" aria-describedby="jeu70033-bib-0099">2022</a>) described the succession of two <em>Minidiscus</em> ASVs (differing by 1 bp) throughout the year, which were classified as <em>M. trioculatus</em> based on available reference databases. When reinvestigated, the <em>16S</em> rRNA sequences from the cp genomes revealed that one of those ASVs is 
<em>M. spinulatus</em>
, highlighting the fundamental principle that microbial community datasets can only be correctly interpreted to the degree that reference databases can be relied on. Ongoing dedication to sequencing uncharacterized nuclear and plastid genomes of important phytoplankton species is crucial for enhancing the accuracy of microbial community analyses.</p></section><section id="jeu70033-sec-0024"><h3 class="pmc_sec_title">4.6. Situating the NWA Distribution of Isolates Within Global Biogeographic Patterns</h3>
<p>Global monitoring programs using metabarcoding (primarily <em>18S</em> rRNA), microscopy, and flow cytometry have identified <em>Minidiscus</em> and <em>Mediolabrus</em> as key players across a range of ecosystems. The <em>Tara Oceans</em> expedition (2009–2013) found that <em>Minidiscus</em> (encompassing <em>Mediolabrus</em>) was one of the most commonly found diatoms with a high level of occupancy determined by both the V4 and V9 <em>18S</em> rRNA markers (Pierella Karlusich et al. <a href="#jeu70033-bib-0093" class="usa-link" aria-describedby="jeu70033-bib-0093">2025</a>; Malviya et al. <a href="#jeu70033-bib-0076" class="usa-link" aria-describedby="jeu70033-bib-0076">2016</a>). In <em>Tara Oceans</em>, <em>Minidiscus</em> were detected primarily in the surface ocean, with some detection at the deep chlorophyll maximum (DCM) (Pierella Karlusich et al. <a href="#jeu70033-bib-0093" class="usa-link" aria-describedby="jeu70033-bib-0093">2025</a>; Malviya et al. <a href="#jeu70033-bib-0076" class="usa-link" aria-describedby="jeu70033-bib-0076">2016</a>). These genera have also been detected in various localized monitoring programs; these include the NBay Plankton Time Series in Narragansett Bay, Rhode Island (2008–2014); the Elbe Estuary in Northern Germany (2021–2022); the LTER‐MC program in the Gulf of Naples (1984–2010, 2011–2013); the SOMLIT‐Astan monitoring program in the Western English Channel (2009–2016); the DeWeX program in the Gulf of Lions, Mediterranean Sea (2013); a survey in Marian Cove, Antarctica (January–February 2019); and natural salt marsh pools in California, USA (2018) (Fontaine and Rynearson <a href="#jeu70033-bib-0031" class="usa-link" aria-describedby="jeu70033-bib-0031">2023</a>; Martens, Biederbick, et al. <a href="#jeu70033-bib-0077" class="usa-link" aria-describedby="jeu70033-bib-0077">2024</a>; Martens, Russnak, et al. <a href="#jeu70033-bib-0078" class="usa-link" aria-describedby="jeu70033-bib-0078">2024</a>; Zingone et al. <a href="#jeu70033-bib-0126" class="usa-link" aria-describedby="jeu70033-bib-0126">2023</a>; Caracciolo, Berney, et al. <a href="#jeu70033-bib-0011" class="usa-link" aria-describedby="jeu70033-bib-0011">2022</a>; Caracciolo, Rigaut‐Jalabert, et al. <a href="#jeu70033-bib-0012" class="usa-link" aria-describedby="jeu70033-bib-0012">2022</a>; Jeon et al. <a href="#jeu70033-bib-0050" class="usa-link" aria-describedby="jeu70033-bib-0050">2023</a>; Arsenieff et al. <a href="#jeu70033-bib-0005" class="usa-link" aria-describedby="jeu70033-bib-0005">2020</a>; Vineis et al. <a href="#jeu70033-bib-0112" class="usa-link" aria-describedby="jeu70033-bib-0112">2025</a>; Leblanc et al. <a href="#jeu70033-bib-0062" class="usa-link" aria-describedby="jeu70033-bib-0062">2018</a>).</p>
<p>Our work highlights the presence of these nanoplanktonic diatoms on the SS and in the LS—areas that have not been sampled by the global <em>Tara Oceans</em> project. Moreover, the detection of <em>M. comicus</em> in the CAG in 2015 reflects earlier records of <em>Minidiscus</em> species in the region (Pierella Karlusich et al. <a href="#jeu70033-bib-0093" class="usa-link" aria-describedby="jeu70033-bib-0093">2025</a>; Malviya et al. <a href="#jeu70033-bib-0076" class="usa-link" aria-describedby="jeu70033-bib-0076">2016</a>). The distribution and seasonal patterns of these nanoplanktonic diatoms suggest that the 
<em>M. spinulatus</em>
 isolate from the Northwest Atlantic (NWA) is primarily restricted to the in‐shore BB and is not commonly found in open ocean areas. The other known circularized cp genome for this species comes from Jiaozhou Bay (China), which is also close to the coastline (Wang and Chen <a href="#jeu70033-bib-0114" class="usa-link" aria-describedby="jeu70033-bib-0114">2022</a>). In contrast, <em>M. trioculatus</em> and <em>M. comicus</em> appear to be more broadly distributed across the NWA. The seasonal trends observed in the BB (Figure <a href="#jeu70033-fig-0007" class="usa-link">7</a>) are reflected in all years of sampling the SS region: <em>M. trioculatus</em> tends to be more abundant in the spring, whereas <em>M. comicus</em> is more prevalent in the fall (Figure <a href="#jeu70033-supitem-0001" class="usa-link">S5</a>).</p></section><section id="jeu70033-sec-0025"><h3 class="pmc_sec_title">4.7. Comparative Seasonal Dynamics of Nanoplanktonic Diatom Genotypes Across Diverse Coastal Systems</h3>
<p>The succession of different <em>Minidiscus</em> genotypes has been observed across multiple monitoring programs, reflecting strong seasonal and regional patterns. In Narragansett Bay (2008–2014), <em>Minidiscus</em> and <em>Mediolabrus</em> species exhibited distinct seasonal trends: 
<em>M. spinulatus</em>
 was more abundant from January to June, <em>M. trioculatus</em> appeared mainly between March and June, and two <em>Mediolabrus comicus</em> ASVs displayed distinct seasonal peaks—one in late fall and another from May to November (Fontaine and Rynearson <a href="#jeu70033-bib-0031" class="usa-link" aria-describedby="jeu70033-bib-0031">2023</a>). In the Gulf of Naples (LTER‐MC station, 1984–2010), <em>M. comicus</em> showed a bimodal pattern, appearing twice annually (Zingone et al. <a href="#jeu70033-bib-0126" class="usa-link" aria-describedby="jeu70033-bib-0126">2023</a>); this pattern was also confirmed with <em>18S</em> rRNA data (2011–2013) at both LTER‐MC and the SOMLIT‐Astan station in the Western English Channel, despite differing environmental conditions (Caracciolo, Berney, et al. <a href="#jeu70033-bib-0011" class="usa-link" aria-describedby="jeu70033-bib-0011">2022</a>). Extended monitoring at SOMLIT‐Astan (2009–2016) further revealed species‐specific seasonality: <em>Minidiscus variabilis
</em> occurred year‐round, 
<em>M. spinulatus</em>
 peaked in the second half of the year, and <em>M. comicus</em> dominated from November to March (Caracciolo, Rigaut‐Jalabert, et al. <a href="#jeu70033-bib-0012" class="usa-link" aria-describedby="jeu70033-bib-0012">2022</a>). In the Elbe Estuary (2021–2022), eight <em>Minidiscus</em> genotypes were identified, with three dominating (
<em>M. spinulatus</em>
 genotypes 1 and 2, and <em>M. comicus</em>), showing salinity‐dependent distributions (Martens, Biederbick, et al. <a href="#jeu70033-bib-0077" class="usa-link" aria-describedby="jeu70033-bib-0077">2024</a>; Martens, Russnak, et al. <a href="#jeu70033-bib-0078" class="usa-link" aria-describedby="jeu70033-bib-0078">2024</a>). These results demonstrate that not all nanoplanktonic diatoms in the <em>Minidiscus</em> and <em>Mediolabrus</em> genera display the same seasonal patterns and are likely controlled by environmental factors and/or seasonal shifts specific to each study area. Moreover, these studies provide evidence that different genotypes of the same species have been shown to have different seasonal patterns (Fontaine and Rynearson <a href="#jeu70033-bib-0031" class="usa-link" aria-describedby="jeu70033-bib-0031">2023</a>; Martens, Russnak, et al. <a href="#jeu70033-bib-0078" class="usa-link" aria-describedby="jeu70033-bib-0078">2024</a>; Martens, Biederbick, et al. <a href="#jeu70033-bib-0077" class="usa-link" aria-describedby="jeu70033-bib-0077">2024</a>).</p>
<p>The BBMP's weekly resolution provides a clearer view of the succession of the three isolates with nine years of metabarcoding data (Figure <a href="#jeu70033-fig-0007" class="usa-link">7</a>). The bimodal seasonal pattern similar to that observed in the Gulf of Naples and the Western English Channel was observed in the BB in 2014, 2016, 2017, 2018, and 2022 (Figure <a href="#jeu70033-supitem-0001" class="usa-link">S6</a>) (Caracciolo, Berney, et al. <a href="#jeu70033-bib-0011" class="usa-link" aria-describedby="jeu70033-bib-0011">2022</a>). In the BBMP, <em>Minidiscus spinulatus</em> peaked in the second half of the year, similar to the 
<em>M. spinulatus</em>
 identified in the SOMLIT‐Astan time series and dissimilar to the trend in Narragansett Bay (Caracciolo, Berney, et al. <a href="#jeu70033-bib-0011" class="usa-link" aria-describedby="jeu70033-bib-0011">2022</a>; Fontaine and Rynearson <a href="#jeu70033-bib-0031" class="usa-link" aria-describedby="jeu70033-bib-0031">2023</a>). We observed <em>M. trioculatus</em> almost exclusively in the first half of the year, from January to June, which is a larger but similar window to that observed in Narragansett Bay (March–June) (Fontaine and Rynearson <a href="#jeu70033-bib-0031" class="usa-link" aria-describedby="jeu70033-bib-0031">2023</a>).</p></section><section id="jeu70033-sec-0026"><h3 class="pmc_sec_title">4.8. Evidence for Biological Carbon Export by Nanoplanktonic Diatoms in the Northwest Atlantic</h3>
<p>It was previously assumed that nanoplanktonic diatoms, due to their small size, were predominantly remineralized within the microbial loop; however, our findings align with more recent studies demonstrating that these diatoms are capable of contributing significantly to carbon export (Leblanc et al. <a href="#jeu70033-bib-0062" class="usa-link" aria-describedby="jeu70033-bib-0062">2018</a>; Malviya et al. <a href="#jeu70033-bib-0076" class="usa-link" aria-describedby="jeu70033-bib-0076">2016</a>; Bolaños et al. <a href="#jeu70033-bib-0009" class="usa-link" aria-describedby="jeu70033-bib-0009">2020</a>). Notably, earlier work has identified species 
<em>M. spinulatus</em>
, <em>M. trioculatus</em>, <em>M. variabilis</em>, and <em>M. comicus</em> within benthic communities along Nova Scotia's coast, suggesting some potential for sedimentary deposition (Krumhansl et al. <a href="#jeu70033-bib-0057" class="usa-link" aria-describedby="jeu70033-bib-0057">2025</a>). Our study extends this knowledge by detecting these species in deeper off‐shore waters, indicating an increased potential for long‐term carbon storage. Although there were no sediment samples taken for any of the off‐shore expeditions, the ASVs for our isolates were often detected below the euphotic layer (&gt; 150 m) (Guidi et al. <a href="#jeu70033-bib-0036" class="usa-link" aria-describedby="jeu70033-bib-0036">2016</a>). On the SS, both <em>M. trioculatus</em> and <em>M. comicus</em> were found throughout the sampled water column from surface to 300 m, and in the LS, <em>Mediolabrus</em> was found in various water samples from the surface to 1000 m. In the Arctic sampling of 2015, both <em>M. comicus</em> and <em>M. trioculatus</em> were found as deep as 1400 m in the LS, and <em>M. comicus</em> was notably detected as deep as 2300 m in the CAG. These observations suggest that nanoplanktonic diatoms may not only escape remineralization in the upper water column but also reach deep ocean layers, enhancing their role in vertical carbon flux.</p></section><section id="jeu70033-sec-0027"><h3 class="pmc_sec_title">4.9. Environmental Drivers of Nanoplanktonic Diatom Abundance in the NWA and Implications for Future Bloom Dynamics</h3>
<p>Temperature was the key factor shaping the relative abundance of the three nanoplankton ASVs in the BB, with <em>M. trioculatus</em> favoring colder waters and 
<em>M. spinulatus</em>
 and <em>M. comicus</em> more abundant in warmer conditions (Figure <a href="#jeu70033-fig-0007" class="usa-link">7</a>). A comparable seasonal pattern was observed on the SS, where <em>M. trioculatus</em> was more dominant in the colder spring months, while <em>M. comicus</em> became more prevalent in the warmer fall. Our findings from both the BB and SS suggest that 
<em>M. spinulatus</em>
 and <em>M. comicus</em> thrive under warm, high‐biomass conditions, indicating greater resilience to future ocean warming compared to <em>M. trioculatus</em>. A notable contradictory finding is the broader distribution of <em>M. comicus</em> into the LS and CAG. If temperature were the primary driver, the cold‐adapted <em>M. trioculatus</em> would be expected to dominate these regions. This discrepancy highlights the complexity of environmental influences, as temperature is not only a direct factor but is also correlated with other oceanographic processes such as stratification, nutrient availability, light availability, and the concentration of dissolved gases. It is therefore possible that other environmental factors may play a more important role in shaping the species distributions in the LS and CAG. Alternatively, cold‐adapted <em>M. comicus</em> strains may exist, with genomic differences undetectable by the <em>16S</em> region alone. Similar strain‐level variation has been observed using the <em>18S</em> rRNA gene, where <em>M. comicus</em> ASVs exhibit distinct seasonal patterns in Narragansett Bay (Fontaine and Rynearson <a href="#jeu70033-bib-0031" class="usa-link" aria-describedby="jeu70033-bib-0031">2023</a>).</p>
<p>Previous surveys have identified intense blooms and blooming capabilities of <em>Minidiscus</em>. During a study in Marian Cove, Antarctica, nano‐diatoms (including <em>Minidiscus</em>) made up approximately 20% of the total phytoplankton abundance, with <em>Minidiscus</em> itself reaching high concentrations ranging from 2.7 × 10<sup>5</sup> to 5.6 × 10<sup>6</sup> cells L<sup>−1</sup> (Jeon et al. <a href="#jeu70033-bib-0050" class="usa-link" aria-describedby="jeu70033-bib-0050">2023</a>). The highest <em>Minidiscus</em> abundances were generally observed near the glacier, associated with surface haline stratification and low‐light conditions. Another bloom was observed in the Mediterranean Sea; these species also reached very high abundances during an April expedition, with cell concentrations of 4–6 × 10<sup>6</sup> cells L<sup>−1</sup> at several stations (Leblanc et al. <a href="#jeu70033-bib-0062" class="usa-link" aria-describedby="jeu70033-bib-0062">2018</a>). Moreover, in a coastal upwelling experiment, two <em>Minidiscus</em> ASVs were identified as stronger blooming species in the mesocosm (Vineis et al. <a href="#jeu70033-bib-0112" class="usa-link" aria-describedby="jeu70033-bib-0112">2025</a>). Although our work does not show direct evidence of large nanoplanktonic blooms in the NWA, the detection of these diatoms across depths, seasons, and years demonstrates the persistent presence of these species in the NWA. Research in the BB has demonstrated that smaller cells tend to dominate under conditions that depart from the typical seasonal cycles (i.e., increased temperature) (Robicheau et al. <a href="#jeu70033-bib-0099" class="usa-link" aria-describedby="jeu70033-bib-0099">2022</a>). We can therefore hypothesize that as climate change continues to cause changes to the ocean, such as rising temperatures, intensified stratification, and altered nutrient availability, small‐sized phytoplankton may become more dominant and potentially contribute to intense blooms as observed previously (Leblanc et al. <a href="#jeu70033-bib-0062" class="usa-link" aria-describedby="jeu70033-bib-0062">2018</a>; Jeon et al. <a href="#jeu70033-bib-0050" class="usa-link" aria-describedby="jeu70033-bib-0050">2023</a>; Bolaños et al. <a href="#jeu70033-bib-0009" class="usa-link" aria-describedby="jeu70033-bib-0009">2020</a>).</p></section></section><section id="jeu70033-sec-0028"><h2 class="pmc_sec_title">5. Conclusions</h2>
<p>Here we presented the organellar genomes for three nanoplanktonic diatoms isolated in the NWA. The pangenomic analyses of the organellar genomes reaffirmed the distinct <em>Minidiscus</em> and <em>Mediolabrus</em> clades proposed by Li et al. (<a href="#jeu70033-bib-0068" class="usa-link" aria-describedby="jeu70033-bib-0068">2020</a>). Additionally, we identified unique features that contributed to the longer length of the <em>M. trioculatus</em> mt genome, including a non‐coding region and seven introns within the <em>cox1</em> gene. Patterns of gene retention and loss were observed in the organellar genomes, such as <em>tatA</em> loss in the <em>Minidiscus</em> isolates' mt genomes and <em>petF</em> retention in all three isolates' cp genomes. The cp genomes from the isolated cultures enabled precise identification of diatom species through the re‐investigation of previous <em>16S</em> rRNA metabarcoding from the NWA. In the BB, we identified seasonal preferences, primarily driven by temperature, that influenced the abundance of the isolates throughout the year. On the SS and in the open NWA, we found <em>M. comicus</em> and <em>M. trioculatus</em> below the euphotic zone and as deep as 1400 m, which indicates their potential contribution to the biological carbon pump. The consistent detection of the isolates across depths, seasons, and regions suggests persistent presence in the NWA; however, no indication of intense <em>Minidiscus</em> or <em>Mediolabrus</em>‐dominated blooms was found. Projected shifts in ocean conditions due to climate change will likely favor smaller phytoplankton, and these isolates are therefore candidates for nanoplanktonic diatom‐dominated blooms in the future.</p></section><section id="sec29"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="jeu70033-supitem-0001"><div class="caption p">
<p>
<strong>Figure S1:</strong> Isolation of nanoplanktonic diatom cultures. Map showing the sampling locations from where three nanoplanktonic diatoms were isolated: <em>Minidiscus spinulatus</em>, collected from the BBMP in July 2018, and <em>Minidiscus trioculatus</em> and <em>Mediolabrus comicus</em>, both collected from the COOGER‐2021 in August 2021 (Mordret et al. 2024).</p>
<p>
<strong>Figure S2:</strong> Schematic representation of complete mitochondrial (a) and chloroplast (b) genomes from the three isolates, annotated using OGDRAW (Greiner et al. <a href="#jeu70033-bib-0034" class="usa-link" aria-describedby="jeu70033-bib-0034">2019</a>). Genetic elements displayed outside the ring are transcribed counterclockwise, while the elements inside the ring are transcribed clockwise. The color of the gene box indicates the type of gene. The GC‐content is shown on the inner ring of each of the genomes.</p>
<p>
<strong>Figure S3:</strong> Yearly identification of nanoplanktonic diatom isolates in environmental metabarcoding samples, including the sampling conducted in AS18 and AS19 in the upper upper photic zone (1–20 m). All of the “AF” samples were collected in the fall; the “AS”, “AZOMP” and “CE” samples were collected in the spring; and the “Arctic15” samples were collected in the summer (see Table 1). Abbreviations: AF, Atlantic Zone Monitoring Program (AZMP) Fall (usually Sept–Oct, but Nov–Dec in 2017); AS, AZMP Spring (Apr–Jun); AZOMP, Atlantic Zone Off‐shore Monitoring Program (Apr–Jun); BELAS‐1, Biological Carbon Export in the Labrador Sea (May–Jun); Arctic15, GEOTRACES mission to CAG (Jul–Aug).</p>
<p>
<strong>Figure S4:</strong> Nanoplanktonic diatom isolates tracked across the NWA and Canadian Arctic using <em>16S</em> metabarcoding of eDNA from any sample collected (1–2500 m; Niskin water samples and marine snow catcher (MSC) samples). Abbreviations: AF, Atlantic Zone Monitoring Program (AZMP) Fall (usually Sept–Oct, but Nov–Dec in 2017); AS, AZMP Spring (Apr–Jun); AZOMP, Atlantic Zone Off‐shore Monitoring Program (Apr–Jun); BELAS‐1, Biological Carbon Export in the Labrador Sea (May–Jun); Arctic15, GEOTRACES mission to CAG (Jul–Aug). Note numbers at the end of mission codes indicate year (see Table 1 for more information). For a more complete figure including limited sampling in AS18 and AS19, see Figure S3.</p>
<p>
<strong>Figure S5:</strong> Proportion of photosynthetic community contributed by nanoplanktonic isolates in the upper photic zone (1–20 m). This demonstrates the AZMP cruises that were completed in both Spring and Fall in order to determine a seasonal pattern for the proportion of the community that is each of the isolates. The environmental metabarcoding data is collected from various depths at each station using the <em>16S</em> V4V5 and <em>16S</em> V6V8 markers. Each data point is the proportion of the community that the ASV takes up in one sample, determined by the sample that had the highest proportion of that ASV. The heatmap represents the relative abundance (%) of the ASV to the rest of the photosynthetic community in that sample.</p>
<p>
<strong>Figure S6:</strong> Vertical distribution of the photosynthetic community in the Bedford Basin. The nanoplankton community was more abundant at 1 m depth compared to 60 m, but all three isolates were present throughout the water column at the four depths sampled.</p>
<p>
<strong>Figure S7:</strong> Proportional abundance of nanoplankton on 3 μM and 0.2 μM filters in Bedford Basin. Nanoplankton constituted a larger proportion of the community on the 3 μM filter compared to the 0.2 μM filter in the years sampled, although they were present in both size fractions.</p>
<p>
<strong>Figure S8:</strong> Distribution of nanoplankton isolates across depths in the Bedford Basin in relation to ocean conditions. Proportion of the community (Rel. Abun.), calculated from the % of the V4V5 region of the cp 16S rRNA, is shown for 1 m and 60 m depths over time (2014–2022) using 0.2‐μm filtered and reconstituted 3 and 0.2 μm size fractions. Environmental variables including temperature (temp), chlorophyll <em>a</em> (chl <em>a</em>), oxygen (O2), nitrate (NO3), nitrite (NO2), phosphate (PO4), silicate (SiO4), and salinity (sal) and flow cytometry counts for nanoeukaryotes (nano), picoeukaryotes (pico), cryptophytes (crypto), and <em>Synechococcus</em> (<em>Syn</em>) are plotted. Missing data is due to COVID‐19 restrictions.</p>
<p>
<strong>Figure S9:</strong> Month of the year versus relative abundance. Dot plot of the relative abundance of each isolate in the photosynthetic community by month from 2014 to 2022. Each dot represents one water sample collected during a select month, year, and depth.</p>
<p>
<strong>Figure S10:</strong> Boxplots showing the distribution of temperatures (°C) at which each species was detected based on non‐zero relative abundance. Significant differences in temperature distributions were detected among species (Kruskal–Wallis test: χ<sup>2</sup> = 461.34, <em>p</em> &lt; 2.2e‐16). Pairwise Wilcoxon rank‐sum tests with Benjamini–Hochberg correction indicated that all species pairs differed significantly. Boxes represent the interquartile range (IQR), with medians shown as horizontal lines; whiskers extend to 1.5× IQR. Outliers are not shown for clarity.</p>
<p>
<strong>Figure S11:</strong> Correlation of variables related to the RDA analyses. (a) Representation of the correlations between environmental variables before reducing the dataset to non‐correlated variables. (b) Reduced dataset used for initial RDA.</p>
<p>
<strong>Figure S12:</strong> BLAST comparison of the <em>Minidiscus spinulatus</em> genome with the previously sequenced genome (Grant et al. <a href="#jeu70033-bib-0033" class="usa-link" aria-describedby="jeu70033-bib-0033">2023</a>; Wang and Chen <a href="#jeu70033-bib-0114" class="usa-link" aria-describedby="jeu70033-bib-0114">2022</a>). The first two inner rings are the annotations of key components of the cp on the forward and reverse strands. The closest ring represents <a href="https://www.ncbi.nlm.nih.gov/nuccore/OL855995" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OL855995</a> <em>Minidiscus spinulatus</em> as downloaded from GenBank. The outer two genomes represent <em>Minidiscus trioculatus</em> and <em>Mediolabrus comicus</em> as a reference.</p>
<p>
<strong>Figure S13:</strong> Heatmap of the 44 protein‐coding genes outlined by Liu et al. (2021a) applied to the three new nanoplanktonic diatom isolates. Blue indicates gene presence, and white indicates gene absence.</p>
<p>
<strong>Data S1:</strong> Summary of Oxford Nanopore sequencing runs.</p>
<p>
<strong>Data S2:</strong> FASTA file of ASV sequences from eDNA metabarcoding aligned to cp 16S.</p>
<p>
<strong>Data S3:</strong> Output of Anvi‘o pangenome analyses.</p>
<p>
<strong>Data S4:</strong> Pairwise Average Nucleotide Identity (ANI) calculated with fastANI for each of the cp and mt genomes.</p>
<p>
<strong>Table S1:</strong> Cell sizes for each species included in Figure 1 with associated reference.</p>
<p>
<strong>Table S2:</strong> Mitochondrial genomes and associated cell sizes in the Order Thalassiosirales.</p>
<p>
<strong>Table S3:</strong> Chloroplast genomes and associated cell sizes in the Order Thalassiosirales.</p>
<p>
<strong>Table S4:</strong> Mitochondrial genes included in the concatenated phylogenetic tree of 16 Thalassiosirales mitochondrial genomes.</p>
<p>
<strong>Table S5:</strong> Chloroplast genes included in the concatenated phylogenetic tree of 37 Thalassiosirales chloroplast genomes.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12326293/bin/JEU-72-e70033-s001.zip" data-ga-action="click_feat_suppl" class="usa-link">JEU-72-e70033-s001.zip</a><sup> (14.2MB, zip) </sup>
</div></div></section></section><section id="jeu70033-sec-0029" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We extend our sincere gratitude to the crews and scientific teams who contributed to the collection, processing, and analysis of the environmental data utilized in this study. We acknowledge those involved in COOGER‐2021 for making the isolation of <em>M. trioculatus</em> and <em>M. comicus</em> possible. We gratefully acknowledge the Bedford Institute of Oceanography (BIO) and Fisheries and Oceans Canada (DFO) for their invaluable support in sampling efforts carried out through the BBMP, AZMP, and AZOMP programs. We would like to acknowledge the GEOTRACES project for the data provided from the Labrador Sea and Canadian Arctic Gateway in 2015. We would also like to thank our collaborators involved in the Ocean Frontier Institute's Northwest Atlantic Biological Carbon Pump (NWA‐BCP) project and the scientists and crew on board the <em>RV Celtic Explorer</em> in 2022 during the BELAS‐1 mission. The crew and scientific teams aboard each of the research vessels are what make research possible. Finally, we would like to thank those involved in the processing of the DNA samples for each of the projects including past LaRoche lab members for DNA extraction and the Integrated Microbiome Resource (IMR) for sequencing efforts.</p></section><section id="notes1"><p>


Stevens‐Green, R.
, Chénard C., Mordret S., MacKinnon J., Robicheau B. M., and LaRoche J.. 2025. “Organellar Genomes of Three Globally Important Nanoplanktonic Diatoms Refine Their Taxon‐Specific Distribution and Succession Patterns in the Northwest Atlantic.” Journal of Eukaryotic Microbiology
72, no. 5: e70033. 10.1111/jeu.70033.

</p>
<section id="jeu70033-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="jeu70033-note-0001"><p>
<strong>Funding:</strong> This study was supported by the NRC‐ACRD Internal Project (A1‐019982) awarded to CC, the NRC Ocean Program Grant and Contribution (OCN‐500) awarded to CC and JLR, the NSERC Discovery Grant (RGPIN/04060‐2021) awarded to JLR and the Canada's First Research Excellence Fund (NWA BCP) awarded to JLR. Through this project, RSG was supported at different times by the NSERC Canada Graduate Scholarship‐Masters' and subsequently Doctoral Award with the Fisheries and Oceans Canada Aquatic Science Supplement provided by DFO, as well as by a N.S. Graduate Scholarship from Dalhousie University. During the course of this study, BMR was supported at different times by NSERC through a Canada Graduate Scholarship‐Doctoral Award and a Postdoctoral Fellowship, as well as by a N.S. Graduate Scholarship and a Killam Predoctoral Award from Dalhousie University.</p></div></div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Rebecca Stevens‐Green, Email: r.stevens-green@dal.ca.</p>
<p>Julie LaRoche, Email: julie.laroche@dal.ca.</p></section><section id="jeu70033-sec-0031"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The data that support the findings of this study are available from the corresponding authors upon request. The isolated cultures are available in the AGSB culture collection maintained by the NRC in Halifax, NS.</p>
<p>Data from the eDNA metabarcoding of the Bedford Basin Monitoring Program from 2014 to 2017 can be found under this BioProject: <a href="https://www.ncbi.nlm.nih.gov/bioproject/785606" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/785606</a>.</p>
<p>Data from the eDNA metabarcoding of the Atlantic Zonal Monitoring Program can be found under this BioProject: <a href="https://www.ncbi.nlm.nih.gov/bioproject/785872" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/785872</a>.</p></section><section id="jeu70033-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<p>References marked with asterisk were cited in Tables <a href="#jeu70033-supitem-0001" class="usa-link">S1–S5</a>.</p>
<section id="jeu70033-bibl-0001_sec1"><section id="jeu70033-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="jeu70033-bib-0001">
<cite>*
Ács, É.
, Ari E., Duleba M., et al. 2016. “
<em>Pantocsekiella</em>, a New Centric Diatom Genus Based on Morphological and Genetic Studies.” Fottea
16, no. 1: 56–78. 10.5507/fot.2015.028.</cite> [<a href="https://doi.org/10.5507/fot.2015.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Fottea&amp;title=Pantocsekiella,%20a%20New%20Centric%20Diatom%20Genus%20Based%20on%20Morphological%20and%20Genetic%20Studies&amp;author=%C3%89.%20%C3%81cs&amp;author=E.%20Ari&amp;author=M.%20Duleba&amp;volume=16&amp;issue=1&amp;publication_year=2016&amp;pages=56-78&amp;doi=10.5507/fot.2015.028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0002">
<cite>

Aké‐Castillo, J. A.
, Hernández‐Becerril D. U., Maria E., del Castillo M., and Bravo‐Sierra E.. 2001. “Species of <em>Minidiscus</em> (Bacillariophyceae) in the Mexican Pacific Ocean.” Cryptogamie Algologie
22, no. 1: 101–107. 10.1016/s0181-1568(00)01051-5.</cite> [<a href="https://doi.org/10.1016/s0181-1568(00)01051-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cryptogamie%20Algologie&amp;title=Species%20of%20Minidiscus%20(Bacillariophyceae)%20in%20the%20Mexican%20Pacific%20Ocean&amp;author=J.%20A.%20Ak%C3%A9%E2%80%90Castillo&amp;author=D.%20U.%20Hern%C3%A1ndez%E2%80%90Becerril&amp;author=E.%20Maria&amp;author=M.%20del%20Castillo&amp;author=E.%20Bravo%E2%80%90Sierra&amp;volume=22&amp;issue=1&amp;publication_year=2001&amp;pages=101-107&amp;doi=10.1016/s0181-1568(00)01051-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0003">
<cite>

Amir, A.
, McDonald D., Navas‐Molina J. A., et al. 2017. “Deblur Rapidly Resolves Single‐Nucleotide Community Sequence Patterns.” mSystems
2, no. 2: e00191‐16. 10.1128/mSystems.00191-16.
</cite> [<a href="https://doi.org/10.1128/mSystems.00191-16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5340863/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28289731/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=Deblur%20Rapidly%20Resolves%20Single%E2%80%90Nucleotide%20Community%20Sequence%20Patterns&amp;author=A.%20Amir&amp;author=D.%20McDonald&amp;author=J.%20A.%20Navas%E2%80%90Molina&amp;volume=2&amp;issue=2&amp;publication_year=2017&amp;pages=e00191%E2%80%9016&amp;pmid=28289731&amp;doi=10.1128/mSystems.00191-16&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0004">
<cite>*
Ando, Y.
, and Katano T.. 2018. “Effect of Cell Size on Growth Rates of Two <em>Skeletonema</em> Species and Their Competitive Interactions.” Journal of Phycology
54, no. 6: 850–859. 10.1111/jpy.12782.
</cite> [<a href="https://doi.org/10.1111/jpy.12782" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30179257/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Phycology&amp;title=Effect%20of%20Cell%20Size%20on%20Growth%20Rates%20of%20Two%20Skeletonema%20Species%20and%20Their%20Competitive%20Interactions&amp;author=Y.%20Ando&amp;author=T.%20Katano&amp;volume=54&amp;issue=6&amp;publication_year=2018&amp;pages=850-859&amp;pmid=30179257&amp;doi=10.1111/jpy.12782&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0005">
<cite>

Arsenieff, L.
, Le Gall F., Rigaut‐Jalabert F., et al. 2020. “Diversity and Dynamics of Relevant Nanoplanktonic Diatoms in the Western English Channel.” ISME Journal
14, no. 8: 1966–1981. 10.1038/s41396-020-0659-6.
</cite> [<a href="https://doi.org/10.1038/s41396-020-0659-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7367886/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32350410/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20Journal&amp;title=Diversity%20and%20Dynamics%20of%20Relevant%20Nanoplanktonic%20Diatoms%20in%20the%20Western%20English%20Channel&amp;author=L.%20Arsenieff&amp;author=F.%20Le%20Gall&amp;author=F.%20Rigaut%E2%80%90Jalabert&amp;volume=14&amp;issue=8&amp;publication_year=2020&amp;pages=1966-1981&amp;pmid=32350410&amp;doi=10.1038/s41396-020-0659-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0006">
<cite>

Bennewitz, B.
, Sharma M., Tannert F., and Klösgen R. B.. 2020. “Dual Targeting of TatA Points to a Chloroplast‐Like Tat Pathway in Plant Mitochondria.” Biochimica et Biophysica Acta, Molecular Cell Research
1867, no. 11: 118816. 10.1016/j.bbamcr.2020.118816.
</cite> [<a href="https://doi.org/10.1016/j.bbamcr.2020.118816" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32768405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta,%20Molecular%20Cell%20Research&amp;title=Dual%20Targeting%20of%20TatA%20Points%20to%20a%20Chloroplast%E2%80%90Like%20Tat%20Pathway%20in%20Plant%20Mitochondria&amp;author=B.%20Bennewitz&amp;author=M.%20Sharma&amp;author=F.%20Tannert&amp;author=R.%20B.%20Kl%C3%B6sgen&amp;volume=1867&amp;issue=11&amp;publication_year=2020&amp;pages=118816&amp;pmid=32768405&amp;doi=10.1016/j.bbamcr.2020.118816&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0007">
<cite>

Benson, G.

1999. “Tandem Repeats Finder: A Program to Analyze DNA Sequences.” Nucleic Acids Research
27, no. 2: 573–580. 10.1093/nar/27.2.573.
</cite> [<a href="https://doi.org/10.1093/nar/27.2.573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC148217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9862982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Tandem%20Repeats%20Finder:%20A%20Program%20to%20Analyze%20DNA%20Sequences&amp;author=G.%20Benson&amp;volume=27&amp;issue=2&amp;publication_year=1999&amp;pages=573-580&amp;pmid=9862982&amp;doi=10.1093/nar/27.2.573&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0008">
<cite>

Boehmke, B. C.

2016. “Reshaping Your Data With Tidyr.” In Data Wrangling With R, 211–218. Springer. 10.1007/978-3-319-45599-0_21.</cite> [<a href="https://doi.org/10.1007/978-3-319-45599-0_21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Data%20Wrangling%20With%20R&amp;author=B.%20C.%20Boehmke&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0009">
<cite>

Bolaños, L. M.
, Karp‐Boss L., Choi C. J., et al. 2020. “Small Phytoplankton Dominate Western North Atlantic Biomass.” ISME Journal
14, no. 7: 1663–1674. 10.1038/s41396-020-0636-0.
</cite> [<a href="https://doi.org/10.1038/s41396-020-0636-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7305139/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32231247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20Journal&amp;title=Small%20Phytoplankton%20Dominate%20Western%20North%20Atlantic%20Biomass&amp;author=L.%20M.%20Bola%C3%B1os&amp;author=L.%20Karp%E2%80%90Boss&amp;author=C.%20J.%20Choi&amp;volume=14&amp;issue=7&amp;publication_year=2020&amp;pages=1663-1674&amp;pmid=32231247&amp;doi=10.1038/s41396-020-0636-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0010">
<cite>

Bolyen, E.
, Rideout J. R., Dillon M. R., et al. 2019. “Reproducible, Interactive, Scalable and Extensible Microbiome Data Science Using QIIME 2.” Nature Biotechnology
37, no. 8: 852–857. 10.1038/s41587-019-0209-9.</cite> [<a href="https://doi.org/10.1038/s41587-019-0209-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7015180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31341288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Reproducible,%20Interactive,%20Scalable%20and%20Extensible%20Microbiome%20Data%20Science%20Using%20QIIME%202&amp;author=E.%20Bolyen&amp;author=J.%20R.%20Rideout&amp;author=M.%20R.%20Dillon&amp;volume=37&amp;issue=8&amp;publication_year=2019&amp;pages=852-857&amp;pmid=31341288&amp;doi=10.1038/s41587-019-0209-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0011">
<cite>

Caracciolo, M.
, Berney C., Alric B., et al. 2022. “Insight Into Diatoms Diversity at Two European Coastal Sites (LTER‐MC in the Mediterranean Sea and SOMLIT‐Astan in the Western English Channel) Using a DNA Metabarcoding Approach.”
<em>bioRxiv</em>. 10.1101/2022.07.01.498404.</cite> [<a href="https://doi.org/10.1101/2022.07.01.498404" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="jeu70033-bib-0012">
<cite>

Caracciolo, M.
, Rigaut‐Jalabert F., Romac S., et al. 2022. “Seasonal Dynamics of Marine Protist Communities in Tidally Mixed Coastal Waters.” Molecular Ecology
31, no. 14: 3761–3783. 10.1111/mec.16539.
</cite> [<a href="https://doi.org/10.1111/mec.16539" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9543310/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35593305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Seasonal%20Dynamics%20of%20Marine%20Protist%20Communities%20in%20Tidally%20Mixed%20Coastal%20Waters&amp;author=M.%20Caracciolo&amp;author=F.%20Rigaut%E2%80%90Jalabert&amp;author=S.%20Romac&amp;volume=31&amp;issue=14&amp;publication_year=2022&amp;pages=3761-3783&amp;pmid=35593305&amp;doi=10.1111/mec.16539&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0013"><cite>

Casault, B.
, Beazley L., Johnson C., Devred E., and Head E.. 2024. “Chemical and Biological Oceanographic Conditions on the Scotian Shelf and in the Eastern Gulf of Maine During 2022.”
<em>Canadian Technical Report of Fisheries and Aquatic Sciences</em> 3589, vi + 72 pp.</cite></li>
<li id="jeu70033-bib-0014">
<cite>

Chan, P. P.
, and Lowe T. M.. 2019. “tRNA‐Scan‐SE: Searching for tRNA Genes in Genomic Sequences.” Methods in Molecular Biology
1962: 1–14. 10.1007/978-1-4939-9173-0_1.
</cite> [<a href="https://doi.org/10.1007/978-1-4939-9173-0_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6768409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31020551/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20in%20Molecular%20Biology&amp;title=tRNA%E2%80%90Scan%E2%80%90SE:%20Searching%20for%20tRNA%20Genes%20in%20Genomic%20Sequences&amp;author=P.%20P.%20Chan&amp;author=T.%20M.%20Lowe&amp;volume=1962&amp;publication_year=2019&amp;pages=1-14&amp;pmid=31020551&amp;doi=10.1007/978-1-4939-9173-0_1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0015">
<cite>*
Chung, M. H.
, Yoon W. D., and Lee J.‐B.. 2010. “Morphological Description of <em>Cyclotella atomus</em> var. <em>marina</em> (Bacillariophyceae): Newly Reported in Korean Waters.” Algae The Korean Phycological Society 25, no. 2: 57–64. 10.4490/algae.2010.25.2.057.</cite> [<a href="https://doi.org/10.4490/algae.2010.25.2.057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Algae&amp;title=Morphological%20Description%20of%20Cyclotella%20atomus%20var.%20marina%20(Bacillariophyceae):%20Newly%20Reported%20in%20Korean%20Waters&amp;author=M.%20H.%20Chung&amp;author=W.%20D.%20Yoon&amp;author=J.%E2%80%90B.%20Lee&amp;volume=25&amp;issue=2&amp;publication_year=2010&amp;pages=57-64&amp;doi=10.4490/algae.2010.25.2.057&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0016">
<cite>

Clark, K.
, Karsch‐Mizrachi I., Lipman D. J., Ostell J., and Sayers E. W.. 2016. “GenBank.” Nucleic Acids Research
44, no. D1: D67–D72. 10.1093/nar/gkv1276.
</cite> [<a href="https://doi.org/10.1093/nar/gkv1276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4702903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26590407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=GenBank&amp;author=K.%20Clark&amp;author=I.%20Karsch%E2%80%90Mizrachi&amp;author=D.%20J.%20Lipman&amp;author=J.%20Ostell&amp;author=E.%20W.%20Sayers&amp;volume=44&amp;issue=D1&amp;publication_year=2016&amp;pages=D67-D72&amp;pmid=26590407&amp;doi=10.1093/nar/gkv1276&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0017">
<cite>

Cline, K.
, and Theg S. M.. 2007. “The Sec and Tat Protein Translocation Pathways in Chloroplasts.” In The Enzymes, vol. 25, chap. 18, 463–492. Elsevier. 10.1016/S1874-6047(07)25018-8.</cite> [<a href="https://doi.org/10.1016/S1874-6047(07)25018-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=The%20Enzymes&amp;author=K.%20Cline&amp;author=S.%20M.%20Theg&amp;publication_year=2007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0018">
<cite>

Comeau, A. M.
, Douglas G. M., and Langille M. G. I.. 2017. “Microbiome Helper: A Custom and Streamlined Workflow for Microbiome Research.” mSystems
2, no. 1: e00127‐16. 10.1128/mSystems.00127-16.</cite> [<a href="https://doi.org/10.1128/mSystems.00127-16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5209531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28066818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=Microbiome%20Helper:%20A%20Custom%20and%20Streamlined%20Workflow%20for%20Microbiome%20Research&amp;author=A.%20M.%20Comeau&amp;author=G.%20M.%20Douglas&amp;author=M.%20G.%20I.%20Langille&amp;volume=2&amp;issue=1&amp;publication_year=2017&amp;pages=e00127%E2%80%9016&amp;pmid=28066818&amp;doi=10.1128/mSystems.00127-16&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0019">
<cite>

Comeau, A. M.
, Li W. K. W., Tremblay J.‐É., Carmack E. C., and Lovejoy C.. 2011. “Arctic Ocean Microbial Community Structure Before and After the 2007 Record Sea Ice Minimum.” PLoS One
6, no. 11: e27492. 10.1371/journal.pone.0027492.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0027492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3212577/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22096583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Arctic%20Ocean%20Microbial%20Community%20Structure%20Before%20and%20After%20the%202007%20Record%20Sea%20Ice%20Minimum&amp;author=A.%20M.%20Comeau&amp;author=W.%20K.%20W.%20Li&amp;author=J.%E2%80%90%C3%89.%20Tremblay&amp;author=E.%20C.%20Carmack&amp;author=C.%20Lovejoy&amp;volume=6&amp;issue=11&amp;publication_year=2011&amp;pages=e27492&amp;pmid=22096583&amp;doi=10.1371/journal.pone.0027492&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0020">
<cite>

Cox, E.
, Tsuchiya M. T. N., Ciufo S., et al. 2024. “NCBI Taxonomy: Enhanced Access via NCBI Datasets.” Nucleic Acids Research
53, no. D1: D1711–D1715. 10.1093/nar/gkae967.</cite> [<a href="https://doi.org/10.1093/nar/gkae967" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11701650/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39470745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=NCBI%20Taxonomy:%20Enhanced%20Access%20via%20NCBI%20Datasets&amp;author=E.%20Cox&amp;author=M.%20T.%20N.%20Tsuchiya&amp;author=S.%20Ciufo&amp;volume=53&amp;issue=D1&amp;publication_year=2024&amp;pages=D1711-D1715&amp;pmid=39470745&amp;doi=10.1093/nar/gkae967&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0021">
<cite>

Cullen, J.
, Doolittle W. F., Levin S., and Li W. K. W.. 2007. “Patterns and Prediction in Microbial Oceanography.” Oceanography
20, no. 2: 34–46. 10.5670/oceanog.2007.46.</cite> [<a href="https://doi.org/10.5670/oceanog.2007.46" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oceanography&amp;title=Patterns%20and%20Prediction%20in%20Microbial%20Oceanography&amp;author=J.%20Cullen&amp;author=W.%20F.%20Doolittle&amp;author=S.%20Levin&amp;author=W.%20K.%20W.%20Li&amp;volume=20&amp;issue=2&amp;publication_year=2007&amp;pages=34-46&amp;doi=10.5670/oceanog.2007.46&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0022">
<cite>

Daniels, C. J.
, Poulton A. J., Esposito M., et al. 2015. “Phytoplankton Dynamics in Contrasting Early Stage North Atlantic Spring Blooms: Composition, Succession, and Potential Drivers.” Biogeosciences
12, no. 8: 2395–2409. 10.5194/bg-12-2395-2015.</cite> [<a href="https://doi.org/10.5194/bg-12-2395-2015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biogeosciences&amp;title=Phytoplankton%20Dynamics%20in%20Contrasting%20Early%20Stage%20North%20Atlantic%20Spring%20Blooms:%20Composition,%20Succession,%20and%20Potential%20Drivers&amp;author=C.%20J.%20Daniels&amp;author=A.%20J.%20Poulton&amp;author=M.%20Esposito&amp;volume=12&amp;issue=8&amp;publication_year=2015&amp;pages=2395-2409&amp;doi=10.5194/bg-12-2395-2015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0023">
<cite>

Decelle, J.
, Romac S., Stern R. F., et al. 2015. “PhytoREF: A Reference Database of the Plastidial 16S rRNA Gene of Photosynthetic Eukaryotes With Curated Taxonomy.” Molecular Ecology Resources
15, no. 6: 1435–1445. 10.1111/1755-0998.12401.
</cite> [<a href="https://doi.org/10.1111/1755-0998.12401" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25740460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Resources&amp;title=PhytoREF:%20A%20Reference%20Database%20of%20the%20Plastidial%2016S%20rRNA%20Gene%20of%20Photosynthetic%20Eukaryotes%20With%20Curated%20Taxonomy&amp;author=J.%20Decelle&amp;author=S.%20Romac&amp;author=R.%20F.%20Stern&amp;volume=15&amp;issue=6&amp;publication_year=2015&amp;pages=1435-1445&amp;pmid=25740460&amp;doi=10.1111/1755-0998.12401&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0024">
<cite>

Dixon, P.

2003. “VEGAN, a Package of R Functions for Community Ecology.” Journal of Vegetation Science
14, no. 6: 927–930. 10.1111/j.1654-1103.2003.tb02228.x.</cite> [<a href="https://doi.org/10.1111/j.1654-1103.2003.tb02228.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Vegetation%20Science&amp;title=VEGAN,%20a%20Package%20of%20R%20Functions%20for%20Community%20Ecology&amp;author=P.%20Dixon&amp;volume=14&amp;issue=6&amp;publication_year=2003&amp;pages=927-930&amp;doi=10.1111/j.1654-1103.2003.tb02228.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0025">
<cite>*
Duleba, M.
, Ector L., Horváth Z., et al. 2014. “Biogeography and Phylogenetic Position of a Warm‐Stenotherm Centric Diatom, 
<em>Skeletonema potamos</em>
 (C.I. Weber) Hasle and Its Long‐Term Dynamics in the River Danube.” Protist
165, no. 5: 715–729. 10.1016/j.protis.2014.08.001.
</cite> [<a href="https://doi.org/10.1016/j.protis.2014.08.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25250953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protist&amp;title=Biogeography%20and%20Phylogenetic%20Position%20of%20a%20Warm%E2%80%90Stenotherm%20Centric%20Diatom,%20Skeletonema%20potamos%20(C.I.%20Weber)%20Hasle%20and%20Its%20Long%E2%80%90Term%20Dynamics%20in%20the%20River%20Danube&amp;author=M.%20Duleba&amp;author=L.%20Ector&amp;author=Z.%20Horv%C3%A1th&amp;volume=165&amp;issue=5&amp;publication_year=2014&amp;pages=715-729&amp;pmid=25250953&amp;doi=10.1016/j.protis.2014.08.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0026">
<cite>

Edgar, R. C.

2004. “MUSCLE: Multiple Sequence Alignment With High Accuracy and High Throughput.” Nucleic Acids Research
32, no. 5: 1792–1797. 10.1093/nar/gkh340.
</cite> [<a href="https://doi.org/10.1093/nar/gkh340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC390337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15034147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=MUSCLE:%20Multiple%20Sequence%20Alignment%20With%20High%20Accuracy%20and%20High%20Throughput&amp;author=R.%20C.%20Edgar&amp;volume=32&amp;issue=5&amp;publication_year=2004&amp;pages=1792-1797&amp;pmid=15034147&amp;doi=10.1093/nar/gkh340&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0027">
<cite>

Eren, A. M.
, Esen Ö. C., Quince C., et al. 2015. “Anvi'o: An Advanced Analysis and Visualization Platform for 'Omics Data.” PeerJ
3: e1319. 10.7717/peerj.1319.
</cite> [<a href="https://doi.org/10.7717/peerj.1319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4614810/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26500826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Anvi'o:%20An%20Advanced%20Analysis%20and%20Visualization%20Platform%20for%20'Omics%20Data&amp;author=A.%20M.%20Eren&amp;author=%C3%96.%20C.%20Esen&amp;author=C.%20Quince&amp;volume=3&amp;publication_year=2015&amp;pages=e1319&amp;pmid=26500826&amp;doi=10.7717/peerj.1319&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0028">
<cite>

Falkowski, P. G.
, Barber R. T., and Smetacek V.. 1998. “Biogeochemical Controls and Feedbacks on Ocean Primary Production.” Science
281, no. 5374: 200–207. 10.1126/science.281.5374.200.
</cite> [<a href="https://doi.org/10.1126/science.281.5374.200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9660741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Biogeochemical%20Controls%20and%20Feedbacks%20on%20Ocean%20Primary%20Production&amp;author=P.%20G.%20Falkowski&amp;author=R.%20T.%20Barber&amp;author=V.%20Smetacek&amp;volume=281&amp;issue=5374&amp;publication_year=1998&amp;pages=200-207&amp;pmid=9660741&amp;doi=10.1126/science.281.5374.200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0029">
<cite>

Fernandes, L. F.
, and Correr‐Da‐Silva F.. 2020. “Diversity and Distribution of Nanoplanktonic <em>Minidiscus</em> (Bacillariophyta) in Shelf Waters of the Southwest Atlantic Ocean.” Diatom Research
35, no. 4: 327–338. 10.1080/0269249x.2020.1845807.</cite> [<a href="https://doi.org/10.1080/0269249x.2020.1845807" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diatom%20Research&amp;title=Diversity%20and%20Distribution%20of%20Nanoplanktonic%20Minidiscus%20(Bacillariophyta)%20in%20Shelf%20Waters%20of%20the%20Southwest%20Atlantic%20Ocean&amp;author=L.%20F.%20Fernandes&amp;author=F.%20Correr%E2%80%90Da%E2%80%90Silva&amp;volume=35&amp;issue=4&amp;publication_year=2020&amp;pages=327-338&amp;doi=10.1080/0269249x.2020.1845807&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0030">
<cite>

Finkel, Z. V.
, Beardall J., Flynn K. J., Quigg A., Rees T. A. V., and Raven J. A.. 2010. “Phytoplankton in a Changing World: Cell Size and Elemental Stoichiometry.” Journal of Plankton Research
32, no. 1: 119–137. 10.1093/plankt/fbp098.</cite> [<a href="https://doi.org/10.1093/plankt/fbp098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Plankton%20Research&amp;title=Phytoplankton%20in%20a%20Changing%20World:%20Cell%20Size%20and%20Elemental%20Stoichiometry&amp;author=Z.%20V.%20Finkel&amp;author=J.%20Beardall&amp;author=K.%20J.%20Flynn&amp;author=A.%20Quigg&amp;author=T.%20A.%20V.%20Rees&amp;volume=32&amp;issue=1&amp;publication_year=2010&amp;pages=119-137&amp;doi=10.1093/plankt/fbp098&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0031">
<cite>

Fontaine, D. N.
, and Rynearson T. A.. 2023. “Multi‐Year Time Series Reveals Temporally Synchronous Diatom Communities With Annual Frequency of Recurrence in a Temperate Estuary.” Limnology and Oceanography
68, no. 9: 1982–1994. 10.1002/lno.12400.</cite> [<a href="https://doi.org/10.1002/lno.12400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Limnology%20and%20Oceanography&amp;title=Multi%E2%80%90Year%20Time%20Series%20Reveals%20Temporally%20Synchronous%20Diatom%20Communities%20With%20Annual%20Frequency%20of%20Recurrence%20in%20a%20Temperate%20Estuary&amp;author=D.%20N.%20Fontaine&amp;author=T.%20A.%20Rynearson&amp;volume=68&amp;issue=9&amp;publication_year=2023&amp;pages=1982-1994&amp;doi=10.1002/lno.12400&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0032">
<cite>*
Genkal, S.

2015. “Morphological Variability, Taxonomy and Ecology of <em>Discostella pseudostelligera</em> (Bacillariophyceae, Centrales) and Similar Species.” Nova Hedwigia
101, no. 3–4: 427–449. 10.1127/nova_hedwigia/2015/0278.</cite> [<a href="https://doi.org/10.1127/nova_hedwigia/2015/0278" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nova%20Hedwigia&amp;title=Morphological%20Variability,%20Taxonomy%20and%20Ecology%20of%20Discostella%20pseudostelligera%20(Bacillariophyceae,%20Centrales)%20and%20Similar%20Species&amp;author=S.%20Genkal&amp;volume=101&amp;issue=3%E2%80%934&amp;publication_year=2015&amp;pages=427-449&amp;doi=10.1127/nova_hedwigia/2015/0278&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0033">
<cite>

Grant, J. R.
, Enns E., Marinier E., et al. 2023. “Proksee: In‐Depth Characterization and Visualization of Bacterial Genomes.” Nucleic Acids Research
51, no. W1: W484–W492. 10.1093/nar/gkad326.
</cite> [<a href="https://doi.org/10.1093/nar/gkad326" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10320063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37140037/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Proksee:%20In%E2%80%90Depth%20Characterization%20and%20Visualization%20of%20Bacterial%20Genomes&amp;author=J.%20R.%20Grant&amp;author=E.%20Enns&amp;author=E.%20Marinier&amp;volume=51&amp;issue=W1&amp;publication_year=2023&amp;pages=W484-W492&amp;pmid=37140037&amp;doi=10.1093/nar/gkad326&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0034">
<cite>

Greiner, S.
, Lehwark P., and Bock R.. 2019. “OrganellarGenomeDRAW (OGDRAW) Version 1.3.1: Expanded Toolkit for the Graphical Visualization of Organellar Genomes.” Nucleic Acids Research
47, no. W1: W59–W64. 10.1093/nar/gkz238.
</cite> [<a href="https://doi.org/10.1093/nar/gkz238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6602502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30949694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=OrganellarGenomeDRAW%20(OGDRAW)%20Version%201.3.1:%20Expanded%20Toolkit%20for%20the%20Graphical%20Visualization%20of%20Organellar%20Genomes&amp;author=S.%20Greiner&amp;author=P.%20Lehwark&amp;author=R.%20Bock&amp;volume=47&amp;issue=W1&amp;publication_year=2019&amp;pages=W59-W64&amp;pmid=30949694&amp;doi=10.1093/nar/gkz238&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0035">
<cite>*
Guerrero, J. M.
, and Echenique R. O.. 2006. “
<em>Discostella</em> Taxa (Bacillariophyta) From the Río Limay Basin (Northwestern Patagonia, Argentina).” European Journal of Phycology
41: 83–96. 10.1080/09670260500512363.</cite> [<a href="https://doi.org/10.1080/09670260500512363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Phycology&amp;title=Discostella%20Taxa%20(Bacillariophyta)%20From%20the%20R%C3%ADo%20Limay%20Basin%20(Northwestern%20Patagonia,%20Argentina)&amp;author=J.%20M.%20Guerrero&amp;author=R.%20O.%20Echenique&amp;volume=41&amp;publication_year=2006&amp;pages=83-96&amp;doi=10.1080/09670260500512363&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0036">
<cite>

Guidi, L.
, Chaffron S., Bittner L., et al. 2016. “Plankton Networks Driving Carbon Export in the Oligotrophic Ocean.” Nature
532, no. 7600: 465–470. 10.1038/nature16942.
</cite> [<a href="https://doi.org/10.1038/nature16942" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4851848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26863193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Plankton%20Networks%20Driving%20Carbon%20Export%20in%20the%20Oligotrophic%20Ocean&amp;author=L.%20Guidi&amp;author=S.%20Chaffron&amp;author=L.%20Bittner&amp;volume=532&amp;issue=7600&amp;publication_year=2016&amp;pages=465-470&amp;pmid=26863193&amp;doi=10.1038/nature16942&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0037">
<cite>

Guindon, S.
, Dufayard J.‐F., Lefort V., Anisimova M., Hordijk W., and Gascuel O.. 2010. “New Algorithms and Methods to Estimate Maximum‐Likelihood Phylogenies: Assessing the Performance of PhyML 3.0.” Systematic Biology
59, no. 3: 307–321. 10.1093/sysbio/syq010.
</cite> [<a href="https://doi.org/10.1093/sysbio/syq010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20525638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Systematic%20Biology&amp;title=New%20Algorithms%20and%20Methods%20to%20Estimate%20Maximum%E2%80%90Likelihood%20Phylogenies:%20Assessing%20the%20Performance%20of%20PhyML%203.0&amp;author=S.%20Guindon&amp;author=J.%E2%80%90F.%20Dufayard&amp;author=V.%20Lefort&amp;author=M.%20Anisimova&amp;author=W.%20Hordijk&amp;volume=59&amp;issue=3&amp;publication_year=2010&amp;pages=307-321&amp;pmid=20525638&amp;doi=10.1093/sysbio/syq010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0038">
<cite>

Guindon, S.
, Lethiec F., Duroux P., and Gascuel O.. 2005. “PHYML Online—A Web Server for Fast Maximum Likelihood‐Based Phylogenetic Inference.” Nucleic Acids Research
33: W557–W559. 10.1093/nar/gki352.
</cite> [<a href="https://doi.org/10.1093/nar/gki352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1160113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15980534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=PHYML%20Online%E2%80%94A%20Web%20Server%20for%20Fast%20Maximum%20Likelihood%E2%80%90Based%20Phylogenetic%20Inference&amp;author=S.%20Guindon&amp;author=F.%20Lethiec&amp;author=P.%20Duroux&amp;author=O.%20Gascuel&amp;volume=33&amp;publication_year=2005&amp;pages=W557-W559&amp;pmid=15980534&amp;doi=10.1093/nar/gki352&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0039">
<cite>

Guiry, M. D.

2012. “How Many Species of Algae Are There?” Journal of Phycology
48, no. 5: 1057–1063. 10.1111/j.1529-8817.2012.01222.x.
</cite> [<a href="https://doi.org/10.1111/j.1529-8817.2012.01222.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27011267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Phycology&amp;title=How%20Many%20Species%20of%20Algae%20Are%20There?&amp;author=M.%20D.%20Guiry&amp;volume=48&amp;issue=5&amp;publication_year=2012&amp;pages=1057-1063&amp;pmid=27011267&amp;doi=10.1111/j.1529-8817.2012.01222.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0040"><cite>

Guiry, M. D.
, and Guiry G. M.. 2015. “Family Thalassiosiraceae.”
<em>Algaebase</em>. October 13, 2015. <a href="https://www.algaebase.org/browse/taxonomy/detail/?taxonid=77608" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.algaebase.org/browse/taxonomy/detail/?taxonid=77608</a>.</cite></li>
<li id="jeu70033-bib-0041"><cite>

Guiry, M. D.
, and Guiry G. M.. 2022. “<em>Mediolabrus</em> Yang Li, 2020.”
<em>Algaebase</em>. June 29, 2022. <a href="https://www.algaebase.org/search/genus/detail/?genus_id=53007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.algaebase.org/search/genus/detail/?genus_id=53007</a>.</cite></li>
<li id="jeu70033-bib-0042"><cite>

Guiry, M. D.
, and Guiry G. M.. 2024. “<em>Minidiscus</em> Hasle, 1973.”
<em>Algaebase</em>. February 5, 2024. <a href="https://www.algaebase.org/search/genus/detail/?genus_id=44422" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.algaebase.org/search/genus/detail/?genus_id=44422</a>.</cite></li>
<li id="jeu70033-bib-0043">
<cite>

Haider, D.
, Tolman J., Beiko R. G., and LaRoche J.. 2025. “Comparison of Spatio‐Temporal Dynamics and Composition in Size‐Fractionated and Unfractionated Northwestern Atlantic Microbial Communities.”
<em>bioRxiv</em>. 10.1101/2025.04.03.647080.</cite> [<a href="https://doi.org/10.1101/2025.04.03.647080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="jeu70033-bib-0044">
<cite>*
Håkansson, H.

2002. “A Compilation and Evaluation of Species in the Genera <em>Stephanodiscus</em>, <em>Cyclostephanos</em>, and <em>Cyclotella</em>, With a New Genus in the Family Stephanodiscaceae.” Diatom Research
17: 1–139. 10.1080/0269249X.2002.9705534.</cite> [<a href="https://doi.org/10.1080/0269249X.2002.9705534" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diatom%20Research&amp;title=A%20Compilation%20and%20Evaluation%20of%20Species%20in%20the%20Genera%20Stephanodiscus,%20Cyclostephanos,%20and%20Cyclotella,%20With%20a%20New%20Genus%20in%20the%20Family%20Stephanodiscaceae&amp;author=H.%20H%C3%A5kansson&amp;volume=17&amp;publication_year=2002&amp;pages=1-139&amp;doi=10.1080/0269249X.2002.9705534&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0045">
<cite>

Hasle, G. R.

1973. “Thalassiosiraceae, a New Diatom Family.” Norwegian Journal of Botany
20: 67–69.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Norwegian%20Journal%20of%20Botany&amp;title=Thalassiosiraceae,%20a%20New%20Diatom%20Family&amp;author=G.%20R.%20Hasle&amp;volume=20&amp;publication_year=1973&amp;pages=67-69&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0046">
<cite>*
Hasle, G. R.
, and Syvertsen E. E.. 1997. “Chapter 2 – Marine Diatoms.” In Identifying Marine Phytoplankton, 5–385. Elsevier. 10.1016/b978-012693018-4/50004-5.</cite> [<a href="https://doi.org/10.1016/b978-012693018-4/50004-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Identifying%20Marine%20Phytoplankton&amp;author=G.%20R.%20Hasle&amp;author=E.%20E.%20Syvertsen&amp;publication_year=1997&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0047">
<cite>

Hernández‐Salmerón, J. E.
, and Moreno‐Hagelsieb G.. 2022. “FastANI, Mash and Dashing Equally Differentiate Between Species.” PeerJ
10, no. July: e13784. 10.7717/peerj.13784.
</cite> [<a href="https://doi.org/10.7717/peerj.13784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9308963/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35891643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=FastANI,%20Mash%20and%20Dashing%20Equally%20Differentiate%20Between%20Species&amp;author=J.%20E.%20Hern%C3%A1ndez%E2%80%90Salmer%C3%B3n&amp;author=G.%20Moreno%E2%80%90Hagelsieb&amp;volume=10&amp;issue=July&amp;publication_year=2022&amp;pages=e13784&amp;pmid=35891643&amp;doi=10.7717/peerj.13784&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0048">
<cite>*
Houk, V.
, and Klee R.. 2004. “The <em>Stelligeroid</em> Taxa of the Genus <em>Cyclotella</em> (Kützing) Brébisson (Bacillariophyceae) and Their Transfer Into the New Genus <em>Discostella</em> gen. nov.” Diatom Research
19: 203–228.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Diatom%20Research&amp;title=The%20Stelligeroid%20Taxa%20of%20the%20Genus%20Cyclotella%20(K%C3%BCtzing)%20Br%C3%A9bisson%20(Bacillariophyceae)%20and%20Their%20Transfer%20Into%20the%20New%20Genus%20Discostella%20gen.%20nov.&amp;author=V.%20Houk&amp;author=R.%20Klee&amp;volume=19&amp;publication_year=2004&amp;pages=203-228&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0049">
<cite>

Huelsenbeck, J. P.
, and Ronquist F.. 2001. “MRBAYES: Bayesian Inference of Phylogenetic Trees.” Bioinformatics
17, no. 8: 754–755. 10.1093/bioinformatics/17.8.754.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/17.8.754" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11524383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MRBAYES:%20Bayesian%20Inference%20of%20Phylogenetic%20Trees&amp;author=J.%20P.%20Huelsenbeck&amp;author=F.%20Ronquist&amp;volume=17&amp;issue=8&amp;publication_year=2001&amp;pages=754-755&amp;pmid=11524383&amp;doi=10.1093/bioinformatics/17.8.754&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0050">
<cite>

Jeon, M.
, Iriarte J. L., Yang E. J., Park J., Alves‐de‐Souza C., and Park S. J.. 2023. “Importance of Nanophytoplankton Biomass During Summer 2019 in a Retreating Marine‐Terminating Glacier‐Fjord System, Marian Cove, West Antarctica (62° S).” Frontiers in Marine Science
10. 10.3389/fmars.2023.1174722.</cite> [<a href="https://doi.org/10.3389/fmars.2023.1174722" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Marine%20Science&amp;title=Importance%20of%20Nanophytoplankton%20Biomass%20During%20Summer%202019%20in%20a%20Retreating%20Marine%E2%80%90Terminating%20Glacier%E2%80%90Fjord%20System,%20Marian%20Cove,%20West%20Antarctica%20(62%C2%B0%E2%80%89S)&amp;author=M.%20Jeon&amp;author=J.%20L.%20Iriarte&amp;author=E.%20J.%20Yang&amp;author=J.%20Park&amp;author=C.%20Alves%E2%80%90de%E2%80%90Souza&amp;volume=10&amp;publication_year=2023&amp;doi=10.3389/fmars.2023.1174722&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0051">
<cite>

Jeong, Y.
, and Lee J.. 2024. “Comparative Analysis of Organelle Genomes Provides Conflicting Evidence Between Morphological Similarity and Phylogenetic Relationship in Diatoms.” Frontiers in Marine Science
10. 10.3389/fmars.2023.1283893.</cite> [<a href="https://doi.org/10.3389/fmars.2023.1283893" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Marine%20Science&amp;title=Comparative%20Analysis%20of%20Organelle%20Genomes%20Provides%20Conflicting%20Evidence%20Between%20Morphological%20Similarity%20and%20Phylogenetic%20Relationship%20in%20Diatoms&amp;author=Y.%20Jeong&amp;author=J.%20Lee&amp;volume=10&amp;publication_year=2024&amp;doi=10.3389/fmars.2023.1283893&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0052">
<cite>

Jin, J.‐J.
, Yu W.‐B., Yang J.‐B., et al. 2020. “GetOrganelle: A Fast and Versatile Toolkit for Accurate de Novo Assembly of Organelle Genomes.” Genome Biology
21, no. 1: 241. 10.1186/s13059-020-02154-5.
</cite> [<a href="https://doi.org/10.1186/s13059-020-02154-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7488116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32912315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=GetOrganelle:%20A%20Fast%20and%20Versatile%20Toolkit%20for%20Accurate%20de%20Novo%20Assembly%20of%20Organelle%20Genomes&amp;author=J.%E2%80%90J.%20Jin&amp;author=W.%E2%80%90B.%20Yu&amp;author=J.%E2%80%90B.%20Yang&amp;volume=21&amp;issue=1&amp;publication_year=2020&amp;pages=241&amp;pmid=32912315&amp;doi=10.1186/s13059-020-02154-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0053">
<cite>

Kaczmarska, I.
, Lovejoy C., Potvin M., and Macgillivary M.. 2009. “Morphological and Molecular Characteristics of Selected Species of <em>Minidiscus</em> (Bacillariophyta, Thalassiosiraceae).” European Journal of Phycology
44, no. 4: 461–475. 10.1080/09670260902855873.</cite> [<a href="https://doi.org/10.1080/09670260902855873" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Phycology&amp;title=Morphological%20and%20Molecular%20Characteristics%20of%20Selected%20Species%20of%20Minidiscus%20(Bacillariophyta,%20Thalassiosiraceae)&amp;author=I.%20Kaczmarska&amp;author=C.%20Lovejoy&amp;author=M.%20Potvin&amp;author=M.%20Macgillivary&amp;volume=44&amp;issue=4&amp;publication_year=2009&amp;pages=461-475&amp;doi=10.1080/09670260902855873&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0054">
<cite>

Kalyaanamoorthy, S.
, Minh B. Q., Wong T. K. F., von Haeseler A., and Jermiin L. S.. 2017. “ModelFinder: Fast Model Selection for Accurate Phylogenetic Estimates.” Nature Methods
14, no. 6: 587–589. 10.1038/nmeth.4285.
</cite> [<a href="https://doi.org/10.1038/nmeth.4285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5453245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28481363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=ModelFinder:%20Fast%20Model%20Selection%20for%20Accurate%20Phylogenetic%20Estimates&amp;author=S.%20Kalyaanamoorthy&amp;author=B.%20Q.%20Minh&amp;author=T.%20K.%20F.%20Wong&amp;author=A.%20von%20Haeseler&amp;author=L.%20S.%20Jermiin&amp;volume=14&amp;issue=6&amp;publication_year=2017&amp;pages=587-589&amp;pmid=28481363&amp;doi=10.1038/nmeth.4285&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0055">
<cite>*
Katrantsiotis, C.
, Risberg J., Norström E., and Holmgren K.. 2016. “Morphological Study of <em>Cyclotella Distinguenda</em> With a Description of a New Fossil Species <em>Cyclotella Paradistinguenda</em> sp. nov. From the Agios Floros Fen, SW Peloponnese, Greece in Relation to Other <em>Cyclotella</em> Species.” Diatom Research
31, no. 3: 243–267. 10.1080/0269249X.2016.1211178.</cite> [<a href="https://doi.org/10.1080/0269249X.2016.1211178" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diatom%20Research&amp;title=Morphological%20Study%20of%20Cyclotella%20Distinguenda%20With%20a%20Description%20of%20a%20New%20Fossil%20Species%20Cyclotella%20Paradistinguenda%20sp.%20nov.%20From%20the%20Agios%20Floros%20Fen,%20SW%20Peloponnese,%20Greece%20in%20Relation%20to%20Other%20Cyclotella%20Species&amp;author=C.%20Katrantsiotis&amp;author=J.%20Risberg&amp;author=E.%20Norstr%C3%B6m&amp;author=K.%20Holmgren&amp;volume=31&amp;issue=3&amp;publication_year=2016&amp;pages=243-267&amp;doi=10.1080/0269249X.2016.1211178&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0056">
<cite>*
Kiss, K. T.
, Iserentant R., Ács È., and Ector L.. 2002. “
<em>Thalassiosira</em> Gessneri Hustedt and <em>T. lacustris</em> (Grunow) Hasle in the Rivers Moselle (Luxembourg), Rhône, Saône (France), Danube (Hungary) and the Channel Main‐Danube (Germany).” Algological Studies
107: 17–37. 10.1127/algol_stud/107/2002/17.</cite> [<a href="https://doi.org/10.1127/algol_stud/107/2002/17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Algological%20Studies&amp;title=Thalassiosira%20Gessneri%20Hustedt%20and%20T.%20lacustris%20(Grunow)%20Hasle%20in%20the%20Rivers%20Moselle%20(Luxembourg),%20Rh%C3%B4ne,%20Sa%C3%B4ne%20(France),%20Danube%20(Hungary)%20and%20the%20Channel%20Main%E2%80%90Danube%20(Germany)&amp;author=K.%20T.%20Kiss&amp;author=R.%20Iserentant&amp;author=%C3%88.%20%C3%81cs&amp;author=L.%20Ector&amp;volume=107&amp;publication_year=2002&amp;pages=17-37&amp;doi=10.1127/algol_stud/107/2002/17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0057"><cite>

Krumhansl, K.
, Brooks C., Kingsbury S., and Dibacco C.. 2025. “<em>Metabarcoding of Benthic Marine Communities in Nova Scotia and New Brunswick</em>.” Version 1.1. Fisheries and Oceans Canada.</cite></li>
<li id="jeu70033-bib-0058">
<cite>*
Kulikovskiy, M.
, Genkal S., Maltsev Y., et al. 2022. “Resurrection of the Diatom Genus <em>Stephanocyclus</em> (Coscinodiscophyceae: Stephanodiscaceae) on the Basis of an Integrated Molecular and Morphological Approach.” Fottea
22, no. 2: 181–191. 10.5507/fot.2021.025.</cite> [<a href="https://doi.org/10.5507/fot.2021.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Fottea&amp;title=Resurrection%20of%20the%20Diatom%20Genus%20Stephanocyclus%20(Coscinodiscophyceae:%20Stephanodiscaceae)%20on%20the%20Basis%20of%20an%20Integrated%20Molecular%20and%20Morphological%20Approach&amp;author=M.%20Kulikovskiy&amp;author=S.%20Genkal&amp;author=Y.%20Maltsev&amp;volume=22&amp;issue=2&amp;publication_year=2022&amp;pages=181-191&amp;doi=10.5507/fot.2021.025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0059">
<cite>

Lang, B. F.
, Beck N., Prince S., Sarrasin M., Rioux P., and Burger G.. 2023. “Mitochondrial Genome Annotation With MFannot: A Critical Analysis of Gene Identification and Gene Model Prediction.” Frontiers in Plant Science
14, no. July: 1222186. 10.3389/fpls.2023.1222186.
</cite> [<a href="https://doi.org/10.3389/fpls.2023.1222186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10352661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37469769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Mitochondrial%20Genome%20Annotation%20With%20MFannot:%20A%20Critical%20Analysis%20of%20Gene%20Identification%20and%20Gene%20Model%20Prediction&amp;author=B.%20F.%20Lang&amp;author=N.%20Beck&amp;author=S.%20Prince&amp;author=M.%20Sarrasin&amp;author=P.%20Rioux&amp;volume=14&amp;issue=July&amp;publication_year=2023&amp;pages=1222186&amp;pmid=37469769&amp;doi=10.3389/fpls.2023.1222186&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0060">
<cite>

Laslett, D.
, and Canback B.. 2004. “ARAGORN, a Program to Detect tRNA Genes and tmRNA Genes in Nucleotide Sequences.” Nucleic Acids Research
32, no. 1: 11–16. 10.1093/nar/gkh152.
</cite> [<a href="https://doi.org/10.1093/nar/gkh152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC373265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14704338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=ARAGORN,%20a%20Program%20to%20Detect%20tRNA%20Genes%20and%20tmRNA%20Genes%20in%20Nucleotide%20Sequences&amp;author=D.%20Laslett&amp;author=B.%20Canback&amp;volume=32&amp;issue=1&amp;publication_year=2004&amp;pages=11-16&amp;pmid=14704338&amp;doi=10.1093/nar/gkh152&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0061">
<cite>*
Leblanc, K.
, Arístegui J., Armand L., et al. 2012. “A Global Diatom Database – Abundance, Biovolume and Biomass in the World Ocean.” Earth System Science Data
4, no. 1: 149–165. 10.5194/essd-4-149-2012.</cite> [<a href="https://doi.org/10.5194/essd-4-149-2012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Earth%20System%20Science%20Data&amp;title=A%20Global%20Diatom%20Database%20%E2%80%93%20Abundance,%20Biovolume%20and%20Biomass%20in%20the%20World%20Ocean&amp;author=K.%20Leblanc&amp;author=J.%20Ar%C3%ADstegui&amp;author=L.%20Armand&amp;volume=4&amp;issue=1&amp;publication_year=2012&amp;pages=149-165&amp;doi=10.5194/essd-4-149-2012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0062">
<cite>

Leblanc, K.
, Quéguiner B., Diaz F., et al. 2018. “Nanoplanktonic Diatoms Are Globally Overlooked but Play a Role in Spring Blooms and Carbon Export.” Nature Communications
9, no. 1: 953. 10.1038/s41467-018-03376-9.</cite> [<a href="https://doi.org/10.1038/s41467-018-03376-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5838239/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29507291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Nanoplanktonic%20Diatoms%20Are%20Globally%20Overlooked%20but%20Play%20a%20Role%20in%20Spring%20Blooms%20and%20Carbon%20Export&amp;author=K.%20Leblanc&amp;author=B.%20Qu%C3%A9guiner&amp;author=F.%20Diaz&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pages=953&amp;pmid=29507291&amp;doi=10.1038/s41467-018-03376-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0063">
<cite>

Lee, J. Y.
, Kong M., Oh J., et al. 2021. “Comparative Evaluation of Nanopore Polishing Tools for Microbial Genome Assembly and Polishing Strategies for Downstream Analysis.” Scientific Reports
11, no. 1: 20740. 10.1038/s41598-021-00178-w.
</cite> [<a href="https://doi.org/10.1038/s41598-021-00178-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8528807/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34671046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Comparative%20Evaluation%20of%20Nanopore%20Polishing%20Tools%20for%20Microbial%20Genome%20Assembly%20and%20Polishing%20Strategies%20for%20Downstream%20Analysis&amp;author=J.%20Y.%20Lee&amp;author=M.%20Kong&amp;author=J.%20Oh&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pages=20740&amp;pmid=34671046&amp;doi=10.1038/s41598-021-00178-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0064">
<cite>*
Lee, J.‐H.
, and Park J.‐S.. 2008. “A Study on the Fine Structure of the Marine Diatoms of Korean Coastal Waters – Genus <em>Thalassiosira</em> 3.” Algae
23, no. 3: 187–199. 10.4490/algae.2008.23.3.187.</cite> [<a href="https://doi.org/10.4490/algae.2008.23.3.187" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Algae&amp;title=A%20Study%20on%20the%20Fine%20Structure%20of%20the%20Marine%20Diatoms%20of%20Korean%20Coastal%20Waters%20%E2%80%93%20Genus%20Thalassiosira%203&amp;author=J.%E2%80%90H.%20Lee&amp;author=J.%E2%80%90S.%20Park&amp;volume=23&amp;issue=3&amp;publication_year=2008&amp;pages=187-199&amp;doi=10.4490/algae.2008.23.3.187&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0065">
<cite>

Li, W. K. W.

1998. “Annual Average Abundance of Heterotrophic Bacteria and <em>Synechococcus</em> in Surface Ocean Waters.” Limnology and Oceanography
43, no. 7: 1746–1753. 10.4319/lo.1998.43.7.1746.</cite> [<a href="https://doi.org/10.4319/lo.1998.43.7.1746" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Limnology%20and%20Oceanography&amp;title=Annual%20Average%20Abundance%20of%20Heterotrophic%20Bacteria%20and%20Synechococcus%20in%20Surface%20Ocean%20Waters&amp;author=W.%20K.%20W.%20Li&amp;volume=43&amp;issue=7&amp;publication_year=1998&amp;pages=1746-1753&amp;doi=10.4319/lo.1998.43.7.1746&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0066">
<cite>

Li, W. K. W.
, and Dickie P. M.. 2001. “Monitoring Phytoplankton, Bacterioplankton, and Virioplankton in a Coastal Inlet (Bedford Basin) by Flow Cytometry.” Cytometry
44, no. 3: 236–246. .
</cite> [<a href="https://doi.org/10.1002/1097-0320(20010701)44:3&amp;#x0003c;236::aid-cyto1116&amp;#x0003e;3.0.co;2-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11429774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cytometry&amp;title=Monitoring%20Phytoplankton,%20Bacterioplankton,%20and%20Virioplankton%20in%20a%20Coastal%20Inlet%20(Bedford%20Basin)%20by%20Flow%20Cytometry&amp;author=W.%20K.%20W.%20Li&amp;author=P.%20M.%20Dickie&amp;volume=44&amp;issue=3&amp;publication_year=2001&amp;pages=236-246&amp;pmid=11429774&amp;doi=10.1002/1097-0320(20010701)44:3&amp;#x0003c;236::aid-cyto1116&amp;#x0003e;3.0.co;2-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0067"><cite>

Li, W. K. W.
, Dickie P. M., and Spry J. A.. 1998. “<em>Plankton Monitoring Programme in the Bedford Basin, 1991–1997</em>.” Can. Data Rep. Fish. Aquat. Sci. 1036, vii + 342 pp.</cite></li>
<li id="jeu70033-bib-0068">
<cite>

Li, Y.
, Guo X.‐H., and Lundholm N.. 2020. “Molecular Phylogeny and Taxonomy of the Genus <em>Minidiscus</em> (Bacillariophyceae), with Description of <em>Mediolabrus</em> gen. nov.” Journal of Phycology
56, no. 6: 1443–1456. 10.1111/jpy.13038.
</cite> [<a href="https://doi.org/10.1111/jpy.13038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32510583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Phycology&amp;title=Molecular%20Phylogeny%20and%20Taxonomy%20of%20the%20Genus%20Minidiscus%20(Bacillariophyceae),%20with%20Description%20of%20Mediolabrus%20gen.%20nov&amp;author=Y.%20Li&amp;author=X.%E2%80%90H.%20Guo&amp;author=N.%20Lundholm&amp;volume=56&amp;issue=6&amp;publication_year=2020&amp;pages=1443-1456&amp;pmid=32510583&amp;doi=10.1111/jpy.13038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0069">
<cite>*
Li, Y.
, Zhao Q., and Lü S.. 2013. “The Genus <em>Thalassiosira</em> off the Guangdong Coast, South China Sea.” Botanica Marina
56: 83–110. 10.1515/bot-2011-0045.</cite> [<a href="https://doi.org/10.1515/bot-2011-0045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Botanica%20Marina&amp;title=The%20Genus%20Thalassiosira%20off%20the%20Guangdong%20Coast,%20South%20China%20Sea&amp;author=Y.%20Li&amp;author=Q.%20Zhao&amp;author=S.%20L%C3%BC&amp;volume=56&amp;publication_year=2013&amp;pages=83-110&amp;doi=10.1515/bot-2011-0045&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0070">
<cite>

Liu, K.
, Chen Y., Cui Z., Liu S., Xu Q., and Chen N.. 2021. “Comparative Analysis of Chloroplast Genomes of <em>Thalassiosira</em> Species.” Frontiers in Marine Science
8: 788307. 10.3389/fmars.2021.788307.</cite> [<a href="https://doi.org/10.3389/fmars.2021.788307" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Marine%20Science&amp;title=Comparative%20Analysis%20of%20Chloroplast%20Genomes%20of%20Thalassiosira%20Species&amp;author=K.%20Liu&amp;author=Y.%20Chen&amp;author=Z.%20Cui&amp;author=S.%20Liu&amp;author=Q.%20Xu&amp;volume=8&amp;publication_year=2021&amp;pages=788307&amp;doi=10.3389/fmars.2021.788307&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0071">
<cite>*
Liu, K.
, Liu S., Chen Y., Liu F., Zhao Y., and Chen N.. 2021. “Complete Mitochondrial Genome of 
<em>Thalassiosira profunda</em>
 (Mediophyceae, Bacillariophyta).” Mitochondrial DNA Part B Resources
6, no. 4: 1560–1562. 10.1080/23802359.2021.1916409.
</cite> [<a href="https://doi.org/10.1080/23802359.2021.1916409" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8079070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33969218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mitochondrial%20DNA%20Part%20B%20Resources&amp;title=Complete%20Mitochondrial%20Genome%20of%20Thalassiosira%20profunda%20(Mediophyceae,%20Bacillariophyta)&amp;author=K.%20Liu&amp;author=S.%20Liu&amp;author=Y.%20Chen&amp;author=F.%20Liu&amp;author=Y.%20Zhao&amp;volume=6&amp;issue=4&amp;publication_year=2021&amp;pages=1560-1562&amp;pmid=33969218&amp;doi=10.1080/23802359.2021.1916409&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0072">
<cite>*
Liu, S.
, Wang Y., Xu Q., Zhang M., and Chen N.. 2021. “Comparative Analysis of Full‐Length Mitochondrial Genomes of Five <em>Skeletonema</em> Species Reveals Conserved Genome Organization and Recent Speciation.” BMC Genomics
22: 746. 10.1186/s12864-021-07999-z.
</cite> [<a href="https://doi.org/10.1186/s12864-021-07999-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8520197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34654361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Comparative%20Analysis%20of%20Full%E2%80%90Length%20Mitochondrial%20Genomes%20of%20Five%20Skeletonema%20Species%20Reveals%20Conserved%20Genome%20Organization%20and%20Recent%20Speciation&amp;author=S.%20Liu&amp;author=Y.%20Wang&amp;author=Q.%20Xu&amp;author=M.%20Zhang&amp;author=N.%20Chen&amp;volume=22&amp;publication_year=2021&amp;pages=746&amp;pmid=34654361&amp;doi=10.1186/s12864-021-07999-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0073">
<cite>

Liu, S.
, Xu Q., Liu K., Zhao Y., and Chen N.. 2021. “Chloroplast Genomes for Five <em>Skeletonema</em> Species: Comparative and Phylogenetic Analysis.” Frontiers in Plant Science
12, no. December: 774617. 10.3389/fpls.2021.774617.
</cite> [<a href="https://doi.org/10.3389/fpls.2021.774617" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8710728/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34966403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Chloroplast%20Genomes%20for%20Five%20Skeletonema%20Species:%20Comparative%20and%20Phylogenetic%20Analysis&amp;author=S.%20Liu&amp;author=Q.%20Xu&amp;author=K.%20Liu&amp;author=Y.%20Zhao&amp;author=N.%20Chen&amp;volume=12&amp;issue=December&amp;publication_year=2021&amp;pages=774617&amp;pmid=34966403&amp;doi=10.3389/fpls.2021.774617&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0074">
<cite>*
Lobban, C. S.
, Schefter M., Jordan R. W., et al. 2012. “Coral‐Reef Diatoms (Bacillariophyta) From Guam: New Records and Preliminary Checklist, With Emphasis on Epiphytic Species From Farmer‐Fish Territories.” Micronesica
43, no. 2: 237–279.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Micronesica&amp;title=Coral%E2%80%90Reef%20Diatoms%20(Bacillariophyta)%20From%20Guam:%20New%20Records%20and%20Preliminary%20Checklist,%20With%20Emphasis%20on%20Epiphytic%20Species%20From%20Farmer%E2%80%90Fish%20Territories&amp;author=C.%20S.%20Lobban&amp;author=M.%20Schefter&amp;author=R.%20W.%20Jordan&amp;volume=43&amp;issue=2&amp;publication_year=2012&amp;pages=237-279&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0075">
<cite>

Lommer, M.
, Roy A.‐S., Schilhabel M., Schreiber S., Rosenstiel P., and LaRoche J.. 2010. “Recent Transfer of an Iron‐Regulated Gene From the Plastid to the Nuclear Genome in an Oceanic Diatom Adapted to Chronic Iron Limitation.” BMC Genomics
11, no. 1: 718. 10.1186/1471-2164-11-718.
</cite> [<a href="https://doi.org/10.1186/1471-2164-11-718" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3022921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21171997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Recent%20Transfer%20of%20an%20Iron%E2%80%90Regulated%20Gene%20From%20the%20Plastid%20to%20the%20Nuclear%20Genome%20in%20an%20Oceanic%20Diatom%20Adapted%20to%20Chronic%20Iron%20Limitation&amp;author=M.%20Lommer&amp;author=A.%E2%80%90S.%20Roy&amp;author=M.%20Schilhabel&amp;author=S.%20Schreiber&amp;author=P.%20Rosenstiel&amp;volume=11&amp;issue=1&amp;publication_year=2010&amp;pages=718&amp;pmid=21171997&amp;doi=10.1186/1471-2164-11-718&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0076">
<cite>

Malviya, S.
, Scalco E., Audic S., et al. 2016. “Insights Into Global Diatom Distribution and Diversity in the World's Ocean.” Proceedings of the National Academy of Sciences of the United States of America
113, no. 11: E1516–E1525. 10.1073/pnas.1509523113.
</cite> [<a href="https://doi.org/10.1073/pnas.1509523113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4801293/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26929361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Insights%20Into%20Global%20Diatom%20Distribution%20and%20Diversity%20in%20the%20World's%20Ocean&amp;author=S.%20Malviya&amp;author=E.%20Scalco&amp;author=S.%20Audic&amp;volume=113&amp;issue=11&amp;publication_year=2016&amp;pages=E1516-E1525&amp;pmid=26929361&amp;doi=10.1073/pnas.1509523113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0077">
<cite>

Martens, N.
, Biederbick J., and Schaum C.‐E.. 2024. “Picophytoplankton Prevail Year‐Round in the Elbe Estuary.” Plant‐Environment Interactions
5, no. 5: e70014. 10.1002/pei3.70014.
</cite> [<a href="https://doi.org/10.1002/pei3.70014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11513158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39464516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%E2%80%90Environment%20Interactions&amp;title=Picophytoplankton%20Prevail%20Year%E2%80%90Round%20in%20the%20Elbe%20Estuary&amp;author=N.%20Martens&amp;author=J.%20Biederbick&amp;author=C.%E2%80%90E.%20Schaum&amp;volume=5&amp;issue=5&amp;publication_year=2024&amp;pages=e70014&amp;pmid=39464516&amp;doi=10.1002/pei3.70014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0078">
<cite>

Martens, N.
, Russnak V., Woodhouse J., Grossart H.‐P., and Schaum C.‐E.. 2024. “Metabarcoding Reveals Potentially Mixotrophic Flagellates and Picophytoplankton as Key Groups of Phytoplankton in the Elbe Estuary.” Environmental Research
252, no. 4: 119126. 10.1016/j.envres.2024.119126.
</cite> [<a href="https://doi.org/10.1016/j.envres.2024.119126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38734293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20Research&amp;title=Metabarcoding%20Reveals%20Potentially%20Mixotrophic%20Flagellates%20and%20Picophytoplankton%20as%20Key%20Groups%20of%20Phytoplankton%20in%20the%20Elbe%20Estuary&amp;author=N.%20Martens&amp;author=V.%20Russnak&amp;author=J.%20Woodhouse&amp;author=H.%E2%80%90P.%20Grossart&amp;author=C.%E2%80%90E.%20Schaum&amp;volume=252&amp;issue=4&amp;publication_year=2024&amp;pages=119126&amp;pmid=38734293&amp;doi=10.1016/j.envres.2024.119126&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0079">
<cite>

McMurdie, P. J.
, and Holmes S.. 2013. “Phyloseq: An R Package for Reproducible Interactive Analysis and Graphics of Microbiome Census Data.” PLoS One
8, no. 4: e61217. 10.1371/journal.pone.0061217.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0061217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3632530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23630581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Phyloseq:%20An%20R%20Package%20for%20Reproducible%20Interactive%20Analysis%20and%20Graphics%20of%20Microbiome%20Census%20Data&amp;author=P.%20J.%20McMurdie&amp;author=S.%20Holmes&amp;volume=8&amp;issue=4&amp;publication_year=2013&amp;pages=e61217&amp;pmid=23630581&amp;doi=10.1371/journal.pone.0061217&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0080">
<cite>

Minh, B. Q.
, Schmidt H. A., Chernomor O., et al. 2020. “IQ‐TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era.” Molecular Biology and Evolution
37, no. 5: 1530–1534. 10.1093/molbev/msaa015.
</cite> [<a href="https://doi.org/10.1093/molbev/msaa015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7182206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32011700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=IQ%E2%80%90TREE%202:%20New%20Models%20and%20Efficient%20Methods%20for%20Phylogenetic%20Inference%20in%20the%20Genomic%20Era&amp;author=B.%20Q.%20Minh&amp;author=H.%20A.%20Schmidt&amp;author=O.%20Chernomor&amp;volume=37&amp;issue=5&amp;publication_year=2020&amp;pages=1530-1534&amp;pmid=32011700&amp;doi=10.1093/molbev/msaa015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0081">
<cite>

Mordret, S.
, MacKinnon J., Behnke J., O'Leary S. J. B., and Chénard C.. 2024. “Identification of Phytoplankton Isolates From the Eastern Canadian Waters Using Long‐Read Sequencing.” Journal of Plankton Research
46, no. 6: 527–541. 10.1093/plankt/fbae043.
</cite> [<a href="https://doi.org/10.1093/plankt/fbae043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11629783/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39664262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Plankton%20Research&amp;title=Identification%20of%20Phytoplankton%20Isolates%20From%20the%20Eastern%20Canadian%20Waters%20Using%20Long%E2%80%90Read%20Sequencing&amp;author=S.%20Mordret&amp;author=J.%20MacKinnon&amp;author=J.%20Behnke&amp;author=S.%20J.%20B.%20O'Leary&amp;author=C.%20Ch%C3%A9nard&amp;volume=46&amp;issue=6&amp;publication_year=2024&amp;pages=527-541&amp;pmid=39664262&amp;doi=10.1093/plankt/fbae043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0082">
<cite>

Nakazato, T.
, and Jinbo U.. 2022. “Cross‐Sectional Use of Barcode of Life Data System and GenBank as DNA Barcoding Databases for the Advancement of Museomics.” Frontiers in Ecology and Evolution
10, no. October: 966605. 10.3389/fevo.2022.966605.</cite> [<a href="https://doi.org/10.3389/fevo.2022.966605" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Ecology%20and%20Evolution&amp;title=Cross%E2%80%90Sectional%20Use%20of%20Barcode%20of%20Life%20Data%20System%20and%20GenBank%20as%20DNA%20Barcoding%20Databases%20for%20the%20Advancement%20of%20Museomics&amp;author=T.%20Nakazato&amp;author=U.%20Jinbo&amp;volume=10&amp;issue=October&amp;publication_year=2022&amp;pages=966605&amp;doi=10.3389/fevo.2022.966605&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0083">
<cite>

Nelson, D. M.
, Tréguer P., Brzezinski M. A., Leynaert A., and Quéguiner B.. 1995. “Production and Dissolution of Biogenic Silica in the Ocean: Revised Global Estimates, Comparison With Regional Data and Relationship to Biogenic Sedimentation.” Global Biogeochemical Cycles
9, no. 3: 359–372. 10.1029/95GB01070.</cite> [<a href="https://doi.org/10.1029/95GB01070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Global%20Biogeochemical%20Cycles&amp;title=Production%20and%20Dissolution%20of%20Biogenic%20Silica%20in%20the%20Ocean:%20Revised%20Global%20Estimates,%20Comparison%20With%20Regional%20Data%20and%20Relationship%20to%20Biogenic%20Sedimentation&amp;author=D.%20M.%20Nelson&amp;author=P.%20Tr%C3%A9guer&amp;author=M.%20A.%20Brzezinski&amp;author=A.%20Leynaert&amp;author=B.%20Qu%C3%A9guiner&amp;volume=9&amp;issue=3&amp;publication_year=1995&amp;pages=359-372&amp;doi=10.1029/95GB01070&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0084">
<cite>

Noakes, M. T.
, Brinkerhoff H., Laszlo A. H., et al. 2019. “Increasing the Accuracy of Nanopore DNA Sequencing Using a Time‐Varying Cross Membrane Voltage.” Nature Biotechnology
37, no. 6: 651–656. 10.1038/s41587-019-0096-0.</cite> [<a href="https://doi.org/10.1038/s41587-019-0096-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6658736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31011178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Increasing%20the%20Accuracy%20of%20Nanopore%20DNA%20Sequencing%20Using%20a%20Time%E2%80%90Varying%20Cross%20Membrane%20Voltage&amp;author=M.%20T.%20Noakes&amp;author=H.%20Brinkerhoff&amp;author=A.%20H.%20Laszlo&amp;volume=37&amp;issue=6&amp;publication_year=2019&amp;pages=651-656&amp;pmid=31011178&amp;doi=10.1038/s41587-019-0096-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0085">
<cite>

Nordberg, H.
, Cantor M., Dusheyko S., et al. 2014. “The Genome Portal of the Department of Energy Joint Genome Institute: 2014 Updates.” Nucleic Acids Research
42, no. D1: D26–D31. 10.1093/nar/gkt1069.
</cite> [<a href="https://doi.org/10.1093/nar/gkt1069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3965075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24225321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=The%20Genome%20Portal%20of%20the%20Department%20of%20Energy%20Joint%20Genome%20Institute:%202014%20Updates&amp;author=H.%20Nordberg&amp;author=M.%20Cantor&amp;author=S.%20Dusheyko&amp;volume=42&amp;issue=D1&amp;publication_year=2014&amp;pages=D26-D31&amp;pmid=24225321&amp;doi=10.1093/nar/gkt1069&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0086"><cite>*
Olenina, I.

2006. “Biovolumes and Size‐Classes of Phytoplankton in the Baltic Sea.”
<em>HELCOM Balt. Sea Environ. Proc</em> no. 106.</cite></li>
<li id="jeu70033-bib-0087">
<cite>

Oudot‐Le Secq, M.‐P.
, and Green B. R.. 2011. “Complex Repeat Structures and Novel Features in the Mitochondrial Genomes of the Diatoms <em>Phaeodactylum Tricornutum</em> and 
<em>Thalassiosira pseudonana</em>

.” Gene
476, no. 1–2: 20–26. 10.1016/j.gene.2011.02.001.
</cite> [<a href="https://doi.org/10.1016/j.gene.2011.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21320580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Complex%20Repeat%20Structures%20and%20Novel%20Features%20in%20the%20Mitochondrial%20Genomes%20of%20the%20Diatoms%20Phaeodactylum%20Tricornutum%20and%20Thalassiosira%20pseudonana&amp;author=M.%E2%80%90P.%20Oudot%E2%80%90Le%20Secq&amp;author=B.%20R.%20Green&amp;volume=476&amp;issue=1%E2%80%932&amp;publication_year=2011&amp;pages=20-26&amp;pmid=21320580&amp;doi=10.1016/j.gene.2011.02.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0088">
<cite>

Parada, A. E.
, Needham D. M., and Fuhrman J. A.. 2016. “Every Base Matters: Assessing Small Subunit rRNA Primers for Marine Microbiomes With Mock Communities, Time Series and Global Field Samples.” Environmental Microbiology
18, no. 5: 1403–1414. 10.1111/1462-2920.13023.
</cite> [<a href="https://doi.org/10.1111/1462-2920.13023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26271760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20Microbiology&amp;title=Every%20Base%20Matters:%20Assessing%20Small%20Subunit%20rRNA%20Primers%20for%20Marine%20Microbiomes%20With%20Mock%20Communities,%20Time%20Series%20and%20Global%20Field%20Samples&amp;author=A.%20E.%20Parada&amp;author=D.%20M.%20Needham&amp;author=J.%20A.%20Fuhrman&amp;volume=18&amp;issue=5&amp;publication_year=2016&amp;pages=1403-1414&amp;pmid=26271760&amp;doi=10.1111/1462-2920.13023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0089">
<cite>*
Park, J. S.
, Jung S. W., Ki J.‐S., et al. 2017. “Transfer of the Small Diatoms <em>Thalassiosira proschkinae</em> and <em>T. spinulata</em> to the Genus <em>Minidiscus</em> and Their Taxonomic Re‐Description.” PLoS One
12, no. 9: e0181980. 10.1371/journal.pone.0181980.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0181980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5584963/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28873451/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Transfer%20of%20the%20Small%20Diatoms%20Thalassiosira%20proschkinae%20and%20T.%20spinulata%20to%20the%20Genus%20Minidiscus%20and%20Their%20Taxonomic%20Re%E2%80%90Description&amp;author=J.%20S.%20Park&amp;author=S.%20W.%20Jung&amp;author=J.%E2%80%90S.%20Ki&amp;volume=12&amp;issue=9&amp;publication_year=2017&amp;pages=e0181980&amp;pmid=28873451&amp;doi=10.1371/journal.pone.0181980&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0090">
<cite>*
Park, J. S.
, and Lee J. H.. 2014. “Description of the Pseudocryptic Species <em>Conticribra weissflogiopsis</em> sp. nov. (Thalassiosirales, Bacillariophyta) Isolated From Brackish Waters in Korea, Based on Its Cingulum Structure and Molecular Analysis.” Phytotaxa
191, no. 1: 115. 10.11646/phytotaxa.191.1.7.</cite> [<a href="https://doi.org/10.11646/phytotaxa.191.1.7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phytotaxa&amp;title=Description%20of%20the%20Pseudocryptic%20Species%20Conticribra%20weissflogiopsis%20sp.%20nov.%20(Thalassiosirales,%20Bacillariophyta)%20Isolated%20From%20Brackish%20Waters%20in%20Korea,%20Based%20on%20Its%20Cingulum%20Structure%20and%20Molecular%20Analysis&amp;author=J.%20S.%20Park&amp;author=J.%20H.%20Lee&amp;volume=191&amp;issue=1&amp;publication_year=2014&amp;pages=115&amp;doi=10.11646/phytotaxa.191.1.7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0091">
<cite>*
Park, J.‐S.
, and Lee J.‐H.. 2010. “A Study on the Fine Structure of Marine Diatoms in Korean Coastal Waters: Genus <em>Thalassiosira</em> 5.” Algae
25, no. 3: 121–131. 10.4490/algae.2010.25.3.121.</cite> [<a href="https://doi.org/10.4490/algae.2010.25.3.121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Algae&amp;title=A%20Study%20on%20the%20Fine%20Structure%20of%20Marine%20Diatoms%20in%20Korean%20Coastal%20Waters:%20Genus%20Thalassiosira%205&amp;author=J.%E2%80%90S.%20Park&amp;author=J.%E2%80%90H.%20Lee&amp;volume=25&amp;issue=3&amp;publication_year=2010&amp;pages=121-131&amp;doi=10.4490/algae.2010.25.3.121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0092">
<cite>

Pesant, S.
, Not F., Picheral M., et al. 2015. “Open Science Resources for the Discovery and Analysis of Tara Oceans Data.” Scientific Data
2, no. 1: 150023. 10.1038/sdata.2015.23.
</cite> [<a href="https://doi.org/10.1038/sdata.2015.23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4443879/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26029378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Data&amp;title=Open%20Science%20Resources%20for%20the%20Discovery%20and%20Analysis%20of%20Tara%20Oceans%20Data&amp;author=S.%20Pesant&amp;author=F.%20Not&amp;author=M.%20Picheral&amp;volume=2&amp;issue=1&amp;publication_year=2015&amp;pages=150023&amp;pmid=26029378&amp;doi=10.1038/sdata.2015.23&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0093">
<cite>

Pierella Karlusich, J. J.
, Cosnier K., Zinger L., et al. 2025. “Patterns and Drivers of Diatom Diversity and Abundance in the Global Ocean.” Nature Communications
16, no. 1: 3452. 10.1038/s41467-025-58027-7.</cite> [<a href="https://doi.org/10.1038/s41467-025-58027-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11992049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40216740/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Patterns%20and%20Drivers%20of%20Diatom%20Diversity%20and%20Abundance%20in%20the%20Global%20Ocean&amp;author=J.%20J.%20Pierella%20Karlusich&amp;author=K.%20Cosnier&amp;author=L.%20Zinger&amp;volume=16&amp;issue=1&amp;publication_year=2025&amp;pages=3452&amp;pmid=40216740&amp;doi=10.1038/s41467-025-58027-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0094">
<cite>

Pogoda, C. S.
, Keepers K. G., Hamsher S. E., Stepanek J. G., Kane N. C., and Kociolek J. P.. 2019. “Comparative Analysis of the Mitochondrial Genomes of Six Newly Sequenced Diatoms Reveals Group II Introns in the Barcoding Region of <em>cox1</em>
.” Mitochondrial DNA Part A, DNA Mapping, Sequencing, and Analysis
30, no. 1: 43–51. 10.1080/24701394.2018.1450397.
</cite> [<a href="https://doi.org/10.1080/24701394.2018.1450397" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29527965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mitochondrial%20DNA%20Part%20A,%20DNA%20Mapping,%20Sequencing,%20and%20Analysis&amp;title=Comparative%20Analysis%20of%20the%20Mitochondrial%20Genomes%20of%20Six%20Newly%20Sequenced%20Diatoms%20Reveals%20Group%20II%20Introns%20in%20the%20Barcoding%20Region%20of%20cox1&amp;author=C.%20S.%20Pogoda&amp;author=K.%20G.%20Keepers&amp;author=S.%20E.%20Hamsher&amp;author=J.%20G.%20Stepanek&amp;author=N.%20C.%20Kane&amp;volume=30&amp;issue=1&amp;publication_year=2019&amp;pages=43-51&amp;pmid=29527965&amp;doi=10.1080/24701394.2018.1450397&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0095">
<cite>

Quast, C.
, Pruesse E., Yilmaz P., et al. 2013. “The SILVA Ribosomal RNA Gene Database Project: Improved Data Processing and Web‐Based Tools.” Nucleic Acids Research
41, no. D1: D590–D596. 10.1093/nar/gks1219.
</cite> [<a href="https://doi.org/10.1093/nar/gks1219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3531112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23193283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=The%20SILVA%20Ribosomal%20RNA%20Gene%20Database%20Project:%20Improved%20Data%20Processing%20and%20Web%E2%80%90Based%20Tools&amp;author=C.%20Quast&amp;author=E.%20Pruesse&amp;author=P.%20Yilmaz&amp;volume=41&amp;issue=D1&amp;publication_year=2013&amp;pages=D590-D596&amp;pmid=23193283&amp;doi=10.1093/nar/gks1219&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0096"><cite>
R Core Team
. 2021. “<em>R: A Language and Environment for Statistical Computing</em>.” Version 4.3.2 (2023‐10‐31). R Foundation for Statistical Computing, Vienna, Austria. <a href="https://www.r-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R‐project.org/</a>.</cite></li>
<li id="jeu70033-bib-0097">
<cite>

Rice, P.
, Longden I., and Bleasby A.. 2000. “EMBOSS: The European Molecular Biology Open Software Suite.” Trends in Genetics
16, no. 6: 276–277. 10.1016/s0168-9525(00)02024-2.
</cite> [<a href="https://doi.org/10.1016/s0168-9525(00)02024-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10827456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Genetics&amp;title=EMBOSS:%20The%20European%20Molecular%20Biology%20Open%20Software%20Suite&amp;author=P.%20Rice&amp;author=I.%20Longden&amp;author=A.%20Bleasby&amp;volume=16&amp;issue=6&amp;publication_year=2000&amp;pages=276-277&amp;pmid=10827456&amp;doi=10.1016/s0168-9525(00)02024-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0098">
<cite>*
Roberts, W. R.
, Siepielski A. M., and Alverson A. J.. 2024. “Diatom Abundance in the Polar Oceans Is Predicted by Genome Size.” PLoS Biology
22, no. 8: e3002733. 10.1371/journal.pbio.3002733.
</cite> [<a href="https://doi.org/10.1371/journal.pbio.3002733" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11309476/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39116044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biology&amp;title=Diatom%20Abundance%20in%20the%20Polar%20Oceans%20Is%20Predicted%20by%20Genome%20Size&amp;author=W.%20R.%20Roberts&amp;author=A.%20M.%20Siepielski&amp;author=A.%20J.%20Alverson&amp;volume=22&amp;issue=8&amp;publication_year=2024&amp;pages=e3002733&amp;pmid=39116044&amp;doi=10.1371/journal.pbio.3002733&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0099">
<cite>

Robicheau, B. M.
, Tolman J., Bertrand E. M., and LaRoche J.. 2022. “Highly‐Resolved Interannual Phytoplankton Community Dynamics of the Coastal Northwest Atlantic.” ISME Communications
2, no. 1: 38. 10.1038/s43705-022-00119-2.
</cite> [<a href="https://doi.org/10.1038/s43705-022-00119-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9723599/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37938666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20Communications&amp;title=Highly%E2%80%90Resolved%20Interannual%20Phytoplankton%20Community%20Dynamics%20of%20the%20Coastal%20Northwest%20Atlantic&amp;author=B.%20M.%20Robicheau&amp;author=J.%20Tolman&amp;author=E.%20M.%20Bertrand&amp;author=J.%20LaRoche&amp;volume=2&amp;issue=1&amp;publication_year=2022&amp;pages=38&amp;pmid=37938666&amp;doi=10.1038/s43705-022-00119-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0100">
<cite>*
Round, F. E.

1988. “A New Species of <em>Thalassiosira, T. cardiophora</em>
.” Diatom Research
3: 123–127. 10.1080/0269249X.1988.9705021.</cite> [<a href="https://doi.org/10.1080/0269249X.1988.9705021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diatom%20Research&amp;title=A%20New%20Species%20of%20Thalassiosira,%20T.%20cardiophora&amp;author=F.%20E.%20Round&amp;volume=3&amp;publication_year=1988&amp;pages=123-127&amp;doi=10.1080/0269249X.1988.9705021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0101">
<cite>

Round, F. E.
, Crawford R. M., and Mann D. G.. 1990. The Diatoms: Biology and Morphology of the Genera. Cambridge University Press.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20Diatoms:%20Biology%20and%20Morphology%20of%20the%20Genera&amp;author=F.%20E.%20Round&amp;author=R.%20M.%20Crawford&amp;author=D.%20G.%20Mann&amp;publication_year=1990&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0102"><cite>
RStudio Team
. 2020. “<em>RStudio: Integrated Development for R</em>.” RStudio, PBC, Boston, MA (version 2023.09.1+494). <a href="http://www.rstudio.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.rstudio.com/</a>.</cite></li>
<li id="jeu70033-bib-0103">
<cite>*
Sarno, D.
, Kooistra W. H. C. F., Balzano S., Hargraves P. E., and Zingone A.. 2007. “Diversity in the Genus <em>Skeletonema</em> (Bacillariophyceae): III. Phylogenetic Position and Morphological Variability of <em>Skeletonema costatum</em> and <em>Skeletonema grevillei</em>, With the Description of <em>Skeletonema ardens</em> sp. nov.” Journal of Phycology
43, no. 1: 156–170. 10.1111/j.1529-8817.2006.00305.x.</cite> [<a href="https://doi.org/10.1111/j.1529-8817.2006.00305.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Phycology&amp;title=Diversity%20in%20the%20Genus%20Skeletonema%20(Bacillariophyceae):%20III.%20Phylogenetic%20Position%20and%20Morphological%20Variability%20of%20Skeletonema%20costatum%20and%20Skeletonema%20grevillei,%20With%20the%20Description%20of%20Skeletonema%20ardens%20sp.%20nov&amp;author=D.%20Sarno&amp;author=W.%20H.%20C.%20F.%20Kooistra&amp;author=S.%20Balzano&amp;author=P.%20E.%20Hargraves&amp;author=A.%20Zingone&amp;volume=43&amp;issue=1&amp;publication_year=2007&amp;pages=156-170&amp;doi=10.1111/j.1529-8817.2006.00305.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0104">
<cite>*
Schultz, K.
, Dreßler M., Jacques O., Frank M., Springer A., and Hübener T.. 2024. “Morphological and Taxonomic Implications of the Formation of Colonies in <em>Cyclostephanos</em> and <em>Stephanodiscus</em> (Bacillariophyta).” European Journal of Phycology
59, no. 1: 71–81. 10.1080/09670262.2023.2222297.</cite> [<a href="https://doi.org/10.1080/09670262.2023.2222297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Phycology&amp;title=Morphological%20and%20Taxonomic%20Implications%20of%20the%20Formation%20of%20Colonies%20in%20Cyclostephanos%20and%20Stephanodiscus%20(Bacillariophyta)&amp;author=K.%20Schultz&amp;author=M.%20Dre%C3%9Fler&amp;author=O.%20Jacques&amp;author=M.%20Frank&amp;author=A.%20Springer&amp;volume=59&amp;issue=1&amp;publication_year=2024&amp;pages=71-81&amp;doi=10.1080/09670262.2023.2222297&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0105">
<cite>

Shafin, K.
, Pesout T., Chang P.‐C., et al. 2021. “Haplotype‐Aware Variant Calling With PEPPER‐Margin‐DeepVariant Enables High Accuracy in Nanopore Long‐Reads.” Nature Methods
18, no. 11: 1322–1332. 10.1038/s41592-021-01299-w.
</cite> [<a href="https://doi.org/10.1038/s41592-021-01299-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8571015/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34725481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Haplotype%E2%80%90Aware%20Variant%20Calling%20With%20PEPPER%E2%80%90Margin%E2%80%90DeepVariant%20Enables%20High%20Accuracy%20in%20Nanopore%20Long%E2%80%90Reads&amp;author=K.%20Shafin&amp;author=T.%20Pesout&amp;author=P.%E2%80%90C.%20Chang&amp;volume=18&amp;issue=11&amp;publication_year=2021&amp;pages=1322-1332&amp;pmid=34725481&amp;doi=10.1038/s41592-021-01299-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0106">
<cite>*
Smucker, N. J.
, Edlund M. B., and Vis M. L.. 2008. “The Distribution, Morphology, and Ecology of a Non‐Native Species, 
<em>Thalassiosira lacustris</em>
 (Bacillariophyceae), from Benthic Stream Habitats in North America.” Nova Hedwigia
87, no. 1–2: 201–220. 10.1127/0029-5035/2008/0087-0201.</cite> [<a href="https://doi.org/10.1127/0029-5035/2008/0087-0201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nova%20Hedwigia&amp;title=The%20Distribution,%20Morphology,%20and%20Ecology%20of%20a%20Non%E2%80%90Native%20Species,%20Thalassiosira%20lacustris%20(Bacillariophyceae),%20from%20Benthic%20Stream%20Habitats%20in%20North%20America&amp;author=N.%20J.%20Smucker&amp;author=M.%20B.%20Edlund&amp;author=M.%20L.%20Vis&amp;volume=87&amp;issue=1%E2%80%932&amp;publication_year=2008&amp;pages=201-220&amp;doi=10.1127/0029-5035/2008/0087-0201&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0107"><cite>

Sudarshan, L. L.

2012. –2019. “<em>Microbiome R Package</em>.”
<a href="http://microbiome.github.io" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://microbiome.github.io</a>.</cite></li>
<li id="jeu70033-bib-0108"><cite>

Therriault, J.‐C.
, Petrie B., Pepin P., et al. 1998. “<em>Proposal for a Northwest Atlantic Zonal Monitoring Program</em>.” Canadian Technical Report of Hydrography and Ocean Sciences 194.</cite></li>
<li id="jeu70033-bib-0109">
<cite>

Tillich, M.
, Lehwark P., Pellizzer T., et al. 2017. “GeSeq – Versatile and Accurate Annotation of Organelle Genomes.” Nucleic Acids Research
45, no. W1: W6–W11. 10.1093/nar/gkx391.
</cite> [<a href="https://doi.org/10.1093/nar/gkx391" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5570176/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28486635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=GeSeq%20%E2%80%93%20Versatile%20and%20Accurate%20Annotation%20of%20Organelle%20Genomes&amp;author=M.%20Tillich&amp;author=P.%20Lehwark&amp;author=T.%20Pellizzer&amp;volume=45&amp;issue=W1&amp;publication_year=2017&amp;pages=W6-W11&amp;pmid=28486635&amp;doi=10.1093/nar/gkx391&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0110">
<cite>

Vaser, R.
, and Šikić M.. 2020. “Raven: A De Novo Genome Assembler for Long Reads.”
<em>bioRxiv</em>, August. 10.1101/2020.08.07.242461.</cite> [<a href="https://doi.org/10.1101/2020.08.07.242461" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="jeu70033-bib-0111">
<cite>

Vaser, R.
, and Šikić M.. 2021. “Time and Memory Efficient Genome Assembly With Raven.” Nature Computational Science
1, no. 5: 332–336. 10.1038/s43588-021-00073-4.
</cite> [<a href="https://doi.org/10.1038/s43588-021-00073-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38217213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Computational%20Science&amp;title=Time%20and%20Memory%20Efficient%20Genome%20Assembly%20With%20Raven&amp;author=R.%20Vaser&amp;author=M.%20%C5%A0iki%C4%87&amp;volume=1&amp;issue=5&amp;publication_year=2021&amp;pages=332-336&amp;pmid=38217213&amp;doi=10.1038/s43588-021-00073-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0112">
<cite>

Vineis, J. H.
, Burger J. M., Fawcett S. E., and Ward B. B.. 2025. “Co‐Occurrence and Successional Patterns Among Diatoms, Dinoflagellates, and Potential Parasites in a Coastal Upwelling Experiment.” Limnology and Oceanography
70: 1481–1498. 10.1002/lno.70048.</cite> [<a href="https://doi.org/10.1002/lno.70048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Limnology%20and%20Oceanography&amp;title=Co%E2%80%90Occurrence%20and%20Successional%20Patterns%20Among%20Diatoms,%20Dinoflagellates,%20and%20Potential%20Parasites%20in%20a%20Coastal%20Upwelling%20Experiment&amp;author=J.%20H.%20Vineis&amp;author=J.%20M.%20Burger&amp;author=S.%20E.%20Fawcett&amp;author=B.%20B.%20Ward&amp;volume=70&amp;publication_year=2025&amp;pages=1481-1498&amp;doi=10.1002/lno.70048&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0113">
<cite>

Walters, W.
, Hyde E. R., Berg‐Lyons D., et al. 2016. “Improved Bacterial 16S rRNA Gene (V4 and V4‐5) and Fungal Internal Transcribed Spacer Marker Gene Primers for Microbial Community Surveys.” mSystems
1: e00009‐15. 10.1128/mSystems.00009-15.</cite> [<a href="https://doi.org/10.1128/mSystems.00009-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5069754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27822518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=Improved%20Bacterial%2016S%20rRNA%20Gene%20(V4%20and%20V4%E2%80%905)%20and%20Fungal%20Internal%20Transcribed%20Spacer%20Marker%20Gene%20Primers%20for%20Microbial%20Community%20Surveys&amp;author=W.%20Walters&amp;author=E.%20R.%20Hyde&amp;author=D.%20Berg%E2%80%90Lyons&amp;volume=1&amp;publication_year=2016&amp;pages=e00009%E2%80%9015&amp;pmid=27822518&amp;doi=10.1128/mSystems.00009-15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0114"><cite>

Wang, Y.
, and Chen N.. 2022. “<em>Minidiscus spinulatus</em> Chloroplast, Complete Genome.”
<a href="https://www.ncbi.nlm.nih.gov/nuccore/OL855995.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/nuccore/OL855995.1</a>.</cite></li>
<li id="jeu70033-bib-0115">
<cite>

Wang, Y.
, Liu S., Wang J., et al. 2022. “Diatom Biodiversity and Speciation Revealed by Comparative Analysis of Mitochondrial Genomes.” Frontiers in Plant Science
13, no. March: 749982. 10.3389/fpls.2022.749982.
</cite> [<a href="https://doi.org/10.3389/fpls.2022.749982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8987724/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35401648/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Diatom%20Biodiversity%20and%20Speciation%20Revealed%20by%20Comparative%20Analysis%20of%20Mitochondrial%20Genomes&amp;author=Y.%20Wang&amp;author=S.%20Liu&amp;author=J.%20Wang&amp;volume=13&amp;issue=March&amp;publication_year=2022&amp;pages=749982&amp;pmid=35401648&amp;doi=10.3389/fpls.2022.749982&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0116">
<cite>

Wang, Y.
, Zhao Y., Bollas A., Wang Y., and Kin Fai A.. 2021. “Nanopore Sequencing Technology, Bioinformatics and Applications.” Nature Biotechnology
39, no. 11: 1348–1365. 10.1038/s41587-021-01108-x.</cite> [<a href="https://doi.org/10.1038/s41587-021-01108-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8988251/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34750572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Nanopore%20Sequencing%20Technology,%20Bioinformatics%20and%20Applications&amp;author=Y.%20Wang&amp;author=Y.%20Zhao&amp;author=A.%20Bollas&amp;author=Y.%20Wang&amp;author=A.%20Kin%20Fai&amp;volume=39&amp;issue=11&amp;publication_year=2021&amp;pages=1348-1365&amp;pmid=34750572&amp;doi=10.1038/s41587-021-01108-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0117">
<cite>

Wick, R. R.
, Judd L. M., Gorrie C. L., and Holt K. E.. 2017. “Completing Bacterial Genome Assemblies With Multiplex MinION Sequencing.” Microbial Genomics
3, no. 10: e000132. 10.1099/mgen.0.000132.
</cite> [<a href="https://doi.org/10.1099/mgen.0.000132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5695209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29177090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbial%20Genomics&amp;title=Completing%20Bacterial%20Genome%20Assemblies%20With%20Multiplex%20MinION%20Sequencing&amp;author=R.%20R.%20Wick&amp;author=L.%20M.%20Judd&amp;author=C.%20L.%20Gorrie&amp;author=K.%20E.%20Holt&amp;volume=3&amp;issue=10&amp;publication_year=2017&amp;pages=e000132&amp;pmid=29177090&amp;doi=10.1099/mgen.0.000132&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0118">
<cite>

Wickham, H.
, Averick M., Bryan J., et al. 2019. “Welcome to the Tidyverse.” Journal of Open Source Software
4, no. 43: 1686. 10.21105/joss.01686.</cite> [<a href="https://doi.org/10.21105/joss.01686" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Open%20Source%20Software&amp;title=Welcome%20to%20the%20Tidyverse&amp;author=H.%20Wickham&amp;author=M.%20Averick&amp;author=J.%20Bryan&amp;volume=4&amp;issue=43&amp;publication_year=2019&amp;pages=1686&amp;doi=10.21105/joss.01686&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0119"><cite>

Wickham, H.
, François R., Henry L., Müller K., and Vaughan D.. 2023. “<em>Dplyr: A Grammar of Data Manipulation</em>.” R Package. <a href="https://cran.r-project.org/web/packages/dplyr/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r‐project.org/web/packages/dplyr/index.html</a>.</cite></li>
<li id="jeu70033-bib-0120">
<cite>

Willis, C.
, Desai D., and LaRoche J.. 2019. “Influence of 16S rRNA Variable Region on Perceived Diversity of Marine Microbial Communities of the Northern North Atlantic.” FEMS Microbiology Letters
366, no. 13: fnz152. 10.1093/femsle/fnz152.
</cite> [<a href="https://doi.org/10.1093/femsle/fnz152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6673769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31344223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiology%20Letters&amp;title=Influence%20of%2016S%20rRNA%20Variable%20Region%20on%20Perceived%20Diversity%20of%20Marine%20Microbial%20Communities%20of%20the%20Northern%20North%20Atlantic&amp;author=C.%20Willis&amp;author=D.%20Desai&amp;author=J.%20LaRoche&amp;volume=366&amp;issue=13&amp;publication_year=2019&amp;pages=fnz152&amp;pmid=31344223&amp;doi=10.1093/femsle/fnz152&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0121">
<cite>*
Wojtal, A. Z.
, and Kwandrans J.. 2006. “Diatoms of the Wyzyna Krakowsko‐Czestochowska Upland (S Poland) – Coscinodiscophyceae (Thalassiosirophycidae).” Polish Botanical Journal
51: 177–207.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Polish%20Botanical%20Journal&amp;title=Diatoms%20of%20the%20Wyzyna%20Krakowsko%E2%80%90Czestochowska%20Upland%20(S%20Poland)%20%E2%80%93%20Coscinodiscophyceae%20(Thalassiosirophycidae)&amp;author=A.%20Z.%20Wojtal&amp;author=J.%20Kwandrans&amp;volume=51&amp;publication_year=2006&amp;pages=177-207&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0122">
<cite>*
Wolf, M.
, Scheffler W., and Nicklisch A.. 1992. “
<em>Stephanodiscus Neoastraea</em> Håkansson et Nickel (Bacillariophyta, Centrales) in Waters in Northeastern Germany.” Archiv für Protistenkunde
142, no. 3–4: 193–205. 10.1016/S0003-9365(11)80084-9.</cite> [<a href="https://doi.org/10.1016/S0003-9365(11)80084-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archiv%20f%C3%BCr%20Protistenkunde&amp;title=Stephanodiscus%20Neoastraea%20H%C3%A5kansson%20et%20Nickel%20(Bacillariophyta,%20Centrales)%20in%20Waters%20in%20Northeastern%20Germany&amp;author=M.%20Wolf&amp;author=W.%20Scheffler&amp;author=A.%20Nicklisch&amp;volume=142&amp;issue=3%E2%80%934&amp;publication_year=1992&amp;pages=193-205&amp;doi=10.1016/S0003-9365(11)80084-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0123">
<cite>

Wu, L.
, Wu M., Cui N., et al. 2021. “Plant Super‐Barcode: A Case Study on Genome‐Based Identification for Closely Related Species of <em>Fritillaria</em>
.” Chinese Medicine
16, no. 1: 52. 10.1186/s13020-021-00460-z.
</cite> [<a href="https://doi.org/10.1186/s13020-021-00460-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8256587/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34225754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chinese%20Medicine&amp;title=Plant%20Super%E2%80%90Barcode:%20A%20Case%20Study%20on%20Genome%E2%80%90Based%20Identification%20for%20Closely%20Related%20Species%20of%20Fritillaria&amp;author=L.%20Wu&amp;author=M.%20Wu&amp;author=N.%20Cui&amp;volume=16&amp;issue=1&amp;publication_year=2021&amp;pages=52&amp;pmid=34225754&amp;doi=10.1186/s13020-021-00460-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0124"><cite>

Yashayaev, I.
, Peterson I., and Wang Z.. 2021. “<em>Meteorological, Sea Ice and Physical Oceanographic Conditions in the Labrador Sea During 2019</em>.” DFO Canadian Science Advisory Secretariat Research Document 2021/074. iv + 38 pp.</cite></li>
<li id="jeu70033-bib-0125">
<cite>*
Zingone, A.
, Percopo I., Sims P. A., and Sarno D.. 2005. “Diversity in the Genus <em>Skeletonema</em> (Bacillariophyceae). I. A Reexamination of the Type Material of <em>S. costatum</em> With the Description of <em>S. grevillei</em> sp. nov.” Journal of Phycology
41, no. 1: 140–150. 10.1111/j.1529-8817.2005.04066.x.</cite> [<a href="https://doi.org/10.1111/j.1529-8817.2005.04066.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Phycology&amp;title=Diversity%20in%20the%20Genus%20Skeletonema%20(Bacillariophyceae).%20I.%20A%20Reexamination%20of%20the%20Type%20Material%20of%20S.%20costatum%20With%20the%20Description%20of%20S.%20grevillei%20sp.%20nov&amp;author=A.%20Zingone&amp;author=I.%20Percopo&amp;author=P.%20A.%20Sims&amp;author=D.%20Sarno&amp;volume=41&amp;issue=1&amp;publication_year=2005&amp;pages=140-150&amp;doi=10.1111/j.1529-8817.2005.04066.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0126">
<cite>

Zingone, A.
, Tortora C., D'Alelio D., Margiotta F., and Sarno D.. 2023. “Assembly Rules Vary Seasonally in Stable Phytoplankton Associations of the Gulf of Naples (Mediterranean Sea).” Marine Ecology
44, no. 3: e12730. 10.1111/maec.12730.</cite> [<a href="https://doi.org/10.1111/maec.12730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Marine%20Ecology&amp;title=Assembly%20Rules%20Vary%20Seasonally%20in%20Stable%20Phytoplankton%20Associations%20of%20the%20Gulf%20of%20Naples%20(Mediterranean%20Sea)&amp;author=A.%20Zingone&amp;author=C.%20Tortora&amp;author=D.%20D'Alelio&amp;author=F.%20Margiotta&amp;author=D.%20Sarno&amp;volume=44&amp;issue=3&amp;publication_year=2023&amp;pages=e12730&amp;doi=10.1111/maec.12730&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70033-bib-0127">
<cite>

Zorz, J.
, Willis C., Comeau A. M., et al. 2019. “Drivers of Regional Bacterial Community Structure and Diversity in the Northwest Atlantic Ocean.” Frontiers in Microbiology
10: 281. 10.3389/fmicb.2019.00281.
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.00281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6393369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30846975/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Drivers%20of%20Regional%20Bacterial%20Community%20Structure%20and%20Diversity%20in%20the%20Northwest%20Atlantic%20Ocean&amp;author=J.%20Zorz&amp;author=C.%20Willis&amp;author=A.%20M.%20Comeau&amp;volume=10&amp;publication_year=2019&amp;pages=281&amp;pmid=30846975&amp;doi=10.3389/fmicb.2019.00281&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>
<strong>Figure S1:</strong> Isolation of nanoplanktonic diatom cultures. Map showing the sampling locations from where three nanoplanktonic diatoms were isolated: <em>Minidiscus spinulatus</em>, collected from the BBMP in July 2018, and <em>Minidiscus trioculatus</em> and <em>Mediolabrus comicus</em>, both collected from the COOGER‐2021 in August 2021 (Mordret et al. 2024).</p>
<p>
<strong>Figure S2:</strong> Schematic representation of complete mitochondrial (a) and chloroplast (b) genomes from the three isolates, annotated using OGDRAW (Greiner et al. <a href="#jeu70033-bib-0034" class="usa-link" aria-describedby="jeu70033-bib-0034">2019</a>). Genetic elements displayed outside the ring are transcribed counterclockwise, while the elements inside the ring are transcribed clockwise. The color of the gene box indicates the type of gene. The GC‐content is shown on the inner ring of each of the genomes.</p>
<p>
<strong>Figure S3:</strong> Yearly identification of nanoplanktonic diatom isolates in environmental metabarcoding samples, including the sampling conducted in AS18 and AS19 in the upper upper photic zone (1–20 m). All of the “AF” samples were collected in the fall; the “AS”, “AZOMP” and “CE” samples were collected in the spring; and the “Arctic15” samples were collected in the summer (see Table 1). Abbreviations: AF, Atlantic Zone Monitoring Program (AZMP) Fall (usually Sept–Oct, but Nov–Dec in 2017); AS, AZMP Spring (Apr–Jun); AZOMP, Atlantic Zone Off‐shore Monitoring Program (Apr–Jun); BELAS‐1, Biological Carbon Export in the Labrador Sea (May–Jun); Arctic15, GEOTRACES mission to CAG (Jul–Aug).</p>
<p>
<strong>Figure S4:</strong> Nanoplanktonic diatom isolates tracked across the NWA and Canadian Arctic using <em>16S</em> metabarcoding of eDNA from any sample collected (1–2500 m; Niskin water samples and marine snow catcher (MSC) samples). Abbreviations: AF, Atlantic Zone Monitoring Program (AZMP) Fall (usually Sept–Oct, but Nov–Dec in 2017); AS, AZMP Spring (Apr–Jun); AZOMP, Atlantic Zone Off‐shore Monitoring Program (Apr–Jun); BELAS‐1, Biological Carbon Export in the Labrador Sea (May–Jun); Arctic15, GEOTRACES mission to CAG (Jul–Aug). Note numbers at the end of mission codes indicate year (see Table 1 for more information). For a more complete figure including limited sampling in AS18 and AS19, see Figure S3.</p>
<p>
<strong>Figure S5:</strong> Proportion of photosynthetic community contributed by nanoplanktonic isolates in the upper photic zone (1–20 m). This demonstrates the AZMP cruises that were completed in both Spring and Fall in order to determine a seasonal pattern for the proportion of the community that is each of the isolates. The environmental metabarcoding data is collected from various depths at each station using the <em>16S</em> V4V5 and <em>16S</em> V6V8 markers. Each data point is the proportion of the community that the ASV takes up in one sample, determined by the sample that had the highest proportion of that ASV. The heatmap represents the relative abundance (%) of the ASV to the rest of the photosynthetic community in that sample.</p>
<p>
<strong>Figure S6:</strong> Vertical distribution of the photosynthetic community in the Bedford Basin. The nanoplankton community was more abundant at 1 m depth compared to 60 m, but all three isolates were present throughout the water column at the four depths sampled.</p>
<p>
<strong>Figure S7:</strong> Proportional abundance of nanoplankton on 3 μM and 0.2 μM filters in Bedford Basin. Nanoplankton constituted a larger proportion of the community on the 3 μM filter compared to the 0.2 μM filter in the years sampled, although they were present in both size fractions.</p>
<p>
<strong>Figure S8:</strong> Distribution of nanoplankton isolates across depths in the Bedford Basin in relation to ocean conditions. Proportion of the community (Rel. Abun.), calculated from the % of the V4V5 region of the cp 16S rRNA, is shown for 1 m and 60 m depths over time (2014–2022) using 0.2‐μm filtered and reconstituted 3 and 0.2 μm size fractions. Environmental variables including temperature (temp), chlorophyll <em>a</em> (chl <em>a</em>), oxygen (O2), nitrate (NO3), nitrite (NO2), phosphate (PO4), silicate (SiO4), and salinity (sal) and flow cytometry counts for nanoeukaryotes (nano), picoeukaryotes (pico), cryptophytes (crypto), and <em>Synechococcus</em> (<em>Syn</em>) are plotted. Missing data is due to COVID‐19 restrictions.</p>
<p>
<strong>Figure S9:</strong> Month of the year versus relative abundance. Dot plot of the relative abundance of each isolate in the photosynthetic community by month from 2014 to 2022. Each dot represents one water sample collected during a select month, year, and depth.</p>
<p>
<strong>Figure S10:</strong> Boxplots showing the distribution of temperatures (°C) at which each species was detected based on non‐zero relative abundance. Significant differences in temperature distributions were detected among species (Kruskal–Wallis test: χ<sup>2</sup> = 461.34, <em>p</em> &lt; 2.2e‐16). Pairwise Wilcoxon rank‐sum tests with Benjamini–Hochberg correction indicated that all species pairs differed significantly. Boxes represent the interquartile range (IQR), with medians shown as horizontal lines; whiskers extend to 1.5× IQR. Outliers are not shown for clarity.</p>
<p>
<strong>Figure S11:</strong> Correlation of variables related to the RDA analyses. (a) Representation of the correlations between environmental variables before reducing the dataset to non‐correlated variables. (b) Reduced dataset used for initial RDA.</p>
<p>
<strong>Figure S12:</strong> BLAST comparison of the <em>Minidiscus spinulatus</em> genome with the previously sequenced genome (Grant et al. <a href="#jeu70033-bib-0033" class="usa-link" aria-describedby="jeu70033-bib-0033">2023</a>; Wang and Chen <a href="#jeu70033-bib-0114" class="usa-link" aria-describedby="jeu70033-bib-0114">2022</a>). The first two inner rings are the annotations of key components of the cp on the forward and reverse strands. The closest ring represents <a href="https://www.ncbi.nlm.nih.gov/nuccore/OL855995" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OL855995</a> <em>Minidiscus spinulatus</em> as downloaded from GenBank. The outer two genomes represent <em>Minidiscus trioculatus</em> and <em>Mediolabrus comicus</em> as a reference.</p>
<p>
<strong>Figure S13:</strong> Heatmap of the 44 protein‐coding genes outlined by Liu et al. (2021a) applied to the three new nanoplanktonic diatom isolates. Blue indicates gene presence, and white indicates gene absence.</p>
<p>
<strong>Data S1:</strong> Summary of Oxford Nanopore sequencing runs.</p>
<p>
<strong>Data S2:</strong> FASTA file of ASV sequences from eDNA metabarcoding aligned to cp 16S.</p>
<p>
<strong>Data S3:</strong> Output of Anvi‘o pangenome analyses.</p>
<p>
<strong>Data S4:</strong> Pairwise Average Nucleotide Identity (ANI) calculated with fastANI for each of the cp and mt genomes.</p>
<p>
<strong>Table S1:</strong> Cell sizes for each species included in Figure 1 with associated reference.</p>
<p>
<strong>Table S2:</strong> Mitochondrial genomes and associated cell sizes in the Order Thalassiosirales.</p>
<p>
<strong>Table S3:</strong> Chloroplast genomes and associated cell sizes in the Order Thalassiosirales.</p>
<p>
<strong>Table S4:</strong> Mitochondrial genes included in the concatenated phylogenetic tree of 16 Thalassiosirales mitochondrial genomes.</p>
<p>
<strong>Table S5:</strong> Chloroplast genes included in the concatenated phylogenetic tree of 37 Thalassiosirales chloroplast genomes.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12326293/bin/JEU-72-e70033-s001.zip" data-ga-action="click_feat_suppl" class="usa-link">JEU-72-e70033-s001.zip</a><sup> (14.2MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are available from the corresponding authors upon request. The isolated cultures are available in the AGSB culture collection maintained by the NRC in Halifax, NS.</p>
<p>Data from the eDNA metabarcoding of the Bedford Basin Monitoring Program from 2014 to 2017 can be found under this BioProject: <a href="https://www.ncbi.nlm.nih.gov/bioproject/785606" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/785606</a>.</p>
<p>Data from the eDNA metabarcoding of the Atlantic Zonal Monitoring Program can be found under this BioProject: <a href="https://www.ncbi.nlm.nih.gov/bioproject/785872" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/785872</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Journal of Eukaryotic Microbiology are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1111/jeu.70033"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/JEU-72-e70033.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (8.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12326293/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12326293/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12326293%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326293/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12326293/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12326293/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40765478/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12326293/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40765478/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12326293/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12326293/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="WQXpb7BM9bZL0ELdUefEW2iPBfyqsSGnqMPLmDxhPkGKIFAtKzUHgd6qb5bxlIuT">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
