
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            DNA glycosylases Ogg1 and Mutyh influence gene expression of PRC2 targets associated with cognition - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="67D7146689B748931C1466002FB30799.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="cmls">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334403/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Cellular and Molecular Life Sciences: CMLS">
<meta name="citation_title" content="DNA glycosylases Ogg1 and Mutyh influence gene expression of PRC2 targets associated with cognition">
<meta name="citation_author" content="Andreas Abentung">
<meta name="citation_author_institution" content="Department of Clinical and Molecular Medicine, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology (NTNU), Trondheim, Norway">
<meta name="citation_author_institution" content="Department of Neurology and Clinical Neurophysiology, University Hospital Trondheim, Trondheim, Norway">
<meta name="citation_author" content="Teri Sakshaug">
<meta name="citation_author_institution" content="Department of Neuromedicine and Movement Science, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology (NTNU), Trondheim, Norway">
<meta name="citation_author" content="Rabina Dumaru">
<meta name="citation_author_institution" content="Department of Clinical and Molecular Medicine, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology (NTNU), Trondheim, Norway">
<meta name="citation_author" content="Nina-Beate Liabakk">
<meta name="citation_author_institution" content="Department of Clinical and Molecular Medicine, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology (NTNU), Trondheim, Norway">
<meta name="citation_author" content="Mingyi Yang">
<meta name="citation_author_institution" content="Department of Microbiology, Oslo University Hospital HF, Rikshospitalet and University of Oslo, Oslo, Norway">
<meta name="citation_author" content="Junbai Wang">
<meta name="citation_author_institution" content="Department of Clinical Molecular Biology (EpiGen), University of Oslo and Akershus University Hospital, Akershus, Norway">
<meta name="citation_author" content="Magnar Bjørås">
<meta name="citation_author_institution" content="Department of Clinical and Molecular Medicine, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology (NTNU), Trondheim, Norway">
<meta name="citation_author_institution" content="Department of Microbiology, Oslo University Hospital HF, Rikshospitalet and University of Oslo, Oslo, Norway">
<meta name="citation_author_institution" content="Centre for Embryology and Healthy Development, University of Oslo, Oslo, Norway">
<meta name="citation_author" content="Katja Scheffler">
<meta name="citation_author_institution" content="Department of Neuromedicine and Movement Science, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology (NTNU), Trondheim, Norway">
<meta name="citation_author_institution" content="Department of Neurology and Clinical Neurophysiology, University Hospital Trondheim, Trondheim, Norway">
<meta name="citation_publication_date" content="2025 Aug 8">
<meta name="citation_volume" content="82">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="307">
<meta name="citation_doi" content="10.1007/s00018-025-05730-9">
<meta name="citation_pmid" content="40779073">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334403/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334403/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334403/pdf/18_2025_Article_5730.pdf">
<meta name="description" content="Base excision repair, initiated by DNA glycosylases, preserves genomic integrity by removing damaged bases. DNA glycosylases Ogg1 and Mutyh were shown to alter the hippocampal transcriptome independently of DNA damage repair. However, the role of ...">
<meta name="og:title" content="DNA glycosylases Ogg1 and Mutyh influence gene expression of PRC2 targets associated with cognition">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Base excision repair, initiated by DNA glycosylases, preserves genomic integrity by removing damaged bases. DNA glycosylases Ogg1 and Mutyh were shown to alter the hippocampal transcriptome independently of DNA damage repair. However, the role of ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334403/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12334403">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1007/s00018-025-05730-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/18_2025_Article_5730.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12334403%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12334403/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12334403/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334403/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-cmls.png" alt="Cellular and Molecular Life Sciences: CMLS logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Cellular and Molecular Life Sciences: CMLS" title="Link to Cellular and Molecular Life Sciences: CMLS" shape="default" href="https://doi.org/10.1007/s00018-025-05730-9" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Cell Mol Life Sci</button></div>. 2025 Aug 8;82(1):307. doi: <a href="https://doi.org/10.1007/s00018-025-05730-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1007/s00018-025-05730-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Cell%20Mol%20Life%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cell%20Mol%20Life%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Cell%20Mol%20Life%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Cell%20Mol%20Life%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>DNA glycosylases Ogg1 and Mutyh influence gene expression of PRC2 targets associated with cognition</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abentung%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Andreas Abentung</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Andreas Abentung</span></h3>
<div class="p">
<sup>1</sup>Department of Clinical and Molecular Medicine, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology (NTNU), Trondheim, Norway </div>
<div class="p">
<sup>3</sup>Department of Neurology and Clinical Neurophysiology, University Hospital Trondheim, Trondheim, Norway </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abentung%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andreas Abentung</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sakshaug%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Teri Sakshaug</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Teri Sakshaug</span></h3>
<div class="p">
<sup>2</sup>Department of Neuromedicine and Movement Science, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology (NTNU), Trondheim, Norway </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sakshaug%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Teri Sakshaug</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dumaru%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Rabina Dumaru</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Rabina Dumaru</span></h3>
<div class="p">
<sup>1</sup>Department of Clinical and Molecular Medicine, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology (NTNU), Trondheim, Norway </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dumaru%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rabina Dumaru</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liabakk%20NB%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Nina-Beate Liabakk</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Nina-Beate Liabakk</span></h3>
<div class="p">
<sup>1</sup>Department of Clinical and Molecular Medicine, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology (NTNU), Trondheim, Norway </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liabakk%20NB%22%5BAuthor%5D" class="usa-link"><span class="name western">Nina-Beate Liabakk</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Mingyi Yang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Mingyi Yang</span></h3>
<div class="p">
<sup>4</sup>Department of Microbiology, Oslo University Hospital HF, Rikshospitalet and University of Oslo, Oslo, Norway </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mingyi Yang</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Junbai Wang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Junbai Wang</span></h3>
<div class="p">
<sup>5</sup>Department of Clinical Molecular Biology (EpiGen), University of Oslo and Akershus University Hospital, Akershus, Norway </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junbai Wang</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bj%C3%B8r%C3%A5s%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Magnar Bjørås</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Magnar Bjørås</span></h3>
<div class="p">
<sup>1</sup>Department of Clinical and Molecular Medicine, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology (NTNU), Trondheim, Norway </div>
<div class="p">
<sup>4</sup>Department of Microbiology, Oslo University Hospital HF, Rikshospitalet and University of Oslo, Oslo, Norway </div>
<div class="p">
<sup>6</sup>Centre for Embryology and Healthy Development, University of Oslo, Oslo, Norway </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bj%C3%B8r%C3%A5s%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Magnar Bjørås</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scheffler%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Katja Scheffler</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Katja Scheffler</span></h3>
<div class="p">
<sup>2</sup>Department of Neuromedicine and Movement Science, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology (NTNU), Trondheim, Norway </div>
<div class="p">
<sup>3</sup>Department of Neurology and Clinical Neurophysiology, University Hospital Trondheim, Trondheim, Norway </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scheffler%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Katja Scheffler</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Clinical and Molecular Medicine, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology (NTNU), Trondheim, Norway </div>
<div id="Aff2">
<sup>2</sup>Department of Neuromedicine and Movement Science, Faculty of Medicine and Health Sciences, Norwegian University of Science and Technology (NTNU), Trondheim, Norway </div>
<div id="Aff3">
<sup>3</sup>Department of Neurology and Clinical Neurophysiology, University Hospital Trondheim, Trondheim, Norway </div>
<div id="Aff4">
<sup>4</sup>Department of Microbiology, Oslo University Hospital HF, Rikshospitalet and University of Oslo, Oslo, Norway </div>
<div id="Aff5">
<sup>5</sup>Department of Clinical Molecular Biology (EpiGen), University of Oslo and Akershus University Hospital, Akershus, Norway </div>
<div id="Aff6">
<sup>6</sup>Centre for Embryology and Healthy Development, University of Oslo, Oslo, Norway </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 29; Revised 2025 Apr 8; Accepted 2025 Apr 29; Collection date 2025 Dec.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12334403  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40779073/" class="usa-link">40779073</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Base excision repair, initiated by DNA glycosylases, preserves genomic integrity by removing damaged bases. DNA glycosylases Ogg1 and Mutyh were shown to alter the hippocampal transcriptome independently of DNA damage repair. However, the role of DNA glycosylases as transcriptional regulators and their involvement in cognition remain unclear. Here, we combine transcriptomic and epigenomic analyses of the hippocampus in mice deficient in DNA glycosylases. We report that the combined deficiency of Ogg1 and Mutyh impairs spatial long-term memory. Mechanistically, Ogg1 and Mutyh modulate DNA methylation of polycomb repressive complex 2 (PRC2) target genes. PRC2 occupancy and associated histone post-translational modifications depend on Ogg1 and Mutyh in neurons and glia. These epigenetic changes correlate with cell-type specific differences in gene expression of PRC2 targets. Finally, human genetic data link DNA glycosylase-regulated genes to cognitive function, highlighting a novel role for Ogg1 and Mutyh in modulating the epigenome to control transcriptional responses relevant for brain-related diseases.</p>
<section id="sec1"><h3 class="pmc_sec_title">Graphical Abstract</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334403_18_2025_5730_Figa_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ac4/12334403/3168651f53c3/18_2025_5730_Figa_HTML.jpg" loading="lazy" id="d33e255" height="686" width="685" alt="graphic file with name 18_2025_5730_Figa_HTML.jpg"></a></p>
<p id="Par245">Created in BioRender. Scheffler, K. (2025). <a href="https://BioRender.com/jwscej4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/jwscej4</a>.</p></section><section id="sec2"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1007/s00018-025-05730-9.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> DNA glycosylase, PRC2, Epigenome, Transcription, Cognition</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">The canonical functions of DNA repair play an essential role in maintaining genomic integrity and stability across organisms [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Genomic instability caused by an accumulation of DNA damage is a hallmark of aging [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>] and has been linked to DNA mutations, altered gene expression and cognitive impairment in the human brain [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Due to its high metabolic rate, high transcriptional activity and long lifespan, brain cells are highly susceptible to DNA damage [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Consistently, mutations in DNA repair genes and impaired DNA repair pathways are associated with pre-mature aging and various neurodegenerative diseases [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Base excision repair (BER) initiated by DNA glycosylases is the major pathway for removal of oxidized bases in the DNA [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. Given its low redox potential, the DNA base guanine is particularly susceptible to oxidation, leading to the formation of the pre-mutagenic base lesion 8-oxoguanine (8-oxoG) [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. 8-oxoG DNA glycosylase 1 (Ogg1) recognizes and facilitates the removal of 8-oxoG within the genome through BER [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Adenine DNA glycosylase (Mutyh) excises mis-incorporated adenine opposite of 8-oxoG during replication, allowing Ogg1 to remove 8-oxoG after insertion of the correct base [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>].</p>
<p id="Par4">Emerging evidence indicates that DNA glycosylases may have additional functions beyond canonical DNA repair by altering the epigenetic landscape and thereby regulating transcription [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. In the mouse brain, Ogg1 and Mutyh regulate gene expression related to learning and memory independent of global accumulation of 8-oxoG [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. In line with these findings, catalytically inactive Ogg1 promotes pro-inflammatory gene expression in other cell-types, highlighting the significant role of DNA glycosylases beyond 8-oxoG repair [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Moreover, several DNA glycosylases were identified as potential readers of epigenetic DNA modifications [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] and have been implicated in active DNA demethylation [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Additionally, DNA glycosylases have been shown to interact with several epigenetic modifiers such as DNA methyltransferases [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>], histone demethylases [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>] and members of the polycomb repressor complex 2 (PRC2) [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p>
<p id="Par5">It has also been demonstrated that 8-oxoG itself exhibits epigenetic-like properties by modulating gene transcription through its accumulation in promoter regions [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. Recent evidence indicates that both 8-oxoG and DNA glycosylases alter gene expression independently of one another [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Nevertheless, the molecular mechanisms through which DNA glycosylases affect brain health, particularly in relation to cognition, are still not fully understood.</p>
<p id="Par6">In this study, we investigate the impact of Ogg1 and Mutyh on the epigenetic landscape in the adult mouse hippocampus. We show that loss of DNA glycosylases affects spatial but not associative long-term memory. Mechanistically, we demonstrate that Ogg1 and Mutyh influence gene expression by modulating DNA methylation levels at regulatory regions of predominantly PRC2 target genes, as well as histone modifications linked to PRC2 activity, in both neurons and glia. In human genetic data, we find genes regulated by DNA glycosylases to be enriched for common variants associated with cognitive function and brain-related diseases. Taken together, our results reveal a previously unrecognized role of DNA glycosylase beyond DNA repair by mediating gene expression important for higher human brain function and health.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Animals</h3>
<p id="Par7">Mice were housed and bred at the Comparative Medicine Core Facility (Trondheim, Norway) in a 12-hour light/dark cycle with access to food and water <em>ad libitum</em> under standard laboratory conditions of 19–22 °C with air humidity of 50–60%. Ogg1 and Mutyh deficient C57BL/6N mice were used for all the experiments included in this study. Double knockout (DKO) mice were obtained by crossing <em>Ogg1</em><sup><em>-\-</em></sup> and <em>Mutyh</em><sup><em>-\-</em></sup> mice and wild type (WT) C57BL/6N mice were used as control groups. For all experiments 6 months old male mice were used. Mouse litters were genotyped using genomic DNA obtained from ear biopsies. All knockout strains have been previously described elsewhere [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>].</p></section><section id="Sec4"><h3 class="pmc_sec_title">Contextual fear conditioning</h3>
<p id="Par8">Mice were handled 5 min for three days prior to the experiment. Contextual fear conditioning was performed in a 17 × 17 × 25 cm chamber with transparent walls and a metal rod floor (Ugo Basile, Fear Conditioning Animal System, 46001), cleaned with soap-water and illuminated to 150 lx, as previously described [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. After a 120 s acclimation period, mice were conditioned with three presentations of a 0.60 mA scrambled foot shock, with a 120 s inter-shock interval. The mice were allowed to remain in the chamber for an additional 120 s following the last stimulus presentation. Short-term and long-term fear memories were assessed by re-exposing the mice for 300 s either 1 h or 24 h later to the conditioning chamber without the administration of a foot shock. Freezing was measured as an index of fear [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>], defined as no visible movement except that required for respiration. Freezing time was scored and analyzed with Any-Maze software (Stoelting Europe) and converted to a percentage [(duration of freezing /total time) × 100].</p></section><section id="Sec5"><h3 class="pmc_sec_title">Flinch-jump test</h3>
<p id="Par9">Reactivity to the foot shock was evaluated in the same apparatus used for contextual fear conditioning. After a 120 s acclimation period, mice were subjected to a series of foot shocks with gradually increasing amperage (0.1 mA every 30 s) starting from 0.1 mA. Mice were scored for their first visible response to the shock (flinch), their first pronounced motor response (run or jump) and their first vocalized distress, as previously described [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>].</p></section><section id="Sec6"><h3 class="pmc_sec_title">Novel object location</h3>
<p id="Par10">Animals were housed under a 12-hour reversed light/dark cycle for at least a week before the start of the paradigm. Prior to training, mice were handled 5 min for five consecutive days and then habituated to the experimental apparatus (50 × 50 × 30 cm open field area) for 5 min for three consecutive days in the absence of objects, as previously described [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. During training, mice were placed into the experimental apparatus containing two identical objects (Duplo blocks) and allowed to explore for 10 min. Short-term and long-term retention tests were conducted by placing the mice back into the experimental apparatus for 10 min either 1 h or 24 h after training. To assess spatial object location-dependent memory, the placement of one familiar object remained unchanged whereas the other familiar object was moved and placed adjacent to a different corner inside the experimental apparatus. Exploration was scored when the head of the animal entered the virtual zone adjacent to the object. All training and testing trials were videotaped and analyzed with Any-Maze software (Stoelting Europe). The relative exploration time (t) was expressed as a percent discrimination index (D.I. = tnovel / (tnovel + tfamiliar) × 100%). Mean exploration times were then calculated and the discrimination indices between experimental groups were compared. Animals that explored each object for less than 3 s during training or explored both objects for less than 10 s total during testing were excluded from the analysis.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Tissue collection</h3>
<p id="Par11">Animals were sacrificed by cervical dislocation and hippocampi were dissected, isolated, snap-frozen in liquid nitrogen and stored at − 80 °C until further use. All mice included in high throughput sequencing experiments were re-genotyped by using genomic DNA obtained from tail biopsies after being sacrificed.</p></section><section id="Sec8"><h3 class="pmc_sec_title">DNA isolation</h3>
<p id="Par12">Total DNA was extracted from snap-frozen hippocampi by using AllPrep DNA/RNA/Protein mini kit (Qiagen, 80004), according to manufacturer’s protocol. Frozen tissue was transferred to tubes with 1.4 mm stainless steel beads containing 350 ml buffer RLT and homogenized using a Roche MagNA Lyser rotor (5000 rpm, 10 s). Samples were further processed according to the respective protocol. DNA concentrations were measured by using a ND-1000 spectrophotometer (Nanodrop Technologies).</p></section><section id="Sec9"><h3 class="pmc_sec_title">Whole genome bisulfite-sequencing</h3>
<p id="Par13">Total DNA from two replicates of each genotype was sent to BGI Tech Solutions Co., Hong Kong. Bisulfite conversion, library preparation and DNA sequencing was performed by BGI as previously described [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Briefly, total hippocampal DNA was fragmented by sonication using a Bioruptor (Diagenode) to a mean size of approximately 250 bp, followed by blunt-ending and dA addition to 3’-end. Finally, adaptor ligation (in this case, with methylated adaptors to protect from bisulfite conversion) was performed according to the manufacturer’s instructions. Ligated DNA was bisulfite converted using the EZ DNA Methylation-Gold kit (Zymo research, D5005). After treatment with sodium bisulfite, unmethylated cytosine residues were converted to uracil whereas 5-methylcytosine (5mC) remains unaffected. Different insert size fragments were excised from the same lane of a 2% TAE agarose gel. Products were purified by using QIAquick Gel Extraction kit (Qiagen, 28704) and amplified by PCR. After PCR amplification, uracil residues were converted to thymine. Pooled libraries were sequenced (paired-end, 100 bp in length) on a BGISEG-500 sequencer with an average depth of 1.3 billion reads per sample.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Fluorescence activated nuclear sorting (FANS)</h3>
<p id="Par14">Hippocampal nuclei were extracted, isolated and sorted as previously described, with some minor modifications [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. In brief, snap-frozen hippocampi from 5 animals of the same genotype were pooled in 1 ml of low sucrose buffer [(0.32 M sucrose, 10 mM HEPES pH 8.0, 5 mM CaCl<sub>2</sub>, 3 mM Mg(CH<sub>3</sub>COO)<sub>2</sub>, 0.1 mM EDTA, 0.1% Triton X-100, 1 mM DTT, 1x Complete EDTA free protease inhibitor cocktail (Roche, 11873580001) and 20 U/ml of RNase Inhibitor (Applied Biosystems, N8080119)]. All steps except the crosslinking were performed at 4 °C or on ice. Hippocampal tissue was homogenized by applying pestle “loose” and “tight” 40 times, respectively. For nuclei fixation, formaldehyde was added at a final concentration of 1% and the samples were incubated at room temperature for 10 min on a rotator. The crosslinking reaction was stopped by adding glycine to a final concentration of 125 mM for 5 min at room temperature. Nuclei were then pelleted by centrifugation, re-suspended and washed twice with 1 ml of low sucrose buffer. Subsequently, nuclei were purified through a sucrose cushion (10 mM HEPES pH 8.0, 1 M sucrose, 3 mM Mg(CH<sub>3</sub>COO)<sub>2</sub>, 1 mM DTT, 6 ml of cushion for 1.5 ml of lysate) by centrifugation at 3200 g for 10 min in a 15 ml Falcon tube. Afterwards, nuclei were re-suspended in nuclei buffer (PBS, 0.1% Tween 20 in PBS, 5% BSA, 1x Complete EDTA free protease inhibitor cocktail (Roche, 11873580001) and 20 U/ml of RNase Inhibitor (Applied Biosystems, N8080119)) containing 3% goat serum and cleared by filtering through a 40 μm cell strainer. The nuclei suspension was stored on 4 °C overnight. On the next day, nuclei were stained with anti-NeuN mouse antibody (Millipore, mab377) diluted 1:500 for 60 min at 4 °C. The samples were then washed twice with nuclei buffer containing 3% goat serum. Subsequently, the samples were stained for 60 min at 4 °C with a secondary antibody (Alexa 488, Life Technologies, A11001) diluted 1:1000. Nuclei were then washed twice with nuclei buffer and stored until sorting. Directly before sorting, nuclei were dissociated by passing them 10 times through a 25G needle and by filtering through a 40 μm cell strainer into ice-cold conical tubes containing 1.8 ml of filtered PBS-BSA (5%). Lastly, 2 µl of Vybrant™ DyeCycle™ Ruby (V10273, Life Technologies) was added to the nuclei suspension. Nuclei were sorted using an 85 μm nozzle on a FACSAria II (BD Bioscience). Nuclei selection was based on DyeCycle Ruby staining, singlet nuclei detection and NeuN + positive staining. Both NeuN stained (NeuN+) and unstained (NeuN−) fractions were collected. The average purity of the sorted nuclei exceeded 95%, yielding highly cell-type specific material. The NeuN positive population contains every neuron expressing NeuN endogenously, which are primary excitatory neurons as well as interneurons. NeuN negative cells consist predominantly of glial cells but also contains other cell-types. Flow cytometry data were analyzed and plotted by using FlowJo (BD Bioscience).</p></section><section id="Sec11"><h3 class="pmc_sec_title">Chromatin Immunoprecipitation (ChIP)</h3>
<p id="Par15">Sorted nuclei were pelleted by centrifugation at 3200 g for 15 min, transferred to Bioruptor Microtubes (Diagenode, C30010016) shearing tubes and carefully re-suspended in RIPA buffer (10 mM Tris-Cl, pH 8.0, 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% sodium deoxycholate, 1% SDS and 1x Complete EDTA free protease inhibitor cocktail (Roche, 11873580001). Samples were incubated for 10 min at 4 °C and then sheared using a Bioruptor (Diagenode) 4 times at 5 cycles and followed by 2 cycles at 30″ ON/OFF high power. Samples were spun down in between every 5 cycles. Sheared chromatin was cleared by centrifugation at 10000 rpm for 10 min and the supernatant was stored in DNA low-binding tubes (Eppendorf, 022431021). To confirm the successful fragmentation of chromatin, DNA from a small aliquot of each sample was purified by phenol-chloroform extraction. The size of the DNA fragments was analyzed through gel electrophoresis on a 1.6% Agarose gel with GelRed<sup>®</sup> nucleic acid stain (Biotium, 41003) and the DNA concentration was determined by a ND-1000 spectrophotometer (Nanodrop Technologies). The rest of the chromatin was snap-frozen and stored at 80 °C until further use. For chromatin immunoprecipitation, sheared chromatin was pre-cleared with BSA-blocked protein G magnetic beads (Dynabeads, Invitrogen, 10004D) for 1 h at 4 °C. For every ChIP-reaction the appropriate amount of chromatin, depending on the different antibodies, was used (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). ChIP-reactions were performed by overnight incubation on a rotating wheel at 4 °C in ChIP buffer (50 mM Tris-HCl at pH 8.0, 150 mM NaCl, 1% NP-40, 0.5% sodium deoxycholate, 20 mM EDTA, 1x Complete EDTA free protease inhibitor cocktail). As input control a 10% aliquot of each sheared chromatin sample was saved and stored at -80 °C. Subsequently, 30 µl of BSA-blocked protein G magnetic beads were added to each ChIP-reaction and the mixture was incubated on a rotator at 4 °C for 4 h. The chromatin-bead complexes were washed twice at 4 °C for 5 min at slow rotation (15 rpm) with ChIP buffer containing 0.1% SDS, twice with wash buffer (100 mM Tris-HCl pH 8.0, 500 mM LiCl, 1% NP-40, 1% sodium deoxycholate, 20 mM EDTA) and twice with TE buffer (10 mM Tris-HCl pH 8.0, 1 mM EDTA). For reverse crosslinking, chromatin-bead complexes and 10% input controls were resuspended in 300 µl elution buffer (100 mM NaHCO<sub>3</sub>, 1% SDS) with 200 mM NaCl and incubated at 65 °C overnight. On the next day, the samples were treated with 1 µl of RNAse A (Thermo Scientific, EN0531) for 1 h at 37 °C. Subsequently, 6 µl 0.5 M EDTA and 1 µl Proteinase K (Roche, 3115828001) were added and the samples were incubated at 45 °C for 1 h. DNA was then purified by phenol-chloroform extraction and samples were eluted in 16 µl Tris buffer (10 mM Tris-HCl pH 8.0) in a DNA low-binding tube and stored at − 80 °C.</p></section><section id="Sec12"><h3 class="pmc_sec_title">ChIP-sequencing</h3>
<p id="Par16">DNA concentrations for ChIP reactions and inputs were determined by using a Qubit dsDNA HS Assay Kit (Thermo Fisher, <a href="https://www.ncbi.nlm.nih.gov/protein/Q32851" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q32851</a>). Library construction was performed by using QIAseq™ Ultralow Input Library Kits (Qiagen, 180492) according to manufacturer’s protocol. The final libraries were purified by using AMPure XP beads (Beckman Coulter, A63880) and eluted in 23 µl nuclease free water. DNA fragment size was determined by using an Agilent 2100 Bioanalyzer and the concentration of the final libraries was measured with a Qubit dsDNA HS Assay Kit (Thermo Fisher, <a href="https://www.ncbi.nlm.nih.gov/protein/Q32851" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q32851</a>). Libraries were further validated by qPCR using a KAPA Library Quantification Kit (Roche, KK4824) and pooled before being sequenced. Sequencing (single-end, 75 bp) was performed at the Genomic Core Facility (GCF, Trondheim) using an Illumina NextSeq500 sequencer with an average depth of 20 million reads per sample.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Nuclear RNA-sequencing</h3>
<p id="Par17">Hippocampal nuclei were pooled from 3 animals per genotype. Nuclear RNA of NeuN + and NeuN- nuclei was isolated by using the RNeasy<sup>®</sup> FFPE Kit (Qiagen, 73504) according to manufacturer’s protocol. RNA concentration and quality was determined using Qubit RNA Assay Kits (Thermo Fisher, <a href="https://www.ncbi.nlm.nih.gov/protein/Q32852" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q32852</a> and <a href="https://www.ncbi.nlm.nih.gov/protein/Q10210" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q10210</a>) and an Agilent 2100 Bioanalyzer, respectively. Libraries were prepared by using a SMARTer<sup>®</sup> Stranded Total RNA-Seq Kit (Takara, 634485) according to the manufacturer’s instructions. Libraries were pooled and sent to BGI Tech Solutions Co., Hong Kong. Sequencing (paired-end, 100 bp) was performed using a MGISEQ2000 sequencer with an average depth of 50 million reads per sample.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Genomic data processing</h3>
<section id="Sec15"><h4 class="pmc_sec_title">Bisulfite-seq data</h4>
<p id="Par18">After trimming and FastQC analysis, the clean GWBS data were aligned to the mouse reference genome GRCm38, followed by a separate extraction of genome-wide 5mC in the context of CG, CHG and CHH by using Bismark [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] (version 0.22.3). The 5mC count data was cleaned by filtering out low reads of coverage &lt; 4 in each cytosine. The differentially methylated regions (DMRs) were identified by HMST-Seq-Analyzer [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. In brief, methylation regions were extracted from predefined genomic areas (e.g. TSS, TES, and gene body) and DMRs were predicted by robust statistical tests (Wilcoxon rank-sum test) between samples. The parameter setting was set to default for gene feature regions, requiring at least 3 mC sites in the TSS and TES, 5mC sites in the gene body, and a maximum distance of 200 bp between adjacent mC sites within a region. Since HMST-Seq-Analyzer is only suited for two sample comparison: replicates of all knockouts (KO1, KO2) were compared to both replicates from WT control samples. The common DMRs among the 4 pair-wise comparisons were defined as the true DMRs. The methylation difference is presented as relative ratio (rratio), which is defined by: &lt; methylation level in KO&gt;– &lt; methylation level in WT&gt; ) / &lt; mean methylation of KO and WT&gt;. DMRs were filtered by rratio &gt; 0.1 as common hyper, and by rratio &lt; -0.1 as common hypo. The methylation profile plots were generated by ggplot2 (version 3.4.0) using the default smoothing method of auto detection in R package. Analysis of upstream regulators of DMRs was done using Ingenuity Pathway Analysis (IPA; Qiagen) and PRC2 gene lists were exported for profile plots in ggplot2.</p></section><section id="Sec16"><h4 class="pmc_sec_title">ChIP-seq data</h4>
<p id="Par19">All datasets were quality assessed using FastQC and FastQ Screen. Fastq reads were aligned to the GRCm38/mm10 genome using Bowtie2 [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>] and files were processed using samtools [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Visualization of bigwigs for genome tracks and profile plots were generated from bam files using deepTools [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. Correlation heatmap and profile plots were generated using deepTools and profile plot values were exported and analyzed using Graph Pad Prism (GraphPad Software, Inc., La Jolla, CA, v10). Peak calling was performed using epic2 [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] with an FDR cutoff of 0.05. Window and gap sizes were 100 bp, 3 gaps for H3K4me3, 200 bp and 3 gaps for H3K27me3 and 50 bp, 2 gaps for Suz12. Reads were counted over consensus peaks, which were defined as peaks present in at least two of three replicates, using the Bioconductor package DiffBind (v.3.10.1) in R (v.4.3.2). Differential enrichment analysis was performed using edgeR (v.3.42.4) between genotypes. Differentially enriched regions (DERs) were annotated to the closest gene and assigned to genomic features using ChIPseeker [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] (v1.36). Volcano plots were generated using the Bioconductor package EnhancedVolcano (v. 1.18.0) in R where differentially enriched regions were defined as having a p-value &lt; 0.05 and an absolute log2FC &gt; 0.5. Pathway analysis for DERs was performed using IPA (Qiagen), pathway heatmaps were made using GraphPad Prism (GraphPad Software, Inc., La Jolla, CA, v10) and all Venn diagrams were made using Venny [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. RPKM values were counted over all DERs using DiffBind and exported to Perseus [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>] where values were z-scored and clustered using average linkage and Euclidean distance.</p></section><section id="Sec17"><h4 class="pmc_sec_title">RNA-seq data</h4>
<p id="Par20">RNAseq datasets were quality assessed using FastQC and FastQ Screen before processing. Paired-end reads were aligned to the GRCm38/mm10 genome using Hisat2 [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>] and files were processed using samtools [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Visualization of bigwigs for genome tracks were generated from bam files using deeptools [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. Quantification of properly aligned read pairs was done using the Rsubread package [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>] in R (v.4.3.2) and the Ensembl GRCm38 release 102 annotation. Bioconductor package edgeR [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>] (v. 3.42.4) was used to normalize counts across libraries and FPKMs were exported for plotting. Identification of differentially enriched genes (DEGs) between genotypes was performed using edgeR where DEGs were defined as having an FDR &lt; 0.05 and an absolute log2FC &gt; 0.5. Volcano plots were created in R using the Bioconductor package EnhancedVolcano (v. 1.18.0) and genotypes were overlayed using Adobe Photoshop. Heatmaps were generated from FPKMs of DEGs using Perseus [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>] where rows were z-scored and hierarchical clustering was performed using average linkage and Euclidean distance. Pathway analysis of DEGs was done using Ingenuity Pathway Analysis (IPA; Qiagen) and heatmaps were made using GraphPad Prism (GraphPad Software, Inc., La Jolla, CA, v10). All Venn diagrams were made using Venny [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>].</p></section><section id="Sec18"><h4 class="pmc_sec_title">PRC2 overrepresentation analysis</h4>
<p id="Par21">PRC2 gene lists were obtained from ChIP Atlas [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Lists for target genes of Suz12, Eed and Ezh2 in mus musculus were used ± 1k from TSS. Lists were filtered for mean peak signal score ≥ 25 and compared to differential analysis result lists from RNA, ChIP and methylation. The expected fraction was calculated as the number of PRC2 genes (<em>n</em> = 8507) over the number of all genes (<em>n</em> = 55204) and the observed fraction was calculated as the number of PRC2 differentially enriched genes over the number of total differentially enriched genes for each individual comparison. The observed/expected ratio was then calculated as the observed fraction over the expected fraction. P values were determined using Fishers exact test.</p></section><section id="Sec19"><h4 class="pmc_sec_title">Correlation analysis</h4>
<p id="Par22">Log2FPKM values from RNA-seq were calculated from the normalized count matrix with edgeR correcting for gene length. Transcription start sites (TSS) for all genes in the RNA count matrix were extracted from biomaRt in R and ChIP-seq reads were counted over 1000 bp bins centered on the TSS using bedtools (v2.27.1). CPMs were calculated using edgeR and log2 read values were used to compute Pearson’s correlation coefficient and generate scatterplots using the R CRAN package ggpubr/ggplot2 (v.0.6/v.3.5).</p></section><section id="Sec20"><h4 class="pmc_sec_title">MAGMA gene set analysis</h4>
<p id="Par23">The gene set analysis tool MAGMA (v.1.10, de Leeuw, 2015) was utilized to compare genetic marker ensembles of human phenotypes to the RNA-seq results. First SNPs were annotated to genes using the GRCh37 (100 kb upstream, 20 kb downstream). Gene sets were created from genes showing a fold-change beyond ± 0.2 and a P. value ≤ 0.05 in KOs compared to WT. Mouse genes were annotated to human using a combination of biomaRt (v. 2.56.1) and NCBI tools (datasets version 16.26.0) where on average 88.3% of genes across all gene sets were mapped to human orthologs. Summary statistics for GWAS data was obtained for cognitive performance [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] based on 257,828 individuals, intelligence [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>] based on 269,867 individuals, hippocampal volume [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] based on 21,297 individuals, bipolar disorder [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>] based on 398,495 individuals, stroke [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>] based on 524,354 individuals, dementia [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>] based on 362,647 individuals, autism spectrum disorder [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>] based on 18,382 individuals, and type 2 diabetes [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>] based on 659,319 individuals. Gene-wise metrics were calculated for each gene in the GWAS data using the SNP-wise mean model, which aggregates SNP-level association statistics using the mean of all SNPs mapped to the same gene. These metrics are then adjusted for linkage disequilibrium using the European panel of the 1000 Genome data as a reference panel producing a gene-level p-value within the GWAS dataset. Competitive gene set analysis was then performed testing the association of genes within the KO gene sets to genes within the GWAS datasets to identify a stronger association with phenotype associated genes over non-associated genes.</p></section></section><section id="Sec21"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par24">Statistical analysis was conducted as indicated in the figure legends using GraphPad Prism (GraphPad Software, Inc., La Jolla, CA, v10). Statistical significance was determined by two-tailed Student’s t-test and one-way ANOVA with Šídák´s multiple comparison test. Data represents mean ± SEM of at least three independent biological replicates. No statistical methods were used to predetermine sample size. The data distribution was assumed to be normal, but it was not formally tested.</p></section></section><section id="Sec22"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec23"><h3 class="pmc_sec_title">Loss of DNA glycosylases impairs spatial but not associative long-term memory</h3>
<p id="Par25">To explore a potential involvement of DNA glycosylases Ogg1 and Mutyh in the regulation of short- and long-term memory, we employed DNA glycosylase-deficient mice, which have previously been characterized [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. First, we subjected 6 months old male wild type (WT), single- (<em>Ogg1</em><sup><em>-/-</em></sup> or <em>Mutyh</em><sup>-/-</sup>) and double-knockout (<em>Ogg1</em><sup>-/-</sup> x <em>Mutyh</em><sup>-/-</sup>, DKO) mice to contextual fear conditioning (CFC), a hippocampus-dependent paradigm for associative learning (Fig. <a href="#Fig1" class="usa-link">1</a>A). DNA glycosylase-deficient mice exhibited typical behavioral and physiological reactions during fear conditioning by displaying a normal response to electric foot shock exposure (Supplementary Fig. <a href="#MOESM1" class="usa-link">1A</a>) and normal fear acquisition (Supplementary Fig. <a href="#MOESM1" class="usa-link">1B</a>). Short-term and long-term contextual fear memory, assessed 1 h and 24 h after training respectively, remained unaffected in DNA glycosylase-deficient mice (Fig. <a href="#Fig1" class="usa-link">1</a>B i and ii), suggesting that DNA glycosylases are not required for the consolidation of associative memories.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ac4/12334403/65e4b57d9a09/18_2025_5730_Fig1_HTML.jpg" loading="lazy" id="d33e740" height="1575" width="784" alt="Fig. 1"></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Loss of DNA glycosylases impairs spatial but not associative long-term memory</p>
<p> (<strong>A</strong>) Experimental design of the contextual fear conditioning (CFC) paradigm. (<strong>B</strong>) Performance in the CFC task measured as freezing 1–24 h after training. At the 1 h (i) as well as at the 24 h (ii) training fear expression test <em>Mutyh</em><sup>-/-</sup>, <em>Ogg1</em><sup>-/-</sup> and DKO mice showed similar levels of freezing when compared to wild-type (WT) (WT, <em>n</em> = 19; <em>Mutyh</em><sup>-/-</sup>, <em>n</em> = 11; <em>Ogg1</em><sup>-/-</sup>, <em>n</em> = 10; DKO, <em>n</em> = 11). (<strong>C</strong>) Experimental design of the novel object location (NOL) memory test. (<strong>D</strong>) (i) DNA glycosylase deficient and WT mice exhibited a similar preference for both objects during training. (ii) <em>Mutyh</em><sup>-/-</sup>, <em>Ogg1</em><sup>-/-</sup> and DKO mice showed similar preference for the novel location and for the familiar location at the 1 h memory retention test compared to WT (WT, <em>n</em> = 8; <em>Mutyh</em><sup>-/-</sup>, <em>n</em> = 8; <em>Ogg1</em><sup>-/-</sup>, <em>n</em> = 7; DKO, <em>n</em> = 7). (<strong>E</strong>) (i) DNA glycosylase deficient and WT mice showed a similar preference for the two objects during the training phase. (ii) DKO mice exhibited impaired long-term location memory by spending less time exploring the novel object at the 24 h memory retention test compared to WT mice (WT, <em>n</em> = 15; <em>Mutyh</em><sup>-/-</sup>, <em>n</em> = 12; <em>Ogg1</em><sup>-/-</sup>, <em>n</em> = 11; DKO, <em>n</em> = 16). The relative exploration time is expressed as a percent discrimination index (D.I. = tnovel location / (tnovel location + tfamiliar location) × 100%). Data are presented as mean ± SEM, n values refer to the number of mice, *<em>p</em> &lt; 0.05</p></figcaption></figure><p id="Par27">Next, we monitored WT, single-knockout and DKO mice in a novel object location (NOL) memory task (Fig. <a href="#Fig1" class="usa-link">1</a>C). All mice displayed identical travel distances, indicating that motor function is not compromised in DNA glycosylase-deficient mice (Supplementary Fig. <a href="#MOESM1" class="usa-link">2A</a>). During the training phase, none of the genotypes showed a significant preference in percentage exploration time (Fig. <a href="#Fig1" class="usa-link">1</a>D i and Fig. <a href="#Fig1" class="usa-link">1</a>E i) and head entries (Supplementary Fig. <a href="#MOESM1" class="usa-link">2B</a>) for either object, suggesting that there is no evident bias towards one of the objects. In the memory retention test conducted 1 h after training, DNA glycosylase-deficient mice exhibited similar exploration of the novel object as WT, indicating that the loss of Ogg1 and/or Mutyh does not impair short-term memory (Fig. <a href="#Fig1" class="usa-link">1</a>D ii). However, in the memory test conducted 24 h after training DKO mice exhibited a significant decrease in the exploration of the novel object location when compared to WT (Fig. <a href="#Fig1" class="usa-link">1</a>E ii). Consistently, DKO mice exhibited no difference in head entries between the familiar and novel object location during the 24 h test (Supplementary Fig. <a href="#MOESM1" class="usa-link">2C</a>). This finding further supports the observation that DKO mice showed less interest in exploring the novel object location compared to other genotypes, indicating a memory deficit. Overall, our results suggest that both Ogg1 and Mutyh are required for long-term spatial memory that relies on spontaneous exploratory behavior.</p></section><section id="Sec24"><h3 class="pmc_sec_title">DNA glycosylases alter DNA methylation of polycomb repressive complex 2 (PRC2) target genes</h3>
<p id="Par28">Given that DNA glycosylases have previously been associated with DNA methylation [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] and DNA methylation in the brain has been identified as a key regulator of memory consolidation [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>, <a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>], we next asked whether Ogg1 and Mutyh affect memory formation by modulating DNA methylation.</p>
<p id="Par29">We therefore performed whole-genome bisulfite sequencing (WGBS) on hippocampal DNA from 6 months old male WT and DNA glycosylase deficient mice to map single-cytosine methylation (5-methylcytosine, 5mC) levels genome-wide. Global mapping of 5mC to genomic locations revealed similar patterns between genotypes (Fig. <a href="#Fig2" class="usa-link">2</a>A). However, CG methylation surrounding 5´ transcription start sites (TSS) were lower in <em>Ogg1</em><sup>-/-</sup> and <em>Mutyh</em><sup>-/-</sup> mice, whereas at gene bodies and transcription end sites (TES) CG methylation was reduced for both single knockout and DKO mice when compared to WT.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334403_18_2025_5730_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ac4/12334403/3bd7c406095e/18_2025_5730_Fig2_HTML.jpg" loading="lazy" id="d33e853" height="1227" width="762" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>DNA glycosylases alter DNA methylation of PRC2 target genes. (<strong>A</strong>) Genome-wide average methylation level in the context of CG at transcription start site (TSS), gene body and transcription end site (TES). (<strong>B</strong>) Identification of differential methylation region (DMRs) and associated genes at TSS, gene body and TES. (<strong>C</strong>) Pie charts represent the percentage of hypomethylated (outer) and hypermethylated (inner) DMRs and associated genes at TSS, gene body and TES. (<strong>D</strong>) Pathways (-log10(p-value) &lt; 0.05) identified by Ingenuity pathway analysis of total DMRs. Upstream regulators (-log10(p-value) &lt; 0.05) identified by Ingenuity pathway analysis of (<strong>E</strong>) total DMRs and (<strong>F</strong>) total DMRs filtered for PRC2 subunits. (<strong>G</strong>) Average methylation level in the context of CG at the TSS of PRC2 target genes</p></figcaption></figure><p id="Par30">Next, we identified differentially methylated regions (DMRs) at specific genomic regions between DNA glycosylase deficient mice and WT hippocampus (Fig. <a href="#Fig2" class="usa-link">2</a>B). We found 1542 regions with differential methylation in <em>Mutyh</em><sup>-/-</sup>, 1279 regions in <em>Ogg1</em><sup>-/-</sup>, and 1548 regions in DKO. Hypermethylated regions were predominantly located within gene bodies across all genotypes. However, the majority of DMRs across the genomic regions were hypomethylated. Interestingly, the highest percentage of hypomethylated regions was found at TSS, with up to 41% for both <em>Ogg1</em><sup>-/-</sup> and DKO (Fig. <a href="#Fig2" class="usa-link">2</a>C). Subsequently, we examined whether DMRs identified at TSS, gene bodies, and TES would intersect among genotypes (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). The most notable overlap was detected for hypermethylated DMRs within gene bodies. The overlap for other DMRs across the genomic regions showed a moderate overlap between genotypes.</p>
<p id="Par32">To understand the biological relevance and functions of altered DNA methylation we mapped all DMRs to their nearby gene annotation and performed ingenuity pathway analysis (IPA). As spatial memory impairment was found exclusively in DKO but not in single knockouts, pathways were filtered based on their significant enrichment in DKO. Among the top 16 most significant pathways for single and DKO we identified an enrichment for pathways that are related to key neuronal functions including axonal guidance signaling, synaptogenesis signaling pathway, synaptic long-term depression and potentiation and CREB signaling in neurons (Fig. <a href="#Fig2" class="usa-link">2</a>D). In addition, targets of several upstream regulators that are key components of the nervous system, including CREB1, SOX2 and FOS, were significantly enriched in DMRs (Fig. <a href="#Fig2" class="usa-link">2</a>E). Interestingly, enhancer of zeste homolog 2 (EZH2), an enzymatic component of the Polycomb Repressive Complex 2 (PRC2), was also found among the candidates with the most significant enrichment. In a previous study, we discovered a significant overlap between differentially expressed genes (DEGs) identified in the hippocampus of Ogg1- and/or Mutyh-deficient mice [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>] and genes marked by H3K27me3, an epigenetic modification mediated by PRC2. Therefore, we examined whether DMRs across all genotypes were enriched for genes targeted by additional PRC2 subunits. In fact, we observed a significant enrichment of the PRC2 subunits suppressor of zeste 12 (SUZ12) and embryonic ectoderm development (EED), in addition to EZH2, among total DMRs across all genotypes (Fig. <a href="#Fig2" class="usa-link">2</a>F). We then mapped the methylation signal to the TSSs of all identified PRC2 target genes and found a significant decrease in methylation across all knockouts in the 500 bp window immediately upstream of TSSs (Fig. <a href="#Fig2" class="usa-link">2</a>G). Taken together, these results indicate that loss of Ogg1 and/or Mutyh alters DNA methylation at regulatory regions of PRC2 target genes in the adult hippocampus.</p></section><section id="Sec25"><h3 class="pmc_sec_title">DNA glycosylases mediate histone post-translational modifications and Suz12 occupancy</h3>
<p id="Par33">Based on these findings, we investigated next whether DNA glycosylases influence PRC2 function. PRC2 is a chromatin-associated protein complex that catalyzes the methylation of mono-, di-, and trimethylation of lysine 27 on histone H3 (H3K27). PRC2-mediated H3K27me3 leads to gene silencing and transcriptional repression important for maintaining cell fate, normal cellular functions and organismal development [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. The core PRC2 subunits, SUZ12, EED, and the methyltransferase EZH2 or its closely related homolog EZH1, are all required for the catalytic activity of the complex [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>, <a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. In addition, PRC2 is involved in the establishment and maintenance of bivalent chromatin domains that typically feature H3K27me3 alongside the methylation of H3 lysine 4 (H3K4me3), an active mark associated with gene transcription [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. Hence, we examined the deposition of histone post-translational modifications (PTMs) that are associated with PRC2 function and the binding of the PRC2 subunit SUZ12.</p>
<p id="Par34">To decipher DNA glycosylase-dependent cell-type specific epigenetic changes, we used fluorescence activated nuclear sorting (FANS) for hippocampal tissue and sorted for NeuN-positive (NeuN+) neuronal and NeuN-negative (NeuN-) non-neuronal nuclei [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] (Fig. <a href="#Fig3" class="usa-link">3</a>A). Cell-type specificity was confirmed by the purity of the sorted cell populations and by immunocytochemistry (ICC) conducted after sorting (Supplementary Fig. <a href="#MOESM1" class="usa-link">4A</a> and <a href="#MOESM1" class="usa-link">4B</a>). Next, we generated ChIP-seq datasets for H3K4me3, H3K27me3, and Suz12 from NeuN + and NeuN- nuclei across all genotypes for three independent replicates. Chromatin signatures of cell-type specific genes demonstrated a successful enrichment for NeuN + and NeuN- nuclei (Fig. <a href="#Fig3" class="usa-link">3</a>B). For instance, the neuronal gene Rbfox3 showed high levels of the activation mark H3K4me3 in NeuN + nuclei, while in NeuN- nuclei H3K4me3 signals were mostly absent and instead an enrichment for the repressive mark H3K27me3 was found. We observed no differences in the global genomic distribution of H3K4me3 or H3K27me3 among genotypes in either neurons or non-neuronal cells (Fig. <a href="#Fig3" class="usa-link">3</a>C). However, the global genomic distribution of Suz12 occupancy at TSS within a range of plus or minus 1 kilobase (kb) was decreased in both NeuN + and NeuN- nuclei for all genotypes compared to WT.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334403_18_2025_5730_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ac4/12334403/28ed7484e603/18_2025_5730_Fig3_HTML.jpg" loading="lazy" id="d33e941" height="675" width="685" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>DNA glycosylases affect histone post-translational modifications and Suz12 occupancy in neurons and non-neuronal cells. (<strong>A</strong>) Experimental design for the isolation of cell-type specific nuclei used in next generation sequencing (NGS) approaches. (<strong>B</strong>) Confirmation of cell-type specific chromatin. Genome browser capture revealed an enrichment of representative H3K4me3 and H3K27me3 signals in NeuN + and NeuN- nuclei at a selected neuronal gene (Rbfox3) and a glial gene (Oaf). (<strong>C</strong>) Profile plots showing mean read distribution of H3K4me3, H3K27me3 and Suz12 over transcription start site (TSS) +/- 1000 bp, scaled gene body and transcription end site (TES) +/- 1000 bp. Boxplots represent the distribution of mean read density over the TSS +/- 1000 bp of all genes</p></figcaption></figure><p id="Par35">We then examined the enrichment of H3K4me3, H3K27me3 and Suz12 across specific gene regions to gain deeper insights into their potential regulatory roles. We identified in total 15,897 consensus peaks for H3K4me3 (12627 in NeuN + and 3270 in NeuN-), 5827 consensus peaks for H3K27me3 (4131 in NeuN + and 1696 in NeuN-) and 1087 consensus peaks for Suz12 (775 in NeuN + and 312 in NeuN-). The genomic annotation of all consensus peaks combined showed that the majority of H3K4me3 peaks were located in promoters for both NeuN + and NeuN- nuclei (Supplementary Fig. <a href="#MOESM1" class="usa-link">5A</a>). For H3K27me3 and Suz12, consensus peaks were mostly identified at promoter and distal intergenic regions, similarly in neurons and non-neuronal cells. However, H3K27me3 demonstrated the highest number of consensus peaks in promoters of non-neuronal cells whereas the majority of Suz12 consensus peaks were located in promoters of neurons. The genomic location of consensus peaks across genotypes revealed a similar genomic distribution (Supplementary Fig. <a href="#MOESM1" class="usa-link">5B</a>). Moreover, the cell-type-specific distribution of consensus peaks revealed that the majority of peaks for H3K4me3, H3K27me3, and SUZ12 were present in NeuN + nuclei, with only a subset of peaks being specific for NeuN- nuclei, which was consistent across all genotypes (Supplementary Fig. <a href="#MOESM1" class="usa-link">5C</a>). Hierarchical clustering analysis of read densities showed a strong correlation among H3K4me3, H3K27me3, and Suz12 across cell-types, demonstrating a high reproducibility between our ChIP-seq replicates (Supplementary Fig. <a href="#MOESM1" class="usa-link">6A</a>). In summary, our data is of high technical quality and indicates that the loss of DNA glycosylases does not impact the global genomic distribution of H3K4me3 and H3K27me3 but alters the global allocation of Suz12 at TSS.</p></section><section id="Sec26"><h3 class="pmc_sec_title">DNA glycosylases regulate PRC2 associated histone PTMs at genes associated with cognitive function</h3>
<p id="Par37">To uncover DNA glycosylase-specific alterations in histone PTMs and Suz12 occupancy, we identified differentially enriched regions (DERs) in our ChIP-seq datasets. In contrast to the global genomic distribution, we detected numerous region-specific alterations in histone PTMs and Suz12 occupancy (Fig. <a href="#Fig4" class="usa-link">4</a>A). The total number of DERs for both over- and under-enriched regions were similar between single-knockouts and DKO mice for H3K4me3, H3K27me3 and Suz12, respectively (Supplementary Fig. <a href="#MOESM1" class="usa-link">6B</a>). On average, we identified 866 DERs for H3K4me3, 342 DERs for H3K27Me3, and 59 DERs for Suz12 across single-knockouts and DKO in both cell-types. The percentage of DERs revealed a similar distribution of over- and under enrichment in both NeuN + and NeuN- nuclei across all genotypes, indicating that both neurons and non-neuronal cells are affected by the loss of DNA glycosylases (Fig. <a href="#Fig4" class="usa-link">4</a>B). Only for Suz12, we found a higher percentage of DERs for <em>Ogg1</em><sup>-/-</sup> and DKO in non-neuronal cells compared to neurons (Fig. <a href="#Fig4" class="usa-link">4</a>B iii). Similar to changes in DNA methylation, we observed a moderate overlap of DERs between genotypes across cell-types (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). However, most of the DERs were still unique for each genotype, suggesting that histone</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334403_18_2025_5730_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ac4/12334403/a15807717292/18_2025_5730_Fig4_HTML.jpg" loading="lazy" id="d33e1003" height="1180" width="637" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>DNA Glycosylases mediate histone post-translational modifications associated with cognition in both neurons and non-neuronal cells. (<strong>A</strong>) Volcano plot representation of differential enriched regions (DERs) of H3K4me3 (i), H3K27me3 (ii) and Suz12 (iii) for NeuN + and NeuN- nuclei. (<strong>B</strong>) Percentage of DERs for H3K4me3 (i), H3K27me3 (ii) and Suz12 (iii) in either NeuN + or NeuN- nuclei across all genotypes. (<strong>C</strong>) Genomic annotations of DERs for H3K4me3 (i), H3K27me3 (ii) and Suz12 (iii) in either NeuN + or NeuN- nuclei of <em>Mutyh</em><sup>-/-</sup>, <em>Ogg1</em><sup>-/-</sup> and DKO when compared to WT. (<strong>D</strong>) Disease and functions (-log10(p-value) &lt; 0.05) identified by Ingenuity pathway analysis of DERs for H3K4me3 in neurons (i) and non-neuronal cells (ii). (<strong>E</strong>) Most significant disease and functions (-log10(p-value) &lt; 0.05) identified by Ingenuity pathway analysis of DERs for H3K27me3 combined for neurons and non-neuronal cells. (<strong>F</strong>) Overrepresentation of PRC2 target genes in DERs of H3K4me3, H3K27me3 and Suz12 for NeuN + and NeuN- nuclei. Dashed lines indicate an observed / expected ratio ≥ 1</p></figcaption></figure><p id="Par39">PTMs and Suz12 occupancy are mainly regulated differently by <em>Mutyh</em><sup>-/-</sup> and/or <em>Ogg1</em><sup>-/-</sup>. Next, we analyzed the genomic distribution of DERs and found that in both NeuN + and NeuN- nuclei the majority of DERs for H3K4me3 were situated in promoter regions (Fig. <a href="#Fig4" class="usa-link">4</a>C i and ii). In contrast, H3K27me3 DERs showed a more ubiquitous distribution and were located both at promoter regions and distal intergenic regions in NeuN + and NeuN- nuclei (Fig. <a href="#Fig4" class="usa-link">4</a>C iii and iv). For Suz12 in neurons, we detected the highest number of DERs at distal intergenic regions across all genotypes (Fig. <a href="#Fig4" class="usa-link">4</a>C v). In addition, we also found a high abundance of DERs at distal intergenic regions for Suz12 in NeuN- nuclei (Fig. <a href="#Fig4" class="usa-link">4</a>C vi). Given that the majority of alterations for H3K27me3 also occurred at distal intergenic regions, these findings indicate a DNA glycosylase-dependent redistribution of Suz12, which consequently affects the genomic distribution of H3K27me3.</p>
<p id="Par40">To comprehend the biological significance of differential H3K4me3, H3K27me3, and Suz12 occupancy, we mapped DERs to their nearby gene annotations and performed IPA (Fig. <a href="#Fig4" class="usa-link">4</a>D and E). Since spatial memory impairment was only observed in DKO, pathways were ranked based on their significant enrichment in DKO. Among the most significant disease and function categories associated with H3K4me3 in neurons, pathways linked to various types of tissue damage, such as neurogenic lesions, soft tissue lesions, and brain lesions were identified across all genotypes (Fig. <a href="#Fig4" class="usa-link">4</a>D I). Moreover, we found myelination signaling pathway and axonal guidance signaling as the top enriched canonical pathways in DKO (Supplementary Fig. <a href="#MOESM1" class="usa-link">8A</a> i). In non-neuronal cells, we found an enrichment of disease and functions associated with cognition including cognitive impairment and mental retardation (Fig. <a href="#Fig4" class="usa-link">4</a>D ii). Unlike H3K4me3, the diseases and functions identified for H3K27me3 showed a strong overlap in both NeuN + and NeuN- nuclei. Interestingly, the majority of the top identified disease and functions were related to gene regulation including transcription, transcription of DNA, expression of RNA and activation of endogenous promoter in all genotypes (Fig. <a href="#Fig4" class="usa-link">4</a>E). In non-neuronal cells, we identified synaptogenesis signaling pathway and activation of NMDA receptors and postsynaptic events as the highly enriched canonical pathways for H3K27me3, most significantly in DKO (Supplementary Fig. <a href="#MOESM1" class="usa-link">8B</a> ii). Because fewer DERs were identified for Suz12, representative pathway analysis could not be obtained.</p>
<p id="Par41">To investigate whether PRC2 target genes are overrepresented in DERs of H3K4me3 and H3K27me3, we compiled a consensus list from publicly available ChIP-seq data in neuronal cells of Eed, Suz12 and Ezh2. The observed versus expected ratio showed a significant overrepresentation of PRC2 target genes in DERs of H3K4me3 and H3K27me3 for both NeuN + and NeuN- nuclei across all genotypes (Fig. <a href="#Fig4" class="usa-link">4</a>F). Taken together, our data suggest that DNA glycosylase-dependent epigenetic changes are particularly prominent for PRC2 targets, which are associated with key functions and pathways relevant for cognition in both neurons and non-neuronal cells.</p></section><section id="Sec27"><h3 class="pmc_sec_title">DNA glycosylases mediate expression of PRC2 target genes important for neuronal function and signaling</h3>
<p id="Par42">Next, we examined whether DNA glycosylase-dependent epigenetic changes affect gene expression in a cell-type specific manner by performing nuclear transcriptome profiling of NeuN + and NeuN- nuclei from DNA glycosylase-deficient and WT hippocampus. Cell-type specific RNA expression was confirmed through gene ontology analysis, which revealed neuronal processes primarily in NeuN + and glial-related processes in NeuN- nuclei (Supplementary Fig. <a href="#MOESM1" class="usa-link">9A</a> and <a href="#MOESM1" class="usa-link">9B</a>). Hierarchical clustering analysis of DEGs revealed that loss of Ogg1 and Mutyh predominantly affected genes with a cell-type specific expression profile (Fig. <a href="#Fig5" class="usa-link">5</a>A). DEGs across all genotypes were primarily downregulated in both neurons (177 up and 454 down) and non-neuronal cells (193 up and 249 down), most prominently in <em>Ogg1</em><sup>-/-</sup> and DKO neurons (Fig. <a href="#Fig5" class="usa-link">5</a>B). The majority of DEGs were either NeuN + or NeuN- specific while only a minority of genes was differentially expressed in both cell-types, indicating that DNA glycosylases regulate gene expression in a cell-type specific manner (Fig. <a href="#Fig5" class="usa-link">5</a>C). Next, we investigated whether DEGs overlapped among genotypes and found again a moderate overlap of DEGs across all genotypes (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>). The most pronounced overlap was observed in downregulated DEGs between <em>Ogg1</em><sup>-/-</sup> and DKO in neurons and non-neuronal cells.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334403_18_2025_5730_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ac4/12334403/41325243302a/18_2025_5730_Fig5_HTML.jpg" loading="lazy" id="d33e1121" height="1152" width="762" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>DNA glycosylases regulate gene expression important for neuronal function and signaling. (<strong>A</strong>) Heatmap over all differentially expressed genes (DEGs) (fold-change ≤ 0.5, FDR ≤ 0.05), clustered using Euclidean distance and average linkage. Values are FPKMs, z-scored by row across cell-type showing that DEGs are predominantly found in cell-type specific genes. (<strong>B</strong>) Volcano plot representation of DEGs across genotypes in NeuN + and NeuN- nuclei. Tables indicate the number of genes either up- or downregulated. (<strong>C</strong>) Cell-type specific DEGs across <em>Mutyh</em><sup>−/−</sup>, <em>Ogg1</em><sup>−/−</sup> and DKO mice. DEGs were detected exclusively in either NeuN + or NeuN- cell populations, or in both. Pathways (-log10(p-value) &lt; 0.05) identified by Ingenuity pathway analysis of DEGs in (<strong>D</strong>) neurons and (<strong>E</strong>) non-neuronal cells. (<strong>F</strong>) Overrepresentation of PRC2 target genes in DEGs for NeuN + and the NeuN-. Dashed lines indicate an observed / expected ratio ≥ 1</p></figcaption></figure><p id="Par44">IPA analysis of DEGs revealed a significant enrichment of several metabolic pathways, including glycolysis, gluconeogenesis, and selenoamino acid metabolism, which was particularly pronounced within neurons in <em>Ogg1</em><sup>-/-</sup> and DKO (Fig. <a href="#Fig5" class="usa-link">5</a>D), suggesting Ogg1 as the driver of this effect. Interestingly, we also discovered pathways associated with NMDA receptors, with the strongest enrichment in DKO. Moreover, diseases and functions annotations showed a significant enrichment of pathways related to neurological disorders such as neuromuscular disease and Huntington disease (Supplementary Fig. <a href="#MOESM1" class="usa-link">11A</a>). In non-neuronal cells, we observed the most significant enrichment of DKO DEGs in diseases and functions related to the entrance of Ca2+, neurite growth, schizophrenia, Huntington’s disease, Parkinsonism, and cognitive impairment (Fig. <a href="#Fig5" class="usa-link">5</a>E). Moreover, significant enriched canonical pathways included cAMP-mediated signaling, ion channel transport and interferon gamma signaling (Supplementary Fig. <a href="#MOESM1" class="usa-link">11B</a>). Interestingly, enrichment analysis using our compiled PRC2 gene list revealed a strong overrepresentation of DEGs from <em>Ogg1</em><sup>-/-</sup> and DKO in neurons, with no significant enrichment observed in non-neuronal cells (Fig. <a href="#Fig5" class="usa-link">5</a>F).</p>
<p id="Par45">In summary, while we observed cell-type specific alterations in gene expression following the loss of DNA glycosylases, DEGs in both cell-types were associated with highly relevant categories for neuronal function and cognition, particularly in DKO. Notably, PRC2 target genes were found to be primarily overrepresented among DEGs in neurons, indicating that DNA glycosylases regulate their expression in a cell-type specific manner.</p></section><section id="Sec28"><h3 class="pmc_sec_title">DNA glycosylases alter gene expression through modulating histone PTMs</h3>
<p id="Par46">We then examined a possible connection between DNA glycosylase-dependent epigenetic modifications and transcriptional changes. Therefore, we performed Spearman correlation analysis between FPKMs (fragments per kilobase million) obtained through RNA-seq and TSS CPMs (counts per million) obtained from ChIP-seq (Fig. <a href="#Fig6" class="usa-link">6</a>A). We observed a consistent positive correlation of H3K4me3 and RNA in neurons and non-neuronal cells across all genotypes. In contrast, H3K27me3, which is associated with transcriptional repression, demonstrated a consistent negative correlation with gene transcription. In addition, we observed a minor negative correlation between Suz12 occupancy and RNA expression for all genotypes and cell-types. To examine a potential correlation between DNA glycosylase-dependent DNA methylation and gene expression changes we intersected DMRs with DEGs and found only a minor overlap across all genotypes (Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334403_18_2025_5730_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ac4/12334403/63eece5e7ee1/18_2025_5730_Fig6_HTML.jpg" loading="lazy" id="d33e1185" height="994" width="732" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Histone post-translational modifications regulated by DNA glycosylases influence gene expression. (<strong>A</strong>) Spearman correlation analysis between ChIP-seq and RNA-seq data. Scatterplot showing the correlation of H3K4me3, H3K27me3, and Suz12 CPMs (counts per million) in the transcription start site (+/- 500 bp) with RNA FPKMs (fragment per kilobase million) for neurons and non-neuronal cells. (<strong>B</strong>) Overlapping pathways for RNA, H3K3me3 and H3K27me3 (-log10(p-value) &lt; 0.05) identified by Ingenuity pathway analysis in neurons. (<strong>C</strong>) Overlapping pathways for RNA, H3K3me3 and H3K27me3 (-log10(p-value) &lt; 0.05) identified by Ingenuity pathway analysis in non-neuronal cells. Differences in gene expression (FPKMs), DNA methylation (mean signal) and occupancy for histone post-translational modifications and Suz12 (CPMs in the TSS) for overlapping PRC2 target genes identified in the most significant pathways in (<strong>D</strong>) neurons and (<strong>E</strong>) non-neuronal cells</p></figcaption></figure><p id="Par48">Since we only identified a minimal gene-specific overlap between DMRs, DERs and DEGs, we wondered whether similar biological processes are affected. Therefore, we intersected the pathways identified in each individual dataset that showed significant enrichment. Interestingly, the overlap revealed numerous pathways linked to critical neuronal processes and neurodegenerative diseases such as axonal guidance signaling, opioid signaling pathway and Huntington’s disease. Notably, DKO exhibited the strongest enrichment in axonal guidance signaling pathway for both RNA and H3K4me3 (Fig. <a href="#Fig6" class="usa-link">6</a>B). In NeuN- nuclei, we found overlapping pathways related to key neuronal processes and diseases including cAMP-mediated signaling, synaptic long-term depression and autism signaling pathway, all showing the most significant enrichment in DKO for RNA (Fig. <a href="#Fig6" class="usa-link">6</a>C). Next, we compared RNA expression (FPKMs), DNA methylation, Suz12 occupancy, and histone PTMs (CPMs) of genes identified in the overlapping neuronal process pathways across all genotypes, focusing on PRC2 target genes. We observed a negative correlation between RNA and H3K27me3 for genes targeted by PRC2 (such as GRIN1, HDAC4, and CAMK2A) in DKO neurons, while a positive correlation was found between RNA and H3K4me3 at PRC2 target genes (such as SEMAC4, GLIS1, and GRIN1) (Fig. <a href="#Fig6" class="usa-link">6</a>D). Moreover, PRC2 target genes that showed higher RNA levels in DKO, including GRIN1, HDAC4, SEMA4C, and GLIS1, also displayed DNA hypomethylation in DKO neurons. In contrast, genes that appeared to be hypermethylated demonstrated a negative correlation with RNA levels in DKO neurons. In non-neuronal cells, we noticed a negative correlation between RNA levels and H3K27me3 for GRM7 and BDNF (Fig. <a href="#Fig6" class="usa-link">6</a>E). Additionally, a decrease in GRM7 RNA levels correlated with DNA hypermethylation, whereas PRC2 target genes NPR3 and WNT6 with elevated RNA levels exhibited DNA hypomethylation.</p>
<p id="Par49">Collectively, our findings suggest that DNA glycosylases Ogg1 and Mutyh regulate gene transcription by modulating the epigenetic landscape through histone PTMs and DNA methylation. Furthermore, our data indicate that DNA glycosylases regulate the expression of genes primarily targeted by PRC2, which are essential for neuronal function and signaling.</p></section><section id="Sec29"><h3 class="pmc_sec_title">DNA glycosylases affect expression of genes relevant for human cognitive function</h3>
<p id="Par50">To further explore a potential contribution of DNA glycosylase-regulated genes to human brain function, we examined their association with cognitive abilities and brain-related diseases in human genetic datasets. Consequently, we employed MAGMA, a fast and versatile tool for gene and gene-set analysis of genome-wide association studies (GWAS) data, to assess a potential enrichment of DNA glycosylase-regulated genes using GWAS summary statistics [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>] from large-scale studies that aggregate data from multiple biobanks and disease-specific consortia.</p>
<p id="Par51">Intriguingly, we discovered that genes regulated by DNA glycosylases in both neurons and non-neuronal cells were significantly associated with cognitive traits and neurological diseases, but not with non-neurological diseases (Fig. <a href="#Fig7" class="usa-link">7</a>). For instance, we found a significant enrichment of genes linked to hippocampal volume for all differentially expressed genes in DKO neurons and upregulated genes in DKO non-neuronal cells whereas for the single KOs only <em>Mutyh</em><sup>-/-</sup> upregulated genes showed a significant association. Cognitive performance was significantly associated with downregulated genes in <em>Mutyh</em><sup><em>-/-</em></sup>, <em>Ogg1</em><sup><em>-/-</em></sup>, and DKO non-neuronal cells, but with the highest enrichment in DKO. In addition, only in the downregulated genes in non-neuronal cells of DKO a significant enrichment of genes linked to human intelligence was found. Interestingly, we also detected a significant enrichment of genes associated with bipolar disorder across all genotypes but particularly among the downregulated genes in <em>Ogg1</em><sup>-/-</sup> and DKO neurons and non-neuronal cells, as well as in all differentially expressed genes in Ogg1 neurons and non-neuronal cells and DKO non-neuronal cells. Taken together, our findings uncover a previously undescribed role of DNA glycosylases beyond DNA repair by mediating gene expression relevant for human cognitive function and neuropsychiatric disorder.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334403_18_2025_5730_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ac4/12334403/c76fd387d690/18_2025_5730_Fig7_HTML.jpg" loading="lazy" id="d33e1255" height="200" width="685" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>DNA glycosylase-regulated genes contribute to human cognitive abilities and bipolar disorder. MAGMA gene set analysis of cell-type specific differentially expressed genes in <em>Mutyh</em><sup>−/−</sup>, <em>Ogg1</em><sup>−/−</sup> and DKO mice using summary statistics from multiple GWAS datasets. GWAS datasets are listed on the right: cognitive performance, intelligence, hippocampal volume, bipolar disorder, stroke, dementia, autism spectrum disorder (ASD) and type II diabetes. Symbols in color indicate a significant enrichment in the corresponding GWAS dataset (p-value &lt; 0.05). DNA glycosylase-regulated gene sets are shown on the y-axis and BETA values (effect sizes) calculated by MAGMA are plotted on the x-axis. Full list of p- and BETA values are supplied in Supplementary Table <a href="#MOESM1" class="usa-link">2</a></p></figcaption></figure></section></section><section id="Sec30"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par53">While conventionally associated with preserving genome integrity, our findings suggest that DNA glycosylases Ogg1 and Mutyh play a significant role in altering the neuronal epigenome thereby contributing to the maintenance of cellular identities and spatial memory formation in the hippocampus. Our study demonstrates that Ogg1 and Mutyh play a role in regulating DNA methylation, particularly affecting the methylation status of PRC2 target genes. Moreover, we find that Ogg1 and Mutyh modulate histone modifications associated with PRC2 activity in neurons and non-neuronal cells of the hippocampus. Our results indicate that alterations in the epigenome, mediated by Ogg1 and Mutyh, correlate with the transcriptional regulation of predominantly PRC2 target genes, which are important for neuronal function and signaling underlying memory formation. Changes were found to primarily affect cell-type specific gene expression indicating that DNA glycosylases may have a role in maintaining cellular identities. Consistent with these findings, human genetic data revealed that epigenetic regulation of gene expression by DNA glycosylases contributes to human cognitive abilities and bipolar disorder.</p>
<section id="Sec31"><h3 class="pmc_sec_title">DNA glycosylases regulate spatial long-term memory</h3>
<p id="Par54">Both our research and other studies have highlighted the involvement of DNA glycosylases in higher brain functions associated with memory formation [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>, <a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. In a previous study, we evaluated the learning and memory performance of single and double knockout mice deficient for Ogg1 and Mutyh by conducting a Morris water maze test. DKO mice were significantly slower in learning the position of the platform compared to wild-type mice but showed no difference in memory retention leaving the question if both Ogg1 and Mutyh affect memory formation unanswered [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Here, we demonstrate that the absence of Ogg1 and/or Mutyh does not influence short-term and long-term memory for contextual fear, suggesting that Ogg1 and Mutyh are not required for the consolidation of associative memories. In a novel object location memory task assessing spontaneous exploratory behavior, DKO mice exhibited no impairment in short-term memory but displayed a deficit in long-term spatial memory. Interestingly, we did not detect any memory formation impairment in Ogg1 and Mutyh single knockouts. However, the simultaneous deletion of both Ogg1 and Mutyh led to a deficit in memory retention, highlighting the importance of both DNA glycosylases in maintaining cognitive function. Our findings suggest that the presence of both Ogg1 and Mutyh is particularly important in memory tasks that recruit an innate behavioral preference for novelty and do not involve exogenous reinforcers such as swim stress or an electric foot shock. Consistent with our results, other studies demonstrate that stressful reinforcement stimuli might trigger alternative mechanisms that could compensate for the absence of both Ogg1 and Mutyh in our mouse model [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>, <a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>].</p></section><section id="Sec32"><h3 class="pmc_sec_title">DNA glycosylases modulate DNA methylation and histone PTMs associated with PRC2 function</h3>
<p id="Par55">Oxidative DNA damage and its repair has been demonstrated to be involved in transcriptional regulation but whether these processes are directly linked has remained largely unexplored. We have previously shown that loss of Ogg1 and/or Mutyh impact gene transcription independent of global accumulation of 8-oxoG in the hippocampus [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. However, the underlying mechanisms concerning how DNA glycosylases mediate gene expression independent of DNA repair remain elusive.</p>
<p id="Par56">Here, we uncover a novel function for Ogg1 and Mutyh, beyond their traditional role in DNA repair, by modulating DNA methylation in the adult mouse hippocampus. DMRs across all genotypes were predominantly hypomethylated and at the TSS the majority of hypomethylated DMRs were significantly enriched in genes targeted by PRC2. Consistent with our findings, increasing evidence indicates that in other cell-types DNA glycosylases play an important role in modulating cellular function by influencing epigenetic alterations. Ogg1 and Mutyh have been demonstrated to bind to, but not to excise, 5mC, potentially preventing DNA demethylation [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. Interestingly, it has been shown that Ogg1 is part of a multiprotein complex that includes DNMT1 and EZH2, which facilitates CpG promoter methylation and induces alterations in histone PTMs such as H3K4me3 and H3K27me3<sup>19</sup>. Consistently, we observed that the absence of Ogg1 and Mutyh disrupts the occupancy of both H3K4me3 and H3K27me3 marks. The precise mechanism by which PRC2 is recruited to its target sites to promote transcriptional silencing remains unclear but our results suggest that Ogg1 and Mutyh may play a role in this process. In line with this, we demonstrated that PRC2 target genes were overrepresented among DERs of H3K4me3 and H3K27me3 in both neurons and non-neuronal cells, suggesting that DNA glycosylases regulate histone modifications predominantly associated with PRC2 activity. In addition, we found a redistribution of H3K27me3 and Suz12 upon loss of Ogg1 and Mutyh, indicating that PRC2 loses its ability to mediate the deposition of H3K27me3 at its target genes and occurs more randomly. We believe that the recruitment of PRC2 by Ogg1 and Mutyh is unlikely to occur through 8-oxoG. First, Mutyh recognizes adenine opposite 8-oxoG which is incorporated after replication [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. Neurons, however, are post-mitotic and as such Mutyh is not required for 8-oxoG repair in those cells. Moreover, in a recent study, we profiled the genome-wide distribution of 8-oxoG and found no overlap of genes with 8-oxoG deposition and genes with altered expression in cells deficient for Ogg1 and Mutyh [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. These findings suggest that 8-oxoG and DNA glycosylases modulate gene transcription independently and argue against an 8-oxoG-dependent mechanism. Alternatively, Ogg1 and Mutyh may recruit PRC2 to its target sites by recognizing secondary DNA structures such as G-quadruplex (G4) structures. G4 structures occur in CpG rich regions and have been implicated in gene regulation [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>, <a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. Although the potential association between DNA glycosylases and G4 structures has been suggested [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>], further research is required to elucidate this potential mechanism.</p></section><section id="Sec329"><h3 class="pmc_sec_title">DNA glycosylases affect the expression of genes targeted by PRC2 and associated with human cognitive abilities</h3>
<p id="Par57">In the adult brain, PRC2 plays a crucial role in maintaining neuronal function and survival by transcriptionally silencing specific bivalent genes. Mice lacking PRC2 in neurons display signs of progressive neurodegeneration, with altered gene expression linked to Huntington’s disease [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. In addition, it has been demonstrated that PRC2 is crucial for the activation of microglia and that loss of PRC2 causes changes in neuronal morphology and complex behaviors associated with neurodegenerative diseases [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. In line with these findings, we found that upon loss of Ogg1 and Mutyh, cell-type specific alterations in gene expression were associated with highly relevant categories for neuronal function and cognition in both neurons and non-neuronal cells. Moreover, a previous study demonstrated that Ogg1 and Mutyh-dependent gene expression changes in a cancer cell line showed the highest enrichment in PRC2 target genes [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. In the mouse hippocampus, a significant overlap between DEGs and genes harboring the epigenetic mark H3K27me3 has been reported [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Consistently, we provide evidence that PRC2 target genes were overrepresented among DEGs, suggesting that DNA glycosylases play a crucial role in regulating their expression. PRC2 is a well-known regulator of cellular differentiation and lineage specification throughout development where disruption of PRC2 regulation results in widespread phenotype changes and developmental delays [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. Here, we find that the loss of DNA glycosylases results in only mild phenotype alterations and no obvious developmental delays indicating that their role lies in the long-term maintenance of epigenetic marks at PRC2 target genes and the preservation of cellular identity. Maintenance of the neuronal epigenome with age is important for cell-specific functionality and contributes to the integrity of cognitive processes like memory formation over time [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>].</p>
<p id="Par58">In addition, we discovered that gene expression changes within neurons were affecting various metabolic pathways. This association was evident in all DNA glycosylase-deficient mice, but most pronounced in Ogg1 and DKO suggesting that the effect is driven mainly by Ogg1. Our findings are consistent with previous studies indicating that Ogg1 plays a role in regulating cellular energy metabolism [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>, <a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. Recent studies have demonstrated the importance of cellular metabolism in the intrinsic activity of neurons and memory formation [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>, <a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. Consequently, the deficit in energy metabolism may partially contribute to the memory impairment seen in DKO mice. This is further supported by the strong association of gene expression changes with brain-related diseases and cognitive impairment upon loss of both Ogg1 and Mutyh. Importantly, several genes involved in these pathways are known PRC2 targets.</p>
<p id="Par59">Notably, analysis of common GWAS variants associated with human cognitive traits and DNA glycosylase-regulated genes revealed an enrichment of genes linked to cognitive performance, intelligence and hippocampal volume. Hippocampal volume has been linked to various cognitive functions including processing speed, working memory, spatial navigation and abstract reasoning [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. Interestingly, we identified the strongest enrichment of genes associated with cognitive performance among the downregulated genes in NeuN- nuclei of DKO, indicating that DNA glycosylases contribute to human cognition partly through glial cells. In the brain, all major types of glial cells have been found to impact neuronal signaling and synaptic transmission through various mechanisms [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>–<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. A strong enrichment of common GWAS variants associated with bipolar disorder was identified across various DEG sets of single KOs and DKO. Bipolar disorders are a class of affect disorders characterized by biphasic episodes of mania and depression. Cognitive symptoms are highly prevalent in patients with bipolar disorder, emerging at the onset of symptoms and worsening as the disease progresses or becomes more severe [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>]. Intriguingly, BER has been implicated in bipolar disorder, with studies showing that OGG1 is downregulated in a cohort of bipolar disorder patients [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>]. Furthermore, a previous study demonstrated that single nucleotide polymorphisms (SNPs) in OGG1 and MUTYH are associated with depression, further emphasizing a significant role of DNA glycosylases in affect regulation and mental health [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]. Although the association with bipolar disorder was significant for DEG sets of single KOs and DKO, the most pronounced enrichment occurring in NeuN- nuclei of DKO, suggesting an additive effect resulting from the simultaneous depletion of Ogg1 and Mutyh.</p>
<p id="Par60">In conclusion, our results indicate that Ogg1 and Mutyh cooperate with PRC2 to modulate the epigenome and, consequently, gene transcription in neurons and non-neuronal cells of the adult hippocampus. We therefore propose a novel role for Ogg1 and Mutyh beyond DNA repair in the epigenetic and transcriptional control of PRC2 target genes that are crucial for neuronal function and signaling. Notably, we observed the most pronounced molecular differences in DKO, which is in line with our finding that only the simultaneous deletion of Ogg1 and Mutyh results in impaired memory formation. In contrast, the absence of Ogg1 or Mutyh in single knockout mice showed no noticeable phenotypic effect on memory formation. Alterations in the epigenome and RNA expression occurred across all genotypes but varied in magnitude, suggesting that Ogg1 and Mutyh have an independent but synergistic effect on gene regulation underlying memory formation. Consistently, human genetic data revealed a previously undescribed link between DNA glycosylases and the etiology of higher cognitive functions and bipolar disorder. Hence, it will be intriguing to explore the role of Ogg1 and Mutyh in epigenetic maintenance underlying neurocognitive and psychiatric disorders. Future studies are needed to elucidate the full mechanisms by which DNA glycosylases Ogg1 and Mutyh cooperate with PRC2 to shape the epigenetic landscape in the brain. A better understanding of DNA glycosylases function in brain-related diseases could pave the way for novel therapeutic interventions.</p></section></section><section id="Sec34"><h2 class="pmc_sec_title">Electronic supplementary material</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12334403/bin/18_2025_5730_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (5.1MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank the Genomic Core Facility (GCF) at NTNU for the help to obtain high-throughput sequencing data.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>AA, RD, NBL and KS performed the experiments. AA, TS, MY, JW and KS analyzed the data. AA, MB and KS designed the work. AA, TS, MB and KS interpreted the analysis. AA and KS wrote the manuscript. All authors participated in the review and editing of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open access funding provided by NTNU Norwegian University of Science and Technology (incl St. Olavs Hospital - Trondheim University Hospital). Open access funding provided by NTNU Norwegian University of Science and Technology (incl St. Olavs Hospital - Trondheim University Hospital). Norwegian University of Science and Technology; Research Council of Norway [275777 to KS, 287911 to MB, 326101 to MB.]; the Central Norway Regional Health Authority of Norway [90172200 to K.S, 90369200 to K.S.]; Funding for open access charge: Norwegian University of Science and Technology.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All genomic data produced in the present project (Bisulfite-seq data, ChIP-seq and RNA-seq) have been deposited in the NCBI GEO database under accession number GSE272603.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar2"><h3 class="pmc_sec_title">Ethical approval</h3>
<p id="Par61">All experiments were approved by the Norwegian Animal Research Authority (NARA) and conducted in accordance with the laws and regulations controlling experimental procedures in live animals in Norway and the European Union’s Directive 86/609/EEC.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Declaration of generative AI and AI-assisted technologies in the writing process</h3>
<p id="Par62">For the preparation of this manuscript, the authors used ChatGPT to revise English grammar and usage. After using this tool, the authors reviewed and edited the content as needed and take full responsibility for the content of the published article.</p></section><section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par63">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p>Andreas Abentung and Teri Sakshaug joint first authorship.</p></div>
<div class="fn p" id="fn2">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
</div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Lindahl T (2000) Repair of endogenous DNA damage. Cold Spring Harb Symp Quant Biol. 10.1101/sqb.2000.65.127
</cite> [<a href="https://doi.org/10.1101/sqb.2000.65.127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12760027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lindahl%20T%20(2000)%20Repair%20of%20endogenous%20DNA%20damage.%20Cold%20Spring%20Harb%20Symp%20Quant%20Biol.%2010.1101/sqb.2000.65.127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Hoeijmakers J (2009) DNA damage, aging, and cancer. N Engl J Med 361:1475–1485. 10.1056/NEJMra0804615</cite> [<a href="https://doi.org/10.1056/NEJMra0804615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19812404/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Lodato MA et al (2018) Aging and neurodegeneration are associated with increased mutations in single human neurons (Science DOI: 10.1126/science.aao4426). Science 361:555–559Erratum</cite> [<a href="https://doi.org/10.1126/science.aao4426" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5831169/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29217584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lodato%20MA%20et%20al%20(2018)%20Aging%20and%20neurodegeneration%20are%20associated%20with%20increased%20mutations%20in%20single%20human%20neurons%20(Science%20DOI:%2010.1126/science.aao4426).%20Science%20361:555%E2%80%93559Erratum" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Lu T et al (2004) Gene regulation and DNA damage in the ageing human brain. Nature 429:883–891
</cite> [<a href="https://doi.org/10.1038/nature02661" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15190254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20T%20et%20al%20(2004)%20Gene%20regulation%20and%20DNA%20damage%20in%20the%20ageing%20human%20brain.%20Nature%20429:883%E2%80%93891" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Halliwell B (1992) Reactive oxygen species and the central nervous system. J Neurochem. 10.1111/j.1471-4159.1992.tb10990.x
</cite> [<a href="https://doi.org/10.1111/j.1471-4159.1992.tb10990.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1402908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Halliwell%20B%20(1992)%20Reactive%20oxygen%20species%20and%20the%20central%20nervous%20system.%20J%20Neurochem.%2010.1111/j.1471-4159.1992.tb10990.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>De Boer J et al (2002) Premature aging in mice deficient in DNA repair and transcription. Science 296:1276–1279
</cite> [<a href="https://doi.org/10.1126/science.1070174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11950998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Boer%20J%20et%20al%20(2002)%20Premature%20aging%20in%20mice%20deficient%20in%20DNA%20repair%20and%20transcription.%20Science%20296:1276%E2%80%931279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Madabhushi R, Pan L, Tsai LH (2014) DNA damage and its links to neurodegeneration. Neuron 83:266–282
</cite> [<a href="https://doi.org/10.1016/j.neuron.2014.06.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5564444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25033177/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Madabhushi%20R,%20Pan%20L,%20Tsai%20LH%20(2014)%20DNA%20damage%20and%20its%20links%20to%20neurodegeneration.%20Neuron%2083:266%E2%80%93282" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Krokan HE, Bjørås M (2013) Base excision repair. Cold Spring Harb Perspect Biol. 10.1101/cshperspect.a012583
</cite> [<a href="https://doi.org/10.1101/cshperspect.a012583" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3683898/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23545420/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krokan%20HE,%20Bj%C3%B8r%C3%A5s%20M%20(2013)%20Base%20excision%20repair.%20Cold%20Spring%20Harb%20Perspect%20Biol.%2010.1101/cshperspect.a012583" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Lindahl T (1993) Instability and decay of the primary structure of DNA. Nature 362:709–715
</cite> [<a href="https://doi.org/10.1038/362709a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8469282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lindahl%20T%20(1993)%20Instability%20and%20decay%20of%20the%20primary%20structure%20of%20DNA.%20Nature%20362:709%E2%80%93715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Hazra TK, Hill JW, Izumi T, Mitra S (2001) Multiple DNA glycosylases for repair of 8-oxoguanine and their potential in vivo functions. Prog Nucl Acid Res Mol Biol. 10.1016/S0079-6603(01)68100-5</cite> [<a href="https://doi.org/10.1016/s0079-6603(01)68100-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11554297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hazra%20TK,%20Hill%20JW,%20Izumi%20T,%20Mitra%20S%20(2001)%20Multiple%20DNA%20glycosylases%20for%20repair%20of%208-oxoguanine%20and%20their%20potential%20in%20vivo%20functions.%20Prog%20Nucl%20Acid%20Res%20Mol%20Biol.%2010.1016/S0079-6603(01)68100-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Russo MT, De Luca G, Degan P, Bignami M (2007) Different DNA repair strategies to combat the threat from 8-oxoguanine. Mutat Res - Fundamental Mol Mech Mutagen. 10.1016/j.mrfmmm.2006.03.007</cite> [<a href="https://doi.org/10.1016/j.mrfmmm.2006.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16769088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Russo%20MT,%20De%20Luca%20G,%20Degan%20P,%20Bignami%20M%20(2007)%20Different%20DNA%20repair%20strategies%20to%20combat%20the%20threat%20from%208-oxoguanine.%20Mutat%20Res%20-%20Fundamental%20Mol%20Mech%20Mutagen.%2010.1016/j.mrfmmm.2006.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Scheffler K, Bjørås KØ, Bjørås M (2019) Diverse functions of DNA glycosylases processing oxidative base lesions in brain. DNA Repair. 81</cite> [<a href="https://doi.org/10.1016/j.dnarep.2019.102665" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31327582/" class="usa-link">PubMed</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Bjørge MD et al (2015) Synergistic actions of Ogg1 and mutyh DNA glycosylases modulate Anxiety-like behavior in mice. Cell Rep. 10.1016/j.celrep.2015.12.001
</cite> [<a href="https://doi.org/10.1016/j.celrep.2015.12.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26711335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bj%C3%B8rge%20MD%20et%20al%20(2015)%20Synergistic%20actions%20of%20Ogg1%20and%20mutyh%20DNA%20glycosylases%20modulate%20Anxiety-like%20behavior%20in%20mice.%20Cell%20Rep.%2010.1016/j.celrep.2015.12.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Hao W et al (2020) Enzymatically inactive OGG1 binds to DNA and steers base excision repair toward gene transcription. FASEB J 34:7427–7441
</cite> [<a href="https://doi.org/10.1096/fj.201902243R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7318607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32378256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hao%20W%20et%20al%20(2020)%20Enzymatically%20inactive%20OGG1%20binds%20to%20DNA%20and%20steers%20base%20excision%20repair%20toward%20gene%20transcription.%20FASEB%20J%2034:7427%E2%80%937441" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Spruijt CG et al (2013) Dynamic readers for 5-(Hydroxy)methylcytosine and its oxidized derivatives. Cell. 10.1016/j.cell.2013.02.004
</cite> [<a href="https://doi.org/10.1016/j.cell.2013.02.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23434322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spruijt%20CG%20et%20al%20(2013)%20Dynamic%20readers%20for%205-(Hydroxy)methylcytosine%20and%20its%20oxidized%20derivatives.%20Cell.%2010.1016/j.cell.2013.02.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Bogdanović O et al (2016) Active DNA demethylation at enhancers during the vertebrate phylotypic period. Nat Genet. 10.1038/ng.3522
</cite> [<a href="https://doi.org/10.1038/ng.3522" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5912259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26928226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bogdanovi%C4%87%20O%20et%20al%20(2016)%20Active%20DNA%20demethylation%20at%20enhancers%20during%20the%20vertebrate%20phylotypic%20period.%20Nat%20Genet.%2010.1038/ng.3522" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Maiti A, Drohat AC (2011) Thymine DNA glycosylase can rapidly excise 5-formylcytosine and 5-carboxylcytosine: potential implications for active demethylation of CpG sites. J Biol Chem 286:35334–35338
</cite> [<a href="https://doi.org/10.1074/jbc.C111.284620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3195571/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21862836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maiti%20A,%20Drohat%20AC%20(2011)%20Thymine%20DNA%20glycosylase%20can%20rapidly%20excise%205-formylcytosine%20and%205-carboxylcytosine:%20potential%20implications%20for%20active%20demethylation%20of%20CpG%20sites.%20J%20Biol%20Chem%20286:35334%E2%80%9335338" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Schomacher L et al (2016) Neil DNA glycosylases promote substrate turnover by Tdg during DNA demethylation. Nat Struct Mol Biology 23:116–124</cite> [<a href="https://doi.org/10.1038/nsmb.3151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4894546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26751644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schomacher%20L%20et%20al%20(2016)%20Neil%20DNA%20glycosylases%20promote%20substrate%20turnover%20by%20Tdg%20during%20DNA%20demethylation.%20Nat%20Struct%20Mol%20Biology%2023:116%E2%80%93124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Zhao G et al (2017) CHD4 has oncogenic functions in initiating and maintaining epigenetic suppression of multiple tumor suppressor genes. Cancer Cell 82:34–44</cite> [<a href="https://doi.org/10.1016/j.ccell.2017.04.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5587180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28486105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20G%20et%20al%20(2017)%20CHD4%20has%20oncogenic%20functions%20in%20initiating%20and%20maintaining%20epigenetic%20suppression%20of%20multiple%20tumor%20suppressor%20genes.%20Cancer%20Cell%2082:34%E2%80%9344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Li YQ, Zhou PZ, Zheng XD, Walsh CP, Xu GL (2007) Association of Dnmt3a and thymine DNA glycosylase links DNA methylation with base-excision repair. Nucleic Acids Res 35:390–400
</cite> [<a href="https://doi.org/10.1093/nar/gkl1052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1802599/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17175537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20YQ,%20Zhou%20PZ,%20Zheng%20XD,%20Walsh%20CP,%20Xu%20GL%20(2007)%20Association%20of%20Dnmt3a%20and%20thymine%20DNA%20glycosylase%20links%20DNA%20methylation%20with%20base-excision%20repair.%20Nucleic%20Acids%20Res%2035:390%E2%80%93400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Perillo B et al (2008) DNA oxidation as triggered by H3K9me2 demethylation drives estrogen-induced gene expression. Science 319:202–206
</cite> [<a href="https://doi.org/10.1126/science.1147674" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18187655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perillo%20B%20et%20al%20(2008)%20DNA%20oxidation%20as%20triggered%20by%20H3K9me2%20demethylation%20drives%20estrogen-induced%20gene%20expression.%20Science%20319:202%E2%80%93206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>O’Hagan HM et al (2011) Oxidative damage targets complexes containing DNA methyltransferases, SIRT1, and polycomb members to promoter CpG Islands. Cancer Cell 20:606–619
</cite> [<a href="https://doi.org/10.1016/j.ccr.2011.09.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3220885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22094255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?O%E2%80%99Hagan%20HM%20et%20al%20(2011)%20Oxidative%20damage%20targets%20complexes%20containing%20DNA%20methyltransferases,%20SIRT1,%20and%20polycomb%20members%20to%20promoter%20CpG%20Islands.%20Cancer%20Cell%2020:606%E2%80%93619" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Giorgio M, Dellino IG, Gambino V, Roda N, Pelicci PG (2020) On the epigenetic role of Guanosine oxidation. Redox Biol 29:101398
</cite> [<a href="https://doi.org/10.1016/j.redox.2019.101398" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6926346/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31926624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Giorgio%20M,%20Dellino%20IG,%20Gambino%20V,%20Roda%20N,%20Pelicci%20PG%20(2020)%20On%20the%20epigenetic%20role%20of%20Guanosine%20oxidation.%20Redox%20Biol%2029:101398" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Pastukh V et al (2015) An oxidative DNA damage and repair mechanism localized in the VEGF promoter is important for hypoxia-induced VEGF mRNA expression. Am J Physiol - Lung Cell Mol Physiol 309:L1367–L1375
</cite> [<a href="https://doi.org/10.1152/ajplung.00236.2015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4669343/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26432868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pastukh%20V%20et%20al%20(2015)%20An%20oxidative%20DNA%20damage%20and%20repair%20mechanism%20localized%20in%20the%20VEGF%20promoter%20is%20important%20for%20hypoxia-induced%20VEGF%20mRNA%20expression.%20Am%20J%20Physiol%20-%20Lung%20Cell%20Mol%20Physiol%20309:L1367%E2%80%93L1375" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Tobias Obermann T, Sakshaug VV, Kanagaraj A, Abentung AS, Bjørås M, Scheffler (2024) Genomic 8-oxoguanine modulates gene transcription independent of its repair by DNA glycosylases OGG1 and MUTYH. BioRxiv Preprint. 10.1101/2024.02.20.581185</cite> [<a href="https://doi.org/10.1016/j.redox.2024.103461" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11697278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39662289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tobias%20Obermann%20T,%20Sakshaug%20VV,%20Kanagaraj%20A,%20Abentung%20AS,%20Bj%C3%B8r%C3%A5s%20M,%20Scheffler%20(2024)%20Genomic%208-oxoguanine%20modulates%20gene%20transcription%20independent%20of%20its%20repair%20by%20DNA%20glycosylases%20OGG1%20and%20MUTYH.%20BioRxiv%20Preprint.%2010.1101/2024.02.20.581185" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Jaitner C et al (2016) Satb2 determines MiRNA expression and long-term memory in the adult central nervous system. eLife. 10.7554/eLife.17361
</cite> [<a href="https://doi.org/10.7554/eLife.17361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5207769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27897969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jaitner%20C%20et%20al%20(2016)%20Satb2%20determines%20MiRNA%20expression%20and%20long-term%20memory%20in%20the%20adult%20central%20nervous%20system.%20eLife.%2010.7554/eLife.17361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Blanchard RJ, Blanchard DC (1969) Crouching as an index of fear. J Comp Physiological Psychol. 10.1037/h0026779</cite> [<a href="https://doi.org/10.1037/h0026779" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5787388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blanchard%20RJ,%20Blanchard%20DC%20(1969)%20Crouching%20as%20an%20index%20of%20fear.%20J%20Comp%20Physiological%20Psychol.%2010.1037/h0026779" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Lehner M et al (2010) The relationship between pain sensitivity and conditioned fear response in rats. Acta Neurobiol Exp 70(1): 56–66. 10.55782/ane-2010-1774</cite> [<a href="https://doi.org/10.55782/ane-2010-1774" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20407487/" class="usa-link">PubMed</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Li N et al (2010) Whole genome DNA methylation analysis based on high throughput sequencing technology. Methods 52:203–212
</cite> [<a href="https://doi.org/10.1016/j.ymeth.2010.04.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20430099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20N%20et%20al%20(2010)%20Whole%20genome%20DNA%20methylation%20analysis%20based%20on%20high%20throughput%20sequencing%20technology.%20Methods%2052:203%E2%80%93212" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Halder R et al (2015) DNA methylation changes in plasticity genes accompany the formation and maintenance of memory. Nat Neurosci. 10.1038/nn.4194
</cite> [<a href="https://doi.org/10.1038/nn.4194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26656643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Halder%20R%20et%20al%20(2015)%20DNA%20methylation%20changes%20in%20plasticity%20genes%20accompany%20the%20formation%20and%20maintenance%20of%20memory.%20Nat%20Neurosci.%2010.1038/nn.4194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Krueger F, Andrews SR, Bismark (2011) A flexible aligner and methylation caller for Bisulfite-Seq applications. Bioinformatics 27:1571–1572
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3102221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21493656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krueger%20F,%20Andrews%20SR,%20Bismark%20(2011)%20A%20flexible%20aligner%20and%20methylation%20caller%20for%20Bisulfite-Seq%20applications.%20Bioinformatics%2027:1571%E2%80%931572" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Farooq A et al (2020) HMST-Seq-Analyzer: A new python tool for differential methylation and hydroxymethylation analysis in various DNA methylation sequencing data. Comput Struct Biotechnol J 18:2877–2889
</cite> [<a href="https://doi.org/10.1016/j.csbj.2020.09.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7593523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33163148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Farooq%20A%20et%20al%20(2020)%20HMST-Seq-Analyzer:%20A%20new%20python%20tool%20for%20differential%20methylation%20and%20hydroxymethylation%20analysis%20in%20various%20DNA%20methylation%20sequencing%20data.%20Comput%20Struct%20Biotechnol%20J%2018:2877%E2%80%932889" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Langmead B, Salzberg SL (2012) Fast gapped-read alignment with bowtie 2. Nat Methods 9:357–359
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead%20B,%20Salzberg%20SL%20(2012)%20Fast%20gapped-read%20alignment%20with%20bowtie%202.%20Nat%20Methods%209:357%E2%80%93359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Li H et al (2009) The sequence alignment/map format and samtools. Bioinformatics 25:2078–2079
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2723002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H%20et%20al%20(2009)%20The%20sequence%20alignment/map%20format%20and%20samtools.%20Bioinformatics%2025:2078%E2%80%932079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Ramírez F et al (2016) deepTools2: a next generation web server for deep-sequencing data analysis. Nucleic Acids Res 44:W160–W165
</cite> [<a href="https://doi.org/10.1093/nar/gkw257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4987876/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27079975/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ram%C3%ADrez%20F%20et%20al%20(2016)%20deepTools2:%20a%20next%20generation%20web%20server%20for%20deep-sequencing%20data%20analysis.%20Nucleic%20Acids%20Res%2044:W160%E2%80%93W165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Stovner EB, Sætrom P (2019) Epic2 efficiently finds diffuse domains in ChIP-seq data. Bioinformatics 35:4392–4393
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30923821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stovner%20EB,%20S%C3%A6trom%20P%20(2019)%20Epic2%20efficiently%20finds%20diffuse%20domains%20in%20ChIP-seq%20data.%20Bioinformatics%2035:4392%E2%80%934393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Qianwen Wang M, Li T, Wu L, Zhan L, Li M, Chen W, Xie Z, Xie E, Hu S, Xu GY (2022) Exploring epigenomic datasets by chipseeker. Curr Protoc. 10.1002/cpz1.585
</cite> [<a href="https://doi.org/10.1002/cpz1.585" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36286622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qianwen%20Wang%20M,%20Li%20T,%20Wu%20L,%20Zhan%20L,%20Li%20M,%20Chen%20W,%20Xie%20Z,%20Xie%20E,%20Hu%20S,%20Xu%20GY%20(2022)%20Exploring%20epigenomic%20datasets%20by%20chipseeker.%20Curr%20Protoc.%2010.1002/cpz1.585" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Oliveros JC An interactive tool for comparing lists with Venn’s diagrams</cite>
</li>
<li id="CR39">
<span class="label">39.</span><cite>Tyanova S et al (2016) The perseus computational platform for comprehensive analysis of (prote)omics data. Nat Methods 13:731–740
</cite> [<a href="https://doi.org/10.1038/nmeth.3901" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27348712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tyanova%20S%20et%20al%20(2016)%20The%20perseus%20computational%20platform%20for%20comprehensive%20analysis%20of%20(prote)omics%20data.%20Nat%20Methods%2013:731%E2%80%93740" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Kim D, Paggi JM, Park C, Bennett C, Salzberg SL (2019) Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Nat Biotechnol 37:907–915
</cite> [<a href="https://doi.org/10.1038/s41587-019-0201-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7605509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31375807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20D,%20Paggi%20JM,%20Park%20C,%20Bennett%20C,%20Salzberg%20SL%20(2019)%20Graph-based%20genome%20alignment%20and%20genotyping%20with%20HISAT2%20and%20HISAT-genotype.%20Nat%20Biotechnol%2037:907%E2%80%93915" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Liao Y, Smyth GK, Shi W (2019) The R package Rsubread is easier, faster, cheaper and better for alignment and quantification of RNA sequencing reads. Nucleic Acids Res 47</cite> [<a href="https://doi.org/10.1093/nar/gkz114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6486549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30783653/" class="usa-link">PubMed</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Robinson MD, McCarthy DJ, Smyth GK, edgeR (2009) A bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26:139–140
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2796818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Robinson%20MD,%20McCarthy%20DJ,%20Smyth%20GK,%20edgeR%20(2009)%20A%20bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data.%20Bioinformatics%2026:139%E2%80%93140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Zou Z, Ohta T, Oki S, ChIP-Atlas (2024) 3.0: a data-mining suite to explore chromosome architecture together with large-scale regulome data. Nucleic Acids Res 1–9. 10.1093/nar/gkae358</cite> [<a href="https://doi.org/10.1093/nar/gkae358" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11223792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38749504/" class="usa-link">PubMed</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Lee JJ et al (2018) Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals. Nat Genet 50:1112–1121
</cite> [<a href="https://doi.org/10.1038/s41588-018-0147-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6393768/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30038396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20JJ%20et%20al%20(2018)%20Gene%20discovery%20and%20polygenic%20prediction%20from%20a%20genome-wide%20association%20study%20of%20educational%20attainment%20in%201.1%20million%20individuals.%20Nat%20Genet%2050:1112%E2%80%931121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Savage JE et al (2018) Genome-wide association meta-analysis in 269,867 individuals identifies new genetic and functional links to intelligence. Nat Genet 50:912–919
</cite> [<a href="https://doi.org/10.1038/s41588-018-0152-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6411041/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29942086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Savage%20JE%20et%20al%20(2018)%20Genome-wide%20association%20meta-analysis%20in%20269,867%20individuals%20identifies%20new%20genetic%20and%20functional%20links%20to%20intelligence.%20Nat%20Genet%2050:912%E2%80%93919" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>van der Meer D et al (2020) Brain scans from 21,297 individuals reveal the genetic architecture of hippocampal subfield volumes. Mol Psychiatry 25:3053–3065
</cite> [<a href="https://doi.org/10.1038/s41380-018-0262-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6445783/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30279459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20der%20Meer%20D%20et%20al%20(2020)%20Brain%20scans%20from%2021,297%20individuals%20reveal%20the%20genetic%20architecture%20of%20hippocampal%20subfield%20volumes.%20Mol%20Psychiatry%2025:3053%E2%80%933065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Richards AL et al (2022) Genetic liabilities differentiating bipolar disorder, schizophrenia, and major depressive disorder, and phenotypic heterogeneity in bipolar disorder. JAMA Psychiatry 79:1032–1039
</cite> [<a href="https://doi.org/10.1001/jamapsychiatry.2022.2594" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9434480/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36044200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Richards%20AL%20et%20al%20(2022)%20Genetic%20liabilities%20differentiating%20bipolar%20disorder,%20schizophrenia,%20and%20major%20depressive%20disorder,%20and%20phenotypic%20heterogeneity%20in%20bipolar%20disorder.%20JAMA%20Psychiatry%2079:1032%E2%80%931039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Malik R et al (2018) Multiancestry genome-wide association study of 520,000 subjects identifies 32 loci associated with stroke and stroke subtypes. Nat Genet 50:524–537
</cite> [<a href="https://doi.org/10.1038/s41588-018-0058-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5968830/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29531354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Malik%20R%20et%20al%20(2018)%20Multiancestry%20genome-wide%20association%20study%20of%20520,000%20subjects%20identifies%2032%20loci%20associated%20with%20stroke%20and%20stroke%20subtypes.%20Nat%20Genet%2050:524%E2%80%93537" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Wightman DP et al (2023) The genetic overlap between Alzheimer’s disease, amyotrophic lateral sclerosis, lewy body dementia, and Parkinson’s disease. Neurobiol Aging 127:99–112
</cite> [<a href="https://doi.org/10.1016/j.neurobiolaging.2023.03.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37045620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wightman%20DP%20et%20al%20(2023)%20The%20genetic%20overlap%20between%20Alzheimer%E2%80%99s%20disease,%20amyotrophic%20lateral%20sclerosis,%20lewy%20body%20dementia,%20and%20Parkinson%E2%80%99s%20disease.%20Neurobiol%20Aging%20127:99%E2%80%93112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Grove J et al (2019) Identification of common genetic risk variants for autism spectrum disorder. Nat Genet 51:431–444
</cite> [<a href="https://doi.org/10.1038/s41588-019-0344-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6454898/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30804558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grove%20J%20et%20al%20(2019)%20Identification%20of%20common%20genetic%20risk%20variants%20for%20autism%20spectrum%20disorder.%20Nat%20Genet%2051:431%E2%80%93444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Xue A et al (2018) Genome-wide association analyses identify 143 risk variants and putative regulatory mechanisms for type 2 diabetes. Nat Commun 9</cite> [<a href="https://doi.org/10.1038/s41467-018-04951-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6063971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30054458/" class="usa-link">PubMed</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Miller CA, Sweatt JD (2008) Covalent modification of DNA regulates memory formation (DOI:10.1016/j.neuron.2007.02.022). Neuron 59:1051</cite> [<a href="https://doi.org/10.1016/j.neuron.2007.02.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17359920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miller%20CA,%20Sweatt%20JD%20(2008)%20Covalent%20modification%20of%20DNA%20regulates%20memory%20formation%20(DOI:10.1016/j.neuron.2007.02.022).%20Neuron%2059:1051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Zovkic IB, Guzman-Karlsson MC, Sweatt JD (2013) Epigenetic regulation of memory formation and maintenance. Learn Memory. 10.1101/lm.026575.112</cite> [<a href="https://doi.org/10.1101/lm.026575.112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3549063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23322554/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zovkic%20IB,%20Guzman-Karlsson%20MC,%20Sweatt%20JD%20(2013)%20Epigenetic%20regulation%20of%20memory%20formation%20and%20maintenance.%20Learn%20Memory.%2010.1101/lm.026575.112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Margueron R, Reinberg D (2011) The polycomb complex PRC2 and its mark in life. Nature. 10.1038/nature09784
</cite> [<a href="https://doi.org/10.1038/nature09784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3760771/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21248841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Margueron%20R,%20Reinberg%20D%20(2011)%20The%20polycomb%20complex%20PRC2%20and%20its%20mark%20in%20life.%20Nature.%2010.1038/nature09784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Ketel CS et al (2005) Subunit contributions to histone methyltransferase activities of fly and worm polycomb group complexes. Mol Cell Biol 25:6857–6868
</cite> [<a href="https://doi.org/10.1128/MCB.25.16.6857-6868.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1190254/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16055700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ketel%20CS%20et%20al%20(2005)%20Subunit%20contributions%20to%20histone%20methyltransferase%20activities%20of%20fly%20and%20worm%20polycomb%20group%20complexes.%20Mol%20Cell%20Biol%2025:6857%E2%80%936868" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Nekrasov M, Wild B, Müller J (2005) Nucleosome binding and histone methyltransferase activity of Drosophila PRC2. EMBO Rep 6:348–353
</cite> [<a href="https://doi.org/10.1038/sj.embor.7400376" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1299286/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15776017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nekrasov%20M,%20Wild%20B,%20M%C3%BCller%20J%20(2005)%20Nucleosome%20binding%20and%20histone%20methyltransferase%20activity%20of%20Drosophila%20PRC2.%20EMBO%20Rep%206:348%E2%80%93353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Jadhav U et al (2016) Acquired tissue-specific promoter bivalency is a basis for PRC2 necessity in adult cells. Cell 165:1389–1400
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.04.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4893000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27212235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jadhav%20U%20et%20al%20(2016)%20Acquired%20tissue-specific%20promoter%20bivalency%20is%20a%20basis%20for%20PRC2%20necessity%20in%20adult%20cells.%20Cell%20165:1389%E2%80%931400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>de Leeuw CA, Mooij JM, Heskes T, Posthuma DMAGMA (2015) Generalized Gene-Set analysis of GWAS data. PLoS Comput Biol 11:1–19</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1004219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4401657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25885710/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Leeuw%20CA,%20Mooij%20JM,%20Heskes%20T,%20Posthuma%20DMAGMA%20(2015)%20Generalized%20Gene-Set%20analysis%20of%20GWAS%20data.%20PLoS%20Comput%20Biol%2011:1%E2%80%9319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Canugovi C et al (2012) Endonuclease VIII-like 1 (NEIL1) promotes short-term Spatial memory retention and protects from ischemic stroke-induced brain dysfunction and death in mice. Proc Natl Acad Sci USA 109:14948–14953
</cite> [<a href="https://doi.org/10.1073/pnas.1204156109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3443144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22927410/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Canugovi%20C%20et%20al%20(2012)%20Endonuclease%20VIII-like%201%20(NEIL1)%20promotes%20short-term%20Spatial%20memory%20retention%20and%20protects%20from%20ischemic%20stroke-induced%20brain%20dysfunction%20and%20death%20in%20mice.%20Proc%20Natl%20Acad%20Sci%20USA%20109:14948%E2%80%9314953" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Regnell CE et al (2012) Hippocampal adult neurogenesis is maintained by Neil3-dependent repair of oxidative DNA lesions in neural progenitor cells. Cell Rep 2:503–510
</cite> [<a href="https://doi.org/10.1016/j.celrep.2012.08.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22959434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Regnell%20CE%20et%20al%20(2012)%20Hippocampal%20adult%20neurogenesis%20is%20maintained%20by%20Neil3-dependent%20repair%20of%20oxidative%20DNA%20lesions%20in%20neural%20progenitor%20cells.%20Cell%20Rep%202:503%E2%80%93510" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Lipinski M et al (2022) CBP is required for Establishing adaptive gene programs in the adult mouse brain. J Neurosci 42:7984–8001
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.0970-22.2022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9617619/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36109165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lipinski%20M%20et%20al%20(2022)%20CBP%20is%20required%20for%20Establishing%20adaptive%20gene%20programs%20in%20the%20adult%20mouse%20brain.%20J%20Neurosci%2042:7984%E2%80%938001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Korzus E, Rosenfeld MG, Mayford M (2021) Memory Consolidation 42:961–972</cite> [<a href="https://doi.org/10.1016/j.neuron.2004.06.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8048715/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15207240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Korzus%20E,%20Rosenfeld%20MG,%20Mayford%20M%20(2021)%20Memory%20Consolidation%2042:961%E2%80%93972" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Spruijt CG et al (2013) Dynamic readers for 5-(Hydroxy)methylcytosine and its oxidized derivatives. Cell 152:1146–1159
</cite> [<a href="https://doi.org/10.1016/j.cell.2013.02.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23434322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spruijt%20CG%20et%20al%20(2013)%20Dynamic%20readers%20for%205-(Hydroxy)methylcytosine%20and%20its%20oxidized%20derivatives.%20Cell%20152:1146%E2%80%931159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Michaels ML, Cruz C, Grollman AP, Miller JH (1992) Evidence that muty and mutm combine to prevent mutations by an oxidatively damaged form of guanine in DNA. Proc Natl Acad Sci USA 89:7022–7025
</cite> [<a href="https://doi.org/10.1073/pnas.89.15.7022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC49637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1495996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Michaels%20ML,%20Cruz%20C,%20Grollman%20AP,%20Miller%20JH%20(1992)%20Evidence%20that%20muty%20and%20mutm%20combine%20to%20prevent%20mutations%20by%20an%20oxidatively%20damaged%20form%20of%20guanine%20in%20DNA.%20Proc%20Natl%20Acad%20Sci%20USA%2089:7022%E2%80%937025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Pan L et al (2017) OGG1-DNA interactions facilitate NF-k B binding to DNA targets. Sci Rep. 10.1038/srep43297
</cite> [<a href="https://doi.org/10.1038/srep43297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5339705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28266569/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan%20L%20et%20al%20(2017)%20OGG1-DNA%20interactions%20facilitate%20NF-k%20B%20binding%20to%20DNA%20targets.%20Sci%20Rep.%2010.1038/srep43297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Lago S et al (2021) Promoter G-quadruplexes and transcription factors cooperate to shape the cell type-specific transcriptome. Nat Commun 12:1–13
</cite> [<a href="https://doi.org/10.1038/s41467-021-24198-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8222265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34162892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lago%20S%20et%20al%20(2021)%20Promoter%20G-quadruplexes%20and%20transcription%20factors%20cooperate%20to%20shape%20the%20cell%20type-specific%20transcriptome.%20Nat%20Commun%2012:1%E2%80%9313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Gorini F, Ambrosio S, Lania L, Majello B, Amente S (2023) The intertwined role of 8-oxodG and G4 in transcription regulation. Int J Mol Sci 24:1–12</cite> [<a href="https://doi.org/10.3390/ijms24032031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9916577/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36768357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gorini%20F,%20Ambrosio%20S,%20Lania%20L,%20Majello%20B,%20Amente%20S%20(2023)%20The%20intertwined%20role%20of%208-oxodG%20and%20G4%20in%20transcription%20regulation.%20Int%20J%20Mol%20Sci%2024:1%E2%80%9312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Von Schimmelmann M et al (2016) Polycomb repressive complex 2 (PRC2) silences genes responsible for neurodegeneration. Nat Neurosci. 10.1038/nn.4360</cite> [<a href="https://doi.org/10.1038/nn.4360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5088783/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27526204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Von%20Schimmelmann%20M%20et%20al%20(2016)%20Polycomb%20repressive%20complex%202%20(PRC2)%20silences%20genes%20responsible%20for%20neurodegeneration.%20Nat%20Neurosci.%2010.1038/nn.4360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Ayata P et al (2018) Epigenetic regulation of brain region-specific microglia clearance activity. Nat Neurosci 21:1049–1060
</cite> [<a href="https://doi.org/10.1038/s41593-018-0192-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6090564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30038282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ayata%20P%20et%20al%20(2018)%20Epigenetic%20regulation%20of%20brain%20region-specific%20microglia%20clearance%20activity.%20Nat%20Neurosci%2021:1049%E2%80%931060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Zocher S (2024) Targeting neuronal epigenomes for brain rejuvenation. EMBO J 43:3312–3326
</cite> [<a href="https://doi.org/10.1038/s44318-024-00148-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11329789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39009672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zocher%20S%20(2024)%20Targeting%20neuronal%20epigenomes%20for%20brain%20rejuvenation.%20EMBO%20J%2043:3312%E2%80%933326" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Magistretti PJ, Allaman I (2015) A cellular perspective on brain energy metabolism and functional imaging. Neuron 86:883–901
</cite> [<a href="https://doi.org/10.1016/j.neuron.2015.03.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25996133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Magistretti%20PJ,%20Allaman%20I%20(2015)%20A%20cellular%20perspective%20on%20brain%20energy%20metabolism%20and%20functional%20imaging.%20Neuron%2086:883%E2%80%93901" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Lundgaard I et al (2015) Direct neuronal glucose uptake heralds activity-dependent increases in cerebral metabolism. Nat Commun 6:1–12</cite> [<a href="https://doi.org/10.1038/ncomms7807" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4410436/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25904018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lundgaard%20I%20et%20al%20(2015)%20Direct%20neuronal%20glucose%20uptake%20heralds%20activity-dependent%20increases%20in%20cerebral%20metabolism.%20Nat%20Commun%206:1%E2%80%9312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Mann K, Deny S, Ganguli S, Clandinin TR (2021) Coupling of activity, metabolism and behaviour across the Drosophila brain. Nature 593:244–248
</cite> [<a href="https://doi.org/10.1038/s41586-021-03497-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10544789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33911283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mann%20K,%20Deny%20S,%20Ganguli%20S,%20Clandinin%20TR%20(2021)%20Coupling%20of%20activity,%20metabolism%20and%20behaviour%20across%20the%20Drosophila%20brain.%20Nature%20593:244%E2%80%93248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Alexandrov YI, Pletnikov MV (2022) Neuronal metabolism in learning and memory: the anticipatory activity perspective. Neurosci Biobehav Rev 137:104664
</cite> [<a href="https://doi.org/10.1016/j.neubiorev.2022.104664" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35439520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alexandrov%20YI,%20Pletnikov%20MV%20(2022)%20Neuronal%20metabolism%20in%20learning%20and%20memory:%20the%20anticipatory%20activity%20perspective.%20Neurosci%20Biobehav%20Rev%20137:104664" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Hardcastle C et al (2020) Contributions of hippocampal volume to cognition in healthy older adults. Front Aging Neurosci 12:1–10
</cite> [<a href="https://doi.org/10.3389/fnagi.2020.593833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7674177/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33250765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hardcastle%20C%20et%20al%20(2020)%20Contributions%20of%20hippocampal%20volume%20to%20cognition%20in%20healthy%20older%20adults.%20Front%20Aging%20Neurosci%2012:1%E2%80%9310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Nave KA (2010) Myelination and support of axonal integrity by glia. Nature 468:244–252
</cite> [<a href="https://doi.org/10.1038/nature09614" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21068833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nave%20KA%20(2010)%20Myelination%20and%20support%20of%20axonal%20integrity%20by%20glia.%20Nature%20468:244%E2%80%93252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Schafer DP et al (2012) Microglia sculpt postnatal neural circuits in an activity and Complement-Dependent manner. Neuron 74:691–705
</cite> [<a href="https://doi.org/10.1016/j.neuron.2012.03.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3528177/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22632727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schafer%20DP%20et%20al%20(2012)%20Microglia%20sculpt%20postnatal%20neural%20circuits%20in%20an%20activity%20and%20Complement-Dependent%20manner.%20Neuron%2074:691%E2%80%93705" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Han X et al (2013) Forebrain engraftment by human glial progenitor cells enhances synaptic plasticity and learning in adult mice. Cell Stem Cell 12:342–353
</cite> [<a href="https://doi.org/10.1016/j.stem.2012.12.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3700554/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23472873/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Han%20X%20et%20al%20(2013)%20Forebrain%20engraftment%20by%20human%20glial%20progenitor%20cells%20enhances%20synaptic%20plasticity%20and%20learning%20in%20adult%20mice.%20Cell%20Stem%20Cell%2012:342%E2%80%93353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Wake H, Lee PR, Fields RD (2011) Control of local protein synthesis and initial events in myelination by action potentials. Science 333:1647–1651
</cite> [<a href="https://doi.org/10.1126/science.1206998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3482340/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21817014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wake%20H,%20Lee%20PR,%20Fields%20RD%20(2011)%20Control%20of%20local%20protein%20synthesis%20and%20initial%20events%20in%20myelination%20by%20action%20potentials.%20Science%20333:1647%E2%80%931651" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Lewandowski KE, Cohen BM, Öngur D (2011) Evolution of neuropsychological dysfunction during the course of schizophrenia and bipolar disorder. Psychol Med 41:225–241
</cite> [<a href="https://doi.org/10.1017/S0033291710001042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20836900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lewandowski%20KE,%20Cohen%20BM,%20%C3%96ngur%20D%20(2011)%20Evolution%20of%20neuropsychological%20dysfunction%20during%20the%20course%20of%20schizophrenia%20and%20bipolar%20disorder.%20Psychol%20Med%2041:225%E2%80%93241" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Arat Çelik HE et al (2024) Oxidatively-induced DNA base damage and base excision repair abnormalities in siblings of individuals with bipolar disorder DNA damage and repair in bipolar disorder. Translational Psychiatry 14</cite> [<a href="https://doi.org/10.1038/s41398-024-02901-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11126633/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38789433/" class="usa-link">PubMed</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Czarny P et al (2015) Association between single nucleotide polymorphisms of MUTYH, hOGG1 and NEIL1 genes, and depression. J Affect Disord 184:90–96
</cite> [<a href="https://doi.org/10.1016/j.jad.2015.05.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26074017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Czarny%20P%20et%20al%20(2015)%20Association%20between%20single%20nucleotide%20polymorphisms%20of%20MUTYH,%20hOGG1%20and%20NEIL1%20genes,%20and%20depression.%20J%20Affect%20Disord%20184:90%E2%80%9396" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12334403/bin/18_2025_5730_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (5.1MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All genomic data produced in the present project (Bisulfite-seq data, ChIP-seq and RNA-seq) have been deposited in the NCBI GEO database under accession number GSE272603.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Cellular and Molecular Life Sciences: CMLS are provided here courtesy of <strong>Springer</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1007/s00018-025-05730-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/18_2025_Article_5730.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12334403/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12334403/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12334403%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334403/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12334403/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12334403/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40779073/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12334403/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40779073/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12334403/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12334403/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="DiGhZX2w1h5Q4Xl8JODcjytrg7hEuEHaS4eBYdgRcw35FOz7xIOkWoKgctdzHfJU">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
