
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Inactivation of branched-chain amino acid uptake halts Staphylococcus aureus growth and induces bacterial quiescence within macrophages - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="FF2083E989B770231A83E90017CFEDB4.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plospath">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333996/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS Pathogens">
<meta name="citation_title" content="Inactivation of branched-chain amino acid uptake halts Staphylococcus aureus growth and induces bacterial quiescence within macrophages">
<meta name="citation_author" content="Adriana Moldovan">
<meta name="citation_author_institution" content="Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany">
<meta name="citation_author" content="Ronald S Flannagan">
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of Western Ontario, London, Ontario, Canada">
<meta name="citation_author" content="Marcel Rühling">
<meta name="citation_author_institution" content="Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany">
<meta name="citation_author" content="Kathrin Stelzner">
<meta name="citation_author_institution" content="Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany">
<meta name="citation_author" content="Clara Hans">
<meta name="citation_author_institution" content="Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany">
<meta name="citation_author" content="Kerstin Paprotka">
<meta name="citation_author_institution" content="Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany">
<meta name="citation_author" content="Tobias C Kunz">
<meta name="citation_author_institution" content="Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany">
<meta name="citation_author" content="David E Heinrichs">
<meta name="citation_author_institution" content="Department of Microbiology and Immunology, University of Western Ontario, London, Ontario, Canada">
<meta name="citation_author" content="Thomas Rudel">
<meta name="citation_author_institution" content="Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany">
<meta name="citation_author" content="Martin J Fraunholz">
<meta name="citation_author_institution" content="Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany">
<meta name="citation_publication_date" content="2025 Aug 8">
<meta name="citation_volume" content="21">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e1013291">
<meta name="citation_doi" content="10.1371/journal.ppat.1013291">
<meta name="citation_pmid" content="40779604">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333996/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333996/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333996/pdf/ppat.1013291.pdf">
<meta name="description" content="Staphylococcus aureus is a notorious human pathogen that thrives in macrophages. It resides in mature phagolysosomes, where a subset of the bacteria eventually begin to proliferate. How S. aureus acquires essential nutrients, such as amino acids, ...">
<meta name="og:title" content="Inactivation of branched-chain amino acid uptake halts Staphylococcus aureus growth and induces bacterial quiescence within macrophages">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Staphylococcus aureus is a notorious human pathogen that thrives in macrophages. It resides in mature phagolysosomes, where a subset of the bacteria eventually begin to proliferate. How S. aureus acquires essential nutrients, such as amino acids, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333996/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12333996">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.ppat.1013291"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ppat.1013291.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12333996%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12333996/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12333996/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333996/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plospath.png" alt="PLOS Pathogens logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS Pathogens" title="Link to PLOS Pathogens" shape="default" href="https://doi.org/10.1371/journal.ppat.1013291" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS Pathog</button></div>. 2025 Aug 8;21(8):e1013291. doi: <a href="https://doi.org/10.1371/journal.ppat.1013291" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.ppat.1013291</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20Pathog%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20Pathog%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20Pathog%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20Pathog%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Inactivation of branched-chain amino acid uptake halts <em>Staphylococcus aureus</em> growth and induces bacterial quiescence within macrophages</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moldovan%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Adriana Moldovan</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Adriana Moldovan</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany</div>
<div>Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Supervision, Validation, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moldovan%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Adriana Moldovan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Flannagan%20RS%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ronald S Flannagan</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ronald S Flannagan</span></h3>
<div class="p">
<sup>2</sup>Department of Microbiology and Immunology, University of Western Ontario, London, Ontario, Canada</div>
<div>Data curation, Formal analysis, Investigation, Methodology, Validation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Flannagan%20RS%22%5BAuthor%5D" class="usa-link"><span class="name western">Ronald S Flannagan</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22R%C3%BChling%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Marcel Rühling</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Marcel Rühling</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany</div>
<div>Data curation, Formal analysis, Funding acquisition, Investigation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22R%C3%BChling%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marcel Rühling</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stelzner%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Kathrin Stelzner</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kathrin Stelzner</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stelzner%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kathrin Stelzner</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hans%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Clara Hans</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Clara Hans</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hans%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Clara Hans</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Paprotka%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Kerstin Paprotka</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Kerstin Paprotka</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Paprotka%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kerstin Paprotka</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kunz%20TC%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Tobias C Kunz</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Tobias C Kunz</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kunz%20TC%22%5BAuthor%5D" class="usa-link"><span class="name western">Tobias C Kunz</span></a>
</div>
</div>
<sup>1,</sup><sup>¤</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heinrichs%20DE%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">David E Heinrichs</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">David E Heinrichs</span></h3>
<div class="p">
<sup>2</sup>Department of Microbiology and Immunology, University of Western Ontario, London, Ontario, Canada</div>
<div>Funding acquisition, Project administration, Supervision, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heinrichs%20DE%22%5BAuthor%5D" class="usa-link"><span class="name western">David E Heinrichs</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rudel%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Thomas Rudel</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Thomas Rudel</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany</div>
<div>Funding acquisition, Project administration, Supervision, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rudel%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thomas Rudel</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fraunholz%20MJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Martin J Fraunholz</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Martin J Fraunholz</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany</div>
<div>Data curation, Funding acquisition, Investigation, Methodology, Project administration, Supervision, Visualization, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fraunholz%20MJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin J Fraunholz</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Alice Prince,</span><sup>3</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Department of Microbiology, Biocenter, University of Würzburg, Würzburg, Germany</div>
<div id="aff002">
<sup>2</sup>Department of Microbiology and Immunology, University of Western Ontario, London, Ontario, Canada</div>
<div id="edit1">
<sup>3</sup>Columbia University, UNITED STATES OF AMERICA</div>
<div class="author-notes p">
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>martin.fraunholz@uni-wuerzburg.de</span> (MJF); <span>thomas.rudel@uni-wuerzburg.de</span> (TR)</p>
</div>
<div class="fn" id="coi001"><p>The authors have declared that no competing interests exist.</p></div>
<div class="fn" id="currentaff001">
<sup>¤</sup><p class="display-inline">Current address: Lonza AG, Visp, Switzerland</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Adriana Moldovan</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Supervision, Validation, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ronald S Flannagan</span></strong>: <span class="role">Data curation, Formal analysis, Investigation, Methodology, Validation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Marcel Rühling</span></strong>: <span class="role">Data curation, Formal analysis, Funding acquisition, Investigation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Kathrin Stelzner</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Clara Hans</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Kerstin Paprotka</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Tobias C Kunz</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">David E Heinrichs</span></strong>: <span class="role">Funding acquisition, Project administration, Supervision, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Thomas Rudel</span></strong>: <span class="role">Funding acquisition, Project administration, Supervision, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Martin J Fraunholz</span></strong>: <span class="role">Data curation, Funding acquisition, Investigation, Methodology, Project administration, Supervision, Visualization, Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Alice Prince,</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 10; Accepted 2025 Jun 10; Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Moldovan et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12333996  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40779604/" class="usa-link">40779604</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p><em>Staphylococcus aureus</em> is a notorious human pathogen that thrives in macrophages. It resides in mature phagolysosomes, where a subset of the bacteria eventually begin to proliferate. How <em>S. aureus</em> acquires essential nutrients, such as amino acids, for growth in this niche is poorly understood. Using a long-term primary human macrophage infection model, we show that branched-chain amino acid (BCAA) uptake mediated by the major transporter BrnQ1 is required by <em>S. aureus</em> for intracellular replication in macrophages and we provide mechanistic insight into the role of BCAAs in the success of intracellular <em>S. aureus.</em> Loss of BrnQ1 function renders intracellular <em>S. aureus</em> non-replicative and non-cytotoxic. The defective intracellular growth of <em>S. aureus brnQ1</em> mutants can be rescued by supplementation with BCAAs or by overexpression of the BCAA transporters BrnQ1 or BcaP. Inactivation of the CodY repressor rescues the ability of <em>S. aureus brnQ1</em> mutants to proliferate intracellularly independent of endogenous BCAA synthesis but dependent on BcaP expression. Non-replicating <em>brnQ1</em> mutants in primary human macrophages become metabolically quiescent and display aberrant gene expression marked by failure to respond to intraphagosomal iron starvation. The bacteria remain, however, viable for an inordinate length of time. This dormant, yet viable bacterial state is distinct from classical persisters and small colony variants.</p></section><section class="abstract" id="abstract2"><h2>Author summary</h2>
<p><em>Staphylococcus aureus</em> is a prominent human pathogen causing acute and chronic disease. It is facultatively intracellular and can reside within many host cell types, including professional phagocytes such as macrophages. The intracellular state contributes to dissemination, recurrence and infection chronicity. Chronic and relapsing infections are often associated with so-called <em>persister</em> phenotypes. Growth arrest and metabolic quiescence, accompanied by antibiotic tolerance, are hallmarks of <em>persistence</em> in bacteria. Antibiotic pressure is a major factor in triggering intracellular <em>persistence</em>. The small colony variant (SCV), an extensively studied form of <em>S. aureus persister</em>, can arise in the absence of antibiotic pressure and exhibits very distinctive phenotypic characteristics.</p>
<p>Here, we describe a different growth-arrested state of <em>S. aureus</em>, which conforms to the definition of a non-antibiotic-driven form of intracellular <em>dormancy,</em> triggered by branched-chain amino acid starvation in macrophages. We show that loss of function of the major branched-chain amino acid transporter BrnQ1 renders intracellular <em>S. aureus</em> non-replicative and metabolically quiescent for an inordinate period of time. Upon stochastic exit from infected macrophages, <em>brnQ1</em> mutants retain full virulence. This <em>dormancy</em> differs from classical <em>persistence</em> or SCVs and uncovers an underestimated role for BCAA uptake in the success of intracellular <em>S. aureus.</em></p></section><section id="sec001"><h2 class="pmc_sec_title">Introduction</h2>
<p><em>Staphylococcus aureus</em> is a Gram-positive prominent human opportunistic pathogen [<a href="#ppat.1013291.ref001" class="usa-link" aria-describedby="ppat.1013291.ref001">1</a>], capable of causing a variety of diseases ranging from superficial skin infections to life-threatening conditions (reviewed in [<a href="#ppat.1013291.ref002" class="usa-link" aria-describedby="ppat.1013291.ref002">2</a>]). It permanently colonizes 20–30% of the human population [<a href="#ppat.1013291.ref003" class="usa-link" aria-describedby="ppat.1013291.ref003">3</a>,<a href="#ppat.1013291.ref004" class="usa-link" aria-describedby="ppat.1013291.ref004">4</a>], with up to 60% carrying <em>S. aureus</em> intermittently during their lifetime [<a href="#ppat.1013291.ref005" class="usa-link" aria-describedby="ppat.1013291.ref005">5</a>].</p>
<p><em>S. aureus</em> survives and replicates in a variety of host cell types including non-professional phagocytes (such as epithelial and endothelial cells) and professional phagocytes such as macrophages (reviewed by [<a href="#ppat.1013291.ref006" class="usa-link" aria-describedby="ppat.1013291.ref006">6</a>,<a href="#ppat.1013291.ref007" class="usa-link" aria-describedby="ppat.1013291.ref007">7</a>]). Recently it has been proposed that the ability of <em>S. aureus</em> to exist intracellularly is a prevailing feature of the <em>S. aureus</em>-host interaction [<a href="#ppat.1013291.ref008" class="usa-link" aria-describedby="ppat.1013291.ref008">8</a>]. Among professional phagocytes, macrophages play a central role in innate immunity due to their ability to phagocytose bacteria and mount potent microbicidal responses. Indeed, macrophages deploy a multitude of antibacterial effectors such as reactive oxygen species, proteases, lipases, and antimicrobial peptides that function to intoxicate and/or degrade phagocytosed prey [<a href="#ppat.1013291.ref009" class="usa-link" aria-describedby="ppat.1013291.ref009">9</a>]. At the same time, macrophages curtail bacterial growth by extruding essential nutrients such as iron and manganese from the phagosome lumen via the host transporter NRAMP1, which is just one aspect of host nutritional immunity [<a href="#ppat.1013291.ref010" class="usa-link" aria-describedby="ppat.1013291.ref010">10</a>]. In addition to metals, macrophage phagolysosomes are presumably depleted of several amino acids owing to the activity of host amino acid transporters (e.g., SLC38A9, SLC66A1/PQLC2, SLC36A1, and SLC66A4/cystinosin) localised to phagosomal or lysosomal membranes, where they extrude amino acids including leucine, neutral amino acids, cationic amino acids and cysteine, respectively, into the host cell cytosol [<a href="#ppat.1013291.ref011" class="usa-link" aria-describedby="ppat.1013291.ref011">11</a>]. Together, metal and amino acid transport should deplete the phagolysosome of nutrients thereby hindering bacterial proliferation within macrophages. Despite this, macrophages fail to eradicate phagocytosed <em>S. aureus</em> [<a href="#ppat.1013291.ref012" class="usa-link" aria-describedby="ppat.1013291.ref012">12</a>–<a href="#ppat.1013291.ref015" class="usa-link" aria-describedby="ppat.1013291.ref015">15</a>] and following engulfment, bacteria can begin to replicate within mature phagolysosomes [<a href="#ppat.1013291.ref016" class="usa-link" aria-describedby="ppat.1013291.ref016">16</a>,<a href="#ppat.1013291.ref017" class="usa-link" aria-describedby="ppat.1013291.ref017">17</a>] independently of toxin production [<a href="#ppat.1013291.ref018" class="usa-link" aria-describedby="ppat.1013291.ref018">18</a>].</p>
<p>Branched-chain amino acids (BCAAs) - isoleucine (Ile), leucine (Leu) and valine (Val) - are essential amino acids that cannot be synthesized by mammals. In <em>S. aureus</em>, BCAAs are not only proteinogenic amino acids, but also serve as precursors for branched-chain fatty acids (BCFAs), which are required for the regulation of membrane fluidity [<a href="#ppat.1013291.ref019" class="usa-link" aria-describedby="ppat.1013291.ref019">19</a>]. In addition, Ile functions as a signalling molecule that, together with GTP [<a href="#ppat.1013291.ref020" class="usa-link" aria-describedby="ppat.1013291.ref020">20</a>,<a href="#ppat.1013291.ref021" class="usa-link" aria-describedby="ppat.1013291.ref021">21</a>], modulates the activation of CodY, a global transcriptional regulator of metabolism and pathogenesis in most low G + C Gram-positive bacteria including <em>S. aureus</em> [<a href="#ppat.1013291.ref022" class="usa-link" aria-describedby="ppat.1013291.ref022">22</a>].</p>
<p>BCAA transport and biosynthesis are therefore closely linked to virulence in <em>S. aureus</em> and other bacterial pathogens, indicating that access to BCAAs as a critical nutrient is required by pathogens to establish infection [<a href="#ppat.1013291.ref023" class="usa-link" aria-describedby="ppat.1013291.ref023">23</a>]. For example, when grown <em>in vitro</em> in the absence of Val, <em>S. aureus</em> displays a growth lag of ~20h, until BCAA biosynthesis is initiated [<a href="#ppat.1013291.ref021" class="usa-link" aria-describedby="ppat.1013291.ref021">21</a>]. To facilitate immediate BCAA uptake from the environment, <em>S. aureus</em> employs the BCAA transporters, BrnQ1 and BcaP, as well as BrnQ2, a dedicated Ile transporter [<a href="#ppat.1013291.ref024" class="usa-link" aria-describedby="ppat.1013291.ref024">24</a>]. In addition, <em>S. aureus</em> harbours BCAA biosynthetic genes encoded by the <em>ilvDBNC-leuABCD-ilvA</em> operon and <em>ilvE</em>. Both BCAA uptake and biosynthesis operons are repressed by CodY when intracellular BCAA levels are sufficient to support growth [<a href="#ppat.1013291.ref021" class="usa-link" aria-describedby="ppat.1013291.ref021">21</a>]. CodY controls the expression of over 200 genes in <em>S. aureus</em> [<a href="#ppat.1013291.ref025" class="usa-link" aria-describedby="ppat.1013291.ref025">25</a>–<a href="#ppat.1013291.ref027" class="usa-link" aria-describedby="ppat.1013291.ref027">27</a>] where it acts primarily as a repressor of metabolic and virulence genes upon binding of Ile and GTP. Upon nutrient depletion, CodY target genes were shown to be de-repressed sequentially, along the BCAA gradient, with genes involved in amino acid uptake (including BCAAs) being turned on much earlier than biosynthesis genes [<a href="#ppat.1013291.ref028" class="usa-link" aria-describedby="ppat.1013291.ref028">28</a>].The requirement for BCAA acquisition for <em>S. aureus</em> pathogenesis is underlined by the fact that <em>brnQ1</em>/<em>bca</em>P mutants are attenuated in murine models of infection [<a href="#ppat.1013291.ref024" class="usa-link" aria-describedby="ppat.1013291.ref024">24</a>,<a href="#ppat.1013291.ref029" class="usa-link" aria-describedby="ppat.1013291.ref029">29</a>].</p>
<p>We here reveal a critical role for high-affinity BCAA transport in the growth and adaptation of <em>S. aureus</em> to the macrophage phagolysosome. Using defined mutants lacking specific BCAA transporters or the ability to synthesize BCAA, we uncover a pivotal role for the high-affinity BCAA importer BrnQ1 in the replication and subsequent survival of <em>S. aureus</em> within macrophages. We provide evidence that <em>brnQ1</em>- deficient <em>S. aureus</em> display a phagolysosomal growth defect that can be overcome by inactivation of the transcription factor CodY, by exogenous supply of BCAAs, or by heterologous expression of either BrnQ1 or the secondary BCAA transporter BcaP. In the absence of BCAA supplementation of the macrophage culture medium, <em>brnQ1</em> bacteria remain intracellular for an inordinate duration and adopt a <em>dormant</em> state, that is distinct from established persister cell or small colony variant (SCV) phenotypes.</p></section><section id="sec002"><h2 class="pmc_sec_title">Results</h2>
<section id="sec003"><h3 class="pmc_sec_title">S<em>. aureus</em> deficient for the BrnQ1 branched-chain amino acid transporter do not replicate in human macrophages</h3>
<p><em>S. aureus</em> are internalized by professional phagocytes such as macrophages, where they survive and replicate within acidic phagolysosomes [<a href="#ppat.1013291.ref018" class="usa-link" aria-describedby="ppat.1013291.ref018">18</a>]. To test the possible involvement of BCAAs in the <em>S. aureus</em> – macrophage interaction we established a macrophage infection model, involving a short phagocytosis time, followed by a lysostaphin pulse-treatment to eradicate extracellular bacteria (<a href="#ppat.1013291.g001" class="usa-link">Fig 1a</a>). Here we purposefully avoided prolonged use of lysostaphin or gentamicin as antimicrobials as these can be internalised by macrophages and thus could impact the survival of intracellular bacteria [<a href="#ppat.1013291.ref012" class="usa-link" aria-describedby="ppat.1013291.ref012">12</a>,<a href="#ppat.1013291.ref031" class="usa-link" aria-describedby="ppat.1013291.ref031">31</a>–<a href="#ppat.1013291.ref033" class="usa-link" aria-describedby="ppat.1013291.ref033">33</a>]. Using macrophage colony-stimulating factor (M-CSF)-differentiated human monocyte-derived macrophages, we analysed the ability of individual <em>S. aureus</em> USA300 mutants retrieved from the Nebraska Transposon Library [<a href="#ppat.1013291.ref034" class="usa-link" aria-describedby="ppat.1013291.ref034">34</a>] to replicate and egress from infected macrophages. We analysed mutants deficient for the BCAA transporters BrnQ1, BrnQ2, BrnQ3 and BcaP, as well as mutants lacking the <em>ilvE</em>, <em>ilvD</em> and <em>leuA</em> genes, which are involved in BCAA biosynthesis [<a href="#ppat.1013291.ref021" class="usa-link" aria-describedby="ppat.1013291.ref021">21</a>]. A mutant in <em>codY</em>, the repressor of the aforementioned genes [<a href="#ppat.1013291.ref026" class="usa-link" aria-describedby="ppat.1013291.ref026">26</a>,<a href="#ppat.1013291.ref027" class="usa-link" aria-describedby="ppat.1013291.ref027">27</a>], known to exhibit a hypervirulent phenotype [<a href="#ppat.1013291.ref035" class="usa-link" aria-describedby="ppat.1013291.ref035">35</a>], was included as control (<a href="#ppat.1013291.g001" class="usa-link">Fig 1b</a>). This analysis revealed that by 24h post-infection (<em>p.i.</em>) all strains except the <em>brnQ1</em> mutant, had emerged from infected macrophages and massively replicated extracellularly, resulting in host cell death and the formation of micro-colonies within the infected wells of cell culture plates (<a href="#ppat.1013291.g001" class="usa-link">Fig 1c</a>). Since the high number of bacteria recovered at 24h <em>p.i.</em> is not solely due to intracellular replication, but rather to extracellular replication of those bacteria that escaped macrophage restriction, we termed this phenomenon <em>unrestricted growth</em> (<em>URG</em>).</p>
<figure class="fig xbox font-sm" id="ppat.1013291.g001"><h4 class="obj_head">Fig 1. The BCAA transporter BrnQ1 is required for <em>S. aureus</em> replication in human macrophages.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333996_ppat.1013291.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3538/12333996/454c5e31c1f1/ppat.1013291.g001.jpg" loading="lazy" height="1066" width="741" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/ppat.1013291.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(a)</strong> Experimental procedure for macrophage infection: primary human macrophages were infected with <em>S. aureus</em>, at a multiplicity of infection (MOI) of 5 for 30 min. Extracellular <em>S. aureus</em> were removed by a 15 min treatment with 30 μg/ml lysostaphin and two subsequent rinses with DPBS. Infected macrophages were further incubated in antibiotic-free media. “T0<em>”</em> corresponds to 45 min <em>p.i</em>. <strong>(</strong><strong>b)</strong> <em>S. aureus</em> mutants in genes involved in either BCAA-uptake (<em>brn</em><em>Q1-3</em> and <em>bca</em><em>P</em>) or biosynthesis (<em>ilv</em><em>E</em>, <em>ilv</em><em>D</em>, <em>leu</em><em>A</em>) were chosen for the analysis. Gene identifiers correspond to USA300_FPR3757(old locus ID) [<a href="#ppat.1013291.ref030" class="usa-link" aria-describedby="ppat.1013291.ref030">30</a>]. <strong>(</strong><strong>c)</strong> Phase-contrast micrographs of macrophages at 24h <em>p.i.,</em> which stayed uninfected, were infected with wild-type <em>S. aureus</em> strains JE2, Cowan I, and RN4220, or mutants in the indicated genes. All mutants were retrieved from the Nebraska Transposon Mutant Library (NTML) generated in the USA300 JE2 background (USA300 LAC-derivative). Unrestricted growth (<em>URG</em>) of the bacteria is observed by formation of bacterial micro-colonies (white arrows). Shown are representative images from two independent experiments. Scale bar = 100 µm. “NE” designates mutant strain identifier in the NTML. <strong>(</strong><strong>d)</strong> Intracellular replication in human macrophages. Data are shown as mean ±SD from independent experiments (JE2 WT and <em>brnQ1</em>: n = 7, <em>brnQ1</em> + p<em>brnQ1</em> and Cowan I: n = 3). <strong>(e)</strong> Macrophage infection with <em>brnQ1</em> mutants in the genomic backgrounds of <em>S. aureus</em> strains 6850, SH100 and RN4220. White arrows indicate bacterial microcolonies. Shown are representative images from three independent experiments (Scale bar: 100 µm). <strong>(</strong><strong>f</strong><strong>)</strong> Live cell imaging of macrophages infected with mRFP-expressing JE2 WT or JE2 <em>brnQ1</em> (magenta, white arrows). Host cell death (green fluorescence, green arrows) is detected by CellEvent Caspase-3/7 Green Detection Reagent. Shown are micrographs extracted from time-lapse series representative of 3 independent experiments (n = 3). Scale bar = 50 µm. (see also <a href="#ppat.1013291.s017" class="usa-link">S1 Movie</a>). <strong>(g)</strong> Transmission electron micrographs of intracellular JE2 WT or JE2 <em>brnQ1</em> confined to the phagosomal compartment (white arrows) within the macrophage, at 10h <em>p.i.</em> Shown are representative micrographs, from one experiment. Statistical analysis: (<strong>d</strong>) two-way ANOVA with Dunnett’s multiple comparisons test (each sample vs JE2 WT, at each time-point), using log10-transformed data; <em>**p &lt; 0.01; ***p &lt; 0.001; ****p &lt; 0.0001</em>. <em>(URG = unrestricted growth).</em>
<strong>(</strong><strong>a</strong><strong>)</strong> contains icons which were modified (colour and design) from <a href="https://bioicons.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bioicons.com/</a>, as follows: petri-dish-lid-yellow icon by Servier <a href="https://smart.servier.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://smart.servier.com/</a> is licensed under CC-BY 3.0 Unported <a href="https://creativecommons.org/licenses/by/3.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://creativecommons.org/licenses/by/3.0/</a>.</p></figcaption></figure><p>Remarkably, even <em>S. aureus</em> strains with reduced virulence, Cowan I and RN4220 [<a href="#ppat.1013291.ref036" class="usa-link" aria-describedby="ppat.1013291.ref036">36</a>], emerged from infected macrophages and exhibited <em>URG</em> (<a href="#ppat.1013291.g001" class="usa-link">Fig 1c</a>).</p>
<p>To further investigate the contribution of <em>brnQ1</em> to <em>S. aureus</em> growth within macrophages, we transduced the <em>brnQ1</em>::Tn transposon insertion (NE945) into a parental <em>S. aureus</em> JE2 strain, hereafter referred to as JE2 <em>brnQ1</em> and again assessed the ability of this strain to infect macrophages and display <em>URG</em>. These data revealed that, in contrast to the parental JE2 strain, JE2 <em>brnQ1</em> failed to replicate. Genetic complementation of <em>brnQ1 in trans</em> (JE2 <em>brnQ1</em> + p<em>brnQ1</em>) (<a href="#ppat.1013291.s001" class="usa-link">S1 Fig</a>) rescued the ability of JE2 <em>brnQ1</em> to egress from macrophages (<a href="#ppat.1013291.g001" class="usa-link">Fig 1d</a>). While at 6h <em>p.i.</em> bacterial loads are comparable among all strains under scrutiny (<a href="#ppat.1013291.s002" class="usa-link">S2 Fig</a>), at 12h <em>p.i.</em> we recovered approximately 2-fold more bacteria compared with “T0” (after lysostaphin pulse, i.e. at 45 min <em>p.i.</em>). Both parental JE2 and JE2 <em>brnQ1</em> + p<em>brnQ1</em> as well as <em>S. aureus</em> Cowan I reached <em>URG</em> by 24h <em>p.i.</em> In contrast, the number of recovered colony-forming units (CFU) for JE2 <em>brnQ1</em> dropped to ~4% of the initial infection inoculum by 24h <em>p.i.</em> (<a href="#ppat.1013291.g001" class="usa-link">Fig 1d</a>). Furthermore, absence of <em>brnQ1</em> abolished <em>URG</em> in several <em>S. aureus</em> strains: 6850, SH1000 and RN4220 (<a href="#ppat.1013291.g001" class="usa-link">Fig 1e</a>). Taken together, these data reveal that a functional BrnQ1 transporter is a general requirement for <em>S. aureus</em> growth inside macrophages, and not simply a JE2-specific phenotype.</p>
<p>To investigate whether the JE2 <em>brnQ1</em> growth defect within macrophages is specific to the intracellular environment, we next analysed bacterial growth in the medium used for macrophage infection assays. In an undefined medium such as TSB, which contains an abundance of amino acids and peptides, growth of JE2 <em>brnQ1</em> is indistinguishable from JE2 WT and the complemented strain JE2 <em>brnQ1 </em>+ p<em>brnQ1</em> (<a href="#ppat.1013291.s003" class="usa-link">S3a Fig</a>). In contrast, in a chemically defined medium (CDM) containing 1 mM BCAA, JE2 <em>brnQ1</em> displayed delayed growth when compared to controls (<a href="#ppat.1013291.s001" class="usa-link">S1 Fig</a>). In RPMI1640, where the concentrations of Val, Leu and Ile are ~ 117 µM (Val) and 131 µM (Leu and Ile, each), JE2 <em>brnQ1</em> also displays a growth delay in comparison to JE2 WT and JE2 <em>brnQ1 </em>+ p<em>brnQ1</em> (<a href="#ppat.1013291.s003" class="usa-link">S3b Fig</a>). This growth delay is still evident in <em>Infection Medium</em> (i.e., RPMI1640 + 10% v/v FBS), indicating that even in the presence of FBS the bacterium’s nutritional requirement for BCAAs is not fully met (<a href="#ppat.1013291.s003" class="usa-link">S3c Fig</a>). Importantly, in all culture media where JE2 <em>brnQ1</em> initially displays delayed growth, the mutant eventually achieves the same bacterial density as parental JE2 (<a href="#ppat.1013291.s003" class="usa-link">S3b</a> and <a href="#ppat.1013291.s003" class="usa-link">S3c Fig</a>). Taken together, these data show that <em>brnQ1</em>-deficient <em>S. aureus</em> displays delayed growth in media with low BCAA availability (≤ 1 mM), however, in macrophages, growth of the mutant is completely halted.</p>
<p>Replication of <em>S. aureus</em> within macrophages precedes cell death of infected phagocytes [<a href="#ppat.1013291.ref012" class="usa-link" aria-describedby="ppat.1013291.ref012">12</a>]. We therefore performed live cell imaging to monitor the intracellular replication of JE2 WT and JE2 <em>brnQ1</em> in the presence of a fluorogenic caspase 3/7 substrate (<a href="#ppat.1013291.g001" class="usa-link">Fig 1f</a> and <a href="#ppat.1013291.s017" class="usa-link">S1 Movie</a>). JE2 WT commenced replicating within macrophages at approximately 8h <em>p.i.</em>, which was followed by bacterial egress from the infected macrophage and replication in the extracellular milieu. Activation of effector caspases was evident as green fluorescence at ~ 11h <em>p.i.</em> Shortly thereafter, caspase activation was detectable in uninfected bystander cells. In contrast, JE2 <em>brnQ1</em> did not replicate, remained confined inside macrophages and failed to trigger caspase activation during the assayed time (i.e., up to 20h <em>p.i.</em>) (<a href="#ppat.1013291.s017" class="usa-link">S1 Movie</a>).</p>
<p>In non-professional phagocytes, such as epithelial cells, JE2 <em>brnQ1</em> exhibited similarly impaired intracellular replication, accompanied by reduced cytotoxicity against the host, despite major differences in infection dynamics in this cell type [i.e., bacterial phagosomal escape precedes cytosolic replication [<a href="#ppat.1013291.ref007" class="usa-link" aria-describedby="ppat.1013291.ref007">7</a>,<a href="#ppat.1013291.ref037" class="usa-link" aria-describedby="ppat.1013291.ref037">37</a>]] (<a href="#ppat.1013291.s004" class="usa-link">S4</a> and <a href="#ppat.1013291.s005" class="usa-link">S5 Figs</a> and <a href="#ppat.1013291.s018" class="usa-link">S2</a>–<a href="#ppat.1013291.s021" class="usa-link">S5 Movies</a> and <a href="#ppat.1013291.s024" class="usa-link">S1 Text</a>).</p>
<p>Notably, culture supernatants from <em>brnQ1</em> bacteria are as cytotoxic as JE2 WT supernatants, indicating that in principle, BrnQ1 is dispensable for secreted toxin production if bacteria grow in a rich medium (<a href="#ppat.1013291.s006" class="usa-link">S6a Fig</a>). Additionally, <em>brnQ1</em> mutants show wild-type haemolysis, on sheep blood agar, which supports the observation that <em>brnQ1</em> mutants can be virulent under the right growth conditions (<a href="#ppat.1013291.s006" class="usa-link">S6b</a> and <a href="#ppat.1013291.s006" class="usa-link">S6c Fig</a>).</p>
<p>Next, we analysed infected macrophages by transmission electron microscopy (TEM). We observed that, at 10h <em>p.i.</em>, both JE2 WT and JE2 <em>brnQ1</em> were localised to the lumen of intact phagosomal compartments (<a href="#ppat.1013291.g001" class="usa-link">Fig 1g</a>). Overall, these data demonstrate that wild-type <em>S. aureus</em> overcomes intra-macrophage confinement whereas the <em>brnQ1</em> mutant cannot. Moreover, under our experimental conditions, 10h <em>p.i.</em> is a timepoint at which both wild-type and the <em>brnQ1</em> mutant, which fails to replicate, are still confined within membrane-bound phagosomes.</p></section><section id="sec004"><h3 class="pmc_sec_title">Transcriptome analysis of intracellular <em>S. aureus</em> reveals dysregulation of iron metabolism in <em>brnQ1</em> mutants</h3>
<p>To identify the cause of the restricted intracellular growth of <em>brnQ1</em> mutants, we next performed dual RNA sequencing (RNAseq) of infected macrophages at 10h <em>p.i.</em> We hypothesized that this time point would be crucial for the bacterial adaptation to the phagolysosomal environment, since <em>S. aureus</em> JE2 WT and <em>brnQ1</em> mutant still reside within a membrane-bound vacuole but only the WT strain can initiate replication shortly thereafter. Consistent with our previous observations [<a href="#ppat.1013291.ref038" class="usa-link" aria-describedby="ppat.1013291.ref038">38</a>], we detected a similar host gene expression signature, irrespective of whether macrophages were infected with JE2 <em>brnQ1</em> or JE2 WT (<a href="#ppat.1013291.g002" class="usa-link">Figs 2a</a> and <a href="#ppat.1013291.s007" class="usa-link">S7a</a>), In contrast, we detected profound differences in gene expression between JE2 WT and JE2 <em>brnQ1</em>. Among bacterial transcripts, we found that 80 genes were significantly downregulated in the JE2 <em>brnQ1</em> mutant inside macrophages as compared to WT. Only 6 transcripts were upregulated in JE2 <em>brnQ1</em> (<a href="#ppat.1013291.g002" class="usa-link">Figs 2b</a> and <a href="#ppat.1013291.s007" class="usa-link">S7b</a> and <a href="#ppat.1013291.t001" class="usa-link">Table 1</a>). Most of the downregulated genes in intracellular JE2 <em>brnQ1</em>, belonged to operons that encode iron acquisition systems (<a href="#ppat.1013291.g002" class="usa-link">Figs 2c</a> and <a href="#ppat.1013291.s008" class="usa-link">S8</a>). These included several <em>isd</em> heme utilization genes (<em>isdA</em>, <em>isdB</em>, <em>isdCDEF</em> and <em>isdG</em>), <em>srtB</em>, staphyloferrin B siderophore biosynthesis genes (<em>sirAB</em> and <em>sbnABCDEFGH</em>) and the staphyloferrin A biosynthesis gene <em>sfaC</em> (<a href="#ppat.1013291.t001" class="usa-link">Table 1</a>). Several other genes, including chemotaxis-inhibiting protein (<em>chp</em>), thermonuclease (<em>nuc</em>), von Willebrand factor-binding protein (<em>vwb</em>), superoxide dismutase (<em>sodM</em>), Panton-Valentine leukocidin (PVL) subunit <em>lukS</em>, were downregulated. The cysteine protease staphopain A (<em>scpA</em>), as well as genes encoding superantigen-like proteins Ssl2, 4, 7, 11, 12, 13, 14, which are known virulence factors, were similarly downregulated as compared to wild-type.</p>
<figure class="fig xbox font-sm" id="ppat.1013291.g002"><h4 class="obj_head">Fig 2. Intracellular <em>S. aureus</em> JE2 <em>brnQ1</em> mutants do not respond to iron-starvation inside macrophages.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333996_ppat.1013291.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3538/12333996/047da8a09de7/ppat.1013291.g002.jpg" loading="lazy" height="873" width="760" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/ppat.1013291.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Dual RNAseq of primary human macrophages infected with either JE2 WT or JE2 <em>brnQ1</em>, at 10h <em>p.i.</em>: <strong>(</strong><strong>a)</strong> Human transcripts and <strong>(</strong><strong>b)</strong> Bacterial transcripts. Horizontal lines represent a p-value cutoff &lt; 0.05. Vertical lines represent cutoffs of log<sub>2</sub>fold changes of either &lt;-1 and &gt; 1 for human transcripts or &lt;-1.5 and &gt; 1.5 for bacterial transcripts. Transcripts of iron uptake systems, which are prominently downregulated in intracellular JE2 <em>brnQ1</em> vs JE2 WT, are depicted in green. Differential transcript abundance is expressed as JE2 <em>brnQ1</em> vs JE2 WT (DESeq2). Data are shown from 3 independent experiments (n = 3). <strong>(c)</strong> Schematic of the ferric uptake regulator Fur regulon of <em>S. aureus</em> (information retrieved from RegPrecise). Coloration of the boxes underneath the open reading frames indicates log<sub>2</sub>fold changes in expression of JE2 <em>brnQ1</em> vs JE2 WT according to the heatmap. Gene identifiers correspond to USA300_FPR3757 (old locus ID). <strong>(</strong><strong>d)</strong> RT-qPCR analysis of selected genes found to be downregulated or upregulated <strong>(</strong><strong>e)</strong> in intracellular JE2 <em>brnQ1</em>. Data are shown as mean -ΔCt values ±SD from independent experiments (n = 4, different donors). <strong>(</strong><strong>f)</strong> Venn diagram showing the overlap between the set of significantly downregulated genes in JE2 <em>brnQ1</em> shown in <strong>b</strong> and the Fur regulon retrieved from RegPrecise. Statistical analysis: (<strong>d</strong>, <strong>e</strong>) one-way ANOVA with Dunnett’s multiple comparisons test (vs JE2 WT for each gene); <em>ns = not significant; **p &lt; 0.01; ****p &lt; 0.0001</em>.</p></figcaption></figure><section class="tw xbox font-sm" id="ppat.1013291.t001"><h4 class="obj_head">Table 1. List of regulated genes for <em>S. aureus</em> JE2 in primary human macrophages at 10h <em>p.i.</em> (JE2 <em>brnQ1</em> compared to JE2 WT).</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead>
<tr><th align="left" colspan="6" rowspan="1">Downregulated Genes</th></tr>
<tr>
<th align="left" rowspan="1" colspan="1">Locus ID (old)</th>
<th align="left" rowspan="1" colspan="1">Locus ID (new)</th>
<th align="left" rowspan="1" colspan="1">Gene</th>
<th align="left" rowspan="1" colspan="1">Annotation</th>
<th align="left" rowspan="1" colspan="1">Log<sub>2</sub>FC</th>
<th align="left" rowspan="1" colspan="1">Adj. P value</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1034</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05565</td>
<td align="left" rowspan="1" colspan="1">
<em>srt</em>
<em>B</em>
</td>
<td align="left" rowspan="1" colspan="1">sortase B</td>
<td align="left" rowspan="1" colspan="1">-7.34</td>
<td align="left" rowspan="1" colspan="1">3.38 x 10<sup>–07</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1920</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS10530</td>
<td align="left" rowspan="1" colspan="1">
<em>chp</em>
</td>
<td align="left" rowspan="1" colspan="1">chemotaxis-inhibiting protein CHIPS</td>
<td align="left" rowspan="1" colspan="1">-6.10</td>
<td align="left" rowspan="1" colspan="1">4.67 x 10<sup>–45</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0407</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS02185</td>
<td align="left" rowspan="1" colspan="1">
<em>ssl</em>
<em>11</em>
</td>
<td align="left" rowspan="1" colspan="1">superantigen-like protein SSL11</td>
<td align="left" rowspan="1" colspan="1">-5.48</td>
<td align="left" rowspan="1" colspan="1">4.10 x 10<sup>–12</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0398</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS02130</td>
<td align="left" rowspan="1" colspan="1">
<em>ssl</em>
<em>4</em>
</td>
<td align="left" rowspan="1" colspan="1">superantigen-like protein SSL4</td>
<td align="left" rowspan="1" colspan="1">-5.05</td>
<td align="left" rowspan="1" colspan="1">2.38 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2291</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS12660</td>
<td align="left" rowspan="1" colspan="1">
<em>glt</em>
<em>S</em>
</td>
<td align="left" rowspan="1" colspan="1">sodium/glutamate symporter</td>
<td align="left" rowspan="1" colspan="1">-4.80</td>
<td align="left" rowspan="1" colspan="1">5.66 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1031</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05550</td>
<td align="left" rowspan="1" colspan="1">
<em>isd</em>
<em>D</em>
</td>
<td align="left" rowspan="1" colspan="1">hypothetical protein</td>
<td align="left" rowspan="1" colspan="1">-4.59</td>
<td align="left" rowspan="1" colspan="1">1.49 x 10<sup>–06</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0408</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS02190</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">FKLRK protein</td>
<td align="left" rowspan="1" colspan="1">-4.48</td>
<td align="left" rowspan="1" colspan="1">2.99 x 10<sup>–22</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0401</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS02150</td>
<td align="left" rowspan="1" colspan="1">
<em>ssl</em>
<em>7</em>
</td>
<td align="left" rowspan="1" colspan="1">superantigen-like protein SSL7</td>
<td align="left" rowspan="1" colspan="1">-4.37</td>
<td align="left" rowspan="1" colspan="1">2.99 x 10<sup>–22</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0120</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS00620</td>
<td align="left" rowspan="1" colspan="1">
<em>sbn</em>
<em>C</em>
</td>
<td align="left" rowspan="1" colspan="1">staphyloferrin B biosynthesis protein SbnC</td>
<td align="left" rowspan="1" colspan="1">-4.36</td>
<td align="left" rowspan="1" colspan="1">1.61 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0774</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS04175</td>
<td align="left" rowspan="1" colspan="1">
<em>emp</em>
</td>
<td align="left" rowspan="1" colspan="1">extracellular matrix protein-binding adhesin Emp</td>
<td align="left" rowspan="1" colspan="1">-4.33</td>
<td align="left" rowspan="1" colspan="1">2.99 x 10<sup>–22</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1022</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05500</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">hypothetical protein</td>
<td align="left" rowspan="1" colspan="1">-4.29</td>
<td align="left" rowspan="1" colspan="1">4.43 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1028</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05535</td>
<td align="left" rowspan="1" colspan="1">
<em>isd</em>
<em>B</em>
</td>
<td align="left" rowspan="1" colspan="1">heme uptake protein IsdB</td>
<td align="left" rowspan="1" colspan="1">-4.28</td>
<td align="left" rowspan="1" colspan="1">3.56 x 10<sup>–20</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0940</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05050</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">DoxX family protein</td>
<td align="left" rowspan="1" colspan="1">-4.20</td>
<td align="left" rowspan="1" colspan="1">6.60 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1032</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05555</td>
<td align="left" rowspan="1" colspan="1">
<em>isdE</em>
</td>
<td align="left" rowspan="1" colspan="1">heme ABC transporter substrate-binding protein IsdE</td>
<td align="left" rowspan="1" colspan="1">-4.16</td>
<td align="left" rowspan="1" colspan="1">4.45 x 10<sup>–05</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1033</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05560</td>
<td align="left" rowspan="1" colspan="1">
<em>isdF</em>
</td>
<td align="left" rowspan="1" colspan="1">iron ABC transporter permease</td>
<td align="left" rowspan="1" colspan="1">-4.15</td>
<td align="left" rowspan="1" colspan="1">2.27 x 10<sup>–06</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0775</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS04180</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">hypothetical protein</td>
<td align="left" rowspan="1" colspan="1">-4.12</td>
<td align="left" rowspan="1" colspan="1">3.56 x 10<sup>–20</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1929</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS10585</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">phage tail protein</td>
<td align="left" rowspan="1" colspan="1">-3.93</td>
<td align="left" rowspan="1" colspan="1">4.86 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0776</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS04185</td>
<td align="left" rowspan="1" colspan="1">
<em>nuc</em>
</td>
<td align="left" rowspan="1" colspan="1">thermonuclease family protein</td>
<td align="left" rowspan="1" colspan="1">-3.86</td>
<td align="left" rowspan="1" colspan="1">1.85 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1030</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05545</td>
<td align="left" rowspan="1" colspan="1">
<em>isd</em>
<em>C</em>
</td>
<td align="left" rowspan="1" colspan="1">heme uptake protein IsdC</td>
<td align="left" rowspan="1" colspan="1">-3.85</td>
<td align="left" rowspan="1" colspan="1">3.09 x 10<sup>–06</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0815</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS04395</td>
<td align="left" rowspan="1" colspan="1">
<em>ear</em>
</td>
<td align="left" rowspan="1" colspan="1">DUF4888 domain-containing protein</td>
<td align="left" rowspan="1" colspan="1">-3.48</td>
<td align="left" rowspan="1" colspan="1">1.03 x 10<sup>–15</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1035</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05570</td>
<td align="left" rowspan="1" colspan="1">
<em>isd</em>
<em>G</em>
</td>
<td align="left" rowspan="1" colspan="1">staphylobilin-forming heme oxygenase IsdG</td>
<td align="left" rowspan="1" colspan="1">-3.45</td>
<td align="left" rowspan="1" colspan="1">1.04 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0215</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS01130</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">isoprenylcysteine carboxyl methyltransferase family protein</td>
<td align="left" rowspan="1" colspan="1">-3.36</td>
<td align="left" rowspan="1" colspan="1">2.36 x 10<sup>–12</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1029</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05540</td>
<td align="left" rowspan="1" colspan="1">
<em>isd</em>
<em>A</em>
</td>
<td align="left" rowspan="1" colspan="1">LPXTG-anchored heme-scavenging protein IsdA</td>
<td align="left" rowspan="1" colspan="1">-3.30</td>
<td align="left" rowspan="1" colspan="1">6.07 x 10<sup>–26</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0773</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS04170</td>
<td align="left" rowspan="1" colspan="1">
<em>vwb</em>
</td>
<td align="left" rowspan="1" colspan="1">von Willebrand factor binding protein Vwb</td>
<td align="left" rowspan="1" colspan="1">-3.29</td>
<td align="left" rowspan="1" colspan="1">3.62 x 10<sup>–16</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1060</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05745</td>
<td align="left" rowspan="1" colspan="1">
<em>ssl</em>
<em>13</em>
</td>
<td align="left" rowspan="1" colspan="1">superantigen-like protein SSL13</td>
<td align="left" rowspan="1" colspan="1">-3.23</td>
<td align="left" rowspan="1" colspan="1">1.84 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS09430</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">hypothetical protein</td>
<td align="left" rowspan="1" colspan="1">-3.01</td>
<td align="left" rowspan="1" colspan="1">4.62 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2528</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS14025</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">epoxyqueuosine reductase QueH</td>
<td align="left" rowspan="1" colspan="1">-2.99</td>
<td align="left" rowspan="1" colspan="1">1.44 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0124</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS00640</td>
<td align="left" rowspan="1" colspan="1">
<em>sbn</em>
<em>G</em>
</td>
<td align="left" rowspan="1" colspan="1">staphyloferrin B biosynthesis citrate synthase SbnG</td>
<td align="left" rowspan="1" colspan="1">-2.98</td>
<td align="left" rowspan="1" colspan="1">5.10 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1061</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05750</td>
<td align="left" rowspan="1" colspan="1">
<em>ssl</em>
<em>14</em>
</td>
<td align="left" rowspan="1" colspan="1">superantigen-like protein SSL14</td>
<td align="left" rowspan="1" colspan="1">-2.97</td>
<td align="left" rowspan="1" colspan="1">1.44 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1059</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05740</td>
<td align="left" rowspan="1" colspan="1">
<em>ssl</em>
<em>12</em>
</td>
<td align="left" rowspan="1" colspan="1">superantigen-like protein SSL12</td>
<td align="left" rowspan="1" colspan="1">-2.96</td>
<td align="left" rowspan="1" colspan="1">1.82 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0237</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS01265</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">nucleoside hydrolase</td>
<td align="left" rowspan="1" colspan="1">-2.93</td>
<td align="left" rowspan="1" colspan="1">4.86 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0119</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS00615</td>
<td align="left" rowspan="1" colspan="1">
<em>sbn</em>
<em>B</em>
</td>
<td align="left" rowspan="1" colspan="1">2,3-diaminopropionate biosynthesis protein SbnB</td>
<td align="left" rowspan="1" colspan="1">-2.65</td>
<td align="left" rowspan="1" colspan="1">4.24 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0116</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS00600</td>
<td align="left" rowspan="1" colspan="1">
<em>sir</em>
<em>B</em>
</td>
<td align="left" rowspan="1" colspan="1">staphyloferrin B ABC transporter permease subunit SirB</td>
<td align="left" rowspan="1" colspan="1">-2.62</td>
<td align="left" rowspan="1" colspan="1">2.94 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0986</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05305</td>
<td align="left" rowspan="1" colspan="1">
<em>cyd</em>
<em>A</em>
</td>
<td align="left" rowspan="1" colspan="1">cytochrome ubiquinol oxidase subunit I</td>
<td align="left" rowspan="1" colspan="1">-2.55</td>
<td align="left" rowspan="1" colspan="1">1.14 x 10<sup>–05</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0122</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS00630</td>
<td align="left" rowspan="1" colspan="1">
<em>sbn</em>
<em>E</em>
</td>
<td align="left" rowspan="1" colspan="1">lucA/IucC family siderophore biosynthesis protein</td>
<td align="left" rowspan="1" colspan="1">-2.55</td>
<td align="left" rowspan="1" colspan="1">1.35 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2453</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS13605</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">ATP-binding cassette domain-containing protein</td>
<td align="left" rowspan="1" colspan="1">-2.49</td>
<td align="left" rowspan="1" colspan="1">3.75 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1786</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS09780</td>
<td align="left" rowspan="1" colspan="1">
<em>ecs</em>
<em>A</em>
</td>
<td align="left" rowspan="1" colspan="1">ABC transporter ATP-binding protein</td>
<td align="left" rowspan="1" colspan="1">-2.48</td>
<td align="left" rowspan="1" colspan="1">1.37 x 10<sup>–05</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0814</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS04390</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">Abi family protein</td>
<td align="left" rowspan="1" colspan="1">-2.46</td>
<td align="left" rowspan="1" colspan="1">3.80 x 10<sup>–09</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0117</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS00605</td>
<td align="left" rowspan="1" colspan="1">
<em>sir</em>
<em>A</em>
</td>
<td align="left" rowspan="1" colspan="1">staphyloferrin B ABC transporter substrate-binding protein SirA</td>
<td align="left" rowspan="1" colspan="1">-2.44</td>
<td align="left" rowspan="1" colspan="1">2.38 x 10<sup>–09</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0123</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS00635</td>
<td align="left" rowspan="1" colspan="1">
<em>sbn</em>
<em>F</em>
</td>
<td align="left" rowspan="1" colspan="1">IucA/IucC family siderophore biosynthesis protein</td>
<td align="left" rowspan="1" colspan="1">-2.42</td>
<td align="left" rowspan="1" colspan="1">9.01 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0307</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS01635</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">5’-nucleotidase, lipoprotein e(P4) family</td>
<td align="left" rowspan="1" colspan="1">-2.41</td>
<td align="left" rowspan="1" colspan="1">4.79 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0902</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS04855</td>
<td align="left" rowspan="1" colspan="1">
<em>pep</em>
<em>F</em>
</td>
<td align="left" rowspan="1" colspan="1">oligoendopeptidase F</td>
<td align="left" rowspan="1" colspan="1">-2.38</td>
<td align="left" rowspan="1" colspan="1">2.11 x 10<sup>–09</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0121</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS00625</td>
<td align="left" rowspan="1" colspan="1">
<em>sbn</em>
<em>D</em>
</td>
<td align="left" rowspan="1" colspan="1">staphyloferrin B export MFS transporter</td>
<td align="left" rowspan="1" colspan="1">-2.33</td>
<td align="left" rowspan="1" colspan="1">3.05 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2409</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS13340</td>
<td align="left" rowspan="1" colspan="1">
<em>cnz</em>
<em>C</em>
</td>
<td align="left" rowspan="1" colspan="1">nickel ABC transporter permease</td>
<td align="left" rowspan="1" colspan="1">-2.33</td>
<td align="left" rowspan="1" colspan="1">1.32 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1171</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS06335</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">insulinase family protein</td>
<td align="left" rowspan="1" colspan="1">-2.30</td>
<td align="left" rowspan="1" colspan="1">2.78 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0825</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS04450</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">nitronate monooxygenase</td>
<td align="left" rowspan="1" colspan="1">-2.26</td>
<td align="left" rowspan="1" colspan="1">1.19 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1675</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS09145</td>
<td align="left" rowspan="1" colspan="1">
<em>tyr</em>
<em>S</em>
</td>
<td align="left" rowspan="1" colspan="1">tyrosine-tRNA ligase</td>
<td align="left" rowspan="1" colspan="1">-2.25</td>
<td align="left" rowspan="1" colspan="1">4.74 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0118</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS00610</td>
<td align="left" rowspan="1" colspan="1">
<em>sbn</em>
<em>A</em>
</td>
<td align="left" rowspan="1" colspan="1">siderophore biosynthesis protein SbnA</td>
<td align="left" rowspan="1" colspan="1">-2.20</td>
<td align="left" rowspan="1" colspan="1">1.29 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2137</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS11770</td>
<td align="left" rowspan="1" colspan="1">
<em>sfa</em>
<em>C</em>
</td>
<td align="left" rowspan="1" colspan="1">staphyloferrin A biosynthesis protein SfaC</td>
<td align="left" rowspan="1" colspan="1">-2.19</td>
<td align="left" rowspan="1" colspan="1">6.69 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2262</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS12500</td>
<td align="left" rowspan="1" colspan="1">
<em>spd</em>
<em>B</em>
</td>
<td align="left" rowspan="1" colspan="1">CPBP family intramembrane metalloprotease SdpB</td>
<td align="left" rowspan="1" colspan="1">-2.17</td>
<td align="left" rowspan="1" colspan="1">1.92 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0125</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS00645</td>
<td align="left" rowspan="1" colspan="1">
<em>sbn</em>
<em>H</em>
</td>
<td align="left" rowspan="1" colspan="1">staphyloferrin B biosynthesis decarboxylase SbnH</td>
<td align="left" rowspan="1" colspan="1">-2.16</td>
<td align="left" rowspan="1" colspan="1">3.36 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1053</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05680</td>
<td align="left" rowspan="1" colspan="1">
<em>flr</em>
</td>
<td align="left" rowspan="1" colspan="1">formyl peptide receptor-like 1 inhibitory protein</td>
<td align="left" rowspan="1" colspan="1">-2.14</td>
<td align="left" rowspan="1" colspan="1">7.19 x 10<sup>–06</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0135</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS00705</td>
<td align="left" rowspan="1" colspan="1">
<em>sod</em>
<em>M</em>
</td>
<td align="left" rowspan="1" colspan="1">superoxide dismutase</td>
<td align="left" rowspan="1" colspan="1">-2.14</td>
<td align="left" rowspan="1" colspan="1">2.62 x 10<sup>–08</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1957</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS10735</td>
<td align="left" rowspan="1" colspan="1">
<em>dna</em>
<em>D2</em>
</td>
<td align="left" rowspan="1" colspan="1">DnaD domain-containing protein</td>
<td align="left" rowspan="1" colspan="1">-2.14</td>
<td align="left" rowspan="1" colspan="1">3.05 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1890</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS10340</td>
<td align="left" rowspan="1" colspan="1">
<em>scp</em>
<em>A</em>
</td>
<td align="left" rowspan="1" colspan="1">cysteine protease staphopain A</td>
<td align="left" rowspan="1" colspan="1">-2.13</td>
<td align="left" rowspan="1" colspan="1">5.77 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2251</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS12430</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">octopine dehydrogenase</td>
<td align="left" rowspan="1" colspan="1">-2.13</td>
<td align="left" rowspan="1" colspan="1">9.70 x 10<sup>–05</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1852</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS10120</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">ABC transporter ATP-binding protein</td>
<td align="left" rowspan="1" colspan="1">-2.13</td>
<td align="left" rowspan="1" colspan="1">3.36 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1964</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS10775</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">DUF771 domain-containing protein</td>
<td align="left" rowspan="1" colspan="1">-2.10</td>
<td align="left" rowspan="1" colspan="1">2.45 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1587</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS08650</td>
<td align="left" rowspan="1" colspan="1">
<em>his</em>
<em>S</em>
</td>
<td align="left" rowspan="1" colspan="1">histidine-tRNA ligase</td>
<td align="left" rowspan="1" colspan="1">-2.04</td>
<td align="left" rowspan="1" colspan="1">2.03 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1797</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS09835</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">helix-turn-helix transcriptional regulator</td>
<td align="left" rowspan="1" colspan="1">-1.99</td>
<td align="left" rowspan="1" colspan="1">5.26 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0396</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS02115</td>
<td align="left" rowspan="1" colspan="1">
<em>ssl</em>
<em>2</em>
</td>
<td align="left" rowspan="1" colspan="1">superantigen-like protein SSL2</td>
<td align="left" rowspan="1" colspan="1">-1.99</td>
<td align="left" rowspan="1" colspan="1">1.19 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0254</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS01355</td>
<td align="left" rowspan="1" colspan="1">
<em>lyt</em>
<em>S</em>
</td>
<td align="left" rowspan="1" colspan="1">sensor histidine kinase</td>
<td align="left" rowspan="1" colspan="1">-1.98</td>
<td align="left" rowspan="1" colspan="1">1.46 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0987</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05310</td>
<td align="left" rowspan="1" colspan="1">
<em>cyd</em>
<em>B</em>
</td>
<td align="left" rowspan="1" colspan="1">cytochrome D ubiquinol oxidase subunit II</td>
<td align="left" rowspan="1" colspan="1">-1.85</td>
<td align="left" rowspan="1" colspan="1">1.67 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2537</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS14075</td>
<td align="left" rowspan="1" colspan="1">
<em>ldh</em>2</td>
<td align="left" rowspan="1" colspan="1">L-lactate dehydrogenase</td>
<td align="left" rowspan="1" colspan="1">-1.82</td>
<td align="left" rowspan="1" colspan="1">2.29 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0432</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS02315</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">sodium-dependent transporter</td>
<td align="left" rowspan="1" colspan="1">-1.81</td>
<td align="left" rowspan="1" colspan="1">6.37 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2210</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS12185</td>
<td align="left" rowspan="1" colspan="1">
<em>glc</em>
<em>U</em>
</td>
<td align="left" rowspan="1" colspan="1">glucose uptake protein GlcU</td>
<td align="left" rowspan="1" colspan="1">-1.79</td>
<td align="left" rowspan="1" colspan="1">1.06 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1382</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS07545</td>
<td align="left" rowspan="1" colspan="1">
<em>luk</em>
<em>S</em>
</td>
<td align="left" rowspan="1" colspan="1">Panton-Valentine bi-component leukocidin subunit LukS-PV</td>
<td align="left" rowspan="1" colspan="1">-1.78</td>
<td align="left" rowspan="1" colspan="1">3.12 x 10<sup>–06</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1882</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS10300</td>
<td align="left" rowspan="1" colspan="1">
<em>gat</em>
<em>C</em>
</td>
<td align="left" rowspan="1" colspan="1">Asp-tRNA(Asn)/Glu-tRNA(Gln) amidotransferase subunit GatC</td>
<td align="left" rowspan="1" colspan="1">-1.75</td>
<td align="left" rowspan="1" colspan="1">2.00 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1381</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS07540</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">Panton-Valentine bi-component leukocidin subunit F</td>
<td align="left" rowspan="1" colspan="1">-1.75</td>
<td align="left" rowspan="1" colspan="1">1.48 x 10<sup>–05</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1471</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS08030</td>
<td align="left" rowspan="1" colspan="1">
<em>xse</em>
<em>B</em>
</td>
<td align="left" rowspan="1" colspan="1">exodeoxyribonuclease VII small subunit</td>
<td align="left" rowspan="1" colspan="1">-1.74</td>
<td align="left" rowspan="1" colspan="1">3.90 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS10500</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">MAP domain-containing protein</td>
<td align="left" rowspan="1" colspan="1">-1.74</td>
<td align="left" rowspan="1" colspan="1">1.04 x 10<sup>–09</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2526</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS14015</td>
<td align="left" rowspan="1" colspan="1">
<em>pyr</em>
<em>D</em>
</td>
<td align="left" rowspan="1" colspan="1">quinone-dependent dihydroorotate dehydrogenase</td>
<td align="left" rowspan="1" colspan="1">-1.73</td>
<td align="left" rowspan="1" colspan="1">2.21 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2250</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS12425</td>
<td align="left" rowspan="1" colspan="1">
<em>nha</em>
<em>C</em>
</td>
<td align="left" rowspan="1" colspan="1">Na + /H+ antiporter NhaC</td>
<td align="left" rowspan="1" colspan="1">-1.69</td>
<td align="left" rowspan="1" colspan="1">3.16 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2328</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS12865</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">DUF4889 domain-containing protein</td>
<td align="left" rowspan="1" colspan="1">-1.67</td>
<td align="left" rowspan="1" colspan="1">2.90 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0107</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS00555</td>
<td align="left" rowspan="1" colspan="1">
<em>npt</em>
<em>A</em>
</td>
<td align="left" rowspan="1" colspan="1">Na/Pi cotransporter family protein</td>
<td align="left" rowspan="1" colspan="1">-1.65</td>
<td align="left" rowspan="1" colspan="1">1.61 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1153</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS06245</td>
<td align="left" rowspan="1" colspan="1">
<em>upp</em>
<em>S</em>
</td>
<td align="left" rowspan="1" colspan="1">isoprenyl transferase</td>
<td align="left" rowspan="1" colspan="1">-1.64</td>
<td align="left" rowspan="1" colspan="1">4.06 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_1170</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS06330</td>
<td align="left" rowspan="1" colspan="1">
<em>–</em>
</td>
<td align="left" rowspan="1" colspan="1">GntR family transcriptional regulator</td>
<td align="left" rowspan="1" colspan="1">-1.63</td>
<td align="left" rowspan="1" colspan="1">2.40 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0025</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS00130</td>
<td align="left" rowspan="1" colspan="1">
<em>ads</em>
<em>A</em>
</td>
<td align="left" rowspan="1" colspan="1">LPXTG-anchored adenosine synthase AdsA</td>
<td align="left" rowspan="1" colspan="1">-1.62</td>
<td align="left" rowspan="1" colspan="1">5.72 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0620</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS03325</td>
<td align="left" rowspan="1" colspan="1">
<em>mnt</em>
<em>A</em>
</td>
<td align="left" rowspan="1" colspan="1">metal ABC transporter ATP-binding protein</td>
<td align="left" rowspan="1" colspan="1">-1.57</td>
<td align="left" rowspan="1" colspan="1">5.35 x 10<sup>–13</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0669</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS03590</td>
<td align="left" rowspan="1" colspan="1">
<em>upp</em>
<em>P</em>
</td>
<td align="left" rowspan="1" colspan="1">undecaprenyl-diphosphate phosphatase</td>
<td align="left" rowspan="1" colspan="1">-1.56</td>
<td align="left" rowspan="1" colspan="1">3.92 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" colspan="5" rowspan="1">
<strong>
<em>Upregulated Genes</em>
</strong>
</td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong><span class="text-underline">Locus ID</span> (<span class="text-underline">old</span>)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>Locus <span class="text-underline">ID</span> (new)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>Gene</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>Annotation</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>Log</strong>
<sub>
<strong>2</strong>
</sub>
<strong>
<span class="text-underline">FC</span>
</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>Adj. P value</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2417</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS13380</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">AbgT family transporter</td>
<td align="left" rowspan="1" colspan="1">1.58</td>
<td align="left" rowspan="1" colspan="1">1.83 x 10<sup>–03</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0955</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS05135</td>
<td align="left" rowspan="1" colspan="1">
<em>atl</em>
</td>
<td align="left" rowspan="1" colspan="1">bifunctional autolysin</td>
<td align="left" rowspan="1" colspan="1">1.60</td>
<td align="left" rowspan="1" colspan="1">2.24 x 10<sup>–04</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0891</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS04805</td>
<td align="left" rowspan="1" colspan="1">
<em>opp</em>-<em>3A</em>
</td>
<td align="left" rowspan="1" colspan="1">peptide ABC transporter substrate-binding protein</td>
<td align="left" rowspan="1" colspan="1">1.66</td>
<td align="left" rowspan="1" colspan="1">4.22 x 10<sup>–07</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_0890</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS04800</td>
<td align="left" rowspan="1" colspan="1">
<em>opp</em>-<em>3F</em>
</td>
<td align="left" rowspan="1" colspan="1">ATP-binding cassette domain-containing protein</td>
<td align="left" rowspan="1" colspan="1">1.73</td>
<td align="left" rowspan="1" colspan="1">6.41 x 10<sup>–05</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2585</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS14375</td>
<td align="left" rowspan="1" colspan="1">
<em>asp</em>
<em>3</em>
</td>
<td align="left" rowspan="1" colspan="1">accessory Sec system protein Asp3</td>
<td align="left" rowspan="1" colspan="1">1.91</td>
<td align="left" rowspan="1" colspan="1">2.45 x 10<sup>–02</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SAUSA300_2607</td>
<td align="left" rowspan="1" colspan="1">SAUSA300_RS14495</td>
<td align="left" rowspan="1" colspan="1">
<em>his</em>
<em>A</em>
</td>
<td align="left" rowspan="1" colspan="1">1-(5-phosphoribosyl)-5-((5-phosphoribosylamino)methylideneamino)imidazole-4-carboxamide isomerase</td>
<td align="left" rowspan="1" colspan="1">1.94</td>
<td align="left" rowspan="1" colspan="1">1.44 x 10<sup>–02</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ppat.1013291.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>We further validated by RT-qPCR the marked downregulation of heme- and siderophore-mediated iron acquisition genes in intracellular JE2 <em>brnQ1</em> as compared to intracellular WT JE2 (<a href="#ppat.1013291.g002" class="usa-link">Fig 2d</a>). It is interesting to note that a few genes showed modest upregulation in the <em>brnQ1</em> mutant (<a href="#ppat.1013291.t001" class="usa-link">Table 1</a>). Again, using RT-qPCR we confirmed that, in the <em>brnQ1</em> mutant, two loci in the <em>opp</em>3 oligopeptide transporter operon [<a href="#ppat.1013291.ref026" class="usa-link" aria-describedby="ppat.1013291.ref026">26</a>] and a gene which encodes a putative transporter for <em>p</em>-aminobenzoyl-glutamate (PABA) (SAUSA300_RS13380/ SAUSA300_2417), a precursor of folic acid [<a href="#ppat.1013291.ref028" class="usa-link" aria-describedby="ppat.1013291.ref028">28</a>], were indeed upregulated in intracellular JE2 <em>brnQ1</em> (<a href="#ppat.1013291.g002" class="usa-link">Fig 2e</a>). Importantly, all changes in gene expression observed in JE2 <em>brnQ1</em> relative to JE2 WT were restored to wild-type levels when JE2 <em>brnQ1</em> was complemented with <em>brnQ1</em> on a plasmid (<a href="#ppat.1013291.g002" class="usa-link">Fig 2d</a> and <a href="#ppat.1013291.g002" class="usa-link">2e</a>).</p>
<p>Taken together, these data indicate that wild-type <em>S. aureus</em> can adapt to the intracellular milieu of the macrophage as evidenced, for instance, by upregulation of many genes including Fe-regulated loci, while the <em>brnQ1</em> mutant displays a global defect in gene expression.</p>
<p>The <em>S. aureus</em>-containing phagosome has been shown to be iron depleted [<a href="#ppat.1013291.ref039" class="usa-link" aria-describedby="ppat.1013291.ref039">39</a>]. By comparing transcriptomes of intracellular bacteria at 10h <em>p.i.</em> to bacteria grown for 10h in <em>Infection Medium,</em> we reaffirm the observation that <em>S. aureus</em> is indeed iron-starved in this niche, as indicated by upregulation of iron-regulated loci by intracellular JE2 WT (<a href="#ppat.1013291.s009" class="usa-link">S9 Fig</a>).</p>
<p>In <em>S. aureus,</em> transcription of genes encoding iron uptake proteins is controlled by the ferric uptake regulator Fur [<a href="#ppat.1013291.ref040" class="usa-link" aria-describedby="ppat.1013291.ref040">40</a>]. Fur is a transcription factor that uses Fe<sup>2+</sup> as corepressor. Under conditions of iron starvation, when Fur is not bound to Fe<sup>2+</sup>, Fur is inactive resulting in de-repression of target genes [<a href="#ppat.1013291.ref041" class="usa-link" aria-describedby="ppat.1013291.ref041">41</a>]. The observation that many of the most significantly downregulated genes in the <em>brnQ1</em> mutant function in iron acquisition, prompted us to consult a list of Fur-regulated targets retrieved from the RegPrecise database [<a href="#ppat.1013291.ref042" class="usa-link" aria-describedby="ppat.1013291.ref042">42</a>]. This analysis revealed that 18 of the 80 (22.5%) genes that were significantly downregulated in JE2 <em>brnQ1</em> indeed belonged to the Fur regulon (<a href="#ppat.1013291.g002" class="usa-link">Fig 2f</a>). To exclude that inadvertent mutations in <em>fur</em> were responsible for our observations, we sequenced the <em>fur</em> locus from JE2 WT and JE2 <em>brnQ1</em> and found that the gene was intact in both strains.</p></section><section id="sec005"><h3 class="pmc_sec_title">Intracellular <em>S. aureus brnQ1</em> mutants are metabolically quiescent at 10h <em>p.i.</em> and fail to respond to environmental cues</h3>
<p>Since the <em>brnQ1</em> mutant carries an intact <em>fur</em> gene, yet fails to regulate iron acquisition loci, we speculated that the <em>brnQ1</em> mutant may lack the ability to respond appropriately to environmental cues within the phagosome. Ostensibly this defect is exclusively due to BCAA starvation. Therefore, we posited that, even upon inactivation of the <em>fur</em> gene, Fur-regulated genes would still not be de-repressed in the intracellular environment.</p>
<p>To test this, we generated <em>fur</em> and <em>brnQ1/fur</em> mutants in the JE2 background. To validate our strains, we first analysed <em>isdB</em> expression, as a representative iron-regulated gene within the Fur regulon, in <em>Infection Medium</em> (iron replete; contains BCAAs). Under this condition, the inactivation of <em>fur</em> resulted in increased expression of <em>isdB</em>, that was comparable between both <em>fur</em> mutant backgrounds (<a href="#ppat.1013291.g003" class="usa-link">Fig 3a</a>). Moreover, when we cultivated both JE2 <em>fur</em> and JE2 <em>brn</em>Q1/<em>fur</em> in iron-rich medium (RPMI1640 supplemented with 20 µM FeSO<sub>4</sub>), total intracellular iron (Fe<sup>2+</sup> and Fe<sup>3+</sup>) was ~ 3-fold higher for both <em>fur</em> backgrounds as compared to either JE2 WT or JE2 <em>brnQ1</em> (<a href="#ppat.1013291.s010" class="usa-link">S10 Fig</a>). This confirms that inactivation of <em>fur</em>, in both wild-type and mutant backgrounds, results in constitutive de-repression of iron-regulated operons in planktonic bacteria.</p>
<figure class="fig xbox font-sm" id="ppat.1013291.g003"><h4 class="obj_head">Fig 3. Loss of BrnQ1 function renders intracellular <em>S. aureus</em> unresponsive to environmental cues and metabolically quiescent, but a supply of exogenous BCAAs restores responsiveness and promotes intracellular replication.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333996_ppat.1013291.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3538/12333996/7ec70fb70017/ppat.1013291.g003.jpg" loading="lazy" height="1017" width="733" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/ppat.1013291.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(a)</strong> RT-qPCR analysis of <em>isdB</em> expression for bacteria grown in <em>Infection Medium</em> for 1 hour or <strong>(<strong>b</strong>)</strong> in intracellular bacteria within infected primary human macrophages, at 10h <em>p.i. isdB</em> was selected as representative iron-regulated gene to report on Fur regulon status. Data are shown as mean -ΔCt values ±SD from independent experiments: (a) n = 3; (b) n = 4. <strong>(c)</strong> Schematic representation of the set-up for experiments using aTc induction of intracellular JE2 with a plasmid allowing anhydrous tetracycline (aTc)-inducible expression of the cyan-fluorescent protein <em>Cerulean</em>. After 1h <em>p.i.</em> or 10h <em>p.i.</em>, infected cells were treated with either vehicle control (-) or 300 ng/mL aTc (+) for 2h to allow <em>Cerulean</em> expression. <strong>(</strong><strong>d)</strong> Representative confocal images of infected primary human macrophages at 1h <em>p.i.</em> and <strong>(f)</strong> 10h <em>p.i.</em> upon induction (Magenta: <em>Cerulean</em>; Green: Vancomycin BODIPY-FL/<em>S. aureus</em>; Gray: Phalloidin/Actin; Blue: Hoechst/DNA. Scale bar: overviews: 5 µm; zoomed inset: 1µm). <strong>(e)</strong> Quantification of intracellular <em>Cerulean</em>-positive bacteria, as percent of total bacteria at 1h <em>p.i.</em> and <strong>(g)</strong> 10h <em>p.i.</em> in samples exposed to aTc (+) or vehicle control (-) for 2h. Data are shown as mean ±SD from independent experiments (n = 3). <strong>(h)</strong> Intracellular replication of JE2 <em>brnQ1</em> or <strong>(i)</strong> JE2 <em>brnQ1/fur</em> in primary human macrophages, without or with BCAA supplementation. Excess BCAAs (1mM each) were supplemented 24h prior to and during the infection. Data are shown as mean ±SD from independent experiments (n = 3). Graphed <em>URG</em> value represents an imputed value (median of all quantified <em>URG</em> experiments which are shown in <a href="#ppat.1013291.g001" class="usa-link">Fig 1d</a>). <strong>(</strong><strong>j)</strong> RT-qPCR analysis of <em>isdB</em> expression in intracellular bacteria in infected primary human macrophages, at 10h <em>p.i.</em> Excess BCAAs (1mM each) were supplemented 24h prior to and during the infection. Data are shown as mean -ΔCt values ±SD from independent experiments: n = 4; except JE2 <em>brn</em>Q1/<em>fur </em>+ BCAA: n = 3. Statistical analysis: (<strong>a, b</strong>) one-way ANOVA with Tukey’s multiple comparisons test; (<strong>e, g, h, i, j</strong>) one-way ANOVA with Sidak’s multiple comparisons test (h and i: for the 10h <em>p.i.</em> time point only); <em>ns = not significant; *p &lt; 0.05; **p &lt; 0.01; ***p &lt; 0.001; (URG = unrestricted growth</em>). (<strong>c</strong>) contains icons which were modified (colour and design) from <a href="https://bioicons.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bioicons.com/</a>, as follows: petri-dish-lid-yellow icon by Servier <a href="https://smart.servier.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://smart.servier.com/</a> is licensed under CC-BY 3.0 Unported <a href="https://creativecommons.org/licenses/by/3.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://creativecommons.org/licenses/by/3.0/</a>. The confocal-scanning-laser-microscope-CSLM icon by DBCLS <a href="https://togotv.dbcls.jp/en/pics.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://togotv.dbcls.jp/en/pics.html</a> is licensed under CC-BY 4.0 Unported <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://creativecommons.org/licenses/by/4.0/</a>.</p></figcaption></figure><p>Using these strains, we analysed transcription of <em>isdB</em> inside macrophages and observed that JE2 WT and JE2 <em>fur</em> show comparable <em>isdB</em> expression. This result was expected as under the iron-starved conditions encountered within the phagosome, <em>isdB</em> should be fully de-repressed in JE2 WT similarly to JE2 <em>fur</em>, where iron-regulated genes are constitutively expressed. Intracellular JE2 <em>brnQ1</em>/<em>fur</em> showed <em>isdB</em> expression levels similar to JE2 <em>brnQ1</em> alone despite lacking the Fur repressor (<a href="#ppat.1013291.g003" class="usa-link">Fig 3b</a>). Furthermore, <em>isdB</em> expression in both <em>brnQ1</em> and <em>brnQ1</em>/<em>fur</em> mutants was significantly decreased relative to JE2 WT or JE2 <em>fur,</em> again indicating that <em>brnQ1</em> deficiency is enough to prevent appropriate induction of gene expression (<a href="#ppat.1013291.g003" class="usa-link">Fig 3b</a>).</p>
<p>The inability of JE2 <em>brnQ1</em> to remodel its gene expression in the intracellular milieu hinted that the bacteria may enter a state of intracellular <em>quiescence.</em> To test this, we assessed whether JE2 <em>brnQ1</em> bacteria were capable of <em>de novo</em> protein synthesis within macrophages. We measured <em>de novo</em> synthesis of the cyan-fluorescent protein <em>Cerulean</em> upon intracellular induction with anhydrotetracycline (aTc) as previously described [<a href="#ppat.1013291.ref043" class="usa-link" aria-describedby="ppat.1013291.ref043">43</a>]. In a control experiment, both JE2 WT and JE2 <em>brnQ1</em> responded to aTc induction in TSB medium, showing that, in principle, both strains can produce <em>Cerulean</em> (<a href="#ppat.1013291.s011" class="usa-link">S11 Fig</a>). To test whether <em>Cerulean</em> production can be induced by intracellular bacteria, aTc was added to infected macrophages at 1h or 10h <em>p.i.</em> for 2h (<a href="#ppat.1013291.g003" class="usa-link">Fig 3c</a>). Induction with aTc at 1h <em>p.i.</em> resulted in <em>Cerulean</em> fluorescence for both JE2 WT and JE2 <em>brnQ1</em> (<a href="#ppat.1013291.g003" class="usa-link">Fig 3d</a>). At this time, approximately 15% of all internalized bacteria, as detected by BODIPY FL Vancomycin staining, also showed <em>Cerulean</em> fluorescence, while in vehicle-treated cells the bacteria remained <em>Cerulean</em> negative (<a href="#ppat.1013291.g003" class="usa-link">Figs 3e</a> and <a href="#ppat.1013291.s012" class="usa-link">S12</a>). In contrast, when <em>Cerulean</em> production was induced at 10h <em>p.i.</em>, ~ 10% of the JE2 WT bacteria became <em>Cerulean-</em>positive, whereas only very rarely did JE2 <em>brnQ1</em> become <em>Cerulean</em>-positive and was comparable to vehicle treated controls (<a href="#ppat.1013291.g003" class="usa-link">Figs 3f</a>, <a href="#ppat.1013291.g003" class="usa-link">3g</a> and <a href="#ppat.1013291.s012" class="usa-link">S12</a>).</p>
<p>The inactivation of <em>brnQ1</em> alone, the dominant BCAA transporter, was sufficient to halt intracellular bacterial growth. This indicated that the <em>S. aureus</em>-containing phagosome must be scarce in BCAAs. To test this directly, we supplemented the <em>Infection Medium</em> with 1 mM of each BCAA and observed that the defect in intracellular replication was rescued, both in the case of JE2 <em>brnQ1</em> (<a href="#ppat.1013291.g003" class="usa-link">Fig 3h</a>) and JE2 <em>brnQ1</em>/<em>fur</em> (<a href="#ppat.1013291.g003" class="usa-link">Fig 3i</a>), which now overcame macrophage restriction and grew unrestrictedly. Moreover, addition of excess BCAAs rendered intracellular JE2 <em>brnQ1</em> and JE2 <em>brnQ1</em>/<em>fur</em> responsive to the iron-deplete state of the phagosome, as shown by significantly higher <em>isdB</em> expression levels 10h <em>p.i.</em> (<a href="#ppat.1013291.g003" class="usa-link">Fig 3j</a>).</p>
<p>During growth in CDM lacking all three BCAAs, the growth of <em>S. aureus</em> is preceded by an extended lag phase of ~20h. However, omission of Val and Leu only, surprisingly completely halted bacterial growth (<a href="#ppat.1013291.s013" class="usa-link">S13a Fig</a>). We have therefore hypothesized that provision of these 2 amino acids alone would be sufficient to rescue the growth deficit of <em>S. aureus brnQ1</em> in macrophages. Indeed, supplementation with 1 mM Val and Leu, but not the single amino acids, restored growth of the <em>brnQ1</em> mutant in RAW 264.7 macrophages (<a href="#ppat.1013291.s013" class="usa-link">S13b</a> and <a href="#ppat.1013291.s013" class="usa-link">S13c Fig</a>).</p>
<p>Taken together, we posit that by 10h <em>p.i. S. aureus brnQ1</em> mutants have assumed a state of <em>quiescenc</em>e where the bacteria fail to adapt to the intracellular niche as evidenced by altered gene expression and impaired <em>de novo</em> protein synthesis. Importantly, this <em>quiescence</em> is caused by BCAA starvation and can be overcome if Val and Leu become available, indicating that BCAA acquisition underpins <em>S. aureus</em> growth and survival inside macrophages.</p></section><section id="sec006"><h3 class="pmc_sec_title">Inactivation of the CodY repressor rescues growth of <em>brnQ1</em>-deficient <em>S. aureus</em> in an IlvD and BcaP dependent manner</h3>
<p>The <em>brnQ1</em> mutant fails to grow in BCAA-depleted phagosomes, although <em>S. aureus</em> carries genes that encode BCAA biosynthesis enzymes (e.g., the <em>ilv</em> operon) [<a href="#ppat.1013291.ref021" class="usa-link" aria-describedby="ppat.1013291.ref021">21</a>], as well as a second BCAA transporter (<em>bcaP</em>) [<a href="#ppat.1013291.ref029" class="usa-link" aria-describedby="ppat.1013291.ref029">29</a>], which are expressed upon inactivation of the repressor protein CodY [<a href="#ppat.1013291.ref021" class="usa-link" aria-describedby="ppat.1013291.ref021">21</a>, <a href="#ppat.1013291.ref026" class="usa-link" aria-describedby="ppat.1013291.ref026">26</a>, <a href="#ppat.1013291.ref029" class="usa-link" aria-describedby="ppat.1013291.ref029">29</a>, <a href="#ppat.1013291.ref044" class="usa-link" aria-describedby="ppat.1013291.ref044">44</a>] and could potentially ensure BCAA supply.</p>
<p>To understand the role that CodY plays in the acquisition of BCAAs during infection, we created a <em>brnQ1/codY</em> double mutant and measured bacterial replication in macrophages. In contrast to the <em>brnQ1</em> single mutant, the <em>brnQ1/codY</em> double mutant behaved similarly to the WT strain and exhibited <em>URG</em> both in primary human (<a href="#ppat.1013291.g004" class="usa-link">Fig 4a</a>) and murine RAW 264.7 macrophages (<a href="#ppat.1013291.g004" class="usa-link">Fig 4b</a>). Restored intracellular growth of <em>brnQ1</em>-deficient <em>S. aureus</em> upon <em>codY</em> inactivation, indicated that the bacteria must now be able to meet their BCAA requirement.</p>
<figure class="fig xbox font-sm" id="ppat.1013291.g004"><h4 class="obj_head">Fig 4. Intracellular growth deficiency of <em>S. aureus brnQ1</em> mutants can be rescued by inactivation of the CodY repressor or BcaP overexpression.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333996_ppat.1013291.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3538/12333996/7dcb13af1bc9/ppat.1013291.g004.jpg" loading="lazy" height="703" width="733" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/ppat.1013291.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Intracellular growth of a <em>brnQ1</em> mutant in the <em>S. aureus</em> USA300 background (∆<em>brnQ1</em>) and a <em>brnQ1</em>/<em>codY</em> double mutant (∆<em>brnQ1/codY</em>::Tn) in <strong>(<strong>a</strong>)</strong> primary human macrophages and <strong>(<strong>b</strong>)</strong> murine RAW 264.7 macrophages. <strong>(c)</strong> Intracellular growth in RAW264.7 macrophages of a triple <em>S. aureus</em> USA300 <em>brnQ1</em>/<em>codY</em>/<em>ilvD</em> mutant (∆<em>brnQ1</em>/<em>codY</em>::Tn/<em>ilvD</em>::Tn-Kan<sup>R</sup>). <strong>(d)</strong> Growth in infected RAW 264.7 macrophages of S<em>. aureus brnQ1</em> carrying either a vector control (pRMC2), plasmid-encoded BCAA transporter BcaP (<em>ΔbrnQ1 </em>+ p<em>bcaP</em>) or plasmid encoded <em>brnQ1</em> (<em>ΔbrnQ1 </em>+ p<em>brnQ1</em>). <strong>(</strong><strong>e)</strong> Growth in infected RAW 264.7 macrophages of <em>S. aureus brnQ1,</em> a double <em>brnQ1/bcaP</em> mutant (<em>ΔbrnQ1/ ΔbcaP)</em>, a triple <em>br</em>n<em>Q1, bcaP, codY</em> mutant (<em>ΔbrnQ1/ΔbcaP/codY::</em>Tn) and a quadruple mutant carrying the additional <em>ilvD</em> mutation (<em>ΔbrnQ1/ΔbcaP/codY::</em>Tn<em>/ilvD::</em>Tn-Kan<sup>R</sup>). USA300 WT serves as control. All data are shown as mean log10 value ± S.D. for the calculated fold change in CFU/mL at 20h relative to 1.5h <em>p.i.</em>, for each bacterial strain. Each data point plotted represents a biological replicate derived from at least three independent experiments (n ≥ 3). Statistical analysis: (<strong>a, b, c, e</strong>) one-way ANOVA with Tukey’s multiple comparison test; (<strong>d</strong>) Brown-Forsythe and Welch ANOVA with Dunnet’s T3 multiple comparisons test. <em>ns = not significant; *p &lt; 0.05, ***p &lt; 0.001, ****p &lt; 0.0001</em>.</p></figcaption></figure><p>Conceivably, this could be accomplished through endogenous BCAA synthesis. To test this, we created a <em>brnQ1/codY/ilvD</em> triple mutant and assessed its ability to grow in CDM without Val. Growth experiments revealed that the <em>brnQ1</em>/<em>codY</em> mutant, as expected, could grow in the absence of exogenous Val yet the <em>brnQ1/codY/ilvD</em> mutant could not (<a href="#ppat.1013291.s014" class="usa-link">S14 Fig</a>). This confirms that, as opposed to the triple mutant, the <em>brnQ1/codY</em> mutant must be able to synthesize Val to allow growth in Val-deplete media.</p>
<p>Next, we infected RAW 264.7 macrophages with the mutants. As expected, <em>brnQ1</em>-deficient <em>S. aureus</em> failed to grow out of macrophages whereas WT and <em>brnQ1/codY S. aureus</em> overcame macrophage restriction and displayed <em>URG</em>. Remarkably, the <em>brnQ1/codY/ ilvD</em> mutant also grew out of macrophages indicating that, despite the inability to synthesize BCAAs, the bacteria still access these amino acids inside the macrophage (<a href="#ppat.1013291.g004" class="usa-link">Fig 4c</a>).</p>
<p>The observation that the intracellular growth of <em>S. aureus brnQ1</em> can be restored by exogenous addition of Val and Leu during infection, indicated that the bacteria must retain the ability for BCAA uptake. BcaP is an obvious candidate as it has previously been shown that it transports all three BCAAs and that it can partially compensate for loss of <em>brnQ1</em> [<a href="#ppat.1013291.ref029" class="usa-link" aria-describedby="ppat.1013291.ref029">29</a>]. Therefore, we next infected macrophages with a <em>brnQ1</em> mutant strain that overexpresses <em>bcaP</em> or a respective vector control and we observed that <em>S. aureus brnQ1</em> overexpressing <em>bcaP</em> could grow unrestrictedly without BCAA supplementation (<a href="#ppat.1013291.g004" class="usa-link">Fig 4d</a>). These data suggested that increased <em>bcaP</em> expression, due to <em>codY</em> inactivation [<a href="#ppat.1013291.ref029" class="usa-link" aria-describedby="ppat.1013291.ref029">29</a>], could explain why the <em>brnQ1/codY</em>/<em>ilvD</em> triple mutant maintained the ability to overcome macrophage restriction. Hence, we utilized a <em>brnQ1</em>/<em>bcaP</em> double mutant, again inactivated either <em>codY</em> alone, or <em>codY</em> and <em>ilvD</em> in this background and infected RAW 264.7 macrophages. Similar to a <em>brnQ1</em> single mutant, the <em>brnQ1</em>/<em>bcaP</em> double mutant was unable to replicate (<a href="#ppat.1013291.g004" class="usa-link">Fig 4e</a>). Inactivation of <em>codY</em> in the <em>brnQ1/bcaP</em> background restored growth in macrophages however, an additional <em>ilvD</em> mutation significantly impaired growth (<a href="#ppat.1013291.g004" class="usa-link">Fig 4e</a>). Taken together, these data indicate that limited phagosomal BCAA levels can in principle be overcome in <em>S. aureus</em> by increased expression of CodY-regulated BCAA synthesis genes and/or BCAA transporters.</p></section><section id="sec007"><h3 class="pmc_sec_title">
<em>S. aureus brnQ1</em> mutants survive for inordinate duration within human macrophages</h3>
<p>Since <em>brnQ1</em>-deficient <em>S. aureus</em> failed to replicate in macrophages during the first 24h of infection, we next assessed long-term intracellular survival in primary human macrophages. Often, studies on long-term intracellular <em>S. aureus</em> infections rely on sustained antibiotic treatment of the infected host cells to limit the effects caused by extracellular bacteria that may egress from host cells during the experiment [<a href="#ppat.1013291.ref013" class="usa-link" aria-describedby="ppat.1013291.ref013">13</a>,<a href="#ppat.1013291.ref045" class="usa-link" aria-describedby="ppat.1013291.ref045">45</a>–<a href="#ppat.1013291.ref049" class="usa-link" aria-describedby="ppat.1013291.ref049">49</a>]. Here we did not maintain antibiotic treatment on infected macrophages, and this allowed us to assess whether <em>URG</em> would occur for <em>S. aureus brnQ1.</em> Bacteria were quantified by CFU counting at 1, 7, 14, 21 and 28 days <em>p.i.</em> unless we observed <em>URG</em> (<a href="#ppat.1013291.g005" class="usa-link">Fig 5a</a>). Within the first 24h of infection, the number of JE2 <em>brnQ1</em> bacteria decreased to ~3.5% of the initially phagocytosed bacterial population (i.e., lysostaphin-protected bacteria at “T0”). After this initial decline, the abundance of JE2 <em>brnQ1</em> remained steady whereas JE2 WT grew unrestrictedly (<em>URG</em>) by day 1 <em>p.i</em>. (<a href="#ppat.1013291.g005" class="usa-link">Fig 5b</a>). Remarkably, viable JE2 <em>brnQ1</em> were recovered from infected macrophages at the endpoint of the assay after 28 days <em>p.i.</em> Across 14 independent experiments, we observed that typically JE2 <em>brnQ1</em> survived for 28 days within the macrophages. However, on occasion, <em>URG</em> was observed at day 2 or 15 <em>p.i</em>. albeit with less frequency (<a href="#ppat.1013291.g005" class="usa-link">Fig 5b</a>). Consistent with the CFU recovery data, only few bacteria were detected microscopically inside macrophages infected with JE2 <em>brnQ1</em> from 24h <em>p.i.</em> onwards and ultrastructural analysis revealed <em>brnQ1</em> mutants to be localised inside intact membrane-enclosed vacuoles (<a href="#ppat.1013291.g005" class="usa-link">Fig 5c</a>). We hypothesized that the variability in occurrence of <em>URG</em> might arise in a stochastic manner, resulting, for example, from loss of macrophage bactericidal capacity, death of individual host cells with subsequent release of bacteria, or heterogeneity within the bacterial subpopulations. To address this, we set up experiments where macrophages derived from the same donor were seeded in multiple technical replicates (e.g., up to 80 wells/experiment), infected with JE2 <em>brnQ1</em> and observed daily for 28 days to monitor <em>URG</em>. We observed variability within technical replicates, with <em>URG</em> occurring at various time points during the infection (<a href="#ppat.1013291.s015" class="usa-link">S15a Fig</a>). Furthermore, <em>brnQ1</em> mutants in different <em>S. aureus</em> backgrounds similarly survived for up to 28 days without eliciting visible cytotoxic effects against host cells (<a href="#ppat.1013291.s015" class="usa-link">S15b</a>, <a href="#ppat.1013291.s015" class="usa-link">S15c</a> and <a href="#ppat.1013291.s015" class="usa-link">S15d Fig</a>). These data collectively indicate that the duration of infection prior to <em>URG</em> is stochastic and donor-independent.</p>
<figure class="fig xbox font-sm" id="ppat.1013291.g005"><h4 class="obj_head">Fig 5. <em>S. aureus brnQ1</em> mutants survive for at least 28 days inside macrophages but are distinct of <em>small colony variants</em> (SCV).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12333996_ppat.1013291.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3538/12333996/94526f51bb8e/ppat.1013291.g005.jpg" loading="lazy" height="963" width="733" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/ppat.1013291.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(a)</strong> JE2 <em>brnQ1</em> intracellular replication in primary human macrophages, assayed over a period of 28 days. Data are shown as mean values ±SD from independent experiments (T<sub>0</sub>: n = 7; T<sub>0.5</sub>: n = 5; T<sub>d1</sub>: n = 6; T<sub>d7</sub> and T<sub>d21</sub>: n = 2; T<sub>d14</sub> and T<sub>d28</sub>: n = 3). <strong>(</strong><strong>b)</strong> Maximum time until unrestricted growth (<em>URG</em>) in macrophages infected with JE2 <em>brnQ1</em> across 14 biological replicates (n = 14), using 13 different blood donors (arrows: same donor, but used in an independent experiment). JE2 WT serves as control (<em>URG</em> at 24h <em>p.i.</em>). The experiment was terminated 28 days <em>p.i.</em> regardless of <em>URG</em> occurrence. <strong>(c)</strong> Transmission electron microscopy of JE2 <em>brnQ1</em>-infected macrophages at 24h <em>p.i.,</em> 48h <em>p.i.,</em> 7 and 28 days <em>p.i.</em> (white arrows). Shown are representative micrographs from one experiment. <strong>(</strong><strong>d)</strong> <em>URG</em> recovery experiment, whereby during a 28-day infection in macrophages, JE2 <em>brnQ1</em> clones (i.e., micro-colonies formed upon overgrowth from infected macrophages) were recovered upon <em>URG</em> (days 21, 23 and 27 <em>p.i.</em> respectively). Recovered JE2 <em>brnQ1</em> clones were re-inoculated in <em>Infection Medium</em> and growth was analysed. OD<sub>600</sub> was measured every 18 min, for 20h. <strong>(e)</strong> Recovered JE2 <em>brnQ1</em> were used for infection experiments in primary human macrophages, for 48h. Graphed <em>URG</em> value for JE2 WT control, represents an imputed value (median of all quantified <em>URG</em> experiments which are shown in <a href="#ppat.1013291.g001" class="usa-link">Fig 1d</a>). <strong>(f)</strong> Circular genome alignment of two JE2 <em>brnQ1</em> clones recovered from macrophages upon <em>URG</em><em> </em>(<em>URG 4</em> and <em>URG5</em> shown in <strong>d</strong> and <strong>e</strong> using the USA300_FPR3757 complete genome and JE2 WT for comparison (parental strain, laboratory stock). <strong>(g)</strong> JE2 <em>brnQ1</em> mutants that did not show <em>URG</em> were recovered from viable macrophages at 28 days <em>p.i.</em> and colony size, haemolysis on sheep-blood agar, and colony pigmentation were assessed. Colony diameter measurements of JE2 <em>brnQ1</em> which were recovered from macrophages, were compared with bacteria grown to late exponential phase in TSB. Data points correspond to single colonies, recovered from independent experiments (n = 2). Statistical analysis: (<strong>g</strong>) two-tailed Mann-Whitney test; <em>ns</em> = not significant.</p></figcaption></figure><p><em>S. aureus</em> can grow in the absence of Leu or Val <em>in vitro</em>, albeit after an extended lag phase (<a href="#ppat.1013291.s013" class="usa-link">S13a Fig</a>). Moreover, growth in the absence of Val was shown to select for heritable loss-of-function mutations in <em>codY</em> or the 5’ untranslated region (5’UTR) of <em>ilvD,</em> to allow <em>de novo</em> BCAA synthesis and subsequent bacterial replication [<a href="#ppat.1013291.ref021" class="usa-link" aria-describedby="ppat.1013291.ref021">21</a>]. Spontaneous selection of such suppressor mutants in intracellular JE2 <em>brnQ1</em> is a possible scenario that could explain the differences in <em>URG</em> observed across experiments and/or technical replicates. Furthermore, our data have revealed that inactivation of CodY is sufficient to rescue the growth of <em>brnQ1 S. aureus</em> in macrophages and in CDM lacking Val (<a href="#ppat.1013291.g004" class="usa-link">Figs 4</a> and <a href="#ppat.1013291.s014" class="usa-link">S14a</a>).</p>
<p>To determine whether <em>brnQ1</em> mutants that initiated <em>URG</em> had acquired genetic changes, we recovered five JE2 <em>brnQ1</em> clones upon egress from infected macrophages, sub-cultured the clones in <em>Infection Medium</em> and infected again macrophages. Each recovered <em>brnQ1</em> isolate again displayed delayed growth in <em>Infection Medium</em> (<a href="#ppat.1013291.g005" class="usa-link">Fig 5d</a>) and impaired growth upon macrophage infection (<a href="#ppat.1013291.g005" class="usa-link">Fig 5e</a>) suggesting that the egress events were not the result of heritable mutations. Consistently, whole genome sequencing of two such <em>brnQ1</em> isolates revealed no mutations in either <em>codY</em> or the 5’UTR of <em>ilvD</em> (<a href="#ppat.1013291.g005" class="usa-link">Fig 5f</a>).</p></section><section id="sec008"><h3 class="pmc_sec_title">Intracellular <em>S. aureus brnQ1</em> mutants are not classical <em>persisters</em>
</h3>
<p>Within a bacterial subpopulation, transient growth arrest and metabolic quiescence, accompanied by antibiotic tolerance, are hallmarks of <em>persistence</em> [<a href="#ppat.1013291.ref050" class="usa-link" aria-describedby="ppat.1013291.ref050">50</a>–<a href="#ppat.1013291.ref053" class="usa-link" aria-describedby="ppat.1013291.ref053">53</a>]. Intracellular persisters of <em>S. aureus</em> can arise upon antibiotic pressure [<a href="#ppat.1013291.ref043" class="usa-link" aria-describedby="ppat.1013291.ref043">43</a>,<a href="#ppat.1013291.ref054" class="usa-link" aria-describedby="ppat.1013291.ref054">54</a>,<a href="#ppat.1013291.ref055" class="usa-link" aria-describedby="ppat.1013291.ref055">55</a>]. Several mechanisms triggering intracellular <em>S. aureus</em> persisters were proposed, such as a transient activation of the stringent response (SR) [<a href="#ppat.1013291.ref043" class="usa-link" aria-describedby="ppat.1013291.ref043">43</a>]. In <em>S. aureus</em>, SR can be induced by amino-acid starvation (especially Leu/Val deprivation [<a href="#ppat.1013291.ref027" class="usa-link" aria-describedby="ppat.1013291.ref027">27</a>]) and results in rapid synthesis of the (p)ppGpp alarmone leading to profound transcriptional reprogramming, especially the repression of proliferation-related genes and activation of the stress-response and starvation-related regulons and a halt of bacterial division [<a href="#ppat.1013291.ref027" class="usa-link" aria-describedby="ppat.1013291.ref027">27</a>]. However, upon consulting our transcriptomic data (S1 Dataset), we did not observe changes in the expression of translation factors, ribosomal proteins or tRNAs in JE2 <em>brnQ1</em> compared to wild-type, at 10h <em>p.i</em>. Similarly, the expression of <em>relA, relP</em> and <em>relQ,</em> encoding the bifunctional Rsh (p)ppGpp synthase/hydrolase (<em>relA</em>) and the RelP and RelQ synthases [<a href="#ppat.1013291.ref056" class="usa-link" aria-describedby="ppat.1013291.ref056">56</a>] remained unaltered (<a href="#ppat.1013291.s008" class="usa-link">S8 Fig</a>). A study found that persistence in intracellular <em>S. aureus</em> in the presence of antibiotics involved multiple protective mechanisms: transcriptional reprogramming of the cell wall stress stimulon, SOS response to preserve bacterial genome integrity or the heat-shock stimulon [<a href="#ppat.1013291.ref043" class="usa-link" aria-describedby="ppat.1013291.ref043">43</a>]. In case of intracellular JE2 <em>brnQ1</em>, however, we found none of these pathways to be differentially regulated (<a href="#ppat.1013291.s022" class="usa-link">S1 Dataset</a> and <a href="#ppat.1013291.s008" class="usa-link">S8 Fig</a>).</p>
<p>A particular type of persistence described for <em>S. aureus</em> is the so-called SCV phenotype [<a href="#ppat.1013291.ref045" class="usa-link" aria-describedby="ppat.1013291.ref045">45</a>,<a href="#ppat.1013291.ref057" class="usa-link" aria-describedby="ppat.1013291.ref057">57</a>–<a href="#ppat.1013291.ref059" class="usa-link" aria-describedby="ppat.1013291.ref059">59</a>]. SCVs exhibit slow growth and metabolic alterations [<a href="#ppat.1013291.ref060" class="usa-link" aria-describedby="ppat.1013291.ref060">60</a>] and are often the result of adaption to an intracellular lifestyle [<a href="#ppat.1013291.ref059" class="usa-link" aria-describedby="ppat.1013291.ref059">59</a>]. SCVs form slow-growing colonies on agar plates and lack haemolysis and pigmentation [<a href="#ppat.1013291.ref060" class="usa-link" aria-describedby="ppat.1013291.ref060">60</a>] however, JE2 <em>brnQ1</em> recovered from macrophages after 28 days of infection exhibited normal colony size as compared to the control (i.e. <em>brnQ1</em> bacteria grown in TSB), displayed haemolysis, and were pigmented (<a href="#ppat.1013291.g005" class="usa-link">Fig 5g</a>) indicating JE2 <em>brn</em><em>Q1</em> recovered from macrophages are not SCVs.</p>
<p>In summary, growth of <em>S. aureus</em> within macrophages requires high affinity transport of BCAAs. BCAA deprivation, for instance mediated through <em>brnQ1</em> inactivation, results in <em>S. aureus</em> entering a state of <em>quiescence</em> that is distinct from SCV phenotypes. In the BCAA-starved state, <em>S. aureus brnQ1</em> stay viable within intact macrophage vacuoles for inordinate duration, even in the absence of antimicrobials.</p></section></section><section id="sec009"><h2 class="pmc_sec_title">Discussion</h2>
<p>At the forefront of host-pathogen interaction is a battle for nutrients (reviewed in [<a href="#ppat.1013291.ref010" class="usa-link" aria-describedby="ppat.1013291.ref010">10</a>,<a href="#ppat.1013291.ref061" class="usa-link" aria-describedby="ppat.1013291.ref061">61</a>,<a href="#ppat.1013291.ref062" class="usa-link" aria-describedby="ppat.1013291.ref062">62</a>]). The host deploys a plethora of effectors to deprive invading bacteria of essential nutrients whereas the ability of bacteria to overcome nutrient restriction underpins their successful colonization and infection. In this study we demonstrate that the availability of BCAAs and their transport by <em>S. aureus</em> are pivotal to the ability of the pathogen to grow inside macrophages.</p>
<p>Here, we found that inactivation of BrnQ1, the main BCAA transporter of <em>S. aureus,</em> is sufficient to arrest intracellular growth within the macrophage phagolysosome. Depriving <em>S. aureus</em> of BCAAs through a <em>brnQ1</em> mutation dramatically alters their physiology within the phagosome, resulting in major defects in gene expression and <em>de novo</em> protein synthesis. Consequently, <em>S. aureus brnQ1</em> mutants do not initiate appropriate patterns of gene expression and thus fail to adapt to the intracellular environment. This is emphasized by the inability of <em>brnQ1</em> bacteria to upregulate the expression of Fur-regulated genes which starkly contrasts wild-type bacteria that appropriately respond to iron limitation after phagocytosis by macrophages (<a href="#ppat.1013291.g002" class="usa-link">Fig 2</a>). Our data suggest that <em>S. aureus</em>, when deprived of Val and Leu, enter a state of <em>quiescence</em> within macrophages but can be “re-activated” once BCAAs become available.</p>
<p>The <em>S. aureus</em> genome encodes a complete BCAA biosynthesis machinery; however, the bacterium prioritises BCAA scavenging over <em>de novo</em> synthesis ostensibly because BCAA synthesis is energetically costly [<a href="#ppat.1013291.ref021" class="usa-link" aria-describedby="ppat.1013291.ref021">21</a>,<a href="#ppat.1013291.ref028" class="usa-link" aria-describedby="ppat.1013291.ref028">28</a>]. <em>S. aureus</em> possesses three functional BCAA transporters: BrnQ1 and BcaP, which import all three BCAAs, and BrnQ2, a designated Ile transporter [<a href="#ppat.1013291.ref024" class="usa-link" aria-describedby="ppat.1013291.ref024">24</a>,<a href="#ppat.1013291.ref029" class="usa-link" aria-describedby="ppat.1013291.ref029">29</a>]. A fourth gene encoding BrnQ3 bears homology to a BCAA transporter but lacks BCAA transport function [<a href="#ppat.1013291.ref024" class="usa-link" aria-describedby="ppat.1013291.ref024">24</a>]. The genes encoding the <em>S. aureus</em> BCAA transporters belong to the CodY regulon [<a href="#ppat.1013291.ref025" class="usa-link" aria-describedby="ppat.1013291.ref025">25</a>] and, as such, their transcription is de-repressed upon CodY inactivation [<a href="#ppat.1013291.ref027" class="usa-link" aria-describedby="ppat.1013291.ref027">27</a>,<a href="#ppat.1013291.ref028" class="usa-link" aria-describedby="ppat.1013291.ref028">28</a>]. Interestingly, there is a hierarchy of de-repression that is affected through CodY: modest decrease in intracellular BCAAs causes de-repression of <em>brnQ1</em>, whereas BCAA biosynthesis, which is also CodY regulated, remains repressed at the same level of CodY inactivation [<a href="#ppat.1013291.ref028" class="usa-link" aria-describedby="ppat.1013291.ref028">28</a>,<a href="#ppat.1013291.ref063" class="usa-link" aria-describedby="ppat.1013291.ref063">63</a>]. This is supported by our data showing that inactivation of <em>brnQ1</em> renders <em>S. aureus</em> unable to grow inside macrophages despite possessing the capability to synthesize the BCAAs required for growth. In this scenario, <em>brnQ1</em> must be expressed to some level by wild-type bacteria while the BCAA biosynthesis genes encoded by the <em>ilv</em> locus are likely not expressed.</p>
<p>In <em>S. aureus,</em> BrnQ1 is the preferred transporter for BCAA uptake [<a href="#ppat.1013291.ref029" class="usa-link" aria-describedby="ppat.1013291.ref029">29</a>]. For instance, when grown in CDM lacking Val or Leu (&lt;34 µM), BrnQ1-mediated BCAA uptake is required for growth [<a href="#ppat.1013291.ref024" class="usa-link" aria-describedby="ppat.1013291.ref024">24</a>]. Although BcaP can also import all three BCAAs and exhibits similar substrate affinity and transport velocities, BrnQ1 remains the predominant transporter likely due to its higher level of expression as compared to BcaP [<a href="#ppat.1013291.ref029" class="usa-link" aria-describedby="ppat.1013291.ref029">29</a>]. Our experiments support this, as the provision of <em>bcaP</em> on a multicopy plasmid is sufficient to rescue the growth of <em>brnQ1</em> bacteria inside macrophages. This indicates that BCAAs are still available to some extent inside the phagosome lumen, but clearly <em>brnQ1</em>-deficient bacteria cannot acquire sufficient Val and Leu to support growth. This suggests that <em>bcaP</em> is either poorly or not at all expressed in <em>S. aureus brnQ1</em> mutants<em>.</em> However, our transcriptome analysis did not show differential <em>bcaP</em> expression between JE2 <em>brnQ1</em> mutant and JE2 WT (<a href="#ppat.1013291.t001" class="usa-link">Table 1</a>).</p>
<p>Since <em>S. aureus brn</em><em>Q1</em> mutants fail to replicate inside macrophages, but do not exhibit growth defects in complex media (TSB or CDM with tryptone), the intraphagosomal milieu must be BCAA-deplete although the exact concentrations are unknown. SLC38A9, a human lysosomal membrane transporter, has been shown to remove Leu from lysosomes and it may contribute to BCAA limitation [<a href="#ppat.1013291.ref064" class="usa-link" aria-describedby="ppat.1013291.ref064">64</a>]. Val must also be limited in the phagosome. This notion is supported by the fact that simultaneous Leu and Val supplementation is required to rescue the <em>brnQ1</em> mutant in macrophages (<a href="#ppat.1013291.s013" class="usa-link">S13 Fig</a>). It is unlikely that the phagosome is depleted of Ile, because the absence of this BCAA would lead to de-repression of CodY targets and subsequent <em>de novo</em> BCAA biosynthesis [<a href="#ppat.1013291.ref021" class="usa-link" aria-describedby="ppat.1013291.ref021">21</a>], followed by bacterial growth.</p>
<p>Our data show that inactivation of <em>codY</em>, and therefore de-repression of the CodY regulon, is sufficient to allow the growth of <em>brnQ1</em>-deficient <em>S. aureus</em> within macrophages. More specifically, our data reveal that CodY inactivation enables growth of <em>S. aureus</em>
<em>brnQ1</em> mutants in an IlvD- and BcaP-dependent manner, indicating that BCAA biosynthesis and transport are enhanced in the <em>codY</em> background. This raises an important question: why wouldn’t the genes that allow growth simply be de-repressed in <em>brnQ1</em>-deficient <em>S. aureus</em>? One possible explanation is the complexity of CodY regulation, which is influenced by intracellular Ile availability and by GTP levels within the bacteria [<a href="#ppat.1013291.ref020" class="usa-link" aria-describedby="ppat.1013291.ref020">20</a>,<a href="#ppat.1013291.ref021" class="usa-link" aria-describedby="ppat.1013291.ref021">21</a>]. Moreover, specific gene targets can be regulated not only by CodY but also by other gene expression modulatory mechanisms. For instance, the <em>ilv</em> operon, which encodes BCAA biosynthesis, is not only subject to CodY regulation but is also influenced by the formation of a leucine-rich attenuator peptide that downregulates the expression of <em>ilv</em> genes [<a href="#ppat.1013291.ref021" class="usa-link" aria-describedby="ppat.1013291.ref021">21</a>]. This peptide is encoded by an RNA, which was termed Rli60 in <em>Listeria monocytogenes</em>, where its role in fine-tuning the CodY response has been demonstrated [<a href="#ppat.1013291.ref065" class="usa-link" aria-describedby="ppat.1013291.ref065">65</a>].</p>
<p>Previous studies had concluded that <em>S. aureus</em> is auxotrophic for BCAAs Leu and Val [<a href="#ppat.1013291.ref066" class="usa-link" aria-describedby="ppat.1013291.ref066">66</a>] because the bacteria delay induction of BCAA synthesis in environments with low BCAA levels [<a href="#ppat.1013291.ref024" class="usa-link" aria-describedby="ppat.1013291.ref024">24</a>], especially if Ile is present as a co-repressor with CodY. Macrophage cultivation and infection in the presence of high BCAA concentrations is sufficient to rescue the replication defect displayed by <em>brnQ1</em> mutants, which highlights the importance of these metabolites for <em>S. aureus</em> pathogenicity. Several human pathogens require functional BCAA transport or biosynthesis for full virulence (reviewed in [<a href="#ppat.1013291.ref023" class="usa-link" aria-describedby="ppat.1013291.ref023">23</a>]), and our study reaffirms the pivotal role of these metabolites at the host - pathogen interface. </p>
<p>Mature phagolysosomes are highly degradative organelles, in part, due to their acidic pH and enrichment with proteolytic enzymes such as cathepsins that degrade proteins thereby liberating peptides [<a href="#ppat.1013291.ref067" class="usa-link" aria-describedby="ppat.1013291.ref067">67</a>]. In addition, <em>S. aureus</em> secretes several proteases (e.g., aureolysin or staphopain) that can degrade various substrates and liberate peptides that could serve as nutritional source [<a href="#ppat.1013291.ref068" class="usa-link" aria-describedby="ppat.1013291.ref068">68</a>]. Indeed, the <em>S. aureus</em>-containing phagosome has been shown to be proteolytic [<a href="#ppat.1013291.ref033" class="usa-link" aria-describedby="ppat.1013291.ref033">33</a>]. Through the activity of the oligopeptide permease Opp3 [<a href="#ppat.1013291.ref068" class="usa-link" aria-describedby="ppat.1013291.ref068">68</a>] and the di-tri-peptide permease DtpT [<a href="#ppat.1013291.ref069" class="usa-link" aria-describedby="ppat.1013291.ref069">69</a>], <em>S. aureus</em> could, at least in theory, utilize peptides to satisfy its amino acid requirements under conditions of nutrient limitation. Nevertheless, we found that inactivation of <em>brnQ1</em> alone is sufficient to inhibit <em>S. aureus</em> growth inside the macrophage phagosome. Although protease expression is enhanced upon <em>codY</em> inactivation [<a href="#ppat.1013291.ref025" class="usa-link" aria-describedby="ppat.1013291.ref025">25</a>,<a href="#ppat.1013291.ref070" class="usa-link" aria-describedby="ppat.1013291.ref070">70</a>], we find that simultaneous mutation of <em>brnQ1, bcaP</em> and <em>ilvD</em> is sufficient to render a <em>S. aureus codY</em> mutant unable to grow intracellularly. Our transcriptomic analysis revealed that genes in the <em>opp3</em> operon were some of the few loci upregulated by intracellular <em>S. aureus brnQ1</em> mutants suggesting that the bacteria may utilize this peptide transporter to maintain viability. The fact that <em>brnQ1-</em> or <em>brnQ1/bcaP/codY/ilvD/</em>-deficient <em>S. aureus</em> still fail to grow inside macrophages, would indicate that peptides are either not being used by the bacteria or that these peptides do not provide enough BCAAs to support growth. Notably, all strains with a <em>codY</em> mutation display increased proteolytic activity <em>in vitro.</em> Still, <em>S. aureus codY</em> mutants which additionally lack <em>brnQ1</em>, <em>bcaP</em> and <em>ilvD</em> (quadruple mutants), remain unable to acquire sufficient BCAAs to support growth withing macrophages (<a href="#ppat.1013291.s016" class="usa-link">S16 Fig</a>), indicating that peptides do not provide enough BCAAs to overcome macrophage restriction. These observations provide important insight into the contribution of free amino acids obtained from the extracellular environment versus protein degradation and peptides within the phagosome as a source of nutrition which allows <em>S. aureus</em> replication within the phagosome lumen.</p>
<p>The impact of <em>brnQ1</em> inactivation on the physiology of intracellular <em>S. aureus</em> is striking and forces the bacteria into a non-replicative state, causing them to fail to regulate adaptiveness to environmental cues. Decreased proliferation rates and/ or growth arrest and metabolic quiescence, accompanied by antibiotic tolerance, are hallmarks of <em>persistence</em> [<a href="#ppat.1013291.ref050" class="usa-link" aria-describedby="ppat.1013291.ref050">50</a>–<a href="#ppat.1013291.ref052" class="usa-link" aria-describedby="ppat.1013291.ref052">52</a>]. So-called <em>persisters</em> can arise stochastically within a bacterial population [<a href="#ppat.1013291.ref050" class="usa-link" aria-describedby="ppat.1013291.ref050">50</a>], or in response to environmental cues, such as exposure to antibiotics [<a href="#ppat.1013291.ref043" class="usa-link" aria-describedby="ppat.1013291.ref043">43</a>], nutrient starvation, oxidative stress [<a href="#ppat.1013291.ref054" class="usa-link" aria-describedby="ppat.1013291.ref054">54</a>], or internalization by host cells, particularly macrophages [<a href="#ppat.1013291.ref053" class="usa-link" aria-describedby="ppat.1013291.ref053">53</a>,<a href="#ppat.1013291.ref071" class="usa-link" aria-describedby="ppat.1013291.ref071">71</a>]. Bacterial persisters are thought to constitute a reservoir for relapsing infections, associated with therapeutic failures [<a href="#ppat.1013291.ref043" class="usa-link" aria-describedby="ppat.1013291.ref043">43</a>,<a href="#ppat.1013291.ref071" class="usa-link" aria-describedby="ppat.1013291.ref071">71</a>].</p>
<p>Various mechanisms were shown to trigger <em>S. aureus persisters</em>, such as antibiotic exposure of intracellular <em>S. aureus</em> [<a href="#ppat.1013291.ref043" class="usa-link" aria-describedby="ppat.1013291.ref043">43</a>,<a href="#ppat.1013291.ref054" class="usa-link" aria-describedby="ppat.1013291.ref054">54</a>]. In macrophages, for example, stringent response induction [<a href="#ppat.1013291.ref043" class="usa-link" aria-describedby="ppat.1013291.ref043">43</a>] and ROS-induced defects in translation and ATP synthesis were proposed as mechanisms of persistence [<a href="#ppat.1013291.ref054" class="usa-link" aria-describedby="ppat.1013291.ref054">54</a>,<a href="#ppat.1013291.ref055" class="usa-link" aria-describedby="ppat.1013291.ref055">55</a>]. Moreover, ROS was shown to mediate intracellular <em>S. aureus</em> antibiotic tolerance by disrupting the TCA cycle [<a href="#ppat.1013291.ref055" class="usa-link" aria-describedby="ppat.1013291.ref055">55</a>].</p>
<p>Antibiotic pressure is a major factor in triggering intracellular persistence of <em>S. aureus</em> [<a href="#ppat.1013291.ref043" class="usa-link" aria-describedby="ppat.1013291.ref043">43</a>,<a href="#ppat.1013291.ref054" class="usa-link" aria-describedby="ppat.1013291.ref054">54</a>,<a href="#ppat.1013291.ref055" class="usa-link" aria-describedby="ppat.1013291.ref055">55</a>,<a href="#ppat.1013291.ref072" class="usa-link" aria-describedby="ppat.1013291.ref072">72</a>]. SCVs, an extensively studied form of <em>S. aureus</em> persisters [<a href="#ppat.1013291.ref051" class="usa-link" aria-describedby="ppat.1013291.ref051">51</a>,<a href="#ppat.1013291.ref057" class="usa-link" aria-describedby="ppat.1013291.ref057">57</a>,<a href="#ppat.1013291.ref060" class="usa-link" aria-describedby="ppat.1013291.ref060">60</a>,<a href="#ppat.1013291.ref073" class="usa-link" aria-describedby="ppat.1013291.ref073">73</a>–<a href="#ppat.1013291.ref076" class="usa-link" aria-describedby="ppat.1013291.ref076">76</a>], may arise in the absence of antibiotic selective pressure [<a href="#ppat.1013291.ref077" class="usa-link" aria-describedby="ppat.1013291.ref077">77</a>]. They comprise a slow-growing subpopulation of bacteria with very distinctive phenotypic characteristics: reduced haemolysis and cytotoxicity, variations in pigmentation as compared to parental strains, enhanced antibiotic tolerance and slower growth rates, often due to auxotrophies (e.g., menadione or haemin) [<a href="#ppat.1013291.ref078" class="usa-link" aria-describedby="ppat.1013291.ref078">78</a>], resulting in non-haemolytic pinpoint-sized colonies on agar plates.</p>
<p><em>S. aureus brnQ1</em> mutants recovered here from macrophages, however, despite being growth-arrested and non-responsive to environmental cues (i.e., iron depletion; aTc-induction of the <em>Cerulean</em> fluorescent protein), do not exhibit SCV characteristics, nor do they display other phenotypes reported for persisters (e.g., induction of stringent response [<a href="#ppat.1013291.ref043" class="usa-link" aria-describedby="ppat.1013291.ref043">43</a>]).</p>
<p>The phenotype reported here, conforms to the definition of <em>dormancy,</em> as stated by Balaban et al.[<a href="#ppat.1013291.ref051" class="usa-link" aria-describedby="ppat.1013291.ref051">51</a>]: “<em>Dormancy reflects the state of a bacterium that does not grow and has decreased activity when compared with growing cells”</em>. This phenotype appears to be a non-antibiotic-driven form of intracellular dormancy triggered by BCAA starvation within macrophage phagolysosomes. This dormancy is reversible, as evidenced by the sporadic outgrowth of the bacteria from infected macrophages, in what appears to be, a stochastic event.</p>
<p>The environmental cues or genetic changes that enable this process are currently unknown and are of research interest. One possible explanation is that the bacteria inactivate CodY, however re-infection of macrophages with <em>brnQ1</em> bacteria that displayed <em>URG</em> does not alter or enhance the course of a subsequent infection. Furthermore, genome sequence analysis revealed no SNPs in <em>codY</em> or <em>ilv</em> genes, which were previously reported to be “hot spots” for the accumulation of suppressor mutations that allow <em>S. aureus</em> to eventually replicate under BCAA-deprived conditions [<a href="#ppat.1013291.ref021" class="usa-link" aria-describedby="ppat.1013291.ref021">21</a>]. Further investigation into how <em>S. aureus brnQ1</em> exits <em>dormancy</em> is ongoing and will provide insight into the mechanisms <em>S. aureus</em> employs to modulate its metabolic program for growth and survival under nutrient-limited conditions.</p>
<p>In summary, our study revealed a pivotal role for BCAA transport in the growth and survival of <em>S. aureus</em> inside the macrophage phagolysosome. Remarkably, inactivation of BrnQ1 alone is sufficient to arrest intracellular growth of <em>S. aureus</em> and force the bacteria into a state of temporary <em>dormancy.</em> These growth-arrested <em>S. aureus</em> reside within viable phagocytes, in the absence of any antibiotic pressure, and could serve as a long-term reservoir of bacteria with complete virulence potential, should BCAA starvation be alleviated.</p>
<p>Understanding the mechanisms underlying this state of dormancy and how the bacteria “reactivate” is important in the rational development anti-staphylococcal therapies especially since bacterial amino acid and peptide transporters are emerging as an attractive target in the fight against virulent or antibiotic-resistant <em>S. aureus</em> [<a href="#ppat.1013291.ref079" class="usa-link" aria-describedby="ppat.1013291.ref079">79</a>]. However, our observations suggest that anti-staphylococcal strategies targeting bacterial amino acid transporters such as BrnQ1 should be carefully tailored to avoid selection of dormant intracellular populations prone to long-term intracellular survival and potential dispersal to secondary infection sites.</p></section><section id="sec010"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="sec011"><h3 class="pmc_sec_title">Ethics Statement</h3>
<p>Leukocyte samples (i.e., leukoreduction system chambers from anonymous healthy donors) were obtained from the Institute for Clinical Transfusion Medicine and Haemotherapy, of the University of Würzburg, Germany for research purposes and require no specific approval by an ethics committee. Whole human blood was also drawn from healthy adult volunteers upon verbal consent in accordance with protocols approved by the University of Western Ontario’s Research Ethics Board, London, Ontario, Canada.</p></section><section id="sec012"><h3 class="pmc_sec_title">Bacterial culture conditions</h3>
<p>Bacterial strains are listed in <a href="#ppat.1013291.s021" class="usa-link">S1 Table</a>. All liquid cultures were grown aerobically at 37°C with 200 rpm shaking. <em>E. coli</em> was grown in Luria-Bertani broth (LB) or on LB agar containing 100 µg/mL ampicillin for plasmid maintenance when necessary. <em>S. aureus</em> strains were routinely grown in tryptic soy broth (TSB), or on TSB agar plates (TSA) if not stated otherwise. Media were supplemented with appropriate antibiotics, when necessary: 10 µg/ml chloramphenicol (plasmid encoded antibiotic resistance); and/or 5 µg/ml erythromycin, 5 µg/mL tetracycline or 50 µg/mL kanamycin (for chromosomally encoded antibiotic resistance).</p></section><section id="sec013"><h3 class="pmc_sec_title">Construction of bacterial strains and plasmids</h3>
<p>All used strains (<a href="#ppat.1013291.s021" class="usa-link">S1 Table</a>), plasmids (<a href="#ppat.1013291.s024" class="usa-link">S2 Table</a>) and oligonucleotides (<a href="#ppat.1013291.s023" class="usa-link">S3 Table</a>) can be found in the Supporting material. Strains used for macrophage infection experiments shown in <a href="#ppat.1013291.g001" class="usa-link">Fig 1c</a> were retrieved from the Nebraska Transposon mutant library [<a href="#ppat.1013291.ref034" class="usa-link" aria-describedby="ppat.1013291.ref034">34</a>].</p>
<p>The <em>S. aureus</em> insertional transposon mutant of <em>brn</em>Q<em>1</em> (NE945) was retrieved from the Nebraska Transposon mutant library, transposon insertion at the correct position was confirmed by PCR, using primers brnQ1_fwd and brnQ1_rev and the mutation was further transduced via phage φ11 or φ80 into the genetic background of wild-type <em>S. aureus</em> strains to yield JE2 <em>brnQ1</em>, 6850 <em>brnQ1</em>, SH1000 <em>brnQ1</em> and RN4220 <em>brnQ1</em> respectively.</p>
<p>Plasmid p<em>brnQ1</em> was generated to complement the <em>brnQ1</em> gene <em>in trans</em> under the transcriptional control of its native promoter, using Fast Cloning [<a href="#ppat.1013291.ref080" class="usa-link" aria-describedby="ppat.1013291.ref080">80</a>]. Briefly, the open reading frame of <em>brnQ1</em> gene (locus ID: SAUSA300_0188/ SAUSA300_RS00985) was amplified from <em>S. aureus</em> JE2 genomic DNA together with its native promoter region included in the 400 bp upstream of the start codon, using primer pairs brnQ1_Comp_Inf_fwd and brnQ1_Comp_Inf_rev. The pALC2085 vector [<a href="#ppat.1013291.ref081" class="usa-link" aria-describedby="ppat.1013291.ref081">81</a>] was linearized by PCR using primer pairs p2085_Inf_for brnQ1_fwd and p2085_Inf_for brnQ1_rev. Both insert and vector were amplified using an 18-cycle amplification protocol and Phusion DNA-polymerase (Thermo Fisher Scientific, Cat. No. F530L). 1 µL DpnI enzyme (Thermo Fisher Scientific, Cat. No. ER1701) was added to each 50 µL un-purified PCR reaction, vector and insert were mixed in a 1:1 (v/v) ratio and DpnI digest was carried out for 1h at 37°C. 2 µL of the reaction was directly transformed into electrocompetent <em>E. coli</em> DH5α and plated on LB agar plates supplemented with 100 µg/mL ampicillin. The plasmid p<em>brnQ1</em> was isolated from <em>E. coli</em> and the sequence was confirmed by Sanger sequencing (SeqLab, Göttingen). Subsequently, plasmids were electroporated into <em>S. aureus</em> RN4220 and further transduced into <em>S. aureus</em> JE2 using phage φ11. A similar procedure was used to generate a control, by amplifying a promoter-less <em>brnQ1</em> using primer pairs brnQ1_Comp_Inf_fwd and brnQ1_P-less_Comp_Inf_rev and cloning it into the pALC2085 backbone to yield pP<sub>less</sub>_<em>brnQ1</em>. Plasmids p<em>bcaP</em> and p<em>brnQ1</em> shown in <a href="#ppat.1013291.g004" class="usa-link">Fig 4d</a> were previously described [<a href="#ppat.1013291.ref024" class="usa-link" aria-describedby="ppat.1013291.ref024">24</a>,<a href="#ppat.1013291.ref029" class="usa-link" aria-describedby="ppat.1013291.ref029">29</a>] and carry <em>bcaP</em> or <em>brnQ1,</em> respectively, that were cloned into the expression plasmid pRMC2 under the control of their respective native promoters.</p>
<p>Red-fluorescent or cyan-fluorescent expressing <em>S. aureus</em> JE2 were generated by transducing the pSarAP1-mRFP or p<em>Cerulean</em>, respectively, from recombinant <em>S. aureus</em> RN4220 strains using phage φ11.</p>
<p><em>S. aureus</em> JE2 <em>fur</em> and JE2 <em>brn</em>Q1/<em>fur</em> were generated by transducing the <em>tet</em> cassette from <em>S</em>. <em>aureus</em> MJH010 <em>fur::tet</em> [<a href="#ppat.1013291.ref040" class="usa-link" aria-describedby="ppat.1013291.ref040">40</a>] using phage φ11.</p>
<p>Strains in USA300 carrying either <em>codY</em> (locus ID: SAUSA300_1148/ SAUSA300_RS06210) or <em>ilvD</em> (locus ID: SAUSA300_2006/ SAUSA300_RS11035) mutations were created by transducing either <em>codY</em>::Tn (NE1555) or <em>ilvD</em>::Tn-Kan<sup>R</sup> using phage φ80 into the recipient strain.</p>
<p>The replacement of the erythromycin resistant cassette in strain JE2 NE718 (<em>ilvD)</em> was done using the plasmid pKAN as previously described [<a href="#ppat.1013291.ref082" class="usa-link" aria-describedby="ppat.1013291.ref082">82</a>]. In brief, pKAN plasmid was transduced into JE2 NE718 (<em>ilvD)</em> using phage φ80. Replacement of the Ery<sup>R</sup> cassette for Kan<sup>R</sup> was confirmed by resistance phenotype (i.e., the strain became Ery<sup>R</sup> and Kan<sup>R</sup>) and by PCR analysis.</p></section><section id="sec014"><h3 class="pmc_sec_title">Isolation of primary human monocytes from human peripheral blood</h3>
<p>Primary human macrophages were derived from peripheral blood mononuclear cells (PBMCs) isolated from whole blood leukoreduction system (LRS) chambers [<a href="#ppat.1013291.ref083" class="usa-link" aria-describedby="ppat.1013291.ref083">83</a>] using the SepMate-50 system (StemCell Technologies, Cat. No. 85450) and Ficoll-Paque PLUS density gradient medium (Sigma Aldrich, Cat. No. GE17-1440-03), according to manufacturer’s instructions. Monocytes were then purified from the PBMC fraction using the EasySep Human CD14 Positive Selection Kit II (StemCell Technologies, Cat. No. 17858) according to the manufacturer’s instructions. In some instances PBMCs were isolated from whole human blood using lympholyte-poly (Cedarlane labs) according to the manufacturer’s instructions and as described [<a href="#ppat.1013291.ref039" class="usa-link" aria-describedby="ppat.1013291.ref039">39</a>].</p></section><section id="sec015"><h3 class="pmc_sec_title">Macrophage differentiation and polarization</h3>
<p>All macrophage cultivation steps, and infection experiments were done in RPMI1640 (Thermo Fisher Scientific, Cat. No. 72400054) supplemented with FBS (10% v/v; heat inactivated: 56°, 30 min) henceforth termed <em>Infection Medium,</em> unless otherwise indicated. Purified monocytes were seeded in Nunc UpCell cell culture plates (Thermo Fisher Scientific, Cat. No. 174902) at a density of 5-7x 10<sup>6</sup> cells/plate in <em>Infection Medium</em> supplemented with 50 ng/mL macrophage-colony-stimulating factor (M-CSF; StemCell Technologies, Cat. No. 78057) and incubated at 37°C in a humidified incubator with 5% CO<sub>2</sub> for 7 days. Following the 7-day differentiation protocol, macrophages were detached from the Nunc UpCell by incubation at 20°C, following the manufacturer’s instructions. Cells were collected, centrifuged, re-suspended in <em>Infection Medium</em>, counted using a Neubauer counting chamber and seeded in cell culture well plates. In some instances isolated PBMCs were adhered directly to 12 well tissue culture treated plates and differentiated in the presence of recombinant human M-CSF at 10 ng/mL (PeproTech, Cat. No. 300–25) as previously described [<a href="#ppat.1013291.ref039" class="usa-link" aria-describedby="ppat.1013291.ref039">39</a>]. Differentiated macrophages were used for infections from day 7 to day 9.</p></section><section id="sec016"><h3 class="pmc_sec_title">Macrophage infection assay</h3>
<p>For standard infection experiments, 10<sup>5</sup> cells were seeded into 24 well microtiter plates, 24h prior to infection. Bacterial overnight cultures (TSB) were diluted to an OD<sub>600</sub> = 0.5 and grown in TSB for 45 min at 37 °C, under continuous shaking at 200 rpm. Bacteria were washed with Dulbecco’s phosphate-buffered saline (DPBS, Thermo Fisher Scientific, Cat. No. 14190169), diluted in <em>Infection Medium</em> and counted using a Thoma counting chamber (Hartenstein, Germany, Cat. No. ZK04). For counting, 10 µL of the above-described suspension were loaded onto the Thoma chamber (<em>small</em> counting square area: 0.0025 mm<sup>2</sup>; depth 0.1 mm). Bacteria were allowed to sediment for 10 min, then counted in at least 16 <em>small</em> squares and the “chamber factor” of 4 x 10<sup>6</sup> was used to estimate no. of bacteria/mL. Non-opsonized bacteria were used to infect macrophages at a multiplicity of infection (MOI) of 5. For the synchronization of infection, plates were centrifuged at 200 x g, for 5 min, without brakes and then incubated for 30 min at 37°C in an incubator with humidified atmosphere with 5% CO<sub>2</sub>. After 30 min, extracellular bacteria were removed by 15 min incubation in <em>Infection Medium</em> containing 30 µg/ml lysostaphin (AMBICIN L, AMBI), followed by washing twice with sterile DPBS and further incubation in <em>Infection Medium</em> without antibiotics until the end of the experiment. For long-term experiments, viable infected macrophages were cultivated for up to 28 days in the absence of lysostaphin (except for the initial 15 min pulse).</p>
<p>In some instances, primary human and RAW 264.7 macrophage infections were performed using 12 well tissue culture treated dishes as previously described [<a href="#ppat.1013291.ref012" class="usa-link" aria-describedby="ppat.1013291.ref012">12</a>]. Here macrophages were infected at an MOI of 5 in serum-free RPMI1640. After synchronization by centrifugation at 277 x g for 2 min, phagocytosis was allowed to occur for 30 min. At this time the medium was replaced with serum-free RPMI1640 containing 100 μg/mL gentamicin for 1 hr, to kill the extracellular bacteria, followed by washing twice with sterile DPBS and further incubated with <em>Infection Medium.</em> For RAW 264.7 macrophage experiments, <em>Infection Medium</em> with 5% (v/v) FBS was used instead.</p></section><section id="sec017"><h3 class="pmc_sec_title">Intracellular bacterial replication assay (CFU)</h3>
<p>To enumerate intracellular bacteria, at the required time points, cells were washed once with DPBS and lysed using alkaline water (pH11) for 5 min at room temperature. Dilution series of the complete cell lysate were plated on TSA and recovered bacterial colony forming units (CFU) were counted. “T0” represented in all graphs corresponds to 45 min <em>p.i.</em> (i.e., 30 min phagocytosis and 15 min lysostaphin treatment). In some instances, after gentamicin treatment and DPBS washing, infected macrophages were lysed using 0.5 mL of 0.1% (v/v) Triton X-100 diluted in DPBS. Lysis at this time equals 1.5h <em>p.i.</em> and the CFU/mL represents the number of phagocytosed or gentamicin protected bacteria. At 20h <em>p.i.</em> supernatants from all wells were collected individually and centrifuged to pellet any bacteria that may have emerged from infected macrophages. While centrifuging, wells containing macrophages were lysed using Triton X-100 as described immediately above, and this lysate was used to resuspend any pelleted bacteria. At each time point bacteria were serially diluted onto TSA and the CFU determined. To calculate the fold change at 20h <em>p.i.,</em> the CFU/mL at 20 h was divided by CFU/mL at 1.5h. The data were expressed as the Log10 fold change.</p></section><section id="sec018"><h3 class="pmc_sec_title">Exogenous BCAA complementation</h3>
<p>Macrophages were seeded 24h prior to infection in either <em>Infection Medium</em> or <em>Infection medium</em> supplemented with 1 mM of each branched-chain amino acid (Ile, Leu, Val) and all infection steps were carried out in the respective medium. Samples were further processed for CFU counting or RNA extraction at 45 min (“T0”), 10h and 24h <em>p.i.</em> (CFU) or 10h <em>p.i.</em> (RNA). In some instances, 1mM of either Val, Leu or both were added to RPMI1640 + 5% FBS that was added back to RAW 264.7 macrophages immediately after gentamicin treatment. CFU determination was performed as described above.</p></section><section id="sec019"><h3 class="pmc_sec_title">Live Cell Imaging</h3>
<p>4.5 x 10<sup>4</sup> macrophages/well were seeded in 8 well chamber µ-slides (ibidi, Cat. No. 80826) 24h prior to infection. Infection with mRFP-expressing bacteria was performed at an MOI of 5 as described above. Activation of effector caspases 3/7 in infected macrophages was visualised 5h <em>p.i.</em> by the addition of 5 µM of the fluorogenic substrate Cell Event Caspase-3/7 Green Detection Reagent (Thermo Fisher Scientific, Cat. No. <a href="https://www.ncbi.nlm.nih.gov/nuccore/C10723" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">C10723</a>).Time-lapse imaging was performed in <em>Imaging Medium</em> (<em>Infection Medium</em> without phenol red), in the absence of antibiotics, on a Leica TCS SP5 confocal laser scanning microscope (Leica GmbH, Wetzlar, Germany; software LAS AF version 2.7.3.9723) using a 40x oil immersion objective (Leica HC PL APO, NA = 1.3), equipped with a pre-warmed (37°C) live-cell incubation chamber (Life Imaging Systems) and perfused with a humidified atmosphere containing 5% CO<sub>2</sub>. The LAS AF software was used for setting adjustment and image acquisition. All images were acquired at a resolution of 1024x1024 and recorded in 8-bit mode, at 30 min time intervals. Z-stacks were imaged with a step size of 0.4 µm. Image-processing was performed using Fiji [<a href="#ppat.1013291.ref084" class="usa-link" aria-describedby="ppat.1013291.ref084">84</a>].</p></section><section id="sec020"><h3 class="pmc_sec_title">Phase contrast microscopy</h3>
<p>Phase contrast microscopic images of live, infected cells were acquired with a LEICA DM IRB microscope connected to a SPOT Pursuit USB camera using a Leica C Plan L 20 x/0.30 PH1 objective and VisiView software (Visitron).</p></section><section id="sec021"><h3 class="pmc_sec_title">Transmission Electron Microscopy (TEM)</h3>
<p>TEM sample preparation and imaging were carried at the Imaging Core Facility of the Biocenter, University of Würzburg, Germany. Macrophages were seeded on cover slips and infection was performed as described. Infected samples were washed once in DPBS, fixed with 2.5% glutaraldehyde (in 50 mM sodium cacodylate [pH 7.2], 50 mM KCl and 2.5 mM MgCl<sub>2</sub>) for 2h at 4°C. Samples were subsequently washed twice for 5 min with 50 mM cacodylate buffer (pH 7.2) and were further fixed with 2% buffered OsO<sub>4</sub> (Science Services) and contrasted with 0.5% uranyl acetate in distilled water and finally embedded in Epon (Serva) after ethanol-based dehydration. Ultrathin sections were generated from embedded samples and contrasted with 2% uranyl acetate in ethanol and lead citrate. TEM imaging was performed on a JEOL JEM-2100 instrument.</p></section><section id="sec022"><h3 class="pmc_sec_title">Preparation of cells for RNA extraction</h3>
<p>For total RNA extraction (host and pathogen), 3 x 10<sup>6</sup> macrophages grown in 6-well cell culture plates (i.e., 1.5 x 10<sup>6</sup> macrophages were seeded per well and 2 wells/condition were pooled) were infected at MOI of 5, as described. At 10h <em>p.i.</em>, cells were washed once with DPBS, and RNAprotect Cell Reagent (Qiagen, Cat. No. 76526) was added immediately (1 part <em>Infection Medium</em> and 5 parts RNAprotect). Samples were stored at 4°C for a maximum of one week before further processing.</p>
<p>Experiments with bacteria grown in <em>Infection Medium</em> were carried out in parallel, using the same bacterial inoculum. Bacteria were similarly grown in 6-well plates at 37°C in a humidified incubator with 5% CO<sub>2</sub>, without shaking. After 10h, bacteria were collected by centrifugation and pellets were flash-frozen in liquid N<sub>2</sub> and stored at -80°C.</p>
<p>For experiments depicted in <a href="#ppat.1013291.g003" class="usa-link">Fig 3a</a>, bacteria were grown overnight in TSB, washed 2x in DPBS and resuspended in <em>Infection Medium</em> at OD<sub>600</sub> = 1. Bacteria were grown for 1h at 37°C with 200 rpm shaking, collected and flash-frozen in liquid N<sub>2</sub> and stored at -80°C.</p></section><section id="sec023"><h3 class="pmc_sec_title">RNA extraction</h3>
<p>RNA extraction from either <em>S. aureus</em>-infected macrophages or cultures of <em>S. aureus</em> was done using the RNeasy Mini Kit (Qiagen, Cat. No. 74104). Infected cells stored in RNAprotect Cell Reagent were collected in RNase-free tubes, by centrifugation (6,200 x g, 5 min, 4°C). Frozen bacterial pellets were allowed to thaw on ice for 5 min. Next, 600 µL RLT Buffer (kit) supplemented with β-mercaptoethanol (10 µL) was added to the pellets, samples were disrupted in a FastPrep FP120 instrument (6 m/s for 45 sec, at 4°C) using Lysing Matrix B tubes (MP Biomedicals, Cat. No. 116911100), immediately cooled down and centrifuged (10,000 x g, 2 min, 4°C). Following lysis, manufacturer’s instructions for the RNeasy Mini kit were followed. TURBO DNA-<em>free</em> DNase (Thermo Fisher Scientific, Cat. No. AM1907) was used to digest contaminating DNA (1h, 37°), according to manufacturer’s instructions. For routine applications, concentration and purity were determined using a Nanodrop instrument and non-denaturing agarose gel electrophoresis, using ethidium bromide staining [<a href="#ppat.1013291.ref085" class="usa-link" aria-describedby="ppat.1013291.ref085">85</a>]. The absence of DNA contamination was verified by qPCR (40 cycles using <em>gyrB</em>-specific primers and RNA samples as template).</p></section><section id="sec024"><h3 class="pmc_sec_title">RNA sequencing</h3>
<p>RNA-seq of <em>S. aureus</em> grown in <em>Infection Medium</em> as well as dual RNAseq of <em>S. aureus</em>-infected macrophages were conducted by the Core Unit SysMed at the University of Würzburg (Germany). RNA quality was checked using a 2100 Bioanalyzer with the RNA 6000 Nano kit (Agilent Technologies, Cat. No. 5067–1511). Ribosomal RNA (rRNA) was depleted from 200 ng of total RNA as input using Lexogen’s RiboCop rRNA Depletion Kit protocol according to manufacturer’s recommendation with the following modifications: for mixed human and bacterial samples, 5 µl of RiboCOP V1.2 probes were mixed with 5 µL of RiboCOP META probes, 4 µl hybridization buffer, and 21 µL of RNA sample. Following rRNA depletion, DNA libraries suitable for sequencing were prepared using CORALL Total RNA-Seq Library Prep protocol according to manufacturer’s recommendation with 14 PCR cycles. After quality control, libraries were sequenced on an <em>Illumina NextSeq</em> platform (1*75 bp single end run). Demultiplexed FASTQ files were generated with bcl2fastq2 v2.20.0.422.</p>
<p>Sequencing reads were trimmed for Illumina adapter sequences using Cutadapt version 2.5 with default parameters. In addition, <em>Cutadapt</em> was given the –<em>nextseq-trim = 20</em> switch to handle two colour sequencing chemistry and reads that were trimmed to length 0 were discarded. Reads were filtered with <em>FastQScreen</em> [<a href="https://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.bioinformatics.babraham.ac.uk/projects/fastq_screen/</a>]. The splice-aware aligner STAR [<a href="#ppat.1013291.ref086" class="usa-link" aria-describedby="ppat.1013291.ref086">86</a>] was used for the mapping of reads originating from human cells to the human genome (GRCh38.p13), whereas the Reademption pipeline [<a href="#ppat.1013291.ref087" class="usa-link" aria-describedby="ppat.1013291.ref087">87</a>] was used to map reads originating from <em>S. aureus</em> to the reference (GCF_000013465.1_ASM1346v1). Read counts for each gene were generated and the count output was utilized to identify differentially expressed genes using DESeq2 v1.24.0 [<a href="#ppat.1013291.ref088" class="usa-link" aria-describedby="ppat.1013291.ref088">88</a>].</p></section><section id="sec025"><h3 class="pmc_sec_title">RT-qPCR</h3>
<p>1 µg total RNA was converted to cDNA using the RevertAid First Strand cDNA Synthesis kit (Thermo Fisher Scientific, Cat. No. K1622) according to manufacturer’s instructions, using random hexamer primers. RT-qPCR was performed one a StepOne Plus instrument (Applied Biosystems). Each reaction mix consisted of GreenMasterMix (Genaxxon, Cat. No. M3052.0500), 900 nM of each primer and 50 ng (infected samples) or 10 ng (bacteria alone) cDNA. Relative gene expression was normalized to expression of the <em>gyrB</em> reference gene and represented as -ΔCt values [<a href="#ppat.1013291.ref089" class="usa-link" aria-describedby="ppat.1013291.ref089">89</a>]. Oligonucleotides used in RT-qPCR experiments are listed in <a href="#ppat.1013291.s023" class="usa-link">S3 Table</a>.</p></section><section id="sec026"><h3 class="pmc_sec_title">Genome sequencing</h3>
<p>DNA was isolated using the Promega One4All kit on a KingFisher (ThermoFisher Scientific) device. DNA concentration was measured using Qubit (dsDNA HS assay kit, Thermo Fisher Scientific, Germany) and sequencing libraries were generated using the Nextera XT library Prep Kit (Illumina, USA). Data acquisition was performed on a NextSeq550Dx (Illumina, USA) during a 2 × 150 bp paired-end sequencing run using a mid-output cassette. The derived FastQ files were quality-controlled, and the genomes were assembled using SKESA (V2.4.0 on Seqsphere+ (client and server version 9.0.0, Ridom GmbH, Münster). Contigs were aligned and reordered using progressiveMauve [<a href="#ppat.1013291.ref090" class="usa-link" aria-describedby="ppat.1013291.ref090">90</a>] and the genomes were annotated using prokka [<a href="#ppat.1013291.ref091" class="usa-link" aria-describedby="ppat.1013291.ref091">91</a>].</p></section><section id="sec027"><h3 class="pmc_sec_title">
<em>Cerulean</em> induction in infected macrophages</h3>
<p>Macrophages were seeded on coverslips and infected as described with JE2 WT or JE2 <em>brn</em><em>Q1</em> carrying p<em>Cerulean</em> (anhydrous tetracycline (aTc)-inducible expression of the codon-adapted fluorescent protein Cerulean [<a href="#ppat.1013291.ref081" class="usa-link" aria-describedby="ppat.1013291.ref081">81</a>]). At either 1h or 10h <em>p.i. Cerulean</em> was induced by the addition of 300 ng/mL aTc, and samples were incubated for further 2h before fixing with 4% paraformaldehyde (PFA, Morphisto, Cat. No. 11762.01000). Fixed cells were washed 3x in DPBS, permeabilized with 0.2% Triton-X100 (in DPBS) for 5 min, followed by incubation with 50 mM ammonium chloride for 5 min. Staining was done for 20 min, with a solution containing 1 µg/mL BODIPY FL Vancomycin (Thermo Fisher Scientific, Cat. No. V34850), Phalloidin-iFluor647 (Abcam, Cat. No. 176759; diluted according to manufacturer’s protocol) and 4 µg/mL Hoechst 34580 (Thermo Fisher Scientific, Cat. No. <a href="https://www.ncbi.nlm.nih.gov/nuccore/H21486" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">H21486</a>), in DPBS. Cover-slips were mounted on microscopy slides with Mowiol and air-dried overnight.</p>
<p>Image acquisition was performed on a Leica TCS SP5 CLSM using a 40x oil immersion objective (NA 1.4). Per sample at least 10 fields of view were recorded with each covering an area of 388 × 388 µm. Images were recorded at a resolution of 1024 x 1024 pixels at 8-bit grey scale depth.</p>
<p>Imaging was done using the following excitation/emission (Ex/Em) settings: Cerulean: Ex/Em = 458/460–494 nm (Argon laser); BODIPY FL Vancomycin: Ex/Em = 488/499–586 nm; (Argon laser); Hoechst 34580: Ex/Em = 405/423–477 nm (diode laser); Phalloidin-iFluor 647 Ex/Em = 633/634–710 nm (HeNe laser).</p>
<p>Image analysis was conducted using Fiji [<a href="#ppat.1013291.ref084" class="usa-link" aria-describedby="ppat.1013291.ref084">84</a>], using an in-house macro toolset, which uses BODIPY-FL- channel to identify the macrophage-associated bacteria, whose positions are saved as individual regions-of-interest (ROI). Subsequently, the mean fluorescence intensities in each ROI were determined for both the Cerulean channel and the BODIPY-FL channel. The results were exported from Fiji as text files with comma-separated values (csv). The csv-files fere transformed into text-format using RStudio (Integrated Development for R. RStudio, PBC, Boston, MA) and subsequently imported the data as dot plots into the flow cytometry software Flowing2 (Turku Bioscience Centre, Turku Finland) to determine percent of total bacteria expressing <em>Cerulean</em>.</p></section><section id="sec028"><h3 class="pmc_sec_title">Fluorescence-based proliferation assay</h3>
<p>Fluorescence proliferation assays were performed as previously described [<a href="#ppat.1013291.ref018" class="usa-link" aria-describedby="ppat.1013291.ref018">18</a>,<a href="#ppat.1013291.ref092" class="usa-link" aria-describedby="ppat.1013291.ref092">92</a>] using GFP expressing <em>S. aureus</em> USA300 and ∆<em>brnQ1</em>. In brief, bacteria were labelled with Cell Proliferation Dye eFluor 670 |(eBioscience, Cat. No. 65-0840-85) and used to infect macrophages that were grown on 18 mm glass coverslips (No. 1) at an MOI of 5 as described above. At 1.5, 10 and 20h <em>p.i.</em> macrophages were stained with 1 µg/mL tetramethylrhodamine-conjugated wheat germ agglutinin (TMR-WGA, Thermo Fischer Scientific, Cat. No. W849). Stained cells were fixed with 4% (v/v) PFA at room temperature for 20 min and mounted on glass microscope slides using ProLong Gold Antifade Mountant (Thermo Fischer Scientific, Cat. No. <a href="https://www.ncbi.nlm.nih.gov/protein/P36934" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P36934</a>). Confocal fluorescence microscopy was performed using a Zeiss LSM 880 with Fast AiryScan equipped with a Plan-Apochromat ×63/1.4 oil DIC M27 objective and Definite Focus 2. GFP, TMR-WGA and eFluor 670 were excited with available laser lines at 488 nm, 561 nm, and 633 nm, respectively. Post-image processing (i.e., contrast enhancement and image cropping) was conducted using Fiji [<a href="#ppat.1013291.ref084" class="usa-link" aria-describedby="ppat.1013291.ref084">84</a>].</p></section><section id="sec029"><h3 class="pmc_sec_title">Bacterial colony morphology analysis</h3>
<p><em>S. aureus</em> JE2 <em>brnQ1</em> were recovered from infected macrophages by lysis with pH11 water. Colony pigmentation was assessed by plating the lysate onto TSA plates. Haemolysis was assessed by plating lysates onto Columbia agar with 5% defibrinated sheep blood. After 30h incubation at 37°C, plates were photographed and colony diameter was measured with Fiji [<a href="#ppat.1013291.ref084" class="usa-link" aria-describedby="ppat.1013291.ref084">84</a>]. Fresh bacteria grown in late-exponential phase in TSB served as control.</p></section><section id="sec030"><h3 class="pmc_sec_title">Bacterial growth curves</h3>
<p>Bacterial growth curves were recorded in 48-well plates using a TECAN MPlex plate reader. <em>S. aureus</em> strains were grown overnight in the appropriate culture medium: TSB, RPMI1640 or <em>Infection Medium</em>. Next day, cultures were diluted to an OD<sub>600</sub> = 0.1 in fresh corresponding medium and used to inoculate 400 µL/well. Sterile medium was used as reference. For experiments depicted in <a href="#ppat.1013291.s001" class="usa-link">S1</a> and <a href="#ppat.1013291.s013" class="usa-link">S13a Figs</a>, bacteria were grown overnight in TSB and washed 4x in DPBS before inoculation at OD<sub>600</sub> = 0.1, in a chemically defined medium (CDM) [[<a href="#ppat.1013291.ref093" class="usa-link" aria-describedby="ppat.1013291.ref093">93</a>] with 25 mM glucose]. BCAAs were excluded from the CDM formulation, as needed. Absorbance at 600 nm (OD<sub>600</sub>) was measured every 18 min for 20h or 50h, under continuous shaking at 200 rpm. For experiments depicted in <a href="#ppat.1013291.s001" class="usa-link">S1 Fig</a>, bacterial doubling times were estimated from growth curves.</p></section><section id="sec031"><h3 class="pmc_sec_title">Bacterial growth in valine-free CDM</h3>
<p>For experiments depicted in <a href="#ppat.1013291.s014" class="usa-link">S14 Fig</a>, CDM was prepared as previously described [<a href="#ppat.1013291.ref024" class="usa-link" aria-describedby="ppat.1013291.ref024">24</a>] and Val was omitted when needed. Bacteria were grown overnight in TSB, pelleted and washed in 1 mL of sterile saline (0.9% w/v NaCl). Bacteria were diluted into saline at an OD<sub>600</sub>nm of 0.1 and 10 μL of this suspension was used to inoculate 2mL culture volumes to give a starting OD<sub>600</sub>nm of ~ 0.0005. The bacteria were grown in 13 mL polypropylene snap cap tubes at 37°C with constant shaking at 250 rpm. The end point OD<sub>600</sub> was measured at 24h. As controls, either 1 mM Val or 0.1% (w/v) tryptone was added to the culture medium.</p></section><section id="sec032"><h3 class="pmc_sec_title">
<em>S. aureus</em> infection of non-professional phagocytic cells</h3>
<p>HeLa cells (HeLa 229, ATCC CCL-2.1) and 16HBE14o- human bronchial epithelial cells were routinely cultured in <em>Infection Medium</em>, at 37 °C and 5% CO<sub>2</sub>. For infection, 10<sup>5</sup> cells/well were seeded in 12-well microtiter plates 24h prior to infection. Infections were carried out with a multiplicity of infection (MOI) of 5 as described for macrophage infections. In this case, however, bacteria and host cells were co-cultivated for 1h, after which extracellular bacteria were removed by a 30 min treatment with 20 µg/ml lysostaphin (AMBICIN L, AMBI) followed by further incubation in medium containing 2 µg/ml lysostaphin until the end of the experiment.</p></section><section id="sec033"><h3 class="pmc_sec_title">Phagosomal escape assay</h3>
<p>Phagosomal escape was determined as previously described [<a href="#ppat.1013291.ref094" class="usa-link" aria-describedby="ppat.1013291.ref094">94</a>]. Briefly, HeLa YFP-CWT or 16HBE14o- YFP-CWT cells were infected with <em>S. aureus</em> at MOI of 5 in 24 well µ-plates (ibidi, Cat. No. 82406). For experiments depicted in <a href="#ppat.1013291.s004" class="usa-link">S4b</a>, <a href="#ppat.1013291.s004" class="usa-link">S4c</a> and <a href="#ppat.1013291.s005" class="usa-link">S5b Figs</a>, mRFP-expressing bacterial strains were used. For experiments shown in <a href="#ppat.1013291.s005" class="usa-link">S5c Fig</a>, bacteria were stained with SNARF-1 (Thermo Fisher Scientific, Cat. No. <a href="https://www.ncbi.nlm.nih.gov/protein/S22801" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S22801</a>) prior to infection. For this, logarithmic growth-phase bacteria grown in TSB were washed once with DPBS and stained with 8 µM SNARF-1 in a final volume of 500 µL DPBS, for 20 min, at RT, protected from light. Afterwards, samples were washed twice in DPBS and de-stained in fresh DPBS for 15 min, at 37°C, protected from light. After de-staining, samples were again washed twice in DPBS, re-suspended in <em>Infection Medium</em>, counted using a Thoma chamber as described, and used for infection. To ensure that the SNARF-1 staining does not interfere with bacterial growth, growth curves using either SNARF-1 stained or unstained bacteria were performed in both TSB medium and RPMI1640. Infection was performed as described. At 3, 6 and 8h post infection, cells were washed, fixed with 4% PFA overnight at 4°C, treated with 50mM NH<sub>4</sub>Cl in DPBS for 5 min, permeabilized with 0.1% (v/v) Triton X-100 and nuclei were stained with 4 µg/mL Hoechst 34580. Images were acquired with an Operetta automated fluorescence microscopy system (Perkin Elmer Operetta High-Content Imaging System) and analysed with the built-in Harmony Software. Phagosomal escape was indicated by the co-localization of the YFP-CWT (<em>phagosomal escape</em>) and mRFP/SNARF signals (<em>total intracellular bacteria</em>).</p>
<p>Live cell imaging experiments shown in <a href="#ppat.1013291.s004" class="usa-link">S4 Fig</a> and <a href="#ppat.1013291.s018" class="usa-link">S2</a>–<a href="#ppat.1013291.s021" class="usa-link">S5 Movies</a> were performed as described for macrophages, using mRFP-expressing bacteria.</p></section><section id="sec034"><h3 class="pmc_sec_title">Cytotoxicity assay (LDH release)</h3>
<p>The cytotoxicity of intracellular bacterial strains against epithelial cells (<a href="#ppat.1013291.s005" class="usa-link">S5e Fig</a>) and the cytotoxicity of bacterial supernatants against HeLa cells (<a href="#ppat.1013291.s006" class="usa-link">S6a Fig</a>), were analysed by measuring the amount of released lactate dehydrogenase (LDH). At the required time points, spent medium of infected/treated cells (~2x10<sup>5</sup> cells/sample) was collected, centrifuged for 5 min at 100 x g and 100µl of supernatant (in technical duplicates) was used to determine LDH release using the Roche Cytotoxicity Detection Kit<sup>PLUS</sup> (Sigma Aldrich, Cat. No. 4744934001), according to manufacturer’s instructions. Spent medium of uninfected cells served as negative control and lysed cells served as positive control.</p></section><section id="sec035"><h3 class="pmc_sec_title">
<em>S. aureus</em> culture supernatant treatment of epithelial cells</h3>
<p>Bacteria were grown for 24h in TSB supplemented with the appropriate antibiotics, if necessary. Cultures were adjusted to an OD<sub>600</sub> = 4, in a final volume of 1 mL, centrifuged at 20,000 x g and supernatants were filtered through 0.22 µm-membrane filters. 2 x 10<sup>5</sup> seeded HeLa cells were treated with 10% (v/v) supernatants (in <em>Infection Medium</em>) and incubated for 4h at 37°C and 5% CO<sub>2</sub>. Cytotoxicity was measured by quantifying LDH released into the supernatant. 10% sterile TSB medium was used as negative control.</p></section><section id="sec036"><h3 class="pmc_sec_title">Iron quantification in bacterial pellets</h3>
<p>A colorimetric Iron Assay kit (Sigma Aldrich Cat. No. MAK025) was used to measure bacteria-associated iron. Overnight <em>S. aureus</em> JE2 cultures in RPMI1640 were transferred to RPMI1640 supplemented with 20 µM FeSO<sub>4</sub> (FeSO<sub>4</sub> heptahydrate dissolved in technical grade H<sub>2</sub>SO<sub>4</sub>), at OD<sub>600</sub> = 0.1 and grown for 24h at 37°C, under shaking at 200 rpm. OD<sub>600</sub> = 0.5 equivalents of bacterial cultures were pelleted by centrifugation and lysed for 30 min at 37°C in 110 µL <em>Lysis Buffer</em> (200 µg/mL lysostaphin, 1.2% Triton-X100, in water). Lysed samples were mixed with 110 µL Assay Buffer (kit) and incubated for 10 min at 80°C. Samples were subsequently centrifuged (13,000 x g, 10 min, 4°C) and 100 µL supernatant was used to measure total iron (Fe<sup>2+</sup> and Fe<sup>3+</sup>), according to manufacturer’s instructions.</p></section><section id="sec037"><h3 class="pmc_sec_title">
<em>Cerulean</em> induction in bacterial cultures</h3>
<p>Induction of the <em>Cerulean</em> fluorescent protein in <em>S. aureus</em> JE2 during growth in TSB medium was measured by monitoring fluorescence and OD<sub>600</sub>, in a TECAN MPlex plate reader, in black 96 well plates (Thermo Fisher Scientific, Cat. No. 165305). Overnight bacterial cultures were diluted to OD<sub>600</sub> = 0.1 in fresh TSB, supplemented with 300, 150 ng/mL aTc or vehicle control (ethanol). Absorbance at 600 nm and fluorescence (Ex/Em<sub>433/504 nm</sub>) were measured every 18 min for 16h. Results are expressed as arbitrary fluorescence units (Cerulean/OD<sub>600</sub> ratio).</p></section><section id="sec038"><h3 class="pmc_sec_title">Random unrestricted growth (<em>URG</em>)</h3>
<p>Bacterial unrestricted growth (<em>URG</em>) observed in infected macrophages, was analysed as follows: Primary human macrophages were seeded in 96-well plates at a density of 1.5 x 10<sup>4</sup> cells/well and infected with <em>S. aureus</em> WT or its isogenic <em>brn</em><em>Q1</em> mutant as described. Macrophages originating from a single donor were employed for each experiment to exclude donor effects on experimental outcome. Plates were observed daily and the number of wells, as well as the time point when <em>URG</em> occurred were recorded and plotted in a Kaplan-Meier survival plot, where each well represented a test subject. Wild-type <em>S. aureus</em>-infected macrophages and uninfected macrophages were used as positive (<em>URG</em> at 24h) and negative controls (no <em>URG</em>), respectively.</p></section><section id="sec039"><h3 class="pmc_sec_title">Assessment of secreted protein profiles of different <em>S. aureus</em> strains</h3>
<p>Single bacterial colonies were grown in TSB overnight. Cultures were diluted to 0.1, drop plated on SCMA plates and incubated at 37°C for 18-20h. Precipitation zones were measured with a standard metric ruler.</p></section><section id="sec040"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p>Data were analysed using GraphPad Prism Software (GraphPad Software, Version 10.2.3). For statistical analysis, at least three independent experiments were performed (sample size is indicated for each graph). All data are presented as means with standard deviation (±SD). P-values ≤0.05 were considered significant. When two groups were analysed, a diagnostic t-test was run first. If residuals were normally distributed (Shapiro-Wilk test), samples were analysed with an unpaired, two-tailed t-test. If residuals were not normally distributed, the non-parametric Mann-Whitney test (unpaired, two-tailed) was used instead.</p>
<p>When more than two groups were compared, a diagnostic one-way ANOVA was carried out first to check for normality of residuals (Shapiro-Wilk test), followed by Brown-Forsythe test to check for equal variance. If the ANOVA assumptions were met, the test was followed by an appropriate post-hoc test, as indicated for each graph. The Brown-Forsythe and Welch ANOVA test were used whenever the condition for equal variance was not met. If the ANOVA assumptions were not met, the non-parametric Kruskal-Wallis test, followed by Dunn’s multiple comparisons test was applied instead.</p>
<p>A two-way ANOVA was applied for time-course CFU experiments on log10-transformed CFU data, followed by an appropriate post-hoc test, as indicated for each graph.</p>
<p>No matching or pairing was assumed in any ANOVA test.</p></section></section><section id="sec041"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="ppat.1013291.s001"><div class="caption p">
<span>S1 Fig. <em>brnQ1</em> complementation <em>in trans</em> restores growth of <em>S. aureus</em> JE2 <em>brnQ1.</em></span><p>A functional copy of the <em>brnQ1</em> gene, under its native promoter region, (SAUSA300_0188/ SAUSA300_RS00985) was cloned into a high-copy plasmid and used for complementing JE2 <em>brnQ1</em> to yield JE2 <em>brnQ1</em> + p<em>brnQ1</em>. To exclude potential negative effects on bacterial fitness caused by gene overexpression from a high-copy plasmid, constructs carrying the <em>brnQ1</em> gene without promoter sequence were created (termed “P<sub>less</sub>”). <strong>(</strong><strong>a</strong><strong>)</strong> Effect of the complementation plasmid (p<em>brnQ1</em>) or promoter-less <em>brnQ1</em> plasmid (P<sub>less</sub>) on bacterial growth in a chemically defined medium with 1 mM of each BCAA (CDM). OD<sub>600</sub> was determined every 18 min, for 20h. Data are shown as mean values from independent experiments ± SD (n = 3). <strong>(</strong><strong>b</strong><strong>)</strong> Doubling times (T<sub>d</sub>) estimated from the growth curve. Dotted line corresponds to mean T<sub>d</sub> for JE2 WT. Statistical analysis: one-way ANOVA, with Dunnett’s multiple comparisons (vs JE2 WT)<em>; ns = not significant</em>; <em>**p &lt; 0.01.</em></p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s001.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s001.tif</a><sup> (338.5KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s002"><div class="caption p">
<span>S2 Fig. CFU numbers recovered from macrophages at 6h <em>p.i.</em> are comparable among the tested <em>S. aureus</em> strains.</span><p><strong>(a)</strong> Bacteria recovered from infected macrophages at 6h <em>p.i.</em> a Raw CFU counts (CFU/mL) plotted from data set shown in Fig 1d. Data are shown as mean ±SD from independent experiments (JE2 WT and JE2 <em>brn</em>Q1: n = 7, JE2 <em>brn</em>Q1 + p<em>brn</em>Q1 and Cowan I: n = 3). <strong>(</strong><strong>b</strong><strong>)</strong> CFU counts shown in a, normalised to “T0” (i.e., 45 min <em>p.i.</em>). Numbers inside bars represent the plotted value (mean). Statistical analysis (<strong>a</strong>): one-way ANOVA, with Dunnett’s multiple comparisons, vs JE2 WT; <em>ns = not significant</em>.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s002.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s002.tif</a><sup> (343KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s003"><div class="caption p">
<span>S3 Fig. <em>In vitro</em> growth of <em>S. aureus</em> JE2 <em>brnQ1</em> mutant in different culture media.</span><p><em>S. aureus</em> growth in <strong>(a)</strong> the nutrient-rich tryptic soy broth medium (TSB), <strong>(b)</strong> RPMI1640 and <strong>(c)</strong>
<em>Infection Medium</em> (RPMI1640 + 10% v/v heat inactivated FBS). OD<sub>600</sub> was measured every 18 min, for 20h. Data are shown as mean values ± SD from independent experiments: (a) JE2 WT and JE2 <em>brnQ1</em>: n = 5, JE2 <em>brnQ1</em> + p<em>brnQ1</em>: n = 2; (b) JE2 WT and JE2 <em>brnQ1</em>: n = 6, JE2 <em>brnQ1</em> + p<em>brnQ1</em>: n = 4; (c) JE2 WT and JE2 <em>brnQ1</em>: n = 3, JE2 <em>brnQ1</em> + p<em>brnQ1</em>: n = 1.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s003.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s003.tif</a><sup> (618.9KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s004"><div class="caption p">
<span>S4 Fig. Phagosomal escape of <em>S. aureus</em> JE2 <em>brnQ1</em> mutant in epithelial cells.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> Dynamics of <em>S. aureus</em> infection in professional phagocytes (e.g., macrophages) vs non-professional phagocytes (e.g., epithelial or endothelial cells). Phenol-soluble modulins (<em>psm</em>), the agr quorum sensing system (<em>agr</em>) and the non-ribosomal peptide synthetase AusAB (<em>ausAB</em>) were reported as bacterial factors involved in the phagosomal escape of <em>S. aureus</em> (modified after [<a href="#ppat.1013291.ref007" class="usa-link" aria-describedby="ppat.1013291.ref007">7</a>]). Live cell imaging of reporter <strong>(b)</strong> HeLa cells or <strong>(c)</strong> human bronchial epithelial cells 16HBE14o- constitutively expressing the phagosomal escape marker YFP-CWT, were infected with mRFP-expressing JE2 WT or JE2 <em>brnQ1</em>. Green arrows indicate phagosomal escape events and white arrows indicate intracellular bacterial replication. Shown are micrographs extracted from time-lapse series, representative of 3 independent experiments (n = 3). Scale bars = 10 µm (see also <a href="#ppat.1013291.s018" class="usa-link">S2</a>–<a href="#ppat.1013291.s021" class="usa-link">S5 Movies</a>). (<em>16HBE14o-: immortalized human bronchial epithelial cells</em>).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s004.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s004.tif</a><sup> (5.7MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s005"><div class="caption p">
<span>S5 Fig. <em>S. aureus brnQ1</em> mutants show delayed phagosomal escape and intracellular replication in non-professional phagocytes.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> Experimental procedure for epithelial cell infections: cells were infected with <em>S. aureus</em> at a multiplicity of infection (MOI) of 5 for 1h, following synchronization by centrifugation. Extracellular <em>S. aureus</em> were removed by a 30 min treatment with 20 µg/µL lysostaphin. Infected cells were further incubated in the presence of 2 µg/mL lysostaphin. “T0” corresponds to 1.5h <em>p.i.</em>
<strong>(</strong><strong>b</strong><strong>)</strong> Phagosomal escape rates of <em>S. aureus</em> JE2 WT and Nebraska Transposon Mutant Library (NTML) <em>S. aureus</em> mutants within several genes associated with either BCAA-uptake (<em>brnQ1-3</em> and <em>bcaP</em>) or biosynthesis (<em>ilvE, ilvD, leuA</em>) were assessed by automated fluorescence microscopy at 4h <em>p.i.,</em> in HeLa YFP-CWT phagosomal escape reporter cells. Phagosomal escape rates of the JE2 WT (percent YFP-CWT positive events of total internalized bacteria) were set to 100% (dotted line). <em>S. aureus</em> Cowan I was used as phagosomal escape-negative control. Data are shown as mean ± SD from independent experiments (n = 2). “NE” designates mutant strain identifier in the NTML. All bacterial strains express the mRFP fluorescent protein. <strong>(</strong><strong>c</strong><strong>)</strong><strong> </strong>Phagosomal escape was quantified in reporter HeLa or 16HBE14o<sup>-</sup> cells expressing the YFP-CWT phagosomal escape marker, by automated fluorescence microscopy, at different time-points, using SNARF-stained bacteria. Data represent the percent YFP-CWT positive events (<em>phagosomal escape</em>) of total internalized bacteria. Data are shown as mean ± SD from independent experiments (n &gt; 3). <strong>(</strong><strong>d</strong><strong>)</strong> Intracellular replication in HeLa and 16HBE14o- epithelial cells. Data are shown as means ± SD from independent experiments (n &gt; 3). <strong>(</strong><strong>e</strong><strong>)</strong> Cytotoxicity of <em>S. aureus</em> infection against HeLa and 16HBE14o- epithelial cells (% LDH release). Data are shown as means ± SD from independent experiments (n &gt; 3). Statistical analysis: (<strong>c</strong>) two-way ANOVA with Dunnett’s multiple comparisons test (each sample vs JE2 WT, for each time-point); (<strong>d</strong>) two-way ANOVA with Dunnett’s multiple comparisons test, using log10-transformed data (each sample vs JE2 WT, for each time-point); (<strong>e</strong>) Kruskal-Wallis test with Dunn’s multiple comparisons test (JE2 WT vs <em>brnQ1</em>, for each time-point); <em>ns = not significant; *p &lt; 0.05; **p &lt; 0.01; ***p &lt; 0.01; ****p &lt; 0.0001</em>. (<em>16HBE14o-: immortalized human bronchial epithelial cells</em>). (<strong>a</strong>) contains icons which were modified (colour and design) from <a href="https://bioicons.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bioicons.com/</a>, as follows: petri-dish-lid-yellow icon by Servier <a href="https://smart.servier.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://smart.servier.com/</a> is licensed under CC-BY 3.0 Unported <a href="https://creativecommons.org/licenses/by/3.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://creativecommons.org/licenses/by/3.0/</a>.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s005.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s005.tif</a><sup> (2MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s006"><div class="caption p">
<span>S6 Fig. Supernatants of <em>S. aureus</em> JE2 <em>brnQ1</em> grown in TSB are cytotoxic to HeLa cells and JE2 <em>brnQ1</em> exhibits normal haemolysis on sheep blood agar.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> Treatment with 10% sterile stationary phase supernatant (24h, TSB) and its effect on cytotoxicity (% LDH release) against HeLa cells (4h of treatment). <em>S. aureus</em> Cowan I is not cytotoxic against epithelial cells and serves as control. Data are shown as mean values ± SD of independent experiments (JE2 WT and JE2 <em>brnQ1</em>: n = 4, JE2 p<em>brnQ1</em> + p<em>brnQ1</em> and Cowan I: n = 3). Statistical analysis: one-way ANOVA, with Dunnett’s multiple comparisons test, vs JE2 WT; <em>ns = not significant; #=below detection limit, set to “0”</em>. <strong>(</strong><strong>b, c</strong><strong>)</strong><strong> </strong><em>S. aureus</em> JE2 haemolysis on Columbia agar with 5% defibrinated sheep blood, for 30h, at 37°C: <strong>(b)</strong> JE2 WT and <strong>(c)</strong> JE2 <em>brnQ1</em>. Representative image is shown (n &gt; 3).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s006.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s006.tif</a><sup> (2.8MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s007"><div class="caption p">
<span>S7 Fig. Dual RNA-seq of <em>S. aureus</em> JE2<em>-</em> infected macrophages.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> Principal component analysis (PCA) of host transcripts and <strong>(b)</strong> intracellular bacterial transcripts from dual RNA-seq, from 3 independent experiments (n = 3) (<em>hMDM: primary human monocyte-derived macrophages</em>).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s007.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s007.tif</a><sup> (287.7KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s008"><div class="caption p">
<span>S8 Fig. Visualization of major <em>S. aureus</em> regulons that are differentially regulated in intracellular <em>S. aureus</em> JE2 <em>brnQ1</em> compared to JE2 WT.</span><p>Displayed squares represent a gene of the indicated regulon and box coloration was assigned according to the heat map in the lower right (red: upregulated in JE2 <em>brnQ1</em> compared to JE2 WT; blue: downregulated in JE2 <em>brnQ1</em> compared to JE2 WT). Gene-boxes with black frames indicate significance of differential gene expression (padj&lt; 0.05) and absolute values of log2 fold-changes &gt;1.5 or &lt;-1.5. Genes regulated upon bacterial stringent response, such as genes coding for ribosomal proteins or tRNAs, were added to the analysis (framed gene set in the upper right). Gene identifiers correspond to USA300_FPR3757 (old locus ID). Regulons were retrieved from RegPrecise.lbl.gov.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s008.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s008.tif</a><sup> (3.8MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s009"><div class="caption p">
<span>S9 Fig. The <em>S. aureus-</em> containing phagosome is an iron deplete environment.</span><p>Transcriptomes of intracellular JE2 WT in primary human macrophages at 10h <em>p.i.</em> compared to bacteria grown in <em>Infection Medium</em> for 10h. Horizontal lines represent a p-value cutoff &lt; 0.05. Vertical lines represent cutoffs of log<sub>2</sub>fold changes of either &lt;-1.5 or &gt; 1.5. Experiment was run in parallel with experiments shown in Fig 2a-b. Data from Fig 2b were used for comparison and expressed as “intracellular vs <em>Infection Medium”</em> using DESeq2. Transcripts of iron uptake systems, which are prominently differentially regulated in intracellular bacteria vs <em>Infection Medium,</em> are depicted in green. Data are shown from 3 independent experiments (n = 3).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s009.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s009.tif</a><sup> (472.2KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s010"><div class="caption p">
<span>S10 Fig. During growth under iron-replete conditions, <em>S. aureus</em> JE2 <em>fur</em> and JE2 <em>brnQ1</em>/<em>fur</em> upregulate iron uptake mechanisms.</span><p>Total iron (Fe<sup>2+</sup> and Fe<sup>3+</sup>) measured in bacteria grown for 24h in RPMI1640 medium supplemented with 20 µM Fe<sup>2+</sup> (provided as FeSO<sub>4</sub>). Data are shown as mean values ±SD of independent experiments (n = 2).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s010.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s010.tif</a><sup> (174.7KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s011"><div class="caption p">
<span>S11 Fig. <em>Cerulean</em> is inducible by aTC in TSB medium in <em>S. aureus</em> JE2.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> JE2 WT and <strong>(b)</strong> JE2 <em>brnQ1</em> were grown in TSB medium supplemented with 300 ng/mL aTc (concentration used in infection experiments), 150 ng/mL or vehicle control (100% ethanol). OD<sub>600</sub> and fluorescence (Ex/Em <sub>433/504 nm</sub>) were measured every 18 min for 16h. Data are shown as mean values ±SD of independent experiments (n = 3). (AFU = arbitrary fluorescence units, expressed as <em>Cerulean</em>/OD<sub>600</sub> ratio).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s011.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s011.tif</a><sup> (898.1KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s012"><div class="caption p">
<span>S12 Fig. <em>Cerulean</em> signal quantification in intracellular <em>S. aureus</em> JE2.</span><p>Dot plots detailing the quantification of the percentage of <em>Cerulean-</em>expressing bacteria of all bacteria (labelled with Vancomycin BODIPY FL). Individual bacteria were identified by fluorescence microscopy and were labelled as regions -of-interest (ROI). For each ROI, fluorescence mean intensities in <em>Cerulean</em> and BODIPY-FL were measured and were plotted to visualize <em>Cerulean</em> expression of single bacteria after induction at 1h and 10h <em>p.i.</em></p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s012.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s012.tif</a><sup> (1.9MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s013"><div class="caption p">
<span>S13 Fig. Valine and leucine rescue the growth defect of <em>brnQ1</em>- deficient <em>S. aureus</em> in macrophages.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> <em>S. aureus</em> JE2 growth in a chemically defined media, either containing 1 mM of each BCAA (CDM) or lacking the stated amino acid(s). OD<sub>600</sub> was measured every 18 min, for 48h. Data are shown as mean values ±SD of independent experiments (n = 4). <strong>(</strong><strong>b</strong><strong>)</strong> Intracellular growth of a <em>S</em><em>. aureus brnQ1</em> in RAW 264.7 macrophages. Macrophages were infected with either WT USA300 or <em>S. aureus brnQ1</em> and, after gentamicin treatment, supplemented with either valine (Val), leucine (Leu), or Leu and Val (each at 1 mM final concentration) for the duration of the experiment. Data are shown as the mean log10 value ± S.D. for the calculated fold change in CFU/mL at 20h relative to 1.5h <em>p.i.</em>, for each bacterial strain. Each data point plotted represents a biological replicate derived from at least three independent experiments (n ≥ 3). Statistical analysis: Brown-Forsythe and Welch ANOVA with Dunnett’s T3 multiple comparison test. <strong>(</strong><strong>c</strong><strong>)</strong> Fluorescence microscopy micrographs of RAW 264.7 macrophages infected with <em>S. aureus</em> USA300 WT and <em>S. aureus brnQ1</em> mutant expressing GFP (green) that were labeled with a fluorescent proliferation dye (blue). The macrophage plasmalemma and extracellular cocci are stained with TMR-WGA (red). 1 mM Val and Leu (each) were added to the medium after gentamicin treatment (1.5h <em>p.i.</em>). At the outset (i.e., 1.5h) all bacteria are GFP and proliferation dye positive; over time and with replication, GFP-positive yet proliferation dye negative bacteria that are devoid of WGA can be seen at 10h and 20h <em>p.i</em>. White arrows indicate unrestricted bacterial replication (<em>URG</em>). Shown are representative micrographs, from 2 independent experiments (n = 2) (scale bars= ~ 10 µm).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s013.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s013.tif</a><sup> (787.6KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s014"><div class="caption p">
<span>S14 Fig. Inactivation of <em>codY</em> rescues the growth of <em>brnQ1-</em>deficient <em>S. aureus</em> in valine-free CDM.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> The ability of <em>S. aureus</em> USA300 WT, a <em>codY</em> mutant (<em>codY::</em>Tn), a <em>brnQ1</em> mutant, a <em>brnQ1/codY</em> double mutant (Δ<em>brnQ1/codY::Tn)</em> and a <em>brnQ1</em>/<em>codY/ilvD</em> triple mutant to grow in Val-free CDM are shown. <strong>(</strong><strong>b</strong><strong>)</strong><strong> </strong>CDM was supplemented with either 1mM Val or <strong>(c)</strong> 0.1% (v/v) tryptone (as a source of peptides) as indicated. Plotted data represent endpoint OD<sub>600</sub> readings, measured at 20h post-inoculation and the bars represent the mean ± SD for the indicated strains in each condition. Individual data points represent independent biological replicates, from three independent experiments (n = 3). Statistical analysis: Kruskal-Wallis with Dunn’s multiple comparisons test performed on the data deriving from each condition (vs USA300 WT); <em>**p &lt; 0.01, ns = not significant.</em></p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s014.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s014.tif</a><sup> (787.6KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s015"><div class="caption p">
<span>S15 Fig. Unrestricted growth <em>(URG)</em> of intracellular <em>S. aureus brnQ1</em> mutants during long-term infection of macrophages occurs in a stochastic manner.</span><p>Macrophages were generated from different donors (JE2 <em>brnQ1</em>: n = 4; 6850 <em>brnQ1</em>, SH1000 <em>brnQ1</em> and RN4220 <em>brnQ1</em>: n = 2) and seeded in well plates as technical replicates (at least 3 wells; JE2 <em>brnQ1</em> Donor #1: 72 wells; JE2 Donor #2: 80 wells). Infected plates were observed daily and the number of wells, as well as the time point when <em>URG</em> occurred were recorded and plotted in a Kaplan-Meier survival curve. Each well represents in this case a <em>test subject</em>. Wells infected with the parental wild-type strain served as positive controls.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s015.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s015.tif</a><sup> (925.1KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s016"><div class="caption p">
<span>S16 Fig. Protease production is increased in quadruple <em>brnQ1/bcaP/codY/ilvD</em> mutants, yet they fail to overcome macrophage restriction.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> Quantitation of the zones of precipitation around the bacteria grown on SCMA plates for the detection of proteolytic activity shown in <strong>(b)</strong>. Data are shown as mean values ± SD, from independent experiments (n &gt; 3). Statistical analysis: Kruskal-Wallis test with Dunn’s multiple comparisons test; <em>**p &lt; 0.01, ***p &lt; 0.001, ns = not significant.</em> (<em>codY = USA300 codY::Tn; ΔbrnQ1/ΔbcaP = USA300 double deletion mutant of brnQ1 and bcaP; ΔbrnQ1/ΔbcaP/codY</em> = USA300 triple mutant <em>ΔbrnQ1/ΔbcaP/codY::Tn;</em> Δ<em>brnQ1/</em>Δ<em>bcaP/codY/ilvD</em> = <em>USA300 quadruple mutant ΔbrnQ1/ΔbcaP/codY::Tn/ilvD::Tn-Kan</em><sup><em>R</em></sup><em>).</em> Asterisks (magenta) designate strains where <em>unrestricted growth</em> (<em>URG</em>) in macrophages was observed (data shown in <a href="#ppat.1013291.g001" class="usa-link">Fig 1c</a> and <a href="#ppat.1013291.g004" class="usa-link">Fig 4e</a>).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s016.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s016.tif</a><sup> (1.4MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s017"><div class="caption p">
<span>S1 Movie. Live cell imaging of <em>S. aureus</em>-infected primary macrophages. mRFP-expressing <em>S. aureus</em> JE2 WT or JE2 <em>brnQ1</em> (red). Host cell death (green fluorescence) is detected by CellEvent Caspase-3/7 Green Detection Reagent. Scale bar=100 µm.</span><p>(MP4)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s017.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (11MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s018"><div class="caption p">
<span>S2 Movie. Phagosomal escape of <em>S. aureus</em> JE2 WT in HeLa cells.</span><p>(red: <em>S. aureus</em>; yellow signal accumulation or ring-like structures: phagosomal escape marker). Shown is a time-lapse movie, representative of 3 independent experiments (n = 3). White frame highlights a region of interest depicted in <a href="#ppat.1013291.s004" class="usa-link">S4b Fig</a>. Scale bar = 50 µm.</p>
<p>(MP4)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s018.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (5MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s019"><div class="caption p">
<span>S3 Movie. Phagosomal escape of <em>S. aureus</em> JE2 <em>brnQ1</em> mutants in HeLa cells.</span><p>(red: S. <em>aureus</em>; yellow signal accumulation or ring-like structures: phagosomal escape marker). Shown is a time-lapse movie, representative of 3 independent experiments (n = 3). White frame highlights a region of interest depicted in <a href="#ppat.1013291.s004" class="usa-link">S4b Fig</a>. Scale bar = 50 µm.</p>
<p>(MP4)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s019.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (5.3MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s020"><div class="caption p">
<span>S4 Movie. Phagosomal escape of <em>S. aureus</em> JE2 WT in bronchial epithelial cells.</span><p>Live cell imaging of reporter 16HBE14o- cells constitutively expressing the phagosomal escape marker YFP-CWT, infected with mRFP-expressing <em>S. aureus</em> JE2 WT. (red: <em>S. aureus</em>; yellow signal accumulation or ring-like structures: phagosomal escape marker). Shown is a time-lapse movie, representative of 3 independent experiments (n = 3). Scale bar = 50 µm.</p>
<p>(MP4)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s020.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (3.1MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s021"><div class="caption p">
<span>S5 Movie. Phagosomal escape of <em>S. aureus</em> JE2 <em>brnQ1</em> mutants in bronchial epithelial cells.</span><p>Live cell imaging of reporter 16HBE14o- cells constitutively expressing the phagosomal escape marker YFP-CWT, infected with mRFP-expressing <em>S. aureus</em>
<em>brnQ1</em>. (red: <em>S. aureus</em>; yellow signal accumulation or ring-like structures: phagosomal escape marker). Shown is a time-lapse movie, representative of 3 independent experiments (n = 3). Scale bar = 50 µm.</p>
<p>(MP4)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s021.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (4.6MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s022"><div class="caption p">
<span>S1 Dataset. Expression profiling by high throughput sequencing.</span><p><em>S. aureus</em> JE2 wild-type (WT) or its isogenic JE2 <em>brnQ1</em> mutant were either used to infect primary human macrophages for 10h or were cultivated in <em>Infection Medium</em> (RPMI1640 + 10% v/v heat-inactivated FBS) for 10h. RNA of both pathogen and host cells were isolated together (dual RNA-Seq of infected macrophages) or isolated from planktonic bacteria (<em>Infection Medium</em>) after 10h of infection/cultivation. DESeq2 was used to analyse the following conditions: host transcripts in human macrophages infected with JE2 <em>brnQ1</em> compared to infected with JE2 WT (<em>HUMAN-ic_Mut_vs_ic_WT</em>); bacterial transcripts for intracellular JE2 <em>brnQ1</em> compared to intracellular JE2 WT (<em>BACT-ic-Mut_vs_ic_WT</em>); bacterial transcripts in intracellular bacteria compared to <em>Infection Medium</em> for JE2 <em>brnQ1</em> (<em>BACT-ic_Mut_vs_RPMI_Mut</em>) or JE2 WT (<em>BACT-ic_Mut_vs_RPMI_WT</em>) (<em>HUMAN: human transcripts; BACT: bacterial transcripts; Mut: JE2 brnQ1; WT: JE2 WT; ic: intracellular; RPMI: Infection Medium</em>).</p>
<p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s022.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s022.xlsx</a><sup> (13.9MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s023"><div class="caption p">
<span>S1 Table. Bacterial strains used in this study.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s023.pdf" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s023.pdf</a><sup> (303.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s024"><div class="caption p">
<span>S2 Table. Plasmids used in this study.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s024.pdf" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s024.pdf</a><sup> (183.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s025"><div class="caption p">
<span>S3 Table. Oligonucleotides used in this study.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s025.pdf" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s025.pdf</a><sup> (168.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="ppat.1013291.s026"><div class="caption p">
<span>S1 Text. Additional information detailing the role of the BrnQ1 transporter in the interaction of <em>S. aureus</em> with epithelial cells (refers to data shown in S4 and S5 Figs and S2-S5 Movies).</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s026.pdf" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s026.pdf</a><sup> (196.8KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The Leica TCS SP5 CLSM, the Perkin Elmer Operetta High-Content Imaging System and the JEOL JEM-2100 TEM were funded by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under project codes 116162193, 237502929 and 218894163, respectively.</p>
<p>RNA-seq was performed by the Core Unit SysMed at the University of Würzburg and was supported by the IZKF at the University of Würzburg (project Z-6).</p>
<p>We thank Jürgen Fritsch (University Hospital of Regensburg, Department of Infection Prevention and Infectious Diseases) for genome sequencing. We thank Simon Foster (University of Sheffield) for providing the <em>S. aureus</em> MJH010 <em>fur</em>::tet strain. We thank Claudia Gehrig-Höhn and Daniela Bunsen for support in transmission electron microscopy sample processing. We thank Nadine Vollmuth for valuable scientific input.</p></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>All relevant data are within the manuscript and its Supporting Information files. RNAseq have been deposited in NCBI's Gene Expression Omnibus under GEO Series accession number GSE235929 (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235929" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235929</a>).</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This work was supported by the Deutsche Forschungsgemeinschaft (German Research Foundation DFG, <a href="https://www.dfg.de" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.dfg.de</a>; RTG 2157 to T.R., RTG 2581 to M.F., CRC1583 to M.F.; DFG RU 631/17-1 to T.R.) and the Canadian Institutes of Health Research (<a href="https://cihr-irsc.gc.ca" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cihr-irsc.gc.ca</a>; PJT-183848 to D.E.H.). A. M. was partially supported by a grant of the German Excellence Initiative to the Graduate School of Life Sciences (GSLS; <a href="https://www.graduateschools.uni-wuerzburg.de/life-sciences/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.graduateschools.uni-wuerzburg.de/life-sciences/</a>), University of Würzburg and the DAAD STIBET Fellowship of the University of Würzburg. M.R. received funds of the Bavarian State Ministry of Science and Arts and the University of Würzburg to the Graduate School of Life Sciences (GSLS), University of Würzburg. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="ppat.1013291.ref001">
<span class="label">1.</span><cite>Proctor RA. Have we outlived the concept of commensalism for staphylococcus aureus?. Clin Infect Dis. 2021;73(1):e267–9. doi: 10.1093/cid/ciaa1431

</cite> [<a href="https://doi.org/10.1093/cid/ciaa1431" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32971535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&amp;title=Have%20we%20outlived%20the%20concept%20of%20commensalism%20for%20staphylococcus%20aureus?&amp;author=RA%20Proctor&amp;volume=73&amp;issue=1&amp;publication_year=2021&amp;pmid=32971535&amp;doi=10.1093/cid/ciaa1431&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref002">
<span class="label">2.</span><cite>Tong SYC, Davis JS, Eichenberger E, Holland TL, Fowler VGJ. Staphylococcus aureus infections: epidemiology, pathophysiology, clinical manifestations, and management. Clin Microbiol Rev. 2015;28(3):603–61.
</cite> [<a href="https://doi.org/10.1128/CMR.00134-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4451395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26016486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Microbiol%20Rev&amp;title=Staphylococcus%20aureus%20infections:%20epidemiology,%20pathophysiology,%20clinical%20manifestations,%20and%20management&amp;author=SYC%20Tong&amp;author=JS%20Davis&amp;author=E%20Eichenberger&amp;author=TL%20Holland&amp;author=VGJ%20Fowler&amp;volume=28&amp;issue=3&amp;publication_year=2015&amp;pages=603-61&amp;pmid=26016486&amp;doi=10.1128/CMR.00134-14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref003">
<span class="label">3.</span><cite>Brown AF, Leech JM, Rogers TR, McLoughlin RM. Staphylococcus aureus colonization: modulation of host immune response and impact on human vaccine design. Front Immunol. 2014;4:507. doi: 10.3389/fimmu.2013.00507

</cite> [<a href="https://doi.org/10.3389/fimmu.2013.00507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3884195/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24409186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=Staphylococcus%20aureus%20colonization:%20modulation%20of%20host%20immune%20response%20and%20impact%20on%20human%20vaccine%20design&amp;author=AF%20Brown&amp;author=JM%20Leech&amp;author=TR%20Rogers&amp;author=RM%20McLoughlin&amp;volume=4&amp;publication_year=2014&amp;pages=507&amp;pmid=24409186&amp;doi=10.3389/fimmu.2013.00507&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref004">
<span class="label">4.</span><cite>Wertheim HF, Melles DC, Vos MC, van Leeuwen W, van Belkum A, Verbrugh HA, et al. The role of nasal carriage in Staphylococcus aureus infections. The Lancet Infectious Diseases. 2005;5(12):751–62. doi: 10.1016/s1473-3099(05)70295-4
</cite> [<a href="https://doi.org/10.1016/s1473-3099(05)70295-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16310147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Lancet%20Infectious%20Diseases&amp;title=The%20role%20of%20nasal%20carriage%20in%20Staphylococcus%20aureus%20infections&amp;author=HF%20Wertheim&amp;author=DC%20Melles&amp;author=MC%20Vos&amp;author=W%20van%20Leeuwen&amp;author=A%20van%20Belkum&amp;volume=5&amp;issue=12&amp;publication_year=2005&amp;pages=751-62&amp;pmid=16310147&amp;doi=10.1016/s1473-3099(05)70295-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref005">
<span class="label">5.</span><cite>Kluytmans J, van Belkum A, Verbrugh H. Nasal carriage of Staphylococcus aureus: epidemiology, underlying mechanisms, and associated risks. Clin Microbiol Rev. 1997;10(3):505–20.
</cite> [<a href="https://doi.org/10.1128/cmr.10.3.505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC172932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9227864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Microbiol%20Rev&amp;title=Nasal%20carriage%20of%20Staphylococcus%20aureus:%20epidemiology,%20underlying%20mechanisms,%20and%20associated%20risks&amp;author=J%20Kluytmans&amp;author=A%20van%20Belkum&amp;author=H%20Verbrugh&amp;volume=10&amp;issue=3&amp;publication_year=1997&amp;pages=505-20&amp;pmid=9227864&amp;doi=10.1128/cmr.10.3.505&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref006">
<span class="label">6.</span><cite>Horn J, Stelzner K, Rudel T, Fraunholz M. Inside job: Staphylococcus aureus host-pathogen interactions. Int J Med Microbiol. 2018;308(6):607–24. doi: 10.1016/j.ijmm.2017.11.009

</cite> [<a href="https://doi.org/10.1016/j.ijmm.2017.11.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29217333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Med%20Microbiol&amp;title=Inside%20job:%20Staphylococcus%20aureus%20host-pathogen%20interactions&amp;author=J%20Horn&amp;author=K%20Stelzner&amp;author=T%20Rudel&amp;author=M%20Fraunholz&amp;volume=308&amp;issue=6&amp;publication_year=2018&amp;pages=607-24&amp;pmid=29217333&amp;doi=10.1016/j.ijmm.2017.11.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref007">
<span class="label">7.</span><cite>Moldovan A, Fraunholz MJ. In or out: Phagosomal escape of Staphylococcus aureus. Cell Microbiol. 2019;21(3):e12997. doi: 10.1111/cmi.12997

</cite> [<a href="https://doi.org/10.1111/cmi.12997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30576050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Microbiol&amp;title=In%20or%20out:%20Phagosomal%20escape%20of%20Staphylococcus%20aureus&amp;author=A%20Moldovan&amp;author=MJ%20Fraunholz&amp;volume=21&amp;issue=3&amp;publication_year=2019&amp;pmid=30576050&amp;doi=10.1111/cmi.12997&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref008">
<span class="label">8.</span><cite>Rodrigues Lopes I, Alcantara LM, Silva RJ, Josse J, Vega EP, Cabrerizo AM, et al. Microscopy-based phenotypic profiling of infection by Staphylococcus aureus clinical isolates reveals intracellular lifestyle as a prevalent feature. Nat Commun. 2022;13(1):7174. doi: 10.1038/s41467-022-34790-9

</cite> [<a href="https://doi.org/10.1038/s41467-022-34790-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9684519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36418309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Microscopy-based%20phenotypic%20profiling%20of%20infection%20by%20Staphylococcus%20aureus%20clinical%20isolates%20reveals%20intracellular%20lifestyle%20as%20a%20prevalent%20feature&amp;author=I%20Rodrigues%20Lopes&amp;author=LM%20Alcantara&amp;author=RJ%20Silva&amp;author=J%20Josse&amp;author=EP%20Vega&amp;volume=13&amp;issue=1&amp;publication_year=2022&amp;pages=7174&amp;pmid=36418309&amp;doi=10.1038/s41467-022-34790-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref009">
<span class="label">9.</span><cite>Flannagan RS, Heit B, Heinrichs DE. Antimicrobial mechanisms of macrophages and the immune evasion strategies of staphylococcus aureus. Pathogens. 2015;4(4):826–68. doi: 10.3390/pathogens4040826

</cite> [<a href="https://doi.org/10.3390/pathogens4040826" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4693167/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26633519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pathogens&amp;title=Antimicrobial%20mechanisms%20of%20macrophages%20and%20the%20immune%20evasion%20strategies%20of%20staphylococcus%20aureus&amp;author=RS%20Flannagan&amp;author=B%20Heit&amp;author=DE%20Heinrichs&amp;volume=4&amp;issue=4&amp;publication_year=2015&amp;pages=826-68&amp;pmid=26633519&amp;doi=10.3390/pathogens4040826&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref010">
<span class="label">10.</span><cite>Murdoch CC, Skaar EP. Nutritional immunity: the battle for nutrient metals at the host-pathogen interface. Nat Rev Microbiol. 2022;20(11):657–70. doi: 10.1038/s41579-022-00745-6

</cite> [<a href="https://doi.org/10.1038/s41579-022-00745-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9153222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35641670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Microbiol&amp;title=Nutritional%20immunity:%20the%20battle%20for%20nutrient%20metals%20at%20the%20host-pathogen%20interface&amp;author=CC%20Murdoch&amp;author=EP%20Skaar&amp;volume=20&amp;issue=11&amp;publication_year=2022&amp;pages=657-70&amp;pmid=35641670&amp;doi=10.1038/s41579-022-00745-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref011">
<span class="label">11.</span><cite>Hu M, Zhou N, Cai W, Xu H. Lysosomal solute and water transport. J Cell Biol. 2022;221(11):e202109133. doi: 10.1083/jcb.202109133

</cite> [<a href="https://doi.org/10.1083/jcb.202109133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9559593/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36219209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&amp;title=Lysosomal%20solute%20and%20water%20transport&amp;author=M%20Hu&amp;author=N%20Zhou&amp;author=W%20Cai&amp;author=H%20Xu&amp;volume=221&amp;issue=11&amp;publication_year=2022&amp;pmid=36219209&amp;doi=10.1083/jcb.202109133&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref012">
<span class="label">12.</span><cite>Flannagan RS, Heit B, Heinrichs DE. Intracellular replication of Staphylococcus aureus in mature phagolysosomes in macrophages precedes host cell death, and bacterial escape and dissemination. Cell Microbiol. 2016;18(4):514–35. doi: 10.1111/cmi.12527

</cite> [<a href="https://doi.org/10.1111/cmi.12527" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26408990/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Microbiol&amp;title=Intracellular%20replication%20of%20Staphylococcus%20aureus%20in%20mature%20phagolysosomes%20in%20macrophages%20precedes%20host%20cell%20death,%20and%20bacterial%20escape%20and%20dissemination&amp;author=RS%20Flannagan&amp;author=B%20Heit&amp;author=DE%20Heinrichs&amp;volume=18&amp;issue=4&amp;publication_year=2016&amp;pages=514-35&amp;pmid=26408990&amp;doi=10.1111/cmi.12527&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref013">
<span class="label">13.</span><cite>Kubica M, Guzik K, Koziel J, Zarebski M, Richter W, Gajkowska B, et al. A potential new pathway for Staphylococcus aureus dissemination: the silent survival of S. aureus phagocytosed by human monocyte-derived macrophages. PLoS One. 2008;3(1):e1409. doi: 10.1371/journal.pone.0001409

</cite> [<a href="https://doi.org/10.1371/journal.pone.0001409" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2169301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18183290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=A%20potential%20new%20pathway%20for%20Staphylococcus%20aureus%20dissemination:%20the%20silent%20survival%20of%20S.%20aureus%20phagocytosed%20by%20human%20monocyte-derived%20macrophages&amp;author=M%20Kubica&amp;author=K%20Guzik&amp;author=J%20Koziel&amp;author=M%20Zarebski&amp;author=W%20Richter&amp;volume=3&amp;issue=1&amp;publication_year=2008&amp;pmid=18183290&amp;doi=10.1371/journal.pone.0001409&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref014">
<span class="label">14.</span><cite>Lehar SM, Pillow T, Xu M, Staben L, Kajihara KK, Vandlen R, et al. Novel antibody-antibiotic conjugate eliminates intracellular S. aureus. Nature. 2015;527(7578):323–8. doi: 10.1038/nature16057

</cite> [<a href="https://doi.org/10.1038/nature16057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26536114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Novel%20antibody-antibiotic%20conjugate%20eliminates%20intracellular%20S.%20aureus&amp;author=SM%20Lehar&amp;author=T%20Pillow&amp;author=M%20Xu&amp;author=L%20Staben&amp;author=KK%20Kajihara&amp;volume=527&amp;issue=7578&amp;publication_year=2015&amp;pages=323-8&amp;pmid=26536114&amp;doi=10.1038/nature16057&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref015">
<span class="label">15.</span><cite>Pidwill GR, Pyrah JF, Sutton JAF, Best A, Renshaw SA, Foster SJ. Clonal population expansion of Staphylococcus aureus occurs due to escape from a finite number of intraphagocyte niches. Sci Rep. 2023;13(1):1188. doi: 10.1038/s41598-023-27928-2

</cite> [<a href="https://doi.org/10.1038/s41598-023-27928-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9867732/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36681703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Clonal%20population%20expansion%20of%20Staphylococcus%20aureus%20occurs%20due%20to%20escape%20from%20a%20finite%20number%20of%20intraphagocyte%20niches&amp;author=GR%20Pidwill&amp;author=JF%20Pyrah&amp;author=JAF%20Sutton&amp;author=A%20Best&amp;author=SA%20Renshaw&amp;volume=13&amp;issue=1&amp;publication_year=2023&amp;pages=1188&amp;pmid=36681703&amp;doi=10.1038/s41598-023-27928-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref016">
<span class="label">16.</span><cite>Flannagan RS, Cosío G, Grinstein S. Antimicrobial mechanisms of phagocytes and bacterial evasion strategies. Nat Rev Microbiol. 2009;7(5):355–66. doi: 10.1038/nrmicro2128

</cite> [<a href="https://doi.org/10.1038/nrmicro2128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19369951/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Microbiol&amp;title=Antimicrobial%20mechanisms%20of%20phagocytes%20and%20bacterial%20evasion%20strategies&amp;author=RS%20Flannagan&amp;author=G%20Cos%C3%ADo&amp;author=S%20Grinstein&amp;volume=7&amp;issue=5&amp;publication_year=2009&amp;pages=355-66&amp;pmid=19369951&amp;doi=10.1038/nrmicro2128&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref017">
<span class="label">17.</span><cite>Surewaard BGJ, Deniset JF, Zemp FJ, Amrein M, Otto M, Conly J, et al. Identification and treatment of the Staphylococcus aureus reservoir in vivo. J Exp Med. 2016;213(7):1141–51. doi: 10.1084/jem.20160334

</cite> [<a href="https://doi.org/10.1084/jem.20160334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4925027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27325887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&amp;title=Identification%20and%20treatment%20of%20the%20Staphylococcus%20aureus%20reservoir%20in%20vivo&amp;author=BGJ%20Surewaard&amp;author=JF%20Deniset&amp;author=FJ%20Zemp&amp;author=M%20Amrein&amp;author=M%20Otto&amp;volume=213&amp;issue=7&amp;publication_year=2016&amp;pages=1141-51&amp;pmid=27325887&amp;doi=10.1084/jem.20160334&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref018">
<span class="label">18.</span><cite>Flannagan RS, Kuiack RC, McGavin MJ, Heinrichs DE. Staphylococcus aureus Uses the GraXRS Regulatory System To Sense and Adapt to the Acidified Phagolysosome in Macrophages. mBio. 2018;9(4):e01143-18. doi: 10.1128/mBio.01143-18

</cite> [<a href="https://doi.org/10.1128/mBio.01143-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6050959/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30018109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Staphylococcus%20aureus%20Uses%20the%20GraXRS%20Regulatory%20System%20To%20Sense%20and%20Adapt%20to%20the%20Acidified%20Phagolysosome%20in%20Macrophages&amp;author=RS%20Flannagan&amp;author=RC%20Kuiack&amp;author=MJ%20McGavin&amp;author=DE%20Heinrichs&amp;volume=9&amp;issue=4&amp;publication_year=2018&amp;pmid=30018109&amp;doi=10.1128/mBio.01143-18&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref019">
<span class="label">19.</span><cite>Singh VK, Hattangady DS, Giotis ES, Singh AK, Chamberlain NR, Stuart MK, et al. Insertional inactivation of branched-chain alpha-keto acid dehydrogenase in Staphylococcus aureus leads to decreased branched-chain membrane fatty acid content and increased susceptibility to certain stresses. Appl Environ Microbiol. 2008;74(19):5882–90. doi: 10.1128/AEM.00882-08

</cite> [<a href="https://doi.org/10.1128/AEM.00882-08" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2565972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18689519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Insertional%20inactivation%20of%20branched-chain%20alpha-keto%20acid%20dehydrogenase%20in%20Staphylococcus%20aureus%20leads%20to%20decreased%20branched-chain%20membrane%20fatty%20acid%20content%20and%20increased%20susceptibility%20to%20certain%20stresses&amp;author=VK%20Singh&amp;author=DS%20Hattangady&amp;author=ES%20Giotis&amp;author=AK%20Singh&amp;author=NR%20Chamberlain&amp;volume=74&amp;issue=19&amp;publication_year=2008&amp;pages=5882-90&amp;pmid=18689519&amp;doi=10.1128/AEM.00882-08&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref020">
<span class="label">20.</span><cite>Han A-R, Kang H-R, Son J, Kwon DH, Kim S, Lee WC, et al. The structure of the pleiotropic transcription regulator CodY provides insight into its GTP-sensing mechanism. Nucleic Acids Res. 2016;44(19):9483–93. doi: 10.1093/nar/gkw775

</cite> [<a href="https://doi.org/10.1093/nar/gkw775" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5100569/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27596595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20structure%20of%20the%20pleiotropic%20transcription%20regulator%20CodY%20provides%20insight%20into%20its%20GTP-sensing%20mechanism&amp;author=A-R%20Han&amp;author=H-R%20Kang&amp;author=J%20Son&amp;author=DH%20Kwon&amp;author=S%20Kim&amp;volume=44&amp;issue=19&amp;publication_year=2016&amp;pages=9483-93&amp;pmid=27596595&amp;doi=10.1093/nar/gkw775&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref021">
<span class="label">21.</span><cite>Kaiser JC, King AN, Grigg JC, Sheldon JR, Edgell DR, Murphy MEP, et al. Repression of branched-chain amino acid synthesis in Staphylococcus aureus is mediated by isoleucine via CodY, and by a leucine-rich attenuator peptide. PLoS Genet. 2018;14(1):e1007159. doi: 10.1371/journal.pgen.1007159

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1007159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5794164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29357354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&amp;title=Repression%20of%20branched-chain%20amino%20acid%20synthesis%20in%20Staphylococcus%20aureus%20is%20mediated%20by%20isoleucine%20via%20CodY,%20and%20by%20a%20leucine-rich%20attenuator%20peptide&amp;author=JC%20Kaiser&amp;author=AN%20King&amp;author=JC%20Grigg&amp;author=JR%20Sheldon&amp;author=DR%20Edgell&amp;volume=14&amp;issue=1&amp;publication_year=2018&amp;pmid=29357354&amp;doi=10.1371/journal.pgen.1007159&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref022">
<span class="label">22.</span><cite>Brinsmade SR. CodY, a master integrator of metabolism and virulence in Gram-positive bacteria. Curr Genet. 2017;63(3):417–25. doi: 10.1007/s00294-016-0656-5

</cite> [<a href="https://doi.org/10.1007/s00294-016-0656-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27744611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Genet&amp;title=CodY,%20a%20master%20integrator%20of%20metabolism%20and%20virulence%20in%20Gram-positive%20bacteria&amp;author=SR%20Brinsmade&amp;volume=63&amp;issue=3&amp;publication_year=2017&amp;pages=417-25&amp;pmid=27744611&amp;doi=10.1007/s00294-016-0656-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref023">
<span class="label">23.</span><cite>Kaiser JC, Heinrichs DE. Branching out: alterations in bacterial physiology and virulence due to branched-chain amino acid deprivation. mBio. 2018;9(5):e01188-18. doi: 10.1128/mBio.01188-18

</cite> [<a href="https://doi.org/10.1128/mBio.01188-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6123439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30181248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Branching%20out:%20alterations%20in%20bacterial%20physiology%20and%20virulence%20due%20to%20branched-chain%20amino%20acid%20deprivation&amp;author=JC%20Kaiser&amp;author=DE%20Heinrichs&amp;volume=9&amp;issue=5&amp;publication_year=2018&amp;pmid=30181248&amp;doi=10.1128/mBio.01188-18&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref024">
<span class="label">24.</span><cite>Kaiser JC, Omer S, Sheldon JR, Welch I, Heinrichs DE. Role of BrnQ1 and BrnQ2 in branched-chain amino acid transport and virulence in Staphylococcus aureus. Infect Immun. 2015;83(3):1019–29. doi: 10.1128/IAI.02542-14

</cite> [<a href="https://doi.org/10.1128/IAI.02542-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4333469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25547798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Immun&amp;title=Role%20of%20BrnQ1%20and%20BrnQ2%20in%20branched-chain%20amino%20acid%20transport%20and%20virulence%20in%20Staphylococcus%20aureus&amp;author=JC%20Kaiser&amp;author=S%20Omer&amp;author=JR%20Sheldon&amp;author=I%20Welch&amp;author=DE%20Heinrichs&amp;volume=83&amp;issue=3&amp;publication_year=2015&amp;pages=1019-29&amp;pmid=25547798&amp;doi=10.1128/IAI.02542-14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref025">
<span class="label">25.</span><cite>Majerczyk CD, Dunman PM, Luong TT, Lee CY, Sadykov MR, Somerville GA, et al. Direct targets of CodY in Staphylococcus aureus. J Bacteriol. 2010;192(11):2861–77. doi: 10.1128/JB.00220-10

</cite> [<a href="https://doi.org/10.1128/JB.00220-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2876493/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20363936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=Direct%20targets%20of%20CodY%20in%20Staphylococcus%20aureus&amp;author=CD%20Majerczyk&amp;author=PM%20Dunman&amp;author=TT%20Luong&amp;author=CY%20Lee&amp;author=MR%20Sadykov&amp;volume=192&amp;issue=11&amp;publication_year=2010&amp;pages=2861-77&amp;pmid=20363936&amp;doi=10.1128/JB.00220-10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref026">
<span class="label">26.</span><cite>Pohl K, Francois P, Stenz L, Schlink F, Geiger T, Herbert S, et al. CodY in Staphylococcus aureus: a regulatory link between metabolism and virulence gene expression. J Bacteriol. 2009;191(9):2953–63. doi: 10.1128/JB.01492-08

</cite> [<a href="https://doi.org/10.1128/JB.01492-08" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2681790/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19251851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=CodY%20in%20Staphylococcus%20aureus:%20a%20regulatory%20link%20between%20metabolism%20and%20virulence%20gene%20expression&amp;author=K%20Pohl&amp;author=P%20Francois&amp;author=L%20Stenz&amp;author=F%20Schlink&amp;author=T%20Geiger&amp;volume=191&amp;issue=9&amp;publication_year=2009&amp;pages=2953-63&amp;pmid=19251851&amp;doi=10.1128/JB.01492-08&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref027">
<span class="label">27.</span><cite>Geiger T, Francois P, Liebeke M, Fraunholz M, Goerke C, Krismer B, et al. The stringent response of Staphylococcus aureus and its impact on survival after phagocytosis through the induction of intracellular PSMs expression. PLoS Pathog. 2012;8(11):e1003016. doi: 10.1371/journal.ppat.1003016

</cite> [<a href="https://doi.org/10.1371/journal.ppat.1003016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3510239/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23209405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&amp;title=The%20stringent%20response%20of%20Staphylococcus%20aureus%20and%20its%20impact%20on%20survival%20after%20phagocytosis%20through%20the%20induction%20of%20intracellular%20PSMs%20expression&amp;author=T%20Geiger&amp;author=P%20Francois&amp;author=M%20Liebeke&amp;author=M%20Fraunholz&amp;author=C%20Goerke&amp;volume=8&amp;issue=11&amp;publication_year=2012&amp;pmid=23209405&amp;doi=10.1371/journal.ppat.1003016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref028">
<span class="label">28.</span><cite>Waters NR, Samuels DJ, Behera RK, Livny J, Rhee KY, Sadykov MR, et al. A spectrum of CodY activities drives metabolic reorganization and virulence gene expression in Staphylococcus aureus. Mol Microbiol. 2016;101(3):495–514. doi: 10.1111/mmi.13404

</cite> [<a href="https://doi.org/10.1111/mmi.13404" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5007081/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27116338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&amp;title=A%20spectrum%20of%20CodY%20activities%20drives%20metabolic%20reorganization%20and%20virulence%20gene%20expression%20in%20Staphylococcus%20aureus&amp;author=NR%20Waters&amp;author=DJ%20Samuels&amp;author=RK%20Behera&amp;author=J%20Livny&amp;author=KY%20Rhee&amp;volume=101&amp;issue=3&amp;publication_year=2016&amp;pages=495-514&amp;pmid=27116338&amp;doi=10.1111/mmi.13404&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref029">
<span class="label">29.</span><cite>Kaiser JC, Sen S, Sinha A, Wilkinson BJ, Heinrichs DE. The role of two branched-chain amino acid transporters in Staphylococcus aureus growth, membrane fatty acid composition and virulence. Mol Microbiol. 2016;102(5):850–64. doi: 10.1111/mmi.13495

</cite> [<a href="https://doi.org/10.1111/mmi.13495" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6225994/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27589208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&amp;title=The%20role%20of%20two%20branched-chain%20amino%20acid%20transporters%20in%20Staphylococcus%20aureus%20growth,%20membrane%20fatty%20acid%20composition%20and%20virulence&amp;author=JC%20Kaiser&amp;author=S%20Sen&amp;author=A%20Sinha&amp;author=BJ%20Wilkinson&amp;author=DE%20Heinrichs&amp;volume=102&amp;issue=5&amp;publication_year=2016&amp;pages=850-64&amp;pmid=27589208&amp;doi=10.1111/mmi.13495&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref030">
<span class="label">30.</span><cite>Fuchs S, Mehlan H, Bernhardt J, Hennig A, Michalik S, Surmann K, et al. AureoWiki - The repository of the Staphylococcus aureus research and annotation community. Int J Med Microbiol. 2018;308(6):558–68. doi: 10.1016/j.ijmm.2017.11.011

</cite> [<a href="https://doi.org/10.1016/j.ijmm.2017.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29198880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Med%20Microbiol&amp;title=AureoWiki%20-%20The%20repository%20of%20the%20Staphylococcus%20aureus%20research%20and%20annotation%20community&amp;author=S%20Fuchs&amp;author=H%20Mehlan&amp;author=J%20Bernhardt&amp;author=A%20Hennig&amp;author=S%20Michalik&amp;volume=308&amp;issue=6&amp;publication_year=2018&amp;pages=558-68&amp;pmid=29198880&amp;doi=10.1016/j.ijmm.2017.11.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref031">
<span class="label">31.</span><cite>Kim J-H, Chaurasia AK, Batool N, Ko KS, Kim KK. Alternative Enzyme Protection Assay To Overcome the Drawbacks of the Gentamicin Protection Assay for Measuring Entry and Intracellular Survival of Staphylococci. Infect Immun. 2019;87(5):e00119-19. doi: 10.1128/IAI.00119-19

</cite> [<a href="https://doi.org/10.1128/IAI.00119-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6479035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30782857/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Immun&amp;title=Alternative%20Enzyme%20Protection%20Assay%20To%20Overcome%20the%20Drawbacks%20of%20the%20Gentamicin%20Protection%20Assay%20for%20Measuring%20Entry%20and%20Intracellular%20Survival%20of%20Staphylococci&amp;author=J-H%20Kim&amp;author=AK%20Chaurasia&amp;author=N%20Batool&amp;author=KS%20Ko&amp;author=KK%20Kim&amp;volume=87&amp;issue=5&amp;publication_year=2019&amp;pmid=30782857&amp;doi=10.1128/IAI.00119-19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref032">
<span class="label">32.</span><cite>VanCleave TT, Pulsifer AR, Connor MG, Warawa JM, Lawrenz MB. Impact of Gentamicin Concentration and Exposure Time on Intracellular Yersinia pestis. Front Cell Infect Microbiol. 2017;7:505. doi: 10.3389/fcimb.2017.00505

</cite> [<a href="https://doi.org/10.3389/fcimb.2017.00505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5732358/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29312891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Infect%20Microbiol&amp;title=Impact%20of%20Gentamicin%20Concentration%20and%20Exposure%20Time%20on%20Intracellular%20Yersinia%20pestis&amp;author=TT%20VanCleave&amp;author=AR%20Pulsifer&amp;author=MG%20Connor&amp;author=JM%20Warawa&amp;author=MB%20Lawrenz&amp;volume=7&amp;publication_year=2017&amp;pages=505&amp;pmid=29312891&amp;doi=10.3389/fcimb.2017.00505&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref033">
<span class="label">33.</span><cite>Flannagan RS, Heinrichs DE. Macrophage-driven nutrient delivery to phagosomal Staphylococcus aureus supports bacterial growth. EMBO reports. 2020;21(8):e50348.</cite> [<a href="https://doi.org/10.15252/embr.202050348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7403714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32452129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20reports&amp;title=Macrophage-driven%20nutrient%20delivery%20to%20phagosomal%20Staphylococcus%20aureus%20supports%20bacterial%20growth&amp;author=RS%20Flannagan&amp;author=DE%20Heinrichs&amp;volume=21&amp;issue=8&amp;publication_year=2020&amp;pmid=32452129&amp;doi=10.15252/embr.202050348&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref034">
<span class="label">34.</span><cite>Fey PD, Endres JL, Yajjala VK, Widhelm TJ, Boissy RJ, Bose JL. A genetic resource for rapid and comprehensive phenotype screening of nonessential Staphylococcus aureus genes. mBio. 2013;4(1):e00537-12.</cite> [<a href="https://doi.org/10.1128/mBio.00537-12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3573662/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23404398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=A%20genetic%20resource%20for%20rapid%20and%20comprehensive%20phenotype%20screening%20of%20nonessential%20Staphylococcus%20aureus%20genes&amp;author=PD%20Fey&amp;author=JL%20Endres&amp;author=VK%20Yajjala&amp;author=TJ%20Widhelm&amp;author=RJ%20Boissy&amp;volume=4&amp;issue=1&amp;publication_year=2013&amp;pmid=23404398&amp;doi=10.1128/mBio.00537-12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref035">
<span class="label">35.</span><cite>Montgomery CP, Boyle-Vavra S, Roux A, Ebine K, Sonenshein AL, Daum RS. CodY deletion enhances in vivo virulence of community-associated methicillin-resistant Staphylococcus aureus clone USA300. Infect Immun. 2012;80(7):2382–9. doi: 10.1128/IAI.06172-11

</cite> [<a href="https://doi.org/10.1128/IAI.06172-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3416461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22526672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Immun&amp;title=CodY%20deletion%20enhances%20in%20vivo%20virulence%20of%20community-associated%20methicillin-resistant%20Staphylococcus%20aureus%20clone%20USA300&amp;author=CP%20Montgomery&amp;author=S%20Boyle-Vavra&amp;author=A%20Roux&amp;author=K%20Ebine&amp;author=AL%20Sonenshein&amp;volume=80&amp;issue=7&amp;publication_year=2012&amp;pages=2382-9&amp;pmid=22526672&amp;doi=10.1128/IAI.06172-11&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref036">
<span class="label">36.</span><cite>Stelzner K, Boyny A, Hertlein T, Sroka A, Moldovan A, Paprotka K, et al. Intracellular Staphylococcus aureus employs the cysteine protease staphopain A to induce host cell death in epithelial cells. PLoS Pathog. 2021;17(9):e1009874. doi: 10.1371/journal.ppat.1009874

</cite> [<a href="https://doi.org/10.1371/journal.ppat.1009874" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8443034/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34473800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&amp;title=Intracellular%20Staphylococcus%20aureus%20employs%20the%20cysteine%20protease%20staphopain%20A%20to%20induce%20host%20cell%20death%20in%20epithelial%20cells&amp;author=K%20Stelzner&amp;author=A%20Boyny&amp;author=T%20Hertlein&amp;author=A%20Sroka&amp;author=A%20Moldovan&amp;volume=17&amp;issue=9&amp;publication_year=2021&amp;pmid=34473800&amp;doi=10.1371/journal.ppat.1009874&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref037">
<span class="label">37.</span><cite>Hommes JW, Surewaard BGJ. Intracellular Habitation of Staphylococcus aureus: Molecular Mechanisms and Prospects for Antimicrobial Therapy. Biomedicines. 2022;10(8):1804. doi: 10.3390/biomedicines10081804

</cite> [<a href="https://doi.org/10.3390/biomedicines10081804" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9405036/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36009351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomedicines&amp;title=Intracellular%20Habitation%20of%20Staphylococcus%20aureus:%20Molecular%20Mechanisms%20and%20Prospects%20for%20Antimicrobial%20Therapy&amp;author=JW%20Hommes&amp;author=BGJ%20Surewaard&amp;volume=10&amp;issue=8&amp;publication_year=2022&amp;pages=1804&amp;pmid=36009351&amp;doi=10.3390/biomedicines10081804&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref038">
<span class="label">38.</span><cite>Lang JC, Seiß EA, Moldovan A, Müsken M, Sauerwein T, Fraunholz M, et al. A Photoconvertible Reporter System for Bacterial Metabolic Activity Reveals That Staphylococcus aureus Enters a Dormant-Like State to Persist within Macrophages. mBio. 2022;13(5):e0231622. doi: 10.1128/mbio.02316-22

</cite> [<a href="https://doi.org/10.1128/mbio.02316-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9600638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36102512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=A%20Photoconvertible%20Reporter%20System%20for%20Bacterial%20Metabolic%20Activity%20Reveals%20That%20Staphylococcus%20aureus%20Enters%20a%20Dormant-Like%20State%20to%20Persist%20within%20Macrophages&amp;author=JC%20Lang&amp;author=EA%20Sei%C3%9F&amp;author=A%20Moldovan&amp;author=M%20M%C3%BCsken&amp;author=T%20Sauerwein&amp;volume=13&amp;issue=5&amp;publication_year=2022&amp;pmid=36102512&amp;doi=10.1128/mbio.02316-22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref039">
<span class="label">39.</span><cite>Flannagan RS, Farrell TJ, Trothen SM, Dikeakos JD, Heinrichs DE. Rapid removal of phagosomal ferroportin in macrophages contributes to nutritional immunity. Blood Adv. 2021;5(2):459–74. doi: 10.1182/bloodadvances.2020002833

</cite> [<a href="https://doi.org/10.1182/bloodadvances.2020002833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7839378/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33496744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Blood%20Adv&amp;title=Rapid%20removal%20of%20phagosomal%20ferroportin%20in%20macrophages%20contributes%20to%20nutritional%20immunity&amp;author=RS%20Flannagan&amp;author=TJ%20Farrell&amp;author=SM%20Trothen&amp;author=JD%20Dikeakos&amp;author=DE%20Heinrichs&amp;volume=5&amp;issue=2&amp;publication_year=2021&amp;pages=459-74&amp;pmid=33496744&amp;doi=10.1182/bloodadvances.2020002833&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref040">
<span class="label">40.</span><cite>Horsburgh MJ, Ingham E, Foster SJ. In Staphylococcus aureus, fur is an interactive regulator with PerR, contributes to virulence, and Is necessary for oxidative stress resistance through positive regulation of catalase and iron homeostasis. J Bacteriol. 2001;183(2):468–75. doi: 10.1128/JB.183.2.468-475.2001

</cite> [<a href="https://doi.org/10.1128/JB.183.2.468-475.2001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC94901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11133939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=In%20Staphylococcus%20aureus,%20fur%20is%20an%20interactive%20regulator%20with%20PerR,%20contributes%20to%20virulence,%20and%20Is%20necessary%20for%20oxidative%20stress%20resistance%20through%20positive%20regulation%20of%20catalase%20and%20iron%20homeostasis&amp;author=MJ%20Horsburgh&amp;author=E%20Ingham&amp;author=SJ%20Foster&amp;volume=183&amp;issue=2&amp;publication_year=2001&amp;pages=468-75&amp;pmid=11133939&amp;doi=10.1128/JB.183.2.468-475.2001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref041">
<span class="label">41.</span><cite>Baichoo N, Wang T, Ye R, Helmann JD. Global analysis of the Bacillus subtilis Fur regulon and the iron starvation stimulon. Mol Microbiol. 2002;45(6):1613–29. doi: 10.1046/j.1365-2958.2002.03113.x

</cite> [<a href="https://doi.org/10.1046/j.1365-2958.2002.03113.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12354229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&amp;title=Global%20analysis%20of%20the%20Bacillus%20subtilis%20Fur%20regulon%20and%20the%20iron%20starvation%20stimulon&amp;author=N%20Baichoo&amp;author=T%20Wang&amp;author=R%20Ye&amp;author=JD%20Helmann&amp;volume=45&amp;issue=6&amp;publication_year=2002&amp;pages=1613-29&amp;pmid=12354229&amp;doi=10.1046/j.1365-2958.2002.03113.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref042">
<span class="label">42.</span><cite>Novichkov PS, Kazakov AE, Ravcheev DA, Leyn SA, Kovaleva GY, Sutormin RA. RegPrecise 3.0--a resource for genome-scale exploration of transcriptional regulation in bacteria. BMC Genomics. 2013;14:745.
</cite> [<a href="https://doi.org/10.1186/1471-2164-14-745" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3840689/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24175918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=RegPrecise%203.0--a%20resource%20for%20genome-scale%20exploration%20of%20transcriptional%20regulation%20in%20bacteria&amp;author=PS%20Novichkov&amp;author=AE%20Kazakov&amp;author=DA%20Ravcheev&amp;author=SA%20Leyn&amp;author=GY%20Kovaleva&amp;volume=14&amp;publication_year=2013&amp;pages=745&amp;pmid=24175918&amp;doi=10.1186/1471-2164-14-745&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref043">
<span class="label">43.</span><cite>Peyrusson F, Varet H, Nguyen TK, Legendre R, Sismeiro O, Coppée J-Y, et al. Intracellular Staphylococcus aureus persisters upon antibiotic exposure. Nat Commun. 2020;11(1):2200. doi: 10.1038/s41467-020-15966-7

</cite> [<a href="https://doi.org/10.1038/s41467-020-15966-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7198484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32366839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Intracellular%20Staphylococcus%20aureus%20persisters%20upon%20antibiotic%20exposure&amp;author=F%20Peyrusson&amp;author=H%20Varet&amp;author=TK%20Nguyen&amp;author=R%20Legendre&amp;author=O%20Sismeiro&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=2200&amp;pmid=32366839&amp;doi=10.1038/s41467-020-15966-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref044">
<span class="label">44.</span><cite>Gao Y, Poudel S, Seif Y, Shen Z, Palsson BO. Elucidating the CodY regulon in Staphylococcus aureus USA300 substrains TCH1516 and LAC. mSystems. 2023;8(4):e00279-23. doi: 10.1128/msystems.00279-23</cite> [<a href="https://doi.org/10.1128/msystems.00279-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10470025/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37310465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=Elucidating%20the%20CodY%20regulon%20in%20Staphylococcus%20aureus%20USA300%20substrains%20TCH1516%20and%20LAC&amp;author=Y%20Gao&amp;author=S%20Poudel&amp;author=Y%20Seif&amp;author=Z%20Shen&amp;author=BO%20Palsson&amp;volume=8&amp;issue=4&amp;publication_year=2023&amp;pmid=37310465&amp;doi=10.1128/msystems.00279-23&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref045">
<span class="label">45.</span><cite>Tuchscherr L, Medina E, Hussain M, Völker W, Heitmann V, Niemann S, et al. Staphylococcus aureus phenotype switching: an effective bacterial strategy to escape host immune response and establish a chronic infection. EMBO Mol Med. 2011;3(3):129–41. doi: 10.1002/emmm.201000115

</cite> [<a href="https://doi.org/10.1002/emmm.201000115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3395110/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21268281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Mol%20Med&amp;title=Staphylococcus%20aureus%20phenotype%20switching:%20an%20effective%20bacterial%20strategy%20to%20escape%20host%20immune%20response%20and%20establish%20a%20chronic%20infection&amp;author=L%20Tuchscherr&amp;author=E%20Medina&amp;author=M%20Hussain&amp;author=W%20V%C3%B6lker&amp;author=V%20Heitmann&amp;volume=3&amp;issue=3&amp;publication_year=2011&amp;pages=129-41&amp;pmid=21268281&amp;doi=10.1002/emmm.201000115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref046">
<span class="label">46.</span><cite>Palma Medina LM, Becker A-K, Michalik S, Yedavally H, Raineri EJM, Hildebrandt P, et al. Metabolic cross-talk between human bronchial epithelial cells and internalized staphylococcus aureus as a driver for infection. Mol Cell Proteomics. 2019;18(5):892–908. doi: 10.1074/mcp.RA118.001138

</cite> [<a href="https://doi.org/10.1074/mcp.RA118.001138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6495256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30808728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Proteomics&amp;title=Metabolic%20cross-talk%20between%20human%20bronchial%20epithelial%20cells%20and%20internalized%20staphylococcus%20aureus%20as%20a%20driver%20for%20infection&amp;author=LM%20Palma%20Medina&amp;author=A-K%20Becker&amp;author=S%20Michalik&amp;author=H%20Yedavally&amp;author=EJM%20Raineri&amp;volume=18&amp;issue=5&amp;publication_year=2019&amp;pages=892-908&amp;pmid=30808728&amp;doi=10.1074/mcp.RA118.001138&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref047">
<span class="label">47.</span><cite>Gunaratnam G, Tuchscherr L, Elhawy MI, Bertram R, Eisenbeis J, Spengler C, et al. ClpC affects the intracellular survival capacity of Staphylococcus aureus in non-professional phagocytic cells. Sci Rep. 2019;9(1):16267. doi: 10.1038/s41598-019-52731-3

</cite> [<a href="https://doi.org/10.1038/s41598-019-52731-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6838064/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31700127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=ClpC%20affects%20the%20intracellular%20survival%20capacity%20of%20Staphylococcus%20aureus%20in%20non-professional%20phagocytic%20cells&amp;author=G%20Gunaratnam&amp;author=L%20Tuchscherr&amp;author=MI%20Elhawy&amp;author=R%20Bertram&amp;author=J%20Eisenbeis&amp;volume=9&amp;issue=1&amp;publication_year=2019&amp;pages=16267&amp;pmid=31700127&amp;doi=10.1038/s41598-019-52731-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref048">
<span class="label">48.</span><cite>Rollin G, Tan X, Tros F, Dupuis M, Nassif X, Charbit A. Intracellular survival of Staphylococcus aureus in endothelial cells: A matter of growth or persistence. Frontiers in Microbiology. 2017;8:1354.
</cite> [<a href="https://doi.org/10.3389/fmicb.2017.01354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5515828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28769913/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Intracellular%20survival%20of%20Staphylococcus%20aureus%20in%20endothelial%20cells:%20A%20matter%20of%20growth%20or%20persistence&amp;author=G%20Rollin&amp;author=X%20Tan&amp;author=F%20Tros&amp;author=M%20Dupuis&amp;author=X%20Nassif&amp;volume=8&amp;publication_year=2017&amp;pages=1354&amp;pmid=28769913&amp;doi=10.3389/fmicb.2017.01354&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref049">
<span class="label">49.</span><cite>Jubrail J, Morris P, Bewley MA, Stoneham S, Johnston SA, Foster SJ. Inability to sustain intraphagolysosomal killing of Staphylococcus aureus predisposes to bacterial persistence in macrophages. Cell Microbiol. 2016;18(1):80–96.
</cite> [<a href="https://doi.org/10.1111/cmi.12485" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4778410/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26248337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Microbiol&amp;title=Inability%20to%20sustain%20intraphagolysosomal%20killing%20of%20Staphylococcus%20aureus%20predisposes%20to%20bacterial%20persistence%20in%20macrophages&amp;author=J%20Jubrail&amp;author=P%20Morris&amp;author=MA%20Bewley&amp;author=S%20Stoneham&amp;author=SA%20Johnston&amp;volume=18&amp;issue=1&amp;publication_year=2016&amp;pages=80-96&amp;pmid=26248337&amp;doi=10.1111/cmi.12485&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref050">
<span class="label">50.</span><cite>Balaban NQ, Merrin J, Chait R, Kowalik L, Leibler S. Bacterial persistence as a phenotypic switch. Science. 2004;305(5690):1622–5. doi: 10.1126/science.1099390

</cite> [<a href="https://doi.org/10.1126/science.1099390" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15308767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Bacterial%20persistence%20as%20a%20phenotypic%20switch&amp;author=NQ%20Balaban&amp;author=J%20Merrin&amp;author=R%20Chait&amp;author=L%20Kowalik&amp;author=S%20Leibler&amp;volume=305&amp;issue=5690&amp;publication_year=2004&amp;pages=1622-5&amp;pmid=15308767&amp;doi=10.1126/science.1099390&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref051">
<span class="label">51.</span><cite>Balaban NQ, Helaine S, Lewis K, Ackermann M, Aldridge B, Andersson DI, et al. Definitions and guidelines for research on antibiotic persistence. Nat Rev Microbiol. 2019;17(7):441–8.
</cite> [<a href="https://doi.org/10.1038/s41579-019-0196-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7136161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30980069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Microbiol&amp;title=Definitions%20and%20guidelines%20for%20research%20on%20antibiotic%20persistence&amp;author=NQ%20Balaban&amp;author=S%20Helaine&amp;author=K%20Lewis&amp;author=M%20Ackermann&amp;author=B%20Aldridge&amp;volume=17&amp;issue=7&amp;publication_year=2019&amp;pages=441-8&amp;pmid=30980069&amp;doi=10.1038/s41579-019-0196-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref052">
<span class="label">52.</span><cite>Chebotar’ IV, Emelyanova MA, Bocharova JA, Mayansky NA, Kopantseva EE, Mikhailovich VM. The classification of bacterial survival strategies in the presence of antimicrobials. Microb Pathog. 2021;155:104901. doi: 10.1016/j.micpath.2021.104901

</cite> [<a href="https://doi.org/10.1016/j.micpath.2021.104901" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33930413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microb%20Pathog&amp;title=The%20classification%20of%20bacterial%20survival%20strategies%20in%20the%20presence%20of%20antimicrobials&amp;author=IV%20Chebotar%E2%80%99&amp;author=MA%20Emelyanova&amp;author=JA%20Bocharova&amp;author=NA%20Mayansky&amp;author=EE%20Kopantseva&amp;volume=155&amp;publication_year=2021&amp;pages=104901&amp;pmid=33930413&amp;doi=10.1016/j.micpath.2021.104901&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref053">
<span class="label">53.</span><cite>Helaine S, Conlon BP, Davis KM, Russell DG. Host stress drives tolerance and persistence: The bane of anti-microbial therapeutics. Cell Host Microbe. 2024;32(6):852–62. doi: 10.1016/j.chom.2024.04.019

</cite> [<a href="https://doi.org/10.1016/j.chom.2024.04.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11446042/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38870901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20Microbe&amp;title=Host%20stress%20drives%20tolerance%20and%20persistence:%20The%20bane%20of%20anti-microbial%20therapeutics&amp;author=S%20Helaine&amp;author=BP%20Conlon&amp;author=KM%20Davis&amp;author=DG%20Russell&amp;volume=32&amp;issue=6&amp;publication_year=2024&amp;pages=852-62&amp;pmid=38870901&amp;doi=10.1016/j.chom.2024.04.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref054">
<span class="label">54.</span><cite>Peyrusson F, Nguyen TK, Najdovski T, Van Bambeke F. Host Cell Oxidative Stress Induces Dormant Staphylococcus aureus Persisters. Microbiol Spectr. 2022;10(1):e0231321. doi: 10.1128/spectrum.02313-21

</cite> [<a href="https://doi.org/10.1128/spectrum.02313-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8865412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35196815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol%20Spectr&amp;title=Host%20Cell%20Oxidative%20Stress%20Induces%20Dormant%20Staphylococcus%20aureus%20Persisters&amp;author=F%20Peyrusson&amp;author=TK%20Nguyen&amp;author=T%20Najdovski&amp;author=F%20Van%20Bambeke&amp;volume=10&amp;issue=1&amp;publication_year=2022&amp;pmid=35196815&amp;doi=10.1128/spectrum.02313-21&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref055">
<span class="label">55.</span><cite>Rowe SE, Wagner NJ, Li L, Beam JE, Wilkinson AD, Radlinski LC, et al. Reactive oxygen species induce antibiotic tolerance during systemic Staphylococcus aureus infection. Nat Microbiol. 2020;5(2):282–90. doi: 10.1038/s41564-019-0627-y

</cite> [<a href="https://doi.org/10.1038/s41564-019-0627-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6992501/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31819212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Microbiol&amp;title=Reactive%20oxygen%20species%20induce%20antibiotic%20tolerance%20during%20systemic%20Staphylococcus%20aureus%20infection&amp;author=SE%20Rowe&amp;author=NJ%20Wagner&amp;author=L%20Li&amp;author=JE%20Beam&amp;author=AD%20Wilkinson&amp;volume=5&amp;issue=2&amp;publication_year=2020&amp;pages=282-90&amp;pmid=31819212&amp;doi=10.1038/s41564-019-0627-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref056">
<span class="label">56.</span><cite>Geiger T, Kästle B, Gratani FL, Goerke C, Wolz C. Two small (p)ppGpp synthases in Staphylococcus aureus mediate tolerance against cell envelope stress conditions. J Bacteriol. 2013;196(4):894–902. 10.1128/JB.01201-13
</cite> [<a href="https://doi.org/10.1128/JB.01201-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3911181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24336937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=Two%20small%20(p)ppGpp%20synthases%20in%20Staphylococcus%20aureus%20mediate%20tolerance%20against%20cell%20envelope%20stress%20conditions&amp;author=T%20Geiger&amp;author=B%20K%C3%A4stle&amp;author=FL%20Gratani&amp;author=C%20Goerke&amp;author=C%20Wolz&amp;volume=196&amp;issue=4&amp;publication_year=2013&amp;pages=894-902&amp;pmid=24336937&amp;doi=10.1128/JB.01201-13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref057">
<span class="label">57.</span><cite>Tuchscherr L, Heitmann V, Hussain M, Viemann D, Roth J, von Eiff C, et al. Staphylococcus aureus small-colony variants are adapted phenotypes for intracellular persistence. J Infect Dis. 2010;202(7):1031–40. doi: 10.1086/656047

</cite> [<a href="https://doi.org/10.1086/656047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20715929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&amp;title=Staphylococcus%20aureus%20small-colony%20variants%20are%20adapted%20phenotypes%20for%20intracellular%20persistence&amp;author=L%20Tuchscherr&amp;author=V%20Heitmann&amp;author=M%20Hussain&amp;author=D%20Viemann&amp;author=J%20Roth&amp;volume=202&amp;issue=7&amp;publication_year=2010&amp;pages=1031-40&amp;pmid=20715929&amp;doi=10.1086/656047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref058">
<span class="label">58.</span><cite>Guérillot R, Kostoulias X, Donovan L, Li L, Carter GP, Hachani A, et al. Unstable chromosome rearrangements in Staphylococcus aureus cause phenotype switching associated with persistent infections. Proc Natl Acad Sci U S A. 2019;116(40):20135–40. doi: 10.1073/pnas.1904861116

</cite> [<a href="https://doi.org/10.1073/pnas.1904861116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6778178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31527262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Unstable%20chromosome%20rearrangements%20in%20Staphylococcus%20aureus%20cause%20phenotype%20switching%20associated%20with%20persistent%20infections&amp;author=R%20Gu%C3%A9rillot&amp;author=X%20Kostoulias&amp;author=L%20Donovan&amp;author=L%20Li&amp;author=GP%20Carter&amp;volume=116&amp;issue=40&amp;publication_year=2019&amp;pages=20135-40&amp;pmid=31527262&amp;doi=10.1073/pnas.1904861116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref059">
<span class="label">59.</span><cite>Löffler B, Tuchscherr L, Niemann S, Peters G. Staphylococcus aureus persistence in non-professional phagocytes. Int J Med Microbiol. 2014;304(2):170–6. doi: 10.1016/j.ijmm.2013.11.011

</cite> [<a href="https://doi.org/10.1016/j.ijmm.2013.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24365645/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Med%20Microbiol&amp;title=Staphylococcus%20aureus%20persistence%20in%20non-professional%20phagocytes&amp;author=B%20L%C3%B6ffler&amp;author=L%20Tuchscherr&amp;author=S%20Niemann&amp;author=G%20Peters&amp;volume=304&amp;issue=2&amp;publication_year=2014&amp;pages=170-6&amp;pmid=24365645&amp;doi=10.1016/j.ijmm.2013.11.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref060">
<span class="label">60.</span><cite>Tuchscherr L, Löffler B, Proctor RA. Persistence of Staphylococcus aureus: Multiple Metabolic Pathways Impact the Expression of Virulence Factors in Small-Colony Variants (SCVs). Front Microbiol. 2020;11:1028. doi: 10.3389/fmicb.2020.01028

</cite> [<a href="https://doi.org/10.3389/fmicb.2020.01028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7253646/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32508801/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Persistence%20of%20Staphylococcus%20aureus:%20Multiple%20Metabolic%20Pathways%20Impact%20the%20Expression%20of%20Virulence%20Factors%20in%20Small-Colony%20Variants%20(SCVs)&amp;author=L%20Tuchscherr&amp;author=B%20L%C3%B6ffler&amp;author=RA%20Proctor&amp;volume=11&amp;publication_year=2020&amp;pages=1028&amp;pmid=32508801&amp;doi=10.3389/fmicb.2020.01028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref061">
<span class="label">61.</span><cite>Kreimendahl S, Pernas L. Metabolic immunity against microbes. Trends Cell Biol. 2024;34(6):496–508. doi: 10.1016/j.tcb.2023.10.013

</cite> [<a href="https://doi.org/10.1016/j.tcb.2023.10.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38030541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Cell%20Biol&amp;title=Metabolic%20immunity%20against%20microbes&amp;author=S%20Kreimendahl&amp;author=L%20Pernas&amp;volume=34&amp;issue=6&amp;publication_year=2024&amp;pages=496-508&amp;pmid=38030541&amp;doi=10.1016/j.tcb.2023.10.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref062">
<span class="label">62.</span><cite>Zhang YJ, Rubin EJ. Feast or famine: the host-pathogen battle over amino acids. Cell Microbiol. 2013;15(7):1079–87. doi: 10.1111/cmi.12140

</cite> [<a href="https://doi.org/10.1111/cmi.12140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6434321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23521858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Microbiol&amp;title=Feast%20or%20famine:%20the%20host-pathogen%20battle%20over%20amino%20acids&amp;author=YJ%20Zhang&amp;author=EJ%20Rubin&amp;volume=15&amp;issue=7&amp;publication_year=2013&amp;pages=1079-87&amp;pmid=23521858&amp;doi=10.1111/cmi.12140&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref063">
<span class="label">63.</span><cite>Brinsmade SR, Alexander EL, Livny J, Stettner AI, Segrè D, Rhee KY, et al. Hierarchical expression of genes controlled by the Bacillus subtilis global regulatory protein CodY. Proc Natl Acad Sci U S A. 2014;111(22):8227–32. doi: 10.1073/pnas.1321308111

</cite> [<a href="https://doi.org/10.1073/pnas.1321308111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4050614/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24843172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Hierarchical%20expression%20of%20genes%20controlled%20by%20the%20Bacillus%20subtilis%20global%20regulatory%20protein%20CodY&amp;author=SR%20Brinsmade&amp;author=EL%20Alexander&amp;author=J%20Livny&amp;author=AI%20Stettner&amp;author=D%20Segr%C3%A8&amp;volume=111&amp;issue=22&amp;publication_year=2014&amp;pages=8227-32&amp;pmid=24843172&amp;doi=10.1073/pnas.1321308111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref064">
<span class="label">64.</span><cite>Wyant GA, Abu-Remaileh M, Wolfson RL, Chen WW, Freinkman E, Danai LV, et al. mTORC1 Activator SLC38A9 Is Required to Efflux Essential Amino Acids from Lysosomes and Use Protein as a Nutrient. Cell. 2017;171(3):642–54.e12.
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.09.046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5704964/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29053970/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=mTORC1%20Activator%20SLC38A9%20Is%20Required%20to%20Efflux%20Essential%20Amino%20Acids%20from%20Lysosomes%20and%20Use%20Protein%20as%20a%20Nutrient&amp;author=GA%20Wyant&amp;author=M%20Abu-Remaileh&amp;author=RL%20Wolfson&amp;author=WW%20Chen&amp;author=E%20Freinkman&amp;volume=171&amp;issue=3&amp;publication_year=2017&amp;pages=642-54.e12&amp;pmid=29053970&amp;doi=10.1016/j.cell.2017.09.046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref065">
<span class="label">65.</span><cite>Brenner M, Lobel L, Borovok I, Sigal N, Herskovits AA. Controlled branched-chain amino acids auxotrophy in Listeria monocytogenes allows isoleucine to serve as a host signal and virulence effector. PLoS Genet. 2018;14(3):e1007283. doi: 10.1371/journal.pgen.1007283

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1007283" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5864092/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29529043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&amp;title=Controlled%20branched-chain%20amino%20acids%20auxotrophy%20in%20Listeria%20monocytogenes%20allows%20isoleucine%20to%20serve%20as%20a%20host%20signal%20and%20virulence%20effector&amp;author=M%20Brenner&amp;author=L%20Lobel&amp;author=I%20Borovok&amp;author=N%20Sigal&amp;author=AA%20Herskovits&amp;volume=14&amp;issue=3&amp;publication_year=2018&amp;pmid=29529043&amp;doi=10.1371/journal.pgen.1007283&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref066">
<span class="label">66.</span><cite>Lincoln RA, Leigh JA, Jones NC. The amino acid requirements of Staphylococcus aureus isolated from cases of bovine mastitis. Vet Microbiol. 1995;45(2–3):275–9. doi: 10.1016/0378-1135(95)00041-8

</cite> [<a href="https://doi.org/10.1016/0378-1135(95)00041-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7571379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vet%20Microbiol&amp;title=The%20amino%20acid%20requirements%20of%20Staphylococcus%20aureus%20isolated%20from%20cases%20of%20bovine%20mastitis&amp;author=RA%20Lincoln&amp;author=JA%20Leigh&amp;author=NC%20Jones&amp;volume=45&amp;publication_year=1995&amp;pages=275-9&amp;pmid=7571379&amp;doi=10.1016/0378-1135(95)00041-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref067">
<span class="label">67.</span><cite>Yadati T, Houben T, Bitorina A, Shiri-Sverdlov R. The Ins and Outs of Cathepsins: Physiological Function and Role in Disease Management. Cells. 2020;9(7):1679. doi: 10.3390/cells9071679

</cite> [<a href="https://doi.org/10.3390/cells9071679" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7407943/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32668602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=The%20Ins%20and%20Outs%20of%20Cathepsins:%20Physiological%20Function%20and%20Role%20in%20Disease%20Management&amp;author=T%20Yadati&amp;author=T%20Houben&amp;author=A%20Bitorina&amp;author=R%20Shiri-Sverdlov&amp;volume=9&amp;issue=7&amp;publication_year=2020&amp;pages=1679&amp;pmid=32668602&amp;doi=10.3390/cells9071679&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref068">
<span class="label">68.</span><cite>Lehman MK, Nuxoll AS, Yamada KJ, Kielian T, Carson SD, Fey PD. Protease-mediated growth of Staphylococcus aureus on host proteins is opp3 dependent. mBio. 2019;10(2).</cite> [<a href="https://doi.org/10.1128/mBio.02553-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6495380/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31040245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Protease-mediated%20growth%20of%20Staphylococcus%20aureus%20on%20host%20proteins%20is%20opp3%20dependent&amp;author=MK%20Lehman&amp;author=AS%20Nuxoll&amp;author=KJ%20Yamada&amp;author=T%20Kielian&amp;author=SD%20Carson&amp;volume=10&amp;issue=2&amp;publication_year=2019&amp;pmid=31040245&amp;doi=10.1128/mBio.02553-18&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref069">
<span class="label">69.</span><cite>Hiron A, Borezée-Durant E, Piard J-C, Juillard V. Only one of four oligopeptide transport systems mediates nitrogen nutrition in Staphylococcus aureus. J Bacteriol. 2007;189(14):5119–29. doi: 10.1128/JB.00274-07

</cite> [<a href="https://doi.org/10.1128/JB.00274-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1951871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17496096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=Only%20one%20of%20four%20oligopeptide%20transport%20systems%20mediates%20nitrogen%20nutrition%20in%20Staphylococcus%20aureus&amp;author=A%20Hiron&amp;author=E%20Borez%C3%A9e-Durant&amp;author=J-C%20Piard&amp;author=V%20Juillard&amp;volume=189&amp;issue=14&amp;publication_year=2007&amp;pages=5119-29&amp;pmid=17496096&amp;doi=10.1128/JB.00274-07&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref070">
<span class="label">70.</span><cite>Gimza BD, Larias MI, Budny BG, Shaw LN. Mapping the Global Network of Extracellular Protease Regulation in Staphylococcus aureus. mSphere. 2019;4(5):e00676-19. doi: 10.1128/mSphere.00676-19

</cite> [<a href="https://doi.org/10.1128/mSphere.00676-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6811363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31645429/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mSphere&amp;title=Mapping%20the%20Global%20Network%20of%20Extracellular%20Protease%20Regulation%20in%20Staphylococcus%20aureus&amp;author=BD%20Gimza&amp;author=MI%20Larias&amp;author=BG%20Budny&amp;author=LN%20Shaw&amp;volume=4&amp;issue=5&amp;publication_year=2019&amp;pmid=31645429&amp;doi=10.1128/mSphere.00676-19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref071">
<span class="label">71.</span><cite>Helaine S, Kugelberg E. Bacterial persisters: formation, eradication, and experimental systems. Trends Microbiol. 2014;22(7):417–24. doi: 10.1016/j.tim.2014.03.008

</cite> [<a href="https://doi.org/10.1016/j.tim.2014.03.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24768561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Microbiol&amp;title=Bacterial%20persisters:%20formation,%20eradication,%20and%20experimental%20systems&amp;author=S%20Helaine&amp;author=E%20Kugelberg&amp;volume=22&amp;issue=7&amp;publication_year=2014&amp;pages=417-24&amp;pmid=24768561&amp;doi=10.1016/j.tim.2014.03.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref072">
<span class="label">72.</span><cite>Conlon BP. Staphylococcus aureus chronic and relapsing infections: Evidence of a role for persister cells: An investigation of persister cells, their formation and their role in S. aureus disease. Bioessays. 2014;36(10):991–6. doi: 10.1002/bies.201400080

</cite> [<a href="https://doi.org/10.1002/bies.201400080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25100240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioessays&amp;title=Staphylococcus%20aureus%20chronic%20and%20relapsing%20infections:%20Evidence%20of%20a%20role%20for%20persister%20cells:%20An%20investigation%20of%20persister%20cells,%20their%20formation%20and%20their%20role%20in%20S.%20aureus%20disease&amp;author=BP%20Conlon&amp;volume=36&amp;issue=10&amp;publication_year=2014&amp;pages=991-6&amp;pmid=25100240&amp;doi=10.1002/bies.201400080&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref073">
<span class="label">73.</span><cite>Vulin C, Leimer N, Huemer M, Ackermann M, Zinkernagel AS. Prolonged bacterial lag time results in small colony variants that represent a sub-population of persisters. Nat Commun. 2018;9(1):4074. doi: 10.1038/s41467-018-06527-0

</cite> [<a href="https://doi.org/10.1038/s41467-018-06527-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6172231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30287875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Prolonged%20bacterial%20lag%20time%20results%20in%20small%20colony%20variants%20that%20represent%20a%20sub-population%20of%20persisters&amp;author=C%20Vulin&amp;author=N%20Leimer&amp;author=M%20Huemer&amp;author=M%20Ackermann&amp;author=AS%20Zinkernagel&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pages=4074&amp;pmid=30287875&amp;doi=10.1038/s41467-018-06527-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref074">
<span class="label">74.</span><cite>Loss G, Simões PM, Valour F, Cortês MF, Gonzaga L, Bergot M, et al. Staphylococcus aureus Small Colony Variants (SCVs): News From a Chronic Prosthetic Joint Infection. Frontiers in Cellular and Infection Microbiology. 2019;9.
</cite> [<a href="https://doi.org/10.3389/fcimb.2019.00363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6817495/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31696062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Cellular%20and%20Infection%20Microbiology&amp;title=Staphylococcus%20aureus%20Small%20Colony%20Variants%20(SCVs):%20News%20From%20a%20Chronic%20Prosthetic%20Joint%20Infection&amp;author=G%20Loss&amp;author=PM%20Sim%C3%B5es&amp;author=F%20Valour&amp;author=MF%20Cort%C3%AAs&amp;author=L%20Gonzaga&amp;publication_year=2019&amp;pages=9&amp;pmid=31696062&amp;doi=10.3389/fcimb.2019.00363&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref075">
<span class="label">75.</span><cite>Kahl BC, Becker K, Löffler B. Clinical Significance and Pathogenesis of Staphylococcal Small Colony Variants in Persistent Infections. Clin Microbiol Rev. 2016;29(2):401–27. doi: 10.1128/CMR.00069-15

</cite> [<a href="https://doi.org/10.1128/CMR.00069-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4786882/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26960941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Microbiol%20Rev&amp;title=Clinical%20Significance%20and%20Pathogenesis%20of%20Staphylococcal%20Small%20Colony%20Variants%20in%20Persistent%20Infections&amp;author=BC%20Kahl&amp;author=K%20Becker&amp;author=B%20L%C3%B6ffler&amp;volume=29&amp;issue=2&amp;publication_year=2016&amp;pages=401-27&amp;pmid=26960941&amp;doi=10.1128/CMR.00069-15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref076">
<span class="label">76.</span><cite>Islam S, Callender AC, Ho QN, Wakeman CA. Iron restriction induces the small-colony variant phenotype in Staphylococcus aureus. Frontiers in Microbiology. 2022;13.</cite> [<a href="https://doi.org/10.3389/fmicb.2022.978859" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9772265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36569073/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Iron%20restriction%20induces%20the%20small-colony%20variant%20phenotype%20in%20Staphylococcus%20aureus&amp;author=S%20Islam&amp;author=AC%20Callender&amp;author=QN%20Ho&amp;author=CA%20Wakeman&amp;volume=13&amp;publication_year=2022&amp;pmid=36569073&amp;doi=10.3389/fmicb.2022.978859&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref077">
<span class="label">77.</span><cite>Garcia LG, Lemaire S, Kahl BC, Becker K, Proctor RA, Denis O, et al. Antibiotic activity against small-colony variants of Staphylococcus aureus: review of in vitro, animal and clinical data. J Antimicrob Chemother. 2013;68(7):1455–64. doi: 10.1093/jac/dkt072

</cite> [<a href="https://doi.org/10.1093/jac/dkt072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23485724/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Antimicrob%20Chemother&amp;title=Antibiotic%20activity%20against%20small-colony%20variants%20of%20Staphylococcus%20aureus:%20review%20of%20in%20vitro,%20animal%20and%20clinical%20data&amp;author=LG%20Garcia&amp;author=S%20Lemaire&amp;author=BC%20Kahl&amp;author=K%20Becker&amp;author=RA%20Proctor&amp;volume=68&amp;issue=7&amp;publication_year=2013&amp;pages=1455-64&amp;pmid=23485724&amp;doi=10.1093/jac/dkt072&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref078">
<span class="label">78.</span><cite>Proctor RA, von Eiff C, Kahl BC, Becker K, McNamara P, Herrmann M, et al. Small colony variants: a pathogenic form of bacteria that facilitates persistent and recurrent infections. Nat Rev Microbiol. 2006;4(4):295–305. doi: 10.1038/nrmicro1384

</cite> [<a href="https://doi.org/10.1038/nrmicro1384" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16541137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Microbiol&amp;title=Small%20colony%20variants:%20a%20pathogenic%20form%20of%20bacteria%20that%20facilitates%20persistent%20and%20recurrent%20infections&amp;author=RA%20Proctor&amp;author=C%20von%20Eiff&amp;author=BC%20Kahl&amp;author=K%20Becker&amp;author=P%20McNamara&amp;volume=4&amp;issue=4&amp;publication_year=2006&amp;pages=295-305&amp;pmid=16541137&amp;doi=10.1038/nrmicro1384&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref079">
<span class="label">79.</span><cite>Zeden MS, Burke Ó, Vallely M, Fingleton C, O’Gara JP. Exploring amino acid and peptide transporters as therapeutic targets to attenuate virulence and antibiotic resistance in Staphylococcus aureus. PLOS Pathogens. 2021;17(1):e1009093. doi: 10.1371/journal.ppat.1009093</cite> [<a href="https://doi.org/10.1371/journal.ppat.1009093" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7808641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33444418/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Pathogens&amp;title=Exploring%20amino%20acid%20and%20peptide%20transporters%20as%20therapeutic%20targets%20to%20attenuate%20virulence%20and%20antibiotic%20resistance%20in%20Staphylococcus%20aureus&amp;author=MS%20Zeden&amp;author=%C3%93%20Burke&amp;author=M%20Vallely&amp;author=C%20Fingleton&amp;author=JP%20O%E2%80%99Gara&amp;volume=17&amp;issue=1&amp;publication_year=2021&amp;pmid=33444418&amp;doi=10.1371/journal.ppat.1009093&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref080">
<span class="label">80.</span><cite>Li C, Wen A, Shen B, Lu J, Huang Y, Chang Y. FastCloning: a highly simplified, purification-free, sequence- and ligation-independent PCR cloning method. BMC Biotechnol. 2011;11:92. doi: 10.1186/1472-6750-11-92

</cite> [<a href="https://doi.org/10.1186/1472-6750-11-92" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3207894/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21992524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Biotechnol&amp;title=FastCloning:%20a%20highly%20simplified,%20purification-free,%20sequence-%20and%20ligation-independent%20PCR%20cloning%20method&amp;author=C%20Li&amp;author=A%20Wen&amp;author=B%20Shen&amp;author=J%20Lu&amp;author=Y%20Huang&amp;volume=11&amp;publication_year=2011&amp;pages=92&amp;pmid=21992524&amp;doi=10.1186/1472-6750-11-92&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref081">
<span class="label">81.</span><cite>Giese B, Dittmann S, Paprotka K, Levin K, Weltrowski A, Biehler D, et al. Staphylococcal alpha-toxin is not sufficient to mediate escape from phagolysosomes in upper-airway epithelial cells. Infect Immun. 2009;77(9):3611–25. doi: 10.1128/IAI.01478-08

</cite> [<a href="https://doi.org/10.1128/IAI.01478-08" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2738027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19564384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Immun&amp;title=Staphylococcal%20alpha-toxin%20is%20not%20sufficient%20to%20mediate%20escape%20from%20phagolysosomes%20in%20upper-airway%20epithelial%20cells&amp;author=B%20Giese&amp;author=S%20Dittmann&amp;author=K%20Paprotka&amp;author=K%20Levin&amp;author=A%20Weltrowski&amp;volume=77&amp;issue=9&amp;publication_year=2009&amp;pages=3611-25&amp;pmid=19564384&amp;doi=10.1128/IAI.01478-08&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref082">
<span class="label">82.</span><cite>Bose JL, Fey PD, Bayles KW. Genetic tools to enhance the study of gene function and regulation in Staphylococcus aureus. Appl Environ Microbiol. 2013;79(7):2218–24. doi: 10.1128/AEM.00136-13

</cite> [<a href="https://doi.org/10.1128/AEM.00136-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3623228/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23354696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Genetic%20tools%20to%20enhance%20the%20study%20of%20gene%20function%20and%20regulation%20in%20Staphylococcus%20aureus&amp;author=JL%20Bose&amp;author=PD%20Fey&amp;author=KW%20Bayles&amp;volume=79&amp;issue=7&amp;publication_year=2013&amp;pages=2218-24&amp;pmid=23354696&amp;doi=10.1128/AEM.00136-13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref083">
<span class="label">83.</span><cite>Kim S, Kim H-S, Lee Y, Song J, Kim HO, Min Y-H. Leukoreduction System Chambers as a Source of Viable Human Peripheral Blood Mononuclear Cells.
Blood. 2007;110(11):4912–4912. doi: 10.1182/blood.v110.11.4912.4912</cite> [<a href="https://doi.org/10.1182/blood.v110.11.4912.4912" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Leukoreduction%20System%20Chambers%20as%20a%20Source%20of%20Viable%20Human%20Peripheral%20Blood%20Mononuclear%20Cells.&amp;author=S%20Kim&amp;author=H-S%20Kim&amp;author=Y%20Lee&amp;author=J%20Song&amp;author=HO%20Kim&amp;volume=110&amp;issue=11&amp;publication_year=2007&amp;pages=4912-4912&amp;doi=10.1182/blood.v110.11.4912.4912&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref084">
<span class="label">84.</span><cite>Schindelin J, Arganda-Carreras I, Frise E, Kaynig V, Longair M, Pietzsch T, et al. Fiji: an open-source platform for biological-image analysis. Nat Methods. 2012;9(7):676–82. doi: 10.1038/nmeth.2019

</cite> [<a href="https://doi.org/10.1038/nmeth.2019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3855844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Fiji:%20an%20open-source%20platform%20for%20biological-image%20analysis&amp;author=J%20Schindelin&amp;author=I%20Arganda-Carreras&amp;author=E%20Frise&amp;author=V%20Kaynig&amp;author=M%20Longair&amp;volume=9&amp;issue=7&amp;publication_year=2012&amp;pages=676-82&amp;pmid=22743772&amp;doi=10.1038/nmeth.2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref085">
<span class="label">85.</span><cite>Rio DC, Ares M Jr, Hannon GJ, Nilsen TW. Nondenaturing agarose gel electrophoresis of RNA. Cold Spring Harb Protoc. 2010;2010(6):pdb.prot5445. doi: 10.1101/pdb.prot5445

</cite> [<a href="https://doi.org/10.1101/pdb.prot5445" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20516183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb%20Protoc&amp;title=Nondenaturing%20agarose%20gel%20electrophoresis%20of%20RNA&amp;author=DC%20Rio&amp;author=Jr%20Ares%20M&amp;author=GJ%20Hannon&amp;author=TW%20Nilsen&amp;volume=2010&amp;issue=6&amp;publication_year=2010&amp;pmid=20516183&amp;doi=10.1101/pdb.prot5445&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref086">
<span class="label">86.</span><cite>Dobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics. 2013;29(1):15–21. doi: 10.1093/bioinformatics/bts635

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3530905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=STAR:%20ultrafast%20universal%20RNA-seq%20aligner&amp;author=A%20Dobin&amp;author=CA%20Davis&amp;author=F%20Schlesinger&amp;author=J%20Drenkow&amp;author=C%20Zaleski&amp;volume=29&amp;issue=1&amp;publication_year=2013&amp;pages=15-21&amp;pmid=23104886&amp;doi=10.1093/bioinformatics/bts635&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref087">
<span class="label">87.</span><cite>Förstner KU, Vogel J, Sharma CM. READemption-a tool for the computational analysis of deep-sequencing-based transcriptome data. Bioinformatics. 2014;30(23):3421–3. doi: 10.1093/bioinformatics/btu533

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu533" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25123900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=READemption-a%20tool%20for%20the%20computational%20analysis%20of%20deep-sequencing-based%20transcriptome%20data&amp;author=KU%20F%C3%B6rstner&amp;author=J%20Vogel&amp;author=CM%20Sharma&amp;volume=30&amp;issue=23&amp;publication_year=2014&amp;pages=3421-3&amp;pmid=25123900&amp;doi=10.1093/bioinformatics/btu533&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref088">
<span class="label">88.</span><cite>Love MI, Huber W, Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 2014;15(12):550. doi: 10.1186/s13059-014-0550-8

</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&amp;author=MI%20Love&amp;author=W%20Huber&amp;author=S%20Anders&amp;volume=15&amp;issue=12&amp;publication_year=2014&amp;pages=550&amp;pmid=25516281&amp;doi=10.1186/s13059-014-0550-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref089">
<span class="label">89.</span><cite>Livak KJ, Schmittgen TD. Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method. Methods. 2001;25(4):402–8.
</cite> [<a href="https://doi.org/10.1006/meth.2001.1262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11846609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202(-Delta%20Delta%20C(T))%20Method&amp;author=KJ%20Livak&amp;author=TD%20Schmittgen&amp;volume=25&amp;issue=4&amp;publication_year=2001&amp;pages=402-8&amp;pmid=11846609&amp;doi=10.1006/meth.2001.1262&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref090">
<span class="label">90.</span><cite>Darling AE, Mau B, Perna NT. progressiveMauve: multiple genome alignment with gene gain, loss and rearrangement. PLoS One. 2010;5(6):e11147. doi: 10.1371/journal.pone.0011147

</cite> [<a href="https://doi.org/10.1371/journal.pone.0011147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2892488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20593022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=progressiveMauve:%20multiple%20genome%20alignment%20with%20gene%20gain,%20loss%20and%20rearrangement&amp;author=AE%20Darling&amp;author=B%20Mau&amp;author=NT%20Perna&amp;volume=5&amp;issue=6&amp;publication_year=2010&amp;pmid=20593022&amp;doi=10.1371/journal.pone.0011147&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref091">
<span class="label">91.</span><cite>Seemann T. Prokka: rapid prokaryotic genome annotation. Bioinformatics. 2014;30(14):2068–9. doi: 10.1093/bioinformatics/btu153

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24642063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Prokka:%20rapid%20prokaryotic%20genome%20annotation&amp;author=T%20Seemann&amp;volume=30&amp;issue=14&amp;publication_year=2014&amp;pages=2068-9&amp;pmid=24642063&amp;doi=10.1093/bioinformatics/btu153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref092">
<span class="label">92.</span><cite>Flannagan RS, Heinrichs DE. A fluorescence based-proliferation assay for the identification of replicating bacteria within host cells. Frontiers in Microbiology. 2018;9.</cite> [<a href="https://doi.org/10.3389/fmicb.2018.03084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6299164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30619165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=A%20fluorescence%20based-proliferation%20assay%20for%20the%20identification%20of%20replicating%20bacteria%20within%20host%20cells&amp;author=RS%20Flannagan&amp;author=DE%20Heinrichs&amp;volume=9&amp;publication_year=2018&amp;pmid=30619165&amp;doi=10.3389/fmicb.2018.03084&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref093">
<span class="label">93.</span><cite>Schoenfelder SMK, Marincola G, Geiger T, Goerke C, Wolz C, Ziebuhr W. Methionine biosynthesis in Staphylococcus aureus is tightly controlled by a hierarchical network involving an initiator tRNA-specific T-box riboswitch. PLoS Pathog. 2013;9(9):e1003606. doi: 10.1371/journal.ppat.1003606

</cite> [<a href="https://doi.org/10.1371/journal.ppat.1003606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3771891/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24068926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&amp;title=Methionine%20biosynthesis%20in%20Staphylococcus%20aureus%20is%20tightly%20controlled%20by%20a%20hierarchical%20network%20involving%20an%20initiator%20tRNA-specific%20T-box%20riboswitch&amp;author=SMK%20Schoenfelder&amp;author=G%20Marincola&amp;author=T%20Geiger&amp;author=C%20Goerke&amp;author=C%20Wolz&amp;volume=9&amp;issue=9&amp;publication_year=2013&amp;pmid=24068926&amp;doi=10.1371/journal.ppat.1003606&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ppat.1013291.ref094">
<span class="label">94.</span><cite>Blättner S, Das S, Paprotka K, Eilers U, Krischke M, Kretschmer D, et al. Staphylococcus aureus Exploits a Non-ribosomal Cyclic Dipeptide to Modulate Survival within Epithelial Cells and Phagocytes. PLoS Pathog. 2016;12(9):e1005857. doi: 10.1371/journal.ppat.1005857

</cite> [<a href="https://doi.org/10.1371/journal.ppat.1005857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5025175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27632173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&amp;title=Staphylococcus%20aureus%20Exploits%20a%20Non-ribosomal%20Cyclic%20Dipeptide%20to%20Modulate%20Survival%20within%20Epithelial%20Cells%20and%20Phagocytes&amp;author=S%20Bl%C3%A4ttner&amp;author=S%20Das&amp;author=K%20Paprotka&amp;author=U%20Eilers&amp;author=M%20Krischke&amp;volume=12&amp;issue=9&amp;publication_year=2016&amp;pmid=27632173&amp;doi=10.1371/journal.ppat.1005857&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section></section><article class="sub-article" id="ppat.1013291.r001"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS Pathog. doi: <a href="https://doi.org/10.1371/journal.ppat.1013291.r001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.ppat.1013291.r001</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 0</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prince%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Alice Prince</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Alice Prince</span></h3>
<div>Section Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prince%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alice Prince</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.c" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.c" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.c" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Alice Prince</span></strong>: <span class="role">Section Editor</span>
</div>
</div>
<div id="clp_a.c" class="d-panel p" style="display: none">
<div>© 2025 Alice Prince</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>PPATHOGENS-D-25-00597</p>
<p>Inactivation of branched-chain amino acid uptake halts <em>Staphylococcus aureus</em> growth and induces bacterial quiescence within macrophages</p>
<p>PLOS Pathogens</p>
<p>Dear Dr. Fraunholz,</p>
<p>Thank you for submitting your manuscript to PLOS Pathogens. After careful consideration, we feel that it has merit but does not fully meet PLOS Pathogens's publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.</p>
<p>Please submit your revised manuscript within 60 days Jun 02 2025 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at plospathogens@plos.org. When you're ready to submit your revision, log on to <a href="https://www.editorialmanager.com/ppathogens/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/ppathogens/</a> and select the 'Submissions Needing Revision' folder to locate your manuscript file.</p>
<p>Please include the following items when submitting your revised manuscript:</p>
<p>* A rebuttal letter that responds to each point raised by the editor and reviewer(s). You should upload this letter as a separate file labeled 'Response to Reviewers'. This file does not need to include responses to any formatting updates and technical items listed in the 'Journal Requirements' section below.</p>
<p>* A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled 'Revised Manuscript with Track Changes'.</p>
<p>* An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled 'Manuscript'.</p>
<p>If you would like to make changes to your financial disclosure, competing interests statement, or data availability statement, please make these updates within the submission form at the time of resubmission. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter.</p>
<p>We look forward to receiving your revised manuscript.</p>
<p>Kind regards,</p>
<p>Alice Prince</p>
<p>Section Editor</p>
<p>PLOS Pathogens</p>
<p>Alice Prince</p>
<p>Section Editor</p>
<p>PLOS Pathogens</p>
<p> Sumita Bhaduri-McIntosh</p>
<p>Editor-in-Chief</p>
<p>PLOS Pathogens</p>
<p>orcid.org/0000-0003-2946-9497</p>
<p> Michael Malim</p>
<p>Editor-in-Chief</p>
<p>PLOS Pathogens</p>
<p>orcid.org/0000-0002-7699-2064</p>
<p>
<strong>Journal Requirements:</strong>
</p>
<p>1) We ask that a manuscript source file is provided at Revision. Please upload your manuscript file as a .doc, .docx, .rtf or .tex. If you are providing a .tex file, please upload it under the item type u2018LaTeX Source Fileu2019 and leave your .pdf version as the item type u2018Manuscriptu2019.</p>
<p>2) We do not publish any copyright or trademark symbols that usually accompany proprietary names, eg ©,  ®, or TM  (e.g. next to drug or reagent names). Therefore please remove all instances of trademark/copyright symbols throughout the text, including:</p>
<p>- ® on pages: 25, 26, and 28.</p>
<p>3) Please upload all main figures as separate Figure files in .tif or .eps format. For more information about how to convert and format your figure files please see our guidelines: </p>
<p>
<a href="https://journals.plos.org/plospathogens/s/figures" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plospathogens/s/figures</a>
</p>
<p>4) We have noticed that you have uploaded Supporting Information files, but you have not included a list of legends. Please add a full list of legends for your Supporting Information files after the references list.</p>
<p>5) Some material included in your submission may be copyrighted. According to PLOSu2019s copyright policy, authors who use figures or other material (e.g., graphics, clipart, maps) from another author or copyright holder must demonstrate or obtain permission to publish this material under the Creative Commons Attribution 4.0 International (CC BY 4.0) License used by PLOS journals. Please closely review the details of PLOSu2019s copyright requirements here: PLOS Licenses and Copyright. If you need to request permissions from a copyright holder, you may use PLOS's Copyright Content Permission form.</p>
<p>Please respond directly to this email and provide any known details concerning your material's license terms and permissions required for reuse, even if you have not yet obtained copyright permissions or are unsure of your material's copyright compatibility. Once you have responded and addressed all other outstanding technical requirements, you may resubmit your manuscript within Editorial Manager. </p>
<p>Potential Copyright Issues:</p>
<p>- Figures 1, 3, and 5. Please confirm whether you drew the images / clip-art within the figure panels by hand. If you did not draw the images, please provide (a) a link to the source of the images or icons and their license / terms of use; or (b) written permission from the copyright holder to publish the images or icons under our CC BY 4.0 license. Alternatively, you may replace the images with open source alternatives. See these open source resources you may use to replace images / clip-art:</p>
<p>- <a href="https://commons.wikimedia.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://commons.wikimedia.org</a></p>
<p>- <a href="https://openclipart.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://openclipart.org/</a>.</p>
<p>6) Please ensure that the funders and grant numbers match between the Financial Disclosure field and the Funding Information tab in your submission form. Note that the funders must be provided in the same order in both places as well.</p>
<p>- State the initials, alongside each funding source, of each author to receive each grant. For example: "This work was supported by the National Institutes of Health (####### to AM; ###### to CJ) and the National Science Foundation (###### to AM)."</p>
<p>- State what role the funders took in the study. If the funders had no role in your study, please state: "The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.".</p>
<p>If you did not receive any funding for this study, please simply state: u201cThe authors received no specific funding for this work.u201d</p>
<p>
<strong>Reviewers' Comments:</strong>
</p>
<p>Reviewer's Responses to Questions</p>
<p>
<strong>Part I - Summary</strong>
</p>
<p>Please use this section to discuss strengths/weaknesses of study, novelty/significance, general execution and scholarship.</p>
<p>Reviewer #1: The study by Moldovan et al describes the consequences of a mutation in the branched chain amino acid transporter BrnQ1 for intracellular growth and protein synthesis in S. aureus. Interestingly, mutation of codY restores growth, showing that repression of BCAA biosynthesis genes occurs in the macrophage, although the mechanism responsible for this is unexplored. Aside from the key finding (S. aureus uses exogenous BCAAs in the phagosome and requires BrnQ1), the rest of the paper is of more limited significance.</p>
<p>Reviewer #2: This manuscript describes the role of branched-chain amino acids (BCAA) in the survival of S. aureus within phagolysosomes after engulfment by macrophages. The authors analyzed the ability of individual S. aureus mutants deficient for the BCAA transporters BrnQ1, BrnQ2, BrnQ3 and BcaP to replicate and survive within macrophages. They also included mutants lacking the ilvE, ilvD and leuA genes involved in BCAA biosynthesis. Of all these tested, only the brnQ1 mutant was found to be defective in intracellular replication within macrophages. Importantly, this phenotype was observed in multiple S. aureus strains and a growth delay was observed in defined medium with low BCAA availability. The brnQ1 mutant also failed to induce death of the macrophages and remained intracellular, unlike other strains, which ultimately escaped and infected neighboring cells.</p>
<p>Transcriptome analysis revealed the downregulation of genes involved in iron acquisition in the brnQ1 mutant compared to the WT strain, along with several other virulence genes, consistent with studies demonstrating the limited iron availability of a phagolysosome. The results here are clear but without more information about the rationale for picking 10 hrs post infection to collect samples for these experiments, it is difficult to completely interpret these results. An interesting aspect of this study was that an iron regulated gene was not expressed in the brnQ1/fur background, suggesting the cells are entering a state of quiescence. This was nicely tested by attempting to induce a reporter gene in the brnQ1 mutant, which demonstrated reduced reporter expression compared with WT cells. In addition, using a combination of mutations that inactivate various genes involved in BCAA regulation, uptake, and/or biosynthesis, the authors conclusively demonstrated that S. aureus could overcome the limited BCAA levels in a phagolysosome by a combination of increased expression of CodY-regulated BCAA transporters or biosynthetic genes. Finally, long-term intracellular survival assays demonstrated the brnQ1 mutant persisted for up to 28 days within a macrophage. These survivors were found to be distinct from classical persisters (SCVs), which exhibit slow growth, antibiotic tolerance, and loss of pigmentation, none of which are characteristics of the brnQ1 mutant cells.</p>
<p>Reviewer #3: This is an interesting manuscript with new insights into the intracellular lifestyle of S. aureus in macrophages depending on BCAA availability.</p>
<p>The manuscript “Inactivation of branched-chain amino acid uptake halts Staphylococcus aureus growth and induces bacterial quiescence within macrophages” by Moldovan et al. investigates the impact of branched-amino acids and the respective transporters of S. aureus on its intracellular fate in macrophages. To show this the authors used macrophages that they developed from monocytes, which they infected with WT S. aureus and different transporter mutants in different S. aureus backgrounds, the complemented mutants and BCAA-synthesis mutants; they performed growth studies with and without external BCAAs; they performed dual RNAseq of infected macrophages revealing that especially iron-dependent genes were differentially expressed in WT and the mutant indicating that brnQ1-mutants are impaired in gene expression, in de novo protein synthesis representing a quiescent status. However, by external BCAAs supplementation the mutant acted comparable to the WT. The authors investigated the role of CodY for the effects observed in the brnQ1-strain by constructing a brnQ1-codY double mutant, which showed intracellular replication and escape of the phagolysosome. The authors also determined the role of CodY as the regulator of BCAA transporters and synthesis genes including the other important BCAA transporter BcaP and the most important synthesis gene ilvD. The authors also performed longterm infection assays revealing that the mutants could survive during this period. The authors tried to show that the persisting mutants were neither classical persisters nor SCVs.</p>
<p>**********</p>
<p>
<strong>Part II – Major Issues: Key Experiments Required for Acceptance</strong>
</p>
<p>Please use this section to detail the key new experiments or modifications of existing experiments that should be <span class="text-underline">absolutely</span> required to validate study conclusions.</p>
<p>Generally, there should be no more than 3 such required experiments or major modifications for a "Major Revision" recommendation. If more than 3 experiments are necessary to validate the study conclusions, then you are encouraged to recommend "Reject".</p>
<p>Reviewer #1: The antibiotic killing experiment is unusual in that it does not include a wild-type strain and the level of killing by rifampicin is not compared with anything. These experiments should be done more completely with the wild-type, mutant and multiple antibiotics, or (as I strongly recommend), removed from the manuscript altogether.</p>
<p>Reviewer #2: None</p>
<p>Reviewer #3: Major points of criticism</p>
<p>1. While the authors describe in the introduction the role of S. aureus and also of branched-chain amino acids, it would be interesting for the reader why the authors got the idea that BCAA could play any role for internalized S. aureus.</p>
<p>2. Also, is this an observation which holds only true for professional phagocytes or is this also a mechanism for intracellular survival in non-professional phagocytes such as epithelial or endothelial cells?</p>
<p>3. Since the authors do not only use BCAA transporter mutants, but also mutants in genes for BCAA synthesis, they should introduce the synthesis genes in the introduction. Furthermore, is it known if there is any phenotype associated with the single BCAA synthesis mutants? Can this be expected, since mostly S. aureus uses BCAAs from the environment as it is described in several studies (e.g. Kaiser et al. PlosGen2018), where it also has been described that it has been suggested that S. aureus would be auxotrophic for leucine and valine.</p>
<p>4. Does it really make sense to test a codY mutant in this regard? Since codY is a negative regulator of the BCAA-synthesis genes, in a codY mutant, these genes would be repressed, thereby acting if at all more in a BCAA synthesis way. Therefore, what should be expected in terms of determining the role of the BCAA-synthesis operon?</p>
<p>5. The term of “unrestricted growth” after escape from the macrophage, is this a term used in the scientific community or is it a term only used in this study? The authors should describe the effect of this more intensely.</p>
<p>6. By first mentioning the Nebraska Transposon Library, the authors should state in which background the library has been established, USA300 LAC.</p>
<p>7. Why did the authors use JE2 as another strain for introducing the brnQ1 mutation as indicated line 124? And surprisingly, in line 133 the authors report that the mutation was also moved into the S. aureus strains 6850, SH1000 and RN4220. They should have indicated this earlier.</p>
<p>8. The authors show that the brnQ1 mutation does not only effect intracellular replication but also growth in cell culture and infection media. This is not entirely new as Kaiser et al have already shown in 2015 that this mutant has a growth defect in chemical defined medium without BCAAs (Kaiser et al in 2015, Inf Immun).</p>
<p>9. The paragraph showing that these mutants are not “classical persisters” is not very convincing. The authors do not show data of the real persisters after 28d, but point at the transcriptomic data at 10h p.i. Also, for the antibiotic tolerance, they test infection after 10h. It would be more convincing to test antibiotic tolerance after long-term infection.</p>
<p>However, convincing is the description that the mutants do not undergo the SCV status during persistence.</p>
<p>A last point of interest would be to search the publicly available S. aureus genome data bases to check if there are any natural brnQ1 existing mutants in clinical isolates present. This would add a clinical role for the interesting mutant.</p>
<p>**********</p>
<p>
<strong>Part III – Minor Issues: Editorial and Data Presentation Modifications</strong>
</p>
<p>Please use this section for editorial suggestions as well as relatively minor modifications of existing data that would enhance clarity.</p>
<p>Reviewer #1: The longterm survival of the mutant is discussed as though it could represent a long-term survival niche, but the mutant doesn't appear to occur naturally, so this behavior is artifactual and of limited significance. The authors should temper their language when discussing this part of the paper. The comparison with SCVs is of limited relevance as they are clearly completely different things and one would not expect them to be similar. A SCV is generally a S. aureus strain that has lost the ability to respire, while this mutant has lost the ability to take up exogenous BCAAs. Comparison of the two seems unusual. It is recommended that this is reduced or removed or at least justified more thoroughly.</p>
<p>In lines 135-145 the authors discuss different media and their BCAA content but they only give actual concentrations of BCAAs for some conditions and are vague for others. Please add quantities for each condition if they are so relevant.</p>
<p>In line 236, recommend removing "Remarkably" as there is no reason to suspect the Fur repressor would be involved in the phenotype.</p>
<p>Fig. 5e and f should be in supplement.</p>
<p>Reviewer #2: Overall, this is a nicely presented body of work that provides important new insight into the survival and growth of S. aureus after engulfment by macrophages. Although this a very thorough study that addresses numerous aspects of growth within a phagolysosome, my one concern is the characterization of the phenotype observed as “dormancy”, akin to persisters. I agree that it is tempting to speculate that this is a new and previously uncharacterized metabolic state, but without any evidence that this state arises in the WT strain or during an infection, I believe any speculation as to the functional relevance of this state should be made with caution.</p>
<p>Questions and comments:</p>
<p>Line 42: Change, “It is facultative intracellular…”, to “It is a facultative intracellular pathogen…”.</p>
<p>Line 256: In this context, what does deplete mean? Clearly there is enough BCAAs for growth, as long as the bacteria can transport them in.</p>
<p>Line 263: I don’t understand this statement: “During growth in CDM lacking all three BCAAs, the growth of S. aureus is preceded by an extended lag phase of ~20h. Omission of Val and Leu, however, completely halted bacterial growth.” If the absence of all three BCAAs causes a lag in growth, why would the absence of only two “completely halt growth”? Please clarify.</p>
<p>Reviewer #3: Minor points of criticism</p>
<p>Line 58: The description “commensal” should be avoided, see the viewpoint article by Proctor, RA: Clin Infect Dis 2021 Jul 1;73(1):e267-e269. doi: 10.1093/cid/ciaa1431.</p>
<p>Have We Outlived the Concept of Commensalism for Staphylococcus aureus?</p>
<p>A legend to Table 1 is missing.</p>
<p>Figure 1a: This figure can be deleted, the assay is well described and not especially new.</p>
<p>Fig. 1f: What do the authors mean by “stills”?</p>
<p>Fig. 1g: In the legend of this figure there is no mentioning of the mutant. What do the authors intend to show here?</p>
<p>Fig. 3c can be deleted, see commentary to Fig. 1a</p>
<p>Reference list: In the reference list, all bacteria and gene names should be written in italics. Also, the references should be in a homogenous style with either all journals abbreviated or not.</p>
<p>Lines 257ff: The authors suggest that they would test the effects of BCAAs to intracellular bacterial growth directly, which is fine. However, next the report at first the results of the effect on isdB expression. It would be more logic to show at first the results of the effects of external BCAAs on growth and then on isdB expression.</p>
<p>Supporting Fig. 12: The authors should also test the effects of “no valine” on a single codY-mutants since the growth of the WT is also affected in this medium.</p>
<p>Materials and Methods</p>
<p>Line 617: Usually, the Thoma counting chamber is used to count eukaryotic cells. How should this work to count bacteria? This should be explained in more detail.</p>
<p>Line 626: This sentence should be corrected “where” to “with”? Also, “mins” should be corrected to “min”.</p>
<p>Also, the authors should use the same abbreviations throughout the manuscript: hours or h with or without a free space.</p>
<p>**********</p>
<p>PLOS authors have the option to publish the peer review history of their article (<a href="https://journals.plos.org/plospathogens/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a> ). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a> .</p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: No</p>
<p>Reviewer #3: <strong>Yes: </strong> Barbara C. Kahl</p>
<p>[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link "View Attachments". If this link does not appear, there are no attachment files.]</p>
<p>
<strong>Figure resubmission:</strong>
</p>
<p>While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool, <a href="https://pacev2.apexcovantage.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pacev2.apexcovantage.com/</a>. PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at figures@plos.org. Please note that Supporting Information files do not need this step. If there are other versions of figure files still present in your submission file inventory at resubmission, please replace them with the PACE-processed versions.</p>
<p>
<strong>Reproducibility:</strong>
</p>
<p>To enhance the reproducibility of your results, we recommend that authors of applicable studies deposit laboratory protocols in protocols.io, where a protocol can be assigned its own identifier (DOI) such that it can be cited independently in the future. Additionally, PLOS ONE offers an option to publish peer-reviewed clinical study protocols. Read more information on sharing protocols at <a href="https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols</a></p></section></article><article class="sub-article" id="ppat.1013291.r002"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS Pathog. 2025 Aug 8;21(8):e1013291. doi: <a href="https://doi.org/10.1371/journal.ppat.1013291.r002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.ppat.1013291.r002</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response to Decision Letter 1</h1></hgroup><ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="anp_a.d" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.d" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="anp_a.d" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub2" class="history"><p>Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a.d" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>6 Jun 2025</em>
</p>
<section class="sm xbox font-sm" id="ppat.1013291.s027"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response to Reviewers.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s027.docx" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s027.docx</a><sup> (1.6MB, docx) </sup>
</div></div></section></section></article><article class="sub-article" id="ppat.1013291.r003"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS Pathog. doi: <a href="https://doi.org/10.1371/journal.ppat.1013291.r003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.ppat.1013291.r003</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 1</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prince%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Alice Prince</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Alice Prince</span></h3>
<div>Section Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prince%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alice Prince</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.e" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.e" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.e" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Alice Prince</span></strong>: <span class="role">Section Editor</span>
</div>
</div>
<div id="clp_a.e" class="d-panel p" style="display: none">
<div>© 2025 Alice Prince</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>Dear Dr. Fraunholz,</p>
<p>We are pleased to inform you that your manuscript 'Inactivation of branched-chain amino acid uptake halts Staphylococcus aureus growth and induces bacterial quiescence within macrophages' has been provisionally accepted for publication in PLOS Pathogens.</p>
<p>Before your manuscript can be formally accepted you will need to complete some formatting changes, which you will receive in a follow up email. A member of our team will be in touch with a set of requests.</p>
<p>Please note that your manuscript will not be scheduled for publication until you have made the required changes, so a swift response is appreciated.</p>
<p>IMPORTANT: The editorial review process is now complete. PLOS will only permit corrections to spelling, formatting or significant scientific errors from this point onwards. Requests for major changes, or any which affect the scientific understanding of your work, will cause delays to the publication date of your manuscript.</p>
<p>Should you, your institution's press office or the journal office choose to press release your paper, you will automatically be opted out of early publication. We ask that you notify us now if you or your institution is planning to press release the article. All press must be co-ordinated with PLOS.</p>
<p>Thank you again for supporting Open Access publishing; we are looking forward to publishing your work in PLOS Pathogens.</p>
<p>Best regards,</p>
<p>Alice Prince</p>
<p>Section Editor</p>
<p>PLOS Pathogens</p>
<p>Alice Prince</p>
<p>Section Editor</p>
<p>PLOS Pathogens</p>
<p>Sumita Bhaduri-McIntosh</p>
<p>Editor-in-Chief</p>
<p>PLOS Pathogens</p>
<p>orcid.org/0000-0003-2946-9497</p>
<p>Michael Malim</p>
<p>Editor-in-Chief</p>
<p>PLOS Pathogens</p>
<p>orcid.org/0000-0002-7699-2064</p>
<p>***********************************************************</p>
<p>The authors have done an excellent job responding to the reviewers' suggestions. This is a nice contribution.</p>
<p>Reviewer Comments (if any, and for reference):</p></section></article><article class="sub-article" id="ppat.1013291.r004"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS Pathog. doi: <a href="https://doi.org/10.1371/journal.ppat.1013291.r004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.ppat.1013291.r004</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Acceptance letter</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prince%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Alice Prince</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Alice Prince</span></h3>
<div>Section Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prince%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alice Prince</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.f" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.f" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.f" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Alice Prince</span></strong>: <span class="role">Section Editor</span>
</div>
</div>
<div id="clp_a.f" class="d-panel p" style="display: none">
<div>© 2025 Alice Prince</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>Dear Dr. Fraunholz,</p>
<p>We are delighted to inform you that your manuscript, "Inactivation of branched-chain amino acid uptake halts Staphylococcus aureus growth and induces bacterial quiescence within macrophages," has been formally accepted for publication in PLOS Pathogens.</p>
<p>We have now passed your article onto the PLOS Production Department who will complete the rest of the pre-publication process. All authors will receive a confirmation email upon publication.</p>
<p>The corresponding author will soon be receiving a typeset proof for review, to ensure errors have not been introduced during production. Please review the PDF proof of your manuscript carefully, as this is the last chance to correct any scientific or type-setting errors. Please note that major changes, or those which affect the scientific understanding of the work, will likely cause delays to the publication date of your manuscript. Note: Proofs for Front Matter articles (Pearls, Reviews, Opinions, etc...) are generated on a different schedule and may not be made available as quickly.</p>
<p>Soon after your final files are uploaded, the early version of your manuscript, if you opted to have an early version of your article, will be published online. The date of the early version will be your article's publication date. The final article will be published to the same URL, and all versions of the paper will be accessible to readers.</p>
<p>Thank you again for supporting open-access publishing; we are looking forward to publishing your work in PLOS Pathogens.</p>
<p>Best regards,</p>
<p>Sumita Bhaduri-McIntosh</p>
<p>Editor-in-Chief</p>
<p>PLOS Pathogens</p>
<p>orcid.org/0000-0003-2946-9497</p>
<p>Michael Malim</p>
<p>Editor-in-Chief</p>
<p>PLOS Pathogens</p>
<p>orcid.org/0000-0002-7699-2064</p></section></article><article class="sub-article" id="_ad93_"><section class="pmc-layout__citation font-secondary font-xs"><div></div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Associated Data</h1></hgroup><ul class="d-buttons inline-list"></ul>
<div class="d-panels font-secondary-light"></div>
<div></div>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
</div></section><section class="body sub-article-body"><section id="_adsm93_" lang="en" class="supplementary-materials"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Fig. <em>brnQ1</em> complementation <em>in trans</em> restores growth of <em>S. aureus</em> JE2 <em>brnQ1.</em></span><p>A functional copy of the <em>brnQ1</em> gene, under its native promoter region, (SAUSA300_0188/ SAUSA300_RS00985) was cloned into a high-copy plasmid and used for complementing JE2 <em>brnQ1</em> to yield JE2 <em>brnQ1</em> + p<em>brnQ1</em>. To exclude potential negative effects on bacterial fitness caused by gene overexpression from a high-copy plasmid, constructs carrying the <em>brnQ1</em> gene without promoter sequence were created (termed “P<sub>less</sub>”). <strong>(</strong><strong>a</strong><strong>)</strong> Effect of the complementation plasmid (p<em>brnQ1</em>) or promoter-less <em>brnQ1</em> plasmid (P<sub>less</sub>) on bacterial growth in a chemically defined medium with 1 mM of each BCAA (CDM). OD<sub>600</sub> was determined every 18 min, for 20h. Data are shown as mean values from independent experiments ± SD (n = 3). <strong>(</strong><strong>b</strong><strong>)</strong> Doubling times (T<sub>d</sub>) estimated from the growth curve. Dotted line corresponds to mean T<sub>d</sub> for JE2 WT. Statistical analysis: one-way ANOVA, with Dunnett’s multiple comparisons (vs JE2 WT)<em>; ns = not significant</em>; <em>**p &lt; 0.01.</em></p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s001.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s001.tif</a><sup> (338.5KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S2 Fig. CFU numbers recovered from macrophages at 6h <em>p.i.</em> are comparable among the tested <em>S. aureus</em> strains.</span><p><strong>(a)</strong> Bacteria recovered from infected macrophages at 6h <em>p.i.</em> a Raw CFU counts (CFU/mL) plotted from data set shown in Fig 1d. Data are shown as mean ±SD from independent experiments (JE2 WT and JE2 <em>brn</em>Q1: n = 7, JE2 <em>brn</em>Q1 + p<em>brn</em>Q1 and Cowan I: n = 3). <strong>(</strong><strong>b</strong><strong>)</strong> CFU counts shown in a, normalised to “T0” (i.e., 45 min <em>p.i.</em>). Numbers inside bars represent the plotted value (mean). Statistical analysis (<strong>a</strong>): one-way ANOVA, with Dunnett’s multiple comparisons, vs JE2 WT; <em>ns = not significant</em>.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s002.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s002.tif</a><sup> (343KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S3 Fig. <em>In vitro</em> growth of <em>S. aureus</em> JE2 <em>brnQ1</em> mutant in different culture media.</span><p><em>S. aureus</em> growth in <strong>(a)</strong> the nutrient-rich tryptic soy broth medium (TSB), <strong>(b)</strong> RPMI1640 and <strong>(c)</strong>
<em>Infection Medium</em> (RPMI1640 + 10% v/v heat inactivated FBS). OD<sub>600</sub> was measured every 18 min, for 20h. Data are shown as mean values ± SD from independent experiments: (a) JE2 WT and JE2 <em>brnQ1</em>: n = 5, JE2 <em>brnQ1</em> + p<em>brnQ1</em>: n = 2; (b) JE2 WT and JE2 <em>brnQ1</em>: n = 6, JE2 <em>brnQ1</em> + p<em>brnQ1</em>: n = 4; (c) JE2 WT and JE2 <em>brnQ1</em>: n = 3, JE2 <em>brnQ1</em> + p<em>brnQ1</em>: n = 1.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s003.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s003.tif</a><sup> (618.9KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>S4 Fig. Phagosomal escape of <em>S. aureus</em> JE2 <em>brnQ1</em> mutant in epithelial cells.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> Dynamics of <em>S. aureus</em> infection in professional phagocytes (e.g., macrophages) vs non-professional phagocytes (e.g., epithelial or endothelial cells). Phenol-soluble modulins (<em>psm</em>), the agr quorum sensing system (<em>agr</em>) and the non-ribosomal peptide synthetase AusAB (<em>ausAB</em>) were reported as bacterial factors involved in the phagosomal escape of <em>S. aureus</em> (modified after [<a href="#ppat.1013291.ref007" class="usa-link" aria-describedby="ppat.1013291.ref007">7</a>]). Live cell imaging of reporter <strong>(b)</strong> HeLa cells or <strong>(c)</strong> human bronchial epithelial cells 16HBE14o- constitutively expressing the phagosomal escape marker YFP-CWT, were infected with mRFP-expressing JE2 WT or JE2 <em>brnQ1</em>. Green arrows indicate phagosomal escape events and white arrows indicate intracellular bacterial replication. Shown are micrographs extracted from time-lapse series, representative of 3 independent experiments (n = 3). Scale bars = 10 µm (see also <a href="#ppat.1013291.s018" class="usa-link">S2</a>–<a href="#ppat.1013291.s021" class="usa-link">S5 Movies</a>). (<em>16HBE14o-: immortalized human bronchial epithelial cells</em>).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s004.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s004.tif</a><sup> (5.7MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>S5 Fig. <em>S. aureus brnQ1</em> mutants show delayed phagosomal escape and intracellular replication in non-professional phagocytes.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> Experimental procedure for epithelial cell infections: cells were infected with <em>S. aureus</em> at a multiplicity of infection (MOI) of 5 for 1h, following synchronization by centrifugation. Extracellular <em>S. aureus</em> were removed by a 30 min treatment with 20 µg/µL lysostaphin. Infected cells were further incubated in the presence of 2 µg/mL lysostaphin. “T0” corresponds to 1.5h <em>p.i.</em>
<strong>(</strong><strong>b</strong><strong>)</strong> Phagosomal escape rates of <em>S. aureus</em> JE2 WT and Nebraska Transposon Mutant Library (NTML) <em>S. aureus</em> mutants within several genes associated with either BCAA-uptake (<em>brnQ1-3</em> and <em>bcaP</em>) or biosynthesis (<em>ilvE, ilvD, leuA</em>) were assessed by automated fluorescence microscopy at 4h <em>p.i.,</em> in HeLa YFP-CWT phagosomal escape reporter cells. Phagosomal escape rates of the JE2 WT (percent YFP-CWT positive events of total internalized bacteria) were set to 100% (dotted line). <em>S. aureus</em> Cowan I was used as phagosomal escape-negative control. Data are shown as mean ± SD from independent experiments (n = 2). “NE” designates mutant strain identifier in the NTML. All bacterial strains express the mRFP fluorescent protein. <strong>(</strong><strong>c</strong><strong>)</strong><strong> </strong>Phagosomal escape was quantified in reporter HeLa or 16HBE14o<sup>-</sup> cells expressing the YFP-CWT phagosomal escape marker, by automated fluorescence microscopy, at different time-points, using SNARF-stained bacteria. Data represent the percent YFP-CWT positive events (<em>phagosomal escape</em>) of total internalized bacteria. Data are shown as mean ± SD from independent experiments (n &gt; 3). <strong>(</strong><strong>d</strong><strong>)</strong> Intracellular replication in HeLa and 16HBE14o- epithelial cells. Data are shown as means ± SD from independent experiments (n &gt; 3). <strong>(</strong><strong>e</strong><strong>)</strong> Cytotoxicity of <em>S. aureus</em> infection against HeLa and 16HBE14o- epithelial cells (% LDH release). Data are shown as means ± SD from independent experiments (n &gt; 3). Statistical analysis: (<strong>c</strong>) two-way ANOVA with Dunnett’s multiple comparisons test (each sample vs JE2 WT, for each time-point); (<strong>d</strong>) two-way ANOVA with Dunnett’s multiple comparisons test, using log10-transformed data (each sample vs JE2 WT, for each time-point); (<strong>e</strong>) Kruskal-Wallis test with Dunn’s multiple comparisons test (JE2 WT vs <em>brnQ1</em>, for each time-point); <em>ns = not significant; *p &lt; 0.05; **p &lt; 0.01; ***p &lt; 0.01; ****p &lt; 0.0001</em>. (<em>16HBE14o-: immortalized human bronchial epithelial cells</em>). (<strong>a</strong>) contains icons which were modified (colour and design) from <a href="https://bioicons.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bioicons.com/</a>, as follows: petri-dish-lid-yellow icon by Servier <a href="https://smart.servier.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://smart.servier.com/</a> is licensed under CC-BY 3.0 Unported <a href="https://creativecommons.org/licenses/by/3.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://creativecommons.org/licenses/by/3.0/</a>.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s005.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s005.tif</a><sup> (2MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>S6 Fig. Supernatants of <em>S. aureus</em> JE2 <em>brnQ1</em> grown in TSB are cytotoxic to HeLa cells and JE2 <em>brnQ1</em> exhibits normal haemolysis on sheep blood agar.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> Treatment with 10% sterile stationary phase supernatant (24h, TSB) and its effect on cytotoxicity (% LDH release) against HeLa cells (4h of treatment). <em>S. aureus</em> Cowan I is not cytotoxic against epithelial cells and serves as control. Data are shown as mean values ± SD of independent experiments (JE2 WT and JE2 <em>brnQ1</em>: n = 4, JE2 p<em>brnQ1</em> + p<em>brnQ1</em> and Cowan I: n = 3). Statistical analysis: one-way ANOVA, with Dunnett’s multiple comparisons test, vs JE2 WT; <em>ns = not significant; #=below detection limit, set to “0”</em>. <strong>(</strong><strong>b, c</strong><strong>)</strong><strong> </strong><em>S. aureus</em> JE2 haemolysis on Columbia agar with 5% defibrinated sheep blood, for 30h, at 37°C: <strong>(b)</strong> JE2 WT and <strong>(c)</strong> JE2 <em>brnQ1</em>. Representative image is shown (n &gt; 3).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s006.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s006.tif</a><sup> (2.8MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p">
<span>S7 Fig. Dual RNA-seq of <em>S. aureus</em> JE2<em>-</em> infected macrophages.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> Principal component analysis (PCA) of host transcripts and <strong>(b)</strong> intracellular bacterial transcripts from dual RNA-seq, from 3 independent experiments (n = 3) (<em>hMDM: primary human monocyte-derived macrophages</em>).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s007.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s007.tif</a><sup> (287.7KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p">
<span>S8 Fig. Visualization of major <em>S. aureus</em> regulons that are differentially regulated in intracellular <em>S. aureus</em> JE2 <em>brnQ1</em> compared to JE2 WT.</span><p>Displayed squares represent a gene of the indicated regulon and box coloration was assigned according to the heat map in the lower right (red: upregulated in JE2 <em>brnQ1</em> compared to JE2 WT; blue: downregulated in JE2 <em>brnQ1</em> compared to JE2 WT). Gene-boxes with black frames indicate significance of differential gene expression (padj&lt; 0.05) and absolute values of log2 fold-changes &gt;1.5 or &lt;-1.5. Genes regulated upon bacterial stringent response, such as genes coding for ribosomal proteins or tRNAs, were added to the analysis (framed gene set in the upper right). Gene identifiers correspond to USA300_FPR3757 (old locus ID). Regulons were retrieved from RegPrecise.lbl.gov.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s008.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s008.tif</a><sup> (3.8MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="caption p">
<span>S9 Fig. The <em>S. aureus-</em> containing phagosome is an iron deplete environment.</span><p>Transcriptomes of intracellular JE2 WT in primary human macrophages at 10h <em>p.i.</em> compared to bacteria grown in <em>Infection Medium</em> for 10h. Horizontal lines represent a p-value cutoff &lt; 0.05. Vertical lines represent cutoffs of log<sub>2</sub>fold changes of either &lt;-1.5 or &gt; 1.5. Experiment was run in parallel with experiments shown in Fig 2a-b. Data from Fig 2b were used for comparison and expressed as “intracellular vs <em>Infection Medium”</em> using DESeq2. Transcripts of iron uptake systems, which are prominently differentially regulated in intracellular bacteria vs <em>Infection Medium,</em> are depicted in green. Data are shown from 3 independent experiments (n = 3).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s009.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s009.tif</a><sup> (472.2KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="caption p">
<span>S10 Fig. During growth under iron-replete conditions, <em>S. aureus</em> JE2 <em>fur</em> and JE2 <em>brnQ1</em>/<em>fur</em> upregulate iron uptake mechanisms.</span><p>Total iron (Fe<sup>2+</sup> and Fe<sup>3+</sup>) measured in bacteria grown for 24h in RPMI1640 medium supplemented with 20 µM Fe<sup>2+</sup> (provided as FeSO<sub>4</sub>). Data are shown as mean values ±SD of independent experiments (n = 2).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s010.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s010.tif</a><sup> (174.7KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="caption p">
<span>S11 Fig. <em>Cerulean</em> is inducible by aTC in TSB medium in <em>S. aureus</em> JE2.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> JE2 WT and <strong>(b)</strong> JE2 <em>brnQ1</em> were grown in TSB medium supplemented with 300 ng/mL aTc (concentration used in infection experiments), 150 ng/mL or vehicle control (100% ethanol). OD<sub>600</sub> and fluorescence (Ex/Em <sub>433/504 nm</sub>) were measured every 18 min for 16h. Data are shown as mean values ±SD of independent experiments (n = 3). (AFU = arbitrary fluorescence units, expressed as <em>Cerulean</em>/OD<sub>600</sub> ratio).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s011.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s011.tif</a><sup> (898.1KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="caption p">
<span>S12 Fig. <em>Cerulean</em> signal quantification in intracellular <em>S. aureus</em> JE2.</span><p>Dot plots detailing the quantification of the percentage of <em>Cerulean-</em>expressing bacteria of all bacteria (labelled with Vancomycin BODIPY FL). Individual bacteria were identified by fluorescence microscopy and were labelled as regions -of-interest (ROI). For each ROI, fluorescence mean intensities in <em>Cerulean</em> and BODIPY-FL were measured and were plotted to visualize <em>Cerulean</em> expression of single bacteria after induction at 1h and 10h <em>p.i.</em></p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s012.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s012.tif</a><sup> (1.9MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="caption p">
<span>S13 Fig. Valine and leucine rescue the growth defect of <em>brnQ1</em>- deficient <em>S. aureus</em> in macrophages.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> <em>S. aureus</em> JE2 growth in a chemically defined media, either containing 1 mM of each BCAA (CDM) or lacking the stated amino acid(s). OD<sub>600</sub> was measured every 18 min, for 48h. Data are shown as mean values ±SD of independent experiments (n = 4). <strong>(</strong><strong>b</strong><strong>)</strong> Intracellular growth of a <em>S</em><em>. aureus brnQ1</em> in RAW 264.7 macrophages. Macrophages were infected with either WT USA300 or <em>S. aureus brnQ1</em> and, after gentamicin treatment, supplemented with either valine (Val), leucine (Leu), or Leu and Val (each at 1 mM final concentration) for the duration of the experiment. Data are shown as the mean log10 value ± S.D. for the calculated fold change in CFU/mL at 20h relative to 1.5h <em>p.i.</em>, for each bacterial strain. Each data point plotted represents a biological replicate derived from at least three independent experiments (n ≥ 3). Statistical analysis: Brown-Forsythe and Welch ANOVA with Dunnett’s T3 multiple comparison test. <strong>(</strong><strong>c</strong><strong>)</strong> Fluorescence microscopy micrographs of RAW 264.7 macrophages infected with <em>S. aureus</em> USA300 WT and <em>S. aureus brnQ1</em> mutant expressing GFP (green) that were labeled with a fluorescent proliferation dye (blue). The macrophage plasmalemma and extracellular cocci are stained with TMR-WGA (red). 1 mM Val and Leu (each) were added to the medium after gentamicin treatment (1.5h <em>p.i.</em>). At the outset (i.e., 1.5h) all bacteria are GFP and proliferation dye positive; over time and with replication, GFP-positive yet proliferation dye negative bacteria that are devoid of WGA can be seen at 10h and 20h <em>p.i</em>. White arrows indicate unrestricted bacterial replication (<em>URG</em>). Shown are representative micrographs, from 2 independent experiments (n = 2) (scale bars= ~ 10 µm).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s013.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s013.tif</a><sup> (787.6KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material14_reqid_"><div class="caption p">
<span>S14 Fig. Inactivation of <em>codY</em> rescues the growth of <em>brnQ1-</em>deficient <em>S. aureus</em> in valine-free CDM.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> The ability of <em>S. aureus</em> USA300 WT, a <em>codY</em> mutant (<em>codY::</em>Tn), a <em>brnQ1</em> mutant, a <em>brnQ1/codY</em> double mutant (Δ<em>brnQ1/codY::Tn)</em> and a <em>brnQ1</em>/<em>codY/ilvD</em> triple mutant to grow in Val-free CDM are shown. <strong>(</strong><strong>b</strong><strong>)</strong><strong> </strong>CDM was supplemented with either 1mM Val or <strong>(c)</strong> 0.1% (v/v) tryptone (as a source of peptides) as indicated. Plotted data represent endpoint OD<sub>600</sub> readings, measured at 20h post-inoculation and the bars represent the mean ± SD for the indicated strains in each condition. Individual data points represent independent biological replicates, from three independent experiments (n = 3). Statistical analysis: Kruskal-Wallis with Dunn’s multiple comparisons test performed on the data deriving from each condition (vs USA300 WT); <em>**p &lt; 0.01, ns = not significant.</em></p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s014.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s014.tif</a><sup> (787.6KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material15_reqid_"><div class="caption p">
<span>S15 Fig. Unrestricted growth <em>(URG)</em> of intracellular <em>S. aureus brnQ1</em> mutants during long-term infection of macrophages occurs in a stochastic manner.</span><p>Macrophages were generated from different donors (JE2 <em>brnQ1</em>: n = 4; 6850 <em>brnQ1</em>, SH1000 <em>brnQ1</em> and RN4220 <em>brnQ1</em>: n = 2) and seeded in well plates as technical replicates (at least 3 wells; JE2 <em>brnQ1</em> Donor #1: 72 wells; JE2 Donor #2: 80 wells). Infected plates were observed daily and the number of wells, as well as the time point when <em>URG</em> occurred were recorded and plotted in a Kaplan-Meier survival curve. Each well represents in this case a <em>test subject</em>. Wells infected with the parental wild-type strain served as positive controls.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s015.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s015.tif</a><sup> (925.1KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material16_reqid_"><div class="caption p">
<span>S16 Fig. Protease production is increased in quadruple <em>brnQ1/bcaP/codY/ilvD</em> mutants, yet they fail to overcome macrophage restriction.</span><p><strong>(</strong><strong>a</strong><strong>)</strong> Quantitation of the zones of precipitation around the bacteria grown on SCMA plates for the detection of proteolytic activity shown in <strong>(b)</strong>. Data are shown as mean values ± SD, from independent experiments (n &gt; 3). Statistical analysis: Kruskal-Wallis test with Dunn’s multiple comparisons test; <em>**p &lt; 0.01, ***p &lt; 0.001, ns = not significant.</em> (<em>codY = USA300 codY::Tn; ΔbrnQ1/ΔbcaP = USA300 double deletion mutant of brnQ1 and bcaP; ΔbrnQ1/ΔbcaP/codY</em> = USA300 triple mutant <em>ΔbrnQ1/ΔbcaP/codY::Tn;</em> Δ<em>brnQ1/</em>Δ<em>bcaP/codY/ilvD</em> = <em>USA300 quadruple mutant ΔbrnQ1/ΔbcaP/codY::Tn/ilvD::Tn-Kan</em><sup><em>R</em></sup><em>).</em> Asterisks (magenta) designate strains where <em>unrestricted growth</em> (<em>URG</em>) in macrophages was observed (data shown in <a href="#ppat.1013291.g001" class="usa-link">Fig 1c</a> and <a href="#ppat.1013291.g004" class="usa-link">Fig 4e</a>).</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s016.tif" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s016.tif</a><sup> (1.4MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material17_reqid_"><div class="caption p">
<span>S1 Movie. Live cell imaging of <em>S. aureus</em>-infected primary macrophages. mRFP-expressing <em>S. aureus</em> JE2 WT or JE2 <em>brnQ1</em> (red). Host cell death (green fluorescence) is detected by CellEvent Caspase-3/7 Green Detection Reagent. Scale bar=100 µm.</span><p>(MP4)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s017.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (11MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material18_reqid_"><div class="caption p">
<span>S2 Movie. Phagosomal escape of <em>S. aureus</em> JE2 WT in HeLa cells.</span><p>(red: <em>S. aureus</em>; yellow signal accumulation or ring-like structures: phagosomal escape marker). Shown is a time-lapse movie, representative of 3 independent experiments (n = 3). White frame highlights a region of interest depicted in <a href="#ppat.1013291.s004" class="usa-link">S4b Fig</a>. Scale bar = 50 µm.</p>
<p>(MP4)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s018.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (5MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material19_reqid_"><div class="caption p">
<span>S3 Movie. Phagosomal escape of <em>S. aureus</em> JE2 <em>brnQ1</em> mutants in HeLa cells.</span><p>(red: S. <em>aureus</em>; yellow signal accumulation or ring-like structures: phagosomal escape marker). Shown is a time-lapse movie, representative of 3 independent experiments (n = 3). White frame highlights a region of interest depicted in <a href="#ppat.1013291.s004" class="usa-link">S4b Fig</a>. Scale bar = 50 µm.</p>
<p>(MP4)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s019.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (5.3MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material20_reqid_"><div class="caption p">
<span>S4 Movie. Phagosomal escape of <em>S. aureus</em> JE2 WT in bronchial epithelial cells.</span><p>Live cell imaging of reporter 16HBE14o- cells constitutively expressing the phagosomal escape marker YFP-CWT, infected with mRFP-expressing <em>S. aureus</em> JE2 WT. (red: <em>S. aureus</em>; yellow signal accumulation or ring-like structures: phagosomal escape marker). Shown is a time-lapse movie, representative of 3 independent experiments (n = 3). Scale bar = 50 µm.</p>
<p>(MP4)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s020.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (3.1MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material21_reqid_"><div class="caption p">
<span>S5 Movie. Phagosomal escape of <em>S. aureus</em> JE2 <em>brnQ1</em> mutants in bronchial epithelial cells.</span><p>Live cell imaging of reporter 16HBE14o- cells constitutively expressing the phagosomal escape marker YFP-CWT, infected with mRFP-expressing <em>S. aureus</em>
<em>brnQ1</em>. (red: <em>S. aureus</em>; yellow signal accumulation or ring-like structures: phagosomal escape marker). Shown is a time-lapse movie, representative of 3 independent experiments (n = 3). Scale bar = 50 µm.</p>
<p>(MP4)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s021.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (4.6MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material22_reqid_"><div class="caption p">
<span>S1 Dataset. Expression profiling by high throughput sequencing.</span><p><em>S. aureus</em> JE2 wild-type (WT) or its isogenic JE2 <em>brnQ1</em> mutant were either used to infect primary human macrophages for 10h or were cultivated in <em>Infection Medium</em> (RPMI1640 + 10% v/v heat-inactivated FBS) for 10h. RNA of both pathogen and host cells were isolated together (dual RNA-Seq of infected macrophages) or isolated from planktonic bacteria (<em>Infection Medium</em>) after 10h of infection/cultivation. DESeq2 was used to analyse the following conditions: host transcripts in human macrophages infected with JE2 <em>brnQ1</em> compared to infected with JE2 WT (<em>HUMAN-ic_Mut_vs_ic_WT</em>); bacterial transcripts for intracellular JE2 <em>brnQ1</em> compared to intracellular JE2 WT (<em>BACT-ic-Mut_vs_ic_WT</em>); bacterial transcripts in intracellular bacteria compared to <em>Infection Medium</em> for JE2 <em>brnQ1</em> (<em>BACT-ic_Mut_vs_RPMI_Mut</em>) or JE2 WT (<em>BACT-ic_Mut_vs_RPMI_WT</em>) (<em>HUMAN: human transcripts; BACT: bacterial transcripts; Mut: JE2 brnQ1; WT: JE2 WT; ic: intracellular; RPMI: Infection Medium</em>).</p>
<p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s022.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s022.xlsx</a><sup> (13.9MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material23_reqid_"><div class="caption p">
<span>S1 Table. Bacterial strains used in this study.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s023.pdf" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s023.pdf</a><sup> (303.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material24_reqid_"><div class="caption p">
<span>S2 Table. Plasmids used in this study.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s024.pdf" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s024.pdf</a><sup> (183.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material25_reqid_"><div class="caption p">
<span>S3 Table. Oligonucleotides used in this study.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s025.pdf" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s025.pdf</a><sup> (168.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material26_reqid_"><div class="caption p">
<span>S1 Text. Additional information detailing the role of the BrnQ1 transporter in the interaction of <em>S. aureus</em> with epithelial cells (refers to data shown in S4 and S5 Figs and S2-S5 Movies).</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s026.pdf" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s026.pdf</a><sup> (196.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material27_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response to Reviewers.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12333996/bin/ppat.1013291.s027.docx" data-ga-action="click_feat_suppl" class="usa-link">ppat.1013291.s027.docx</a><sup> (1.6MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>All relevant data are within the manuscript and its Supporting Information files. RNAseq have been deposited in NCBI's Gene Expression Omnibus under GEO Series accession number GSE235929 (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235929" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235929</a>).</p></section></section></article><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS Pathogens are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.ppat.1013291"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ppat.1013291.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12333996/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12333996/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12333996%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12333996/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12333996/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12333996/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40779604/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12333996/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40779604/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12333996/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12333996/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="xDEjcmoQZQQUSH8ZyO0SpncDMBhCHvXfg6CDHY0aBfPKAe89Wz5IcDKWmhuYRwkw">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
