
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Imaging Ligand-Driven PPAR Activities Using Single-Chain Bioluminescent Probes - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="67D7131F89B72E431C131F0029121EA5.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsomega">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332630/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Omega">
<meta name="citation_title" content="Imaging Ligand-Driven PPAR Activities Using Single-Chain Bioluminescent Probes">
<meta name="citation_author" content="Sung-Bae Kim">
<meta name="citation_author_institution" content="Environmental Management Research Institute, National Institute of Advanced Industrial Science and Technology (AIST), 16-1 Onogawa, Tsukuba 305-8569, Japan">
<meta name="citation_author" content="Tadaomi Furuta">
<meta name="citation_author_institution" content="School of Life Science and Technology, Institute of Science Tokyo, B-62 4259 Nagatsuta-cho, Midori-ku, Yokohama 226-8501, Japan">
<meta name="citation_author" content="Genta Kamiya">
<meta name="citation_author_institution" content="Department of Engineering Science, Graduate School of Informatics and Engineering, The University of Electro-Communications, Chofu, Tokyo 182-8585, Japan">
<meta name="citation_author" content="Shojiro A Maki">
<meta name="citation_author_institution" content="Department of Engineering Science, Graduate School of Informatics and Engineering, The University of Electro-Communications, Chofu, Tokyo 182-8585, Japan">
<meta name="citation_author" content="Mariko Orioka">
<meta name="citation_author_institution" content="Department of Applied Chemistry, Faculty of Science and Technology, Keio University, 3-14-1 Hiyoshi, Kohoku-ku, Yokohama, Kanagawa 223-8522, Japan">
<meta name="citation_author" content="Ryuji Watanabe">
<meta name="citation_author_institution" content="Department of Applied Chemistry, Faculty of Science and Technology, Keio University, 3-14-1 Hiyoshi, Kohoku-ku, Yokohama, Kanagawa 223-8522, Japan">
<meta name="citation_author" content="Yuki Hiruta">
<meta name="citation_author_institution" content="Department of Applied Chemistry, Faculty of Science and Technology, Keio University, 3-14-1 Hiyoshi, Kohoku-ku, Yokohama, Kanagawa 223-8522, Japan">
<meta name="citation_author" content="Suresh Thangudu">
<meta name="citation_author_institution" content="Molecular Imaging Program at Stanford, Bio-X Program, Stanford University School of Medicine, Palo Alto, California 94304, United States">
<meta name="citation_author" content="Arutselvan Natarajan">
<meta name="citation_author_institution" content="Molecular Imaging Program at Stanford, Bio-X Program, Stanford University School of Medicine, Palo Alto, California 94304, United States">
<meta name="citation_author" content="Ramasamy Paulmurugan">
<meta name="citation_author_institution" content="Molecular Imaging Program at Stanford, Bio-X Program, Stanford University School of Medicine, Palo Alto, California 94304, United States">
<meta name="citation_publication_date" content="2025 Jul 24">
<meta name="citation_volume" content="10">
<meta name="citation_issue" content="30">
<meta name="citation_firstpage" content="33850">
<meta name="citation_doi" content="10.1021/acsomega.5c04665">
<meta name="citation_pmid" content="40787404">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332630/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332630/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332630/pdf/ao5c04665.pdf">
<meta name="description" content="Nuclear receptors (NRs) play pivotal roles in functionally diverse cell signaling cascades, regulating metabolism and homeostasis. This study introduces a broadly applicable molecular imaging platform for NR activities based on four rationally ...">
<meta name="og:title" content="Imaging Ligand-Driven PPAR Activities Using Single-Chain Bioluminescent Probes">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Nuclear receptors (NRs) play pivotal roles in functionally diverse cell signaling cascades, regulating metabolism and homeostasis. This study introduces a broadly applicable molecular imaging platform for NR activities based on four rationally ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332630/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332630">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsomega.5c04665"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ao5c04665.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332630%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332630/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332630/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332630/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsomega.png" alt="ACS Omega logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Omega" title="Link to ACS Omega" shape="default" href="https://pubs.acs.org/journal/acsodf" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Omega</button></div>. 2025 Jul 24;10(30):33850–33861. doi: <a href="https://doi.org/10.1021/acsomega.5c04665" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsomega.5c04665</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Omega%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Omega%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Imaging Ligand-Driven
PPAR Activities Using Single-Chain
Bioluminescent Probes</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20SB%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Sung-Bae Kim</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sung-Bae Kim</span></h3>
<div class="p">
<sup>†</sup>
Environmental
Management Research Institute, National
Institute of Advanced Industrial Science and Technology (AIST), 16-1 Onogawa, Tsukuba
305-8569, Japan
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20SB%22%5BAuthor%5D" class="usa-link"><span class="name western">Sung-Bae Kim</span></a>
</div>
</div>
<sup>†,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Furuta%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Tadaomi Furuta</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Tadaomi Furuta</span></h3>
<div class="p">
<sup>‡</sup>
School
of Life Science and Technology, Institute of Science Tokyo, B-62 4259 Nagatsuta-cho, Midori-ku, Yokohama
226-8501, Japan
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Furuta%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tadaomi Furuta</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kamiya%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Genta Kamiya</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Genta Kamiya</span></h3>
<div class="p">
<sup>§</sup>
Department
of Engineering Science, Graduate School of Informatics and Engineering, The University of Electro-Communications, Chofu, Tokyo
182-8585, Japan
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kamiya%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Genta Kamiya</span></a>
</div>
</div>
<sup>§</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maki%20SA%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Shojiro A Maki</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Shojiro A Maki</span></h3>
<div class="p">
<sup>§</sup>
Department
of Engineering Science, Graduate School of Informatics and Engineering, The University of Electro-Communications, Chofu, Tokyo
182-8585, Japan
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maki%20SA%22%5BAuthor%5D" class="usa-link"><span class="name western">Shojiro A Maki</span></a>
</div>
</div>
<sup>§</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Orioka%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Mariko Orioka</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Mariko Orioka</span></h3>
<div class="p">
<sup>∥</sup>
Department
of Applied Chemistry, Faculty of Science and Technology, Keio University, 3-14-1 Hiyoshi, Kohoku-ku, Yokohama, Kanagawa
223-8522, Japan
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Orioka%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mariko Orioka</span></a>
</div>
</div>
<sup>∥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Watanabe%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Ryuji Watanabe</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Ryuji Watanabe</span></h3>
<div class="p">
<sup>∥</sup>
Department
of Applied Chemistry, Faculty of Science and Technology, Keio University, 3-14-1 Hiyoshi, Kohoku-ku, Yokohama, Kanagawa
223-8522, Japan
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Watanabe%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ryuji Watanabe</span></a>
</div>
</div>
<sup>∥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hiruta%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Yuki Hiruta</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Yuki Hiruta</span></h3>
<div class="p">
<sup>∥</sup>
Department
of Applied Chemistry, Faculty of Science and Technology, Keio University, 3-14-1 Hiyoshi, Kohoku-ku, Yokohama, Kanagawa
223-8522, Japan
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hiruta%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuki Hiruta</span></a>
</div>
</div>
<sup>∥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thangudu%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Suresh Thangudu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Suresh Thangudu</span></h3>
<div class="p">
<sup>⊥</sup>
Molecular
Imaging Program at Stanford, Bio-X Program, Stanford University School of Medicine, Palo Alto, California
94304, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thangudu%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Suresh Thangudu</span></a>
</div>
</div>
<sup>⊥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Natarajan%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Arutselvan Natarajan</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Arutselvan Natarajan</span></h3>
<div class="p">
<sup>⊥</sup>
Molecular
Imaging Program at Stanford, Bio-X Program, Stanford University School of Medicine, Palo Alto, California
94304, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Natarajan%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Arutselvan Natarajan</span></a>
</div>
</div>
<sup>⊥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Paulmurugan%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Ramasamy Paulmurugan</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Ramasamy Paulmurugan</span></h3>
<div class="p">
<sup>⊥</sup>
Molecular
Imaging Program at Stanford, Bio-X Program, Stanford University School of Medicine, Palo Alto, California
94304, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Paulmurugan%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ramasamy Paulmurugan</span></a>
</div>
</div>
<sup>⊥</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Environmental
Management Research Institute, National
Institute of Advanced Industrial Science and Technology (AIST), 16-1 Onogawa, Tsukuba
305-8569, Japan
</div>
<div id="aff2">
<sup>‡</sup>
School
of Life Science and Technology, Institute of Science Tokyo, B-62 4259 Nagatsuta-cho, Midori-ku, Yokohama
226-8501, Japan
</div>
<div id="aff3">
<sup>§</sup>
Department
of Engineering Science, Graduate School of Informatics and Engineering, The University of Electro-Communications, Chofu, Tokyo
182-8585, Japan
</div>
<div id="aff4">
<sup>∥</sup>
Department
of Applied Chemistry, Faculty of Science and Technology, Keio University, 3-14-1 Hiyoshi, Kohoku-ku, Yokohama, Kanagawa
223-8522, Japan
</div>
<div id="aff5">
<sup>⊥</sup>
Molecular
Imaging Program at Stanford, Bio-X Program, Stanford University School of Medicine, Palo Alto, California
94304, United States
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>kimu-sb@aist.go.jp</span>.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jun 3; Accepted 2025 Jul 11; Revised 2025 Jul 4; Collection date 2025 Aug 5.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332630  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40787404/" class="usa-link">40787404</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Nuclear receptors
(NRs) play pivotal roles in functionally
diverse
cell signaling cascades, regulating metabolism and homeostasis. This
study introduces a broadly applicable molecular imaging platform for
NR activities based on four rationally designed single-chain bioluminescent
probes named <strong>P1</strong>–<strong>P4</strong>. As all the ligand
binding domains (LBDs) of NRs are highly conserved, the probe portfolio
was exemplified using the LBD of peroxisome proliferator-activated
receptor γ (PPARγ-LBD), i.e., the LBD was centrally positioned
and fused with luciferases and/or fluorescent proteins based on the
schemes of the protein-fragment complementation assay (PCA), molecular
strain (MS) probe, circular permutation (CP) probe, and bioluminescence
resonance energy transfer (BRET) system. In the developing process,
we identified the optimal dissection sites for the marine luciferase
RLuc for PCA systems and validated them in in vitro and in vivo studies
in response to various ligands. Among the probes, <strong>P3</strong> and <strong>P4</strong> series probes exhibited strong BL intensities in response
to a PPARγ agonist with signal-to-background ratios of maximally
14-fold. The animal study using furimazine (FMZ) substrate analogs
such as Ad-FMZ showed that the probes can sustain agonist-dependent
BL signals for up to 24 h in animal models. Considering the biological
importance of NRs, the molecular imaging platform with the portfolio
of probes developed in this study can contribute to interrogate many
NR-related cell signaling pathways by replacing the PPARγ-LBD
with LBDs from other NRs in the probes.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332630_ao5c04665_0008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/34d8/12332630/26332d14df4b/ao5c04665_0008.jpg" loading="lazy" id="abs1" height="280" width="680" alt="graphic file with name ao5c04665_0008.jpg"></a></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332630_ao5c04665_0006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/34d8/12332630/26332d14df4b/ao5c04665_0006.jpg" loading="lazy" id="tgr1" height="280" width="680" alt="graphic file with name ao5c04665_0006.jpg"></a></p></section><section id="sec1"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Nuclear receptors (NRs)
are one of the most abundant classes of
transcriptional factors acting as molecular switches in regulating
the cellular signaling pathway in normal and pathological conditions.
NRs provide a direct link between signaling molecules that regulate
homeostasis, reproduction, development, and metabolism.<em>
,
</em>
</p>
<p>NRs consist of three functionally distinct domains called
the N-terminal
domain (NTD), DNA-binding domain (DBD), and ligand-binding domain
(LBD). The NTD is variable among NRs, whereas the LBD is highly conserved. In addition, the LBD is mainly responsible for
the key roles in recognizing the endogenous ligand as a first responder
of the signal transduction pathways. This unique feature of the LBD
inspired us to create a portfolio of four rationally designed single-chain
bioluminescent probes based on the LBD of NRs. Considering that the
size and functions of the LBD of NRs are similar to each other, we
speculated that a portfolio of probes based on a particular NR could
generalize the applicability of the probes for other NR-LBD-based
molecular imaging probes.</p>
<p>As a representative of NRs, we paid
attention to peroxisome proliferator-activated
receptor (PPAR), which controls a great number of gene networks involved
in adipogenesis, lipid metabolism, inflammation, and maintenance of
metabolic homeostasis. PPAR has three
isoforms, PPARα, PPARβ, and PPARγ, each containing
several domains including the LBD, i.e., PPAR-LBD. The ligand-induced
conformational changes in the PPAR-LBD control the downstream regulatory
functions via its homo- and hetero dimerizations with retinoid X receptor
(RXR) and by recruiting coactivators.<em>
,
</em> This PPAR–RXR
dimerization was previously investigated using a bioluminescence resonance
energy transfer (BRET) assay. Because
of diverse functions in the biological activities, PPAR has been persistently
investigated as a drug target in various diseases. Some PPAR agonists have been approved as clinical agents
for the treatment of diabetes and dyslipidemia.<em>
,
</em> Thus,
it is important to have a proper evaluation of PPAR activities in
vivo and in vitro for identifying potential ligands with no or minimal
adverse activities using high-throughput assays before they can be
used for clinical applications.</p>
<p>Conventionally, NR activities
have been determined preclinically
using mammalian two-hybrid assays, where the NR-LBD is fused to the
GAL4-DNA binding domain of yeast and VP16 transactivator domain of
HSV1 (GAL4-DBD-LBD-VP16). This fusion protein is designed to convert
ligand activities of NRs eventually to the expression of a reporter
protein (luciferase) codelivered with the system. Thus, the ligand activity is converted to the expression
levels of a reporter protein, such as firefly luciferase. The expression
levels are finally determined using luciferin, a specific substrate
for luciferase. NR activities have also been investigated using time-resolved
fluorescence resonance energy transfer (TR-FRET) system for screening
potential NR ligands, where the NR-LBD is fused to GAL4-DBD.<em>
−

</em> All the methods generally require a long stimulation time, typically
12–24 h, to accumulate sufficient amount of reporter proteins
to quantify and decode NR activity. This feature limits the determination
of the temporal dynamics of molecular events of interest by the receptor
and its respective ligand. Moreover, the detectable molecular events
are confined only to nuclear proteins by using this approach.</p>
<p>Alternatively, ligand-activated NR–coactivator interaction
has been indexed with various cell-free assay systems such as immunohistochemistry,
surface plasmon resonance (SPR), and real-time quantitative reverse
transcriptase polymerase chain reaction (RT-qPCR).<em>
,
</em> However, these assays are simply designed to represent a snapshot
of a complex biological process under artificially designed experimental
conditions. Further, the results do not reflect the complexity of
the contexts in cells within living subjects.</p>
<p>We previously
demonstrated the fundamental strategies for imaging
molecular events in live mammalian cells using “single-chain”
bioluminescence (BL) probes.<em>
−


</em> These single-chain probes are uniquely designed by
integrating both functions of ligand recognition and optical reporter
within a single fusion protein. We have applied for determining biological
activities of steroid hormones such as androgen, estrogen, and cortisol
upon their interactions with respective receptors.<em>
−

</em> We also invented
a concept of “molecular strain probes” for indexing
molecular strains appended on marine luciferases by ligand-activated
protein–protein interactions (PPIs), where a full-length luciferase
is sandwiched between two proteins of interest with the use of small
flexible linkers.<em>
,
</em> However, these probes can conceptually
determine only the antiparallel PPIs, since the protein partners assembled
within the single-chain probe can only interact antiparallelly. Another
problem with the single-chain strain probe is the background BL intensities,
which are relatively high because full-length luciferases are embedded
in the probe backbone, where high backgound signals are developed
by the reporter under normal physiological conditions.</p>
<p>In the
present study, we leveraged the great physiological impact
of NRs and developed a portfolio of four representative molecular
imaging probes based on the ligand-induced conformational changes,
coactivator recruitment, and receptor homo- and heterodimerizations.
The probes were contructed by centering NR-LBD on tandemly modified
luciferase and/or a fluorescent protein. We designed the probes based
on the schemes of protein-fragment complementation assay (PCA), molecular
strain (MS) probe, circular permutation (CP) probe, and BRET system,
as illustrated in <a href="#fig1" class="usa-link">Figure 
</a>A. We exemplified the utility of the portfolio with the PPARγ-LBD,
which is useful for imaging the intracellular molecular activities
of NRs in vitro and in vivo. In the probe developing process, we have
specified the insights used to develop “single-chain”
probes through optimizing (i) the dissection sites of <em>Renilla</em> luciferase (RLuc) for the protein-fragment complementation-based
probes and (ii) the interface regions between the domains of the resonance
energy transfer-based probes for improving the signal-to-background
(S/B) ratios. The ligand-activated conformation changes of PPARγ-LBD
and the corresponding working mechanism of the probes are computationally
modeled to address the rational design. We finally elucidated in vivo
imaging of the probe in animals using xenografts expressing the optical
probes. Considering the impact of NRs as a triggering mechanism in
many important molecular events in cells, the present study guides
on how to image the ligand-activated cell signaling pathways of NRs
in cells and in animal models.</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332630_ao5c04665_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/34d8/12332630/c4d8b8ebee46/ao5c04665_0001.jpg" loading="lazy" id="gr1" height="694" width="666" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Working mechanism of the four single-chain
probes in response
to agonist stimulation. Ligand-induced conformational changes in the
PPARγ ligand-binding domain (LBD) trigger one of the following
events: (i) complementation of split luciferases in Group 1 (PCA probe)
and Group 3 (CP probe); (ii) development of molecular tension that
affects the sandwiched luciferase in Group 2 (MS probe); and (iii)
increased BRET efficiency due to the close proximity between the luciferase
and fluorescent protein in Group 4 (BRET probe). (B) Conformational
changes in PPARγ-LBD upon agonist binding (structures: PDB <a href="https://www.rcsb.org/structure/2PRG" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2PRG</a> and <a href="https://www.rcsb.org/structure/6C5T" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6C5T</a>). The N-terminal
region (putative α-helix) and C-terminal helix 12 (H12) are
shown in blue and red, respectively. The chemical structure of the
PPARγ agonist GW1929 is shown on the right. (C) Schematic representation
of cDNA constructs encoding single-chain PPARγ probes (<strong>P1</strong>–<strong>P4</strong>). Group 1 (PCA) and Group 3 (CP)
probes utilize split-luciferase fragments (ALuc16 and RLuc8.6-535,
respectively). Group 2 (MS) includes a full-length ALuc23 luciferase
sandwiched between PPARγ-LBD and SH2. Group 4 (BRET) consists
of PPARγ-LBD inserted between luciferase (NanoLuc or RLuc86SG)
and fluorescent proteins (mNeptune or iRFP).</p></figcaption></figure></section><section id="sec2"><h2 class="pmc_sec_title">2. Materials and Methods</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">2.1. Materials</h3>
<p>Native
coelenterazine (nCTZ)
was purchased from Nanolight (Pinetop, AZ, USA). The CTZ analogs,
T1, T2, TS1, and TS2, were obtained from our previous study, while coelenterazine-<em>h</em> (CTZh)
was purchased from FujiFilm Wako Pure Chemical Co. (Osaka, Japan).
The furimazine (FMZ) analogs, pivaloyl-FMZ (Piv-FMZ) and 1-adamantanecarbonyl-FMZ
(Ad-FMZ), were from our previous studies.<em>
,
</em> The ligands of PPARγ such as GW1929, rosiglitazone, pioglitazone
hydrochloride, and rosiglitazone maleate were obtained from FujiFilm
Wako Pure Chemical Co. The cDNA constructs of various marine luciferases
cloned in pcDNA3.1­(+) vector backbone were obtained from our previous
studies, i.e., <em>Renilla</em> luciferase 8.6-535 (RLuc8.6-535), <em>Renilla</em> luciferase 8.6-535SG (RLuc86SG), Artificial luciferase
16 (ALuc16, GenBank <a href="https://www.ncbi.nlm.nih.gov/nuccore/MF817967" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MF817967</a>), and Artificial luciferase 23 (ALuc23;
GenBank <a href="https://www.ncbi.nlm.nih.gov/nuccore/MF817968" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MF817968</a>).<em>
,
</em>
</p>
<p>The cDNAs encoding human
PPARγ-LBD (193–475 AA or 204–475 AA; Genbank <a href="https://www.ncbi.nlm.nih.gov/nuccore/X90563" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">X90563</a>),
the Src homology-2 (SH2) domain derived from <em>v</em>-Src
(150–248 AA), mNeptune C158D, near-infrared fluorescent protein (iRFP), and the Ornithine decarboxylase-PEST sequence were custom-synthesized by Eurofins Genomics
(Tokyo, Japan) based on the sequence information available on the
public database, National Center for Biotechnology Information (NCBI).</p>
<p>BALB/c nude mice (female, 6 weeks old, ca. 16 g) were obtained
from Charles River (Wilmington, MA). The Administrative Panel on Laboratory
Animal Care of Stanford University approved all procedures using laboratory
animals used in this study, and all experiments were conducted in
accordance with the Guidelines for the Care and Use of Laboratory
Animals (APLAC-26748, Approval Date: September 11, 2023).</p></section><section id="sec2.2"><h3 class="pmc_sec_title">2.2. Modeling of the Molecular Structure of PPARγ-LBD</h3>
<p>We obtained PPARγ-LBD structure information from the AlphaFold
database (<a href="https://alphafold.ebi.ac.uk" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://alphafold.ebi.ac.uk</a>, AF-<a href="https://www.ncbi.nlm.nih.gov/protein/P37231" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P37231</a>). The N-terminal region was rotated to make contacts
with the helix 9 (H9) as it is not bound to DNA. The agonist was placed
based on the crystal structure (PDB: <a href="https://www.rcsb.org/structure/2PRG" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2PRG</a>). The unbound
structure used in this study was based on the antagonist-bound structure
(PDB: <a href="https://www.rcsb.org/structure/6C5T" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6C5T</a>) with the above N-terminal region that was manually
placed.</p></section><section id="sec2.3"><h3 class="pmc_sec_title">2.3. Construction of a Series of Plasmids Encoding
Single-Chain Probes to Image PPARγ Activities</h3>
<p>The single-chain
probes categorized into Groups 1–4 in <a href="#fig1" class="usa-link">Figure 
</a>A were generated by a tandem linkage of cDNA
fragments, as shown in <a href="#fig1" class="usa-link">Figure 
</a>B,C: in Group 1 (<strong>P1</strong>), the cDNA fragments encoding
the N- and C-terminal domains of ALuc16 (19–129 AA for N-ALuc16
and 130–212 AA for C-ALuc16) were generated by polymerase chain
reaction (PCR) to introduce the specific dissection sites <em>Hind</em>III/<em>Kpn</em>I and <em>Bam</em>HI/<em>Xho</em>I at the 5′ and 3′ ends, respectively.
Separately, the cDNA fragment encoding PPARγ-LBD (193–475
AA) comprising a glycine-serine (GS) linker sequence was synthesized
by PCR to introduce the specific dissection sites <em>Kpn</em>I/<em>Bam</em>HI at the 5′ and 3′ ends, respectively.
After PCR, the cDNA blocks were sequentially digested with the respective
sets of restriction enzymes, purified, ligated, and subcloned into
the appropriately digested pcDNA3.1­(+) vector backbone (Invitrogen,
Carlsbad, CA, USA).</p>
<p>The design of the <strong>P1</strong> probe
was inspired by an original single-chain probe concept based on the
″protein-fragment complementation assay (PCA)″. In this
design, a full-length luciferase was dissected into N- and C-terminal
fragments, between of which PPARγ-LBD was sandwiched. In the
presence of an agonist, the PPARγ-LBD undergoes an internal
conformation change and results in the reconstitution of the adjacent
N- and C-terminal fragments. Consequently, the BL activities are ligand-dependently
resurrected. In the <strong>P1</strong> design, the N- and C-terminal
fragments of ALuc16 (19–129 and 130–212 AA) were used
because these fragments were one of the most successful fragments
for PCA based on our previous study.
</p>
<p>The Group 2 (<strong>P2</strong>) version of probe was created from
the idea of nongenomic activities of steroid hormones using “molecular
strain (MS)” probes.<em>
,
</em> For the construction
of plasmid vector, cDNA fragment encoding PPARγ-LBD (193–475
AA) comprising a GS linker sequence at the C-terminus with the unique
restriction enzyme sites such as <em>Hind</em>III and <em>Bam</em>HI at the 5′ and 3′ ends, respectively,
was generated by PCR. In parallel, cDNA fragments encoding full-length
ALuc23 and the SH2 domain of <em>v</em>-Src (150–248
AA) with unique restriction enzyme sites, <em>Bam</em>HI/<em>Kpn</em>I and <em>Kpn</em>I/<em>Xho</em>I, at
the 5′ and 3′ ends, respectively, were synthesized by
PCR. The PCR-amplified cDNA blocks were stepwise digested with the
respective sets of restriction enzymes, purified, ligated, and subcloned
into the corresponding restriction enzyme-digested pcDNA3.1­(+) vector
backbone (Invitrogen) to construct a <strong>P2</strong> probe (<a href="#fig1" class="usa-link">Figure 
</a>C).</p>
<p>It was
previously reported that ligand-activated PPARγ-LBD
is phosphorylated and binds the SH2 domain of coactivators including <em>v</em>-Src. Hence, we designed the <strong>P2</strong> version probe to exploit this PPI mechanism and constructed
the single-chain probe carrying PPARγ-LBD and SH2 domain flanking
the full-length ALuc23 on either side. The hinge region tyrosine of
α-Helix 12 of PPARγ-LBD is phosphorylated and recognized
by the adjacent SH2 domain. This intramolecular PPI elevates the BL
intensity by appending strain to ALuc23 (<a href="#fig1" class="usa-link">Figure 
</a>A).</p>
<p>In the Group 3 version of probes
(<strong>P3</strong>), the cDNA fragments
encoding the dissection pairs of RLuc8.6-535 were generated by PCR
as stated previously to introduce the unique restriction enzyme sites <em>Bam</em>HI/<em>Age</em>I and <em>Age</em>I/<em>Kpn</em>I at the 5′ and 3′ ends, respectively,
as shown in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04665/suppl_file/ao5c04665_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S1</a>. In the fabrication
of the cDNA construct for <strong>P3.2</strong>, the cDNA templates encoding
the α-helical LXXLL and LXXII motifs were obtained from our
precedent study, where L, I, and X mean
leucine, isoleucine, and any amino acid, respectively. After PCR amplification,
the cDNA fragments were digested stepwise with the respective sets
of restriction enzymes, purified, ligated, and subcloned into the
corresponding restriction enzyme-digested pcDNA3.1­(+) vector backbone
(Invitrogen) to construct a set of <strong>P3</strong> probes.</p>
<p>The <strong>P3</strong> probes were designed based on the “circular
permutation (CP)” method, which essentially combined ideas
from the <strong>P1</strong> probe and our previous glucocorticoid receptor
(GR) probe. In this strategy, RLuc8.6-535
was dissected into N- and C-terminal fragments, which were deployed
in the C- and N-terminal order and fused via a GS linker to create
circular permutated RLuc8.6-535. These luciferase fragments were then
sandwiched between the LXXLL motif and PPARγ-LBD. In the presence
of the ligand agonist, PPARγ-LBD binds the LXXLL motif. This
intramolecular folding enhances the BL intensities of reconstituted
RLuc8.6-535. We prepared two different versions of <strong>P3</strong> probes: one named <strong>P3.1</strong> was constructed without the
LXXII consensus motif of corepressors of NRs, while the other one
named <strong>P3.2</strong> was with the LXXII motif. We attempted the
consensus motif “LXXII” of corepressors of NRs with
the hope of improving the signal-to-background (S/B) ratios through
reducing the background intensities derived from nonspecific binding
between the probe components as speculated in our previous study.
</p>
<p>For the construction of Group 4 (<strong>P4</strong>) probes, cDNA
fragments encoding a fluorescent protein (mNeptune (C158D) or iRFP)
and a marine luciferase (NanoLuc or RLuc86SG) were created by PCR
to introduce the specific dissection sites, <em>Hind</em>III/<em>Bam</em>HI and <em>Kpn</em>I/<em>Xho</em>I, at
the 5′ and 3′ ends, respectively. In parallel, a cDNA
fragment encoding PPARγ-LBD or GS-linker-fused PPARγ-LBD
was generated by PCR to make the unique dissection sites, <em>Bam</em>HI/<em>Kpn</em>I, at the 5′ and 3′
ends, respectively. After PCR amplification, the cDNA fragments were
stepwise digested with the respective sets of restriction enzymes,
purified, ligated, and subcloned into the corresponding restriction
enzyme-digested pcDNA3.1­(+) vector backbone (Invitrogen) to construct
the <strong>P4</strong> version of probes.</p>
<p>The <strong>P4</strong> probes
were inspired by the basic design of
our previous reported concept of BRET probes.<em>
,
</em> The key components of <strong>P4</strong> probes are luciferase and
fluorescent protein as the resonance energy donor and acceptor, respectively,
between which PPARγ-LBD was inserted. In the <strong>P4</strong>-series probes upon agonistic ligand-mediated activation, PPARγ-LBD
transforms through conformational changes of H12 and enhances the
BRET efficiency by proximally positioning the fluorescent and luciferase
proteins.</p>
<p>The primers used for the PCR reactions were listed
in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04665/suppl_file/ao5c04665_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>. We tested the fidelity of
the cDNA
constructs by a genetic sequence analyzer (Eurofin Genomics) before
using them for further experiments.</p></section><section id="sec2.4"><h3 class="pmc_sec_title">2.4. Characterization
of the Imaging Probes (P1–P4)
for Studying PPARγ Activities</h3>
<p>All versions of the probes
were characterized in mammalian cells according to the following method.
We used COS-7 cells grown in Dulbecco’s modified Eagle’s
medium (DMEM) supplemented with 10% (v/v) FBS and 1% (v/v) P/S in
6-well microplates. We transiently transfected the cells with pcDNA3.1­(+)
vector encoding <strong>P1</strong>, <strong>P2</strong>, <strong>P3.1</strong>, or <strong>P3.2</strong> probes. Twenty-four h post transfection, we
subcultured the cells to a 96-well black-wall clear bottom microplate
to study the ligand-induced activation of the probes. The cells in
the wells were stimulated overnight using 10<sup>–6</sup> M
GW1929 or this vehicle (0.1% (v/v) DMSO dissolved in PBS). The culture
media in the wells were carefully eliminated, and the remaining cells
in the wells were simultaneously immersed with nCTZ dissolved in PBS.
The corresponding BL intensities from the live cells were immediately
determined with an IVIS Spectrum imaging system (PerkinElmer, Waltham,
MA, USA) and analyzed with Living Image software version 4.7, as shown
in <a href="#fig2" class="usa-link">Figure 
</a>A. The S/B
ratios were calculated by dividing the BL intensity of the ligand-stimulated
cells (signal) with that without ligand stimulation (background).</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332630_ao5c04665_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/34d8/12332630/eb061fc25c17/ao5c04665_0002.jpg" loading="lazy" id="gr2" height="1003" width="729" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Comparison
of the sensor properties of probes <strong>P1</strong>, <strong>P2</strong>, <strong>P3.1</strong>, and <strong>P3.2</strong>. Error
bars indicate the standard deviation from the mean of triplicate measurements
(<em>n</em> = 3). The upper-right panel (Inset <em>a</em>) displays BL images before and after ligand stimulation. The bottom-right
panel (Inset <em>b</em>) illustrates the proposed working
mechanism of <strong>P3.2</strong> upon ligand binding. (B) Model structure
of the dissection sites of RLuc8.6-535 suitable for PPARγ-LBD
probes. The α-helices and β-sheets are highlighted in
red and green, respectively (left panel). The right panel depicts
the corresponding working mechanism of the <strong>P3</strong>-series
probes. (C) BL intensities of a series of PPARγ probes containing
different N- and C-terminal fragment pairs of RLuc8.6-535. The <em>X</em>-axis numbers represent the dissection sites of RLuc8.6-535
within each probe. The numbers on the bars indicate signal-to-background
(S/B) ratios after ligand addition. Error bars indicate the standard
deviation from triplicate measurements (<em>n</em> = 3). The
left panel (Inset <em>a</em>) shows a proposed working mechanism
of the probes in response to ligand stimulation. The right panel (Inset <em>b</em>) displays the corresponding BL images of the probe set.</p></figcaption></figure><p>As <strong>P3.1</strong> showed the highest absolute
BL intensity and
S/B ratio in response to the PPAR<strong>γ</strong> agonist, whereas
probes <strong>P1</strong> and <strong>P2</strong> (Groups 1 and 2) did not
elevate the BL signal, and we further investigated the optimal dissection
site for RLuc8.6-535 inside the <strong>P3.1</strong> probe, as illustrated
in <a href="#fig2" class="usa-link">Figure 
</a>B,C.</p>
<p>COS-7 cells were grown in DMEM supplemented with 10% (v/v) FBS
and 1% (v/v) P/S in 6-well microplates for transfection experiments.
The cells were then transiently transfected with pcDNA3.1­(+) vector
encoding each <strong>P3</strong>-series of probes developed with the
N- and C-terminal fragments of RLuc8.6-535 at different dissection
sites. Twenty-four h post transfection, the cells were subcultured
into a 96-well black-wall clear bottom microplate. The cells in the
wells were overnight stimulated using 10<sup>–6</sup> M GW1929
or this vehicle. The cells were lysed in 40 μL of lysis buffer
(Promega) by incubating for 15 min at room temperature. The cell lysates
were simultaneously mixed with 40 μL of nCTZ dissolved in PBS
using a 12-channel microplate, and the corresponding BL images were
immediately determined using an IVIS Spectrum optical imaging system
(PerkinElmer).</p></section><section id="sec2.5"><h3 class="pmc_sec_title">2.5. Characterization of P4-Series
of BRET Probes
for Imaging PPARγ Activities</h3>
<p>The optical properties
of the <strong>P4</strong>-series of BRET probes were determined in the
presence or absence of an agonist GW1929 (<a href="#fig3" class="usa-link">Figure 
</a>). We used COS-7 cells transfected with plasmid
vectors encoding probes <strong>P4.1</strong>–<strong>P4.6</strong> by following the same protocol stated above in <a href="#fig2" class="usa-link">Figure 
</a>A. The cells in the wells of
96-well black-frame microplates were stimulated with the vehicle (0
h) or 10 μM of GW1929 for 1, 5, and 16 h. The culture media
in the wells were removed, and the wells were simultaneously treated
with 40 μL of nCTZ dissolved in PBS using a 12-channel micropipette.
The corresponding BL images were determined with the IVIS Spectrum
imaging system (PerkinElmer) and analyzed with Living Image software
version 4.7.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332630_ao5c04665_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/34d8/12332630/3768662593ef/ao5c04665_0003.jpg" loading="lazy" id="gr3" height="699" width="666" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Development of <strong>P4</strong>-series BRET probes (<strong>P4.1–P4.6</strong>) based on PPARγ-LBD. (A) Illustration
of the proposed working
mechanism of a BRET probe. Upon agonist binding, the single-chain
probe binds the specific ligand and undergoes a conformational change,
enhancing BRET efficiency between the approximated mNeptune (mNep)
and NanoLuc (NLuc). The bottom panel (Inset <em>a</em>) shows
representative BL images with and without agonist stimulation. (B)
Characterization of <strong>P4</strong>-series BRET probes with varying
molecular designs and stimulation times. Error bars represent the
standard deviation from the mean of quadruplicate measurements (<em>n</em> = 4). The bottom panel (Inset <em>a</em>) shows
the corresponding BL images of the BRET probes. The numbers next to
the probe names indicate the specific amino acid numbers of the PPARγ-LBD
and the presence of the GS linker. (C) Evaluation of the sensor properties
of BRET probes <strong>P4.5</strong> and <strong>P4.6</strong> in response
to ligand stimulation. The left panel (inset <em>a</em>) shows
the optical intensities with and without GW1929. Error bars represent
the standard deviation from quadruplicate measurements (<em>n</em> = 4). Statistical significance was assessed using a Student’s <em>t</em>-test (<strong>
<em>p</em>
</strong> ≤ 0.01).
The right panel (inset <em>b</em>) demonstrates an application
of probes <strong>P4.5</strong> (white wells) and <strong>P4.6</strong> (gray
wells) in forming a visible pattern resembling handwritten letters.</p></figcaption></figure></section><section id="sec2.6"><h3 class="pmc_sec_title">2.6. Determination of the Sensor
Performance of <strong>P3.1</strong> and <strong>P4.1</strong> according to
CTZ Analogs</h3>
<p>We also tested specificity of various CTZ analogs
in the sensor performance
using probes <strong>P3.1</strong> and <strong>P4.4</strong> stably expressed
in COS-7 cells (<a href="#fig4" class="usa-link">Figure 
</a>). We cultured COS-7 cells in the DMEM complete medium in 10 cm culture
dishes. The cells were transiently transfected with pcDNA3.1­(+) vector
encoding the probes <strong>P3.1</strong> or <strong>P4.4</strong>, where
the Neomycin ORF in the original vector backbone was replaced with
puromycin antibiotic-coding gene. The cells were incubated for 24
h in a CO<sub>2</sub> incubator (5% (v/v) CO<sub>2</sub> with 95%
(v/v) O<sub>2</sub>), and then, the culture media were replaced with
fresh medium containing 1 μg/mL puromycin. The puromycin-resistant
clones were selected by culturing for 2 weeks and used for further
experiments.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332630_ao5c04665_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/34d8/12332630/4d3d148ee752/ao5c04665_0004.jpg" loading="lazy" id="gr4" height="579" width="666" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effects of CTZ analogs on the sensor performance of <strong>P3.1</strong> and <strong>P4.4</strong>. (A) BL intensities of <strong>P3.1</strong> with
and without GW1929 stimulation. The BL signals were developed with
the CTZ analogs. The numbers on the bar graphs indicate the S/B ratios.
Error bars represent the standard deviation (SD) from triplicate measurements
(<em>n</em> = 3). The right panel (inset <em>a</em>) shows the corresponding BL image. (B) BL spectra of <strong>P4.4</strong> in the presence of various CTZ analogs. The left and middle panels
show the absolute and normalized spectra, respectively. In each spectrum,
the dotted and solid lines indicate the absence and presence of GW1929
stimulation in cells expressing <strong>P4.4</strong>, respectively. The
right panel (inset <em>b</em>) presents the chemical structures
of the CTZ analogs.</p></figcaption></figure><p>COS-7 cells stably expressing <strong>P3.1</strong> or <strong>P4.4</strong> were seeded in a 96-well microplate
and incubated
overnight. The
cells in the wells were then stimulated overnight using 10<sup>–6</sup> M GW1929 or its vehicle (0.1% (v/v) DMSO). The wells were then conceptually
divided into two sections: The culture media in the first section
were eliminated completely, and the remaining cells were lysed with
40 μL of the lysis buffer by incubating for 20 min, whereas
the wells in the second section remained sealed for preventing evaporation
after completely decanting the culture media. Separately, a series
of substrate solutions were prepared by dissolving the stocks of nCTZ
or its analogs (CTZh, T1, T2, TS1, or TS2) in PBS buffer to a final
concentration of 100 μM. The wells were then simultaneously
injected with 40 μL of the substrate solutions using a 12-channel
micropipette, and the corresponding BL images were obtained using
the IVIS Spectrum imaging system and analyzed with Living Image software
version 4.7.</p>
<p>In parallel, the live COS-7 cells containing <strong>P4.4</strong> probe
according to CTZ analogs were prepared as shown in the above paragraphs
of this section, and the corresponding BL spectra were determined
by the following method. The live cells were harvested by trypsinization
and centrifugation. The precipitated cells were resuspended with a
fresh PBS buffer and counted with an automatic cell counter (Countess
3, Thermo Fisher Scientific). The cell number was finally adjusted
to 2 × 10 <sup>6</sup> /mL. Forty μL of the cell solution
was deployed into 200 μL PCR tubes, and the corresponding BL
spectra were measured with a precision spectrophotometer (AB-1850,
ATTO, Tokyo, Japan) (integration time: 30 s) immediately after injection
of 40 μL of each substrate solution.</p></section><section id="sec2.7"><h3 class="pmc_sec_title">2.7. BL Imaging
of GW1929 Activity with <strong>P4.4</strong> in Living Mice</h3>
<p>We used BALB/C mice implanted with cells
expressing the <strong>P4.4</strong> probe for in vivo imaging of PPARγ
activity in response to agonist GW1929. The BL images of mice bearing
xenografts of the <strong>P4.4</strong> probe were monitored after administration
of caged luciferins and PPARγ agonist GW1929 (<a href="#fig5" class="usa-link">Figure 
</a>).</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332630_ao5c04665_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/34d8/12332630/322b881ec02f/ao5c04665_0005.jpg" loading="lazy" id="gr5" height="420" width="697" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Long-term BL imaging
of ligand-stimulated PPARγ activity
in living mice. (A) Schematic illustration of the in vivo mouse study
design and the chemical structures of the caged luciferins used for
long-term imaging. (B) Time-course BL imaging of mice bearing xenografts
expressing the <strong>P4.4</strong> probe following administration of
the substrate Piv-FMZ and optional injection of the ligand GW1929.
The upper panel shows representative mouse images over time (preinjection
and 30–240 min postinjection). The lower panel (inset <em>a</em>) presents the corresponding ex vivo images of harvested
organs. The bar graph on the right shows the time course BL intensities
of the xenograft sites measured in vivo over time. Error bars represent
the standard deviation (SD) from triplicate measurements (<em>n</em> = 3). (C) Time-course BL imaging of mice bearing xenografts
expressing the <strong>P4.4</strong> probe following administration of
Ad-FMZ. The upper panel shows representative mouse images over time
(preinjection and 30 to 240 min postinjection). The lower panel (inset <em>a</em>) presents the corresponding ex vivo images of harvested
organs. The bar graph on the right shows the BL intensities of each
ROI measured in vivo over time. Error bars represent the SD from triplicate
measurements (<em>n</em> = 3).</p></figcaption></figure><p>COS-7 cells stably expressing the <strong>P4.4</strong> probe were
implanted into female BALB/c mice (5–7 weeks old). To image
the tumor sites in vivo, the mice were randomly divided into two groups.
The left tumors received PBS, while the right tumors were intratumorally
(<em>i.t.</em>) injected with 20 μL of GW1929 (1 μM).
The two groups were administered intraperitoneally (<em>i.p.</em>) with 100 μL of a caged luciferin, Piv-FMZ or Ad-FMZ in PBS,
respectively (<em>n</em> = 3 per group). These animals were
anesthetized with 2% (v/v) isoflurane in (1 L/min oxygen), and the
BL images were acquired at various time points using the LagoX-Spectra
optical imaging system (Spectral Instruments Imaging, Tucson, AZ,
USA), i.e., at 5 min, 30 min, 1, 2, and 4 h post ligand injection
(30 s integration per image). The images were analyzed using Living
Image software, version 4.7. Regions of interest (ROIs) were drawn
around each xenograft site, and the total number of photons within
each ROI was recorded. The ROI position and size were held constant
across all of the images.</p>
<p>To analyze the ex vivo images, after
2 h postinjection (<em>p.i.</em>) of the substrates <em>i.t.</em> or intravenously
(<em>i.v.</em>), the mouse organs of each group were exercised
and imaged over time, and the corresponding images were analyzed using
the same Living Image software as described above.</p></section></section><section id="sec3"><h2 class="pmc_sec_title">3. Results and Discussion</h2>
<section id="sec3.1"><h3 class="pmc_sec_title">3.1. Design of Single-Chain
Probes for Imaging
Ligand-Driven PPARγ Activities</h3>
<p>To date, we have devoted
systematic efforts in creating unique single-chain bioluminescent
imaging systems for studying various PPIs in cells.<em>
,,,
</em> In this study,
for the first time, we applied the basic concepts of single-chain
probes to image ligand-driven PPARγ activities in mammalian
cells and animal models, as illustrated in <a href="#fig1" class="usa-link">Figure 
</a>A,B. The probe designs and working mechanisms
have been briefly described in <a href="#sec2.3" class="usa-link">Section 
</a> of the Methods.</p></section><section id="sec3.2"><h3 class="pmc_sec_title">3.2. Characterization
of Single-Chain Probes for
Studying PPARγ Activities</h3>
<p>We determined the absolute
BL intensities and the S/B ratios of probes <strong>P1</strong>, <strong>P2</strong>, <strong>P3.1</strong>, and <strong>P3.2</strong> in the presence
or absence of the PPARγ agonist GW1929 (<a href="#fig2" class="usa-link">Figure 
</a>A). The highest BL intensities were found
with <strong>P3.1</strong>, and its S/B ratio was ca. 2.9, whereas no
significant BL signals were observed with the probes <strong>P1</strong> and <strong>P2</strong> in both the presence and absence of GW1929.</p>
<p>It is difficult to pinpoint the exact reason for the poor performance
of the <strong>P1</strong> and <strong>P2</strong> probes in measuring ligand-driven
PPARγ activities, although similar molecular designs reported
in our previous studies have successfully measured ohter steroid hormone
activities.<em>
−

</em> This may suggest that the ligand-activated PPARγ-LBD
adopts a unique conformation, which could be distinct from those of
other NRs such as the androgen receptor (AR) and estrogen receptor
(ER), potentially requiring optimal linker lengths and amino acid
sequences different from those used in our previous probe designs.
However, the primary objective of this study was to develop a broadly
applicable molecular imaging platform for NR activities with optimal
designs and linkers. For instance, the relative position and distance
between the N- and C-termini of the PPARγ-LBD upon ligand activation
may be unique and different from those observed in the LBDs of the
other NRs. Even minor variations in the linker length can markedly
influence the sensor performance of <strong>P1</strong> and <strong>P2</strong>. Therefore, the overall performance of <strong>P1</strong> and <strong>P2</strong> could likely be improved through further optimization of
probe designs, including adjustments to linker lengths, component
arrangement, and interface regions of the PPARγ-LBD.</p></section><section id="sec3.3"><h3 class="pmc_sec_title">3.3. Characterization of Optimal Dissection Sites
for the P3-Series Probes</h3>
<p>To optimize PCA-based single-chain
probes (<strong>P3</strong>-series), we investigated suitable dissection
sites within RLuc8.6-535, which are essential for improving both S/B
ratios and absolute optical intensities of the probes. A series of
N- and C-terminal fragments of RLuc8.6-535 derived from various dissection
sites were deployed between the LXXLL motif and PPARγ-LBD (<a href="#fig2" class="usa-link">Figures 
</a>B,C and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04665/suppl_file/ao5c04665_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S1</a>).</p>
<p>We tested the functionality of the
probes in transiently transfected COS-7 cells upon exposure to PPARγ
agonist GW1929. The results showed a significant level of BL intensities
from the <strong>P3</strong>-series of probes carrying the N- and C-terminal
fragments based on the dissection sites of RLuc8.6-535 at 209/210
AA; 225/226 AA; 228/229 AA; and 245/246 AA. The highest BL intensities
were produced from the probe carrying the N- and C-terminal fragments,
which were dissected at 228/229 AA, followed by the pair dissected
at 209/210 AA. Interestingly, the distance between the dissection
sites was 17 AA (<a href="#fig2" class="usa-link">Figure 
</a>C). We named the single-chain probe carrying the N- and C-terminal
fragments of RLuc8.6-535 dissected at 228 and 229 AA as <strong>P3.1</strong>.</p>
<p>It is noteworthy that the four probes constructed using the
dissection
sites at 209/210, 225/226, 229/230, and 245/246 with high BL intensities
are located in the loop region of RLuc8.6-535 as shown in <a href="#fig2" class="usa-link">Figure 
</a>B. In contrast, the
dissection sites of the low intensity probes are mostly located within
secondary structures positioned within α-helices and β-strand
(with the exception of 249/250) of RLuc8.6-535. The overall results
can be explained with the statement that disruption of the secondary
structures of the luciferase protein can destabilize the enzyme structure
upon it being located around the active site, resulting in the poor
reconstitution of the reporter activity. We also noted that the split
sites 209/210 and 228/229 each harbor glycine residues (G210 and G229),
whose inherent flexibility and neutrality are likely to contribute
to the superior BL intensities by providing a margin for the approximation
of the split interface.</p>
<p>Although the PCA-based single-chain
probes showcase a bioassay
based on NR-LBD, the overall S/B ratios and optical performance fell
short of our expectations. Hence, we further designed BRET-based probes
as a new architecture based on the ligand-induced structural changes
to NR-LBD.</p></section><section id="sec3.4"><h3 class="pmc_sec_title">3.4. Characterization of P4-Series
BRET Probes
for Imaging PPARγ Activities</h3>
<p>Furthermore, we tested
the sensor properties of the BRET-based <strong>P4</strong>-series of
probes in COS-7 cells (<a href="#fig3" class="usa-link">Figure 
</a>). The sensor properties of probes <strong>P4.1</strong> through <strong>4.4</strong> were evaluated after exposure to ligand GW1929 for different
time points. The strongest S/B ratios appeared after overnight stimulation
with GW1929, followed by a 5 h exposure. Although the BRET probes
themselves reflect real-time PPIs, the ligand GW1929 contains hydrophilic
functional groups (p<em>K</em>
<sub>a</sub> 3.91), and it takes
a long time to pass through the cell membrane of animal cells to show
the intended ligand-mediated NR activation. This is why many previous
publications used such long stimulation times such as 9 h and 24 h in their
experimental protocols with GW1929. The present study also showed
the highest effect with the longest stimulation time; however, it
also showed efficient early activation.</p>
<p>On the other hand, the
probe <strong>P4.3</strong> showed slightly enhanced background BL intensities
with the poor S/B ratios. We observed the BRET signal after screening
using a 640 nm bandpass filter, which is the expected emission peak
of mNeptune C162D in <strong>P4</strong>-series probes. Among the various
P-series probes, <strong>P4.2</strong> and <strong>P4.4</strong> showed the
highest S/B ratios (<a href="#fig3" class="usa-link">Figure 
</a>B).</p>
<p>We further constructed <strong>P4.5</strong> and <strong>P4.6</strong> through
tagging a PEST sequence or replacing the fluorescent protein and luciferase
with iRFP and RLuc86SG, respectively, to the <strong>P4.4</strong> probe.
The PEST sequence was introduced to the probes to promote proteasomal
degradation, shortening probe half-life and lowering the background
intensity. iRFP and RLuc86SG were chosen because of their red-shifted
optical properties. Interestingly, the agonist facilitated a reduction
in the BL intensities of probe <strong>P4.5</strong> by half of the negative
control, whereas the same agonist boosted the BL intensities of probe <strong>P4.6</strong> by 1.5-fold (<a href="#fig3" class="usa-link">Figure 
</a>C). Ultimately, <strong>P4.4</strong> was selected for
the following experiments in <a href="#fig4" class="usa-link">Figure 
</a> because it showed the highest S/B ratio and the strongest
optical intensities.</p></section><section id="sec3.5"><h3 class="pmc_sec_title">3.5. CTZ Analogs Significantly
Modulate the Sensor
Performance of <strong>P3.1</strong> and <strong>P4.4</strong> Probes</h3>
<p>To evaluate substrate contribution to the sensor performance, we
tested <strong>P3.1</strong> (PCA probe) and <strong>P4.4</strong> (BRET probe)
using a range of coelenterazine (CTZ) analogs in both live cells and
lysates (<a href="#fig4" class="usa-link">Figure 
</a>).
The overall BL intensities of probe <strong>P3.1</strong> were higher
in live cells than in lysates. In live cells, CTZh yielded the highest
absolute BL intensity, followed by T1 and nCTZ. The lowest BL intensity
was recorded for <strong>P3.1</strong> probes with substrates TS1 and
TS2 (<a href="#fig4" class="usa-link">Figure 
</a>A). These
results indicate that the split-luciferase based on RLuc8.6-535 positioned
within the <strong>P3.1</strong> probe shows the best enzymatic performance
with substrate CTZh. This result aligns with our previous report where
we have observed the strongest BL intensities with substrate CTZh
for RLuc86SG, compared to the same with other CTZ analogs. The chemical structure of CTZh also explains
that the highest BL intensity is partly contributed by the fast diffusion
of CTZh into the live cells, facilitated by (i) the absence of the
hydroxy group (OH) at the C-2 position (high hydrophobicity) and (ii)
the absence of the double bond at the C-6 position of the CTZh backbone
(low bulkiness).</p>
<p>The overall S/B ratios appeared to be predominantly
influenced by variance in the baseline intensities in the absence
of a ligand. T1 and nCTZ delivered the highest S/B ratios (∼6.8
fold) for <strong>P3.1</strong> due to reduced background signal (0.3
× 10<sup>6</sup> p/s/cm<sup>2</sup>/sr) in live cells, whereas
CTZh, despite high BL, showed lower S/B (∼2.0 fold) owing to
elevated baseline (2.3 × 10<sup>6</sup> p/s/cm<sup>2</sup>/sr).</p>
<p>It remains challenging to fully elucidate why the S/B ratios are
diverse, according to the substrate–probe <strong>P3.1</strong> combinations. The S/B ratios as a consequence of the substrate–probe <strong>P3.1</strong> reaction can be influenced by many factors, e.g., the
functional groups of each substrate determine (i) the specificity
to the luciferase present within the probes, (ii) the hydrophobicity
of the substrate facilitating permeability across the plasma membrane,
and (iii) the steric hindrance upon substrate interaction with the
luciferase in the probes. All the intrinsic factors of the substrate
collectively determine the unique strength of the signal intensities
and the S/B ratios of the substrate–<strong>P3.1</strong> pairs.</p>
<p>We also tested the BL spectral properties of the BRET probes. In <a href="#fig4" class="usa-link">Figure 
</a>B, we show that the
peak positions and intensities of the BL spectra of <strong>P4.4</strong> probe are heavily influenced by CTZ analogs. The strongest BL spectrum
for the probe was obtained with the substrate CTZh followed by S5
and nCTZ. The spectral peaks obtained for probe <strong>P4.4</strong> with
nCTZ and CTZh were in blue wavelengths, 468 and 473 nm, respectively.
The most red-shifted peak of 507 nm was obtained for the same probe
with the substrate S5. The wavelength gap between nCTZ and S5 (around
39 nm) is attributable to the presence of the sulfur (S) ankle at
the C-8 position that is the only unique feature of substrate S5 compared
to nCTZ. Hence, the red-shifted BL spectrum of the <strong>P4.4</strong> probe with S5 can be interpreted as the spectral shift solely contributed
by the extended π-conjugation of the sulfur at the C-8 position.
This result corresponds with those of the previous study using S-series
of CTZ analogs.
</p>
<p>The <strong>P4</strong>-series BRET probes are conceptually different
from a conventional NanoBRET platform using a NanoLuc-labeled target
protein or a split-NanoLuc-labeled protein pair.<em>
,
</em> In the present probes, the target protein (PPARγ-LBD) is designed
to be sandwiched between a fluorescent protein and a luciferase (i.e.,
an integrated form), whereas the NanoBRET probes make use of simply
NanoLuc-labeled protein(s) (i.e., two separated molecule form). These
two designsintegrated versus separatedoffer complementary
advantages depending on the PPI context under investigation.</p></section><section id="sec3.6"><h3 class="pmc_sec_title">3.6. In Vivo Imaging of the <strong>P4.4</strong> BRET
Probe in Response to the Treatment of Agonist GW1929 in a Mouse Model</h3>
<p>We further conducted in vivo imaging of the GW1929-driven PPARγ
activity in COS-7 xenografts expressing the <strong>P4.4</strong> probe.
We used specific substrates such as Piv-FMZ and Ad-FMZ, where Piv
and Ad groups serve as protecting groups, which were introduced at
the C-3 position of FMZ via an ether linkage (<a href="#fig5" class="usa-link">Figure 
</a>). FMZ was originally optimized for NanoLuc,
but we used it in this study because of its small size and compatibility
with the protecting groups.</p>
<p>We used mice bearing tumor xenografts
on both lower flanks of the mouse for this study. After <em>i.p.</em> administration of Piv-FMZ, the left tumor injected with PBS did
not show any increase in the BL signal, whereas the right tumor that
received <em>i.t.</em> injection of the GW1929 ligand exhibited
long-lasting and high BL signals (<a href="#fig5" class="usa-link">Figure 
</a>B). In the right tumor sites, the BL intensity
significantly increased at 30 min <em>p.i.</em> and peaked
at 120 min <em>p.i.</em> and became progressively weaker after
240 min. A similar trend was observed with another substrate, Ad-FMZ
(<a href="#fig5" class="usa-link">Figure 
</a>C). ROI measurements
of the BL signals at each time point for the mouse groups are presented
as bar graphs adjacent to the corresponding mouse image panels. These
data clearly indicated that Ad-FMZ is a better substrate than Piv-FMZ
with respect to signal stability and durability; specifically, Ad-FMZ
displays approximately 2-fold higher BL signals at all the measurement
time points <em>p.i</em>. with GW1929 than with PBS. Ad-FMZ
shows considerable BL intensity even after 24 h <em>p.i.</em>
</p>
<p>It is interesting to discuss why Ad-FMZ is more durable than
Piv-FMZ.
Although both protective groups, Ad and Piv, are linked to the same
imidazopyrazinone backbone of FMZ via an ester linker, Ad group is
more bulky and lipophilic than Piv group and easily stabilized by
binding albumin as shown in refs <em>
 and 
</em>. These chemical properties should delay the hydrolysis of Ad-FMZ
in animal models compared to the case of Piv-FMZ and thus result in
the longer durability of Ad-FMZ than that of Piv-FMZ in animal models.</p>
<p>At the early time points (within 5 min), Piv-FMZ exhibited a high
auto luminescence baseline. As discussed in previous studies, this
elevated baseline is likely attributable to the initial instability
of the imidazopyrazinone backbone in the serum and plasma. The acute
and nonspecific oxidation of this backbone generates BL signals that
decay quickly, known as not conducive to prolonged real-time imaging
in vitro and in vivo.<em>
,
</em> Stabilization of the imidazopyrazinone
structure can be achieved by introducing protecting groups at the
reaction site, such as Ad-FMZ and Piv-FMZ. However, the Piv group
offers less steric protection effects compared to the Ad group. This
chemical difference reasonably explains the higher auto luminescence
baseline observed for Piv-FMZ, particularly at the early time points
in the in vivo experiments.</p>
<p>One may consider that animal experiments
using only 6 mice (<em>n</em> = 3) may have an insufficient
sample size and can affect
the reliability of the conclusions. However, in most probe imaging
studies, we observed a minimum variability between animal imaging
results. Hence, we used fewer numbers in this study to address the
use of these probes for in vivo imaging applications. Although preclinical
experiments are often conducted with five mice to achieve statistical
significance, in this study, we utilized a minimal number of mice
(<em>n</em> = 3) to draw reasonable conclusions.</p>
<p>We
obtained corresponding ex vivo images of tumors 24 h <em>p.i</em>. to determine the potential distribution of tumor cells
and the biodistribution of the substrates (<a href="#fig5" class="usa-link">Figure 
</a>B inset “<em>a</em>”
(the bottom panel) and <a href="#fig5" class="usa-link">Figure 
</a>C inset “<em>a</em>” (the bottom panel)).
The BL images indicated that BL signals corresponding to the tumors
were weak, but upon activation by GW1929, the tumors showed a brighter
signal compared to the one activated by PBS. We did not observe any
nonspecific signal due to substrate autoluminescence in other tissues
and organs, which is common with FMZ and its analogs.</p>
<p>The ex
vivo results support the following conclusion: (i) the in
vivo BL signals are specific to the activation by the ligand GW1929
compared to the negative controls; (ii) the <em>i.t.</em> injection
of ligand GW1929 does not result in significant nonspecific activation
in the contralateral tumor; and (iii) the two FMZ analogs (Piv-FMZ
and Ad-FMZ) are useful for imaging with a long circulation time of
up to 24 h while preserving activity specific to the luciferase expressed
by the tumor cells. We consider that the reason for the nondiffused
nature of GW1929 could be partially due to the ligand formulation
(using mineral oil and PEG400) used for the study.</p>
<p>Overall,
the in vivo imaging study clearly demonstrated that the <strong>P4.4</strong> probe provides an improved BL intensity that can be imaged
by the use of long-acting substrates such as Ad-FMZ for up to 24 h
without reinjection. This <strong>P4.4</strong>-based in vivo imaging
provides an orthogonal methodology to directly assess ligand-activated
PPARγ activities and offers advantages over conventional in
vivo imaging methods using a Fluorine-18-labeled PPARγ ligand and a PPAR-targeting carrier with fluorescent
Cyanine 5.5 particularly in terms of
quantification and phototoxicity-free imaging.</p></section></section><section id="sec4"><h2 class="pmc_sec_title">4. Conclusion</h2>
<p>NRs are one of the most
abundant classes of transcription factors
present in mammalian systems, acting as molecular switches that directly
connect signaling molecules. Hence, the NR-based imaging probes facilitate
preclinical investigations of the molecular events in cells under
both normal and abnormal metabolic conditions.</p>
<p>In the present
study, we developed a molecular imaging platform
with four differently designed “single-chain” bioluminescent
probes that can be readily adapted for other molecular imaging applications.
We demonstrated four different categories of unique molecular imaging
probes in vitro and in vivo. We reported efficient molecular imaging
probes such as <strong>P3.1</strong> and <strong>P4.4</strong> that can assay
ligand-driven PPARγ activity with high sensitivity and specificity.
In the developing process, we identified the prominent dissection
sites for RLuc that interestingly appear at every 17 AA distance in
the AA sequence. We also found that the <strong>P4</strong> series of
probes efficiently generate optical signals whose S/B ratios are up
to 14-fold. This study also demonstrated long-term in vivo imaging
of ligand-driven PPARγ activity in living mice using the optical
probes and novel long-acting substrates Piv-FMZ and Ad-FMZ. The entire
probe development process above provides guidance on how to develop
efficient single-chain BL probes based on NRs.</p>
<p>As noted in the
introduction, NR activity has conventionally been
assessed using mammalian two-hybrid assays and SPR. BRET systems and
other probe platforms have also been developed but are validated in
limited case studies. Given the importance of NR-LBDs, the assay platform
developed in this study demonstrates how to bioassay the NR-related
molecular events in cell signaling pathways under various physiological
conditions, including in living animal models. Given that the size
and function of NR LBDs are highly conserved, researchers can readily
develop their own probe portfolios by simply replacing PPARγ-LBD
with other LBDs, including isoforms such as PPARα-LBD and PPARβ-LBD.
In addition to the general significance of NRs, PPARs have been especially
investigated as a drug target in various diseases. For example, several
PPAR agonists are currently being used as clinical agents for the
treatment of such as diabetes and dyslipidemia. The assay portfolio
developed in this study contributes to the rapid preclinical evaluation
of PPAR ligand activity both in vivo and in vitro prior to potential
clinical applications.</p></section><section id="sec18"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12332630/bin/ao5c04665_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c04665_si_001.pdf</a><sup> (900.9KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>This
work was partly supported by JSPS KAKENHI and
JST SPRING Grants: Numbers 23KK0101, 23K17389, 25H01207, 25K22883,
and JPMJSP2131.</p></section><section id="notes1"><p>The
Supporting Information is
available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsomega.5c04665?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsomega.5c04665</a>.</p>
<ul id="silist" class="list" style="list-style-type:none"><li><p>Schematic diagrams of
the N- and C-terminal fragments
of RLuc8.6-535; relative BL intensities of COS-7 cells carrying <strong>P4.1</strong> after administration of various PPARγ agonists;
time course of the BL intensities of the xenograft sites of mice <em>p.i.</em> of the substrates, FMZ, Piv-FMZ, or Ad-FMZ; and primers
used in the construction of the plasmids encoding the probes <strong>P2</strong>–<strong>P4</strong> (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04665/suppl_file/ao5c04665_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes2"><p>The authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Magner D. B., Antebi A.. Caenorhabditis elegans nuclear receptors: insights
into life traits. Trends Endocrinol. Metab. 2008;19(5):153–160. doi: 10.1016/j.tem.2008.02.005.</cite> [<a href="https://doi.org/10.1016/j.tem.2008.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2744080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18406164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Endocrinol.%20Metab.&amp;title=Caenorhabditis%20elegans%20nuclear%20receptors:%20insights%20into%20life%20traits&amp;author=D.%20B.%20Magner&amp;author=A.%20Antebi&amp;volume=19&amp;issue=5&amp;publication_year=2008&amp;pages=153-160&amp;pmid=18406164&amp;doi=10.1016/j.tem.2008.02.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Weikum E. R., Liu X., Ortlund E. A.. The nuclear
receptor superfamily: A structural perspective. Protein Sci. 2018;27(11):1876–1892. doi: 10.1002/pro.3496.</cite> [<a href="https://doi.org/10.1002/pro.3496" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6201731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30109749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&amp;title=The%20nuclear%20receptor%20superfamily:%20A%20structural%20perspective&amp;author=E.%20R.%20Weikum&amp;author=X.%20Liu&amp;author=E.%20A.%20Ortlund&amp;volume=27&amp;issue=11&amp;publication_year=2018&amp;pages=1876-1892&amp;pmid=30109749&amp;doi=10.1002/pro.3496&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Ahmadian M., Suh J. M., Hah N., Liddle C., Atkins A. R., Downes M., Evans R. M.. PPARgamma
signaling and metabolism:
the good, the bad and the future. Nat. Med. 2013;19(5):557–566. doi: 10.1038/nm.3159.</cite> [<a href="https://doi.org/10.1038/nm.3159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3870016/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23652116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&amp;title=PPARgamma%20signaling%20and%20metabolism:%20the%20good,%20the%20bad%20and%20the%20future&amp;author=M.%20Ahmadian&amp;author=J.%20M.%20Suh&amp;author=N.%20Hah&amp;author=C.%20Liddle&amp;author=A.%20R.%20Atkins&amp;volume=19&amp;issue=5&amp;publication_year=2013&amp;pages=557-566&amp;pmid=23652116&amp;doi=10.1038/nm.3159&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Tyagi S., Gupta P., Saini A., Kaushal C., Sharma S.. The peroxisome
proliferator-activated receptor: A family of nuclear receptors role
in various diseases. J. Adv. Pharm. Technol.
Res. 2011;2(4):236–240. doi: 10.4103/2231-4040.90879.</cite> [<a href="https://doi.org/10.4103/2231-4040.90879" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3255347/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22247890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Adv.%20Pharm.%20Technol.%20Res.&amp;title=The%20peroxisome%20proliferator-activated%20receptor:%20A%20family%20of%20nuclear%20receptors%20role%20in%20various%20diseases&amp;author=S.%20Tyagi&amp;author=P.%20Gupta&amp;author=A.%20Saini&amp;author=C.%20Kaushal&amp;author=S.%20Sharma&amp;volume=2&amp;issue=4&amp;publication_year=2011&amp;pages=236-240&amp;pmid=22247890&amp;doi=10.4103/2231-4040.90879&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Wang T. T., Xu J., Yu X. F., Yang R. C., Han Z. C.. Peroxisome proliferator-activated
receptor gamma in malignant diseases. Crit.
Rev. Oncol. Hematol. 2006;58(1):1–14. doi: 10.1016/j.critrevonc.2005.08.011.</cite> [<a href="https://doi.org/10.1016/j.critrevonc.2005.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16388966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crit.%20Rev.%20Oncol.%20Hematol.&amp;title=Peroxisome%20proliferator-activated%20receptor%20gamma%20in%20malignant%20diseases&amp;author=T.%20T.%20Wang&amp;author=J.%20Xu&amp;author=X.%20F.%20Yu&amp;author=R.%20C.%20Yang&amp;author=Z.%20C.%20Han&amp;volume=58&amp;issue=1&amp;publication_year=2006&amp;pages=1-14&amp;pmid=16388966&amp;doi=10.1016/j.critrevonc.2005.08.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Mulero M., Perroy J., Federici C., Cabello G., Ollendorff V.. Analysis of
RXR/THR and RXR/PPARG2 Heterodimerization by Bioluminescence Resonance
Energy Transfer (BRET) PLoS One. 2013;8(12):e84569.  doi: 10.1371/journal.pone.0084569.</cite> [<a href="https://doi.org/10.1371/journal.pone.0084569" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3877338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24391967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Analysis%20of%20RXR/THR%20and%20RXR/PPARG2%20Heterodimerization%20by%20Bioluminescence%20Resonance%20Energy%20Transfer%20(BRET)&amp;author=M.%20Mulero&amp;author=J.%20Perroy&amp;author=C.%20Federici&amp;author=G.%20Cabello&amp;author=V.%20Ollendorff&amp;volume=8&amp;issue=12&amp;publication_year=2013&amp;pages=e84569&amp;pmid=24391967&amp;doi=10.1371/journal.pone.0084569&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Xi Y., Zhang Y. H., Zhu S. R., Luo Y. P., Xu P. F., Huang Z. Y.. PPAR-Mediated Toxicology
and Applied Pharmacology. Cells. 2020;9(2):352. doi: 10.3390/cells9020352.</cite> [<a href="https://doi.org/10.3390/cells9020352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7072218/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32028670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=PPAR-Mediated%20Toxicology%20and%20Applied%20Pharmacology&amp;author=Y.%20Xi&amp;author=Y.%20H.%20Zhang&amp;author=S.%20R.%20Zhu&amp;author=Y.%20P.%20Luo&amp;author=P.%20F.%20Xu&amp;volume=9&amp;issue=2&amp;publication_year=2020&amp;pages=352&amp;pmid=32028670&amp;doi=10.3390/cells9020352&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Boeckmans J., Natale A., Rombaut M., Buyl K., Rogiers V., De Kock J., Vanhaecke T., Rodrigues R. M.. Anti-NASH
Drug Development Hitches a Lift on PPAR Agonist. Cells. 2020;9(1):37. doi: 10.3390/cells9010037.</cite> [<a href="https://doi.org/10.3390/cells9010037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7016963/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31877771/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=Anti-NASH%20Drug%20Development%20Hitches%20a%20Lift%20on%20PPAR%20Agonist&amp;author=J.%20Boeckmans&amp;author=A.%20Natale&amp;author=M.%20Rombaut&amp;author=K.%20Buyl&amp;author=V.%20Rogiers&amp;volume=9&amp;issue=1&amp;publication_year=2020&amp;pages=37&amp;pmid=31877771&amp;doi=10.3390/cells9010037&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Seimandi M., Lemaire G., Pillon A., Perrin A., Carlavan I., Voegel J. J., Vignon F., Nicolas J. C., Balaguer P.. Differential
responses of PPAR alpha, PPAR delta, and PPAR gamma reporter cell
lines to selective PPAR synthetic ligands. Anal.
Biochem. 2005;344(1):8–15. doi: 10.1016/j.ab.2005.06.010.</cite> [<a href="https://doi.org/10.1016/j.ab.2005.06.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16038868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Biochem.&amp;title=Differential%20responses%20of%20PPAR%20alpha,%20PPAR%20delta,%20and%20PPAR%20gamma%20reporter%20cell%20lines%20to%20selective%20PPAR%20synthetic%20ligands&amp;author=M.%20Seimandi&amp;author=G.%20Lemaire&amp;author=A.%20Pillon&amp;author=A.%20Perrin&amp;author=I.%20Carlavan&amp;volume=344&amp;issue=1&amp;publication_year=2005&amp;pages=8-15&amp;pmid=16038868&amp;doi=10.1016/j.ab.2005.06.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Shi Y. H., Hon M., Evans R. M.. The peroxisome
proliferator-activated receptor δ,
an integrator of transcriptional repression and nuclear receptor signaling. Proc. Natl. Acad. Sci. U.S.A. 2002;99(5):2613–2618. doi: 10.1073/pnas.052707099.</cite> [<a href="https://doi.org/10.1073/pnas.052707099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC122396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11867749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=The%20peroxisome%20proliferator-activated%20receptor%20%CE%B4,%20an%20integrator%20of%20transcriptional%20repression%20and%20nuclear%20receptor%20signaling&amp;author=Y.%20H.%20Shi&amp;author=M.%20Hon&amp;author=R.%20M.%20Evans&amp;volume=99&amp;issue=5&amp;publication_year=2002&amp;pages=2613-2618&amp;pmid=11867749&amp;doi=10.1073/pnas.052707099&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Nevin D. K., Peters M. B., Carta G., Fayne D., Lloyd D. G.. Integrated
Virtual Screening for the Identification of Novel and Selective Peroxisome
Proliferator-Activated Receptor (PPAR) Scaffolds. J. Med. Chem. 2012;55(11):4978–4989. doi: 10.1021/jm300068n.</cite> [<a href="https://doi.org/10.1021/jm300068n" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22582973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=Integrated%20Virtual%20Screening%20for%20the%20Identification%20of%20Novel%20and%20Selective%20Peroxisome%20Proliferator-Activated%20Receptor%20(PPAR)%20Scaffolds&amp;author=D.%20K.%20Nevin&amp;author=M.%20B.%20Peters&amp;author=G.%20Carta&amp;author=D.%20Fayne&amp;author=D.%20G.%20Lloyd&amp;volume=55&amp;issue=11&amp;publication_year=2012&amp;pages=4978-4989&amp;pmid=22582973&amp;doi=10.1021/jm300068n&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Malek M. A., Hoang M. H., Jia Y. Y., Lee J. H., Jun H. J., Lee D. H., Lee H. J., Lee C., Lee M. K., Hwang B. Y.. et al. Ombuin-3-O-β-d-glucopyranoside from Gynostemma
pentaphyllum is a dual agonistic ligand of peroxisome proliferator-activated
receptors α and δ/β. Biochem.
Biophys. Res. Commun. 2013;430(4):1322–1328. doi: 10.1016/j.bbrc.2012.12.020.</cite> [<a href="https://doi.org/10.1016/j.bbrc.2012.12.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23247028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;title=Ombuin-3-O-%CE%B2-d-glucopyranoside%20from%20Gynostemma%20pentaphyllum%20is%20a%20dual%20agonistic%20ligand%20of%20peroxisome%20proliferator-activated%20receptors%20%CE%B1%20and%20%CE%B4/%CE%B2&amp;author=M.%20A.%20Malek&amp;author=M.%20H.%20Hoang&amp;author=Y.%20Y.%20Jia&amp;author=J.%20H.%20Lee&amp;author=H.%20J.%20Jun&amp;volume=430&amp;issue=4&amp;publication_year=2013&amp;pages=1322-1328&amp;pmid=23247028&amp;doi=10.1016/j.bbrc.2012.12.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Cuzzocrea S., Pisano B., Dugo L., Ianaro A., Maffia P., Patel N. S. A., Di Paola R., Ialenti A., Genovese T., Chatterjee P. K.. et al. Rosiglitazone, a ligand
of the peroxisome proliferator-activated
receptor-gamma, reduces acute inflammation. Eur. J. Pharmacol. 2004;483(1):79–93. doi: 10.1016/j.ejphar.2003.10.056.</cite> [<a href="https://doi.org/10.1016/j.ejphar.2003.10.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14709329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pharmacol.&amp;title=Rosiglitazone,%20a%20ligand%20of%20the%20peroxisome%20proliferator-activated%20receptor-gamma,%20reduces%20acute%20inflammation&amp;author=S.%20Cuzzocrea&amp;author=B.%20Pisano&amp;author=L.%20Dugo&amp;author=A.%20Ianaro&amp;author=P.%20Maffia&amp;volume=483&amp;issue=1&amp;publication_year=2004&amp;pages=79-93&amp;pmid=14709329&amp;doi=10.1016/j.ejphar.2003.10.056&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Kim S. B., Otani Y., Umezawa Y., Tao H.. Bioluminescent indicator
for determining protein-protein interactions using intramolecular
complementation of split click beetle luciferase. Anal. Chem. 2007;79(13):4820–4826. doi: 10.1021/ac0621571.</cite> [<a href="https://doi.org/10.1021/ac0621571" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17539598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=Bioluminescent%20indicator%20for%20determining%20protein-protein%20interactions%20using%20intramolecular%20complementation%20of%20split%20click%20beetle%20luciferase&amp;author=S.%20B.%20Kim&amp;author=Y.%20Otani&amp;author=Y.%20Umezawa&amp;author=H.%20Tao&amp;volume=79&amp;issue=13&amp;publication_year=2007&amp;pages=4820-4826&amp;pmid=17539598&amp;doi=10.1021/ac0621571&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Kim S. B., Awais M., Sato M., Umezawa Y., Tao H.. Integrated
molecule-format bioluminescent probe for visualizing androgenicity
of ligands based on the intramolecular association of androgen receptor
with its recognition peptide. Anal. Chem. 2007;79(5):1874–1880. doi: 10.1021/ac061934u.</cite> [<a href="https://doi.org/10.1021/ac061934u" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17269793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=Integrated%20molecule-format%20bioluminescent%20probe%20for%20visualizing%20androgenicity%20of%20ligands%20based%20on%20the%20intramolecular%20association%20of%20androgen%20receptor%20with%20its%20recognition%20peptide&amp;author=S.%20B.%20Kim&amp;author=M.%20Awais&amp;author=M.%20Sato&amp;author=Y.%20Umezawa&amp;author=H.%20Tao&amp;volume=79&amp;issue=5&amp;publication_year=2007&amp;pages=1874-1880&amp;pmid=17269793&amp;doi=10.1021/ac061934u&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Kim S. B., Takenaka Y., Torimura M.. A Bioluminescent
Probe for Salivary
Cortisol. Bioconjugate Chem. 2011;22(9):1835–1841. doi: 10.1021/bc200220k.</cite> [<a href="https://doi.org/10.1021/bc200220k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21838298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioconjugate%20Chem.&amp;title=A%20Bioluminescent%20Probe%20for%20Salivary%20Cortisol&amp;author=S.%20B.%20Kim&amp;author=Y.%20Takenaka&amp;author=M.%20Torimura&amp;volume=22&amp;issue=9&amp;publication_year=2011&amp;pages=1835-1841&amp;pmid=21838298&amp;doi=10.1021/bc200220k&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Kim S. B., Umezawa Y., Kanno K. A., Tao H.. An integrated-molecule-format
multicolor probe for monitoring multiple activities of a bioactive
small molecule. ACS Chem. Biol. 2008;3(6):359–372. doi: 10.1021/cb800004s.</cite> [<a href="https://doi.org/10.1021/cb800004s" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18570355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Chem.%20Biol.&amp;title=An%20integrated-molecule-format%20multicolor%20probe%20for%20monitoring%20multiple%20activities%20of%20a%20bioactive%20small%20molecule&amp;author=S.%20B.%20Kim&amp;author=Y.%20Umezawa&amp;author=K.%20A.%20Kanno&amp;author=H.%20Tao&amp;volume=3&amp;issue=6&amp;publication_year=2008&amp;pages=359-372&amp;pmid=18570355&amp;doi=10.1021/cb800004s&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Kim S. B., Nishihara R., Citterio D., Suzuki K.. Genetically Encoded
Molecular Tension Probe for Tracing Protein - Protein Interactions
in Mammalian Cells. Bioconjugate Chem. 2016;27(2):354–362. doi: 10.1021/acs.bioconjchem.5b00421.</cite> [<a href="https://doi.org/10.1021/acs.bioconjchem.5b00421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26322739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioconjugate%20Chem.&amp;title=Genetically%20Encoded%20Molecular%20Tension%20Probe%20for%20Tracing%20Protein%20-%20Protein%20Interactions%20in%20Mammalian%20Cells&amp;author=S.%20B.%20Kim&amp;author=R.%20Nishihara&amp;author=D.%20Citterio&amp;author=K.%20Suzuki&amp;volume=27&amp;issue=2&amp;publication_year=2016&amp;pages=354-362&amp;pmid=26322739&amp;doi=10.1021/acs.bioconjchem.5b00421&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Kamiya G., Kitada N., Furuta T., Hirano T., Maki S. A., Kim S. B.. S-Series Coelenterazine-Driven
Combinatorial Bioluminescence
Imaging Systems for Mammalian Cells. Int. J.
Mol. Sci. 2023;24(2):1420. doi: 10.3390/ijms24021420.</cite> [<a href="https://doi.org/10.3390/ijms24021420" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9865520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36674934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=S-Series%20Coelenterazine-Driven%20Combinatorial%20Bioluminescence%20Imaging%20Systems%20for%20Mammalian%20Cells&amp;author=G.%20Kamiya&amp;author=N.%20Kitada&amp;author=T.%20Furuta&amp;author=T.%20Hirano&amp;author=S.%20A.%20Maki&amp;volume=24&amp;issue=2&amp;publication_year=2023&amp;pages=1420&amp;pmid=36674934&amp;doi=10.3390/ijms24021420&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Nishihara R., Abe M., Nishiyama S., Citterio D., Suzuki K., Kim S. B.. Luciferase-Specific
Coelenterazine Analogues for Optical Contamination-Free Bioassays. Sci. Rep. 2017;7:908. doi: 10.1038/s41598-017-00955-6.</cite> [<a href="https://doi.org/10.1038/s41598-017-00955-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5430434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28424463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Luciferase-Specific%20Coelenterazine%20Analogues%20for%20Optical%20Contamination-Free%20Bioassays&amp;author=R.%20Nishihara&amp;author=M.%20Abe&amp;author=S.%20Nishiyama&amp;author=D.%20Citterio&amp;author=K.%20Suzuki&amp;volume=7&amp;publication_year=2017&amp;pages=908&amp;pmid=28424463&amp;doi=10.1038/s41598-017-00955-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Orioka M., Eguchi M., Mizui Y., Ikeda Y., Sakama A., Li Q. J., Yoshimura H., Ozawa T., Citterio D., Hiruta Y.. A Series of Furimazine
Derivatives for Sustained Live-Cell
Bioluminescence Imaging and Application to the Monitoring of Myogenesis
at the Single-Cell Level. Bioconjugate Chem. 2022;33(3):496–504. doi: 10.1021/acs.bioconjchem.2c00035.</cite> [<a href="https://doi.org/10.1021/acs.bioconjchem.2c00035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35184558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioconjugate%20Chem.&amp;title=A%20Series%20of%20Furimazine%20Derivatives%20for%20Sustained%20Live-Cell%20Bioluminescence%20Imaging%20and%20Application%20to%20the%20Monitoring%20of%20Myogenesis%20at%20the%20Single-Cell%20Level&amp;author=M.%20Orioka&amp;author=M.%20Eguchi&amp;author=Y.%20Mizui&amp;author=Y.%20Ikeda&amp;author=A.%20Sakama&amp;volume=33&amp;issue=3&amp;publication_year=2022&amp;pages=496-504&amp;pmid=35184558&amp;doi=10.1021/acs.bioconjchem.2c00035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Mizui Y., Eguchi M., Tanaka M., Ikeda Y., Yoshimura H., Ozawa T., Citterio D., Hiruta Y.. Long-term single cell
bioluminescence imaging with C-3 position protected coelenterazine
analogues. Org. Biomol. Chem. 2021;19(3):579–586. doi: 10.1039/D0OB02020F.</cite> [<a href="https://doi.org/10.1039/D0OB02020F" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33140803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Org.%20Biomol.%20Chem.&amp;title=Long-term%20single%20cell%20bioluminescence%20imaging%20with%20C-3%20position%20protected%20coelenterazine%20analogues&amp;author=Y.%20Mizui&amp;author=M.%20Eguchi&amp;author=M.%20Tanaka&amp;author=Y.%20Ikeda&amp;author=H.%20Yoshimura&amp;volume=19&amp;issue=3&amp;publication_year=2021&amp;pages=579-586&amp;pmid=33140803&amp;doi=10.1039/D0OB02020F&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Kim S. B., Torimura M., Tao H.. Creation of
artificial luciferases
for bioassays. Bioconjugate Chem. 2013;24(12):2067–2075. doi: 10.1021/bc400411h.</cite> [<a href="https://doi.org/10.1021/bc400411h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24237362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioconjugate%20Chem.&amp;title=Creation%20of%20artificial%20luciferases%20for%20bioassays&amp;author=S.%20B.%20Kim&amp;author=M.%20Torimura&amp;author=H.%20Tao&amp;volume=24&amp;issue=12&amp;publication_year=2013&amp;pages=2067-2075&amp;pmid=24237362&amp;doi=10.1021/bc400411h&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Nishihara R., Paulmurugan R., Nakajima T., Yamamoto E., Natarajan A., Afjei R., Hiruta Y., Iwasawa N., Nishiyama S., Citterio D.. et al. Highly bright and stable NIR-BRET with blue-shifted
coelenterazine derivatives for deep-tissue imaging of molecular events
in vivo. Theranostics. 2019;9(9):2646–2661. doi: 10.7150/thno.32219.</cite> [<a href="https://doi.org/10.7150/thno.32219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6525985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31131059/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theranostics&amp;title=Highly%20bright%20and%20stable%20NIR-BRET%20with%20blue-shifted%20coelenterazine%20derivatives%20for%20deep-tissue%20imaging%20of%20molecular%20events%20in%20vivo&amp;author=R.%20Nishihara&amp;author=R.%20Paulmurugan&amp;author=T.%20Nakajima&amp;author=E.%20Yamamoto&amp;author=A.%20Natarajan&amp;volume=9&amp;issue=9&amp;publication_year=2019&amp;pages=2646-2661&amp;pmid=31131059&amp;doi=10.7150/thno.32219&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Piatkevich K. D., Malashkevich V. N., Almo S. C., Verkhusha V. V.. Engineering
ESPT Pathways Based on Structural Analysis of LSSmKate Red Fluorescent
Proteins with Large Stokes Shift. J. Am. Chem.
Soc. 2010;132(31):10762–10770. doi: 10.1021/ja101974k.</cite> [<a href="https://doi.org/10.1021/ja101974k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2919691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20681709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Engineering%20ESPT%20Pathways%20Based%20on%20Structural%20Analysis%20of%20LSSmKate%20Red%20Fluorescent%20Proteins%20with%20Large%20Stokes%20Shift&amp;author=K.%20D.%20Piatkevich&amp;author=V.%20N.%20Malashkevich&amp;author=S.%20C.%20Almo&amp;author=V.%20V.%20Verkhusha&amp;volume=132&amp;issue=31&amp;publication_year=2010&amp;pages=10762-10770&amp;pmid=20681709&amp;doi=10.1021/ja101974k&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Rechsteiner M., Rogers S. W.. PEST sequences
and regulation by proteolysis. Trends Biochem.
Sci. 1996;21(7):267–271. doi: 10.1016/S0968-0004(96)10031-1.</cite> [<a href="https://doi.org/10.1016/S0968-0004(96)10031-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8755249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem.%20Sci.&amp;title=PEST%20sequences%20and%20regulation%20by%20proteolysis&amp;author=M.%20Rechsteiner&amp;author=S.%20W.%20Rogers&amp;volume=21&amp;issue=7&amp;publication_year=1996&amp;pages=267-271&amp;pmid=8755249&amp;doi=10.1016/S0968-0004(96)10031-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Nolte R. T., Wisely G. B., Westin S., Cobb J. E., Lambert M. H., Kurokawa R., Rosenfeld M. G., Willson T. M., Glass C. K., Milburn M. V.. Ligand binding and
co-activator assembly of the peroxisome
proliferator-activated receptor-γ. Nature. 1998;395(6698):137–143. doi: 10.1038/25931.</cite> [<a href="https://doi.org/10.1038/25931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9744270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Ligand%20binding%20and%20co-activator%20assembly%20of%20the%20peroxisome%20proliferator-activated%20receptor-%CE%B3&amp;author=R.%20T.%20Nolte&amp;author=G.%20B.%20Wisely&amp;author=S.%20Westin&amp;author=J.%20E.%20Cobb&amp;author=M.%20H.%20Lambert&amp;volume=395&amp;issue=6698&amp;publication_year=1998&amp;pages=137-143&amp;pmid=9744270&amp;doi=10.1038/25931&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Frkic R. L., Marshall A. C., Blayo A. L., Pukala T. L., Kamenecka T. M., Griffin P. R., Bruning J. B.. PPARγ
in Complex with an Antagonist
and Inverse Agonist: a Tumble and Trap Mechanism of the Activation
Helix. Iscience. 2018;5:69–79. doi: 10.1016/j.isci.2018.06.012.</cite> [<a href="https://doi.org/10.1016/j.isci.2018.06.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6095676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30123887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Iscience&amp;title=PPAR%CE%B3%20in%20Complex%20with%20an%20Antagonist%20and%20Inverse%20Agonist:%20a%20Tumble%20and%20Trap%20Mechanism%20of%20the%20Activation%20Helix&amp;author=R.%20L.%20Frkic&amp;author=A.%20C.%20Marshall&amp;author=A.%20L.%20Blayo&amp;author=T.%20L.%20Pukala&amp;author=T.%20M.%20Kamenecka&amp;volume=5&amp;publication_year=2018&amp;pages=69-79&amp;pmid=30123887&amp;doi=10.1016/j.isci.2018.06.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Burns K. A., Heuvel J. P. V.. Modulation of PPAR activity via phosphorylation. Biochim. Biophys. Acta Mol. Cell Biol. Lipids. 2007;1771(8):952–960. doi: 10.1016/j.bbalip.2007.04.018.</cite> [<a href="https://doi.org/10.1016/j.bbalip.2007.04.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2712836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17560826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta%20Mol.%20Cell%20Biol.%20Lipids&amp;title=Modulation%20of%20PPAR%20activity%20via%20phosphorylation&amp;author=K.%20A.%20Burns&amp;author=J.%20P.%20V.%20Heuvel&amp;volume=1771&amp;issue=8&amp;publication_year=2007&amp;pages=952-960&amp;pmid=17560826&amp;doi=10.1016/j.bbalip.2007.04.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Kim S. B., Sato M., Tao H.. Genetically
Encoded Bioluminescent
Indicators for Stress Hormones. Anal. Chem. 2009;81(10):3760–3768. doi: 10.1021/ac802674w.</cite> [<a href="https://doi.org/10.1021/ac802674w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19388668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=Genetically%20Encoded%20Bioluminescent%20Indicators%20for%20Stress%20Hormones&amp;author=S.%20B.%20Kim&amp;author=M.%20Sato&amp;author=H.%20Tao&amp;volume=81&amp;issue=10&amp;publication_year=2009&amp;pages=3760-3768&amp;pmid=19388668&amp;doi=10.1021/ac802674w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Kim S. B., Fujii R., Natarajan A., Massoud T. F., Paulmurugan R.. Ligand-activated
BRET9 imaging for measuring protein-protein interactions in living
mice. Chem. Commun. 2020;56(2):281–284. doi: 10.1039/c9cc07634d.</cite> [<a href="https://doi.org/10.1039/c9cc07634d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31807738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Commun.&amp;title=Ligand-activated%20BRET9%20imaging%20for%20measuring%20protein-protein%20interactions%20in%20living%20mice&amp;author=S.%20B.%20Kim&amp;author=R.%20Fujii&amp;author=A.%20Natarajan&amp;author=T.%20F.%20Massoud&amp;author=R.%20Paulmurugan&amp;volume=56&amp;issue=2&amp;publication_year=2020&amp;pages=281-284&amp;pmid=31807738&amp;doi=10.1039/c9cc07634d&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Kim S. B., Sato M., Tao H.. Circularly Permutated Bioluminescent
Probes for Illuminating Ligand-Activated Protein Dynamics. Bioconjugate Chem. 2008;19(12):2480–2486. doi: 10.1021/bc800378a.</cite> [<a href="https://doi.org/10.1021/bc800378a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19049389/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioconjugate%20Chem.&amp;title=Circularly%20Permutated%20Bioluminescent%20Probes%20for%20Illuminating%20Ligand-Activated%20Protein%20Dynamics&amp;author=S.%20B.%20Kim&amp;author=M.%20Sato&amp;author=H.%20Tao&amp;volume=19&amp;issue=12&amp;publication_year=2008&amp;pages=2480-2486&amp;pmid=19049389&amp;doi=10.1021/bc800378a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Mayi T. H., Duhem C., Copin C., Bouhlel M. A., Rigamonti E., Pattou F., Staels B., Chinetti-Gbaguidi G.. Visfatin is
induced by peroxisome proliferator-activated receptor gamma in human
macrophages. FEBS J. 2010;277(16):3308–3320. doi: 10.1111/j.1742-4658.2010.07729.x.</cite> [<a href="https://doi.org/10.1111/j.1742-4658.2010.07729.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3183437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20608974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20J.&amp;title=Visfatin%20is%20induced%20by%20peroxisome%20proliferator-activated%20receptor%20gamma%20in%20human%20macrophages&amp;author=T.%20H.%20Mayi&amp;author=C.%20Duhem&amp;author=C.%20Copin&amp;author=M.%20A.%20Bouhlel&amp;author=E.%20Rigamonti&amp;volume=277&amp;issue=16&amp;publication_year=2010&amp;pages=3308-3320&amp;pmid=20608974&amp;doi=10.1111/j.1742-4658.2010.07729.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Han S., Roman J.. Suppression of prostaglandin
E2 receptor subtype EP2 by PPARgamma
ligands inhibits human lung carcinoma cell growth. Biochem. Biophys. Res. Commun. 2004;314(4):1093–1099. doi: 10.1016/j.bbrc.2004.01.007.</cite> [<a href="https://doi.org/10.1016/j.bbrc.2004.01.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14751245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;title=Suppression%20of%20prostaglandin%20E2%20receptor%20subtype%20EP2%20by%20PPARgamma%20ligands%20inhibits%20human%20lung%20carcinoma%20cell%20growth&amp;author=S.%20Han&amp;author=J.%20Roman&amp;volume=314&amp;issue=4&amp;publication_year=2004&amp;pages=1093-1099&amp;pmid=14751245&amp;doi=10.1016/j.bbrc.2004.01.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Kamiya G., Kitada N., Furuta T., Hirano T., Maki S., Kim S. B.. C-Series Coelenterazine-Driven Bioluminescence
Signature
Imaging. Int. J. Mol. Sci. 2022;23(21):13047. doi: 10.3390/ijms232113047.</cite> [<a href="https://doi.org/10.3390/ijms232113047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9658928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36361837/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=C-Series%20Coelenterazine-Driven%20Bioluminescence%20Signature%20Imaging&amp;author=G.%20Kamiya&amp;author=N.%20Kitada&amp;author=T.%20Furuta&amp;author=T.%20Hirano&amp;author=S.%20Maki&amp;volume=23&amp;issue=21&amp;publication_year=2022&amp;pages=13047&amp;pmid=36361837&amp;doi=10.3390/ijms232113047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Johnstone E. K. M., See H. B., Abhayawardana R. S., Song A., Rosengren K. J., Hill S. J., Pfleger K. D. G.. Investigation of Receptor Heteromers
Using NanoBRET Ligand Binding. Int. J. Mol.
Sci. 2021;22(3):1082. doi: 10.3390/ijms22031082.</cite> [<a href="https://doi.org/10.3390/ijms22031082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7866079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33499147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Investigation%20of%20Receptor%20Heteromers%20Using%20NanoBRET%20Ligand%20Binding&amp;author=E.%20K.%20M.%20Johnstone&amp;author=H.%20B.%20See&amp;author=R.%20S.%20Abhayawardana&amp;author=A.%20Song&amp;author=K.%20J.%20Rosengren&amp;volume=22&amp;issue=3&amp;publication_year=2021&amp;pages=1082&amp;pmid=33499147&amp;doi=10.3390/ijms22031082&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Huggins R. J., Hosfield D., Ishag-Osman A., Lee K., Ton-That E., Greene G. L.. Evaluating steroid hormone receptor
interactions using
the live-cell NanoBRET proximity assay. bioRxiv. 2023:2023.07.25.550078. doi: 10.1101/2023.07.25.550078.</cite> [<a href="https://doi.org/10.1101/2023.07.25.550078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Evaluating%20steroid%20hormone%20receptor%20interactions%20using%20the%20live-cell%20NanoBRET%20proximity%20assay&amp;author=R.%20J.%20Huggins&amp;author=D.%20Hosfield&amp;author=A.%20Ishag-Osman&amp;author=K.%20Lee&amp;author=E.%20Ton-That&amp;publication_year=2023&amp;pages=2023.07.25.550078&amp;doi=10.1101/2023.07.25.550078&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Waring M. J.. Lipophilicity
in drug discovery. Expert Opin. Drug Discovery. 2010;5(3):235–248. doi: 10.1517/17460441003605098.</cite> [<a href="https://doi.org/10.1517/17460441003605098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22823020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Opin.%20Drug%20Discovery&amp;title=Lipophilicity%20in%20drug%20discovery&amp;author=M.%20J.%20Waring&amp;volume=5&amp;issue=3&amp;publication_year=2010&amp;pages=235-248&amp;pmid=22823020&amp;doi=10.1517/17460441003605098&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Valko K., Nunhuck S., Bevan C., Abraham M. H., Reynolds D. P.. Fast gradient
HPLC method to determine compounds binding to human serum albumin.
Relationships with octanol/water and immobilized artificial membrane
lipophilicity. J. Pharm. Sci. 2003;92(11):2236–2248. doi: 10.1002/jps.10494.</cite> [<a href="https://doi.org/10.1002/jps.10494" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14603509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Sci.&amp;title=Fast%20gradient%20HPLC%20method%20to%20determine%20compounds%20binding%20to%20human%20serum%20albumin.%20Relationships%20with%20octanol/water%20and%20immobilized%20artificial%20membrane%20lipophilicity&amp;author=K.%20Valko&amp;author=S.%20Nunhuck&amp;author=C.%20Bevan&amp;author=M.%20H.%20Abraham&amp;author=D.%20P.%20Reynolds&amp;volume=92&amp;issue=11&amp;publication_year=2003&amp;pages=2236-2248&amp;pmid=14603509&amp;doi=10.1002/jps.10494&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Vassel N., Cox C. D., Naseem R., Morse V., Evans R. T., Power R. L., Brancale A., Wann K. T., Campbell A. K.. Enzymatic
activity of albumin shown by coelenterazine chemiluminescence. Luminescence. 2012;27(3):234–241. doi: 10.1002/bio.2357.</cite> [<a href="https://doi.org/10.1002/bio.2357" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22362656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Luminescence&amp;title=Enzymatic%20activity%20of%20albumin%20shown%20by%20coelenterazine%20chemiluminescence&amp;author=N.%20Vassel&amp;author=C.%20D.%20Cox&amp;author=R.%20Naseem&amp;author=V.%20Morse&amp;author=R.%20T.%20Evans&amp;volume=27&amp;issue=3&amp;publication_year=2012&amp;pages=234-241&amp;pmid=22362656&amp;doi=10.1002/bio.2357&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Yuan M., Ma X., Jiang T., Gao Y., Cui Y., Zhang C., Yang X., Huang Y., Du L., Yampolsky I.. et al. Prolonged bioluminescence imaging in living cells and mice using
novel pro-substrates for Renilla luciferase. Org. Biomol. Chem. 2017;15(48):10238–10244. doi: 10.1039/C7OB01656E.</cite> [<a href="https://doi.org/10.1039/C7OB01656E" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29177293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Org.%20Biomol.%20Chem.&amp;title=Prolonged%20bioluminescence%20imaging%20in%20living%20cells%20and%20mice%20using%20novel%20pro-substrates%20for%20Renilla%20luciferase&amp;author=M.%20Yuan&amp;author=X.%20Ma&amp;author=T.%20Jiang&amp;author=Y.%20Gao&amp;author=Y.%20Cui&amp;volume=15&amp;issue=48&amp;publication_year=2017&amp;pages=10238-10244&amp;pmid=29177293&amp;doi=10.1039/C7OB01656E&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Lee H., Chen D. L., Rothfuss J. M., Welch M. J., Gropler R. J., Mach R. H.. Synthesis and evaluation of 18F-labeled PPARγ
antagonists. Nucl. Med. Biol. 2012;39(1):77–87. doi: 10.1016/j.nucmedbio.2011.07.002.</cite> [<a href="https://doi.org/10.1016/j.nucmedbio.2011.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3248633/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21907585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucl.%20Med.%20Biol.&amp;title=Synthesis%20and%20evaluation%20of%2018F-labeled%20PPAR%CE%B3%20antagonists&amp;author=H.%20Lee&amp;author=D.%20L.%20Chen&amp;author=J.%20M.%20Rothfuss&amp;author=M.%20J.%20Welch&amp;author=R.%20J.%20Gropler&amp;volume=39&amp;issue=1&amp;publication_year=2012&amp;pages=77-87&amp;pmid=21907585&amp;doi=10.1016/j.nucmedbio.2011.07.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Choi J. Y., Ryu J., Kim H. J., Song J. W., Jeon J. H., Lee D. H., Oh D. J., Gweon D. G., Oh W. Y., Yoo H.. et al. Therapeutic
Effects of Targeted PPAR. Activation on Inflamed High-Risk
Plaques Assessed by Serial Optical Imaging In Vivo. Theranostics. 2018;8(1):45–60. doi: 10.7150/thno.20885.</cite> [<a href="https://doi.org/10.7150/thno.20885" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5743459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29290792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theranostics&amp;title=Therapeutic%20Effects%20of%20Targeted%20PPAR.%20Activation%20on%20Inflamed%20High-Risk%20Plaques%20Assessed%20by%20Serial%20Optical%20Imaging%20In%20Vivo&amp;author=J.%20Y.%20Choi&amp;author=J.%20Ryu&amp;author=H.%20J.%20Kim&amp;author=J.%20W.%20Song&amp;author=J.%20H.%20Jeon&amp;volume=8&amp;issue=1&amp;publication_year=2018&amp;pages=45-60&amp;pmid=29290792&amp;doi=10.7150/thno.20885&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12332630/bin/ao5c04665_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c04665_si_001.pdf</a><sup> (900.9KB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Omega are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsomega.5c04665"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ao5c04665.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332630/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332630/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332630%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332630/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332630/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332630/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40787404/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332630/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40787404/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332630/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332630/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="UQtlkdAuY2hsvxxLFeiguJfDqsxo9Hw5xWELfTu9sedeTIxrsNXbuh5b5db0vOGV">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
