
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genetic analysis of predicted vegetative biomass and biomass‐related traits from digital phenotyping of strawberry - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="37B2C35C7FFF9D4306C35C0006E41798.m_1" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958871/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Genetic analysis of predicted vegetative biomass and biomass‐related traits from digital phenotyping of strawberry">
<meta name="citation_author" content="Cheryl Dalid">
<meta name="citation_author_institution" content="Horticultural Sciences Department, IFAS Gulf Coast Research and Education Center, University of Florida, Wimauma, Florida, USA">
<meta name="citation_author" content="Caiwang Zheng">
<meta name="citation_author_institution" content="School of Forest Resources and Conservation Geomatics, IFAS Gulf Coast Research and Education Center, University of Florida, Plant City, Florida, USA">
<meta name="citation_author" content="Luis Osorio">
<meta name="citation_author_institution" content="Horticultural Sciences Department, IFAS Gulf Coast Research and Education Center, University of Florida, Wimauma, Florida, USA">
<meta name="citation_author" content="Sujeet Verma">
<meta name="citation_author_institution" content="Fall Creek Farm and Nursery Inc. Lowell, Oregon, USA">
<meta name="citation_author" content="Amr Abd‐Elrahman">
<meta name="citation_author_institution" content="School of Forest Resources and Conservation Geomatics, IFAS Gulf Coast Research and Education Center, University of Florida, Plant City, Florida, USA">
<meta name="citation_author" content="Xu Wang">
<meta name="citation_author_institution" content="Agricultural and Biological Engineering Department, IFAS Gulf Coast Research and Education Center, University of Florida, Wimauma, Florida, USA">
<meta name="citation_author" content="Vance M Whitaker">
<meta name="citation_author_institution" content="Horticultural Sciences Department, IFAS Gulf Coast Research and Education Center, University of Florida, Wimauma, Florida, USA">
<meta name="citation_publication_date" content="2025 Mar 31">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="2">
<meta name="citation_firstpage" content="e70018">
<meta name="citation_doi" content="10.1002/tpg2.70018">
<meta name="citation_pmid" content="40164966">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958871/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958871/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958871/pdf/TPG2-18-e70018.pdf">
<meta name="description" content="High‐throughput digital phenotyping (DP) has been widely explored in plant breeding to assess large numbers of genotypes with minimal manual labor and reduced cost and time. DP platforms using high‐resolution images captured by drones and ...">
<meta name="og:title" content="Genetic analysis of predicted vegetative biomass and biomass‐related traits from digital phenotyping of strawberry">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="High‐throughput digital phenotyping (DP) has been widely explored in plant breeding to assess large numbers of genotypes with minimal manual labor and reduced cost and time. DP platforms using high‐resolution images captured by drones and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958871/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11958871">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70018"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70018.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11958871%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11958871/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11958871/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958871/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70018" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Mar 31;18(2):e70018. doi: <a href="https://doi.org/10.1002/tpg2.70018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70018</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genetic analysis of predicted vegetative biomass and biomass‐related traits from digital phenotyping of strawberry</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dalid%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Cheryl Dalid</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Cheryl Dalid</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Horticultural Sciences Department, IFAS Gulf Coast Research and Education Center, University of Florida, Wimauma, Florida, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dalid%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Cheryl Dalid</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Caiwang Zheng</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Caiwang Zheng</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>School of Forest Resources and Conservation Geomatics, IFAS Gulf Coast Research and Education Center, University of Florida, Plant City, Florida, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Caiwang Zheng</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Osorio%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Luis Osorio</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Luis Osorio</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Horticultural Sciences Department, IFAS Gulf Coast Research and Education Center, University of Florida, Wimauma, Florida, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Osorio%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Luis Osorio</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Verma%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Sujeet Verma</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Sujeet Verma</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Fall Creek Farm and Nursery Inc. Lowell, Oregon, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Verma%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sujeet Verma</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abd%E2%80%90Elrahman%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Amr Abd‐Elrahman</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Amr Abd‐Elrahman</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>School of Forest Resources and Conservation Geomatics, IFAS Gulf Coast Research and Education Center, University of Florida, Plant City, Florida, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abd%E2%80%90Elrahman%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Amr Abd‐Elrahman</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Xu Wang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Xu Wang</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Agricultural and Biological Engineering Department, IFAS Gulf Coast Research and Education Center, University of Florida, Wimauma, Florida, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xu Wang</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Whitaker%20VM%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Vance M Whitaker</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Vance M Whitaker</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Horticultural Sciences Department, IFAS Gulf Coast Research and Education Center, University of Florida, Wimauma, Florida, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Whitaker%20VM%22%5BAuthor%5D" class="usa-link"><span class="name western">Vance M Whitaker</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270018-aff-0001">
<sup>
<sup>1</sup>
</sup>Horticultural Sciences Department, IFAS Gulf Coast Research and Education Center, University of Florida, Wimauma, Florida, USA</div>
<div id="tpg270018-aff-0002">
<sup>
<sup>2</sup>
</sup>School of Forest Resources and Conservation Geomatics, IFAS Gulf Coast Research and Education Center, University of Florida, Plant City, Florida, USA</div>
<div id="tpg270018-aff-0003">
<sup>
<sup>3</sup>
</sup>Fall Creek Farm and Nursery Inc. Lowell, Oregon, USA</div>
<div id="tpg270018-aff-0004">
<sup>
<sup>4</sup>
</sup>Agricultural and Biological Engineering Department, IFAS Gulf Coast Research and Education Center, University of Florida, Wimauma, Florida, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Vance M. Whitaker, Horticultural Sciences Department, IFAS Gulf Coast Research and Education Center, University of Florida, Wimauma, FL 33598, USA. Email: <span>vwhitaker@ufl.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2024 Dec 30; Received 2024 Sep 26; Accepted 2025 Feb 23; Issue date 2025 Jun.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11958871  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40164966/" class="usa-link">40164966</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>High‐throughput digital phenotyping (DP) has been widely explored in plant breeding to assess large numbers of genotypes with minimal manual labor and reduced cost and time. DP platforms using high‐resolution images captured by drones and tractor‐based platforms have recently allowed the University of Florida strawberry (<em>Fragaria × ananassa</em>) breeding program to assess vegetative biomass at scale. Biomass has not previously been explored in a strawberry breeding context due to the labor required and the need to destroy the plant. This study aims to understand the genetic basis of predicted vegetative biomass and biomass‐related traits and to chart a path for the combined use of DP and genomics in strawberry breeding. Aboveground dry vegetative biomass was estimated by adapting a previously published model using ground‐truth data on a subset of breeding germplasm. High‐resolution images were collected on clonally replicated trials at different time points during the fruiting season. There was moderate to high heritability (<em>h</em>
<sup>2 </sup>= 0.26–0.56) for predicted vegetative biomass, and genetic correlations between vegetative biomass and marketable yield were mostly positive (<em>r<sub>G</sub>
</em> = −0.13–0.47). Fruit yield traits scaled on a vegetative biomass basis also had moderate to high heritability (<em>h</em>
<sup>2 </sup>= 0.25–0.64). This suggests that vegetative biomass can be decreased or increased through selection, and that marketable fruit yield can be improved without simultaneously increasing plant size. No consistent marker‐trait associations were discovered via genome‐wide association studies. On the other hand, predictive abilities from genomic selection ranged from 0.15 to 0.46 across traits and years, suggesting that genomic prediction will be an effective breeding tool for vegetative biomass in strawberry.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270018-list-10001" class="list" style="list-style-type:disc">
<li><p>University of Florida strawberry breeding program nondestructively measured biomass through digital phenotyping (DP) methods.</p></li>
<li><p>These measurements allowed us to examine the inheritance of biomass and balance selection for yield and plant size.</p></li>
<li><p>Predictive abilities from genomic selection were moderate and will allow effective predictive breeding for biomass.</p></li>
<li><p>Improvement of DP methods for biomass estimation could also improve selection accuracy for biomass in the future.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270018-dl-0001">
<dt>ACCB</dt>
<dd><p>accumulated predicted vegetative biomass</p></dd>
<dt>AWT</dt>
<dd><p>average fruit weight</p></dd>
<dt>AWTAC</dt>
<dd><p>average fruit weight on an accumulated predicted vegetative biomass basis</p></dd>
<dt>AWTB</dt>
<dd><p>average fruit weight on a predicted biomass basis</p></dd>
<dt>BIO</dt>
<dd><p>predicted vegetative biomass</p></dd>
<dt>DP</dt>
<dd><p>digital phenotyping</p></dd>
<dt>EMY</dt>
<dd><p>early marketable yield</p></dd>
<dt>FRM</dt>
<dd><p>firmness</p></dd>
<dt>GS</dt>
<dd><p>genomic selection</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association studies</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
<dt>SSC</dt>
<dd><p>soluble solids content</p></dd>
<dt>TC</dt>
<dd><p>total culls</p></dd>
<dt>TMY</dt>
<dd><p>total marketable yield</p></dd>
<dt>UAV</dt>
<dd><p>unmanned aerial vehicle</p></dd>
<dt>UF</dt>
<dd><p>University of Florida</p></dd>
</dl>
<section id="tpg270018-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Cultivated strawberry (<em>Fragaria</em> × <em>ananassa</em>) is the most widely consumed fruit crop in the world with an annual production of 9.5 million metric ton in 2022 (FAO‐UN, <a href="#tpg270018-bib-0011" class="usa-link" aria-describedby="tpg270018-bib-0011"><sup>2022</sup></a>). Although the production of strawberries increased by 1168% from 1961 to 2022 due to an increase in acreage, only a 200% increase in yield was observed (FAO‐UN, <a href="#tpg270018-bib-0011" class="usa-link" aria-describedby="tpg270018-bib-0011"><sup>2022</sup></a>). To meet the growing global demands for strawberries, it is vital for strawberry breeding programs to develop high‐yielding cultivars while also improving ease of harvest, fruit quality, and disease resistance.</p>
<p>In the United States, most strawberries are produced in California and Florida. As of 2022, California produced 89.1% of strawberries in the United States, while Florida, known for its peak strawberry production during winter, produced 10.9% (USDA‐NASS, <a href="#tpg270018-bib-0039" class="usa-link" aria-describedby="tpg270018-bib-0039"><sup>2023</sup></a>). The University of Florida (UF) breeding program develops cultivars that are adapted for fruit production during the winter season in west‐central Florida. Currently, strawberry fruit production in west‐central Florida begins in late November and peaks during February and March as the weather warms and the plants continue to accumulate crowns and leaves (Whitaker &amp; Fan, <a href="#tpg270018-bib-0045" class="usa-link" aria-describedby="tpg270018-bib-0045">2020</a>; Wu et al., <a href="#tpg270018-bib-0047" class="usa-link" aria-describedby="tpg270018-bib-0047"><sup>2015</sup></a>). Development of early yielding strawberries is an important breeding objective at UF to help Florida farmers take advantage of high prices from late November until the end of January.</p>
<p>Harvest efficiency is also an important breeding objective since harvesting is a labor‐intensive and costly activity that represents about 57% of the total cost of strawberry production (Guan et al., <a href="#tpg270018-bib-0016" class="usa-link" aria-describedby="tpg270018-bib-0016">2018</a>). Selecting for a moderate plant size helps increase efficiency of manual harvesting and allows strawberry plants to fit within desired in‐row spacings. On the other hand, per plant yield tends to be positively correlated to leaf area and plant size (Agehara, <a href="#tpg270018-bib-0001" class="usa-link" aria-describedby="tpg270018-bib-0001">2021</a>). Plant biomass is the total mass of a plant, including aboveground plant parts such as stems, leaves, flowers, and fruits, as well as the root system (Poorter &amp; Nagel, <a href="#tpg270018-bib-0030" class="usa-link" aria-describedby="tpg270018-bib-0030">2000</a>). Plant vegetative biomass, which is our focus here, refers to only leaves and stems. High vegetative biomass is associated with the potential for higher yields across plant species (Poorter et al., <a href="#tpg270018-bib-0031" class="usa-link" aria-describedby="tpg270018-bib-0031">2009</a>). With an increase in leaf biomass, higher photosynthesis is expected, which in turn increases biomass accumulation that can lead to greater yield (Burgess et al., <a href="#tpg270018-bib-0004" class="usa-link" aria-describedby="tpg270018-bib-0004">2023</a>; Poorter et al., <a href="#tpg270018-bib-0031" class="usa-link" aria-describedby="tpg270018-bib-0031"><sup>2009</sup></a>; Zhu et al., <a href="#tpg270018-bib-0051" class="usa-link" aria-describedby="tpg270018-bib-0051"><sup>2010</sup></a>). However, direct measurement of vegetative biomass is labor‐intensive, time consuming, and requires the destruction of the plant. Implementation of high‐throughput phenotyping approaches, including digital phenotyping (DP), can help address these limitations.</p>
<p>Recent advances in DP technologies have enabled rapid and detailed extraction of plant traits on a large scale. Image‐based DP using ground‐based and aerial systems are the two most common approaches (Guan et al., <a href="#tpg270018-bib-0015" class="usa-link" aria-describedby="tpg270018-bib-0015">2020</a>; Krause et al., <a href="#tpg270018-bib-0020" class="usa-link" aria-describedby="tpg270018-bib-0020"><sup>2020</sup></a>; Pérez‐Ruiz et al., <a href="#tpg270018-bib-0028" class="usa-link" aria-describedby="tpg270018-bib-0028"><sup>2020</sup></a>; H. Zheng et al., <a href="#tpg270018-bib-0050" class="usa-link" aria-describedby="tpg270018-bib-0050"><sup>2019</sup></a>). In strawberry, Guan et al. (<a href="#tpg270018-bib-0015" class="usa-link" aria-describedby="tpg270018-bib-0015">2020</a>) used high‐resolution imagery obtained from a ground‐based imaging system mounted on a tractor to develop a model predicting vegetative biomass with an accuracy (<em>R</em>
<sup>2</sup>) of 0.79. This previous work has provided an opportunity to estimate vegetative biomass at scale in strawberry and thus to study the genetics of this trait.</p>
<p>Because the ability to measure vegetative biomass at scale is a new development in strawberry, the genetics of this trait have not yet been explored. Genome‐wide association studies (GWAS) represent an effective method to discover genetic variants associated with traits, potentially leading to the development of tools for marker‐assisted selection (Ibrahim et al., <a href="#tpg270018-bib-0018" class="usa-link" aria-describedby="tpg270018-bib-0018">2020</a>). However, this approach is most beneficial when the trait is controlled by one or few large‐effect loci. For complex traits, a genomic selection (GS) strategy has been effective for increasing genetic gains in plants (Crossa et al., <a href="#tpg270018-bib-0007" class="usa-link" aria-describedby="tpg270018-bib-0007">2010</a>; de Bem Oliveira et al., <a href="#tpg270018-bib-0008" class="usa-link" aria-describedby="tpg270018-bib-0008"><sup>2019</sup></a>; Gezan et al., <a href="#tpg270018-bib-0013" class="usa-link" aria-describedby="tpg270018-bib-0013"><sup>2017</sup></a>; Merrick &amp; Carter, <a href="#tpg270018-bib-0021" class="usa-link" aria-describedby="tpg270018-bib-0021"><sup>2021</sup></a>; Pincot et al., <a href="#tpg270018-bib-0029" class="usa-link" aria-describedby="tpg270018-bib-0029"><sup>2020</sup></a>; Tessema et al., <a href="#tpg270018-bib-0036" class="usa-link" aria-describedby="tpg270018-bib-0036"><sup>2020</sup></a>). GS uses genome‐wide marker data to predict the performance of untested individuals (Meuwissen et al., <a href="#tpg270018-bib-0022" class="usa-link" aria-describedby="tpg270018-bib-0022">2001</a>), a successful approach in various breeding programs, including in strawberry (Crain et al., <a href="#tpg270018-bib-0006" class="usa-link" aria-describedby="tpg270018-bib-0006">2018</a>; O'Connor et al., <a href="#tpg270018-bib-0024" class="usa-link" aria-describedby="tpg270018-bib-0024"><sup>2021</sup></a>; Osorio et al., <a href="#tpg270018-bib-0025" class="usa-link" aria-describedby="tpg270018-bib-0025"><sup>2021</sup></a>; Peixoto et al., <a href="#tpg270018-bib-0026" class="usa-link" aria-describedby="tpg270018-bib-0026"><sup>2024</sup></a>).</p>
<p>To date, vegetative biomass has not been measured or studied at scale in any strawberry breeding program due to the lack of a high‐throughput and nondestructive method. Now that we have a DP method to estimate vegetative biomass, we are able to explore the genetic basis of this trait in relation to important fruit yield and quality traits. Here, we have combined predicted vegetative biomass data with genomic information from the UF strawberry breeding program to estimate heritabilities and genetic correlations for vegetative biomass and yield‐related traits. We also performed GWAS to examine the genetic architecture of vegetative biomass. Lastly, we evaluated the potential of GS, demonstrating the combined use of DP and genomics in strawberry breeding.</p></section><section id="tpg270018-sec-0020"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270018-sec-0030"><h3 class="pmc_sec_title">2.1. Plant material and field trials</h3>
<p>The genotypes used in this study represent the elite strawberry germplasm of the Strawberry Breeding and Genetics Program of the UF including released cultivars and advanced selections. These were planted in replicated yield trials during the 2017–2018, 2018–2019, 2020–2021, and 2021–2022 seasons on October 9, October 10, October 7, and October 6, respectively, at the UF/IFAS Gulf Coast Research and Education Center, in Balm, FL (27° 45′ 37.98″ N, 82° 13′ 32.49″ W) (Figure <a href="#tpg270018-fig-0001" class="usa-link">1</a>). Other data from these field trials were also utilized in previous studies (Fan &amp; Whitaker, <a href="#tpg270018-bib-0010" class="usa-link" aria-describedby="tpg270018-bib-0010">2024</a>; Fan et al., <a href="#tpg270018-bib-0009" class="usa-link" aria-describedby="tpg270018-bib-0009"><sup>2024</sup></a>; Osorio et al., <a href="#tpg270018-bib-0025" class="usa-link" aria-describedby="tpg270018-bib-0025"><sup>2021</sup></a>).</p>
<figure class="fig xbox font-sm" id="tpg270018-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d601/11958871/cf9e0e25e92d/TPG2-18-e70018-g001.jpg" loading="lazy" id="jats-graphic-1" height="677" width="800" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270018-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Location of the phenomics trial used to develop the model to predict vegetative biomass in relation to the advanced selection trial where vegetative biomass was predicted and used in this study.</p></figcaption></figure><p>Bare‐root clonal replicates of 1748 different entries across the years were planted in a randomized complete block design with five replications that consists of two beds per replication. The 1748 phenotypes represented 1460 unique individuals including advanced selections and varieties that were repeated across trials (Table <a href="#tpg270018-tbl-0001" class="usa-link">1</a>). Each bed was divided into five to eight plots with one cultivar check per plot to account for variation within the bed (columns). Each replication consisted of a single plant (clonal replicate) per genotype. Overlapping genotypes were included across years. The trial site was prepared and maintained following the standard strawberry growing practices in west‐central Florida. Fertilization, weed management, and pest and disease control varied among different seasons according to environmental conditions.</p>
<section class="tw xbox font-sm" id="tpg270018-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Number of tested genotypes across years.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Year</th>
<th align="left" rowspan="1" colspan="1">2017</th>
<th align="left" rowspan="1" colspan="1">2018</th>
<th align="left" rowspan="1" colspan="1">2020</th>
<th align="left" rowspan="1" colspan="1">2021</th>
<th align="left" rowspan="1" colspan="1">
<em>N</em>
<a href="#tpg270018-tbl1-note-0002" class="usa-link">
<sup>a</sup>
</a>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="left" rowspan="1" colspan="1">
<strong>390</strong>
<a href="#tpg270018-tbl1-note-0003" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="left" rowspan="1" colspan="1">64<a href="#tpg270018-tbl1-note-0004" class="usa-link">
<sup>c</sup>
</a>
</td>
<td align="left" rowspan="1" colspan="1">19</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">394</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="left" rowspan="1" colspan="1">64</td>
<td align="left" rowspan="1" colspan="1">
<strong>413</strong>
</td>
<td align="left" rowspan="1" colspan="1">29</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">445</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2020</td>
<td align="left" rowspan="1" colspan="1">19</td>
<td align="left" rowspan="1" colspan="1">29</td>
<td align="left" rowspan="1" colspan="1">
<strong>413</strong>
</td>
<td align="left" rowspan="1" colspan="1">102</td>
<td align="left" rowspan="1" colspan="1">443</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2021</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">102</td>
<td align="left" rowspan="1" colspan="1">
<strong>444</strong>
</td>
<td align="left" rowspan="1" colspan="1">466</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270018-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270018-tbl1-note-0001"><p>
<em>Note</em>: The numbers in bold shows the number of unique genotypes that were included in the analysis.</p></div>
<div class="fn" id="tpg270018-tbl1-note-0002">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Total number of tested genotypes across years (total <em>N</em> = 1748).</p>
</div>
<div class="fn" id="tpg270018-tbl1-note-0003">
<sup>
<sup>b</sup>
</sup><p class="display-inline">Number of analyzed genotypes (diagonal, 1460 unique genotypes).</p>
</div>
<div class="fn" id="tpg270018-tbl1-note-0004">
<sup>
<sup>c</sup>
</sup><p class="display-inline">Number of common genotypes within years (off‐diagonal).</p>
</div>
</div></section><section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270018-sec-0040"><h4 class="pmc_sec_title">Core Ideas</h4>
<ul id="tpg270018-list-0001" class="list" style="list-style-type:disc">
<li><p>University of Florida strawberry breeding program nondestructively measured biomass through digital phenotyping (DP) methods.</p></li>
<li><p>These measurements allowed us to examine the inheritance of biomass and balance selection for yield and plant size.</p></li>
<li><p>Predictive abilities from genomic selection were moderate and will allow effective predictive breeding for biomass.</p></li>
<li><p>Improvement of DP methods for biomass estimation could also improve selection accuracy for biomass in the future.</p></li>
</ul></section></section></section><section id="tpg270018-sec-0050"><h3 class="pmc_sec_title">2.2. Phenotypic data collection</h3>
<section id="tpg270018-sec-0060"><h4 class="pmc_sec_title">2.2.1. Yield and fruit quality</h4>
<p>Weekly yield and fruit quality data were collected from mid‐November until mid‐March for all trials. During each harvest week, all ripe fruits per plant were removed and evaluated for six yield and fruit quality traits that were divided into two timepoints, early and total season. Early season is the harvest period between late November and January, while the total season is the harvest period between late November and March. The traits evaluated were early marketable yield (EMY; g/plant), total marketable yield (TMY; g/plant), average fruit weight (AWT; g/fruit), total culls (TC; number/plant), soluble solids content (SSC; %), and firmness (FRM; 1–5 score). All marketable fruit were considered as EMY if harvested before the first day of February. TMY was calculated as the marketable fruit collected until the first week of March. AWT in grams was estimated as the TMY divided by the number of marketable fruits. TC, or unmarketable fruit, were counted and expressed as a proportion of the total number of fruits per plant. SSC was measured five times during the season within each trial and was calculated as the mean of all measurements. At each measurement, one ripe fruit from each plant was squeezed by hand onto a handheld digital refractometer. FRM was measured as a score from a scale of 1–5 (Osorio et al., <a href="#tpg270018-bib-0025" class="usa-link" aria-describedby="tpg270018-bib-0025">2021</a>; Whitaker et al., <a href="#tpg270018-bib-0046" class="usa-link" aria-describedby="tpg270018-bib-0046"><sup>2012</sup></a>).</p></section><section id="tpg270018-sec-0070"><h4 class="pmc_sec_title">2.2.2. Digital phenotyping</h4>
<p>A ground‐based imaging platform described by Guan et al. (<a href="#tpg270018-bib-0015" class="usa-link" aria-describedby="tpg270018-bib-0015">2020</a>) and Abd‐Elrahman et al. (<a href="#tpg270018-bib-0052" class="usa-link" aria-describedby="tpg270018-bib-0052">2020</a>) was utilized to collect high‐resolution RGB and NIR images from the field trials during the 2017–2018 and 2018–2019. Image data were collected at two timepoints that represent the end points of the early season and total season. Canopy size metrics such as plant canopy area, volume, average height, and standard deviation of height were computed using a model used for all datasets acquired throughout the season as described in Abd‐Elrahman et al. (<a href="#tpg270018-bib-0052" class="usa-link" aria-describedby="tpg270018-bib-0052">2020</a>).</p>
<p>In the 2020–2021 and 2021–2022 seasons, an unmanned aerial vehicle (UAV) platform was developed to replace the ground‐based imaging platform for image acquisition. A MicaSense RedEdge‐M multispectral camera with a downwelling light sensor (DLS) mounted on a Da‐Jiang Innovations (DJI) Inspire 2 drone was used in this study. Since the UAV is easy to operate and control remotely, it was used to collect image data at more time points (13 and 26 for the 2020–2021 and 2021–2022 seasons, respectively) to track the growth of strawberry plants over time. Canopy size metrics and related vegetation indexes were extracted from the UAV imagery for vegetative biomass prediction. Image data processing and comparison between the UAV and ground‐based platform in this study were discussed in C. Zheng et al. (<a href="#tpg270018-bib-0048" class="usa-link" aria-describedby="tpg270018-bib-0048">2022a</a>).</p></section><section id="tpg270018-sec-0080"><h4 class="pmc_sec_title">2.2.3. Vegetative biomass prediction</h4>
<p>Biomass data were estimated using a modified regression model previously published by Guan et al. (<a href="#tpg270018-bib-0015" class="usa-link" aria-describedby="tpg270018-bib-0015">2020</a>). Canopy size metrics utilized in the model were generated from high‐resolution images acquired from genotypes where dry biomass using destructive methods was also measured to serve as the ground truth. This experiment was done at the UF/IFAS Gulf Coast Research and Education Center, in Balm, FL, on a subset of the UF breeding germplasm called the “phenomics trial” (Figure <a href="#tpg270018-fig-0001" class="usa-link">1</a>) following the formula:
</p>
<table class="disp-formula p" id="tpg270018-disp-0001"><tr><td class="formula"><math id="jats-math-1" display="block"><mrow><mtable displaystyle="true"><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mrow><mrow><mi>Vegetative</mi><mspace width="0.28em"></mspace><mi>biomass</mi><mspace width="0.28em"></mspace></mrow><mfenced open="(" close=")"><mi mathvariant="normal">g</mi></mfenced><mo linebreak="badbreak">=</mo><mo>−</mo><mn>22.95</mn><mo linebreak="goodbreak">+</mo><mfenced separators="" open="(" close=")"><mrow><mn>156.72</mn><mspace width="0.28em"></mspace><mo>×</mo><mspace width="0.28em"></mspace><mrow><mi>Canopy</mi><mspace width="0.28em"></mspace><mi>area</mi></mrow></mrow></mfenced></mrow></mtd></mtr><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mrow><mspace width="1em"></mspace><mo linebreak="badbreak">+</mo><mfenced separators="" open="(" close=")"><mrow><mn>266.50</mn><mo>×</mo><mrow><mi>Mean</mi><mspace width="0.28em"></mspace><mi>canopy</mi><mspace width="0.28em"></mspace><mi>height</mi></mrow></mrow></mfenced></mrow></mtd></mtr><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mrow><mspace width="1em"></mspace><mo linebreak="badbreak">+</mo><mfenced separators="" open="(" close=")"><mrow><mn>0.8</mn><mo>×</mo><mrow><mi>Weeks</mi><mspace width="0.28em"></mspace><mi>after</mi><mspace width="0.28em"></mspace><mi>planting</mi></mrow></mrow></mfenced><mo>.</mo></mrow></mtd></mtr></mtable></mrow></math></td></tr></table>
<p>Vegetative biomass that represents the early season was labeled as EBIO. Predicted vegetative biomass at the end of the season was labeled as BIO. Fruit yield data such as EMY, TMY, and AWT were then scaled on a biomass basis by dividing EMY, TMY, and AWT by the predicted vegetative biomass, which resulted in new traits of interest, namely, early AWT on a predicted biomass basis (EAWTB), AWT on a predicted biomass basis (AWTB), early marketable yield on a predicted biomass basis (EMYB), and total marketable yield on a predicted biomass basis (TMYB).</p></section><section id="tpg270018-sec-0090"><h4 class="pmc_sec_title">2.2.4. Accumulated predicted vegetative biomass</h4>
<p>In the 2020–2022 seasons, vegetative biomass was estimated using machine learning models previously proposed by C. Zheng et al. (<a href="#tpg270018-bib-0048" class="usa-link" aria-describedby="tpg270018-bib-0048">2022a</a>, 2022<a href="#tpg270018-bib-0049" class="usa-link" aria-describedby="tpg270018-bib-0049"><sup>b</sup></a>) using image‐derived canopy size metrics and several vegetation indices extracted from a MicaSense RedEdge‐M multispectral camera and a DLS mounted on a DJI Inspire 2 drone operated autonomously, with field flight parameters pre‐set using the Litchi flight planning software to cover the whole strawberry experiment area, which were collected 13 times during 2020–2021 and 26 times during 2021–2022. Using the estimated vegetative biomass data at multiple time points, the logistic function was adopted to fit the growth curve of each strawberry plant, which contains three parameters, as shown below:
</p>
<table class="disp-formula p" id="tpg270018-disp-0002"><tr><td class="formula"><math id="jats-math-2" display="block"><mrow><mrow><mi>y</mi><mo linebreak="badbreak">=</mo><mfrac><mi>k</mi><mrow><mn>1</mn><mo>+</mo><msup><mi>e</mi><mrow><msub><mi mathvariant="normal">ρ</mi><mn>0</mn></msub><mo>+</mo><mi mathvariant="normal">γ</mi><mi>x</mi></mrow></msup></mrow></mfrac><mo>,</mo></mrow></mrow></math></td></tr></table>
<p>where <em>x</em> represents the days after planting, <em>y</em> represents the predicted dry biomass data, and there are three coefficients (<em>k</em>, <em>ρ</em>
<sub>0</sub>, and <em>γ</em>). The logistic model can be dated back to the work of Verhulst (<a href="#tpg270018-bib-0041" class="usa-link" aria-describedby="tpg270018-bib-0041">1838</a>), who proposed a self‐limiting model of population growth with a function or curve representing an S‐shaped curve (Kawano et al., <a href="#tpg270018-bib-0019" class="usa-link" aria-describedby="tpg270018-bib-0019">2020</a>).</p>
<p>As shown in Figure <a href="#tpg270018-fig-0002" class="usa-link">2</a>, the fitted logistic function can reflect the growth trend of strawberry plants. The accumulated biomass was defined as the sum of biomass between the first and last days of data collection during the strawberry winter growing season.</p>
<figure class="fig xbox font-sm" id="tpg270018-fig-0002"><h5 class="obj_head">FIGURE 2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d601/11958871/a1b76945e723/TPG2-18-e70018-g002.jpg" loading="lazy" id="jats-graphic-3" height="365" width="400" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270018-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>An example of strawberry plant growth curve where the solid red line is the fitted logistic curve, and the discrete blue points are the biomass estimated using machine learning methods. The green shaded area represents the accumulated biomass during the strawberry winter growing season, spanning from the initial to the final days of data collection.</p></figcaption></figure><p>During the 2020–2021 and 2021–2022 seasons, the accumulated predicted vegetative biomass (ACCB) was recorded, and AWT as well as TMY were scaled on an ACCB basis (AWT/ACCB and TMY/ACCB, respectively) generating two new traits, AWT on an accumulated predicted vegetative biomass basis (AWTAC) and TMYAC.</p></section></section><section id="tpg270018-sec-0100"><h3 class="pmc_sec_title">2.3. SNP genotyping</h3>
<p>Two sets of single nucleotide polymorphism (SNP) data were utilized in this study. First, a reference population of 1228 individuals out of a total 1297 genotypes was genotyped using common markers across the Affymetrix Axiom IStraw90 (Bassil et al., <a href="#tpg270018-bib-0002" class="usa-link" aria-describedby="tpg270018-bib-0002">2015</a>) and IStraw35 arrays (Verma et al., <a href="#tpg270018-bib-0042" class="usa-link" aria-describedby="tpg270018-bib-0042">2017</a>). Quality control was implemented on a total of 14,332 segregating SNP markers in which SNPs with minor allele frequency (MAF) &lt; 0.05, and missing marker data &gt; 0.05 were removed, leaving a total of 5300 markers for the analyses. These markers were used for the 2017–2018 and 2018–2019 seasons. Missing values for each of the markers were imputed based on average allele frequency. Genotyping of the individuals from seasons 2020–2021 and 2021–2022 was performed utilizing 8749 molecular markers from the newer FanaSNP 50K array (Hardigan et al., <a href="#tpg270018-bib-0017" class="usa-link" aria-describedby="tpg270018-bib-0017">2020</a>), and a similar quality control was implemented. Due to the difference in the marker platforms used, and since different DP methods were used in the first 2 years versus the last 2 years, we took the conservative approach of conducting two analyses, one for 2017–2018 and 2018–2019 seasons and another for 2020–2021 and 2021–2022 seasons, as opposed to combining data across 4 years.</p></section><section id="tpg270018-sec-0110"><h3 class="pmc_sec_title">2.4. Genetic parameter estimation</h3>
<p>The estimation of the genetic parameters from the advanced selection trials and multi‐year trials followed the two‐stage approach proposed by Smith, Cullis, and Thompson (<a href="#tpg270018-bib-0035" class="usa-link" aria-describedby="tpg270018-bib-0035">2001</a>) and implemented in the ASRtriala software (Gezan et al., <a href="#tpg270018-bib-0012" class="usa-link" aria-describedby="tpg270018-bib-0012">2022</a>). These analyses were based on multiple linear mixed models, including the modeling and/or not of spatial model terms, row, column, autoregressive correlated errors, or independent residual units. All spatially adjusted phenotypic data used in this study are available in Table <a href="#tpg270018-supinfo-0001" class="usa-link">S1</a>.</p>
<p>In the first‐stage analysis, field observations of the single advanced trials were analyzed according to the following general linear mixed model:
</p>
<table class="disp-formula p" id="tpg270018-disp-0003"><tr><td class="formula"><math id="jats-math-3" display="block"><mrow><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>B</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><mi>c</mi><msub><mfenced open="(" close=")"><mi>B</mi></mfenced><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">+</mo><mi>r</mi><msub><mfenced open="(" close=")"><mi>B</mi></mfenced><mrow><mi>i</mi><mi>k</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>g</mi><mi>l</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub><mo>,</mo></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><mi mathvariant="normal">μ</mi></mrow></math></span> is the general mean, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-5" display="inline"><mrow><mrow><msub><mi>B</mi><mi>i</mi></msub><mspace width="0.28em"></mspace></mrow></mrow></math></span> is the fixed replication effect, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><mrow><mi>c</mi><msub><mrow><mo stretchy="false">(</mo><mi>B</mi><mo stretchy="false">)</mo></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></span> is the column within replication random effect, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><mrow><mi>r</mi><msub><mrow><mo stretchy="false">(</mo><mi>B</mi><mo stretchy="false">)</mo></mrow><mrow><mi>i</mi><mi>k</mi></mrow></msub></mrow></mrow></math></span> is the random row within replication effect, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><msub><mi>g</mi><mi>l</mi></msub></mrow></math></span> is the random clonal effect, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub></mrow></math></span> is the residual random effect of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><mrow><mspace width="0.28em"></mspace><mi>l</mi></mrow></mrow></math></span> clone in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline"><mrow><mrow><mi>j</mi><mspace width="0.28em"></mspace></mrow></mrow></math></span> column and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-12" display="inline"><mrow><mi>k</mi></mrow></math></span> row of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline"><mrow><mi>i</mi></mrow></math></span> replication. The residual effect (<em>R</em>) for the spatial models of the <em>t</em> trial fitted the autoregressive variance‐covariance matrix of order 1 of the form:
</p>
<table class="disp-formula p" id="tpg270018-disp-0004"><tr><td class="formula"><math id="jats-math-14" display="block"><mrow><mrow><msub><mi>R</mi><mi>t</mi></msub><mo linebreak="badbreak">=</mo><msubsup><mi mathvariant="normal">σ</mi><mi>t</mi><mn>2</mn></msubsup><mspace width="0.28em"></mspace><munder><mo>∑</mo><msub><mi>c</mi><mi>t</mi></msub></munder><mfenced separators="" open="(" close=")"><msub><msub><mi mathvariant="normal">ρ</mi><mi>c</mi></msub><mi>t</mi></msub></mfenced><mo>⊗</mo><munder><mo>∑</mo><msub><mi>r</mi><mi>t</mi></msub></munder><mfenced separators="" open="(" close=")"><msub><msub><mi mathvariant="normal">ρ</mi><mi>r</mi></msub><mi>t</mi></msub></mfenced></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-15" display="inline"><mrow><mrow><msub><mo>∑</mo><msub><mi>c</mi><mi>t</mi></msub></msub><mrow><mo stretchy="false">(</mo><msub><msub><mi mathvariant="normal">ρ</mi><mi>c</mi></msub><mi>t</mi></msub><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-16" display="inline"><mrow><mrow><msub><mo>∑</mo><msub><mi>r</mi><mi>t</mi></msub></msub><mrow><mo stretchy="false">(</mo><msub><msub><mi mathvariant="normal">ρ</mi><mi>r</mi></msub><mi>t</mi></msub><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span> are spatial correlation structures in the column and row directions, respectively.</p>
<p>There were 14 models tested for each trait within the early and total periods of each year, including the presence or absence of some model factors (Table <a href="#tpg270018-supinfo-0001" class="usa-link">S2</a>). The best‐fitted model for each trait, period, and year was selected based on the heritability of the prediction error variance (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-17" display="inline"><mrow><msubsup><mi>h</mi><mi>PEV</mi><mn>2</mn></msubsup></mrow></math></span>), a goodness of fit criterion that is commonly used to compare different models with and/or without spatial factors (Table <a href="#tpg270018-supinfo-0001" class="usa-link">S3</a>). These single‐stage analyses provide variance components for the random effects, genetic predictions for all genotypes and their corresponding weights (Gezan et al., <a href="#tpg270018-bib-0012" class="usa-link" aria-describedby="tpg270018-bib-0012">2022</a>).</p>
<p>The genotype by year interaction (<em>G</em> × <em>Y</em>) of EBIO, BIO, and the scaled traits EAWTB, AWTB, EMYB, and TMYB were estimated in the second‐stage analyses. The genetic predictions from the first‐stage model, with random genotype effects, were Best Linear Unbiased Predictions (BLUPs) and cannot be used in a multi‐year trial because they were shrunken (Smith, Cullis, &amp; Thompson, <a href="#tpg270018-bib-0035" class="usa-link" aria-describedby="tpg270018-bib-0035">2001</a>). Therefore, the previous model was fitted with genotype as a random effect, and the variance components from the first stage were fixed, once it ran, the model is refitted again with genotype fixed to obtain unshrunken genotype predictions. These new predictions are Best Linear Unbiased Estimates (BLUEs), comparable across trials, and the estimated weights from the stage 1 analyses are included into a weighted linear mixed model in the across‐year analyses (Gezan et al., <a href="#tpg270018-bib-0012" class="usa-link" aria-describedby="tpg270018-bib-0012">2022</a>). Details of the statistical model can be found in Smith, Cullis, and Gilmour (<a href="#tpg270018-bib-0034" class="usa-link" aria-describedby="tpg270018-bib-0034">2001</a>) and Gogel et al. (<a href="#tpg270018-bib-0014" class="usa-link" aria-describedby="tpg270018-bib-0014">2018</a>). The same procedure was followed to estimate the <em>G</em> × <em>Y</em> interaction of the same traits from seasons 2020 to 2021 and 2021 to 2022, plus the additional ACCB trait and the scaled traits AWTAC and TMYAC.</p>
<p>The genetic correlations among multiple traits within a season were estimated using the Bayesian multi‐environment model (BME) function of the Bayesian multi‐trait and multi‐environment (BMTME) R software developed by Montesinos‐López et al. (<a href="#tpg270018-bib-0023" class="usa-link" aria-describedby="tpg270018-bib-0023">2019</a>). Details on the linear mixed model and the script used to estimate the genetic correlations among traits are presented in the cited article.</p></section><section id="tpg270018-sec-0120"><h3 class="pmc_sec_title">2.5. Genome‐wide association study for predicted vegetative biomass and accumulated biomass</h3>
<p>Spatially adjusted data generated from the two‐stage analysis of predicted vegetative biomass from 2017 to 2018, 2018 to 2019, and 2020 to 2021 and accumulated biomass from 2020 to 2021 and 2021 to 2022 through ASRtriala were used as the phenotype input for GWAS. Analyses were conducted separately by season, using SNP genotypes from a set of common markers across the Affymetrix Axiom IStraw90 (Bassil et al., <a href="#tpg270018-bib-0002" class="usa-link" aria-describedby="tpg270018-bib-0002">2015</a>), IStraw35 (Verma et al., <a href="#tpg270018-bib-0042" class="usa-link" aria-describedby="tpg270018-bib-0042">2017</a>), and FanaSNP 50k (Hardigan et al., <a href="#tpg270018-bib-0017" class="usa-link" aria-describedby="tpg270018-bib-0017">2020</a>) SNP arrays. GWAS analysis was conducted using a Bayesian information and linkage disequilibrium iteratively nested keyway method implemented in GAPIT3 (Genome Association and Prediction Integrated Tool 3) R package (Wang &amp; Zhang, <a href="#tpg270018-bib-0043" class="usa-link" aria-describedby="tpg270018-bib-0043">2021</a>) for both predicted vegetative biomass and accumulated biomass with the inclusion of the top five principal components and markers with MAF &gt; 0.05. Relatedness was accounted for using a kinship matrix estimated with the “VanRaden” algorithm implemented in GAPIT. Significant marker‐trait associations were determined using both 1% and 5% Bonferroni correction.</p></section><section id="tpg270018-sec-0130"><h3 class="pmc_sec_title">2.6. Genomic selection for predicted vegetative biomass and accumulated biomass</h3>
<p>GS was implemented using 2017–2018 as a training population to predict 2018–2019 and using 2020–2021 to predict 2021–2022, in an independent validation approach (separate years as opposed to cross‐validation). Common individuals between seasons were masked in the testing set, and genotypes not previously tested were predicted. The Bayesian ridge regression (BRR) model was fitted in the R programming language (R Core Team, 2021), and run in Rstudio (Rstudio Team, 2019), using the R package BGLR (Bayesian Generalized Linear Regression) (Pérez &amp; De Los Campos, <a href="#tpg270018-bib-0027" class="usa-link" aria-describedby="tpg270018-bib-0027">2014</a>). The package implemented a Markov Chain Monte Carlo method with 15,000 iterations, where the first 5000 were used as a burn‐in. Predictive ability (PA), reported as the Pearson correlation coefficient <em>r</em> between the observed spatially adjusted phenotypes and the genomic‐estimated breeding values (GEBVs) of the testing sets (Osorio et al., <a href="#tpg270018-bib-0025" class="usa-link" aria-describedby="tpg270018-bib-0025">2021</a>), was estimated for both seasons 2018–2019 and 2021–2022. Genomic narrow‐sense heritability was estimated using BRR, as follows:
</p>
<table class="disp-formula p" id="tpg270018-disp-0005"><tr><td class="formula"><math id="jats-math-18" display="block"><mrow><mrow><msup><mi>h</mi><mn>2</mn></msup><mo linebreak="badbreak">=</mo><mfrac><mrow><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>p</mi><mspace width="0.28em"></mspace><mo>−</mo><mspace width="0.28em"></mspace></mrow><mn>2</mn></msubsup><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow><msubsup><mi mathvariant="normal">σ</mi><mi>p</mi><mn>2</mn></msubsup></mfrac><mo>,</mo></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-19" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>p</mi><mn>2</mn></msubsup></mrow></math></span> is the phenotypic variance and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-20" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow></math></span> is the estimated residual variance.</p></section></section><section id="tpg270018-sec-0140"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270018-sec-0150"><h3 class="pmc_sec_title">3.1. Phenotypic variability and trait correlations</h3>
<p>The phenotypic variation of the predicted vegetative dry biomass during the early and the total season is represented in Table <a href="#tpg270018-supinfo-0001" class="usa-link">S4</a>. Predicted EBIO ranged from 8.5 to 47 g/plant across years. Predicted BIO ranged from 12.2 to 75.7 g/plant across years. Scaled traits such as EAWTB, EMYB, AWTB, and TMYB ranged from 0.3 to 1.8 g fruit weight/g biomass, 0.0 to 18.7 g marketable weight/g biomass, 0.3 to 1.6 g fruit weight/g biomass, and 0.5 to 31.8 g marketable weight/g biomass, respectively.</p>
<p>The genetic correlations between predicted biomass and marketable yield were positive and low to moderate (<em>r<sub>G </sub>
</em>= 0.16) for the total season in 2017–2018 (Figure <a href="#tpg270018-fig-0003" class="usa-link">3</a>). A similar trend was also seen in the total season in 2018–2019 (<em>r<sub>G </sub>
</em>= 0.40) but was slightly higher than the first season. The genetic correlations between the two seasons for predicted biomass (<em>r<sub>G×Y</sub> </em>= 0.57, 0.80), AWT (<em>r<sub>G×Y</sub> </em>= 0.69, 0.87), and marketable yield (<em>r<sub>G×Y</sub> </em>= 0.60, 0.75) indicated moderate to low genotype by year interactions (Figure <a href="#tpg270018-supinfo-0001" class="usa-link">S1</a>). Scatter plots, histograms, and phenotypic correlations for the predicted biomass and biomass‐related traits with the fruit yield and quality traits are shown in Figure <a href="#tpg270018-supinfo-0001" class="usa-link">S2</a>.</p>
<figure class="fig xbox font-sm" id="tpg270018-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d601/11958871/cd2c362c3f12/TPG2-18-e70018-g003.jpg" loading="lazy" id="jats-graphic-5" height="632" width="600" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270018-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genetic correlations of predicted biomass and biomass‐related traits (BIO, predicted vegetative biomass; ACCB, accumulated predicted vegetative biomass; AWTB, average fruit weight on a predicted biomass basis; MYB, marketable fruit yield on a predicted biomass basis; AWTAC, average fruit weight on an accumulated predicted vegetative biomass basis; TMYAC, total marketable fruit yield on an accumulated predicted vegetative biomass basis) with yield and fruit traits (AWT, average weight; TMY, total marketable yield) during (a) the total season 2017–2018, (b) total season 2018–2019, (c) total season 2020–2021, and (e) total season 2021–2022. Early season showed similar trend as the total season. TMYB, total marketable yield on a predicted biomass basis.</p></figcaption></figure><p>During 2020–2021 and 2021–2022, aside from a different SNP marker set, we were able to access weekly drone image data, which generated another trait, the accumulated predicted vegetative biomass (ACCB), a measure of biomass over several timepoints in a growing season. This is presented as a growth‐related curve in Figure <a href="#tpg270018-supinfo-0001" class="usa-link">S3</a>. The genetic correlations between biomass and marketable yield were negative and low (<em>r<sub>G </sub>
</em>= −0.13) in 2020–2021 but positive and low in 2021–2022 (<em>r<sub>G </sub>
</em>= 0.14) season (Figure <a href="#tpg270018-fig-0003" class="usa-link">3</a>). The phenotypic correlation for 2020–2021 (Figure <a href="#tpg270018-supinfo-0001" class="usa-link">S2</a>) between biomass and yield was positive and low, in contrast to the genetic correlation for that season. For ACCB, the genetic correlation with marketable yield was low in 2020–2021 (<em>r<sub>G </sub>
</em>= 0.17) and moderate in 2021–2022 (<em>r<sub>G </sub>
</em>= 0.43). The genetic correlations between the two seasons for ACCB (<em>r<sub>G×Y</sub> </em>= 0.88), predicted biomass (<em>r<sub>G×Y</sub> </em>= 0.91), AWT (<em>r<sub>G×Y</sub> </em>= 0.9), and marketable yield (<em>r<sub>G×Y</sub> </em>= 0.61) also indicated moderate to low genotype by year interactions (Figure <a href="#tpg270018-supinfo-0001" class="usa-link">S4</a>).</p>
<p>The heritability of the predicted biomass and biomass‐related traits were moderate to high and comparable with yield traits, with <em>h</em>
<sup>2</sup> = 0.48 and <em>h</em>
<sup>2</sup> = 0.49 for the EBIO 2017–2018 and 2018–2019 seasons, respectively, and with <em>h</em>
<sup>2</sup> = 0.28 and <em>h</em>
<sup>2</sup> = 0.26 for EBIO 2020–2021 and 2021–2022, respectively (Table <a href="#tpg270018-tbl-0002" class="usa-link">2</a>). For BIO, heritabilities were similar, with <em>h</em>
<sup>2</sup> = 0.50 and <em>h</em>
<sup>2</sup> = 0.56 for the 2017–2018 and 2018–2019 seasons, respectively, and <em>h</em>
<sup>2</sup> = 0.34 and <em>h</em>
<sup>2</sup> = 0.30 for 2020–2021 and 2021–2022, respectively. For ACCB, heritabilities were <em>h</em>
<sup>2</sup> = 0.27 and <em>h</em>
<sup>2</sup> = 0.27 for 2020–2021 and 2021–2022, respectively. For the predicted biomass scaled traits, the heritability estimates were also moderate to high, with heritability ranging from <em>h</em>
<sup>2</sup> = 0.25–0.58 across seasons for EAWTB, <em>h</em>
<sup>2</sup> = 0.39–0.60 across seasons for EMYB, <em>h</em>
<sup>2</sup> = 0.44–0.64 across seasons for AWTB, and <em>h</em>
<sup>2</sup> = 0.49–0.64 across seasons for TMYB.</p>
<section class="tw xbox font-sm" id="tpg270018-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Narrow‐sense heritability (<em>h</em>
<sup>2</sup>) of predicted vegetative biomass, biomass‐related traits, and yield traits in the University of Florida strawberry breeding population during the 2017–2018, 2018–2019, 2020–2021, and 2021–2022 seasons.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead>
<tr>
<th align="left" rowspan="1" colspan="1"></th>
<th colspan="4" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Heritability <em>h</em>
<sup>2</sup> (SE)</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Traits</th>
<th style="border-bottom:solid 1px #000000" align="left" rowspan="1" colspan="1">2017–2018</th>
<th style="border-bottom:solid 1px #000000" align="left" rowspan="1" colspan="1">2018–2019</th>
<th style="border-bottom:solid 1px #000000" align="left" rowspan="1" colspan="1">2020–2021</th>
<th style="border-bottom:solid 1px #000000" align="left" rowspan="1" colspan="1">2021–2022</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td colspan="4" align="left" rowspan="1">Early</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Predicted biomass traits</strong>
</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EBIO</td>
<td align="left" rowspan="1" colspan="1">0.48 (0.40)</td>
<td align="left" rowspan="1" colspan="1">0.49 (0.44)</td>
<td align="left" rowspan="1" colspan="1">0.28 (0.41)</td>
<td align="left" rowspan="1" colspan="1">0.26 (0.41)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EAWTB</td>
<td align="left" rowspan="1" colspan="1">0.45 (0.09)</td>
<td align="left" rowspan="1" colspan="1">0.58 (0.08)</td>
<td align="left" rowspan="1" colspan="1">0.41 (0.10)</td>
<td align="left" rowspan="1" colspan="1">0.25 (0.09)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EMYB</td>
<td align="left" rowspan="1" colspan="1">0.44 (0.17)</td>
<td align="left" rowspan="1" colspan="1">0.39 (0.12)</td>
<td align="left" rowspan="1" colspan="1">0.47 (0.14)</td>
<td align="left" rowspan="1" colspan="1">0.60 (0.12)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ACCB</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Yield and fruit traits</strong>
</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EAWT</td>
<td align="left" rowspan="1" colspan="1">0.45 (0.16)</td>
<td align="left" rowspan="1" colspan="1">0.36 (0.21)</td>
<td align="left" rowspan="1" colspan="1">0.44 (0.35)</td>
<td align="left" rowspan="1" colspan="1">0.65 (0.30)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EMKN</td>
<td align="left" rowspan="1" colspan="1">0.44 (0.18)</td>
<td align="left" rowspan="1" colspan="1">0.36 (0.12)</td>
<td align="left" rowspan="1" colspan="1">0.41 (0.13)</td>
<td align="left" rowspan="1" colspan="1">0.54 (0.15)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EMY</td>
<td align="left" rowspan="1" colspan="1">0.39 (0.09)</td>
<td align="left" rowspan="1" colspan="1">0.33 (0.08)</td>
<td align="left" rowspan="1" colspan="1">0.41 (0.47)</td>
<td align="left" rowspan="1" colspan="1">0.49 (0.55)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ESSC</td>
<td align="left" rowspan="1" colspan="1">0.32 (0.09)</td>
<td align="left" rowspan="1" colspan="1">0.28 (0.08)</td>
<td align="left" rowspan="1" colspan="1">0.23 (0.09)</td>
<td align="left" rowspan="1" colspan="1">0.27 (0.09)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ETC</td>
<td align="left" rowspan="1" colspan="1">0.20 (0.39)</td>
<td align="left" rowspan="1" colspan="1">0.14 (0.23)</td>
<td align="left" rowspan="1" colspan="1">0.20 (0.08)</td>
<td align="left" rowspan="1" colspan="1">0.08 (0.06)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FRM</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td colspan="4" align="left" rowspan="1">Total</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Predicted biomass traits</strong>
</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">BIO</td>
<td align="left" rowspan="1" colspan="1">0.50 (0.90)</td>
<td align="left" rowspan="1" colspan="1">0.56 (0.55)</td>
<td align="left" rowspan="1" colspan="1">0.34 (0.73)</td>
<td align="left" rowspan="1" colspan="1">0.30 (0.50)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AWTB</td>
<td align="left" rowspan="1" colspan="1">0.57 (0.09)</td>
<td align="left" rowspan="1" colspan="1">0.64 (0.07)</td>
<td align="left" rowspan="1" colspan="1">0.50 (0.10)</td>
<td align="left" rowspan="1" colspan="1">0.44 (0.10)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TMYB</td>
<td align="left" rowspan="1" colspan="1">0.54 (0.17)</td>
<td align="left" rowspan="1" colspan="1">0.49 (0.18)</td>
<td align="left" rowspan="1" colspan="1">0.63 (0.28)</td>
<td align="left" rowspan="1" colspan="1">0.64 (0.22)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ACCB</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.27 (0.38)</td>
<td align="left" rowspan="1" colspan="1">0.27 (0.61)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Yield and fruit traits</strong>
</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AWT</td>
<td align="left" rowspan="1" colspan="1">0.56 (0.18)</td>
<td align="left" rowspan="1" colspan="1">0.70 (0.13)</td>
<td align="left" rowspan="1" colspan="1">0.46 (0.37)</td>
<td align="left" rowspan="1" colspan="1">0.83 (0.21)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">MKN</td>
<td align="left" rowspan="1" colspan="1">0.53 (0.36)</td>
<td align="left" rowspan="1" colspan="1">0.44 (0.48)</td>
<td align="left" rowspan="1" colspan="1">0.55 (0.32)</td>
<td align="left" rowspan="1" colspan="1">0.50 (0.43)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TMY</td>
<td align="left" rowspan="1" colspan="1">0.48 (0.10)</td>
<td align="left" rowspan="1" colspan="1">0.43 (0.10)</td>
<td align="left" rowspan="1" colspan="1">0.50 (0.11)</td>
<td align="left" rowspan="1" colspan="1">0.46 (0.11)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SSC</td>
<td align="left" rowspan="1" colspan="1">0.37 (0.09)</td>
<td align="left" rowspan="1" colspan="1">0.37 (0.08)</td>
<td align="left" rowspan="1" colspan="1">0.44 (0.10)</td>
<td align="left" rowspan="1" colspan="1">0.11 (0.06)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TC</td>
<td align="left" rowspan="1" colspan="1">0.33 (0.41)</td>
<td align="left" rowspan="1" colspan="1">0.17 (0.18)</td>
<td align="left" rowspan="1" colspan="1">0.19 (0.08)</td>
<td align="left" rowspan="1" colspan="1">0.16 (0.07)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FRM</td>
<td align="left" rowspan="1" colspan="1">0.50 (0.08)</td>
<td align="left" rowspan="1" colspan="1">0.68 (0.07)</td>
<td align="left" rowspan="1" colspan="1">0.57 (0.09)</td>
<td align="left" rowspan="1" colspan="1">0.44 (0.09)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270018-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270018-tbl2-note-0001"><p>
<em>Note</em>: Values given in parentheses are standard error.</p></div>
<div class="fn" id="tpg270018-tbl2-note-0002"><p>Abbreviations: ACCB, accumulated predicted vegetative biomass; AWT, average fruit weight; AWTB, average fruit weight on a predicted vegetative biomass basis; BIO, predicted vegetative biomass; EAWTB, early average fruit weight on a predicted vegetative biomass basis; EBIO, vegetative biomass that represents the early season; EMKN, early marketable number; EMY, early marketable yield; EMYB, early marketable yield on a predicted vegetative biomass basis; ESSC, early solubale solidf content; ETC; early total culls; FRM, firmness; MKN, marketable number per plant; SSC, soluble solids content; TC, total culls; TMY, total marketable yield; TMYB, total marketable yield on a predicted vegetative biomass basis.</p></div>
</div></section></section><section id="tpg270018-sec-0160"><h3 class="pmc_sec_title">3.2. Genome‐wide association studies</h3>
<p>To assess the genetic architecture of predicted biomass, we conducted a genome‐wide association analysis of BIO for the 2017–2018 and 2018–2019 seasons (Figure <a href="#tpg270018-fig-0004" class="usa-link">4</a>) and ACCB for the 2020–2021 and 2021–2022 seasons (Figure <a href="#tpg270018-fig-0005" class="usa-link">5</a>). Results showed low‐level peaks with no consistent SNPs associated with BIO and ACCB across years. This is evident from the lack of consistent peaks across environments in the Manhattan plots shown in Figures <a href="#tpg270018-fig-0004" class="usa-link">4</a> and <a href="#tpg270018-fig-0005" class="usa-link">5</a> as compared to the high‐level peaks observed from strawberry disease resistance studies like in <em>FaRCa1</em> (Salinas et al., <a href="#tpg270018-bib-0033" class="usa-link" aria-describedby="tpg270018-bib-0033">2019</a>), a major locus that confers resistance to anthracnose fruit rot.</p>
<figure class="fig xbox font-sm" id="tpg270018-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d601/11958871/a892255edab2/TPG2-18-e70018-g004.jpg" loading="lazy" id="jats-graphic-7" height="239" width="800" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270018-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome‐wide association studies (GWAS) analysis of predicted biomass during (a) the early season 2017–2018, (b) total season 2017–2018, (c) early season 2018–2019, and (d) total season 2018–2019 showing no consistent peaks across years.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270018-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d601/11958871/aac317826282/TPG2-18-e70018-g005.jpg" loading="lazy" id="jats-graphic-9" height="510" width="798" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270018-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome‐wide association studies (GWAS) analysis of accumulated biomass during (a) the total season 2020–2021 and (b) total season 2021–2022 showing no consistent peaks across years.</p></figcaption></figure></section><section id="tpg270018-sec-0170"><h3 class="pmc_sec_title">3.3. Genomic selection</h3>
<p>GS was tested as a breeding strategy for predicted vegetative biomass and biomass‐related traits in our breeding program (Table <a href="#tpg270018-tbl-0003" class="usa-link">3</a>). Since two different sets of SNP markers were used across years, we generated separate PAs with BRR as the model. The PA for BIO and BIO‐related traits in 2017–2018 (2017–2018 season was used as the training set and 2018–2019 season was used as the testing set) ranged from 0.24 to 0.46. The PA for BIO and BIO related traits in 2020–2021 (2020–2021 season was used as the training set and 2021–2022 season was used as the testing set) ranged from 0.25 to 0.36. The correlation of heritability in 2018 and PA in 2017–2018 was 0.67, while the correlation of heritability in 2021 and PA in 2020–2021 was 0.92, with ACCB included. PA for EAWT, EMY, AWT, and TMY were also included for comparison since those are traits routinely used in GS in the breeding program (Table <a href="#tpg270018-tbl-0003" class="usa-link">3</a>).</p>
<section class="tw xbox font-sm" id="tpg270018-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Predictive ability for predicted vegetative biomass, biomass‐related traits, and yield and fruit traits in the University of Florida strawberry breeding population during the 2017–2018, 2018–2019, 2020–2021, and 2021–2022 seasons.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead>
<tr>
<th align="left" rowspan="1" colspan="1"></th>
<th colspan="2" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Predictive ability (PA)</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Traits</th>
<th align="left" rowspan="1" colspan="1">2017–2018/2018−2019</th>
<th align="left" rowspan="1" colspan="1">2020–2021/2021−2022</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td colspan="2" align="left" rowspan="1">Early</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Predicted biomass traits</strong>
</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EBIO</td>
<td align="left" rowspan="1" colspan="1">0.24</td>
<td align="left" rowspan="1" colspan="1">0.20</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EAWTB</td>
<td align="left" rowspan="1" colspan="1">0.31</td>
<td align="left" rowspan="1" colspan="1">0.15</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EMYB</td>
<td align="left" rowspan="1" colspan="1">0.24</td>
<td align="left" rowspan="1" colspan="1">0.31</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ACCB</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Yield and fruit traits</strong>
</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EAWT</td>
<td align="left" rowspan="1" colspan="1">0.35</td>
<td align="left" rowspan="1" colspan="1">0.30</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EMY</td>
<td align="left" rowspan="1" colspan="1">0.22</td>
<td align="left" rowspan="1" colspan="1">0.28</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td colspan="2" align="left" rowspan="1">Total</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Predicted biomass traits</strong>
</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">BIO</td>
<td align="left" rowspan="1" colspan="1">0.41</td>
<td align="left" rowspan="1" colspan="1">0.23</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AWTB</td>
<td align="left" rowspan="1" colspan="1">0.46</td>
<td align="left" rowspan="1" colspan="1">0.21</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TMYB</td>
<td align="left" rowspan="1" colspan="1">0.33</td>
<td align="left" rowspan="1" colspan="1">0.36</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ACCB</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.24</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Yield and fruit traits</strong>
</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AWT</td>
<td align="left" rowspan="1" colspan="1">0.41</td>
<td align="left" rowspan="1" colspan="1">0.45</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TMY</td>
<td align="left" rowspan="1" colspan="1">0.25</td>
<td align="left" rowspan="1" colspan="1">0.23</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270018-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270018-tbl3-note-0001"><p>
<em>Note</em>: The first year as the training set and the second year as the testing set for independent validation.</p></div>
<div class="fn" id="tpg270018-tbl3-note-0002"><p>Abbreviations: ACCB, accumulated predicted vegetative biomass; AWT, average fruit weight; AWTB, average fruit weight on a predicted biomass basis; BIO, predicted vegetative biomass; EAWTB, early average fruit weight on a predicted biomass basis; EBIO, vegetative biomass that represents the early season; EMY, early marketable yield; EMYB, early marketable yield on a predicted biomass basis; TMY, total marketable yield; TMYB, total marketable yield on a predicted biomass basis.</p></div>
</div></section></section></section><section id="tpg270018-sec-0180"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>Vegetative biomass is an important trait to help assess a plant's yield potential. However, traditional measurement of vegetative biomass is destructive and time consuming. In a breeding program, it is essential to collect yield and fruit quality data at every harvest. Thus, destroying field plots to measure biomass manually is not a viable option. With the emergence of DP technologies, it has become possible to estimate vegetative biomass at scale.</p>
<section id="tpg270018-sec-0190"><h3 class="pmc_sec_title">4.1. Trait correlations and heritability</h3>
<p>The phenotypic and genetic correlations between predicted biomass and marketable yield in both the early season and total season were generally positive but low in magnitude (Figure <a href="#tpg270018-fig-0003" class="usa-link">3</a>). This suggests that fruit yield can be improved without a corresponding increase in plant biomass. The moderate heritability of biomass‐scaled yield also suggests that improvements in yield on a plant biomass basis are quite feasible. Overall, heritability estimates for biomass traits were in similar ranges with yield traits, suggesting that directional selection for biomass will also result in genetic gains (Table <a href="#tpg270018-tbl-0002" class="usa-link">2</a>). This is noteworthy since the biomass traits are not ground‐truth measurements. EBIO and BIO are based on size metrics extracted from high‐resolution images that were used to develop a regression model modified from Guan et al. (<a href="#tpg270018-bib-0015" class="usa-link" aria-describedby="tpg270018-bib-0015">2020</a>) with an adjusted <em>R</em>
<sup>2</sup> of 0.77. If the accuracy of biomass estimation can be increased, we should expect increases in heritability above what are reported here.</p>
<p>It is important to reiterate that our goal in strawberry breeding is not to increase vegetative biomass. In the strawberry industry, cost of labor is a significant bottleneck with the majority being used on harvesting. Thus, the goal is to select for high‐yielding varieties without simultaneously increasing plant biomass. More biomass could mean a larger canopy and/or increased canopy density. A moderate plant size allows strawberry plants to fit within the desired in‐row spacings that could maximize yield. In addition, plant density must not be increased too much, as air circulation within the plant and the ability for harvesters to visualize fruit under the canopy is critical. Currently, the recommended plant populations in Florida are 16,000–20,000 plants per acre on a two‐row bed with distance of 12–16 in. within row and 12–14 in. between row (Whitaker et al., <a href="#tpg270018-bib-0044" class="usa-link" aria-describedby="tpg270018-bib-0044">2023</a>). A moderate plant size also allows fruits to be more visible and thus helps increase efficiency of manual harvesting in contrast to larger plant size, which tends to hide fruits under its canopy. Thus, it was important to examine scaled traits representing yield‐related traits on plant biomass basis. These scaled traits are meant to capture a roughly similar concept to harvest index, which is the ratio of yield to total harvested aboveground biomass in grain crops (Brandán et al., <a href="#tpg270018-bib-0003" class="usa-link" aria-describedby="tpg270018-bib-0003">2009</a>; Porker et al., <a href="#tpg270018-bib-0032" class="usa-link" aria-describedby="tpg270018-bib-0032"><sup>2020</sup></a>; Van Hooijdonk et al., <a href="#tpg270018-bib-0040" class="usa-link" aria-describedby="tpg270018-bib-0040"><sup>2018</sup></a>). However, we should also reiterate that the yield traits that we measure are only from marketable fruits, and that our ground truth biomass data do not include reproductive parts and runners but only leaves and stems. The heritability estimates for the biomass‐scaled traits are moderate and thus can be improved in our breeding program.</p>
<p>In 2020–2021 and 2021–2022 seasons, we were able to obtain UAV‐derived biomass data (C. Zheng et al., <a href="#tpg270018-bib-0048" class="usa-link" aria-describedby="tpg270018-bib-0048"><sup>2022a</sup></a>). Unlike the ground‐based method, the UAV could be flown weekly to collect image data. This allowed the calculation of ACCB. This trait better reflects the vegetative growth trajectory of a strawberry plant over the season, which may give more in‐depth insights into how vegetative biomass relates to fruit production. Interestingly, in this study, the accumulated biomass traits did not have higher heritabilities or predictive abilities compared to the biomass estimates at single time points. Thus, it is not clear whether weekly UAV imaging can provide additional value over single flights at the end of the early season and total season when predicting vegetative biomass. If more time points do not provide an advantage, they would represent a waste of personnel and computing resources. A study on the rate of growth of individual genotypes using weekly UAV imaging could be a potential trait to explore that could be used in phenomic selection.</p></section><section id="tpg270018-sec-0200"><h3 class="pmc_sec_title">4.2. Genome‐wide studies</h3>
<p>GWAS analysis did not reveal consistent peaks across trials. This was not unexpected, as biomass should be controlled by a large number of physiological, biochemical, and genetic factors. This was also a strong indication that genomic prediction would be the best strategy for DNA‐informed breeding for these traits (Clark et al., <a href="#tpg270018-bib-0005" class="usa-link" aria-describedby="tpg270018-bib-0005">2019</a>; Tilhou &amp; Casler, <a href="#tpg270018-bib-0037" class="usa-link" aria-describedby="tpg270018-bib-0037"><sup>2022</sup></a>; Toda et al., <a href="#tpg270018-bib-0038" class="usa-link" aria-describedby="tpg270018-bib-0038"><sup>2021</sup></a>). Predictive abilities for the predicted biomass and biomass‐related traits in this study were similar to the yield traits in a scenario in which one field trial was used to predict the next year's trial. This suggests that genomic prediction can effectively be applied for these traits when predicting untested genotypes in new environments. Therefore, inclusion of vegetative biomass in the multi‐trait index or the use of biomass‐scaled yield traits in GS should help balance genetic gains for fruit and plant traits.</p></section></section><section id="tpg270018-sec-0210"><h2 class="pmc_sec_title">5. CONCLUSIONS</h2>
<p>With the advent of DP technologies, we were able to measure aboveground vegetative biomass on a large scale, a trait that we have not measured before in the UF strawberry breeding program. This approach is allowing our program to efficiently balance selection for fruit yield and plant size within the context of GS for the first time. We expect that as additional data are accumulated over seasons, increasing the training population size, predictive abilities will continue to improve. Further improvements in the accuracy of DP methods for biomass estimation could also help improve the accuracy of selection for vegetative biomass traits in the future.</p></section><section id="tpg270018-sec-0220"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Cheryl Dalid</strong>: Conceptualization; data curation; formal analysis; project administration; visualization; writing—original draft. <strong>Caiwang Zheng</strong>: Data curation; formal analysis; writing—review and editing. <strong>Luis Osorio</strong>: Data curation; formal analysis; writing—review and editing. <strong>Sujeet Verma</strong>: Formal analysis; writing—review and editing. <strong>Amr Abd‐Elrahman</strong>: Resources; writing—review and editing. <strong>Xu Wang</strong>: Resources; writing—review and editing. <strong>Vance M. Whitaker</strong>: Conceptualization; funding acquisition; project administration; supervision; writing—original draft.</p></section><section id="tpg270018-sec-0240"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec24"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270018-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270018-supitem-0001"><div class="caption p">
<p>Supplemental Figure S1. Genotype by year interaction for (a) early season predicted biomass, (b) early season average fruit weight, (c) early season marketable yield, (d) total season predicted biomass, (e) total season average fruit weight, and (f) total season marketable yield during the 2017–2018 and 2018–2019 seasons.</p>
<p>Supplemental Figure S2. Phenotypic correlations of predicted biomass and biomass‐related traits with yield and fruit traits during (a) the total season 2017–2018, (b) total season 2018–2019, (c) total season 2020–2021, and (d) total season 2021–2022.</p>
<p>Supplemental Figure S3. Growth related curves representing (a) the logistic curve of the advanced selections and (b) the absolute growth rate using weekly vegetative biomass data from the 2021 to 2022 season, showing variability among genotypes for these traits.</p>
<p>Supplemental Figure S4. Genotype by year interaction for (a) total season accumulated predicted biomass, (b) total season predicted biomass, (c) total season average fruit weight, and (d) total season marketable yield during the 2020–2021, and 2021–2022 seasons.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11958871/bin/TPG2-18-e70018-s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70018-s002.pdf</a><sup> (725.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270018-supitem-0002"><div class="caption p">
<p>Supplemental Table S1. Phenotypic data from spatial analyses through ASRtriala for predicted vegetative biomass, biomass‐related traits, and yield traits in the University of Florida strawberry breeding population during the 2017–2018, 2018–2019, 2020–2021, and 2021–2022 seasons.</p>
<p>Supplemental Table S2. Spatial analyses models generated by ASRtriala.</p>
<p>Supplemental Table S3. Model selection from spatial analyses through ASRtriala for predicted vegetative biomass, biomass‐related traits, and yield traits in the University of Florida strawberry breeding population during the 2017–2018, 2018–2019, 2020–2021, and 2021–2022 seasons.</p>
<p>Supplemental Table S4. Descriptive statistics for predicted vegetative biomass and biomass‐related traits in the University of Florida strawberry breeding population from the early and total seasons during 2017–2018, 2018–2019, and 2020–2021.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11958871/bin/TPG2-18-e70018-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70018-s001.xlsx</a><sup> (317.7KB, xlsx) </sup>
</div></div></section></section><section id="tpg270018-sec-0230" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We thank the staff of the UF/IFAS Gulf Coast Research and Education Center Strawberry Breeding and Genetics laboratory and former interns, Juan David Jimenez Pardo and Mariel Andrea Gallardo Salazar for their work in data collection and trial maintenance. We also thank the UF/IFAS Geomatics and Plant Phenomics laboratories for their work in high‐resolution image data collection and processing. This research was partially supported by grants awarded to Vance M. Whitaker from the Florida Strawberry Research and Education Foundation (FSREF) and support from the Florida Agricultural Experiment Station (FAES).</p></section><section id="notes1"><p>


Dalid, C.
, 
Zheng, C.
, 
Osorio, L.
, 
Verma, S.
, 
Abd‐Elrahman, A.
, 
Wang, X.
, &amp; 
Whitaker, V. M.
 (2025). Genetic analysis of predicted vegetative biomass and biomass‐related traits from digital phenotyping of strawberry. The Plant Genome, 18, e70018. 10.1002/tpg2.70018


</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270018-note-0001"><p>Assigned to Associate Editor Qingyi Yu.</p></div></div></section></section><section id="tpg270018-sec-0260"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All phenotypic data and spatial analysis models are included in the <a href="#tpg270018-supinfo-0001" class="usa-link">Supporting Information tables</a>.</p></section><section id="tpg270018-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270018-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270018-bib-0052">
<cite>

Abd‐Elrahman, A.
, 
Guan, Z.
, 
Dalid, C.
, 
Whitaker, V.
, 
Britt, K.
, 
Wilkinson, B.
, &amp; 
Gonzalez, A.
 (2020). Automated canopy delineation and size metrics extraction for strawberry dry weight modeling using raster analysis of high‐resolution imagery. Remote Sensing 2020, 12(21), 3632. 10.3390/RS12213632
</cite> [<a href="https://doi.org/10.3390/RS12213632" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Remote%20Sensing%202020&amp;title=Automated%20canopy%20delineation%20and%20size%20metrics%20extraction%20for%20strawberry%20dry%20weight%20modeling%20using%20raster%20analysis%20of%20high%E2%80%90resolution%20imagery&amp;author=A.%20Abd%E2%80%90Elrahman&amp;author=Z.%20Guan&amp;author=C.%20Dalid&amp;author=V.%20Whitaker&amp;author=K.%20Britt&amp;volume=12&amp;issue=21&amp;publication_year=2020&amp;pages=3632&amp;doi=10.3390/RS12213632&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0001">
<cite>

Agehara, S.
 (2021). Characterizing early‐season nitrogen fertilization rate effects on growth, yield, and quality of strawberry. Agronomy, 11(5), 905. 10.3390/AGRONOMY11050905
</cite> [<a href="https://doi.org/10.3390/AGRONOMY11050905" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=Characterizing%20early%E2%80%90season%20nitrogen%20fertilization%20rate%20effects%20on%20growth,%20yield,%20and%20quality%20of%20strawberry&amp;author=S.%20Agehara&amp;volume=11&amp;issue=5&amp;publication_year=2021&amp;pages=905&amp;doi=10.3390/AGRONOMY11050905&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0002">
<cite>

Bassil, N. V.
, 
Davis, T. M.
, 
Zhang, H.
, 
Ficklin, S.
, 
Mittmann, M.
, 
Webster, T.
, 
Mahoney, L.
, 
Wood, D.
, 
Alperin, E. S.
, 
Rosyara, U. R.
, 
Koehorst‐vanc Putten, H.
, 
Monfort, A.
, 
Sargent, D. J.
, 
Amaya, I.
, 
Denoyes, B.
, 
Bianco, L.
, 
van Dijk, T.
, 
Pirani, A.
, 
Iezzoni, A.
, … 
van de Weg, E.
 (2015). Development and preliminary evaluation of a 90 K Axiom® SNP array for the allo‐octoploid cultivated strawberry Fragaria × ananassa. BMC Genomics, 16(1), Article 155. 10.1186/S12864-015-1310-1

</cite> [<a href="https://doi.org/10.1186/S12864-015-1310-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4374422/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25886969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Development%20and%20preliminary%20evaluation%20of%20a%2090%20K%20Axiom%C2%AE%20SNP%20array%20for%20the%20allo%E2%80%90octoploid%20cultivated%20strawberry%20Fragaria%20%C3%97%20ananassa&amp;author=N.%20V.%20Bassil&amp;author=T.%20M.%20Davis&amp;author=H.%20Zhang&amp;author=S.%20Ficklin&amp;author=M.%20Mittmann&amp;volume=16&amp;issue=1&amp;publication_year=2015&amp;pages=155&amp;pmid=25886969&amp;doi=10.1186/S12864-015-1310-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0003">
<cite>

Brandán, E. Z.
, 
Fernández, R. R.
, 
Villagra, E. L.
, 
Salazar, S. M.
, 
Cruz, C.
, &amp; 
Ponce De León, R.
 (2009). Effects of latitude in morphological and physiologic parameters and plant harvest index in fresh plants of strawberry cv. “Camarosa.”
Acta Horticulturae, 842, 711–714. 10.17660/ActaHortic.2009.842.154
</cite> [<a href="https://doi.org/10.17660/ActaHortic.2009.842.154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Horticulturae&amp;title=Effects%20of%20latitude%20in%20morphological%20and%20physiologic%20parameters%20and%20plant%20harvest%20index%20in%20fresh%20plants%20of%20strawberry%20cv.%20%E2%80%9CCamarosa.%E2%80%9D&amp;author=E.%20Z.%20Brand%C3%A1n&amp;author=R.%20R.%20Fern%C3%A1ndez&amp;author=E.%20L.%20Villagra&amp;author=S.%20M.%20Salazar&amp;author=C.%20Cruz&amp;volume=842&amp;publication_year=2009&amp;pages=711-714&amp;doi=10.17660/ActaHortic.2009.842.154&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0004">
<cite>

Burgess, A. J.
, 
Masclaux‐Daubresse, C.
, 
Strittmatter, G.
, 
Weber, A. P. M.
, 
Taylor, S. H.
, 
Harbinson, J.
, 
Yin, X.
, 
Long, S.
, 
Paul, M. J.
, 
Westhoff, P.
, 
Loreto, F.
, 
Ceriotti, A.
, 
Saltenis, V. L. R.
, 
Pribil, M.
, 
Nacry, P.
, 
Scharff, L. B.
, 
Jensen, P. E.
, 
Muller, B.
, 
Cohan, J. P.
, … 
Baekelandt, A.
 (2023). Improving crop yield potential: Underlying biological processes and future prospects. Food and Energy Security, 12(1), e435. 10.1002/FES3.435

</cite> [<a href="https://doi.org/10.1002/FES3.435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10078444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37035025/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20and%20Energy%20Security&amp;title=Improving%20crop%20yield%20potential:%20Underlying%20biological%20processes%20and%20future%20prospects&amp;author=A.%20J.%20Burgess&amp;author=C.%20Masclaux%E2%80%90Daubresse&amp;author=G.%20Strittmatter&amp;author=A.%20P.%20M.%20Weber&amp;author=S.%20H.%20Taylor&amp;volume=12&amp;issue=1&amp;publication_year=2023&amp;pages=e435&amp;pmid=37035025&amp;doi=10.1002/FES3.435&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0005">
<cite>

Clark, L. V.
, 
Dwiyanti, M. S.
, 
Anzoua, K. G.
, 
Brummer, J. E.
, 
Ghimire, B. K.
, 
Głowacka, K.
, 
Hall, M.
, 
Heo, K.
, 
Jin, X.
, 
Lipka, A. E.
, 
Peng, J.
, 
Yamada, T.
, 
Yoo, J. H.
, 
Yu, C. Y.
, 
Zhao, H.
, 
Long, S. P.
, &amp; 
Sacks, E. J.
 (2019). Genome‐wide association and genomic prediction for biomass yield in a genetically diverse <em>Miscanthus sinensis</em> germplasm panel phenotyped at five locations in Asia and North America. GCB Bioenergy, 11(8), 988–1007. 10.1111/GCBB.12620
</cite> [<a href="https://doi.org/10.1111/GCBB.12620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GCB%20Bioenergy&amp;title=Genome%E2%80%90wide%20association%20and%20genomic%20prediction%20for%20biomass%20yield%20in%20a%20genetically%20diverse%20Miscanthus%20sinensis%20germplasm%20panel%20phenotyped%20at%20five%20locations%20in%20Asia%20and%20North%20America&amp;author=L.%20V.%20Clark&amp;author=M.%20S.%20Dwiyanti&amp;author=K.%20G.%20Anzoua&amp;author=J.%20E.%20Brummer&amp;author=B.%20K.%20Ghimire&amp;volume=11&amp;issue=8&amp;publication_year=2019&amp;pages=988-1007&amp;doi=10.1111/GCBB.12620&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0006">
<cite>

Crain, J.
, 
Mondal, S.
, 
Rutkoski, J.
, 
Singh, R. P.
, &amp; 
Poland, J.
 (2018). Combining high‐throughput phenotyping and genomic information to increase prediction and selection accuracy in wheat breeding. The Plant Genome, 11(1), 170043. 10.3835/PLANTGENOME2017.05.0043
</cite> [<a href="https://doi.org/10.3835/PLANTGENOME2017.05.0043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29505641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Combining%20high%E2%80%90throughput%20phenotyping%20and%20genomic%20information%20to%20increase%20prediction%20and%20selection%20accuracy%20in%20wheat%20breeding&amp;author=J.%20Crain&amp;author=S.%20Mondal&amp;author=J.%20Rutkoski&amp;author=R.%20P.%20Singh&amp;author=J.%20Poland&amp;volume=11&amp;issue=1&amp;publication_year=2018&amp;pages=170043&amp;pmid=29505641&amp;doi=10.3835/PLANTGENOME2017.05.0043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0007">
<cite>

Crossa, J.
, 
De Los Campos, G.
, 
Pérez, P.
, 
Gianola, D.
, 
Burgueño, J.
, 
Araus, J. L.
, 
Makumbi, D.
, 
Singh, R. P.
, 
Dreisigacker, S.
, 
Yan, J.
, 
Arief, V.
, 
Banziger, M.
, &amp; 
Braun, H. J.
 (2010). Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers. Genetics, 186(2), 713–724. 10.1534/genetics.110.118521

</cite> [<a href="https://doi.org/10.1534/genetics.110.118521" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2954475/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20813882/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Prediction%20of%20genetic%20values%20of%20quantitative%20traits%20in%20plant%20breeding%20using%20pedigree%20and%20molecular%20markers&amp;author=J.%20Crossa&amp;author=G.%20De%20Los%20Campos&amp;author=P.%20P%C3%A9rez&amp;author=D.%20Gianola&amp;author=J.%20Burgue%C3%B1o&amp;volume=186&amp;issue=2&amp;publication_year=2010&amp;pages=713-724&amp;pmid=20813882&amp;doi=10.1534/genetics.110.118521&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0008">
<cite>

de Bem Oliveira, I.
, 
Resende, M. F. R.
, 
Ferrão, L. F. V.
, 
Amadeu, R. R.
, 
Endelman, J. B.
, 
Kirst, M.
, 
Coelho, A. S. G.
, &amp; 
Munoz, P. R.
 (2019). Genomic prediction of autotetraploids; influence of relationship matrices, allele dosage, and continuous genotyping calls in phenotype prediction. G3: Genes, Genomes, Genetics, 9(4), 1189–1198. 10.1534/g3.119.400059

</cite> [<a href="https://doi.org/10.1534/g3.119.400059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6469427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30782769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=Genomic%20prediction%20of%20autotetraploids;%20influence%20of%20relationship%20matrices,%20allele%20dosage,%20and%20continuous%20genotyping%20calls%20in%20phenotype%20prediction&amp;author=I.%20de%20Bem%20Oliveira&amp;author=M.%20F.%20R.%20Resende&amp;author=L.%20F.%20V.%20Ferr%C3%A3o&amp;author=R.%20R.%20Amadeu&amp;author=J.%20B.%20Endelman&amp;volume=9&amp;issue=4&amp;publication_year=2019&amp;pages=1189-1198&amp;pmid=30782769&amp;doi=10.1534/g3.119.400059&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0009">
<cite>

Fan, Z.
, 
Verma, S.
, 
Lee, H.
, 
Jang, Y. J.
, 
Wang, Y.
, 
Lee, S.
, &amp; 
Whitaker, V. M.
 (2024). Strawberry soluble solids QTL with inverse effects on yield. Horticulture Research, 11(2), uhad271. 10.1093/HR/UHAD271

</cite> [<a href="https://doi.org/10.1093/HR/UHAD271" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10873791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38371635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=Strawberry%20soluble%20solids%20QTL%20with%20inverse%20effects%20on%20yield&amp;author=Z.%20Fan&amp;author=S.%20Verma&amp;author=H.%20Lee&amp;author=Y.%20J.%20Jang&amp;author=Y.%20Wang&amp;volume=11&amp;issue=2&amp;publication_year=2024&amp;pages=uhad271&amp;pmid=38371635&amp;doi=10.1093/HR/UHAD271&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0010">
<cite>

Fan, Z.
, &amp; 
Whitaker, V. M.
 (2024). Genomic signatures of strawberry domestication and diversification. The Plant Cell, 36(5), 1622–1636. 10.1093/PLCELL/KOAD314

</cite> [<a href="https://doi.org/10.1093/PLCELL/KOAD314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11062436/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38113879/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Cell&amp;title=Genomic%20signatures%20of%20strawberry%20domestication%20and%20diversification&amp;author=Z.%20Fan&amp;author=V.%20M.%20Whitaker&amp;volume=36&amp;issue=5&amp;publication_year=2024&amp;pages=1622-1636&amp;pmid=38113879&amp;doi=10.1093/PLCELL/KOAD314&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0011">
<cite>
FAO‐UN
. (2022). <em>Data</em>. FAO. <a href="https://www.fao.org/faostat/en/#data" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fao.org/faostat/en/#data</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?FAO%E2%80%90UN%20.%20(2022).%20Data.%20FAO.%20https://www.fao.org/faostat/en/#data" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0012">
<cite>

Gezan, S. A.
, 
Galli, G.
, &amp; 
Murray, D.
 (2022). 
User's manual for ASRtriala v. 1.0.0: An R package with complementary functions for single and multi‐environment trial analyses
. VSN International Ltd. <a href="https://vsni.co.uk/free-software/asrtriala" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://vsni.co.uk/free‐software/asrtriala</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Gezan,%20S.%20A.%20,%20Galli,%20G.%20,%20&amp;%20Murray,%20D.%20(2022).%20User's%20manual%20for%20ASRtriala%20v.%201.0.0:%20An%20R%20package%20with%20complementary%20functions%20for%20single%20and%20multi%E2%80%90environment%20trial%20analyses%20.%20VSN%20International%20Ltd.%20https://vsni.co.uk/free%E2%80%90software/asrtriala" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0013">
<cite>

Gezan, S. A.
, 
Osorio, L. F.
, 
Verma, S.
, &amp; 
Whitaker, V. M.
 (2017). An experimental validation of genomic selection in octoploid strawberry. Horticulture Research, 4, 16070. 10.1038/hortres.2016.70

</cite> [<a href="https://doi.org/10.1038/hortres.2016.70" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5225750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28090334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=An%20experimental%20validation%20of%20genomic%20selection%20in%20octoploid%20strawberry&amp;author=S.%20A.%20Gezan&amp;author=L.%20F.%20Osorio&amp;author=S.%20Verma&amp;author=V.%20M.%20Whitaker&amp;volume=4&amp;publication_year=2017&amp;pages=16070&amp;pmid=28090334&amp;doi=10.1038/hortres.2016.70&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0014">
<cite>

Gogel, B.
, 
Smith, A.
, &amp; 
Cullis, B.
 (2018). Comparison of a one‐ and two‐stage mixed model analysis of Australia's National Variety Trial Southern Region wheat data. Euphytica, 214(2), Article 44. 10.1007/S10681-018-2116-4
</cite> [<a href="https://doi.org/10.1007/S10681-018-2116-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Comparison%20of%20a%20one%E2%80%90%20and%20two%E2%80%90stage%20mixed%20model%20analysis%20of%20Australia's%20National%20Variety%20Trial%20Southern%20Region%20wheat%20data&amp;author=B.%20Gogel&amp;author=A.%20Smith&amp;author=B.%20Cullis&amp;volume=214&amp;issue=2&amp;publication_year=2018&amp;pages=44&amp;doi=10.1007/S10681-018-2116-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0015">
<cite>

Guan, Z.
, 
Abd‐Elrahman, A.
, 
Fan, Z.
, 
Whitaker, V. M.
, &amp; 
Wilkinson, B.
 (2020). Modeling strawberry biomass and leaf area using object‐based analysis of high‐resolution images. ISPRS Journal of Photogrammetry and Remote Sensing, 163, 171–186. 10.1016/j.isprsjprs.2020.02.021
</cite> [<a href="https://doi.org/10.1016/j.isprsjprs.2020.02.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISPRS%20Journal%20of%20Photogrammetry%20and%20Remote%20Sensing&amp;title=Modeling%20strawberry%20biomass%20and%20leaf%20area%20using%20object%E2%80%90based%20analysis%20of%20high%E2%80%90resolution%20images&amp;author=Z.%20Guan&amp;author=A.%20Abd%E2%80%90Elrahman&amp;author=Z.%20Fan&amp;author=V.%20M.%20Whitaker&amp;author=B.%20Wilkinson&amp;volume=163&amp;publication_year=2020&amp;pages=171-186&amp;doi=10.1016/j.isprsjprs.2020.02.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0016">
<cite>

Guan, Z.
, 
Wu, F.
, &amp; 
Whidden, A. J.
 (2018). Florida strawberry production costs and trends. EDIS, 2018(1), 5. 10.32473/EDIS-FE1013-2017
</cite> [<a href="https://doi.org/10.32473/EDIS-FE1013-2017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EDIS&amp;title=Florida%20strawberry%20production%20costs%20and%20trends&amp;author=Z.%20Guan&amp;author=F.%20Wu&amp;author=A.%20J.%20Whidden&amp;volume=2018&amp;issue=1&amp;publication_year=2018&amp;pages=5&amp;doi=10.32473/EDIS-FE1013-2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0017">
<cite>

Hardigan, M. A.
, 
Feldmann, M. J.
, 
Lorant, A.
, 
Bird, K. A.
, 
Famula, R.
, 
Acharya, C.
, 
Cole, G.
, 
Edger, P. P.
, &amp; 
Knapp, S. J.
 (2020). Genome synteny has been conserved among the octoploid progenitors of cultivated strawberry over millions of years of evolution. Frontiers in Plant Science, 10, Article 1789. 10.3389/fpls.2019.01789

</cite> [<a href="https://doi.org/10.3389/fpls.2019.01789" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7020885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32158449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%20synteny%20has%20been%20conserved%20among%20the%20octoploid%20progenitors%20of%20cultivated%20strawberry%20over%20millions%20of%20years%20of%20evolution&amp;author=M.%20A.%20Hardigan&amp;author=M.%20J.%20Feldmann&amp;author=A.%20Lorant&amp;author=K.%20A.%20Bird&amp;author=R.%20Famula&amp;volume=10&amp;publication_year=2020&amp;pages=1789&amp;pmid=32158449&amp;doi=10.3389/fpls.2019.01789&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0018">
<cite>

Ibrahim, A. K.
, 
Zhang, L.
, 
Niyitanga, S.
, 
Afzal, M. Z.
, 
Xu, Y.
, 
Zhang, L.
, 
Zhang, L.
, &amp; 
Qi, J.
 (2020). Principles and approaches of association mapping in plant breeding. Tropical Plant Biology, 13(3), 212–224. 10.1007/S12042-020-09261-4
</cite> [<a href="https://doi.org/10.1007/S12042-020-09261-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Tropical%20Plant%20Biology&amp;title=Principles%20and%20approaches%20of%20association%20mapping%20in%20plant%20breeding&amp;author=A.%20K.%20Ibrahim&amp;author=L.%20Zhang&amp;author=S.%20Niyitanga&amp;author=M.%20Z.%20Afzal&amp;author=Y.%20Xu&amp;volume=13&amp;issue=3&amp;publication_year=2020&amp;pages=212-224&amp;doi=10.1007/S12042-020-09261-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0019">
<cite>

Kawano, T.
, 
Wallbridge, N.
, &amp; 
Plummer, C.
 (2020). Logistic models for simulating the growth of plants by defining the maximum plant size as the limit of information flow. Plant Signaling &amp; Behavior, 15(2), 1709718. 10.1080/15592324.2019.1709718

</cite> [<a href="https://doi.org/10.1080/15592324.2019.1709718" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7053935/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31984861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signaling%20&amp;%20Behavior&amp;title=Logistic%20models%20for%20simulating%20the%20growth%20of%20plants%20by%20defining%20the%20maximum%20plant%20size%20as%20the%20limit%20of%20information%20flow&amp;author=T.%20Kawano&amp;author=N.%20Wallbridge&amp;author=C.%20Plummer&amp;volume=15&amp;issue=2&amp;publication_year=2020&amp;pages=1709718&amp;pmid=31984861&amp;doi=10.1080/15592324.2019.1709718&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0020">
<cite>

Krause, M. R.
, 
Mondal, S.
, 
Crossa, J.
, 
Singh, R. P.
, 
Pinto, F.
, 
Haghighattalab, A.
, 
Shrestha, S.
, 
Rutkoski, J.
, 
Gore, M. A.
, 
Sorrells, M. E.
, &amp; 
Poland, J.
 (2020). Aerial high‐throughput phenotyping enables indirect selection for grain yield at the early generation, seed‐limited stages in breeding programs. Crop Science, 60(6), 3096–3114. 10.1002/CSC2.20259
</cite> [<a href="https://doi.org/10.1002/CSC2.20259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Aerial%20high%E2%80%90throughput%20phenotyping%20enables%20indirect%20selection%20for%20grain%20yield%20at%20the%20early%20generation,%20seed%E2%80%90limited%20stages%20in%20breeding%20programs&amp;author=M.%20R.%20Krause&amp;author=S.%20Mondal&amp;author=J.%20Crossa&amp;author=R.%20P.%20Singh&amp;author=F.%20Pinto&amp;volume=60&amp;issue=6&amp;publication_year=2020&amp;pages=3096-3114&amp;doi=10.1002/CSC2.20259&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0021">
<cite>

Merrick, L. F.
, &amp; 
Carter, A. H.
 (2021). Comparison of genomic selection models for exploring predictive ability of complex traits in breeding programs. The Plant Genome, 14(3), e20158. 10.1002/tpg2.20158

</cite> [<a href="https://doi.org/10.1002/tpg2.20158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34719886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Comparison%20of%20genomic%20selection%20models%20for%20exploring%20predictive%20ability%20of%20complex%20traits%20in%20breeding%20programs&amp;author=L.%20F.%20Merrick&amp;author=A.%20H.%20Carter&amp;volume=14&amp;issue=3&amp;publication_year=2021&amp;pages=e20158&amp;pmid=34719886&amp;doi=10.1002/tpg2.20158&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0022">
<cite>

Meuwissen, T. H. E.
, 
Hayes, B. J.
, &amp; 
Goddard, M. E.
 (2001). Prediction of total genetic value using genome‐wide dense marker maps. Genetics, 157(4), 1819–1829. 10.1093/GENETICS/157.4.1819

</cite> [<a href="https://doi.org/10.1093/GENETICS/157.4.1819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11290733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Prediction%20of%20total%20genetic%20value%20using%20genome%E2%80%90wide%20dense%20marker%20maps&amp;author=T.%20H.%20E.%20Meuwissen&amp;author=B.%20J.%20Hayes&amp;author=M.%20E.%20Goddard&amp;volume=157&amp;issue=4&amp;publication_year=2001&amp;pages=1819-1829&amp;pmid=11290733&amp;doi=10.1093/GENETICS/157.4.1819&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0023">
<cite>

Montesinos‐López, O. A.
, 
Montesinos‐López, A.
, 
Luna‐Vázquez, F. J.
, 
Toledo, F. H.
, 
Pérez‐Rodríguez, P.
, 
Lillemo, M.
, &amp; 
Crossa, J.
 (2019). An R package for Bayesian analysis of multi‐environment and multi‐trait multi‐environment data for genome‐based prediction. G3: Genes, Genomes, Genetics, 9(5), 1355–1369. 10.1534/G3.119.400126

</cite> [<a href="https://doi.org/10.1534/G3.119.400126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6505148/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30819822/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=An%20R%20package%20for%20Bayesian%20analysis%20of%20multi%E2%80%90environment%20and%20multi%E2%80%90trait%20multi%E2%80%90environment%20data%20for%20genome%E2%80%90based%20prediction&amp;author=O.%20A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=F.%20J.%20Luna%E2%80%90V%C3%A1zquez&amp;author=F.%20H.%20Toledo&amp;author=P.%20P%C3%A9rez%E2%80%90Rodr%C3%ADguez&amp;volume=9&amp;issue=5&amp;publication_year=2019&amp;pages=1355-1369&amp;pmid=30819822&amp;doi=10.1534/G3.119.400126&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0024">
<cite>

O'Connor, K. M.
, 
Hayes, B. J.
, 
Hardner, C. M.
, 
Alam, M.
, 
Henry, R. J.
, &amp; 
Topp, B. L.
 (2021). Genomic selection and genetic gain for nut yield in an Australian macadamia breeding population. BMC Genomics, 22(1), Article 370. 10.1186/S12864-021-07694-Z

</cite> [<a href="https://doi.org/10.1186/S12864-021-07694-Z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8139092/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34016055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Genomic%20selection%20and%20genetic%20gain%20for%20nut%20yield%20in%20an%20Australian%20macadamia%20breeding%20population&amp;author=K.%20M.%20O'Connor&amp;author=B.%20J.%20Hayes&amp;author=C.%20M.%20Hardner&amp;author=M.%20Alam&amp;author=R.%20J.%20Henry&amp;volume=22&amp;issue=1&amp;publication_year=2021&amp;pages=370&amp;pmid=34016055&amp;doi=10.1186/S12864-021-07694-Z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0025">
<cite>

Osorio, L. F.
, 
Gezan, S. A.
, 
Verma, S.
, &amp; 
Whitaker, V. M.
 (2021). Independent validation of genomic prediction in strawberry over multiple cycles. Frontiers in Genetics, 11, Article 596258. 10.3389/FGENE.2020.596258

</cite> [<a href="https://doi.org/10.3389/FGENE.2020.596258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7862747/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33552121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Independent%20validation%20of%20genomic%20prediction%20in%20strawberry%20over%20multiple%20cycles&amp;author=L.%20F.%20Osorio&amp;author=S.%20A.%20Gezan&amp;author=S.%20Verma&amp;author=V.%20M.%20Whitaker&amp;volume=11&amp;publication_year=2021&amp;pages=596258&amp;pmid=33552121&amp;doi=10.3389/FGENE.2020.596258&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0026">
<cite>

Peixoto, M. A.
, 
Leach, K. A.
, 
Jarquin, D.
, 
Flannery, P.
, 
Zystro, J.
, 
Tracy, W. F.
, 
Bhering, L.
, &amp; 
Resende, M. F. R.
 (2024). Utilizing genomic prediction to boost hybrid performance in a sweet corn breeding program. Frontiers in Plant Science, 15, 1293307. 10.3389/FPLS.2024.1293307

</cite> [<a href="https://doi.org/10.3389/FPLS.2024.1293307" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11080654/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38726298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Utilizing%20genomic%20prediction%20to%20boost%20hybrid%20performance%20in%20a%20sweet%20corn%20breeding%20program&amp;author=M.%20A.%20Peixoto&amp;author=K.%20A.%20Leach&amp;author=D.%20Jarquin&amp;author=P.%20Flannery&amp;author=J.%20Zystro&amp;volume=15&amp;publication_year=2024&amp;pages=1293307&amp;pmid=38726298&amp;doi=10.3389/FPLS.2024.1293307&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0027">
<cite>

Pérez, P.
, &amp; 
De Los Campos, G.
 (2014). Genome‐wide regression and prediction with the BGLR statistical package. Genetics, 198(2), 483–495. 10.1534/GENETICS.114.164442

</cite> [<a href="https://doi.org/10.1534/GENETICS.114.164442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4196607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25009151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genome%E2%80%90wide%20regression%20and%20prediction%20with%20the%20BGLR%20statistical%20package&amp;author=P.%20P%C3%A9rez&amp;author=G.%20De%20Los%20Campos&amp;volume=198&amp;issue=2&amp;publication_year=2014&amp;pages=483-495&amp;pmid=25009151&amp;doi=10.1534/GENETICS.114.164442&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0028">
<cite>

Pérez‐Ruiz, M.
, 
Prior, A.
, 
Martinez‐Guanter, J.
, 
Apolo‐Apolo, O. E.
, 
Andrade‐Sanchez, P.
, &amp; 
Egea, G.
 (2020). Development and evaluation of a self‐propelled electric platform for high‐throughput field phenotyping in wheat breeding trials. Computers and Electronics in Agriculture, 169, 105237. 10.1016/J.COMPAG.2020.105237
</cite> [<a href="https://doi.org/10.1016/J.COMPAG.2020.105237" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Computers%20and%20Electronics%20in%20Agriculture&amp;title=Development%20and%20evaluation%20of%20a%20self%E2%80%90propelled%20electric%20platform%20for%20high%E2%80%90throughput%20field%20phenotyping%20in%20wheat%20breeding%20trials&amp;author=M.%20P%C3%A9rez%E2%80%90Ruiz&amp;author=A.%20Prior&amp;author=J.%20Martinez%E2%80%90Guanter&amp;author=O.%20E.%20Apolo%E2%80%90Apolo&amp;author=P.%20Andrade%E2%80%90Sanchez&amp;volume=169&amp;publication_year=2020&amp;pages=105237&amp;doi=10.1016/J.COMPAG.2020.105237&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0029">
<cite>

Pincot, D. D. A.
, 
Hardigan, M. A.
, 
Cole, G. S.
, 
Famula, R. A.
, 
Henry, P. M.
, 
Gordon, T. R.
, &amp; 
Knapp, S. J.
 (2020). Accuracy of genomic selection and long‐term genetic gain for resistance to Verticillium wilt in strawberry. The Plant Genome, 13(3), e20054. 10.1002/tpg2.20054

</cite> [<a href="https://doi.org/10.1002/tpg2.20054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33217217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Accuracy%20of%20genomic%20selection%20and%20long%E2%80%90term%20genetic%20gain%20for%20resistance%20to%20Verticillium%20wilt%20in%20strawberry&amp;author=D.%20D.%20A.%20Pincot&amp;author=M.%20A.%20Hardigan&amp;author=G.%20S.%20Cole&amp;author=R.%20A.%20Famula&amp;author=P.%20M.%20Henry&amp;volume=13&amp;issue=3&amp;publication_year=2020&amp;pages=e20054&amp;pmid=33217217&amp;doi=10.1002/tpg2.20054&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0030">
<cite>

Poorter, H.
, &amp; 
Nagel, O.
 (2000). The role of biomass allocation in the growth response of plants to different levels of light, CO<sub>2</sub>, nutrients and water: A quantitative review. Functional Plant Biology, 27(12), 1191. 10.1071/PP99173_CO
</cite> [<a href="https://doi.org/10.1071/PP99173_CO" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20Plant%20Biology&amp;title=The%20role%20of%20biomass%20allocation%20in%20the%20growth%20response%20of%20plants%20to%20different%20levels%20of%20light,%20CO2,%20nutrients%20and%20water:%20A%20quantitative%20review&amp;author=H.%20Poorter&amp;author=O.%20Nagel&amp;volume=27&amp;issue=12&amp;publication_year=2000&amp;pages=1191&amp;doi=10.1071/PP99173_CO&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0031">
<cite>

Poorter, H.
, 
Niinemets, Ü.
, 
Poorter, L.
, 
Wright, I. J.
, &amp; 
Villar, R.
 (2009). Causes and consequences of variation in leaf mass per area (LMA): A meta‐analysis. New Phytologist, 182(3), 565–588. 10.1111/J.1469-8137.2009.02830.X

</cite> [<a href="https://doi.org/10.1111/J.1469-8137.2009.02830.X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19434804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Causes%20and%20consequences%20of%20variation%20in%20leaf%20mass%20per%20area%20(LMA):%20A%20meta%E2%80%90analysis&amp;author=H.%20Poorter&amp;author=%C3%9C.%20Niinemets&amp;author=L.%20Poorter&amp;author=I.%20J.%20Wright&amp;author=R.%20Villar&amp;volume=182&amp;issue=3&amp;publication_year=2009&amp;pages=565-588&amp;pmid=19434804&amp;doi=10.1111/J.1469-8137.2009.02830.X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0032">
<cite>

Porker, K.
, 
Straight, M.
, &amp; 
Hunt, J. R.
 (2020). Evaluation of G × E × M interactions to increase harvest index and yield of early sown wheat. Frontiers in Plant Science, 11, Article 994. 10.3389/FPLS.2020.00994

</cite> [<a href="https://doi.org/10.3389/FPLS.2020.00994" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7366857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32754174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Evaluation%20of%20G%20%C3%97%20E%20%C3%97%20M%20interactions%20to%20increase%20harvest%20index%20and%20yield%20of%20early%20sown%20wheat&amp;author=K.%20Porker&amp;author=M.%20Straight&amp;author=J.%20R.%20Hunt&amp;volume=11&amp;publication_year=2020&amp;pages=994&amp;pmid=32754174&amp;doi=10.3389/FPLS.2020.00994&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0053">
<cite>
R Core Team
. (2021). R: A language and environment for statistical computing. R Foundation for Statistical Computing. <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R‐project.org/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=R:%20A%20language%20and%20environment%20for%20statistical%20computing&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0054">
<cite>
RStudio Team
. (2019). RStudio: Integrated development for R. RStudio. <a href="http://www.rstudio.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.rstudio.com/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=RStudio:%20Integrated%20development%20for%20R&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0033">
<cite>

Salinas, N.
, 
Verma, S.
, 
Peres, N.
, &amp; 
Whitaker, V. M.
 (2019). FaRCa1: A major subgenome‐specific locus conferring resistance to <em>Colletotrichum acutatum</em> in strawberry. Theoretical and Applied Genetics, 132(4), 1109–1120. 10.1007/s00122-018-3263-7

</cite> [<a href="https://doi.org/10.1007/s00122-018-3263-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6449309/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30564908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=FaRCa1:%20A%20major%20subgenome%E2%80%90specific%20locus%20conferring%20resistance%20to%20Colletotrichum%20acutatum%20in%20strawberry&amp;author=N.%20Salinas&amp;author=S.%20Verma&amp;author=N.%20Peres&amp;author=V.%20M.%20Whitaker&amp;volume=132&amp;issue=4&amp;publication_year=2019&amp;pages=1109-1120&amp;pmid=30564908&amp;doi=10.1007/s00122-018-3263-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0034">
<cite>

Smith, A.
, 
Cullis, B.
, &amp; 
Gilmour, A.
 (2001). Applications: The analysis of crop variety evaluation data in Australia. Australian &amp; New Zealand Journal of Statistics, 43(2), 129–145. 10.1111/1467-842X.00163
</cite> [<a href="https://doi.org/10.1111/1467-842X.00163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Australian%20&amp;%20New%20Zealand%20Journal%20of%20Statistics&amp;title=Applications:%20The%20analysis%20of%20crop%20variety%20evaluation%20data%20in%20Australia&amp;author=A.%20Smith&amp;author=B.%20Cullis&amp;author=A.%20Gilmour&amp;volume=43&amp;issue=2&amp;publication_year=2001&amp;pages=129-145&amp;doi=10.1111/1467-842X.00163&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0035">
<cite>

Smith, A.
, 
Cullis, B.
, &amp; 
Thompson, R.
 (2001). Analyzing variety by environment data using multiplicative mixed models and adjustments for spatial field trend. Biometrics, 57(4), 1138–1147. 10.1111/J.0006-341X.2001.01138.X

</cite> [<a href="https://doi.org/10.1111/J.0006-341X.2001.01138.X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11764254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Analyzing%20variety%20by%20environment%20data%20using%20multiplicative%20mixed%20models%20and%20adjustments%20for%20spatial%20field%20trend&amp;author=A.%20Smith&amp;author=B.%20Cullis&amp;author=R.%20Thompson&amp;volume=57&amp;issue=4&amp;publication_year=2001&amp;pages=1138-1147&amp;pmid=11764254&amp;doi=10.1111/J.0006-341X.2001.01138.X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0036">
<cite>

Tessema, B. B.
, 
Liu, H.
, 
Sørensen, A. C.
, 
Andersen, J. R.
, &amp; 
Jensen, J.
 (2020). Strategies using genomic selection to increase genetic gain in breeding programs for wheat. Frontiers in Genetics, 11, Article 578123. 10.3389/fgene.2020.578123

</cite> [<a href="https://doi.org/10.3389/fgene.2020.578123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7748061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33343626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Strategies%20using%20genomic%20selection%20to%20increase%20genetic%20gain%20in%20breeding%20programs%20for%20wheat&amp;author=B.%20B.%20Tessema&amp;author=H.%20Liu&amp;author=A.%20C.%20S%C3%B8rensen&amp;author=J.%20R.%20Andersen&amp;author=J.%20Jensen&amp;volume=11&amp;publication_year=2020&amp;pages=578123&amp;pmid=33343626&amp;doi=10.3389/fgene.2020.578123&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0037">
<cite>

Tilhou, N. W.
, &amp; 
Casler, M. D.
 (2022). Biomass yield improvement in switchgrass through genomic prediction of flowering time. GCB Bioenergy, 14(9), 1023–1034. 10.1111/GCBB.12983
</cite> [<a href="https://doi.org/10.1111/GCBB.12983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GCB%20Bioenergy&amp;title=Biomass%20yield%20improvement%20in%20switchgrass%20through%20genomic%20prediction%20of%20flowering%20time&amp;author=N.%20W.%20Tilhou&amp;author=M.%20D.%20Casler&amp;volume=14&amp;issue=9&amp;publication_year=2022&amp;pages=1023-1034&amp;doi=10.1111/GCBB.12983&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0038">
<cite>

Toda, Y.
, 
Kaga, A.
, 
Kajiya‐Kanegae, H.
, 
Hattori, T.
, 
Yamaoka, S.
, 
Okamoto, M.
, 
Tsujimoto, H.
, &amp; 
Iwata, H.
 (2021). Genomic prediction modeling of soybean biomass using UAV‐based remote sensing and longitudinal model parameters. The Plant Genome, 14(3), e20157. 10.1002/tpg2.20157

</cite> [<a href="https://doi.org/10.1002/tpg2.20157" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34595846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20prediction%20modeling%20of%20soybean%20biomass%20using%20UAV%E2%80%90based%20remote%20sensing%20and%20longitudinal%20model%20parameters&amp;author=Y.%20Toda&amp;author=A.%20Kaga&amp;author=H.%20Kajiya%E2%80%90Kanegae&amp;author=T.%20Hattori&amp;author=S.%20Yamaoka&amp;volume=14&amp;issue=3&amp;publication_year=2021&amp;pages=e20157&amp;pmid=34595846&amp;doi=10.1002/tpg2.20157&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0039"><cite>
USDA‐NASS
. (2023). 
Noncitrus fruits and nuts 2022 summary
. <a href="https://usda.library.cornell.edu/concern/publications/zs25x846c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://usda.library.cornell.edu/concern/publications/zs25x846c</a>
</cite></li>
<li id="tpg270018-bib-0040">
<cite>

Van Hooijdonk, B. M.
, 
Tustin, D. S.
, 
Dayatilake, G. A.
, 
Oliver, M. J.
, &amp; 
Richardson, A.
 (2018). Harvest index of apple and kiwifruit—How high can we go by optimising plant architecture and canopy management?
Acta Horticulturae, 1228, 307–313. 10.17660/ActaHortic.2018.1228.46
</cite> [<a href="https://doi.org/10.17660/ActaHortic.2018.1228.46" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Horticulturae&amp;title=Harvest%20index%20of%20apple%20and%20kiwifruit%E2%80%94How%20high%20can%20we%20go%20by%20optimising%20plant%20architecture%20and%20canopy%20management?&amp;author=B.%20M.%20Van%20Hooijdonk&amp;author=D.%20S.%20Tustin&amp;author=G.%20A.%20Dayatilake&amp;author=M.%20J.%20Oliver&amp;author=A.%20Richardson&amp;volume=1228&amp;publication_year=2018&amp;pages=307-313&amp;doi=10.17660/ActaHortic.2018.1228.46&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0041">
<cite>

Verhulst, P. F.
 (1838). Notice sur la loi que la population poursuit dans son accroissement. Correspondance mathématique et physique, 10, 113–121. <a href="https://www.sciepub.com/reference/6960" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sciepub.com/reference/6960</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Correspondance%20math%C3%A9matique%20et%20physique&amp;title=Notice%20sur%20la%20loi%20que%20la%20population%20poursuit%20dans%20son%20accroissement&amp;author=P.%20F.%20Verhulst&amp;volume=10&amp;publication_year=1838&amp;pages=113-121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0042">
<cite>

Verma, S.
, 
Bassil, N. V.
, 
Van De Weg, E.
, 
Harrison, R. J.
, 
Monfort, A.
, 
Hidalgo, J. M.
, 
Amaya, I.
, 
Denoyes, B.
, 
Mahoney, L. L.
, 
Davis, T. M.
, 
Fan, Z.
, 
Knapp, S.
, &amp; 
Whitaker, V. M.
 (2017). Development and evaluation of the Axiom® IStraw35 384HT array for the allo‐octoploid cultivated strawberry Fragaria ×ananassa. Acta Horticulturae, 1156, 75–81. 10.17660/ACTAHORTIC.2017.1156.10
</cite> [<a href="https://doi.org/10.17660/ACTAHORTIC.2017.1156.10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Horticulturae&amp;title=Development%20and%20evaluation%20of%20the%20Axiom%C2%AE%20IStraw35%20384HT%20array%20for%20the%20allo%E2%80%90octoploid%20cultivated%20strawberry%20Fragaria%20%C3%97ananassa&amp;author=S.%20Verma&amp;author=N.%20V.%20Bassil&amp;author=E.%20Van%20De%20Weg&amp;author=R.%20J.%20Harrison&amp;author=A.%20Monfort&amp;volume=1156&amp;publication_year=2017&amp;pages=75-81&amp;doi=10.17660/ACTAHORTIC.2017.1156.10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0043">
<cite>

Wang, J.
, &amp; 
Zhang, Z.
 (2021). GAPIT version 3: Boosting power and accuracy for genomic association and prediction. Genomics, Proteomics &amp; Bioinformatics, 19(4), 629–640. 10.1016/j.gpb.2021.08.005
</cite> [<a href="https://doi.org/10.1016/j.gpb.2021.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9121400/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34492338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics,%20Proteomics%20&amp;%20Bioinformatics&amp;title=GAPIT%20version%203:%20Boosting%20power%20and%20accuracy%20for%20genomic%20association%20and%20prediction&amp;author=J.%20Wang&amp;author=Z.%20Zhang&amp;volume=19&amp;issue=4&amp;publication_year=2021&amp;pages=629-640&amp;pmid=34492338&amp;doi=10.1016/j.gpb.2021.08.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0044">
<cite>

Whitaker, V. M.
, 
Boyd, N. S.
, 
Peres, N. A.
, 
Desaeger, J.
, 
Lahiri, S.
, &amp; 
Agehara, S.
 (2023). Strawberry production (HS736). University of Florida Institute of Food and Agricultural Sciences. 10.32473/edis-cv134-2023
</cite> [<a href="https://doi.org/10.32473/edis-cv134-2023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Strawberry%20production&amp;author=V.%20M.%20Whitaker&amp;author=N.%20S.%20Boyd&amp;author=N.%20A.%20Peres&amp;author=J.%20Desaeger&amp;author=S.%20Lahiri&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0045">
<cite>

Whitaker, V. M.
, &amp; 
Fan, Z.
 (2020). What will Florida strawberry varieties look like in the future? A breeder's perspective. International Journal of Fruit Science, 20(Suppl 1), 992–996. 10.1080/15538362.2020.1755770
</cite> [<a href="https://doi.org/10.1080/15538362.2020.1755770" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Fruit%20Science&amp;title=What%20will%20Florida%20strawberry%20varieties%20look%20like%20in%20the%20future?%20A%20breeder's%20perspective&amp;author=V.%20M.%20Whitaker&amp;author=Z.%20Fan&amp;volume=20&amp;issue=1&amp;publication_year=2020&amp;pages=992-996&amp;doi=10.1080/15538362.2020.1755770&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0046">
<cite>

Whitaker, V. M.
, 
Osorio, L. F.
, 
Hasing, T.
, &amp; 
Gezan, S.
 (2012). Estimation of genetic parameters for 12 fruit and vegetative traits in the University of Florida strawberry breeding population. Journal of the American Society for Horticultural Science, 137(5), 316–324. 10.21273/JASHS.137.5.316
</cite> [<a href="https://doi.org/10.21273/JASHS.137.5.316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Horticultural%20Science&amp;title=Estimation%20of%20genetic%20parameters%20for%2012%20fruit%20and%20vegetative%20traits%20in%20the%20University%20of%20Florida%20strawberry%20breeding%20population&amp;author=V.%20M.%20Whitaker&amp;author=L.%20F.%20Osorio&amp;author=T.%20Hasing&amp;author=S.%20Gezan&amp;volume=137&amp;issue=5&amp;publication_year=2012&amp;pages=316-324&amp;doi=10.21273/JASHS.137.5.316&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0047">
<cite>

Wu, F.
, 
Guan, Z.
, &amp; 
Whitaker, V.
 (2015). Optimizing yield distribution under biological and economic constraints: Florida strawberries as a model for perishable commodities. Agricultural Systems, 141, 113–120. 10.1016/J.AGSY.2015.10.002
</cite> [<a href="https://doi.org/10.1016/J.AGSY.2015.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agricultural%20Systems&amp;title=Optimizing%20yield%20distribution%20under%20biological%20and%20economic%20constraints:%20Florida%20strawberries%20as%20a%20model%20for%20perishable%20commodities&amp;author=F.%20Wu&amp;author=Z.%20Guan&amp;author=V.%20Whitaker&amp;volume=141&amp;publication_year=2015&amp;pages=113-120&amp;doi=10.1016/J.AGSY.2015.10.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0048">
<cite>

Zheng, C.
, 
Abd‐Elrahman, A.
, 
Whitaker, V.
, &amp; 
Dalid, C.
 (2022a). Prediction of strawberry dry biomass from UAV multispectral imagery using multiple machine learning methods. Remote Sensing, 14(18), 4511. 10.3390/rs14184511
</cite> [<a href="https://doi.org/10.3390/rs14184511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Remote%20Sensing&amp;title=Prediction%20of%20strawberry%20dry%20biomass%20from%20UAV%20multispectral%20imagery%20using%20multiple%20machine%20learning%20methods&amp;author=C.%20Zheng&amp;author=A.%20Abd%E2%80%90Elrahman&amp;author=V.%20Whitaker&amp;author=C.%20Dalid&amp;volume=14&amp;issue=18&amp;publication_year=2022a&amp;pages=4511&amp;doi=10.3390/rs14184511&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0049">
<cite>

Zheng, C.
, 
Abd‐Elrahman, A.
, 
Whitaker, V. M.
, &amp; 
Dalid, C.
 (2022b). Deep learning for strawberry canopy delineation and biomass prediction from high‐resolution images. Plant Phenomics, 2022, 9850486. 10.34133/2022/9850486

</cite> [<a href="https://doi.org/10.34133/2022/9850486" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9595049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36320455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Phenomics&amp;title=Deep%20learning%20for%20strawberry%20canopy%20delineation%20and%20biomass%20prediction%20from%20high%E2%80%90resolution%20images&amp;author=C.%20Zheng&amp;author=A.%20Abd%E2%80%90Elrahman&amp;author=V.%20M.%20Whitaker&amp;author=C.%20Dalid&amp;volume=2022&amp;publication_year=2022b&amp;pages=9850486&amp;pmid=36320455&amp;doi=10.34133/2022/9850486&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0050">
<cite>

Zheng, H.
, 
Cheng, T.
, 
Zhou, M.
, 
Li, D.
, 
Yao, X.
, 
Tian, Y.
, 
Cao, W.
, &amp; 
Zhu, Y.
 (2019). Improved estimation of rice aboveground biomass combining textural and spectral analysis of UAV imagery. Precision Agriculture, 20(3), 611–629. 10.1007/s11119-018-9600-7
</cite> [<a href="https://doi.org/10.1007/s11119-018-9600-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Precision%20Agriculture&amp;title=Improved%20estimation%20of%20rice%20aboveground%20biomass%20combining%20textural%20and%20spectral%20analysis%20of%20UAV%20imagery&amp;author=H.%20Zheng&amp;author=T.%20Cheng&amp;author=M.%20Zhou&amp;author=D.%20Li&amp;author=X.%20Yao&amp;volume=20&amp;issue=3&amp;publication_year=2019&amp;pages=611-629&amp;doi=10.1007/s11119-018-9600-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270018-bib-0051">
<cite>

Zhu, X. G.
, 
Long, S. P.
, &amp; 
Ort, D. R.
 (2010). Improving photosynthetic efficiency for greater yield. Annual Review of Plant Biology, 61, 235–261. 10.1146/ANNUREV-ARPLANT-042809-112206
</cite> [<a href="https://doi.org/10.1146/ANNUREV-ARPLANT-042809-112206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20192734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Plant%20Biology&amp;title=Improving%20photosynthetic%20efficiency%20for%20greater%20yield&amp;author=X.%20G.%20Zhu&amp;author=S.%20P.%20Long&amp;author=D.%20R.%20Ort&amp;volume=61&amp;publication_year=2010&amp;pages=235-261&amp;pmid=20192734&amp;doi=10.1146/ANNUREV-ARPLANT-042809-112206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>Supplemental Figure S1. Genotype by year interaction for (a) early season predicted biomass, (b) early season average fruit weight, (c) early season marketable yield, (d) total season predicted biomass, (e) total season average fruit weight, and (f) total season marketable yield during the 2017–2018 and 2018–2019 seasons.</p>
<p>Supplemental Figure S2. Phenotypic correlations of predicted biomass and biomass‐related traits with yield and fruit traits during (a) the total season 2017–2018, (b) total season 2018–2019, (c) total season 2020–2021, and (d) total season 2021–2022.</p>
<p>Supplemental Figure S3. Growth related curves representing (a) the logistic curve of the advanced selections and (b) the absolute growth rate using weekly vegetative biomass data from the 2021 to 2022 season, showing variability among genotypes for these traits.</p>
<p>Supplemental Figure S4. Genotype by year interaction for (a) total season accumulated predicted biomass, (b) total season predicted biomass, (c) total season average fruit weight, and (d) total season marketable yield during the 2020–2021, and 2021–2022 seasons.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11958871/bin/TPG2-18-e70018-s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70018-s002.pdf</a><sup> (725.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<p>Supplemental Table S1. Phenotypic data from spatial analyses through ASRtriala for predicted vegetative biomass, biomass‐related traits, and yield traits in the University of Florida strawberry breeding population during the 2017–2018, 2018–2019, 2020–2021, and 2021–2022 seasons.</p>
<p>Supplemental Table S2. Spatial analyses models generated by ASRtriala.</p>
<p>Supplemental Table S3. Model selection from spatial analyses through ASRtriala for predicted vegetative biomass, biomass‐related traits, and yield traits in the University of Florida strawberry breeding population during the 2017–2018, 2018–2019, 2020–2021, and 2021–2022 seasons.</p>
<p>Supplemental Table S4. Descriptive statistics for predicted vegetative biomass and biomass‐related traits in the University of Florida strawberry breeding population from the early and total seasons during 2017–2018, 2018–2019, and 2020–2021.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11958871/bin/TPG2-18-e70018-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70018-s001.xlsx</a><sup> (317.7KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All phenotypic data and spatial analysis models are included in the <a href="#tpg270018-supinfo-0001" class="usa-link">Supporting Information tables</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70018"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70018.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11958871/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11958871/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11958871%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958871/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11958871/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11958871/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40164966/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11958871/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40164966/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11958871/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11958871/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="wVexUhY8mUfqVfcvgjZA2cWYqumcji0TUgvepqi5p4HwwWcp3Q30bpdMnNhfK7eD">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
