
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Development of a cost‐effective high‐throughput mid‐density 5K genotyping assay for germplasm characterization and breeding in groundnut - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="61E606D27FFF9D030706D2000A355BF7.m_1" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958872/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Development of a cost‐effective high‐throughput mid‐density 5K genotyping assay for germplasm characterization and breeding in groundnut">
<meta name="citation_author" content="Manish K Pandey">
<meta name="citation_author_institution" content="Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India">
<meta name="citation_author" content="Vinay Sharma">
<meta name="citation_author_institution" content="Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India">
<meta name="citation_author" content="Aamir W Khan">
<meta name="citation_author_institution" content="Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India">
<meta name="citation_author" content="Pushpesh Joshi">
<meta name="citation_author_institution" content="Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India">
<meta name="citation_author" content="Sunil S Gangurde">
<meta name="citation_author_institution" content="Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India">
<meta name="citation_author" content="Prasad Bajaj">
<meta name="citation_author_institution" content="Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India">
<meta name="citation_author" content="Pasupuleti Janila">
<meta name="citation_author_institution" content="Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India">
<meta name="citation_author" content="Annapurna Chitikineni">
<meta name="citation_author_institution" content="Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India">
<meta name="citation_author_institution" content="WA State Agricultural Biotechnology Centre, Centre for Crop and Food Innovation, Murdoch University, Murdoch, WA, Australia">
<meta name="citation_author" content="Ramesh Bhat">
<meta name="citation_author_institution" content="Department of Biotechnology, University of Agricultural Sciences, Dharwad, India">
<meta name="citation_author" content="Babu N Motagi">
<meta name="citation_author_institution" content="Department of Biotechnology, University of Agricultural Sciences, Dharwad, India">
<meta name="citation_author" content="Chandramohan Sangh">
<meta name="citation_author_institution" content="ICAR‐Indian Institute of Groundnut Research (IIGR), Junagadh, India">
<meta name="citation_author" content="Thankappan Radhakrishnan">
<meta name="citation_author_institution" content="ICAR‐Indian Institute of Groundnut Research (IIGR), Junagadh, India">
<meta name="citation_author" content="Sandip K Bera">
<meta name="citation_author_institution" content="ICAR‐Indian Institute of Groundnut Research (IIGR), Junagadh, India">
<meta name="citation_author" content="Gregor Gorjanc">
<meta name="citation_author_institution" content="The Roslin Institute and The Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, UK">
<meta name="citation_author" content="Krishna Reddy Gujjula">
<meta name="citation_author_institution" content="Thermo Fisher Scientific, Waltham, Massachusetts, USA">
<meta name="citation_author" content="Nathan Hall">
<meta name="citation_author_institution" content="Thermo Fisher Scientific, Waltham, Massachusetts, USA">
<meta name="citation_author" content="Claudio D Carrasco">
<meta name="citation_author_institution" content="Thermo Fisher Scientific, Waltham, Massachusetts, USA">
<meta name="citation_author" content="Kandalam Arjun">
<meta name="citation_author_institution" content="Thermo Fisher Scientific, Waltham, Massachusetts, USA">
<meta name="citation_author" content="Srinivas Chandram">
<meta name="citation_author_institution" content="Thermo Fisher Scientific, Waltham, Massachusetts, USA">
<meta name="citation_author" content="Rajeev K Varshney">
<meta name="citation_author_institution" content="Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India">
<meta name="citation_author_institution" content="WA State Agricultural Biotechnology Centre, Centre for Crop and Food Innovation, Murdoch University, Murdoch, WA, Australia">
<meta name="citation_publication_date" content="2025 Mar 31">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="2">
<meta name="citation_firstpage" content="e70019">
<meta name="citation_doi" content="10.1002/tpg2.70019">
<meta name="citation_pmid" content="40164965">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958872/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958872/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958872/pdf/TPG2-18-e70019.pdf">
<meta name="description" content="Groundnut (Arachis hypogaea L.), also known as peanut, is an allotetraploid legume crop composed of two different progenitor sub‐genomes. This crop is an important source for food, feed, and confectioneries. Leveraging translational genomics ...">
<meta name="og:title" content="Development of a cost‐effective high‐throughput mid‐density 5K genotyping assay for germplasm characterization and breeding in groundnut">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Groundnut (Arachis hypogaea L.), also known as peanut, is an allotetraploid legume crop composed of two different progenitor sub‐genomes. This crop is an important source for food, feed, and confectioneries. Leveraging translational genomics ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958872/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11958872">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70019"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70019.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11958872%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11958872/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11958872/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958872/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70019" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Mar 31;18(2):e70019. doi: <a href="https://doi.org/10.1002/tpg2.70019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70019</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Development of a cost‐effective high‐throughput mid‐density 5K genotyping assay for germplasm characterization and breeding in groundnut</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pandey%20MK%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Manish K Pandey</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Manish K Pandey</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pandey%20MK%22%5BAuthor%5D" class="usa-link"><span class="name western">Manish K Pandey</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Vinay Sharma</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Vinay Sharma</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Vinay Sharma</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khan%20AW%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Aamir W Khan</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Aamir W Khan</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khan%20AW%22%5BAuthor%5D" class="usa-link"><span class="name western">Aamir W Khan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joshi%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Pushpesh Joshi</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Pushpesh Joshi</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joshi%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pushpesh Joshi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gangurde%20SS%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Sunil S Gangurde</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Sunil S Gangurde</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gangurde%20SS%22%5BAuthor%5D" class="usa-link"><span class="name western">Sunil S Gangurde</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bajaj%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Prasad Bajaj</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Prasad Bajaj</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bajaj%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Prasad Bajaj</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Janila%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Pasupuleti Janila</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Pasupuleti Janila</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Janila%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pasupuleti Janila</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chitikineni%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Annapurna Chitikineni</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Annapurna Chitikineni</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India</div>
<div class="p">
<sup>
<sup>6</sup>
</sup>WA State Agricultural Biotechnology Centre, Centre for Crop and Food Innovation, Murdoch University, Murdoch, WA, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chitikineni%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Annapurna Chitikineni</span></a>
</div>
</div>
<sup>1,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhat%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Ramesh Bhat</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Ramesh Bhat</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Biotechnology, University of Agricultural Sciences, Dharwad, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhat%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ramesh Bhat</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Motagi%20BN%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Babu N Motagi</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Babu N Motagi</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Biotechnology, University of Agricultural Sciences, Dharwad, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Motagi%20BN%22%5BAuthor%5D" class="usa-link"><span class="name western">Babu N Motagi</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sangh%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Chandramohan Sangh</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Chandramohan Sangh</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>ICAR‐Indian Institute of Groundnut Research (IIGR), Junagadh, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sangh%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chandramohan Sangh</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Radhakrishnan%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Thankappan Radhakrishnan</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Thankappan Radhakrishnan</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>ICAR‐Indian Institute of Groundnut Research (IIGR), Junagadh, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Radhakrishnan%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thankappan Radhakrishnan</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bera%20SK%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Sandip K Bera</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Sandip K Bera</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>ICAR‐Indian Institute of Groundnut Research (IIGR), Junagadh, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bera%20SK%22%5BAuthor%5D" class="usa-link"><span class="name western">Sandip K Bera</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gorjanc%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Gregor Gorjanc</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Gregor Gorjanc</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>The Roslin Institute and The Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gorjanc%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gregor Gorjanc</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gujjula%20KR%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Krishna Reddy Gujjula</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Krishna Reddy Gujjula</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Thermo Fisher Scientific, Waltham, Massachusetts, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gujjula%20KR%22%5BAuthor%5D" class="usa-link"><span class="name western">Krishna Reddy Gujjula</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hall%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Nathan Hall</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Nathan Hall</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Thermo Fisher Scientific, Waltham, Massachusetts, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hall%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nathan Hall</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carrasco%20CD%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Claudio D Carrasco</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Claudio D Carrasco</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Thermo Fisher Scientific, Waltham, Massachusetts, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carrasco%20CD%22%5BAuthor%5D" class="usa-link"><span class="name western">Claudio D Carrasco</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arjun%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Kandalam Arjun</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Kandalam Arjun</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Thermo Fisher Scientific, Waltham, Massachusetts, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arjun%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kandalam Arjun</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chandram%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Srinivas Chandram</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Srinivas Chandram</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Thermo Fisher Scientific, Waltham, Massachusetts, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chandram%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Srinivas Chandram</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Varshney%20RK%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Rajeev K Varshney</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Rajeev K Varshney</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India</div>
<div class="p">
<sup>
<sup>6</sup>
</sup>WA State Agricultural Biotechnology Centre, Centre for Crop and Food Innovation, Murdoch University, Murdoch, WA, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Varshney%20RK%22%5BAuthor%5D" class="usa-link"><span class="name western">Rajeev K Varshney</span></a>
</div>
</div>
<sup>1,</sup><sup>6,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270019-aff-0001">
<sup>
<sup>1</sup>
</sup>Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India</div>
<div id="tpg270019-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Biotechnology, University of Agricultural Sciences, Dharwad, India</div>
<div id="tpg270019-aff-0003">
<sup>
<sup>3</sup>
</sup>ICAR‐Indian Institute of Groundnut Research (IIGR), Junagadh, India</div>
<div id="tpg270019-aff-0004">
<sup>
<sup>4</sup>
</sup>The Roslin Institute and The Royal (Dick) School of Veterinary Studies, University of Edinburgh, Edinburgh, UK</div>
<div id="tpg270019-aff-0005">
<sup>
<sup>5</sup>
</sup>Thermo Fisher Scientific, Waltham, Massachusetts, USA</div>
<div id="tpg270019-aff-0006">
<sup>
<sup>6</sup>
</sup>WA State Agricultural Biotechnology Centre, Centre for Crop and Food Innovation, Murdoch University, Murdoch, WA, Australia</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Manish K. Pandey and Rajeev K. Varshney, Center for Excellence in Genomics and Systems Biology (CEGSB) and Center for Pre ‐Breeding Research (CPBR), International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT), Hyderabad, India. Email: <span>Manish.Pandey@icrisat.org</span> and <span>Rajeev.Varshney@murdoch.edu.au</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Feb 6; Received 2024 Oct 14; Accepted 2025 Feb 24; Issue date 2025 Jun.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 ICRISAT. <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11958872  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40164965/" class="usa-link">40164965</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Groundnut (<em>Arachis hypogaea</em> L.), also known as peanut, is an allotetraploid legume crop composed of two different progenitor sub‐genomes. This crop is an important source for food, feed, and confectioneries. Leveraging translational genomics research has expedited the precision and speed in making selections of progenies in several crops through either marker‐assisted selection or genomic selection, including groundnut. The availability of foundational genomic resources such as reference genomes for diploid progenitors and cultivated tetraploids, offered substantial opportunities for genomic interventions, including the development of genotyping assays. Here, a cost‐effective and high‐throughput genotyping assay has been developed with 5,081 single nucleotide polymorphisms (SNPs) referred to as “mid‐density assay.” This multi‐purpose assay includes 5,000 highly informative SNPs selected based on higher polymorphism information content (PIC) from our previously developed high‐density “Axiom_<em>Arachis</em>” array containing 58,233 SNPs. Additionally 82 SNPs associated with five resilience and quality traits were included for marker‐assisted selection. To test the utility of the mid‐density genotyping (MDG) assay, 2,573 genotypes from distinct sets of breeding populations were genotyped with the 5,081 SNPs. PIC of the SNPs in the MDG ranged from 0.34 to 0.37 among diverse sets. The first three principal components collectively explained 82.08% of the variance among these genotypes. The mid‐density assay demonstrated a proficient ability to distinguish between the genotypes, offering a high level of genome‐wide nucleotide diversity. This assay holds promise for possible deployment in the identification of varietal seed mixtures, genetic purity within gene bank germplasms and seed systems, foreground and background selection in backcross breeding programs, genomic selection, and sparse trait mapping studies in groundnut.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270019-list-10001" class="list" style="list-style-type:disc">
<li><p>Developed a mid‐density genotyping assay with 5,081 highly informative SNPs.</p></li>
<li><p>A total of 82 trait‐associated SNPs were also included in this assay for foreground selection.</p></li>
<li><p>A comprehensive diversity analysis study was performed using mid‐density assay on 2,573 genotypes.</p></li>
<li><p>The efficiency of mid‐density (5K) assay was tested for foreground and background selection and compared with high density.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270019-dl-0001">
<dt>ABLs</dt>
<dd><p>advanced breeding lines</p></dd>
<dt>FP</dt>
<dd><p>founder parents</p></dd>
<dt>GBS</dt>
<dd><p>genotyping‐by‐sequencing</p></dd>
<dt>MABC</dt>
<dd><p>marker‐assisted backcrossing</p></dd>
<dt>MAF</dt>
<dd><p>minor allele frequency</p></dd>
<dt>MAS</dt>
<dd><p>marker‐assisted selection</p></dd>
<dt>MDG</dt>
<dd><p>mid‐density genotyping</p></dd>
<dt>PCA</dt>
<dd><p>principal component analysis</p></dd>
<dt>PIC</dt>
<dd><p>polymorphism information content</p></dd>
<dt>QC</dt>
<dd><p>quality control</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus</p></dd>
<dt>RS</dt>
<dd><p>reference set</p></dd>
<dt>RV</dt>
<dd><p>released varieties</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
<dt>TASSEL</dt>
<dd><p>Trait Analysis by Association, Evolution, and Linkage</p></dd>
<dt>T‐GBS</dt>
<dd><p>targeted genotyping‐by‐sequencing</p></dd>
<dt>TP</dt>
<dd><p>training population</p></dd>
<dt>TVC</dt>
<dd><p>torrent variant caller</p></dd>
</dl>
<section id="tpg270019-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Groundnut (<em>Arachis hypogaea</em> L.) is an allotetraploid crop, composed of two progenitor sub‐genomes, with chromosome number 2<em>n</em> = 2<em>x</em> = 20 and genome size of ∼2.7 Gb for cultivated groundnut. Groundnut is an important source of protein, oil, and minerals cultivated in more than 100 countries worldwide. Notably, groundnut improvement aims to increase productivity, nutritional value, and resilience against biotic and abiotic stresses. Translational genomics expedited selection process with precision in several crops, including groundnut, through either marker‐assisted selection (MAS) (Bera et al., <a href="#tpg270019-bib-0001" class="usa-link" aria-describedby="tpg270019-bib-0001">2018</a>, <a href="#tpg270019-bib-0002" class="usa-link" aria-describedby="tpg270019-bib-0002">2019</a>; Janila et al., <a href="#tpg270019-bib-0015" class="usa-link" aria-describedby="tpg270019-bib-0015">2016</a>; Kolekar et al., <a href="#tpg270019-bib-0016" class="usa-link" aria-describedby="tpg270019-bib-0016">2017</a>; Pandey et al., <a href="#tpg270019-bib-0024" class="usa-link" aria-describedby="tpg270019-bib-0024">2024</a>; Shasidhar et al., <a href="#tpg270019-bib-0027" class="usa-link" aria-describedby="tpg270019-bib-0027">2020</a>; Varshney et al., <a href="#tpg270019-bib-0032" class="usa-link" aria-describedby="tpg270019-bib-0032">2014</a>; Yeri &amp; Bhat, <a href="#tpg270019-bib-0035" class="usa-link" aria-describedby="tpg270019-bib-0035">2016</a>) or genomic selection (Pandey et al., <a href="#tpg270019-bib-0022" class="usa-link" aria-describedby="tpg270019-bib-0022">2020</a>). The development of high‐quality reference genomes for both diploid progenitor species (<em>A. duranensis</em> &amp; <em>A. ipaensis</em>) and the cultivated subspecies (subsp. <em>hypogaea</em> and subsp. <em>fastigiata</em>) of tetraploid groundnut mark as an important milestone in groundnut genomics (Bertioli et al., <a href="#tpg270019-bib-0003" class="usa-link" aria-describedby="tpg270019-bib-0003">2016</a>; X. Chen et al., <a href="#tpg270019-bib-0008" class="usa-link" aria-describedby="tpg270019-bib-0008">2016</a>; Bertioli et al., <a href="#tpg270019-bib-0004" class="usa-link" aria-describedby="tpg270019-bib-0004">2019</a>; X. Chen et al., <a href="#tpg270019-bib-0009" class="usa-link" aria-describedby="tpg270019-bib-0009">2019</a>; Zhuang et al., <a href="#tpg270019-bib-0036" class="usa-link" aria-describedby="tpg270019-bib-0036">2019</a>).</p>
<p>Various genotyping assays, with varied marker densities, are needed based on the scale and type of application in a breeding program. While whole‐genome sequencing with varied depth and coverage provides one of the most flexible and informative options for genotyping, it is not cost‐effective. Furthermore, the requirement of sound technical expertise to analyze sequencing data hinders the deployment of large‐scale genotyping of samples in gene bank, breeding, and seed systems. High‐density genotyping (HDG) arrays, such as single nucleotide polymorphism (SNP) arrays (ranging from 10,000 SNPs to 250,000 SNPs), are available for multiple crops. Due to substantial similarity between the sub‐genomes of tetraploid groundnut, an HDG platform was designed for genetic diversity and trait mapping studies in groundnut. Specifically, International Crops Research Institute for the Semi‐Arid Tropics (ICRISAT) led the development of a HDG array (Axiom_ <em>Arachis</em> Array) with 58K genome‐wide SNPs in collaboration with the University of Georgia and Thermo Fisher Scientific (Pandey et al., <a href="#tpg270019-bib-0021" class="usa-link" aria-describedby="tpg270019-bib-0021">2017</a>). So far, the international groundnut research community has used HDG array in &gt;25 studies of genetic diversity, trait mapping, and background genome recovery in backcross breeding (Pandey et al., <a href="#tpg270019-bib-0022" class="usa-link" aria-describedby="tpg270019-bib-0022">2020</a>; Shasidhar et al., <a href="#tpg270019-bib-0027" class="usa-link" aria-describedby="tpg270019-bib-0027">2020</a>).</p>
<p>While the Axiom_<em>Arachis</em> array is sufficient for studies related to genetic diversity and trait mapping through genetic and association analysis, it is not cost‐effective for large‐scale application in groundnut breeding. Therefore, a cost‐effective mid‐density genotyping (MDG) assay is necessary with around 2,000–6,000 SNPs to be deployed in large‐scale applications with reduced per sample cost of about 10 USD. HDG arrays, typically consisting of 50K–80K of SNPs, are significantly more expensive, with costs ranging from 50 to 60 USD per sample, including DNA extraction, depending on the platform. MDG assay can be used in genetic diversity, background genome recovery, genomic selection, genetic purity, and tracking linkage drag in wide crosses.</p>
<p>MDG is an established tool for large‐scale adoption in breeding programs. It is desirable to keep MDG assay dynamic so that the useful SNPs from genomics research can be incorporated, making the MDG assay more and more informative over time for breeders and researchers. Recently, a new MDG assay with 4,000 SNPs has been developed in pearl millet using the AgriSeq platform with targeted genotyping‐by‐sequencing (T‐GBS) (Semalaiyappan et al., <a href="#tpg270019-bib-0026" class="usa-link" aria-describedby="tpg270019-bib-0026">2023</a>). Here, we developed and validated a new cost‐effective MDG assay in groundnut with 5,081 SNP markers for routine application of genomics in genetic diversity, background genome recovery in recurrent parents, checking duplications and genetic mixture, and most importantly, genomic selection. The AgriSeq T‐GBS assay was developed with 5,081 SNPs including trait‐associated SNPs for seven to eight resilience and quality traits in groundnut. We are deploying this assay for large‐scale genomic selection in groundnut at ICRISAT and National Agricultural Research System (NARS) locations. This assay will enable widespread adoption in several other new applications in addition to genetic diversity and trait mapping, background selection, and genomic selection, thereby contributing to the advancement of future groundnut breeding programs.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270019-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270019-list-0001" class="list" style="list-style-type:disc">
<li><p>Developed a mid‐density genotyping assay with 5,081 highly informative SNPs.</p></li>
<li><p>A total of 82 trait‐associated SNPs were also included in this assay for foreground selection.</p></li>
<li><p>A comprehensive diversity analysis study was performed using mid‐density assay on 2,573 genotypes.</p></li>
<li><p>The efficiency of mid‐density (5K) assay was tested for foreground and background selection and compared with high density.</p></li>
</ul></section></section></section><section id="tpg270019-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270019-sec-0040"><h3 class="pmc_sec_title">2.1. Sources for SNPs and selection of best set of informative SNPs</h3>
<p>The genotyping data were generated using the high‐density Axiom_<em>Arachis</em> array, containing 58,233 unique and informative SNPs (Pandey et al., <a href="#tpg270019-bib-0021" class="usa-link" aria-describedby="tpg270019-bib-0021">2017</a>) on a diverse panel of 722 cultivated groundnut genotypes, which had 254 from the ICRISAT reference set (RS), 285 from the ICRISAT training population (TP), and 174 released varieties (RV) across India. The seven genotypes were common to the RS and TP. We analyzed these data and selected 8,200 high‐quality SNPs based on the criteria of high polymorphism information content (PIC; between 0.374 and above) and uniform genome coverage. Previous studies also reported SNP markers with similar PIC values as informative for genetic diversity and population structure analysis (H. Chen et al., <a href="#tpg270019-bib-0007" class="usa-link" aria-describedby="tpg270019-bib-0007">2011</a>; Garcia et al., <a href="#tpg270019-bib-0013" class="usa-link" aria-describedby="tpg270019-bib-0013">2018</a>; Singh et al., <a href="#tpg270019-bib-0029" class="usa-link" aria-describedby="tpg270019-bib-0029">2013</a>). All the SNPs were initially subjected to check for missing calls &lt;0.2, and those with heterozygous calls &gt;0.05 (&gt;5% heterozygous calls) were removed. Next, priority was given to SNPs that did not have another SNP within the flanking region of 200 bp. Finally, a total of 82 trait‐associated SNPs identified from different studies were included, such as those associated with resistance to rust and late leaf spot, high oleic acid, seed weight, fresh seed dormancy, and shelling percentage (Figure <a href="#tpg270019-fig-0001" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="tpg270019-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d2e3/11958872/a776a611fba1/TPG2-18-e70019-g003.jpg" loading="lazy" id="jats-graphic-1" height="335" width="603" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270019-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Flowchart for developing mid‐density genotyping assay in groundnut with 5,081 informative single nucleotide polymorphisms (SNPs).</p></figcaption></figure></section><section id="tpg270019-sec-0050"><h3 class="pmc_sec_title">2.2. Assay design and validation</h3>
<p>The AgriSeq T‐GBS assay design comprises three sequential steps, such as (1) pre‐design quality control (QC), (2) AgriSeq Primer designs, (3) post‐design QC, and (4) final assay development. Initially, ∼8,027 SNP markers were selected and further filtered for the assay. Using the “Tifrunner” reference genome (Bertioli et al., <a href="#tpg270019-bib-0004" class="usa-link" aria-describedby="tpg270019-bib-0004">2019</a>), the uniqueness of the context sequence for each candidate marker was examined by mapping it to the reference genome.</p>
<p>Since the 8,027 SNPs were initially identified on diploid progenitor genomes (Bertioli et al., <a href="#tpg270019-bib-0003" class="usa-link" aria-describedby="tpg270019-bib-0003">2016</a>) for the development of a 58K Axiom_<em>Arachis</em> array (Pandey et al., <a href="#tpg270019-bib-0021" class="usa-link" aria-describedby="tpg270019-bib-0021">2017</a>), additional computational work was undertaken to identify marker position in the tetraploid reference genome, Tifrunner (Bertioli et al., <a href="#tpg270019-bib-0004" class="usa-link" aria-describedby="tpg270019-bib-0004">2019</a>). Each marker was mapped to an individual sub‐genome separately to identify the location of the marker with the mapping threshold of 95%. For each marker, results of mapping on separate sub‐genomes were classified into (a) mapping uniquely and the marker position is identified, (b) mapping uniquely, with region identified but the marker position is not identified, (c) mapping to multiple locations on the same sub‐genome, and last (d) not mapping to any sub‐genome.</p>
<p>Based on the mapping results against the reference genome, markers were categorized as follows: (a) unique to a particular sub‐genome (A or B); (b) mapping to two loci (one each in sub‐genome A and B); and (c) mapping to more than two loci on the reference genome or mapping to exactly two loci on the same sub‐genome (A or B). These markers were then divided into three categories. In Category 1, the marker sequence maps uniquely to one of the sub‐genomes, while the Category 2 marker sequence maps to two loci on the genome (one on sub‐genome A and another on sub‐genome B), and in Category 3 marker sequence maps to two loci on the genome, either both on sub‐genome A or both on sub‐genome B or maps to more than two loci. The primer designing for Category 3 markers (associated with traits) was removed from the design process at this stage and primer design was done separately.</p>
<p>The primers were designed for Category 1 and 2 markers. The designer pipeline runs iteratively, modifying primer design and amplicon parameters (e.g., GC [guanine and cytosine] content and Tm [melting temperature] values) sequentially to find all possible amplicons, which consist of primer pairs and an insertion sequence, for each marker. The designer pipeline chooses the best amplicon that contains the marker. During this process, the specificity of the primers was checked on the reference genome. Some of the primers were discarded due to specificity. Post‐design quality check was performed for Category 1 markers with primer designs amplifying one sub‐genome (either A or B), while Category 2 markers with primer designs amplifying both sub‐genomes (A and B) (Figure <a href="#tpg270019-fig-0002" class="usa-link">2</a>). After filtering, approximately 5,000 SNPs were finalized from the initially selected set of 8,027 SNPs. Additionally, 82 highly informative SNPs were finalized from a subset of 213 trait‐associated SNPs. In total, 5,081 highly informative SNPs were used for assay design and validation.</p>
<figure class="fig xbox font-sm" id="tpg270019-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d2e3/11958872/79469fb15dd4/TPG2-18-e70019-g005.jpg" loading="lazy" id="jats-graphic-3" height="443" width="800" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270019-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Quality matrix for validation samples and call types for markers from progenitor diploid and cultivated tetraploid reference genome. (a) Quality metrics for 192 samples (&gt;20X coverage); (b) total 963 markers from progenitor diploid genome expected allele frequencies of 0.0, 0.5, 1 for AA, AB, BB, respectively; (c) total 3393 markers from tetraploid genome with expected allele frequencies of 0.5, 0.75, 1 for AABB, ABBB, BBBB, respectively; (d) total 669 markers from tetraploid genome expected allele frequencies of 0.5, 0.75, 1 for AAAA, ABAA, BBAA, respectively.</p></figcaption></figure></section><section id="tpg270019-sec-0060"><h3 class="pmc_sec_title">2.3. Assay validation</h3>
<p>The AgriSeq T‐GBS workflow starts with sample concentration, initially checked using the Quant‐IT Ds DNA assay (<a href="https://www.ncbi.nlm.nih.gov/protein/Q32851" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q32851</a>, Invitrogen), followed by sample dilutions for normalization of uniform DNA concentrations. Subsequently, 3 µL of normalized DNA was used for library amplification, employing AgriSeq HTS library kit (<a href="https://www.ncbi.nlm.nih.gov/protein/A34143" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">A34143</a>, Life Technologies) followed by 16 min amp/cycles (12 cycles) and 20 min pre‐ligation of IonCode Barcode Adapters (IonCode_0101 to IonCode_0296). Genotyping was performed using the Ion 550 Chef kit (A43542‐Life Technologies) and Ion 550 chip (∼120 million reads) (A42962‐Life Technologies). This assay was validated on a set of 386 diverse lines.</p>
<p>To ensure high‐quality genotyping‐by‐sequencing (GBS), the number of reads per amplicon was monitored during sequencing, aiming for a mean coverage depth of at least 100X to increase the likelihood that most markers having a minimum coverage of 5X. The attention was given to assessing the percentage of mapped reads and ensuring that on‐target reads aligned correctly over a target region, with a focus on achieving metric values that minimize off‐target alignments, indicating potential issues such as sample contamination and uncharacterized genetic variation. Note that the low uniformity indicates poor sequencing across the entire amplicon. Therefore, the uniformity was maintained by ensuring a percentage of target bases with at least 0.2 times the mean depth. This feature is a measure of end‐to‐end coverage over the target region. Finally, the Marker Call Rate was assessed by calculating the percentage of all samples generating a genotype call for a specific marker. The emphasis was given to designing assays with high‐performing markers, targeting an overall mean marker call rates &gt;90% (Figure <a href="#tpg270019-fig-0003" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="tpg270019-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d2e3/11958872/37d037904336/TPG2-18-e70019-g004.jpg" loading="lazy" id="jats-graphic-5" height="490" width="700" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270019-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Workflow for groundnut AgriSeq targeted genotyping‐by‐sequencing‐based mid‐density assay genotyping. GBS, genotyping‐by‐sequencing.</p></figcaption></figure></section><section id="tpg270019-sec-0070"><h3 class="pmc_sec_title">2.4. Genotyping</h3>
<p>Genotyping was performed using Torrent Variant Caller (TVC) for 963 category‐1 markers and AgriSeqPoly and 4,118 category‐2 markers. TVC is suitable for diploid markers and AgriSeqPoly is utilized to genotype both allo‐ and auto‐polyploid markers. The sequenced reads were de‐multiplexed to individual samples using the barcode sequences. For each sample, the sequenced reads from the targeted regions were mapped to the reference genome using Torrent Mapping Alignment Program (<a href="https://github.com/iontorrent/TMAP" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/iontorrent/TMAP</a>), followed by genotyping using TVC (<a href="https://github.com/iontorrent/Torrent-Variant-Caller-stable" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/iontorrent/Torrent‐Variant‐Caller‐stable</a>). Both TVC and AgriSeqPoly implement a Bayesian model, using the number of reads supporting each allele as input to output a genotype call.</p></section><section id="tpg270019-sec-0080"><h3 class="pmc_sec_title">2.5. Assay deployment for performing genetic diversity in a set of 2,573 genotypes</h3>
<p>MDG assay was used for genotyping 1776 genotypes on the AgriSeq T‐GBS platform. This diverse set included 270 founder parents (FP) from breeding programs (ICRISAT, UAS‐Dharwad, and ICAR‐DGR), along with 1,506 advanced breeding lines (ABLs). Additionally, SNP calls for 797 genotypes were extracted from genotyping data generated using the high‐density Axiom_<em>Arachis</em> array, comprising 297 RSs, 294 TPs, and 206 RV. The genotyping data from this set of 2,573 genotypes were then analyzed to assess genetic diversity.</p>
<p>Genotyping data were filtered before being uploaded into Trait Analysis by Association, Evolution, and Linkage (TASSEL) v5.0 (Bradbury et al., <a href="#tpg270019-bib-0006" class="usa-link" aria-describedby="tpg270019-bib-0006">2007</a>), which was used to further filter samples and sites, and produce a dendrogram of samples (<a href="https://www.maizegenetics.net/tassel" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.maizegenetics.net/tassel</a>). Thorough filtering helped to exclude uncalled sites, enhancing the accuracy of the predicted tree. Before uploading data to TASSEL, filtering was performed and the ambiguous SNP IDs and associated calls were identified and removed, considering instances where two or more IDs occurred at the same position or when one ID had two or more positions associated with it. Sample names were converted to a standardized format compatible with downstream analyses, and a conversion table was created. The conversion table associates the original sample name with the new standardized name. All samples were added to a single data matrix, saved as a comma‐separated value file, and modified using Microsoft Excel to be a TASSEL‐compatible hapmap format. Positions with multi‐SNP calls were excluded, as TASSEL does not process multi‐SNP data. Following filtration, the data file was loaded into TASSEL for further filtration using three approaches (unfiltered, less strict, and strict). The stringent dataset was generated by filtering sites to include only loci with a ≥2,000 calls, followed by retaining samples with a proportion of 0.95 of sites present, resulting in 2153 samples with 4714 sites. Alternatively, the less stringent dataset was generated by filtering loci with calls of ≥2,700, retaining only samples with a proportion of 0.90 sites present, resulting in 2,639 samples with 2,988 sites. The resulting datasets were used for subsequent analysis. For each dataset, a distance matrix was constructed, and multidimensional scaling (MDS), similar to a principal component analysis (PCA), was performed using TASSEL. Neighbor‐joining trees were constructed for the less strict and strict datasets. Attempts to create a tree from the unfiltered dataset yielded no informative result. For data visualizations, MDS outputs were visualized with Plotly in Python 3.9 and saved as an interactive graphic embedded in an html document. Additionally, trees were rendered using ete3 in Python 3.9 and saved as scalable vector graphics for comprehensive visualization and analysis.</p></section></section><section id="tpg270019-sec-0090"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270019-sec-0100"><h3 class="pmc_sec_title">3.1. SNP selection and assay design</h3>
<p>Previously, we developed an HDG array containing 58,233 unique and informative SNPs (Pandey et al., <a href="#tpg270019-bib-0021" class="usa-link" aria-describedby="tpg270019-bib-0021">2017</a>) and used it to genotype 722 cultivated groundnut genotypes (Table <a href="#tpg270019-supinfo-0001" class="usa-link">S1</a>). This diverse panel comprised accessions from the ICRISAT RS (254 cultivated accessions), the ICRISAT TP (285 accessions), and RV across India (174 varieties), with seven common genotypes shared across the three panels. Based on the PIC and uniform genome coverage, 8,027 SNPs were selected, with a PIC value of 0.374 and above (Table <a href="#tpg270019-supinfo-0001" class="usa-link">S2</a>). Of these, 5,000 SNPs were prioritized, while the remaining 3,027 were kept in reserve, considering their PIC and genome coverage. Further, as an essential requirement for this platform, priority was given to only those SNPs that did not have any another SNP within flanking region of 200 bp. The primer sequence was analyzed using blast analysis against the reference genome for priority one markers, 863 markers mapped with zero mismatche and uniquely to one sub‐genome of Tifrunner, 3,991 markers mapped with zero mismatches but to two locations (one on sub‐genome A and another on sub‐genome B), and the remaining 1099 designs/primers have mismatches with the Tifrunner reference.</p>
<p>In addition to 8,027 SNPs, a total of 82 traits associated with SNPs from different studies were included, covering traits such as resistance to rust and late leaf spot, high oleic acid, seed weight, fresh seed dormancy, and shelling percentage. These 82 SNPs were identified through trait mapping using reference genomes (Table <a href="#tpg270019-supinfo-0001" class="usa-link">S3</a>), specifically 97 SNPs (58 Category 1, 25 Category 2, and 14 Category 3) from <em>A. duranensis</em> (Bertioli et al., <a href="#tpg270019-bib-0003" class="usa-link" aria-describedby="tpg270019-bib-0003">2016</a>), 104 (47 Category 1, 18 Category 2, and 39 Category 3) from Shitouqi (Zhuang et al., <a href="#tpg270019-bib-0036" class="usa-link" aria-describedby="tpg270019-bib-0036">2019</a>), and 12 (6 Category 1, 1 Category 2, and 5 Category 3) from Tiffrunner (Bertioli et al., <a href="#tpg270019-bib-0004" class="usa-link" aria-describedby="tpg270019-bib-0004">2019</a>) assembly. A similar analysis was performed for these markers, which showed 111 SNPs as Category 1, 44 SNPs as Category 2 markers, and 58 Category 3 markers.</p>
<p>The final assay comprised 5,081 SNPs, consisting of 963 Category 1 markers (amplifying either A or B sub‐genome) and 4,118 Category 2 markers (amplifying both sub‐genome A and B) (Table <a href="#tpg270019-supinfo-0001" class="usa-link">S4</a>). In total, the final assay included 5,063 SNPs, eight deletions, and 10 insertions. Of the total 5,081 SNP markers, 2,428 SNP markers were selected from A sub‐genome (Chromosomes 1–10), and 2,653 SNP markers from B sub‐genome (Chromosomes 11–20) (Table <a href="#tpg270019-tbl-0001" class="usa-link">1</a>) (Figure <a href="#tpg270019-fig-0004" class="usa-link">4</a>). The distribution of markers across chromosomes varied, from 187 (Chromosome 17) to 393 (Chromosome 3), with an average of 254 SNP markers per chromosome.</p>
<section class="tw xbox font-sm" id="tpg270019-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Number of single nucleotide polymorphism (SNP) markers from different chromosomes in mid‐density genotyping assay.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Chromosome</th>
<th align="left" rowspan="1" colspan="1">Number of markers</th>
<th align="left" rowspan="1" colspan="1">Chromosome</th>
<th align="left" rowspan="1" colspan="1">Number of markers</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Arahy.01</td>
<td align="left" rowspan="1" colspan="1">264</td>
<td align="left" rowspan="1" colspan="1">Arahy.11</td>
<td align="left" rowspan="1" colspan="1">211</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Arahy.02</td>
<td align="left" rowspan="1" colspan="1">278</td>
<td align="left" rowspan="1" colspan="1">Arahy.12</td>
<td align="left" rowspan="1" colspan="1">347</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Arahy.03</td>
<td align="left" rowspan="1" colspan="1">393</td>
<td align="left" rowspan="1" colspan="1">Arahy.13</td>
<td align="left" rowspan="1" colspan="1">338</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Arahy.04</td>
<td align="left" rowspan="1" colspan="1">227</td>
<td align="left" rowspan="1" colspan="1">Arahy.14</td>
<td align="left" rowspan="1" colspan="1">272</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Arahy.05</td>
<td align="left" rowspan="1" colspan="1">225</td>
<td align="left" rowspan="1" colspan="1">Arahy.15</td>
<td align="left" rowspan="1" colspan="1">237</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Arahy.06</td>
<td align="left" rowspan="1" colspan="1">216</td>
<td align="left" rowspan="1" colspan="1">Arahy.16</td>
<td align="left" rowspan="1" colspan="1">281</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Arahy.07</td>
<td align="left" rowspan="1" colspan="1">196</td>
<td align="left" rowspan="1" colspan="1">Arahy.17</td>
<td align="left" rowspan="1" colspan="1">261</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Arahy.08</td>
<td align="left" rowspan="1" colspan="1">196</td>
<td align="left" rowspan="1" colspan="1">Arahy.18</td>
<td align="left" rowspan="1" colspan="1">187</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Arahy.09</td>
<td align="left" rowspan="1" colspan="1">252</td>
<td align="left" rowspan="1" colspan="1">Arahy.19</td>
<td align="left" rowspan="1" colspan="1">285</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Arahy.10</td>
<td align="left" rowspan="1" colspan="1">181</td>
<td align="left" rowspan="1" colspan="1">Arahy.20</td>
<td align="left" rowspan="1" colspan="1">234</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Sub‐genome A</td>
<td align="left" rowspan="1" colspan="1">2428</td>
<td align="left" rowspan="1" colspan="1">Sub‐genome B</td>
<td align="left" rowspan="1" colspan="1">2653</td>
</tr>
<tr>
<td colspan="3" align="left" rowspan="1">Total markers on the groundnut Agriseq mid‐density genotyping assay</td>
<td align="left" rowspan="1" colspan="1">5081</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270019-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="tpg270019-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d2e3/11958872/441222cfd24b/TPG2-18-e70019-g002.jpg" loading="lazy" id="jats-graphic-7" height="402" width="603" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270019-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome‐wide distribution of single nucleotide polymorphism (SNP) markers on mid‐density genotyping assay on 20 chromosomes of cultivated groundnut genome. The triangles indicate the SNP positions of trait‐associated SNPs for five quality‐ and resilience‐related traits as well as 20 quality control (QC) SNPs.</p></figcaption></figure></section><section id="tpg270019-sec-0110"><h3 class="pmc_sec_title">3.2. Verification and validation of new mid‐density genotyping assay</h3>
<p>The high‐quality GBS matrix was ensured during sequencing by keeping coverage depth of at least 100X to increase the likelihood of each marker with a minimum coverage of 5X. The emphasis was given to design assays with high‐performing markers with overall mean marker call rates &gt;90%. This assay was initially genotyped for 386 samples to check the quality of genotyping data, wherein many of the samples are being duplicated to check the consistency of call data. After ensuring high‐quality data, this newly developed genotyping assay was used for genotyping a set of 2,573 genotypes (270 FP of breeding programs [ICRISAT, UAS‐Dharwad, and ICAR‐DGR], 1506 ABLs, 297 RS, 294 TPs, and 206 RV). A distance matrix was constructed for each dataset, and MDS, similar to a PCA, was performed using TASSEL for all datasets (Figure <a href="#tpg270019-fig-0005" class="usa-link">5</a>). The first three principal components collectively explain 82.08% of the variance, with PC1 contributing 42.16%, PC2 explaining 24.56%, and PC3 explaining 15.36%, all indicating substantial variation among the genotypes. The results showed clear clustering of the RS, TP, and RV, while mixed for FP and ABLs. The genetic relationship information, along with genetic distance will help in selecting parents to sample the high genetic diversity for future breeding programs. Analysis of the PIC values for each set indicates that the markers are moderately informative, with PIC varying from 0.34 to 0.37. The proportion of heterozygosity across the different sets ranged from 0.027 to 0.294, with ABLs set 1 exhibited the lowest, whereas the RS showed the highest heterozygosity. Meanwhile, average minor allele frequency (MAF) values were calculated for each set, depicting the ABLs set 1 had the lowest MAF (0.062), while the RS had the highest (0.216). The calculated proportion of missing data for the six distinct sets showed that the RS had the lowest proportion (0.031), whereas the ABLs set 2 had the highest proportion (0.082) (Table <a href="#tpg270019-tbl-0002" class="usa-link">2</a>). This information provides a significant understanding of the genetic diversity and variability within different sets, resulting in valuable clues for the characterization and selection of breeding lines.</p>
<figure class="fig xbox font-sm" id="tpg270019-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d2e3/11958872/a2be9249d3c8/TPG2-18-e70019-g006.jpg" loading="lazy" id="jats-graphic-9" height="380" width="800" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270019-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Deployment of mid‐density assay for genetic diversity analysis of a diverse set containing genotypes from founder parents (FP) of ICRISAT breeding program, training population (TP), ICRISAT reference set (RS), two sets of advanced breeding lines (ABLs), and released varieties (RV).</p></figcaption></figure><section class="tw xbox font-sm" id="tpg270019-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Summary of genetic diversity indices among distinct diverse sets of groundnut germplasm.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">Number of samples</th>
<th align="left" rowspan="1" colspan="1">Released varieties</th>
<th align="left" rowspan="1" colspan="1">Training population</th>
<th align="left" rowspan="1" colspan="1">Reference set</th>
<th align="left" rowspan="1" colspan="1">Founder parents</th>
<th align="left" rowspan="1" colspan="1">Advanced breeding lines set 2</th>
<th align="left" rowspan="1" colspan="1">Advanced breeding lines set 1</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Number of samples</td>
<td align="center" rowspan="1" colspan="1">2573</td>
<td align="center" rowspan="1" colspan="1">206</td>
<td align="center" rowspan="1" colspan="1">294</td>
<td align="center" rowspan="1" colspan="1">297</td>
<td align="center" rowspan="1" colspan="1">270</td>
<td align="center" rowspan="1" colspan="1">805</td>
<td align="center" rowspan="1" colspan="1">701</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PIC</td>
<td align="center" rowspan="1" colspan="1">0.32</td>
<td align="center" rowspan="1" colspan="1">0.34</td>
<td align="center" rowspan="1" colspan="1">0.34</td>
<td align="center" rowspan="1" colspan="1">0.36</td>
<td align="center" rowspan="1" colspan="1">0.36</td>
<td align="center" rowspan="1" colspan="1">0.37</td>
<td align="center" rowspan="1" colspan="1">0.35</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Average minor allele frequency</td>
<td align="center" rowspan="1" colspan="1">0.192</td>
<td align="center" rowspan="1" colspan="1">0.076</td>
<td align="center" rowspan="1" colspan="1">0.102</td>
<td align="center" rowspan="1" colspan="1">0.216</td>
<td align="center" rowspan="1" colspan="1">0.066</td>
<td align="center" rowspan="1" colspan="1">0.064</td>
<td align="center" rowspan="1" colspan="1">0.062</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Proportion missing</td>
<td align="center" rowspan="1" colspan="1">0.065</td>
<td align="center" rowspan="1" colspan="1">0.034</td>
<td align="center" rowspan="1" colspan="1">0.041</td>
<td align="center" rowspan="1" colspan="1">0.031</td>
<td align="center" rowspan="1" colspan="1">0.061</td>
<td align="center" rowspan="1" colspan="1">0.082</td>
<td align="center" rowspan="1" colspan="1">0.081</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Proportion not missing</td>
<td align="center" rowspan="1" colspan="1">0.934</td>
<td align="center" rowspan="1" colspan="1">0.966</td>
<td align="center" rowspan="1" colspan="1">0.958</td>
<td align="center" rowspan="1" colspan="1">0.968</td>
<td align="center" rowspan="1" colspan="1">0.938</td>
<td align="center" rowspan="1" colspan="1">0.917</td>
<td align="center" rowspan="1" colspan="1">0.918</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Proportion heterozygous</td>
<td align="center" rowspan="1" colspan="1">0.068</td>
<td align="center" rowspan="1" colspan="1">0.044</td>
<td align="center" rowspan="1" colspan="1">0.09</td>
<td align="center" rowspan="1" colspan="1">0.294</td>
<td align="center" rowspan="1" colspan="1">0.03</td>
<td align="center" rowspan="1" colspan="1">0.031</td>
<td align="center" rowspan="1" colspan="1">0.027</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270019-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270019-tbl2-note-0001"><p>Abbreviation: PIC, polymorphism information content.</p></div></div></section></section><section id="tpg270019-sec-0120"><h3 class="pmc_sec_title">3.3. Background genome recovery using mid‐density assay versus high‐density array</h3>
<p>A set of four marker‐assisted backcrossing (MABC) lines in BC<sub>3</sub>F<sub>4</sub> generation, derived from the cross GPBD4 and GJG9 were genotyped using the 58K high‐density array and 5K mid‐density assay to compare the background genome recovery (Figure <a href="#tpg270019-fig-0006" class="usa-link">6a</a>). GJG9 was recurrent parent (background genome) susceptible to late leaf spot and GPBD4 was used as donor for late leaf spot resistance. A total of 726 SNPs (13%) in 5K assay were polymorphic between parents GPBD4 and GJG9 of MABC lines, as compared to 2,900 SNPs (5%) in 58K array (Figure <a href="#tpg270019-fig-0006" class="usa-link">6b</a>). The background genome recovery was more densely visualized using a circos plot with a high‐density SNP array, in comparison to a mid‐density assay. Interestingly, uniform recovery of background genome was achieved from each of the 20 chromosomes using mid‐density assay. Therefore, mid‐density assay presents a cost‐effective option for background genome recovery in a large number of breeding populations.</p>
<figure class="fig xbox font-sm" id="tpg270019-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d2e3/11958872/b6f69637816e/TPG2-18-e70019-g001.jpg" loading="lazy" id="jats-graphic-11" height="852" width="680" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270019-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Background genome recovery using 58K array and 5K assay. (a) Circos plot illustrates background genome recovery of four marker‐assisted backcross lines using 58 and 5K assays. The tracks from inside to outside (a–d) illustrate background genome recovery of four marker‐assisted backcrossing (MABC) lines using 58K array, whereas tracks (e–h) illustrate background genome recovery of the same set of four MABC lines using 5K assay. Uniform whole genome recovery across 20 chromosomes of groundnut was achieved using 5K mid‐density assay. (b) Number of polymorphic single nucleotide polymorphisms (SNPs) recovered from 5K assay and 58K array. 13% in 5K assay and 5% in 58K array were identified as polymorphic SNPs between parents of MABC lines.</p></figcaption></figure></section></section><section id="tpg270019-sec-0130"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>Genomics advancements have facilitated the development of cost‐effective and highly precise SNP genotyping arrays. These arrays have proven important in accelerating breeding innovations and providing essential support for fundamental research in the area of crop breeding. In 2017, we previously developed high‐density genotyping array containing 58,233 unique and informative SNPs (Pandey et al., <a href="#tpg270019-bib-0021" class="usa-link" aria-describedby="tpg270019-bib-0021">2017</a>). This high‐density SNP array “Axiom<em>_Arachis</em>” has been extensively used for performing diversity analysis (Clevenger et al., <a href="#tpg270019-bib-0010" class="usa-link" aria-describedby="tpg270019-bib-0010">2017</a>), trait mapping (Dodia et al., <a href="#tpg270019-bib-0011" class="usa-link" aria-describedby="tpg270019-bib-0011">2019</a>; Luo et al., <a href="#tpg270019-bib-0018" class="usa-link" aria-describedby="tpg270019-bib-0018">2019</a>; Pandey et al., <a href="#tpg270019-bib-0022" class="usa-link" aria-describedby="tpg270019-bib-0022">2020</a>), background selection in backcross breeding program (Shasidhar et al., <a href="#tpg270019-bib-0027" class="usa-link" aria-describedby="tpg270019-bib-0027">2020</a>), and genomic prediction (Pandey et al., <a href="#tpg270019-bib-0022" class="usa-link" aria-describedby="tpg270019-bib-0022">2020</a>) research in groundnut by multiple research groups. We have used this genotyping assay to genotype a total of 722 cultivated groundnut genotypes. After analyzing these data, we selected 5,000 SNPs using the criteria of PIC and uniform genome coverage. SNPs having PIC value of 0.374 and above were shortlisted. For the final assay design, 5,081 SNPs were selected. Such customized SNP assays, comprising SNPs chosen from sequencing data and past association research, offer a valuable genotyping platform due to its efficacy, accessibility, and precision (Morales et al., <a href="#tpg270019-bib-0020" class="usa-link" aria-describedby="tpg270019-bib-0020">2020</a>; Sun et al., <a href="#tpg270019-bib-0030" class="usa-link" aria-describedby="tpg270019-bib-0030">2020</a>; Tian et al., <a href="#tpg270019-bib-0031" class="usa-link" aria-describedby="tpg270019-bib-0031">2021</a>).</p>
<p>The groundnut AgriSeq T‐GBS MDG assay has 5,081 SNP markers with balanced sub‐genome representation of 2,428 SNPs on the A sub‐genome (Chromosome 1–10) and 2,653 SNPs on the B sub‐genome (Chromosome 11–20). Of these, 82 SNPs have been associated with important traits such as rust and late leaf spot resistance, high oleic acid, seed weight, fresh seed dormancy, and shelling percentage for the purpose of foreground selection. Trait‐specific SNPs serve as effective tools in MAS, allowing breeders to find and select traits of interest more efficiently. Kompetitive Allele Specific polymerase chain reaction markers have been successfully employed, aiding in the selection of traits like disease resistance and quality enhancement (Mbanjo et al., <a href="#tpg270019-bib-0019" class="usa-link" aria-describedby="tpg270019-bib-0019">2024</a>). The reliability of SNP marker predictions is significantly influenced by their association with the trait. Major‐effect SNPs linked to key genes often show higher prediction accuracy, while complex traits governed by multiple genes and influenced by environmental factors may result in lower accuracy (Heffner et al., <a href="#tpg270019-bib-0014" class="usa-link" aria-describedby="tpg270019-bib-0014">2009</a>). SNP markers are especially valuable for traits that are difficult to evaluate phenotypically, such as drought tolerance or nutritional content. In such instances, trait‐specific SNP facilitates indirect selection without extensive and time‐consuming phenotypic evaluation (Xu &amp; Crouch, <a href="#tpg270019-bib-0034" class="usa-link" aria-describedby="tpg270019-bib-0034">2008</a>). Integrating validated SNP markers allows into MAS pipelines enables breeders to more effectively target traits of interest, pyramid favorable alleles, which can improve both the accuracy and success rate of breeding programs.</p>
<p>This study presents an investigation of molecular diversity by employing the developed panel on a diverse set of groundnut genotypes. The findings from the analysis of 2,573 genotypes provided support for the efficacy of the newly developed SNP assay in identifying genetic variation among genotypes with varied genomic constitutions. The assay facilitated an understanding of the genetic diversity present within various sets of groundnut collections, enabling a comprehensive assessment of genetic diversity indices. This will facilitate the identification of conserved alleles specific to sets of genotypes, as well as with potential implications in breeding and evolutionary research.</p>
<p>This assay, characterized by its cost‐effectiveness and faster data turnover, demonstrates great potential for routine applications as a MDG platform in groundnut research programs such as (a) checking genetic purity among FP of the breeding program, (b) checking genetic purity within and between accessions, including the identification of duplicate germplasm in genebank collections, (c) DNA fingerprinting of breeding material for legal validation, (d) tracing genetic linkage drag from wilds and distant relatives in wide crossing pre‐breeding programs, (e) facilitating background genome recovery in backcross breeding program, (f) enabling genomic prediction based selection strategies in genomic selection breeding, and (g) verifying genetic purity in RV in seed system.</p>
<p>Measures of genetic indices from the AgriSeq MDG assay provided insights into the diversity aspects of diverse sets of groundnut collection. Both the PIC values and the proportion of heterozygosity provide measures of genetic diversity among genotypes. These metrics offer insights into the evolutionary pressure on alleles and the mutation rate experienced by a locus through time. When evaluating the inheritance from parents to offspring by linkage analysis, the PIC values provide a reliable measure of the markers' efficacy (Shete et al., <a href="#tpg270019-bib-0028" class="usa-link" aria-describedby="tpg270019-bib-0028">2000</a>). In contrast, heterozygosity serves as a means to determine genetic distance among individuals within a population, in addition to determining gene diversity for haploid markers. Using the AgriSeq MDG assay, the calculated average PIC values of markers within each group were less than 0.5 (PIC = 0.34–0.37), indicating a moderately informative. This level of PIC value is expected for SNP markers utilized in genotyping. For bi‐allelic SNPs and low mutation rates on SNP loci, the value is constrained to 0.5 (Eltaher et al., <a href="#tpg270019-bib-0012" class="usa-link" aria-describedby="tpg270019-bib-0012">2018</a>). The MAF of each estimated diverse set also showed the difference among common and rare alleles in genotypes, resulting in MAF as an important factor in selection and evolution. The mean MAF from the inferred set comprising ABLs set 1 is 0.062 (relatively low), which means that the common alleles dominate most of the alleles in this set. This observation, however, is in contrast to findings in diverse RS where we observed the mean MAF of 0.216, indicating greater diversity in this set compared to the breeding set.</p>
<p>The advent of various genotyping technologies in recent years has made it simpler to perform SNP genotyping. The developed groundnut AgriSeq T‐GBS mid‐density assay fulfills the requirement for such SNP genotyping in groundnut more readily. This assay has facilitated a precise assessment of the structural complexity and genetic diversity in groundnut, holding importance for germplasm characterization, genomics studies, and marker‐assisted breeding. The diverse sets analyzed in this study, including RSs, FP, RV, TPs, and ABLs from stage 1 and stage 2 trials, contribute valuable insights into genetic variability and SNP profile, particularly for groundnut breeding. Researchers now have the opportunity to leverage these markers for investigating and examining the genetic diversity in germplasm from various origins and explore the genetic foundation of complex traits.</p>
<p>Moreover, two genetic mapping studies utilizing the AgriSeq T‐GBS mid‐density assay have revealed important findings for fresh seed dormancy and iron (Fe) and zinc (Zn) content traits in groundnut. The first study involved a recombinant inbred line ICGV 02266 × ICGV 97045 population, identified five major quantitative trait loci (QTLs) for fresh seed dormancy on five chromosomes (Ah01, Ah11, Ah06, Ah16, and Ah17). These QTLs explained phenotypic variance ranging from 53.2% to 60.4% (QTL Cartographer software) and 69.3% to 74.7% (IciMapping software). Gene mining in the vicinity of these genomic regions highlighted key regulators, like histone deacetylases and ethylene‐responsive transcription factors associated with hormonal regulation of dormancy (Bomireddy et al., <a href="#tpg270019-bib-0005" class="usa-link" aria-describedby="tpg270019-bib-0005">2022</a>). In another study, utilizing a biparental mapping population (ICGV 00440 × ICGV 06040), five major main‐effect QTLs for Fe (explaining phenotypic variance ranging from 22% to 30.0%) and four major main‐effect QTLs for Zn content (explaining from 21.8% to 32.8% phenotypic variance) were identified (Parmar et al., <a href="#tpg270019-bib-0025" class="usa-link" aria-describedby="tpg270019-bib-0025">2023</a>). High‐resolution QTL mapping requires high‐density marker arrays, which is limitation in our mid‐density assay. Due to the limited number of SNPs in assay, adoption in trait mapping would requires a high population size to provide the appropriate statistical accuracy for QTL detection (Wang et al., <a href="#tpg270019-bib-0033" class="usa-link" aria-describedby="tpg270019-bib-0033">2019</a>). This mid‐density assay has notable potential for examining population structure, assessing genetic diversity, and helping for QC in the breeding program. Therefore, the extensive genomic SNP resources and fundamental knowledge generated from this research would establish a robust framework, thereby promoting swift advancements in groundnut genomic resources and facilitating the acceleration of groundnut breeding programs.</p></section><section id="tpg270019-sec-0140"><h2 class="pmc_sec_title">5. SUMMARY</h2>
<p>A cost‐effective high‐throughput MDG assay is essential for large‐scale genomic breeding applications in addition to genetic purity and duplication detection in gene bank and genetic purity among the seed system. The newly developed mid‐density assay with 5,081 SNPs in groundnut provides new opportunities for deployment in these applications on a large scale. Being dynamic, AgriSeq T‐GBS assays can be easily modified by adding informative SNPs in the future to further increase the utilization of this assay in breeding programs globally.</p></section><section id="tpg270019-sec-0150"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Manish K. Pandey</strong>: Conceptualization; data curation; formal analysis; funding acquisition; investigation; resources; writing—original draft; writing—review and editing. <strong>Vinay Sharma</strong>: Data curation; formal analysis; methodology; software; visualization; writing—original draft; writing—review and editing. <strong>Aamir W. Khan</strong>: Data curation; formal analysis; software. <strong>Pushpesh Joshi</strong>: Data curation; methodology. <strong>Sunil S. Gangurde</strong>: Data curation; formal analysis; methodology; software; writing—original draft; writing—review and editing. <strong>Prasad Bajaj</strong>: Formal analysis; methodology. <strong>Pasupuleti Janila</strong>: Investigation, resources. <strong>Annapurna Chitikineni</strong>: Investigation; methodology. <strong>Ramesh Bhat</strong>: Investigation, resources. <strong>Babu N. Motagi</strong>: Investigation, resources. <strong>Chandramohan Sangh</strong>: Investigation; resources. <strong>Thankappan Radhakrishnan</strong>: Investigation; resources; writing—review and editing. <strong>Sandip K. Bera</strong>: Investigation, resources. <strong>Gregor Gorjanc</strong>: Investigation; resources; writing—review and editing. <strong>Krishna Reddy Gujjula</strong>: Data curation, formal analysis. <strong>Nathan Hall</strong>: Data curation; formal analysis. <strong>Claudio D. Carrasco</strong>: Data curation, formal analysis. <strong>Kandalam Arjun</strong>: Data curation; formal analysis. <strong>Srinivas Chandram</strong>: Data curation, formal analysis. <strong>Rajeev K. Varshney</strong>: Conceptualization; investigation; resources; writing—original draft; writing—review and editing.</p></section><section id="tpg270019-sec-0170"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec17"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270019-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270019-supitem-0001"><div class="caption p">
<p>
<strong>Table S1</strong>: List of 722 genotypes used for genotyping using high density SNP array</p>
<p>
<strong>Table S2</strong>: List of 8,027 SNPs selected based on PIC values</p>
<p>
<strong>Table S3</strong>: List of 82 SNPs associated with various traits with allele information and flanking sequences</p>
<p>
<strong>Table S4</strong>: List of 5,081 SNPs with details of chromosome number, position in diploid and tetraploid genome</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11958872/bin/TPG2-18-e70019-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70019-s001.xlsx</a><sup> (2.1MB, xlsx) </sup>
</div></div></section></section><section id="tpg270019-sec-0160" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>The authors would like to acknowledge major funding source Department of Biotechnology (DBT) of Government of India along with minor funding contributions from the Indian Council of Agricultural Research (ICAR), India and the Bill &amp; Melinda Gates Foundation (BMGF), USA.</p></section><section id="notes1"><p>


Pandey, M. K.
, 
Sharma, V.
, 
Khan, A. W.
, 
Joshi, P.
, 
Gangurde, S. S.
, 
Bajaj, P.
, 
Janila, P.
, 
Chitikineni, A.
, 
Bhat, R.
, 
Motagi, B. N.
, 
Sangh, C.
, 
Radhakrishnan, T.
, 
Bera, S. K.
, 
Gorjanc, G.
, 
Gujjula, K. R.
, 
Hall, N.
, 
Carrasco, C. D.
, 
Arjun, K.
, 
Chandram, S.
, &amp; 
Varshney, R. K.
 (2025). Development of a cost‐effective high‐throughput mid‐density 5K genotyping assay for germplasm characterization and breeding in groundnut. The Plant Genome, 18, e70019. 10.1002/tpg2.70019


</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270019-note-0001"><p>Assigned to Associate Editor Caio Vieira.</p></div></div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Manish K. Pandey, Email: Manish.Pandey@icrisat.org.</p>
<p>Rajeev K. Varshney, Email: Rajeev.Varshney@murdoch.edu.au.</p></section><section id="tpg270019-sec-0190"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All the data generated in this study are provided in the supplementary material.</p></section><section id="tpg270019-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270019-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270019-bib-0001">
<cite>

Bera, S. K.
, 
Kamdar, J. H.
, 
Kasundra, S. V.
, 
Dash, P.
, 
Maurya, A. K.
, 
Jasani, M. D.
, 
Chandrashekar, A. B.
, 
Manivannan, N.
, 
Vasanthi, R. P.
, 
Dobariya, K. L.
, 
Pandey, M. K.
, 
Janila, P.
, 
Radhakrishnan, T.
, &amp; 
Varshney, R. K.
 (2018). Improving oil quality by altering levels of fatty acids through marker‐assisted selection of <em>ahfad2</em> alleles in peanut (<em>Arachis hypogaea</em> L.) varieties. Euphytica, 214, Article 162. 10.1007/s10681-018-2241-0
</cite> [<a href="https://doi.org/10.1007/s10681-018-2241-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Improving%20oil%20quality%20by%20altering%20levels%20of%20fatty%20acids%20through%20marker%E2%80%90assisted%20selection%20of%20ahfad2%20alleles%20in%20peanut%20(Arachis%20hypogaea%20L.)%20varieties&amp;author=S.%20K.%20Bera&amp;author=J.%20H.%20Kamdar&amp;author=S.%20V.%20Kasundra&amp;author=P.%20Dash&amp;author=A.%20K.%20Maurya&amp;volume=214&amp;publication_year=2018&amp;pages=162&amp;doi=10.1007/s10681-018-2241-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0002">
<cite>

Bera, S. K.
, 
Kamdar, J. H.
, 
Kasundra, S. V.
, 
Patel, S. V.
, 
Jasani, M. D.
, 
Maurya, A. K.
, 
Dash, P.
, 
Chandrashekar, A. B.
, 
Rani, K.
, 
Manivannan, N.
, 
Janila, P.
, 
Pandey, M. K.
, 
Vasanthi, R. P.
, 
Dobariya, K. L.
, 
Radhakrishnan, T.
, &amp; 
Varshney, R. K.
 (2019). Steady expression of high oleic acid across environments in peanut bred by MABC breeding for fatty acid desaturase mutant alleles (<em>ahfad2</em>) and effect of high oleic acid content in seed germination and other seedling traits. PLoS One, 14(12), e0226252. 10.1371/journal.pone.0226252

</cite> [<a href="https://doi.org/10.1371/journal.pone.0226252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6910123/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31830093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Steady%20expression%20of%20high%20oleic%20acid%20across%20environments%20in%20peanut%20bred%20by%20MABC%20breeding%20for%20fatty%20acid%20desaturase%20mutant%20alleles%20(ahfad2)%20and%20effect%20of%20high%20oleic%20acid%20content%20in%20seed%20germination%20and%20other%20seedling%20traits&amp;author=S.%20K.%20Bera&amp;author=J.%20H.%20Kamdar&amp;author=S.%20V.%20Kasundra&amp;author=S.%20V.%20Patel&amp;author=M.%20D.%20Jasani&amp;volume=14&amp;issue=12&amp;publication_year=2019&amp;pages=e0226252&amp;pmid=31830093&amp;doi=10.1371/journal.pone.0226252&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0003">
<cite>

Bertioli, D. J.
, 
Cannon, S. B.
, 
Froenicke, L.
, 
Huang, G.
, 
Farmer, A. D.
, 
Cannon, E. K. S.
, 
Liu, X.
, 
Gao, D.
, 
Clevenger, J.
, 
Dash, S.
, 
Ren, L.
, 
Moretzsohn, M. C.
, 
Shirasawa, K.
, 
Huang, W.
, 
Vidigal, B.
, 
Abernathy, B.
, 
Chu, Y. E.
, 
Niederhuth, C. E.
, 
Umale, P.
, … 
Ozias‐Akins, P.
 (2016). The genome sequences of <em>Arachis duranensis</em> and <em>Arachis ipaensis</em>, the diploid ancestors of cultivated peanut. Nature Genetics, 48, 438–446. 10.1038/ng.3517

</cite> [<a href="https://doi.org/10.1038/ng.3517" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26901068/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=The%20genome%20sequences%20of%20Arachis%20duranensis%20and%20Arachis%20ipaensis,%20the%20diploid%20ancestors%20of%20cultivated%20peanut&amp;author=D.%20J.%20Bertioli&amp;author=S.%20B.%20Cannon&amp;author=L.%20Froenicke&amp;author=G.%20Huang&amp;author=A.%20D.%20Farmer&amp;volume=48&amp;publication_year=2016&amp;pages=438-446&amp;pmid=26901068&amp;doi=10.1038/ng.3517&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0004">
<cite>

Bertioli, D. J.
, 
Jenkins, J.
, 
Clevenger, J.
, 
Dudchenko, O.
, 
Gao, D.
, 
Seijo, G.
, 
Leal‐Bertioli, S. C. M.
, 
Ren, L.
, 
Farmer, A. D.
, 
Pandey, M. K.
, 
Samoluk, S. S.
, 
Abernathy, B.
, 
Agarwal, G.
, 
Ballén‐Taborda, C.
, 
Cameron, C.
, 
Campbell, J.
, 
Chavarro, C.
, 
Chitikineni, A.
, 
Chu, Y. E.
, … 
Schmutz, J.
 (2019). The genome sequence of peanut (<em>Arachis hypogaea</em>), a segmental allotetraploid. Nature Genetics, 51, 877–884. 10.1038/s41588-019-0405-z

</cite> [<a href="https://doi.org/10.1038/s41588-019-0405-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31043755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=The%20genome%20sequence%20of%20peanut%20(Arachis%20hypogaea),%20a%20segmental%20allotetraploid&amp;author=D.%20J.%20Bertioli&amp;author=J.%20Jenkins&amp;author=J.%20Clevenger&amp;author=O.%20Dudchenko&amp;author=D.%20Gao&amp;volume=51&amp;publication_year=2019&amp;pages=877-884&amp;pmid=31043755&amp;doi=10.1038/s41588-019-0405-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0005">
<cite>

Bomireddy, D.
, 
Gangurde, S. S.
, 
Variath, M. T.
, 
Janila, P.
, 
Manohar, S. S.
, 
Sharma, V.
, 
Parmar, S.
, 
Deshmukh, D.
, 
Reddisekhar, M.
, 
Reddy, D. M.
, 
Sudhakar, P.
, 
Reddy, B. V. B.
, 
Varshney, R. K.
, 
Guo, B.
, &amp; 
Pandey, M. K.
 (2022). Discovery of major quantitative trait loci and candidate genes for fresh seed dormancy in groundnut. Agronomy, 12(2), 404. 10.3390/agronomy12020404
</cite> [<a href="https://doi.org/10.3390/agronomy12020404" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=Discovery%20of%20major%20quantitative%20trait%20loci%20and%20candidate%20genes%20for%20fresh%20seed%20dormancy%20in%20groundnut&amp;author=D.%20Bomireddy&amp;author=S.%20S.%20Gangurde&amp;author=M.%20T.%20Variath&amp;author=P.%20Janila&amp;author=S.%20S.%20Manohar&amp;volume=12&amp;issue=2&amp;publication_year=2022&amp;pages=404&amp;doi=10.3390/agronomy12020404&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0006">
<cite>

Bradbury, P. J.
, 
Zhang, Z.
, 
Kroon, D. E.
, 
Casstevens, T. M.
, 
Ramdoss, Y.
, &amp; 
Buckler, E. S.
 (2007). TASSEL: Software for association mapping of complex traits in diverse samples. Bioinformatics, 23(19), 2633–2635. 10.1093/bioinformatics/btm308

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btm308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17586829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=TASSEL:%20Software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples&amp;author=P.%20J.%20Bradbury&amp;author=Z.%20Zhang&amp;author=D.%20E.%20Kroon&amp;author=T.%20M.%20Casstevens&amp;author=Y.%20Ramdoss&amp;volume=23&amp;issue=19&amp;publication_year=2007&amp;pages=2633-2635&amp;pmid=17586829&amp;doi=10.1093/bioinformatics/btm308&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0007">
<cite>

Chen, H.
, 
He, H.
, 
Zou, Y.
, 
Chen, W.
, 
Yu, R.
, 
Liu, X.
, 
Yang, Y.
, 
Gao, Y.‐M.
, 
Xu, J.‐L.
, 
Fan, L.‐M.
, 
Li, Y. I.
, 
Li, Z.‐K.
, &amp; 
Deng, X. W.
 (2011). Development and application of a set of breeder‐friendly SNP markers for genetic analyses and molecular breeding of rice (<em>Oryza sativa</em> L.). Theoretical and Applied Genetics, 123, 869–879. 10.1007/s00122-011-1633-5

</cite> [<a href="https://doi.org/10.1007/s00122-011-1633-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21681488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Development%20and%20application%20of%20a%20set%20of%20breeder%E2%80%90friendly%20SNP%20markers%20for%20genetic%20analyses%20and%20molecular%20breeding%20of%20rice%20(Oryza%20sativa%20L.)&amp;author=H.%20Chen&amp;author=H.%20He&amp;author=Y.%20Zou&amp;author=W.%20Chen&amp;author=R.%20Yu&amp;volume=123&amp;publication_year=2011&amp;pages=869-879&amp;pmid=21681488&amp;doi=10.1007/s00122-011-1633-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0008">
<cite>

Chen, X.
, 
Li, H.
, 
Pandey, M. K.
, 
Yang, Q.
, 
Wang, X.
, 
Garg, V.
, 
Li, H.
, 
Chi, X.
, 
Doddamani, D.
, 
Hong, Y.
, 
Upadhyaya, H.
, 
Guo, H.
, 
Khan, A. W.
, 
Zhu, F.
, 
Zhang, X.
, 
Pan, L.
, 
Pierce, G. J.
, 
Zhou, G.
, 
Krishnamohan, K. A. V. S.
, … 
Yu, S.
 (2016). Draft genome of the peanut A‐genome progenitor (<em>Arachis duranensis</em>) provides insights into geocarpy, oil biosynthesis and allergens. Proceedings of the National Academy of Sciences USA, 113, 6785–6790. 10.1073/pnas.1600899113
</cite> [<a href="https://doi.org/10.1073/pnas.1600899113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4914189/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27247390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20USA&amp;title=Draft%20genome%20of%20the%20peanut%20A%E2%80%90genome%20progenitor%20(Arachis%20duranensis)%20provides%20insights%20into%20geocarpy,%20oil%20biosynthesis%20and%20allergens&amp;author=X.%20Chen&amp;author=H.%20Li&amp;author=M.%20K.%20Pandey&amp;author=Q.%20Yang&amp;author=X.%20Wang&amp;volume=113&amp;publication_year=2016&amp;pages=6785-6790&amp;pmid=27247390&amp;doi=10.1073/pnas.1600899113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0009">
<cite>

Chen, X.
, 
Lu, Q.
, 
Liu, H.
, 
Zhang, J.
, 
Hong, Y.
, 
Lan, H.
, 
Li, H.
, 
Wang, J.
, 
Liu, H.
, 
Li, H.
, 
Pandey, M. K.
, 
Zhang, Z.
, 
Zhou, G.
, 
Yu, J.
, 
Zhang, G.
, 
Yuan, J.
, 
Li, X.
, 
Wen, S.
, 
Meng, F.
, … 
Liang, X.
 (2019). Sequencing of cultivated peanut, <em>Arachis hypogaea</em>, yields insights into genome evolution and oil improvement. Molecular Plant, 12(7), 920–934. 10.1016/j.molp.2019.03.005

</cite> [<a href="https://doi.org/10.1016/j.molp.2019.03.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30902685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=Sequencing%20of%20cultivated%20peanut,%20Arachis%20hypogaea,%20yields%20insights%20into%20genome%20evolution%20and%20oil%20improvement&amp;author=X.%20Chen&amp;author=Q.%20Lu&amp;author=H.%20Liu&amp;author=J.%20Zhang&amp;author=Y.%20Hong&amp;volume=12&amp;issue=7&amp;publication_year=2019&amp;pages=920-934&amp;pmid=30902685&amp;doi=10.1016/j.molp.2019.03.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0010">
<cite>

Clevenger, J.
, 
Chu, Y.
, 
Chavarro, C.
, 
Agarwal, G.
, 
Bertioli, D. J.
, 
Leal‐Bertioli, S. C. M.
, 
Pandey, M. K.
, 
Vaughn, J.
, 
Abernathy, B.
, 
Barkley, N.
, 
Hovav, R.
, 
Burow, M.
, 
Nayak, S. N.
, 
Chitikineni, A.
, 
Isleib, T.
, 
Holbrook, C.
, 
Jackson, S. A.
, 
Varshney, R. K.
, &amp; 
Ozias‐Akins, P.
 (2017). Genome‐wide SNP genotyping resolves signatures of selection and tetrasomic recombination in peanut. Molecular Plant, 10, 309–322. 10.1016/j.molp.2016.11.015

</cite> [<a href="https://doi.org/10.1016/j.molp.2016.11.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5315502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27993622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=Genome%E2%80%90wide%20SNP%20genotyping%20resolves%20signatures%20of%20selection%20and%20tetrasomic%20recombination%20in%20peanut&amp;author=J.%20Clevenger&amp;author=Y.%20Chu&amp;author=C.%20Chavarro&amp;author=G.%20Agarwal&amp;author=D.%20J.%20Bertioli&amp;volume=10&amp;publication_year=2017&amp;pages=309-322&amp;pmid=27993622&amp;doi=10.1016/j.molp.2016.11.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0011">
<cite>

Dodia, S. M.
, 
Joshi, B.
, 
Gangurde, S. S.
, 
Thirumalaisamy, P. P.
, 
Mishra, G. P.
, 
Narandrakumar, D.
, 
Soni, P.
, 
Rathnakumar, A. L.
, 
Dobaria, J. R.
, 
Sangh, C.
, 
Chitikineni, A.
, 
Chanda, S. V.
, 
Pandey, M. K.
, 
Varshney, R. K.
, &amp; 
Thankappan, R.
 (2019). Genotyping‐by‐sequencing based genetic mapping reveals large number of epistatic interactions for stem rot resistance in groundnut. Theoretical and Applied Genetics, 132(4), 1001–1016. 10.1007/s00122-018-3255-7

</cite> [<a href="https://doi.org/10.1007/s00122-018-3255-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30539317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genotyping%E2%80%90by%E2%80%90sequencing%20based%20genetic%20mapping%20reveals%20large%20number%20of%20epistatic%20interactions%20for%20stem%20rot%20resistance%20in%20groundnut&amp;author=S.%20M.%20Dodia&amp;author=B.%20Joshi&amp;author=S.%20S.%20Gangurde&amp;author=P.%20P.%20Thirumalaisamy&amp;author=G.%20P.%20Mishra&amp;volume=132&amp;issue=4&amp;publication_year=2019&amp;pages=1001-1016&amp;pmid=30539317&amp;doi=10.1007/s00122-018-3255-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0012">
<cite>

Eltaher, S.
, 
Sallam, A.
, 
Belamkar, V.
, 
Emara, H. A.
, 
Nower, A. A.
, 
Salem, K. F.
, 
Poland, J.
, &amp; 
Baenziger, P. S.
 (2018). Genetic diversity and population structure of F<sub>3:6</sub> Nebraska winter wheat genotypes using genotyping‐by‐sequencing. Frontiers in Genetics, 9, 76. 10.3389/fgene.2018.00076

</cite> [<a href="https://doi.org/10.3389/fgene.2018.00076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5857551/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29593779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Genetic%20diversity%20and%20population%20structure%20of%20F3:6%20Nebraska%20winter%20wheat%20genotypes%20using%20genotyping%E2%80%90by%E2%80%90sequencing&amp;author=S.%20Eltaher&amp;author=A.%20Sallam&amp;author=V.%20Belamkar&amp;author=H.%20A.%20Emara&amp;author=A.%20A.%20Nower&amp;volume=9&amp;publication_year=2018&amp;pages=76&amp;pmid=29593779&amp;doi=10.3389/fgene.2018.00076&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0013">
<cite>

García, C.
, 
Guichoux, E.
, &amp; 
Hampe, A.
 (2018). A comparative analysis between SNPs and SSRs to investigate genetic variation in a juniper species (<em>Juniperus phoenicea</em> ssp. turbinata). Tree Genetics Genomics, 14, Article 87. 10.1007/s11295-018-1301-x
</cite> [<a href="https://doi.org/10.1007/s11295-018-1301-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Tree%20Genetics%20Genomics&amp;title=A%20comparative%20analysis%20between%20SNPs%20and%20SSRs%20to%20investigate%20genetic%20variation%20in%20a%20juniper%20species%20(Juniperus%20phoenicea%20ssp.%20turbinata)&amp;author=C.%20Garc%C3%ADa&amp;author=E.%20Guichoux&amp;author=A.%20Hampe&amp;volume=14&amp;publication_year=2018&amp;pages=87&amp;doi=10.1007/s11295-018-1301-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0014">
<cite>

Heffner, E. L.
, 
Sorrells, M. E.
, &amp; 
Jannink, J. L.
 (2009). Genomic selection for crop improvement. Crop Science, 49, 1–12. 10.2135/cropsci2008.08.0512
</cite> [<a href="https://doi.org/10.2135/cropsci2008.08.0512" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genomic%20selection%20for%20crop%20improvement&amp;author=E.%20L.%20Heffner&amp;author=M.%20E.%20Sorrells&amp;author=J.%20L.%20Jannink&amp;volume=49&amp;publication_year=2009&amp;pages=1-12&amp;doi=10.2135/cropsci2008.08.0512&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0015">
<cite>

Janila, P.
, 
Pandey, M. K.
, 
Shasidhar, Y.
, 
Variath, M. T.
, 
Sriswathi, M.
, 
Khera, P.
, 
Manohar, S. S.
, 
Nagesh, P.
, 
Vishwakarma, M. K.
, 
Mishra, G. P.
, 
Radhakrishnan, T.
, 
Manivannan, N.
, 
Dobariya, K. L.
, 
Vasanthi, R. P.
, &amp; 
Varshney, R. K.
 (2016). Molecular breeding for introgression of fatty acid desaturase mutant alleles (<em>ahFAD2A</em> and <em>ahFAD2B</em>) enhances oil quality in high and low oil containing peanut genotypes. Plant Science, 242, 203–213. 10.1016/j.plantsci.2015.08.013

</cite> [<a href="https://doi.org/10.1016/j.plantsci.2015.08.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26566838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Science&amp;title=Molecular%20breeding%20for%20introgression%20of%20fatty%20acid%20desaturase%20mutant%20alleles%20(ahFAD2A%20and%20ahFAD2B)%20enhances%20oil%20quality%20in%20high%20and%20low%20oil%20containing%20peanut%20genotypes&amp;author=P.%20Janila&amp;author=M.%20K.%20Pandey&amp;author=Y.%20Shasidhar&amp;author=M.%20T.%20Variath&amp;author=M.%20Sriswathi&amp;volume=242&amp;publication_year=2016&amp;pages=203-213&amp;pmid=26566838&amp;doi=10.1016/j.plantsci.2015.08.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0016">
<cite>

Kolekar, R. M.
, 
Sukruth, M.
, 
Shirasawa, K.
, 
Nadaf, H. L.
, 
Motagi, B. N.
, 
Lingaraju, S.
, 
Patil, P. V.
, &amp; 
Bhat, R. S.
 (2017). Marker‐assisted backcrossing to develop foliar disease‐resistant genotypes in TMV 2 variety of peanut (<em>Arachis hypogaea</em> L.). Plant Breeding, 136(6), 948–953. 10.1111/pbr.12549
</cite> [<a href="https://doi.org/10.1111/pbr.12549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Breeding&amp;title=Marker%E2%80%90assisted%20backcrossing%20to%20develop%20foliar%20disease%E2%80%90resistant%20genotypes%20in%20TMV%202%20variety%20of%20peanut%20(Arachis%20hypogaea%20L.)&amp;author=R.%20M.%20Kolekar&amp;author=M.%20Sukruth&amp;author=K.%20Shirasawa&amp;author=H.%20L.%20Nadaf&amp;author=B.%20N.%20Motagi&amp;volume=136&amp;issue=6&amp;publication_year=2017&amp;pages=948-953&amp;doi=10.1111/pbr.12549&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0018">
<cite>

Luo, H.
, 
Pandey, M. K.
, 
Khan, A. W.
, 
Wu, B.
, 
Guo, J.
, 
Ren, X.
, 
Zhou, X.
, 
Chen, Y.
, 
Chen, W.
, 
Huang, L. I.
, 
Liu, N.
, 
Lei, Y.
, 
Liao, B.
, 
Varshney, R. K.
, &amp; 
Jiang, H.
 (2019). Next‐generation sequencing identified genomic region and diagnostic markers for resistance to bacterial wilt on chromosome B02 in peanut (<em>Arachis hypogaea</em> L.). Plant Biotechnology Journal, 17, 2356–2369. 10.1111/pbi.13153

</cite> [<a href="https://doi.org/10.1111/pbi.13153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6835129/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31087470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=Next%E2%80%90generation%20sequencing%20identified%20genomic%20region%20and%20diagnostic%20markers%20for%20resistance%20to%20bacterial%20wilt%20on%20chromosome%20B02%20in%20peanut%20(Arachis%20hypogaea%20L.)&amp;author=H.%20Luo&amp;author=M.%20K.%20Pandey&amp;author=A.%20W.%20Khan&amp;author=B.%20Wu&amp;author=J.%20Guo&amp;volume=17&amp;publication_year=2019&amp;pages=2356-2369&amp;pmid=31087470&amp;doi=10.1111/pbi.13153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0019">
<cite>

Mbanjo, E. G. N.
, 
Ogungbesan, A.
, 
Agbona, A.
, 
Akpotuzor, P.
, 
Toyinbo, S.
, 
Iluebbey, P.
, 
Rabbi, I. Y.
, 
Peteti, P.
, 
Wages, S. A.
, 
Norton, J.
, 
Zhang, X.
, 
Bohórquez‐Chaux, A.
, 
Mushoriwa, H.
, 
Egesi, C.
, 
Kulakow, P.
, &amp; 
Parkes, E.
 (2024). Validation of SNP markers for diversity analysis, quality control, and trait selection in a biofortified cassava population. Plants, 13, 2328. 10.3390/plants13162328

</cite> [<a href="https://doi.org/10.3390/plants13162328" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11359368/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39204764/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Validation%20of%20SNP%20markers%20for%20diversity%20analysis,%20quality%20control,%20and%20trait%20selection%20in%20a%20biofortified%20cassava%20population&amp;author=E.%20G.%20N.%20Mbanjo&amp;author=A.%20Ogungbesan&amp;author=A.%20Agbona&amp;author=P.%20Akpotuzor&amp;author=S.%20Toyinbo&amp;volume=13&amp;publication_year=2024&amp;pages=2328&amp;pmid=39204764&amp;doi=10.3390/plants13162328&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0020">
<cite>

Morales, K. Y.
, 
Singh, N.
, 
Perez, F. A.
, 
Ignacio, J. C.
, 
Thapa, R.
, 
Arbelaez, J. D.
, 
Tabien, R. E.
, 
Famoso, A.
, 
Wang, D. R.
, 
Septiningsih, E. M.
, 
Shi, Y.
, 
Kretzschmar, T.
, 
Mccouch, S. R.
, &amp; 
Thomson, M. J.
 (2020). An improved 7K SNP array, the C7AIR, provides a wealth of validated SNP markers for rice breeding and genetics studies. PLoS One, 15(5), e0232479. 10.1371/journal.pone.0232479

</cite> [<a href="https://doi.org/10.1371/journal.pone.0232479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7224494/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32407369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=An%20improved%207K%20SNP%20array,%20the%20C7AIR,%20provides%20a%20wealth%20of%20validated%20SNP%20markers%20for%20rice%20breeding%20and%20genetics%20studies&amp;author=K.%20Y.%20Morales&amp;author=N.%20Singh&amp;author=F.%20A.%20Perez&amp;author=J.%20C.%20Ignacio&amp;author=R.%20Thapa&amp;volume=15&amp;issue=5&amp;publication_year=2020&amp;pages=e0232479&amp;pmid=32407369&amp;doi=10.1371/journal.pone.0232479&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0021">
<cite>

Pandey, M. K.
, 
Agarwal, G.
, 
Kale, S. M.
, 
Clevenger, J.
, 
Nayak, S. N.
, 
Sriswathi, M.
, 
Chitikineni, A.
, 
Chavarro, C.
, 
Chen, X.
, 
Upadhyaya, H. D.
, 
Vishwakarma, M. K.
, 
Leal‐Bertioli, S.
, 
Liang, X.
, 
Bertioli, D. J.
, 
Guo, B.
, 
Jackson, S. A.
, 
Ozias‐Akins, P.
, &amp; 
Varshney, R. K.
 (2017a). Development and evaluation of a high density genotyping ‘Axiom_<em>Arachis</em>’ array with 58 K SNPs for accelerating genetics and breeding in groundnut. Scientific Reports, 7, Article 40577. 10.1038/srep40577

</cite> [<a href="https://doi.org/10.1038/srep40577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5238394/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28091575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Development%20and%20evaluation%20of%20a%20high%20density%20genotyping%20%E2%80%98Axiom_Arachis%E2%80%99%20array%20with%2058%20K%20SNPs%20for%20accelerating%20genetics%20and%20breeding%20in%20groundnut&amp;author=M.%20K.%20Pandey&amp;author=G.%20Agarwal&amp;author=S.%20M.%20Kale&amp;author=J.%20Clevenger&amp;author=S.%20N.%20Nayak&amp;volume=7&amp;publication_year=2017a&amp;pages=40577&amp;pmid=28091575&amp;doi=10.1038/srep40577&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0022">
<cite>

Pandey, M. K.
, 
Chaudhari, S.
, 
Jarquin, D.
, 
Janila, P.
, 
Crossa, J.
, 
Patil, S. C.
, 
Sundravadana, S.
, 
Khare, D.
, 
Bhat, R. S.
, 
Radhakrishnan, T.
, 
Hickey, J. M.
, &amp; 
Varshney, R. K.
 (2020). Genome‐based trait prediction in multi‐environment breeding trials in groundnut. Theoretical and Applied Genetics, 133, 3101–3117. 10.1007/s00122-020-03658-1

</cite> [<a href="https://doi.org/10.1007/s00122-020-03658-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7547976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32809035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genome%E2%80%90based%20trait%20prediction%20in%20multi%E2%80%90environment%20breeding%20trials%20in%20groundnut&amp;author=M.%20K.%20Pandey&amp;author=S.%20Chaudhari&amp;author=D.%20Jarquin&amp;author=P.%20Janila&amp;author=J.%20Crossa&amp;volume=133&amp;publication_year=2020&amp;pages=3101-3117&amp;pmid=32809035&amp;doi=10.1007/s00122-020-03658-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0023">
<cite>

Pandey, M. K.
, 
Gangurde, S. S.
, 
Sharma, V.
, 
Pattanashetti, S. K.
, 
Naidu, G. K.
, 
Faye, I.
, 
Hamidou, F.
, 
Desmae, H.
, 
Kane, N. A.
, 
Yuan, M.
, 
Vadez, V.
, 
Nigam, S. N.
, &amp; 
Varshney, R. K.
 (2020). Improved genetic map identified major QTLs for drought tolerance‐and iron deficiency tolerance‐related traits in groundnut. Genes, 12(1), 37. 10.3390/genes12010037

</cite> [<a href="https://doi.org/10.3390/genes12010037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7824586/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33396649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes&amp;title=Improved%20genetic%20map%20identified%20major%20QTLs%20for%20drought%20tolerance%E2%80%90and%20iron%20deficiency%20tolerance%E2%80%90related%20traits%20in%20groundnut&amp;author=M.%20K.%20Pandey&amp;author=S.%20S.%20Gangurde&amp;author=V.%20Sharma&amp;author=S.%20K.%20Pattanashetti&amp;author=G.%20K.%20Naidu&amp;volume=12&amp;issue=1&amp;publication_year=2020&amp;pages=37&amp;pmid=33396649&amp;doi=10.3390/genes12010037&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0024">
<cite>

Pandey, M. K.
, 
Gangurde, S. S.
, 
Shasidhar, Y.
, 
Sharma, V.
, 
Kale, S. M.
, 
Khan, A. W.
, 
Shah, P.
, 
Joshi, P.
, 
Bhat, R. S.
, 
Janila, P.
, 
Bera, S. K.
, &amp; 
Varshney, R. K.
 (2024). High‐throughput diagnostic markers for foliar fungal disease resistance and high oleic acid content in groundnut. BMC Plant Biology, 24, Article 262. 10.1186/s12870-024-04987-9

</cite> [<a href="https://doi.org/10.1186/s12870-024-04987-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11005153/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38594614/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=High%E2%80%90throughput%20diagnostic%20markers%20for%20foliar%20fungal%20disease%20resistance%20and%20high%20oleic%20acid%20content%20in%20groundnut&amp;author=M.%20K.%20Pandey&amp;author=S.%20S.%20Gangurde&amp;author=Y.%20Shasidhar&amp;author=V.%20Sharma&amp;author=S.%20M.%20Kale&amp;volume=24&amp;publication_year=2024&amp;pages=262&amp;pmid=38594614&amp;doi=10.1186/s12870-024-04987-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0025">
<cite>

Parmar, S.
, 
Janila, P.
, 
Gangurde, S. S.
, 
Variath, M. T.
, 
Sharma, V.
, 
Bomireddy, D.
, 
Manohar, S. S.
, 
Varshney, R. K.
, 
Singam, P.
, &amp; 
Pandey, M. K.
 (2023). Genetic mapping identified major main‐effect and three co‐localized quantitative trait loci controlling high iron and zinc content in groundnut. The Plant Genome, 5, e20361. 10.1002/tpg2.20361
</cite> [<a href="https://doi.org/10.1002/tpg2.20361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37408143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genetic%20mapping%20identified%20major%20main%E2%80%90effect%20and%20three%20co%E2%80%90localized%20quantitative%20trait%20loci%20controlling%20high%20iron%20and%20zinc%20content%20in%20groundnut&amp;author=S.%20Parmar&amp;author=P.%20Janila&amp;author=S.%20S.%20Gangurde&amp;author=M.%20T.%20Variath&amp;author=V.%20Sharma&amp;volume=5&amp;publication_year=2023&amp;pages=e20361&amp;pmid=37408143&amp;doi=10.1002/tpg2.20361&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0026">
<cite>

Semalaiyappan, J.
, 
Selvanayagam, S.
, 
Rathore, A.
, 
Gupta, S. K.
, 
Chakraborty, A.
, 
Gujjula, K. R.
, 
Haktan, S.
, 
Viswanath, A.
, 
Malipatil, R.
, 
Shah, P.
, 
Govindaraj, M.
, 
Ignacio, J. C.
, 
Reddy, S.
, 
Singh, A. K.
, &amp; 
Thirunavukkarasu, N.
 (2023). Development of a new AgriSeq 4K mid‐density SNP genotyping panel and its utility in pearl millet breeding. Frontiers in Plant Science, 13, 1068883. 10.3389/fpls.2022.1068883

</cite> [<a href="https://doi.org/10.3389/fpls.2022.1068883" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9871632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36704175/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Development%20of%20a%20new%20AgriSeq%204K%20mid%E2%80%90density%20SNP%20genotyping%20panel%20and%20its%20utility%20in%20pearl%20millet%20breeding&amp;author=J.%20Semalaiyappan&amp;author=S.%20Selvanayagam&amp;author=A.%20Rathore&amp;author=S.%20K.%20Gupta&amp;author=A.%20Chakraborty&amp;volume=13&amp;publication_year=2023&amp;pages=1068883&amp;pmid=36704175&amp;doi=10.3389/fpls.2022.1068883&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0027">
<cite>

Shasidhar, Y.
, 
Variath, M. T.
, 
Vishwakarma, M. K.
, 
Manohar, S. S.
, 
Gangurde, S. S.
, 
Sriswathi, M.
, 
Sudini, H. K.
, 
Dobariya, K. L.
, 
Bera, S. K.
, 
Radhakrishnan, T.
, 
Pandey, M. K.
, 
Janila, P.
, &amp; 
Varshney, R. K.
 (2020). Improvement of three Indian popular groundnut varieties for foliar disease resistance and high oleic acid using SSR markers and SNP array in marker‐assisted backcrossing. Crop Journal, 8, 1–15. 10.1016/j.cj.2019.07.001
</cite> [<a href="https://doi.org/10.1016/j.cj.2019.07.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Journal&amp;title=Improvement%20of%20three%20Indian%20popular%20groundnut%20varieties%20for%20foliar%20disease%20resistance%20and%20high%20oleic%20acid%20using%20SSR%20markers%20and%20SNP%20array%20in%20marker%E2%80%90assisted%20backcrossing&amp;author=Y.%20Shasidhar&amp;author=M.%20T.%20Variath&amp;author=M.%20K.%20Vishwakarma&amp;author=S.%20S.%20Manohar&amp;author=S.%20S.%20Gangurde&amp;volume=8&amp;publication_year=2020&amp;pages=1-15&amp;doi=10.1016/j.cj.2019.07.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0028">
<cite>

Shete, S.
, 
Tiwari, H.
, &amp; 
Elston, R. C.
 (2000). On estimating the heterozygosity and polymorphism information content value. Theoretical Population Biology, 57(3), 265–271. 10.1006/tpbi.2000.1452

</cite> [<a href="https://doi.org/10.1006/tpbi.2000.1452" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10828218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20Population%20Biology&amp;title=On%20estimating%20the%20heterozygosity%20and%20polymorphism%20information%20content%20value&amp;author=S.%20Shete&amp;author=H.%20Tiwari&amp;author=R.%20C.%20Elston&amp;volume=57&amp;issue=3&amp;publication_year=2000&amp;pages=265-271&amp;pmid=10828218&amp;doi=10.1006/tpbi.2000.1452&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0029">
<cite>

Singh, N.
, 
Choudhury, D. R.
, 
Singh, A. K.
, 
Kumar, S.
, 
Srinivasan, K.
, 
Tyagi, R. K.
, 
Singh, N. K.
, &amp; 
Singh, R.
 (2013). Comparison of SSR and SNP markers in estimation of genetic diversity and population structure of Indian rice varieties. PLoS One, 8, e84136. 10.1371/journal.pone.0084136

</cite> [<a href="https://doi.org/10.1371/journal.pone.0084136" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3868579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24367635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Comparison%20of%20SSR%20and%20SNP%20markers%20in%20estimation%20of%20genetic%20diversity%20and%20population%20structure%20of%20Indian%20rice%20varieties&amp;author=N.%20Singh&amp;author=D.%20R.%20Choudhury&amp;author=A.%20K.%20Singh&amp;author=S.%20Kumar&amp;author=K.%20Srinivasan&amp;volume=8&amp;publication_year=2013&amp;pages=e84136&amp;pmid=24367635&amp;doi=10.1371/journal.pone.0084136&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0030">
<cite>

Sun, C.
, 
Dong, Z.
, 
Zhao, L.
, 
Ren, Y.
, 
Zhang, N.
, &amp; 
Chen, F.
 (2020). The Wheat 660K SNP array demonstrates great potential for marker‐assisted selection in polyploid wheat. Plant Biotechnology Journal, 18(6), 1354–1360. 10.1111/pbi.13361

</cite> [<a href="https://doi.org/10.1111/pbi.13361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7206996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32065714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=The%20Wheat%20660K%20SNP%20array%20demonstrates%20great%20potential%20for%20marker%E2%80%90assisted%20selection%20in%20polyploid%20wheat&amp;author=C.%20Sun&amp;author=Z.%20Dong&amp;author=L.%20Zhao&amp;author=Y.%20Ren&amp;author=N.%20Zhang&amp;volume=18&amp;issue=6&amp;publication_year=2020&amp;pages=1354-1360&amp;pmid=32065714&amp;doi=10.1111/pbi.13361&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0031">
<cite>

Tian, H.
, 
Yang, Y.
, 
Yi, H.
, 
Xu, L.
, 
He, H.
, 
Fan, Y.
, 
Wang, L.
, 
Ge, J.
, 
Liu, Y.
, 
Wang, F.
, &amp; 
Zhao, J.
 (2021). New resources for genetic studies in maize (<em>Zea mays</em> L.): A genome‐wide Maize6H‐60K single nucleotide polymorphism array and its application. The Plant Journal, 105(4), 1113–1122. 10.1111/tpj.15089

</cite> [<a href="https://doi.org/10.1111/tpj.15089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33225500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=New%20resources%20for%20genetic%20studies%20in%20maize%20(Zea%20mays%20L.):%20A%20genome%E2%80%90wide%20Maize6H%E2%80%9060K%20single%20nucleotide%20polymorphism%20array%20and%20its%20application&amp;author=H.%20Tian&amp;author=Y.%20Yang&amp;author=H.%20Yi&amp;author=L.%20Xu&amp;author=H.%20He&amp;volume=105&amp;issue=4&amp;publication_year=2021&amp;pages=1113-1122&amp;pmid=33225500&amp;doi=10.1111/tpj.15089&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0032">
<cite>

Varshney, R. K.
, 
Pandey, M. K.
, 
Janila, P.
, 
Nigam, S. N.
, 
Sudini, H.
, 
Gowda, M. V. C.
, 
Sriswathi, M.
, 
Radhakrishnan, T.
, 
Manohar, S. S.
, &amp; 
Nagesh, P.
 (2014). Marker‐assisted introgression of a QTL region to improve rust resistance in three elite and popular varieties of peanut (<em>Arachis hypogaea</em> L.). Theoretical and Applied Genetics, 127(8), 1771–1781. 10.1007/s00122-014-2338-3

</cite> [<a href="https://doi.org/10.1007/s00122-014-2338-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4110420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24927821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Marker%E2%80%90assisted%20introgression%20of%20a%20QTL%20region%20to%20improve%20rust%20resistance%20in%20three%20elite%20and%20popular%20varieties%20of%20peanut%20(Arachis%20hypogaea%20L.)&amp;author=R.%20K.%20Varshney&amp;author=M.%20K.%20Pandey&amp;author=P.%20Janila&amp;author=S.%20N.%20Nigam&amp;author=H.%20Sudini&amp;volume=127&amp;issue=8&amp;publication_year=2014&amp;pages=1771-1781&amp;pmid=24927821&amp;doi=10.1007/s00122-014-2338-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0033">
<cite>

Wang, M.
, &amp; 
Xu, S.
 (2019). Statistical power in genome‐wide association studies and quantitative trait locus mapping. Heredity, 123, 287–306. 10.1038/s41437-019-0205-3

</cite> [<a href="https://doi.org/10.1038/s41437-019-0205-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6781134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30858595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Statistical%20power%20in%20genome%E2%80%90wide%20association%20studies%20and%20quantitative%20trait%20locus%20mapping&amp;author=M.%20Wang&amp;author=S.%20Xu&amp;volume=123&amp;publication_year=2019&amp;pages=287-306&amp;pmid=30858595&amp;doi=10.1038/s41437-019-0205-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0034">
<cite>

Xu, Y.
, &amp; 
Crouch, J. H.
 (2008). Marker‐assisted selection in plant breeding: From publications to practice. Crop Science, 48, 391–407. 10.2135/cropsci2007.04.0191
</cite> [<a href="https://doi.org/10.2135/cropsci2007.04.0191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Marker%E2%80%90assisted%20selection%20in%20plant%20breeding:%20From%20publications%20to%20practice&amp;author=Y.%20Xu&amp;author=J.%20H.%20Crouch&amp;volume=48&amp;publication_year=2008&amp;pages=391-407&amp;doi=10.2135/cropsci2007.04.0191&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0035">
<cite>

Yeri, S. B.
, &amp; 
Bhat, R. S.
 (2016). Development of late leaf spot and rust resistant backcross lines in JL 24 variety of groundnut (<em>Arachis hypogaea</em> L.). Electronic Journal of Plant Breeding, 7, 37–41. 10.5958/0975-928X.2016.00005.3
</cite> [<a href="https://doi.org/10.5958/0975-928X.2016.00005.3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Electronic%20Journal%20of%20Plant%20Breeding&amp;title=Development%20of%20late%20leaf%20spot%20and%20rust%20resistant%20backcross%20lines%20in%20JL%2024%20variety%20of%20groundnut%20(Arachis%20hypogaea%20L.)&amp;author=S.%20B.%20Yeri&amp;author=R.%20S.%20Bhat&amp;volume=7&amp;publication_year=2016&amp;pages=37-41&amp;doi=10.5958/0975-928X.2016.00005.3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270019-bib-0036">
<cite>

Zhuang, W.
, 
Chen, H.
, 
Yang, M.
, 
Wang, J.
, 
Pandey, M. K.
, 
Zhang, C.
, 
Chang, W.‐C.
, 
Zhang, L.
, 
Zhang, X.
, 
Tang, R.
, 
Garg, V.
, 
Wang, X.
, 
Tang, H.
, 
Chow, C.‐N.
, 
Wang, J.
, 
Deng, Y. E.
, 
Wang, D.
, 
Khan, A. W.
, 
Yang, Q.
, … 
Varshney, R. K.
 (2019). The <em>Arachis hypogaea</em> genome elucidates legume karyotypes, polyploid evolution and crop domestication. Nature Genetics, 51, 865–876. 10.1038/s41588-019-0402-2

</cite> [<a href="https://doi.org/10.1038/s41588-019-0402-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7188672/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31043757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=The%20Arachis%20hypogaea%20genome%20elucidates%20legume%20karyotypes,%20polyploid%20evolution%20and%20crop%20domestication&amp;author=W.%20Zhuang&amp;author=H.%20Chen&amp;author=M.%20Yang&amp;author=J.%20Wang&amp;author=M.%20K.%20Pandey&amp;volume=51&amp;publication_year=2019&amp;pages=865-876&amp;pmid=31043757&amp;doi=10.1038/s41588-019-0402-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>
<strong>Table S1</strong>: List of 722 genotypes used for genotyping using high density SNP array</p>
<p>
<strong>Table S2</strong>: List of 8,027 SNPs selected based on PIC values</p>
<p>
<strong>Table S3</strong>: List of 82 SNPs associated with various traits with allele information and flanking sequences</p>
<p>
<strong>Table S4</strong>: List of 5,081 SNPs with details of chromosome number, position in diploid and tetraploid genome</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11958872/bin/TPG2-18-e70019-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70019-s001.xlsx</a><sup> (2.1MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All the data generated in this study are provided in the supplementary material.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70019"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70019.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11958872/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11958872/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11958872%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958872/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11958872/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11958872/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40164965/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11958872/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40164965/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11958872/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11958872/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="LPKSXlVt3zmBGbdcZnasRCyODcIlPhizovGvJWeN6KUyIrI1xX6tc3pp4zojIIEL">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
